{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "import context\n",
    "from palimpzest.constants import PZ_DIR\n",
    "import palimpzest as pz\n",
    "\n",
    "import gradio as gr\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "import argparse\n",
    "import requests\n",
    "import json\n",
    "import time\n",
    "import os\n",
    "\n",
    "pz.DataDirectory().clearCache(keep_registry=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "class ScientificPaper(pz.PDFFile):\n",
    "   \"\"\"Represents a scientific research paper, which in practice is usually from a PDF file\"\"\"\n",
    "   title = pz.Field(desc=\"The title of the paper. This is a natural language title, not a number or letter.\", required=True)\n",
    "   publicationYear = pz.Field(desc=\"The year the paper was published. This is a number.\", required=False)\n",
    "   author = pz.Field(desc=\"The name of the first author of the paper\", required=True)\n",
    "   journal = pz.Field(desc=\"The name of the journal the paper was published in\", required=True)\n",
    "   subject = pz.Field(desc=\"A summary of the paper contribution in one sentence\", required=False)\n",
    "   doiURL = pz.Field(desc=\"The DOI URL for the paper\", required=True)\n",
    "\n",
    "class TabularRow(pz.Schema):\n",
    "    \"\"\"A Row is a list of cells. For generality, we assume that all cell values are strings.\"\"\"\n",
    "    cells = pz.ListField(element_type=pz.StringField, desc=\"The cells in the row\", required=True)\n",
    "\n",
    "class Table(pz.Schema):\n",
    "    \"\"\"A Table is an object composed of a header and rows.\"\"\"\n",
    "    name = pz.StringField(desc=\"The name of the table\", required=False)\n",
    "    header = pz.ListField(element_type=pz.StringField, desc=\"The header of the table\", required=True)\n",
    "    rows = pz.ListField(element_type=pz.TabularRow, desc=\"The rows of the table\", required=True)\n",
    "\n",
    "def execute(output, policy):\n",
    "    execution = pz.SimpleExecution(output, policy)\n",
    "    physicalTree = execution.executeAndOptimize(verbose=True)\n",
    "\n",
    "    for table in physicalTree:\n",
    "        header = table.header\n",
    "        subset_rows = table.rows[:3]\n",
    "\n",
    "        print(\"Table name:\", table.name)\n",
    "        print(\" | \".join(header)[:100], \"...\")\n",
    "        for row in subset_rows:\n",
    "            print(\" | \".join(row.cells)[:100], \"...\")\n",
    "        print()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 0.178 -- Cost est: 0.000 -- Quality est: 0.240\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> TextFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. TextFile -> InduceFromCandidateOp -> URL \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (url...)\n",
      "\n",
      " 3. URL -> ApplyUserFunctionOp -> Download \n",
      "    (url...) -> (content,timesta...)\n",
      "\n",
      " 4. Download -> InduceFromCandidateOp -> File \n",
      "    Using Model.GPT_3_5\n",
      "    (content,timesta...) -> (contents,filena...)\n",
      "\n",
      " 5. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 6. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (header,name,row...)\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc2.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Patient_ID | Unique ID of each patient.  ...\n",
      "tumor_code | Tumor acronym (BRCA--Breast Cancer; ccRCC--clear cell renal cell carcinoma;  COAD--colo ...\n",
      "tumor_sample_id_protein | Sample ID for the tumor tissue ( used as column names in the combined prot ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc2.xlsx - Supplementary Table 1\n",
      "Case_ID | tumor_code | tumor_sample_id_protein | normal_sample_id_protein | is_excluded_from_pancanc ...\n",
      "11BR047 | BRCA | 11BR047-T | nan | nan | nan | 0.0 | nan | 81116212-b7e6-454b-9579-105cf3 | ef52c640 ...\n",
      "11BR043 | BRCA | 11BR043-T | nan | nan | nan | 0.0 | nan | 6d34d499-167e-42aa-9790-316fca_D2 | 6d34d ...\n",
      "11BR049 | BRCA | 11BR049-T | nan | nan | nan | 0.0 | nan | 2e700669-85b0-43fa-a9c7-3eaf5a_D2 | 2e700 ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Supplementary Table 2\n",
      "Case_ID | Specimen_ID | Slide_ID | Tumor | Topographic_Site | Specimen_Type | Weight | Tumor_Histolo ...\n",
      "C3L-00004 | C3L-00004-01 | C3L-00004-21 | ccRCC | Kidney | tumor_tissue | 484.0 | Clear cell renal c ...\n",
      "C3L-00004 | C3L-00004-06 | C3L-00004-26 | ccRCC | Kidney | normal_tissue | 252.0 | nan | -1.0 | -1.0 ...\n",
      "C3L-00010 | C3L-00010-01 | C3L-00010-21 | ccRCC | Kidney | tumor_tissue | 575.0 | Clear cell renal c ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Supplementary Table 2\n",
      "Case_ID | Specimen_ID | Slide_ID | Tumor | Topographic_Site | Specimen_Type | Weight | Tumor_Histolo ...\n",
      "C3L-00004 | C3L-00004-01 | C3L-00004-21 | ccRCC | Kidney | tumor_tissue | 484.0 | Clear cell renal c ...\n",
      "C3L-00004 | C3L-00004-06 | C3L-00004-26 | ccRCC | Kidney | normal_tissue | 252.0 | nan | -1.0 | -1.0 ...\n",
      "C3L-00010 | C3L-00010-01 | C3L-00010-21 | ccRCC | Kidney | tumor_tissue | 575.0 | Clear cell renal c ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "policy = pz.MinCost()\n",
    "\n",
    "papers = pz.Dataset(\"biofabric-pdf\", schema=ScientificPaper)\n",
    "paperURLs = papers.convert(pz.URL, desc=\"The DOI url of the paper\") \n",
    "htmlDOI = paperURLs.map(pz.DownloadHTMLFunction())\n",
    "tableURLS = htmlDOI.convert(pz.URL, desc=\"The URLs of the XLS tables from the page\", cardinality=\"oneToMany\")\n",
    "\n",
    "urlFile = pz.Dataset(\"biofabric-urls\", schema=pz.TextFile)\n",
    "tableURLS = urlFile.convert(pz.URL, desc=\"The URLs of the tables\")\n",
    "binary_tables = tableURLS.map(pz.DownloadBinaryFunction())\n",
    "tables = binary_tables.convert(pz.File)\n",
    "xls = tables.convert(pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "\n",
    "output = patient_tables\n",
    "execute(output, policy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 0.114 -- Cost est: 0.000 -- Quality est: 0.841\n",
      " 0. CacheScanDataOp -> Table \n",
      "\n",
      " 1. Table -> FilterCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    Filter: \"The table explains the meaning of attributes\"\n",
      "    (header,name,row...) -> (header,name,row...)\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc2.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Patient_ID | Unique ID of each patient.  ...\n",
      "tumor_code | Tumor acronym (BRCA--Breast Cancer; ccRCC--clear cell renal cell carcinoma;  COAD--colo ...\n",
      "tumor_sample_id_protein | Sample ID for the tumor tissue ( used as column names in the combined prot ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc2.xlsx - Supplementary Table 1\n",
      "Case_ID | tumor_code | tumor_sample_id_protein | normal_sample_id_protein | is_excluded_from_pancanc ...\n",
      "11BR047 | BRCA | 11BR047-T | nan | nan | nan | 0.0 | nan | 81116212-b7e6-454b-9579-105cf3 | ef52c640 ...\n",
      "11BR043 | BRCA | 11BR043-T | nan | nan | nan | 0.0 | nan | 6d34d499-167e-42aa-9790-316fca_D2 | 6d34d ...\n",
      "11BR049 | BRCA | 11BR049-T | nan | nan | nan | 0.0 | nan | 2e700669-85b0-43fa-a9c7-3eaf5a_D2 | 2e700 ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Supplementary Table 2\n",
      "Case_ID | Specimen_ID | Slide_ID | Tumor | Topographic_Site | Specimen_Type | Weight | Tumor_Histolo ...\n",
      "C3L-00004 | C3L-00004-01 | C3L-00004-21 | ccRCC | Kidney | tumor_tissue | 484.0 | Clear cell renal c ...\n",
      "C3L-00004 | C3L-00004-06 | C3L-00004-26 | ccRCC | Kidney | normal_tissue | 252.0 | nan | -1.0 | -1.0 ...\n",
      "C3L-00010 | C3L-00010-01 | C3L-00010-21 | ccRCC | Kidney | tumor_tissue | 575.0 | Clear cell renal c ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Supplementary Table 2\n",
      "Case_ID | Specimen_ID | Slide_ID | Tumor | Topographic_Site | Specimen_Type | Weight | Tumor_Histolo ...\n",
      "C3L-00004 | C3L-00004-01 | C3L-00004-21 | ccRCC | Kidney | tumor_tissue | 484.0 | Clear cell renal c ...\n",
      "C3L-00004 | C3L-00004-06 | C3L-00004-26 | ccRCC | Kidney | normal_tissue | 252.0 | nan | -1.0 | -1.0 ...\n",
      "C3L-00010 | C3L-00010-01 | C3L-00010-21 | ccRCC | Kidney | tumor_tissue | 575.0 | Clear cell renal c ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "policy = pz.MinCost()\n",
    "\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table explains the meaning of attributes\")\n",
    "\n",
    "output = patient_tables\n",
    "\n",
    "execute(output, policy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Policy is: Maximum Quality\n",
      "Chosen plan: Time est: 0.853 -- Cost est: 0.003 -- Quality est: 0.518\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> TextFile \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. TextFile -> InduceFromCandidateOp -> URL \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (url...)\n",
      "\n",
      " 3. URL -> ApplyUserFunctionOp -> Download \n",
      "    (url...) -> (content,timesta...)\n",
      "\n",
      " 4. Download -> InduceFromCandidateOp -> File \n",
      "    Using Model.GPT_4\n",
      "    (content,timesta...) -> (contents,filena...)\n",
      "\n",
      " 5. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 6. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_4\n",
      "    (contents,filena...) -> (header,name,row...)\n",
      "\n",
      " 7. Table -> FilterCandidateOp -> Table \n",
      "    Using Model.GPT_4\n",
      "    Filter: \"The table explains the meaning of attributes\"\n",
      "    (header,name,row...) -> (header,name,row...)\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc2.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Patient_ID | Unique ID of each patient.  ...\n",
      "tumor_code | Tumor acronym (BRCA--Breast Cancer; ccRCC--clear cell renal cell carcinoma;  COAD--colo ...\n",
      "tumor_sample_id_protein | Sample ID for the tumor tissue ( used as column names in the combined prot ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n",
      "Table name: 1-s2.0-S1535610823002192-mmc3.xlsx - Data_dictionary\n",
      "Column | Description ...\n",
      "Case_ID | Case identifier that is assigned by TSS after completing the \"Screening and Eligibility\" a ...\n",
      "Specimen_ID | Unique identifier for each tumor segement collected at TSS ...\n",
      "Slide_ID | Unique identifier for the slide created from the tissue ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "pz.DataDirectory().clearCache(keep_registry=True)\n",
    "policy = pz.MaxQuality()\n",
    "\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table explains the meaning of attributes\")\n",
    "\n",
    "output = patient_tables\n",
    "execute(output, policy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------\n",
      "Policy is: Minimum Cost\n",
      "Chosen plan: Time est: 486657.457 -- Cost est: 277.019 -- Quality est: 0.412\n",
      " 0. MarshalAndScanDataOp -> File \n",
      "\n",
      " 1. File -> InduceFromCandidateOp -> XLSFile \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (contents,filena...)\n",
      "\n",
      " 2. XLSFile -> InduceFromCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    (contents,filena...) -> (header,name,row...)\n",
      "\n",
      " 3. Table -> FilterCandidateOp -> Table \n",
      "    Using Model.GPT_3_5\n",
      "    Filter: \"The table records if the patient is excluded from the study\"\n",
      "    (header,name,row...) -> (header,name,row...)\n",
      "\n",
      "Table name: cao_mmc1.xlsx - Clinical_data\n",
      "case_id | tumor_included_for_the_study | normal_included_for_the_study | histology_diagnosis | age | ...\n",
      "C3L-00102 | yes | yes | PDAC | 42 | Male | White | United States | head | Unifocal | 3.0 | Not ident ...\n",
      "C3L-00189 | yes | yes | PDAC | 68 | Female | nan | Canada | head | Unifocal | 2.7 | Not identified | ...\n",
      "C3L-00277 | yes | yes | PDAC | 69 | Male | White | Other | tail | Unifocal | 5.0 | Not identified |  ...\n",
      "\n",
      "Table name: cao_mmc1.xlsx - TMT_labeling_scheme\n",
      "TMT_set | channel_name | case_id | sample_type ...\n",
      "1 | 128C | C3L-03123 | NAT ...\n",
      "1 | 131N | C3N-03884 | NAT ...\n",
      "2 | 128C | C3L-03630 | NAT ...\n",
      "\n",
      "Table name: cao_mmc4.xlsx - Glycoprotein_vs_IGP\n",
      "Gene | Peptide | Glycan | IGP_FC | IGP_T | Protein_FC | Protein_T | is_HM | is_F | is_S ...\n",
      "ORM2 | QNQCFYNSSYLNVQR | N5H6F0S3G0 | 0.990997178339351 | 888 | 0.562795647604764 | 2833 | 0 | 0 | 1 ...\n",
      "LEMD2 | CIPVMEAQEYIANVTSSSSAK | N5H3F1S0G0 | -1.12467524206323 | 263 | 0.42514544907711 | 3184 | 0 | ...\n",
      "DSC2 | NGIYNITVLASDQGGR | N2H5F0S0G0 | -0.709938361491043 | 470 | -0.203303693250363 | 1622 | 1 | 0  ...\n",
      "\n",
      "Table name: cao_mmc7.xlsx - C) NMF features C1\n",
      "Accession | logFC.C1.over.C2 | AveExpr.C1.over.C2 | t.C1.over.C2 | P.Value.C1.over.C2 | adj.P.Val.C1 ...\n",
      "ZNF281 | -1.29292838503 | -1.9615353232542e-05 | -11.0448241119811 | 2.47908022818637e-19 | 8.076843 ...\n",
      "ARSI | -2.74579811894165 | -0.0604085333120224 | -9.63917991384953 | 3.73184239304364e-16 | 4.769885 ...\n",
      "DSE | -1.41241347937131 | -0.02254913305671 | -9.56468188612356 | 5.37252666011987e-16 | 4.769885473 ...\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table name: clark_S044_CPTAC_CCRCC_Discovery_Cohort_Specimens_r1_Sept2018.xlsx - CCRCC_TMT\n",
      "Batch | TMT plex | TMT channel | ParticipantID | Parent Sample ID(s) | Aliquot ID | Group | OCT | TC ...\n",
      "1 | 1 | 126 | C3L-01287 | C3L-01287-06 | CPT0079430001 | Normal | No | C3L-01287-26 | https://pathol ...\n",
      "1 | 1 | 127N | C3L-00561 | C3L-00561-06 | CPT0023360001 | Normal | No | C3L-00561-26 | https://patho ...\n",
      "1 | 1 | 127C | C3L-00561 | C3L-00561-01 | CPT0023350003 | Tumor | No | C3L-00561-21 | https://pathol ...\n",
      "\n",
      "Table name: dou_mmc3.xlsx - E-Mutation-adjacent\n",
      "Sample | CHROM | POS | ID | REF | ALT | QUAL | FILTER | INFO | FORMAT | NORMAL | TUMOR ...\n",
      "C3L-00006 | chr9 | 94325425 | . | T | C | . | PASS | AC=1;AF=0.25;AN=4;DP=364;SOMATIC;set=varscan-mu ...\n",
      "C3L-00006 | chr16 | 1256631 | . | C | T | . | PASS | AC=1;AF=0.25;AN=4;DP=227;SOMATIC;set=varscan-mu ...\n",
      "C3L-00932 | chr1 | 22597319 | . | A | C | . | PASS | AC=1;AF=0.25;AN=4;DP=96;SOMATIC;set=varscan-mut ...\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table name: dou_mmc4.xlsx - B-Novel Splice Junctions\n",
      "Peptide | P-value | Q-value | MassDiff | Hyperscore | Expectation | Chromosome | 5' Splice Site | 3' ...\n",
      "MGEEEALQSQVTK | 0.0 | 0.0 | -0.0041 | 42.033 | 3.521e-26 | chr9 | 136474486 | 136475198 | 3 | S078,S ...\n",
      "WCFGPDGTGPNLLTDLTK | 0.0 | 0.0 | 1.0087 | 34.943 | 6.72799999999999e-24 | chr19 | 3964834 | 3977873  ...\n",
      "VEAGDEADYYCQVWDTR | 0.0 | 0.0 | 0.0063 | 59.0569999999999 | 1.699e-23 | chr22 | 22713183 | 22993459  ...\n",
      "\n",
      "Table name: gillette_mmc7.xlsx - Table S7J\n",
      "Signature.set.id | TN.STK11.outlier | TN.EGFR.outlier | TN.TP53.outlier | TN.KRAS.outlier | TN.STK11 ...\n",
      "PERT-PSP_ERLOTINIB | 1 | 1 | 0 | 1 | 3.7034 | -2.7473 | -0.0373 | 2.0159 ...\n",
      "PERT-PSP_WORTMANNIN | 1 | 0 | 0 | 1 | 2.7349 | -1.5854 | -0.4037 | 2.0359 ...\n",
      "PERT-PSP_GEFITINIB | 0 | 0 | 0 | 0 | 0.9997 | -1.4017 | -0.4305 | 0.7878 ...\n",
      "\n",
      "Table name: huang_mmc2.xlsx - Data availabilty\n",
      "case_id | WES | WES_BN | WGS | WGS_BN | RNA-Seq | RNA-Seq_AN | MiRNA | MiRNA_AN | Methy | Methy_AN | ...\n",
      "C3L-00977 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 0 | 1 | 0 | nan | _BN: blood normal; _AN: ad ...\n",
      "C3L-00987 | 1 | 1 | 1 | 1 | 1 | 0 | 1 | 0 | 0 | 0 | 1 | 0 | 1 | 0 | nan | nan ...\n",
      "C3L-00994 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | nan | nan ...\n",
      "\n",
      "Table name: huang_mmc2.xlsx - Removed samples\n",
      "Case_id | Sample_type | Reason to be removed ...\n",
      "C3N-01643 | Tumor | QC failed ...\n",
      "C3N-02693 | Tumor | HPV positive ...\n",
      "C3L-01138 | NAT | QC failed ...\n",
      "\n",
      "Table name: huang_mmc4.xlsx - Description\n",
      "Sheet | Description ...\n",
      "protein_differential_TumorNorm | paired tumor-NAT abundance changes in proteomics data ...\n",
      "protein_ORA | over-representation analysis for 2x up proteins and 2x down proteins ...\n",
      "proteinMarkers | 63 proteins increased >2x fold in tumor compared to NAT even after adjusting for no ...\n",
      "\n",
      "Table name: krug_mmc1.xlsx - B) QC RNA-seq\n",
      "Sample.ID | ACGT tag ID | Total reads | QC | mapping rates(%) | No. of transcripts | Notes ...\n",
      "X01BR001 | 25-R | 33444989 | pass | 0.9611 | 77112 | nan ...\n",
      "X01BR008 | 350-R | 38418218 | pass | 0.9761 | 78337 | nan ...\n",
      "X01BR009 | 345-R | 35745238 | pass | 0.968 | 84231 | nan ...\n",
      "\n",
      "Table name: krug_mmc4.xlsx - C) NMF LumB-I\n",
      "Accession | logFC.C3.over.rest | P.Value.C3.over.rest | adj.P.Val.C3.over.rest | Type | Direction |  ...\n",
      "EGFR | -3.32953534348444 | 2.48409635522064e-18 | 4.11047642175189e-17 | RNA | down | EGFR | 0.96876 ...\n",
      "MPZL2 | -2.00903015866333 | 4.58018662612264e-17 | 5.08458692545514e-16 | RNA | down | MPZL2 | 0.757 ...\n",
      "RBMS1 | -1.32015392725479 | 1.90105933776291e-16 | 1.70125412165109e-15 | RNA | down | RBMS1 | 0.794 ...\n",
      "\n",
      "Table name: li_mmc1.xlsx - Clinical_data\n",
      "case_id | tumor_included_for_the_study | normal_included_for_the_study | histology_diagnosis | age | ...\n",
      "C3L-00102 | yes | yes | PDAC | 42 | Male | White | United States | head | Unifocal | 3.0 | Not ident ...\n",
      "C3L-00189 | yes | yes | PDAC | 68 | Female | nan | Canada | head | Unifocal | 2.7 | Not identified | ...\n",
      "C3L-00277 | yes | yes | PDAC | 69 | Male | White | Other | tail | Unifocal | 5.0 | Not identified |  ...\n",
      "\n",
      "Table name: li_mmc2.xlsx - Supplementary Table 1\n",
      "Case_ID | tumor_code | tumor_sample_id_protein | normal_sample_id_protein | is_excluded_from_pancanc ...\n",
      "11BR047 | BRCA | 11BR047-T | nan | nan | nan | 0.0 | nan | 81116212-b7e6-454b-9579-105cf3 | ef52c640 ...\n",
      "11BR043 | BRCA | 11BR043-T | nan | nan | nan | 0.0 | nan | 6d34d499-167e-42aa-9790-316fca_D2 | 6d34d ...\n",
      "11BR049 | BRCA | 11BR049-T | nan | nan | nan | 0.0 | nan | 2e700669-85b0-43fa-a9c7-3eaf5a_D2 | 2e700 ...\n",
      "\n",
      "Table name: mcdermott_mmc10.xlsx - Clinical variables survival\n",
      "Variable | Category | Test | p value ...\n",
      "Site of disease | Fallopian tube | Fishers | 0.637 ...\n",
      "Site of disease | Ovary | Fishers | 0.673 ...\n",
      "Site of disease | Peritoneum | Fishers | 0.518 ...\n",
      "\n",
      "Table name: satpathy_mmc4.xlsx - Table S4G\n",
      "CMAP Enriched Gene | Sample Annotation | Sample Annotation Category | Fisher Test p-value ...\n",
      "IKBKAP.AMP | CIMP.status.meth | CIMP-high | 1.55566883736238e-06 ...\n",
      "PIN1.AMP | NMF.cluster | Classical | 1.39140310684105e-05 ...\n",
      "NR2F6.AMP | NMF.cluster | Classical | 2.99594780705427e-05 ...\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n",
      "/home/gerardo/.local/lib/python3.10/site-packages/openpyxl/worksheet/_reader.py:329: UserWarning: Unknown extension is not supported and will be removed\n",
      "  warn(msg)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Table name: satpathy_mmc7.xlsx - Table S7B\n",
      "Gene | medianDifference | pvalue | Normal_N | Tumor_N | pair | FDR | signedFDR | RNAevidence | Phosp ...\n",
      "S100A2 | 6.517 | 5.63e-14 | 99 | 99 | 99 | 1.8e-13 | 12.74391 | 1 | 0 | 1 ...\n",
      "KRT6A | 5.9276 | 4e-15 | 99 | 99 | 99 | 1.53e-14 | 13.81486 | 1 | 1 | 1 ...\n",
      "SERPINB5 | 5.6931 | 1.34e-16 | 99 | 99 | 99 | 7.07e-16 | 15.15054 | 1 | 0 | 0 ...\n",
      "\n",
      "Table name: satpathy_mmc7.xlsx - Table S7C\n",
      "ID | Gene Type | Measurement type | Change | Basal inclusive | Classical | EMT enriched | Inflamed s ...\n",
      "TGFBR2 | Tumor Suppressor | Global protein | Downregulated | X | X | X | X | X | 4.89274980284654e-2 ...\n",
      "MSI2 | Oncogene | Global protein | Upregulated | X | X | X | X | X | 1.77339394872147e-18 ...\n",
      "SPRED1 | Tumor Suppressor | Global protein | Downregulated | X | X | X | X | X | 6.43750606951931e-1 ...\n",
      "\n",
      "Table name: vasaikar_mmc3.xlsx - A-somatic_events\n",
      "Chr | Start | End | Ref | Alt | Variant_Type | Variant_Function | Gene | mRNA | mRNA_Change | Protei ...\n",
      "chr19 | 58864357 | 58864357 | A | G | nonsynonymous SNV | protein-altering | A1BG | NM_130786 | T277 ...\n",
      "chr19 | 58864307 | 58864307 | C | G | nonsynonymous SNV | protein-altering | A1BG | NM_130786 | G327 ...\n",
      "chr19 | 58858758 | 58858758 | C | T | nonsynonymous SNV | protein-altering | A1BG | NM_130786 | G144 ...\n",
      "\n",
      "Table name: vasaikar_mmc4.xlsx - A-Driver_SCNAs\n",
      "Gene_Symbol | cytoband | frequency | cnv2rna.cor | cnv2rna.corP | cnv2rna.fdr | cnv2tmt.cor | cnv2tm ...\n",
      "GOT1 | 10q24.2 | 0.0285714285714286 | 0.319677754014785 | 0.00158918600062735 | 0.00160664958305183  ...\n",
      "HIF1AN | 10q24.31 | 0.019047619047619 | 0.593763670496332 | 2.26887529411555e-10 | 5.64152775834138e ...\n",
      "SLK | 10q24.33 | 0.142857142857143 | 0.393361960250686 | 8.03102043906535e-05 | 9.72176158413174e-05 ...\n",
      "\n",
      "Table name: wang_mmc2.xlsx - cbttc_sample_info\n",
      "aliquot_id | sample_id | original_aliquot_id | batch | plex | tmt_channel | research_id | kids_first ...\n",
      "7316-288-TISS [894482] | 7316-288 | 7316-288-TISS [894482] | 1 | 1 | 127N | C54735 | PT_84Q11Z5X | A ...\n",
      "7316-89-TISS [894435] | 7316-89 | 7316-89-TISS [894435] | 1 | 1 | 128N | C25461 | PT_N32BYY8A | Aliv ...\n",
      "7316-2176-TISS [894450] | 7316-2176 | 7316-2176-TISS [894450] | 1 | 1 | 129N | C624225 | PT_VAJN5QP8 ...\n",
      "\n",
      "Table name: wang_mmc3.xlsx - structural_variant_manta\n",
      "case | CHROM_A | START_A | END_A | CHROM_B | START_B | END_B | SV_ID | QUAL | STRAND_A | STRAND_B |  ...\n",
      "C3L-00104 | chr1 | 203360603 | 203360604 | chr1 | 205331009 | 205331010 | MantaDEL:152977:0:1:1:0:0  ...\n",
      "C3L-00104 | chr1 | 203360608 | 203360609 | chr1 | 205331105 | 205331106 | MantaDUP:TANDEM:152977:0:1 ...\n",
      "C3L-00104 | chr1 | 203432963 | 203432964 | chr1 | 203484851 | 203484852 | MantaBND:153015:0:2:0:0:0: ...\n",
      "\n",
      "Table name: wang_mmc5.xlsx - im4_pathway\n",
      "p-value | q-value | pathway | source | external_id | members_input_overlap | members_input_overlap_g ...\n",
      "2.5987843984e-18 | 9.745441494e-16 | Neuronal System | Reactome | R-HSA-112316 | SLC6A1; SLC32A1; ST ...\n",
      "5.27156352118e-16 | 9.88418160221e-14 | Neurotransmitter release cycle | Reactome | R-HSA-112310 | S ...\n",
      "3.09057941746e-15 | 3.86322427182e-13 | Transmission across Chemical Synapses | Reactome | R-HSA-112 ...\n",
      "\n",
      "Table name: wang_mmc6.xlsx - mes_causative_interactions\n",
      "source | interaction_type | target | evidence | phosphosites ...\n",
      "CSNK2A1 | phosphorylates | STX1A | http://pathwaycommons.org/pc2/Catalysis_42fc58d79a5f94c3b3b4785a0 ...\n",
      "MAPK1 | phosphorylates | EIF4EBP1 | http://pathwaycommons.org/pc2/Catalysis_4d482934642080193320ff95 ...\n",
      "FYN | phosphorylates | PRKCB | http://pathwaycommons.org/pc2/BiochemicalReaction_2d62137ed93eba71da0 ...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "policy = pz.MinCost()\n",
    "\n",
    "xls = pz.Dataset('biofabric-xls', schema=pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table records if the patient is excluded from the study\")\n",
    "\n",
    "output = patient_tables\n",
    "execute(output, policy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "policy = pz.MinCost()\n",
    "\n",
    "xls = pz.Dataset('biofabric-xls', schema=pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table contains patient biometric data\")\n",
    "\n",
    "output = patient_tables\n",
    "execute(output, policy)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "policy = pz.MinCost()\n",
    "\n",
    "xls = pz.Dataset('biofabric-xls', schema=pz.XLSFile)\n",
    "patient_tables = xls.convert(pz.Table, desc=\"All tables in the file\", cardinality=\"oneToMany\")\n",
    "patient_tables = patient_tables.filterByStr(\"The table contains proteomic data\")\n",
    "\n",
    "output = patient_tables\n",
    "execute(output, policy)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "palimpzest",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
