<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="da7d341a-dd9d-4e61-aa50-58e1eba66a9c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867419311237;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text;issue:issue:pii\:S0092867418X00233"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(19)31123-7" />
    



        <meta property="og:site_name" content="Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg" />
    


        <meta property="og:description" content="Comprehensive proteogenomic characterization in 103 treatment-naive clear cell renal
cell carcinoma patient samples highlights tumor-specific alterations at the proteomic
level that are unrevealed by transcriptomic profiling and proposes a revised subtyping
scheme based on integrated omics analysis." />
    











        <meta name="twitter:title" content="Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg" />
    


        <meta name="twitter:description" content="Comprehensive proteogenomic characterization in 103 treatment-naive clear cell renal
cell carcinoma patient samples highlights tumor-specific alterations at the proteomic
level that are unrevealed by transcriptomic profiling and proposes a revised subtyping
scheme based on integrated omics analysis." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(19)31123-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419311237%3Fshowall%3Dtrue">
    


<title>Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma: Cell</title><meta name="citation_journal_title" content="Cell"/> <meta name="citation_journal_abbrev" content="Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma"/> <meta name="citation_date" content="2019/10/31"/> <meta name="citation_volume" content="179"/> <meta name="citation_issue" content="4"/> <meta name="citation_firstpage" content="964"/> <meta name="citation_lastpage" content="983.e31"/> <meta name="citation_pmid" content="31675502"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S0092867419311237/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S0092867419311237/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S0092867419311237/pdf"/> <meta name="citation_issn" content="0092-8674"/> <meta name="citation_issn" content="1097-4172"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.cell.2019.10.007"/> <meta name="citation_pmid" content="31675502"/> <meta name="citation_author" content="David J. Clark"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Saravana M. Dhanasekaran"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Francesca Petralia"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Jianbo Pan"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Xiaoyu Song"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Yingwei Hu"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Felipe da Veiga Leprevost"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Boris Reva"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Tung-Shing M. Lih"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Hui-Yin Chang"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Weiping Ma"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Christopher J. Ricketts"></meta><meta name="citation_author_institution" content="Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Lijun Chen"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Azra Krek"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Yize Li"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Dmitry Rykunov"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Qing Kay Li"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Lin S. Chen"></meta><meta name="citation_author_institution" content="Department of Public Health Sciences, University of Chicago, Chicago, IL 60637, USA"></meta><meta name="citation_author" content="Umut Ozbek"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Suhas Vasaikar"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yige Wu"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Seungyeul Yoo"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Shrabanti Chowdhury"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Matthew A. Wyczalkowski"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Jiayi Ji"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Michael Schnaubelt"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Andy Kong"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Sunantha Sethuraman"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Dmitry M. Avtonomov"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Minghui Ao"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Antonio Colaprico"></meta><meta name="citation_author_institution" content="Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Song Cao"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Kyung-Cho Cho"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Selim Kalayci"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Shiyong Ma"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Wenke Liu"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Kelly Ruggles"></meta><meta name="citation_author_institution" content="Department of Medicine, New York University School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Anna Calinawan"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Zeynep H. Gümüş"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Daniel Geiszler"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Emily Kawaler"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Guo Ci Teo"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yuping Zhang"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Sarah Keegan"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Kai Li"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Feng Chen"></meta><meta name="citation_author_institution" content="Departments of Medicine and Cell Biology and Physiology, Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Nathan Edwards"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Cellular Biology, Georgetown University, Washington, DC 20007, USA"></meta><meta name="citation_author" content="Phillip M. Pierorazio"></meta><meta name="citation_author_institution" content="Brady Urological Institute and Department of Urology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Xi Steven Chen"></meta><meta name="citation_author_institution" content="Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Christian P. Pavlovich"></meta><meta name="citation_author_institution" content="Brady Urological Institute and Department of Urology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="A. Ari Hakimi"></meta><meta name="citation_author_institution" content="Department of Surgery, Urology Service, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"></meta><meta name="citation_author" content="Gabriel Brominski"></meta><meta name="citation_author_institution" content="Department of Urology, Poznań University of Medical Sciences, Szwajcarska 3, Poznań 61-285, Poland"></meta><meta name="citation_author" content="James J. Hsieh"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Andrzej Antczak"></meta><meta name="citation_author_institution" content="Department of Urology, Poznań University of Medical Sciences, Szwajcarska 3, Poznań 61-285, Poland"></meta><meta name="citation_author" content="Tatiana Omelchenko"></meta><meta name="citation_author_institution" content="Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"></meta><meta name="citation_author" content="Jan Lubinski"></meta><meta name="citation_author_institution" content="Department of Genetics and Pathology, Pomeranian Medical University, Szczecin 71-252, Poland"></meta><meta name="citation_author" content="Maciej Wiznerowicz"></meta><meta name="citation_author_institution" content="International Institute for Molecular Oncology, Poznań 60-203, Poland"></meta><meta name="citation_author_institution" content="Poznań University of Medical Sciences, Poznan 60-701, Poland"></meta><meta name="citation_author" content="W. Marston Linehan"></meta><meta name="citation_author_institution" content="Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Jiang Qian"></meta><meta name="citation_author_institution" content="Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="David Fenyö"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="Washington University School of Medicine, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Eric Schadt"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Sema4, Stamford, CT 06902, USA"></meta><meta name="citation_author" content="Arul M. Chinnaiyan"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Gilbert S. Omenn"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Internal Medicine, Human Genetics, and School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Marcin Cieslik"></meta><meta name="citation_author_email" content="mcieslik@med.mich.edu"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author_email" content="dchan@jhmi.edu"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Alexey I. Nesvizhskii"></meta><meta name="citation_author_email" content="nesvi@med.umich.edu"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Pei Wang"></meta><meta name="citation_author_email" content="pei.wang@mssm.edu"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author_email" content="huizhang@jhu.edu"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Abdul Samad Hashimi"></meta><meta name="citation_author" content="Alexander R. Pico"></meta><meta name="citation_author" content="Alla Karpova"></meta><meta name="citation_author" content="Alyssa Charamut"></meta><meta name="citation_author" content="Amanda G. Paulovich"></meta><meta name="citation_author" content="Amy M. Perou"></meta><meta name="citation_author" content="Anna Malovannaya"></meta><meta name="citation_author" content="Annette Marrero-Oliveras"></meta><meta name="citation_author" content="Anupriya Agarwal"></meta><meta name="citation_author" content="Barbara Hindenach"></meta><meta name="citation_author" content="Barbara Pruetz"></meta><meta name="citation_author" content="Beom-Jun Kim"></meta><meta name="citation_author" content="Brian J. Druker"></meta><meta name="citation_author" content="Chelsea J. Newton"></meta><meta name="citation_author" content="Chet Birger"></meta><meta name="citation_author" content="Corbin D. Jones"></meta><meta name="citation_author" content="Cristina Tognon"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author" content="Dana R. Valley"></meta><meta name="citation_author" content="Daniel C. Rohrer"></meta><meta name="citation_author" content="Daniel C. Zhou"></meta><meta name="citation_author" content="Darlene Tansil"></meta><meta name="citation_author" content="David Chesla"></meta><meta name="citation_author" content="David Heiman"></meta><meta name="citation_author" content="David Wheeler"></meta><meta name="citation_author" content="Donghui Tan"></meta><meta name="citation_author" content="Doug Chan"></meta><meta name="citation_author" content="Emek Demir"></meta><meta name="citation_author" content="Ewa Malc"></meta><meta name="citation_author" content="Francesmary Modugno"></meta><meta name="citation_author" content="Gaddy Getz"></meta><meta name="citation_author" content="Galen Hostetter"></meta><meta name="citation_author" content="George D. Wilson"></meta><meta name="citation_author" content="Gerald W. Hart"></meta><meta name="citation_author" content="Heng Zhu"></meta><meta name="citation_author" content="Hongwei Liu"></meta><meta name="citation_author" content="Houston Culpepper"></meta><meta name="citation_author" content="Hua Sun"></meta><meta name="citation_author" content="Hua Zhou"></meta><meta name="citation_author" content="Jacob Day"></meta><meta name="citation_author" content="James Suh"></meta><meta name="citation_author" content="Jasmine Huang"></meta><meta name="citation_author" content="Jason McDermott"></meta><meta name="citation_author" content="Jeffrey R. Whiteaker"></meta><meta name="citation_author" content="Jeffrey W. Tyner"></meta><meta name="citation_author" content="Jennifer Eschbacher"></meta><meta name="citation_author" content="Jin Chen"></meta><meta name="citation_author" content="John McGee"></meta><meta name="citation_author" content="Jun Zhu"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author" content="Karl Clauser"></meta><meta name="citation_author" content="Karna Robinson"></meta><meta name="citation_author" content="Karsten Krug"></meta><meta name="citation_author" content="Katherine A. Hoadley"></meta><meta name="citation_author" content="Ki Sung Um"></meta><meta name="citation_author" content="Kim Elburn"></meta><meta name="citation_author" content="Kimberly Holloway"></meta><meta name="citation_author" content="Liang-Bo Wang"></meta><meta name="citation_author" content="Lili Blumenberg"></meta><meta name="citation_author" content="Linda Hannick"></meta><meta name="citation_author" content="Liqun Qi"></meta><meta name="citation_author" content="Lori J. Sokoll"></meta><meta name="citation_author" content="MacIntosh Cornwell"></meta><meta name="citation_author" content="Marc Loriaux"></meta><meta name="citation_author" content="Marcin J. Domagalski"></meta><meta name="citation_author" content="Marina A. Gritsenko"></meta><meta name="citation_author" content="Matthew Anderson"></meta><meta name="citation_author" content="Matthew E. Monroe"></meta><meta name="citation_author" content="Matthew J. Ellis"></meta><meta name="citation_author" content="Maureen Dyer"></meta><meta name="citation_author" content="Meenakshi Anurag"></meta><meta name="citation_author" content="Meghan C. Burke"></meta><meta name="citation_author" content="Melissa Borucki"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author" content="Michael J. Birrer"></meta><meta name="citation_author" content="Michael Lewis"></meta><meta name="citation_author" content="Michael M. Ittmann"></meta><meta name="citation_author" content="Michael Smith"></meta><meta name="citation_author" content="Michael Vernon"></meta><meta name="citation_author" content="Michelle Chaikin"></meta><meta name="citation_author" content="Milan G. Chheda"></meta><meta name="citation_author" content="Munziba Khan"></meta><meta name="citation_author" content="Nancy Roche"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Negin Vatanian"></meta><meta name="citation_author" content="Nicole Tignor"></meta><meta name="citation_author" content="Noam Beckmann"></meta><meta name="citation_author" content="Pamela Grady"></meta><meta name="citation_author" content="Patricia Castro"></meta><meta name="citation_author" content="Paul Piehowski"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="Piotr Mieczkowski"></meta><meta name="citation_author" content="Pushpa Hariharan"></meta><meta name="citation_author" content="Qingsong Gao"></meta><meta name="citation_author" content="Rajiv Dhir"></meta><meta name="citation_author" content="Ramani Bhupendra Kothadia"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Rebecca Montgomery"></meta><meta name="citation_author" content="Reyka G. Jayasinghe"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author" content="Robert Edwards"></meta><meta name="citation_author" content="Robert Zelt"></meta><meta name="citation_author" content="Ross Bremner"></meta><meta name="citation_author" content="Ruiyang Liu"></meta><meta name="citation_author" content="Runyu Hong"></meta><meta name="citation_author" content="Sailaja Mareedu"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author" content="Sandra Cottingham"></meta><meta name="citation_author" content="Sanford P. Markey"></meta><meta name="citation_author" content="Scott D. Jewell"></meta><meta name="citation_author" content="Shalin Patel"></meta><meta name="citation_author" content="Shankha Satpathy"></meta><meta name="citation_author" content="Shannon Richey"></meta><meta name="citation_author" content="Sherri R. Davies"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Simina M. Boca"></meta><meta name="citation_author" content="Snehal Patil"></meta><meta name="citation_author" content="Sohini Sengupta"></meta><meta name="citation_author" content="Sonya Carter"></meta><meta name="citation_author" content="Stacey Gabriel"></meta><meta name="citation_author" content="Stefani N. Thomas"></meta><meta name="citation_author" content="Stephanie De Young"></meta><meta name="citation_author" content="Stephen E. Stein"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author" content="Steven M. Foltz"></meta><meta name="citation_author" content="Sue Hilsenbeck"></meta><meta name="citation_author" content="Tanya Krubit"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author" content="Tara Skelly"></meta><meta name="citation_author" content="Thomas Westbrook"></meta><meta name="citation_author" content="Uma Borate"></meta><meta name="citation_author" content="Uma Velvulou"></meta><meta name="citation_author" content="Vladislav A. Petyuk"></meta><meta name="citation_author" content="William E. Bocik"></meta><meta name="citation_author" content="Xi Chen"></meta><meta name="citation_author" content="Yan Shi"></meta><meta name="citation_author" content="Yifat Geffen"></meta><meta name="citation_author" content="Yihao Lu"></meta><meta name="citation_author" content="Ying Wang"></meta><meta name="citation_author" content="Yosef Maruvka"></meta><meta name="citation_author" content="Zhi Li"></meta><meta name="citation_author" content="Zhiao Shi"></meta><meta name="citation_author" content="Zhidong Tu"></meta><meta name="citation_keywords" content="ccRCC; renal carcinoma; proteogenomics; proteomics; immune infiltration; tumor microenvironment; phosphoproteomics; drug targets; chromosomal translocation; CPTAC"></meta><meta name="citation_reference" content="citation_title=From big data analysis to personalized medicine for all: challenges and opportunities; citation_author=A. Alyass; citation_author=M. Turcotte; citation_author=D. Meyre; citation_journal_title=BMC Med. Genomics; citation_volume=8; citation_firstpage=33; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Obstacles Posed by the Tumor Microenvironment to T cell Activity: A Case for Synergistic Therapies; citation_author=K.G. Anderson; citation_author=I.M. Stromnes; citation_author=P.D. Greenberg; citation_journal_title=Cancer Cell; citation_volume=31; citation_firstpage=311; citation_lastpage=325; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=CD39 and CD73 in immunity and inflammation; citation_author=L. Antonioli; citation_author=P. Pacher; citation_author=E.S. Vizi; citation_author=G. Haskó; citation_journal_title=Trends Mol. Med.; citation_volume=19; citation_firstpage=355; citation_lastpage=367; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Single-CpG-resolution methylome analysis identifies clinicopathologically aggressive CpG island methylator phenotype clear cell renal cell carcinomas; citation_author=E. Arai; citation_author=S. Chiku; citation_author=T. Mori; citation_author=M. Gotoh; citation_author=T. Nakagawa; citation_author=H. Fujimoto; citation_author=Y. Kanai; citation_journal_title=Carcinogenesis; citation_volume=33; citation_firstpage=1487; citation_lastpage=1493; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=xCell: digitally portraying the tissue cellular heterogeneity landscape; citation_author=D. Aran; citation_author=Z. Hu; citation_author=A.J. Butte; citation_journal_title=Genome Biol.; citation_volume=18; citation_firstpage=220; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans; citation_author=K.G. Ardlie; citation_author=D.S. Deluca; citation_author=A.V. Segre; citation_author=T.J. Sullivan; citation_author=T.R. Young; citation_author=E.T. Gelfand; citation_author=C.A. Trowbridge; citation_author=J.B. Maller; citation_author=T. Tukiainen; citation_author=M. Lek; citation_author=et al.; citation_journal_title=Science; citation_volume=348; citation_firstpage=648; citation_lastpage=660; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=moFF: a robust and automated approach to extract peptide ion intensities; citation_author=A. Argentini; citation_author=L.J.E. Goeminne; citation_author=K. Verheggen; citation_author=N. Hulstaert; citation_author=A. Staes; citation_author=L. Clement; citation_author=L. Martens; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=964; citation_lastpage=966; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=MEK162 for patients with advanced melanoma harbouring NRAS or Val600 BRAF mutations: a non-randomised, open-label phase 2 study; citation_author=P.A. Ascierto; citation_author=D. Schadendorf; citation_author=C. Berking; citation_author=S.S. Agarwala; citation_author=C.M.L. van Herpen; citation_author=P. Queirolo; citation_author=C.U. Blank; citation_author=A. Hauschild; citation_author=J.T. Beck; citation_author=A. St-Pierre; citation_author=et al.; citation_journal_title=Lancet Oncol.; citation_volume=14; citation_firstpage=249; citation_lastpage=256; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Current and emerging therapies for first-line treatment of metastatic clear cell renal cell carcinoma; citation_author=M.B. Atkins; citation_author=N.M. Tannir; citation_journal_title=Cancer Treat. Rev.; citation_volume=70; citation_firstpage=127; citation_lastpage=137; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=YY1 can inhibit c-Myc function through a mechanism requiring DNA binding of YY1 but neither its transactivation domain nor direct interaction with c-Myc; citation_author=M. Austen; citation_author=C. Cerni; citation_author=J.M. Lüscher-Firzlaff; citation_author=B. Lüscher; citation_journal_title=Oncogene; citation_volume=17; citation_firstpage=511; citation_lastpage=520; citation_publication_date=1998; "></meta><meta name="citation_reference" content="citation_title=MYC activation cooperates with Vhl and Ink4a/Arf loss to induce clear cell renal cell carcinoma; citation_author=S.T. Bailey; citation_author=A.M. Smith; citation_author=J. Kardos; citation_author=S.E. Wobker; citation_author=H.L. Wilson; citation_author=B. Krishnan; citation_author=R. Saito; citation_author=H.J. Lee; citation_author=J. Zhang; citation_author=S.C. Eaton; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=15770; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A decade of exploring the cancer epigenome - biological and translational implications; citation_author=S.B. Baylin; citation_author=P.A. Jones; citation_journal_title=Nat. Rev. Cancer; citation_volume=11; citation_firstpage=726; citation_lastpage=734; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=On the approximation of curves by line segments using dynamic programming; citation_author=R. Bellman; citation_journal_title=Commun. ACM; citation_volume=4; citation_firstpage=284; citation_publication_date=1961; "></meta><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "></meta><meta name="citation_reference" content="citation_title=Molecular subtypes of clear cell renal cell carcinoma are associated with sunitinib response in the metastatic setting; citation_author=B. Beuselinck; citation_author=S. Job; citation_author=E. Becht; citation_author=A. Karadimou; citation_author=V. Verkarre; citation_author=G. Couchy; citation_author=N. Giraldo; citation_author=N. Rioux-Leclercq; citation_author=V. Molinié; citation_author=M. Sibony; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=21; citation_firstpage=1329; citation_lastpage=1339; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=The endothelial transcription factor ERG promotes vascular stability and growth through Wnt/β-catenin signaling; citation_author=G.M. Birdsey; citation_author=A.V. Shah; citation_author=N. Dufton; citation_author=L.E. Reynolds; citation_author=L. Osuna Almagro; citation_author=Y. Yang; citation_author=I.M. Aspalter; citation_author=S.T. Khan; citation_author=J.C. Mason; citation_author=E. Dejana; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=32; citation_firstpage=82; citation_lastpage=96; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Role of radiation therapy in the management of renal cell cancer; citation_author=A.I. Blanco; citation_author=B.S. Teh; citation_author=R.J. Amato; citation_journal_title=Cancers (Basel); citation_volume=3; citation_firstpage=4010; citation_lastpage=4023; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=G2 checkpoint abrogation and checkpoint kinase-1 targeting in the treatment of cancer; citation_author=N. Bucher; citation_author=C.D. Britten; citation_journal_title=Br. J. Cancer; citation_volume=98; citation_firstpage=523; citation_lastpage=528; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Multifarious functions of PDGFs and PDGFRs in tumor growth and metastasis; citation_author=Y. Cao; citation_journal_title=Trends Mol. Med.; citation_volume=19; citation_firstpage=460; citation_lastpage=473; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=PKIDB: A curated, annotated and updated database of protein kinase inhibitors in clinical trials; citation_author=F. Carles; citation_author=S. Bourg; citation_author=C. Meyer; citation_author=P. Bonnet; citation_journal_title=Molecules; citation_volume=23; citation_firstpage=1; citation_lastpage=18; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Absolute quantification of somatic DNA alterations in human cancer; citation_author=S.L. Carter; citation_author=K. Cibulskis; citation_author=E. Helman; citation_author=A. McKenna; citation_author=H. Shen; citation_author=T. Zack; citation_author=P.W. Laird; citation_author=R.C. Onofrio; citation_author=W. Winckler; citation_author=B.A. Weir; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=30; citation_firstpage=413; citation_lastpage=421; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Elements of cancer immunity and the cancer-immune set point; citation_author=D.S. Chen; citation_author=I. Mellman; citation_journal_title=Nature; citation_volume=541; citation_firstpage=321; citation_lastpage=330; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Interferon-γ-induced PD-L1 surface expression on human oral squamous carcinoma via PKD2 signal pathway; citation_author=J. Chen; citation_author=Y. Feng; citation_author=L. Lu; citation_author=H. Wang; citation_author=L. Dai; citation_author=Y. Li; citation_author=P. Zhang; citation_journal_title=Immunobiology; citation_volume=217; citation_firstpage=385; citation_lastpage=393; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Multilevel Genomics-Based Taxonomy of Renal Cell Carcinoma; citation_author=F. Chen; citation_author=Y. Zhang; citation_author=Y. Şenbabaoğlu; citation_author=G. Ciriello; citation_author=L. Yang; citation_author=E. Reznik; citation_author=B. Shuch; citation_author=G. Micevic; citation_author=G. De Velasco; citation_author=E. Shinbrot; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=14; citation_firstpage=2476; citation_lastpage=2489; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications; citation_author=X. Chen; citation_author=O. Schulz-Trieglaff; citation_author=R. Shaw; citation_author=B. Barnes; citation_author=F. Schlesinger; citation_author=M. Källberg; citation_author=A.J. Cox; citation_author=S. Kruglyak; citation_author=C.T. Saunders; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=1220; citation_lastpage=1222; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=CD38-mediated immunosuppression as a mechanism of tumor cell escape from PD-1/PD-l1 blockade; citation_author=L. Chen; citation_author=L. Diao; citation_author=Y. Yang; citation_author=X. Yi; citation_author=B.L. Rodriguez; citation_author=Y. Li; citation_author=P.A. Villalobos; citation_author=T. Cascone; citation_author=X. Liu; citation_author=L. Tan; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=8; citation_firstpage=1156; citation_lastpage=1175; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Distinct functions for Rap1 signaling in vascular morphogenesis and dysfunction; citation_author=M. Chrzanowska-Wodnicka; citation_journal_title=Exp. Cell Res.; citation_volume=319; citation_firstpage=2350; citation_lastpage=2359; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics; citation_author=D.J. Clark; citation_author=Y. Hu; citation_author=W. Bocik; citation_author=L. Chen; citation_author=M. Schnaubelt; citation_author=R. Roberts; citation_author=P. Shah; citation_author=G. Whiteley; citation_author=H. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=17; citation_firstpage=2205; citation_lastpage=2215; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Comprehensive molecular characterization of clear cell renal cell carcinoma; citation_author=C.J. Creighton; citation_author=M. Morgan; citation_author=P.H. Gunaratne; citation_author=D.A. Wheeler; citation_author=R.A. Gibbs; citation_author=G. Robertson; citation_author=A. Chu; citation_author=R. Beroukhim; citation_author=K. Cibulskis; citation_author=S. Signoretti; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Nature; citation_volume=499; citation_firstpage=43; citation_lastpage=49; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=The somatic genomic landscape of chromophobe renal cell carcinoma; citation_author=C.F. Davis; citation_author=C.J. Ricketts; citation_author=M. Wang; citation_author=L. Yang; citation_author=A.D. Cherniack; citation_author=H. Shen; citation_author=C. Buhay; citation_author=H. Kang; citation_author=S.C. Kim; citation_author=C.C. Fahey; citation_author=The Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=26; citation_firstpage=319; citation_lastpage=330; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics; citation_author=E.W. Deutsch; citation_author=L. Mendoza; citation_author=D. Shteynberg; citation_author=J. Slagel; citation_author=Z. Sun; citation_author=R.L. Moritz; citation_journal_title=Proteomics Clin. Appl.; citation_volume=9; citation_firstpage=745; citation_lastpage=754; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Combination therapy targeting ribosome biogenesis and mRNA translation synergistically extends survival in MYC-driven lymphoma; citation_author=J.R. Devlin; citation_author=K.M. Hannan; citation_author=N. Hein; citation_author=C. Cullinane; citation_author=E. Kusnadi; citation_author=P.Y. Ng; citation_author=A.J. George; citation_author=J. Shortt; citation_author=M.J. Bywater; citation_author=G. Poortinga; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=6; citation_firstpage=59; citation_lastpage=70; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Cytotoxic chemotherapy in the treatment of advanced renal cell carcinoma in the era of targeted therapy; citation_author=E. Diamond; citation_author=A.M. Molina; citation_author=M. Carbonaro; citation_author=N.H. Akhtar; citation_author=P. Giannakakou; citation_author=S.T. Tagawa; citation_author=D.M. Nanus; citation_journal_title=Crit. Rev. Oncol. Hematol.; citation_volume=96; citation_firstpage=518; citation_lastpage=526; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Epigenetic alterations of chromosome 3 revealed by NotI-microarrays in clear cell renal cell carcinoma; citation_author=A.A. Dmitriev; citation_author=E.E. Rudenko; citation_author=A.V. Kudryavtseva; citation_author=G.S. Krasnov; citation_author=V.V. Gordiyuk; citation_author=N.V. Melnikova; citation_author=E.O. Stakhovsky; citation_author=O.A. Kononenko; citation_author=L.S. Pavlova; citation_author=T.T. Kondratieva; citation_author=et al.; citation_journal_title=BioMed Res. Int.; citation_volume=2014; citation_firstpage=735292; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=AJCC cancer staging manual; citation_author=S. Edge; citation_author=D. Byrd; citation_author=C. Compton; citation_author=A. Fritz; citation_author=F. Greene; citation_author=A. Trotti; citation_publication_date=2010; citation_publisher=Springer US, New York, NY; "></meta><meta name="citation_reference" content="citation_title=Sorafenib in advanced clear-cell renal-cell carcinoma; citation_author=B. Escudier; citation_author=T. Eisen; citation_author=W.M. Stadler; citation_author=C. Szczylik; citation_author=S. Oudard; citation_author=M. Siebels; citation_author=S. Negrier; citation_author=C. Chevreau; citation_author=E. Solska; citation_author=A.A. Desai; citation_author=TARGET Study Group; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=356; citation_firstpage=125; citation_lastpage=134; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Bevacizumab plus interferon alfa-2a for treatment of metastatic renal cell carcinoma: a randomised, double-blind phase III trial; citation_author=B. Escudier; citation_author=A. Pluzanska; citation_author=P. Koralewski; citation_author=A. Ravaud; citation_author=S. Bracarda; citation_author=C. Szczylik; citation_author=C. Chevreau; citation_author=M. Filipek; citation_author=B. Melichar; citation_author=E. Bajetta; citation_author=AVOREN Trial investigators; citation_author=et al.; citation_journal_title=Lancet; citation_volume=370; citation_firstpage=2103; citation_lastpage=2111; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Kinase inhibitors: the road ahead; citation_author=F.M. Ferguson; citation_author=N.S. Gray; citation_journal_title=Nat. Rev. Drug Discov.; citation_volume=17; citation_firstpage=353; citation_lastpage=377; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries; citation_author=S. Fisher; citation_author=A. Barry; citation_author=J. Abreu; citation_author=B. Minie; citation_author=J. Nolan; citation_author=T.M. Delorey; citation_author=G. Young; citation_author=T.J. Fennell; citation_author=A. Allen; citation_author=L. Ambrogio; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R1; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Functional normalization of 450k methylation array data improves replication in large cancer studies; citation_author=J.P. Fortin; citation_author=A. Labbe; citation_author=M. Lemire; citation_author=B.W. Zanke; citation_author=T.J. Hudson; citation_author=E.J. Fertig; citation_author=C.M.T. Greenwood; citation_author=K.D. Hansen; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=503; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi; citation_author=J.P. Fortin; citation_author=T.J. Triche; citation_author=K.D. Hansen; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=558; citation_lastpage=560; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A clearer view of the molecular complexity of clear cell renal cell carcinoma; citation_author=I.J. Frew; citation_author=H. Moch; citation_journal_title=Annu. Rev. Pathol.; citation_volume=10; citation_firstpage=263; citation_lastpage=289; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Interferon Receptor Signaling Pathways Regulating PD-L1 and PD-L2 Expression; citation_author=A. Garcia-Diaz; citation_author=D.S. Shin; citation_author=B.H. Moreno; citation_author=J. Saco; citation_author=H. Escuin-Ordinas; citation_author=G.A. Rodriguez; citation_author=J.M. Zaretsky; citation_author=L. Sun; citation_author=W. Hugo; citation_author=X. Wang; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=19; citation_firstpage=1189; citation_lastpage=1201; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Carcinoma-associated fibroblasts: orchestrating the composition of malignancy; citation_author=P. Gascard; citation_author=T.D. Tlsty; citation_journal_title=Genes Dev.; citation_volume=30; citation_firstpage=1002; citation_lastpage=1019; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Genomic architecture and evolution of clear cell renal cell carcinomas defined by multiregion sequencing; citation_author=M. Gerlinger; citation_author=S. Horswell; citation_author=J. Larkin; citation_author=A.J. Rowan; citation_author=M.P. Salm; citation_author=I. Varela; citation_author=R. Fisher; citation_author=N. McGranahan; citation_author=N. Matthews; citation_author=C.R. Santos; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=225; citation_lastpage=233; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Using CART to Detect Multiple Change Points in the Mean for Large Sample; citation_author=S. Gey; citation_author=E. Lebarbier; citation_journal_title=Stat. Syst. Biol.; citation_volume=12; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=HIF-2α promotes hypoxic cell proliferation by enhancing c-myc transcriptional activity; citation_author=J.D. Gordan; citation_author=J.A. Bertout; citation_author=C.J. Hu; citation_author=J.A. Diehl; citation_author=M.C. Simon; citation_journal_title=Cancer Cell; citation_volume=11; citation_firstpage=335; citation_lastpage=347; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=HIF and c-Myc: sibling rivals for control of cancer cell metabolism and proliferation; citation_author=J.D. Gordan; citation_author=C.B. Thompson; citation_author=M.C. Simon; citation_journal_title=Cancer Cell; citation_volume=12; citation_firstpage=108; citation_lastpage=113; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=HIF-α effects on c-Myc distinguish two subtypes of sporadic VHL-deficient clear cell renal carcinoma; citation_author=J.D. Gordan; citation_author=P. Lal; citation_author=V.R. Dondeti; citation_author=R. Letrero; citation_author=K.N. Parekh; citation_author=C.E. Oquendo; citation_author=R.A. Greenberg; citation_author=K.T. Flaherty; citation_author=W.K. Rathmell; citation_author=B. Keith; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=14; citation_firstpage=435; citation_lastpage=446; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Frequent mutations of genes encoding ubiquitin-mediated proteolysis pathway components in clear cell renal cell carcinoma; citation_author=G. Guo; citation_author=Y. Gui; citation_author=S. Gao; citation_author=A. Tang; citation_author=X. Hu; citation_author=Y. Huang; citation_author=W. Jia; citation_author=Z. Li; citation_author=M. He; citation_author=L. Sun; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=44; citation_firstpage=17; citation_lastpage=19; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=A clear picture of renal cell carcinoma; citation_author=A.A. Hakimi; citation_author=C.G. Pham; citation_author=J.J. Hsieh; citation_journal_title=Nat. Genet.; citation_volume=45; citation_firstpage=849; citation_lastpage=850; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma; citation_author=A.A. Hakimi; citation_author=E. Reznik; citation_author=C.H. Lee; citation_author=C.J. Creighton; citation_author=A.R. Brannon; citation_author=A. Luna; citation_author=B.A. Aksoy; citation_author=E.M. Liu; citation_author=R. Shen; citation_author=W. Lee; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=29; citation_firstpage=104; citation_lastpage=116; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=GSVA: gene set variation analysis for microarray and RNA-seq data; citation_author=S. Hänzelmann; citation_author=R. Castelo; citation_author=J. Guinney; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=7; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Expression pattern of immune checkpoint-associated molecules in radical nephrectomy specimens as a prognosticator in patients with metastatic renal cell carcinoma treated with tyrosine kinase inhibitors; citation_author=T. Hara; citation_author=H. Miyake; citation_author=M. Fujisawa; citation_journal_title=Urol. Oncol.; citation_volume=35; citation_firstpage=363; citation_lastpage=369; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Interphase APC/C-Cdc20 inhibition by cyclin A2-Cdk2 ensures efficient mitotic entry; citation_author=J.B. Hein; citation_author=J. Nilsson; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=10975; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis; citation_author=M. Hellström; citation_author=L.K. Phng; citation_author=J.J. Hofmann; citation_author=E. Wallgard; citation_author=L. Coultas; citation_author=P. Lindblom; citation_author=J. Alva; citation_author=A.K. Nilsson; citation_author=L. Karlsson; citation_author=N. Gaiano; citation_author=et al.; citation_journal_title=Nature; citation_volume=445; citation_firstpage=776; citation_lastpage=780; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=HIF and reactive oxygen species regulate oxidative phosphorylation in cancer; citation_author=E. Hervouet; citation_author=A. Cízková; citation_author=J. Demont; citation_author=A. Vojtísková; citation_author=P. Pecina; citation_author=N.L.W. Franssen-van Hal; citation_author=J. Keijer; citation_author=H. Simonnet; citation_author=R. Ivánek; citation_author=S. Kmoch; citation_author=et al.; citation_journal_title=Carcinogenesis; citation_volume=29; citation_firstpage=1528; citation_lastpage=1537; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Benchmarking common quantification strategies for large-scale phosphoproteomics; citation_author=A. Hogrebe; citation_author=L. von Stechow; citation_author=D.B. Bekker-Jensen; citation_author=B.T. Weinert; citation_author=C.D. Kelstrup; citation_author=J.V. Olsen; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1045; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations; citation_author=P.V. Hornbeck; citation_author=B. Zhang; citation_author=B. Murray; citation_author=J.M. Kornhauser; citation_author=V. Latham; citation_author=E. Skrzypek; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D512; citation_lastpage=D520; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Renal cell carcinoma; citation_author=J.J. Hsieh; citation_author=M.P. Purdue; citation_author=S. Signoretti; citation_author=C. Swanton; citation_author=L. Albiges; citation_author=M. Schmidinger; citation_author=D.Y. Heng; citation_author=J. Larkin; citation_author=V. Ficarra; citation_journal_title=Nat. Rev. Dis. Primers; citation_volume=3; citation_firstpage=17009; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Chromosome 3p Loss-Orchestrated VHL, HIF, and Epigenetic Deregulation in Clear Cell Renal Cell Carcinoma; citation_author=J.J. Hsieh; citation_author=V.H. Le; citation_author=T. Oyama; citation_author=C.J. Ricketts; citation_author=T.H. Ho; citation_author=E.H. Cheng; citation_journal_title=J. Clin. Oncol.; citation_volume=36; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Genomic classifications of renal cell carcinoma: a critical step towards the future application of personalized kidney cancer care with pan-omics precision; citation_author=J.J. Hsieh; citation_author=V. Le; citation_author=D. Cao; citation_author=E.H. Cheng; citation_author=C.J. Creighton; citation_journal_title=J. Pathol.; citation_volume=244; citation_firstpage=525; citation_lastpage=537; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=OmicsOne: Associate Omics Data with Phenotypes in One-Click; citation_author=Y. Hu; citation_author=M. Ao; citation_author=H. Zhang; citation_journal_title=bioRxiv; citation_doi=10.1101/756544; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Inhibition of MAPK kinase signaling pathways suppressed renal cell carcinoma growth and angiogenesis in vivo; citation_author=D. Huang; citation_author=Y. Ding; citation_author=W.M. Luo; citation_author=S. Bender; citation_author=C.N. Qian; citation_author=E. Kort; citation_author=Z.F. Zhang; citation_author=K. VandenBeldt; citation_author=N.S. Duesbery; citation_author=J.H. Resau; citation_author=B.T. Teh; citation_journal_title=Cancer Res.; citation_volume=68; citation_firstpage=81; citation_lastpage=88; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic integration reveals therapeutic targets in breast cancer xenografts; citation_author=K.L. Huang; citation_author=S. Li; citation_author=P. Mertins; citation_author=S. Cao; citation_author=H.P. Gunawardena; citation_author=K.V. Ruggles; citation_author=D.R. Mani; citation_author=K.R. Clauser; citation_author=M. Tanioka; citation_author=J. Usary; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=14864; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Temsirolimus, interferon alfa, or both for advanced renal-cell carcinoma; citation_author=G. Hudes; citation_author=M. Carducci; citation_author=P. Tomczak; citation_author=J. Dutcher; citation_author=R. Figlin; citation_author=A. Kapoor; citation_author=E. Staroslawska; citation_author=J. Sosman; citation_author=D. McDermott; citation_author=I. Bodrogi; citation_author=Global ARCC Trial; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=356; citation_firstpage=2271; citation_lastpage=2281; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Tumor grade of clear cell renal cell carcinoma assessed by contrast-enhanced computed tomography; citation_author=K. Ishigami; citation_author=L.V. Leite; citation_author=M.G. Pakalniskis; citation_author=D.K. Lee; citation_author=D.G. Holanda; citation_author=D.M. Kuehn; citation_journal_title=Springerplus; citation_volume=3; citation_firstpage=694; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Adjusting batch effects in microarray expression data using empirical Bayes methods; citation_author=W.E. Johnson; citation_author=C. Li; citation_author=A. Rabinovic; citation_journal_title=Biostatistics; citation_volume=8; citation_firstpage=118; citation_lastpage=127; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Reactome: a knowledgebase of biological pathways; citation_author=G. Joshi-Tope; citation_author=M. Gillespie; citation_author=I. Vastrik; citation_author=P. D’Eustachio; citation_author=E. Schmidt; citation_author=B. de Bono; citation_author=B. Jassal; citation_author=G.R. Gopinath; citation_author=G.R. Wu; citation_author=L. Matthews; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=33; citation_firstpage=D428; citation_lastpage=D432; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data; citation_author=V. Jurtz; citation_author=S. Paul; citation_author=M. Andreatta; citation_author=P. Marcatili; citation_author=B. Peters; citation_author=M. Nielsen; citation_journal_title=J. Immunol.; citation_volume=199; citation_firstpage=3360; citation_lastpage=3368; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Exploring Biological Networks in 3D, Stereoscopic 3D, and Immersive 3D with iCAVE; citation_author=S. Kalayci; citation_author=Z.H. Gümüş; citation_journal_title=Curr. Protoc. Bioinformatics; citation_volume=61; citation_firstpage=8.27.1; citation_lastpage=8.27.26; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes; citation_author=M. Kanehisa; citation_author=S. Goto; citation_journal_title=Nucleic Acids Res.; citation_volume=28; citation_firstpage=27; citation_lastpage=30; citation_publication_date=2000; "></meta><meta name="citation_reference" content="citation_title=Effects on survival of BAP1 and PBRM1 mutations in sporadic clear-cell renal-cell carcinoma: A retrospective analysis with independent validation; citation_author=P. Kapur; citation_author=S. Peña-Llopis; citation_author=A. Christie; citation_author=L. Zhrebker; citation_author=A. Pavía-Jiménez; citation_author=W.K. Rathmell; citation_author=X.J. Xie; citation_author=J. Brugarolas; citation_journal_title=Lancet Oncol.; citation_volume=14; citation_firstpage=159; citation_lastpage=167; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search; citation_author=A. Keller; citation_author=A.I. Nesvizhskii; citation_author=E. Kolker; citation_author=R. Aebersold; citation_journal_title=Anal. Chem.; citation_volume=74; citation_firstpage=5383; citation_lastpage=5392; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=ProteoWizard: Open source software for rapid proteomics tools development; citation_author=D. Kessner; citation_author=M. Chambers; citation_author=R. Burke; citation_author=D. Agus; citation_author=P. Mallick; citation_journal_title=Bioinformatics; citation_volume=24; citation_firstpage=2534; citation_lastpage=2536; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Strelka2: fast and accurate calling of germline and somatic variants; citation_author=S. Kim; citation_author=K. Scheffler; citation_author=A.L. Halpern; citation_author=M.A. Bekritsky; citation_author=E. Noh; citation_author=M. Källberg; citation_author=X. Chen; citation_author=Y. Kim; citation_author=D. Beyter; citation_author=P. Krusche; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=591; citation_lastpage=594; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=D.C. Koboldt; citation_author=Q. Zhang; citation_author=D.E. Larson; citation_author=D. Shen; citation_author=M.D. McLellan; citation_author=L. Lin; citation_author=C.A. Miller; citation_author=E.R. Mardis; citation_author=L. Ding; citation_author=R.K. Wilson; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=568; citation_lastpage=576; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Coupling and coordination in gene expression processes: a systems biology view; citation_author=S. Komili; citation_author=P.A. Silver; citation_journal_title=Nat. Rev. Genet.; citation_volume=9; citation_firstpage=38; citation_lastpage=48; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics; citation_author=A.T. Kong; citation_author=F.V. Leprevost; citation_author=D.M. Avtonomov; citation_author=D. Mellacheruvu; citation_author=A.I. Nesvizhskii; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=513; citation_lastpage=520; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Up-regulation of hypoxia-inducible factors HIF-1α and HIF-2α under normoxic conditions in renal carcinoma cells by von Hippel-Lindau tumor suppressor gene loss of function; citation_author=M. Krieg; citation_author=R. Haas; citation_author=H. Brauch; citation_author=T. Acker; citation_author=I. Flamme; citation_author=K.H. Plate; citation_journal_title=Oncogene; citation_volume=19; citation_firstpage=5435; citation_lastpage=5443; citation_publication_date=2000; "></meta><meta name="citation_reference" content="citation_title=Mutations in TSC1, TSC2, and MTOR are associated with response to rapalogs in patients with metastatic renal cell carcinoma; citation_author=D.J. Kwiatkowski; citation_author=T.K. Choueiri; citation_author=A.P. Fay; citation_author=B.I. Rini; citation_author=A.R. Thorner; citation_author=G. de Velasco; citation_author=M.E. Tyburczy; citation_author=L. Hamieh; citation_author=L. Albiges; citation_author=N. Agarwal; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=22; citation_firstpage=2445; citation_lastpage=2452; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=VarDict: A novel and versatile variant caller for next-generation sequencing in cancer research; citation_author=Z. Lai; citation_author=A. Markovets; citation_author=M. Ahdesmaki; citation_author=B. Chapman; citation_author=O. Hofmann; citation_author=R. Mcewen; citation_author=J. Johnson; citation_author=B. Dougherty; citation_author=J.C. Barrett; citation_author=J.R. Dry; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=1; citation_lastpage=11; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Cancer-associated fibroblasts induce antigen-specific deletion of CD8 + T Cells to protect tumour cells; citation_author=M.A. Lakins; citation_author=E. Ghorani; citation_author=H. Munir; citation_author=C.P. Martins; citation_author=J.D. Shields; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=948; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=LUMPY: a probabilistic framework for structural variant discovery; citation_author=R.M. Layer; citation_author=C. Chiang; citation_author=A.R. Quinlan; citation_author=I.M. Hall; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=R84; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Detecting multiple change-points in the mean of Gaussian process by model selection; citation_author=E. Lebarbier; citation_journal_title=Signal Processing; citation_volume=85; citation_firstpage=717; citation_lastpage=736; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=The OSBP-related proteins: a novel protein family involved in vesicle transport, cellular lipid metabolism, and cell signalling; citation_author=M. Lehto; citation_author=V.M. Olkkonen; citation_journal_title=Biochim. Biophys. Acta; citation_volume=1631; citation_firstpage=1; citation_lastpage=11; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=PDV: an integrative proteomics data viewer; citation_author=K. Li; citation_author=M. Vaudel; citation_author=B. Zhang; citation_author=Y. Ren; citation_author=B. Wen; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=1249; citation_lastpage=1251; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The roles of chromatin-remodelers and epigenetic modifiers in kidney cancer; citation_author=L. Liao; citation_author=J.R. Testa; citation_author=H. Yang; citation_journal_title=Cancer Genet.; citation_volume=208; citation_firstpage=206; citation_lastpage=214; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Molecular signatures database (MSigDB) 3.0; citation_author=A. Liberzon; citation_author=A. Subramanian; citation_author=R. Pinchback; citation_author=H. Thorvaldsdóttir; citation_author=P. Tamayo; citation_author=J.P. Mesirov; citation_journal_title=Bioinformatics; citation_volume=27; citation_firstpage=1739; citation_lastpage=1740; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Omic-Sig: Utilizing Omics Data to Explore and Visualize Kinase-Substrate Interactions; citation_author=T.M. Lih; citation_author=D.J. Clark; citation_author=H. Zhang; citation_journal_title=bioRxiv; citation_doi=10.1101/746123; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=iCAVE: an open source tool for visualizing biomolecular networks in 3D, stereoscopic 3D and immersive 3D; citation_author=V. Liluashvili; citation_author=S. Kalayci; citation_author=E. Fluder; citation_author=M. Wilson; citation_author=A. Gabow; citation_author=Z.H. Gümüs; citation_journal_title=Gigascience; citation_volume=6; citation_firstpage=1; citation_lastpage=13; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=EGFR signaling enhances aerobic glycolysis in triple-negative breast cancer cells to promote tumor growth and immune escape; citation_author=S.O. Lim; citation_author=C.W. Li; citation_author=W. Xia; citation_author=H.H. Lee; citation_author=S.S. Chang; citation_author=J. Shen; citation_author=J.L. Hsu; citation_author=D. Raftery; citation_author=D. Djukovic; citation_author=H. Gu; citation_author=et al.; citation_journal_title=Cancer Res.; citation_volume=76; citation_firstpage=1284; citation_lastpage=1296; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Cell-Type-Specific Gene Programs of the Normal Human Nephron Define Kidney Cancer Subtypes; citation_author=D. Lindgren; citation_author=P. Eriksson; citation_author=K. Krawczyk; citation_author=H. Nilsson; citation_author=J. Hansson; citation_author=S. Veerla; citation_author=J. Sjölund; citation_author=M. Höglund; citation_author=M.E. Johansson; citation_author=H. Axelson; citation_journal_title=Cell Rep.; citation_volume=20; citation_firstpage=1476; citation_lastpage=1489; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Comprehensive Molecular Characterization of Papillary Renal-Cell Carcinoma; citation_author=W.M. Linehan; citation_author=P.T. Spellman; citation_author=C.J. Ricketts; citation_author=C.J. Creighton; citation_author=S.S. Fei; citation_author=C. Davis; citation_author=D.A. Wheeler; citation_author=B.A. Murray; citation_author=L. Schmidt; citation_author=C.D. Vocke; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=374; citation_firstpage=135; citation_lastpage=145; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia; citation_author=W. Luo; citation_author=I. Chen; citation_author=Y. Chen; citation_author=D. Alkam; citation_author=Y. Wang; citation_author=G.L. Semenza; citation_journal_title=Oncotarget; citation_volume=7; citation_firstpage=6379; citation_lastpage=6397; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Integrative genomic analysis identifies NDRG2 as a candidate tumor suppressor gene frequently inactivated in clinically aggressive meningioma; citation_author=E.A. Lusis; citation_author=M.A. Watson; citation_author=M.R. Chicoine; citation_author=M. Lyman; citation_author=P. Roerig; citation_author=G. Reifenberger; citation_author=D.H. Gutmann; citation_author=A. Perry; citation_journal_title=Cancer Res.; citation_volume=65; citation_firstpage=7121; citation_lastpage=7126; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=A cross-package Bioconductor workflow for analysing methylation array data; citation_author=J. Maksimovic; citation_author=B. Phipson; citation_author=A. Oshlack; citation_journal_title=F1000Research; citation_volume=5; citation_firstpage=1; citation_lastpage=53; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation Article Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation; citation_author=T.M. Malta; citation_author=A. Sokolov; citation_author=A.J. Gentles; citation_author=T. Burzykowski; citation_author=L. Poisson; citation_author=J.N. Weinstein; citation_author=B. Kaminska; citation_author=J. Huelsken; citation_author=L. Omberg; citation_author=O. Gevaert; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=338; citation_lastpage=354; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=HIF pathway and c-Myc as biomarkers for response to sunitinib in metastatic clear-cell renal cell carcinoma; citation_author=P. Maroto; citation_author=E. Esteban; citation_author=E.F. Parra; citation_author=M.J. Mendez-Vidal; citation_author=M. Domenech; citation_author=B. Pérez-Valderrama; citation_author=V. Calderero; citation_author=J.L. Pérez-Gracia; citation_author=E. Grande; citation_author=F. Algaba; citation_journal_title=OncoTargets Ther.; citation_volume=10; citation_firstpage=4635; citation_lastpage=4643; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Targeting WEE1 Kinase in Cancer; citation_author=C.J. Matheson; citation_author=D.S. Backos; citation_author=P. Reigan; citation_journal_title=Trends Pharmacol. Sci.; citation_volume=37; citation_firstpage=872; citation_lastpage=881; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Neoantigen Load, Antigen Presentation Machinery, and Immune Signatures Determine Prognosis in Clear Cell Renal Cell Carcinoma; citation_author=H. Matsushita; citation_author=Y. Sato; citation_author=T. Karasaki; citation_author=T. Nakagawa; citation_author=H. Kume; citation_author=S. Ogawa; citation_author=Y. Homma; citation_author=K. Kakimi; citation_journal_title=Cancer Immunol. Res.; citation_volume=4; citation_firstpage=463; citation_lastpage=471; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Increasing the multiplexing capacity of TMTs using reporter ion isotopologues with isobaric masses; citation_author=G.C. McAlister; citation_author=E.L. Huttlin; citation_author=W. Haas; citation_author=L. Ting; citation_author=M.P. Jedrychowski; citation_author=J.C. Rogers; citation_author=K. Kuhn; citation_author=I. Pike; citation_author=R.A. Grothe; citation_author=J.D. Blethrow; citation_author=S.P. Gygi; citation_journal_title=Anal. Chem.; citation_volume=84; citation_firstpage=7469; citation_lastpage=7478; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma; citation_author=D.F. McDermott; citation_author=M.A. Huseni; citation_author=M.B. Atkins; citation_author=R.J. Motzer; citation_author=B.I. Rini; citation_author=B. Escudier; citation_author=L. Fong; citation_author=R.W. Joseph; citation_author=S.K. Pal; citation_author=J.A. Reeves; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=749; citation_lastpage=757; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Biallelic Alteration and Dysregulation of the Hippo Pathway in Mucinous Tubular and Spindle Cell Carcinoma of the Kidney; citation_author=R. Mehra; citation_author=P. Vats; citation_author=M. Cieslik; citation_author=X. Cao; citation_author=F. Su; citation_author=S. Shukla; citation_author=A.M. Udager; citation_author=R. Wang; citation_author=J. Pan; citation_author=K. Kasaian; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=6; citation_firstpage=1258; citation_lastpage=1266; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Somatic Bi-allelic Loss of TSC Genes in Eosinophilic Solid and Cystic Renal Cell Carcinoma; citation_author=R. Mehra; citation_author=P. Vats; citation_author=X. Cao; citation_author=F. Su; citation_author=N.D. Lee; citation_author=R. Lonigro; citation_author=K. Premkumar; citation_author=K. Trpkov; citation_author=J.K. McKenney; citation_author=S.M. Dhanasekaran; citation_author=A.M. Chinnaiyan; citation_journal_title=Eur. Urol.; citation_volume=74; citation_firstpage=483; citation_lastpage=486; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Integrated proteomic analysis of post-translational modifications by serial enrichment; citation_author=P. Mertins; citation_author=J.W. Qiao; citation_author=J. Patel; citation_author=N.D. Udeshi; citation_author=K.R. Clauser; citation_author=D.R. Mani; citation_author=M.W. Burgess; citation_author=M.A. Gillette; citation_author=J.D. Jaffe; citation_author=S.A. Carr; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=634; citation_lastpage=637; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry; citation_author=P. Mertins; citation_author=L.C. Tang; citation_author=K. Krug; citation_author=D.J. Clark; citation_author=M.A. Gritsenko; citation_author=L. Chen; citation_author=K.R. Clauser; citation_author=T.R. Clauss; citation_author=P. Shah; citation_author=M.A. Gillette; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=13; citation_firstpage=1632; citation_lastpage=1661; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Hypoxia: A Key Feature of the Tumor Microenvironment Triggers Several Mechanisms of Evasion from Natural Killer and Cytotoxic T Lymphocytes Surveillance; citation_author=T. Mgrditchian; citation_author=T. Arakelian; citation_author=J. Paggetti; citation_author=E. Viry; citation_author=A. Al-Absi; citation_author=S. Medves; citation_author=E. Moussay; citation_author=G. Berchem; citation_author=C. Thomas; citation_author=B. Janji; citation_journal_title=J. Immun. Res.; citation_volume=1; citation_firstpage=7; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal; citation_author=T.J. Mitchell; citation_author=S. Turajlic; citation_author=A. Rowan; citation_author=D. Nicol; citation_author=J.H.R. Farmery; citation_author=T. O’Brien; citation_author=I. Martincorena; citation_author=P. Tarpey; citation_author=N. Angelopoulos; citation_author=L.R. Yates; citation_author=TRACERx Renal Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=611; citation_lastpage=623; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Consensus Clustering A Resampling-Based Method for Class Discovery and Visualization of Gene Expression Microarray Data; citation_author=S. Monti; citation_author=P. Tamayo; citation_author=J. Mesirov; citation_author=T. Golub; citation_journal_title=Mach. Learn.; citation_volume=52; citation_firstpage=91; citation_lastpage=118; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=clusterMaker: a multi-algorithm clustering plugin for Cytoscape; citation_author=J.H. Morris; citation_author=L. Apeltsin; citation_author=A.M. Newman; citation_author=J. Baumbach; citation_author=T. Wittkop; citation_author=G. Su; citation_author=G.D. Bader; citation_author=T.E. Ferrin; citation_journal_title=BMC Bioinformatics; citation_volume=12; citation_firstpage=436; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Sunitinib versus interferon alfa in metastatic renal-cell carcinoma; citation_author=R.J. Motzer; citation_author=T.E. Hutson; citation_author=P. Tomczak; citation_author=M.D. Michaelson; citation_author=R.M. Bukowski; citation_author=O. Rixe; citation_author=S. Oudard; citation_author=S. Negrier; citation_author=C. Szczylik; citation_author=S.T. Kim; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=356; citation_firstpage=115; citation_lastpage=124; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Efficacy of everolimus in advanced renal cell carcinoma: a double-blind, randomised, placebo-controlled phase III trial; citation_author=R.J. Motzer; citation_author=B. Escudier; citation_author=S. Oudard; citation_author=T.E. Hutson; citation_author=C. Porta; citation_author=S. Bracarda; citation_author=V. Grünwald; citation_author=J.A. Thompson; citation_author=R.A. Figlin; citation_author=N. Hollaender; citation_author=RECORD-1 Study Group; citation_author=et al.; citation_journal_title=Lancet; citation_volume=372; citation_firstpage=449; citation_lastpage=456; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Nivolumab plus Ipilimumab versus Sunitinib in Advanced Renal-Cell Carcinoma; citation_author=R.J. Motzer; citation_author=N.M. Tannir; citation_author=D.F. McDermott; citation_author=O. Arén Frontera; citation_author=B. Melichar; citation_author=T.K. Choueiri; citation_author=E.R. Plimack; citation_author=P. Barthélémy; citation_author=C. Porta; citation_author=S. George; citation_author=CheckMate 214 Investigators; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=378; citation_firstpage=1277; citation_lastpage=1290; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PEGylated IL-10 (Pegilodecakin) Induces Systemic Immune Activation, CD8+ T Cell Invigoration and Polyclonal T Cell Expansion in Cancer Patients; citation_author=A. Naing; citation_author=J.R. Infante; citation_author=K.P. Papadopoulos; citation_author=I.H. Chan; citation_author=C. Shen; citation_author=N.P. Ratti; citation_author=B. Rojo; citation_author=K.A. Autio; citation_author=D.J. Wong; citation_author=M.R. Patel; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=34; citation_firstpage=775; citation_lastpage=791.e3; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A statistical model for identifying proteins by tandem mass spectrometry; citation_author=A.I. Nesvizhskii; citation_author=A. Keller; citation_author=E. Kolker; citation_author=R. Aebersold; citation_journal_title=Anal. Chem.; citation_volume=75; citation_firstpage=4646; citation_lastpage=4658; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=Comparative analysis of different label-free mass spectrometry based protein abundance estimates and their correlation with RNA-Seq gene expression data; citation_author=K. Ning; citation_author=D. Fermin; citation_author=A.I. Nesvizhskii; citation_journal_title=J. Proteome Res.; citation_volume=11; citation_firstpage=2261; citation_lastpage=2271; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=KEGG: Kyoto encyclopedia of genes and genomes; citation_author=H. Ogata; citation_author=S. Goto; citation_author=K. Sato; citation_author=W. Fujibuchi; citation_author=H. Bono; citation_author=M. Kanehisa; citation_journal_title=Nucleic Acids Res.; citation_volume=27; citation_firstpage=29; citation_lastpage=34; citation_publication_date=1999; "></meta><meta name="citation_reference" content="citation_title=Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation; citation_author=S.Y. Ow; citation_author=M. Salim; citation_author=J. Noirel; citation_author=C. Evans; citation_author=P.C. Wright; citation_journal_title=Proteomics; citation_volume=11; citation_firstpage=2341; citation_lastpage=2346; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=OmicsX: a web server for integrated OMICS analysis; citation_author=J. Pan; citation_author=J. Qian; citation_author=H. Zhang; citation_journal_title=bioRxiv; citation_doi=10.1101/755918; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Endogenous retrovirus expression is associated with response to immune checkpoint blockade in clear cell renal cell carcinoma; citation_author=A. Panda; citation_author=A.A. de Cubas; citation_author=M. Stein; citation_author=G. Riedlinger; citation_author=J. Kra; citation_author=T. Mayer; citation_author=C.C. Smith; citation_author=B.G. Vincent; citation_author=J.S. Serody; citation_author=K.E. Beckermann; citation_author=et al.; citation_journal_title=JCI Insight; citation_volume=3; citation_firstpage=121522; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=HIF-1 mediates adaptation to hypoxia by actively downregulating mitochondrial oxygen consumption; citation_author=I. Papandreou; citation_author=R.A. Cairns; citation_author=L. Fontana; citation_author=A.L. Lim; citation_author=N.C. Denko; citation_journal_title=Cell Metab.; citation_volume=3; citation_firstpage=187; citation_lastpage=197; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=Patterns of aneuploidy in stage IV clear cell renal cell carcinoma revealed by comparative genomic hybridization and spectral karyotyping; citation_author=C.P. Pavlovich; citation_author=H. Padilla-Nash; citation_author=D. Wangsa; citation_author=M.L. Nickerson; citation_author=V. Matrosova; citation_author=W.M. Linehan; citation_author=T. Ried; citation_author=J.L. Phillips; citation_journal_title=Genes Chromosomes Cancer; citation_volume=37; citation_firstpage=252; citation_lastpage=260; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=Scikit-learn: Machine Learning in Python; citation_author=F. Pedregosa; citation_author=G. Varoquaux; citation_author=A. Gramfort; citation_author=V. Michel; citation_author=B. Thirion; citation_author=O. Grisel; citation_author=M. Blondel; citation_author=P. Prettenhofer; citation_author=R. Weiss; citation_author=V. Dubourg; citation_author=et al.; citation_journal_title=J. Mach. Learn. Res.; citation_volume=12; citation_firstpage=2825; citation_lastpage=2830; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Ribosome biogenesis in cancer: new players and therapeutic avenues; citation_author=J. Pelletier; citation_author=G. Thomas; citation_author=S. Volarević; citation_journal_title=Nat. Rev. Cancer; citation_volume=18; citation_firstpage=51; citation_lastpage=63; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Integrative random forest for gene regulatory network inference; citation_author=F. Petralia; citation_author=P. Wang; citation_author=J. Yang; citation_author=Z. Tu; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=i197; citation_lastpage=i205; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer; citation_author=F. Petralia; citation_author=W.M. Song; citation_author=Z. Tu; citation_author=P. Wang; citation_journal_title=J. Proteome Res.; citation_volume=15; citation_firstpage=743; citation_lastpage=754; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity; citation_author=F. Petralia; citation_author=L. Wang; citation_author=J. Peng; citation_author=A. Yan; citation_author=J. Zhu; citation_author=P. Wang; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=i528; citation_lastpage=i536; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PTEN suppression of YY1 induces HIF-2α activity in von-Hippel-Lindau-null renal-cell carcinoma; citation_author=B.L. Petrella; citation_author=C.E. Brinckerhoff; citation_journal_title=Cancer Biol. Ther.; citation_volume=8; citation_firstpage=1389; citation_lastpage=1401; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Performance evaluation of DNA copy number segmentation methods; citation_author=M. Pierre-Jean; citation_author=G. Rigaill; citation_author=P. Neuvial; citation_journal_title=Brief. Bioinform.; citation_volume=16; citation_firstpage=600; citation_lastpage=615; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Isobaric labeling-based relative quantification in shotgun proteomics; citation_author=N. Rauniyar; citation_author=J.R. Yates; citation_journal_title=J. Proteome Res.; citation_volume=13; citation_firstpage=5293; citation_lastpage=5309; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Lapatinib versus hormone therapy in patients with advanced renal cell carcinoma: a randomized phase III clinical trial; citation_author=A. Ravaud; citation_author=R. Hawkins; citation_author=J.P. Gardner; citation_author=H. von der Maase; citation_author=N. Zantl; citation_author=P. Harper; citation_author=F. Rolland; citation_author=B. Audhuy; citation_author=J.P. Machiels; citation_author=F. Pétavy; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=26; citation_firstpage=2285; citation_lastpage=2291; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Loss of the RNA-binding protein TACO1 causes late-onset mitochondrial dysfunction in mice; citation_author=T.R. Richman; citation_author=H. Spåhr; citation_author=J.A. Ermer; citation_author=S.M.K. Davies; citation_author=H.M. Viola; citation_author=K.A. Bates; citation_author=J. Papadimitriou; citation_author=L.C. Hool; citation_author=J. Rodger; citation_author=N.G. Larsson; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=7; citation_firstpage=11884; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=The Cancer Genome Atlas Comprehensive Molecular Characterization of Renal Cell Carcinoma; citation_author=C.J. Ricketts; citation_author=A.A. De Cubas; citation_author=H. Fan; citation_author=C.C. Smith; citation_author=M. Lang; citation_author=E. Reznik; citation_author=R. Bowlby; citation_author=E.A. Gibb; citation_author=R. Akbani; citation_author=R. Beroukhim; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=23; citation_firstpage=313; citation_lastpage=326; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PureCN: copy number calling and SNV classification using targeted short read sequencing; citation_author=M. Riester; citation_author=A.P. Singh; citation_author=A.R. Brannon; citation_author=K. Yu; citation_author=C.D. Campbell; citation_author=D.Y. Chiang; citation_author=M.P. Morrissey; citation_journal_title=Source Code Biol. Med.; citation_volume=11; citation_firstpage=13; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Integrative clinical genomics of metastatic cancer; citation_author=D.R. Robinson; citation_author=Y.M. Wu; citation_author=R.J. Lonigro; citation_author=P. Vats; citation_author=E. Cobain; citation_author=J. Everett; citation_author=X. Cao; citation_author=E. Rabban; citation_author=C. Kumar-Sinha; citation_author=V. Raymond; citation_author=et al.; citation_journal_title=Nature; citation_volume=548; citation_firstpage=297; citation_lastpage=303; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents; citation_author=P.L. Ross; citation_author=Y.N. Huang; citation_author=J.N. Marchese; citation_author=B. Williamson; citation_author=K. Parker; citation_author=S. Hattan; citation_author=N. Khainovski; citation_author=S. Pillai; citation_author=S. Dey; citation_author=S. Daniels; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=3; citation_firstpage=1154; citation_lastpage=1169; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=An Analysis of the Sensitivity of Proteogenomic Mapping of Somatic Mutations and Novel Splicing Events in Cancer; citation_author=K.V. Ruggles; citation_author=Z. Tang; citation_author=X. Wang; citation_author=H. Grover; citation_author=M. Askenazi; citation_author=J. Teubl; citation_author=S. Cao; citation_author=M.D. McLellan; citation_author=K.R. Clauser; citation_author=D.L. Tabb; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=15; citation_firstpage=1060; citation_lastpage=1071; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma; citation_author=M. Sade-Feldman; citation_author=K. Yizhak; citation_author=S.L. Bjorgaard; citation_author=J.P. Ray; citation_author=C.G. de Boer; citation_author=R.W. Jenkins; citation_author=D.J. Lieb; citation_author=J.H. Chen; citation_author=D.T. Frederick; citation_author=M. Barzily-Rokni; citation_author=et al.; citation_journal_title=Cell; citation_volume=175; citation_firstpage=998; citation_lastpage=1013.e20; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets; citation_author=M.M. Savitski; citation_author=M. Wilhelm; citation_author=H. Hahne; citation_author=B. Kuster; citation_author=M. Bantscheff; citation_journal_title=Mol. Cell. Proteomics; citation_volume=14; citation_firstpage=2394; citation_lastpage=2404; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=GOLPH3 modulates mTOR signalling and rapamycin sensitivity in cancer; citation_author=K.L. Scott; citation_author=O. Kabbarah; citation_author=M.C. Liang; citation_author=E. Ivanova; citation_author=V. Anagnostou; citation_author=J. Wu; citation_author=S. Dhakal; citation_author=M. Wu; citation_author=S. Chen; citation_author=T. Feinberg; citation_author=et al.; citation_journal_title=Nature; citation_volume=459; citation_firstpage=1085; citation_lastpage=1090; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Platelet-derived Growth Factor Is Chemotactic for Fibroblasts; citation_author=H. Seppä; citation_author=G. Grotendorst; citation_author=S. Seppä; citation_author=E. Schiffmann; citation_author=G.R. Martin; citation_journal_title=J. Cell Biol.; citation_volume=92; citation_firstpage=584; citation_lastpage=588; citation_publication_date=1982; "></meta><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; citation_author=P. Shannon; citation_author=A. Markiel; citation_author=O. Ozier; citation_author=N.S. Baliga; citation_author=J.T. Wang; citation_author=D. Ramage; citation_author=N. Amin; citation_author=B. Schwikowski; citation_author=T. Ideker; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=2498; citation_lastpage=2504; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=4E-BP1 is a key effector of the oncogenic activation of the AKT and ERK signaling pathways that integrates their function in tumors; citation_author=Q.B. She; citation_author=E. Halilovic; citation_author=Q. Ye; citation_author=W. Zhen; citation_author=S. Shirasawa; citation_author=T. Sasazuki; citation_author=D.B. Solit; citation_author=N. Rosen; citation_journal_title=Cancer Cell; citation_volume=18; citation_firstpage=39; citation_lastpage=51; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Genetic and functional studies implicate HIF1α as a 14q kidney cancer suppressor gene; citation_author=C. Shen; citation_author=R. Beroukhim; citation_author=S.E. Schumacher; citation_author=J. Zhou; citation_author=M. Chang; citation_author=S. Signoretti; citation_author=W.G. Kaelin; citation_journal_title=Cancer Discov.; citation_volume=1; citation_firstpage=222; citation_lastpage=235; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Role of DNA methylation in renal cell carcinoma; citation_author=N. Shenoy; citation_author=N. Vallumsetla; citation_author=Y. Zou; citation_author=J.N. Galeas; citation_author=M. Shrivastava; citation_author=C. Hu; citation_author=K. Susztak; citation_author=A. Verma; citation_journal_title=J. Hematol. Oncol.; citation_volume=8; citation_firstpage=88; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Cancer-associated fibroblasts: Their characteristics and their roles in tumor growth; citation_author=K. Shiga; citation_author=M. Hara; citation_author=T. Nagasaki; citation_author=T. Sato; citation_author=H. Takahashi; citation_author=H. Takeyama; citation_journal_title=Cancers (Basel); citation_volume=7; citation_firstpage=2443; citation_lastpage=2458; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=The Association Between PD-L1 Expression and the Clinical Outcomes to Vascular Endothelial Growth Factor-Targeted Therapy in Patients With Metastatic Clear Cell Renal Cell Carcinoma; citation_author=S.-J. Shin; citation_author=Y.K. Jeon; citation_author=Y.M. Cho; citation_author=J.-L. Lee; citation_author=D.H. Chung; citation_author=J.Y. Park; citation_author=H. Go; citation_journal_title=Oncologist; citation_volume=20; citation_firstpage=1253; citation_lastpage=1260; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Cancer statistics, 2018; citation_author=R.L. Siegel; citation_author=K.D. Miller; citation_author=A. Jemal; citation_journal_title=CA Cancer J. Clin.; citation_volume=68; citation_firstpage=7; citation_lastpage=30; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=TCGA Workflow: Analyze cancer genomics and epigenomics data using Bioconductor packages; citation_author=T.C. Silva; citation_author=A. Colaprico; citation_author=C. Olsen; citation_author=F. D’Angelo; citation_author=G. Bontempi; citation_author=M. Ceccarelli; citation_author=H. Noushmehr; citation_journal_title=F1000Res.; citation_volume=5; citation_firstpage=1542; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Endogenous retroviral signatures predict immunotherapy response in clear cell renal cell carcinoma; citation_author=C.C. Smith; citation_author=K.E. Beckermann; citation_author=D.S. Bortone; citation_author=A.A. De Cubas; citation_author=L.M. Bixby; citation_author=S.J. Lee; citation_author=A. Panda; citation_author=S. Ganesan; citation_author=G. Bhanot; citation_author=E.M. Wallen; citation_author=et al.; citation_journal_title=J. Clin. Invest.; citation_volume=128; citation_firstpage=4804; citation_lastpage=4820; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=One-Class Detection of Cell States in Tumor Subtypes; citation_author=A. Sokolov; citation_author=E.O. Paull; citation_author=J.M. Stuart; citation_journal_title=Pac. Symp. Biocomput.; citation_volume=21; citation_firstpage=405; citation_lastpage=416; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis; citation_author=X. Song; citation_author=J. Ji; citation_author=K.J. Gleason; citation_author=F. Yang; citation_author=J.A. Martignetti; citation_author=L.S. Chen; citation_author=P. Wang; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_number=Suppl 1; citation_firstpage=S52; citation_lastpage=S65; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Robust and tissue-independent gender-specific transcript biomarkers; citation_author=F. Staedtler; citation_author=N. Hartmann; citation_author=M. Letzkus; citation_author=S. Bongiovanni; citation_author=A. Scherer; citation_author=P. Marc; citation_author=K.J. Johnson; citation_author=M.M. Schumacher; citation_journal_title=Biomarkers; citation_volume=18; citation_firstpage=436; citation_lastpage=445; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Pazopanib in locally advanced or metastatic renal cell carcinoma: results of a randomized phase III trial; citation_author=C.N. Sternberg; citation_author=I.D. Davis; citation_author=J. Mardiak; citation_author=C. Szczylik; citation_author=E. Lee; citation_author=J. Wagstaff; citation_author=C.H. Barrios; citation_author=P. Salman; citation_author=O.A. Gladkov; citation_author=A. Kavina; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=28; citation_firstpage=1061; citation_lastpage=1068; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=GLay: community structure analysis of biological networks; citation_author=G. Su; citation_author=A. Kuchinsky; citation_author=J.H. Morris; citation_author=D.J. States; citation_author=F. Meng; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=3135; citation_lastpage=3137; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles; citation_author=A. Subramanian; citation_author=P. Tamayo; citation_author=V.K. Mootha; citation_author=S. Mukherjee; citation_author=B.L. Ebert; citation_author=M.A. Gillette; citation_author=A. Paulovich; citation_author=S.L. Pomeroy; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15545; citation_lastpage=15550; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=OptiType: precision HLA typing from next-generation sequencing data; citation_author=A. Szolek; citation_author=B. Schubert; citation_author=C. Mohr; citation_author=M. Sturm; citation_author=M. Feldhahn; citation_author=O. Kohlbacher; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=3310; citation_lastpage=3316; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Tandem mass tags: a novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS; citation_author=A. Thompson; citation_author=J. Schäfer; citation_author=K. Kuhn; citation_author=S. Kienle; citation_author=J. Schwarz; citation_author=G. Schmidt; citation_author=T. Neumann; citation_author=R. Johnstone; citation_author=A.K. Mohammed; citation_author=C. Hamon; citation_journal_title=Anal. Chem.; citation_volume=75; citation_firstpage=1895; citation_lastpage=1904; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics; citation_author=C.C. Tsou; citation_author=D. Avtonomov; citation_author=B. Larsen; citation_author=M. Tucholska; citation_author=H. Choi; citation_author=A.C. Gingras; citation_author=A.I. Nesvizhskii; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=258; citation_lastpage=264; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Deterministic Evolutionary Trajectories Influence Primary Tumor Growth: TRACERx Renal; citation_author=S. Turajlic; citation_author=H. Xu; citation_author=K. Litchfield; citation_author=A. Rowan; citation_author=S. Horswell; citation_author=T. Chambers; citation_author=T. O’Brien; citation_author=J.I. Lopez; citation_author=T.B.K. Watkins; citation_author=D. Nicol; citation_author=TRACERx Renal Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=595; citation_lastpage=610; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Tracking Cancer Evolution Reveals Constrained Routes to Metastases: TRACERx Renal; citation_author=S. Turajlic; citation_author=H. Xu; citation_author=K. Litchfield; citation_author=A. Rowan; citation_author=T. Chambers; citation_author=J.I. Lopez; citation_author=D. Nicol; citation_author=T. O’Brien; citation_author=J. Larkin; citation_author=S. Horswell; citation_author=PEACE; citation_author=TRACERx Renal Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=581; citation_lastpage=594; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response; citation_author=V.G. Tusher; citation_author=R. Tibshirani; citation_author=G. Chu; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=98; citation_firstpage=5116; citation_lastpage=5121; citation_publication_date=2001; "></meta><meta name="citation_reference" content="citation_title=Mechanisms of specificity in protein phosphorylation; citation_author=J.A. Ubersax; citation_author=J.E. Ferrell; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=8; citation_firstpage=530; citation_lastpage=541; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Insulin signalling to mTOR mediated by the Akt/PKB substrate PRAS40; citation_author=E. Vander Haar; citation_author=S.I. Lee; citation_author=S. Bandhakavi; citation_author=T.J. Griffin; citation_author=D.H. Kim; citation_journal_title=Nat. Cell Biol.; citation_volume=9; citation_firstpage=316; citation_lastpage=323; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Protein identification using customized protein sequence databases derived from RNA-Seq data; citation_author=X. Wang; citation_author=R.J.C. Slebos; citation_author=D. Wang; citation_author=P.J. Halvey; citation_author=D.L. Tabb; citation_author=D.C. Liebler; citation_author=B. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=11; citation_firstpage=1009; citation_lastpage=1017; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit; citation_author=J. Wang; citation_author=S. Vasaikar; citation_author=Z. Shi; citation_author=M. Greer; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=W1; citation_firstpage=W130; citation_lastpage=W137; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=VSTM2A over-expression is a sensitive and specific biomarker for mucinous tubular and spindle cell carcinoma (MTSCC) of the kidney; citation_author=L. Wang; citation_author=Y. Zhang; citation_author=Y.-B. Chen; citation_author=S.L. Skala; citation_author=H.A. Al-Ahmadie; citation_author=X. Wang; citation_author=X. Cao; citation_author=B.A. Veeneman; citation_author=J. Chen; citation_author=M. Cieślik; citation_author=et al.; citation_journal_title=Am. J. Surg. Pathol.; citation_volume=42; citation_firstpage=1571; citation_lastpage=1584; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=An empirical approach leveraging tumorgrafts to dissect the tumor microenvironment in renal cell carcinoma identifies missing link to prognostic inflammatory factors; citation_author=T. Wang; citation_author=R. Lu; citation_author=P. Kapur; citation_author=B.S. Jaiswal; citation_author=R. Hannan; citation_author=Z. Zhang; citation_author=I. Pedrosa; citation_author=J.J. Luke; citation_author=H. Zhang; citation_author=L.D. Goldstein; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=8; citation_firstpage=1142; citation_lastpage=1155; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations; citation_author=B. Wen; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=485; citation_lastpage=493; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=High-resolution enabled TMT 8-plexing; citation_author=T. Werner; citation_author=I. Becher; citation_author=G. Sweetman; citation_author=C. Doce; citation_author=M.M. Savitski; citation_author=M. Bantscheff; citation_journal_title=Anal. Chem.; citation_volume=84; citation_firstpage=7188; citation_lastpage=7194; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Metabolic reprogramming in clear cell renal cell carcinoma; citation_author=H.I. Wettersten; citation_author=O.A. Aboud; citation_author=P.N. Lara; citation_author=R.H. Weiss; citation_journal_title=Nat. Rev. Nephrol.; citation_volume=13; citation_firstpage=410; citation_lastpage=419; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking; citation_author=M.D. Wilkerson; citation_author=D.N. Hayes; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=1572; citation_lastpage=1573; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Transcription factor YY1 contributes to tumor growth by stabilizing hypoxia factor HIF-1α in a p53-independent manner; citation_author=S. Wu; citation_author=V. Kasim; citation_author=M.R. Kano; citation_author=S. Tanaka; citation_author=S. Ohba; citation_author=Y. Miura; citation_author=K. Miyata; citation_author=X. Liu; citation_author=A. Matsuhashi; citation_author=U.I. Chung; citation_author=et al.; citation_journal_title=Cancer Res.; citation_volume=73; citation_firstpage=1787; citation_lastpage=1799; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization; citation_author=T. Xu; citation_author=T.D. Le; citation_author=L. Liu; citation_author=N. Su; citation_author=R. Wang; citation_author=B. Sun; citation_author=A. Colaprico; citation_author=G. Bontempi; citation_author=J. Li; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=3131; citation_lastpage=3133; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Pindel: A pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads; citation_author=K. Ye; citation_author=M.H. Schulz; citation_author=Q. Long; citation_author=R. Apweiler; citation_author=Z. Ning; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2865; citation_lastpage=2871; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=MODMatcher: multi-omics data matcher for integrative genomic analysis; citation_author=S. Yoo; citation_author=T. Huang; citation_author=J.D. Campbell; citation_author=E. Lee; citation_author=Z. Tu; citation_author=M.W. Geraci; citation_author=C.A. Powell; citation_author=E.E. Schadt; citation_author=A. Spira; citation_author=J. Zhu; citation_journal_title=PLoS Comput. Biol.; citation_volume=10; citation_firstpage=e1003790; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Inferring tumour purity and stromal and immune cell admixture from expression data; citation_author=K. Yoshihara; citation_author=M. Shahmoradgoli; citation_author=E. Martínez; citation_author=R. Vegesna; citation_author=H. Kim; citation_author=W. Torres-Garcia; citation_author=V. Treviño; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=D.A. Levine; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2612; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of human colon and rectal cancer; citation_author=B. Zhang; citation_author=J. Wang; citation_author=X. Wang; citation_author=J. Zhu; citation_author=Q. Liu; citation_author=Z. Shi; citation_author=M.C. Chambers; citation_author=L.J. Zimmerman; citation_author=K.F. Shaddox; citation_author=S. Kim; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=382; citation_lastpage=387; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=VGLL4 functions as a new tumor suppressor in lung cancer by negatively regulating the YAP-TEAD transcriptional complex; citation_author=W. Zhang; citation_author=Y. Gao; citation_author=P. Li; citation_author=Z. Shi; citation_author=T. Guo; citation_author=F. Li; citation_author=X. Han; citation_author=Y. Feng; citation_author=C. Zheng; citation_author=Z. Wang; citation_author=et al.; citation_journal_title=Cell Res.; citation_volume=24; citation_firstpage=331; citation_lastpage=343; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer; citation_author=H. Zhang; citation_author=T. Liu; citation_author=Z. Zhang; citation_author=S.H. Payne; citation_author=B. Zhang; citation_author=J.E. McDermott; citation_author=J.Y. Zhou; citation_author=V.A. Petyuk; citation_author=L. Chen; citation_author=D. Ray; citation_author=CPTAC Investigators; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=755; citation_lastpage=765; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=SUMOylation of VEGFR2 regulates its intracellular trafficking and pathological angiogenesis; citation_author=H.J. Zhou; citation_author=Z. Xu; citation_author=Z. Wang; citation_author=H. Zhang; citation_author=Z.W. Zhuang; citation_author=M. Simons; citation_author=W. Min; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=3303; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=SUMOylation negatively regulates angiogenesis by targeting endothelial NOTCH signaling; citation_author=X. Zhu; citation_author=S. Ding; citation_author=C. Qiu; citation_author=Y. Shi; citation_author=L. Song; citation_author=Y. Wang; citation_author=Y. Wang; citation_author=J. Li; citation_author=Y. Wang; citation_author=Y. Sun; citation_author=et al.; citation_journal_title=Circ. Res.; citation_volume=121; citation_firstpage=636; citation_lastpage=649; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Alteration of the antitumor immune response by cancer-associated fibroblasts; citation_author=L. Ziani; citation_author=S. Chouaib; citation_author=J. Thiery; citation_journal_title=Front. Immunol.; citation_volume=9; citation_firstpage=414; citation_publication_date=2018; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;To elucidate the deregulated functional modules that drive clear cell renal cell carcinoma (ccRCC), we performed comprehensive genomic, epigenomic, transcriptomic, proteomic, and phosphoproteomic characterization of treatment-naive ccRCC and paired normal adjacent tissue samples. Genomic analyses identified a distinct molecular subgroup associated with genomic instability. Integration of proteogenomic measurements uniquely identified protein dysregulation of cellular mechanisms impacted by genomic alterations, including oxidative phosphorylation-related metabolism, protein translation processes, and phospho-signaling modules. To assess the degree of immune infiltration in individual tumors, we identified microenvironment cell signatures that delineated four immune-based ccRCC subtypes characterized by distinct cellular pathways. This study reports a large-scale proteogenomic analysis of ccRCC to discern the functional impact of genomic alterations and provides evidence for rational treatment selection stemming from ccRCC pathobiology.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931123-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419311237%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(19)31123-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419311237%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(19)31123-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419311237%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931123-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419311237%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931123-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419311237%253Fshowall%253Dtrue&amp;pii=S0092-8674%2819%2931123-7" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=cell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931123-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419311237%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931123-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419311237%253Fshowall%253Dtrue&amp;pii=S0092-8674%2819%2931123-7" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2019.10.007">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(23)X0017-8"><img src="/cms/asset/atypon:cms:attachment:img:d18e6:rev:1722883367683-3936:pii:S0092867423X00178/cover.tif.jpg" alt="Cover Image - Cell, Volume 187, Issue 16"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S0092-8674(19)31123-7"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Resource</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cell/issue?pii=S0092-8674(18)X0023-3" class="article-header__vol faded">
                                                    Volume 179, ISSUE 4</a><span></span><span>, </span><span class="article-header__pages faded">P964-983.e31, </span><span class="article-header__date faded">October 31, 2019</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=179&amp;issue=4&amp;pii=S0092-8674%2818%29X0023-3" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2819%2931123-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF [12 MB]</span><span class="alternative">PDF [12 MB]</span></a></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cell/ppt/S0092-8674(19)31123-7.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.cell.2019.10.007" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2019.10.007&amp;pii=S0092-8674%2819%2931123-7" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2019.10.007" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-926" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-926">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2819%2931123-7"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S0092867419311237&amp;issn=00928674&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867419311237&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David J. Clark">David J. Clark</a> <a href="#" data-db-target-for="crosref0015" aria-expanded="false" aria-controls="crosref0015" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0015" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David J. Clark</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clark%2C+David J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Saravana M. Dhanasekaran">Saravana M. Dhanasekaran</a> <a href="#" data-db-target-for="crosref0025" aria-expanded="false" aria-controls="crosref0025" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0025" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Saravana M. Dhanasekaran</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dhanasekaran%2C+Saravana M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Francesca Petralia">Francesca Petralia</a> <a href="#" data-db-target-for="crosref0035" aria-expanded="false" aria-controls="crosref0035" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0035" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Francesca Petralia</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petralia%2C+Francesca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jianbo Pan">Jianbo Pan</a> <a href="#" data-db-target-for="crosref0045" aria-expanded="false" aria-controls="crosref0045" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0045" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jianbo Pan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Pan%2C+Jianbo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaoyu Song">Xiaoyu Song</a> <a href="#" data-db-target-for="crosref0060" aria-expanded="false" aria-controls="crosref0060" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0060" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaoyu Song</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Song%2C+Xiaoyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yingwei Hu">Yingwei Hu</a> <a href="#" data-db-target-for="crosref0070" aria-expanded="false" aria-controls="crosref0070" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0070" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yingwei Hu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hu%2C+Yingwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Felipe da Veiga Leprevost">Felipe da Veiga Leprevost</a> <a href="#" data-db-target-for="crosref0080" aria-expanded="false" aria-controls="crosref0080" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0080" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Felipe da Veiga Leprevost</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=da Veiga Leprevost%2C+Felipe" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Boris Reva">Boris Reva</a> <a href="#" data-db-target-for="crosref0090" aria-expanded="false" aria-controls="crosref0090" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0090" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Boris Reva</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Reva%2C+Boris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tung-Shing M. Lih">Tung-Shing M. Lih</a> <a href="#" data-db-target-for="crosref0100" aria-expanded="false" aria-controls="crosref0100" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>32</sup></span></a><div data-db-target-of="crosref0100" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tung-Shing M. Lih</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">32 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lih%2C+Tung-Shing M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Hui-Yin Chang">Hui-Yin Chang</a><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Hui-Yin Chang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chang%2C+Hui-Yin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Weiping Ma">Weiping Ma</a><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Weiping Ma</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ma%2C+Weiping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher J. Ricketts">Christopher J. Ricketts</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher J. Ricketts</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ricketts%2C+Christopher J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lijun Chen">Lijun Chen</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lijun Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Lijun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Azra Krek">Azra Krek</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Azra Krek</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krek%2C+Azra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yize Li">Yize Li</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yize Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Yize" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Dmitry Rykunov">Dmitry Rykunov</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Dmitry Rykunov</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rykunov%2C+Dmitry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Qing Kay Li">Qing Kay Li</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Qing Kay Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Qing Kay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lin S. Chen">Lin S. Chen</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lin S. Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Public Health Sciences, University of Chicago, Chicago, IL 60637, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Lin S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Umut Ozbek">Umut Ozbek</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Umut Ozbek</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ozbek%2C+Umut" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Suhas Vasaikar">Suhas Vasaikar</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Suhas Vasaikar</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Vasaikar%2C+Suhas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yige Wu">Yige Wu</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yige Wu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wu%2C+Yige" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Seungyeul Yoo">Seungyeul Yoo</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Seungyeul Yoo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Yoo%2C+Seungyeul" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shrabanti Chowdhury">Shrabanti Chowdhury</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shrabanti Chowdhury</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chowdhury%2C+Shrabanti" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew A. Wyczalkowski">Matthew A. Wyczalkowski</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew A. Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiayi Ji">Jiayi Ji</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiayi Ji</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ji%2C+Jiayi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael Schnaubelt">Michael Schnaubelt</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael Schnaubelt</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schnaubelt%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Andy Kong">Andy Kong</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Andy Kong</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kong%2C+Andy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sunantha Sethuraman">Sunantha Sethuraman</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sunantha Sethuraman</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Sethuraman%2C+Sunantha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Dmitry M. Avtonomov">Dmitry M. Avtonomov</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Dmitry M. Avtonomov</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Avtonomov%2C+Dmitry M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Minghui Ao">Minghui Ao</a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Minghui Ao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ao%2C+Minghui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Antonio Colaprico">Antonio Colaprico</a><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Antonio Colaprico</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Colaprico%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au33-authorcard" aria-expanded="false" aria-controls="au33" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Song Cao">Song Cao</a><div data-db-target-of="au33-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Song Cao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cao%2C+Song" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au34-authorcard" aria-expanded="false" aria-controls="au34" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kyung-Cho Cho">Kyung-Cho Cho</a><div data-db-target-of="au34-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kyung-Cho Cho</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cho%2C+Kyung-Cho" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au35-authorcard" aria-expanded="false" aria-controls="au35" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Selim Kalayci">Selim Kalayci</a><div data-db-target-of="au35-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Selim Kalayci</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kalayci%2C+Selim" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au36-authorcard" aria-expanded="false" aria-controls="au36" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shiyong Ma">Shiyong Ma</a><div data-db-target-of="au36-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shiyong Ma</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ma%2C+Shiyong" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au37-authorcard" aria-expanded="false" aria-controls="au37" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wenke Liu">Wenke Liu</a><div data-db-target-of="au37-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wenke Liu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Wenke" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au38-authorcard" aria-expanded="false" aria-controls="au38" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kelly Ruggles">Kelly Ruggles</a><div data-db-target-of="au38-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kelly Ruggles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, New York University School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ruggles%2C+Kelly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au39-authorcard" aria-expanded="false" aria-controls="au39" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Anna Calinawan">Anna Calinawan</a><div data-db-target-of="au39-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Anna Calinawan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Calinawan%2C+Anna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au40-authorcard" aria-expanded="false" aria-controls="au40" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zeynep H. Gümüş">Zeynep H. Gümüş</a><div data-db-target-of="au40-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zeynep H. Gümüş</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gümüş%2C+Zeynep H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au41-authorcard" aria-expanded="false" aria-controls="au41" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel Geiszler">Daniel Geiszler</a><div data-db-target-of="au41-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel Geiszler</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Geiszler%2C+Daniel" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au42-authorcard" aria-expanded="false" aria-controls="au42" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily Kawaler">Emily Kawaler</a><div data-db-target-of="au42-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily Kawaler</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kawaler%2C+Emily" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au43-authorcard" aria-expanded="false" aria-controls="au43" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Guo Ci Teo">Guo Ci Teo</a><div data-db-target-of="au43-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Guo Ci Teo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Teo%2C+Guo Ci" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au44-authorcard" aria-expanded="false" aria-controls="au44" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bo Wen">Bo Wen</a><div data-db-target-of="au44-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au45-authorcard" aria-expanded="false" aria-controls="au45" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yuping Zhang">Yuping Zhang</a><div data-db-target-of="au45-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yuping Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Yuping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au46-authorcard" aria-expanded="false" aria-controls="au46" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sarah Keegan">Sarah Keegan</a><div data-db-target-of="au46-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sarah Keegan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Keegan%2C+Sarah" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au47-authorcard" aria-expanded="false" aria-controls="au47" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kai Li">Kai Li</a><div data-db-target-of="au47-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kai Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Kai" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au48-authorcard" aria-expanded="false" aria-controls="au48" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Feng Chen">Feng Chen</a><div data-db-target-of="au48-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Feng Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Departments of Medicine and Cell Biology and Physiology, Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Feng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au49-authorcard" aria-expanded="false" aria-controls="au49" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Nathan Edwards">Nathan Edwards</a><div data-db-target-of="au49-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Nathan Edwards</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry and Cellular Biology, Georgetown University, Washington, DC 20007, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au50-authorcard" aria-expanded="false" aria-controls="au50" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Phillip M. Pierorazio">Phillip M. Pierorazio</a><div data-db-target-of="au50-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Phillip M. Pierorazio</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Brady Urological Institute and Department of Urology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Pierorazio%2C+Phillip M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au51-authorcard" aria-expanded="false" aria-controls="au51" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xi Steven Chen">Xi Steven Chen</a><div data-db-target-of="au51-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xi Steven Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Public Health Sciences, University of Miami Miller School of Medicine, Miami, FL 33136, USA<p></p>Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xi Steven" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au52-authorcard" aria-expanded="false" aria-controls="au52" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christian P. Pavlovich">Christian P. Pavlovich</a><div data-db-target-of="au52-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christian P. Pavlovich</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Brady Urological Institute and Department of Urology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Pavlovich%2C+Christian P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au53-authorcard" aria-expanded="false" aria-controls="au53" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author A. Ari Hakimi">A. Ari Hakimi</a><div data-db-target-of="au53-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">A. Ari Hakimi</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Surgery, Urology Service, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hakimi%2C+A. Ari" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au54-authorcard" aria-expanded="false" aria-controls="au54" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gabriel Brominski">Gabriel Brominski</a><div data-db-target-of="au54-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gabriel Brominski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Urology, Poznań University of Medical Sciences, Szwajcarska 3, Poznań 61-285, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Brominski%2C+Gabriel" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au55-authorcard" aria-expanded="false" aria-controls="au55" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author James J. Hsieh">James J. Hsieh</a><div data-db-target-of="au55-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">James J. Hsieh</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hsieh%2C+James J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au56-authorcard" aria-expanded="false" aria-controls="au56" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Andrzej Antczak">Andrzej Antczak</a><div data-db-target-of="au56-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Andrzej Antczak</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Urology, Poznań University of Medical Sciences, Szwajcarska 3, Poznań 61-285, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Antczak%2C+Andrzej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au57-authorcard" aria-expanded="false" aria-controls="au57" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tatiana Omelchenko">Tatiana Omelchenko</a><div data-db-target-of="au57-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tatiana Omelchenko</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Omelchenko%2C+Tatiana" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au58-authorcard" aria-expanded="false" aria-controls="au58" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jan Lubinski">Jan Lubinski</a><div data-db-target-of="au58-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jan Lubinski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Pathology, Pomeranian Medical University, Szczecin 71-252, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Lubinski%2C+Jan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au59-authorcard" aria-expanded="false" aria-controls="au59" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Maciej Wiznerowicz">Maciej Wiznerowicz</a><div data-db-target-of="au59-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Maciej Wiznerowicz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">International Institute for Molecular Oncology, Poznań 60-203, Poland<p></p>Poznań University of Medical Sciences, Poznan 60-701, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Wiznerowicz%2C+Maciej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au60-authorcard" aria-expanded="false" aria-controls="au60" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author W. Marston Linehan">W. Marston Linehan</a><div data-db-target-of="au60-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">W. Marston Linehan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Linehan%2C+W. Marston" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au61-authorcard" aria-expanded="false" aria-controls="au61" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au61-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au62-authorcard" aria-expanded="false" aria-controls="au62" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au62-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au63-authorcard" aria-expanded="false" aria-controls="au63" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily S. Boja">Emily S. Boja</a><div data-db-target-of="au63-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au64-authorcard" aria-expanded="false" aria-controls="au64" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mehdi Mesri">Mehdi Mesri</a><div data-db-target-of="au64-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au65-authorcard" aria-expanded="false" aria-controls="au65" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tara Hiltke">Tara Hiltke</a><div data-db-target-of="au65-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au66-authorcard" aria-expanded="false" aria-controls="au66" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au66-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au67-authorcard" aria-expanded="false" aria-controls="au67" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au67-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au68-authorcard" aria-expanded="false" aria-controls="au68" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiang Qian">Jiang Qian</a><div data-db-target-of="au68-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiang Qian</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Qian%2C+Jiang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au69-authorcard" aria-expanded="false" aria-controls="au69" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David Fenyö">David Fenyö</a><div data-db-target-of="au69-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David Fenyö</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, New York University School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Fenyö%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au70-authorcard" aria-expanded="false" aria-controls="au70" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang</a><div data-db-target-of="au70-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au71-authorcard" aria-expanded="false" aria-controls="au71" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au71-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Washington University School of Medicine, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au72-authorcard" aria-expanded="false" aria-controls="au72" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Eric Schadt">Eric Schadt</a><div data-db-target-of="au72-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Eric Schadt</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Sema4, Stamford, CT 06902, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schadt%2C+Eric" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au73-authorcard" aria-expanded="false" aria-controls="au73" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Arul M. Chinnaiyan">Arul M. Chinnaiyan</a><div data-db-target-of="au73-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Arul M. Chinnaiyan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chinnaiyan%2C+Arul M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au74-authorcard" aria-expanded="false" aria-controls="au74" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhen Zhang">Zhen Zhang</a><div data-db-target-of="au74-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au75-authorcard" aria-expanded="false" aria-controls="au75" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gilbert S. Omenn">Gilbert S. Omenn</a><div data-db-target-of="au75-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gilbert S. Omenn</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Internal Medicine, Human Genetics, and School of Public Health, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Omenn%2C+Gilbert S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au76-authorcard" aria-expanded="false" aria-controls="au76" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marcin Cieslik">Marcin Cieslik<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0470" aria-expanded="false" aria-controls="crosref0470" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>33</sup></span></a><div data-db-target-of="crosref0470" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">33 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au76-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marcin Cieslik</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#5538363c3026393c3e153830317b383c363d7b303120" title="Send email to Marcin Cieslik"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">33 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cieslik%2C+Marcin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Marcin Cieslik" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#9df0fef4f8eef1f4f6ddf0f8f9b3f0f4fef5b3f8f9e8"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au77-authorcard" aria-expanded="false" aria-controls="au77" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel W. Chan">Daniel W. Chan<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0485" aria-expanded="false" aria-controls="crosref0485" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>33</sup></span></a><div data-db-target-of="crosref0485" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">33 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au77-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel W. Chan</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#debabdb6bfb09eb4b6b3b7f0bbbaab" title="Send email to Daniel W. Chan"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">33 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Daniel W. Chan" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#96f2f5fef7f8d6fcfefbffb8f3f2e3"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au78-authorcard" aria-expanded="false" aria-controls="au78" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexey I. Nesvizhskii">Alexey I. Nesvizhskii<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0500" aria-expanded="false" aria-controls="crosref0500" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>33</sup></span></a><div data-db-target-of="crosref0500" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">33 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au78-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexey I. Nesvizhskii</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#305e55434659705d55541e455d5953581e555445" title="Send email to Alexey I. Nesvizhskii"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">33 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Nesvizhskii%2C+Alexey I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Alexey I. Nesvizhskii" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#b5dbd0c6c3dcf5d8d0d19bc0d8dcd6dd9bd0d1c0"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au79-authorcard" aria-expanded="false" aria-controls="au79" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pei Wang">Pei Wang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0515" aria-expanded="false" aria-controls="crosref0515" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>33</sup></span></a><div data-db-target-of="crosref0515" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">33 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au79-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pei Wang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#4f3f2a2661382e21280f223c3c22612a2b3a" title="Send email to Pei Wang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">33 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Pei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Pei Wang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#e0908589ce97818e87a08d93938dce858495"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au80-authorcard" aria-expanded="false" aria-controls="au80" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Hui Zhang">Hui Zhang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0530" aria-expanded="false" aria-controls="crosref0530" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>33</sup></span></a><div data-db-target-of="crosref0530" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">33 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>, </sup><a href="#" data-db-target-for="crosref0535" aria-expanded="false" aria-controls="crosref0535" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>34</sup></span></a><div data-db-target-of="crosref0535" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">34 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au80-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Hui Zhang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#92fae7fbe8faf3fcf5d2f8fae7bcf7f6e7" title="Send email to Hui Zhang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">33 These authors contributed equally<br/>34 Lead Contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Hui Zhang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#7b130e1201131a151c3b11130e551e1f0e"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Abdul Samad Hashimi" class="dropBlock__slider__ctrl">Abdul Samad Hashimi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Abdul Samad back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Abdul Samad Hashimi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hashimi%2C+Abdul Samad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alexander R. Pico" class="dropBlock__slider__ctrl">Alexander R. Pico</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alexander R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alexander R. Pico</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pico%2C+Alexander R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alla Karpova" class="dropBlock__slider__ctrl">Alla Karpova</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alla back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alla Karpova</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Karpova%2C+Alla" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alyssa Charamut" class="dropBlock__slider__ctrl">Alyssa Charamut</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alyssa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alyssa Charamut</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Charamut%2C+Alyssa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amanda G. Paulovich" class="dropBlock__slider__ctrl">Amanda G. Paulovich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amanda G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amanda G. Paulovich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amy M. Perou" class="dropBlock__slider__ctrl">Amy M. Perou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amy M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amy M. Perou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Perou%2C+Amy M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anna Malovannaya" class="dropBlock__slider__ctrl">Anna Malovannaya</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anna Malovannaya</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malovannaya%2C+Anna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Annette Marrero-Oliveras" class="dropBlock__slider__ctrl">Annette Marrero-Oliveras</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Annette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Annette Marrero-Oliveras</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Marrero-Oliveras%2C+Annette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anupriya Agarwal" class="dropBlock__slider__ctrl">Anupriya Agarwal</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anupriya back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anupriya Agarwal</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Agarwal%2C+Anupriya" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Hindenach" class="dropBlock__slider__ctrl">Barbara Hindenach</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Hindenach</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hindenach%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Pruetz" class="dropBlock__slider__ctrl">Barbara Pruetz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Pruetz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pruetz%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Beom-Jun Kim" class="dropBlock__slider__ctrl">Beom-Jun Kim</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Beom-Jun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Beom-Jun Kim</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kim%2C+Beom-Jun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Brian J. Druker" class="dropBlock__slider__ctrl">Brian J. Druker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Brian J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Brian J. Druker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Druker%2C+Brian J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chelsea J. Newton" class="dropBlock__slider__ctrl">Chelsea J. Newton</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chelsea J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chelsea J. Newton</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Newton%2C+Chelsea J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chet Birger" class="dropBlock__slider__ctrl">Chet Birger</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chet back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chet Birger</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birger%2C+Chet" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Corbin D. Jones" class="dropBlock__slider__ctrl">Corbin D. Jones</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Corbin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Corbin D. Jones</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jones%2C+Corbin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Cristina Tognon" class="dropBlock__slider__ctrl">Cristina Tognon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Cristina back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Cristina Tognon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tognon%2C+Cristina" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="D.R. Mani" class="dropBlock__slider__ctrl">D.R. Mani</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="D.R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">D.R. Mani</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dana R. Valley" class="dropBlock__slider__ctrl">Dana R. Valley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dana R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dana R. Valley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Valley%2C+Dana R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel C. Rohrer" class="dropBlock__slider__ctrl">Daniel C. Rohrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel C. Rohrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rohrer%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel C. Zhou" class="dropBlock__slider__ctrl">Daniel C. Zhou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel C. Zhou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhou%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Darlene Tansil" class="dropBlock__slider__ctrl">Darlene Tansil</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Darlene back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Darlene Tansil</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tansil%2C+Darlene" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Chesla" class="dropBlock__slider__ctrl">David Chesla</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Chesla</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chesla%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Heiman" class="dropBlock__slider__ctrl">David Heiman</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Heiman</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Heiman%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Wheeler" class="dropBlock__slider__ctrl">David Wheeler</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Wheeler</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wheeler%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Donghui Tan" class="dropBlock__slider__ctrl">Donghui Tan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Donghui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Donghui Tan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tan%2C+Donghui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Doug Chan" class="dropBlock__slider__ctrl">Doug Chan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Doug back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Doug Chan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Doug" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Emek Demir" class="dropBlock__slider__ctrl">Emek Demir</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Emek back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Emek Demir</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Demir%2C+Emek" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ewa Malc" class="dropBlock__slider__ctrl">Ewa Malc</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ewa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ewa Malc</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malc%2C+Ewa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Francesmary Modugno" class="dropBlock__slider__ctrl">Francesmary Modugno</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Francesmary back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Francesmary Modugno</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Modugno%2C+Francesmary" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gaddy Getz" class="dropBlock__slider__ctrl">Gaddy Getz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gaddy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gaddy Getz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Getz%2C+Gaddy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Galen Hostetter" class="dropBlock__slider__ctrl">Galen Hostetter</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Galen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Galen Hostetter</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hostetter%2C+Galen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="George D. Wilson" class="dropBlock__slider__ctrl">George D. Wilson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="George D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">George D. Wilson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wilson%2C+George D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gerald W. Hart" class="dropBlock__slider__ctrl">Gerald W. Hart</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gerald W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gerald W. Hart</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hart%2C+Gerald W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Heng Zhu" class="dropBlock__slider__ctrl">Heng Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Heng back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Heng Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Heng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hongwei Liu" class="dropBlock__slider__ctrl">Hongwei Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hongwei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hongwei Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Hongwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Houston Culpepper" class="dropBlock__slider__ctrl">Houston Culpepper</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Houston back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Houston Culpepper</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Culpepper%2C+Houston" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hua Sun" class="dropBlock__slider__ctrl">Hua Sun</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hua back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hua Sun</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sun%2C+Hua" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hua Zhou" class="dropBlock__slider__ctrl">Hua Zhou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hua back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hua Zhou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhou%2C+Hua" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jacob Day" class="dropBlock__slider__ctrl">Jacob Day</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jacob back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jacob Day</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Day%2C+Jacob" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="James Suh" class="dropBlock__slider__ctrl">James Suh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="James back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">James Suh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Suh%2C+James" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jasmine Huang" class="dropBlock__slider__ctrl">Jasmine Huang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jasmine back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jasmine Huang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Jasmine" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jason McDermott" class="dropBlock__slider__ctrl">Jason McDermott</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jason back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jason McDermott</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey R. Whiteaker" class="dropBlock__slider__ctrl">Jeffrey R. Whiteaker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey R. Whiteaker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteaker%2C+Jeffrey R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey W. Tyner" class="dropBlock__slider__ctrl">Jeffrey W. Tyner</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey W. Tyner</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tyner%2C+Jeffrey W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jennifer Eschbacher" class="dropBlock__slider__ctrl">Jennifer Eschbacher</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jennifer back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jennifer Eschbacher</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Eschbacher%2C+Jennifer" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jin Chen" class="dropBlock__slider__ctrl">Jin Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jin Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Jin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="John McGee" class="dropBlock__slider__ctrl">John McGee</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="John back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">John McGee</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGee%2C+John" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jun Zhu" class="dropBlock__slider__ctrl">Jun Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jun Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Jun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karen A. Ketchum" class="dropBlock__slider__ctrl">Karen A. Ketchum</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karen A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karin D. Rodland" class="dropBlock__slider__ctrl">Karin D. Rodland</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karin D. Rodland</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karl Clauser" class="dropBlock__slider__ctrl">Karl Clauser</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karl back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karl Clauser</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Clauser%2C+Karl" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karna Robinson" class="dropBlock__slider__ctrl">Karna Robinson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karna Robinson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Robinson%2C+Karna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karsten Krug" class="dropBlock__slider__ctrl">Karsten Krug</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karsten back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karsten Krug</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Krug%2C+Karsten" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Katherine A. Hoadley" class="dropBlock__slider__ctrl">Katherine A. Hoadley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Katherine A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Katherine A. Hoadley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoadley%2C+Katherine A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ki Sung Um" class="dropBlock__slider__ctrl">Ki Sung Um</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ki Sung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ki Sung Um</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Um%2C+Ki Sung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kim Elburn" class="dropBlock__slider__ctrl">Kim Elburn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kim back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kim Elburn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Elburn%2C+Kim" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kimberly Holloway" class="dropBlock__slider__ctrl">Kimberly Holloway</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kimberly back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kimberly Holloway</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Holloway%2C+Kimberly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liang-Bo Wang" class="dropBlock__slider__ctrl">Liang-Bo Wang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liang-Bo back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liang-Bo Wang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Liang-Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lili Blumenberg" class="dropBlock__slider__ctrl">Lili Blumenberg</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lili back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lili Blumenberg</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Blumenberg%2C+Lili" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Linda Hannick" class="dropBlock__slider__ctrl">Linda Hannick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Linda back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Linda Hannick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hannick%2C+Linda" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liqun Qi" class="dropBlock__slider__ctrl">Liqun Qi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liqun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liqun Qi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Qi%2C+Liqun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lori J. Sokoll" class="dropBlock__slider__ctrl">Lori J. Sokoll</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lori J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lori J. Sokoll</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sokoll%2C+Lori J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="MacIntosh Cornwell" class="dropBlock__slider__ctrl">MacIntosh Cornwell</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="MacIntosh back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">MacIntosh Cornwell</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cornwell%2C+MacIntosh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marc Loriaux" class="dropBlock__slider__ctrl">Marc Loriaux</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marc back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marc Loriaux</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Loriaux%2C+Marc" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marcin J. Domagalski" class="dropBlock__slider__ctrl">Marcin J. Domagalski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marcin J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marcin J. Domagalski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Domagalski%2C+Marcin J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marina A. Gritsenko" class="dropBlock__slider__ctrl">Marina A. Gritsenko</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marina A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marina A. Gritsenko</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gritsenko%2C+Marina A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew Anderson" class="dropBlock__slider__ctrl">Matthew Anderson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew Anderson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Anderson%2C+Matthew" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew E. Monroe" class="dropBlock__slider__ctrl">Matthew E. Monroe</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew E. Monroe</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Monroe%2C+Matthew E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew J. Ellis" class="dropBlock__slider__ctrl">Matthew J. Ellis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew J. Ellis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Maureen Dyer" class="dropBlock__slider__ctrl">Maureen Dyer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Maureen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Maureen Dyer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dyer%2C+Maureen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Meenakshi Anurag" class="dropBlock__slider__ctrl">Meenakshi Anurag</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Meenakshi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Meenakshi Anurag</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Anurag%2C+Meenakshi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Meghan C. Burke" class="dropBlock__slider__ctrl">Meghan C. Burke</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Meghan C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Meghan C. Burke</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Burke%2C+Meghan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melissa Borucki" class="dropBlock__slider__ctrl">Melissa Borucki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melissa Borucki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borucki%2C+Melissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael A. Gillette" class="dropBlock__slider__ctrl">Michael A. Gillette</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael A. Gillette</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael J. Birrer" class="dropBlock__slider__ctrl">Michael J. Birrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael J. Birrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birrer%2C+Michael J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Lewis" class="dropBlock__slider__ctrl">Michael Lewis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Lewis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Lewis%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael M. Ittmann" class="dropBlock__slider__ctrl">Michael M. Ittmann</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael M. Ittmann</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ittmann%2C+Michael M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Smith" class="dropBlock__slider__ctrl">Michael Smith</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Smith</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Vernon" class="dropBlock__slider__ctrl">Michael Vernon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Vernon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vernon%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michelle Chaikin" class="dropBlock__slider__ctrl">Michelle Chaikin</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michelle back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michelle Chaikin</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chaikin%2C+Michelle" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Milan G. Chheda" class="dropBlock__slider__ctrl">Milan G. Chheda</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Milan G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Milan G. Chheda</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chheda%2C+Milan G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Munziba Khan" class="dropBlock__slider__ctrl">Munziba Khan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Munziba back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Munziba Khan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Khan%2C+Munziba" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nancy Roche" class="dropBlock__slider__ctrl">Nancy Roche</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nancy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nancy Roche</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Roche%2C+Nancy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Negin Vatanian" class="dropBlock__slider__ctrl">Negin Vatanian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Negin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Negin Vatanian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vatanian%2C+Negin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nicole Tignor" class="dropBlock__slider__ctrl">Nicole Tignor</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nicole back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nicole Tignor</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tignor%2C+Nicole" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Noam Beckmann" class="dropBlock__slider__ctrl">Noam Beckmann</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Noam back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Noam Beckmann</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Beckmann%2C+Noam" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pamela Grady" class="dropBlock__slider__ctrl">Pamela Grady</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pamela back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pamela Grady</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Grady%2C+Pamela" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Patricia Castro" class="dropBlock__slider__ctrl">Patricia Castro</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Patricia back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Patricia Castro</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Castro%2C+Patricia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Paul Piehowski" class="dropBlock__slider__ctrl">Paul Piehowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Paul back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Paul Piehowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Piehowski%2C+Paul" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Piotr Mieczkowski" class="dropBlock__slider__ctrl">Piotr Mieczkowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Piotr back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Piotr Mieczkowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mieczkowski%2C+Piotr" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pushpa Hariharan" class="dropBlock__slider__ctrl">Pushpa Hariharan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pushpa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pushpa Hariharan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hariharan%2C+Pushpa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Qingsong Gao" class="dropBlock__slider__ctrl">Qingsong Gao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Qingsong back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Qingsong Gao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gao%2C+Qingsong" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rajiv Dhir" class="dropBlock__slider__ctrl">Rajiv Dhir</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rajiv back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rajiv Dhir</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dhir%2C+Rajiv" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ramani Bhupendra Kothadia" class="dropBlock__slider__ctrl">Ramani Bhupendra Kothadia</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ramani Bhupendra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ramani Bhupendra Kothadia</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kothadia%2C+Ramani Bhupendra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rebecca Montgomery" class="dropBlock__slider__ctrl">Rebecca Montgomery</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rebecca back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rebecca Montgomery</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Montgomery%2C+Rebecca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Reyka G. Jayasinghe" class="dropBlock__slider__ctrl">Reyka G. Jayasinghe</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Reyka G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Reyka G. Jayasinghe</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jayasinghe%2C+Reyka G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Richard D. Smith" class="dropBlock__slider__ctrl">Richard D. Smith</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Richard D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert Edwards" class="dropBlock__slider__ctrl">Robert Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Robert" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert Zelt" class="dropBlock__slider__ctrl">Robert Zelt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert Zelt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zelt%2C+Robert" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ross Bremner" class="dropBlock__slider__ctrl">Ross Bremner</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ross back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ross Bremner</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bremner%2C+Ross" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ruiyang Liu" class="dropBlock__slider__ctrl">Ruiyang Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ruiyang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ruiyang Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Ruiyang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Runyu Hong" class="dropBlock__slider__ctrl">Runyu Hong</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Runyu back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Runyu Hong</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hong%2C+Runyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sailaja Mareedu" class="dropBlock__slider__ctrl">Sailaja Mareedu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sailaja back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sailaja Mareedu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mareedu%2C+Sailaja" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Samuel H. Payne" class="dropBlock__slider__ctrl">Samuel H. Payne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Samuel H. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sandra Cottingham" class="dropBlock__slider__ctrl">Sandra Cottingham</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sandra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sandra Cottingham</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cottingham%2C+Sandra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sanford P. Markey" class="dropBlock__slider__ctrl">Sanford P. Markey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sanford P. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sanford P. Markey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Markey%2C+Sanford P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Scott D. Jewell" class="dropBlock__slider__ctrl">Scott D. Jewell</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Scott D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Scott D. Jewell</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jewell%2C+Scott D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shalin Patel" class="dropBlock__slider__ctrl">Shalin Patel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shalin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shalin Patel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Patel%2C+Shalin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shankha Satpathy" class="dropBlock__slider__ctrl">Shankha Satpathy</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shankha back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shankha Satpathy</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Satpathy%2C+Shankha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shannon Richey" class="dropBlock__slider__ctrl">Shannon Richey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shannon back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shannon Richey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Richey%2C+Shannon" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sherri R. Davies" class="dropBlock__slider__ctrl">Sherri R. Davies</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sherri R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sherri R. Davies</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Davies%2C+Sherri R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Simina M. Boca" class="dropBlock__slider__ctrl">Simina M. Boca</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Simina M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Simina M. Boca</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Boca%2C+Simina M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Snehal Patil" class="dropBlock__slider__ctrl">Snehal Patil</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Snehal back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Snehal Patil</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Patil%2C+Snehal" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sohini Sengupta" class="dropBlock__slider__ctrl">Sohini Sengupta</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sohini back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sohini Sengupta</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sengupta%2C+Sohini" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sonya Carter" class="dropBlock__slider__ctrl">Sonya Carter</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sonya back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sonya Carter</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Carter%2C+Sonya" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stacey Gabriel" class="dropBlock__slider__ctrl">Stacey Gabriel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stacey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stacey Gabriel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabriel%2C+Stacey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stefani N. Thomas" class="dropBlock__slider__ctrl">Stefani N. Thomas</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stefani N. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stefani N. Thomas</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thomas%2C+Stefani N" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephanie De Young" class="dropBlock__slider__ctrl">Stephanie De Young</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephanie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephanie De Young</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=De Young%2C+Stephanie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephen E. Stein" class="dropBlock__slider__ctrl">Stephen E. Stein</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephen E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephen E. Stein</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stein%2C+Stephen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Steven A. Carr" class="dropBlock__slider__ctrl">Steven A. Carr</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Steven A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Steven A. Carr</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Steven M. Foltz" class="dropBlock__slider__ctrl">Steven M. Foltz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Steven M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Steven M. Foltz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Foltz%2C+Steven M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sue Hilsenbeck" class="dropBlock__slider__ctrl">Sue Hilsenbeck</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sue back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sue Hilsenbeck</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hilsenbeck%2C+Sue" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tanya Krubit" class="dropBlock__slider__ctrl">Tanya Krubit</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tanya back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tanya Krubit</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Krubit%2C+Tanya" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tao Liu" class="dropBlock__slider__ctrl">Tao Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tao Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tara Skelly" class="dropBlock__slider__ctrl">Tara Skelly</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tara Skelly</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Skelly%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Thomas Westbrook" class="dropBlock__slider__ctrl">Thomas Westbrook</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Thomas back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Thomas Westbrook</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Westbrook%2C+Thomas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Borate" class="dropBlock__slider__ctrl">Uma Borate</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Borate</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borate%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Velvulou" class="dropBlock__slider__ctrl">Uma Velvulou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Velvulou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Velvulou%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Vladislav A. Petyuk" class="dropBlock__slider__ctrl">Vladislav A. Petyuk</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Vladislav A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Vladislav A. Petyuk</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Petyuk%2C+Vladislav A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="William E. Bocik" class="dropBlock__slider__ctrl">William E. Bocik</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="William E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">William E. Bocik</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bocik%2C+William E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Xi Chen" class="dropBlock__slider__ctrl">Xi Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Xi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Xi Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yan Shi" class="dropBlock__slider__ctrl">Yan Shi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yan Shi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Yan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yifat Geffen" class="dropBlock__slider__ctrl">Yifat Geffen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yifat back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yifat Geffen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Geffen%2C+Yifat" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yihao Lu" class="dropBlock__slider__ctrl">Yihao Lu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yihao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yihao Lu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Lu%2C+Yihao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ying Wang" class="dropBlock__slider__ctrl">Ying Wang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ying back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ying Wang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Ying" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yosef Maruvka" class="dropBlock__slider__ctrl">Yosef Maruvka</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yosef back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yosef Maruvka</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maruvka%2C+Yosef" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhi Li" class="dropBlock__slider__ctrl">Zhi Li</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhi Li</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Zhi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhiao Shi" class="dropBlock__slider__ctrl">Zhiao Shi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhiao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhiao Shi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Zhiao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhidong Tu" class="dropBlock__slider__ctrl">Zhidong Tu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhidong back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhidong Tu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tu%2C+Zhidong" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">32 These authors contributed equally<br/>33 These authors contributed equally<br/>34 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.cell.2019.10.007">https://doi.org/10.1016/j.cell.2019.10.007</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.cell.2019.10.007"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2019.10.007" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cell/fulltext/S0092-8674(19)31125-0" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Active and Repressed Chromatin Domains Exhibit Distinct Nucleosome Segregation during DNA Replication</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="/cell/fulltext/S0092-8674(19)31120-1" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">Uganda Genome Resource Enables Insights into Population History and Genomic Discovery in Africa</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        

    

        


    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Integrated proteogenomic characterization in 103 ccRCC cases</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Delineation of chromosomal translocation events leading to chromosome 3p loss</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Tumor-specific proteomic/phosphoproteomic alterations unrevealed by mRNA analysis</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Immune-based subtypes of ccRCC defined by mRNA, proteome, and phosphoproteome</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">To elucidate the deregulated functional modules that drive clear cell renal cell carcinoma (ccRCC), we performed comprehensive genomic, epigenomic, transcriptomic, proteomic, and phosphoproteomic characterization of treatment-naive ccRCC and paired normal adjacent tissue samples. Genomic analyses identified a distinct molecular subgroup associated with genomic instability. Integration of proteogenomic measurements uniquely identified protein dysregulation of cellular mechanisms impacted by genomic alterations, including oxidative phosphorylation-related metabolism, protein translation processes, and phospho-signaling modules. To assess the degree of immune infiltration in individual tumors, we identified microenvironment cell signatures that delineated four immune-based ccRCC subtypes characterized by distinct cellular pathways. This study reports a large-scale proteogenomic analysis of ccRCC to discern the functional impact of genomic alterations and provides evidence for rational treatment selection stemming from ccRCC pathobiology.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical Abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical Abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/49ffad7e-4330-4639-9ab9-9f3941f9e0cc/fx1_lrg.jpg" data-large-image="/cms/attachment/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/4dbceec2-0cff-4f19-abe3-3469163764a0/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/49ffad7e-4330-4639-9ab9-9f3941f9e0cc/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;ccRCC&#34;&amp;ISSN=0092-8674">ccRCC</a></li><li><a href="/action/doSearch?AllField=&#34;renal carcinoma&#34;&amp;ISSN=0092-8674">renal carcinoma</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=0092-8674">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=0092-8674">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;immune infiltration&#34;&amp;ISSN=0092-8674">immune infiltration</a></li><li><a href="/action/doSearch?AllField=&#34;tumor microenvironment&#34;&amp;ISSN=0092-8674">tumor microenvironment</a></li><li><a href="/action/doSearch?AllField=&#34;phosphoproteomics&#34;&amp;ISSN=0092-8674">phosphoproteomics</a></li><li><a href="/action/doSearch?AllField=&#34;drug targets&#34;&amp;ISSN=0092-8674">drug targets</a></li><li><a href="/action/doSearch?AllField=&#34;chromosomal translocation&#34;&amp;ISSN=0092-8674">chromosomal translocation</a></li><li><a href="/action/doSearch?AllField=&#34;CPTAC&#34;&amp;ISSN=0092-8674">CPTAC</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">Renal cell carcinoma (RCC) is among the top ten most commonly diagnosed cancers worldwide (<div class="dropBlock reference-citations"><a href="#bib156" id="back-bib156" data-db-target-for="bib156-d218962e1878" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib156">Siegel et al., 2018</a><div data-db-target-of="bib156-d218962e1878" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Siegel R.L. </li><li class="loa__item">Miller K.D. </li><li class="loa__item">Jemal A. </li></ul><div class="reference-citations__detials">Cancer statistics, 2018.<div class="popup-ref"><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3322%2Fcaac.21442&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29313949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85040049759&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3369)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2018">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Clear cell renal cell carcinoma (ccRCC) is the predominant histology of renal cell carcinoma, representing 75% of all cases and the majority of cancer-associated deaths (<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d218962e1881" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Hsieh et al., 2017</a><div data-db-target-of="bib64-d218962e1881" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hsieh J.J. </li><li class="loa__item">Purdue M.P. </li><li class="loa__item">Signoretti S. </li><li class="loa__item">Swanton C. </li><li class="loa__item">Albiges L. </li><li class="loa__item">Schmidinger M. </li><li class="loa__item">Heng D.Y. </li><li class="loa__item">Larkin J. </li><li class="loa__item">Ficarra V. </li></ul><div class="reference-citations__detials">Renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Dis. Primers.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">17009</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrdp.2017.9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28276433&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85014927523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (370)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=17009&journal=Nat.+Rev.+Dis.+Primers&author=Hsieh+J.J.&author=Purdue+M.P.&author=Signoretti+S.&author=Swanton+C.&author=Albiges+L.&author=Schmidinger+M.&author=Heng+D.Y.&author=Larkin+J.&author=Ficarra+V.&title=Renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To understand the underlying molecular alterations that drive ccRCC oncogenesis, The Cancer Genome Atlas (TCGA) has performed extensive genomic, epigenomic, and transcriptomic profiling, identifying discriminating features of ccRCC that include loss of various tumor suppressor genes (<div class="dropBlock reference-citations"><a href="#" id="back-bib31" data-db-target-for="bib31-d218962e1884" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e1884" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib141" data-db-target-for="bib141-d218962e1884" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Ricketts et al., 2018</a><div data-db-target-of="bib141-d218962e1884" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ricketts C.J. </li><li class="loa__item">De Cubas A.A. </li><li class="loa__item">Fan H. </li><li class="loa__item">Smith C.C. </li><li class="loa__item">Lang M. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Bowlby R. </li><li class="loa__item">Gibb E.A. </li><li class="loa__item">Akbani R. </li><li class="loa__item">Beroukhim R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Cancer Genome Atlas Comprehensive Molecular Characterization of Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">313-326</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29617669&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044860979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=313-326&journal=Cell+Rep.&author=Cancer+Genome+Atlas+Research+Network&title=The+Cancer+Genome+Atlas+Comprehensive+Molecular+Characterization+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Aberrant dysregulation of the <em>VHL</em> gene is a nearly universal founding event. Subsequent genomic alterations involving <em>PBRM1</em>, <em>SETD2</em>, <em>KDM5C</em>, or <em>BAP1</em> are required for disease progression and are associated with aggressive phenotypes (<div class="dropBlock reference-citations"><a href="#" id="back-bib197" data-db-target-for="bib197-d218962e1898" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib197">Hakimi et al., 2013</a><div data-db-target-of="bib197-d218962e1898" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hakimi A.A. </li><li class="loa__item">Pham C.G. </li><li class="loa__item">Hsieh J.J. </li></ul><div class="reference-citations__detials">A clear picture of renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">849-850</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.2708&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23892664&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84881016018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=849-850&journal=Nat.+Genet.&author=Hakimi+A.A.&author=Pham+C.G.&author=Hsieh+J.J.&title=A+clear+picture+of+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib198" data-db-target-for="bib198-d218962e1898" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib198">Kapur et al., 2013</a><div data-db-target-of="bib198-d218962e1898" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kapur P. </li><li class="loa__item">Peña-Llopis S. </li><li class="loa__item">Christie A. </li><li class="loa__item">Zhrebker L. </li><li class="loa__item">Pavía-Jiménez A. </li><li class="loa__item">Rathmell W.K. </li><li class="loa__item">Xie X.J. </li><li class="loa__item">Brugarolas J. </li></ul><div class="reference-citations__detials">Effects on survival of BAP1 and PBRM1 mutations in sporadic clear-cell renal-cell carcinoma: A retrospective analysis with independent validation.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">159-167</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23333114&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84873081430&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=159-167&journal=Lancet+Oncol.&author=Kapur+P.&author=Pe%C3%B1a-Llopis+S.&author=Christie+A.&author=Zhrebker+L.&author=Pav%C3%ADa-Jim%C3%A9nez+A.&author=Rathmell+W.K.&author=Xie+X.J.&author=Brugarolas+J.&title=Effects+on+survival+of+BAP1+and+PBRM1+mutations+in+sporadic+clear-cell+renal-cell+carcinoma%3A+A+retrospective+analysis+with+independent+validation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These studies have highlighted the value of molecular characterization, in addition to histological assessment, to stratify ccRCC patients, while identifying genomic features unique to ccRCC tumorigenesis (<div class="dropBlock reference-citations"><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d218962e1901" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Chen et al., 2016a</a><div data-db-target-of="bib25-d218962e1901" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen F. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Şenbabaoğlu Y. </li><li class="loa__item">Ciriello G. </li><li class="loa__item">Yang L. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Shuch B. </li><li class="loa__item">Micevic G. </li><li class="loa__item">De Velasco G. </li><li class="loa__item">Shinbrot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Multilevel Genomics-Based Taxonomy of Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2476-2489</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26947078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2476-2489&journal=Cell+Rep.&author=Chen+F.&author=Zhang+Y.&author=%C5%9Eenbabao%C4%9Flu+Y.&author=Ciriello+G.&author=Yang+L.&author=Reznik+E.&author=Shuch+B.&author=Micevic+G.&author=De+Velasco+G.&author=Shinbrot+E.&title=Multilevel+Genomics-Based+Taxonomy+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Historically, ccRCC has been considered resistant to conventional chemotherapy and radiotherapy, with surgical resection as the primary treatment for localized tumors (<div class="dropBlock reference-citations"><a href="#" id="back-bib16" data-db-target-for="bib16-d218962e1906" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Blanco et al., 2011</a><div data-db-target-of="bib16-d218962e1906" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blanco A.I. </li><li class="loa__item">Teh B.S. </li><li class="loa__item">Amato R.J. </li></ul><div class="reference-citations__detials">Role of radiation therapy in the management of renal cell cancer.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">4010-4023</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fcancers3044010&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24213122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84055193229&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=4010-4023&journal=Cancers+%28Basel%29&author=Blanco+A.I.&author=Teh+B.S.&author=Amato+R.J.&title=Role+of+radiation+therapy+in+the+management+of+renal+cell+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib36" data-db-target-for="bib36-d218962e1906" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Diamond et al., 2015</a><div data-db-target-of="bib36-d218962e1906" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Diamond E. </li><li class="loa__item">Molina A.M. </li><li class="loa__item">Carbonaro M. </li><li class="loa__item">Akhtar N.H. </li><li class="loa__item">Giannakakou P. </li><li class="loa__item">Tagawa S.T. </li><li class="loa__item">Nanus D.M. </li></ul><div class="reference-citations__detials">Cytotoxic chemotherapy in the treatment of advanced renal cell carcinoma in the era of targeted therapy.<div class="popup-ref"><span class="ref__series"><i>Crit. Rev. Oncol. Hematol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">96</span></span>: <span class="ref__seriesPages">518-526</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.critrevonc.2015.08.007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26321263&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2015&pages=518-526&journal=Crit.+Rev.+Oncol.+Hematol.&author=Diamond+E.&author=Molina+A.M.&author=Carbonaro+M.&author=Akhtar+N.H.&author=Giannakakou+P.&author=Tagawa+S.T.&author=Nanus+D.M.&title=Cytotoxic+chemotherapy+in+the+treatment+of+advanced+renal+cell+carcinoma+in+the+era+of+targeted+therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Despite several Food and Drug Administration (FDA)-approved agents that target cellular pathways prioritized by genomic analyses, response of ccRCC patients to these treatments has been limited (<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d218962e1909" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Hsieh et al., 2018a</a><div data-db-target-of="bib65-d218962e1909" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hsieh J.J. </li><li class="loa__item">Le V.H. </li><li class="loa__item">Oyama T. </li><li class="loa__item">Ricketts C.J. </li><li class="loa__item">Ho T.H. </li><li class="loa__item">Cheng E.H. </li></ul><div class="reference-citations__detials">Chromosome 3p Loss-Orchestrated VHL, HIF, and Epigenetic Deregulation in Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">36</span> (<span class="refComment">JCO2018792549</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2018.79.2549&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30372397&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85058568679&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&journal=J.%C2%A0Clin.+Oncol.&author=Hsieh+J.J.&author=Le+V.H.&author=Oyama+T.&author=Ricketts+C.J.&author=Ho+T.H.&author=Cheng+E.H.&title=Chromosome+3p+Loss-Orchestrated+VHL%2C+HIF%2C+and+Epigenetic+Deregulation+in+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These results illustrate the complexity of tumorigenesis processes and suggest that genomic, epigenomic, and transcriptomic profiling alone may be insufficient to interrogate this cancer type fully for identifying effective curative treatments. In this study, the Clinical Proteomics Tumor Analysis Consortium (CPTAC) has performed a comprehensive proteogenomic characterization of treatment-naive tumors and paired normal adjacent tissues (NATs) to elucidate the impact of genomic alterations driving phenotypic perturbations and to delineate the mechanisms of ccRCC pathobiology for prospective exploration of personalized, precision-based clinical care.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic Analyses of Tumor and NAT Specimens</span></h3><div class="section-paragraph">In this study, 110 treatment-naive RCC and 84 paired-matched NAT samples were analyzed using a proteogenomic approach wherein each tissue was homogenized via cryopulverization and aliquoted to facilitate genomic, transcriptomic, and proteomic analyses on the same tissue sample (<a href="#sec4" class="scroll-into-link">STAR Methods</a>). Patient characteristics, including age, gender, race, and tumor grade and stage, were recorded for all cases and summarized in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>. Proteomics and phosphoproteomics analyses identified a total of 11,355 proteins and 42,889 phosphopeptides, respectively, of which 7,150 proteins and 20,976 phosphopeptides were quantified across all samples (<a href="#sec4" class="scroll-into-link">STAR Methods</a>). To enable multi-omics data integration and proteogenomic analysis, whole genome sequencing (WGS), whole exome sequencing (WES), and total RNA sequencing (RNA-seq) were performed for all 110 tumor samples, while 107 tumor samples had quality DNA methylation profiling data (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>). NAT samples with mRNA of sufficient quality were subjected to total RNA-seq (n = 75). One NAT sample that displayed discordant proteogenomic profiles was found to contain significant histological evidence of tumor tissue and was excluded from downstream analyses (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>). In addition to the initial pathological diagnosis, we leveraged the molecular information available for RCCs by TCGA and others to verify further the histological classification of tumor samples (<a href="#sec4" class="scroll-into-link">STAR Methods</a>; <div class="dropBlock reference-citations"><a href="#" id="back-bib31" data-db-target-for="bib31-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib32" data-db-target-for="bib32-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Davis et al., 2014</a><div data-db-target-of="bib32-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Davis C.F. </li><li class="loa__item">Ricketts C.J. </li><li class="loa__item">Wang M. </li><li class="loa__item">Yang L. </li><li class="loa__item">Cherniack A.D. </li><li class="loa__item">Shen H. </li><li class="loa__item">Buhay C. </li><li class="loa__item">Kang H. </li><li class="loa__item">Kim S.C. </li><li class="loa__item">Fahey C.C. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>The Cancer Genome Atlas Research Network</span><br />The somatic genomic landscape of chromophobe renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">319-330</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25155756&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84908695210&amp;cf=" onclick="newWindow(this.href);return false">Scopus (358)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=319-330&journal=Cancer+Cell&author=The+Cancer+Genome+Atlas+Research+Network&title=The+somatic+genomic+landscape+of+chromophobe+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib205" data-db-target-for="bib205-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib205">Malta et al., 2018</a><div data-db-target-of="bib205-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kaminska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation Article Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kaminska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation+Article+Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib206" data-db-target-for="bib206-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib206">Mehra et al., 2016</a><div data-db-target-of="bib206-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mehra R. </li><li class="loa__item">Vats P. </li><li class="loa__item">Cieslik M. </li><li class="loa__item">Cao X. </li><li class="loa__item">Su F. </li><li class="loa__item">Shukla S. </li><li class="loa__item">Udager A.M. </li><li class="loa__item">Wang R. </li><li class="loa__item">Pan J. </li><li class="loa__item">Kasaian K. </li><li>et al.</li></ul><div class="reference-citations__detials">Biallelic Alteration and Dysregulation of the Hippo Pathway in Mucinous Tubular and Spindle Cell Carcinoma of the Kidney.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1258-1266</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-16-0267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27604489&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=1258-1266&journal=Cancer+Discov.&author=Mehra+R.&author=Vats+P.&author=Cieslik+M.&author=Cao+X.&author=Su+F.&author=Shukla+S.&author=Udager+A.M.&author=Wang+R.&author=Pan+J.&author=Kasaian+K.&title=Biallelic+Alteration+and+Dysregulation+of+the+Hippo+Pathway+in+Mucinous+Tubular+and+Spindle+Cell+Carcinoma+of+the+Kidney">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib102" data-db-target-for="bib102-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Mehra et al., 2018</a><div data-db-target-of="bib102-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mehra R. </li><li class="loa__item">Vats P. </li><li class="loa__item">Cao X. </li><li class="loa__item">Su F. </li><li class="loa__item">Lee N.D. </li><li class="loa__item">Lonigro R. </li><li class="loa__item">Premkumar K. </li><li class="loa__item">Trpkov K. </li><li class="loa__item">McKenney J.K. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Chinnaiyan A.M. </li></ul><div class="reference-citations__detials">Somatic Bi-allelic Loss of TSC Genes in Eosinophilic Solid and Cystic Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Eur. Urol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">483-486</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=abstract&amp;site=eururo-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=fulltext&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=pdf&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29941307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048882704&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=483-486&journal=Eur.+Urol.&author=Mehra+R.&author=Vats+P.&author=Cao+X.&author=Su+F.&author=Lee+N.D.&author=Lonigro+R.&author=Premkumar+K.&author=Trpkov+K.&author=McKenney+J.K.&author=Dhanasekaran+S.M.&author=Chinnaiyan+A.M.&title=Somatic+Bi-allelic+Loss+of+TSC+Genes+in+Eosinophilic+Solid+and+Cystic+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib94" data-db-target-for="bib94-d218962e1946" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib94">Linehan et al., 2016</a><div data-db-target-of="bib94-d218962e1946" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linehan W.M. </li><li class="loa__item">Spellman P.T. </li><li class="loa__item">Ricketts C.J. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">Fei S.S. </li><li class="loa__item">Davis C. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Murray B.A. </li><li class="loa__item">Schmidt L. </li><li class="loa__item">Vocke C.D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive Molecular Characterization of Papillary Renal-Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">374</span></span>: <span class="ref__seriesPages">135-145</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa1505917&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26536169&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84954286974&amp;cf=" onclick="newWindow(this.href);return false">Scopus (480)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=135-145&journal=N.%C2%A0Engl.+J.+Med.&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+Molecular+Characterization+of+Papillary+Renal-Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Sample-wise assessment of genomic profiles identified seven tumors with molecular aberrations atypical for ccRCC, such as lacking the characteristic bi-allelic loss of tumor suppressor genes on 3p (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>B–S1D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>). While these seven non-ccRCC samples and their corresponding NATs (n = 3) were excluded from most subsequent analyses, the non-ccRCC samples served as useful controls to highlight ccRCC-specific features. Overall, data from 103 ccRCC and 80 NAT tissue samples (with RNA-seq profiles available for 72 samples) were examined for comprehensive proteogenomic characterization (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>).<div class="floatDisplay"><figure id="figs1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/73a1da85-22bb-42f2-b5de-4454330575ec/figs1.jpg" alt="Figure thumbnail figs1" data-locator="figs1" data-high-res="/cms/attachment/f8fe33f5-9eb3-4daa-b552-b43da23acddf/figs1_lrg.jpg" data-large-image="/cms/attachment/73a1da85-22bb-42f2-b5de-4454330575ec/figs1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S1" class="label figure__label">Figure S1</span><span class="figure__title__text">Impact of Genomic Alterations on Transcriptomic and Proteomic Abundances, Related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2670" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) The number of RCC and NAT samples subjected to “WGS” (whole genome sequencing), “WES” (whole exome sequencing), “DNA Methylation,” “RNA-seq,” “Proteomics,” and “Phosphoproteomics” analyses. The presented numbers reflect the number of samples processed that passed data quality control (QC) metrics. (B) Principal Component Analysis (PCA) of RNA-seq data for all CPTAC tumors (n = 110). Identified outliers are annotated by color coding (right). (C) Outlier samples were further investigated for copy number variations (WGS) and somatic mutations (WES). (D) B-allele frequency plots (derived from WES (black dots) and WGS (colored)) for two samples indicate loss of heterozygosity by uniparental disomy for chr 9 and 10 for two index samples. (E) Frequency of <em>SETD2</em> mutation distribution across tumors without translocation events and tumors with t(3:2) (ch3/2), t(3:5) (ch3/5), and other (ch3/O) translocation events. (F) SETD2 protein abundance distribution across tumors without translocation events and tumors with t(3:2) (ch3/2), t(3:5) (ch3/5), and other (ch3/O) translocation events. (G) Co-occurrence of <em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, and <em>KDM5C</em> gene mutations in ccRCC tumors (n = 103). (H) Heatmap of 14 CpG probe beta-values associated with the CpG island methylator phenotype (CIMP) profile of ccRCC tumors. CIMP status and Stage are annotated (top).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/73a1da85-22bb-42f2-b5de-4454330575ec/figs1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/f8fe33f5-9eb3-4daa-b552-b43da23acddf/figs1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic Landscape of the CPTAC ccRCC Cohort</span></h3><div class="section-paragraph">Our study represents a large WGS analysis of ccRCC, revealing arm-level loss of chromosome 3p as the most frequent event (93%), followed by chromosome 5q gain (54%), chromosome 14q loss (42%), chromosome 7 gain (34%), and chromosome 9 loss (21%) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>). Strikingly, we observed fourteen tumors in our cohort displayed extensive CNVs across all chromosomes, indicating a high degree of genomic instability. A molecular subset with these characteristics was not identified in the initial TCGA ccRCC study, possibly due to the limited number of tumors examined via WGS (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e1971" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e1971" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Moreover, a recent pan-cancer analysis of three common RCC subtypes and studies investigating ccRCC tumor ploidy via an intra-tumor heterogeneity approach showed a significant association between high genome aneuploidy and poor prognosis (<div class="dropBlock reference-citations"><a href="#" id="back-bib25" data-db-target-for="bib25-d218962e1974" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Chen et al., 2016a</a><div data-db-target-of="bib25-d218962e1974" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen F. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Şenbabaoğlu Y. </li><li class="loa__item">Ciriello G. </li><li class="loa__item">Yang L. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Shuch B. </li><li class="loa__item">Micevic G. </li><li class="loa__item">De Velasco G. </li><li class="loa__item">Shinbrot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Multilevel Genomics-Based Taxonomy of Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2476-2489</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26947078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2476-2489&journal=Cell+Rep.&author=Chen+F.&author=Zhang+Y.&author=%C5%9Eenbabao%C4%9Flu+Y.&author=Ciriello+G.&author=Yang+L.&author=Reznik+E.&author=Shuch+B.&author=Micevic+G.&author=De+Velasco+G.&author=Shinbrot+E.&title=Multilevel+Genomics-Based+Taxonomy+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib169" data-db-target-for="bib169-d218962e1974" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib169">Turajlic et al., 2018a</a><div data-db-target-of="bib169-d218962e1974" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Turajlic S. </li><li class="loa__item">Xu H. </li><li class="loa__item">Litchfield K. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Horswell S. </li><li class="loa__item">Chambers T. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Lopez J.I. </li><li class="loa__item">Watkins T.B.K. </li><li class="loa__item">Nicol D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TRACERx Renal Consortium</span><br />Deterministic Evolutionary Trajectories Influence Primary Tumor Growth: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">595-610</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656894&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045306946&amp;cf=" onclick="newWindow(this.href);return false">Scopus (110)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=595-610&journal=Cell&author=TRACERx+Renal+Consortium&title=Deterministic+Evolutionary+Trajectories+Influence+Primary+Tumor+Growth%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib170" data-db-target-for="bib170-d218962e1974" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib170">Turajlic et al., 2018b</a><div data-db-target-of="bib170-d218962e1974" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Turajlic S. </li><li class="loa__item">Xu H. </li><li class="loa__item">Litchfield K. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Chambers T. </li><li class="loa__item">Lopez J.I. </li><li class="loa__item">Nicol D. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Larkin J. </li><li class="loa__item">Horswell S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>PEACE</span><span>TRACERx Renal Consortium</span><br />Tracking Cancer Evolution Reveals Constrained Routes to Metastases: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">581-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656895&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045083551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=581-594&journal=Cell&author=PEACETRACERx+Renal+Consortium&title=Tracking+Cancer+Evolution+Reveals+Constrained+Routes+to+Metastases%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which is consistent with the enrichment of genomically instable high grade tumors identified in our study (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A).<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/a3bd0f3f-59fa-4db9-8d7e-cd69f6793edc/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/48a7f1fd-06f2-49cc-8ec6-da8fff9ea9c1/gr1_lrg.jpg" data-large-image="/cms/attachment/a3bd0f3f-59fa-4db9-8d7e-cd69f6793edc/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Genomic Alterations and their Associations with mRNA, Protein, and Phosphoprotein Abundances</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Profiling of absolute copy number estimates observed in the CPTAC cohort. Genomically defined non-ccRCC tumors are above ccRCC tumors; translocation event, grade, CpG island methylator phenotype (CIMP) status, genome instability, and CNV loss/gain are indicated by color coding. ccRCC tumors with evidence of 3p loss of heterozygosity (LOH) are indicated by three asterisks (<sup>∗∗∗</sup>).</div><div class="section-paragraph">(B) Circos plots of translocation events involving chromosomes 3 and either chromosomes 5 (red), 2 (blue), 8 (purple) or all other chromosomes (gray), including chromosomal inversion within chromosome 3 (green). Percentage of involved tumors with re-arrangement for each chromosome is annotated below each plot.</div><div class="section-paragraph">(C) Heatmap of multi-omic data for the five key tumor suppressor genes (<em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, and <em>KDM5C</em>) (n = 103). Tumor samples were ordered by 3p CNV alteration (loss to neutral). Non-ccRCC tumors are separated (right). CNV event, <em>Z</em> score, CNV loss/gain, translocation status, CpG island methylator phenotype (CIMP) status, genome instability, grade, and gender are indicated by color coding (bottom).</div><div class="section-paragraph">See also <a data-locator="figs1" aria-expanded="false" aria-controls="crosref2565" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs1">Figure S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2570" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2575" href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" target="_blank">S2</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/a3bd0f3f-59fa-4db9-8d7e-cd69f6793edc/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/48a7f1fd-06f2-49cc-8ec6-da8fff9ea9c1/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">A previous report identified chromosome translocation as a mechanism of concurrent 3p loss and 5q gain in ccRCC (<div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d218962e1982" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Mitchell et al., 2018</a><div data-db-target-of="bib109-d218962e1982" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mitchell T.J. </li><li class="loa__item">Turajlic S. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Nicol D. </li><li class="loa__item">Farmery J.H.R. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Martincorena I. </li><li class="loa__item">Tarpey P. </li><li class="loa__item">Angelopoulos N. </li><li class="loa__item">Yates L.R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TRACERx Renal Consortium</span><br />Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">611-623</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044780018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=611-623&journal=Cell&author=TRACERx+Renal+Consortium&title=Timing+the+Landmark+Events+in+the+Evolution+of+Clear+Cell+Renal+Cell+Cancer%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Utilizing the WGS data in our study, we investigated the frequency and types of chromosomal rearrangements present in our cohort. We observed that 61% of ccRCC cases displayed one or more translocation events, predominantly involving the chromosome 3p locus and chromosomes 5 (20%), 2 (11%), and 8 (7%) among others (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>). The novel chromosome t(3:2) rearrangement, largely observed in cases with concurrent 3p loss and 2q gain, was the second most common translocation event and was nearly mutually exclusive with t(3:5) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). We also detected chromosomal inversion within chromosome 3 (n = 2) that resulted in concurrent 3q gain and 3p loss (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). A previous study suggested chromothripsis as a likely mechanism of concurrent 3p loss and 5q gain in ccRCC (<div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d218962e1998" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Mitchell et al., 2018</a><div data-db-target-of="bib109-d218962e1998" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mitchell T.J. </li><li class="loa__item">Turajlic S. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Nicol D. </li><li class="loa__item">Farmery J.H.R. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Martincorena I. </li><li class="loa__item">Tarpey P. </li><li class="loa__item">Angelopoulos N. </li><li class="loa__item">Yates L.R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TRACERx Renal Consortium</span><br />Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">611-623</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044780018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=611-623&journal=Cell&author=TRACERx+Renal+Consortium&title=Timing+the+Landmark+Events+in+the+Evolution+of+Clear+Cell+Renal+Cell+Cancer%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); we noted a similar genomic feature of oscillating copy number patterns near the breakpoint (data not shown). In addition, 3p loss of heterozygosity (LOH) was observed in six tumors (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>). In total, 102 of 103 ccRCC tumors in the CPTAC cohort showed evidence of 3p dysregulation.</div><div class="section-paragraph">The profile of somatic mutations in the CPTAC ccRCC cohort was reflective of that previously described by TCGA (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e2006" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e2006" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Dysregulation of <em>VHL</em> was the most frequent alteration and was observed in 85% of tumors. <em>PBRM1</em>, <em>BAP1</em>, <em>KDM5C</em>, and <em>SETD2</em> followed with mutation rates of 43%, 17%, 18%, and 16%, respectively (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C). We further examined the impact of mutation and methylation of these genes on their respective levels of mRNA and protein. Overall, inactivating genomic events resulted in reduced expression of mRNA and protein (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C), indicating loss-of-function and supporting the classification of these genes as ccRCC tumor suppressors (<div class="dropBlock reference-citations"><a href="#" id="back-bib46" data-db-target-for="bib46-d218962e2026" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Frew and Moch, 2015</a><div data-db-target-of="bib46-d218962e2026" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Frew I.J. </li><li class="loa__item">Moch H. </li></ul><div class="reference-citations__detials">A clearer view of the molecular complexity of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Annu. Rev. Pathol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">263-289</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1146%2Fannurev-pathol-012414-040306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25387056&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84921913287&amp;cf=" onclick="newWindow(this.href);return false">Scopus (97)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=263-289&journal=Annu.+Rev.+Pathol.&author=Frew+I.J.&author=Moch+H.&title=A+clearer+view+of+the+molecular+complexity+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib88" data-db-target-for="bib88-d218962e2026" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib88">Liao et al., 2015</a><div data-db-target-of="bib88-d218962e2026" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao L. </li><li class="loa__item">Testa J.R. </li><li class="loa__item">Yang H. </li></ul><div class="reference-citations__detials">The roles of chromatin-remodelers and epigenetic modifiers in kidney cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Genet.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">208</span></span>: <span class="ref__seriesPages">206-214</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=abstract&amp;site=cgen-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=fulltext&amp;site=cgen-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=pdf&amp;site=cgen-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25873528&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2015&pages=206-214&journal=Cancer+Genet.&author=Liao+L.&author=Testa+J.R.&author=Yang+H.&title=The+roles+of+chromatin-remodelers+and+epigenetic+modifiers+in+kidney+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Unique to SETD2 was the relationship of gene inactivation due to t(3:2), with a higher frequency of mutations and reduced protein expression relative to other translocation events involving chromosome 3p (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>E and S1F). Investigation into the rate of mutation co-occurrence revealed that <em>VHL</em> mutations were associated with the presence of other mutations (<em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, and <em>KDM5C</em>), while <em>PBRM1</em> mutations were associated with mutation of <em>SETD2</em> and <em>KDM5C</em> but not <em>BAP1</em> (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>G). These results support a model of branched evolution in ccRCC and the largely mutual exclusivity of somatic mutations involving <em>PBRM1</em> and <em>BAP1</em> (<div class="dropBlock reference-citations"><a href="#" id="back-bib50" data-db-target-for="bib50-d218962e2059" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Gerlinger et al., 2014</a><div data-db-target-of="bib50-d218962e2059" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gerlinger M. </li><li class="loa__item">Horswell S. </li><li class="loa__item">Larkin J. </li><li class="loa__item">Rowan A.J. </li><li class="loa__item">Salm M.P. </li><li class="loa__item">Varela I. </li><li class="loa__item">Fisher R. </li><li class="loa__item">McGranahan N. </li><li class="loa__item">Matthews N. </li><li class="loa__item">Santos C.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Genomic architecture and evolution of clear cell renal cell carcinomas defined by multiregion sequencing.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">225-233</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.2891&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24487277&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84895876130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (690)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=225-233&journal=Nat.+Genet.&author=Gerlinger+M.&author=Horswell+S.&author=Larkin+J.&author=Rowan+A.J.&author=Salm+M.P.&author=Varela+I.&author=Fisher+R.&author=McGranahan+N.&author=Matthews+N.&author=Santos+C.R.&title=Genomic+architecture+and+evolution+of+clear+cell+renal+cell+carcinomas+defined+by+multiregion+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib169" data-db-target-for="bib169-d218962e2059" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib169">Turajlic et al., 2018a</a><div data-db-target-of="bib169-d218962e2059" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Turajlic S. </li><li class="loa__item">Xu H. </li><li class="loa__item">Litchfield K. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Horswell S. </li><li class="loa__item">Chambers T. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Lopez J.I. </li><li class="loa__item">Watkins T.B.K. </li><li class="loa__item">Nicol D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TRACERx Renal Consortium</span><br />Deterministic Evolutionary Trajectories Influence Primary Tumor Growth: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">595-610</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656894&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045306946&amp;cf=" onclick="newWindow(this.href);return false">Scopus (110)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=595-610&journal=Cell&author=TRACERx+Renal+Consortium&title=Deterministic+Evolutionary+Trajectories+Influence+Primary+Tumor+Growth%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Increased methylation of promoter CpG islands contributes to an oncogenic phenotype (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d218962e2064" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Baylin and Jones, 2011</a><div data-db-target-of="bib11-d218962e2064" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Baylin S.B. </li><li class="loa__item">Jones P.A. </li></ul><div class="reference-citations__detials">A decade of exploring the cancer epigenome - biological and translational implications.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">726-734</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrc3130&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21941284&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80053144962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1619)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=726-734&journal=Nat.+Rev.+Cancer&author=Baylin+S.B.&author=Jones+P.A.&title=A+decade+of+exploring+the+cancer+epigenome+-+biological+and+translational+implications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Querying DNA methylation status of CpG positions with a previously reported CpG island methylator phenotype (CIMP) marker panel specific for ccRCC (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d218962e2067" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Arai et al., 2012</a><div data-db-target-of="bib4-d218962e2067" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Arai E. </li><li class="loa__item">Chiku S. </li><li class="loa__item">Mori T. </li><li class="loa__item">Gotoh M. </li><li class="loa__item">Nakagawa T. </li><li class="loa__item">Fujimoto H. </li><li class="loa__item">Kanai Y. </li></ul><div class="reference-citations__detials">Single-CpG-resolution methylome analysis identifies clinicopathologically aggressive CpG island methylator phenotype clear cell renal cell carcinomas.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1487-1493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fcarcin%2Fbgs177&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22610075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865339838&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=1487-1493&journal=Carcinogenesis&author=Arai+E.&author=Chiku+S.&author=Mori+T.&author=Gotoh+M.&author=Nakagawa+T.&author=Fujimoto+H.&author=Kanai+Y.&title=Single-CpG-resolution+methylome+analysis+identifies+clinicopathologically+aggressive+CpG+island+methylator+phenotype+clear+cell+renal+cell+carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we classified tumors in our cohort into CIMP<sup>−</sup> (negative) and CIMP<sup>+</sup> (positive) categories (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>H). We identified 36 tumors (34%) with CIMP<sup>+</sup> status, which was associated with higher grade (p &lt; 9.0 e−05) and stage (p &lt; 0.001), and higher frequency of genomic instability (p &lt; 0.004) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>).</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Integrated Proteogenomic Analyses of Genomic Alterations</span></h3><div class="section-paragraph">Genomic alterations can impact mRNA and protein abundance at the same locus (<em>cis</em>-effects), as well as other loci (<em>trans</em>-effects). Through integration of mRNA, protein, and phosphopeptide levels, we identified genomic alterations preserved through transcriptional, translational, and post-translational levels (<em>cis</em>-effect cascades), prioritizing gene targets associated with tumor grade and differential abundance between tumors and NATs (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A, <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>A, and S2B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>). Examples of identified CNV <em>cis</em>-cascades included <em>SQSTM1</em> (5q35.3), <em>OSBPL3</em> (7p15.3), and <em>GOLPH3</em> (5q13.3), previously associated with PI3K-mTOR signaling (<div class="dropBlock reference-citations"><a href="#" id="back-bib31" data-db-target-for="bib31-d218962e2121" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e2121" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib86" data-db-target-for="bib86-d218962e2121" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Lehto and Olkkonen, 2003</a><div data-db-target-of="bib86-d218962e2121" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lehto M. </li><li class="loa__item">Olkkonen V.M. </li></ul><div class="reference-citations__detials">The OSBP-related proteins: a novel protein family involved in vesicle transport, cellular lipid metabolism, and cell signalling.<div class="popup-ref"><span class="ref__series"><i>Biochim. Biophys. Acta.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">1631</span></span>: <span class="ref__seriesPages">1-11</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1388-1981%2802%2900364-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12573443&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0037455984&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1631&publication_year=2003&pages=1-11&journal=Biochim.+Biophys.+Acta&author=Lehto+M.&author=Olkkonen+V.M.&title=The+OSBP-related+proteins%3A+a+novel+protein+family+involved+in+vesicle+transport%2C+cellular+lipid+metabolism%2C+and+cell+signalling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib148" data-db-target-for="bib148-d218962e2121" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib148">Scott et al., 2009</a><div data-db-target-of="bib148-d218962e2121" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Scott K.L. </li><li class="loa__item">Kabbarah O. </li><li class="loa__item">Liang M.C. </li><li class="loa__item">Ivanova E. </li><li class="loa__item">Anagnostou V. </li><li class="loa__item">Wu J. </li><li class="loa__item">Dhakal S. </li><li class="loa__item">Wu M. </li><li class="loa__item">Chen S. </li><li class="loa__item">Feinberg T. </li><li>et al.</li></ul><div class="reference-citations__detials">GOLPH3 modulates mTOR signalling and rapamycin sensitivity in cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">459</span></span>: <span class="ref__seriesPages">1085-1090</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature08109&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19553991&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-67649500009&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=1085-1090&journal=Nature&author=Scott+K.L.&author=Kabbarah+O.&author=Liang+M.C.&author=Ivanova+E.&author=Anagnostou+V.&author=Wu+J.&author=Dhakal+S.&author=Wu+M.&author=Chen+S.&author=Feinberg+T.&title=GOLPH3+modulates+mTOR+signalling+and+rapamycin+sensitivity+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We also identified a <em>cis</em>-cascade involving the multifunctional transcription factor <em>YY1</em> (14q32.2), which is a stabilizer of HIF-1α, co-repressor of HIF-2α activity, and inhibitor of <em>MYC</em> function (<div class="dropBlock reference-citations"><a href="#" id="back-bib9" data-db-target-for="bib9-d218962e2130" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Austen et al., 1998</a><div data-db-target-of="bib9-d218962e2130" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Austen M. </li><li class="loa__item">Cerni C. </li><li class="loa__item">Lüscher-Firzlaff J.M. </li><li class="loa__item">Lüscher B. </li></ul><div class="reference-citations__detials">YY1 can inhibit c-Myc function through a mechanism requiring DNA binding of YY1 but neither its transactivation domain nor direct interaction with c-Myc.<div class="popup-ref"><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">511-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.onc.1201968&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=9696045&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=511-520&journal=Oncogene&author=Austen+M.&author=Cerni+C.&author=L%C3%BCscher-Firzlaff+J.M.&author=L%C3%BCscher+B.&title=YY1+can+inhibit+c-Myc+function+through+a+mechanism+requiring+DNA+binding+of+YY1+but+neither+its+transactivation+domain+nor+direct+interaction+with+c-Myc">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib135" data-db-target-for="bib135-d218962e2130" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Petrella and Brinckerhoff, 2009</a><div data-db-target-of="bib135-d218962e2130" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petrella B.L. </li><li class="loa__item">Brinckerhoff C.E. </li></ul><div class="reference-citations__detials">PTEN suppression of YY1 induces HIF-2α activity in von-Hippel-Lindau-null renal-cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Biol. Ther.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1389-1401</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.4161%2Fcbt.8.14.8880&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19483472&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-76249096569&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1389-1401&journal=Cancer+Biol.+Ther.&author=Petrella+B.L.&author=Brinckerhoff+C.E.&title=PTEN+suppression+of+YY1+induces+HIF-2%CE%B1+activity+in+von-Hippel-Lindau-null+renal-cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib183" data-db-target-for="bib183-d218962e2130" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib183">Wu et al., 2013</a><div data-db-target-of="bib183-d218962e2130" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wu S. </li><li class="loa__item">Kasim V. </li><li class="loa__item">Kano M.R. </li><li class="loa__item">Tanaka S. </li><li class="loa__item">Ohba S. </li><li class="loa__item">Miura Y. </li><li class="loa__item">Miyata K. </li><li class="loa__item">Liu X. </li><li class="loa__item">Matsuhashi A. </li><li class="loa__item">Chung U.I. </li><li>et al.</li></ul><div class="reference-citations__detials">Transcription factor YY1 contributes to tumor growth by stabilizing hypoxia factor HIF-1α in a p53-independent manner.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">1787-1799</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-12-0366&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23328582&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84875473756&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1787-1799&journal=Cancer+Res.&author=Wu+S.&author=Kasim+V.&author=Kano+M.R.&author=Tanaka+S.&author=Ohba+S.&author=Miura+Y.&author=Miyata+K.&author=Liu+X.&author=Matsuhashi+A.&author=Chung+U.I.&title=Transcription+factor+YY1+contributes+to+tumor+growth+by+stabilizing+hypoxia+factor+HIF-1%CE%B1+in+a+p53-independent+manner">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The latter observation suggests a mechanism whereby <em>YY1</em> loss links the HIF-2α signaling with MYC expression in ccRCC (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d218962e2136" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Gordan et al., 2008</a><div data-db-target-of="bib54-d218962e2136" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gordan J.D. </li><li class="loa__item">Lal P. </li><li class="loa__item">Dondeti V.R. </li><li class="loa__item">Letrero R. </li><li class="loa__item">Parekh K.N. </li><li class="loa__item">Oquendo C.E. </li><li class="loa__item">Greenberg R.A. </li><li class="loa__item">Flaherty K.T. </li><li class="loa__item">Rathmell W.K. </li><li class="loa__item">Keith B. </li><li>et al.</li></ul><div class="reference-citations__detials">HIF-α effects on c-Myc distinguish two subtypes of sporadic VHL-deficient clear cell renal carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">435-446</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19061835&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-56849096837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=435-446&journal=Cancer+Cell&author=Gordan+J.D.&author=Lal+P.&author=Dondeti+V.R.&author=Letrero+R.&author=Parekh+K.N.&author=Oquendo+C.E.&author=Greenberg+R.A.&author=Flaherty+K.T.&author=Rathmell+W.K.&author=Keith+B.&title=HIF-%CE%B1+effects+on+c-Myc+distinguish+two+subtypes+of+sporadic+VHL-deficient+clear+cell+renal+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Investigation into methylation events proximal to <em>VHL</em> identified three genes, <em>VGLL4</em> (3p25.2), <em>PLCL2</em> (3p24.3), and <em>IQSEC1</em> (3p25.2) with associated <em>cis</em>-regulated effects (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C). Methylation of <em>PLCL2</em> and <em>IQSEC1</em> has been noted previously (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d218962e2157" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">Dmitriev et al., 2014</a><div data-db-target-of="bib37-d218962e2157" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dmitriev A.A. </li><li class="loa__item">Rudenko E.E. </li><li class="loa__item">Kudryavtseva A.V. </li><li class="loa__item">Krasnov G.S. </li><li class="loa__item">Gordiyuk V.V. </li><li class="loa__item">Melnikova N.V. </li><li class="loa__item">Stakhovsky E.O. </li><li class="loa__item">Kononenko O.A. </li><li class="loa__item">Pavlova L.S. </li><li class="loa__item">Kondratieva T.T. </li><li>et al.</li></ul><div class="reference-citations__detials">Epigenetic alterations of chromosome 3 revealed by NotI-microarrays in clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>BioMed Res. Int.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">2014</span></span>: <span class="ref__seriesPages">735292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1155%2F2014%2F735292&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24977159&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84903597797&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=735292&journal=BioMed+Res.+Int.&author=Dmitriev+A.A.&author=Rudenko+E.E.&author=Kudryavtseva+A.V.&author=Krasnov+G.S.&author=Gordiyuk+V.V.&author=Melnikova+N.V.&author=Stakhovsky+E.O.&author=Kononenko+O.A.&author=Pavlova+L.S.&author=Kondratieva+T.T.&title=Epigenetic+alterations+of+chromosome+3+revealed+by+NotI-microarrays+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); however, methylation of <em>VGLL4</em> has not been reported. Its functional role inhibiting YAP-induced cell proliferation (<div class="dropBlock reference-citations"><a href="#bib189" id="back-bib189" data-db-target-for="bib189-d218962e2162" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib189">Zhang et al., 2014b</a><div data-db-target-of="bib189-d218962e2162" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang W. </li><li class="loa__item">Gao Y. </li><li class="loa__item">Li P. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Guo T. </li><li class="loa__item">Li F. </li><li class="loa__item">Han X. </li><li class="loa__item">Feng Y. </li><li class="loa__item">Zheng C. </li><li class="loa__item">Wang Z. </li><li>et al.</li></ul><div class="reference-citations__detials">VGLL4 functions as a new tumor suppressor in lung cancer by negatively regulating the YAP-TEAD transcriptional complex.<div class="popup-ref"><span class="ref__series"><i>Cell Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">331-343</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fcr.2014.10&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24458094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84897584171&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=331-343&journal=Cell+Res.&author=Zhang+W.&author=Gao+Y.&author=Li+P.&author=Shi+Z.&author=Guo+T.&author=Li+F.&author=Han+X.&author=Feng+Y.&author=Zheng+C.&author=Wang+Z.&title=VGLL4+functions+as+a+new+tumor+suppressor+in+lung+cancer+by+negatively+regulating+the+YAP-TEAD+transcriptional+complex">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) may represent another tumor suppressor gene in ccRCC. Interestingly, unique to <em>IQSEC1</em>, we found that gene methylation was associated with reduced protein and phosphorylation levels, but not mRNA. This distinctive feature of methylation affecting protein but not mRNA expression was also observed in several other genes, including <em>BCL9L</em> (11q23.3) and <em>AHDC1</em> (1p35.3) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>), and may indicate a post-translational regulatory mechanism.<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/601f35a6-324c-4aa9-ac4f-e9cb44587cf9/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/26e88ddf-3769-48d8-9a20-df8ad644ed7e/gr2_lrg.jpg" data-large-image="/cms/attachment/601f35a6-324c-4aa9-ac4f-e9cb44587cf9/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">Impact of Copy Number Variation (CNV) on Protein Abundance</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) DNA variations (CNV baf, B-allele frequencies; CNV lr, adjusted log coverage ratios; DNA methylation) with cascading <em>cis</em>-association (associations with all types of mRNA, global protein, and phosphopeptide abundances), overlapped with phosphopeptides significantly differentiated by clinical features (tumor versus NAT and tumor grade). Genes in bold are associated with CNV events involving chromosome 5 or 7 gain and 14 loss.</div><div class="section-paragraph">(B) The <em>cis</em> and <em>trans</em> associations of chromosome arms (3p, 5q, 7p, 9p, and 14q) and CpG island methylator phenotype (CIMP). Significant (adjusted p &lt; 0.1) positive (red) and negative (blue) associations for individual chromosomes (left), summed associations (middle), and corresponding enriched upregulated (red) and downregulated (blue) pathways (adjusted p &lt; 0.05) are annotated (right).</div><div class="section-paragraph">See also <a data-locator="figs2" aria-expanded="false" aria-controls="crosref2580" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a> and <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2585" href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" target="_blank">Table S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/601f35a6-324c-4aa9-ac4f-e9cb44587cf9/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/26e88ddf-3769-48d8-9a20-df8ad644ed7e/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/f50420e5-69ff-47c6-9192-e4a6a31b143b/figs2.jpg" alt="Figure thumbnail figs2" data-locator="figs2" data-high-res="/cms/attachment/42c1d858-7892-4b7b-a239-cb48ca8cc683/figs2_lrg.jpg" data-large-image="/cms/attachment/f50420e5-69ff-47c6-9192-e4a6a31b143b/figs2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S2" class="label figure__label">Figure S2</span><span class="figure__title__text">Impact of Copy Number Variation (CNV) on Protein Abundance, Related to <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2675" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Results of iProFun associations of CNV lr (adjusted log coverage-ratios), CNV baf (B-allele frequencies), and DNA methylation. Average posterior probability was calculated for association of genomic alteration events and mRNA, global protein, and phosphosite abundance. (B) Venn diagram of CNV (lr), CNV (baf), and DNA methylation genomic alterations for mRNA, protein, and phosphosite association overlap. (C) Heatmap of multi-omic data for methylation <em>cis</em>-cascade event (<em>VGLL4</em>, <em>PLCL2</em>) and for DNA methylation not detected at the mRNA level (protein and phosphosite only) (<em>IQSEC1</em>, <em>BCL9L</em>, <em>AHDC1</em>). Tumor samples were ordered by increasing DNA methylation z-score (left to right). (D) Associated enriched upregulated (red) and downregulated (blue) pathways of <em>cis</em>- and <em>trans</em>-effects for five key tumor suppressor genes (<em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, and <em>KDM5C</em>) (adjusted p &lt; 0.05).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/f50420e5-69ff-47c6-9192-e4a6a31b143b/figs2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/42c1d858-7892-4b7b-a239-cb48ca8cc683/figs2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Next, we focused on identifying cellular processes correlated with <em>cis-</em> and <em>trans</em>-effects driven by major CNV or mutation events in ccRCC and identified multiple pathways that were disparate or commonly dysregulated among distinct genomic alterations (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>). Loss of chromosome 3p was associated with upregulation of hypoxic signaling, cell-cycle regulation and glycolysis, downregulation of oxidative phosphorylation (OXPHOS), fatty acid metabolism, and the TCA cycle. Increased expression of HIF-1α and HIF-2α is a mechanism for upregulating hypoxic signaling upon <em>VHL</em> loss (<div class="dropBlock reference-citations"><a href="#" id="back-bib55" data-db-target-for="bib55-d218962e2196" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Guo et al., 2011</a><div data-db-target-of="bib55-d218962e2196" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guo G. </li><li class="loa__item">Gui Y. </li><li class="loa__item">Gao S. </li><li class="loa__item">Tang A. </li><li class="loa__item">Hu X. </li><li class="loa__item">Huang Y. </li><li class="loa__item">Jia W. </li><li class="loa__item">Li Z. </li><li class="loa__item">He M. </li><li class="loa__item">Sun L. </li><li>et al.</li></ul><div class="reference-citations__detials">Frequent mutations of genes encoding ubiquitin-mediated proteolysis pathway components in clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">17-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.1014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22138691&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84655176646&amp;cf=" onclick="newWindow(this.href);return false">Scopus (218)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=17-19&journal=Nat.+Genet.&author=Guo+G.&author=Gui+Y.&author=Gao+S.&author=Tang+A.&author=Hu+X.&author=Huang+Y.&author=Jia+W.&author=Li+Z.&author=He+M.&author=Sun+L.&title=Frequent+mutations+of+genes+encoding+ubiquitin-mediated+proteolysis+pathway+components+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib81" data-db-target-for="bib81-d218962e2196" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">Krieg et al., 2000</a><div data-db-target-of="bib81-d218962e2196" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krieg M. </li><li class="loa__item">Haas R. </li><li class="loa__item">Brauch H. </li><li class="loa__item">Acker T. </li><li class="loa__item">Flamme I. </li><li class="loa__item">Plate K.H. </li></ul><div class="reference-citations__detials">Up-regulation of hypoxia-inducible factors HIF-1α and HIF-2α under normoxic conditions in renal carcinoma cells by von Hippel-Lindau tumor suppressor gene loss of function.<div class="popup-ref"><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">5435-5443</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.onc.1203938&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=11114720&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0034676331&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=5435-5443&journal=Oncogene&author=Krieg+M.&author=Haas+R.&author=Brauch+H.&author=Acker+T.&author=Flamme+I.&author=Plate+K.H.&title=Up-regulation+of+hypoxia-inducible+factors+HIF-1%CE%B1+and+HIF-2%CE%B1+under+normoxic+conditions+in+renal+carcinoma+cells+by+von+Hippel-Lindau+tumor+suppressor+gene+loss+of+function">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In cancer cells, HIF1 signaling alters mitochondrial activity and drives a metabolic shift from OXPHOS to glycolysis (<div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d218962e2199" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Papandreou et al., 2006</a><div data-db-target-of="bib126-d218962e2199" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Papandreou I. </li><li class="loa__item">Cairns R.A. </li><li class="loa__item">Fontana L. </li><li class="loa__item">Lim A.L. </li><li class="loa__item">Denko N.C. </li></ul><div class="reference-citations__detials">HIF-1 mediates adaptation to hypoxia by actively downregulating mitochondrial oxygen consumption.<div class="popup-ref"><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">187-197</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16517406&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33644622570&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1259)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=187-197&journal=Cell+Metab.&author=Papandreou+I.&author=Cairns+R.A.&author=Fontana+L.&author=Lim+A.L.&author=Denko+N.C.&title=HIF-1+mediates+adaptation+to+hypoxia+by+actively+downregulating+mitochondrial+oxygen+consumption">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Investigation of <em>trans</em>-effects involving chromosome 3p genes revealed that <em>VHL</em> mutations resulted in dysregulation of similar pathways as 3p loss including downregulation of metabolic pathways and upregulation of G1/S cell-cycle transition and interferon-α response. <em>PBRM1</em> mutations drove downregulation of OXPHOS and upregulation of G2/M cell-cycle transition, mitogen-activated protein kinase (MAPK) signaling, and focal adhesion pathways. <em>BAP1</em> mutations were associated with upregulation of protein translation pathways and interferon-γ signaling, with the latter feature also associated with <em>SETD2</em> mutations (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>). <em>KD5MC</em> mutations shared a similar <em>trans</em>-effect profile as 3p locus/VHL loss, including downregulation of select metabolic pathways and increased cell-cycle regulation. 5q gain resulted in increased mTORC1 and MYC signaling, while gain of 7p drove increased protein translation and epithelial mesenchymal transition (EMT) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B). 9p loss, which includes the tumor suppressor <em>CDKN2A</em>, was associated with upregulation of translation initiation, mTOR and MYC signaling, consistent with loss of <em>CDKN2A</em> and <em>MYC</em> activation previously reported in ccRCC progression (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d218962e2233" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Bailey et al., 2017</a><div data-db-target-of="bib10-d218962e2233" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bailey S.T. </li><li class="loa__item">Smith A.M. </li><li class="loa__item">Kardos J. </li><li class="loa__item">Wobker S.E. </li><li class="loa__item">Wilson H.L. </li><li class="loa__item">Krishnan B. </li><li class="loa__item">Saito R. </li><li class="loa__item">Lee H.J. </li><li class="loa__item">Zhang J. </li><li class="loa__item">Eaton S.C. </li><li>et al.</li></ul><div class="reference-citations__detials">MYC activation cooperates with Vhl and Ink4a/Arf loss to induce clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">15770</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms15770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28593993&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85020380928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15770&journal=Nat.+Commun.&author=Bailey+S.T.&author=Smith+A.M.&author=Kardos+J.&author=Wobker+S.E.&author=Wilson+H.L.&author=Krishnan+B.&author=Saito+R.&author=Lee+H.J.&author=Zhang+J.&author=Eaton+S.C.&title=MYC+activation+cooperates+with+Vhl+and+Ink4a%2FArf+loss+to+induce+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Loss of 14q, involving the potential tumor suppressors <em>NDRG2</em> (14q11.2) and <em>HIF1A</em> (14q23.2) (<div class="dropBlock reference-citations"><a href="#" id="back-bib96" data-db-target-for="bib96-d218962e2241" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Lusis et al., 2005</a><div data-db-target-of="bib96-d218962e2241" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lusis E.A. </li><li class="loa__item">Watson M.A. </li><li class="loa__item">Chicoine M.R. </li><li class="loa__item">Lyman M. </li><li class="loa__item">Roerig P. </li><li class="loa__item">Reifenberger G. </li><li class="loa__item">Gutmann D.H. </li><li class="loa__item">Perry A. </li></ul><div class="reference-citations__detials">Integrative genomic analysis identifies NDRG2 as a candidate tumor suppressor gene frequently inactivated in clinically aggressive meningioma.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">65</span></span>: <span class="ref__seriesPages">7121-7126</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-05-0043&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16103061&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-23844451672&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=7121-7126&journal=Cancer+Res.&author=Lusis+E.A.&author=Watson+M.A.&author=Chicoine+M.R.&author=Lyman+M.&author=Roerig+P.&author=Reifenberger+G.&author=Gutmann+D.H.&author=Perry+A.&title=Integrative+genomic+analysis+identifies+NDRG2+as+a+candidate+tumor+suppressor+gene+frequently+inactivated+in+clinically+aggressive+meningioma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib152" data-db-target-for="bib152-d218962e2241" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib152">Shen et al., 2011</a><div data-db-target-of="bib152-d218962e2241" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shen C. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Zhou J. </li><li class="loa__item">Chang M. </li><li class="loa__item">Signoretti S. </li><li class="loa__item">Kaelin Jr., W.G. </li></ul><div class="reference-citations__detials">Genetic and functional studies implicate HIF1α as a 14q kidney cancer suppressor gene.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">222-235</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-11-0098&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22037472&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80054771537&amp;cf=" onclick="newWindow(this.href);return false">Scopus (212)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=222-235&journal=Cancer+Discov.&author=Shen+C.&author=Beroukhim+R.&author=Schumacher+S.E.&author=Zhou+J.&author=Chang+M.&author=Signoretti+S.&author=Kaelin+W.G.&title=Genetic+and+functional+studies+implicate+HIF1%CE%B1+as+a+14q+kidney+cancer+suppressor+gene">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), displayed decreased WNT signaling expression and upregulation of MYC signaling, <em>N</em>-linked glycosylation, and interferon-γ response. We detected a high percentage of CIMP<sup>+</sup> tumors with 14q loss (75%) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C). CIMP<sup>+</sup> tumors shared a similar <em>trans</em>-effect profile, including increased MYC signaling and protein translation, with a unique signature associated with increased OXPHOS and decreased focal adhesion (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>).</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Correlation of Transcriptome and Proteome Expression</span></h3><div class="section-paragraph">To understand the regulatory relationship between transcriptomic and proteomic processes, we calculated gene-wise (inter-sample) and sample-wise (intra-sample) correlation of 7,027 mRNA-protein pairs for the 103 ccRCC tumors and 72 NATs. NATs displayed a median gene-wise correlation value of 0.34, while tumors displayed a higher median value of 0.43, which is similar to previous studies investigating colorectal and high grade serous ovarian cancers (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>; <div class="dropBlock reference-citations"><a href="#" id="back-bib188" data-db-target-for="bib188-d218962e2276" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib188">Zhang et al., 2014a</a><div data-db-target-of="bib188-d218962e2276" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (628)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib190" data-db-target-for="bib190-d218962e2276" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib190">Zhang et al., 2016</a><div data-db-target-of="bib190-d218962e2276" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (305)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For ccRCC tumors and NATs, 74% and 52% of mRNA-protein pairs had significant positive Spearman correlations, respectively (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; Benjamini-Hochberg adjusted p &lt; 0.01), with OXPHOS, spliceosome, and ribosome pathways poorly correlated in tumors. In NATs, the housekeeping activities of the spliceosome and ribosome displayed a distinct correlation pattern, representing a differentially regulated axis to maintain cellular homeostasis (<div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d218962e2282" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Komili and Silver, 2008</a><div data-db-target-of="bib79-d218962e2282" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Komili S. </li><li class="loa__item">Silver P.A. </li></ul><div class="reference-citations__detials">Coupling and coordination in gene expression processes: a systems biology view.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Genet.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">38-48</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrg2223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18071322&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-37249034083&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=38-48&journal=Nat.+Rev.+Genet.&author=Komili+S.&author=Silver+P.A.&title=Coupling+and+coordination+in+gene+expression+processes%3A+a+systems+biology+view">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/519dea2d-831d-460c-bb7a-bfe9ab98ef92/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/cc417c44-fb1e-4129-91a7-b56534e496fa/gr3_lrg.jpg" data-large-image="/cms/attachment/519dea2d-831d-460c-bb7a-bfe9ab98ef92/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Correlations between Transcriptomic and Proteomic Abundance</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Gene-wise correlations of mRNA and protein expression in tumors (left) and NATs (right). Annotated cellular pathways and corresponding Spearman gene-wise correlation (bottom).</div><div class="section-paragraph">(B) Sample-wise correlation of tumors ranked from high to low with corresponding NAT sample-wise correlation (top). Tumors were evenly distributed into three groups: high (blue), middle (gray), and low (gold). BAP1 mutation, chromosome 14 loss status, and tumor grade are annotated (bottom).</div><div class="section-paragraph">(C) Boxplots of ribosome and translation factor gene expression and Pol I-associated regulation in tumor samples (left) and corresponding NATs (right) (<sup>∗</sup>p &lt; 0.05).</div><div class="section-paragraph"><a data-locator="figs3" aria-expanded="false" aria-controls="crosref2590" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs3">Figure S3</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2595" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2600" href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" target="_blank">S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/519dea2d-831d-460c-bb7a-bfe9ab98ef92/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/cc417c44-fb1e-4129-91a7-b56534e496fa/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Examination of sample-wise mRNA-protein correlation indicated a lower median sample-wise correlation for tumor samples (0.44) than for NAT samples (0.52), which had lower variance (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">S3</a>A; Benjamini-Hochberg adjusted p &lt; 0.001). Adjusting for tumor purity (<a href="#sec4" class="scroll-into-link">STAR Methods</a>), we detected a trend of higher sample-wise correlation in tumors associated with clinical features such as higher grade (p = 0.006), chromosome 14 loss (p = 0.0006), and <em>BAP1</em> mutations (p = 0.00004) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>). Linking high sample correlation to increased protein translation, we identified a concordant increase of ribosome and translation factor protein expression (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). Delineating a mechanism of increased ribosome biogenesis via Pol I transcription regulation (<div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d218962e2309" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Pelletier et al., 2018</a><div data-db-target-of="bib130-d218962e2309" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pelletier J. </li><li class="loa__item">Thomas G. </li><li class="loa__item">Volarević S. </li></ul><div class="reference-citations__detials">Ribosome biogenesis in cancer: new players and therapeutic avenues.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">51-63</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrc.2017.104&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29192214&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85038859866&amp;cf=" onclick="newWindow(this.href);return false">Scopus (96)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=51-63&journal=Nat.+Rev.+Cancer&author=Pelletier+J.&author=Thomas+G.&author=Volarevi%C4%87+S.&title=Ribosome+biogenesis+in+cancer%3A+new+players+and+therapeutic+avenues">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we detected a corresponding increase of protein expression for MYC-targets and mTORC1 signaling genes (p &lt; 0.05), increased mRNA levels of the Pol I transcription activator, ECT2 (p &lt; 0.05), and decreased mRNA levels for the Pol I transcription inhibitor, FGF13, in tumors with high sample-wise correlation compared to those with low sample-wise correlation (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). In contrast, corresponding NATs did not display differences between the tumor-based classifications of high/low sample-wise correlation or express differential levels of ribosome and translation-related proteins (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>C, <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">S3</a>B, and S3C). Interestingly, there was a concerted shift of discordant ribosome protein-mRNA levels in tumors (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). However, the dysregulated mechanisms that contribute to the uncoupling of ribosomal mRNA and protein expression in tumors are less clear, suggesting the importance of protein evidence when evaluating ribosome biogenesis as a therapeutic target for intervention in ccRCC (<div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d218962e2325" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Devlin et al., 2016</a><div data-db-target-of="bib35-d218962e2325" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Devlin J.R. </li><li class="loa__item">Hannan K.M. </li><li class="loa__item">Hein N. </li><li class="loa__item">Cullinane C. </li><li class="loa__item">Kusnadi E. </li><li class="loa__item">Ng P.Y. </li><li class="loa__item">George A.J. </li><li class="loa__item">Shortt J. </li><li class="loa__item">Bywater M.J. </li><li class="loa__item">Poortinga G. </li><li>et al.</li></ul><div class="reference-citations__detials">Combination therapy targeting ribosome biogenesis and mRNA translation synergistically extends survival in MYC-driven lymphoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">59-70</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-14-0673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26490423&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=59-70&journal=Cancer+Discov.&author=Devlin+J.R.&author=Hannan+K.M.&author=Hein+N.&author=Cullinane+C.&author=Kusnadi+E.&author=Ng+P.Y.&author=George+A.J.&author=Shortt+J.&author=Bywater+M.J.&author=Poortinga+G.&title=Combination+therapy+targeting+ribosome+biogenesis+and+mRNA+translation+synergistically+extends+survival+in+MYC-driven+lymphoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="figs3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/e56efb22-e109-4494-b783-7ee5f5c39613/figs3.jpg" alt="Figure thumbnail figs3" data-locator="figs3" data-high-res="/cms/attachment/021175c5-e06e-4ee8-8baf-b2ecaea78437/figs3_lrg.jpg" data-large-image="/cms/attachment/e56efb22-e109-4494-b783-7ee5f5c39613/figs3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S3" class="label figure__label">Figure S3</span><span class="figure__title__text">Correlation of mRNA and Protein Abundance in ccRCC and NATs, Related to <a data-locator="fig3" aria-expanded="false" aria-controls="crosref2680" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig3">Figure 3</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Violin plots of tumor and NAT sample-wise correlation frequency and median distribution. (B) Sample-wise correlation of NATs ranked from high to low. NATs were evenly distributed into three groups: high (blue), middle (gray), and low (gold). Corresponding tumor grade is indicated below. (C) Boxplots of ribosomal gene expression (mRNA and protein), translation factor expression (mRNA and protein), and Pol I associated regulatory factors based on NAT high/low sample-wise correlation ranking (<sup>∗</sup>p &lt; 0.05).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/e56efb22-e109-4494-b783-7ee5f5c39613/figs3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/021175c5-e06e-4ee8-8baf-b2ecaea78437/figs3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic Alterations of ccRCC Compared to NATs</span></h3><div class="section-paragraph">Visualization of the abundance of identified proteins by principal component analysis (PCA) and hierarchical clustering showed clear discrimination between ccRCC (n = 103) and NAT samples (n = 80) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>A). In total, 820 proteins showed significant differential expression in a group comparison of tissue samples (log2 fold-change &gt;1; Benjamini-Hochberg adjusted p &lt; 0.05), with 565 proteins downregulated and 255 upregulated in ccRCC tumors relative to NATs (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). Enrichment analysis revealed immune response, EMT, and multiple signaling pathways (hypoxia, glycolysis, and mTOR) to be upregulated in tumors, and TCA cycle, fatty acid metabolism, and OXPHOS to be downregulated (adjusted p &lt; 0.05; <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>B). Select cellular pathways were maintained even when accounting for tissue heterogeneity in both tumors and NAT samples (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). ccRCC tumors are characterized by particular genomic alterations that have resulted in their classification as a metabolic disease (<div class="dropBlock reference-citations"><a href="#bib180" id="back-bib180" data-db-target-for="bib180-d218962e2354" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib180">Wettersten et al., 2017</a><div data-db-target-of="bib180-d218962e2354" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wettersten H.I. </li><li class="loa__item">Aboud O.A. </li><li class="loa__item">Lara Jr., P.N. </li><li class="loa__item">Weiss R.H. </li></ul><div class="reference-citations__detials">Metabolic reprogramming in clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Nephrol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">410-419</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrneph.2017.59&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28480903&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85019246775&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=410-419&journal=Nat.+Rev.+Nephrol.&author=Wettersten+H.I.&author=Aboud+O.A.&author=Lara+P.N.&author=Weiss+R.H.&title=Metabolic+reprogramming+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which prompted us to identify and annotate differentially abundant mRNA transcripts and proteins involved in cellular metabolism (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>). Proteins in the glycolysis pathway and their cognate mRNAs were upregulated in this analysis, whereas proteins associated with OXPHOS were downregulated. A non-linear correlation has been previously observed between metabolic mRNA levels and corresponding glycolytic and OXPHOS metabolites in ccRCC (<div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d218962e2363" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Hakimi et al., 2016</a><div data-db-target-of="bib56-d218962e2363" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hakimi A.A. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Lee C.H. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">Brannon A.R. </li><li class="loa__item">Luna A. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Liu E.M. </li><li class="loa__item">Shen R. </li><li class="loa__item">Lee W. </li><li>et al.</li></ul><div class="reference-citations__detials">An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">104-116</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26766592&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84958749958&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=104-116&journal=Cancer+Cell&author=Hakimi+A.A.&author=Reznik+E.&author=Lee+C.H.&author=Creighton+C.J.&author=Brannon+A.R.&author=Luna+A.&author=Aksoy+B.A.&author=Liu+E.M.&author=Shen+R.&author=Lee+W.&title=An+Integrated+Metabolic+Atlas+of+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Analyzing the differential abundances of mRNA and protein levels between tumors and NAT revealed a prominent uncoupling of OXPHOS mRNA and protein expression that was disparate from other cellular pathways (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>C–4E and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>C), which reflects the regulation of select OXPHOS components at the translational level (<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d218962e2373" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Richman et al., 2016</a><div data-db-target-of="bib140-d218962e2373" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Richman T.R. </li><li class="loa__item">Spåhr H. </li><li class="loa__item">Ermer J.A. </li><li class="loa__item">Davies S.M.K. </li><li class="loa__item">Viola H.M. </li><li class="loa__item">Bates K.A. </li><li class="loa__item">Papadimitriou J. </li><li class="loa__item">Hool L.C. </li><li class="loa__item">Rodger J. </li><li class="loa__item">Larsson N.G. </li><li>et al.</li></ul><div class="reference-citations__detials">Loss of the RNA-binding protein TACO1 causes late-onset mitochondrial dysfunction in mice.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">11884</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms11884&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27319982&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84975502603&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11884&journal=Nat.+Commun.&author=Richman+T.R.&author=Sp%C3%A5hr+H.&author=Ermer+J.A.&author=Davies+S.M.K.&author=Viola+H.M.&author=Bates+K.A.&author=Papadimitriou+J.&author=Hool+L.C.&author=Rodger+J.&author=Larsson+N.G.&title=Loss+of+the+RNA-binding+protein+TACO1+causes+late-onset+mitochondrial+dysfunction+in+mice">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Together, these results show that the functional consequences of the Warburg effect are not fully captured at the transcriptional level, which could impact the clinical use of transcription-based metabolic signatures for prognosis of ccRCC (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e2376" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e2376" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/b4992771-de07-41fb-81d6-c695ebfbfc96/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/de6371bf-aab4-496f-8544-ab77307f2de9/gr4_lrg.jpg" data-large-image="/cms/attachment/b4992771-de07-41fb-81d6-c695ebfbfc96/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Differential Expression of Transcriptomic and Proteomic Profiles between ccRCC Tumors and NAT Protein Expression</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) PCA visualization of protein expression in ccRCC tumors and NATs.</div><div class="section-paragraph">(B) Analysis of significantly differentially regulated pathways (adjusted p &lt; 0.05) between ccRCC tumors and NATs.</div><div class="section-paragraph">(C) Schema of metabolic pathways (glycolysis and electron transport chain [OXPHOS]) with select differential gene expression of mRNA and protein levels between ccRCC tumors and NATs.</div><div class="section-paragraph">(D) Scatterplots depicting expression of mRNA (x axis) and protein (y axis). Linear regression of all mRNA-protein pairs (gray dotted line) and OXPHOS mRNA-gene pairs (red dotted line) are shown. Metabolism-related genes are indicated.</div><div class="section-paragraph">(E) Boxplot of representative OXPHOS genes from complex I (<em>NDUFV2</em>), IV (<em>COX6C</em>), and V (<em>ATP6V1F</em>) displaying discordant mRNA-protein expression (n.s., not significant, <sup>∗</sup>adjusted p &lt; 0.05).</div><div class="section-paragraph">See also <a data-locator="figs4" aria-expanded="false" aria-controls="crosref2605" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs4">Figure S4</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2610" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a>, <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2615" href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" target="_blank">S4</a>, and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2620" href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" target="_blank">S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/b4992771-de07-41fb-81d6-c695ebfbfc96/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/de6371bf-aab4-496f-8544-ab77307f2de9/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/5232b22c-cc3c-413b-8f1e-63fb6499692b/figs4.jpg" alt="Figure thumbnail figs4" data-locator="figs4" data-high-res="/cms/attachment/6dbc6c43-beb1-4986-882c-9387aa62641e/figs4_lrg.jpg" data-large-image="/cms/attachment/5232b22c-cc3c-413b-8f1e-63fb6499692b/figs4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S4" class="label figure__label">Figure S4</span><span class="figure__title__text">Differential Expression of Transcriptomic and Proteomic Profiles between ccRCC and NAT Protein Expression, Related to <a data-locator="fig4" aria-expanded="false" aria-controls="crosref2685" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig4">Figure 4</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmap of protein expression abundances between ccRCC and NAT samples. Hierarchical clustering discriminated tumors (left) from NAT (right) (n = 5745) (B) Volcano plot of significantly differentially expressed proteins (log2 fold-change &gt; 1; Benjamini-Hochberg adjusted p &lt; 0.05) (n = 820; 565 downregulated, 255 upregulated). (C) Scatterplot of differential gene expression between ccRCC tumors and NATs at mRNA and protein levels using log2-fold changes. Each dot represents one gene; x axis: protein, y axis: mRNA, respectively. Negative values indicate a decrease and positive values an increase in gene expression between tumor and NAT. Upregulated pathways in ccRCC samples were shown.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/5232b22c-cc3c-413b-8f1e-63fb6499692b/figs4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/6dbc6c43-beb1-4986-882c-9387aa62641e/figs4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomic Analysis of Kinase and Substrate Regulatory Pathways</span></h3><div class="section-paragraph">Phosphorylation impacts multiple cellular processes, with site occupancy tightly regulated by the activity of kinases and phosphatases on their respective substrates (<div class="dropBlock reference-citations"><a href="#bib172" id="back-bib172" data-db-target-for="bib172-d218962e2384" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib172">Ubersax and Ferrell, 2007</a><div data-db-target-of="bib172-d218962e2384" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ubersax J.A. </li><li class="loa__item">Ferrell Jr., J.E. </li></ul><div class="reference-citations__detials">Mechanisms of specificity in protein phosphorylation.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">530-541</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrm2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17585314&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34250878954&amp;cf=" onclick="newWindow(this.href);return false">Scopus (801)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=530-541&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Ubersax+J.A.&author=Ferrell+J.E.&title=Mechanisms+of+specificity+in+protein+phosphorylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We analyzed differential phosphopeptide abundance between 80 tumor/NAT paired tissues to stratify phospho-substrates corresponding to different kinases and their inhibitors, and identified CDK1 and MAPK1 (ERK2) as two highly ranked phospho-substrate events in most tumors. (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a>). A more comprehensive investigation of the cell-cycle regulatory network using phosphosite abundance revealed that phosphorylated substrates associated with S-phase entry/progression (CDK7-MCM2) and the G2/M checkpoint (WEE1-CDK1) were elevated across the majority of tumors (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A). The G2/M checkpoint is the final safeguard of genomic fidelity prior to mitosis; our data support a mechanism of G2-stalling that prevents mitotic arrest-induced apoptosis in tumors (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d218962e2397" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Bucher and Britten, 2008</a><div data-db-target-of="bib18-d218962e2397" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bucher N. </li><li class="loa__item">Britten C.D. </li></ul><div class="reference-citations__detials">G2 checkpoint abrogation and checkpoint kinase-1 targeting in the treatment of cancer.<div class="popup-ref"><span class="ref__series"><i>Br. J. Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">523-528</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.bjc.6604208&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18231106&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-38949147038&amp;cf=" onclick="newWindow(this.href);return false">Scopus (227)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2008&pages=523-528&journal=Br.+J.+Cancer&author=Bucher+N.&author=Britten+C.D.&title=G2+checkpoint+abrogation+and+checkpoint+kinase-1+targeting+in+the+treatment+of+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), evidenced by elevated levels of the inhibitory CDK1-Y15 phosphorylation, especially in more aggressive tumors (p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>B). Comprehensive examination of the signaling network involving MAPK1 revealed increased protein and phospho-peptide expression of the upstream receptor tyrosine kinase epidermal growth factor receptor (EGFR) in almost all tumors, while vascular endothelial growth factor (VEGF) receptors such as FLT and KDR were more selectively expressed and phosphorylated in tumors (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>A). Additionally, we inferred activated signaling from elevated substrate phosphosite occupancy. This analysis indicated that activation of the EGFR/VEGF downstream signaling pathways MAPK/ERK and AKT-mTOR converged on the downstream substrate EIF4EBP1 (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and 5B), an important regulator of protein translation (<div class="dropBlock reference-citations"><a href="#bib151" id="back-bib151" data-db-target-for="bib151-d218962e2414" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib151">She et al., 2010</a><div data-db-target-of="bib151-d218962e2414" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">She Q.B. </li><li class="loa__item">Halilovic E. </li><li class="loa__item">Ye Q. </li><li class="loa__item">Zhen W. </li><li class="loa__item">Shirasawa S. </li><li class="loa__item">Sasazuki T. </li><li class="loa__item">Solit D.B. </li><li class="loa__item">Rosen N. </li></ul><div class="reference-citations__detials">4E-BP1 is a key effector of the oncogenic activation of the AKT and ERK signaling pathways that integrates their function in tumors.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">39-51</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20609351&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954296394&amp;cf=" onclick="newWindow(this.href);return false">Scopus (280)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=39-51&journal=Cancer+Cell&author=She+Q.B.&author=Halilovic+E.&author=Ye+Q.&author=Zhen+W.&author=Shirasawa+S.&author=Sasazuki+T.&author=Solit+D.B.&author=Rosen+N.&title=4E-BP1+is+a+key+effector+of+the+oncogenic+activation+of+the+AKT+and+ERK+signaling+pathways+that+integrates+their+function+in+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This observation of cell signaling redundancy may explain the limited clinical response of ccRCC patients to mTOR-targeted therapies such as everolimus and temsirolimus (<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d218962e2418" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">Kwiatkowski et al., 2016</a><div data-db-target-of="bib82-d218962e2418" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Choueiri T.K. </li><li class="loa__item">Fay A.P. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Thorner A.R. </li><li class="loa__item">de Velasco G. </li><li class="loa__item">Tyburczy M.E. </li><li class="loa__item">Hamieh L. </li><li class="loa__item">Albiges L. </li><li class="loa__item">Agarwal N. </li><li>et al.</li></ul><div class="reference-citations__detials">Mutations in TSC1, TSC2, and MTOR are associated with response to rapalogs in patients with metastatic renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">2445-2452</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F1078-0432.CCR-15-2631&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26831717&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84968902640&amp;cf=" onclick="newWindow(this.href);return false">Scopus (105)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=2445-2452&journal=Clin.+Cancer+Res.&author=Kwiatkowski+D.J.&author=Choueiri+T.K.&author=Fay+A.P.&author=Rini+B.I.&author=Thorner+A.R.&author=de+Velasco+G.&author=Tyburczy+M.E.&author=Hamieh+L.&author=Albiges+L.&author=Agarwal+N.&title=Mutations+in+TSC1%2C+TSC2%2C+and+MTOR+are+associated+with+response+to+rapalogs+in+patients+with+metastatic+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggesting that combinational therapy targeting both mTOR and MAPK/ERK pathways may be a more effective approach. In addition, activation of mTOR signaling via the phosphorylation of the mTORC1 subunit, AKT1S1 (<div class="dropBlock reference-citations"><a href="#bib173" id="back-bib173" data-db-target-for="bib173-d218962e2421" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib173">Vander Haar et al., 2007</a><div data-db-target-of="bib173-d218962e2421" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vander Haar E. </li><li class="loa__item">Lee S.I. </li><li class="loa__item">Bandhakavi S. </li><li class="loa__item">Griffin T.J. </li><li class="loa__item">Kim D.H. </li></ul><div class="reference-citations__detials">Insulin signalling to mTOR mediated by the Akt/PKB substrate PRAS40.<div class="popup-ref"><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">316-323</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncb1547&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17277771&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33847397874&amp;cf=" onclick="newWindow(this.href);return false">Scopus (779)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=316-323&journal=Nat.+Cell+Biol.&author=Vander+Haar+E.&author=Lee+S.I.&author=Bandhakavi+S.&author=Griffin+T.J.&author=Kim+D.H.&title=Insulin+signalling+to+mTOR+mediated+by+the+Akt%2FPKB+substrate+PRAS40">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was observed in phosphoproteomic analysis but was not captured at the transcriptomic level (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>C), highlighting the added value of phosphoproteomics in integrative analyses. Independent of EGFR-mediated MAPK/AKT signaling, PKM phosphorylation was highly ranked in approximately half of our ccRCC tumor cohort and associated with lower tumor grade (p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>B), reflecting a secondary, EGFR-mediated mechanism of glycolytic reprogramming in a subset of ccRCC tumors (<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d218962e2433" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Lim et al., 2016</a><div data-db-target-of="bib92-d218962e2433" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lim S.O. </li><li class="loa__item">Li C.W. </li><li class="loa__item">Xia W. </li><li class="loa__item">Lee H.H. </li><li class="loa__item">Chang S.S. </li><li class="loa__item">Shen J. </li><li class="loa__item">Hsu J.L. </li><li class="loa__item">Raftery D. </li><li class="loa__item">Djukovic D. </li><li class="loa__item">Gu H. </li><li>et al.</li></ul><div class="reference-citations__detials">EGFR signaling enhances aerobic glycolysis in triple-negative breast cancer cells to promote tumor growth and immune escape.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">76</span></span>: <span class="ref__seriesPages">1284-1296</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-15-2478&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26759242&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84961825288&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1284-1296&journal=Cancer+Res.&author=Lim+S.O.&author=Li+C.W.&author=Xia+W.&author=Lee+H.H.&author=Chang+S.S.&author=Shen+J.&author=Hsu+J.L.&author=Raftery+D.&author=Djukovic+D.&author=Gu+H.&title=EGFR+signaling+enhances+aerobic+glycolysis+in+triple-negative+breast+cancer+cells+to+promote+tumor+growth+and+immune+escape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/24de9bce-9537-48ad-8ca8-328c3469bd89/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/c506a178-46a6-44c2-b429-88376dcf1712/gr5_lrg.jpg" data-large-image="/cms/attachment/24de9bce-9537-48ad-8ca8-328c3469bd89/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Phospho-Substrates with Associated Kinases and a Network Module Specific to Phospho-Tumor Data</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Ranked phospho-substrate events of kinases with inhibitors and fold-change at global- and phospho-levels for kinases and substrates, respectively.</div><div class="section-paragraph">(B) Pathways based on the selected phospho-substrates and kinases, with relevant drugs shown by targets (red). Current FDA-approved drugs for ccRCC (gray).</div><div class="section-paragraph">(C) Pairwise correlation of nodes at multi-omics levels of “cell cycle” co-expression network module.</div><div class="section-paragraph">(D) Heatmap of “cell cycle” module expression with grade, BAP1 and chromosome 14 loss, and genome instability distribution annotated.</div><div class="section-paragraph">See also <a data-locator="figs5" aria-expanded="false" aria-controls="crosref2625" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs5">Figure S5</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2630" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2635" href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" target="_blank">S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/24de9bce-9537-48ad-8ca8-328c3469bd89/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/c506a178-46a6-44c2-b429-88376dcf1712/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/b77a5237-bfdb-4f91-80c5-f43f2da62ce5/figs5.jpg" alt="Figure thumbnail figs5" data-locator="figs5" data-high-res="/cms/attachment/36b57dd5-df74-4df9-9963-7e3498550835/figs5_lrg.jpg" data-large-image="/cms/attachment/b77a5237-bfdb-4f91-80c5-f43f2da62ce5/figs5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S5" class="label figure__label">Figure S5</span><span class="figure__title__text">Expression Changes of Phospho-Substrates in Multiple Omics Data and the Network Module of Cell Cycle, Related to <a data-locator="fig5" aria-expanded="false" aria-controls="crosref2690" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Fold change (log2 scale) of 80 tumor/NAT pairs in phospho-substrates based on mRNA and global proteome expression. (B) Comparison of phosphorylation levels of PKM (top) and CDK1 (bottom) phosphopeptides between low grade/stage tumors and high grade/stage tumors (Mann–Whitney U test, p &lt; 0.05). (C) Two phospho-substrates, AKT1S1 and SMAD3 showed substantial differences in mRNA and phosphopeptide expression when comparing tumors to NATs. (D) Phosphopeptide co-expression network module displaying “Cell Cycle” pathway enrichment. (E) Phosphopeptide co-expression network module displaying “Angiogenesis” pathway enrichment. (F) Pairwise correlation of nodes at multi-omics levels of the “Angiogenesis” co-expression network module. (G) Heatmap of “Angiogenesis” module expression with grade, BAP1 and chromosome 14 loss, and genome instability distribution annotated.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/b77a5237-bfdb-4f91-80c5-f43f2da62ce5/figs5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/36b57dd5-df74-4df9-9963-7e3498550835/figs5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Leveraging differential phosphopeptide abundance across all tumor samples, we identified several phosphopeptide co-expression networks including two modules (cell cycle and angiogenesis) that were independent of global proteomic and transcriptomic profiles (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>C, 5D, and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>D–S5G; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a>). The cell-cycle module included multiple cell-cycle checkpoint proteins involved in the G1/S-phase transition (CDKN1B, SKP2), S-phase regulatory elements (MCM4, MCM6), and the G2/M phase (CDK1, TK1, CDC20) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>D), with phosphorylation of CDC20 representing another mechanism of mitotic-arrest (<div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d218962e2450" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Hein and Nilsson, 2016</a><div data-db-target-of="bib59-d218962e2450" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hein J.B. </li><li class="loa__item">Nilsson J. </li></ul><div class="reference-citations__detials">Interphase APC/C-Cdc20 inhibition by cyclin A2-Cdk2 ensures efficient mitotic entry.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10975</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms10975&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26960431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960912526&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10975&journal=Nat.+Commun.&author=Hein+J.B.&author=Nilsson+J.&title=Interphase+APC%2FC-Cdc20+inhibition+by+cyclin+A2-Cdk2+ensures+efficient+mitotic+entry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Interestingly, we observed tumors with genomic instability that correlated with this module, as well as phospho-events involved in DNA damage response (e.g., FANCD2, PSME3, CLSPN, and BRCA1) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>D and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>D), representing a mechanism by which a subset of tumors engage cellular processes in response to loss of genomic fidelity. The angiogenesis module included multiple elements associated with VEGF-response (ELK3, ERG), Notch-associated signaling (LDB2, SOX18), and vasculature development (PECAM-1, CCM2L) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>E). This module was inversely correlated with <em>BAP1</em> and chromosome 14 loss and associated with lower-grade tumors (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>G). Our phosphoproteomic analysis thus identified multiple signal transduction pathways activated in tumors and provided evidence for expanding treatment selection beyond the current FDA-approved therapies targeting VEGF and mTOR (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B; <div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d218962e2472" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Hsieh et al., 2018b</a><div data-db-target-of="bib66-d218962e2472" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hsieh J.J. </li><li class="loa__item">Le V. </li><li class="loa__item">Cao D. </li><li class="loa__item">Cheng E.H. </li><li class="loa__item">Creighton C.J. </li></ul><div class="reference-citations__detials">Genomic classifications of renal cell carcinoma: a critical step towards the future application of personalized kidney cancer care with pan-omics precision.<div class="popup-ref"><span class="ref__series"><i>J. Pathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">244</span></span>: <span class="ref__seriesPages">525-537</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fpath.5022&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29266437&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044317167&amp;cf=" onclick="newWindow(this.href);return false">Scopus (36)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2018&pages=525-537&journal=J.%C2%A0Pathol.&author=Hsieh+J.J.&author=Le+V.&author=Cao+D.&author=Cheng+E.H.&author=Creighton+C.J.&title=Genomic+classifications+of+renal+cell+carcinoma%3A+a+critical+step+towards+the+future+application+of+personalized+kidney+cancer+care+with+pan-omics+precision">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Characterization of Immune Infiltration in ccRCC</span></h3><div class="section-paragraph">To gain insight into features of immune infiltration in ccRCC, we analyzed the transcriptomic profiles of 103 tumors and 72 NATs and deconvoluted immune, stromal, and microenvironmental cell gene signatures using xCell (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d218962e2480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d218962e2480" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These molecularly based cell-type classifications were supported by histopathological assessment, DNA promoter methylation-based deconvolution analysis, and ESTIMATE analysis (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e2483" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e2483" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with the latter showing a Pearson correlation higher than 0.75 between protein and mRNA data for immune- and stromal-derived signatures (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>A and S6B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). ESTIMATE generated RNA-seq stromal and immune signatures in this cohort were comparable to those observed in TCGA ccRCC and Genotype-Tissue Expression (GTEx) kidney-cortex datasets (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>C). Consensus clustering of the cell signatures identified two NAT subtypes with distinct enrichment of cell signatures relative to ccRCC tumor tissues and four ccRCC tumor subtypes (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). The latter were discriminated by the presence or absence of specific cell types related to immune (CD8<sup>+</sup> T cells, macrophages, dendritic cells) and stromal (fibroblast, endothelial) signatures. Adopting general features of immune-based groupings described previously (<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d218962e2504" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Chen and Mellman, 2017</a><div data-db-target-of="bib23-d218962e2504" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen D.S. </li><li class="loa__item">Mellman I. </li></ul><div class="reference-citations__detials">Elements of cancer immunity and the cancer-immune set point.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">541</span></span>: <span class="ref__seriesPages">321-330</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature21349&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28102259&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016548232&amp;cf=" onclick="newWindow(this.href);return false">Scopus (983)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=541&publication_year=2017&pages=321-330&journal=Nature&author=Chen+D.S.&author=Mellman+I.&title=Elements+of+cancer+immunity+and+the+cancer-immune+set+point">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and incorporating transcriptomic and proteomic features, we defined four tumor subtypes in this ccRCC cohort: (1) CD8<sup>+</sup> inflamed, (2) CD8<sup>−</sup> inflamed, (3) VEGF immune desert, and (4) metabolic immune desert (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). These subtypes were characterized by unique genomic alterations and tumor microenvironment (TME) signatures and discriminating signaling pathways that could be leveraged to predict therapeutic response (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B, 6C, and 6E).<div class="floatDisplay"><figure id="figs6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/ded39348-08f4-4789-ad59-e5061fe3c4ba/figs6.jpg" alt="Figure thumbnail figs6" data-locator="figs6" data-high-res="/cms/attachment/56933e4e-15c8-4e11-a8b3-5df10034424b/figs6_lrg.jpg" data-large-image="/cms/attachment/ded39348-08f4-4789-ad59-e5061fe3c4ba/figs6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S6" class="label figure__label">Figure S6</span><span class="figure__title__text">Immune-Based Subtyping of ccRCC Tumors, Related to <a data-locator="fig6" aria-expanded="false" aria-controls="crosref2695" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig6">Figure 6</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Histologic assessment of xCell-generated cell signatures. Representative H&amp;E stained tissue sections verifying mesangial cell, immune infiltration, and stromal infiltration signatures. (B) Comparison between RNA-seq and global proteomics in estimating immune and stromal cell infiltration based on 103 ccRCC tumor samples and 72 NAT samples. Correlation between scores based on RNA-seq and global proteomics data was higher than 0.75 for both tissue types. Stromal and immune scores were computed via ESTIMATE. (C) Comparison of stromal and immune scores between CPTAC, TCGA, and GTEx data. Stromal and immune scores were computed via ESTIMATE using RNA-seq data. (D) IHC profiling of immune cell populations in ccRCC. FFPE sections were stained for CD8, CD4, and CD163 protein markers to determine CD8+ T cell, CD4+ T cell, and macrophage abundance in ccRCC tumor tissues. The scale bar indicates 200 μm. (E) Boxplot of pathway scores for different immune groups based on RNA-seq and proteomics data from 103 CPTAC ccRCC tumor samples and 80 CPTAC NAT samples. Pathway scores were computed via combined z-score. Pathways upregulated in CD8+ Inflamed tumors included “APM,” while pathways upregulated in CD8- Inflamed tumors included “ECM” and “Platelet Degranulation.” Pathways upregulated in VEGF Immune Desert tumors included “RAP1 Signaling,” while pathways upregulated in Metabolic Immune Desert tumors included “MTOR Signaling” and “Mitochondrial.” (F) Z-Score based on Angiogenesis (<em>VEGFA</em>, <em>KDR</em>, <em>ESM1</em>, <em>PECAM1</em>, <em>ANGPTL4</em>, <em>CD34</em>) and T-Effector (<em>CD8A</em>, <em>EOMES</em>, <em>PRF1</em>, <em>IFNG</em>, <em>CD274</em>) signatures from <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d218962e353" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">McDermott et al., 2018</a><div data-db-target-of="bib101-d218962e353" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McDermott D.F. </li><li class="loa__item">Huseni M.A. </li><li class="loa__item">Atkins M.B. </li><li class="loa__item">Motzer R.J. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Fong L. </li><li class="loa__item">Joseph R.W. </li><li class="loa__item">Pal S.K. </li><li class="loa__item">Reeves J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>. Upregulation of each signature in a particular immune group was assessed via multivariate model (<sup>∗∗∗</sup> adjusted p &lt; 0.001). (G) Distribution of CD8+ T cells, endothelial, and fibroblast cell concentrations for different immune groups based on 103 CPTAC ccRCC samples and 495 TCGA KIRC samples; RNA-seq-derived cell signatures. (H) Kaplan-Meier curves of survival data stratified by immune groups derived based on 495 TCGA KIRC samples. VEGF Immune Desert samples displayed increased overall compared to other groups (p value = 0.0004). CD8+ Inflamed subgroup samples displayed decreased overall survival compared to all the other groups (p value = 5e-04).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/ded39348-08f4-4789-ad59-e5061fe3c4ba/figs6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/56933e4e-15c8-4e11-a8b3-5df10034424b/figs6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/3c52997f-6940-4b1e-8d52-1cb204c7dd66/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/7415647e-d822-4651-b5c1-fbab4cdd69b5/gr6_lrg.jpg" data-large-image="/cms/attachment/3c52997f-6940-4b1e-8d52-1cb204c7dd66/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Immune-Based Subtyping of ccRCC Tumors</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Transcriptome-based deconvolution of mRNA transcript cell signatures in 103 ccRCC tumors and 72 NATs using xCell.</div><div class="section-paragraph">(B) Molecular characteristics (transcriptomic, proteomic) stratified tumors into four immune subtypes: CD8<sup>+</sup> inflamed (red), CD8<sup>−</sup> inflamed (blue), VEGF immune desert (yellow), metabolic immune desert (green), and NATs into two subtypes (pink and gray).</div><div class="section-paragraph">(C) Proportion of <em>BAP1</em> mutation, <em>PBRM1</em> mutation, chromosome 14 loss, and chromosome 7 gains within each of the immune groups.</div><div class="section-paragraph">(D) Proportion of high tumor grade tumors (i.e., grade 3 and grade 4) in each of the immune groups for CPTAC and TCGA datasets. High-grade tumors were significantly enriched in CD8<sup>+</sup> inflamed group compared to VEGF immune desert group.</div><div class="section-paragraph">(E) Density contours of immune and stroma scores of each immune subtype. Pathways upregulated based on RNA-seq and global proteomics data are labeled with “R” or “P,” respectively.</div><div class="section-paragraph">See also <a data-locator="figs6" aria-expanded="false" aria-controls="crosref2640" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs6">Figure S6</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2645" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2650" href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" target="_blank">S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/3c52997f-6940-4b1e-8d52-1cb204c7dd66/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/7415647e-d822-4651-b5c1-fbab4cdd69b5/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">CD8<sup>+</sup> inflamed tumors were characterized by a high degree of CD8<sup>+</sup> T cell infiltration (t test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>D), increased expression of the immune evasion markers <em>PD1</em>, <em>PD-L1</em>, <em>PD-L2</em>, and <em>CTLA4</em> (t test adjusted p &lt; 0.05), and high frequency of chromosome 14 loss (chi-square test p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A–6C). Corresponding to the elevated CD8<sup>+</sup> T cell presence was a higher frequency of <em>BAP1</em> mutations, a feature previously associated with increased immune infiltration in a kidney cancer xenograft model (<div class="dropBlock reference-citations"><a href="#bib177" id="back-bib177" data-db-target-for="bib177-d218962e2554" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib177">Wang et al., 2018b</a><div data-db-target-of="bib177-d218962e2554" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang T. </li><li class="loa__item">Lu R. </li><li class="loa__item">Kapur P. </li><li class="loa__item">Jaiswal B.S. </li><li class="loa__item">Hannan R. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Pedrosa I. </li><li class="loa__item">Luke J.J. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Goldstein L.D. </li><li>et al.</li></ul><div class="reference-citations__detials">An empirical approach leveraging tumorgrafts to dissect the tumor microenvironment in renal cell carcinoma identifies missing link to prognostic inflammatory factors.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1142-1155</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-17-1246&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29884728&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053203753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1142-1155&journal=Cancer+Discov.&author=Wang+T.&author=Lu+R.&author=Kapur+P.&author=Jaiswal+B.S.&author=Hannan+R.&author=Zhang+Z.&author=Pedrosa+I.&author=Luke+J.J.&author=Zhang+H.&author=Goldstein+L.D.&title=An+empirical+approach+leveraging+tumorgrafts+to+dissect+the+tumor+microenvironment+in+renal+cell+carcinoma+identifies+missing+link+to+prognostic+inflammatory+factors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Proteomic analysis showed upregulation of CD38 expression and pathways involved in antigen processing/presentation (APM) and interferon-γ signaling (Fisher’s exact test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>E; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Phosphoproteomic analysis confirmed active interferon-γ signaling via elevated phosphorylation of the downstream effector STAT1 (t test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B). Interferon-γ signaling has been shown to regulate PD-L1 expression in cancer cells (<div class="dropBlock reference-citations"><a href="#" id="back-bib24" data-db-target-for="bib24-d218962e2570" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Chen et al., 2012</a><div data-db-target-of="bib24-d218962e2570" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen J. </li><li class="loa__item">Feng Y. </li><li class="loa__item">Lu L. </li><li class="loa__item">Wang H. </li><li class="loa__item">Dai L. </li><li class="loa__item">Li Y. </li><li class="loa__item">Zhang P. </li></ul><div class="reference-citations__detials">Interferon-γ-induced PD-L1 surface expression on human oral squamous carcinoma via PKD2 signal pathway.<div class="popup-ref"><span class="ref__series"><i>Immunobiology.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">217</span></span>: <span class="ref__seriesPages">385-393</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.imbio.2011.10.016&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22204817&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84858007569&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2012&pages=385-393&journal=Immunobiology&author=Chen+J.&author=Feng+Y.&author=Lu+L.&author=Wang+H.&author=Dai+L.&author=Li+Y.&author=Zhang+P.&title=Interferon-%CE%B3-induced+PD-L1+surface+expression+on+human+oral+squamous+carcinoma+via+PKD2+signal+pathway">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib47" data-db-target-for="bib47-d218962e2570" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Garcia-Diaz et al., 2017</a><div data-db-target-of="bib47-d218962e2570" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Garcia-Diaz A. </li><li class="loa__item">Shin D.S. </li><li class="loa__item">Moreno B.H. </li><li class="loa__item">Saco J. </li><li class="loa__item">Escuin-Ordinas H. </li><li class="loa__item">Rodriguez G.A. </li><li class="loa__item">Zaretsky J.M. </li><li class="loa__item">Sun L. </li><li class="loa__item">Hugo W. </li><li class="loa__item">Wang X. </li><li>et al.</li></ul><div class="reference-citations__detials">Interferon Receptor Signaling Pathways Regulating PD-L1 and PD-L2 Expression.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">1189-1201</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28494868&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85019192700&amp;cf=" onclick="newWindow(this.href);return false">Scopus (332)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1189-1201&journal=Cell+Rep.&author=Garcia-Diaz+A.&author=Shin+D.S.&author=Moreno+B.H.&author=Saco+J.&author=Escuin-Ordinas+H.&author=Rodriguez+G.A.&author=Zaretsky+J.M.&author=Sun+L.&author=Hugo+W.&author=Wang+X.&title=Interferon+Receptor+Signaling+Pathways+Regulating+PD-L1+and+PD-L2+Expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and the combination of increased CD38 protein and mRNA signature (<em>PD-L1</em>, <em>PD-L2</em>, and <em>CTLA4</em>) associated with T cell exhaustion are representative of multiple mechanisms of immune evasion in this tumor type, with implications for immune checkpoint therapy (<div class="dropBlock reference-citations"><a href="#" id="back-bib27" data-db-target-for="bib27-d218962e2580" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Chen et al., 2018</a><div data-db-target-of="bib27-d218962e2580" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen L. </li><li class="loa__item">Diao L. </li><li class="loa__item">Yang Y. </li><li class="loa__item">Yi X. </li><li class="loa__item">Rodriguez B.L. </li><li class="loa__item">Li Y. </li><li class="loa__item">Villalobos P.A. </li><li class="loa__item">Cascone T. </li><li class="loa__item">Liu X. </li><li class="loa__item">Tan L. </li><li>et al.</li></ul><div class="reference-citations__detials">CD38-mediated immunosuppression as a mechanism of tumor cell escape from PD-1/PD-l1 blockade.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1156-1175</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-17-1033&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30012853&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053178618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (75)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1156-1175&journal=Cancer+Discov.&author=Chen+L.&author=Diao+L.&author=Yang+Y.&author=Yi+X.&author=Rodriguez+B.L.&author=Li+Y.&author=Villalobos+P.A.&author=Cascone+T.&author=Liu+X.&author=Tan+L.&title=CD38-mediated+immunosuppression+as+a+mechanism+of+tumor+cell+escape+from+PD-1%2FPD-l1+blockade">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib146" data-db-target-for="bib146-d218962e2580" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Sade-Feldman et al., 2018</a><div data-db-target-of="bib146-d218962e2580" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sade-Feldman M. </li><li class="loa__item">Yizhak K. </li><li class="loa__item">Bjorgaard S.L. </li><li class="loa__item">Ray J.P. </li><li class="loa__item">de Boer C.G. </li><li class="loa__item">Jenkins R.W. </li><li class="loa__item">Lieb D.J. </li><li class="loa__item">Chen J.H. </li><li class="loa__item">Frederick D.T. </li><li class="loa__item">Barzily-Rokni M. </li><li>et al.</li></ul><div class="reference-citations__detials">Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">175</span></span>: <span class="ref__seriesPages">998-1013.e20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30388456&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85055560004&amp;cf=" onclick="newWindow(this.href);return false">Scopus (156)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=998-1013.e20&journal=Cell&author=Sade-Feldman+M.&author=Yizhak+K.&author=Bjorgaard+S.L.&author=Ray+J.P.&author=de+Boer+C.G.&author=Jenkins+R.W.&author=Lieb+D.J.&author=Chen+J.H.&author=Frederick+D.T.&author=Barzily-Rokni+M.&title=Defining+T+Cell+States+Associated+with+Response+to+Checkpoint+Immunotherapy+in+Melanoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">CD8<sup>−</sup> inflamed tumors were characterized by an innate immune signature, evidenced by dendritic and macrophage cells in the TME compared to the VEGF and metabolic immune desert groups and by increased complement and coagulation cascade protein expression (Fisher’s exact test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A, 6B, and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). A fibroblast signature that included elevated <em>PDGFRA</em>, extracellular matrix (ECM) and EMT-associated protein abundance, and expression of cancer associated fibroblasts (CAFs) features (POSTN protein and gene <em>FAP</em> mRNA) (t test adjusted p &lt; 0.05) was a unique feature of the CD8<sup>−</sup> inflamed tumor subtype (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A, 6B, and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>E; <div class="dropBlock reference-citations"><a href="#" id="back-bib154" data-db-target-for="bib154-d218962e2611" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib154">Shiga et al., 2015</a><div data-db-target-of="bib154-d218962e2611" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shiga K. </li><li class="loa__item">Hara M. </li><li class="loa__item">Nagasaki T. </li><li class="loa__item">Sato T. </li><li class="loa__item">Takahashi H. </li><li class="loa__item">Takeyama H. </li></ul><div class="reference-citations__detials">Cancer-associated fibroblasts: Their characteristics and their roles in tumor growth.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">2443-2458</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fcancers7040902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26690480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84949775551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (289)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2443-2458&journal=Cancers+%28Basel%29&author=Shiga+K.&author=Hara+M.&author=Nagasaki+T.&author=Sato+T.&author=Takahashi+H.&author=Takeyama+H.&title=Cancer-associated+fibroblasts%3A+Their+characteristics+and+their+roles+in+tumor+growth">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib193" data-db-target-for="bib193-d218962e2611" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib193">Ziani et al., 2018</a><div data-db-target-of="bib193-d218962e2611" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ziani L. </li><li class="loa__item">Chouaib S. </li><li class="loa__item">Thiery J. </li></ul><div class="reference-citations__detials">Alteration of the antitumor immune response by cancer-associated fibroblasts.<div class="popup-ref"><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">414</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3389%2Ffimmu.2018.00414&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29545811&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042694394&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=414&journal=Front.+Immunol.&author=Ziani+L.&author=Chouaib+S.&author=Thiery+J.&title=Alteration+of+the+antitumor+immune+response+by+cancer-associated+fibroblasts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Together, these associated pathways are representative of TME-tumor crosstalk, with PDGF signaling driving fibroblast recruitment and activation, and CAFs subsequently inducing an EMT-phenotype in tumors (<div class="dropBlock reference-citations"><a href="#" id="back-bib48" data-db-target-for="bib48-d218962e2614" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Gascard and Tlsty, 2016</a><div data-db-target-of="bib48-d218962e2614" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gascard P. </li><li class="loa__item">Tlsty T.D. </li></ul><div class="reference-citations__detials">Carcinoma-associated fibroblasts: orchestrating the composition of malignancy.<div class="popup-ref"><span class="ref__series"><i>Genes Dev.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">1002-1019</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgad.279737.116&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27151975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84965082206&amp;cf=" onclick="newWindow(this.href);return false">Scopus (227)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=1002-1019&journal=Genes+Dev.&author=Gascard+P.&author=Tlsty+T.D.&title=Carcinoma-associated+fibroblasts%3A+orchestrating+the+composition+of+malignancy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib149" data-db-target-for="bib149-d218962e2614" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Seppä et al., 1982</a><div data-db-target-of="bib149-d218962e2614" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Seppä H. </li><li class="loa__item">Grotendorst G. </li><li class="loa__item">Seppä S. </li><li class="loa__item">Schiffmann E. </li><li class="loa__item">Martin G.R. </li></ul><div class="reference-citations__detials">Platelet-derived Growth Factor Is Chemotactic for Fibroblasts.<div class="popup-ref"><span class="ref__series"><i>J. Cell Biol.</i> <span class="ref__seriesDate">1982; </span><span class="ref__seriesVolume">92</span></span>: <span class="ref__seriesPages">584-588</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1083%2Fjcb.92.2.584&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=7061598&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=584-588&journal=J.%C2%A0Cell+Biol.&author=Sepp%C3%A4+H.&author=Grotendorst+G.&author=Sepp%C3%A4+S.&author=Schiffmann+E.&author=Martin+G.R.&title=Platelet-derived+Growth+Factor+Is+Chemotactic+for+Fibroblasts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Interestingly, <em>PD-L2</em> mRNA levels were elevated independent of <em>PD-L1</em> expression (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B), suggesting a CAF-mediated mechanism of T cell death via PD-L2 antigen presentation (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d218962e2625" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Lakins et al., 2018</a><div data-db-target-of="bib83-d218962e2625" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lakins M.A. </li><li class="loa__item">Ghorani E. </li><li class="loa__item">Munir H. </li><li class="loa__item">Martins C.P. </li><li class="loa__item">Shields J.D. </li></ul><div class="reference-citations__detials">Cancer-associated fibroblasts induce antigen-specific deletion of CD8 <sup>+</sup> T Cells to protect tumour cells.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">948</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-03347-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29507342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042940461&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=948&journal=Nat.+Commun.&author=Lakins+M.A.&author=Ghorani+E.&author=Munir+H.&author=Martins+C.P.&author=Shields+J.D.&title=Cancer-associated+fibroblasts+induce+antigen-specific+deletion+of+CD8+%2B+T+Cells+to+protect+tumour+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Independently, increased mRNA expression of <em>ENTPD1</em> (<em>CD39</em>) and <em>NT5E</em> (<em>CD73</em>) in the CD8<sup>−</sup> inflamed tumor subtype could signal an adenosine-rich TME (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d218962e2640" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Antonioli et al., 2013</a><div data-db-target-of="bib3-d218962e2640" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Antonioli L. </li><li class="loa__item">Pacher P. </li><li class="loa__item">Vizi E.S. </li><li class="loa__item">Haskó G. </li></ul><div class="reference-citations__detials">CD39 and CD73 in immunity and inflammation.<div class="popup-ref"><span class="ref__series"><i>Trends Mol. Med.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">355-367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23601906&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84878614399&amp;cf=" onclick="newWindow(this.href);return false">Scopus (427)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=355-367&journal=Trends+Mol.+Med.&author=Antonioli+L.&author=Pacher+P.&author=Vizi+E.S.&author=Hask%C3%B3+G.&title=CD39+and+CD73+in+immunity+and+inflammation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), further contributing to CD8<sup>+</sup> T cell exclusion in these tumors (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>D).</div><div class="section-paragraph">CD8<sup>−</sup> inflamed tumors and VEGF immune desert tumors shared several features including an elevated stromal score (t test adjusted p &lt; 0.05) and enrichment of endothelial cells (t test adjusted p &lt; 0.05 – VEGF immune desert) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Interestingly, these tumor subtypes had a higher frequency of chromosome 7 gain and lower frequency of chromosome 14 loss, with the latter feature inversely correlated with endothelial cell presence (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6C). Although angiogenic signaling was elevated in both subtypes (Fisher’s exact test adjusted p &lt; 0.05), angiogenesis and platelet degranulation were higher in CD8<sup>−</sup> inflamed tumors (Fisher’s exact test adjusted p &lt; 0.05) and corresponded to upregulation of the PDGF-related signaling pathway (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). In contrast, higher expression of SUMOylation (protein level) and Wnt/β-catenin, RAP1, and Notch signaling pathways (mRNA level) were observed in VEGF immune desert tumors and corresponded to the highest endothelial cell signature (t test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>E). The differential expression of VEGFR1 (FLT1) (t test adjusted p &lt; 0.05) and multiple angiogenic-related signaling pathways may be representative of distinct mechanisms of endothelial cell recruitment and tumor vasculature formation between the two subtypes (<div class="dropBlock reference-citations"><a href="#" id="back-bib15" data-db-target-for="bib15-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Birdsey et al., 2015</a><div data-db-target-of="bib15-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Birdsey G.M. </li><li class="loa__item">Shah A.V. </li><li class="loa__item">Dufton N. </li><li class="loa__item">Reynolds L.E. </li><li class="loa__item">Osuna Almagro L. </li><li class="loa__item">Yang Y. </li><li class="loa__item">Aspalter I.M. </li><li class="loa__item">Khan S.T. </li><li class="loa__item">Mason J.C. </li><li class="loa__item">Dejana E. </li><li>et al.</li></ul><div class="reference-citations__detials">The endothelial transcription factor ERG promotes vascular stability and growth through Wnt/β-catenin signaling.<div class="popup-ref"><span class="ref__series"><i>Dev. Cell.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">82-96</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25584796&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84922364577&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=82-96&journal=Dev.+Cell&author=Birdsey+G.M.&author=Shah+A.V.&author=Dufton+N.&author=Reynolds+L.E.&author=Osuna+Almagro+L.&author=Yang+Y.&author=Aspalter+I.M.&author=Khan+S.T.&author=Mason+J.C.&author=Dejana+E.&title=The+endothelial+transcription+factor+ERG+promotes+vascular+stability+and+growth+through+Wnt%2F%CE%B2-catenin+signaling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib20" data-db-target-for="bib20-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Cao, 2013</a><div data-db-target-of="bib20-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cao Y. </li></ul><div class="reference-citations__detials">Multifarious functions of PDGFs and PDGFRs in tumor growth and metastasis.<div class="popup-ref"><span class="ref__series"><i>Trends Mol. Med.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">460-473</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23773831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=460-473&journal=Trends+Mol.+Med.&author=Cao+Y.&title=Multifarious+functions+of+PDGFs+and+PDGFRs+in+tumor+growth+and+metastasis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib28" data-db-target-for="bib28-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Chrzanowska-Wodnicka, 2013</a><div data-db-target-of="bib28-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chrzanowska-Wodnicka M. </li></ul><div class="reference-citations__detials">Distinct functions for Rap1 signaling in vascular morphogenesis and dysfunction.<div class="popup-ref"><span class="ref__series"><i>Exp. Cell Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">319</span></span>: <span class="ref__seriesPages">2350-2359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.yexcr.2013.07.022&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23911990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84883210190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2013&pages=2350-2359&journal=Exp.+Cell+Res.&author=Chrzanowska-Wodnicka+M.&title=Distinct+functions+for+Rap1+signaling+in+vascular+morphogenesis+and+dysfunction">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib60" data-db-target-for="bib60-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Hellström et al., 2007</a><div data-db-target-of="bib60-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hellström M. </li><li class="loa__item">Phng L.K. </li><li class="loa__item">Hofmann J.J. </li><li class="loa__item">Wallgard E. </li><li class="loa__item">Coultas L. </li><li class="loa__item">Lindblom P. </li><li class="loa__item">Alva J. </li><li class="loa__item">Nilsson A.K. </li><li class="loa__item">Karlsson L. </li><li class="loa__item">Gaiano N. </li><li>et al.</li></ul><div class="reference-citations__detials">Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">445</span></span>: <span class="ref__seriesPages">776-780</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature05571&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17259973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33847046849&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1080)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=445&publication_year=2007&pages=776-780&journal=Nature&author=Hellstr%C3%B6m+M.&author=Phng+L.K.&author=Hofmann+J.J.&author=Wallgard+E.&author=Coultas+L.&author=Lindblom+P.&author=Alva+J.&author=Nilsson+A.K.&author=Karlsson+L.&author=Gaiano+N.&title=Dll4+signalling+through+Notch1+regulates+formation+of+tip+cells+during+angiogenesis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib191" data-db-target-for="bib191-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib191">Zhou et al., 2018</a><div data-db-target-of="bib191-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhou H.J. </li><li class="loa__item">Xu Z. </li><li class="loa__item">Wang Z. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Zhuang Z.W. </li><li class="loa__item">Simons M. </li><li class="loa__item">Min W. </li></ul><div class="reference-citations__detials">SUMOylation of VEGFR2 regulates its intracellular trafficking and pathological angiogenesis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">3303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-05812-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30120232&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85051730064&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3303&journal=Nat.+Commun.&author=Zhou+H.J.&author=Xu+Z.&author=Wang+Z.&author=Zhang+H.&author=Zhuang+Z.W.&author=Simons+M.&author=Min+W.&title=SUMOylation+of+VEGFR2+regulates+its+intracellular+trafficking+and+pathological+angiogenesis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib192" data-db-target-for="bib192-d218962e2680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib192">Zhu et al., 2017</a><div data-db-target-of="bib192-d218962e2680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhu X. </li><li class="loa__item">Ding S. </li><li class="loa__item">Qiu C. </li><li class="loa__item">Shi Y. </li><li class="loa__item">Song L. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Li J. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Sun Y. </li><li>et al.</li></ul><div class="reference-citations__detials">SUMOylation negatively regulates angiogenesis by targeting endothelial NOTCH signaling.<div class="popup-ref"><span class="ref__series"><i>Circ. Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">121</span></span>: <span class="ref__seriesPages">636-649</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1161%2FCIRCRESAHA.117.310696&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28760777&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85028807516&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2017&pages=636-649&journal=Circ.+Res.&author=Zhu+X.&author=Ding+S.&author=Qiu+C.&author=Shi+Y.&author=Song+L.&author=Wang+Y.&author=Wang+Y.&author=Li+J.&author=Wang+Y.&author=Sun+Y.&title=SUMOylation+negatively+regulates+angiogenesis+by+targeting+endothelial+NOTCH+signaling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Metabolic immune desert tumors displayed low immune, stromal, and microenvironment scores (t test adjusted p &lt; 0.05), an elevated <em>MYC</em> target mRNA signature (Fisher’s exact test adjusted p &lt; 0.05), increased mTOR signaling (Fisher’s exact test adjusted p &lt; 0.05), and a unique metabolic profile that included elevated mitochondrial, OXPHOS, glycolysis protein expression (Fisher’s exact test p &lt; 0.05), and <em>PKM</em> mRNA expression (t test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>E; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Previous reports have established a relationship between HIF-2α and MYC-induced transcription in renal carcinomas (<div class="dropBlock reference-citations"><a href="#" id="back-bib52" data-db-target-for="bib52-d218962e2699" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Gordan et al., 2007a</a><div data-db-target-of="bib52-d218962e2699" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gordan J.D. </li><li class="loa__item">Bertout J.A. </li><li class="loa__item">Hu C.J. </li><li class="loa__item">Diehl J.A. </li><li class="loa__item">Simon M.C. </li></ul><div class="reference-citations__detials">HIF-2α promotes hypoxic cell proliferation by enhancing c-myc transcriptional activity.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">335-347</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17418410&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34047156190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=335-347&journal=Cancer+Cell&author=Gordan+J.D.&author=Bertout+J.A.&author=Hu+C.J.&author=Diehl+J.A.&author=Simon+M.C.&title=HIF-2%CE%B1+promotes+hypoxic+cell+proliferation+by+enhancing+c-myc+transcriptional+activity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib54" data-db-target-for="bib54-d218962e2699" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Gordan et al., 2008</a><div data-db-target-of="bib54-d218962e2699" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gordan J.D. </li><li class="loa__item">Lal P. </li><li class="loa__item">Dondeti V.R. </li><li class="loa__item">Letrero R. </li><li class="loa__item">Parekh K.N. </li><li class="loa__item">Oquendo C.E. </li><li class="loa__item">Greenberg R.A. </li><li class="loa__item">Flaherty K.T. </li><li class="loa__item">Rathmell W.K. </li><li class="loa__item">Keith B. </li><li>et al.</li></ul><div class="reference-citations__detials">HIF-α effects on c-Myc distinguish two subtypes of sporadic VHL-deficient clear cell renal carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">435-446</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19061835&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-56849096837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=435-446&journal=Cancer+Cell&author=Gordan+J.D.&author=Lal+P.&author=Dondeti+V.R.&author=Letrero+R.&author=Parekh+K.N.&author=Oquendo+C.E.&author=Greenberg+R.A.&author=Flaherty+K.T.&author=Rathmell+W.K.&author=Keith+B.&title=HIF-%CE%B1+effects+on+c-Myc+distinguish+two+subtypes+of+sporadic+VHL-deficient+clear+cell+renal+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with MYC regulation of metabolism functioning in concert with and independent of HIF-1 signaling (<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d218962e2702" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Gordan et al., 2007b</a><div data-db-target-of="bib53-d218962e2702" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gordan J.D. </li><li class="loa__item">Thompson C.B. </li><li class="loa__item">Simon M.C. </li></ul><div class="reference-citations__detials">HIF and c-Myc: sibling rivals for control of cancer cell metabolism and proliferation.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">108-113</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17692803&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34547580590&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=108-113&journal=Cancer+Cell&author=Gordan+J.D.&author=Thompson+C.B.&author=Simon+M.C.&title=HIF+and+c-Myc%3A+sibling+rivals+for+control+of+cancer+cell+metabolism+and+proliferation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Further contributing to this unique metabolism signature was the increased expression of <em>PRDX4</em> (t test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B), which has been shown to impair the binding of HIF-1α/2α to the hypoxia response element in select glycolytic genes (<em>SLC2A3</em>, <em>PDK3</em>, <em>GPI</em>) (<div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d218962e2717" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">Luo et al., 2016</a><div data-db-target-of="bib95-d218962e2717" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Luo W. </li><li class="loa__item">Chen I. </li><li class="loa__item">Chen Y. </li><li class="loa__item">Alkam D. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Semenza G.L. </li></ul><div class="reference-citations__detials">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia.<div class="popup-ref"><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">6379-6397</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.18632%2Foncotarget.6395&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26837221&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84957675600&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6379-6397&journal=Oncotarget&author=Luo+W.&author=Chen+I.&author=Chen+Y.&author=Alkam+D.&author=Wang+Y.&author=Semenza+G.L.&title=PRDX2+and+PRDX4+are+negative+regulators+of+hypoxia-inducible+factors+under+conditions+of+prolonged+hypoxia">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The minimal presence of any immune cells in metabolic immune desert tumors is particularly striking and supports the hypothesis that a hypoxic, nutrient-poor microenvironment can be immunosuppressive (<div class="dropBlock reference-citations"><a href="#" id="back-bib2" data-db-target-for="bib2-d218962e2720" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Anderson et al., 2017</a><div data-db-target-of="bib2-d218962e2720" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anderson K.G. </li><li class="loa__item">Stromnes I.M. </li><li class="loa__item">Greenberg P.D. </li></ul><div class="reference-citations__detials">Obstacles Posed by the Tumor Microenvironment to T cell Activity: A Case for Synergistic Therapies.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">311-325</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28292435&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85015064348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (126)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=311-325&journal=Cancer+Cell&author=Anderson+K.G.&author=Stromnes+I.M.&author=Greenberg+P.D.&title=Obstacles+Posed+by+the+Tumor+Microenvironment+to+T%C2%A0cell+Activity%3A+A+Case+for+Synergistic+Therapies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib107" data-db-target-for="bib107-d218962e2720" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib107">Mgrditchian et al., 2014</a><div data-db-target-of="bib107-d218962e2720" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mgrditchian T. </li><li class="loa__item">Arakelian T. </li><li class="loa__item">Paggetti J. </li><li class="loa__item">Viry E. </li><li class="loa__item">Al-Absi A. </li><li class="loa__item">Medves S. </li><li class="loa__item">Moussay E. </li><li class="loa__item">Berchem G. </li><li class="loa__item">Thomas C. </li><li class="loa__item">Janji B. </li></ul><div class="reference-citations__detials">Hypoxia: A Key Feature of the Tumor Microenvironment Triggers Several Mechanisms of Evasion from Natural Killer and Cytotoxic T Lymphocytes Surveillance.<div class="popup-ref"><span class="ref__series"><i>J. Immun. Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=7&journal=J.%C2%A0Immun.+Res.&author=Mgrditchian+T.&author=Arakelian+T.&author=Paggetti+J.&author=Viry+E.&author=Al-Absi+A.&author=Medves+S.&author=Moussay+E.&author=Berchem+G.&author=Thomas+C.&author=Janji+B.&title=Hypoxia%3A+A+Key+Feature+of+the+Tumor+Microenvironment+Triggers+Several+Mechanisms+of+Evasion+from+Natural+Killer+and+Cytotoxic+T+Lymphocytes+Surveillance">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">This analysis discriminated four subtypes of ccRCC and linked unique cellular pathways to observed TME compositions, with select protein features validated using an orthogonal mass spectrometry approach (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A, 6E, and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>D; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). We hypothesized that the delineated molecular signatures may predict patient responses to select therapies and survival. To examine the former, we characterized tumors within each subtype using two gene signatures that have been previously associated with patient response to immune checkpoint and anti-VEGF therapies (T-effector [T<sub>eff</sub>] and angiogenesis [Angio]), respectively (<div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d218962e2737" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">McDermott et al., 2018</a><div data-db-target-of="bib101-d218962e2737" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McDermott D.F. </li><li class="loa__item">Huseni M.A. </li><li class="loa__item">Atkins M.B. </li><li class="loa__item">Motzer R.J. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Fong L. </li><li class="loa__item">Joseph R.W. </li><li class="loa__item">Pal S.K. </li><li class="loa__item">Reeves J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). CD8<sup>+</sup> inflamed tumors displayed an elevated T<sub>eff</sub> signature relative to other tumor types, while VEGF immune desert tumors displayed an elevated Angio signature (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>F). The remaining two subtypes displayed a minimal T<sub>eff</sub> signature but had a moderate Angio signature, suggesting a potential response to therapeutics targeting VEGF signaling. Leveraging the gene signatures from our subtypes, we explored the TCGA dataset and observed similar distribution patterns of tumor grade and CD8<sup>+</sup> T cell, endothelial, and fibroblast cell composition (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>D and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>G; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Patient stratification based on the four subtypes revealed that VEGF immune desert tumors were associated with improved patient survival, while CD8<sup>+</sup> Inflamed tumors were associated with poor patient outcome (log-rank test adjusted p &lt; 0.05) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>H). The latter result reflects the aggregation of multiple features in the CD8<sup>+</sup> Inflamed subtype that are considered as poor prognosticators in ccRCC, including higher frequency of <em>BAP1</em> mutations (chi-square test adjusted p &lt; 0.05), increased proportion of higher grade tumors, and increased <em>PD-1</em>/<em>PD-L1</em> expression (t test adjusted p &lt; 0.05). We did not detect an association of tumor mutational burden or neoantigen load with any of these immune subtypes (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>), confirming previous reports that indicate that these features do not correlate with ccRCC prognosis (<div class="dropBlock reference-citations"><a href="#" id="back-bib99" data-db-target-for="bib99-d218962e2785" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Matsushita et al., 2016</a><div data-db-target-of="bib99-d218962e2785" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matsushita H. </li><li class="loa__item">Sato Y. </li><li class="loa__item">Karasaki T. </li><li class="loa__item">Nakagawa T. </li><li class="loa__item">Kume H. </li><li class="loa__item">Ogawa S. </li><li class="loa__item">Homma Y. </li><li class="loa__item">Kakimi K. </li></ul><div class="reference-citations__detials">Neoantigen Load, Antigen Presentation Machinery, and Immune Signatures Determine Prognosis in Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Immunol. Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">463-471</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2326-6066.CIR-15-0225&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26980598&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=463-471&journal=Cancer+Immunol.+Res.&author=Matsushita+H.&author=Sato+Y.&author=Karasaki+T.&author=Nakagawa+T.&author=Kume+H.&author=Ogawa+S.&author=Homma+Y.&author=Kakimi+K.&title=Neoantigen+Load%2C+Antigen+Presentation+Machinery%2C+and+Immune+Signatures+Determine+Prognosis+in+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib101" data-db-target-for="bib101-d218962e2785" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">McDermott et al., 2018</a><div data-db-target-of="bib101-d218962e2785" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McDermott D.F. </li><li class="loa__item">Huseni M.A. </li><li class="loa__item">Atkins M.B. </li><li class="loa__item">Motzer R.J. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Fong L. </li><li class="loa__item">Joseph R.W. </li><li class="loa__item">Pal S.K. </li><li class="loa__item">Reeves J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteome Inter-Tumoral Heterogeneity of ccRCC</span></h3><div class="section-paragraph">Tumor grade is an important prognosticator in ccRCC, which is often associated with higher-stage and larger tumors (<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d218962e2793" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Ishigami et al., 2014</a><div data-db-target-of="bib71-d218962e2793" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ishigami K. </li><li class="loa__item">Leite L.V. </li><li class="loa__item">Pakalniskis M.G. </li><li class="loa__item">Lee D.K. </li><li class="loa__item">Holanda D.G. </li><li class="loa__item">Kuehn D.M. </li></ul><div class="reference-citations__detials">Tumor grade of clear cell renal cell carcinoma assessed by contrast-enhanced computed tomography.<div class="popup-ref"><span class="ref__series"><i>Springerplus.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">694</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F2193-1801-3-694&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25806147&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925254491&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=694&journal=Springerplus&author=Ishigami+K.&author=Leite+L.V.&author=Pakalniskis+M.G.&author=Lee+D.K.&author=Holanda+D.G.&author=Kuehn+D.M.&title=Tumor+grade+of+clear+cell+renal+cell+carcinoma+assessed+by+contrast-enhanced+computed+tomography">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Multivariate analysis integrating clinical features and CPTAC “omics” data found that numerous genes within each data type were associated with immune and stromal scores and with tumor grade (Benjamini-Hochberg adjusted p &lt; 0.10; <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). Further investigation of these genes revealed distinct pathways associated with tumor grade. In high-grade tumors, translation, mTOR signaling, and EMT were increased at the mRNA and protein levels. Other cellular pathways displayed disparate upregulation at the transcriptomic and proteomic levels. For example, increased mRNA levels were associated with cell-cycle regulation and DNA repair, while increased OXPHOS and <em>N</em>-linked glycosylation were detected only at the protein level (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">S7</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). In contrast, low-grade tumors displayed increased expression of receptor tyrosine kinase, RAS, MAPK, Notch, and RAP1 cell signaling pathways at both the mRNA and protein levels (Wilcoxon rank-sum test, Benjamini-Hochberg adjusted p &lt; 0.01; <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">S7</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). Low-grade tumors showed increased protein expression of transcription-related processes (e.g., chromatin reorganization) that was not captured at the transcript level (Wilcoxon rank-sum test, Benjamini-Hochberg adjusted p &lt; 0.01; <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). The divergence of cellular processes between high- and low-grade tumors reflects the disparate tumoral needs, with higher grade tumors upregulating mechanisms to adapt to increased genomic alterations and a changing tumor microenvironment.<div class="floatDisplay"><figure id="figs7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/c699e4e7-f531-4e1a-acef-a1d84943251e/figs7.jpg" alt="Figure thumbnail figs7" data-locator="figs7" data-high-res="/cms/attachment/3226f7a7-ed7a-4336-b91c-ab3e5700aafa/figs7_lrg.jpg" data-large-image="/cms/attachment/c699e4e7-f531-4e1a-acef-a1d84943251e/figs7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S7" class="label figure__label">Figure S7</span><span class="figure__title__text">Proteomic Inter-Tumor Heterogeneity of ccRCC and Associated Functional Pathways, Related to <a data-locator="fig7" aria-expanded="false" aria-controls="crosref2705" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig7">Figure 7</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Results from multivariate analysis association of multiple “omics” levels. RNA-based Immune Score, RNA-based Stromal Score, and grade associated with multiple data types (B) Cellular pathways (right) with positive (red) or negative (blue) associations (adjusted p &lt; 0.05) at protein or mRNA level (left). Heatmap of mRNA transcript expression associated with high and low grade tumors (center) (Benjamini-Hochberg adjusted p &lt; 0.05).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/c699e4e7-f531-4e1a-acef-a1d84943251e/figs7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/3226f7a7-ed7a-4336-b91c-ab3e5700aafa/figs7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=figs7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/3211cf16-9e2e-4706-a186-a567f0b95669/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/d6ba72fd-ac9f-49ac-bc81-fe5ba922ecf4/gr7_lrg.jpg" data-large-image="/cms/attachment/3211cf16-9e2e-4706-a186-a567f0b95669/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Proteomic Inter-Tumor Heterogeneity of ccRCC and Associated Functional Pathways</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Cellular pathways (right) with positive (red) or negative (blue) associations with grade (adjusted p &lt; 0.05) at protein or mRNA level (left). Heatmap of protein expression associated with high- and low-grade tumors (center) (Benjamini-Hochberg adjusted p &lt; 0.05).</div><div class="section-paragraph">(B) Heatmap of global proteomic abundances. For subtype identification, protein features (n = 3,567) were selected based on highest variance. Color indicates <em>Z</em> score of protein in each sample: red is increased, blue is decreased. Clinical and molecular features are indicated above the heatmap. Cluster-derived modules are annotated according to pathway enrichment using Hallmark Gene signature, REACTOME, and KEGG ontologies (adjusted p &lt; 0.05).</div><div class="section-paragraph">See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref2655" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2660" href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" target="_blank">Tables S1</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2665" href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" target="_blank">S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/3211cf16-9e2e-4706-a186-a567f0b95669/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/d6ba72fd-ac9f-49ac-bc81-fe5ba922ecf4/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)31123-7&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">To determine ccRCC inter-tumor heterogeneity of the CPTAC cohort, we constructed an unbiased proteomic grouping of ccRCC. Three major proteomic ccRCC groups emerged from this analysis (ccRCC1-3), which were discriminated by seven major protein clusters (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). Tumors in ccRCC2 had a higher degree of protein expression associated with innate immunity and platelet degranulation (adjusted p &lt; 0.05), while those in ccRCC3 displayed increased protein expression associated with glycolysis, mTOR signaling, and hypoxia (adjusted p &lt; 0.05). ccRCC2 and ccRCC3 were associated with lower tumor grade (p &lt; 0.01 and p &lt; 0.02, respectively), while only ccRCC2 was associated with lower stage (p &lt; 0.001). Tumors in ccRCC1 presented with higher grade (p &lt; 0.001) and stage (p &lt; 0.01), characterized by elevated adaptive immune response, <em>N</em>-linked glycosylation, OXPHOS protein expression, and fatty acid metabolism (adjusted p &lt; 0.05). Tumors in ccRCC3 displayed a higher frequency of <em>PBRM1</em> mutations (p &lt; 0.05), whereas those in ccRCC1 had a higher frequency of <em>BAP1</em> mutations (p &lt; 0.0001), CIMP<sup>+</sup> status (p &lt; 0.007), and genomic instability (p &lt; 0.0001) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B). As highlighted in our immune analysis, ccRCC tumors had variable TME compositions, with immune and stromal signatures impacting observed protein expression patterns (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A). The distribution of the immune subtypes across the proteomic groupings contributed to the discriminating gene clusters described in <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B that were also delineated in the immune-based subtyping of ccRCC (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B), capturing the more dominant molecular signatures of CD8<sup>+</sup> inflamed (interferon-γ signaling), CD8<sup>−</sup> inflamed (platelet degranulation), and VEGF immune desert (hypoxic signaling) tumors, which had a higher prevalence in ccRCC1 (p &lt; 5.0 e−07), ccRCC2 (p &lt; 6.0 e−05), and ccRCC3 (p &lt; 0.0001), respectively.</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0080" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">This comprehensive proteogenomic characterization of ccRCC provides insight into the differential impact of underlying genomic and epigenomic events on the transcriptome, proteome, and phosphoproteome. The integration of chromosome-level alterations and comparative, multi-level profiling of treatment-naive tumors and NATs connects genomic aberrations to active mechanisms that drive ccRCC tumorigenesis. The identification of a subset of ccRCC patients who display genomic instability could have clinical utility that warrants further investigation, as this group may have worse prognosis and benefit from continual surveillance post-treatment. The <em>trans</em>-effects of chromosome 3p fully capture the dysregulated VHL/HIF-1 axis that is a hallmark of ccRCC, while the widespread translocation events involving chromosome 3 observed in this study and others (<div class="dropBlock reference-citations"><a href="#" id="back-bib109" data-db-target-for="bib109-d218962e2873" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Mitchell et al., 2018</a><div data-db-target-of="bib109-d218962e2873" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mitchell T.J. </li><li class="loa__item">Turajlic S. </li><li class="loa__item">Rowan A. </li><li class="loa__item">Nicol D. </li><li class="loa__item">Farmery J.H.R. </li><li class="loa__item">O’Brien T. </li><li class="loa__item">Martincorena I. </li><li class="loa__item">Tarpey P. </li><li class="loa__item">Angelopoulos N. </li><li class="loa__item">Yates L.R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TRACERx Renal Consortium</span><br />Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">611-623</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044780018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=611-623&journal=Cell&author=TRACERx+Renal+Consortium&title=Timing+the+Landmark+Events+in+the+Evolution+of+Clear+Cell+Renal+Cell+Cancer%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib128" data-db-target-for="bib128-d218962e2873" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Pavlovich et al., 2003</a><div data-db-target-of="bib128-d218962e2873" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pavlovich C.P. </li><li class="loa__item">Padilla-Nash H. </li><li class="loa__item">Wangsa D. </li><li class="loa__item">Nickerson M.L. </li><li class="loa__item">Matrosova V. </li><li class="loa__item">Linehan W.M. </li><li class="loa__item">Ried T. </li><li class="loa__item">Phillips J.L. </li></ul><div class="reference-citations__detials">Patterns of aneuploidy in stage IV clear cell renal cell carcinoma revealed by comparative genomic hybridization and spectral karyotyping.<div class="popup-ref"><span class="ref__series"><i>Genes Chromosomes Cancer.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">252-260</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fgcc.10209&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12759923&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038546730&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2003&pages=252-260&journal=Genes+Chromosomes+Cancer&author=Pavlovich+C.P.&author=Padilla-Nash+H.&author=Wangsa+D.&author=Nickerson+M.L.&author=Matrosova+V.&author=Linehan+W.M.&author=Ried+T.&author=Phillips+J.L.&title=Patterns+of+aneuploidy+in+stage+IV+clear+cell+renal+cell+carcinoma+revealed+by+comparative+genomic+hybridization+and+spectral+karyotyping">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) portray ccRCC as a disorder defined by genomic rearrangements.</div><div class="section-paragraph">Our analysis supports and elaborates on the metabolic shift that occurs within ccRCC tumors, illustrated at the protein level by the upregulation of glycolysis and the corresponding downregulation of the Krebs cycle and the electron transport chain (OXPHOS) associated with the Warburg effect. Notably, the downregulation of components of the Krebs cycle and the majority of the nuclear-encoded OXPHOS proteins were not observed at the mRNA level and would have not been reported by RNA-seq analysis alone. This finding is significant, as recent large-scale ccRCC studies have focused on mRNA expression data to depict the metabolic shift in ccRCC and have evaluated transcriptomic signatures to stratify patients with more aggressive disease (<div class="dropBlock reference-citations"><a href="#" id="back-bib25" data-db-target-for="bib25-d218962e2878" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Chen et al., 2016a</a><div data-db-target-of="bib25-d218962e2878" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen F. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Şenbabaoğlu Y. </li><li class="loa__item">Ciriello G. </li><li class="loa__item">Yang L. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Shuch B. </li><li class="loa__item">Micevic G. </li><li class="loa__item">De Velasco G. </li><li class="loa__item">Shinbrot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Multilevel Genomics-Based Taxonomy of Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2476-2489</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26947078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2476-2489&journal=Cell+Rep.&author=Chen+F.&author=Zhang+Y.&author=%C5%9Eenbabao%C4%9Flu+Y.&author=Ciriello+G.&author=Yang+L.&author=Reznik+E.&author=Shuch+B.&author=Micevic+G.&author=De+Velasco+G.&author=Shinbrot+E.&title=Multilevel+Genomics-Based+Taxonomy+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib31" data-db-target-for="bib31-d218962e2878" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e2878" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). HIF1-signaling has been shown to decrease OXPHOS activity through multiple mechanisms (<div class="dropBlock reference-citations"><a href="#" id="back-bib61" data-db-target-for="bib61-d218962e2881" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Hervouet et al., 2008</a><div data-db-target-of="bib61-d218962e2881" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hervouet E. </li><li class="loa__item">Cízková A. </li><li class="loa__item">Demont J. </li><li class="loa__item">Vojtísková A. </li><li class="loa__item">Pecina P. </li><li class="loa__item">Franssen-van Hal N.L.W. </li><li class="loa__item">Keijer J. </li><li class="loa__item">Simonnet H. </li><li class="loa__item">Ivánek R. </li><li class="loa__item">Kmoch S. </li><li>et al.</li></ul><div class="reference-citations__detials">HIF and reactive oxygen species regulate oxidative phosphorylation in cancer.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">1528-1537</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fcarcin%2Fbgn125&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18515279&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-49649107272&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1528-1537&journal=Carcinogenesis&author=Hervouet+E.&author=C%C3%ADzkov%C3%A1+A.&author=Demont+J.&author=Vojt%C3%ADskov%C3%A1+A.&author=Pecina+P.&author=Franssen-van+Hal+N.L.W.&author=Keijer+J.&author=Simonnet+H.&author=Iv%C3%A1nek+R.&author=Kmoch+S.&title=HIF+and+reactive+oxygen+species+regulate+oxidative+phosphorylation+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib126" data-db-target-for="bib126-d218962e2881" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Papandreou et al., 2006</a><div data-db-target-of="bib126-d218962e2881" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Papandreou I. </li><li class="loa__item">Cairns R.A. </li><li class="loa__item">Fontana L. </li><li class="loa__item">Lim A.L. </li><li class="loa__item">Denko N.C. </li></ul><div class="reference-citations__detials">HIF-1 mediates adaptation to hypoxia by actively downregulating mitochondrial oxygen consumption.<div class="popup-ref"><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">187-197</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16517406&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33644622570&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1259)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=187-197&journal=Cell+Metab.&author=Papandreou+I.&author=Cairns+R.A.&author=Fontana+L.&author=Lim+A.L.&author=Denko+N.C.&title=HIF-1+mediates+adaptation+to+hypoxia+by+actively+downregulating+mitochondrial+oxygen+consumption">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Interestingly, our proteomic analysis and previous metabolic profiling of ccRCC show evidence of late-stage tumors upregulating the OXPHOS pathway relative to earlier-stage tumors (<div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d218962e2884" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Hakimi et al., 2016</a><div data-db-target-of="bib56-d218962e2884" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hakimi A.A. </li><li class="loa__item">Reznik E. </li><li class="loa__item">Lee C.H. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">Brannon A.R. </li><li class="loa__item">Luna A. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Liu E.M. </li><li class="loa__item">Shen R. </li><li class="loa__item">Lee W. </li><li>et al.</li></ul><div class="reference-citations__detials">An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">104-116</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26766592&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84958749958&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=104-116&journal=Cancer+Cell&author=Hakimi+A.A.&author=Reznik+E.&author=Lee+C.H.&author=Creighton+C.J.&author=Brannon+A.R.&author=Luna+A.&author=Aksoy+B.A.&author=Liu+E.M.&author=Shen+R.&author=Lee+W.&title=An+Integrated+Metabolic+Atlas+of+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and may reflect the dysregulation of HIF-1α expression resulting from 14q loss or the aberrant methylation profiles associated with CIMP<sup>+</sup> status. The maintenance of OXPHOS transcription levels similar to those seen in normoxic cells may provide a mechanism for rapid induction of OXPHOS activity when it is advantageous to fulfill tumor energy requirements. This hypothesis warrants deeper exploration and expanded investigation in other cancer types.</div><div class="section-paragraph">Current first-line therapies for advanced ccRCC target VEGF and mTOR (<div class="dropBlock reference-citations"><a href="#" id="back-bib41" data-db-target-for="bib41-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">Escudier et al., 2007a</a><div data-db-target-of="bib41-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Escudier B. </li><li class="loa__item">Eisen T. </li><li class="loa__item">Stadler W.M. </li><li class="loa__item">Szczylik C. </li><li class="loa__item">Oudard S. </li><li class="loa__item">Siebels M. </li><li class="loa__item">Negrier S. </li><li class="loa__item">Chevreau C. </li><li class="loa__item">Solska E. </li><li class="loa__item">Desai A.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>TARGET Study Group</span><br />Sorafenib in advanced clear-cell renal-cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">125-134</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa060655&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17215530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33846148701&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3992)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=125-134&journal=N.%C2%A0Engl.+J.+Med.&author=TARGET+Study+Group&title=Sorafenib+in+advanced+clear-cell+renal-cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib42" data-db-target-for="bib42-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Escudier et al., 2007b</a><div data-db-target-of="bib42-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Escudier B. </li><li class="loa__item">Pluzanska A. </li><li class="loa__item">Koralewski P. </li><li class="loa__item">Ravaud A. </li><li class="loa__item">Bracarda S. </li><li class="loa__item">Szczylik C. </li><li class="loa__item">Chevreau C. </li><li class="loa__item">Filipek M. </li><li class="loa__item">Melichar B. </li><li class="loa__item">Bajetta E. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>AVOREN Trial investigators</span><br />Bevacizumab plus interferon alfa-2a for treatment of metastatic renal cell carcinoma: a randomised, double-blind phase III trial.<div class="popup-ref"><span class="ref__series"><i>Lancet.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">370</span></span>: <span class="ref__seriesPages">2103-2111</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18156031&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-37349080670&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1872)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2007&pages=2103-2111&journal=Lancet&author=AVOREN+Trial+investigators&title=Bevacizumab+plus+interferon+alfa-2a+for+treatment+of+metastatic+renal+cell+carcinoma%3A+a+randomised%2C+double-blind+phase+III+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib70" data-db-target-for="bib70-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Hudes et al., 2007</a><div data-db-target-of="bib70-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hudes G. </li><li class="loa__item">Carducci M. </li><li class="loa__item">Tomczak P. </li><li class="loa__item">Dutcher J. </li><li class="loa__item">Figlin R. </li><li class="loa__item">Kapoor A. </li><li class="loa__item">Staroslawska E. </li><li class="loa__item">Sosman J. </li><li class="loa__item">McDermott D. </li><li class="loa__item">Bodrogi I. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Global ARCC Trial</span><br />Temsirolimus, interferon alfa, or both for advanced renal-cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">2271-2281</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa066838&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17538086&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34249779568&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2992)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=2271-2281&journal=N.%C2%A0Engl.+J.+Med.&author=Global+ARCC+Trial&title=Temsirolimus%2C+interferon+alfa%2C+or+both+for+advanced+renal-cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib114" data-db-target-for="bib114-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Motzer et al., 2007</a><div data-db-target-of="bib114-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Motzer R.J. </li><li class="loa__item">Hutson T.E. </li><li class="loa__item">Tomczak P. </li><li class="loa__item">Michaelson M.D. </li><li class="loa__item">Bukowski R.M. </li><li class="loa__item">Rixe O. </li><li class="loa__item">Oudard S. </li><li class="loa__item">Negrier S. </li><li class="loa__item">Szczylik C. </li><li class="loa__item">Kim S.T. </li><li>et al.</li></ul><div class="reference-citations__detials">Sunitinib versus interferon alfa in metastatic renal-cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">115-124</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa065044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17215529&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33846181370&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4540)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=115-124&journal=N.%C2%A0Engl.+J.+Med.&author=Motzer+R.J.&author=Hutson+T.E.&author=Tomczak+P.&author=Michaelson+M.D.&author=Bukowski+R.M.&author=Rixe+O.&author=Oudard+S.&author=Negrier+S.&author=Szczylik+C.&author=Kim+S.T.&title=Sunitinib+versus+interferon+alfa+in+metastatic+renal-cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib115" data-db-target-for="bib115-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib115">Motzer et al., 2008</a><div data-db-target-of="bib115-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Motzer R.J. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Oudard S. </li><li class="loa__item">Hutson T.E. </li><li class="loa__item">Porta C. </li><li class="loa__item">Bracarda S. </li><li class="loa__item">Grünwald V. </li><li class="loa__item">Thompson J.A. </li><li class="loa__item">Figlin R.A. </li><li class="loa__item">Hollaender N. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>RECORD-1 Study Group</span><br />Efficacy of everolimus in advanced renal cell carcinoma: a double-blind, randomised, placebo-controlled phase III trial.<div class="popup-ref"><span class="ref__series"><i>Lancet.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">372</span></span>: <span class="ref__seriesPages">449-456</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18653228&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-48649107474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2384)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2008&pages=449-456&journal=Lancet&author=RECORD-1+Study+Group&title=Efficacy+of+everolimus+in+advanced+renal+cell+carcinoma%3A+a+double-blind%2C+randomised%2C+placebo-controlled+phase+III+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib162" data-db-target-for="bib162-d218962e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib162">Sternberg et al., 2010</a><div data-db-target-of="bib162-d218962e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sternberg C.N. </li><li class="loa__item">Davis I.D. </li><li class="loa__item">Mardiak J. </li><li class="loa__item">Szczylik C. </li><li class="loa__item">Lee E. </li><li class="loa__item">Wagstaff J. </li><li class="loa__item">Barrios C.H. </li><li class="loa__item">Salman P. </li><li class="loa__item">Gladkov O.A. </li><li class="loa__item">Kavina A. </li><li>et al.</li></ul><div class="reference-citations__detials">Pazopanib in locally advanced or metastatic renal cell carcinoma: results of a randomized phase III trial.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1061-1068</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2009.23.9764&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20100962&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77949890945&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1792)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1061-1068&journal=J.%C2%A0Clin.+Oncol.&author=Sternberg+C.N.&author=Davis+I.D.&author=Mardiak+J.&author=Szczylik+C.&author=Lee+E.&author=Wagstaff+J.&author=Barrios+C.H.&author=Salman+P.&author=Gladkov+O.A.&author=Kavina+A.&title=Pazopanib+in+locally+advanced+or+metastatic+renal+cell+carcinoma%3A+results+of+a+randomized+phase+III+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with ongoing exploration of immune checkpoint inhibitors (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d218962e2895" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Atkins and Tannir, 2018</a><div data-db-target-of="bib8-d218962e2895" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Atkins M.B. </li><li class="loa__item">Tannir N.M. </li></ul><div class="reference-citations__detials">Current and emerging therapies for first-line treatment of metastatic clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Treat. Rev.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">127-137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=abstract&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=fulltext&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=pdf&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30173085&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85052524008&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=127-137&journal=Cancer+Treat.+Rev.&author=Atkins+M.B.&author=Tannir+N.M.&title=Current+and+emerging+therapies+for+first-line+treatment+of+metastatic+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Through investigation of transcriptomic, proteomic, and phosphoproteomic signatures of treatment-naive tumors, we propose a rational stratification of ccRCC patients for personalized therapeutic interventions. Immune-based subtyping of ccRCC tumors via deconvolution of cell composition identified TME and cellular pathways that delineated patients who displayed a pro-angiogenic phenotype from those with an immune-evasive phenotype. With evidence demonstrating the lack of efficacy of anti-VEGF therapy in patients with elevated levels of immune checkpoint signatures (<div class="dropBlock reference-citations"><a href="#" id="back-bib58" data-db-target-for="bib58-d218962e2898" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Hara et al., 2017</a><div data-db-target-of="bib58-d218962e2898" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hara T. </li><li class="loa__item">Miyake H. </li><li class="loa__item">Fujisawa M. </li></ul><div class="reference-citations__detials">Expression pattern of immune checkpoint-associated molecules in radical nephrectomy specimens as a prognosticator in patients with metastatic renal cell carcinoma treated with tyrosine kinase inhibitors.<div class="popup-ref"><span class="ref__series"><i>Urol. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">363-369</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.urolonc.2017.01.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28169111&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85011333471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=363-369&journal=Urol.+Oncol.&author=Hara+T.&author=Miyake+H.&author=Fujisawa+M.&title=Expression+pattern+of+immune+checkpoint-associated+molecules+in+radical+nephrectomy+specimens+as+a+prognosticator+in+patients+with+metastatic+renal+cell+carcinoma+treated+with+tyrosine+kinase+inhibitors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib155" data-db-target-for="bib155-d218962e2898" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib155">Shin et al., 2015</a><div data-db-target-of="bib155-d218962e2898" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shin S.-J. </li><li class="loa__item">Jeon Y.K. </li><li class="loa__item">Cho Y.M. </li><li class="loa__item">Lee J.-L. </li><li class="loa__item">Chung D.H. </li><li class="loa__item">Park J.Y. </li><li class="loa__item">Go H. </li></ul><div class="reference-citations__detials">The Association Between PD-L1 Expression and the Clinical Outcomes to Vascular Endothelial Growth Factor-Targeted Therapy in Patients With Metastatic Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Oncologist.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1253-1260</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1634%2Ftheoncologist.2015-0151&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26424759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1253-1260&journal=Oncologist&author=Shin+S.-J.&author=Jeon+Y.K.&author=Cho+Y.M.&author=Lee+J.-L.&author=Chung+D.H.&author=Park+J.Y.&author=Go+H.&title=The+Association+Between+PD-L1+Expression+and+the+Clinical+Outcomes+to+Vascular+Endothelial+Growth+Factor-Targeted+Therapy+in+Patients+With+Metastatic+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and preliminary clinical studies showing encouraging results when treating RCC using PD-1/CTLA4-targeting therapies (<div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d218962e2901" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">Motzer et al., 2018</a><div data-db-target-of="bib116-d218962e2901" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Motzer R.J. </li><li class="loa__item">Tannir N.M. </li><li class="loa__item">McDermott D.F. </li><li class="loa__item">Arén Frontera O. </li><li class="loa__item">Melichar B. </li><li class="loa__item">Choueiri T.K. </li><li class="loa__item">Plimack E.R. </li><li class="loa__item">Barthélémy P. </li><li class="loa__item">Porta C. </li><li class="loa__item">George S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CheckMate 214 Investigators</span><br />Nivolumab plus Ipilimumab versus Sunitinib in Advanced Renal-Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">378</span></span>: <span class="ref__seriesPages">1277-1290</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa1712126&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29562145&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045136401&amp;cf=" onclick="newWindow(this.href);return false">Scopus (914)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2018&pages=1277-1290&journal=N.%C2%A0Engl.+J.+Med.&author=CheckMate+214+Investigators&title=Nivolumab+plus+Ipilimumab+versus+Sunitinib+in+Advanced+Renal-Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), it is vital to identify theranostic markers to improve patient outcome and overall survival. However, mechanisms that drive the observed differences in TME signatures warrant further exploration. Our results support recent reports that tumor mutational burden or neoantigen load may not be associated with immune infiltration or response to immune checkpoint therapy in ccRCC (<div class="dropBlock reference-citations"><a href="#" id="back-bib99" data-db-target-for="bib99-d218962e2904" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Matsushita et al., 2016</a><div data-db-target-of="bib99-d218962e2904" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matsushita H. </li><li class="loa__item">Sato Y. </li><li class="loa__item">Karasaki T. </li><li class="loa__item">Nakagawa T. </li><li class="loa__item">Kume H. </li><li class="loa__item">Ogawa S. </li><li class="loa__item">Homma Y. </li><li class="loa__item">Kakimi K. </li></ul><div class="reference-citations__detials">Neoantigen Load, Antigen Presentation Machinery, and Immune Signatures Determine Prognosis in Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Immunol. Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">463-471</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2326-6066.CIR-15-0225&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26980598&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=463-471&journal=Cancer+Immunol.+Res.&author=Matsushita+H.&author=Sato+Y.&author=Karasaki+T.&author=Nakagawa+T.&author=Kume+H.&author=Ogawa+S.&author=Homma+Y.&author=Kakimi+K.&title=Neoantigen+Load%2C+Antigen+Presentation+Machinery%2C+and+Immune+Signatures+Determine+Prognosis+in+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib101" data-db-target-for="bib101-d218962e2904" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">McDermott et al., 2018</a><div data-db-target-of="bib101-d218962e2904" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McDermott D.F. </li><li class="loa__item">Huseni M.A. </li><li class="loa__item">Atkins M.B. </li><li class="loa__item">Motzer R.J. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Fong L. </li><li class="loa__item">Joseph R.W. </li><li class="loa__item">Pal S.K. </li><li class="loa__item">Reeves J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), although alternative explanations, such as endogenous retroviral expression, were not examined in our study (<div class="dropBlock reference-citations"><a href="#" id="back-bib125" data-db-target-for="bib125-d218962e2908" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Panda et al., 2018</a><div data-db-target-of="bib125-d218962e2908" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Panda A. </li><li class="loa__item">de Cubas A.A. </li><li class="loa__item">Stein M. </li><li class="loa__item">Riedlinger G. </li><li class="loa__item">Kra J. </li><li class="loa__item">Mayer T. </li><li class="loa__item">Smith C.C. </li><li class="loa__item">Vincent B.G. </li><li class="loa__item">Serody J.S. </li><li class="loa__item">Beckermann K.E. </li><li>et al.</li></ul><div class="reference-citations__detials">Endogenous retrovirus expression is associated with response to immune checkpoint blockade in clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>JCI Insight.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">121522</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1172%2Fjci.insight.121522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30135306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85062245416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=121522&journal=JCI+Insight&author=Panda+A.&author=de+Cubas+A.A.&author=Stein+M.&author=Riedlinger+G.&author=Kra+J.&author=Mayer+T.&author=Smith+C.C.&author=Vincent+B.G.&author=Serody+J.S.&author=Beckermann+K.E.&title=Endogenous+retrovirus+expression+is+associated+with+response+to+immune+checkpoint+blockade+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib158" data-db-target-for="bib158-d218962e2908" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib158">Smith et al., 2018</a><div data-db-target-of="bib158-d218962e2908" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Smith C.C. </li><li class="loa__item">Beckermann K.E. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">De Cubas A.A. </li><li class="loa__item">Bixby L.M. </li><li class="loa__item">Lee S.J. </li><li class="loa__item">Panda A. </li><li class="loa__item">Ganesan S. </li><li class="loa__item">Bhanot G. </li><li class="loa__item">Wallen E.M. </li><li>et al.</li></ul><div class="reference-citations__detials">Endogenous retroviral signatures predict immunotherapy response in clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Invest.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">128</span></span>: <span class="ref__seriesPages">4804-4820</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1172%2FJCI121476&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30137025&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053394932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=4804-4820&journal=J.%C2%A0Clin.+Invest.&author=Smith+C.C.&author=Beckermann+K.E.&author=Bortone+D.S.&author=De+Cubas+A.A.&author=Bixby+L.M.&author=Lee+S.J.&author=Panda+A.&author=Ganesan+S.&author=Bhanot+G.&author=Wallen+E.M.&title=Endogenous+retroviral+signatures+predict+immunotherapy+response+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Independent of current first-line regimens and immune checkpoint inhibition, the ubiquitous activation of EGFR and downstream signaling cascades (MAPK1), as well as cell-cycle checkpoint regulation (WEE1-CDK1) revealed by our phosphoproteomic analysis, provide additional therapeutic targets that have been evaluated extensively in other cancer types but minimally in ccRCC (<div class="dropBlock reference-citations"><a href="#" id="back-bib7" data-db-target-for="bib7-d218962e2911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Ascierto et al., 2013</a><div data-db-target-of="bib7-d218962e2911" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ascierto P.A. </li><li class="loa__item">Schadendorf D. </li><li class="loa__item">Berking C. </li><li class="loa__item">Agarwala S.S. </li><li class="loa__item">van Herpen C.M.L. </li><li class="loa__item">Queirolo P. </li><li class="loa__item">Blank C.U. </li><li class="loa__item">Hauschild A. </li><li class="loa__item">Beck J.T. </li><li class="loa__item">St-Pierre A. </li><li>et al.</li></ul><div class="reference-citations__detials">MEK162 for patients with advanced melanoma harbouring NRAS or Val600 BRAF mutations: a non-randomised, open-label phase 2 study.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">249-256</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23414587&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84875235432&amp;cf=" onclick="newWindow(this.href);return false">Scopus (430)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=249-256&journal=Lancet+Oncol.&author=Ascierto+P.A.&author=Schadendorf+D.&author=Berking+C.&author=Agarwala+S.S.&author=van+Herpen+C.M.L.&author=Queirolo+P.&author=Blank+C.U.&author=Hauschild+A.&author=Beck+J.T.&author=St-Pierre+A.&title=MEK162+for+patients+with+advanced+melanoma+harbouring+NRAS+or+Val600+BRAF+mutations%3A+a+non-randomised%2C+open-label+phase+2+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib68" data-db-target-for="bib68-d218962e2911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Huang et al., 2008</a><div data-db-target-of="bib68-d218962e2911" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang D. </li><li class="loa__item">Ding Y. </li><li class="loa__item">Luo W.M. </li><li class="loa__item">Bender S. </li><li class="loa__item">Qian C.N. </li><li class="loa__item">Kort E. </li><li class="loa__item">Zhang Z.F. </li><li class="loa__item">VandenBeldt K. </li><li class="loa__item">Duesbery N.S. </li><li class="loa__item">Resau J.H. </li><li class="loa__item">Teh B.T. </li></ul><div class="reference-citations__detials">Inhibition of MAPK kinase signaling pathways suppressed renal cell carcinoma growth and angiogenesis in vivo.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">81-88</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-07-5311&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18172299&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-39149099155&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=81-88&journal=Cancer+Res.&author=Huang+D.&author=Ding+Y.&author=Luo+W.M.&author=Bender+S.&author=Qian+C.N.&author=Kort+E.&author=Zhang+Z.F.&author=VandenBeldt+K.&author=Duesbery+N.S.&author=Resau+J.H.&author=Teh+B.T.&title=Inhibition+of+MAPK+kinase+signaling+pathways+suppressed+renal+cell+carcinoma+growth+and+angiogenesis+in%C2%A0vivo">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib98" data-db-target-for="bib98-d218962e2911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Matheson et al., 2016</a><div data-db-target-of="bib98-d218962e2911" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matheson C.J. </li><li class="loa__item">Backos D.S. </li><li class="loa__item">Reigan P. </li></ul><div class="reference-citations__detials">Targeting WEE1 Kinase in Cancer.<div class="popup-ref"><span class="ref__series"><i>Trends Pharmacol. Sci.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">872-881</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27427153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84978879508&amp;cf=" onclick="newWindow(this.href);return false">Scopus (97)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=872-881&journal=Trends+Pharmacol.+Sci.&author=Matheson+C.J.&author=Backos+D.S.&author=Reigan+P.&title=Targeting+WEE1+Kinase+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib139" data-db-target-for="bib139-d218962e2911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib139">Ravaud et al., 2008</a><div data-db-target-of="bib139-d218962e2911" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ravaud A. </li><li class="loa__item">Hawkins R. </li><li class="loa__item">Gardner J.P. </li><li class="loa__item">von der Maase H. </li><li class="loa__item">Zantl N. </li><li class="loa__item">Harper P. </li><li class="loa__item">Rolland F. </li><li class="loa__item">Audhuy B. </li><li class="loa__item">Machiels J.P. </li><li class="loa__item">Pétavy F. </li><li>et al.</li></ul><div class="reference-citations__detials">Lapatinib versus hormone therapy in patients with advanced renal cell carcinoma: a randomized phase III clinical trial.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">2285-2291</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2007.14.5029&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18467719&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-44249127263&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=2285-2291&journal=J.%C2%A0Clin.+Oncol.&author=Ravaud+A.&author=Hawkins+R.&author=Gardner+J.P.&author=von+der+Maase+H.&author=Zantl+N.&author=Harper+P.&author=Rolland+F.&author=Audhuy+B.&author=Machiels+J.P.&author=P%C3%A9tavy+F.&title=Lapatinib+versus+hormone+therapy+in+patients+with+advanced+renal+cell+carcinoma%3A+a+randomized+phase+III+clinical+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Application to ccRCC would be especially important given our identification of a subset of ccRCC tumors that are predicted or shown to be immune checkpoint/VEGF non-responders (<div class="dropBlock reference-citations"><a href="#" id="back-bib14" data-db-target-for="bib14-d218962e2914" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Beuselinck et al., 2015</a><div data-db-target-of="bib14-d218962e2914" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beuselinck B. </li><li class="loa__item">Job S. </li><li class="loa__item">Becht E. </li><li class="loa__item">Karadimou A. </li><li class="loa__item">Verkarre V. </li><li class="loa__item">Couchy G. </li><li class="loa__item">Giraldo N. </li><li class="loa__item">Rioux-Leclercq N. </li><li class="loa__item">Molinié V. </li><li class="loa__item">Sibony M. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular subtypes of clear cell renal cell carcinoma are associated with sunitinib response in the metastatic setting.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1329-1339</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F1078-0432.CCR-14-1128&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25583177&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84927592892&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1329-1339&journal=Clin.+Cancer+Res.&author=Beuselinck+B.&author=Job+S.&author=Becht+E.&author=Karadimou+A.&author=Verkarre+V.&author=Couchy+G.&author=Giraldo+N.&author=Rioux-Leclercq+N.&author=Molini%C3%A9+V.&author=Sibony+M.&title=Molecular+subtypes+of+clear+cell+renal+cell+carcinoma+are+associated+with+sunitinib+response+in+the+metastatic+setting">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib97" data-db-target-for="bib97-d218962e2914" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Maroto et al., 2017</a><div data-db-target-of="bib97-d218962e2914" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maroto P. </li><li class="loa__item">Esteban E. </li><li class="loa__item">Parra E.F. </li><li class="loa__item">Mendez-Vidal M.J. </li><li class="loa__item">Domenech M. </li><li class="loa__item">Pérez-Valderrama B. </li><li class="loa__item">Calderero V. </li><li class="loa__item">Pérez-Gracia J.L. </li><li class="loa__item">Grande E. </li><li class="loa__item">Algaba F. </li></ul><div class="reference-citations__detials">HIF pathway and c-Myc as biomarkers for response to sunitinib in metastatic clear-cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>OncoTargets Ther.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">4635-4643</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.2147%2FOTT.S137677&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29033582&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85030245220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=4635-4643&journal=OncoTargets+Ther.&author=Maroto+P.&author=Esteban+E.&author=Parra+E.F.&author=Mendez-Vidal+M.J.&author=Domenech+M.&author=P%C3%A9rez-Valderrama+B.&author=Calderero+V.&author=P%C3%A9rez-Gracia+J.L.&author=Grande+E.&author=Algaba+F.&title=HIF+pathway+and+c-Myc+as+biomarkers+for+response+to+sunitinib+in+metastatic+clear-cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) that may benefit from therapies that activate anti-tumor T cell expansion (<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d218962e2917" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Naing et al., 2018</a><div data-db-target-of="bib117-d218962e2917" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Naing A. </li><li class="loa__item">Infante J.R. </li><li class="loa__item">Papadopoulos K.P. </li><li class="loa__item">Chan I.H. </li><li class="loa__item">Shen C. </li><li class="loa__item">Ratti N.P. </li><li class="loa__item">Rojo B. </li><li class="loa__item">Autio K.A. </li><li class="loa__item">Wong D.J. </li><li class="loa__item">Patel M.R. </li><li>et al.</li></ul><div class="reference-citations__detials">PEGylated IL-10 (Pegilodecakin) Induces Systemic Immune Activation, CD8<sup>+</sup> T Cell Invigoration and Polyclonal T Cell Expansion in Cancer Patients.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">775-791.e3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30423297&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85055890244&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=775-791.e3&journal=Cancer+Cell&author=Naing+A.&author=Infante+J.R.&author=Papadopoulos+K.P.&author=Chan+I.H.&author=Shen+C.&author=Ratti+N.P.&author=Rojo+B.&author=Autio+K.A.&author=Wong+D.J.&author=Patel+M.R.&title=PEGylated+IL-10+%28Pegilodecakin%29+Induces+Systemic+Immune+Activation%2C+CD8%2B+T+Cell+Invigoration+and+Polyclonal+T+Cell+Expansion+in+Cancer+Patients">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) or combinatorial therapeutic approaches, such as concurrent cell-cycle checkpoint and mTOR inhibition.</div><div class="section-paragraph">Overall, this study reveals unique biological insights that are gained only when combining complementary proteomic and genomic analyses that link the functional consequences of genomic aberrations with proteomic outcomes. The integration of comprehensive genomic, epigenomic, transcriptomic, proteomic, and phosphoproteomic measurements for tumors and corresponding NATs provides an invaluable bioinformatic resource for the deeper examination of ccRCC tumorigenesis. Our multi-level “omics” analysis identifies underlying molecular mechanisms that are not fully captured at the genomic and transcriptomic levels and defines proteomic, phosphoproteomic, and immune signatures necessary to stratify ccRCC patients, with the goal of developing rational therapeutic interventions.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec5.1" data-locator="sec5.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key Resources Table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Antibodies</strong></td></tr><tr><td>Mouse monoclonal anti-CD8 (C8/144B)</td><td>Cellmarque</td><td>Catalog #108M-96; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_1158208" target="_blank">AB_1158208</a></td></tr><tr><td>Rabbit monoclonal anti-CD4 (SP35)</td><td>Roche</td><td>Catalog #790-4423</td></tr><tr><td class="rowsep">Liquid Concentrated Monoclonal Antibody anti-CD163</td><td class="rowsep">Leica Biosystems</td><td class="rowsep">Catalog #NCL-L-163; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2756375" target="_blank">AB_2756375</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Biological Samples</strong></td></tr><tr><td class="rowsep">Primary tumor samples</td><td class="rowsep">See <a href="#sec5.3" class="scroll-into-link">Experimental Model and Subject Details</a></td><td class="rowsep">N/A</td></tr><tr><td class="rowsep" colspan="3"><strong>Chemicals, Peptides, and Recombinant Proteins</strong></td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog:93482</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>Phosphatase Inhibitor Cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase Inhibitor Cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Urea</td><td>Sigma</td><td>Catalog: U0631</td></tr><tr><td>Tris(hydroxymethyl)aminomethane</td><td>Invitrogen</td><td>Catalog: AM9855G</td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Sodium chloride</td><td>Santa Cruz Biotechnology</td><td>Catalog: sc-295833</td></tr><tr><td>PUGNAc</td><td>Sigma</td><td>Catalog: A7229</td></tr><tr><td>Dithiothretiol</td><td>ThermoFisher Scientific</td><td>Catalog: 20291</td></tr><tr><td>Iodoacetamide</td><td>ThermoFisher Scientific</td><td>Catalog: A3221</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Catalog: V511X</td></tr><tr><td>Lysyl endopeptidase, aass spectrometry grade</td><td>Wako Chemicals</td><td>Catalog: 125-05061</td></tr><tr><td>Formic acid</td><td>Fisher Chemical</td><td>Catalog: A117-50</td></tr><tr><td>Reversed-phase C18 SepPak</td><td>Waters</td><td>Catalog: WAT054925</td></tr><tr><td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</td><td>Alfa Aesar</td><td>Catalog: J63218</td></tr><tr><td>Tandem mass tags – 10plex</td><td>ThermoFisher Scientific</td><td>Catalog: 90110</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>Catalog: 302031</td></tr><tr><td>Ammonium Hydroxide solution</td><td>Sigma</td><td>Catalog: 338818</td></tr><tr><td>Hydroxylamine solution</td><td>Aldrich</td><td>Catalog: 467804</td></tr><tr><td>Ni-NTA agarose beads</td><td>QIAGEN</td><td>Catalog: 30410</td></tr><tr><td>Iron (III) chloride</td><td>Sigma</td><td>Catalog:451649</td></tr><tr><td>iVIEW DAB Detection Kit</td><td>Roche</td><td>Catalog: 760-091</td></tr><tr><td class="rowsep">Bond Polymer Refine Detection Kit</td><td class="rowsep">Leica Biosystems</td><td class="rowsep">Catalog: DS9800</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical Commercial Assays</strong></td></tr><tr><td>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold</td><td>Illumina</td><td>Catalog: RS-122-2301</td></tr><tr><td>Infinium MethylationEPIC Kit</td><td>Illumina</td><td>Catalog: WG-317-1003</td></tr><tr><td>Nextera DNA Exosome Kit</td><td>Illumina</td><td>Catalog: 20020617</td></tr><tr><td>KAPA Hyper Prep Kit, PCR-free</td><td>Roche</td><td>Catalog: 07962371001</td></tr><tr><td class="rowsep">BCA Protein Assay Kit</td><td class="rowsep">ThermoFisher Scientific</td><td class="rowsep">Catalog: 23225</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited Data</strong></td></tr><tr><td>PhosphoSitePlus</td><td><div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d218962e3217" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Hornbeck et al., 2015</a><div data-db-target-of="bib63-d218962e3217" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (898)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.phosphosite.org" target="_blank">https://www.phosphosite.org</a></td></tr><tr><td>GTEx</td><td><div class="dropBlock reference-citations"><a href="#bib194" id="back-bib194" data-db-target-for="bib194-d218962e3226" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib194">Ardlie et al., 2015</a><div data-db-target-of="bib194-d218962e3226" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ardlie K.G. </li><li class="loa__item">Deluca D.S. </li><li class="loa__item">Segre A.V. </li><li class="loa__item">Sullivan T.J. </li><li class="loa__item">Young T.R. </li><li class="loa__item">Gelfand E.T. </li><li class="loa__item">Trowbridge C.A. </li><li class="loa__item">Maller J.B. </li><li class="loa__item">Tukiainen T. </li><li class="loa__item">Lek M. </li><li>et al.</li></ul><div class="reference-citations__detials">The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">348</span></span>: <span class="ref__seriesPages">648-660</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1126%2Fscience.1262110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25954001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84929001104&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1921)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=648-660&journal=Science&author=Ardlie+K.G.&author=Deluca+D.S.&author=Segre+A.V.&author=Sullivan+T.J.&author=Young+T.R.&author=Gelfand+E.T.&author=Trowbridge+C.A.&author=Maller+J.B.&author=Tukiainen+T.&author=Lek+M.&title=The+Genotype-Tissue+Expression+%28GTEx%29+pilot+analysis%3A+Multitissue+gene+regulation+in+humans">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://gtexportal.org/home/" target="_blank">https://gtexportal.org/home/</a></td></tr><tr><td class="rowsep">TCGA - ccRCC</td><td class="rowsep"><div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e3235" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e3235" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td class="rowsep"><a class="inter-ref-link" href="https://portal.gdc.cancer.gov/" target="_blank">https://portal.gdc.cancer.gov/</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>methylationArrayAnalysis (version 3.9)</td><td><div class="dropBlock reference-citations"><a href="#bib204" id="back-bib204" data-db-target-for="bib204-d218962e3248" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib204">Maksimovic et al., 2016</a><div data-db-target-of="bib204-d218962e3248" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maksimovic J. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Oshlack A. </li></ul><div class="reference-citations__detials">A cross-package Bioconductor workflow for analysing methylation array data.<div class="popup-ref"><span class="ref__series"><i>F1000Research.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1-53</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.12688%2Ff1000research.8839.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1-53&journal=F1000Research&author=Maksimovic+J.&author=Phipson+B.&author=Oshlack+A.&title=A+cross-package+Bioconductor+workflow+for+analysing+methylation+array+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html" target="_blank">https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html</a></td></tr><tr><td>Illumina EPIC methylation array (3.9)</td><td><div class="dropBlock reference-citations"><a href="#bib196" id="back-bib196" data-db-target-for="bib196-d218962e3257" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib196">Fortin et al., 2017</a><div data-db-target-of="bib196-d218962e3257" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fortin J.P. </li><li class="loa__item">Triche T.J. </li><li class="loa__item">Hansen K.D. </li></ul><div class="reference-citations__detials">Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">558-560</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtw691&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28035024&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016984334&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=558-560&journal=Bioinformatics&author=Fortin+J.P.&author=Triche+T.J.&author=Hansen+K.D.&title=Preprocessing%2C+normalization+and+integration+of+the+Illumina+HumanMethylationEPIC+array+with+minfi">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html" target="_blank">https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html</a></td></tr><tr><td>VarDict</td><td><div class="dropBlock reference-citations"><a href="#bib203" id="back-bib203" data-db-target-for="bib203-d218962e3266" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib203">Lai et al., 2016</a><div data-db-target-of="bib203-d218962e3266" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lai Z. </li><li class="loa__item">Markovets A. </li><li class="loa__item">Ahdesmaki M. </li><li class="loa__item">Chapman B. </li><li class="loa__item">Hofmann O. </li><li class="loa__item">Mcewen R. </li><li class="loa__item">Johnson J. </li><li class="loa__item">Dougherty B. </li><li class="loa__item">Barrett J.C. </li><li class="loa__item">Dry J.R. </li></ul><div class="reference-citations__detials">VarDict: A novel and versatile variant caller for next-generation sequencing in cancer research.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">1-11</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkw227&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26621913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84976448371&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1-11&journal=Nucleic+Acids+Res.&author=Lai+Z.&author=Markovets+A.&author=Ahdesmaki+M.&author=Chapman+B.&author=Hofmann+O.&author=Mcewen+R.&author=Johnson+J.&author=Dougherty+B.&author=Barrett+J.C.&author=Dry+J.R.&title=VarDict%3A+A+novel+and+versatile+variant+caller+for+next-generation+sequencing+in+cancer+research">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/AstraZeneca-NGS/VarDict" target="_blank">https://github.com/AstraZeneca-NGS/VarDict</a></td></tr><tr><td>Strelka2</td><td><div class="dropBlock reference-citations"><a href="#bib201" id="back-bib201" data-db-target-for="bib201-d218962e3275" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib201">Kim et al., 2018</a><div data-db-target-of="bib201-d218962e3275" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li>et al.</li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/Illumina/strelka" target="_blank">https://github.com/Illumina/strelka</a></td></tr><tr><td>MUTECT2</td><td><div class="dropBlock reference-citations"><a href="#bib195" id="back-bib195" data-db-target-for="bib195-d218962e3284" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib195">Cibulskis et al., 2013</a><div data-db-target-of="bib195-d218962e3284" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1869)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.php" target="_blank">https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.php</a></td></tr><tr><td>VarScan2.3.8</td><td><div class="dropBlock reference-citations"><a href="#bib202" id="back-bib202" data-db-target-for="bib202-d218962e3293" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib202">Koboldt et al., 2012</a><div data-db-target-of="bib202-d218962e3293" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1937)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+Somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://varscan.sourceforge.net" target="_blank">http://varscan.sourceforge.net</a></td></tr><tr><td>Pindel0.2.5</td><td><div class="dropBlock reference-citations"><a href="#bib207" id="back-bib207" data-db-target-for="bib207-d218962e3302" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib207">Ye et al., 2009</a><div data-db-target-of="bib207-d218962e3302" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: A pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1065)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+A+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://gmt.genome.wustl.edu/packages/pindel/" target="_blank">http://gmt.genome.wustl.edu/packages/pindel/</a></td></tr><tr><td>CNVEX</td><td>Marcin Cieslik Lab</td><td><a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a></td></tr><tr><td>CRISP</td><td>Marcin Cieslik Lab</td><td><a class="inter-ref-link" href="https://github.com/mcieslik-mctp/crisp-build" target="_blank">https://github.com/mcieslik-mctp/crisp-build</a></td></tr><tr><td>Proteowizard</td><td><div class="dropBlock reference-citations"><a href="#bib199" id="back-bib199" data-db-target-for="bib199-d218962e3327" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib199">Kessner et al., 2008</a><div data-db-target-of="bib199-d218962e3327" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kessner D. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Burke R. </li><li class="loa__item">Agus D. </li><li class="loa__item">Mallick P. </li></ul><div class="reference-citations__detials">ProteoWizard: Open source software for rapid proteomics tools development.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (870)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+Open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://proteowizard.sourceforge.net/" target="_blank">http://proteowizard.sourceforge.net/</a></td></tr><tr><td>MSFragger</td><td><div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d218962e3336" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Kong et al., 2017</a><div data-db-target-of="bib80-d218962e3336" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://msfragger.nesvilab.org/" target="_blank">https://msfragger.nesvilab.org/</a></td></tr><tr><td>Philosopher</td><td>Alexey Nesvizhskii Lab</td><td><a class="inter-ref-link" href="https://philosopher.nesvilab.org/" target="_blank">https://philosopher.nesvilab.org/</a></td></tr><tr><td>PeptideProphet</td><td><div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d218962e3354" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Keller et al., 2002</a><div data-db-target-of="bib78-d218962e3354" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keller A. </li><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3457)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal.+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://peptideprophet.sourceforge.net/" target="_blank">http://peptideprophet.sourceforge.net/</a></td></tr><tr><td>ProteinProphet</td><td><div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d218962e3363" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Nesvizhskii et al., 2003</a><div data-db-target-of="bib118-d218962e3363" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Keller A. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">A statistical model for identifying proteins by tandem mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3129)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal.+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://proteinprophet.sourceforge.net/prot-proph.pdf" target="_blank">http://proteinprophet.sourceforge.net/prot-proph.pdf</a></td></tr><tr><td>PTMProphet</td><td><div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d218962e3372" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Deutsch et al., 2015</a><div data-db-target-of="bib34-d218962e3372" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Deutsch E.W. </li><li class="loa__item">Mendoza L. </li><li class="loa__item">Shteynberg D. </li><li class="loa__item">Slagel J. </li><li class="loa__item">Sun Z. </li><li class="loa__item">Moritz R.L. </li></ul><div class="reference-citations__detials">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics.<div class="popup-ref"><span class="ref__series"><i>Proteomics Clin. Appl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">745-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fprca.201400164&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25631240&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84948987628&amp;cf=" onclick="newWindow(this.href);return false">Scopus (125)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=745-754&journal=Proteomics+Clin.+Appl.&author=Deutsch+E.W.&author=Mendoza+L.&author=Shteynberg+D.&author=Slagel+J.&author=Sun+Z.&author=Moritz+R.L.&title=Trans-Proteomic+Pipeline%2C+a+standardized+data+processing+pipeline+for+large-scale+reproducible+proteomics+informatics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.tppms.org/tools/ptm/" target="_blank">http://www.tppms.org/tools/ptm/</a></td></tr><tr><td>TMT-Integrator</td><td>Alexey Nesvizhskii Lab</td><td><a class="inter-ref-link" href="https://github.com/Nesvilab/TMT-Integrator" target="_blank">https://github.com/Nesvilab/TMT-Integrator</a></td></tr><tr><td>DIA-Umpire</td><td><div class="dropBlock reference-citations"><a href="#bib168" id="back-bib168" data-db-target-for="bib168-d218962e3389" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib168">Tsou et al., 2015</a><div data-db-target-of="bib168-d218962e3389" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tsou C.C. </li><li class="loa__item">Avtonomov D. </li><li class="loa__item">Larsen B. </li><li class="loa__item">Tucholska M. </li><li class="loa__item">Choi H. </li><li class="loa__item">Gingras A.C. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">258-264</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.3255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25599550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923838580&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=258-264&journal=Nat.+Methods&author=Tsou+C.C.&author=Avtonomov+D.&author=Larsen+B.&author=Tucholska+M.&author=Choi+H.&author=Gingras+A.C.&author=Nesvizhskii+A.I.&title=DIA-Umpire%3A+comprehensive+computational+framework+for+data-independent+acquisition+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/Nesvilab/DIA-Umpire" target="_blank">https://github.com/Nesvilab/DIA-Umpire</a></td></tr><tr><td>msproteomicstools</td><td><a class="inter-ref-link" href="http://msproteomicstools.roestlab.org/" target="_blank">http://msproteomicstools.roestlab.org/</a></td><td><a class="inter-ref-link" href="https://github.com/msproteomicstools" target="_blank">https://github.com/msproteomicstools</a></td></tr><tr><td>ComBat (v3.20.0)</td><td><div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d218962e3407" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Johnson et al., 2007</a><div data-db-target-of="bib72-d218962e3407" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johnson W.E. </li><li class="loa__item">Li C. </li><li class="loa__item">Rabinovic A. </li></ul><div class="reference-citations__detials">Adjusting batch effects in microarray expression data using empirical Bayes methods.<div class="popup-ref"><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">118-127</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbiostatistics%2Fkxj037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16632515&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33845432928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2518)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=118-127&journal=Biostatistics&author=Johnson+W.E.&author=Li+C.&author=Rabinovic+A.&title=Adjusting+batch+effects+in+microarray+expression+data+using+empirical+Bayes+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/sva.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/sva.html</a></td></tr><tr><td>DreamAI</td><td>Pei Wang Lab</td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/DreamAI" target="_blank">https://github.com/WangLab-MSSM/DreamAI</a></td></tr><tr><td>GISTIC2.0</td><td><div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d218962e3424" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Mermel et al., 2011</a><div data-db-target-of="bib103-d218962e3424" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (959)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz" target="_blank">ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz</a></td></tr><tr><td>iProFun</td><td><div class="dropBlock reference-citations"><a href="#bib160" id="back-bib160" data-db-target-for="bib160-d218962e3433" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib160">Song et al., 2019</a><div data-db-target-of="bib160-d218962e3433" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+Impact+of+DNA+Copy+Number+Alteration+and+Methylation+on+the+Proteogenomic+Landscape+of+Human+Ovarian+Cancer+via+a+Multi-omics+Integrative+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/iProFun" target="_blank">https://github.com/WangLab-MSSM/iProFun</a></td></tr><tr><td>ESTIMATE</td><td><div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e3443" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e3443" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioinformatics.mdanderson.org/public-software/estimate/" target="_blank">https://bioinformatics.mdanderson.org/public-software/estimate/</a></td></tr><tr><td>WebGestaltR</td><td><div class="dropBlock reference-citations"><a href="#bib175" id="back-bib175" data-db-target-for="bib175-d218962e3452" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib175">Wang et al., 2017</a><div data-db-target-of="bib175-d218962e3452" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Greer M. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (345)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt+2017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.webgestalt.org/" target="_blank">http://www.webgestalt.org/</a></td></tr><tr><td>Joint Random Forest</td><td><div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d218962e3461" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Petralia et al., 2016</a><div data-db-target-of="bib133-d218962e3461" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Song W.M. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">743-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.5b00925&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26733076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960353766&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=743-754&journal=J.%C2%A0Proteome+Res.&author=Petralia+F.&author=Song+W.M.&author=Tu+Z.&author=Wang+P.&title=New+Method+for+Joint+Network+Analysis+Reveals+Common+and+Different+Coexpression+Patterns+among+Genes+and+Proteins+in+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/ptmJRF" target="_blank">https://github.com/WangLab-MSSM/ptmJRF</a></td></tr><tr><td>GSVA</td><td><div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d218962e3470" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Hänzelmann et al., 2013</a><div data-db-target-of="bib57-d218962e3470" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (722)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/GSVA.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/GSVA.html</a></td></tr><tr><td>TCGAbiolinks</td><td><div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d218962e3479" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d218962e3479" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html</a></td></tr><tr><td>Cytoscape</td><td><div class="dropBlock reference-citations"><a href="#bib150" id="back-bib150" data-db-target-for="bib150-d218962e3488" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib150">Shannon et al., 2003</a><div data-db-target-of="bib150-d218962e3488" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shannon P. </li><li class="loa__item">Markiel A. </li><li class="loa__item">Ozier O. </li><li class="loa__item">Baliga N.S. </li><li class="loa__item">Wang J.T. </li><li class="loa__item">Ramage D. </li><li class="loa__item">Amin N. </li><li class="loa__item">Schwikowski B. </li><li class="loa__item">Ideker T. </li></ul><div class="reference-citations__detials">Cytoscape: a software environment for integrated models of biomolecular interaction networks.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13696)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cytoscape.org/" target="_blank">https://cytoscape.org/</a></td></tr><tr><td>TSNet</td><td><div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d218962e3497" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Petralia et al., 2018</a><div data-db-target-of="bib134-d218962e3497" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng J. </li><li class="loa__item">Yan A. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/TSNet" target="_blank">https://github.com/WangLab-MSSM/TSNet</a></td></tr><tr><td>xCell</td><td><div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d218962e3506" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d218962e3506" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://xcell.ucsf.edu/" target="_blank">http://xcell.ucsf.edu/</a></td></tr><tr><td>CPTAC Network Exploration Portal</td><td>Pei Wang Lab</td><td><a class="inter-ref-link" href="http://ccrcc.cptac-network-view.org/" target="_blank">http://ccrcc.cptac-network-view.org/</a></td></tr><tr><td>CPTAC Data Viewer</td><td>Pei Wang Lab</td><td><a class="inter-ref-link" href="http://ccrcc.cptac-data-view.org/" target="_blank">http://ccrcc.cptac-data-view.org/</a></td></tr><tr><td>iCAVE</td><td><div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d218962e3531" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Liluashvili et al., 2017</a><div data-db-target-of="bib91-d218962e3531" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liluashvili V. </li><li class="loa__item">Kalayci S. </li><li class="loa__item">Fluder E. </li><li class="loa__item">Wilson M. </li><li class="loa__item">Gabow A. </li><li class="loa__item">Gümüs Z.H. </li></ul><div class="reference-citations__detials">iCAVE: an open source tool for visualizing biomolecular networks in 3D, stereoscopic 3D and immersive 3D.<div class="popup-ref"><span class="ref__series"><i>Gigascience.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1-13</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fgigascience%2Fgix054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28814063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042149548&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1-13&journal=Gigascience&author=Liluashvili+V.&author=Kalayci+S.&author=Fluder+E.&author=Wilson+M.&author=Gabow+A.&author=G%C3%BCm%C3%BCs+Z.H.&title=iCAVE%3A+an+open+source+tool+for+visualizing+biomolecular+networks+in+3D%2C+stereoscopic+3D+and+immersive+3D">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://labs.icahn.mssm.edu/gumuslab/software" target="_blank">http://labs.icahn.mssm.edu/gumuslab/software</a></td></tr><tr><td>MODMatcher</td><td><div class="dropBlock reference-citations"><a href="#bib185" id="back-bib185" data-db-target-for="bib185-d218962e3541" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib185">Yoo et al., 2014</a><div data-db-target-of="bib185-d218962e3541" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoo S. </li><li class="loa__item">Huang T. </li><li class="loa__item">Campbell J.D. </li><li class="loa__item">Lee E. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Geraci M.W. </li><li class="loa__item">Powell C.A. </li><li class="loa__item">Schadt E.E. </li><li class="loa__item">Spira A. </li><li class="loa__item">Zhu J. </li></ul><div class="reference-citations__detials">MODMatcher: multi-omics data matcher for integrative genomic analysis.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/integrativenetworkbiology/Modmatcher" target="_blank">https://github.com/integrativenetworkbiology/Modmatcher</a></td></tr><tr><td>ConsensusClusterPlus</td><td><div class="dropBlock reference-citations"><a href="#" id="back-bib111" data-db-target-for="bib111-d218962e3550" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Monti et al., 2003</a><div data-db-target-of="bib111-d218962e3550" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monti S. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mesirov J. </li><li class="loa__item">Golub T. </li></ul><div class="reference-citations__detials">Consensus Clustering A Resampling-Based Method for Class Discovery and Visualization of Gene Expression Microarray Data.<div class="popup-ref"><span class="ref__series"><i>Mach. Learn.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">91-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1023%2FA%3A1023949509487&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038724494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=91-118&journal=Mach.+Learn.&author=Monti+S.&author=Tamayo+P.&author=Mesirov+J.&author=Golub+T.&title=Consensus+Clustering+A+Resampling-Based+Method+for+Class+Discovery+and+Visualization+of+Gene+Expression+Microarray+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib182" data-db-target-for="bib182-d218962e3550" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib182">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib182-d218962e3550" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (442)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html</a></td></tr><tr><td>OmicsX</td><td><div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d218962e3559" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Pan et al., 2019</a><div data-db-target-of="bib124-d218962e3559" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pan J. </li><li class="loa__item">Qian J. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">OmicsX: a web server for integrated OMICS analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/755918">https://doi.org/10.1101/755918</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F755918&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095490240&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Pan+J.&author=Qian+J.&author=Zhang+H.&title=OmicsX%3A+a+web+server+for+integrated+OMICS+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bioinfo.wilmer.jhu.edu/OmicsX/" target="_blank">http://bioinfo.wilmer.jhu.edu/OmicsX/</a></td></tr><tr><td>Omic-Sig</td><td><div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d218962e3568" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Lih et al., 2019</a><div data-db-target-of="bib90-d218962e3568" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lih T.M. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Omic-Sig: Utilizing Omics Data to Explore and Visualize Kinase-Substrate Interactions.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/746123">https://doi.org/10.1101/746123</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F746123&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095527091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Lih+T.M.&author=Clark+D.J.&author=Zhang+H.&title=Omic-Sig%3A+Utilizing+Omics+Data+to+Explore+and+Visualize+Kinase-Substrate+Interactions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/hzhangjhu/Omic-Sig" target="_blank">https://github.com/hzhangjhu/Omic-Sig</a></td></tr><tr><td>OmicsOne</td><td><div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d218962e3577" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Hu et al., 2019</a><div data-db-target-of="bib67-d218962e3577" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hu Y. </li><li class="loa__item">Ao M. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">OmicsOne: Associate Omics Data with Phenotypes in One-Click.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/756544">https://doi.org/10.1101/756544</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F756544&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095531613&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Hu+Y.&author=Ao+M.&author=Zhang+H.&title=OmicsOne%3A+Associate+Omics+Data+with+Phenotypes+in+One-Click">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/HuiZhangLab-JHU/OmicsOne" target="_blank">https://github.com/HuiZhangLab-JHU/OmicsOne</a></td></tr><tr><td>pyQUILTS (v1.0)</td><td><div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d218962e3586" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Ruggles et al., 2016</a><div data-db-target-of="bib145-d218962e3586" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Tang Z. </li><li class="loa__item">Wang X. </li><li class="loa__item">Grover H. </li><li class="loa__item">Askenazi M. </li><li class="loa__item">Teubl J. </li><li class="loa__item">Cao S. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Tabb D.L. </li><li>et al.</li></ul><div class="reference-citations__detials">An Analysis of the Sensitivity of Proteogenomic Mapping of Somatic Mutations and Novel Splicing Events in Cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+Analysis+of+the+Sensitivity+of+Proteogenomic+Mapping+of+Somatic+Mutations+and+Novel+Splicing+Events+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl" target="_blank">http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl</a></td></tr><tr><td>MS-GF+</td><td><div class="dropBlock reference-citations"><a href="#bib200" id="back-bib200" data-db-target-for="bib200-d218962e3595" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib200">Kim and Pevzner, 2014</a><div data-db-target-of="bib200-d218962e3595" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (390)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/MSGFPlus/msgfplus" target="_blank">https://github.com/MSGFPlus/msgfplus</a></td></tr><tr><td>NeoFlow</td><td>Bing Zhang Lab</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a></td></tr><tr><td>netMHCpan</td><td><div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d218962e3612" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Jurtz et al., 2017</a><div data-db-target-of="bib75-d218962e3612" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (216)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.cbs.dtu.dk/services/NetMHCpan/" target="_blank">http://www.cbs.dtu.dk/services/NetMHCpan/</a></td></tr><tr><td>Optitype</td><td><div class="dropBlock reference-citations"><a href="#bib165" id="back-bib165" data-db-target-for="bib165-d218962e3621" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib165">Szolek et al., 2014</a><div data-db-target-of="bib165-d218962e3621" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/FRED-2/OptiType" target="_blank">https://github.com/FRED-2/OptiType</a></td></tr><tr><td>Customprodbj</td><td>Bing Zhang Lab</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a></td></tr><tr><td>PDV</td><td><div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d218962e3639" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Li et al., 2019</a><div data-db-target-of="bib87-d218962e3639" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/wenbostar/PDV" target="_blank">https://github.com/wenbostar/PDV</a></td></tr><tr><td>PepQuery</td><td><div class="dropBlock reference-citations"><a href="#bib178" id="back-bib178" data-db-target-for="bib178-d218962e3648" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib178">Wen et al., 2019</a><div data-db-target-of="bib178-d218962e3648" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://pepquery.org" target="_blank">http://pepquery.org</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2819%2931123-7">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec5.2" data-locator="sec5.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead Contact and Materials Availability</span></h3><div class="section-paragraph">This study did not generate new unique reagents. Further information and requests for resources should be directed to and will be fulfilled by the Lead Contact, Hui Zhang (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#94fce1fdeefcf5faf3d4fefce1baf1f0e1" target="_blank"><span class="__cf_email__" data-cfemail="d9b1acb0a3b1b8b7be99b3b1acf7bcbdac">[email&#160;protected]</span></a>).</div></section><section id="sec5.3" data-locator="sec5.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental Model and Subject Details</span></h3><section id="sec5.3.1" data-locator="sec5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Human Subjects</span></h4><div class="section-paragraph">A total of 110 participants, with an age range of 30-90, were included in this study. This cohort contained males (n = 81) and females (n = 29) and reflects the gender distribution of clear cell renal cell carcinoma (ccRCC) (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e3669" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e3669" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Only histopathologically defined adult ccRCC tumors were only included in the analysis. Institutional review boards at each Tissue Source Site (TSS) reviewed protocols and consent documentation, in adherence to Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines.</div></section><section id="sec5.3.2" data-locator="sec5.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Clinical Data Annotation</span></h4><div class="section-paragraph">Clinical data were obtained from TSS and aggregated by the Biospecimen Core Resource (BCR, Van Andel Research Institute (Grand Rapids, MI)). Data forms were stored as Microsoft Excel files (.xls). Clinical data can be accessed and downloaded from the CPTAC Data Portal at <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/documents/S044_CPTAC_CCRCC_Discovery_Cohort_Clinical_Data_r1_Sept2018" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/documents/S044_CPTAC_CCRCC_Discovery_Cohort_Clinical_Data_r1_Sept2018</a>.xlsx. Patients with any prior history of other malignancies within twelve months or any systemic treatment (chemotherapy, radiotherapy, of immune-related therapy) were excluded from this study. Demographics, histopathologic information, and treatment details were collected and summarized in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>. The characteristics of the CPTAC ccRCC cohort reflect the general incidence of ccRCC (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d218962e3683" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Creighton et al., 2013</a><div data-db-target-of="bib31-d218962e3683" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Creighton C.J. </li><li class="loa__item">Morgan M. </li><li class="loa__item">Gunaratne P.H. </li><li class="loa__item">Wheeler D.A. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Chu A. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Signoretti S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive molecular characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), including patient age distributions [30-49 (17.4%), 50-69 (60.6%), and &gt; 70 (22.0%)], grade distributions [G1 (5.5%), G2 (48.6%), G3 (37.6%), and G4 (8.3%)], and stage distributions [I (46.8%), II (11.9%), III (30.3%), and IV (11.0%)].</div></section></section><section id="sec5.4" data-locator="sec5.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method Details</span></h3><section id="sec5.4.1" data-locator="sec5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimen Acquisition</span></h4><div class="section-paragraph">The tumor, adjacent normal tissue and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly-diagnosed patients with ccRCC who were undergoing surgical resection and had received no prior treatment for their disease, including radiotherapy or chemotherapy. All cases had ccRCC histology but were collected regardless of histologic grade or surgical stage. Cases were then graded using the Fuhrman Nuclear Grading System and staged using the AJCC cancer staging system 7<sup>th</sup> edition (<div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d218962e3697" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Edge et al., 2010</a><div data-db-target-of="bib38-d218962e3697" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Edge S. </li><li class="loa__item">Byrd D. </li><li class="loa__item">Compton C. </li><li class="loa__item">Fritz A. </li><li class="loa__item">Greene F. </li><li class="loa__item">Trotti A. </li></ul><div class="reference-citations__detials">AJCC cancer staging manual.<div class="popup-ref"> <span class="refPublisherName">Springer US</span>,
        <span class="refPublisherLoc">New York, NY</span><span class="refDate">2010</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=Edge+S.&author=Byrd+D.&author=Compton+C.&author=Fritz+A.&author=Greene+F.&author=Trotti+A.&title=AJCC+cancer+staging+manual">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Tumors specimens weighed between 125 and 3,000 mg. For most cases, three to four tumor specimens were collected. Each tissue specimen endured cold ischemia for 30 minutes or less prior to freezing in liquid nitrogen. The specimens were collected with an average total ischemic time of fifteen minutes from resection/collection to freezing. Specimens were either flash-frozen in liquid nitrogen or embedded in optimal cutting temperature (OCT) medium, with histologic sections obtained from top and bottom portions for review. Each case was reviewed by a board-certified pathologist to confirm the assigned pathology. The top and bottom sections were required to contain an average of 80% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the TSS to the BCR using a cryoport that maintained an average temperature of less than −140°C. At the BCR, the specimens were confirmed by pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized, and material was aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome characterization centers (GCC). The DNA sequencing and methylation analyses were performed at the Broad Institute (Cambridge, MA) and RNA sequencing was performed at the University of North Carolina (Chapel Hill, NC). Material for proteomic analyses was sent to the Proteomic Characterization Center (PCC) at Johns Hopkins Medical Institutions of Johns Hopkins University (Baltimore, MD)</div></section><section id="sec5.4.2" data-locator="sec5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic and Transcriptomic Sample Preparation and Data Acquisition</span></h4><section id="sec5.4.2.1" data-locator="sec5.4.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample Processing for Genomic DNA and total RNA Extraction</span></h4><div class="section-paragraph">Our study sampled a single site of the primary tumor. All DNA and RNA were isolated using a co-isolation protocol in which nucleic acids were isolated from the same cryopulverized aliquot that was used for both proteomics and genomics. Tumor samples were from surgical resections due to the requirement to process a minimum of 125mg of tumor issue and 50mg of adjacent normal tissue. RNA and DNA were extracted from tumor and adjacent normal specimens using QIAGEN’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3-5 mL) to serve as matched benign reference material. The Qubit dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorometer to determine the concentration of dsDNA in an aqueous solution. A sample that passed quality control and produced sufficient DNA yield various genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality was assessed using Agilent Bioanalyzer. A sample that passed RNA quality control and had a minimum RNA integrity Number (RIN) score of 8 was subjected to RNA sequencing.</div></section></section><section id="sec5.4.3" data-locator="sec5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole Exome Sequencing Methods</span></h4><section id="sec5.4.3.1" data-locator="sec5.4.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Library Construction</span></h4><div class="section-paragraph">Library construction was performed as described in <div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d218962e3716" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Fisher et al., 2011</a><div data-db-target-of="bib44-d218962e3716" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fisher S. </li><li class="loa__item">Barry A. </li><li class="loa__item">Abreu J. </li><li class="loa__item">Minie B. </li><li class="loa__item">Nolan J. </li><li class="loa__item">Delorey T.M. </li><li class="loa__item">Young G. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Allen A. </li><li class="loa__item">Ambrogio L. </li><li>et al.</li></ul><div class="reference-citations__detials">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, with the following modifications: initial genomic DNA input into shearing was reduced from 3 μg to 250 ng in 50 μL of solution. For adaptor ligation, Illumina paired end adapters were replaced with palindromic forked adapters obtained from Integrated DNA Technologies, with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents were used in a 96-reaction kit format for end repair/A- tailing, adaptor ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30 μL to maximize library concentration followed by a vortexing step to maximize the amount of template eluted.</div></section><section id="sec5.4.3.2" data-locator="sec5.4.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">In-solution Hybrid Selection</span></h4><div class="section-paragraph">Following library construction, products were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s recommended protocol, with a few exceptions. First, all libraries within a library construction plate were pooled prior to hybridization. Second, to facilitate automation the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate. All hybridization and capture steps were automated utilizing the Agilent Bravo liquid handling system.</div></section><section id="sec5.4.3.3" data-locator="sec5.4.3.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of Libraries for Cluster Amplification and Sequencing</span></h4><div class="section-paragraph">After post-capture enrichment, library pools were quantified by qPCR using a kit obtained from KAPA Biosystems with probes specific to the ends of the adapters. The assay was automated on the Agilent Bravo liquid handling system. Based on qPCR quantification, libraries were normalized to 2 nM.</div></section><section id="sec5.4.3.4" data-locator="sec5.4.3.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster Amplification and Sequencing</span></h4><div class="section-paragraph">Cluster amplification of DNA libraries was performed according to the manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flowcells were sequenced utilizing Sequencing-by-Synthesis chemistry. The flowcells are then analyzed using RTA v.2.7.3 or a later version. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two eight-cycle index reads across the number of lanes necessary to meet coverage for all libraries in the pool. Pooled libraries were processed using HiSeq4000 as paired end runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adaptor and low-quality sequences were trimmed. The raw reads were mapped to the hg38 human reference genome. The validated Binary Alignment Map (BAM) files were used for downstream analysis and variant calling. FASTQ files of all reads were then uploaded to the Genomic Data Commons (GDC) repository.</div></section></section><section id="sec5.4.4" data-locator="sec5.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole Genome Sequencing, PCR-Free</span></h4><section id="sec5.4.4.1" data-locator="sec5.4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of Libraries for Cluster Amplification and Sequencing</span></h4><div class="section-paragraph">Input genomic DNA (350 ng in 50 μL) was acoustically sheared using a Covaris focused-ultrasonicator (∼385 bp fragment range). Following shearing, an additional size selection was performed using a SPRI cleanup. Library preparation was performed using KAPA Hyper Prep without amplification module (KAPA Biosystems) with palindromic forked adapters containing unique eight-base index sequences embedded within the adaptor (Integrated DNA Technologies). Libraries were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters. The assay was automated on the Agilent Bravo liquid handling system. Based on qPCR quantification, libraries were normalized to 1.7 nM and pooled into 24-plexes.</div></section><section id="sec5.4.4.2" data-locator="sec5.4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster Amplification and Sequencing (HiSeqX)</span></h4><div class="section-paragraph">Sample pools were combined with HiSeqX Cluster Amp Regents EPX1, EPX2 and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flowcells were sequenced for a minimum of 15x coverage on HiSeqX utilizing sequencing-by-synthesis to produce 151 base pair paired-end reads. Outputs from Illumina software were processed by the Picard data-processing pipeline to yield BAM files containing demultiplexed, aggregated aligned reads. All sample information tracking was performed by automated LIMS messaging. FASTQ files of all reads were then uploaded to the GDC.</div></section><section id="sec5.4.4.3" data-locator="sec5.4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Illumina Infinium MethylationEPIC BeadChip Array</span></h4><div class="section-paragraph">The MethylationEPIC array uses an eight-sample version of the Illumina BeadChip that captures &gt; 850,000 methylation sites per sample. 250 ng of DNA was used for bisulfite conversion using the Infinium MethylationEPIC BeadChip Kit. The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data were processed through an automated genotype calling pipeline, generating raw idat files and a sample sheet. For 109 out of the 110 samples, a sufficient amount of material was obtained to perform methylation profiling. Two out of the 109 samples showed a missing rate greater than 95% and were excluded from all downstream analyses.</div></section></section><section id="sec5.4.5" data-locator="sec5.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA Sequencing</span></h4><section id="sec5.4.5.1" data-locator="sec5.4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantitation and Quality Assessment QC of total RNA Samples</span></h4><div class="section-paragraph">All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent). Samples with RINs &gt; 8.0 were considered to be of high quality and were processed further.</div></section><section id="sec5.4.5.2" data-locator="sec5.4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA-seq Library Construction</span></h4><div class="section-paragraph">Total RNA-seq libraries were generated using 300 ng of total RNA, analyzed using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer’s instructions (Illumina). Libraries were prepared on an Agilent Bravo Automated Liquid Handling System. Quality control was performed at every step, and the libraries were quantified using a TapeStation system.</div></section><section id="sec5.4.5.3" data-locator="sec5.4.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA Sequencing</span></h4><div class="section-paragraph">Indexed RNA-seq libraries were sequenced using the HiSeq4000 platform to generate a minimum of 120 million paired end reads (75 base pairs) per library with a target of greater than 90% mapped reads. The sequence data were demultiplexed and converted to FASTQ files, and adaptor and low-quality sequences were quantified/trimmed. Samples were then assessed for quality by mapping reads to the hg38 reference genome, estimating the total number of reads that mapped, assessing the amount of RNA that mapped to coding regions, the amount of rRNA in the sample, the number of genes expressed, and the relative expression of housekeeping genes. Samples that passed the quality criteria were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns, including pathological status (i.e., normal adjacent versus tumor tissue) and tissue-origin specificity. FASTQ files of all reads were then uploaded to the GDC repository.</div></section></section><section id="sec5.4.6" data-locator="sec5.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic Sample Preparation and Data Acquisition</span></h4><section id="sec5.4.6.1" data-locator="sec5.4.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample Processing for Protein Extraction and Tryptic Digestion</span></h4><div class="section-paragraph">All samples for the current study were prospectively collected as described above and processed for mass spectrometric (MS) analysis at the PCC. Tissue lysis and downstream sample preparation for global proteomic and phosphoproteomic analysis were carried out as previously described (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d218962e3778" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Mertins et al., 2018</a><div data-db-target-of="bib106-d218962e3778" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (49)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Approximately 25-120 mg of each cryopulverized renal tumor tissues or NATs were homogenized separately in an appropriate volume of lysis buffer (8 M urea, 75 mM NaCl, 50 mM Tris, pH 8.0, 1 mM EDTA, 2 μg/mL aprotinin, 10 μg/mL leupeptin, 1 mM PMSF, 10 mM NaF, Phosphatase Inhibitor Cocktail 2 and Phosphatase Inhibitor Cocktail 3 [1:100 dilution], and 20 mM PUGNAc) by repeated vortexing. Lysates were clarified by centrifugation at 20,000 x g for 10 min at 4°C, and protein concentrations determined by BCA assay (Pierce). Lysates were diluted to a final concentration of 8 mg/mL with lysis buffer, and 800 μg of protein was reduced with 5 mM dithiothreitol (DTT) for 1 h at 37°C and subsequently alkylated with 10 mM iodoacetamide for 45 min at RT (room temperature) in the dark. Samples were diluted 1:3 with 50 mM Tris-HCl (pH 8.0) and subjected to proteolytic digestion with LysC (Wako Chemicals) at 1 mAU:50 μg enzyme-to-substrate ratio for 2 h at RT, followed by the addition of sequencing-grade modified trypsin (Promega) at a 1:50 enzyme-to-substrate ratio and overnight incubation at RT. The digested samples were then acidified with 50% trifluoroacetic acid (TFA, Sigma) to a pH value of approximately 2.0. Tryptic peptides were desalted on reversed-phase C18 SPE columns (Waters) and dried using a Speed-Vac (Thermo Scientific).</div></section><section id="sec5.4.6.2" data-locator="sec5.4.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT Labeling of Peptides</span></h4><div class="section-paragraph">Tandem-mass-tag (TMT) quantitation utilizes reporter ion intensities to determine protein abundance and facilitate quantitative proteomic analysis (<div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d218962e3786" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Ross et al., 2004</a><div data-db-target-of="bib144-d218962e3786" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ross P.L. </li><li class="loa__item">Huang Y.N. </li><li class="loa__item">Marchese J.N. </li><li class="loa__item">Williamson B. </li><li class="loa__item">Parker K. </li><li class="loa__item">Hattan S. </li><li class="loa__item">Khainovski N. </li><li class="loa__item">Pillai S. </li><li class="loa__item">Dey S. </li><li class="loa__item">Daniels S. </li><li>et al.</li></ul><div class="reference-citations__detials">Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">1154-1169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M400129-MCP200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15385600&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-19944432197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=1154-1169&journal=Mol.+Cell.+Proteomics&author=Ross+P.L.&author=Huang+Y.N.&author=Marchese+J.N.&author=Williamson+B.&author=Parker+K.&author=Hattan+S.&author=Khainovski+N.&author=Pillai+S.&author=Dey+S.&author=Daniels+S.&title=Multiplexed+protein+quantitation+in+Saccharomyces+cerevisiae+using+amine-reactive+isobaric+tagging+reagents">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Previously, CPTAC used two major LC-MS/MS-based methods for quantitative proteomics: label-free quantification (LFQ) and isobaric tag for relative and absolute quantitation (iTRAQ) for proteogenomic characterization of colorectal, breast, and ovarian cancers (<div class="dropBlock reference-citations"><a href="#" id="back-bib105" data-db-target-for="bib105-d218962e3789" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Mertins et al., 2016</a><div data-db-target-of="bib105-d218962e3789" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (533)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib188" data-db-target-for="bib188-d218962e3789" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib188">Zhang et al., 2014a</a><div data-db-target-of="bib188-d218962e3789" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (628)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib190" data-db-target-for="bib190-d218962e3789" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib190">Zhang et al., 2016</a><div data-db-target-of="bib190-d218962e3789" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (305)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Inherent benefits of isobaric tag approaches over LFQ for protein quantitation include decreasing peptide/protein quantification differences attributed to variation in instrument performance, reducing the number “missing values,” and facilitating integrated measurements of global protein and post-transcriptional modifications (PTM) levels (<div class="dropBlock reference-citations"><a href="#" id="back-bib62" data-db-target-for="bib62-d218962e3792" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Hogrebe et al., 2018</a><div data-db-target-of="bib62-d218962e3792" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hogrebe A. </li><li class="loa__item">von Stechow L. </li><li class="loa__item">Bekker-Jensen D.B. </li><li class="loa__item">Weinert B.T. </li><li class="loa__item">Kelstrup C.D. </li><li class="loa__item">Olsen J.V. </li></ul><div class="reference-citations__detials">Benchmarking common quantification strategies for large-scale phosphoproteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1045</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-03309-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29535314&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044218439&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1045&journal=Nat.+Commun.&author=Hogrebe+A.&author=von+Stechow+L.&author=Bekker-Jensen+D.B.&author=Weinert+B.T.&author=Kelstrup+C.D.&author=Olsen+J.V.&title=Benchmarking+common+quantification+strategies+for+large-scale+phosphoproteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib166" data-db-target-for="bib166-d218962e3792" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Thompson et al., 2003</a><div data-db-target-of="bib166-d218962e3792" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thompson A. </li><li class="loa__item">Schäfer J. </li><li class="loa__item">Kuhn K. </li><li class="loa__item">Kienle S. </li><li class="loa__item">Schwarz J. </li><li class="loa__item">Schmidt G. </li><li class="loa__item">Neumann T. </li><li class="loa__item">Johnstone R. </li><li class="loa__item">Mohammed A.K. </li><li class="loa__item">Hamon C. </li></ul><div class="reference-citations__detials">Tandem mass tags: a novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">1895-1904</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac0262560&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12713048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-12444345515&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1275)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=1895-1904&journal=Anal.+Chem.&author=Thompson+A.&author=Sch%C3%A4fer+J.&author=Kuhn+K.&author=Kienle+S.&author=Schwarz+J.&author=Schmidt+G.&author=Neumann+T.&author=Johnstone+R.&author=Mohammed+A.K.&author=Hamon+C.&title=Tandem+mass+tags%3A+a+novel+quantification+strategy+for+comparative+analysis+of+complex+protein+mixtures+by+MS%2FMS">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Recent developments in isobaric tag technology have enabled an increase in the number of independent samples that can be analyzed in parallel, thereby, increasing throughput and facilitating deeper quantification of respective proteomes (<div class="dropBlock reference-citations"><a href="#" id="back-bib100" data-db-target-for="bib100-d218962e3795" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">McAlister et al., 2012</a><div data-db-target-of="bib100-d218962e3795" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McAlister G.C. </li><li class="loa__item">Huttlin E.L. </li><li class="loa__item">Haas W. </li><li class="loa__item">Ting L. </li><li class="loa__item">Jedrychowski M.P. </li><li class="loa__item">Rogers J.C. </li><li class="loa__item">Kuhn K. </li><li class="loa__item">Pike I. </li><li class="loa__item">Grothe R.A. </li><li class="loa__item">Blethrow J.D. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">Increasing the multiplexing capacity of TMTs using reporter ion isotopologues with isobaric masses.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">84</span></span>: <span class="ref__seriesPages">7469-7478</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac301572t&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22880955&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865718193&amp;cf=" onclick="newWindow(this.href);return false">Scopus (284)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=7469-7478&journal=Anal.+Chem.&author=McAlister+G.C.&author=Huttlin+E.L.&author=Haas+W.&author=Ting+L.&author=Jedrychowski+M.P.&author=Rogers+J.C.&author=Kuhn+K.&author=Pike+I.&author=Grothe+R.A.&author=Blethrow+J.D.&author=Gygi+S.P.&title=Increasing+the+multiplexing+capacity+of+TMTs+using+reporter+ion+isotopologues+with+isobaric+masses">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib179" data-db-target-for="bib179-d218962e3795" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib179">Werner et al., 2012</a><div data-db-target-of="bib179-d218962e3795" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Werner T. </li><li class="loa__item">Becher I. </li><li class="loa__item">Sweetman G. </li><li class="loa__item">Doce C. </li><li class="loa__item">Savitski M.M. </li><li class="loa__item">Bantscheff M. </li></ul><div class="reference-citations__detials">High-resolution enabled TMT 8-plexing.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">84</span></span>: <span class="ref__seriesPages">7188-7194</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac301553x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22881393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865454296&amp;cf=" onclick="newWindow(this.href);return false">Scopus (115)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=7188-7194&journal=Anal.+Chem.&author=Werner+T.&author=Becher+I.&author=Sweetman+G.&author=Doce+C.&author=Savitski+M.M.&author=Bantscheff+M.&title=High-resolution+enabled+TMT+8-plexing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Desalted peptides from each sample were labeled with 10-plex TMT (Tandem Mass Tag) reagents (Thermo Fisher Scientific). Peptides (300 μg) from each ccRCC and NAT sample were dissolved in 300 μL of 50 mM HEPES, pH 8.5 solution. Five milligrams of TMT reagent was dissolved in 256 μL of anhydrous acetonitrile, and 123 μL of each TMT reagent was added to the corresponding aliquot of peptides. After 1 h incubation at RT, the reaction was quenched by acidification with 50% TFA to pH &lt; 3. A reference sample was created by pooling an aliquot from individual ccRCC tumors and NAT samples (90 tumors and 72 NATs, representing ∼90% of the sample cohort), labeled with the TMT-131 reagent, and included in all TMT 10-plexes as a pooled reference channel. Two internal quality control (QC) samples, a single, independently-acquired chromophobe renal cell carcinoma (chRCC) tumor sample and an NCI-7 Cell Line Panel sample (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d218962e3798" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Clark et al., 2018</a><div data-db-target-of="bib29-d218962e3798" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Bocik W. </li><li class="loa__item">Chen L. </li><li class="loa__item">Schnaubelt M. </li><li class="loa__item">Roberts R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Whiteley G. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">2205-2215</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.8b00165&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29718670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85046635566&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2205-2215&journal=J.%C2%A0Proteome+Res.&author=Clark+D.J.&author=Hu+Y.&author=Bocik+W.&author=Chen+L.&author=Schnaubelt+M.&author=Roberts+R.&author=Shah+P.&author=Whiteley+G.&author=Zhang+H.&title=Evaluation+of+NCI-7+Cell+Line+Panel+as+a+Reference+Material+for+Clinical+Proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), were prepared and interspersed among all TMT 10-plex sets. 110 ccRCC tumor and 84 NAT samples with eight chromophobe QC aliquots and five NCI-7 QC aliquots were co-randomized to 23 TMT 10-plex sets. The sample-to-TMT channel mapping is shown in <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/documents/S044_CPTAC_CCRCC_Discovery_Cohort_Specimens_r1_Sept2018.xlsx" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/documents/S044_CPTAC_CCRCC_Discovery_Cohort_Specimens_r1_Sept2018.xlsx</a> . Following labeling, peptides were mixed according to the sample-to-TMT channel mapping, concentrated and desalted on reversed-phase C18 SPE columns (Waters) and dried using a Speed-Vac (Thermo Scientific).</div></section><section id="sec5.4.6.3" data-locator="sec5.4.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide Fractionation by Basic Reversed-phase Liquid Chromatography (bRPLC)</span></h4><div class="section-paragraph">To reduce the likelihood of peptides co-isolating and co-fragmenting in these highly complex samples, we employed extensive, high-resolution fractionation via basic reversed-phase liquid chromatography (bRPLC). Previous reports indicate that this approach can reduce the incidence of isobaric reporter ion ratio distortion effects, which would impact downstream quantitation (<div class="dropBlock reference-citations"><a href="#" id="back-bib123" data-db-target-for="bib123-d218962e3810" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Ow et al., 2011</a><div data-db-target-of="bib123-d218962e3810" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ow S.Y. </li><li class="loa__item">Salim M. </li><li class="loa__item">Noirel J. </li><li class="loa__item">Evans C. </li><li class="loa__item">Wright P.C. </li></ul><div class="reference-citations__detials">Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2341-2346</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fpmic.201000752&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21548092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79956270174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=2341-2346&journal=Proteomics&author=Ow+S.Y.&author=Salim+M.&author=Noirel+J.&author=Evans+C.&author=Wright+P.C.&title=Minimising+iTRAQ+ratio+compression+through+understanding+LC-MS+elution+dependence+and+high-resolution+HILIC+fractionation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib138" data-db-target-for="bib138-d218962e3810" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Rauniyar and Yates, 2014</a><div data-db-target-of="bib138-d218962e3810" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rauniyar N. </li><li class="loa__item">Yates 3rd, J.R. </li></ul><div class="reference-citations__detials">Isobaric labeling-based relative quantification in shotgun proteomics.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">5293-5309</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr500880b&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25337643&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84915751035&amp;cf=" onclick="newWindow(this.href);return false">Scopus (229)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=5293-5309&journal=J.%C2%A0Proteome+Res.&author=Rauniyar+N.&author=Yates+J.R.&title=Isobaric+labeling-based+relative+quantification+in+shotgun+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The desalted, TMT-labeled samples were reconstituted in 900 μL of 20 mM ammonium formate (pH 10) and 2% acetonitrile (ACN) and loaded onto a 4.6 mm x 250 mm RP Zorbax 300 A Extend-C18 column with 3.5 μm size beads (Agilent). Peptides were separated at a flow-rate of 1mL/min using an Agilent 1200 Series HPLC instrument via bHPLC with Solvent A (2% ACN, 5 mM ammonium formate, pH 10) and a non-linear gradient of Solvent B (90% ACN, 5 mM ammonium formate, pH 10) as follows: 0% Solvent B (9 min), 6% Solvent B (4 min), 6% to 28.5% Solvent B (50 min), 28.% to 34% Solvent B (5.5 min), 34% to 60% Solvent B (13 min), and holding at 60% Solvent B for 8.5 min. Collected fractions were concatenated into 24 fractions by combining four fractions that are 24 fractions apart (i.e., combining fractions #1, #25, #49, and #73; #2, #26, #50, and #74; and so on); a 5% aliquot of each of the 24 fractions was used for global proteomic analysis, dried in a Speed-Vac, and resuspended in 3% ACN/0.1% formic acid prior to ESI-LC-MS/MS analysis. The remaining sample was utilized for phosphopeptide enrichment.</div></section><section id="sec5.4.6.4" data-locator="sec5.4.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Enrichment of Phosphopeptides by Fe-IMAC</span></h4><div class="section-paragraph">The remaining 95% of the sample was further concatenated before being subjected to phosphopeptide enrichment using immobilized metal affinity chromatography (IMAC) as previously described (<div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d218962e3818" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Mertins et al., 2013</a><div data-db-target-of="bib104-d218962e3818" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Patel J. </li><li class="loa__item">Udeshi N.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Burgess M.W. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Jaffe J.D. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Integrated proteomic analysis of post-translational modifications by serial enrichment.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">634-637</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.2518&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23749302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879637572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (300)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=634-637&journal=Nat.+Methods&author=Mertins+P.&author=Qiao+J.W.&author=Patel+J.&author=Udeshi+N.D.&author=Clauser+K.R.&author=Mani+D.R.&author=Burgess+M.W.&author=Gillette+M.A.&author=Jaffe+J.D.&author=Carr+S.A.&title=Integrated+proteomic+analysis+of+post-translational+modifications+by+serial+enrichment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In brief, Ni-NTA agarose beads were used to prepare Fe<sup>3+</sup>-NTA agarose beads, and 300 μg of peptides were reconstituted in 80% ACN/0.1% trifluoroacetic acid and incubated with 10 μL of the Fe<sup>3+</sup>-IMAC beads for 30 min. Samples were then centrifuged, and the supernatant containing unbound peptides was removed. The beads were washed twice and then transferred onto equilibrated C-18 Stage Tips with 80% ACN/0.1% trifluoroacetic acid. Tips were rinsed twice with 1% formic acid and eluted from the Fe<sup>3+</sup>-IMAC beads onto the C-18 Stage Tips with 70 μL of 500 mM dibasic potassium phosphate, pH 7.0 a total of three times. C-18 Stage Tips were then washed twice with 1% formic acid, followed by elution of the phosphopeptides from the C-18 Stage Tips with 50% ACN/0.1% formic acid twice. Samples were dried down and resuspended in 3% ACN/0.1% formic acid prior to ESI-LC-MS/MS analysis.</div></section><section id="sec5.4.6.5" data-locator="sec5.4.6.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">ESI-LC-MS/MS for Global Proteome and Phosphoproteome Analysis</span></h4><div class="section-paragraph">Global proteome and phosphoproteome fractions were analyzed using the same instrumentation and methodology. Peptides (∼0.8 μg) were separated on an Easy nLC 1200 UHPLC system (Thermo Scientific) on an in-house packed 20 cm x 75 μm diameter C18 column (1.9 μm Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10 μm opening (New Objective)). The column was heated to 50°C using a column heater (Phoenix-ST). The flow rate was 0.200 μl/min with 0.1% formic acid and 2% acetonitrile in water (A) and 0.1% formic acid, 90% acetonitrile (B). The peptides were separated with a 6%–30% B gradient in 84 min and analyzed using the Thermo Fusion Lumos mass spectrometer (Thermo Scientific). Parameters were as follows: MS1: resolution – 60,000, mass range – 350 to 1800 m/z, RF Lens – 30%, AGC Target 4.0e<sup>5</sup>, Max IT – 50 ms, charge state include - 2-6, dynamic exclusion – 45 s, top 20 ions selected for MS2; MS2: resolution – 50,000, high-energy collision dissociation activation energy (HCD) – 37, isolation width (m/z) – 0.7, AGC Target – 2.0e<sup>5</sup>, Max IT – 105 ms.</div></section><section id="sec5.4.6.6" data-locator="sec5.4.6.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">ESI-LC-MS/MS for Global Proteome Data-Independent Acquisition Analysis</span></h4><div class="section-paragraph">Unlabeled, digested peptide material from individual tissue samples (ccRCC and NAT) was spiked with index Retention Time (iRT) peptides (Biognosys) and subjected to data-independent acquisition (DIA) analysis. Peptides (∼0.8 μg) were separated on an Easy nLC 1200 UHPLC system (Thermo Scientific) on an in-house packed 20 cm x 75 μm diameter C18 column (1.9 μm Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10 μm opening (New Objective)). The column was heated to 50°C using a column heater (Phoenix-ST). The flow rate was 0.200 μl/min with 0.1% formic acid and 3% acetonitrile in water (A) and 0.1% formic acid, 90% acetonitrile (B). The peptides were separated with a 7%–30% B gradient in 84 mins and analyzed using the Thermo Fusion Lumos mass spectrometer (Thermo Scientific). The DIA segment consisted of one MS1 scan (350-1650 m/z range, 120K resolution) followed by 30 MS2 scans (variable m/z range, 30K resolution). Additional parameters were as follows: MS1: RF Lens – 30%, AGC Target 4.0e<sup>5</sup>, Max IT – 50 ms, charge state include - 2-6; MS2: isolation width (m/z) – 0.7, AGC Target – 2.0e<sup>5</sup>, Max IT – 120 ms.</div></section><section id="sec5.4.6.7" data-locator="sec5.4.6.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Spectral Library generation for Data-Independent Acquisition Analysis</span></h4><div class="section-paragraph">For spectral library generation, an aliquot (2 μg) of unlabeled, digested peptide material from individual tissue samples (ccRCC and NAT) was pooled and subjected to bRPLC as previously described. Collected fractions were concatenated into eight fractions by combining twelve fractions that are eight fractions apart (i.e., combining fractions #1, #9, #17, #25, #33, #41, #49, #57, #65, #73, #81, and #89; #2, #18, #26, #34, #42, #58, #66, #74, #82, and #90; and so on); dried down in a Speed-Vac, resuspended in 3% ACN, 0.1% formic acid, and was spiked with index Retention Time (iRT) peptides (Biognosys) prior to ESI-LC-MS/MS analysis. Parameters were the same as previously described for ESI-LC-MS/MS for Global Proteome and Phosphoproteome Analysis with a high-energy collision dissociation activation energy (HCD) – 34.</div></section></section><section id="sec5.4.7" data-locator="sec5.4.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic Data Processing</span></h4><section id="sec5.4.7.1" data-locator="sec5.4.7.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Harmonized Somatic Variant Calling</span></h4><div class="section-paragraph">Details regarding somatic variant calling performed by the GDC are available at: <a class="inter-ref-link" href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/" target="_blank">https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/</a>, <a class="inter-ref-link" href="https://gdc.cancer.gov/about-gdc/variant-calling-gdc" target="_blank">https://gdc.cancer.gov/about-gdc/variant-calling-gdc</a>. The University of Michigan aligned FASTQ files to the GRCh38 references, including alternate haplotypes. Variant calling was performed using VarDict (germline &amp; somatic) and Strelka2 (somatic). Variant callers were run with default settings, but custom filters were applied. Strelka was used to generate the primary somatic call-set. Variants called by Strelka had to be either (FILTER = = ”PASS”) or meet the following threshold criteria: allele frequency in the tumor &gt; 0.05, allele frequency in the normal &lt; 0.01, at least five variant reads, depth in normal &gt; 50, Somatic Evidence Score (EVS) &gt; 90th percentile of overall EVS distribution. These calls were supplemented by variants called confidently (FILTER = = ”PASS” and manual review) by VarDict in genes recurrently mutated in ccRCC: <em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, <em>KDM5C</em>, <em>PTEN</em>, <em>MTOR</em>, <em>TP53</em>, <em>PIK3CA</em>, <em>ARID1A</em>, <em>STAG2</em>, <em>KDM6A</em>, <em>KMT2C</em>, <em>KMT2D</em>. This strategy improved sensitivity in ccRCC-mutated genes without sacrificing the accuracy of variant calls genome wide. Washington University in St. Louis called somatic variants using four tools: Strelka2, Mutect2, VarScan2.3.8, and Pindel0.2.5. SNVs and indels from the four tools were then merged with SNVs identified by the University of Michigan and GDC pipelines using the following strategy: SNVs called by any two callers among Mutect2, VarScan2.3.8, and Strelka2 and indels called by any two callers among MUTECT2, VarScan2.3.8, Strelka2, and Pindel 0.2.5. For the merged SNVs and indels, we applied a cut-off of 14X and 8X coverage for tumor and normal respectively. SNVs and indels were filtered using a minimal variant allele frequency (VAF) of 0.05 in tumors and a maximal VAF of 0.02 in normal tissues. Any SNV within 10 bps of an indel identified in the same tumor sample was filtered.</div></section><section id="sec5.4.7.2" data-locator="sec5.4.7.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Structural Variant Analysis</span></h4><div class="section-paragraph">Structural variants (SVs) and indels were called from the whole-genome mapped paired-end sequencing reads by Manta (<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d218962e3906" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Chen et al., 2016b</a><div data-db-target-of="bib26-d218962e3906" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Schulz-Trieglaff O. </li><li class="loa__item">Shaw R. </li><li class="loa__item">Barnes B. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Cox A.J. </li><li class="loa__item">Kruglyak S. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (253)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with the default record- and sample-level filters. Record-level filters included a QUAL score &lt; 20; somatic variant quality score &lt; 30; depth greater than 3x the median chromosome depth near one or both variant breakends; for variants significantly larger than the paired read fragment size, no paired reads support the alternate allele in any sample). Sample-level filters included a Genotype Quality &lt; 15. This approach optimizes the analysis of somatic variation in tumor/normal sample pairs. The paired and split-read evidence was combined during the SV discovery and scoring to improve accuracy but did not require split-reads or successful breakpoint assemblies to report a variant in cases where there is strong evidence otherwise. Calls were prioritized with three confidence levels based on whether the read evidence included spanning read pairs (level 1: no spanning read pairs, only split-reads; level 2: one spanning read pair with or without split-reads; level 3: two or more spanning read pairs with or without split-reads) (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>). We mainly focused on levels 2 and 3 SVs with spanning read pair evidence. We extracted the chr3 translocation events from the SV calls and calculated the prevalence of each chr3 translocation types.</div><div class="section-paragraph">We used an independent structural variant calling method LUMPY to validate the translocation events (<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d218962e3914" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Layer et al., 2014</a><div data-db-target-of="bib84-d218962e3914" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Layer R.M. </li><li class="loa__item">Chiang C. </li><li class="loa__item">Quinlan A.R. </li><li class="loa__item">Hall I.M. </li></ul><div class="reference-citations__detials">LUMPY: a probabilistic framework for structural variant discovery.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">R84</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2014-15-6-r84&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24970577&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84912096454&amp;cf=" onclick="newWindow(this.href);return false">Scopus (388)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=R84&journal=Genome+Biol.&author=Layer+R.M.&author=Chiang+C.&author=Quinlan+A.R.&author=Hall+I.M.&title=LUMPY%3A+a+probabilistic+framework+for+structural+variant+discovery">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), selecting the LUMPY Express mode for automated breakpoint detection by using the default parameters. LUMPY Express expected BWA-MEM aligned BAM files as input and automatically parsed sample, library, and read group. Before running LUMPY, we extracted the split and discordant read-pairs based on the whole-genome mapped paired-end sequencing reads of each sample, which were needed by LUMPY. The VCF output file contained the number of supporting reads for each SV event. For those translocation events that were also detected in LUMPY supported by spanning reads, we labeled them ‘Yes’ in the <em>Validated_by_LUMPY</em> column (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>) indicating that they were validated.</div></section><section id="sec5.4.7.3" data-locator="sec5.4.7.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Methylation Analysis</span></h4><div class="section-paragraph">Raw data from Illumina’s EPIC methylation array were made available by GCCs as IDAT files from the CPTAC consortium. The methylation analysis was performed using the cross-package workflow “methylationArrayAnalysis” (<a class="inter-ref-link" href="https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html" target="_blank">https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html</a>) available on Bioconductor. In brief, the raw data files (IDAT files) were processed to obtain the methylated (M) and unmethylated (U) signal intensities for each locus. The processing step included an unsupervised functional normalization step that has been previously implemented for Illumina 450K methylation arrays (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d218962e3930" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Fortin et al., 2014</a><div data-db-target-of="bib45-d218962e3930" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fortin J.P. </li><li class="loa__item">Labbe A. </li><li class="loa__item">Lemire M. </li><li class="loa__item">Zanke B.W. </li><li class="loa__item">Hudson T.J. </li><li class="loa__item">Fertig E.J. </li><li class="loa__item">Greenwood C.M.T. </li><li class="loa__item">Hansen K.D. </li></ul><div class="reference-citations__detials">Functional normalization of 450k methylation array data improves replication in large cancer studies.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (232)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.T.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A detection p value was also calculated for each locus, and this p value captured the quality of detection at the locus with respect to negative control background probes included in the array. Loci having common SNPs (with MAF &gt; 0.01), as per dbSNP build 132 through 147 via snp132 through snp147 common tracks at the UCSC Browser, were removed from further analysis. Beta values were calculated as M/(M+U), which is equal to the fraction methylated at each locus. Beta values of loci with detection p value &gt; 0.01 were assigned NA in the output file. All loci were annotated with the annotation information file ‘MethylationEPIC_v-1-0_B2.csv’ from the zip archive ‘infinium-methylationepic-v1-0-b2-manifest-file-csv.zip’ (<a class="inter-ref-link" href="https://www.illumina.com" target="_blank">https://www.illumina.com</a>) through the Bioconductor IlluminaHumanMethylationEPICanno.ilm10b2.hg19 package.</div></section><section id="sec5.4.7.4" data-locator="sec5.4.7.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Classification of Samples with CpG Island Methylator Phenotype (CIMP)</span></h4><div class="section-paragraph">To classify the tumor samples into CpG island methylator phenotypes (CIMP+ and CIMP-), we performed hierarchical clustering of the methylation data using CpG probes previously established to distinguish these epigenetic states in ccRCC (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d218962e3941" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Arai et al., 2012</a><div data-db-target-of="bib4-d218962e3941" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Arai E. </li><li class="loa__item">Chiku S. </li><li class="loa__item">Mori T. </li><li class="loa__item">Gotoh M. </li><li class="loa__item">Nakagawa T. </li><li class="loa__item">Fujimoto H. </li><li class="loa__item">Kanai Y. </li></ul><div class="reference-citations__detials">Single-CpG-resolution methylome analysis identifies clinicopathologically aggressive CpG island methylator phenotype clear cell renal cell carcinomas.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1487-1493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fcarcin%2Fbgs177&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22610075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865339838&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=1487-1493&journal=Carcinogenesis&author=Arai+E.&author=Chiku+S.&author=Mori+T.&author=Gotoh+M.&author=Nakagawa+T.&author=Fujimoto+H.&author=Kanai+Y.&title=Single-CpG-resolution+methylome+analysis+identifies+clinicopathologically+aggressive+CpG+island+methylator+phenotype+clear+cell+renal+cell+carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The parameters used for the hierarchical clustering were “Euclidean” for distance, “complete” for method, and “none” for scale. The resulting two clusters were verified using the histopathological stage data and well-known gene methylation markers (<div class="dropBlock reference-citations"><a href="#bib153" id="back-bib153" data-db-target-for="bib153-d218962e3944" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib153">Shenoy et al., 2015</a><div data-db-target-of="bib153-d218962e3944" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shenoy N. </li><li class="loa__item">Vallumsetla N. </li><li class="loa__item">Zou Y. </li><li class="loa__item">Galeas J.N. </li><li class="loa__item">Shrivastava M. </li><li class="loa__item">Hu C. </li><li class="loa__item">Susztak K. </li><li class="loa__item">Verma A. </li></ul><div class="reference-citations__detials">Role of DNA methylation in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>J. Hematol. Oncol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">88</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13045-015-0180-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26198328&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84937417017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=88&journal=J.%C2%A0Hematol.+Oncol.&author=Shenoy+N.&author=Vallumsetla+N.&author=Zou+Y.&author=Galeas+J.N.&author=Shrivastava+M.&author=Hu+C.&author=Susztak+K.&author=Verma+A.&title=Role+of+DNA+methylation+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The CIMP+ group was enriched with late-stage tumors and the CIMP+/− differential marker genes were recapitulated based on our classification (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>H; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>).</div></section><section id="sec5.4.7.5" data-locator="sec5.4.7.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Copy-Number Analysis</span></h4><div class="section-paragraph">Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing data of the tumor and germline DNA, using CNVEX (<a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a>). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals and whole-genome and whole-exome variant calls to compute B-allele frequencies (BAFs) at variable positions (we used VarDict germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3-0.7, with a span of 0.5 (the target and configuration files are provided with CNVEX). The adjusted log coverage-ratios (LR) and BAFs were jointly segmented by a custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d218962e3961" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gey and Lebarbier, 2008</a><div data-db-target-of="bib51-d218962e3961" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gey S. </li><li class="loa__item">Lebarbier E. </li></ul><div class="reference-citations__detials">Using CART to Detect Multiple Change Points in the Mean for Large Sample.<div class="popup-ref"><span class="ref__series"><i>Stat. Syst. Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">12</span></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&journal=Stat.+Syst.+Biol.&author=Gey+S.&author=Lebarbier+E.&title=Using+CART+to+Detect+Multiple+Change+Points+in+the+Mean+for+Large+Sample">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and dynamic programming (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d218962e3964" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Bellman, 1961</a><div data-db-target-of="bib12-d218962e3964" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bellman R. </li></ul><div class="reference-citations__detials">On the approximation of curves by line segments using dynamic programming.<div class="popup-ref"><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), as implemented in the R-package jointseg (<div class="dropBlock reference-citations"><a href="#bib136" id="back-bib136" data-db-target-for="bib136-d218962e3967" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib136">Pierre-Jean et al., 2015</a><div data-db-target-of="bib136-d218962e3967" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pierre-Jean M. </li><li class="loa__item">Rigaill G. </li><li class="loa__item">Neuvial P. </li></ul><div class="reference-citations__detials">Performance evaluation of DNA copy number segmentation methods.<div class="popup-ref"><span class="ref__series"><i>Brief. Bioinform.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">600-615</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbib%2Fbbu026&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25202135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925419952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=600-615&journal=Brief.+Bioinform.&author=Pierre-Jean+M.&author=Rigaill+G.&author=Neuvial+P.&title=Performance+evaluation+of+DNA+copy+number+segmentation+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha = 0.01, trim = 0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs, and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d218962e3970" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Lebarbier, 2005</a><div data-db-target-of="bib85-d218962e3970" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lebarbier E. </li></ul><div class="reference-citations__detials">Detecting multiple change-points in the mean of Gaussian process by model selection.<div class="popup-ref"><span class="ref__series"><i>Signal Processing.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (103)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Processing&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the final set of CNV segments, we chose the CBS-based results as they did not require specifying a prior number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data. The resulting segmented copy-number profiles were then subject to joint inference of tumor purity and ploidy and absolute copy number states, implemented in CNVEX, which is most similar to the mathematical formalism of ABSOLUTE (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d218962e3974" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Carter et al., 2012</a><div data-db-target-of="bib22-d218962e3974" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Carter S.L. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Helman E. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Shen H. </li><li class="loa__item">Zack T. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Onofrio R.C. </li><li class="loa__item">Winckler W. </li><li class="loa__item">Weir B.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Absolute quantification of somatic DNA alterations in human cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">413-421</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnbt.2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22544022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84860782006&amp;cf=" onclick="newWindow(this.href);return false">Scopus (809)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=413-421&journal=Nat.+Biotechnol.&author=Carter+S.L.&author=Cibulskis+K.&author=Helman+E.&author=McKenna+A.&author=Shen+H.&author=Zack+T.&author=Laird+P.W.&author=Onofrio+R.C.&author=Winckler+W.&author=Weir+B.A.&title=Absolute+quantification+of+somatic+DNA+alterations+in+human+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and PureCN (<div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d218962e3977" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Riester et al., 2016</a><div data-db-target-of="bib142-d218962e3977" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Riester M. </li><li class="loa__item">Singh A.P. </li><li class="loa__item">Brannon A.R. </li><li class="loa__item">Yu K. </li><li class="loa__item">Campbell C.D. </li><li class="loa__item">Chiang D.Y. </li><li class="loa__item">Morrissey M.P. </li></ul><div class="reference-citations__detials">PureCN: copy number calling and SNV classification using targeted short read sequencing.<div class="popup-ref"><span class="ref__series"><i>Source Code Biol. Med.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">13</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13029-016-0060-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27999612&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85006152885&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=13&journal=Source+Code+Biol.+Med.&author=Riester+M.&author=Singh+A.P.&author=Brannon+A.R.&author=Yu+K.&author=Campbell+C.D.&author=Chiang+D.Y.&author=Morrissey+M.P.&title=PureCN%3A+copy+number+calling+and+SNV+classification+using+targeted+short+read+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, the algorithm inputs the observed log-ratios (of 10kb bins) and BAFs of individual SNPs. LRs and BAFs are assigned to their joint segments and their likelihood is determined given a particular purity, ploidy, absolute segment copy number, and number of minor alleles. To identify candidate combinations with a high likelihood, we followed a multi-step optimization procedure that includes grid-search (across purity-ploidy combinations), greedy optimization of absolute copy numbers, and maximum-likelihood inferences of minor allele counts. Following optimization, CNVEX ranks candidate solutions. Because the copy-number inference problem can have multiple equally likely solutions, further biological insights are necessary to choose the most parsimonious result. The solutions have been reviewed by independent analysts following a set of guidelines. Solutions implying whole-genome duplication must be supported by at least one large segment that cannot be explained by a low-ploidy solution, inferred purity must be consistent with the variant-allele-frequencies of somatic mutations, and large homozygous segments are not allowed. Clonal loss or LOH of 3p is very likely in ccRCC, whereas near-haploid and very high ploidy solutions are unlikely.</div></section><section id="sec5.4.7.6" data-locator="sec5.4.7.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Classification of Samples with Genome instability</span></h4><div class="section-paragraph">To classify the tumor samples into genome instability+ samples and genome stability- samples, we calculated the proportion of genome altered (PGA), which is defined as the length of all segments that have copy number (as inferred by CNVEX) different from C = 2 K = 1; i.e., diploid heterozygous divided by the total length of the genome. The resulting PGA score is a bimodal distribution with one peak &lt; 25% and the other peak near 100%. We dichotomized the samples into two genome instability categories using a cutoff of 0.85, which has the local minimal density separating two peaks. The genome instability+ group was enriched with late-stage tumors and the CIMP+/− differential marker genes were recapitulated based on our classification (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>).</div></section></section><section id="sec5.4.8" data-locator="sec5.4.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Transcriptomic Data Processing</span></h4><div class="section-paragraph">Transcriptomic data were analyzed as described previously (<div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d218962e3996" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Robinson et al., 2017</a><div data-db-target-of="bib143-d218962e3996" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson D.R. </li><li class="loa__item">Wu Y.M. </li><li class="loa__item">Lonigro R.J. </li><li class="loa__item">Vats P. </li><li class="loa__item">Cobain E. </li><li class="loa__item">Everett J. </li><li class="loa__item">Cao X. </li><li class="loa__item">Rabban E. </li><li class="loa__item">Kumar-Sinha C. </li><li class="loa__item">Raymond V. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative clinical genomics of metastatic cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">548</span></span>: <span class="ref__seriesPages">297-303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature23306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28783718&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85027893400&amp;cf=" onclick="newWindow(this.href);return false">Scopus (215)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2017&pages=297-303&journal=Nature&author=Robinson+D.R.&author=Wu+Y.M.&author=Lonigro+R.J.&author=Vats+P.&author=Cobain+E.&author=Everett+J.&author=Cao+X.&author=Rabban+E.&author=Kumar-Sinha+C.&author=Raymond+V.&title=Integrative+clinical+genomics+of+metastatic+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), using the Clinical RNA-seq Pipeline (CRISP) developed at University of Michigan (<a class="inter-ref-link" href="https://github.com/mcieslik-mctp/crisp-build" target="_blank">https://github.com/mcieslik-mctp/crisp-build</a>). Briefly, raw sequencing data were trimmed, merged using BBMap, and aligned to GRCh38 using STAR. The resulting BAM files were analyzed for expression using feature counts against a transcriptomic reference based on Gencode 26. The resulting gene-level counts for protein-coding genes were transformed into FPKMs using edgeR.</div></section><section id="sec5.4.9" data-locator="sec5.4.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic Data Processing</span></h4><section id="sec5.4.9.1" data-locator="sec5.4.9.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein database searching and quantification of global and phosphoproteomic data</span></h4><div class="section-paragraph">Raw mass spectrometry files were converted into open mzML format using the msconvert utility of the Proteowizard software suite. MS/MS spectra were searched using the MSFragger database search tool (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d218962e4010" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Kong et al., 2017</a><div data-db-target-of="bib80-d218962e4010" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) against a CPTAC harmonized RefSeq protein sequence database appended with an equal number of decoy sequences. For the analysis of whole proteome data, MS/MS spectra were searched using a precursor-ion mass tolerance of 20 ppm, fragment mass tolerance of 20 ppm, and allowing C12/C13 isotope errors (−1/0/1/2/3). Cysteine carbamidomethylation (+57.0215) and lysine TMT labeling (+229.1629) were specified as fixed modifications, and methionine oxidation (+15.9949), N-terminal protein acetylation (+42.0106), and TMT labeling of peptide N terminus and serine residues were specified as variable modifications. The search was restricted to fully tryptic peptides, allowing up to two missed cleavage sites. For the analysis of phosphopeptide enriched data, the set of variable modifications also included phosphorylation (+79.9663) of serine, threonine, and tyrosine residues.</div><div class="section-paragraph">The search results were further processed using the Philosopher pipeline (<a class="inter-ref-link" href="https://github.com/Nesvilab/philosopher" target="_blank">https://github.com/Nesvilab/philosopher</a>). Whole proteome and phosphopeptide-enriched datasets were processed separately but using the same pipeline steps, except when noted. First, MSFragger output files (in pepXML format) were processed using PeptideProphet (<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d218962e4018" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Keller et al., 2002</a><div data-db-target-of="bib78-d218962e4018" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keller A. </li><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3457)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal.+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (with the high–mass accuracy binning and semi-parametric mixture modeling options) to compute the posterior probability of correct identification for each peptide to spectrum match (PSM). In the case of the phosphopeptide-enriched dataset, PeptideProphet files were additionally processed using PTMProphet (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d218962e4021" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Deutsch et al., 2015</a><div data-db-target-of="bib34-d218962e4021" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Deutsch E.W. </li><li class="loa__item">Mendoza L. </li><li class="loa__item">Shteynberg D. </li><li class="loa__item">Slagel J. </li><li class="loa__item">Sun Z. </li><li class="loa__item">Moritz R.L. </li></ul><div class="reference-citations__detials">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics.<div class="popup-ref"><span class="ref__series"><i>Proteomics Clin. Appl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">745-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fprca.201400164&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25631240&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84948987628&amp;cf=" onclick="newWindow(this.href);return false">Scopus (125)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=745-754&journal=Proteomics+Clin.+Appl.&author=Deutsch+E.W.&author=Mendoza+L.&author=Shteynberg+D.&author=Slagel+J.&author=Sun+Z.&author=Moritz+R.L.&title=Trans-Proteomic+Pipeline%2C+a+standardized+data+processing+pipeline+for+large-scale+reproducible+proteomics+informatics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to localize the phosphorylation sites. The resulting pepXML files from PeptideProphet (or PTMProphet) from all 23 TMT 10-plex experiments were then processed together to assemble peptides into proteins (protein inference) and to create a combined file (in protXML format) of high confidence protein groups. Corresponding peptides were assigned to each group. The combined protXML file and the individual PSM lists for each TMT 10-plex were further processed using Philosopher filter command as follows. Each peptide was assigned either as a unique peptide to a particular protein group or assigned as a razor peptide to a single protein group that had the most peptide evidence. The protein groups assembled by ProteinProphet (<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d218962e4024" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Nesvizhskii et al., 2003</a><div data-db-target-of="bib118-d218962e4024" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Keller A. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">A statistical model for identifying proteins by tandem mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3129)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal.+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were filtered to 1% protein-level False Discovery Rate (FDR) using the chosen FDR target-decoy strategy and the best peptide approach (allowing both unique and razor peptides) and applying the picked FDR strategy (<div class="dropBlock reference-citations"><a href="#bib147" id="back-bib147" data-db-target-for="bib147-d218962e4027" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib147">Savitski et al., 2015</a><div data-db-target-of="bib147-d218962e4027" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Savitski M.M. </li><li class="loa__item">Wilhelm M. </li><li class="loa__item">Hahne H. </li><li class="loa__item">Kuster B. </li><li class="loa__item">Bantscheff M. </li></ul><div class="reference-citations__detials">A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (125)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell.+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+Scalable+Approach+for+Protein+False+Discovery+Rate+Estimation+in+Large+Proteomic+Data+Sets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In each TMT 10-plex, the PSM lists were filtered using a stringent, sequential FDR strategy, retaining only those PSMs with PeptideProphet probability of 0.9 or higher (which in these data corresponded to less than 1% PSM-level FDR) and mapped to proteins that also passed the global 1% protein-level FDR filter. For each PSM that passed these filters, MS1 intensity of the corresponding precursor-ion was extracted using the Philosopher label-free quantification module based on the moFF method (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d218962e4031" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Argentini et al., 2016</a><div data-db-target-of="bib6-d218962e4031" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Argentini A. </li><li class="loa__item">Goeminne L.J.E. </li><li class="loa__item">Verheggen K. </li><li class="loa__item">Hulstaert N. </li><li class="loa__item">Staes A. </li><li class="loa__item">Clement L. </li><li class="loa__item">Martens L. </li></ul><div class="reference-citations__detials">moFF: a robust and automated approach to extract peptide ion intensities.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">964-966</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4075&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27898063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84999277470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=964-966&journal=Nat.+Methods&author=Argentini+A.&author=Goeminne+L.J.E.&author=Verheggen+K.&author=Hulstaert+N.&author=Staes+A.&author=Clement+L.&author=Martens+L.&title=moFF%3A+a+robust+and+automated+approach+to+extract+peptide+ion+intensities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (using 10 p.p.m mass tolerance and 0.4 min retention time window for extracted ion chromatogram peak tracing). In addition, for all PSMs corresponding to a TMT-labeled peptide, ten TMT reporter ion intensities were extracted from the MS/MS scans (using 0.002 Da window) and the precursor ion purity scores were calculated using the intensity of the sequenced precursor ion and that of other interfering ions observed in MS1 data (within a 0.7 Da isolation window). All supporting information for each PSM, including the accession numbers and names of the protein/gene selected based on the protein inference approach with razor peptide assignment and quantification information (MS1 precursor-ion intensity and the TMT reporter ion intensities) was summarized in the output PSM.tsv files, one file for each TMT 10-plex experiment. The PSM.tsv files were further processed using TMT-Integrator (<a class="inter-ref-link" href="https://github.com/Nesvilab/TMT-Integrator" target="_blank">https://github.com/Nesvilab/TMT-Integrator</a>) to generate summary reports at the gene and protein level and, for phosphopeptide enriched data, also at the peptide and modification site levels. In the quantitation step, TMT-Integrator used as input the PSM tables generated by the Philosopher pipeline as described above and created integrated reports with quantification across all samples at each level. First, PSM from PSM.tsv files were filtered to remove all entries that did not pass at least one of the quality filters, such as PSMs with (a) no TMT label; (b) missing quantification in the Reference sample; (c) precursor-ion purity less than 50%; (d) summed reporter ion intensity (across all ten channels) in the lower 5% percentile of all PSMs in the corresponding PSM.tsv file (2.5% for phosphopeptide enriched data); (e) peptides without phosphorylation (for phosphopeptide enriched data). In the case of redundant PSMs (i.e., multiple PSMs in the same MS run sample corresponding the same peptide ion), only the single PSM with the highest summed TMT intensity was retained for subsequent analysis. Both unique and razor peptides were used for quantification, while PSMs mapping to common external contaminant proteins (that were included in the searched protein sequence database) were excluded. Next, in each TMT 10-plex experiment, for each PSM the intensity in each TMT channel was log2 transformed, and the reference channel intensity (pooled reference sample) was subtracted from that for the other nine channels (samples), thus converting the data into log2-based ratio to the reference scale (referred to as ‘ratios’ below). After the ratio-to-reference conversion, the PSMs were grouped on the basis of a predefined level (gene, protein, and also peptide and site-level for phosphopeptide enriched data; see below for details). At each level, and in each sample, the interquartile range (IQR) algorithm was applied to remove the outliers in the corresponding PSM group. The first quantile (Q1), the third quantile (Q3), and the interquartile range (IQR, i.e., Q3-Q1) of the sample ratios were calculated, and the PSMs with ratios outside of the boundaries of Q1-1.5<sup>∗</sup>IQR and Q3+1.5<sup>∗</sup>IQR were excluded. Then, the median was calculated from the remaining ratios to represent the ratio for each sample, at every level. In the next step, the ratios were normalized using the median absolute deviation (MAD). Briefly, independently at each level of data summarization (gene, protein, peptide, or site), given the p by n table of ratios for entry j in sample i, R<sub>ij</sub>, the median ratio M<sub>i</sub> = median(R<sub>ij</sub>, j = 1,…,p), and the global median across all n samples, M<sub>0</sub> = median(M<sub>i</sub>, i = 1,…,n), were calculated. The ratios in each sample were median centered, R<sup>C</sup><sub>ij</sub> = R<sub>ij</sub> – M<sub>i</sub>. The median absolute deviation of centered values in each sample, MAD<sub>i</sub> = median(abs(R<sup>C</sup><sub>ij</sub>), j = 1…p) was calculated along with the global absolute deviation, MAD<sub>0</sub> = median(MAD<sub>i</sub>, i = 1,…,n). All ratios were then scaled to derive the final normalized measures: R<sup>N</sup><sub>ij</sub> = (R<sup>C</sup><sub>ij</sub>/ MAD<sub>i</sub>) × MAD<sub>0</sub> + M<sub>0</sub>. As a final step, the normalized ratios were converted back to the absolute intensity scale using the estimated intensity of each entry (at each level, gene/protein/peptide/site) in the Reference sample. The Reference Intensity of entry i measured in TMT 10-plex k (k = 1,…,q), REF<sub>ik</sub>, was estimated using the weighted sum of the MS1 intensities of the top three most intense peptide ions (<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d218962e4109" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Ning et al., 2012</a><div data-db-target-of="bib119-d218962e4109" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ning K. </li><li class="loa__item">Fermin D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">Comparative analysis of different label-free mass spectrometry based protein abundance estimates and their correlation with RNA-Seq gene expression data.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2261-2271</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr201052x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22329341&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84859611662&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=2261-2271&journal=J.%C2%A0Proteome+Res.&author=Ning+K.&author=Fermin+D.&author=Nesvizhskii+A.I.&title=Comparative+analysis+of+different+label-free+mass+spectrometry+based+protein+abundance+estimates+and+their+correlation+with+RNA-Seq+gene+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) quantified for that entry in the TMT 10-plex k. The weighting factor for each PSM was taken as the proportion of the reference channel TMT intensity to the total summed TMT channel intensity. The overall Reference Intensity for entry i was then computed as REF<sub>i</sub> = Mean(REF<sub>ik</sub>, k = 1,…,q). In doing so, the missing intensity values (i.e., no identified and/or quantified PSMs in a particular TMT 10-plex experiment) were imputed with a global minimum intensity value. The final abundance (intensity) of entry i in sample j (log2 transformed) was computed as A<sub>ij</sub> = R<sup>N</sup><sub>ij</sub> + log2(REF<sub>i</sub>). The ratio and intensity tables described above were calculated separately for each level (gene and protein for whole proteome, and also peptide and site-level for phosphopeptide enriched data). PSMs were grouped as follows. At the gene level, all PSMs were grouped based on the gene symbol of the corresponding protein to which they were assigned as either unique or razor peptides. In the protein tables, identified proteins that mapped to the same gene were kept as separate entries. To generate peptide-level and site-level tables, additional post-processing was applied to generate all non-conflicting phosphosite configurations using a strategy similar to that described in <div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d218962e4130" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Huang et al., 2017</a><div data-db-target-of="bib69-d218962e4130" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang K.L. </li><li class="loa__item">Li S. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Cao S. </li><li class="loa__item">Gunawardena H.P. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Tanioka M. </li><li class="loa__item">Usary J. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic integration reveals therapeutic targets in breast cancer xenografts.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">14864</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms14864&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28348404&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016396817&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14864&journal=Nat.+Commun.&author=Huang+K.L.&author=Li+S.&author=Mertins+P.&author=Cao+S.&author=Gunawardena+H.P.&author=Ruggles+K.V.&author=Mani+D.R.&author=Clauser+K.R.&author=Tanioka+M.&author=Usary+J.&title=Proteogenomic+integration+reveals+therapeutic+targets+in+breast+cancer+xenografts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. In doing so, confidently localized sites were defined as sites with PTMProphet localization probability of 0.9 or higher. The same peptide sequences but with different site configurations, i.e., different site localization configurations or peptides with unlocalized sites, were retained as separate entries in the site-level tables. In the peptide-level tables, different site-level configurations were combined into a single peptide-level index, grouping PSMs with all site configurations together if they corresponded to the same peptide sequence. The tutorial describing all steps of the analysis, including specific input parameter files, command-line option, and all software tools necessary to replicate the results are available at <a class="inter-ref-link" href="https://github.com/Nesvilab" target="_blank">https://github.com/Nesvilab</a>.</div></section><section id="sec5.4.9.2" data-locator="sec5.4.9.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Creation of a Patient-Specific Protein Sequence Database</span></h4><div class="section-paragraph">The proteogenomic database tool pyQUILTS (<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d218962e4141" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Ruggles et al., 2016</a><div data-db-target-of="bib145-d218962e4141" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Tang Z. </li><li class="loa__item">Wang X. </li><li class="loa__item">Grover H. </li><li class="loa__item">Askenazi M. </li><li class="loa__item">Teubl J. </li><li class="loa__item">Cao S. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Tabb D.L. </li><li>et al.</li></ul><div class="reference-citations__detials">An Analysis of the Sensitivity of Proteogenomic Mapping of Somatic Mutations and Novel Splicing Events in Cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+Analysis+of+the+Sensitivity+of+Proteogenomic+Mapping+of+Somatic+Mutations+and+Novel+Splicing+Events+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), available at <a class="inter-ref-link" href="http://quilts.fenyolab.org" target="_blank">http://quilts.fenyolab.org</a>, was used to incorporate the germline and somatic SNVs and RNA-seq-predicted junctions into a searchable protein database. The human RefSeq protein database (downloaded 2018/06/29) was used as a reference for the hg38 proteome and genome.</div></section><section id="sec5.4.9.3" data-locator="sec5.4.9.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Variant Peptide Identification and Neoantigen Prediction</span></h4><div class="section-paragraph">NeoFlow (<a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a>) was used for neoantigen prediction. Specifically, Optitype (<div class="dropBlock reference-citations"><a href="#bib165" id="back-bib165" data-db-target-for="bib165-d218962e4155" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib165">Szolek et al., 2014</a><div data-db-target-of="bib165-d218962e4155" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to identify human leukocyte antigens (HLA) in the WES data. netMHCpan (<div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d218962e4158" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Jurtz et al., 2017</a><div data-db-target-of="bib75-d218962e4158" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (216)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was then used to predict HLA peptide binding affinity for somatic mutation–derived variant peptides with a length between 8-11 amino acids. The cutoff of IC<sub>50</sub> binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at the mRNA and protein levels using RNA-seq data and MS/MS data, respectively. To identify variant peptides, we used a customized protein sequence database approach (<div class="dropBlock reference-citations"><a href="#bib174" id="back-bib174" data-db-target-for="bib174-d218962e4164" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib174">Wang et al., 2012</a><div data-db-target-of="bib174-d218962e4164" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang X. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Wang D. </li><li class="loa__item">Halvey P.J. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Liebler D.C. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Protein identification using customized protein sequence databases derived from RNA-Seq data.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (87)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.C.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases+derived+from+RNA-Seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Two different workflows were used. In the first workflow, we derived customized protein sequence databases from matched WES data and performed database searching using the customized databases for individual TMT experiments. We built a customized database for each TMT experiment based on somatic variants from WES data. Customprodbj (<a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a>) was used to construct customized databases. MS-GF+ was used to identify variant peptides for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at PSM level. Remaining variant peptides were further filtered using PepQuery (<a class="inter-ref-link" href="http://www.pepquery.org" target="_blank">http://www.pepquery.org</a>) (<div class="dropBlock reference-citations"><a href="#bib178" id="back-bib178" data-db-target-for="bib178-d218962e4174" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib178">Wen et al., 2019</a><div data-db-target-of="bib178-d218962e4174" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with the p value cutoff ≤ 0.01. Variant peptide spectra were annotated using PDV (<a class="inter-ref-link" href="http://www.zhang-lab.org/" target="_blank">http://www.zhang-lab.org/</a>) (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d218962e4180" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Li et al., 2019</a><div data-db-target-of="bib87-d218962e4180" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In the second workflow, the RefSeq-based protein database used in the main analysis was trypsin digested <em>in silico</em> allowing up to one missed cleavage and treating N-terminal methionine excision as a variable modification to produce two sets of N-terminal peptides (methionine excised and methionine retained). One additional missed cleavage was retained for peptides containing KP and RP amino acids. Isoleucine and leucine occurrences were set to leucine as they are indistinguishable during peptide sequencing. QUILT-derived, patient-specific protein FASTA files corresponding to all samples within a given TMT-plex were combined. From these files, a set of unique peptides was generated from each protein, and any of these peptides that occurred within the RefSeq database were removed. The result of this process was a protein FASTA file, in which every peptide present within the combined personalized QUILT FASTA that was not found within the RefSeq database was retained for searching. The new customized databases were used to search each corresponding dataset against spectra previously classified as having a PeptideProphet score lower than 0.5. The new database searches were carried out using the MSFragger methodology described previously (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d218962e4186" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Kong et al., 2017</a><div data-db-target-of="bib80-d218962e4186" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Resulting PSMs were filtered, keeping only charge states 2, 3 and 4 and PSMs with delta mass between &gt; -.05 and 2.5. After scoring all identifications with FDRs, the best PSM from each experiment was selected, generating quantitation tables of raw abundance and ratio to common reference. In global and phosphorylation-enriched datasets, post processing was completed to annotate misidentified novel peptides that are contained within UniProt Swiss-Prot (ret. 22 June 2019) and to identify the patients from which each novel peptide was derived. In the phosphorylation-enriched dataset, a column (called isMatch) was added to check whether mutation sites overlap with novel peptide phosphorylation sites. The somatic variant peptides identified by either of the two workflows were used for downstream analysis.</div></section><section id="sec5.4.9.4" data-locator="sec5.4.9.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein database searching and quantification of global data independent acquisition data</span></h4><div class="section-paragraph">Raw mass spectrometry files from DIA (n = 194) and DDA (n = 8) platforms were processed using the DIA-Umpire (<div class="dropBlock reference-citations"><a href="#bib168" id="back-bib168" data-db-target-for="bib168-d218962e4194" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib168">Tsou et al., 2015</a><div data-db-target-of="bib168-d218962e4194" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tsou C.C. </li><li class="loa__item">Avtonomov D. </li><li class="loa__item">Larsen B. </li><li class="loa__item">Tucholska M. </li><li class="loa__item">Choi H. </li><li class="loa__item">Gingras A.C. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">258-264</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.3255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25599550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923838580&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=258-264&journal=Nat.+Methods&author=Tsou+C.C.&author=Avtonomov+D.&author=Larsen+B.&author=Tucholska+M.&author=Choi+H.&author=Gingras+A.C.&author=Nesvizhskii+A.I.&title=DIA-Umpire%3A+comprehensive+computational+framework+for+data-independent+acquisition+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) based pipeline to generate a combined spectral library that integrated DDA and DIA search results. In brief, DIA data were first processed using DIA-Umpire to generate deconvoluted (pseudo-MS/MS) spectra. DDA and pseudo-MS/MS DIA spectra were then searched using the MSFragger (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d218962e4197" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Kong et al., 2017</a><div data-db-target-of="bib80-d218962e4197" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) search engine against the same CPTAC harmonized RefSeq protein sequence database as for TMT data analysis (with an addition of the sequences of iRT peptides). The search was completed using tryptic peptides only, allowing up to two missed cleavages, allowing methionine oxidation and protein N-terminal acetylation as variable modifications, and cysteine carbamylation as a fixed modification. The search results were further processed using the Philosopher pipeline, including PeptideProphet and iProphet (run using the same settings as for TMT data as described above). Retention times of peptide identifications from all DIA and DDA runs were aligned to a single reference DIA run using high quality peptide identifications. ProteinProphet was run using all iProphet pepXML files (i.e., from all DIA and DDA runs) to generate a single combined protXML file, and the protein list was filtered to 1% protein-level FDR. PSMs identified in each individual data subset (DDA or DIA) were then filtered using the Philosopher filter utility to 1% peptide ion-level FDR separately in each subset. Only those PSMs that mapped to proteins in the 1% protein-level FDR filtered combined DDA plus DIA protein list were retained. These filtered PSMs, with aligned retention times, were used to generate two spectral libraries using SpectraST, one for DIA and one for DDA data subsets. The retention times were further transformed to the indexed retention time (iRT) scale using standard peptides spiked into the samples. The consensus MS/MS spectra were generated for each peptide ion, and the spectral libraries were refined to contain only selected peptide fragments using the spectrast2tsv script from the msproteomicstools resource (<a class="inter-ref-link" href="https://github.com/msproteomicstools" target="_blank">https://github.com/msproteomicstools</a>), requiring a minimum of three and a maximum of six fragments per spectrum, fragment m/z values between 250 and 2000 Da only, and <em>b</em> and <em>y</em> ion types only, but allowing neutral losses of water or ammonia. The resulting DIA and DDA spectral libraries were combined. When the same peptide ion was present in both libraries, the DIA spectrum was selected for the combined library. The combined library was then converted to Spectronaut (Biognosys) format for subsequent targeted re-extraction of quantification information. The combined spectrum library was loaded into Spectronaut, and targeted quantification was performed using default settings. For protein quantification, all abundances were calculated as the area under the extracted ion chromatogram (XIC) of all selected fragments that passed filtering. The data were filtered using the global protein-level FDR value of 1%, and proteins were quantified in each individual DIA run using peptides that passed the run-specific peptide ion q-value of 0.01. Protein abundances for select candidates (PECAM1, VEGFA, PKM, HLA-C, C5, HLA-A, PGM1, HLA-B, POSTN, STAT1) were extracted and reported in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>. DIA expression matrix of select proteins (ccRCC-DIA_selected_proteins.csv) is available at <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a></div></section><section id="sec5.4.9.5" data-locator="sec5.4.9.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preprocessing of ccRCC proteomics data</span></h4><div class="section-paragraph">Global protein and phosphosite abundances from 194 samples were measured in 23 TMT-10plexes in this experiment, which identified 11,355 unique genes in global protein abundance along with 100,730 phosphosites. There were 18.4% data missing in global protein abundance with 7,150 genes completely observed. In the phosphoproteomic analyses, 67.5% of data was missing in phosphosite abundance, with 5,584 phosphosites from 2,443 genes completely observed. Before performing any downstream analysis, we applied batch correction on global and phosphoproteome abundance to remove the technical difference between different TMT 10-plexes. An R tool, ComBat, with tumor/normal status adjustment was applied to remove batch effects (<div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d218962e4218" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Johnson et al., 2007</a><div data-db-target-of="bib72-d218962e4218" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johnson W.E. </li><li class="loa__item">Li C. </li><li class="loa__item">Rabinovic A. </li></ul><div class="reference-citations__detials">Adjusting batch effects in microarray expression data using empirical Bayes methods.<div class="popup-ref"><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">118-127</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbiostatistics%2Fkxj037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16632515&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33845432928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2518)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=118-127&journal=Biostatistics&author=Johnson+W.E.&author=Li+C.&author=Rabinovic+A.&title=Adjusting+batch+effects+in+microarray+expression+data+using+empirical+Bayes+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To impute missing values, we used DreamAI (<a class="inter-ref-link" href="https://github.com/WangLab-MSSM/DreamAI" target="_blank">https://github.com/WangLab-MSSM/DreamAI</a>), an ensemble algorithm developed during the NCI-CPTAC Dream Proteomics Imputation Challenge (<a class="inter-ref-link" href="https://www.synapse.org/#!Synapse:syn8228304/wiki/413428" target="_blank">https://www.synapse.org/#!Synapse:syn8228304/wiki/413428</a>). Only those phosphosites and proteins with a missing rate less than 50% were imputed. After imputation, the number of phosphosites deemed valid for downstream analyses was boosted to 26,814 (from 5,571 genes).</div></section><section id="sec5.4.9.6" data-locator="sec5.4.9.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample labeling check of ccRCC dataset</span></h4><div class="section-paragraph">Integrating multiple layers of omics data enhances our understanding of complex molecular mechanisms in biological systems. However, unintended errors in annotations and sample mislabeling often occur when generating and managing large-scale data (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d218962e4232" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">Alyass et al., 2015</a><div data-db-target-of="bib1-d218962e4232" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alyass A. </li><li class="loa__item">Turcotte M. </li><li class="loa__item">Meyre D. </li></ul><div class="reference-citations__detials">From big data analysis to personalized medicine for all: challenges and opportunities.<div class="popup-ref"><span class="ref__series"><i>BMC Med. Genomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">33</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs12920-015-0108-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26112054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84933530616&amp;cf=" onclick="newWindow(this.href);return false">Scopus (221)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=33&journal=BMC+Med.+Genomics&author=Alyass+A.&author=Turcotte+M.&author=Meyre+D.&title=From+big+data+analysis+to+personalized+medicine+for+all%3A+challenges+and+opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Since integrative analysis based on erroneous data could lead to inaccurate scientific conclusions, a sample-labeling check is a critical QC step before integration. In this study, we performed a systematic QC procedure to confirm that all annotations in clinical information and sample names were consistent. We checked tissue annotations (tumor or normal), gender annotations (male or female), and sample matching among RNA-seq, proteomics and phosphorylation data.<ul class="ce-list--remove-bullets" id="olist0010"><li class="ce-list--remove-bullets__list-item" id="o0010"><span class="label">1.</span><div class="ce-list--remove-bullets__list-item__text">Tissue Annotation. We performed PCA independently in RNA-seq, proteomics and phosphorylation data. As expected, normal and tumor tissues were well separated without exception, suggesting that tissue information is consistent with given annotation.</div></li><li class="ce-list--remove-bullets__list-item" id="o0015"><span class="label">2.</span><div class="ce-list--remove-bullets__list-item__text">Gender Annotation. Expression of marker genes in X and Y chromosomes can help to infer gender of the samples from which they are analyzed (<div class="dropBlock reference-citations"><a href="#bib161" id="back-bib161" data-db-target-for="bib161-d218962e4244" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib161">Staedtler et al., 2013</a><div data-db-target-of="bib161-d218962e4244" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Staedtler F. </li><li class="loa__item">Hartmann N. </li><li class="loa__item">Letzkus M. </li><li class="loa__item">Bongiovanni S. </li><li class="loa__item">Scherer A. </li><li class="loa__item">Marc P. </li><li class="loa__item">Johnson K.J. </li><li class="loa__item">Schumacher M.M. </li></ul><div class="reference-citations__detials">Robust and tissue-independent gender-specific transcript biomarkers.<div class="popup-ref"><span class="ref__series"><i>Biomarkers.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">436-445</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3109%2F1354750X.2013.811538&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23829492&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84880540860&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=436-445&journal=Biomarkers&author=Staedtler+F.&author=Hartmann+N.&author=Letzkus+M.&author=Bongiovanni+S.&author=Scherer+A.&author=Marc+P.&author=Johnson+K.J.&author=Schumacher+M.M.&title=Robust+and+tissue-independent+gender-specific+transcript+biomarkers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). After combining all normal and tumor RNA-seq samples, male- and female-annotated samples were compared on a t test. Two Y chromosome genes (<em>RPS4Y1</em> and <em>DDX3Y</em>) showed the strongest separation (t test p values = 6.1 × 10<sup>−140</sup> and 4.4 × 10<sup>−136</sup> respectively). When using the expression of these two genes, genders agreed between annotation and expression in all samples. When a similar approach was applied to proteomics and phosphorylation data, the signal was less clear than in RNA-seq data. Hence, it proved difficult to check gender consistency using proteomics and phosphorylation data.</div></li><li class="ce-list--remove-bullets__list-item" id="o0020"><span class="label">3.</span><div class="ce-list--remove-bullets__list-item__text">Sample Alignment. Although we assessed the accuracy of tissues and gender information, sample mislabeling could still occur by swapping, shifting, or duplicating two samples from the same tissue or genders. Therefore, it is necessary to confirm that complementary samples used in RNA-seq, proteomics, and phosphorylation analyses are from the same individuals. We applied a pairwise alignment procedure developed by us previously (<div class="dropBlock reference-citations"><a href="#bib185" id="back-bib185" data-db-target-for="bib185-d218962e4261" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib185">Yoo et al., 2014</a><div data-db-target-of="bib185-d218962e4261" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoo S. </li><li class="loa__item">Huang T. </li><li class="loa__item">Campbell J.D. </li><li class="loa__item">Lee E. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Geraci M.W. </li><li class="loa__item">Powell C.A. </li><li class="loa__item">Schadt E.E. </li><li class="loa__item">Spira A. </li><li class="loa__item">Zhu J. </li></ul><div class="reference-citations__detials">MODMatcher: multi-omics data matcher for integrative genomic analysis.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using all samples that compared global proteomics and phosphoproteomics, RNA-seq and global proteomics, and RNA-seq and phosphoproteomics. First, the top 500 <em>cis</em> genes were identified based on the correlation coefficient for each of proteomics-phosphorylation, RNA-seq-proteomics, and RNA-seq-phosphorylation pair. Then the values of the 500 genes were rank-transformed to obtain a sample-wise similarity score. If a sample matches between two types of data, its sample similarity score should be higher than the score when compared to other cases, which have null distribution with mean 0. Using this approach, we confirmed that all 110 tumor and 84 normal tissues were perfectly matched between global proteomic and phosphoproteomic data. All tumor samples in RNA-seq were well matched with their corresponding global proteomic and phosphoproteomic data except for one normal sample, CPT0012090003, whose self-similarity score did not differ from scores corresponding to other samples (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>A; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>). Indeed, the RNA expression of this sample did not match its proteomic profile (both global and phosphoprotein abundance). While this error could arise from either RNA-seq or global-proteomic/phosphorylation data since global proteomic and phosphorylation data for this sample were well aligned. This sample was removed from all further downstream analysis as the error-source remained unresolved.</div></li></ul></div></section></section><section id="sec5.4.10" data-locator="sec5.4.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CNV Integrated Analysis</span></h4><section id="sec5.4.10.1" data-locator="sec5.4.10.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic determination of 103 ccRCC versus 7 non-ccRCC samples from 110 pathologically defined ccRCC tumors</span></h4><div class="section-paragraph">Outlier samples identified PCA of by RNA-seq data using Omics Explorer (Qlucore, Lund, Sweden) also exhibited low expression of ccRCC biomarkers such as <em>ANGPTL4</em>, <em>CA9</em>, and <em>NDUFA4L2,</em> among others and were subject to further critical evaluation for genomic aberrations (copy number variations (CNVs) and mutations) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>B–S1D). Samples C3N-00492 and C3N-00175 showed one copy loss of chromosomes 1, 2, 6, 10, 13 and 17 along with <em>TP53</em> mutations and contained high expression of several biomarkers (such as FOXI1, RHCG) (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d218962e4292" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Lindgren et al., 2017</a><div data-db-target-of="bib93-d218962e4292" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lindgren D. </li><li class="loa__item">Eriksson P. </li><li class="loa__item">Krawczyk K. </li><li class="loa__item">Nilsson H. </li><li class="loa__item">Hansson J. </li><li class="loa__item">Veerla S. </li><li class="loa__item">Sjölund J. </li><li class="loa__item">Höglund M. </li><li class="loa__item">Johansson M.E. </li><li class="loa__item">Axelson H. </li></ul><div class="reference-citations__detials">Cell-Type-Specific Gene Programs of the Normal Human Nephron Define Kidney Cancer Subtypes.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1476-1489</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28793269&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85026887625&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=1476-1489&journal=Cell+Rep.&author=Lindgren+D.&author=Eriksson+P.&author=Krawczyk+K.&author=Nilsson+H.&author=Hansson+J.&author=Veerla+S.&author=Sj%C3%B6lund+J.&author=H%C3%B6glund+M.&author=Johansson+M.E.&author=Axelson+H.&title=Cell-Type-Specific+Gene+Programs+of+the+Normal+Human+Nephron+Define+Kidney+Cancer+Subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) that are characteristic of chromophobe RCC (chRCC). In addition, samples C3N-00832 and C3N-00313 contained <em>PTEN</em> mutations, and the latter also showed outlier expression of the papillary RCC biomarker <em>VSTM2A</em> (<div class="dropBlock reference-citations"><a href="#bib176" id="back-bib176" data-db-target-for="bib176-d218962e4299" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib176">Wang et al., 2018a</a><div data-db-target-of="bib176-d218962e4299" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang L. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Chen Y.-B. </li><li class="loa__item">Skala S.L. </li><li class="loa__item">Al-Ahmadie H.A. </li><li class="loa__item">Wang X. </li><li class="loa__item">Cao X. </li><li class="loa__item">Veeneman B.A. </li><li class="loa__item">Chen J. </li><li class="loa__item">Cieślik M. </li><li>et al.</li></ul><div class="reference-citations__detials">VSTM2A over-expression is a sensitive and specific biomarker for mucinous tubular and spindle cell carcinoma (MTSCC) of the kidney.<div class="popup-ref"><span class="ref__series"><i>Am. J. Surg. Pathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">1571-1584</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1097%2FPAS.0000000000001150&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30285995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85054473283&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2018&pages=1571-1584&journal=Am.+J.+Surg.+Pathol.&author=Wang+L.&author=Zhang+Y.&author=Chen+Y.-B.&author=Skala+S.L.&author=Al-Ahmadie+H.A.&author=Wang+X.&author=Cao+X.&author=Veeneman+B.A.&author=Chen+J.&author=Cie%C5%9Blik+M.&title=VSTM2A+over-expression+is+a+sensitive+and+specific+biomarker+for+mucinous+tubular+and+spindle+cell+carcinoma+%28MTSCC%29+of+the+kidney">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), along with gain of chr7, and <em>PIK3R1</em> mutation and were thus categorized as likely papillary RCCs. Sample C3L-00359 contained bi-allelic loss of <em>TSC1</em> along with histologic features of eosinophilic solid cystic RCC (ESC-RCC) (<div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d218962e4307" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Mehra et al., 2018</a><div data-db-target-of="bib102-d218962e4307" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mehra R. </li><li class="loa__item">Vats P. </li><li class="loa__item">Cao X. </li><li class="loa__item">Su F. </li><li class="loa__item">Lee N.D. </li><li class="loa__item">Lonigro R. </li><li class="loa__item">Premkumar K. </li><li class="loa__item">Trpkov K. </li><li class="loa__item">McKenney J.K. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Chinnaiyan A.M. </li></ul><div class="reference-citations__detials">Somatic Bi-allelic Loss of TSC Genes in Eosinophilic Solid and Cystic Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Eur. Urol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">483-486</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=abstract&amp;site=eururo-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=fulltext&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=pdf&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29941307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048882704&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=483-486&journal=Eur.+Urol.&author=Mehra+R.&author=Vats+P.&author=Cao+X.&author=Su+F.&author=Lee+N.D.&author=Lonigro+R.&author=Premkumar+K.&author=Trpkov+K.&author=McKenney+J.K.&author=Dhanasekaran+S.M.&author=Chinnaiyan+A.M.&title=Somatic+Bi-allelic+Loss+of+TSC+Genes+in+Eosinophilic+Solid+and+Cystic+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), while C3N-01180 had a <em>SFPQ-TFE3</em> gene fusion, a hallmark of translocation RCC. Finally, C3N-00435 contained 3p loss with a <em>PIK3CA</em> hotspot mutation. These seven samples were therefore annotated as non-ccRCC samples and excluded from the ccRCC cohort in all downstream analyses (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1" class="scroll-into-link">Table S1</a>).</div></section><section id="sec5.4.10.2" data-locator="sec5.4.10.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Detection of Chromosomal Arm-Level Aberration</span></h4><div class="section-paragraph">GISTIC2 (<div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d218962e4322" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Mermel et al., 2011</a><div data-db-target-of="bib103-d218962e4322" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (959)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to process the segment-level CNV log ratios and define chromosome arm-level gain/loss events for 3p, 5q, 7p, 9p, and 14q, using the default threshold setting (i.e., -ta and -td were both set to 0.1). From the GISTC2 arm-level outputs, we defined the positive and negative values as arm gain and loss events, respectively. Since GISTIC2 by design ignores arm-level CNVs, we next identified arm-level recurrence of gains/losses using an alternate approach. For each chromosome arm, we calculated the average (weighted) maximum-likelihood copy number (clonal or subclonal) relative to 2 (diploid). Similarly, for each chromosome arm, we calculated the proportion of the arm that shows LOH, including copy-neutral LOH. Following thresholding, (for CN gain (&gt; 0.5) or loss (&lt; −0.5)) these two proportions were used to estimate population-level recurrence of arm-level losses/gains and LOH, respectively. For chromosome 3p, only the loss (&lt; −0.5) was investigated for LOH (defined as &gt; 0.5). The detailed annotation of all chromosome arm events is provided (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2" class="scroll-into-link">Table S2</a>).</div></section><section id="sec5.4.10.3" data-locator="sec5.4.10.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Heatmap Web Server</span></h4><div class="section-paragraph">We developed an online application that allows researchers to query the dataset for genes of interest, rendering a downloadable table and corresponding heatmap visualization of the select data. The underlying data consist of quantitative information on copy number alteration, mutation, methylation, RNA-seq gene expression, protein expression, and phosphosite expression for 22,867 genes across 110 samples. Data tracks for each gene are labeled as: “Mut”–mutation data (“Yes” is any types of mutation, “No” is no mutation), “Methy”–beta value of CpG island in the promoter region of the gene (standardized), “CNV (lr)”–the log ratio of copy number variation, “CNV (baf)”–the b-allele frequency of the copy number variation (standardized), “mRNA”–gene expression levels (standardized), “Protein”–gene-level protein abundance (standardized), and “Phospho”–gene-level phosphoprotein abundance (standardized). Genomic and clinical annotation data are displayed on the top and bottom of the visualization. Tracks on the top include genomically confirmed ccRCC and non-ccRCC, 3p copy number variation, and an immune grouping for each sample. Bottom-placed tracks visualize CNV data for chromosomes 5q, 7p, 9p, and 14q, t(3;2) and t(3;5) chromosome translocations, CIMP status, and genomic instability status, as well as grade, stage, and gender information. The application can be accessed at <a class="inter-ref-link" href="http://ccrcc.cptac-data-view.org" target="_blank">http://ccrcc.cptac-data-view.org</a>. This is an entirely web-based application, and users do not need to download any software outside of a web browser to visualize and access the data. Users begin by entering official symbols for up to thirty genes into a text field. For convenience, the input gene list may be separated by tabs, commas, semicolons, single spaces, or line breaks. All gene symbols found in the underlying dataset will be used to generate an Excel file (.xls) and corresponding heatmap visualization. The raw data and graphic visualization (.png) can be downloaded to a local computer for further exploration or for use in publication. The application can also be used for interactive visual exploration of the data. Users may click any data point on the interactive heatmap to view the underlying values, including the sample identifier, data type, and value. By clicking a link next to the sample identifier, users can view direct links to the histological images corresponding to the sample, organized by tissue location. When available, the user can click gene symbols on the heatmap to view summary descriptions from the NCBI. Users can sort the entire heatmap by a single data track, in ascending or descending order. The sorted visualization can then be downloaded with a button. This functionality enables users to explore dynamically the relationships and patterns among different tracks.</div></section><section id="sec5.4.10.4" data-locator="sec5.4.10.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Downstream Analysis of Chromosome Arm 3p translocation</span></h4><div class="section-paragraph">This cohort featured frequent chromosome 3p arm translocation (e.g., to chromosome 5q and results in 3p loss and 5q gain) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). An association analysis of chromosome 3p arm translocation events was performed with each of the 9,190 protein abundances (with missing rate &lt; 50%) across 110 ccRCC tumor samples to identify the proteomic functional impacts of the translocation events. Specifically, three types of translocation categories, Chr3-Chr2, Chr3-Chr5, and Chr3-other, were simultaneously considered in the regression models, and an ANOVA test was applied to assess whether any translocation was associated with the protein abundances in the tumors. Age, gender, ischemic time, OCT status and tumor purity were adjusted as covariates. The most significant protein was SETD2 (p = 8E – 6; FDR &lt; 0.05). Post-ANOVA assessment of each event indicated that Chr3-Chr2 translocation was significantly associated with decreased abundance in SETD protein abundance (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>F), which is consistent with the high mutation rate observed in Chr3-Chr2 group (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>E).</div></section><section id="sec5.4.10.5" data-locator="sec5.4.10.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">iProFun-Based <em>Cis</em> Association Analysis</span></h4><div class="section-paragraph">The integrative analysis tool, iProFun (<div class="dropBlock reference-citations"><a href="#bib160" id="back-bib160" data-db-target-for="bib160-d218962e4357" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib160">Song et al., 2019</a><div data-db-target-of="bib160-d218962e4357" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+Impact+of+DNA+Copy+Number+Alteration+and+Methylation+on+the+Proteogenomic+Landscape+of+Human+Ovarian+Cancer+via+a+Multi-omics+Integrative+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was used to identify functional molecular quantitative traits perturbed by DNA-level variations (<a class="inter-ref-link" href="https://github.com/WangLab-MSSM/iProFun" target="_blank">https://github.com/WangLab-MSSM/iProFun</a>). Compared to analyzing each molecular trait separately, iProFun models multi-omic data jointly, thereby enhancing the power for detecting significant <em>cis</em>-associations shared across different omics data types and achieves better accuracy in inferring <em>cis</em>-associations unique to certain types of molecular traits. Specifically, we considered three functional molecular quantitative traits (mRNA expression levels, global protein abundances, and phosphopeptide abundances) for their associations with four DNA-level variations (copy number alterations measured by log ratios, copy number alterations measured by b-allele frequency, DNA methylations and somatic mutations).</div></section><section id="sec5.4.10.6" data-locator="sec5.4.10.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and preprocessing:</span></h4><div class="section-paragraph">We analyzed data from 110 tumors in the CPTAC cohort. mRNA expression levels measured with RNA-seq were available for 19,293 genes (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; RNA_rpkm_tumor_normal.tsv), while global protein abundance measurements were available for 11,355 genes (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Whole_abundance_gene_protNorm = 2_CB.tsv) and the phosphopeptide abundances were available for 42,893 peptides (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Phospho_abundance_phosphopeptide_protNorm%3D2_CB_1211.tsv) from 8,502 genes. The log ratios and b-allele frequencies of copy number alterations were obtained from WGS and WES data using a segmentation method for 19,285 and 19,293 genes, respectively. The DNA methylation levels (beta values) averaging the CpG islands located in the upstream and nearby transition starting site (TSS) regions, including 5UTR, first exon, and upstream TSS were available for 15,885 genes. Somatic mutations were called using WES (see SNV calling section above). All data types were preprocessed to eliminate potential issues such as batch effects, missing data, and major unmeasured confounding effects. All seven types of data were filtered with a missing rate &gt; 50%. mRNA expression levels and global protein and phosphoprotein abundances were also normalized to a standard normal distribution. Somatic mutations with &gt; 5% frequency were considered in iProFun for their functional consequences on molecular quantitative traits (QTs). To account for potential confounding factors, we considered age, gender and tumor purity. Tumor purity was determined from RNA-seq data using ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e4381" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e4381" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.4.10.7" data-locator="sec5.4.10.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">iProFun procedure:</span></h4><div class="section-paragraph">The iProFun procedure was applied to 4,009 genes measured across all six data types (mRNA, global protein, phosphoprotein, CNA – lr, CNA – baf, DNA methylation) for their <em>cis</em> regulatory patterns in tumors. Thirteen genes with frequent somatic mutations (mutation rate &gt; 5%) were also considered for their effects on <em>cis</em> molecular QTs. Specifically, for the remaining 3986 genes, we considered the following three regressions:<ul class="ce-list--remove-bullets" id="ulist0015"><li class="ce-list--remove-bullets__list-item" id="u0030"><div class="ce-list--remove-bullets__list-item__text">mRNA ∼CNV (lr) + CNV (baf) + methy + covariates,</div></li><li class="ce-list--remove-bullets__list-item" id="u0035"><div class="ce-list--remove-bullets__list-item__text">global ∼CNV (lr) + CNV (baf) + methy + covariates, and</div></li><li class="ce-list--remove-bullets__list-item" id="u0040"><div class="ce-list--remove-bullets__list-item__text">phosphor ∼CNV (lr) + CNV (baf) + methy + covariates.</div></li></ul></div><div class="section-paragraph">For the thirteen genes with frequent somatic mutations we considered the following three regressions:<ul class="ce-list--remove-bullets" id="ulist0020"><li class="ce-list--remove-bullets__list-item" id="u0045"><div class="ce-list--remove-bullets__list-item__text">mRNA ∼CNV (lr) + CNV (baf) +methy + mutation + covariates,</div></li><li class="ce-list--remove-bullets__list-item" id="u0050"><div class="ce-list--remove-bullets__list-item__text">global ∼CNV (lr) + CNV (baf) +methy + mutation + covariates, and</div></li><li class="ce-list--remove-bullets__list-item" id="u0055"><div class="ce-list--remove-bullets__list-item__text">phosphor ∼CNV (lr) + CNV (baf) +methy + mutation + covariates,</div></li></ul></div><div class="section-paragraph">The association summary statistics of CNV (lr), CNV (baf), and methy from two sets of regression frameworks were combined and applied to iProFun to call the posterior probability of belonging to each of the eight possible configurations (“None,” “mRNA only,” “global only,” “phosphor only,” “mRNA &amp; global,” “mRNA &amp; phosphor,” “global &amp; phosphor,” and “all three”) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>A), to estimate the variation of molecular QTs explained by each DNA variations (Rˆ2), and to determine significance associations.</div><div class="section-paragraph"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a> and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>B present the significant genes that pass three criteria: (1) satisfying the biologic filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR) &lt; 5%. Specifically, we posed the biological filtering criterion for CNV and DNA methylations. Only CNV (lr) that were positively associated with all the types of molecular QTs, DNA methylations that were negatively associated with all types of molecular QTs, and CNVs (baf) with associations of consistent direction (either positive or negative) with all types of molecular QTs were considered for significance calling. A significance was then called only if the posterior probability &gt; 75% of a predictor being associated with a molecular QT was greater than 75%, by summing over all configurations that are consistent with the association of interest. For example, the posterior probability of a DNA methylation being associated with mRNA expression levels was obtained by summing up the posterior probabilities in the following four association patterns – “mRNA only,” “mRNA &amp; global,” “mRNA &amp; phosphor,” and “all three,” all of which were consistent with the association of DNA methylation with mRNA expression. Lastly, we calculated empirical FDR (eFDR) via 100 permutations per molecular QTs by shuffling the label of the molecular QTs and requesting eFDR &lt; 5% by selecting a minimal cutoff value of alpha such that 75% &lt; alpha &lt; 100%. The eFDR is calculated by:<div class="labeled-equation clearfix"><div id="ufd1" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif"><mrow><mi mathvariant="normal">eFDR=</mi><mrow><mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Averaged</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">number</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">genes</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">with</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">posterior</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">probabilities&gt;alpha</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">in</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">permuted</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">data</mi></mrow><mo>)</mo></mrow></mrow><mo>/</mo><mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Averaged</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">number</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">genes</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">with</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">posterior</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">probabilities&gt;alpha</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">in</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">original</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">data</mi></mrow><mo>)</mo></mrow></mrow></mrow><mi mathvariant="normal">.</mi></mrow></math></div></div><br></br>Among all the genes whose phosphoproteins were significantly associated with tumor versus normal and with tumor grade, CNV, and methylation with cascade effects, i.e., demonstration of significant association with all of the three traits (mRNA levels, protein and phosphopeptide abundances) were plotted in <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a> catalogs Rˆ2 range (0, 1) by providing the percentage of adjusted variances explained by each type of DNA alterations. These Rˆ2 were obtained by contrasting multiple regression Rˆ2 values between the full model and models without the predictor of interest. For example, to calculate Rˆ2 of DNA methylation of a gene with a low mutation rate on its <em>cis</em> mRNA, the Rˆ2_full value is generated from model mRNA ∼CNV (lr) + CNV (baf) + methy + covariates, whereas the Rˆ2_reduced value is from model mRNA ∼CNV (lr) + CNV (baf) + covariates. The difference, Rˆ2_diff is calculated as Rˆ2_full – Rˆ2_reduced and represents Rˆ2 of mRNA explained by DNA methylation in this gene. For the additional thirteen somatic mutations, posterior probabilities could not be called robustly using iProFun, as alternative densities used in iProFun cannot be inferred accurately using only thirteen observations. We only required eFDR &lt; 5% and consistent direction in association with all types of molecular QTs to call significance. The Rˆ2 can be obtained in the same way as in other DNA variations. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a> presents the iProFun results based on measured mRNA and protein, without considering phosphoprotein. This exercise begins with a large number of genes that are measured across mRNA and protein.</div></section><section id="sec5.4.10.8" data-locator="sec5.4.10.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Trans Association Analysis</span></h4><div class="section-paragraph">We analyzed ten genomic features to understand their <em>cis</em> and <em>trans</em> associations with multi-omic molecular QTs in 110 cases in this cohort (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>D). Specifically, we considered multiple linear regression to evaluate the association between each pair of genomic feature and molecular trait adjusting for potential confounding factors. The mRNA expression and global protein abundances were considered at gene-level, and the phosphopeptide data were considered at the peptide level. In ccRCC, a total of 19,293 mRNA expression levels, 11,355 global protein abundances and 42,893 peptide-level phosphoprotein abundances were quantified, respectively. We required a missing rate &lt; 50% for consideration in association analyses, with 17,461 mRNA expression levels, 9,190 protein abundances and 21,245 phosphopeptide abundances being analyzed. All outcomes were normalized to match to a standard normal distribution before entering the regression models. We considered five chromosome arm-level genomic features (3p, 5q, 7p, 9p and 14q) that are most prevalent in the genome, one global methylation feature (CpG Island Methylator Phenotype (CIMP)), and five key mutations (<em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, <em>KDM5C</em>) for ccRCC (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3" class="scroll-into-link">Table S3</a>). For chromosome arms 3p and 14q, we compared copy loss with copy neutral. For chromosome arm 5q and 7p, we compared amplification with neutral. In 7p, we excluded four samples with loss from the analysis. CIMP was treated as a categorical variable; CIMP+ and CIMP- were compared in regression. For five mutations, we grouped all mutation types for power purposes and compared mutation versus no mutation in the analysis. We adjusted for potential confounding factors that could affect the association between <em>trans</em> genomic features and molecular traits in the analyses, including age, gender, OCT embedding, ischemic time, and tumor purity. Ischemic time was calculated as minutes from the initial clamping to collection and minutes from collection to freezing, if a clamp was used in the surgery. If no clamp was used in the surgery, the ischemic time was defined as minutes from collection to freezing. We considered eFDR to call for significance to account for complex unknown gene/gene correlations. Specifically, we first calculated t statistics for the association between a genomic feature and all genes/peptides of a given data type (e.g., mRNA) via multiple regression, thus adjusting for confounding factors. We then permuted our sample 100 times by shuffling the sample label of the outcomes to re-calculate the t statistics. For a pre-specified t statistics cut-off value, T, a gene was considered positive if it’s absolute t statistics were greater than T. Empirical FDR was calculated as noted previously for a pre-specified T value. The smallest T values that allow an averaged empirical FDR &lt; 10% were used as the final cutoffs, and all genomic feature and molecular trait pairs with absolute t statistics greater than the cutoff were considered significant associations. The significant <em>trans</em> associations of the selected key features (3p loss, 5q amplification, 7p amplification, 9p loss, 14q loss, CIMP, <em>VHL</em>, <em>PBRM1</em>, <em>BAP1</em>, <em>SETD2</em>, <em>KDM5C</em>) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>D) were binarized to +1 (positive) and −1 (negative) according to the calculated association values above and visualized in <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">S2</a>D using OmicsOne (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d218962e4582" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Hu et al., 2019</a><div data-db-target-of="bib67-d218962e4582" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hu Y. </li><li class="loa__item">Ao M. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">OmicsOne: Associate Omics Data with Phenotypes in One-Click.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/756544">https://doi.org/10.1101/756544</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F756544&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095531613&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Hu+Y.&author=Ao+M.&author=Zhang+H.&title=OmicsOne%3A+Associate+Omics+Data+with+Phenotypes+in+One-Click">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), a toolkit for data visualization and analysis of multi-omic data (<a class="inter-ref-link" href="https://github.com/HuiZhangLab-JHU/OmicsOne" target="_blank">https://github.com/HuiZhangLab-JHU/OmicsOne</a>). The cumulative density of the positive and negative associations on each individual chromosome was counted to illustrate the propagation heterogeneity of <em>trans</em> associations on different chromosomal locations. For each one of the eleven genomic events, pathway analyses were considered based on their association summary statistics with 9,190 genes that were observed in more than 50% of the 110 clinical samples. Specifically, a quantity T = - sign(beta) log(p value) was considered for the association between each pair of genomic event and protein abundance. The T value will be extremely high if the genomic event is significantly upregulated in the protein abundance, extreme negative values if the genomic event is significantly downregulated in the protein abundance, and values close to zero if the genomic event is not associated with the protein abundance. For each genomic event and each pathway under consideration, we tested if T_in = T_out using a two-sided Wilcoxon rank-score test, where T_in are the T values for all proteins inside of the pathway and T_out are the T values for all proteins outside of the pathway. Databases from Hallmark (MsigDB Collections), KEGG, and Reactome were combined for pathway analysis (<div class="dropBlock reference-citations"><a href="#" id="back-bib74" data-db-target-for="bib74-d218962e4591" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Joshi-Tope et al., 2005</a><div data-db-target-of="bib74-d218962e4591" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Joshi-Tope G. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Vastrik I. </li><li class="loa__item">D’Eustachio P. </li><li class="loa__item">Schmidt E. </li><li class="loa__item">de Bono B. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Gopinath G.R. </li><li class="loa__item">Wu G.R. </li><li class="loa__item">Matthews L. </li><li>et al.</li></ul><div class="reference-citations__detials">Reactome: a knowledgebase of biological pathways.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">D428-D432</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgki072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15608231&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13444259844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (846)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D428-D432&journal=Nucleic+Acids+Res.&author=Joshi-Tope+G.&author=Gillespie+M.&author=Vastrik+I.&author=D%E2%80%99Eustachio+P.&author=Schmidt+E.&author=de+Bono+B.&author=Jassal+B.&author=Gopinath+G.R.&author=Wu+G.R.&author=Matthews+L.&title=Reactome%3A+a+knowledgebase+of+biological+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib89" data-db-target-for="bib89-d218962e4591" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">Liberzon et al., 2011</a><div data-db-target-of="bib89-d218962e4591" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Pinchback R. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Molecular signatures database (MSigDB) 3.0.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1739-1740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtr260&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21546393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79958135926&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1170)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1739-1740&journal=Bioinformatics&author=Liberzon+A.&author=Subramanian+A.&author=Pinchback+R.&author=Thorvaldsd%C3%B3ttir+H.&author=Tamayo+P.&author=Mesirov+J.P.&title=Molecular+signatures+database+%28MSigDB%29+3.0">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib120" data-db-target-for="bib120-d218962e4591" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Ogata et al., 1999</a><div data-db-target-of="bib120-d218962e4591" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ogata H. </li><li class="loa__item">Goto S. </li><li class="loa__item">Sato K. </li><li class="loa__item">Fujibuchi W. </li><li class="loa__item">Bono H. </li><li class="loa__item">Kanehisa M. </li></ul><div class="reference-citations__detials">KEGG: Kyoto encyclopedia of genes and genomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">29-34</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2F27.1.29&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=9847135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0032919364&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2390)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=29-34&journal=Nucleic+Acids+Res.&author=Ogata+H.&author=Goto+S.&author=Sato+K.&author=Fujibuchi+W.&author=Bono+H.&author=Kanehisa+M.&title=KEGG%3A+Kyoto+encyclopedia+of+genes+and+genomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The significance threshold was set as FDR &lt; 0.05 to identify pathways that were most significantly associated with the genomic features. The direction of regulation (up versus down) was given by test statistics, with T_in &gt; T_out indicating upregulation and T_in &lt; T_out indicating downregulation.</div></section><section id="sec5.4.10.9" data-locator="sec5.4.10.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Correlation of mRNA and Protein Expression</span></h4><div class="section-paragraph">A total of 7,026 genes with complete mRNA (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; RNA_rpkm_tumor_normal.tsv) and protein (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Whole_abundance_gene_protNorm = 2_CB.tsv) data across all 103 ccRCCs and 72 NAT samples were used to measure gene-wise (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A) and sample-wise (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">S3</a>A) mRNA and protein correlations. Spearman correlation was calculated for each mRNA-protein pair across tumors and NATs separately and for each individual sample across 7,026 genes (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>). GSEA was used for gene-wise investigation on the correlation-ranked list of genes to determine if functional groups (pathways or complexes) were non-randomly distributed in terms of mRNA-protein correlation (<div class="dropBlock reference-citations"><a href="#bib164" id="back-bib164" data-db-target-for="bib164-d218962e4618" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib164">Subramanian et al., 2005</a><div data-db-target-of="bib164-d218962e4618" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mootha V.K. </li><li class="loa__item">Mukherjee S. </li><li class="loa__item">Ebert B.L. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Paulovich A. </li><li class="loa__item">Pomeroy S.L. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15633)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Functional classes were obtained from the MSIGDB (<a class="inter-ref-link" href="https://www.broadinstitute.org/gsea/msigdb/index.jsp" target="_blank">https://www.broadinstitute.org/gsea/msigdb/index.jsp</a>) and were chosen from the most significant non-redundant functions that were biologically informative. The individual proteins associated with pathways highlighted in the text as being significantly differentially present in more- or less-correlated protein-mRNA pairs are presented. To determine which pathway or functional activity may contribute to the sample-wise mRNA-protein correlation, we calculated the Spearman correlation between index and protein expressions across 103 tumor samples for 7,026 proteins using the correlation as index for each sample. Similarly, functional pathways were enriched and selected in those high correlations by GSEA enrichment analysis using the correlation-ranked list of proteins (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>B, 3C, <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">S3</a>B, and S3C). In addition, we assessed the association between DNA aberrations such as CNV data and mutations and clinical phenotypes such as tumor grade. A univariate linear model was utilized in which the pairwise correlation was modeled as a linear function of DNA aberrations and clinical phenotypes (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>). To assess whether the association between pairwise correlation and those covariates was induced by their shared dependence on tumor purity, tumor purity was included in the linear model as a covariate (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>). All figures were visualized using OmicsX (<div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d218962e4637" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Pan et al., 2019</a><div data-db-target-of="bib124-d218962e4637" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pan J. </li><li class="loa__item">Qian J. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">OmicsX: a web server for integrated OMICS analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/755918">https://doi.org/10.1101/755918</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F755918&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095490240&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Pan+J.&author=Qian+J.&author=Zhang+H.&title=OmicsX%3A+a+web+server+for+integrated+OMICS+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), a webserver for data analysis and visualization (<a class="inter-ref-link" href="http://bioinfo.wilmer.jhu.edu/OmicsX/" target="_blank">http://bioinfo.wilmer.jhu.edu/OmicsX/</a>).</div></section></section><section id="sec5.4.11" data-locator="sec5.4.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differential Abundance of mRNA, Protein, and Phosphoproteome Measurements</span></h4><section id="sec5.4.11.1" data-locator="sec5.4.11.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Principal Components Analysis</span></h4><div class="section-paragraph">We performed PCA on 103 tumor samples and 80 normal adjacent (NAT) samples to illustrate the global proteomic difference between tumor and NAT samples (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A). The PCA function under the scikit-learn R package (<div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d218962e4655" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Pedregosa et al., 2011</a><div data-db-target-of="bib129-d218962e4655" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pedregosa F. </li><li class="loa__item">Varoquaux G. </li><li class="loa__item">Gramfort A. </li><li class="loa__item">Michel V. </li><li class="loa__item">Thirion B. </li><li class="loa__item">Grisel O. </li><li class="loa__item">Blondel M. </li><li class="loa__item">Prettenhofer P. </li><li class="loa__item">Weiss R. </li><li class="loa__item">Dubourg V. </li><li>et al.</li></ul><div class="reference-citations__detials">Scikit-learn: Machine Learning in Python.<div class="popup-ref"><span class="ref__series"><i>J. Mach. Learn. Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">2825-2830</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&journal=J.%C2%A0Mach.+Learn.+Res.&author=Pedregosa+F.&author=Varoquaux+G.&author=Gramfort+A.&author=Michel+V.&author=Thirion+B.&author=Grisel+O.&author=Blondel+M.&author=Prettenhofer+P.&author=Weiss+R.&author=Dubourg+V.&title=Scikit-learn%3A+Machine+Learning+in+Python">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was implemented for unsupervised clustering analysis with the parameter ‘n_components = 2′ on the expression matrix of global proteomic data containing 7000+ proteins (features). The 95% confidence coverage was represented by a colored ellipse for each group, which was calculated based on the mean and covariance of points in each specific group (tumor and NAT).</div></section><section id="sec5.4.11.2" data-locator="sec5.4.11.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Global Heatmap</span></h4><div class="section-paragraph">Two-way hierarchical clustering was applied to the global proteomic data on samples and proteins to identify the global differential protein expression and protein co-expression patterns (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>A). Each gene expression value in the global proteomic expression matrix was transformed to a z-score across all samples. For the sample-wise and protein-wise clustering, distance was set as “Euclidean” distance, and the weight method was “complete.” The z-score-transformed matrix was clustered using R package: pheatmap (version 1.0.10).</div></section><section id="sec5.4.11.3" data-locator="sec5.4.11.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor versus Normal Differential Proteomic Analysis</span></h4><div class="section-paragraph">TMT-based global proteomic data were used to perform differential proteome analysis between tumor and normal samples (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>B, <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>B, and S4C; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). A Wilcoxon rank sum test was performed to determine differential abundance of proteins between tumor and normal samples. The significantly differentially expressed gene lists (fold change &gt; = 2 and FDR &lt; 0.05) were used to perform overrepresentation enrichment analysis (ORA) implemented in WebGestaltR (<div class="dropBlock reference-citations"><a href="#bib175" id="back-bib175" data-db-target-for="bib175-d218962e4680" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib175">Wang et al., 2017</a><div data-db-target-of="bib175-d218962e4680" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Greer M. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (345)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt+2017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), in which the parameters were set to use 9190 background genes and the combined KEGG/HALLMARK/Reactome database as described above.</div></section><section id="sec5.4.11.4" data-locator="sec5.4.11.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Accounting for Tissue Purity in Differential Analysis based on Proteomic Data</span></h4><div class="section-paragraph">NAT and tumor tissues represent mixtures of epithelial, stromal and immune cells. TSNet (<div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d218962e4688" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Petralia et al., 2018</a><div data-db-target-of="bib134-d218962e4688" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng J. </li><li class="loa__item">Yan A. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to account for this tumor heterogeneity and identify proteins that are differentially expressed between pure-tumor and pure-NAT cells. TSNet models the global abundance of each protein as a mixture of pure component and a component that captures the immune and stromal infiltration in a particular tissue. This algorithm estimated a mean parameter for pure component and immune/stromal infiltrated component for each protein. TSNet was applied to tumor (T) and NAT (N) samples separately by estimating the following two models:<div class="labeled-equation clearfix"><div id="ufd2" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif"><mrow><msub><mi>x</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><msub><mi>π</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi></mrow></msub><msub><mi>y</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo linebreak="badbreak">−</mo><msub><mi>π</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><msub><mi>z</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></div></div><br></br><div class="labeled-equation clearfix"><div id="ufd3" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif"><mrow><msub><mi>x</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><msub><mi>π</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi></mrow></msub><msub><mi>y</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo linebreak="badbreak">−</mo><msub><mi>π</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><msub><mi>z</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></div></div><br></br>with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si4.gif"><mrow><msub><mi>y</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">∼</mo><mi>N</mi><mi>o</mi><mi>r</mi><mi>m</mi><mi>a</mi><mi>l</mi><mrow><mo>(</mo><mrow><msubsup><mi>μ</mi><mrow><mi>T</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup><mo>,</mo><msubsup><mi>σ</mi><mrow><mi>T</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup></mrow><mo>)</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si5.gif"><mrow><msub><mi>z</mi><mrow><mi>T</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">∼</mo><mi>N</mi><mi>o</mi><mi>r</mi><mi>m</mi><mi>a</mi><mi>l</mi><mrow><mo>(</mo><mrow><msubsup><mi>μ</mi><mrow><mi>T</mi><mo>,</mo><mi>j</mi></mrow><mi>Z</mi></msubsup><mo>,</mo><msubsup><mi>σ</mi><mrow><mi>T</mi><mo>,</mo><mi>j</mi></mrow><mi>Z</mi></msubsup></mrow><mo>)</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si6.gif"><mrow><msub><mi>y</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">∼</mo><mi>N</mi><mi>o</mi><mi>r</mi><mi>m</mi><mi>a</mi><mi>l</mi><mrow><mo>(</mo><mrow><msubsup><mi>μ</mi><mrow><mi>N</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup><mo>,</mo><msubsup><mi>σ</mi><mrow><mi>N</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup></mrow><mo>)</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si7.gif"><mrow><msub><mi>z</mi><mrow><mi>N</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">∼</mo><mi>N</mi><mi>o</mi><mi>r</mi><mi>m</mi><mi>a</mi><mi>l</mi><mrow><mo>(</mo><mrow><msubsup><mi>μ</mi><mrow><mi>N</mi><mo>,</mo><mi>j</mi></mrow><mi>Z</mi></msubsup><mo>,</mo><msubsup><mi>σ</mi><mrow><mi>N</mi><mo>,</mo><mi>j</mi></mrow><mi>Z</mi></msubsup></mrow><mo>)</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si8.gif"><mrow><msub><mi>x</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math> being the observed global abundance of sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math> and gene <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math> for tissue <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si11.gif"><mrow><mi>t</mi><mo linebreak="goodbreak" linebreakstyle="after">∈</mo><mrow><mo>{</mo><mrow><mi>T</mi><mo>,</mo><mi>N</mi></mrow><mo>}</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si12.gif"><mrow><msub><mi>π</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi></mrow></msub></mrow></math> being the tissue purity for sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math> and tissue <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si13.gif"><mi>t</mi></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si14.gif"><mrow><mrow><mo>(</mo><mrow><msub><mi>y</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msub><mi>z</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></math> being latent variables corresponding to the protein abundance that would be observed in pure-tissue (i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si15.gif"><mrow><msub><mi>y</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math>) and immune/stromal cells (i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si16.gif"><mrow><msub><mi>z</mi><mrow><mi>t</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math>). Given the consistency of purity values estimated by TSNet and ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e5244" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e5244" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for this analysis, purity was inferred via ESTIMATE and considered as fixed (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Before implementing TSNet, each protein was z-score normalized across NAT and tumor samples. Once that the signal was deconvolved into pure-tissue and immune/stromal components, we identified proteins that were differentially expressed in pure tumor component compared to pure NAT component. Mathematically, this was achieved by assessing the significance of the difference between the means <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si17.gif"><mrow><msubsup><mi>μ</mi><mrow><mi>N</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup></mrow></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si50.gif"><mrow><msubsup><mi>μ</mi><mrow><mi>T</mi><mo>,</mo><mi>j</mi></mrow><mi>Y</mi></msubsup></mrow></math> for each protein <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math>. Significance was assessed via permutation. Specifically, TSNet was implemented under permuted data, where the labels of NAT and tumor samples were randomly shuffled. For this analysis, 200 permutations were considered. Using the null density of mean difference derived based on permutated data, we assessed the significance of the up/downregulation in tumor compared to NAT. In particular, at a specific FDR cut-off of 10%, true associations were computed using the strategy illustrated by <div class="dropBlock reference-citations"><a href="#bib171" id="back-bib171" data-db-target-for="bib171-d218962e5285" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib171">Tusher et al., 2001</a><div data-db-target-of="bib171-d218962e5285" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tusher V.G. </li><li class="loa__item">Tibshirani R. </li><li class="loa__item">Chu G. </li></ul><div class="reference-citations__detials">Significance analysis of microarrays applied to the ionizing radiation response.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">5116-5121</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1073%2Fpnas.091062498&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=11309499&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0035942271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9173)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=5116-5121&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Tusher+V.G.&author=Tibshirani+R.&author=Chu+G.&title=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. To be consistent with the differential analysis based on un-deconvolved data, among the selected proteins at FDR 10%, only mean differences with a fold change greater than two were considered significant. Reported pathways are listed in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>.</div></section><section id="sec5.4.11.5" data-locator="sec5.4.11.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Accounting for Anatomic Region of NAT</span></h4><div class="section-paragraph">Using published gene signatures (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d218962e5296" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Lindgren et al., 2017</a><div data-db-target-of="bib93-d218962e5296" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lindgren D. </li><li class="loa__item">Eriksson P. </li><li class="loa__item">Krawczyk K. </li><li class="loa__item">Nilsson H. </li><li class="loa__item">Hansson J. </li><li class="loa__item">Veerla S. </li><li class="loa__item">Sjölund J. </li><li class="loa__item">Höglund M. </li><li class="loa__item">Johansson M.E. </li><li class="loa__item">Axelson H. </li></ul><div class="reference-citations__detials">Cell-Type-Specific Gene Programs of the Normal Human Nephron Define Kidney Cancer Subtypes.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1476-1489</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28793269&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85026887625&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=1476-1489&journal=Cell+Rep.&author=Lindgren+D.&author=Eriksson+P.&author=Krawczyk+K.&author=Nilsson+H.&author=Hansson+J.&author=Veerla+S.&author=Sj%C3%B6lund+J.&author=H%C3%B6glund+M.&author=Johansson+M.E.&author=Axelson+H.&title=Cell-Type-Specific+Gene+Programs+of+the+Normal+Human+Nephron+Define+Kidney+Cancer+Subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), NAT samples were allocated to different anatomic regions, including cortex, medulla, corticomedulla (Tal), inflammatory (Infla), and endothelial and/or smooth muscle cells or fibroblasts (SMC). Gene expression of all the genes (TPM) was z-scored normalized across samples and then averaged across genes mapping to each anatomic region subtype. Each sample was then allocated to the anatomic group with the highest score. A one-sided Wilcoxon test was performed to compare ccRCC versus NAT samples allocated to the cortex anatomic region. P values were adjusted for multiple comparisons using a Benjamini Hochberg adjustment. Only proteins with an adjusted p value less than 5% and fold change greater than two were considered significant. Reported pathways are listed in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>.</div></section><section id="sec5.4.11.6" data-locator="sec5.4.11.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Metabolic Reprogramming in ccRCC</span></h4><div class="section-paragraph">A Wilcoxon rank sum test was performed to explore tumor-normal differential analysis for RNA and protein at the gene level (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). Genes associated with glycolysis, the TCA cycle (Krebs Cycle), and oxidative phosphorylation (electron transport chain) were focused for metabolic reprogramming. In tumor samples, metabolic reprogramming-associated genes were selected, and z-score transformation was performed. For genes detected at both the mRNA and protein levels, t tests were performed to compare the gene/protein expression between tumor and NAT, separately. The log2 fold changes were used to measure the expression difference and significance and the concordance between mRNA level and protein measurements. Finally, the difference of log2 fold change between mRNA and protein were input into GSEA (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4" class="scroll-into-link">Table S4</a>) to investigate enriched pathways. The enriched concepts indicates the discordance between mRNA and protein for tumor/normal difference (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>D and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>C). <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>D and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>C were visualized using OmicsX (<a class="inter-ref-link" href="http://bioinfo.wilmer.jhu.edu/OmicsX/" target="_blank">http://bioinfo.wilmer.jhu.edu/OmicsX/</a>) (<div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d218962e5329" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Pan et al., 2019</a><div data-db-target-of="bib124-d218962e5329" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pan J. </li><li class="loa__item">Qian J. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">OmicsX: a web server for integrated OMICS analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/755918">https://doi.org/10.1101/755918</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F755918&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095490240&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Pan+J.&author=Qian+J.&author=Zhang+H.&title=OmicsX%3A+a+web+server+for+integrated+OMICS+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section></section><section id="sec5.4.12" data-locator="sec5.4.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomic Analysis</span></h4><section id="sec5.4.12.1" data-locator="sec5.4.12.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide Analysis – Kinase and Substrate Regulation</span></h4><div class="section-paragraph">To discover the phosphorylation events that were relevant to ccRCC, we utilized phosphopeptide-level data to examine the overall relationship between phospho-substrates and their associated kinases (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>A). The kinase-substrate association was first extracted from PhosphoSitePlus (<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d218962e5346" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Hornbeck et al., 2015</a><div data-db-target-of="bib63-d218962e5346" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (898)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to eliminate phosphopeptides (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Phospho_abundance_phosphopeptide_protNorm = 2_CB_imputed_1211.tsv) containing phosphosites (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Phospho_abundance_phosphosite_protNorm = 2_CB_imputed.tsv) that were not reported as well as those without associated kinases identified in our global dataset (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>; CPTAC3_CCRCC_Whole_abundance_gene_protNorm = 2_CB.tsv). Next, we inspected any substantial differences among 80 tumor/NAT pairs, especially those that showed higher changes in tumors, by calculating the fold change (log2 scale). We then ranked each tumor (&gt; 1.5 fold increase) among different kinase substrates to obtain the highest ranked phospho-substrate events in the majority of tumors (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a>). Finally, we identified nine phospho-substrate events of eight kinases with inhibitors that are either FDA-approved or in clinical trials (<div class="dropBlock reference-citations"><a href="#" id="back-bib21" data-db-target-for="bib21-d218962e5362" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Carles et al., 2018</a><div data-db-target-of="bib21-d218962e5362" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Carles F. </li><li class="loa__item">Bourg S. </li><li class="loa__item">Meyer C. </li><li class="loa__item">Bonnet P. </li></ul><div class="reference-citations__detials">PKIDB: A curated, annotated and updated database of protein kinase inhibitors in clinical trials.<div class="popup-ref"><span class="ref__series"><i>Molecules.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1-18</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fmolecules23040908&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045639618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1-18&journal=Molecules&author=Carles+F.&author=Bourg+S.&author=Meyer+C.&author=Bonnet+P.&title=PKIDB%3A+A+curated%2C+annotated+and+updated+database+of+protein+kinase+inhibitors+in+clinical+trials">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib43" data-db-target-for="bib43-d218962e5362" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Ferguson and Gray, 2018</a><div data-db-target-of="bib43-d218962e5362" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ferguson F.M. </li><li class="loa__item">Gray N.S. </li></ul><div class="reference-citations__detials">Kinase inhibitors: the road ahead.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Drug Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">353-377</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrd.2018.21&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29545548&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045991926&amp;cf=" onclick="newWindow(this.href);return false">Scopus (188)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&journal=Nat.+Rev.+Drug+Discov.&author=Ferguson+F.M.&author=Gray+N.S.&title=Kinase+inhibitors%3A+the+road+ahead">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We also calculated the fold change of the selected phospho-substrates and kinases in other omics data (e.g., mRNA) to examine any difference in expression level among multiple omics data types (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>A–S5C). In addition, we compared the phosphopeptide expression between low-grade tumors (Grades 1 and 2) and high-grade tumors (Grades 3 and 4) as well as between low-stage tumors (Stages 1 and 2) and high-stage tumors (Stages 3 and 4). A p value &lt; 0.05 (Mann–Whitney U test) was considered as significant (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>B). Data were visualized using Omic-Sig (<a class="inter-ref-link" href="https://github.com/hzhangjhu/Omic-Sig" target="_blank">https://github.com/hzhangjhu/Omic-Sig</a>) (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d218962e5375" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Lih et al., 2019</a><div data-db-target-of="bib90-d218962e5375" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lih T.M. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Omic-Sig: Utilizing Omics Data to Explore and Visualize Kinase-Substrate Interactions.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/746123">https://doi.org/10.1101/746123</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F746123&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095527091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Lih+T.M.&author=Clark+D.J.&author=Zhang+H.&title=Omic-Sig%3A+Utilizing+Omics+Data+to+Explore+and+Visualize+Kinase-Substrate+Interactions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.4.12.2" data-locator="sec5.4.12.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomic Co-expression Network Inference</span></h4><div class="section-paragraph">Network inference was utilized to characterize co-expression patterns among phosphopeptides in ccRCC. Due to the high dimensionality of phosphorylation data, which contained approximately 20,000 different peptides, phosphopeptides were first clustered into three groups, and then co-expression networks were estimated for each group, separately (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>C, 5D, and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">S5</a>D–S5G). Specifically, we first summarized multiple phosphopeptides mapping to the same protein using their leading principal component, which was derived based on 103 ccRCC tumor samples and 80 NAT samples. k-means clustering was then implemented on the gene-level matrix to cluster proteins into three groups. This procedure resulted in one group containing 1,842 genes mapping to 6,182 phosphopeptides, a second group containing 1,963 genes mapping to 6,976 phosphopeptides, and a third group containing 2,047 genes mapping to 7,818 phosphopeptides. For each group of genes, a co-expression network was estimated based on phospho-peptide level data through a random-forest-based algorithm (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d218962e5389" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Petralia et al., 2015</a><div data-db-target-of="bib132-d218962e5389" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang P. </li><li class="loa__item">Yang J. </li><li class="loa__item">Tu Z. </li></ul><div class="reference-citations__detials">Integrative random forest for gene regulatory network inference.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">i197-i205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtv268&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26072483&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84931051773&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=i197-i205&journal=Bioinformatics&author=Petralia+F.&author=Wang+P.&author=Yang+J.&author=Tu+Z.&title=Integrative+random+forest+for+gene+regulatory+network+inference">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In particular, co-expression networks were estimated using missing data-imputed peptide-level phosphorylation data (https://cptac-data-portal.georgetown.edu/cptac/s/S050; CPTAC3_CCRCC_Phospho_abundance_phosphopeptide_protNorm = 2_CB_imputed_1211.tsv). Let <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si18.gif"><mi>p</mi></math> represent the total number of phosphopeptides measured for <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si19.gif"><mi>n</mi></math> samples. <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si20.gif"><mrow><msubsup><mi>x</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mi>s</mi></msubsup></mrow></math> represents the abundance of the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math>-th peptide mapping to the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si21.gif"><mi>s</mi></math>-th protein for the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>-th sample. <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si20.gif"><mrow><msubsup><mi>x</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mi>s</mi></msubsup></mrow></math> was modeled as a function of other protein phosphopeptides, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si22.gif"><mrow><msub><mrow><mrow><mo>{</mo><mrow><msubsup><mi>x</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mi>k</mi></msubsup></mrow><mo>}</mo></mrow></mrow><mrow><mi>k</mi><mo>≠</mo><mi>s</mi></mrow></msub></mrow></math>, via random forest. To facilitate the comparison with networks inferred based on RNA-seq and global proteomic data, the network was obtained at the gene-level with nodes corresponding to genes. Basically, an edge between two genes was drawn if at least some peptides mapping to the two genes were found to be associated. This was achieved by using an extension of the random forest algorithm (<a class="inter-ref-link" href="https://github.com/WangLab-MSSM/ptmJRF" target="_blank">https://github.com/WangLab-MSSM/ptmJRF</a>) (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d218962e5477" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Petralia et al., 2016</a><div data-db-target-of="bib133-d218962e5477" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Song W.M. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">743-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.5b00925&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26733076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960353766&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=743-754&journal=J.%C2%A0Proteome+Res.&author=Petralia+F.&author=Song+W.M.&author=Tu+Z.&author=Wang+P.&title=New+Method+for+Joint+Network+Analysis+Reveals+Common+and+Different+Coexpression+Patterns+among+Genes+and+Proteins+in+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Basically, for each protein <em>j,</em> the abundance of each phosphopeptide mapping to that protein was modeled as a function of other proteins’ phosphopeptides via random forest. At each node in the random forest tree, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si23.gif"><mi>M</mi></math> proteins were randomly sampled and proposed as candidates for the splitting rule. Then, across all phosphopeptides mapping to the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si23.gif"><mi>M</mi></math> proteins, the phosphopeptide resulting in the lowest node impurity was utilized for the splitting rule. A separate collection of T trees was estimated for each phosphopeptide mapping to protein <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math>. Based on each ensemble tree, an importance score capturing the association between protein <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math> was derived (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d218962e5503" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Petralia et al., 2015</a><div data-db-target-of="bib132-d218962e5503" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang P. </li><li class="loa__item">Yang J. </li><li class="loa__item">Tu Z. </li></ul><div class="reference-citations__detials">Integrative random forest for gene regulatory network inference.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">i197-i205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtv268&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26072483&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84931051773&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=i197-i205&journal=Bioinformatics&author=Petralia+F.&author=Wang+P.&author=Yang+J.&author=Tu+Z.&title=Integrative+random+forest+for+gene+regulatory+network+inference">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The final weight assigned to the relationship <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si25.gif"><mrow><mrow><mo>(</mo><mrow><mi>k</mi><mo linebreak="badbreak">→</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></math> was derived by taking the maximum of the importance scores across different tree ensembles. This procedure was repeated for each protein <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math>. The final importance score assigned to the edge <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si26.gif"><mrow><mrow><mo>(</mo><mrow><mi>k</mi><mo linebreak="badbreak">−</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></math> was derived as the average between the importance score corresponding to <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si25.gif"><mrow><mrow><mo>(</mo><mrow><mi>k</mi><mo linebreak="badbreak">→</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></math> and the importance score corresponding to <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si27.gif"><mrow><mrow><mo>(</mo><mrow><mi>j</mi><mo linebreak="badbreak">→</mo><mi>k</mi></mrow><mo>)</mo></mrow></mrow></math> (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d218962e5571" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Petralia et al., 2016</a><div data-db-target-of="bib133-d218962e5571" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Song W.M. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">743-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.5b00925&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26733076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960353766&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=743-754&journal=J.%C2%A0Proteome+Res.&author=Petralia+F.&author=Song+W.M.&author=Tu+Z.&author=Wang+P.&title=New+Method+for+Joint+Network+Analysis+Reveals+Common+and+Different+Coexpression+Patterns+among+Genes+and+Proteins+in+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To derive the final unweighted networks, a proper cut-off value was chosen via permutation techniques (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d218962e5574" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Petralia et al., 2016</a><div data-db-target-of="bib133-d218962e5574" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Song W.M. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">743-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.5b00925&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26733076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960353766&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=743-754&journal=J.%C2%A0Proteome+Res.&author=Petralia+F.&author=Song+W.M.&author=Tu+Z.&author=Wang+P.&title=New+Method+for+Joint+Network+Analysis+Reveals+Common+and+Different+Coexpression+Patterns+among+Genes+and+Proteins+in+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, 40 permutations and a FDR cut-off of 5% were considered to derive the final network. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a> contains the list of network edges of genes obtained at 5% FDR cut-off.</div></section><section id="sec5.4.12.3" data-locator="sec5.4.12.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomics Co-expression Network Modules</span></h4><div class="section-paragraph">Network-modules were derived using Glay (<div class="dropBlock reference-citations"><a href="#bib163" id="back-bib163" data-db-target-for="bib163-d218962e5585" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib163">Su et al., 2010</a><div data-db-target-of="bib163-d218962e5585" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Su G. </li><li class="loa__item">Kuchinsky A. </li><li class="loa__item">Morris J.H. </li><li class="loa__item">States D.J. </li><li class="loa__item">Meng F. </li></ul><div class="reference-citations__detials">GLay: community structure analysis of biological networks.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">3135-3137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq596&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21123224&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79951764932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=3135-3137&journal=Bioinformatics&author=Su+G.&author=Kuchinsky+A.&author=Morris+J.H.&author=States+D.J.&author=Meng+F.&title=GLay%3A+community+structure+analysis+of+biological+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), a community clustering algorithm available through Cytoscape (<div class="dropBlock reference-citations"><a href="#" id="back-bib113" data-db-target-for="bib113-d218962e5588" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Morris et al., 2011</a><div data-db-target-of="bib113-d218962e5588" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Morris J.H. </li><li class="loa__item">Apeltsin L. </li><li class="loa__item">Newman A.M. </li><li class="loa__item">Baumbach J. </li><li class="loa__item">Wittkop T. </li><li class="loa__item">Su G. </li><li class="loa__item">Bader G.D. </li><li class="loa__item">Ferrin T.E. </li></ul><div class="reference-citations__detials">clusterMaker: a multi-algorithm clustering plugin for Cytoscape.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">436</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-12-436&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22070249&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80655132169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=436&journal=BMC+Bioinformatics&author=Morris+J.H.&author=Apeltsin+L.&author=Newman+A.M.&author=Baumbach+J.&author=Wittkop+T.&author=Su+G.&author=Bader+G.D.&author=Ferrin+T.E.&title=clusterMaker%3A+a+multi-algorithm+clustering+plugin+for+Cytoscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib150" data-db-target-for="bib150-d218962e5588" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib150">Shannon et al., 2003</a><div data-db-target-of="bib150-d218962e5588" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shannon P. </li><li class="loa__item">Markiel A. </li><li class="loa__item">Ozier O. </li><li class="loa__item">Baliga N.S. </li><li class="loa__item">Wang J.T. </li><li class="loa__item">Ramage D. </li><li class="loa__item">Amin N. </li><li class="loa__item">Schwikowski B. </li><li class="loa__item">Ideker T. </li></ul><div class="reference-citations__detials">Cytoscape: a software environment for integrated models of biomolecular interaction networks.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13696)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thirty network-modules containing at least twenty genes were identified (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a>). Considering the list of genes mapping to each network module, pathway enrichment analysis was performed to identify biological pathways over-represented in each network module via Fisher's exact test. For this analysis, pathways from the KEGG (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d218962e5594" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Kanehisa and Goto, 2000</a><div data-db-target-of="bib77-d218962e5594" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kanehisa M. </li><li class="loa__item">Goto S. </li></ul><div class="reference-citations__detials">KEGG: Kyoto Encyclopedia of Genes and Genomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">27-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2F28.1.27&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10592173&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=27-30&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Goto+S.&title=KEGG%3A+Kyoto+Encyclopedia+of+Genes+and+Genomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and Reactome (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d218962e5597" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Joshi-Tope et al., 2005</a><div data-db-target-of="bib74-d218962e5597" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Joshi-Tope G. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Vastrik I. </li><li class="loa__item">D’Eustachio P. </li><li class="loa__item">Schmidt E. </li><li class="loa__item">de Bono B. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Gopinath G.R. </li><li class="loa__item">Wu G.R. </li><li class="loa__item">Matthews L. </li><li>et al.</li></ul><div class="reference-citations__detials">Reactome: a knowledgebase of biological pathways.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">D428-D432</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgki072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15608231&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13444259844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (846)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D428-D432&journal=Nucleic+Acids+Res.&author=Joshi-Tope+G.&author=Gillespie+M.&author=Vastrik+I.&author=D%E2%80%99Eustachio+P.&author=Schmidt+E.&author=de+Bono+B.&author=Jassal+B.&author=Gopinath+G.R.&author=Wu+G.R.&author=Matthews+L.&title=Reactome%3A+a+knowledgebase+of+biological+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) database were considered. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a> shows the list of enriched pathways for each network-module. A one-sided t test was used to identify 18 network modules whose nodes were more correlated under the Phospho-Tumor network than under other data types (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6" class="scroll-into-link">Table S6</a>). To visualize the network modules (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>D and S5E) the software iCAVE (<div class="dropBlock reference-citations"><a href="#" id="back-bib76" data-db-target-for="bib76-d218962e5610" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Kalayci and Gümüş, 2018</a><div data-db-target-of="bib76-d218962e5610" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kalayci S. </li><li class="loa__item">Gümüş Z.H. </li></ul><div class="reference-citations__detials">Exploring Biological Networks in 3D, Stereoscopic 3D, and Immersive 3D with iCAVE.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">61</span></span>: <span class="ref__seriesPages">8.27.1-8.27.26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fcpbi.47&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050801906&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8.27.1-8.27.26&journal=Curr.+Protoc.+Bioinformatics&author=Kalayci+S.&author=G%C3%BCm%C3%BC%C5%9F+Z.H.&title=Exploring+Biological+Networks+in+3D%2C+Stereoscopic+3D%2C+and+Immersive+3D+with+iCAVE">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib91" data-db-target-for="bib91-d218962e5610" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Liluashvili et al., 2017</a><div data-db-target-of="bib91-d218962e5610" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liluashvili V. </li><li class="loa__item">Kalayci S. </li><li class="loa__item">Fluder E. </li><li class="loa__item">Wilson M. </li><li class="loa__item">Gabow A. </li><li class="loa__item">Gümüs Z.H. </li></ul><div class="reference-citations__detials">iCAVE: an open source tool for visualizing biomolecular networks in 3D, stereoscopic 3D and immersive 3D.<div class="popup-ref"><span class="ref__series"><i>Gigascience.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1-13</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fgigascience%2Fgix054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28814063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042149548&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1-13&journal=Gigascience&author=Liluashvili+V.&author=Kalayci+S.&author=Fluder+E.&author=Wilson+M.&author=Gabow+A.&author=G%C3%BCm%C3%BCs+Z.H.&title=iCAVE%3A+an+open+source+tool+for+visualizing+biomolecular+networks+in+3D%2C+stereoscopic+3D+and+immersive+3D">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and Cytoscape were utilized.</div></section><section id="sec5.4.12.4" data-locator="sec5.4.12.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Interactive Network Exploration Portal</span></h4><div class="section-paragraph">We developed a web portal that allows researchers to interactively explore tumor phosphoproteomics co-expression network and its modules (<a class="inter-ref-link" href="http://ccrcc.cptac-network-view.org" target="_blank">http://ccrcc.cptac-network-view.org</a>). This web-based application does not require users to download any software outside of a web browser to access and explore the data. The main page features a panel on the left that enables 3D viewing and exploration of the tumor phosphoproteomics co-expression network. Nodes are colored accordingly with associated modules that are listed to the right of the viewing panel. Users can search for a certain gene within the network by entering the HUGO symbol in the search box provided and clicking the Search button. If found, the gene is highlighted in the 3D panel in red, and the associated information about the gene is provided under the search box. Gene information includes the list of genes that are directly connected to the queried gene and also association of the gene with clinical variables (FDR value and p value for grade, gender, age, and stage). Users can click on the name of a module to view it in detail. A separate page opens and displays module-specific network and associated details. Module-specific pages provide the 3D network view and exploration panel at the center of the page. Users can interactively explore the network in 3D using this panel. Node sizes are proportional to the number of connections. Hovering over a node displays the gene name and highlights it in red. Clicking on a node will highlight the edges connected to the node in red and also display associated information on the right side. Gene-associated information includes the list of genes that are directly connected to the gene, peptides associated with the gene, and the association of the gene with clinical variables (FDR value and p value for grade, gender, age, and stage). Above the network-view panel, another panel can perform phenotype-related search operations. Users can select the phenotype of interest (grade, gender, age, or stage), enter an FDR cutoff value, and click the Submit button. Genes that satisfy the search metrics are listed in the text box and are also highlighted in the network panel in red. Users can click the Reset button to return to the original network. On the left side of the network view panel, an interactive table showing the list of enriched pathways (if any) is provided. Users can click on a pathway name, and the genes within the pathway will be listed in the text box as well as highlighted in the network panel in red. Users can click the Reset button to return to the original network. The interactive network exploration portal utilizes multiple client-side Javascript libraries (e.g., three.js, D3.js, JQuery) to facilitate visualization and user interaction with large volumes of data in real time. For 3D visualizations that are displayed within the interface, we incorporated 3D layouts from iCAVE (<div class="dropBlock reference-citations"><a href="#" id="back-bib76" data-db-target-for="bib76-d218962e5621" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Kalayci and Gümüş, 2018</a><div data-db-target-of="bib76-d218962e5621" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kalayci S. </li><li class="loa__item">Gümüş Z.H. </li></ul><div class="reference-citations__detials">Exploring Biological Networks in 3D, Stereoscopic 3D, and Immersive 3D with iCAVE.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">61</span></span>: <span class="ref__seriesPages">8.27.1-8.27.26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fcpbi.47&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050801906&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8.27.1-8.27.26&journal=Curr.+Protoc.+Bioinformatics&author=Kalayci+S.&author=G%C3%BCm%C3%BC%C5%9F+Z.H.&title=Exploring+Biological+Networks+in+3D%2C+Stereoscopic+3D%2C+and+Immersive+3D+with+iCAVE">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib91" data-db-target-for="bib91-d218962e5621" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Liluashvili et al., 2017</a><div data-db-target-of="bib91-d218962e5621" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liluashvili V. </li><li class="loa__item">Kalayci S. </li><li class="loa__item">Fluder E. </li><li class="loa__item">Wilson M. </li><li class="loa__item">Gabow A. </li><li class="loa__item">Gümüs Z.H. </li></ul><div class="reference-citations__detials">iCAVE: an open source tool for visualizing biomolecular networks in 3D, stereoscopic 3D and immersive 3D.<div class="popup-ref"><span class="ref__series"><i>Gigascience.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1-13</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fgigascience%2Fgix054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28814063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042149548&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1-13&journal=Gigascience&author=Liluashvili+V.&author=Kalayci+S.&author=Fluder+E.&author=Wilson+M.&author=Gabow+A.&author=G%C3%BCm%C3%BCs+Z.H.&title=iCAVE%3A+an+open+source+tool+for+visualizing+biomolecular+networks+in+3D%2C+stereoscopic+3D+and+immersive+3D">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and customized them to serve the specific needs of our tool. We also incorporated other utility libraries (e.g., dataTables.js) for data manipulation and interaction. For web interface styling, we relied primarily relied on Bootstrap v3.3.7, integrated with our custom CSS elements. Since our implementation utilizes only standard libraries and does not necessitate any external plug-ins, the portal runs on all modern web browsers.</div></section></section><section id="sec5.4.13" data-locator="sec5.4.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">ccRCC Inter-Tumor Proteome Heterogeneity</span></h4><section id="sec5.4.13.1" data-locator="sec5.4.13.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic Subtyping</span></h4><div class="section-paragraph">We investigated the molecular subtyping of all tumor samples based mainly on global proteomic expression to identify the associations between the multi-omics expression and clinical phenotypes, such as tumor stage and grade (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). The 3,567 (50%) most variable global proteins without missing values were analyzed by CancerSubtypes (<div class="dropBlock reference-citations"><a href="#bib184" id="back-bib184" data-db-target-for="bib184-d218962e5638" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib184">Xu et al., 2017</a><div data-db-target-of="bib184-d218962e5638" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xu T. </li><li class="loa__item">Le T.D. </li><li class="loa__item">Liu L. </li><li class="loa__item">Su N. </li><li class="loa__item">Wang R. </li><li class="loa__item">Sun B. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Li J. </li></ul><div class="reference-citations__detials">CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3131-3133</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtx378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28605519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85030680397&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3131-3133&journal=Bioinformatics&author=Xu+T.&author=Le+T.D.&author=Liu+L.&author=Su+N.&author=Wang+R.&author=Sun+B.&author=Colaprico+A.&author=Bontempi+G.&author=Li+J.&title=CancerSubtypes%3A+an+R%2FBioconductor+package+for+molecular+cancer+subtype+identification%2C+validation+and+visualization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for consensus clustering (<div class="dropBlock reference-citations"><a href="#bib111" id="back-bib111" data-db-target-for="bib111-d218962e5641" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Monti et al., 2003</a><div data-db-target-of="bib111-d218962e5641" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monti S. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mesirov J. </li><li class="loa__item">Golub T. </li></ul><div class="reference-citations__detials">Consensus Clustering A Resampling-Based Method for Class Discovery and Visualization of Gene Expression Microarray Data.<div class="popup-ref"><span class="ref__series"><i>Mach. Learn.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">91-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1023%2FA%3A1023949509487&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038724494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=91-118&journal=Mach.+Learn.&author=Monti+S.&author=Tamayo+P.&author=Mesirov+J.&author=Golub+T.&title=Consensus+Clustering+A+Resampling-Based+Method+for+Class+Discovery+and+Visualization+of+Gene+Expression+Microarray+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) of tumor subtypes. Specifically, 80% of the original sample pool was randomly subsampled without replacement and partitioned into three major clusters using hierarchical clustering, which was repeated 500 times (<div class="dropBlock reference-citations"><a href="#bib182" id="back-bib182" data-db-target-for="bib182-d218962e5644" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib182">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib182-d218962e5644" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (442)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The expression values were transformed into Z scores at the gene level using the built-in standardization function of R. The consensus-clustered samples were ordered according to the calculated distance and associated with stage and grade, four key mutations (<em>VHL</em>, <em>PBRM1</em>, <em>SETD2</em>, and <em>BAP1</em>), and the consensus clustering results from other omics data, including immune subtypes. Proteins were grouped into three clusters using hierarchical clustering. The overrepresentation analysis (ORA) was performed on the gene list of each protein cluster using WebGestaltR (the R Version of WebGestalt). The parameters were set as described above. The significance threshold was set as FDR &lt; 0.05 to identify and annotate the pathways most-associated with each protein cluster.</div></section><section id="sec5.4.13.2" data-locator="sec5.4.13.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multivariate Analysis</span></h4><div class="section-paragraph">To investigate the possible associations between genome-/proteome-wide data and clinical features, we utilized multiple omics data including RNA-seq, proteome, and phosphoproteome of tumor and NAT samples to conduct the association analysis (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A, <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">S7</a>A, and S7B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>). We also incorporated CNV and DNA methylation data from tumors only in the analysis. Our set of clinical features consists of tumor characteristics (e.g., stage, grade, margin status, left-right kidney laterality, presence of necrosis), patient properties (e.g., age, gender, BMI, country of origin), lifestyle, medical history (e.g., smoking, alcohol, diabetes), and sample handling parameters (ischemic time). We also included Immune and Stromal scores computed by ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e5670" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e5670" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) as predictors because they reflect crucial TME properties (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E). In our linear regression analysis, stage, grade, age, BMI, ischemic time, and Immune and Stromal score were used as numerical variables, whereas the remaining variables were treated as categorical. For alcohol and smoking status, lifetime non-drinkers or non-smokers were compared to the rest of the population. For country of origin, European countries (in this case, Poland and Ukraine) were compared to all other countries. p values obtained for each gene in multivariate linear regression were corrected using Benjamini-Hochberg adjustment (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d218962e5677" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib13-d218962e5677" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the false discovery rate: a practical and powerful approach to multiple testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The values of all adjusted p values for all clinical features and all data types in both tumor and adjacent normal are provided in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a>. <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A contains the number of genes with adjusted p value below 10%. To identify molecular pathways associated with tumor grade, we began with the pathway gene sets obtained from KEGG, Reactome, and Hallmark databases (<div class="dropBlock reference-citations"><a href="#" id="back-bib74" data-db-target-for="bib74-d218962e5686" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Joshi-Tope et al., 2005</a><div data-db-target-of="bib74-d218962e5686" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Joshi-Tope G. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Vastrik I. </li><li class="loa__item">D’Eustachio P. </li><li class="loa__item">Schmidt E. </li><li class="loa__item">de Bono B. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Gopinath G.R. </li><li class="loa__item">Wu G.R. </li><li class="loa__item">Matthews L. </li><li>et al.</li></ul><div class="reference-citations__detials">Reactome: a knowledgebase of biological pathways.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">D428-D432</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgki072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15608231&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13444259844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (846)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D428-D432&journal=Nucleic+Acids+Res.&author=Joshi-Tope+G.&author=Gillespie+M.&author=Vastrik+I.&author=D%E2%80%99Eustachio+P.&author=Schmidt+E.&author=de+Bono+B.&author=Jassal+B.&author=Gopinath+G.R.&author=Wu+G.R.&author=Matthews+L.&title=Reactome%3A+a+knowledgebase+of+biological+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib77" data-db-target-for="bib77-d218962e5686" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Kanehisa and Goto, 2000</a><div data-db-target-of="bib77-d218962e5686" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kanehisa M. </li><li class="loa__item">Goto S. </li></ul><div class="reference-citations__detials">KEGG: Kyoto Encyclopedia of Genes and Genomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">27-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2F28.1.27&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10592173&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=27-30&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Goto+S.&title=KEGG%3A+Kyoto+Encyclopedia+of+Genes+and+Genomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib89" data-db-target-for="bib89-d218962e5686" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">Liberzon et al., 2011</a><div data-db-target-of="bib89-d218962e5686" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Pinchback R. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Molecular signatures database (MSigDB) 3.0.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1739-1740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtr260&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21546393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79958135926&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1170)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1739-1740&journal=Bioinformatics&author=Liberzon+A.&author=Subramanian+A.&author=Pinchback+R.&author=Thorvaldsd%C3%B3ttir+H.&author=Tamayo+P.&author=Mesirov+J.P.&title=Molecular+signatures+database+%28MSigDB%29+3.0">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and removed pathways that contained more than 500 genes. For proteome and mRNA data, we computed log10(FDR) for genes with positive associations and negative associations, where FDR is the adjusted p value for a gene’s association with grade. On this dataset, we ran a one-sided Wilcoxon rank-sum test for each pathway gene set versus all other genes. These p values were then corrected using Benjamini-Hochberg adjustment. A score for each pathway was computed as (+/−)log10(p-adj), depending on whether the rank-sum test p value showed it to be more up- or downregulated compared to other genes. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5" class="scroll-into-link">Table S5</a> lists the scores of all pathways with adjusted p value &lt; 0.01. <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">S7</a>B present a subset of pathways significantly associated with grade that were selected to be representative, non-redundant, and as informative as possible regarding the biological functions contained in the full set.</div></section></section><section id="sec5.4.14" data-locator="sec5.4.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune-based Clustering of ccRCC tumors</span></h4><section id="sec5.4.14.1" data-locator="sec5.4.14.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Subtype identification based on cell type composition</span></h4><div class="section-paragraph">The abundances of 64 different cell types in 175 ccRCC samples (103 tumor samples and 72 NAT samples) were computed via xCell (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d218962e5707" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d218962e5707" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For this analysis, FPKM (Fragments Per Kilobase Million) mRNA expression values were utilized. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a> contains the final score computed by xCell for different cell types for the 175 samples. Based on these 64 signatures, consensus clustering was performed to identify groups of samples with the same immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus (<div class="dropBlock reference-citations"><a href="#" id="back-bib111" data-db-target-for="bib111-d218962e5713" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Monti et al., 2003</a><div data-db-target-of="bib111-d218962e5713" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monti S. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mesirov J. </li><li class="loa__item">Golub T. </li></ul><div class="reference-citations__detials">Consensus Clustering A Resampling-Based Method for Class Discovery and Visualization of Gene Expression Microarray Data.<div class="popup-ref"><span class="ref__series"><i>Mach. Learn.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">91-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1023%2FA%3A1023949509487&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038724494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=91-118&journal=Mach.+Learn.&author=Monti+S.&author=Tamayo+P.&author=Mesirov+J.&author=Golub+T.&title=Consensus+Clustering+A+Resampling-Based+Method+for+Class+Discovery+and+Visualization+of+Gene+Expression+Microarray+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib182" data-db-target-for="bib182-d218962e5713" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib182">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib182-d218962e5713" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (442)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) within the Bioconductor package CancerSubtypes (<div class="dropBlock reference-citations"><a href="#bib184" id="back-bib184" data-db-target-for="bib184-d218962e5716" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib184">Xu et al., 2017</a><div data-db-target-of="bib184-d218962e5716" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xu T. </li><li class="loa__item">Le T.D. </li><li class="loa__item">Liu L. </li><li class="loa__item">Su N. </li><li class="loa__item">Wang R. </li><li class="loa__item">Sun B. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Li J. </li></ul><div class="reference-citations__detials">CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3131-3133</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtx378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28605519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85030680397&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3131-3133&journal=Bioinformatics&author=Xu+T.&author=Le+T.D.&author=Liu+L.&author=Su+N.&author=Wang+R.&author=Sun+B.&author=Colaprico+A.&author=Bontempi+G.&author=Li+J.&title=CancerSubtypes%3A+an+R%2FBioconductor+package+for+molecular+cancer+subtype+identification%2C+validation+and+visualization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, 80% of the original 175 samples were randomly subsampled without replacement and partitioned into six major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 200 times (<div class="dropBlock reference-citations"><a href="#bib182" id="back-bib182" data-db-target-for="bib182-d218962e5719" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib182">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib182-d218962e5719" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (442)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). <a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A shows the heatmap of scores for key cell types from the 175 ccRCC samples. The four tumor sample-based subtypes were tested for association with clinical variables (e.g., tumor grade) and genomic aberrations such as chr14 loss via a Chi-Square test of independence (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). The upregulation of immune and stromal cells in a particular immune group based on tumor samples was assessed using a multivariate linear regression in which the score of each immune and stromal cell was modeled as a function of immune groups. First, every score was normalized across tumor samples by subtracting the mean and dividing by the standard deviation. Then, the score of the <em>j</em>-th cell for sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math> was modeled as:<div class="labeled-equation clearfix"><div id="fd1" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si28.gif"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><munderover><mo>∑</mo><mrow><mi>k</mi><mo linebreak="badbreak">=</mo><mn>1</mn></mrow><mn>4</mn></munderover><mrow><msub><mi>β</mi><mrow><mi>k</mi><mo>,</mo><mi>j</mi></mrow></msub><mn>1</mn><mrow><mo>(</mo><mrow><mi>i</mi><mo linebreak="badbreak">∈</mo><msub><mi>I</mi><mi>k</mi></msub></mrow><mo>)</mo></mrow><mo>+</mo><msub><mi mathvariant="italic">ε</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></mrow></math></div><div class="formulaLabel">(1) </div></div><br></br>with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si29.gif"><mrow><msub><mi mathvariant="italic">ε</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">∼</mo><mi>N</mi><mrow><mo>(</mo><mrow><mn>0</mn><mo>,</mo><msub><mi>σ</mi><mi>j</mi></msub></mrow><mo>)</mo></mrow></mrow></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si30.gif"><mrow><msub><mi>I</mi><mi>k</mi></msub></mrow></math>being the set of samples belonging to the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th immune cluster, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si31.gif"><mrow><mn>1</mn><mrow><mo>(</mo><mi>A</mi><mo>)</mo></mrow></mrow></math>being an indicator function equal to 1 if the event <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si32.gif"><mi>A</mi></math> occurs and 0 otherwise, and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si33.gif"><mrow><msub><mi>β</mi><mrow><mi>k</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math> being the coefficient capturing the association between gene <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math> and the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th immune group. Benjamini-adjusted p values can be found in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>.</div></section><section id="sec5.4.14.2" data-locator="sec5.4.14.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Estimation of Stromal and Immune Scores</span></h4><div class="section-paragraph">ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d218962e5896" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Yoshihara et al., 2013</a><div data-db-target-of="bib186-d218962e5896" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was also used to infer tumor purity and immune and stromal scores based on RNA-seq data and global proteomic data (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>B; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). For the analysis of global proteomic data, only proteins with no missing values across all samples were considered. As shown in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>, immune and stromal scores based on global proteomic data and RNA-seq data were highly correlated (i.e., a Pearson correlation between immune scores based on RNA-seq and proteomic data higher than 0.85 and Pearson correlation of stromal scores higher than 0.75 for both NAT samples and ccRCC tumor samples). For this comparison, only samples overlapping between the two data types were considered (i.e., 103 ccRCC tumor samples and 72 NAT samples).</div></section><section id="sec5.4.14.3" data-locator="sec5.4.14.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Validation of microenvironment scores using DNA methylation data</span></h4><div class="section-paragraph">Edec was used to infer the tumor composition from DNA methylation data (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Edec is based on the principle that DNA methylation measured from whole bulk tumor is the linear combination of measurements from individual cell types weighted by their cell proportions. For the reference methylation profiles, we collected DNA methylation data (represented as beta-values) for five cell types – kidney cancer epithelial cells, kidney normal epithelial cells, fibroblasts, endothelial cells and immune cells (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). Using a one-versus-all t test, we selected the methylation probes that distinguish the given cell type from other cell types. The probes were then mixed for the data deconvolution.</div></section><section id="sec5.4.14.4" data-locator="sec5.4.14.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immunohistochemistry (IHC) validation of immune cell compositions</span></h4><div class="section-paragraph">Formalin-fixed, paraffin-embedded 5 μm tissue sections were stained in batches for CD4, CD8, and CD163 in a central laboratory at the Johns Hopkins Hospital according to standard automated protocols. Deparaffinization and rehydration were performed, followed by antigen retrieval and antibody staining. CD4 and CD8 IHC was performed using the Ventana Benchmark Ultra autostaining system (Roche) using mouse monoclonal anti-CD8 (C8144B) antibody (Cell marque) and rabbit monoclonal anti-CD4(Sp35) antibody (Roche), followed by detection with the iVIEW DAB Detection Kit (Roche). CD163 IHC was performed on the Leica Bond MAX autostaining system (Leica Biosystems) using anti-CD163 (10D6) antibody (Leica Biosystems) followed by detection with Bond Polymer Refine Detection kit (Leica Biosystems). For tissue section imaging, slides were imaged using a Ventana iScan HT slide scanner (Roche) and processed using the Ventana Virtuoso software (Roche) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>D).</div></section><section id="sec5.4.14.5" data-locator="sec5.4.14.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Analysis of Differentially-Expressed Genes and Pathways</span></h4><div class="section-paragraph">Genes that were upregulated and downregulated in each of the four immune clusters were identified based on 103 tumor samples. For each data type, every feature vector was normalized by subtracting the mean and dividing by the standard deviation. For each data type, the expression level of gene <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math> and ccRCC sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math> (i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si34.gif"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math>) was modeled via Equation (<a href="#fd1" data-locator="fd1" class="scroll-into-link">1</a>). Model [1] was implemented for each gene <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif"><mi>j</mi></math>. <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a> shows upregulated and downregulated genes identified based on different data types. Considering genes that were up- and downregulated with Benjamini’s adjusted p value lower than 10%, a Fisher’s exact test was implemented to derive enriched pathways (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B and <a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>E; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>) (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d218962e5973" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib13-d218962e5973" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the false discovery rate: a practical and powerful approach to multiple testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For this analysis, pathways from the Reactome, KEGG and Hallmark databases were considered and as background the full list of gene/proteins observed under each data type was utilized. Pathway scores for 103 ccRCC tumor samples and 80 NAT samples were computed based on combined z-score using the R package GSVA (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d218962e5976" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Hänzelmann et al., 2013</a><div data-db-target-of="bib57-d218962e5976" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (722)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Pathway scores based on different data types can be found in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>. Only combined z-scores of some key enriched pathways (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>E) were included.</div></section><section id="sec5.4.14.6" data-locator="sec5.4.14.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Angiogenesis and T-Effector Signatures</span></h4><div class="section-paragraph">Using package GSVA (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d218962e5991" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Hänzelmann et al., 2013</a><div data-db-target-of="bib57-d218962e5991" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (722)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), Angiogenesis (VEGFA, KDR, ESM1, PECAM1, ANGPTL4, and CD34) and T-Effector (CD8A, EOMES, PRF1, IFNG, and CD274) signatures (<div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d218962e5994" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">McDermott et al., 2018</a><div data-db-target-of="bib101-d218962e5994" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McDermott D.F. </li><li class="loa__item">Huseni M.A. </li><li class="loa__item">Atkins M.B. </li><li class="loa__item">Motzer R.J. </li><li class="loa__item">Rini B.I. </li><li class="loa__item">Escudier B. </li><li class="loa__item">Fong L. </li><li class="loa__item">Joseph R.W. </li><li class="loa__item">Pal S.K. </li><li class="loa__item">Reeves J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were computed for 103 ccRCC samples. Upregulation of these signatures in a particular immune group was assessed via (1) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>F; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>).</div></section><section id="sec5.4.14.7" data-locator="sec5.4.14.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune-based clustering on The Cancer Genome Atlas (TCGA) data</span></h4><div class="section-paragraph">Based on 103 ccRCC samples, we selected genes that were differentially expressed in each of the four immune groups (CD8+ Inflamed, CD8- Inflamed, VEGF Immune Desert, Metabolic Immune Desert) using the function TCGAanalyze_DEA from the package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d218962e6008" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d218962e6008" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and following our previously-described workflow (<div class="dropBlock reference-citations"><a href="#bib157" id="back-bib157" data-db-target-for="bib157-d218962e6011" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib157">Silva et al., 2016</a><div data-db-target-of="bib157-d218962e6011" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Silva T.C. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">D’Angelo F. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Ceccarelli M. </li><li class="loa__item">Noushmehr H. </li></ul><div class="reference-citations__detials"><em>TCGA Workflow</em>: Analyze cancer genomics and epigenomics data using Bioconductor packages.<div class="popup-ref"><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1542</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.12688%2Ff1000research.8923.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28232861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85013753091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1542&journal=F1000Res.&author=Silva+T.C.&author=Colaprico+A.&author=Olsen+C.&author=D%E2%80%99Angelo+F.&author=Bontempi+G.&author=Ceccarelli+M.&author=Noushmehr+H.&title=TCGA+Workflow%3A+Analyze+cancer+genomics+and+epigenomics+data+using+Bioconductor+packages">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In particular, only genes with 10% FDR cut-off and log fold change greater than 1 were selected. Following this procedure, 2,252 unique genes were selected across different immune groups (i.e., 1,067 for CD8+ Inflamed, 721 for CD8- Inflamed, 1,054 for VEGF Immune Desert and 898 for Metabolic Immune Desert). Based on this set of genes, the one-class regression model, OCRL (<div class="dropBlock reference-citations"><a href="#bib159" id="back-bib159" data-db-target-for="bib159-d218962e6014" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib159">Sokolov et al., 2016</a><div data-db-target-of="bib159-d218962e6014" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sokolov A. </li><li class="loa__item">Paull E.O. </li><li class="loa__item">Stuart J.M. </li></ul><div class="reference-citations__detials">One-Class Detection of Cell States in Tumor Subtypes.<div class="popup-ref"><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-Class+Detection+of+Cell+States+in+Tumor+Subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was applied to construct a CPTAC data-based classifier for each immune group. The logistic regression model was trained using the R CRAN package, gelnet. The OCRL pipeline returned a 2,252 dimensional vector of weights for each immune group, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si35.gif"><mrow><msub><mi>W</mi><mi>i</mi></msub></mrow></math> with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si36.gif"><mrow><mo linebreak="goodbreak" linebreakstyle="after">∈</mo><mo>{</mo><mn>1,2</mn></mrow></math>,3,4} Then, TCGA kidney renal clear cell carcinoma (KIRC) samples were allocated into immune groups based on two scores that were computed using the set of 2,252 pre-selected genes. Specifically, for the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th sample in TCGA data and each immune group <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>, the following two scores were computed:<ul class="ce-list--remove-bullets" id="ulist0025"><li class="ce-list--remove-bullets__list-item" id="u0060"><div class="ce-list--remove-bullets__list-item__text"><span class="underline">Score 1:</span> Spearman correlation between the model’s weight vector and the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample’s expression profile, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si37.gif"><mrow><msub><mi>Z</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mi>c</mi><mi>o</mi><mi>r</mi><mrow><mo>(</mo><mrow><msub><mi>W</mi><mi>i</mi></msub><mo>,</mo><msub><mi>X</mi><mi>k</mi></msub></mrow><mo>)</mo></mrow></mrow></math> with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si38.gif"><mrow><msub><mi>X</mi><mi>k</mi></msub></mrow></math> being a 2,252 dimensional vector containing expression levels of the 2,252 genes for the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample. The correlation between <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si35.gif"><mrow><msub><mi>W</mi><mi>i</mi></msub></mrow></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si38.gif"><mrow><msub><mi>X</mi><mi>k</mi></msub></mrow></math> would be high if the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample belonged to the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>-th immune group. Scores <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si39.gif"><mrow><mrow><mo>{</mo><mrow><msub><mi>Z</mi><mrow><mi>s</mi><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow></math> were normalized to be in the unit interval {0,1} by subtracting the lowest value and then dividing by the maximum value.</div></li><li class="ce-list--remove-bullets__list-item" id="u0065"><div class="ce-list--remove-bullets__list-item__text"><span class="underline">Score 2:</span> Spearman correlation between the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample’s expression profile and the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si21.gif"><mi>s</mi></math>-th CPTAC sample’s expression profile of 2,252 genes, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si40.gif"><mrow><msub><mi>S</mi><mrow><mi>s</mi><mi>k</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mi>c</mi><mi>o</mi><mi>r</mi><mrow><mo>(</mo><mrow><msub><mi>Y</mi><mi>s</mi></msub><mo>,</mo><msub><mi>X</mi><mi>k</mi></msub></mrow><mo>)</mo></mrow></mrow></math>with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si41.gif"><mrow><msub><mi>Y</mi><mi>s</mi></msub></mrow></math> being a 2,252 dimensional vector containing expression levels of the pre-selected 2,252 genes for the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si21.gif"><mi>s</mi></math>-th CPTAC sample, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si38.gif"><mrow><msub><mi>X</mi><mi>k</mi></msub></mrow></math> being a 2,252 dimensional vector containing expression levels of the 2,252 genes for the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample. This score was computed for each CPTAC sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si21.gif"><mi>s</mi></math> belonging to the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>-th immune group. Scores <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si42.gif"><mrow><msub><mrow><mrow><mo>{</mo><mrow><msub><mi>S</mi><mrow><mi>s</mi><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow><mrow><mi>s</mi><mo>∈</mo><msub><mi>I</mi><mi>i</mi></msub></mrow></msub></mrow></math> were normalized to be in the unit interval {0,1}. The final score measuring the association between the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>-th TCGA sample and the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>-immune group (i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si43.gif"><mrow><mrow><mrow><msub><mi>Q</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow></math> was obtained by averaging scores <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si42.gif"><mrow><msub><mrow><mrow><mo>{</mo><mrow><msub><mi>S</mi><mrow><mi>s</mi><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow><mrow><mi>s</mi><mo>∈</mo><msub><mi>I</mi><mi>i</mi></msub></mrow></msub></mrow></math> with <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si44.gif"><mrow><msub><mi>I</mi><mi>i</mi></msub></mrow></math> being the set of samples in the <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif"><mi>i</mi></math>-th immune group.</div></li></ul></div><div class="section-paragraph">The first score (i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si45.gif"><mrow><msub><mi>Z</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow></math>) was utilized previously to classify samples (<div class="dropBlock reference-citations"><a href="#bib205" id="back-bib205" data-db-target-for="bib205-d218962e6332" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib205">Malta et al., 2018</a><div data-db-target-of="bib205-d218962e6332" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kaminska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation Article Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kaminska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation+Article+Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In this study, a second score was considered to avoid cases in which multiple immune categories resulted in the same score. In particular, the final score was derived by averaging scores <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si46.gif"><mrow><mrow><mo>{</mo><mrow><msub><mi>Z</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si47.gif"><mrow><mrow><mo>{</mo><mrow><msub><mi>Q</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow></math>, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si48.gif"><mrow><msub><mi>X</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mrow><mrow><mrow><mo>(</mo><mrow><msub><mi>Z</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">+</mo><msub><mi>Q</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mo>/</mo><mn>2</mn></mrow><mo>.</mo></mrow></math> Finally, to each TCGA sample <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si24.gif"><mi>k</mi></math>, the immune group with the highest score was assigned, i.e., <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si49.gif"><mrow><msub><mi>G</mi><mi>k</mi></msub><mo linebreak="goodbreak" linebreakstyle="after">=</mo><mi>a</mi><mi>r</mi><mi>g</mi><mspace width="0.25em"></mspace><mi>m</mi><mi>a</mi><msub><mi>x</mi><mi>i</mi></msub><msub><mi>X</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow></math>. This final score can be found in <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>. This classification resulted in 126 samples allocated to the CD8- Inflamed group, 156 samples allocated to the CD8+ Inflamed group, 135 samples allocated to the Metabolic Immune Desert and 78 samples allocated to the VEGF Immune Desert group (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). This TCGA-based classification was compared to that based on CPTAC data in terms of immune and stromal cell infiltration, pathway activities and key markers that were found to be upregulated in different immune groups based on CPTAC data. Similarly to CPTAC data, the concentration of different immune and stromal cells was computed via xCell (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d218962e6459" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d218962e6459" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>), while the activity of key pathways was derived via a combined z-score (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d218962e6466" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Hänzelmann et al., 2013</a><div data-db-target-of="bib57-d218962e6466" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (722)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>E; <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>). To identify immune and stromal cells upregulated in different immune groups, the strategy adopted for CPTAC data was utilized (Model 1, <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>).</div></section><section id="sec5.4.14.8" data-locator="sec5.4.14.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Clinical Outcome of Immune Groups</span></h4><div class="section-paragraph">Immune groups based on TCGA data were utilized to better understand the clinical outcome and expected survival for different immune groups (<a data-link="modal" data-target="#image-S0092-8674(19)31123-7figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>G and S6H). Overall survival data and tumor grade information for 495 TCGA KIRC samples, deposited in the Genomic Data Commons (GDC) Data Portal, were downloaded using the function GDCquery_clinic from the package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d218962e6486" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d218962e6486" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). <a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a> shows Benjamini’s adjusted p values (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d218962e6492" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib13-d218962e6492" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the false discovery rate: a practical and powerful approach to multiple testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from a pairwise Log Rank test comparing survival curves that correspond to different immune groups. Kaplan-Meier overall survival curves were generated using the function TCGAanalyze_survival from the package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d218962e6495" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Colaprico et al., 2016</a><div data-db-target-of="bib30-d218962e6495" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The association between high-grade tumors (i.e., grade 3 and 4) and immune groups was assessed via a Chi-Square test of independence (<a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7" class="scroll-into-link">Table S7</a>).</div></section></section></section><section id="sec5.5" data-locator="sec5.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and Statistical Analysis</span></h3><section id="sec5.5.1" data-locator="sec5.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Transcriptomic Quantitation</span></h4><div class="section-paragraph">The resulting BAM files were analyzed for expression using feature counts against a transcriptomic reference based on Gencode 26. The resulting gene-level counts for protein-coding genes were transformed into FPKMs using edgeR.</div></section><section id="sec5.5.2" data-locator="sec5.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic and Phosphoproteomic Quantitation</span></h4><div class="section-paragraph">Whole proteome and phosphopeptide-enriched datasets were processed separately but using the same pipeline steps as described in the “Protein database searching and quantification of global and phosphoproteomic data” section of the <a href="#sec4" class="scroll-into-link">STAR Methods</a>. A tutorial describing all steps of the analysis, including specific input parameter files, command-line option, and all software tools necessary to replicate the results are available at <a class="inter-ref-link" href="https://github.com/Nesvilab" target="_blank">https://github.com/Nesvilab</a>.</div><div class="section-paragraph">The statistical details of all experiments have been reported in the manuscript text, figure legends and corresponding <a href="#sec4" class="scroll-into-link">STAR Methods</a> section descriptions. Data analysis was performed in Excel, R, and Python.</div></section></section><section id="sec5.6" data-locator="sec5.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and Code Availability</span></h3><div class="section-paragraph">Raw data files for proteomic analysis reported in this paper are hosted by the CPTAC Data Portal and can be accessed at: <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S044" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S044</a> and <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>. Genomic and transcriptomic data files can be accessed at: <a class="inter-ref-link" href="https://portal.gdc.cancer.gov/" target="_blank">https://portal.gdc.cancer.gov/</a>. Processed data utilized for this publication can be accessed at: <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S050" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S050</a>.</div><div class="section-paragraph">Several custom coding softwares were generated as part of this study and have been referenced in the corresponding STAR Methods section and listed with links to the coding script in the <a href="#sec5.1" class="scroll-into-link">Key Resources Table</a>: software codes generated by the Cieslik laboratory for genomic analyses (CNVEX and CRISP), by the Nesvizhskii laboratory for proteomic data processing (Philosopher and TMT-Integrator) by the Wang lab for data imputation (DreamAI), and by the Zhang lab for data processing and neoantigen detection (NeoFlow and PepQuery).</div><div class="section-paragraph">Interactive data analysis tools were generated by the Wang lab: a web-based application for visualizing a heatmap of 22,867 genes across 110 samples can be accessed at: <a class="inter-ref-link" href="http://ccrcc.cptac-data-view.org" target="_blank">http://ccrcc.cptac-data-view.org</a>; a web-based application for interactively exploring ccRCC phosphoproteomic co-expression networks (3,614 nodes, 11,200 edges) and their modules is available at: <a class="inter-ref-link" href="http://ccrcc.cptac-network-view.org/" target="_blank">http://ccrcc.cptac-network-view.org/</a>.</div><section id="sec5.6.1" data-locator="sec5.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Additional Resources</span></h4><div class="section-paragraph">The CPTAC program website, which includes details about program initiatives, investigators, and datasets, can be accessed at: <a class="inter-ref-link" href="https://proteomics.cancer.gov/programs/cptac" target="_blank">https://proteomics.cancer.gov/programs/cptac</a></div></section></section></section><section id="sec6" data-locator="sec6" tabindex="0"><h2 class="top" id="secsectitle0515" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the National Cancer Institute Clinical Proteomic Tumor Analysis Consortium are Matthew J. Ellis, Thomas Westbrook, Michael Lewis, Kimberly Holloway, Beom-Jun Kim, Meenakshi Anurag, David Wheeler, Doug Chan, Sue Hilsenbeck, Bing Zhang, Suhas Vasaikar, Chen Huang, Anna Malovannaya, Bo Wen, Zhiao Shi, Stacey Gabriel, Samuel H. Payne, Steven A. Carr, Michael A. Gillette, Gaddy Getz, Chet Birger, D. R. Mani, Shankha Satpathy, Karl Clauser, Karsten Krug, Yifat Geffen, Ramani Bhupendra Kothadia, David Heiman, Yosef Maruvka, Karen A. Ketchum, Shuang Cai, Marcin J. Domagalski, Ratna R. Thangudu, William E. Bocik, Amanda G. Paulovich, Jeffrey R. Whiteaker, Alexander R. Pico, Nathan J. Edwards, Peter B. McGarvey, Simina M. Boca, Eric E. Schadt, Pei Wang, Francesca Petralia, Xiaoyu Song, Boris Reva, Dmitry Rykunov, Weiping Ma, Shrabanti Chowdhury, Azra Krek, Jiayi Ji, Anna Calinawan, Noam Beckmann, Umut Ozbek, Seungyeul Yoo, Jun Zhu, Zhidong Tu, Selim Kalayci, Zeynep Gümüş, Nicole Tignor, Shalin Patel, Lin S. Chen, Yihao Lu, Tatiana Omelchenko, Daniel W. Chan, Hui Zhang, Zhen Zhang, David J. Clark, Lori J. Sokoll, Qing Kay Li, Lijun Chen, Jiang Qian, Heng Zhu, Gerald W. Hart, Yingwei Hu, Jianbo Pan, Tung-Shing Lih, Michael Schnaubelt, Kyung-Cho Cho, Stefani N. Thomas, Ana I. Robles, Emily Boja, Tara Hiltke, Christopher R. Kinsinger, Mehdi Mesri, Henry Rodriguez, Annette Marrero-Oliveras, Linda Hannick, Nancy Roche, Maureen Dyer, Melissa Borucki, Negin Vatanian, Kim Elburn, James Suh, Karna Robinson, Munziba Khan, Tanya Krubit, Michelle Chaikin, Sonya Carter, Liqun Qi, Pushpa Hariharan, Sailaja Mareedu, Ki Sung Um, Stephen E. Stein, Sanford P. Markey, Meghan C. Burke, David Fenyö, Kelly V. Ruggles, Lili Blumenberg, MacIntosh Cornwell, Runyu Hong, Emily Kawaler, Zhi Li, Wenke Liu, Ying Wang, Hua Zhou, Brian Druker, Jeffrey Tyner, Anupriya Agarwal, Cristina Tognon, Uma Borate, Emek Demir, Marc Loriaux, Tao Liu, Karin D. Rodland, Richard D. Smith, Vladislav A. Petyuk, Marina A. Gritsenko, Matthew E. Monroe, Paul Piehowski, Jason McDermott, Xi Chen, Antonio Colaprico, Alexey Nesvizhskii, Marcin P. Cieslik, Guo Ci Teo, Snehal Patil, Yuping Zhang, Jin Chen, Saravana Mohan Dhanasekaran, Arul M. Chinnaiyan, Gilbert S. Omenn, Corbin D. Jones, Katherine A. Hoadley, Barbara Hindenach, Amy M. Perou, Piotr Mieczkowski, Ewa Malc, Stephanie De Young, Yan Shi, Michael Vernon, Alyssa Charamut, Donghui Tan, Jacob Day, Tara Skelly, Uma Velvulou, Houston Culpepper, Hongwei Liu, John McGee, Robert Zelt, Michael J. Birrer, Scott D. Jewell, Galen Hostetter, Daniel C. Rohrer, Dana R. Valley, Chelsea J. Newton, Sandra Cottingham, David Chesla, Pamela Grady, Jennifer Eschbacher, Michael Smith, Jasmine Huang, Abdul Samad Hashimi, Ross Bremner, Matthew Anderson, Michael M Ittmann, Patricia Castro, Rajiv Dhir, Robert Edwards, Francesmary Modugno, George D. Wilson, Barbara Pruetz, Shannon Richey, Rebecca Montgomery, Darlene Tansil, Li Ding, Matthew A. Wyczalkowski, Song Cao, Daniel C. Zhou, Alla Karpova, Yige Wu, Liang-Bo Wang, Sunantha Sethuraman, Qingsong Gao, Hua Sun, Sohini Sengupta, Reyka G. Jayasinghe, YizeLi, Steven M. Foltz, Feng Chen, James J. Hsieh, Milan G. Chheda, Sherri R. Davies, and Ruiyang Liu.</div></section><section><h2 class="top" id="secsectitle0520" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by the NIH, National Cancer Institute’s Clinical Proteomic Tumor Analysis Consortium (CPTAC) grants U24CA210985, U24CA210993, U24CA210967, U24CA210954, and U24CA210972.</div><h3>Author Contributions</h3><div class="section-paragraph">Conceptualization, H.R., D.C., A.I.N., P.W., and H.Z.; Methodology, D.J.C. and L.C.; Software, F.P., J.P., X.S., Y.H., F.d.V.L., B.R., T.-S.M.L., H.-Y.C., W.M., C.H., A. Krek, Y.L., D.R., L.S.C., U.O., S.V., S.Y., S. Chowdhury, J.J., A. Kong, S.S., D.M.A., N.E., Z.Z., M.C., A.I.N., and P.W.; Validation, S.M.D., K.L.Q., and K.-C.C.; Formal Analysis, D.J.C., S.M.D., F.P., J.P., X.S., Y.H., B.R., T.-S.M.L., H.-Y.C., W.M., C.H., A. Krek, Y.L., D.R., L.S.C., U.O., S.V., Y.W., S.Y., S. Chowdhury, J.J., A. Kong, S.S., D.M.A., M.A., A. Colaprico, S. Cao, S. Kalayci, S.M., W.L., K.R., D.G., E.K., G.C.T., B.W., Y.Z., S. Keegan, K.L., F.C., N.E., A.I.N., P.W., and H.Z.; Investigation, D.J.C., L.C., M.S., K.-C.C., D.W.C., and H.Z.; Resources, Q.K.L., C.P.P., G.B., A.A., J.L., and M.T.; Data Curation, D.J.C., S.M.D., W.M., M.A.W., M.S., M.A., M.C., A.I.N., P.W., and H.Z.; Writing – Original Draft, D.J.C., A.I.N., P.W., and H.Z.; Writing – Review &amp; Editing, all authors; Visualization, D.J.C., S.M.D., F.P., J.P., X.S., Y.H., B.R., T.-S.M.L., C.H., C.J.R., A. Krek, Y.L., D.R., U.O., S.V., M.A., A. Calinawan, Z.H.G., Y.Z., and M.C.; Supervision, D.J.C., S.M.D., C.J.R., P.M.P., X.S.C., C.P.P., A.A.H, G.B., J.H., A.A., T.O., J.L., M.W., W.M.L., J.Q., D.F., B.Z., L.D., E.S., A.M.C., Z.Z., G.S.O., D.W.C., A.I.N., P.W., and H.Z.; Project Administration, D.J.C., C.R.K., M.T., M.M., E.S.B., M.M., T.H., A.I.R., H.R., D.W.C., A.I.N., P.W., and H.Z.; Funding Acquisition, B.Z., L.D., D.F., E.S., A.M.C., Z.Z., D.W.C., A.I.N., P.W., and H.Z.</div><h3>Declaration of Interests</h3><div class="section-paragraph">Dr. Eric Schadt serves as Chief Executive Officer for Sema4 and has an equity interest in this company.</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental Information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental Information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/ea95ac43-bc30-42c9-9119-1801852141b7/mmc1.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.11
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. ccRCC Patient Clinical Characteristics, CPTAC Data Annotation, and Genomic and Transcriptomic Sequencing Depth, Related to Figures 1, 3, 4, 5, 6, 7, S1, S3, S4, S5, and S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/2499de6d-f9aa-475e-bf43-234b0cc3411f/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.5
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Genomic Alteration Annotation, Chromosome Translocation Events, and CpG Island Methylator Phenotype (CIMP) Analysis, Related to Figures 1 and S1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/67963163-9672-4725-aaaa-694550b807c6/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (2.48
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. iProFun Results, trans-Effects, Related to Figures 2 and S2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/c4c3b033-43e7-4ded-a07e-a5a12566a7ef/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.2
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. mRNA and Protein Correlation, Related to Figures 3, 4, S3, and S4</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/52f30d5d-db03-4bb9-9ab3-74321b1d2604/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.0
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Tumor Protein Expression Profiles, Related to Figures 4, 7, S4, and S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/0c2f9545-9a42-4cac-9235-6ad01640cf81/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.18
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Phosphopeptide Analysis, Related to Figures 5 and S5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.10.007/attachment/a357423a-6d1e-45e4-9dd1-9d0ad2b38433/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (2.93
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. Immune Analysis, Related to Figures 6 and S6</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0545" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Alyass A. </li><li class="ref__authors__name">Turcotte M. </li><li class="ref__authors__name">Meyre D. </li></ul><div class="ref__title">From big data analysis to personalized medicine for all: challenges and opportunities.</div><span class="ref__series"><i>BMC Med. Genomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">33</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84933530616&amp;cf=" onclick="newWindow(this.href);return false">Scopus (221)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26112054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs12920-015-0108-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=33&journal=BMC+Med.+Genomics&author=Alyass+A.&author=Turcotte+M.&author=Meyre+D.&title=From+big+data+analysis+to+personalized+medicine+for+all%3A+challenges+and+opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anderson K.G. </li><li class="ref__authors__name">Stromnes I.M. </li><li class="ref__authors__name">Greenberg P.D. </li></ul><div class="ref__title">Obstacles Posed by the Tumor Microenvironment to T cell Activity: A Case for Synergistic Therapies.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">311-325</span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85015064348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (126)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28292435&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2017.02.008&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=311-325&journal=Cancer+Cell&author=Anderson+K.G.&author=Stromnes+I.M.&author=Greenberg+P.D.&title=Obstacles+Posed+by+the+Tumor+Microenvironment+to+T%C2%A0cell+Activity%3A+A+Case+for+Synergistic+Therapies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Antonioli L. </li><li class="ref__authors__name">Pacher P. </li><li class="ref__authors__name">Vizi E.S. </li><li class="ref__authors__name">Haskó G. </li></ul><div class="ref__title">CD39 and CD73 in immunity and inflammation.</div><span class="ref__series"><i>Trends Mol. Med.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">355-367</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84878614399&amp;cf=" onclick="newWindow(this.href);return false">Scopus (427)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23601906&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.03.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=355-367&journal=Trends+Mol.+Med.&author=Antonioli+L.&author=Pacher+P.&author=Vizi+E.S.&author=Hask%C3%B3+G.&title=CD39+and+CD73+in+immunity+and+inflammation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Arai E. </li><li class="ref__authors__name">Chiku S. </li><li class="ref__authors__name">Mori T. </li><li class="ref__authors__name">Gotoh M. </li><li class="ref__authors__name">Nakagawa T. </li><li class="ref__authors__name">Fujimoto H. </li><li class="ref__authors__name">Kanai Y. </li></ul><div class="ref__title">Single-CpG-resolution methylome analysis identifies clinicopathologically aggressive CpG island methylator phenotype clear cell renal cell carcinomas.</div><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1487-1493</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865339838&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22610075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fcarcin%2Fbgs177&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=1487-1493&journal=Carcinogenesis&author=Arai+E.&author=Chiku+S.&author=Mori+T.&author=Gotoh+M.&author=Nakagawa+T.&author=Fujimoto+H.&author=Kanai+Y.&title=Single-CpG-resolution+methylome+analysis+identifies+clinicopathologically+aggressive+CpG+island+methylator+phenotype+clear+cell+renal+cell+carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Aran D. </li><li class="ref__authors__name">Hu Z. </li><li class="ref__authors__name">Butte A.J. </li></ul><div class="ref__title">xCell: digitally portraying the tissue cellular heterogeneity landscape.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ardlie K.G. </li><li class="ref__authors__name">Deluca D.S. </li><li class="ref__authors__name">Segre A.V. </li><li class="ref__authors__name">Sullivan T.J. </li><li class="ref__authors__name">Young T.R. </li><li class="ref__authors__name">Gelfand E.T. </li><li class="ref__authors__name">Trowbridge C.A. </li><li class="ref__authors__name">Maller J.B. </li><li class="ref__authors__name">Tukiainen T. </li><li class="ref__authors__name">Lek M. </li><li>et al.</li></ul><div class="ref__title">The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">348</span></span>: <span class="ref__seriesPages">648-660</span><div class="ref__controls"><a href="#back-bib194" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84929001104&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1921)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25954001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1126%2Fscience.1262110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=648-660&journal=Science&author=Ardlie+K.G.&author=Deluca+D.S.&author=Segre+A.V.&author=Sullivan+T.J.&author=Young+T.R.&author=Gelfand+E.T.&author=Trowbridge+C.A.&author=Maller+J.B.&author=Tukiainen+T.&author=Lek+M.&title=The+Genotype-Tissue+Expression+%28GTEx%29+pilot+analysis%3A+Multitissue+gene+regulation+in+humans">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Argentini A. </li><li class="ref__authors__name">Goeminne L.J.E. </li><li class="ref__authors__name">Verheggen K. </li><li class="ref__authors__name">Hulstaert N. </li><li class="ref__authors__name">Staes A. </li><li class="ref__authors__name">Clement L. </li><li class="ref__authors__name">Martens L. </li></ul><div class="ref__title">moFF: a robust and automated approach to extract peptide ion intensities.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">964-966</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84999277470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27898063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4075&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=964-966&journal=Nat.+Methods&author=Argentini+A.&author=Goeminne+L.J.E.&author=Verheggen+K.&author=Hulstaert+N.&author=Staes+A.&author=Clement+L.&author=Martens+L.&title=moFF%3A+a+robust+and+automated+approach+to+extract+peptide+ion+intensities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ascierto P.A. </li><li class="ref__authors__name">Schadendorf D. </li><li class="ref__authors__name">Berking C. </li><li class="ref__authors__name">Agarwala S.S. </li><li class="ref__authors__name">van Herpen C.M.L. </li><li class="ref__authors__name">Queirolo P. </li><li class="ref__authors__name">Blank C.U. </li><li class="ref__authors__name">Hauschild A. </li><li class="ref__authors__name">Beck J.T. </li><li class="ref__authors__name">St-Pierre A. </li><li>et al.</li></ul><div class="ref__title">MEK162 for patients with advanced melanoma harbouring NRAS or Val600 BRAF mutations: a non-randomised, open-label phase 2 study.</div><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">249-256</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84875235432&amp;cf=" onclick="newWindow(this.href);return false">Scopus (430)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23414587&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2813%2970024-X&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=249-256&journal=Lancet+Oncol.&author=Ascierto+P.A.&author=Schadendorf+D.&author=Berking+C.&author=Agarwala+S.S.&author=van+Herpen+C.M.L.&author=Queirolo+P.&author=Blank+C.U.&author=Hauschild+A.&author=Beck+J.T.&author=St-Pierre+A.&title=MEK162+for+patients+with+advanced+melanoma+harbouring+NRAS+or+Val600+BRAF+mutations%3A+a+non-randomised%2C+open-label+phase+2+study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Atkins M.B. </li><li class="ref__authors__name">Tannir N.M. </li></ul><div class="ref__title">Current and emerging therapies for first-line treatment of metastatic clear cell renal cell carcinoma.</div><span class="ref__series"><i>Cancer Treat. Rev.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">127-137</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85052524008&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30173085&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=abstract&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=fulltext&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ctrv.2018.07.009&amp;cf=pdf&amp;site=yctrv-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=127-137&journal=Cancer+Treat.+Rev.&author=Atkins+M.B.&author=Tannir+N.M.&title=Current+and+emerging+therapies+for+first-line+treatment+of+metastatic+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Austen M. </li><li class="ref__authors__name">Cerni C. </li><li class="ref__authors__name">Lüscher-Firzlaff J.M. </li><li class="ref__authors__name">Lüscher B. </li></ul><div class="ref__title">YY1 can inhibit c-Myc function through a mechanism requiring DNA binding of YY1 but neither its transactivation domain nor direct interaction with c-Myc.</div><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">511-520</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=9696045&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.onc.1201968&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=511-520&journal=Oncogene&author=Austen+M.&author=Cerni+C.&author=L%C3%BCscher-Firzlaff+J.M.&author=L%C3%BCscher+B.&title=YY1+can+inhibit+c-Myc+function+through+a+mechanism+requiring+DNA+binding+of+YY1+but+neither+its+transactivation+domain+nor+direct+interaction+with+c-Myc">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bailey S.T. </li><li class="ref__authors__name">Smith A.M. </li><li class="ref__authors__name">Kardos J. </li><li class="ref__authors__name">Wobker S.E. </li><li class="ref__authors__name">Wilson H.L. </li><li class="ref__authors__name">Krishnan B. </li><li class="ref__authors__name">Saito R. </li><li class="ref__authors__name">Lee H.J. </li><li class="ref__authors__name">Zhang J. </li><li class="ref__authors__name">Eaton S.C. </li><li>et al.</li></ul><div class="ref__title">MYC activation cooperates with Vhl and Ink4a/Arf loss to induce clear cell renal cell carcinoma.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">15770</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85020380928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28593993&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms15770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15770&journal=Nat.+Commun.&author=Bailey+S.T.&author=Smith+A.M.&author=Kardos+J.&author=Wobker+S.E.&author=Wilson+H.L.&author=Krishnan+B.&author=Saito+R.&author=Lee+H.J.&author=Zhang+J.&author=Eaton+S.C.&title=MYC+activation+cooperates+with+Vhl+and+Ink4a%2FArf+loss+to+induce+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Baylin S.B. </li><li class="ref__authors__name">Jones P.A. </li></ul><div class="ref__title">A decade of exploring the cancer epigenome - biological and translational implications.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">726-734</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80053144962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1619)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21941284&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrc3130&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=726-734&journal=Nat.+Rev.+Cancer&author=Baylin+S.B.&author=Jones+P.A.&title=A+decade+of+exploring+the+cancer+epigenome+-+biological+and+translational+implications">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bellman R. </li></ul><div class="ref__title">On the approximation of curves by line segments using dynamic programming.</div><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benjamini Y. </li><li class="ref__authors__name">Hochberg Y. </li></ul><div class="ref__title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</div><span class="ref__series"><i>J. R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Beuselinck B. </li><li class="ref__authors__name">Job S. </li><li class="ref__authors__name">Becht E. </li><li class="ref__authors__name">Karadimou A. </li><li class="ref__authors__name">Verkarre V. </li><li class="ref__authors__name">Couchy G. </li><li class="ref__authors__name">Giraldo N. </li><li class="ref__authors__name">Rioux-Leclercq N. </li><li class="ref__authors__name">Molinié V. </li><li class="ref__authors__name">Sibony M. </li><li>et al.</li></ul><div class="ref__title">Molecular subtypes of clear cell renal cell carcinoma are associated with sunitinib response in the metastatic setting.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1329-1339</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84927592892&amp;cf=" onclick="newWindow(this.href);return false">Scopus (121)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25583177&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F1078-0432.CCR-14-1128&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1329-1339&journal=Clin.+Cancer+Res.&author=Beuselinck+B.&author=Job+S.&author=Becht+E.&author=Karadimou+A.&author=Verkarre+V.&author=Couchy+G.&author=Giraldo+N.&author=Rioux-Leclercq+N.&author=Molini%C3%A9+V.&author=Sibony+M.&title=Molecular+subtypes+of+clear+cell+renal+cell+carcinoma+are+associated+with+sunitinib+response+in+the+metastatic+setting">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Birdsey G.M. </li><li class="ref__authors__name">Shah A.V. </li><li class="ref__authors__name">Dufton N. </li><li class="ref__authors__name">Reynolds L.E. </li><li class="ref__authors__name">Osuna Almagro L. </li><li class="ref__authors__name">Yang Y. </li><li class="ref__authors__name">Aspalter I.M. </li><li class="ref__authors__name">Khan S.T. </li><li class="ref__authors__name">Mason J.C. </li><li class="ref__authors__name">Dejana E. </li><li>et al.</li></ul><div class="ref__title">The endothelial transcription factor ERG promotes vascular stability and growth through Wnt/β-catenin signaling.</div><span class="ref__series"><i>Dev. Cell.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">82-96</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84922364577&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25584796&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.devcel.2014.11.016&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=82-96&journal=Dev.+Cell&author=Birdsey+G.M.&author=Shah+A.V.&author=Dufton+N.&author=Reynolds+L.E.&author=Osuna+Almagro+L.&author=Yang+Y.&author=Aspalter+I.M.&author=Khan+S.T.&author=Mason+J.C.&author=Dejana+E.&title=The+endothelial+transcription+factor+ERG+promotes+vascular+stability+and+growth+through+Wnt%2F%CE%B2-catenin+signaling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Blanco A.I. </li><li class="ref__authors__name">Teh B.S. </li><li class="ref__authors__name">Amato R.J. </li></ul><div class="ref__title">Role of radiation therapy in the management of renal cell cancer.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">4010-4023</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84055193229&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24213122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fcancers3044010&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=4010-4023&journal=Cancers+%28Basel%29&author=Blanco+A.I.&author=Teh+B.S.&author=Amato+R.J.&title=Role+of+radiation+therapy+in+the+management+of+renal+cell+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bucher N. </li><li class="ref__authors__name">Britten C.D. </li></ul><div class="ref__title">G2 checkpoint abrogation and checkpoint kinase-1 targeting in the treatment of cancer.</div><span class="ref__series"><i>Br. J. Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">523-528</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-38949147038&amp;cf=" onclick="newWindow(this.href);return false">Scopus (227)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18231106&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.bjc.6604208&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2008&pages=523-528&journal=Br.+J.+Cancer&author=Bucher+N.&author=Britten+C.D.&title=G2+checkpoint+abrogation+and+checkpoint+kinase-1+targeting+in+the+treatment+of+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cao Y. </li></ul><div class="ref__title">Multifarious functions of PDGFs and PDGFRs in tumor growth and metastasis.</div><span class="ref__series"><i>Trends Mol. Med.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">460-473</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23773831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.molmed.2013.05.002&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=460-473&journal=Trends+Mol.+Med.&author=Cao+Y.&title=Multifarious+functions+of+PDGFs+and+PDGFRs+in+tumor+growth+and+metastasis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Carles F. </li><li class="ref__authors__name">Bourg S. </li><li class="ref__authors__name">Meyer C. </li><li class="ref__authors__name">Bonnet P. </li></ul><div class="ref__title">PKIDB: A curated, annotated and updated database of protein kinase inhibitors in clinical trials.</div><span class="ref__series"><i>Molecules.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1-18</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045639618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fmolecules23040908&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1-18&journal=Molecules&author=Carles+F.&author=Bourg+S.&author=Meyer+C.&author=Bonnet+P.&title=PKIDB%3A+A+curated%2C+annotated+and+updated+database+of+protein+kinase+inhibitors+in+clinical+trials">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Helman E. </li><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Zack T. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Onofrio R.C. </li><li class="ref__authors__name">Winckler W. </li><li class="ref__authors__name">Weir B.A. </li><li>et al.</li></ul><div class="ref__title">Absolute quantification of somatic DNA alterations in human cancer.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">413-421</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84860782006&amp;cf=" onclick="newWindow(this.href);return false">Scopus (809)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22544022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnbt.2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=413-421&journal=Nat.+Biotechnol.&author=Carter+S.L.&author=Cibulskis+K.&author=Helman+E.&author=McKenna+A.&author=Shen+H.&author=Zack+T.&author=Laird+P.W.&author=Onofrio+R.C.&author=Winckler+W.&author=Weir+B.A.&title=Absolute+quantification+of+somatic+DNA+alterations+in+human+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen D.S. </li><li class="ref__authors__name">Mellman I. </li></ul><div class="ref__title">Elements of cancer immunity and the cancer-immune set point.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">541</span></span>: <span class="ref__seriesPages">321-330</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016548232&amp;cf=" onclick="newWindow(this.href);return false">Scopus (983)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28102259&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature21349&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=541&publication_year=2017&pages=321-330&journal=Nature&author=Chen+D.S.&author=Mellman+I.&title=Elements+of+cancer+immunity+and+the+cancer-immune+set+point">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen J. </li><li class="ref__authors__name">Feng Y. </li><li class="ref__authors__name">Lu L. </li><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Dai L. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Zhang P. </li></ul><div class="ref__title">Interferon-γ-induced PD-L1 surface expression on human oral squamous carcinoma via PKD2 signal pathway.</div><span class="ref__series"><i>Immunobiology.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">217</span></span>: <span class="ref__seriesPages">385-393</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84858007569&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22204817&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.imbio.2011.10.016&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2012&pages=385-393&journal=Immunobiology&author=Chen+J.&author=Feng+Y.&author=Lu+L.&author=Wang+H.&author=Dai+L.&author=Li+Y.&author=Zhang+P.&title=Interferon-%CE%B3-induced+PD-L1+surface+expression+on+human+oral+squamous+carcinoma+via+PKD2+signal+pathway">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen F. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Şenbabaoğlu Y. </li><li class="ref__authors__name">Ciriello G. </li><li class="ref__authors__name">Yang L. </li><li class="ref__authors__name">Reznik E. </li><li class="ref__authors__name">Shuch B. </li><li class="ref__authors__name">Micevic G. </li><li class="ref__authors__name">De Velasco G. </li><li class="ref__authors__name">Shinbrot E. </li><li>et al.</li></ul><div class="ref__title">Multilevel Genomics-Based Taxonomy of Renal Cell Carcinoma.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2476-2489</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26947078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2016.02.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=2476-2489&journal=Cell+Rep.&author=Chen+F.&author=Zhang+Y.&author=%C5%9Eenbabao%C4%9Flu+Y.&author=Ciriello+G.&author=Yang+L.&author=Reznik+E.&author=Shuch+B.&author=Micevic+G.&author=De+Velasco+G.&author=Shinbrot+E.&title=Multilevel+Genomics-Based+Taxonomy+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Schulz-Trieglaff O. </li><li class="ref__authors__name">Shaw R. </li><li class="ref__authors__name">Barnes B. </li><li class="ref__authors__name">Schlesinger F. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Cox A.J. </li><li class="ref__authors__name">Kruglyak S. </li><li class="ref__authors__name">Saunders C.T. </li></ul><div class="ref__title">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (253)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Diao L. </li><li class="ref__authors__name">Yang Y. </li><li class="ref__authors__name">Yi X. </li><li class="ref__authors__name">Rodriguez B.L. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Villalobos P.A. </li><li class="ref__authors__name">Cascone T. </li><li class="ref__authors__name">Liu X. </li><li class="ref__authors__name">Tan L. </li><li>et al.</li></ul><div class="ref__title">CD38-mediated immunosuppression as a mechanism of tumor cell escape from PD-1/PD-l1 blockade.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1156-1175</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053178618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (75)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30012853&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-17-1033&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1156-1175&journal=Cancer+Discov.&author=Chen+L.&author=Diao+L.&author=Yang+Y.&author=Yi+X.&author=Rodriguez+B.L.&author=Li+Y.&author=Villalobos+P.A.&author=Cascone+T.&author=Liu+X.&author=Tan+L.&title=CD38-mediated+immunosuppression+as+a+mechanism+of+tumor+cell+escape+from+PD-1%2FPD-l1+blockade">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chrzanowska-Wodnicka M. </li></ul><div class="ref__title">Distinct functions for Rap1 signaling in vascular morphogenesis and dysfunction.</div><span class="ref__series"><i>Exp. Cell Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">319</span></span>: <span class="ref__seriesPages">2350-2359</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84883210190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23911990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.yexcr.2013.07.022&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2013&pages=2350-2359&journal=Exp.+Cell+Res.&author=Chrzanowska-Wodnicka+M.&title=Distinct+functions+for+Rap1+signaling+in+vascular+morphogenesis+and+dysfunction">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Jaffe D. </li><li class="ref__authors__name">Sougnez C. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span><div class="ref__controls"><a href="#back-bib195" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1869)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Bocik W. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Schnaubelt M. </li><li class="ref__authors__name">Roberts R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Whiteley G. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">2205-2215</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85046635566&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29718670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.8b00165&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2205-2215&journal=J.%C2%A0Proteome+Res.&author=Clark+D.J.&author=Hu+Y.&author=Bocik+W.&author=Chen+L.&author=Schnaubelt+M.&author=Roberts+R.&author=Shah+P.&author=Whiteley+G.&author=Zhang+H.&title=Evaluation+of+NCI-7+Cell+Line+Panel+as+a+Reference+Material+for+Clinical+Proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Garofano L. </li><li class="ref__authors__name">Cava C. </li><li class="ref__authors__name">Garolini D. </li><li class="ref__authors__name">Sabedot T.S. </li><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Pagnotta S.M. </li><li class="ref__authors__name">Castiglioni I. </li><li>et al.</li></ul><div class="ref__title">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (363)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">Morgan M. </li><li class="ref__authors__name">Gunaratne P.H. </li><li class="ref__authors__name">Wheeler D.A. </li><li class="ref__authors__name">Gibbs R.A. </li><li class="ref__authors__name">Robertson G. </li><li class="ref__authors__name">Chu A. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Signoretti S. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive molecular characterization of clear cell renal cell carcinoma.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">43-49</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879890360&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1589)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23792563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature12222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=43-49&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+molecular+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Davis C.F. </li><li class="ref__authors__name">Ricketts C.J. </li><li class="ref__authors__name">Wang M. </li><li class="ref__authors__name">Yang L. </li><li class="ref__authors__name">Cherniack A.D. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Buhay C. </li><li class="ref__authors__name">Kang H. </li><li class="ref__authors__name">Kim S.C. </li><li class="ref__authors__name">Fahey C.C. </li><li>et al.</li><li>The Cancer Genome Atlas Research Network</li></ul><div class="ref__title">The somatic genomic landscape of chromophobe renal cell carcinoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">319-330</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84908695210&amp;cf=" onclick="newWindow(this.href);return false">Scopus (358)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25155756&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2014.07.014&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=319-330&journal=Cancer+Cell&author=The+Cancer+Genome+Atlas+Research+Network&title=The+somatic+genomic+landscape+of+chromophobe+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Deutsch E.W. </li><li class="ref__authors__name">Mendoza L. </li><li class="ref__authors__name">Shteynberg D. </li><li class="ref__authors__name">Slagel J. </li><li class="ref__authors__name">Sun Z. </li><li class="ref__authors__name">Moritz R.L. </li></ul><div class="ref__title">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics.</div><span class="ref__series"><i>Proteomics Clin. Appl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">745-754</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84948987628&amp;cf=" onclick="newWindow(this.href);return false">Scopus (125)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25631240&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fprca.201400164&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=745-754&journal=Proteomics+Clin.+Appl.&author=Deutsch+E.W.&author=Mendoza+L.&author=Shteynberg+D.&author=Slagel+J.&author=Sun+Z.&author=Moritz+R.L.&title=Trans-Proteomic+Pipeline%2C+a+standardized+data+processing+pipeline+for+large-scale+reproducible+proteomics+informatics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Devlin J.R. </li><li class="ref__authors__name">Hannan K.M. </li><li class="ref__authors__name">Hein N. </li><li class="ref__authors__name">Cullinane C. </li><li class="ref__authors__name">Kusnadi E. </li><li class="ref__authors__name">Ng P.Y. </li><li class="ref__authors__name">George A.J. </li><li class="ref__authors__name">Shortt J. </li><li class="ref__authors__name">Bywater M.J. </li><li class="ref__authors__name">Poortinga G. </li><li>et al.</li></ul><div class="ref__title">Combination therapy targeting ribosome biogenesis and mRNA translation synergistically extends survival in MYC-driven lymphoma.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">59-70</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26490423&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-14-0673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=59-70&journal=Cancer+Discov.&author=Devlin+J.R.&author=Hannan+K.M.&author=Hein+N.&author=Cullinane+C.&author=Kusnadi+E.&author=Ng+P.Y.&author=George+A.J.&author=Shortt+J.&author=Bywater+M.J.&author=Poortinga+G.&title=Combination+therapy+targeting+ribosome+biogenesis+and+mRNA+translation+synergistically+extends+survival+in+MYC-driven+lymphoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Diamond E. </li><li class="ref__authors__name">Molina A.M. </li><li class="ref__authors__name">Carbonaro M. </li><li class="ref__authors__name">Akhtar N.H. </li><li class="ref__authors__name">Giannakakou P. </li><li class="ref__authors__name">Tagawa S.T. </li><li class="ref__authors__name">Nanus D.M. </li></ul><div class="ref__title">Cytotoxic chemotherapy in the treatment of advanced renal cell carcinoma in the era of targeted therapy.</div><span class="ref__series"><i>Crit. Rev. Oncol. Hematol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">96</span></span>: <span class="ref__seriesPages">518-526</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26321263&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.critrevonc.2015.08.007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2015&pages=518-526&journal=Crit.+Rev.+Oncol.+Hematol.&author=Diamond+E.&author=Molina+A.M.&author=Carbonaro+M.&author=Akhtar+N.H.&author=Giannakakou+P.&author=Tagawa+S.T.&author=Nanus+D.M.&title=Cytotoxic+chemotherapy+in+the+treatment+of+advanced+renal+cell+carcinoma+in+the+era+of+targeted+therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dmitriev A.A. </li><li class="ref__authors__name">Rudenko E.E. </li><li class="ref__authors__name">Kudryavtseva A.V. </li><li class="ref__authors__name">Krasnov G.S. </li><li class="ref__authors__name">Gordiyuk V.V. </li><li class="ref__authors__name">Melnikova N.V. </li><li class="ref__authors__name">Stakhovsky E.O. </li><li class="ref__authors__name">Kononenko O.A. </li><li class="ref__authors__name">Pavlova L.S. </li><li class="ref__authors__name">Kondratieva T.T. </li><li>et al.</li></ul><div class="ref__title">Epigenetic alterations of chromosome 3 revealed by NotI-microarrays in clear cell renal cell carcinoma.</div><span class="ref__series"><i>BioMed Res. Int.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">2014</span></span>: <span class="ref__seriesPages">735292</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84903597797&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24977159&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1155%2F2014%2F735292&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=735292&journal=BioMed+Res.+Int.&author=Dmitriev+A.A.&author=Rudenko+E.E.&author=Kudryavtseva+A.V.&author=Krasnov+G.S.&author=Gordiyuk+V.V.&author=Melnikova+N.V.&author=Stakhovsky+E.O.&author=Kononenko+O.A.&author=Pavlova+L.S.&author=Kondratieva+T.T.&title=Epigenetic+alterations+of+chromosome+3+revealed+by+NotI-microarrays+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Edge S. </li><li class="ref__authors__name">Byrd D. </li><li class="ref__authors__name">Compton C. </li><li class="ref__authors__name">Fritz A. </li><li class="ref__authors__name">Greene F. </li><li class="ref__authors__name">Trotti A. </li></ul><div class="ref__title">AJCC cancer staging manual.</div> <span class="refPublisherName">Springer US</span>,
        <span class="refPublisherLoc">New York, NY</span><span class="refDate">2010</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=Edge+S.&author=Byrd+D.&author=Compton+C.&author=Fritz+A.&author=Greene+F.&author=Trotti+A.&title=AJCC+cancer+staging+manual">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Escudier B. </li><li class="ref__authors__name">Eisen T. </li><li class="ref__authors__name">Stadler W.M. </li><li class="ref__authors__name">Szczylik C. </li><li class="ref__authors__name">Oudard S. </li><li class="ref__authors__name">Siebels M. </li><li class="ref__authors__name">Negrier S. </li><li class="ref__authors__name">Chevreau C. </li><li class="ref__authors__name">Solska E. </li><li class="ref__authors__name">Desai A.A. </li><li>et al.</li><li>TARGET Study Group</li></ul><div class="ref__title">Sorafenib in advanced clear-cell renal-cell carcinoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">125-134</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33846148701&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3992)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17215530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa060655&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=125-134&journal=N.%C2%A0Engl.+J.+Med.&author=TARGET+Study+Group&title=Sorafenib+in+advanced+clear-cell+renal-cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Escudier B. </li><li class="ref__authors__name">Pluzanska A. </li><li class="ref__authors__name">Koralewski P. </li><li class="ref__authors__name">Ravaud A. </li><li class="ref__authors__name">Bracarda S. </li><li class="ref__authors__name">Szczylik C. </li><li class="ref__authors__name">Chevreau C. </li><li class="ref__authors__name">Filipek M. </li><li class="ref__authors__name">Melichar B. </li><li class="ref__authors__name">Bajetta E. </li><li>et al.</li><li>AVOREN Trial investigators</li></ul><div class="ref__title">Bevacizumab plus interferon alfa-2a for treatment of metastatic renal cell carcinoma: a randomised, double-blind phase III trial.</div><span class="ref__series"><i>Lancet.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">370</span></span>: <span class="ref__seriesPages">2103-2111</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-37349080670&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1872)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18156031&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2807%2961904-7&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2007&pages=2103-2111&journal=Lancet&author=AVOREN+Trial+investigators&title=Bevacizumab+plus+interferon+alfa-2a+for+treatment+of+metastatic+renal+cell+carcinoma%3A+a+randomised%2C+double-blind+phase+III+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ferguson F.M. </li><li class="ref__authors__name">Gray N.S. </li></ul><div class="ref__title">Kinase inhibitors: the road ahead.</div><span class="ref__series"><i>Nat. Rev. Drug Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">353-377</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045991926&amp;cf=" onclick="newWindow(this.href);return false">Scopus (188)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29545548&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrd.2018.21&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=353-377&journal=Nat.+Rev.+Drug+Discov.&author=Ferguson+F.M.&author=Gray+N.S.&title=Kinase+inhibitors%3A+the+road+ahead">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fisher S. </li><li class="ref__authors__name">Barry A. </li><li class="ref__authors__name">Abreu J. </li><li class="ref__authors__name">Minie B. </li><li class="ref__authors__name">Nolan J. </li><li class="ref__authors__name">Delorey T.M. </li><li class="ref__authors__name">Young G. </li><li class="ref__authors__name">Fennell T.J. </li><li class="ref__authors__name">Allen A. </li><li class="ref__authors__name">Ambrogio L. </li><li>et al.</li></ul><div class="ref__title">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fortin J.P. </li><li class="ref__authors__name">Labbe A. </li><li class="ref__authors__name">Lemire M. </li><li class="ref__authors__name">Zanke B.W. </li><li class="ref__authors__name">Hudson T.J. </li><li class="ref__authors__name">Fertig E.J. </li><li class="ref__authors__name">Greenwood C.M.T. </li><li class="ref__authors__name">Hansen K.D. </li></ul><div class="ref__title">Functional normalization of 450k methylation array data improves replication in large cancer studies.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (232)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.T.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fortin J.P. </li><li class="ref__authors__name">Triche T.J. </li><li class="ref__authors__name">Hansen K.D. </li></ul><div class="ref__title">Preprocessing, normalization and integration of the Illumina HumanMethylationEPIC array with minfi.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">558-560</span><div class="ref__controls"><a href="#back-bib196" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016984334&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28035024&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtw691&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=558-560&journal=Bioinformatics&author=Fortin+J.P.&author=Triche+T.J.&author=Hansen+K.D.&title=Preprocessing%2C+normalization+and+integration+of+the+Illumina+HumanMethylationEPIC+array+with+minfi">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Frew I.J. </li><li class="ref__authors__name">Moch H. </li></ul><div class="ref__title">A clearer view of the molecular complexity of clear cell renal cell carcinoma.</div><span class="ref__series"><i>Annu. Rev. Pathol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">263-289</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84921913287&amp;cf=" onclick="newWindow(this.href);return false">Scopus (97)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25387056&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1146%2Fannurev-pathol-012414-040306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=263-289&journal=Annu.+Rev.+Pathol.&author=Frew+I.J.&author=Moch+H.&title=A+clearer+view+of+the+molecular+complexity+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Garcia-Diaz A. </li><li class="ref__authors__name">Shin D.S. </li><li class="ref__authors__name">Moreno B.H. </li><li class="ref__authors__name">Saco J. </li><li class="ref__authors__name">Escuin-Ordinas H. </li><li class="ref__authors__name">Rodriguez G.A. </li><li class="ref__authors__name">Zaretsky J.M. </li><li class="ref__authors__name">Sun L. </li><li class="ref__authors__name">Hugo W. </li><li class="ref__authors__name">Wang X. </li><li>et al.</li></ul><div class="ref__title">Interferon Receptor Signaling Pathways Regulating PD-L1 and PD-L2 Expression.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">1189-1201</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85019192700&amp;cf=" onclick="newWindow(this.href);return false">Scopus (332)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28494868&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.04.031&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1189-1201&journal=Cell+Rep.&author=Garcia-Diaz+A.&author=Shin+D.S.&author=Moreno+B.H.&author=Saco+J.&author=Escuin-Ordinas+H.&author=Rodriguez+G.A.&author=Zaretsky+J.M.&author=Sun+L.&author=Hugo+W.&author=Wang+X.&title=Interferon+Receptor+Signaling+Pathways+Regulating+PD-L1+and+PD-L2+Expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gascard P. </li><li class="ref__authors__name">Tlsty T.D. </li></ul><div class="ref__title">Carcinoma-associated fibroblasts: orchestrating the composition of malignancy.</div><span class="ref__series"><i>Genes Dev.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">1002-1019</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84965082206&amp;cf=" onclick="newWindow(this.href);return false">Scopus (227)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27151975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgad.279737.116&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=1002-1019&journal=Genes+Dev.&author=Gascard+P.&author=Tlsty+T.D.&title=Carcinoma-associated+fibroblasts%3A+orchestrating+the+composition+of+malignancy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gerlinger M. </li><li class="ref__authors__name">Horswell S. </li><li class="ref__authors__name">Larkin J. </li><li class="ref__authors__name">Rowan A.J. </li><li class="ref__authors__name">Salm M.P. </li><li class="ref__authors__name">Varela I. </li><li class="ref__authors__name">Fisher R. </li><li class="ref__authors__name">McGranahan N. </li><li class="ref__authors__name">Matthews N. </li><li class="ref__authors__name">Santos C.R. </li><li>et al.</li></ul><div class="ref__title">Genomic architecture and evolution of clear cell renal cell carcinomas defined by multiregion sequencing.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">225-233</span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84895876130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (690)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24487277&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.2891&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=225-233&journal=Nat.+Genet.&author=Gerlinger+M.&author=Horswell+S.&author=Larkin+J.&author=Rowan+A.J.&author=Salm+M.P.&author=Varela+I.&author=Fisher+R.&author=McGranahan+N.&author=Matthews+N.&author=Santos+C.R.&title=Genomic+architecture+and+evolution+of+clear+cell+renal+cell+carcinomas+defined+by+multiregion+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gey S. </li><li class="ref__authors__name">Lebarbier E. </li></ul><div class="ref__title">Using CART to Detect Multiple Change Points in the Mean for Large Sample.</div><span class="ref__series"><i>Stat. Syst. Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">12</span></span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&journal=Stat.+Syst.+Biol.&author=Gey+S.&author=Lebarbier+E.&title=Using+CART+to+Detect+Multiple+Change+Points+in+the+Mean+for+Large+Sample">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gordan J.D. </li><li class="ref__authors__name">Bertout J.A. </li><li class="ref__authors__name">Hu C.J. </li><li class="ref__authors__name">Diehl J.A. </li><li class="ref__authors__name">Simon M.C. </li></ul><div class="ref__title">HIF-2α promotes hypoxic cell proliferation by enhancing c-myc transcriptional activity.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">335-347</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34047156190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17418410&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.02.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=335-347&journal=Cancer+Cell&author=Gordan+J.D.&author=Bertout+J.A.&author=Hu+C.J.&author=Diehl+J.A.&author=Simon+M.C.&title=HIF-2%CE%B1+promotes+hypoxic+cell+proliferation+by+enhancing+c-myc+transcriptional+activity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gordan J.D. </li><li class="ref__authors__name">Thompson C.B. </li><li class="ref__authors__name">Simon M.C. </li></ul><div class="ref__title">HIF and c-Myc: sibling rivals for control of cancer cell metabolism and proliferation.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">108-113</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34547580590&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17692803&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2007.07.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=108-113&journal=Cancer+Cell&author=Gordan+J.D.&author=Thompson+C.B.&author=Simon+M.C.&title=HIF+and+c-Myc%3A+sibling+rivals+for+control+of+cancer+cell+metabolism+and+proliferation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gordan J.D. </li><li class="ref__authors__name">Lal P. </li><li class="ref__authors__name">Dondeti V.R. </li><li class="ref__authors__name">Letrero R. </li><li class="ref__authors__name">Parekh K.N. </li><li class="ref__authors__name">Oquendo C.E. </li><li class="ref__authors__name">Greenberg R.A. </li><li class="ref__authors__name">Flaherty K.T. </li><li class="ref__authors__name">Rathmell W.K. </li><li class="ref__authors__name">Keith B. </li><li>et al.</li></ul><div class="ref__title">HIF-α effects on c-Myc distinguish two subtypes of sporadic VHL-deficient clear cell renal carcinoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">435-446</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-56849096837&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19061835&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2008.10.016&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=435-446&journal=Cancer+Cell&author=Gordan+J.D.&author=Lal+P.&author=Dondeti+V.R.&author=Letrero+R.&author=Parekh+K.N.&author=Oquendo+C.E.&author=Greenberg+R.A.&author=Flaherty+K.T.&author=Rathmell+W.K.&author=Keith+B.&title=HIF-%CE%B1+effects+on+c-Myc+distinguish+two+subtypes+of+sporadic+VHL-deficient+clear+cell+renal+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Guo G. </li><li class="ref__authors__name">Gui Y. </li><li class="ref__authors__name">Gao S. </li><li class="ref__authors__name">Tang A. </li><li class="ref__authors__name">Hu X. </li><li class="ref__authors__name">Huang Y. </li><li class="ref__authors__name">Jia W. </li><li class="ref__authors__name">Li Z. </li><li class="ref__authors__name">He M. </li><li class="ref__authors__name">Sun L. </li><li>et al.</li></ul><div class="ref__title">Frequent mutations of genes encoding ubiquitin-mediated proteolysis pathway components in clear cell renal cell carcinoma.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">17-19</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84655176646&amp;cf=" onclick="newWindow(this.href);return false">Scopus (218)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22138691&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.1014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2011&pages=17-19&journal=Nat.+Genet.&author=Guo+G.&author=Gui+Y.&author=Gao+S.&author=Tang+A.&author=Hu+X.&author=Huang+Y.&author=Jia+W.&author=Li+Z.&author=He+M.&author=Sun+L.&title=Frequent+mutations+of+genes+encoding+ubiquitin-mediated+proteolysis+pathway+components+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hakimi A.A. </li><li class="ref__authors__name">Pham C.G. </li><li class="ref__authors__name">Hsieh J.J. </li></ul><div class="ref__title">A clear picture of renal cell carcinoma.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">849-850</span><div class="ref__controls"><a href="#back-bib197" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84881016018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23892664&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fng.2708&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=849-850&journal=Nat.+Genet.&author=Hakimi+A.A.&author=Pham+C.G.&author=Hsieh+J.J.&title=A+clear+picture+of+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hakimi A.A. </li><li class="ref__authors__name">Reznik E. </li><li class="ref__authors__name">Lee C.H. </li><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">Brannon A.R. </li><li class="ref__authors__name">Luna A. </li><li class="ref__authors__name">Aksoy B.A. </li><li class="ref__authors__name">Liu E.M. </li><li class="ref__authors__name">Shen R. </li><li class="ref__authors__name">Lee W. </li><li>et al.</li></ul><div class="ref__title">An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">104-116</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84958749958&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26766592&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=104-116&journal=Cancer+Cell&author=Hakimi+A.A.&author=Reznik+E.&author=Lee+C.H.&author=Creighton+C.J.&author=Brannon+A.R.&author=Luna+A.&author=Aksoy+B.A.&author=Liu+E.M.&author=Shen+R.&author=Lee+W.&title=An+Integrated+Metabolic+Atlas+of+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hänzelmann S. </li><li class="ref__authors__name">Castelo R. </li><li class="ref__authors__name">Guinney J. </li></ul><div class="ref__title">GSVA: gene set variation analysis for microarray and RNA-seq data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (722)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hara T. </li><li class="ref__authors__name">Miyake H. </li><li class="ref__authors__name">Fujisawa M. </li></ul><div class="ref__title">Expression pattern of immune checkpoint-associated molecules in radical nephrectomy specimens as a prognosticator in patients with metastatic renal cell carcinoma treated with tyrosine kinase inhibitors.</div><span class="ref__series"><i>Urol. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">363-369</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85011333471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28169111&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.urolonc.2017.01.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=363-369&journal=Urol.+Oncol.&author=Hara+T.&author=Miyake+H.&author=Fujisawa+M.&title=Expression+pattern+of+immune+checkpoint-associated+molecules+in+radical+nephrectomy+specimens+as+a+prognosticator+in+patients+with+metastatic+renal+cell+carcinoma+treated+with+tyrosine+kinase+inhibitors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hein J.B. </li><li class="ref__authors__name">Nilsson J. </li></ul><div class="ref__title">Interphase APC/C-Cdc20 inhibition by cyclin A2-Cdk2 ensures efficient mitotic entry.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">10975</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960912526&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26960431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms10975&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10975&journal=Nat.+Commun.&author=Hein+J.B.&author=Nilsson+J.&title=Interphase+APC%2FC-Cdc20+inhibition+by+cyclin+A2-Cdk2+ensures+efficient+mitotic+entry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hellström M. </li><li class="ref__authors__name">Phng L.K. </li><li class="ref__authors__name">Hofmann J.J. </li><li class="ref__authors__name">Wallgard E. </li><li class="ref__authors__name">Coultas L. </li><li class="ref__authors__name">Lindblom P. </li><li class="ref__authors__name">Alva J. </li><li class="ref__authors__name">Nilsson A.K. </li><li class="ref__authors__name">Karlsson L. </li><li class="ref__authors__name">Gaiano N. </li><li>et al.</li></ul><div class="ref__title">Dll4 signalling through Notch1 regulates formation of tip cells during angiogenesis.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">445</span></span>: <span class="ref__seriesPages">776-780</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33847046849&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1080)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17259973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature05571&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=445&publication_year=2007&pages=776-780&journal=Nature&author=Hellstr%C3%B6m+M.&author=Phng+L.K.&author=Hofmann+J.J.&author=Wallgard+E.&author=Coultas+L.&author=Lindblom+P.&author=Alva+J.&author=Nilsson+A.K.&author=Karlsson+L.&author=Gaiano+N.&title=Dll4+signalling+through+Notch1+regulates+formation+of+tip+cells+during+angiogenesis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hervouet E. </li><li class="ref__authors__name">Cízková A. </li><li class="ref__authors__name">Demont J. </li><li class="ref__authors__name">Vojtísková A. </li><li class="ref__authors__name">Pecina P. </li><li class="ref__authors__name">Franssen-van Hal N.L.W. </li><li class="ref__authors__name">Keijer J. </li><li class="ref__authors__name">Simonnet H. </li><li class="ref__authors__name">Ivánek R. </li><li class="ref__authors__name">Kmoch S. </li><li>et al.</li></ul><div class="ref__title">HIF and reactive oxygen species regulate oxidative phosphorylation in cancer.</div><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">1528-1537</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-49649107272&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18515279&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fcarcin%2Fbgn125&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=1528-1537&journal=Carcinogenesis&author=Hervouet+E.&author=C%C3%ADzkov%C3%A1+A.&author=Demont+J.&author=Vojt%C3%ADskov%C3%A1+A.&author=Pecina+P.&author=Franssen-van+Hal+N.L.W.&author=Keijer+J.&author=Simonnet+H.&author=Iv%C3%A1nek+R.&author=Kmoch+S.&title=HIF+and+reactive+oxygen+species+regulate+oxidative+phosphorylation+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hogrebe A. </li><li class="ref__authors__name">von Stechow L. </li><li class="ref__authors__name">Bekker-Jensen D.B. </li><li class="ref__authors__name">Weinert B.T. </li><li class="ref__authors__name">Kelstrup C.D. </li><li class="ref__authors__name">Olsen J.V. </li></ul><div class="ref__title">Benchmarking common quantification strategies for large-scale phosphoproteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1045</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044218439&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29535314&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-03309-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1045&journal=Nat.+Commun.&author=Hogrebe+A.&author=von+Stechow+L.&author=Bekker-Jensen+D.B.&author=Weinert+B.T.&author=Kelstrup+C.D.&author=Olsen+J.V.&title=Benchmarking+common+quantification+strategies+for+large-scale+phosphoproteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Skrzypek E. </li></ul><div class="ref__title">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (898)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hsieh J.J. </li><li class="ref__authors__name">Purdue M.P. </li><li class="ref__authors__name">Signoretti S. </li><li class="ref__authors__name">Swanton C. </li><li class="ref__authors__name">Albiges L. </li><li class="ref__authors__name">Schmidinger M. </li><li class="ref__authors__name">Heng D.Y. </li><li class="ref__authors__name">Larkin J. </li><li class="ref__authors__name">Ficarra V. </li></ul><div class="ref__title">Renal cell carcinoma.</div><span class="ref__series"><i>Nat. Rev. Dis. Primers.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">17009</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85014927523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (370)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28276433&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrdp.2017.9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=17009&journal=Nat.+Rev.+Dis.+Primers&author=Hsieh+J.J.&author=Purdue+M.P.&author=Signoretti+S.&author=Swanton+C.&author=Albiges+L.&author=Schmidinger+M.&author=Heng+D.Y.&author=Larkin+J.&author=Ficarra+V.&title=Renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hsieh J.J. </li><li class="ref__authors__name">Le V.H. </li><li class="ref__authors__name">Oyama T. </li><li class="ref__authors__name">Ricketts C.J. </li><li class="ref__authors__name">Ho T.H. </li><li class="ref__authors__name">Cheng E.H. </li></ul><div class="ref__title">Chromosome 3p Loss-Orchestrated VHL, HIF, and Epigenetic Deregulation in Clear Cell Renal Cell Carcinoma.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">36</span> (<span class="refComment">JCO2018792549</span>)</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85058568679&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30372397&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2018.79.2549&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&journal=J.%C2%A0Clin.+Oncol.&author=Hsieh+J.J.&author=Le+V.H.&author=Oyama+T.&author=Ricketts+C.J.&author=Ho+T.H.&author=Cheng+E.H.&title=Chromosome+3p+Loss-Orchestrated+VHL%2C+HIF%2C+and+Epigenetic+Deregulation+in+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hsieh J.J. </li><li class="ref__authors__name">Le V. </li><li class="ref__authors__name">Cao D. </li><li class="ref__authors__name">Cheng E.H. </li><li class="ref__authors__name">Creighton C.J. </li></ul><div class="ref__title">Genomic classifications of renal cell carcinoma: a critical step towards the future application of personalized kidney cancer care with pan-omics precision.</div><span class="ref__series"><i>J. Pathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">244</span></span>: <span class="ref__seriesPages">525-537</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044317167&amp;cf=" onclick="newWindow(this.href);return false">Scopus (36)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29266437&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fpath.5022&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=2018&pages=525-537&journal=J.%C2%A0Pathol.&author=Hsieh+J.J.&author=Le+V.&author=Cao+D.&author=Cheng+E.H.&author=Creighton+C.J.&title=Genomic+classifications+of+renal+cell+carcinoma%3A+a+critical+step+towards+the+future+application+of+personalized+kidney+cancer+care+with+pan-omics+precision">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Ao M. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">OmicsOne: Associate Omics Data with Phenotypes in One-Click.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/756544">https://doi.org/10.1101/756544</a></span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095531613&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F756544&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Hu+Y.&author=Ao+M.&author=Zhang+H.&title=OmicsOne%3A+Associate+Omics+Data+with+Phenotypes+in+One-Click">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huang D. </li><li class="ref__authors__name">Ding Y. </li><li class="ref__authors__name">Luo W.M. </li><li class="ref__authors__name">Bender S. </li><li class="ref__authors__name">Qian C.N. </li><li class="ref__authors__name">Kort E. </li><li class="ref__authors__name">Zhang Z.F. </li><li class="ref__authors__name">VandenBeldt K. </li><li class="ref__authors__name">Duesbery N.S. </li><li class="ref__authors__name">Resau J.H. </li><li class="ref__authors__name">Teh B.T. </li></ul><div class="ref__title">Inhibition of MAPK kinase signaling pathways suppressed renal cell carcinoma growth and angiogenesis in vivo.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">81-88</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-39149099155&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18172299&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-07-5311&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=81-88&journal=Cancer+Res.&author=Huang+D.&author=Ding+Y.&author=Luo+W.M.&author=Bender+S.&author=Qian+C.N.&author=Kort+E.&author=Zhang+Z.F.&author=VandenBeldt+K.&author=Duesbery+N.S.&author=Resau+J.H.&author=Teh+B.T.&title=Inhibition+of+MAPK+kinase+signaling+pathways+suppressed+renal+cell+carcinoma+growth+and+angiogenesis+in%C2%A0vivo">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huang K.L. </li><li class="ref__authors__name">Li S. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Gunawardena H.P. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Tanioka M. </li><li class="ref__authors__name">Usary J. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic integration reveals therapeutic targets in breast cancer xenografts.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">14864</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85016396817&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28348404&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms14864&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14864&journal=Nat.+Commun.&author=Huang+K.L.&author=Li+S.&author=Mertins+P.&author=Cao+S.&author=Gunawardena+H.P.&author=Ruggles+K.V.&author=Mani+D.R.&author=Clauser+K.R.&author=Tanioka+M.&author=Usary+J.&title=Proteogenomic+integration+reveals+therapeutic+targets+in+breast+cancer+xenografts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hudes G. </li><li class="ref__authors__name">Carducci M. </li><li class="ref__authors__name">Tomczak P. </li><li class="ref__authors__name">Dutcher J. </li><li class="ref__authors__name">Figlin R. </li><li class="ref__authors__name">Kapoor A. </li><li class="ref__authors__name">Staroslawska E. </li><li class="ref__authors__name">Sosman J. </li><li class="ref__authors__name">McDermott D. </li><li class="ref__authors__name">Bodrogi I. </li><li>et al.</li><li>Global ARCC Trial</li></ul><div class="ref__title">Temsirolimus, interferon alfa, or both for advanced renal-cell carcinoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">2271-2281</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34249779568&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2992)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17538086&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa066838&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=2271-2281&journal=N.%C2%A0Engl.+J.+Med.&author=Global+ARCC+Trial&title=Temsirolimus%2C+interferon+alfa%2C+or+both+for+advanced+renal-cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ishigami K. </li><li class="ref__authors__name">Leite L.V. </li><li class="ref__authors__name">Pakalniskis M.G. </li><li class="ref__authors__name">Lee D.K. </li><li class="ref__authors__name">Holanda D.G. </li><li class="ref__authors__name">Kuehn D.M. </li></ul><div class="ref__title">Tumor grade of clear cell renal cell carcinoma assessed by contrast-enhanced computed tomography.</div><span class="ref__series"><i>Springerplus.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">694</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925254491&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25806147&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F2193-1801-3-694&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=694&journal=Springerplus&author=Ishigami+K.&author=Leite+L.V.&author=Pakalniskis+M.G.&author=Lee+D.K.&author=Holanda+D.G.&author=Kuehn+D.M.&title=Tumor+grade+of+clear+cell+renal+cell+carcinoma+assessed+by+contrast-enhanced+computed+tomography">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Johnson W.E. </li><li class="ref__authors__name">Li C. </li><li class="ref__authors__name">Rabinovic A. </li></ul><div class="ref__title">Adjusting batch effects in microarray expression data using empirical Bayes methods.</div><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">118-127</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33845432928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2518)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16632515&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbiostatistics%2Fkxj037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=118-127&journal=Biostatistics&author=Johnson+W.E.&author=Li+C.&author=Rabinovic+A.&title=Adjusting+batch+effects+in+microarray+expression+data+using+empirical+Bayes+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Joshi-Tope G. </li><li class="ref__authors__name">Gillespie M. </li><li class="ref__authors__name">Vastrik I. </li><li class="ref__authors__name">D’Eustachio P. </li><li class="ref__authors__name">Schmidt E. </li><li class="ref__authors__name">de Bono B. </li><li class="ref__authors__name">Jassal B. </li><li class="ref__authors__name">Gopinath G.R. </li><li class="ref__authors__name">Wu G.R. </li><li class="ref__authors__name">Matthews L. </li><li>et al.</li></ul><div class="ref__title">Reactome: a knowledgebase of biological pathways.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">D428-D432</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13444259844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (846)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15608231&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgki072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D428-D432&journal=Nucleic+Acids+Res.&author=Joshi-Tope+G.&author=Gillespie+M.&author=Vastrik+I.&author=D%E2%80%99Eustachio+P.&author=Schmidt+E.&author=de+Bono+B.&author=Jassal+B.&author=Gopinath+G.R.&author=Wu+G.R.&author=Matthews+L.&title=Reactome%3A+a+knowledgebase+of+biological+pathways">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jurtz V. </li><li class="ref__authors__name">Paul S. </li><li class="ref__authors__name">Andreatta M. </li><li class="ref__authors__name">Marcatili P. </li><li class="ref__authors__name">Peters B. </li><li class="ref__authors__name">Nielsen M. </li></ul><div class="ref__title">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.</div><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (216)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kalayci S. </li><li class="ref__authors__name">Gümüş Z.H. </li></ul><div class="ref__title">Exploring Biological Networks in 3D, Stereoscopic 3D, and Immersive 3D with iCAVE.</div><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">61</span></span>: <span class="ref__seriesPages">8.27.1-8.27.26</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050801906&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fcpbi.47&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8.27.1-8.27.26&journal=Curr.+Protoc.+Bioinformatics&author=Kalayci+S.&author=G%C3%BCm%C3%BC%C5%9F+Z.H.&title=Exploring+Biological+Networks+in+3D%2C+Stereoscopic+3D%2C+and+Immersive+3D+with+iCAVE">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kanehisa M. </li><li class="ref__authors__name">Goto S. </li></ul><div class="ref__title">KEGG: Kyoto Encyclopedia of Genes and Genomes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">27-30</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10592173&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2F28.1.27&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=27-30&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Goto+S.&title=KEGG%3A+Kyoto+Encyclopedia+of+Genes+and+Genomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kapur P. </li><li class="ref__authors__name">Peña-Llopis S. </li><li class="ref__authors__name">Christie A. </li><li class="ref__authors__name">Zhrebker L. </li><li class="ref__authors__name">Pavía-Jiménez A. </li><li class="ref__authors__name">Rathmell W.K. </li><li class="ref__authors__name">Xie X.J. </li><li class="ref__authors__name">Brugarolas J. </li></ul><div class="ref__title">Effects on survival of BAP1 and PBRM1 mutations in sporadic clear-cell renal-cell carcinoma: A retrospective analysis with independent validation.</div><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">159-167</span><div class="ref__controls"><a href="#back-bib198" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84873081430&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23333114&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1470-2045%2812%2970584-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=159-167&journal=Lancet+Oncol.&author=Kapur+P.&author=Pe%C3%B1a-Llopis+S.&author=Christie+A.&author=Zhrebker+L.&author=Pav%C3%ADa-Jim%C3%A9nez+A.&author=Rathmell+W.K.&author=Xie+X.J.&author=Brugarolas+J.&title=Effects+on+survival+of+BAP1+and+PBRM1+mutations+in+sporadic+clear-cell+renal-cell+carcinoma%3A+A+retrospective+analysis+with+independent+validation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Keller A. </li><li class="ref__authors__name">Nesvizhskii A.I. </li><li class="ref__authors__name">Kolker E. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3457)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal.+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kessner D. </li><li class="ref__authors__name">Chambers M. </li><li class="ref__authors__name">Burke R. </li><li class="ref__authors__name">Agus D. </li><li class="ref__authors__name">Mallick P. </li></ul><div class="ref__title">ProteoWizard: Open source software for rapid proteomics tools development.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span><div class="ref__controls"><a href="#back-bib199" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (870)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+Open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib200" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (390)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Scheffler K. </li><li class="ref__authors__name">Halpern A.L. </li><li class="ref__authors__name">Bekritsky M.A. </li><li class="ref__authors__name">Noh E. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Kim Y. </li><li class="ref__authors__name">Beyter D. </li><li class="ref__authors__name">Krusche P. </li><li>et al.</li></ul><div class="ref__title">Strelka2: fast and accurate calling of germline and somatic variants.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span><div class="ref__controls"><a href="#back-bib201" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Koboldt D.C. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Larson D.E. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Lin L. </li><li class="ref__authors__name">Miller C.A. </li><li class="ref__authors__name">Mardis E.R. </li><li class="ref__authors__name">Ding L. </li><li class="ref__authors__name">Wilson R.K. </li></ul><div class="ref__title">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span><div class="ref__controls"><a href="#back-bib202" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1937)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+Somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Komili S. </li><li class="ref__authors__name">Silver P.A. </li></ul><div class="ref__title">Coupling and coordination in gene expression processes: a systems biology view.</div><span class="ref__series"><i>Nat. Rev. Genet.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">38-48</span><div class="ref__controls"><a href="#back-bib79" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-37249034083&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18071322&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrg2223&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=38-48&journal=Nat.+Rev.+Genet.&author=Komili+S.&author=Silver+P.A.&title=Coupling+and+coordination+in+gene+expression+processes%3A+a+systems+biology+view">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kong A.T. </li><li class="ref__authors__name">Leprevost F.V. </li><li class="ref__authors__name">Avtonomov D.M. </li><li class="ref__authors__name">Mellacheruvu D. </li><li class="ref__authors__name">Nesvizhskii A.I. </li></ul><div class="ref__title">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span><div class="ref__controls"><a href="#back-bib80" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Krieg M. </li><li class="ref__authors__name">Haas R. </li><li class="ref__authors__name">Brauch H. </li><li class="ref__authors__name">Acker T. </li><li class="ref__authors__name">Flamme I. </li><li class="ref__authors__name">Plate K.H. </li></ul><div class="ref__title">Up-regulation of hypoxia-inducible factors HIF-1α and HIF-2α under normoxic conditions in renal carcinoma cells by von Hippel-Lindau tumor suppressor gene loss of function.</div><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2000; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">5435-5443</span><div class="ref__controls"><a href="#back-bib81" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0034676331&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=11114720&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fsj.onc.1203938&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=5435-5443&journal=Oncogene&author=Krieg+M.&author=Haas+R.&author=Brauch+H.&author=Acker+T.&author=Flamme+I.&author=Plate+K.H.&title=Up-regulation+of+hypoxia-inducible+factors+HIF-1%CE%B1+and+HIF-2%CE%B1+under+normoxic+conditions+in+renal+carcinoma+cells+by+von+Hippel-Lindau+tumor+suppressor+gene+loss+of+function">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kwiatkowski D.J. </li><li class="ref__authors__name">Choueiri T.K. </li><li class="ref__authors__name">Fay A.P. </li><li class="ref__authors__name">Rini B.I. </li><li class="ref__authors__name">Thorner A.R. </li><li class="ref__authors__name">de Velasco G. </li><li class="ref__authors__name">Tyburczy M.E. </li><li class="ref__authors__name">Hamieh L. </li><li class="ref__authors__name">Albiges L. </li><li class="ref__authors__name">Agarwal N. </li><li>et al.</li></ul><div class="ref__title">Mutations in TSC1, TSC2, and MTOR are associated with response to rapalogs in patients with metastatic renal cell carcinoma.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">2445-2452</span><div class="ref__controls"><a href="#back-bib82" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84968902640&amp;cf=" onclick="newWindow(this.href);return false">Scopus (105)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26831717&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F1078-0432.CCR-15-2631&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=2445-2452&journal=Clin.+Cancer+Res.&author=Kwiatkowski+D.J.&author=Choueiri+T.K.&author=Fay+A.P.&author=Rini+B.I.&author=Thorner+A.R.&author=de+Velasco+G.&author=Tyburczy+M.E.&author=Hamieh+L.&author=Albiges+L.&author=Agarwal+N.&title=Mutations+in+TSC1%2C+TSC2%2C+and+MTOR+are+associated+with+response+to+rapalogs+in+patients+with+metastatic+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lai Z. </li><li class="ref__authors__name">Markovets A. </li><li class="ref__authors__name">Ahdesmaki M. </li><li class="ref__authors__name">Chapman B. </li><li class="ref__authors__name">Hofmann O. </li><li class="ref__authors__name">Mcewen R. </li><li class="ref__authors__name">Johnson J. </li><li class="ref__authors__name">Dougherty B. </li><li class="ref__authors__name">Barrett J.C. </li><li class="ref__authors__name">Dry J.R. </li></ul><div class="ref__title">VarDict: A novel and versatile variant caller for next-generation sequencing in cancer research.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">1-11</span><div class="ref__controls"><a href="#back-bib203" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84976448371&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26621913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkw227&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1-11&journal=Nucleic+Acids+Res.&author=Lai+Z.&author=Markovets+A.&author=Ahdesmaki+M.&author=Chapman+B.&author=Hofmann+O.&author=Mcewen+R.&author=Johnson+J.&author=Dougherty+B.&author=Barrett+J.C.&author=Dry+J.R.&title=VarDict%3A+A+novel+and+versatile+variant+caller+for+next-generation+sequencing+in+cancer+research">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lakins M.A. </li><li class="ref__authors__name">Ghorani E. </li><li class="ref__authors__name">Munir H. </li><li class="ref__authors__name">Martins C.P. </li><li class="ref__authors__name">Shields J.D. </li></ul><div class="ref__title">Cancer-associated fibroblasts induce antigen-specific deletion of CD8 <sup>+</sup> T Cells to protect tumour cells.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">948</span><div class="ref__controls"><a href="#back-bib83" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042940461&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29507342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-03347-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=948&journal=Nat.+Commun.&author=Lakins+M.A.&author=Ghorani+E.&author=Munir+H.&author=Martins+C.P.&author=Shields+J.D.&title=Cancer-associated+fibroblasts+induce+antigen-specific+deletion+of+CD8+%2B+T+Cells+to+protect+tumour+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Layer R.M. </li><li class="ref__authors__name">Chiang C. </li><li class="ref__authors__name">Quinlan A.R. </li><li class="ref__authors__name">Hall I.M. </li></ul><div class="ref__title">LUMPY: a probabilistic framework for structural variant discovery.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">R84</span><div class="ref__controls"><a href="#back-bib84" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84912096454&amp;cf=" onclick="newWindow(this.href);return false">Scopus (388)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24970577&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2014-15-6-r84&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=R84&journal=Genome+Biol.&author=Layer+R.M.&author=Chiang+C.&author=Quinlan+A.R.&author=Hall+I.M.&title=LUMPY%3A+a+probabilistic+framework+for+structural+variant+discovery">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lebarbier E. </li></ul><div class="ref__title">Detecting multiple change-points in the mean of Gaussian process by model selection.</div><span class="ref__series"><i>Signal Processing.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span><div class="ref__controls"><a href="#back-bib85" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (103)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Processing&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lehto M. </li><li class="ref__authors__name">Olkkonen V.M. </li></ul><div class="ref__title">The OSBP-related proteins: a novel protein family involved in vesicle transport, cellular lipid metabolism, and cell signalling.</div><span class="ref__series"><i>Biochim. Biophys. Acta.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">1631</span></span>: <span class="ref__seriesPages">1-11</span><div class="ref__controls"><a href="#back-bib86" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0037455984&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12573443&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS1388-1981%2802%2900364-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1631&publication_year=2003&pages=1-11&journal=Biochim.+Biophys.+Acta&author=Lehto+M.&author=Olkkonen+V.M.&title=The+OSBP-related+proteins%3A+a+novel+protein+family+involved+in+vesicle+transport%2C+cellular+lipid+metabolism%2C+and+cell+signalling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li K. </li><li class="ref__authors__name">Vaudel M. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Ren Y. </li><li class="ref__authors__name">Wen B. </li></ul><div class="ref__title">PDV: an integrative proteomics data viewer.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span><div class="ref__controls"><a href="#back-bib87" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liao L. </li><li class="ref__authors__name">Testa J.R. </li><li class="ref__authors__name">Yang H. </li></ul><div class="ref__title">The roles of chromatin-remodelers and epigenetic modifiers in kidney cancer.</div><span class="ref__series"><i>Cancer Genet.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">208</span></span>: <span class="ref__seriesPages">206-214</span><div class="ref__controls"><a href="#back-bib88" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25873528&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=abstract&amp;site=cgen-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=fulltext&amp;site=cgen-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cancergen.2015.02.008&amp;cf=pdf&amp;site=cgen-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2015&pages=206-214&journal=Cancer+Genet.&author=Liao+L.&author=Testa+J.R.&author=Yang+H.&title=The+roles+of+chromatin-remodelers+and+epigenetic+modifiers+in+kidney+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Pinchback R. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Molecular signatures database (MSigDB) 3.0.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">1739-1740</span><div class="ref__controls"><a href="#back-bib89" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79958135926&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1170)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21546393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtr260&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=1739-1740&journal=Bioinformatics&author=Liberzon+A.&author=Subramanian+A.&author=Pinchback+R.&author=Thorvaldsd%C3%B3ttir+H.&author=Tamayo+P.&author=Mesirov+J.P.&title=Molecular+signatures+database+%28MSigDB%29+3.0">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lih T.M. </li><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">Omic-Sig: Utilizing Omics Data to Explore and Visualize Kinase-Substrate Interactions.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/746123">https://doi.org/10.1101/746123</a></span><div class="ref__controls"><a href="#back-bib90" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095527091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F746123&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Lih+T.M.&author=Clark+D.J.&author=Zhang+H.&title=Omic-Sig%3A+Utilizing+Omics+Data+to+Explore+and+Visualize+Kinase-Substrate+Interactions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liluashvili V. </li><li class="ref__authors__name">Kalayci S. </li><li class="ref__authors__name">Fluder E. </li><li class="ref__authors__name">Wilson M. </li><li class="ref__authors__name">Gabow A. </li><li class="ref__authors__name">Gümüs Z.H. </li></ul><div class="ref__title">iCAVE: an open source tool for visualizing biomolecular networks in 3D, stereoscopic 3D and immersive 3D.</div><span class="ref__series"><i>Gigascience.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1-13</span><div class="ref__controls"><a href="#back-bib91" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042149548&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28814063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fgigascience%2Fgix054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1-13&journal=Gigascience&author=Liluashvili+V.&author=Kalayci+S.&author=Fluder+E.&author=Wilson+M.&author=Gabow+A.&author=G%C3%BCm%C3%BCs+Z.H.&title=iCAVE%3A+an+open+source+tool+for+visualizing+biomolecular+networks+in+3D%2C+stereoscopic+3D+and+immersive+3D">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lim S.O. </li><li class="ref__authors__name">Li C.W. </li><li class="ref__authors__name">Xia W. </li><li class="ref__authors__name">Lee H.H. </li><li class="ref__authors__name">Chang S.S. </li><li class="ref__authors__name">Shen J. </li><li class="ref__authors__name">Hsu J.L. </li><li class="ref__authors__name">Raftery D. </li><li class="ref__authors__name">Djukovic D. </li><li class="ref__authors__name">Gu H. </li><li>et al.</li></ul><div class="ref__title">EGFR signaling enhances aerobic glycolysis in triple-negative breast cancer cells to promote tumor growth and immune escape.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">76</span></span>: <span class="ref__seriesPages">1284-1296</span><div class="ref__controls"><a href="#back-bib92" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84961825288&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26759242&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-15-2478&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1284-1296&journal=Cancer+Res.&author=Lim+S.O.&author=Li+C.W.&author=Xia+W.&author=Lee+H.H.&author=Chang+S.S.&author=Shen+J.&author=Hsu+J.L.&author=Raftery+D.&author=Djukovic+D.&author=Gu+H.&title=EGFR+signaling+enhances+aerobic+glycolysis+in+triple-negative+breast+cancer+cells+to+promote+tumor+growth+and+immune+escape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lindgren D. </li><li class="ref__authors__name">Eriksson P. </li><li class="ref__authors__name">Krawczyk K. </li><li class="ref__authors__name">Nilsson H. </li><li class="ref__authors__name">Hansson J. </li><li class="ref__authors__name">Veerla S. </li><li class="ref__authors__name">Sjölund J. </li><li class="ref__authors__name">Höglund M. </li><li class="ref__authors__name">Johansson M.E. </li><li class="ref__authors__name">Axelson H. </li></ul><div class="ref__title">Cell-Type-Specific Gene Programs of the Normal Human Nephron Define Kidney Cancer Subtypes.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1476-1489</span><div class="ref__controls"><a href="#back-bib93" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85026887625&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28793269&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2017.07.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2017&pages=1476-1489&journal=Cell+Rep.&author=Lindgren+D.&author=Eriksson+P.&author=Krawczyk+K.&author=Nilsson+H.&author=Hansson+J.&author=Veerla+S.&author=Sj%C3%B6lund+J.&author=H%C3%B6glund+M.&author=Johansson+M.E.&author=Axelson+H.&title=Cell-Type-Specific+Gene+Programs+of+the+Normal+Human+Nephron+Define+Kidney+Cancer+Subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Linehan W.M. </li><li class="ref__authors__name">Spellman P.T. </li><li class="ref__authors__name">Ricketts C.J. </li><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">Fei S.S. </li><li class="ref__authors__name">Davis C. </li><li class="ref__authors__name">Wheeler D.A. </li><li class="ref__authors__name">Murray B.A. </li><li class="ref__authors__name">Schmidt L. </li><li class="ref__authors__name">Vocke C.D. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive Molecular Characterization of Papillary Renal-Cell Carcinoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">374</span></span>: <span class="ref__seriesPages">135-145</span><div class="ref__controls"><a href="#back-bib94" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84954286974&amp;cf=" onclick="newWindow(this.href);return false">Scopus (480)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26536169&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa1505917&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=135-145&journal=N.%C2%A0Engl.+J.+Med.&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+Molecular+Characterization+of+Papillary+Renal-Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Luo W. </li><li class="ref__authors__name">Chen I. </li><li class="ref__authors__name">Chen Y. </li><li class="ref__authors__name">Alkam D. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Semenza G.L. </li></ul><div class="ref__title">PRDX2 and PRDX4 are negative regulators of hypoxia-inducible factors under conditions of prolonged hypoxia.</div><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">6379-6397</span><div class="ref__controls"><a href="#back-bib95" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84957675600&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26837221&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.18632%2Foncotarget.6395&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=6379-6397&journal=Oncotarget&author=Luo+W.&author=Chen+I.&author=Chen+Y.&author=Alkam+D.&author=Wang+Y.&author=Semenza+G.L.&title=PRDX2+and+PRDX4+are+negative+regulators+of+hypoxia-inducible+factors+under+conditions+of+prolonged+hypoxia">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lusis E.A. </li><li class="ref__authors__name">Watson M.A. </li><li class="ref__authors__name">Chicoine M.R. </li><li class="ref__authors__name">Lyman M. </li><li class="ref__authors__name">Roerig P. </li><li class="ref__authors__name">Reifenberger G. </li><li class="ref__authors__name">Gutmann D.H. </li><li class="ref__authors__name">Perry A. </li></ul><div class="ref__title">Integrative genomic analysis identifies NDRG2 as a candidate tumor suppressor gene frequently inactivated in clinically aggressive meningioma.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">65</span></span>: <span class="ref__seriesPages">7121-7126</span><div class="ref__controls"><a href="#back-bib96" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-23844451672&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16103061&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-05-0043&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=7121-7126&journal=Cancer+Res.&author=Lusis+E.A.&author=Watson+M.A.&author=Chicoine+M.R.&author=Lyman+M.&author=Roerig+P.&author=Reifenberger+G.&author=Gutmann+D.H.&author=Perry+A.&title=Integrative+genomic+analysis+identifies+NDRG2+as+a+candidate+tumor+suppressor+gene+frequently+inactivated+in+clinically+aggressive+meningioma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Maksimovic J. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Oshlack A. </li></ul><div class="ref__title">A cross-package Bioconductor workflow for analysing methylation array data.</div><span class="ref__series"><i>F1000Research.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1-53</span><div class="ref__controls"><a href="#back-bib204" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.12688%2Ff1000research.8839.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1-53&journal=F1000Research&author=Maksimovic+J.&author=Phipson+B.&author=Oshlack+A.&title=A+cross-package+Bioconductor+workflow+for+analysing+methylation+array+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Burzykowski T. </li><li class="ref__authors__name">Poisson L. </li><li class="ref__authors__name">Weinstein J.N. </li><li class="ref__authors__name">Kaminska B. </li><li class="ref__authors__name">Huelsken J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Gevaert O. </li><li>et al.</li></ul><div class="ref__title">Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation Article Machine Learning Identifies Stemness Features Associated with Oncogenic Dedifferentiation.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354</span><div class="ref__controls"><a href="#back-bib205" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kaminska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation+Article+Machine+Learning+Identifies+Stemness+Features+Associated+with+Oncogenic+Dedifferentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Maroto P. </li><li class="ref__authors__name">Esteban E. </li><li class="ref__authors__name">Parra E.F. </li><li class="ref__authors__name">Mendez-Vidal M.J. </li><li class="ref__authors__name">Domenech M. </li><li class="ref__authors__name">Pérez-Valderrama B. </li><li class="ref__authors__name">Calderero V. </li><li class="ref__authors__name">Pérez-Gracia J.L. </li><li class="ref__authors__name">Grande E. </li><li class="ref__authors__name">Algaba F. </li></ul><div class="ref__title">HIF pathway and c-Myc as biomarkers for response to sunitinib in metastatic clear-cell renal cell carcinoma.</div><span class="ref__series"><i>OncoTargets Ther.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">4635-4643</span><div class="ref__controls"><a href="#back-bib97" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85030245220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29033582&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.2147%2FOTT.S137677&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=4635-4643&journal=OncoTargets+Ther.&author=Maroto+P.&author=Esteban+E.&author=Parra+E.F.&author=Mendez-Vidal+M.J.&author=Domenech+M.&author=P%C3%A9rez-Valderrama+B.&author=Calderero+V.&author=P%C3%A9rez-Gracia+J.L.&author=Grande+E.&author=Algaba+F.&title=HIF+pathway+and+c-Myc+as+biomarkers+for+response+to+sunitinib+in+metastatic+clear-cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Matheson C.J. </li><li class="ref__authors__name">Backos D.S. </li><li class="ref__authors__name">Reigan P. </li></ul><div class="ref__title">Targeting WEE1 Kinase in Cancer.</div><span class="ref__series"><i>Trends Pharmacol. Sci.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">872-881</span><div class="ref__controls"><a href="#back-bib98" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84978879508&amp;cf=" onclick="newWindow(this.href);return false">Scopus (97)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27427153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.tips.2016.06.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=872-881&journal=Trends+Pharmacol.+Sci.&author=Matheson+C.J.&author=Backos+D.S.&author=Reigan+P.&title=Targeting+WEE1+Kinase+in+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Matsushita H. </li><li class="ref__authors__name">Sato Y. </li><li class="ref__authors__name">Karasaki T. </li><li class="ref__authors__name">Nakagawa T. </li><li class="ref__authors__name">Kume H. </li><li class="ref__authors__name">Ogawa S. </li><li class="ref__authors__name">Homma Y. </li><li class="ref__authors__name">Kakimi K. </li></ul><div class="ref__title">Neoantigen Load, Antigen Presentation Machinery, and Immune Signatures Determine Prognosis in Clear Cell Renal Cell Carcinoma.</div><span class="ref__series"><i>Cancer Immunol. Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">463-471</span><div class="ref__controls"><a href="#back-bib99" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26980598&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2326-6066.CIR-15-0225&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=463-471&journal=Cancer+Immunol.+Res.&author=Matsushita+H.&author=Sato+Y.&author=Karasaki+T.&author=Nakagawa+T.&author=Kume+H.&author=Ogawa+S.&author=Homma+Y.&author=Kakimi+K.&title=Neoantigen+Load%2C+Antigen+Presentation+Machinery%2C+and+Immune+Signatures+Determine+Prognosis+in+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McAlister G.C. </li><li class="ref__authors__name">Huttlin E.L. </li><li class="ref__authors__name">Haas W. </li><li class="ref__authors__name">Ting L. </li><li class="ref__authors__name">Jedrychowski M.P. </li><li class="ref__authors__name">Rogers J.C. </li><li class="ref__authors__name">Kuhn K. </li><li class="ref__authors__name">Pike I. </li><li class="ref__authors__name">Grothe R.A. </li><li class="ref__authors__name">Blethrow J.D. </li><li class="ref__authors__name">Gygi S.P. </li></ul><div class="ref__title">Increasing the multiplexing capacity of TMTs using reporter ion isotopologues with isobaric masses.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">84</span></span>: <span class="ref__seriesPages">7469-7478</span><div class="ref__controls"><a href="#back-bib100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865718193&amp;cf=" onclick="newWindow(this.href);return false">Scopus (284)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22880955&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac301572t&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=7469-7478&journal=Anal.+Chem.&author=McAlister+G.C.&author=Huttlin+E.L.&author=Haas+W.&author=Ting+L.&author=Jedrychowski+M.P.&author=Rogers+J.C.&author=Kuhn+K.&author=Pike+I.&author=Grothe+R.A.&author=Blethrow+J.D.&author=Gygi+S.P.&title=Increasing+the+multiplexing+capacity+of+TMTs+using+reporter+ion+isotopologues+with+isobaric+masses">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McDermott D.F. </li><li class="ref__authors__name">Huseni M.A. </li><li class="ref__authors__name">Atkins M.B. </li><li class="ref__authors__name">Motzer R.J. </li><li class="ref__authors__name">Rini B.I. </li><li class="ref__authors__name">Escudier B. </li><li class="ref__authors__name">Fong L. </li><li class="ref__authors__name">Joseph R.W. </li><li class="ref__authors__name">Pal S.K. </li><li class="ref__authors__name">Reeves J.A. </li><li>et al.</li></ul><div class="ref__title">Clinical activity and molecular correlates of response to atezolizumab alone or in combination with bevacizumab versus sunitinib in renal cell carcinoma.</div><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">749-757</span><div class="ref__controls"><a href="#back-bib101" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048001150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (210)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29867230&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41591-018-0053-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=749-757&journal=Nat.+Med.&author=McDermott+D.F.&author=Huseni+M.A.&author=Atkins+M.B.&author=Motzer+R.J.&author=Rini+B.I.&author=Escudier+B.&author=Fong+L.&author=Joseph+R.W.&author=Pal+S.K.&author=Reeves+J.A.&title=Clinical+activity+and+molecular+correlates+of+response+to+atezolizumab+alone+or+in+combination+with+bevacizumab+versus+sunitinib+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mehra R. </li><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Cieslik M. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Su F. </li><li class="ref__authors__name">Shukla S. </li><li class="ref__authors__name">Udager A.M. </li><li class="ref__authors__name">Wang R. </li><li class="ref__authors__name">Pan J. </li><li class="ref__authors__name">Kasaian K. </li><li>et al.</li></ul><div class="ref__title">Biallelic Alteration and Dysregulation of the Hippo Pathway in Mucinous Tubular and Spindle Cell Carcinoma of the Kidney.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">1258-1266</span><div class="ref__controls"><a href="#back-bib206" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27604489&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-16-0267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=1258-1266&journal=Cancer+Discov.&author=Mehra+R.&author=Vats+P.&author=Cieslik+M.&author=Cao+X.&author=Su+F.&author=Shukla+S.&author=Udager+A.M.&author=Wang+R.&author=Pan+J.&author=Kasaian+K.&title=Biallelic+Alteration+and+Dysregulation+of+the+Hippo+Pathway+in+Mucinous+Tubular+and+Spindle+Cell+Carcinoma+of+the+Kidney">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mehra R. </li><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Su F. </li><li class="ref__authors__name">Lee N.D. </li><li class="ref__authors__name">Lonigro R. </li><li class="ref__authors__name">Premkumar K. </li><li class="ref__authors__name">Trpkov K. </li><li class="ref__authors__name">McKenney J.K. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Chinnaiyan A.M. </li></ul><div class="ref__title">Somatic Bi-allelic Loss of TSC Genes in Eosinophilic Solid and Cystic Renal Cell Carcinoma.</div><span class="ref__series"><i>Eur. Urol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">483-486</span><div class="ref__controls"><a href="#back-bib102" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85048882704&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29941307&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=abstract&amp;site=eururo-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=fulltext&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.eururo.2018.06.007&amp;cf=pdf&amp;site=eururo-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2018&pages=483-486&journal=Eur.+Urol.&author=Mehra+R.&author=Vats+P.&author=Cao+X.&author=Su+F.&author=Lee+N.D.&author=Lonigro+R.&author=Premkumar+K.&author=Trpkov+K.&author=McKenney+J.K.&author=Dhanasekaran+S.M.&author=Chinnaiyan+A.M.&title=Somatic+Bi-allelic+Loss+of+TSC+Genes+in+Eosinophilic+Solid+and+Cystic+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib103" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (959)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Patel J. </li><li class="ref__authors__name">Udeshi N.D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Burgess M.W. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Jaffe J.D. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Integrated proteomic analysis of post-translational modifications by serial enrichment.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">634-637</span><div class="ref__controls"><a href="#back-bib104" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84879637572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (300)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23749302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.2518&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=634-637&journal=Nat.+Methods&author=Mertins+P.&author=Qiao+J.W.&author=Patel+J.&author=Udeshi+N.D.&author=Clauser+K.R.&author=Mani+D.R.&author=Burgess+M.W.&author=Gillette+M.A.&author=Jaffe+J.D.&author=Carr+S.A.&title=Integrated+proteomic+analysis+of+post-translational+modifications+by+serial+enrichment">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (533)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Tang L.C. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Clauss T.R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Gillette M.A. </li><li>et al.</li></ul><div class="ref__title">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.</div><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span><div class="ref__controls"><a href="#back-bib106" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (49)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mgrditchian T. </li><li class="ref__authors__name">Arakelian T. </li><li class="ref__authors__name">Paggetti J. </li><li class="ref__authors__name">Viry E. </li><li class="ref__authors__name">Al-Absi A. </li><li class="ref__authors__name">Medves S. </li><li class="ref__authors__name">Moussay E. </li><li class="ref__authors__name">Berchem G. </li><li class="ref__authors__name">Thomas C. </li><li class="ref__authors__name">Janji B. </li></ul><div class="ref__title">Hypoxia: A Key Feature of the Tumor Microenvironment Triggers Several Mechanisms of Evasion from Natural Killer and Cytotoxic T Lymphocytes Surveillance.</div><span class="ref__series"><i>J. Immun. Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">7</span><div class="ref__controls"><a href="#back-bib107" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=7&journal=J.%C2%A0Immun.+Res.&author=Mgrditchian+T.&author=Arakelian+T.&author=Paggetti+J.&author=Viry+E.&author=Al-Absi+A.&author=Medves+S.&author=Moussay+E.&author=Berchem+G.&author=Thomas+C.&author=Janji+B.&title=Hypoxia%3A+A+Key+Feature+of+the+Tumor+Microenvironment+Triggers+Several+Mechanisms+of+Evasion+from+Natural+Killer+and+Cytotoxic+T+Lymphocytes+Surveillance">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mitchell T.J. </li><li class="ref__authors__name">Turajlic S. </li><li class="ref__authors__name">Rowan A. </li><li class="ref__authors__name">Nicol D. </li><li class="ref__authors__name">Farmery J.H.R. </li><li class="ref__authors__name">O’Brien T. </li><li class="ref__authors__name">Martincorena I. </li><li class="ref__authors__name">Tarpey P. </li><li class="ref__authors__name">Angelopoulos N. </li><li class="ref__authors__name">Yates L.R. </li><li>et al.</li><li>TRACERx Renal Consortium</li></ul><div class="ref__title">Timing the Landmark Events in the Evolution of Clear Cell Renal Cell Cancer: TRACERx Renal.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">611-623</span><div class="ref__controls"><a href="#back-bib109" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044780018&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.02.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=611-623&journal=Cell&author=TRACERx+Renal+Consortium&title=Timing+the+Landmark+Events+in+the+Evolution+of+Clear+Cell+Renal+Cell+Cancer%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Monti S. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Mesirov J. </li><li class="ref__authors__name">Golub T. </li></ul><div class="ref__title">Consensus Clustering A Resampling-Based Method for Class Discovery and Visualization of Gene Expression Microarray Data.</div><span class="ref__series"><i>Mach. Learn.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">91-118</span><div class="ref__controls"><a href="#back-bib111" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038724494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1023%2FA%3A1023949509487&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=91-118&journal=Mach.+Learn.&author=Monti+S.&author=Tamayo+P.&author=Mesirov+J.&author=Golub+T.&title=Consensus+Clustering+A+Resampling-Based+Method+for+Class+Discovery+and+Visualization+of+Gene+Expression+Microarray+Data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Morris J.H. </li><li class="ref__authors__name">Apeltsin L. </li><li class="ref__authors__name">Newman A.M. </li><li class="ref__authors__name">Baumbach J. </li><li class="ref__authors__name">Wittkop T. </li><li class="ref__authors__name">Su G. </li><li class="ref__authors__name">Bader G.D. </li><li class="ref__authors__name">Ferrin T.E. </li></ul><div class="ref__title">clusterMaker: a multi-algorithm clustering plugin for Cytoscape.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">436</span><div class="ref__controls"><a href="#back-bib113" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80655132169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22070249&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2F1471-2105-12-436&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=436&journal=BMC+Bioinformatics&author=Morris+J.H.&author=Apeltsin+L.&author=Newman+A.M.&author=Baumbach+J.&author=Wittkop+T.&author=Su+G.&author=Bader+G.D.&author=Ferrin+T.E.&title=clusterMaker%3A+a+multi-algorithm+clustering+plugin+for+Cytoscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Motzer R.J. </li><li class="ref__authors__name">Hutson T.E. </li><li class="ref__authors__name">Tomczak P. </li><li class="ref__authors__name">Michaelson M.D. </li><li class="ref__authors__name">Bukowski R.M. </li><li class="ref__authors__name">Rixe O. </li><li class="ref__authors__name">Oudard S. </li><li class="ref__authors__name">Negrier S. </li><li class="ref__authors__name">Szczylik C. </li><li class="ref__authors__name">Kim S.T. </li><li>et al.</li></ul><div class="ref__title">Sunitinib versus interferon alfa in metastatic renal-cell carcinoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">356</span></span>: <span class="ref__seriesPages">115-124</span><div class="ref__controls"><a href="#back-bib114" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33846181370&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4540)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17215529&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa065044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2007&pages=115-124&journal=N.%C2%A0Engl.+J.+Med.&author=Motzer+R.J.&author=Hutson+T.E.&author=Tomczak+P.&author=Michaelson+M.D.&author=Bukowski+R.M.&author=Rixe+O.&author=Oudard+S.&author=Negrier+S.&author=Szczylik+C.&author=Kim+S.T.&title=Sunitinib+versus+interferon+alfa+in+metastatic+renal-cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Motzer R.J. </li><li class="ref__authors__name">Escudier B. </li><li class="ref__authors__name">Oudard S. </li><li class="ref__authors__name">Hutson T.E. </li><li class="ref__authors__name">Porta C. </li><li class="ref__authors__name">Bracarda S. </li><li class="ref__authors__name">Grünwald V. </li><li class="ref__authors__name">Thompson J.A. </li><li class="ref__authors__name">Figlin R.A. </li><li class="ref__authors__name">Hollaender N. </li><li>et al.</li><li>RECORD-1 Study Group</li></ul><div class="ref__title">Efficacy of everolimus in advanced renal cell carcinoma: a double-blind, randomised, placebo-controlled phase III trial.</div><span class="ref__series"><i>Lancet.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">372</span></span>: <span class="ref__seriesPages">449-456</span><div class="ref__controls"><a href="#back-bib115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-48649107474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2384)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18653228&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2FS0140-6736%2808%2961039-9&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2008&pages=449-456&journal=Lancet&author=RECORD-1+Study+Group&title=Efficacy+of+everolimus+in+advanced+renal+cell+carcinoma%3A+a+double-blind%2C+randomised%2C+placebo-controlled+phase+III+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Motzer R.J. </li><li class="ref__authors__name">Tannir N.M. </li><li class="ref__authors__name">McDermott D.F. </li><li class="ref__authors__name">Arén Frontera O. </li><li class="ref__authors__name">Melichar B. </li><li class="ref__authors__name">Choueiri T.K. </li><li class="ref__authors__name">Plimack E.R. </li><li class="ref__authors__name">Barthélémy P. </li><li class="ref__authors__name">Porta C. </li><li class="ref__authors__name">George S. </li><li>et al.</li><li>CheckMate 214 Investigators</li></ul><div class="ref__title">Nivolumab plus Ipilimumab versus Sunitinib in Advanced Renal-Cell Carcinoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">378</span></span>: <span class="ref__seriesPages">1277-1290</span><div class="ref__controls"><a href="#back-bib116" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045136401&amp;cf=" onclick="newWindow(this.href);return false">Scopus (914)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29562145&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1056%2FNEJMoa1712126&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2018&pages=1277-1290&journal=N.%C2%A0Engl.+J.+Med.&author=CheckMate+214+Investigators&title=Nivolumab+plus+Ipilimumab+versus+Sunitinib+in+Advanced+Renal-Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Naing A. </li><li class="ref__authors__name">Infante J.R. </li><li class="ref__authors__name">Papadopoulos K.P. </li><li class="ref__authors__name">Chan I.H. </li><li class="ref__authors__name">Shen C. </li><li class="ref__authors__name">Ratti N.P. </li><li class="ref__authors__name">Rojo B. </li><li class="ref__authors__name">Autio K.A. </li><li class="ref__authors__name">Wong D.J. </li><li class="ref__authors__name">Patel M.R. </li><li>et al.</li></ul><div class="ref__title">PEGylated IL-10 (Pegilodecakin) Induces Systemic Immune Activation, CD8<sup>+</sup> T Cell Invigoration and Polyclonal T Cell Expansion in Cancer Patients.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">775-791.e3</span><div class="ref__controls"><a href="#back-bib117" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85055890244&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30423297&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccell.2018.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=775-791.e3&journal=Cancer+Cell&author=Naing+A.&author=Infante+J.R.&author=Papadopoulos+K.P.&author=Chan+I.H.&author=Shen+C.&author=Ratti+N.P.&author=Rojo+B.&author=Autio+K.A.&author=Wong+D.J.&author=Patel+M.R.&title=PEGylated+IL-10+%28Pegilodecakin%29+Induces+Systemic+Immune+Activation%2C+CD8%2B+T+Cell+Invigoration+and+Polyclonal+T+Cell+Expansion+in+Cancer+Patients">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nesvizhskii A.I. </li><li class="ref__authors__name">Keller A. </li><li class="ref__authors__name">Kolker E. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">A statistical model for identifying proteins by tandem mass spectrometry.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span><div class="ref__controls"><a href="#back-bib118" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3129)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal.+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ning K. </li><li class="ref__authors__name">Fermin D. </li><li class="ref__authors__name">Nesvizhskii A.I. </li></ul><div class="ref__title">Comparative analysis of different label-free mass spectrometry based protein abundance estimates and their correlation with RNA-Seq gene expression data.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2261-2271</span><div class="ref__controls"><a href="#back-bib119" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84859611662&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22329341&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr201052x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=2261-2271&journal=J.%C2%A0Proteome+Res.&author=Ning+K.&author=Fermin+D.&author=Nesvizhskii+A.I.&title=Comparative+analysis+of+different+label-free+mass+spectrometry+based+protein+abundance+estimates+and+their+correlation+with+RNA-Seq+gene+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ogata H. </li><li class="ref__authors__name">Goto S. </li><li class="ref__authors__name">Sato K. </li><li class="ref__authors__name">Fujibuchi W. </li><li class="ref__authors__name">Bono H. </li><li class="ref__authors__name">Kanehisa M. </li></ul><div class="ref__title">KEGG: Kyoto encyclopedia of genes and genomes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">1999; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">29-34</span><div class="ref__controls"><a href="#back-bib120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0032919364&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2390)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=9847135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2F27.1.29&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=29-34&journal=Nucleic+Acids+Res.&author=Ogata+H.&author=Goto+S.&author=Sato+K.&author=Fujibuchi+W.&author=Bono+H.&author=Kanehisa+M.&title=KEGG%3A+Kyoto+encyclopedia+of+genes+and+genomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ow S.Y. </li><li class="ref__authors__name">Salim M. </li><li class="ref__authors__name">Noirel J. </li><li class="ref__authors__name">Evans C. </li><li class="ref__authors__name">Wright P.C. </li></ul><div class="ref__title">Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation.</div><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2341-2346</span><div class="ref__controls"><a href="#back-bib123" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79956270174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21548092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fpmic.201000752&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=2341-2346&journal=Proteomics&author=Ow+S.Y.&author=Salim+M.&author=Noirel+J.&author=Evans+C.&author=Wright+P.C.&title=Minimising+iTRAQ+ratio+compression+through+understanding+LC-MS+elution+dependence+and+high-resolution+HILIC+fractionation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pan J. </li><li class="ref__authors__name">Qian J. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">OmicsX: a web server for integrated OMICS analysis.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/755918">https://doi.org/10.1101/755918</a></span><div class="ref__controls"><a href="#back-bib124" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85095490240&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2F755918&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=bioRxiv&author=Pan+J.&author=Qian+J.&author=Zhang+H.&title=OmicsX%3A+a+web+server+for+integrated+OMICS+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Panda A. </li><li class="ref__authors__name">de Cubas A.A. </li><li class="ref__authors__name">Stein M. </li><li class="ref__authors__name">Riedlinger G. </li><li class="ref__authors__name">Kra J. </li><li class="ref__authors__name">Mayer T. </li><li class="ref__authors__name">Smith C.C. </li><li class="ref__authors__name">Vincent B.G. </li><li class="ref__authors__name">Serody J.S. </li><li class="ref__authors__name">Beckermann K.E. </li><li>et al.</li></ul><div class="ref__title">Endogenous retrovirus expression is associated with response to immune checkpoint blockade in clear cell renal cell carcinoma.</div><span class="ref__series"><i>JCI Insight.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">121522</span><div class="ref__controls"><a href="#back-bib125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85062245416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30135306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1172%2Fjci.insight.121522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=121522&journal=JCI+Insight&author=Panda+A.&author=de+Cubas+A.A.&author=Stein+M.&author=Riedlinger+G.&author=Kra+J.&author=Mayer+T.&author=Smith+C.C.&author=Vincent+B.G.&author=Serody+J.S.&author=Beckermann+K.E.&title=Endogenous+retrovirus+expression+is+associated+with+response+to+immune+checkpoint+blockade+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Papandreou I. </li><li class="ref__authors__name">Cairns R.A. </li><li class="ref__authors__name">Fontana L. </li><li class="ref__authors__name">Lim A.L. </li><li class="ref__authors__name">Denko N.C. </li></ul><div class="ref__title">HIF-1 mediates adaptation to hypoxia by actively downregulating mitochondrial oxygen consumption.</div><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">187-197</span><div class="ref__controls"><a href="#back-bib126" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33644622570&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1259)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16517406&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cmet.2006.01.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=187-197&journal=Cell+Metab.&author=Papandreou+I.&author=Cairns+R.A.&author=Fontana+L.&author=Lim+A.L.&author=Denko+N.C.&title=HIF-1+mediates+adaptation+to+hypoxia+by+actively+downregulating+mitochondrial+oxygen+consumption">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pavlovich C.P. </li><li class="ref__authors__name">Padilla-Nash H. </li><li class="ref__authors__name">Wangsa D. </li><li class="ref__authors__name">Nickerson M.L. </li><li class="ref__authors__name">Matrosova V. </li><li class="ref__authors__name">Linehan W.M. </li><li class="ref__authors__name">Ried T. </li><li class="ref__authors__name">Phillips J.L. </li></ul><div class="ref__title">Patterns of aneuploidy in stage IV clear cell renal cell carcinoma revealed by comparative genomic hybridization and spectral karyotyping.</div><span class="ref__series"><i>Genes Chromosomes Cancer.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">252-260</span><div class="ref__controls"><a href="#back-bib128" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0038546730&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12759923&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1002%2Fgcc.10209&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2003&pages=252-260&journal=Genes+Chromosomes+Cancer&author=Pavlovich+C.P.&author=Padilla-Nash+H.&author=Wangsa+D.&author=Nickerson+M.L.&author=Matrosova+V.&author=Linehan+W.M.&author=Ried+T.&author=Phillips+J.L.&title=Patterns+of+aneuploidy+in+stage+IV+clear+cell+renal+cell+carcinoma+revealed+by+comparative+genomic+hybridization+and+spectral+karyotyping">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pedregosa F. </li><li class="ref__authors__name">Varoquaux G. </li><li class="ref__authors__name">Gramfort A. </li><li class="ref__authors__name">Michel V. </li><li class="ref__authors__name">Thirion B. </li><li class="ref__authors__name">Grisel O. </li><li class="ref__authors__name">Blondel M. </li><li class="ref__authors__name">Prettenhofer P. </li><li class="ref__authors__name">Weiss R. </li><li class="ref__authors__name">Dubourg V. </li><li>et al.</li></ul><div class="ref__title">Scikit-learn: Machine Learning in Python.</div><span class="ref__series"><i>J. Mach. Learn. Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">2825-2830</span><div class="ref__controls"><a href="#back-bib129" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&journal=J.%C2%A0Mach.+Learn.+Res.&author=Pedregosa+F.&author=Varoquaux+G.&author=Gramfort+A.&author=Michel+V.&author=Thirion+B.&author=Grisel+O.&author=Blondel+M.&author=Prettenhofer+P.&author=Weiss+R.&author=Dubourg+V.&title=Scikit-learn%3A+Machine+Learning+in+Python">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pelletier J. </li><li class="ref__authors__name">Thomas G. </li><li class="ref__authors__name">Volarević S. </li></ul><div class="ref__title">Ribosome biogenesis in cancer: new players and therapeutic avenues.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">51-63</span><div class="ref__controls"><a href="#back-bib130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85038859866&amp;cf=" onclick="newWindow(this.href);return false">Scopus (96)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29192214&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrc.2017.104&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=51-63&journal=Nat.+Rev.+Cancer&author=Pelletier+J.&author=Thomas+G.&author=Volarevi%C4%87+S.&title=Ribosome+biogenesis+in+cancer%3A+new+players+and+therapeutic+avenues">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Yang J. </li><li class="ref__authors__name">Tu Z. </li></ul><div class="ref__title">Integrative random forest for gene regulatory network inference.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">i197-i205</span><div class="ref__controls"><a href="#back-bib132" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84931051773&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26072483&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtv268&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=i197-i205&journal=Bioinformatics&author=Petralia+F.&author=Wang+P.&author=Yang+J.&author=Tu+Z.&title=Integrative+random+forest+for+gene+regulatory+network+inference">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Song W.M. </li><li class="ref__authors__name">Tu Z. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">New Method for Joint Network Analysis Reveals Common and Different Coexpression Patterns among Genes and Proteins in Breast Cancer.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">743-754</span><div class="ref__controls"><a href="#back-bib133" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84960353766&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26733076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Facs.jproteome.5b00925&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=743-754&journal=J.%C2%A0Proteome+Res.&author=Petralia+F.&author=Song+W.M.&author=Tu+Z.&author=Wang+P.&title=New+Method+for+Joint+Network+Analysis+Reveals+Common+and+Different+Coexpression+Patterns+among+Genes+and+Proteins+in+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Peng J. </li><li class="ref__authors__name">Yan A. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span><div class="ref__controls"><a href="#back-bib134" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petrella B.L. </li><li class="ref__authors__name">Brinckerhoff C.E. </li></ul><div class="ref__title">PTEN suppression of YY1 induces HIF-2α activity in von-Hippel-Lindau-null renal-cell carcinoma.</div><span class="ref__series"><i>Cancer Biol. Ther.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1389-1401</span><div class="ref__controls"><a href="#back-bib135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-76249096569&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19483472&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.4161%2Fcbt.8.14.8880&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=1389-1401&journal=Cancer+Biol.+Ther.&author=Petrella+B.L.&author=Brinckerhoff+C.E.&title=PTEN+suppression+of+YY1+induces+HIF-2%CE%B1+activity+in+von-Hippel-Lindau-null+renal-cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pierre-Jean M. </li><li class="ref__authors__name">Rigaill G. </li><li class="ref__authors__name">Neuvial P. </li></ul><div class="ref__title">Performance evaluation of DNA copy number segmentation methods.</div><span class="ref__series"><i>Brief. Bioinform.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">600-615</span><div class="ref__controls"><a href="#back-bib136" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925419952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25202135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbib%2Fbbu026&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=600-615&journal=Brief.+Bioinform.&author=Pierre-Jean+M.&author=Rigaill+G.&author=Neuvial+P.&title=Performance+evaluation+of+DNA+copy+number+segmentation+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rauniyar N. </li><li class="ref__authors__name">Yates 3rd, J.R. </li></ul><div class="ref__title">Isobaric labeling-based relative quantification in shotgun proteomics.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">5293-5309</span><div class="ref__controls"><a href="#back-bib138" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84915751035&amp;cf=" onclick="newWindow(this.href);return false">Scopus (229)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25337643&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr500880b&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=5293-5309&journal=J.%C2%A0Proteome+Res.&author=Rauniyar+N.&author=Yates+J.R.&title=Isobaric+labeling-based+relative+quantification+in+shotgun+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ravaud A. </li><li class="ref__authors__name">Hawkins R. </li><li class="ref__authors__name">Gardner J.P. </li><li class="ref__authors__name">von der Maase H. </li><li class="ref__authors__name">Zantl N. </li><li class="ref__authors__name">Harper P. </li><li class="ref__authors__name">Rolland F. </li><li class="ref__authors__name">Audhuy B. </li><li class="ref__authors__name">Machiels J.P. </li><li class="ref__authors__name">Pétavy F. </li><li>et al.</li></ul><div class="ref__title">Lapatinib versus hormone therapy in patients with advanced renal cell carcinoma: a randomized phase III clinical trial.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">2285-2291</span><div class="ref__controls"><a href="#back-bib139" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-44249127263&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=18467719&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2007.14.5029&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=2285-2291&journal=J.%C2%A0Clin.+Oncol.&author=Ravaud+A.&author=Hawkins+R.&author=Gardner+J.P.&author=von+der+Maase+H.&author=Zantl+N.&author=Harper+P.&author=Rolland+F.&author=Audhuy+B.&author=Machiels+J.P.&author=P%C3%A9tavy+F.&title=Lapatinib+versus+hormone+therapy+in+patients+with+advanced+renal+cell+carcinoma%3A+a+randomized+phase+III+clinical+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Richman T.R. </li><li class="ref__authors__name">Spåhr H. </li><li class="ref__authors__name">Ermer J.A. </li><li class="ref__authors__name">Davies S.M.K. </li><li class="ref__authors__name">Viola H.M. </li><li class="ref__authors__name">Bates K.A. </li><li class="ref__authors__name">Papadimitriou J. </li><li class="ref__authors__name">Hool L.C. </li><li class="ref__authors__name">Rodger J. </li><li class="ref__authors__name">Larsson N.G. </li><li>et al.</li></ul><div class="ref__title">Loss of the RNA-binding protein TACO1 causes late-onset mitochondrial dysfunction in mice.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">11884</span><div class="ref__controls"><a href="#back-bib140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84975502603&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27319982&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms11884&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11884&journal=Nat.+Commun.&author=Richman+T.R.&author=Sp%C3%A5hr+H.&author=Ermer+J.A.&author=Davies+S.M.K.&author=Viola+H.M.&author=Bates+K.A.&author=Papadimitriou+J.&author=Hool+L.C.&author=Rodger+J.&author=Larsson+N.G.&title=Loss+of+the+RNA-binding+protein+TACO1+causes+late-onset+mitochondrial+dysfunction+in+mice">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ricketts C.J. </li><li class="ref__authors__name">De Cubas A.A. </li><li class="ref__authors__name">Fan H. </li><li class="ref__authors__name">Smith C.C. </li><li class="ref__authors__name">Lang M. </li><li class="ref__authors__name">Reznik E. </li><li class="ref__authors__name">Bowlby R. </li><li class="ref__authors__name">Gibb E.A. </li><li class="ref__authors__name">Akbani R. </li><li class="ref__authors__name">Beroukhim R. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">The Cancer Genome Atlas Comprehensive Molecular Characterization of Renal Cell Carcinoma.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">313-326</span><div class="ref__controls"><a href="#back-bib141" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85044860979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29617669&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.celrep.2018.03.075&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=313-326&journal=Cell+Rep.&author=Cancer+Genome+Atlas+Research+Network&title=The+Cancer+Genome+Atlas+Comprehensive+Molecular+Characterization+of+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Riester M. </li><li class="ref__authors__name">Singh A.P. </li><li class="ref__authors__name">Brannon A.R. </li><li class="ref__authors__name">Yu K. </li><li class="ref__authors__name">Campbell C.D. </li><li class="ref__authors__name">Chiang D.Y. </li><li class="ref__authors__name">Morrissey M.P. </li></ul><div class="ref__title">PureCN: copy number calling and SNV classification using targeted short read sequencing.</div><span class="ref__series"><i>Source Code Biol. Med.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">13</span><div class="ref__controls"><a href="#back-bib142" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85006152885&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27999612&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13029-016-0060-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=13&journal=Source+Code+Biol.+Med.&author=Riester+M.&author=Singh+A.P.&author=Brannon+A.R.&author=Yu+K.&author=Campbell+C.D.&author=Chiang+D.Y.&author=Morrissey+M.P.&title=PureCN%3A+copy+number+calling+and+SNV+classification+using+targeted+short+read+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Robinson D.R. </li><li class="ref__authors__name">Wu Y.M. </li><li class="ref__authors__name">Lonigro R.J. </li><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Cobain E. </li><li class="ref__authors__name">Everett J. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Rabban E. </li><li class="ref__authors__name">Kumar-Sinha C. </li><li class="ref__authors__name">Raymond V. </li><li>et al.</li></ul><div class="ref__title">Integrative clinical genomics of metastatic cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">548</span></span>: <span class="ref__seriesPages">297-303</span><div class="ref__controls"><a href="#back-bib143" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85027893400&amp;cf=" onclick="newWindow(this.href);return false">Scopus (215)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28783718&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature23306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2017&pages=297-303&journal=Nature&author=Robinson+D.R.&author=Wu+Y.M.&author=Lonigro+R.J.&author=Vats+P.&author=Cobain+E.&author=Everett+J.&author=Cao+X.&author=Rabban+E.&author=Kumar-Sinha+C.&author=Raymond+V.&title=Integrative+clinical+genomics+of+metastatic+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ross P.L. </li><li class="ref__authors__name">Huang Y.N. </li><li class="ref__authors__name">Marchese J.N. </li><li class="ref__authors__name">Williamson B. </li><li class="ref__authors__name">Parker K. </li><li class="ref__authors__name">Hattan S. </li><li class="ref__authors__name">Khainovski N. </li><li class="ref__authors__name">Pillai S. </li><li class="ref__authors__name">Dey S. </li><li class="ref__authors__name">Daniels S. </li><li>et al.</li></ul><div class="ref__title">Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">1154-1169</span><div class="ref__controls"><a href="#back-bib144" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-19944432197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3363)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=15385600&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M400129-MCP200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=1154-1169&journal=Mol.+Cell.+Proteomics&author=Ross+P.L.&author=Huang+Y.N.&author=Marchese+J.N.&author=Williamson+B.&author=Parker+K.&author=Hattan+S.&author=Khainovski+N.&author=Pillai+S.&author=Dey+S.&author=Daniels+S.&title=Multiplexed+protein+quantitation+in+Saccharomyces+cerevisiae+using+amine-reactive+isobaric+tagging+reagents">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Tang Z. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Grover H. </li><li class="ref__authors__name">Askenazi M. </li><li class="ref__authors__name">Teubl J. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Tabb D.L. </li><li>et al.</li></ul><div class="ref__title">An Analysis of the Sensitivity of Proteogenomic Mapping of Somatic Mutations and Novel Splicing Events in Cancer.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1060-1071</span><div class="ref__controls"><a href="#back-bib145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84962585103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26631509&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M115.056226&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=1060-1071&journal=Mol.+Cell.+Proteomics&author=Ruggles+K.V.&author=Tang+Z.&author=Wang+X.&author=Grover+H.&author=Askenazi+M.&author=Teubl+J.&author=Cao+S.&author=McLellan+M.D.&author=Clauser+K.R.&author=Tabb+D.L.&title=An+Analysis+of+the+Sensitivity+of+Proteogenomic+Mapping+of+Somatic+Mutations+and+Novel+Splicing+Events+in+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sade-Feldman M. </li><li class="ref__authors__name">Yizhak K. </li><li class="ref__authors__name">Bjorgaard S.L. </li><li class="ref__authors__name">Ray J.P. </li><li class="ref__authors__name">de Boer C.G. </li><li class="ref__authors__name">Jenkins R.W. </li><li class="ref__authors__name">Lieb D.J. </li><li class="ref__authors__name">Chen J.H. </li><li class="ref__authors__name">Frederick D.T. </li><li class="ref__authors__name">Barzily-Rokni M. </li><li>et al.</li></ul><div class="ref__title">Defining T Cell States Associated with Response to Checkpoint Immunotherapy in Melanoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">175</span></span>: <span class="ref__seriesPages">998-1013.e20</span><div class="ref__controls"><a href="#back-bib146" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85055560004&amp;cf=" onclick="newWindow(this.href);return false">Scopus (156)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30388456&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.10.038&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=998-1013.e20&journal=Cell&author=Sade-Feldman+M.&author=Yizhak+K.&author=Bjorgaard+S.L.&author=Ray+J.P.&author=de+Boer+C.G.&author=Jenkins+R.W.&author=Lieb+D.J.&author=Chen+J.H.&author=Frederick+D.T.&author=Barzily-Rokni+M.&title=Defining+T+Cell+States+Associated+with+Response+to+Checkpoint+Immunotherapy+in+Melanoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Savitski M.M. </li><li class="ref__authors__name">Wilhelm M. </li><li class="ref__authors__name">Hahne H. </li><li class="ref__authors__name">Kuster B. </li><li class="ref__authors__name">Bantscheff M. </li></ul><div class="ref__title">A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span><div class="ref__controls"><a href="#back-bib147" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (125)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell.+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+Scalable+Approach+for+Protein+False+Discovery+Rate+Estimation+in+Large+Proteomic+Data+Sets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Scott K.L. </li><li class="ref__authors__name">Kabbarah O. </li><li class="ref__authors__name">Liang M.C. </li><li class="ref__authors__name">Ivanova E. </li><li class="ref__authors__name">Anagnostou V. </li><li class="ref__authors__name">Wu J. </li><li class="ref__authors__name">Dhakal S. </li><li class="ref__authors__name">Wu M. </li><li class="ref__authors__name">Chen S. </li><li class="ref__authors__name">Feinberg T. </li><li>et al.</li></ul><div class="ref__title">GOLPH3 modulates mTOR signalling and rapamycin sensitivity in cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">459</span></span>: <span class="ref__seriesPages">1085-1090</span><div class="ref__controls"><a href="#back-bib148" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-67649500009&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19553991&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature08109&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=1085-1090&journal=Nature&author=Scott+K.L.&author=Kabbarah+O.&author=Liang+M.C.&author=Ivanova+E.&author=Anagnostou+V.&author=Wu+J.&author=Dhakal+S.&author=Wu+M.&author=Chen+S.&author=Feinberg+T.&title=GOLPH3+modulates+mTOR+signalling+and+rapamycin+sensitivity+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Seppä H. </li><li class="ref__authors__name">Grotendorst G. </li><li class="ref__authors__name">Seppä S. </li><li class="ref__authors__name">Schiffmann E. </li><li class="ref__authors__name">Martin G.R. </li></ul><div class="ref__title">Platelet-derived Growth Factor Is Chemotactic for Fibroblasts.</div><span class="ref__series"><i>J. Cell Biol.</i> <span class="ref__seriesDate">1982; </span><span class="ref__seriesVolume">92</span></span>: <span class="ref__seriesPages">584-588</span><div class="ref__controls"><a href="#back-bib149" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=7061598&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1083%2Fjcb.92.2.584&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=584-588&journal=J.%C2%A0Cell+Biol.&author=Sepp%C3%A4+H.&author=Grotendorst+G.&author=Sepp%C3%A4+S.&author=Schiffmann+E.&author=Martin+G.R.&title=Platelet-derived+Growth+Factor+Is+Chemotactic+for+Fibroblasts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shannon P. </li><li class="ref__authors__name">Markiel A. </li><li class="ref__authors__name">Ozier O. </li><li class="ref__authors__name">Baliga N.S. </li><li class="ref__authors__name">Wang J.T. </li><li class="ref__authors__name">Ramage D. </li><li class="ref__authors__name">Amin N. </li><li class="ref__authors__name">Schwikowski B. </li><li class="ref__authors__name">Ideker T. </li></ul><div class="ref__title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span><div class="ref__controls"><a href="#back-bib150" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13696)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">She Q.B. </li><li class="ref__authors__name">Halilovic E. </li><li class="ref__authors__name">Ye Q. </li><li class="ref__authors__name">Zhen W. </li><li class="ref__authors__name">Shirasawa S. </li><li class="ref__authors__name">Sasazuki T. </li><li class="ref__authors__name">Solit D.B. </li><li class="ref__authors__name">Rosen N. </li></ul><div class="ref__title">4E-BP1 is a key effector of the oncogenic activation of the AKT and ERK signaling pathways that integrates their function in tumors.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">39-51</span><div class="ref__controls"><a href="#back-bib151" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954296394&amp;cf=" onclick="newWindow(this.href);return false">Scopus (280)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20609351&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.ccr.2010.05.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=39-51&journal=Cancer+Cell&author=She+Q.B.&author=Halilovic+E.&author=Ye+Q.&author=Zhen+W.&author=Shirasawa+S.&author=Sasazuki+T.&author=Solit+D.B.&author=Rosen+N.&title=4E-BP1+is+a+key+effector+of+the+oncogenic+activation+of+the+AKT+and+ERK+signaling+pathways+that+integrates+their+function+in+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shen C. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Zhou J. </li><li class="ref__authors__name">Chang M. </li><li class="ref__authors__name">Signoretti S. </li><li class="ref__authors__name">Kaelin Jr., W.G. </li></ul><div class="ref__title">Genetic and functional studies implicate HIF1α as a 14q kidney cancer suppressor gene.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">222-235</span><div class="ref__controls"><a href="#back-bib152" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-80054771537&amp;cf=" onclick="newWindow(this.href);return false">Scopus (212)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22037472&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-11-0098&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2011&pages=222-235&journal=Cancer+Discov.&author=Shen+C.&author=Beroukhim+R.&author=Schumacher+S.E.&author=Zhou+J.&author=Chang+M.&author=Signoretti+S.&author=Kaelin+W.G.&title=Genetic+and+functional+studies+implicate+HIF1%CE%B1+as+a+14q+kidney+cancer+suppressor+gene">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shenoy N. </li><li class="ref__authors__name">Vallumsetla N. </li><li class="ref__authors__name">Zou Y. </li><li class="ref__authors__name">Galeas J.N. </li><li class="ref__authors__name">Shrivastava M. </li><li class="ref__authors__name">Hu C. </li><li class="ref__authors__name">Susztak K. </li><li class="ref__authors__name">Verma A. </li></ul><div class="ref__title">Role of DNA methylation in renal cell carcinoma.</div><span class="ref__series"><i>J. Hematol. Oncol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">88</span><div class="ref__controls"><a href="#back-bib153" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84937417017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26198328&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1186%2Fs13045-015-0180-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=88&journal=J.%C2%A0Hematol.+Oncol.&author=Shenoy+N.&author=Vallumsetla+N.&author=Zou+Y.&author=Galeas+J.N.&author=Shrivastava+M.&author=Hu+C.&author=Susztak+K.&author=Verma+A.&title=Role+of+DNA+methylation+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shiga K. </li><li class="ref__authors__name">Hara M. </li><li class="ref__authors__name">Nagasaki T. </li><li class="ref__authors__name">Sato T. </li><li class="ref__authors__name">Takahashi H. </li><li class="ref__authors__name">Takeyama H. </li></ul><div class="ref__title">Cancer-associated fibroblasts: Their characteristics and their roles in tumor growth.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">2443-2458</span><div class="ref__controls"><a href="#back-bib154" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84949775551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (289)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26690480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3390%2Fcancers7040902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=2443-2458&journal=Cancers+%28Basel%29&author=Shiga+K.&author=Hara+M.&author=Nagasaki+T.&author=Sato+T.&author=Takahashi+H.&author=Takeyama+H.&title=Cancer-associated+fibroblasts%3A+Their+characteristics+and+their+roles+in+tumor+growth">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shin S.-J. </li><li class="ref__authors__name">Jeon Y.K. </li><li class="ref__authors__name">Cho Y.M. </li><li class="ref__authors__name">Lee J.-L. </li><li class="ref__authors__name">Chung D.H. </li><li class="ref__authors__name">Park J.Y. </li><li class="ref__authors__name">Go H. </li></ul><div class="ref__title">The Association Between PD-L1 Expression and the Clinical Outcomes to Vascular Endothelial Growth Factor-Targeted Therapy in Patients With Metastatic Clear Cell Renal Cell Carcinoma.</div><span class="ref__series"><i>Oncologist.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1253-1260</span><div class="ref__controls"><a href="#back-bib155" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26424759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1634%2Ftheoncologist.2015-0151&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1253-1260&journal=Oncologist&author=Shin+S.-J.&author=Jeon+Y.K.&author=Cho+Y.M.&author=Lee+J.-L.&author=Chung+D.H.&author=Park+J.Y.&author=Go+H.&title=The+Association+Between+PD-L1+Expression+and+the+Clinical+Outcomes+to+Vascular+Endothelial+Growth+Factor-Targeted+Therapy+in+Patients+With+Metastatic+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Siegel R.L. </li><li class="ref__authors__name">Miller K.D. </li><li class="ref__authors__name">Jemal A. </li></ul><div class="ref__title">Cancer statistics, 2018.</div><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7-30</span><div class="ref__controls"><a href="#back-bib156" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85040049759&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3369)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29313949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3322%2Fcaac.21442&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2018">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">D’Angelo F. </li><li class="ref__authors__name">Bontempi G. </li><li class="ref__authors__name">Ceccarelli M. </li><li class="ref__authors__name">Noushmehr H. </li></ul><div class="ref__title"><em>TCGA Workflow</em>: Analyze cancer genomics and epigenomics data using Bioconductor packages.</div><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1542</span><div class="ref__controls"><a href="#back-bib157" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85013753091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28232861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.12688%2Ff1000research.8923.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1542&journal=F1000Res.&author=Silva+T.C.&author=Colaprico+A.&author=Olsen+C.&author=D%E2%80%99Angelo+F.&author=Bontempi+G.&author=Ceccarelli+M.&author=Noushmehr+H.&title=TCGA+Workflow%3A+Analyze+cancer+genomics+and+epigenomics+data+using+Bioconductor+packages">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Smith C.C. </li><li class="ref__authors__name">Beckermann K.E. </li><li class="ref__authors__name">Bortone D.S. </li><li class="ref__authors__name">De Cubas A.A. </li><li class="ref__authors__name">Bixby L.M. </li><li class="ref__authors__name">Lee S.J. </li><li class="ref__authors__name">Panda A. </li><li class="ref__authors__name">Ganesan S. </li><li class="ref__authors__name">Bhanot G. </li><li class="ref__authors__name">Wallen E.M. </li><li>et al.</li></ul><div class="ref__title">Endogenous retroviral signatures predict immunotherapy response in clear cell renal cell carcinoma.</div><span class="ref__series"><i>J. Clin. Invest.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">128</span></span>: <span class="ref__seriesPages">4804-4820</span><div class="ref__controls"><a href="#back-bib158" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053394932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30137025&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1172%2FJCI121476&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=4804-4820&journal=J.%C2%A0Clin.+Invest.&author=Smith+C.C.&author=Beckermann+K.E.&author=Bortone+D.S.&author=De+Cubas+A.A.&author=Bixby+L.M.&author=Lee+S.J.&author=Panda+A.&author=Ganesan+S.&author=Bhanot+G.&author=Wallen+E.M.&title=Endogenous+retroviral+signatures+predict+immunotherapy+response+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Paull E.O. </li><li class="ref__authors__name">Stuart J.M. </li></ul><div class="ref__title">One-Class Detection of Cell States in Tumor Subtypes.</div><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span><div class="ref__controls"><a href="#back-bib159" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-Class+Detection+of+Cell+States+in+Tumor+Subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Ji J. </li><li class="ref__authors__name">Gleason K.J. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Martignetti J.A. </li><li class="ref__authors__name">Chen L.S. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span><div class="ref__controls"><a href="#back-bib160" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+Impact+of+DNA+Copy+Number+Alteration+and+Methylation+on+the+Proteogenomic+Landscape+of+Human+Ovarian+Cancer+via+a+Multi-omics+Integrative+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Staedtler F. </li><li class="ref__authors__name">Hartmann N. </li><li class="ref__authors__name">Letzkus M. </li><li class="ref__authors__name">Bongiovanni S. </li><li class="ref__authors__name">Scherer A. </li><li class="ref__authors__name">Marc P. </li><li class="ref__authors__name">Johnson K.J. </li><li class="ref__authors__name">Schumacher M.M. </li></ul><div class="ref__title">Robust and tissue-independent gender-specific transcript biomarkers.</div><span class="ref__series"><i>Biomarkers.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">436-445</span><div class="ref__controls"><a href="#back-bib161" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84880540860&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23829492&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3109%2F1354750X.2013.811538&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=436-445&journal=Biomarkers&author=Staedtler+F.&author=Hartmann+N.&author=Letzkus+M.&author=Bongiovanni+S.&author=Scherer+A.&author=Marc+P.&author=Johnson+K.J.&author=Schumacher+M.M.&title=Robust+and+tissue-independent+gender-specific+transcript+biomarkers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sternberg C.N. </li><li class="ref__authors__name">Davis I.D. </li><li class="ref__authors__name">Mardiak J. </li><li class="ref__authors__name">Szczylik C. </li><li class="ref__authors__name">Lee E. </li><li class="ref__authors__name">Wagstaff J. </li><li class="ref__authors__name">Barrios C.H. </li><li class="ref__authors__name">Salman P. </li><li class="ref__authors__name">Gladkov O.A. </li><li class="ref__authors__name">Kavina A. </li><li>et al.</li></ul><div class="ref__title">Pazopanib in locally advanced or metastatic renal cell carcinoma: results of a randomized phase III trial.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1061-1068</span><div class="ref__controls"><a href="#back-bib162" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77949890945&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1792)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20100962&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1200%2FJCO.2009.23.9764&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=1061-1068&journal=J.%C2%A0Clin.+Oncol.&author=Sternberg+C.N.&author=Davis+I.D.&author=Mardiak+J.&author=Szczylik+C.&author=Lee+E.&author=Wagstaff+J.&author=Barrios+C.H.&author=Salman+P.&author=Gladkov+O.A.&author=Kavina+A.&title=Pazopanib+in+locally+advanced+or+metastatic+renal+cell+carcinoma%3A+results+of+a+randomized+phase+III+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Su G. </li><li class="ref__authors__name">Kuchinsky A. </li><li class="ref__authors__name">Morris J.H. </li><li class="ref__authors__name">States D.J. </li><li class="ref__authors__name">Meng F. </li></ul><div class="ref__title">GLay: community structure analysis of biological networks.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">3135-3137</span><div class="ref__controls"><a href="#back-bib163" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-79951764932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (113)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=21123224&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq596&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=3135-3137&journal=Bioinformatics&author=Su+G.&author=Kuchinsky+A.&author=Morris+J.H.&author=States+D.J.&author=Meng+F.&title=GLay%3A+community+structure+analysis+of+biological+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Mootha V.K. </li><li class="ref__authors__name">Mukherjee S. </li><li class="ref__authors__name">Ebert B.L. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Paulovich A. </li><li class="ref__authors__name">Pomeroy S.L. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span><div class="ref__controls"><a href="#back-bib164" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15633)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Szolek A. </li><li class="ref__authors__name">Schubert B. </li><li class="ref__authors__name">Mohr C. </li><li class="ref__authors__name">Sturm M. </li><li class="ref__authors__name">Feldhahn M. </li><li class="ref__authors__name">Kohlbacher O. </li></ul><div class="ref__title">OptiType: precision HLA typing from next-generation sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span><div class="ref__controls"><a href="#back-bib165" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (192)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Thompson A. </li><li class="ref__authors__name">Schäfer J. </li><li class="ref__authors__name">Kuhn K. </li><li class="ref__authors__name">Kienle S. </li><li class="ref__authors__name">Schwarz J. </li><li class="ref__authors__name">Schmidt G. </li><li class="ref__authors__name">Neumann T. </li><li class="ref__authors__name">Johnstone R. </li><li class="ref__authors__name">Mohammed A.K. </li><li class="ref__authors__name">Hamon C. </li></ul><div class="ref__title">Tandem mass tags: a novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">1895-1904</span><div class="ref__controls"><a href="#back-bib166" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-12444345515&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1275)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=12713048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac0262560&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=1895-1904&journal=Anal.+Chem.&author=Thompson+A.&author=Sch%C3%A4fer+J.&author=Kuhn+K.&author=Kienle+S.&author=Schwarz+J.&author=Schmidt+G.&author=Neumann+T.&author=Johnstone+R.&author=Mohammed+A.K.&author=Hamon+C.&title=Tandem+mass+tags%3A+a+novel+quantification+strategy+for+comparative+analysis+of+complex+protein+mixtures+by+MS%2FMS">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tsou C.C. </li><li class="ref__authors__name">Avtonomov D. </li><li class="ref__authors__name">Larsen B. </li><li class="ref__authors__name">Tucholska M. </li><li class="ref__authors__name">Choi H. </li><li class="ref__authors__name">Gingras A.C. </li><li class="ref__authors__name">Nesvizhskii A.I. </li></ul><div class="ref__title">DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">258-264</span><div class="ref__controls"><a href="#back-bib168" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84923838580&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25599550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnmeth.3255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=258-264&journal=Nat.+Methods&author=Tsou+C.C.&author=Avtonomov+D.&author=Larsen+B.&author=Tucholska+M.&author=Choi+H.&author=Gingras+A.C.&author=Nesvizhskii+A.I.&title=DIA-Umpire%3A+comprehensive+computational+framework+for+data-independent+acquisition+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Turajlic S. </li><li class="ref__authors__name">Xu H. </li><li class="ref__authors__name">Litchfield K. </li><li class="ref__authors__name">Rowan A. </li><li class="ref__authors__name">Horswell S. </li><li class="ref__authors__name">Chambers T. </li><li class="ref__authors__name">O’Brien T. </li><li class="ref__authors__name">Lopez J.I. </li><li class="ref__authors__name">Watkins T.B.K. </li><li class="ref__authors__name">Nicol D. </li><li>et al.</li><li>TRACERx Renal Consortium</li></ul><div class="ref__title">Deterministic Evolutionary Trajectories Influence Primary Tumor Growth: TRACERx Renal.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">595-610</span><div class="ref__controls"><a href="#back-bib169" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045306946&amp;cf=" onclick="newWindow(this.href);return false">Scopus (110)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656894&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.043&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=595-610&journal=Cell&author=TRACERx+Renal+Consortium&title=Deterministic+Evolutionary+Trajectories+Influence+Primary+Tumor+Growth%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Turajlic S. </li><li class="ref__authors__name">Xu H. </li><li class="ref__authors__name">Litchfield K. </li><li class="ref__authors__name">Rowan A. </li><li class="ref__authors__name">Chambers T. </li><li class="ref__authors__name">Lopez J.I. </li><li class="ref__authors__name">Nicol D. </li><li class="ref__authors__name">O’Brien T. </li><li class="ref__authors__name">Larkin J. </li><li class="ref__authors__name">Horswell S. </li><li>et al.</li><li>PEACE</li><li>TRACERx Renal Consortium</li></ul><div class="ref__title">Tracking Cancer Evolution Reveals Constrained Routes to Metastases: TRACERx Renal.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">581-594</span><div class="ref__controls"><a href="#back-bib170" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85045083551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29656895&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2018.03.057&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=581-594&journal=Cell&author=PEACETRACERx+Renal+Consortium&title=Tracking+Cancer+Evolution+Reveals+Constrained+Routes+to+Metastases%3A+TRACERx+Renal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tusher V.G. </li><li class="ref__authors__name">Tibshirani R. </li><li class="ref__authors__name">Chu G. </li></ul><div class="ref__title">Significance analysis of microarrays applied to the ionizing radiation response.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">5116-5121</span><div class="ref__controls"><a href="#back-bib171" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-0035942271&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9173)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=11309499&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1073%2Fpnas.091062498&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=5116-5121&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Tusher+V.G.&author=Tibshirani+R.&author=Chu+G.&title=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ubersax J.A. </li><li class="ref__authors__name">Ferrell Jr., J.E. </li></ul><div class="ref__title">Mechanisms of specificity in protein phosphorylation.</div><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">530-541</span><div class="ref__controls"><a href="#back-bib172" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-34250878954&amp;cf=" onclick="newWindow(this.href);return false">Scopus (801)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17585314&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrm2203&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=530-541&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Ubersax+J.A.&author=Ferrell+J.E.&title=Mechanisms+of+specificity+in+protein+phosphorylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vander Haar E. </li><li class="ref__authors__name">Lee S.I. </li><li class="ref__authors__name">Bandhakavi S. </li><li class="ref__authors__name">Griffin T.J. </li><li class="ref__authors__name">Kim D.H. </li></ul><div class="ref__title">Insulin signalling to mTOR mediated by the Akt/PKB substrate PRAS40.</div><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">316-323</span><div class="ref__controls"><a href="#back-bib173" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-33847397874&amp;cf=" onclick="newWindow(this.href);return false">Scopus (779)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=17277771&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncb1547&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=316-323&journal=Nat.+Cell+Biol.&author=Vander+Haar+E.&author=Lee+S.I.&author=Bandhakavi+S.&author=Griffin+T.J.&author=Kim+D.H.&title=Insulin+signalling+to+mTOR+mediated+by+the+Akt%2FPKB+substrate+PRAS40">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Slebos R.J.C. </li><li class="ref__authors__name">Wang D. </li><li class="ref__authors__name">Halvey P.J. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Liebler D.C. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Protein identification using customized protein sequence databases derived from RNA-Seq data.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span><div class="ref__controls"><a href="#back-bib174" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (87)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.C.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases+derived+from+RNA-Seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Greer M. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span><div class="ref__controls"><a href="#back-bib175" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (345)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt+2017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Chen Y.-B. </li><li class="ref__authors__name">Skala S.L. </li><li class="ref__authors__name">Al-Ahmadie H.A. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Veeneman B.A. </li><li class="ref__authors__name">Chen J. </li><li class="ref__authors__name">Cieślik M. </li><li>et al.</li></ul><div class="ref__title">VSTM2A over-expression is a sensitive and specific biomarker for mucinous tubular and spindle cell carcinoma (MTSCC) of the kidney.</div><span class="ref__series"><i>Am. J. Surg. Pathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">1571-1584</span><div class="ref__controls"><a href="#back-bib176" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85054473283&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30285995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1097%2FPAS.0000000000001150&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2018&pages=1571-1584&journal=Am.+J.+Surg.+Pathol.&author=Wang+L.&author=Zhang+Y.&author=Chen+Y.-B.&author=Skala+S.L.&author=Al-Ahmadie+H.A.&author=Wang+X.&author=Cao+X.&author=Veeneman+B.A.&author=Chen+J.&author=Cie%C5%9Blik+M.&title=VSTM2A+over-expression+is+a+sensitive+and+specific+biomarker+for+mucinous+tubular+and+spindle+cell+carcinoma+%28MTSCC%29+of+the+kidney">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang T. </li><li class="ref__authors__name">Lu R. </li><li class="ref__authors__name">Kapur P. </li><li class="ref__authors__name">Jaiswal B.S. </li><li class="ref__authors__name">Hannan R. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Pedrosa I. </li><li class="ref__authors__name">Luke J.J. </li><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Goldstein L.D. </li><li>et al.</li></ul><div class="ref__title">An empirical approach leveraging tumorgrafts to dissect the tumor microenvironment in renal cell carcinoma identifies missing link to prognostic inflammatory factors.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1142-1155</span><div class="ref__controls"><a href="#back-bib177" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85053203753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29884728&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F2159-8290.CD-17-1246&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1142-1155&journal=Cancer+Discov.&author=Wang+T.&author=Lu+R.&author=Kapur+P.&author=Jaiswal+B.S.&author=Hannan+R.&author=Zhang+Z.&author=Pedrosa+I.&author=Luke+J.J.&author=Zhang+H.&author=Goldstein+L.D.&title=An+empirical+approach+leveraging+tumorgrafts+to+dissect+the+tumor+microenvironment+in+renal+cell+carcinoma+identifies+missing+link+to+prognostic+inflammatory+factors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span><div class="ref__controls"><a href="#back-bib178" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Werner T. </li><li class="ref__authors__name">Becher I. </li><li class="ref__authors__name">Sweetman G. </li><li class="ref__authors__name">Doce C. </li><li class="ref__authors__name">Savitski M.M. </li><li class="ref__authors__name">Bantscheff M. </li></ul><div class="ref__title">High-resolution enabled TMT 8-plexing.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">84</span></span>: <span class="ref__seriesPages">7188-7194</span><div class="ref__controls"><a href="#back-bib179" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84865454296&amp;cf=" onclick="newWindow(this.href);return false">Scopus (115)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=22881393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1021%2Fac301553x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=7188-7194&journal=Anal.+Chem.&author=Werner+T.&author=Becher+I.&author=Sweetman+G.&author=Doce+C.&author=Savitski+M.M.&author=Bantscheff+M.&title=High-resolution+enabled+TMT+8-plexing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wettersten H.I. </li><li class="ref__authors__name">Aboud O.A. </li><li class="ref__authors__name">Lara Jr., P.N. </li><li class="ref__authors__name">Weiss R.H. </li></ul><div class="ref__title">Metabolic reprogramming in clear cell renal cell carcinoma.</div><span class="ref__series"><i>Nat. Rev. Nephrol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">410-419</span><div class="ref__controls"><a href="#back-bib180" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85019246775&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28480903&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnrneph.2017.59&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=410-419&journal=Nat.+Rev.+Nephrol.&author=Wettersten+H.I.&author=Aboud+O.A.&author=Lara+P.N.&author=Weiss+R.H.&title=Metabolic+reprogramming+in+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Hayes D.N. </li></ul><div class="ref__title">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span><div class="ref__controls"><a href="#back-bib182" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (442)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wu S. </li><li class="ref__authors__name">Kasim V. </li><li class="ref__authors__name">Kano M.R. </li><li class="ref__authors__name">Tanaka S. </li><li class="ref__authors__name">Ohba S. </li><li class="ref__authors__name">Miura Y. </li><li class="ref__authors__name">Miyata K. </li><li class="ref__authors__name">Liu X. </li><li class="ref__authors__name">Matsuhashi A. </li><li class="ref__authors__name">Chung U.I. </li><li>et al.</li></ul><div class="ref__title">Transcription factor YY1 contributes to tumor growth by stabilizing hypoxia factor HIF-1α in a p53-independent manner.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">1787-1799</span><div class="ref__controls"><a href="#back-bib183" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84875473756&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=23328582&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1158%2F0008-5472.CAN-12-0366&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1787-1799&journal=Cancer+Res.&author=Wu+S.&author=Kasim+V.&author=Kano+M.R.&author=Tanaka+S.&author=Ohba+S.&author=Miura+Y.&author=Miyata+K.&author=Liu+X.&author=Matsuhashi+A.&author=Chung+U.I.&title=Transcription+factor+YY1+contributes+to+tumor+growth+by+stabilizing+hypoxia+factor+HIF-1%CE%B1+in+a+p53-independent+manner">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Xu T. </li><li class="ref__authors__name">Le T.D. </li><li class="ref__authors__name">Liu L. </li><li class="ref__authors__name">Su N. </li><li class="ref__authors__name">Wang R. </li><li class="ref__authors__name">Sun B. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Bontempi G. </li><li class="ref__authors__name">Li J. </li></ul><div class="ref__title">CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3131-3133</span><div class="ref__controls"><a href="#back-bib184" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85030680397&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28605519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtx378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3131-3133&journal=Bioinformatics&author=Xu+T.&author=Le+T.D.&author=Liu+L.&author=Su+N.&author=Wang+R.&author=Sun+B.&author=Colaprico+A.&author=Bontempi+G.&author=Li+J.&title=CancerSubtypes%3A+an+R%2FBioconductor+package+for+molecular+cancer+subtype+identification%2C+validation+and+visualization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Schulz M.H. </li><li class="ref__authors__name">Long Q. </li><li class="ref__authors__name">Apweiler R. </li><li class="ref__authors__name">Ning Z. </li></ul><div class="ref__title">Pindel: A pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span><div class="ref__controls"><a href="#back-bib207" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1065)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+A+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoo S. </li><li class="ref__authors__name">Huang T. </li><li class="ref__authors__name">Campbell J.D. </li><li class="ref__authors__name">Lee E. </li><li class="ref__authors__name">Tu Z. </li><li class="ref__authors__name">Geraci M.W. </li><li class="ref__authors__name">Powell C.A. </li><li class="ref__authors__name">Schadt E.E. </li><li class="ref__authors__name">Spira A. </li><li class="ref__authors__name">Zhu J. </li></ul><div class="ref__title">MODMatcher: multi-omics data matcher for integrative genomic analysis.</div><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span><div class="ref__controls"><a href="#back-bib185" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoshihara K. </li><li class="ref__authors__name">Shahmoradgoli M. </li><li class="ref__authors__name">Martínez E. </li><li class="ref__authors__name">Vegesna R. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Torres-Garcia W. </li><li class="ref__authors__name">Treviño V. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Levine D.A. </li><li>et al.</li></ul><div class="ref__title">Inferring tumour purity and stromal and immune cell admixture from expression data.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span><div class="ref__controls"><a href="#back-bib186" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (661)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Zimmerman L.J. </li><li class="ref__authors__name">Shaddox K.F. </li><li class="ref__authors__name">Kim S. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomic characterization of human colon and rectal cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span><div class="ref__controls"><a href="#back-bib188" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (628)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang W. </li><li class="ref__authors__name">Gao Y. </li><li class="ref__authors__name">Li P. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Guo T. </li><li class="ref__authors__name">Li F. </li><li class="ref__authors__name">Han X. </li><li class="ref__authors__name">Feng Y. </li><li class="ref__authors__name">Zheng C. </li><li class="ref__authors__name">Wang Z. </li><li>et al.</li></ul><div class="ref__title">VGLL4 functions as a new tumor suppressor in lung cancer by negatively regulating the YAP-TEAD transcriptional complex.</div><span class="ref__series"><i>Cell Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">331-343</span><div class="ref__controls"><a href="#back-bib189" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84897584171&amp;cf=" onclick="newWindow(this.href);return false">Scopus (144)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=24458094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fcr.2014.10&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=331-343&journal=Cell+Res.&author=Zhang+W.&author=Gao+Y.&author=Li+P.&author=Shi+Z.&author=Guo+T.&author=Li+F.&author=Han+X.&author=Feng+Y.&author=Zheng+C.&author=Wang+Z.&title=VGLL4+functions+as+a+new+tumor+suppressor+in+lung+cancer+by+negatively+regulating+the+YAP-TEAD+transcriptional+complex">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Liu T. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">McDermott J.E. </li><li class="ref__authors__name">Zhou J.Y. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Ray D. </li><li>et al.</li><li>CPTAC Investigators</li></ul><div class="ref__title">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span><div class="ref__controls"><a href="#back-bib190" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (305)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhou H.J. </li><li class="ref__authors__name">Xu Z. </li><li class="ref__authors__name">Wang Z. </li><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Zhuang Z.W. </li><li class="ref__authors__name">Simons M. </li><li class="ref__authors__name">Min W. </li></ul><div class="ref__title">SUMOylation of VEGFR2 regulates its intracellular trafficking and pathological angiogenesis.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">3303</span><div class="ref__controls"><a href="#back-bib191" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85051730064&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=30120232&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1038%2Fs41467-018-05812-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3303&journal=Nat.+Commun.&author=Zhou+H.J.&author=Xu+Z.&author=Wang+Z.&author=Zhang+H.&author=Zhuang+Z.W.&author=Simons+M.&author=Min+W.&title=SUMOylation+of+VEGFR2+regulates+its+intracellular+trafficking+and+pathological+angiogenesis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhu X. </li><li class="ref__authors__name">Ding S. </li><li class="ref__authors__name">Qiu C. </li><li class="ref__authors__name">Shi Y. </li><li class="ref__authors__name">Song L. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Li J. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Sun Y. </li><li>et al.</li></ul><div class="ref__title">SUMOylation negatively regulates angiogenesis by targeting endothelial NOTCH signaling.</div><span class="ref__series"><i>Circ. Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">121</span></span>: <span class="ref__seriesPages">636-649</span><div class="ref__controls"><a href="#back-bib192" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85028807516&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=28760777&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.1161%2FCIRCRESAHA.117.310696&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2017&pages=636-649&journal=Circ.+Res.&author=Zhu+X.&author=Ding+S.&author=Qiu+C.&author=Shi+Y.&author=Song+L.&author=Wang+Y.&author=Wang+Y.&author=Li+J.&author=Wang+Y.&author=Sun+Y.&title=SUMOylation+negatively+regulates+angiogenesis+by+targeting+endothelial+NOTCH+signaling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ziani L. </li><li class="ref__authors__name">Chouaib S. </li><li class="ref__authors__name">Thiery J. </li></ul><div class="ref__title">Alteration of the antitumor immune response by cancer-associated fibroblasts.</div><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">414</span><div class="ref__controls"><a href="#back-bib193" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.10.007&amp;key=2-s2.0-85042694394&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.10.007&amp;key=29545811&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.10.007&amp;key=10.3389%2Ffimmu.2018.00414&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=414&journal=Front.+Immunol.&author=Ziani+L.&author=Chouaib+S.&author=Thiery+J.&title=Alteration+of+the+antitumor+immune+response+by+cancer-associated+fibroblasts">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: October 31, 2019; corrected online December 20, 2020</div><div class="article-info__date">Accepted:
            October 7,
            2019</div><div class="article-info__date">Received in revised form:
            July 15,
            2019</div><div class="article-info__date">Received:
            February 5,
            2019</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.cell.2019.10.007">https://doi.org/10.1016/j.cell.2019.10.007</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2019 Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution (CC BY 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution (CC BY 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>
<li>Text &amp; data mine</li>
<li>Translate the article</li>
<li>Reuse portions or extracts from the article in other works</li>
<li>Sell or re-use for commercial purposes</li>
</ul>


<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S0092867419311237">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList"><h2 id="section-e587233e-22aa-4221-8ae1-22df489d46f7" class="top">Linked Article</h2><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(19)31388-1" title="Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma</a><div><ul class="meta__authors rlist--inline comma" title="list of authors">Clark et al.</ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">January 09, 2020</span></div><ul class="rlist rlist--inline linked-article__links clearfix"><li class="bulleted preview-label"><a href="#" data-db-target-for="related-article-in-brief-S0092-8674(19)31388-1" class="link--underlined" title="Preview Abstract">In Brief</a><div data-db-target-of="related-article-in-brief-S0092-8674(19)31388-1" class="dropBlock__holder__article-preview dropBlock__holder"><div class="dropBlock__body"><span>(Cell <em>179</em>, 964–983.e1–e31; October 31, 2019)</span></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></li><li class="bulleted"><a title="Full-Text" class="faded link--underlined" href="/cell/fulltext/S0092-8674(19)31388-1">
                Full-Text
            </a></li><li class="bulleted"><a title="PDF" class="link--underlined" target="_blank" href="https://www.cell.com/cell/pdf/S0092-8674(19)31388-1.pdf">
                    PDF
                </a></li></ul><span class="article-header__access">Open Access</span></li></ul></div></section>

            

        

    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867419311237" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)31123-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419311237%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)31123-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419311237%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1723231768730","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"179","issueNumber":"4","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867419311237","coverDate":"201910","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
