<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="117b32b2-fd3e-490f-8cf8-8965b5aa3aef"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;requestedJournal:journal:ccell;website:website:cell-site;ctype:string:Journal Content;pageGroup:string:Article View;page:string:Show Full Text;issue:issue:pii\:S1535610820X00043;wgroup:string:Default Website Group;subPage:string:Full Text;article:article:pii\:S1535610820306553;journal:journal:ccell"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cancer-cell/abstract/S1535-6108(20)30655-3" />
    



        <meta property="og:site_name" content="Cancer Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg" />
    


        <meta property="og:description" content="Huang et&nbsp;al. report a proteogenomic study on 108 HPV-negative head and neck squamous
cell carcinomas (HNSCCs). In addition to creating a comprehensive resource for pathogenic
insights, multi-omic analysis identifies therapeutic hypotheses that may inform more
precise approaches to treatment." />
    











        <meta name="twitter:title" content="Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg" />
    


        <meta name="twitter:description" content="Huang et&nbsp;al. report a proteogenomic study on 108 HPV-negative head and neck squamous
cell carcinomas (HNSCCs). In addition to creating a comprehensive resource for pathogenic
insights, multi-omic analysis identifies therapeutic hypotheses that may inform more
precise approaches to treatment." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(20)30655-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610820306553%3Fshowall%3Dtrue">
    


<title>Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma: Cancer Cell</title><meta name="citation_journal_title" content="Cancer Cell"/> <meta name="citation_journal_abbrev" content="Cancer Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma"/> <meta name="citation_date" content="2021/03/08"/> <meta name="citation_online_date" content="2021/01/07"/> <meta name="citation_volume" content="39"/> <meta name="citation_issue" content="3"/> <meta name="citation_firstpage" content="361"/> <meta name="citation_lastpage" content="379.e16"/> <meta name="citation_pmid" content="33417831"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S1535610820306553/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S1535610820306553/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S1535610820306553/pdf"/> <meta name="citation_issn" content="1535-6108"/> <meta name="citation_issn" content="1878-3686"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.ccell.2020.12.007"/> <meta name="citation_pmid" content="33417831"/> <meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Lijun Chen"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Sara R. Savage"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Rodrigo Vargas Eguez"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Yongchao Dou"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yize Li"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Felipe da Veiga Leprevost"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Eric J. Jaehnig"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Jonathan T. Lei"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Michael Schnaubelt"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Karsten Krug"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Xiaoyu Song"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Marcin Cieślik"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Hui-Yin Chang"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Matthew A. Wyczalkowski"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Kai Li"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Antonio Colaprico"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Qing Kay Li"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="David J. Clark"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Yingwei Hu"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Liwei Cao"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Jianbo Pan"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author_institution" content="Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Yuefan Wang"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Kyung-Cho Cho"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Zhiao Shi"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yuxing Liao"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Wen Jiang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Meenakshi Anurag"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Jiayi Ji"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Seungyeul Yoo"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Daniel Cui Zhou"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Wen-Wei Liang"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Michael Wendl"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Pankaj Vats"></meta><meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Jiang Qian"></meta><meta name="citation_author_institution" content="Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Xi S. Chen"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Alexander R. Pico"></meta><meta name="citation_author_institution" content="Institute of Data Science and Biotechnology, Gladstone Institutes, San Francisco, CA 94158, USA"></meta><meta name="citation_author" content="Pei Wang"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Arul M. Chinnaiyan"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author_institution" content="ESAC, Inc., Rockville, MD 20850, United States"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Eunkyung An"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick NaVonal Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Alissa M. Weaver"></meta><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Andrew G. Sikora"></meta><meta name="citation_author_institution" content="Department of Head and Neck Surgery, University of Texas M.D. Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Jan Lubiński"></meta><meta name="citation_author_institution" content="Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, 71-252 Szczecin, Poland"></meta><meta name="citation_author_institution" content="International Institute for Molecular Oncology, 60-203 Poznań, Poland"></meta><meta name="citation_author" content="Małgorzata Wierzbicka"></meta><meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland"></meta><meta name="citation_author_institution" content="Institute of Human Genetics Polish Academy of Sciences, 60-479 Poznań, Poland"></meta><meta name="citation_author" content="Maciej Wiznerowicz"></meta><meta name="citation_author_institution" content="International Institute for Molecular Oncology, 60-203 Poznań, Poland"></meta><meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland"></meta><meta name="citation_author" content="Shankha Satpathy"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02114, USA"></meta><meta name="citation_author" content="George Miles"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Matthew J. Ellis"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Gilbert S. Omenn"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Saravana M. Dhanasekaran"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Alexey I. Nesvizhskii"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Adel K. El-Naggar"></meta><meta name="citation_author_institution" content="Department of Pathology, Division of Pathology and Laboratory Medicine, MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author_email" content="dchan@jhmi.edu"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author_email" content="huizhang@jhu.edu"></meta><meta name="citation_author_institution" content="Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_email" content="bing.zhang@bcm.edu"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Anupriya Agarwal"></meta><meta name="citation_author" content="Matthew L. Anderson"></meta><meta name="citation_author" content="Shayan C. Avanessian"></meta><meta name="citation_author" content="Dmitry Avtonomov"></meta><meta name="citation_author" content="Oliver F. Bathe"></meta><meta name="citation_author" content="Chet Birger"></meta><meta name="citation_author" content="Michael J. Birrer"></meta><meta name="citation_author" content="Lili Blumenberg"></meta><meta name="citation_author" content="William E. Bocik"></meta><meta name="citation_author" content="Uma Borate"></meta><meta name="citation_author" content="Melissa Borucki"></meta><meta name="citation_author" content="Meghan C. Burke"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Anna Pamela Calinawan"></meta><meta name="citation_author" content="Sandra Cerda"></meta><meta name="citation_author" content="Alyssa Charamut"></meta><meta name="citation_author" content="Lin S. Chen"></meta><meta name="citation_author" content="Shrabanti Chowdhury"></meta><meta name="citation_author" content="Karl R. Clauser"></meta><meta name="citation_author" content="Houston Culpepper"></meta><meta name="citation_author" content="Tomasz Czernicki"></meta><meta name="citation_author" content="Fulvio D'Angelo"></meta><meta name="citation_author" content="Jacob Day"></meta><meta name="citation_author" content="Stephanie De Young"></meta><meta name="citation_author" content="Emek Demir"></meta><meta name="citation_author" content="Fei Ding"></meta><meta name="citation_author" content="Marcin J. Domagalski"></meta><meta name="citation_author" content="Joseph C. Dort"></meta><meta name="citation_author" content="Brian Druker"></meta><meta name="citation_author" content="Elizabeth Duffy"></meta><meta name="citation_author" content="Maureen Dyer"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Kimberly Elburn"></meta><meta name="citation_author" content="Tatiana S. Ermakova"></meta><meta name="citation_author" content="David Fenyo"></meta><meta name="citation_author" content="Renata Ferrarotto"></meta><meta name="citation_author" content="Alicia Francis"></meta><meta name="citation_author" content="Stacey Gabriel"></meta><meta name="citation_author" content="Luciano Garofano"></meta><meta name="citation_author" content="Yifat Geffen"></meta><meta name="citation_author" content="Gad Getz"></meta><meta name="citation_author" content="Charles A. Goldthwaite"></meta><meta name="citation_author" content="Linda I. Hannick"></meta><meta name="citation_author" content="Pushpa Hariharan"></meta><meta name="citation_author" content="David N. Hayes"></meta><meta name="citation_author" content="David Heiman"></meta><meta name="citation_author" content="Barbara Hindenach"></meta><meta name="citation_author" content="Katherine A. Hoadley"></meta><meta name="citation_author" content="Galen Hostetter"></meta><meta name="citation_author" content="Martin Hyrcza"></meta><meta name="citation_author" content="Scott D. Jewell"></meta><meta name="citation_author" content="Corbin D. Jones"></meta><meta name="citation_author" content="M. Harry Kane"></meta><meta name="citation_author" content="Alicia Karz"></meta><meta name="citation_author" content="Ramani B. Kothadia"></meta><meta name="citation_author" content="Azra Krek"></meta><meta name="citation_author" content="Chandan Kumar-Sinha"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author" content="Hongwei Liu"></meta><meta name="citation_author" content="Weiping Ma"></meta><meta name="citation_author" content="Ewa Malc"></meta><meta name="citation_author" content="Anna Malovannaya"></meta><meta name="citation_author" content="Sailaja Mareedu"></meta><meta name="citation_author" content="Sanford P. Markey"></meta><meta name="citation_author" content="Annette Marrero-Oliveras"></meta><meta name="citation_author" content="Nicollette Maunganidze"></meta><meta name="citation_author" content="Jason E. McDermott"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="John McGee"></meta><meta name="citation_author" content="Piotr Mieczkowski"></meta><meta name="citation_author" content="Simona Migliozzi"></meta><meta name="citation_author" content="Rebecca Montgomery"></meta><meta name="citation_author" content="Chelsea J. Newton"></meta><meta name="citation_author" content="Umut Ozbek"></meta><meta name="citation_author" content="Amanda G. Paulovich"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author" content="Dimitar Dimitrov Pazardzhikliev"></meta><meta name="citation_author" content="Amy M. Perou"></meta><meta name="citation_author" content="Francesca Petralia"></meta><meta name="citation_author" content="Lyudmila Petrenko"></meta><meta name="citation_author" content="Paul D. Piehowski"></meta><meta name="citation_author" content="Dmitris Placantonakis"></meta><meta name="citation_author" content="Larisa Polonskaya"></meta><meta name="citation_author" content="Elena V. Ponomareva"></meta><meta name="citation_author" content="Olga Potapova"></meta><meta name="citation_author" content="Liqun Qi"></meta><meta name="citation_author" content="Ning Qu"></meta><meta name="citation_author" content="Shakti Ramkissoon"></meta><meta name="citation_author" content="Boris Reva"></meta><meta name="citation_author" content="Shannon Richey"></meta><meta name="citation_author" content="Karna Robinson"></meta><meta name="citation_author" content="Nancy Roche"></meta><meta name="citation_author" content="Karin Rodland"></meta><meta name="citation_author" content="Daniel C. Rohrer"></meta><meta name="citation_author" content="Dmitry Rykunov"></meta><meta name="citation_author" content="Eric E. Schadt"></meta><meta name="citation_author" content="Yan Shi"></meta><meta name="citation_author" content="Yvonne Shutack"></meta><meta name="citation_author" content="Shilpi Singh"></meta><meta name="citation_author" content="Tara Skelly"></meta><meta name="citation_author" content="Richard Smith"></meta><meta name="citation_author" content="Lori J. Sokoll"></meta><meta name="citation_author" content="Jakub Stawicki"></meta><meta name="citation_author" content="Stephen E. Stein"></meta><meta name="citation_author" content="James Suh"></meta><meta name="citation_author" content="Wojciech Szopa"></meta><meta name="citation_author" content="Dave Tabor"></meta><meta name="citation_author" content="Donghui Tan"></meta><meta name="citation_author" content="Darlene Tansil"></meta><meta name="citation_author" content="Guo Ci Teo"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Cristina Tognon"></meta><meta name="citation_author" content="Elie Traer"></meta><meta name="citation_author" content="Shirley Tsang"></meta><meta name="citation_author" content="Jeffrey Tyner"></meta><meta name="citation_author" content="Ki Sung Um"></meta><meta name="citation_author" content="Dana R. Valley"></meta><meta name="citation_author" content="Lyubomir Valkov Vasilev"></meta><meta name="citation_author" content="Negin Vatanian"></meta><meta name="citation_author" content="Uma Velvulou"></meta><meta name="citation_author" content="Michael Vernon"></meta><meta name="citation_author" content="Thomas F. Westbrook"></meta><meta name="citation_author" content="Jeffrey R. Whiteaker"></meta><meta name="citation_author" content="Yige Wu"></meta><meta name="citation_author" content="Midie Xu"></meta><meta name="citation_author" content="Lijun Yao"></meta><meta name="citation_author" content="Xinpei Yi"></meta><meta name="citation_author" content="Fengchao Yu"></meta><meta name="citation_author" content="Kakhaber Zaalishvili"></meta><meta name="citation_author" content="Yuriy Zakhartsev"></meta><meta name="citation_author" content="Robert Zelt"></meta><meta name="citation_author" content="Grace Zhao"></meta><meta name="citation_author" content="Jun Zhu"></meta><meta name="citation_keywords" content="HNSCC; proteogenomics; proteomics; immunotherapy; EGFR mAb; immune evasion; combination immune checkpoint blockade; CPTAC; CDK inhibitor; predictive biomarker"></meta><meta name="citation_reference" content="citation_title=Palbociclib and cetuximab in platinum-resistant and in cetuximab-resistant human papillomavirus-unrelated head and neck cancer: a multicentre, multigroup, phase 2 trial; citation_author=D. Adkins; citation_author=J. Ley; citation_author=P. Neupane; citation_author=F. Worden; citation_author=A.G. Sacco; citation_author=K. Palka; citation_author=J.E. Grilley-Olson; citation_author=R. Maggiore; citation_author=N.N. Salama; citation_author=K. Trinkaus; citation_author=et al.; citation_journal_title=Lancet Oncol.; citation_volume=20; citation_firstpage=1295; citation_lastpage=1305; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens; citation_author=L.G. Almeida; citation_author=N.J. Sakabe; citation_author=A.R. deOliveira; citation_author=M.C.C. Silva; citation_author=A.S. Mundstein; citation_author=T. Cohen; citation_author=Y.-T. Chen; citation_author=R. Chua; citation_author=S. Gurung; citation_author=S. Gnjatic; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=37; citation_firstpage=D816; citation_lastpage=D819; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Functional characterization of somatic mutations in cancer using network-based inference of protein activity; citation_author=M.J. Alvarez; citation_author=Y. Shen; citation_author=F.M. Giorgi; citation_author=A. Lachmann; citation_author=B.B. Ding; citation_author=B.H. Ye; citation_author=A. Califano; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=838; citation_lastpage=847; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Randomized phase III trial of concurrent accelerated radiation plus cisplatin with or without cetuximab for stage III to IV head and neck carcinoma: RTOG 0522; citation_author=K.K. Ang; citation_author=Q. Zhang; citation_author=D.I. Rosenthal; citation_author=P.F. Nguyen-Tan; citation_author=E.J. Sherman; citation_author=R.S. Weber; citation_author=J.M. Galvin; citation_author=J.A. Bonner; citation_author=J. Harris; citation_author=A.K. El-Naggar; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=32; citation_firstpage=2940; citation_lastpage=2950; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=xCell: digitally portraying the tissue cellular heterogeneity landscape; citation_author=D. Aran; citation_author=Z. Hu; citation_author=A.J. Butte; citation_journal_title=Genome Biol.; citation_volume=18; citation_firstpage=220; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=moFF: a robust and automated approach to extract peptide ion intensities; citation_author=A. Argentini; citation_author=L.J.E. Goeminne; citation_author=K. Verheggen; citation_author=N. Hulstaert; citation_author=A. Staes; citation_author=L. Clement; citation_author=L. Martens; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=964; citation_lastpage=966; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Recurrent chimeric fusion RNAs in non-cancer tissues and cells; citation_author=M. Babiceanu; citation_author=F. Qin; citation_author=Z. Xie; citation_author=Y. Jia; citation_author=K. Lopez; citation_author=N. Janus; citation_author=L. Facemire; citation_author=S. Kumar; citation_author=Y. Pang; citation_author=Y. Qi; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=2859; citation_lastpage=2872; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1; citation_author=D.A. Barbie; citation_author=P. Tamayo; citation_author=J.S. Boehm; citation_author=S.Y. Kim; citation_author=S.E. Moody; citation_author=I.F. Dunn; citation_author=A.C. Schinzel; citation_author=P. Sandy; citation_author=E. Meylan; citation_author=C. Scholl; citation_author=et al.; citation_journal_title=Nature; citation_volume=462; citation_firstpage=108; citation_lastpage=112; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Phase II multicenter study of the antiepidermal growth factor receptor monoclonal antibody cetuximab in combination with platinum-based chemotherapy in patients with platinum-refractory metastatic and/or recurrent squamous cell carcinoma of the head and neck; citation_author=J. Baselga; citation_author=J.M. Trigo; citation_author=J. Bourhis; citation_author=J. Tortochaux; citation_author=H. Cortés-Funes; citation_author=R. Hitt; citation_author=P. Gascón; citation_author=N. Amellal; citation_author=A. Harstrick; citation_author=A. Eckardt; citation_journal_title=J. Clin. Oncol.; citation_volume=23; citation_firstpage=5568; citation_lastpage=5577; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens; citation_author=F.M. Behan; citation_author=F. Iorio; citation_author=G. Picco; citation_author=E. Gonçalves; citation_author=C.M. Beaver; citation_author=G. Migliardi; citation_author=R. Santos; citation_author=Y. Rao; citation_author=F. Sassi; citation_author=M. Pinnelli; citation_author=et al.; citation_journal_title=Nature; citation_volume=568; citation_firstpage=511; citation_lastpage=516; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=On the approximation of curves by line segments using dynamic programming; citation_author=R. Bellman; citation_journal_title=Commun. ACM; citation_volume=4; citation_firstpage=284; citation_publication_date=1961; "></meta><meta name="citation_reference" content="citation_title=Polo-like kinase 1 inhibits DNA damage response during mitosis; citation_author=J. Benada; citation_author=K. Burdová; citation_author=T. Lidak; citation_author=P. von Morgen; citation_author=L. Macurek; citation_journal_title=Cell Cycle; citation_volume=14; citation_firstpage=219; citation_lastpage=231; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript; citation_author=M. Benelli; citation_author=C. Pescucci; citation_author=G. Marseglia; citation_author=M. Severgnini; citation_author=F. Torricelli; citation_author=A. Magi; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=3232; citation_lastpage=3239; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J. R. Stat. Soc. Ser. B Methodol.; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "></meta><meta name="citation_reference" content="citation_title=CHFR promoter hypermethylation in colon cancer correlates with the microsatellite instability phenotype; citation_author=J.C. Brandes; citation_author=M. van Engeland; citation_author=K.A.D. Wouters; citation_author=M.P. Weijenberg; citation_author=J.G. Herman; citation_journal_title=Carcinogenesis; citation_volume=26; citation_firstpage=1152; citation_lastpage=1156; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries; citation_author=F. Bray; citation_author=J. Ferlay; citation_author=I. Soerjomataram; citation_author=R.L. Siegel; citation_author=L.A. Torre; citation_author=A. Jemal; citation_journal_title=Cancer J. Clin.; citation_volume=68; citation_firstpage=394; citation_lastpage=424; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Optimization of experimental parameters in data-independent mass spectrometry significantly increases depth and reproducibility of results; citation_author=R. Bruderer; citation_author=O.M. Bernhardt; citation_author=T. Gandhi; citation_author=Y. Xuan; citation_author=J. Sondermann; citation_author=M. Schmidt; citation_author=D. Gomez-Varela; citation_author=L. Reiter; citation_journal_title=Mol. Cell. Proteomics; citation_volume=16; citation_firstpage=2296; citation_lastpage=2309; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Metagenes and molecular pattern discovery using matrix factorization; citation_author=J.-P. Brunet; citation_author=P. Tamayo; citation_author=T.R. Golub; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. U S A; citation_volume=101; citation_firstpage=4164; citation_lastpage=4169; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Phase III randomized trial of cisplatin plus placebo compared with cisplatin plus cetuximab in metastatic/recurrent head and neck cancer: an Eastern Cooperative Oncology Group study; citation_author=B. Burtness; citation_author=M.A. Goldwasser; citation_author=W. Flood; citation_author=B. Mattar; citation_author=A.A. Forastiere; citation_author=G. Eastern Cooperative Oncology; citation_journal_title=J. Clin. Oncol.; citation_volume=23; citation_firstpage=8646; citation_lastpage=8654; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=A test metric for assessing single-cell RNA-seq batch correction; citation_author=M. Büttner; citation_author=Z. Miao; citation_author=F.A. Wolf; citation_author=S.A. Teichmann; citation_author=F.J. Theis; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=43; citation_lastpage=49; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Epigenetic silencing of neurofilament genes promotes an aggressive phenotype in breast cancer; citation_author=M.F. Calmon; citation_author=J. Jeschke; citation_author=W. Zhang; citation_author=M. Dhir; citation_author=C. Siebenkäs; citation_author=A. Herrera; citation_author=H.-C. Tsai; citation_author=H.M. O'Hagan; citation_author=E.P. Pappou; citation_author=C.M. Hooker; citation_author=et al.; citation_journal_title=Epigenetics; citation_volume=10; citation_firstpage=622; citation_lastpage=632; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Comprehensive genomic characterization of head and neck squamous cell carcinomas; citation_author=N. Cancer Genome Atlas; citation_journal_title=Nature; citation_volume=517; citation_firstpage=576; citation_lastpage=582; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Divergent viral presentation among human tumors and adjacent normal tissues; citation_author=S. Cao; citation_author=M.C. Wendl; citation_author=M.A. Wyczalkowski; citation_author=K. Wylie; citation_author=K. Ye; citation_author=R. Jayasinghe; citation_author=M. Xie; citation_author=S. Wu; citation_author=B. Niu; citation_author=R. Grubb; citation_author=et al.; citation_journal_title=Sci. Rep.; citation_volume=6; citation_firstpage=28294; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data; citation_author=E. Cerami; citation_author=J. Gao; citation_author=U. Dogrusoz; citation_author=B.E. Gross; citation_author=S.O. Sumer; citation_author=B.A. Aksoy; citation_author=A. Jacobsen; citation_author=C.J. Byrne; citation_author=M.L. Heuer; citation_author=E. Larsson; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=2; citation_firstpage=401; citation_lastpage=404; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=MS-PyCloud: an open-source, cloud computing-based pipeline for LC-MS/MS data analysis; citation_author=L. Chen; citation_author=B. Zhang; citation_author=M. Schnaubelt; citation_author=P. Shah; citation_author=P. Aiyetan; citation_author=D. Chan; citation_author=H. Zhang; citation_author=Z. Zhang; citation_journal_title=bioRxiv; citation_firstpage=320887; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications; citation_author=X. Chen; citation_author=O. Schulz-Trieglaff; citation_author=R. Shaw; citation_author=B. Barnes; citation_author=F. Schlesinger; citation_author=M. Källberg; citation_author=A.J. Cox; citation_author=S. Kruglyak; citation_author=C.T. Saunders; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=1220; citation_lastpage=1222; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Large-scale profiling of microRNAs for the cancer genome Atlas; citation_author=A. Chu; citation_author=G. Robertson; citation_author=D. Brooks; citation_author=A.J. Mungall; citation_author=I. Birol; citation_author=R. Coope; citation_author=Y. Ma; citation_author=S. Jones; citation_author=M.A. Marra; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e3; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Molecular classification of head and neck squamous cell carcinomas using patterns of gene expression; citation_author=C.H. Chung; citation_author=J.S. Parker; citation_author=G. Karaca; citation_author=J. Wu; citation_author=W.K. Funkhouser; citation_author=D. Moore; citation_author=D. Butterfoss; citation_author=D. Xiang; citation_author=A. Zanation; citation_author=X. Yin; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=5; citation_firstpage=489; citation_lastpage=500; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Mutual exclusivity analysis identifies oncogenic network modules; citation_author=G. Ciriello; citation_author=E. Cerami; citation_author=C. Sander; citation_author=N. Schultz; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=398; citation_lastpage=406; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Integrated proteogenomic characterization of clear cell renal cell carcinoma; citation_author=D.J. Clark; citation_author=S.M. Dhanasekaran; citation_author=F. Petralia; citation_author=J. Pan; citation_author=X. Song; citation_author=Y. Hu; citation_author=F. da Veiga Leprevost; citation_author=B. Reva; citation_author=T.-S.M. Lih; citation_author=H.-Y. Chang; citation_author=et al.; citation_journal_title=Cell; citation_volume=179; citation_firstpage=964; citation_lastpage=983.e931; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Evaluation of NCI-7 cell line panel as a reference material for clinical proteomics; citation_author=D.J. Clark; citation_author=Y. Hu; citation_author=W. Bocik; citation_author=L. Chen; citation_author=M. Schnaubelt; citation_author=R. Roberts; citation_author=P. Shah; citation_author=G. Whiteley; citation_author=H. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=17; citation_firstpage=2205; citation_lastpage=2215; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Interpreting pathways to discover cancer driver genes with Moonlight; citation_author=A. Colaprico; citation_author=C. Olsen; citation_author=M.H. Bailey; citation_author=G.J. Odom; citation_author=T. Terkelsen; citation_author=T.C. Silva; citation_author=A.V. Olsen; citation_author=L. Cantini; citation_author=A. Zinovyev; citation_author=E. Barillot; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=69; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Use of the humanized anti-epidermal growth factor receptor monoclonal antibody h-R3 in combination with radiotherapy in the treatment of locally advanced head and neck cancer patients; citation_author=T. Crombet; citation_author=M. Osorio; citation_author=T. Cruz; citation_author=C. Roca; citation_author=R. del Castillo; citation_author=R. Mon; citation_author=N. Iznaga-Escobar; citation_author=R. Figueredo; citation_author=J. Koropatnick; citation_author=E. Renginfo; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=22; citation_firstpage=1646; citation_lastpage=1654; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives; citation_author=K. Daily; citation_author=S.J. Ho Sui; citation_author=L.M. Schriml; citation_author=P.J. Dexheimer; citation_author=N. Salomonis; citation_author=R. Schroll; citation_author=S. Bush; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=S. Lotia; citation_author=et al.; citation_journal_title=Sci. Data; citation_volume=4; citation_firstpage=170030; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Plk1 regulates contraction of postmitotic smooth muscle cells and is required for vascular homeostasis; citation_author=G. de Cárcer; citation_author=P. Wachowicz; citation_author=S. Martínez-Martínez; citation_author=J. Oller; citation_author=N. Méndez-Barbero; citation_author=B. Escobar; citation_author=A. González-Loyola; citation_author=T. Takaki; citation_author=A. El Bakkali; citation_author=J.A. Cámara; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=23; citation_firstpage=964; citation_lastpage=974; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics; citation_author=E.W. Deutsch; citation_author=L. Mendoza; citation_author=D. Shteynberg; citation_author=J. Slagel; citation_author=Z. Sun; citation_author=R.L. Moritz; citation_journal_title=Proteomics Clin. Appl.; citation_volume=9; citation_firstpage=745; citation_lastpage=754; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=NAA80 is actin's N-terminal acetyltransferase and regulates cytoskeleton assembly and cell motility; citation_author=A. Drazic; citation_author=H. Aksnes; citation_author=M. Marie; citation_author=M. Boczkowska; citation_author=S. Varland; citation_author=E. Timmerman; citation_author=H. Foyn; citation_author=N. Glomnes; citation_author=G. Rebowski; citation_author=F. Impens; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. U S A; citation_volume=115; citation_firstpage=4399; citation_lastpage=4404; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Ki67 proliferation index as a tool for chemotherapy decisions during and after neoadjuvant aromatase inhibitor treatment of breast cancer: results from the American College of Surgeons Oncology Group Z1031 trial (alliance); citation_author=M.J. Ellis; citation_author=V.J. Suman; citation_author=J. Hoog; citation_author=R. Goncalves; citation_author=S. Sanati; citation_author=C.J. Creighton; citation_author=K. DeSchryver; citation_author=E. Crouch; citation_author=A. Brink; citation_author=M. Watson; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=35; citation_firstpage=1061; citation_lastpage=1069; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Durvalumab with or without tremelimumab in patients with recurrent or metastatic head and neck squamous cell carcinoma: EAGLE, a randomized, open-label phase III study; citation_author=R.L. Ferris; citation_author=R. Haddad; citation_author=C. Even; citation_author=M. Tahara; citation_author=M. Dvorkin; citation_author=T.E. Ciuleanu; citation_author=P.M. Clement; citation_author=R. Mesia; citation_author=S. Kutukova; citation_author=L. Zholudeva; citation_author=et al.; citation_journal_title=Ann. Oncol.; citation_volume=31; citation_firstpage=942; citation_lastpage=950; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries; citation_author=S. Fisher; citation_author=A. Barry; citation_author=J. Abreu; citation_author=B. Minie; citation_author=J. Nolan; citation_author=T.M. Delorey; citation_author=G. Young; citation_author=T.J. Fennell; citation_author=A. Allen; citation_author=L. Ambrogio; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R1; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Functional normalization of 450k methylation array data improves replication in large cancer studies; citation_author=J.-P. Fortin; citation_author=A. Labbe; citation_author=M. Lemire; citation_author=B.W. Zanke; citation_author=T.J. Hudson; citation_author=E.J. Fertig; citation_author=C.M. Greenwood; citation_author=K.D. Hansen; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=503; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Clustering by passing messages between data points; citation_author=B.J. Frey; citation_author=D. Dueck; citation_journal_title=Science (New York, NY); citation_volume=315; citation_firstpage=972; citation_lastpage=976; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal; citation_author=J. Gao; citation_author=B.A. Aksoy; citation_author=U. Dogrusoz; citation_author=G. Dresdner; citation_author=B. Gross; citation_author=S.O. Sumer; citation_author=Y. Sun; citation_author=A. Jacobsen; citation_author=R. Sinha; citation_author=E. Larsson; citation_author=et al.; citation_journal_title=Sci. Signal.; citation_volume=6; citation_firstpage=pl1; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Driver fusions and their implications in the development and treatment of human cancers; citation_author=Q. Gao; citation_author=W.-W. Liang; citation_author=S.M. Foltz; citation_author=G. Mutharasu; citation_author=R.G. Jayasinghe; citation_author=S. Cao; citation_author=W.-W. Liao; citation_author=S.M. Reynolds; citation_author=M.A. Wyczalkowski; citation_author=L. Yao; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=23; citation_firstpage=227; citation_lastpage=238.e223; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=CIRI: an efficient and unbiased algorithm for de novo circular RNA identification; citation_author=Y. Gao; citation_author=J. Wang; citation_author=F. Zhao; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=4; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Benchmark and integration of resources for the estimation of human transcription factor activities; citation_author=L. Garcia-Alonso; citation_author=C.H. Holland; citation_author=M.M. Ibrahim; citation_author=D. Turei; citation_author=J. Saez-Rodriguez; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=1363; citation_lastpage=1375; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=A flexible R package for nonnegative matrix factorization; citation_author=R. Gaujoux; citation_author=C. Seoighe; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=367; citation_publication_date=2010; "></meta><meta name="citation_reference" content="Gey, S., and Lebarbier, E. (2008). Using CART to Detect Multiple Change Points in the Mean for Large Sample, https://hal.archives-ouvertes.fr/hal-00327146/."></meta><meta name="citation_reference" content="citation_title=Tobacco and alcohol-induced epigenetic changes in oral carcinoma; citation_author=Y. Ghantous; citation_author=J.L. Schussel; citation_author=M. Brait; citation_journal_title=Curr. Opin. Oncol.; citation_volume=30; citation_firstpage=152; citation_lastpage=158; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods; citation_author=B.J. Haas; citation_author=A. Dobin; citation_author=B. Li; citation_author=N. Stransky; citation_author=N. Pochet; citation_author=A. Regev; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=213; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=GSVA: gene set variation analysis for microarray and RNA-seq data; citation_author=S. Hänzelmann; citation_author=R. Castelo; citation_author=J. Guinney; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=7; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Cetuximab: an epidermal growth factor receptor chimeric human-murine monoclonal antibody; citation_author=J. Harding; citation_author=B. Burtness; citation_journal_title=Drugs Today; citation_volume=41; citation_firstpage=107; citation_lastpage=127; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Distinct kinetics of serine and threonine dephosphorylation are essential for mitosis; citation_author=J.B. Hein; citation_author=E.P.T. Hertz; citation_author=D.H. Garvanska; citation_author=T. Kruse; citation_author=J. Nilsson; citation_journal_title=Nat. Cell Biol.; citation_volume=19; citation_firstpage=1433; citation_lastpage=1440; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Phase II multicenter study of the epidermal growth factor receptor antibody cetuximab and cisplatin for recurrent and refractory squamous cell carcinoma of the head and neck; citation_author=R.S. Herbst; citation_author=M. Arquette; citation_author=D.M. Shin; citation_author=K. Dicke; citation_author=E.E. Vokes; citation_author=N. Azarnia; citation_author=W.K. Hong; citation_author=M.S. Kies; citation_journal_title=J. Clin. Oncol.; citation_volume=23; citation_firstpage=5578; citation_lastpage=5587; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations; citation_author=P.V. Hornbeck; citation_author=B. Zhang; citation_author=B. Murray; citation_author=J.M. Kornhauser; citation_author=V. Latham; citation_author=E. Skrzypek; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D512; citation_lastpage=D520; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=NetMHCpan-4.0: improved peptide-MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data; citation_author=V. Jurtz; citation_author=S. Paul; citation_author=M. Andreatta; citation_author=P. Marcatili; citation_author=B. Peters; citation_author=M. Nielsen; citation_journal_title=J. Immunol.; citation_volume=199; citation_firstpage=3360; citation_lastpage=3368; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Oncogenic activity of Ect2 is regulated through protein kinase C iota-mediated phosphorylation; citation_author=V. Justilien; citation_author=L. Jameison; citation_author=C.J. Der; citation_author=K.L. Rossman; citation_author=A.P. Fields; citation_journal_title=J. Biol. Chem.; citation_volume=286; citation_firstpage=8149; citation_lastpage=8157; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Patient-derived xenografts for prognostication and personalized treatment for head and neck squamous cell carcinoma; citation_author=C. Karamboulas; citation_author=J.P. Bruce; citation_author=A.J. Hope; citation_author=J. Meens; citation_author=S.H. Huang; citation_author=N. Erdmann; citation_author=E. Hyatt; citation_author=K. Pereira; citation_author=D.P. Goldstein; citation_author=I. Weinreb; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=25; citation_firstpage=1318; citation_lastpage=1331.e1314; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution; citation_author=S. Kasar; citation_author=J. Kim; citation_author=R. Improgo; citation_author=G. Tiao; citation_author=P. Polak; citation_author=N. Haradhvala; citation_author=M.S. Lawrence; citation_author=A. Kiezun; citation_author=S.M. Fernandes; citation_author=S. Bahl; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=8866; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Integrative analysis of head and neck cancer identifies two biologically distinct HPV and three non-HPV subtypes; citation_author=M.K. Keck; citation_author=Z. Zuo; citation_author=A. Khattri; citation_author=T.P. Stricker; citation_author=C.D. Brown; citation_author=M. Imanguli; citation_author=D. Rieke; citation_author=K. Endhardt; citation_author=P. Fang; citation_author=J. Brägelmann; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=21; citation_firstpage=870; citation_lastpage=881; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search; citation_author=A. Keller; citation_author=A.I. Nesvizhskii; citation_author=E. Kolker; citation_author=R. Aebersold; citation_journal_title=Anal Chem.; citation_volume=74; citation_firstpage=5383; citation_lastpage=5392; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors; citation_author=J. Kim; citation_author=K.W. Mouw; citation_author=P. Polak; citation_author=L.Z. Braunstein; citation_author=A. Kamburov; citation_author=D.J. Kwiatkowski; citation_author=J.E. Rosenberg; citation_author=E.M. Van Allen; citation_author=A. D'Andrea; citation_author=G. Getz; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=600; citation_lastpage=606; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Strelka2: fast and accurate calling of germline and somatic variants; citation_author=S. Kim; citation_author=K. Scheffler; citation_author=A.L. Halpern; citation_author=M.A. Bekritsky; citation_author=E. Noh; citation_author=M. Källberg; citation_author=X. Chen; citation_author=Y. Kim; citation_author=D. Beyter; citation_author=P. Krusche; citation_author=C.T. Saunders; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=591; citation_lastpage=594; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Basal subtype is predictive for response to cetuximab treatment in patient-derived xenografts of squamous cell head and neck cancer; citation_author=K. Klinghammer; citation_author=R. Otto; citation_author=J.-D. Raguse; citation_author=A.E. Albers; citation_author=I. Tinhofer; citation_author=I. Fichtner; citation_author=U. Leser; citation_author=U. Keilholz; citation_author=J. Hoffmann; citation_journal_title=Int. J. Cancer; citation_volume=141; citation_firstpage=1215; citation_lastpage=1221; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The body language of cells: the intimate connection between cell adhesion and behavior; citation_author=M.W. Klymkowsky; citation_author=B. Parr; citation_journal_title=Cell; citation_volume=83; citation_firstpage=5; citation_lastpage=8; citation_publication_date=1995; "></meta><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=D.C. Koboldt; citation_author=Q. Zhang; citation_author=D.E. Larson; citation_author=D. Shen; citation_author=M.D. McLellan; citation_author=L. Lin; citation_author=C.A. Miller; citation_author=E.R. Mardis; citation_author=L. Ding; citation_author=R.K. Wilson; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=568; citation_lastpage=576; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics; citation_author=A.T. Kong; citation_author=F.V. Leprevost; citation_author=D.M. Avtonomov; citation_author=D. Mellacheruvu; citation_author=A.I. Nesvizhskii; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=513; citation_lastpage=520; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A synthetic lethal screen identifies FAT1 as an antagonist of caspase-8 in extrinsic apoptosis; citation_author=D. Kranz; citation_author=M. Boutros; citation_journal_title=EMBO J.; citation_volume=33; citation_firstpage=181; citation_lastpage=197; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Human papillomavirus types in head and neck squamous cell carcinomas worldwide: a systematic review; citation_author=A.R. Kreimer; citation_author=G.M. Clifford; citation_author=P. Boyle; citation_author=S. Franceschi; citation_journal_title=Cancer Epidemiol. Biomarkers Prev.; citation_volume=14; citation_firstpage=467; citation_lastpage=475; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=lmerTest package: tests in linear mixed effects models; citation_author=A. Kuznetsova; citation_author=P.B. Brockhoff; citation_author=R.H.B. Christensen; citation_journal_title=J. Stat. Softw.; citation_volume=82; citation_firstpage=1; citation_lastpage=26; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=ARACNe-AP: gene network reverse engineering through adaptive partitioning inference of mutual information; citation_author=A. Lachmann; citation_author=F.M. Giorgi; citation_author=G. Lopez; citation_author=A. Califano; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=2233; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B. Langmead; citation_author=C. Trapnell; citation_author=M. Pop; citation_author=S.L. Salzberg; citation_journal_title=Genome Biol.; citation_volume=10; citation_firstpage=R25; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Detecting multiple change-points in the mean of Gaussian process by model selection; citation_author=E. Lebarbier; citation_journal_title=Signal Process.; citation_volume=85; citation_firstpage=717; citation_lastpage=736; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-seq data with or without a reference genome; citation_author=B. Li; citation_author=C.N. Dewey; citation_journal_title=BMC Bioinformatics; citation_volume=12; citation_firstpage=323; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=S. Genome Project Data Processing; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=PDV: an integrative proteomics data viewer; citation_author=K. Li; citation_author=M. Vaudel; citation_author=B. Zhang; citation_author=Y. Ren; citation_author=B. Wen; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=1249; citation_lastpage=1251; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Quantifying circular RNA expression from RNA-seq data using model-based framework; citation_author=M. Li; citation_author=X. Xie; citation_author=J. Zhou; citation_author=M. Sheng; citation_author=X. Yin; citation_author=E.-A. Ko; citation_author=T. Zhou; citation_author=W. Gu; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=2131; citation_lastpage=2139; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs; citation_author=Y. Liao; citation_author=J. Wang; citation_author=E.J. Jaehnig; citation_author=Z. Shi; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=W199; citation_lastpage=W205; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=The rules and impact of nonsense-mediated mRNA decay in human cancers; citation_author=R.G.H. Lindeboom; citation_author=F. Supek; citation_author=B. Lehner; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=1112; citation_lastpage=1118; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Systematic discovery of in vivo phosphorylation networks; citation_author=R. Linding; citation_author=L.J. Jensen; citation_author=G.J. Ostheimer; citation_author=M.A.T.M. van Vugt; citation_author=C. Jørgensen; citation_author=I.M. Miron; citation_author=F. Diella; citation_author=K. Colwill; citation_author=L. Taylor; citation_author=K. Elder; citation_author=et al.; citation_journal_title=Cell; citation_volume=129; citation_firstpage=1415; citation_lastpage=1426; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=NetworKIN: a resource for exploring cellular phosphorylation networks; citation_author=R. Linding; citation_author=L.J. Jensen; citation_author=A. Pasculescu; citation_author=M. Olhovsky; citation_author=K. Colwill; citation_author=P. Bork; citation_author=M.B. Yaffe; citation_author=T. Pawson; citation_journal_title=Nucleic Acids Res.; citation_volume=36; citation_firstpage=D695; citation_lastpage=D699; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Regulation of Chk2 ubiquitination and signaling through autophosphorylation of serine 379; citation_author=C.M. Lovly; citation_author=L. Yan; citation_author=C.E. Ryan; citation_author=S. Takada; citation_author=H. Piwnica-Worms; citation_journal_title=Mol. Cell Biol.; citation_volume=28; citation_firstpage=5874; citation_lastpage=5885; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Cortactin in cell migration and cancer at a glance; citation_author=S.M. MacGrath; citation_author=A.J. Koleske; citation_journal_title=J. Cell Sci; citation_volume=125; citation_firstpage=1621; citation_lastpage=1626; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Machine learning identifies stemness features associated with oncogenic dedifferentiation; citation_author=T.M. Malta; citation_author=A. Sokolov; citation_author=A.J. Gentles; citation_author=T. Burzykowski; citation_author=L. Poisson; citation_author=J.N. Weinstein; citation_author=B. Kamińska; citation_author=J. Huelsken; citation_author=L. Omberg; citation_author=O. Gevaert; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=338; citation_lastpage=354.e315; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Class II PI3Ks at the intersection between signal transduction and membrane trafficking; citation_author=J.P. Margaria; citation_author=E. Ratto; citation_author=L. Gozzelino; citation_author=H. Li; citation_author=E. Hirsch; citation_journal_title=Biomolecules; citation_volume=9; citation_firstpage=104; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Assembly and activation of the Hippo signalome by FAT1 tumor suppressor; citation_author=D. Martin; citation_author=M.S. Degese; citation_author=L. Vitale-Cross; citation_author=R. Iglesias-Bartolome; citation_author=J.L.C. Valera; citation_author=Z. Wang; citation_author=X. Feng; citation_author=H. Yeerna; citation_author=V. Vadmal; citation_author=T. Moroishi; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=2372; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration; citation_author=J.M. McFarland; citation_author=Z.V. Ho; citation_author=G. Kugener; citation_author=J.M. Dempster; citation_author=P.G. Montgomery; citation_author=J.G. Bryan; citation_author=J.M. Krill-Burger; citation_author=T.M. Green; citation_author=F. Vazquez; citation_author=J.S. Boehm; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=4610; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=M.A. DePristo; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Integrated proteomic analysis of post-translational modifications by serial enrichment; citation_author=P. Mertins; citation_author=J.W. Qiao; citation_author=J. Patel; citation_author=N.D. Udeshi; citation_author=K.R. Clauser; citation_author=D.R. Mani; citation_author=M.W. Burgess; citation_author=M.A. Gillette; citation_author=J.D. Jaffe; citation_author=S.A. Carr; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=634; citation_lastpage=637; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry; citation_author=P. Mertins; citation_author=L.C. Tang; citation_author=K. Krug; citation_author=D.J. Clark; citation_author=M.A. Gritsenko; citation_author=L. Chen; citation_author=K.R. Clauser; citation_author=T.R. Clauss; citation_author=P. Shah; citation_author=M.A. Gillette; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=13; citation_firstpage=1632; citation_lastpage=1661; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Panitumumab, a monoclonal anti epidermal growth factor receptor antibody in colorectal cancer: another one or the one?; citation_author=W.A. Messersmith; citation_author=M. Hidalgo; citation_journal_title=Clin. Cancer Res.; citation_volume=13; citation_firstpage=4664; citation_lastpage=4666; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Epigenetic silencing of the putative tumor suppressor gene GLDC (glycine dehydrogenase) in gastric carcinoma; citation_author=H.L. Min; citation_author=J. Kim; citation_author=W.H. Kim; citation_author=B.G. Jang; citation_author=M.A. Kim; citation_journal_title=Anticancer Res.; citation_volume=36; citation_firstpage=179; citation_lastpage=187; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx; citation_author=M. Mounir; citation_author=M. Lucchetta; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=G. Bontempi; citation_author=X. Chen; citation_author=H. Noushmehr; citation_author=A. Colaprico; citation_author=E. Papaleo; citation_journal_title=PLoS Comput. Biol.; citation_volume=15; citation_firstpage=e1006701; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=A statistical model for identifying proteins by tandem mass spectrometry; citation_author=A.I. Nesvizhskii; citation_author=A. Keller; citation_author=E. Kolker; citation_author=R. Aebersold; citation_journal_title=Anal Chem.; citation_volume=75; citation_firstpage=4646; citation_lastpage=4658; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=Quantitative proteomics of the cancer cell line encyclopedia; citation_author=D.P. Nusinow; citation_author=J. Szpyt; citation_author=M. Ghandi; citation_author=C.M. Rose; citation_author=E.R. McDonald; citation_author=M. Kalocsay; citation_author=J. Jané-Valbuena; citation_author=E. Gelfand; citation_author=D.K. Schweppe; citation_author=M. Jedrychowski; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=387; citation_lastpage=402.e316; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation; citation_author=S.Y. Ow; citation_author=M. Salim; citation_author=J. Noirel; citation_author=C. Evans; citation_author=P.C. Wright; citation_journal_title=Proteomics; citation_volume=11; citation_firstpage=2341; citation_lastpage=2346; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Impaired H3K36 methylation defines a subset of head and neck squamous cell carcinomas; citation_author=S. Papillon-Cavanagh; citation_author=C. Lu; citation_author=T. Gayden; citation_author=L.G. Mikael; citation_author=D. Bechet; citation_author=C. Karamboulas; citation_author=L. Ailles; citation_author=J. Karamchandani; citation_author=D.M. Marchione; citation_author=B.A. Garcia; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=180; citation_lastpage=185; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Performance evaluation of DNA copy number segmentation methods; citation_author=M. Pierre-Jean; citation_author=G. Rigaill; citation_author=P. Neuvial; citation_journal_title=Brief Bioinform; citation_volume=16; citation_firstpage=600; citation_lastpage=615; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Prognostic biomarkers in phase II trial of cetuximab-containing induction and chemoradiation in resectable HNSCC: Eastern Cooperative Oncology Group E2303; citation_author=A. Psyrri; citation_author=J.-W. Lee; citation_author=E. Pectasides; citation_author=M. Vassilakopoulou; citation_author=E.K. Kosmidis; citation_author=B.A. Burtness; citation_author=D.L. Rimm; citation_author=H.J. Wanebo; citation_author=A.A. Forastiere; citation_journal_title=Clin. Cancer Res.; citation_volume=20; citation_firstpage=3023; citation_lastpage=3032; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Isobaric labeling-based relative quantification in shotgun proteomics; citation_author=N. Rauniyar; citation_author=J.R. Yates; citation_journal_title=J. Proteome Res.; citation_volume=13; citation_firstpage=5293; citation_lastpage=5309; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=mProphet: automated data processing and statistical validation for large-scale SRM experiments; citation_author=L. Reiter; citation_author=O. Rinner; citation_author=P. Picotti; citation_author=R. Hüttenhain; citation_author=M. Beck; citation_author=M.-Y. Brusniak; citation_author=M.O. Hengartner; citation_author=R. Aebersold; citation_journal_title=Nat. Methods; citation_volume=8; citation_firstpage=430; citation_lastpage=435; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=M.E. Ritchie; citation_author=B. Phipson; citation_author=D. Wu; citation_author=Y. Hu; citation_author=C.W. Law; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=e47; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=MGMT promoter methylation is predictive of response to radiotherapy and prognostic in the absence of adjuvant alkylating chemotherapy for glioblastoma; citation_author=A.L. Rivera; citation_author=C.E. Pelloski; citation_author=M.R. Gilbert; citation_author=H. Colman; citation_author=C. De La Cruz; citation_author=E.P. Sulman; citation_author=B.N. Bekele; citation_author=K.D. Aldape; citation_journal_title=Neuro Oncol.; citation_volume=12; citation_firstpage=116; citation_lastpage=121; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Integrated genomic analysis of diverse induced pluripotent stem cells from the Progenitor Cell Biology Consortium; citation_author=N. Salomonis; citation_author=P.J. Dexheimer; citation_author=L. Omberg; citation_author=R. Schroll; citation_author=S. Bush; citation_author=J. Huo; citation_author=L. Schriml; citation_author=S. Ho Sui; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=et al.; citation_journal_title=Stem Cell Rep.; citation_volume=7; citation_firstpage=110; citation_lastpage=125; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=New therapies in head and neck cancer; citation_author=R.T. Santuray; citation_author=D.E. Johnson; citation_author=J.R. Grandis; citation_journal_title=Trends Cancer; citation_volume=4; citation_firstpage=385; citation_lastpage=396; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs; citation_author=C.T. Saunders; citation_author=W.S.W. Wong; citation_author=S. Swamy; citation_author=J. Becq; citation_author=L.J. Murray; citation_author=R.K. Cheetham; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=1811; citation_lastpage=1817; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Graph algorithms for condensing and consolidating gene set analysis results; citation_author=S.R. Savage; citation_author=Z. Shi; citation_author=Y. Liao; citation_author=B. Zhang; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=S141; citation_lastpage=S152; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=A scalable approach for protein false discovery rate estimation in large proteomic data sets; citation_author=M.M. Savitski; citation_author=M. Wilhelm; citation_author=H. Hahne; citation_author=B. Kuster; citation_author=M. Bantscheff; citation_journal_title=Mol. Cell Proteomics; citation_volume=14; citation_firstpage=2394; citation_lastpage=2404; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Perturbation-response genes reveal signaling footprints in cancer gene expression; citation_author=M. Schubert; citation_author=B. Klinger; citation_author=M. Klünemann; citation_author=A. Sieber; citation_author=F. Uhlitz; citation_author=S. Sauer; citation_author=M.J. Garnett; citation_author=N. Blüthgen; citation_author=J. Saez-Rodriguez; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=20; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial; citation_author=T.Y. Seiwert; citation_author=B. Burtness; citation_author=R. Mehra; citation_author=J. Weiss; citation_author=R. Berger; citation_author=J.P. Eder; citation_author=K. Heath; citation_author=T. McClanahan; citation_author=J. Lunceford; citation_author=C. Gause; citation_author=et al.; citation_journal_title=Lancet Oncol.; citation_volume=17; citation_firstpage=956; citation_lastpage=965; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=The mTOR/PI3K and MAPK pathways converge on eIF4B to control its phosphorylation and activity; citation_author=D. Shahbazian; citation_author=P.P. Roux; citation_author=V. Mieulet; citation_author=M.S. Cohen; citation_author=B. Raught; citation_author=J. Taunton; citation_author=J.W.B. Hershey; citation_author=J. Blenis; citation_author=M. Pende; citation_author=N. Sonenberg; citation_journal_title=EMBO J.; citation_volume=25; citation_firstpage=2781; citation_lastpage=2791; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=PTMProphet: fast and accurate mass modification localization for the trans-proteomic pipeline; citation_author=D.D. Shteynberg; citation_author=E.W. Deutsch; citation_author=D.S. Campbell; citation_author=M.R. Hoopmann; citation_author=U. Kusebauch; citation_author=D. Lee; citation_author=L. Mendoza; citation_author=M.K. Midha; citation_author=Z. Sun; citation_author=A.D. Whetton; citation_author=R.L. Moritz; citation_journal_title=J. Proteome Res.; citation_volume=18; citation_firstpage=4262; citation_lastpage=4272; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Gene signatures and expression of miRNAs associated with efficacy of panitumumab in a head and neck cancer phase II trial; citation_author=M. Siano; citation_author=V. Espeli; citation_author=N. Mach; citation_author=P. Bossi; citation_author=L. Licitra; citation_author=M. Ghielmini; citation_author=M. Frattini; citation_author=S. Canevari; citation_author=L. De Cecco; citation_journal_title=Oral Oncol.; citation_volume=82; citation_firstpage=144; citation_lastpage=151; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Emerging functions of the EGFR in cancer; citation_author=S. Sigismund; citation_author=D. Avanzato; citation_author=L. Lanzetti; citation_journal_title=Mol. Oncol.; citation_volume=12; citation_firstpage=3; citation_lastpage=20; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=EGF receptor ligands: recent advances; citation_author=B. Singh; citation_author=G. Carpenter; citation_author=R.J. Coffey; citation_journal_title=F1000Res; citation_volume=5; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=One-class detection of cell states in tumor subtypes; citation_author=A. Sokolov; citation_author=E.O. Paull; citation_author=J.M. Stuart; citation_journal_title=Pac. Symp. Biocomput.; citation_volume=21; citation_firstpage=405; citation_lastpage=416; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis; citation_author=X. Song; citation_author=J. Ji; citation_author=K.J. Gleason; citation_author=F. Yang; citation_author=J.A. Martignetti; citation_author=L.S. Chen; citation_author=P. Wang; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=S52; citation_lastpage=S65; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Contact inhibitory Eph signaling suppresses EGF-promoted cell migration by decoupling EGFR activity from vesicular recycling; citation_author=W. Stallaert; citation_author=Y. Brüggemann; citation_author=O. Sabet; citation_author=L. Baak; citation_author=M. Gattiglio; citation_author=P.I.H. Bastiaens; citation_journal_title=Sci. Signal.; citation_volume=11; citation_firstpage=eaat0114; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=OptiType: precision HLA typing from next-generation sequencing data; citation_author=A. Szolek; citation_author=B. Schubert; citation_author=C. Mohr; citation_author=M. Sturm; citation_author=M. Feldhahn; citation_author=O. Kohlbacher; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=3310; citation_lastpage=3316; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Automatic relevance determination in nonnegative matrix factorization with the β-divergence; citation_author=V.Y.F. Tan; citation_author=C. Févotte; citation_journal_title=IEEE Trans. Pattern Anal. Mach. Intell.; citation_volume=35; citation_firstpage=1592; citation_lastpage=1605; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Mammalian Fat1 cadherin regulates actin dynamics and cell-cell contact; citation_author=T. Tanoue; citation_author=M. Takeichi; citation_journal_title=J. Cell Biol.; citation_volume=165; citation_firstpage=517; citation_lastpage=528; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=COSMIC: the catalogue of somatic mutations in cancer; citation_author=J.G. Tate; citation_author=S. Bamford; citation_author=H.C. Jubb; citation_author=Z. Sondka; citation_author=D.M. Beare; citation_author=N. Bindal; citation_author=H. Boutselakis; citation_author=C.G. Cole; citation_author=C. Creatore; citation_author=E. Dawson; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=D941; citation_lastpage=D947; citation_publication_date=2019; "></meta><meta name="citation_reference" content="Therneau, T.M. (2020). A Package for Survival Analysis in R. https://cran.r-project.org/web/packages/survival/index.html"></meta><meta name="citation_reference" content="citation_title=Defining a cancer dependency map; citation_author=A. Tsherniak; citation_author=F. Vazquez; citation_author=P.G. Montgomery; citation_author=B.A. Weir; citation_author=G. Kryukov; citation_author=G.S. Cowley; citation_author=S. Gill; citation_author=W.F. Harrington; citation_author=S. Pantel; citation_author=J.M. Krill-Burger; citation_author=et al.; citation_journal_title=Cell; citation_volume=170; citation_firstpage=564; citation_lastpage=576.e16; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics; citation_author=C.-C. Tsou; citation_author=D. Avtonomov; citation_author=B. Larsen; citation_author=M. Tucholska; citation_author=H. Choi; citation_author=A.-C. Gingras; citation_author=A.I. Nesvizhskii; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=258; citation_lastpage=264; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Proteomics. Tissue-based map of the human proteome; citation_author=M. Uhlén; citation_author=L. Fagerberg; citation_author=B.M. Hallström; citation_author=C. Lindskog; citation_author=P. Oksvold; citation_author=A. Mardinoglu; citation_author=Å. Sivertsson; citation_author=C. Kampf; citation_author=E. Sjöstedt; citation_author=A. Asplund; citation_author=et al.; citation_journal_title=Science (New York, NY); citation_volume=347; citation_firstpage=1260419; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O.A. Arshad; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=LinkedOmics: analyzing multi-omics data within and across 32 cancer types; citation_author=S.V. Vasaikar; citation_author=P. Straub; citation_author=J. Wang; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_firstpage=D956; citation_lastpage=D963; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Platinum-based chemotherapy plus cetuximab in head and neck cancer; citation_author=J.B. Vermorken; citation_author=R. Mesia; citation_author=F. Rivera; citation_author=E. Remenar; citation_author=A. Kawecki; citation_author=S. Rottey; citation_author=J. Erfan; citation_author=D. Zabolotnyy; citation_author=H.-R. Kienzer; citation_author=D. Cupissol; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=359; citation_firstpage=1116; citation_lastpage=1127; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Open-label, uncontrolled, multicenter phase II study to evaluate the efficacy and toxicity of cetuximab as a single agent in patients with recurrent and/or metastatic squamous cell carcinoma of the head and neck who failed to respond to platinum-based therapy; citation_author=J.B. Vermorken; citation_author=J. Trigo; citation_author=R. Hitt; citation_author=P. Koralewski; citation_author=E. Diaz-Rubio; citation_author=F. Rolland; citation_author=R. Knecht; citation_author=N. Amellal; citation_author=A. Schueler; citation_author=J. Baselga; citation_journal_title=J. Clin. Oncol.; citation_volume=25; citation_firstpage=2171; citation_lastpage=2177; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes; citation_author=V. Walter; citation_author=X. Yin; citation_author=M.D. Wilkerson; citation_author=C.R. Cabanski; citation_author=N. Zhao; citation_author=Y. Du; citation_author=M.K. Ang; citation_author=M.C. Hayward; citation_author=A.H. Salazar; citation_author=K.A. Hoadley; citation_author=et al.; citation_journal_title=PLoS One; citation_volume=8; citation_firstpage=e56823; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Ckmeans.1d.dp: optimal k-means clustering in one dimension by dynamic programming; citation_author=H. Wang; citation_author=M. Song; citation_journal_title=R. J.; citation_volume=3; citation_firstpage=29; citation_lastpage=33; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Proteome profiling outperforms transcriptome profiling for coexpression based gene function prediction; citation_author=J. Wang; citation_author=Z. Ma; citation_author=S.A. Carr; citation_author=P. Mertins; citation_author=H. Zhang; citation_author=Z. Zhang; citation_author=D.W. Chan; citation_author=M.J.C. Ellis; citation_author=R.R. Townsend; citation_author=R.D. Smith; citation_author=et al.; citation_journal_title=Mol. Cell Proteomics; citation_volume=16; citation_firstpage=121; citation_lastpage=134; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Mammalian target of rapamycin complex 1 (mTORC1) activity is associated with phosphorylation of raptor by mTOR; citation_author=L. Wang; citation_author=J.C. Lawrence; citation_author=T.W. Sturgill; citation_author=T.E. Harris; citation_journal_title=J. Biol. Chem.; citation_volume=284; citation_firstpage=14693; citation_lastpage=14697; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Targeting Cdc20 as a novel cancer therapeutic strategy; citation_author=L. Wang; citation_author=J. Zhang; citation_author=L. Wan; citation_author=X. Zhou; citation_author=Z. Wang; citation_author=W. Wei; citation_journal_title=Pharmacol. Ther.; citation_volume=151; citation_firstpage=141; citation_lastpage=151; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Protein identification using customized protein sequence databases derived from RNA-seq data; citation_author=X. Wang; citation_author=R.J.C. Slebos; citation_author=D. Wang; citation_author=P.J. Halvey; citation_author=D.L. Tabb; citation_author=D.C. Liebler; citation_author=B. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=11; citation_firstpage=1009; citation_lastpage=1017; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=3235; citation_lastpage=3237; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Epidermal growth factor receptor cell proliferation signaling pathways; citation_author=P. Wee; citation_author=Z. Wang; citation_journal_title=Cancers (Basel); citation_volume=9; citation_firstpage=52; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis; citation_author=B. Wen; citation_author=K. Li; citation_author=Y. Zhang; citation_author=B. Zhang; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=1759; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations; citation_author=B. Wen; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=485; citation_lastpage=493; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Identification of genes periodically expressed in the human cell cycle and their expression in tumors; citation_author=M.L. Whitfield; citation_author=G. Sherlock; citation_author=A.J. Saldanha; citation_author=J.I. Murray; citation_author=C.A. Ball; citation_author=K.E. Alexander; citation_author=J.C. Matese; citation_author=C.M. Perou; citation_author=M.M. Hurt; citation_author=P.O. Brown; citation_author=D. Botstein; citation_journal_title=Mol. Biol. Cell; citation_volume=13; citation_firstpage=1977; citation_lastpage=2000; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads; citation_author=K. Ye; citation_author=M.H. Schulz; citation_author=Q. Long; citation_author=R. Apweiler; citation_author=Z. Ning; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2865; citation_lastpage=2871; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Inferring tumour purity and stromal and immune cell admixture from expression data; citation_author=K. Yoshihara; citation_author=M. Shahmoradgoli; citation_author=E. Martínez; citation_author=R. Vegesna; citation_author=H. Kim; citation_author=W. Torres-Garcia; citation_author=V. Treviño; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=D.A. Levine; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2612; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach; citation_author=J. Zecha; citation_author=S. Satpathy; citation_author=T. Kanashova; citation_author=S.C. Avanessian; citation_author=M.H. Kane; citation_author=K.R. Clauser; citation_author=P. Mertins; citation_author=S.A. Carr; citation_author=B. Kuster; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=1468; citation_lastpage=1478; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=INTEGRATE: gene fusion discovery using whole genome and transcriptome data; citation_author=J. Zhang; citation_author=N.M. White; citation_author=H.K. Schmidt; citation_author=R.S. Fulton; citation_author=C. Tomlinson; citation_author=W.C. Warren; citation_author=R.K. Wilson; citation_author=C.A. Maher; citation_journal_title=Genome Res.; citation_volume=26; citation_firstpage=108; citation_lastpage=118; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=QuickMIRSeq: a pipeline for quick and accurate quantification of both known miRNAs and isomiRs by jointly processing multiple samples from microRNA sequencing; citation_author=S. Zhao; citation_author=W. Gordon; citation_author=S. Du; citation_author=C. Zhang; citation_author=W. He; citation_author=L. Xi; citation_author=S. Mathur; citation_author=M. Agostino; citation_author=T. Paradis; citation_author=D. von Schack; citation_author=et al.; citation_journal_title=BMC Bioinformatics; citation_volume=18; citation_firstpage=180; citation_publication_date=2017; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;We present a proteogenomic study of 108 human papilloma virus (HPV)-negative head and neck squamous cell carcinomas (HNSCCs). Proteomic analysis systematically catalogs HNSCC-associated proteins and phosphosites, prioritizes copy number drivers, and highlights an oncogenic role for RNA processing genes. Proteomic investigation of mutual exclusivity between &lt;i&gt;FAT1&lt;/i&gt; truncating mutations and 11q13.3 amplifications reveals dysregulated actin dynamics as a common functional consequence. Phosphoproteomics characterizes two modes of EGFR activation, suggesting a new strategy to stratify HNSCCs based on EGFR ligand abundance for effective treatment with inhibitory EGFR monoclonal antibodies. Widespread deletion of immune modulatory genes accounts for low immune infiltration in immune-cold tumors, whereas concordant upregulation of multiple immune checkpoint proteins may underlie resistance to anti-programmed cell death protein 1 monotherapy in immune-hot tumors. Multi-omic analysis identifies three molecular subtypes with high potential for treatment with CDK inhibitors, anti-EGFR antibody therapy, and immunotherapy, respectively. Altogether, proteogenomics provides a systematic framework to inform HNSCC biology and treatment.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-ccell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2820%2930655-3%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610820306553%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cancer-cell/fulltext/S1535-6108(20)30655-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610820306553%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cancer-cell/fulltext/S1535-6108(20)30655-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610820306553%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2820%2930655-3%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610820306553%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="ccell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cancer-cell/home" title="Cancer Cell"><img alt="Cancer Cell" src="/pb-assets/ux3/logos/cell/CancerCell_white-1532516635240.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="f0df5ebf-b1ca-4821-b16c-caadb221e181" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cancer-cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2820%2930655-3%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610820306553%253Fshowall%253Dtrue&amp;pii=S1535-6108%2820%2930655-3" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=ccell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2820%2930655-3%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610820306553%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cancer-cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2820%2930655-3%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610820306553%253Fshowall%253Dtrue&amp;pii=S1535-6108%2820%2930655-3" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cancer-cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cancer-cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.ccell.2020.12.007">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S1535-6108(23)X0008-7"><img src="/cms/asset/atypon:cms:attachment:img:d3e6:rev:1720427783745-1677:pii:S1535610823X00087/cover.tif.jpg" alt="Cover Image - Cancer Cell, Volume 42, Issue 7"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cancer-cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.14.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=ccell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=ccell&amp;ISSN=1535-6108&amp;journalCode=ccell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="ccell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="ccell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=ccell&amp;ISSN=1535-6108&amp;journalCode=ccell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="dc43d918-6255-4b51-a3b2-9725dca931ea" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S1535-6108(20)30655-3"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Article</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cancer-cell/issue?pii=S1535-6108(20)X0004-3" class="article-header__vol faded">
                                                    Volume 39, ISSUE 3</a><span></span><span>, </span><span class="article-header__pages faded">P361-379.e16, </span><span class="article-header__date faded">March 08, 2021</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=ccell&amp;volume=39&amp;issue=3&amp;pii=S1535-6108%2820%29X0004-3" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__pdf"><a href="#" title="open PDF menu" data-db-target-for="article-tools-pdf" aria-expanded="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF</span><span class="alternative">PDF</span></a><div data-db-target-of="article-tools-pdf" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="/action/showPdf?pii=S1535-6108%2820%2930655-3" title="Standard PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayStandardPdf ">Standard PDF [7 MB]</a></li><li class="article-tools__sub-item"><a href="/cancer-cell/pdfExtended/S1535-6108(20)30655-3" title="Extended PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayExtendedPdf ">Extended PDF [10 MB]</a></li></ul></div></div></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cancer-cell/ppt/S1535-6108(20)30655-3.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.ccell.2020.12.007" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.ccell.2020.12.007&amp;pii=S1535-6108%2820%2930655-3" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.ccell.2020.12.007" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-524" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-524">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cancer-cell/fulltext/S1535-6108%2820%2930655-3"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S1535610820306553&amp;issn=15356108&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1535610820306553&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a> <a href="#" data-db-target-for="crosref0020" aria-expanded="false" aria-controls="crosref0020" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>28</sup></span></a><div data-db-target-of="crosref0020" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">28 The authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">28 The authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lijun Chen">Lijun Chen</a> <a href="#" data-db-target-for="crosref0030" aria-expanded="false" aria-controls="crosref0030" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>28</sup></span></a><div data-db-target-of="crosref0030" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">28 The authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lijun Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">28 The authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Lijun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sara R. Savage">Sara R. Savage</a> <a href="#" data-db-target-for="crosref0045" aria-expanded="false" aria-controls="crosref0045" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>28</sup></span></a><div data-db-target-of="crosref0045" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">28 The authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sara R. Savage</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">28 The authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Savage%2C+Sara R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rodrigo Vargas Eguez">Rodrigo Vargas Eguez</a><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rodrigo Vargas Eguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Eguez%2C+Rodrigo Vargas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yongchao Dou">Yongchao Dou</a><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yongchao Dou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dou%2C+Yongchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yize Li">Yize Li</a><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yize Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Yize" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Felipe da Veiga Leprevost">Felipe da Veiga Leprevost</a><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Felipe da Veiga Leprevost</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=da Veiga Leprevost%2C+Felipe" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Eric J. Jaehnig">Eric J. Jaehnig</a><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Eric J. Jaehnig</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jaehnig%2C+Eric J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jonathan T. Lei">Jonathan T. Lei</a><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jonathan T. Lei</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lei%2C+Jonathan T" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bo Wen">Bo Wen</a><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael Schnaubelt">Michael Schnaubelt</a><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael Schnaubelt</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schnaubelt%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karsten Krug">Karsten Krug</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karsten Krug</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krug%2C+Karsten" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaoyu Song">Xiaoyu Song</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaoyu Song</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Song%2C+Xiaoyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marcin Cieślik">Marcin Cieślik</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marcin Cieślik</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cieślik%2C+Marcin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Hui-Yin Chang">Hui-Yin Chang</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Hui-Yin Chang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chang%2C+Hui-Yin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew A. Wyczalkowski">Matthew A. Wyczalkowski</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew A. Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kai Li">Kai Li</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kai Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Kai" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Antonio Colaprico">Antonio Colaprico</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Antonio Colaprico</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA<p></p>Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Colaprico%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Qing Kay Li">Qing Kay Li</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Qing Kay Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Qing Kay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David J. Clark">David J. Clark</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David J. Clark</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clark%2C+David J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yingwei Hu">Yingwei Hu</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yingwei Hu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hu%2C+Yingwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Liwei Cao">Liwei Cao</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Liwei Cao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cao%2C+Liwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jianbo Pan">Jianbo Pan</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jianbo Pan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA<p></p>Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Pan%2C+Jianbo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yuefan Wang">Yuefan Wang</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yuefan Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Yuefan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kyung-Cho Cho">Kyung-Cho Cho</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kyung-Cho Cho</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cho%2C+Kyung-Cho" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhiao Shi">Zhiao Shi</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhiao Shi</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Zhiao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yuxing Liao">Yuxing Liao</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yuxing Liao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liao%2C+Yuxing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wen Jiang">Wen Jiang</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wen Jiang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jiang%2C+Wen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Meenakshi Anurag">Meenakshi Anurag</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Meenakshi Anurag</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Anurag%2C+Meenakshi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiayi Ji">Jiayi Ji</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiayi Ji</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ji%2C+Jiayi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Seungyeul Yoo">Seungyeul Yoo</a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Seungyeul Yoo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Yoo%2C+Seungyeul" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel Cui Zhou">Daniel Cui Zhou</a><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel Cui Zhou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhou%2C+Daniel Cui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au33-authorcard" aria-expanded="false" aria-controls="au33" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wen-Wei Liang">Wen-Wei Liang</a><div data-db-target-of="au33-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wen-Wei Liang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liang%2C+Wen-Wei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au34-authorcard" aria-expanded="false" aria-controls="au34" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael Wendl">Michael Wendl</a><div data-db-target-of="au34-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael Wendl</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wendl%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au35-authorcard" aria-expanded="false" aria-controls="au35" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pankaj Vats">Pankaj Vats</a><div data-db-target-of="au35-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pankaj Vats</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Vats%2C+Pankaj" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au36-authorcard" aria-expanded="false" aria-controls="au36" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Steven A. Carr">Steven A. Carr</a><div data-db-target-of="au36-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Steven A. Carr</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au37-authorcard" aria-expanded="false" aria-controls="au37" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author D.R. Mani">D.R. Mani</a><div data-db-target-of="au37-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">D.R. Mani</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au38-authorcard" aria-expanded="false" aria-controls="au38" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhen Zhang">Zhen Zhang</a><div data-db-target-of="au38-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au39-authorcard" aria-expanded="false" aria-controls="au39" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiang Qian">Jiang Qian</a><div data-db-target-of="au39-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiang Qian</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Ophthalmology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Qian%2C+Jiang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au40-authorcard" aria-expanded="false" aria-controls="au40" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xi S. Chen">Xi S. Chen</a><div data-db-target-of="au40-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xi S. Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami Miller School of Medicine, Miami, FL 33136, USA<p></p>Division of Biostatistics, Department of Public Health Science, University of Miami Miller School of Medicine, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xi S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au41-authorcard" aria-expanded="false" aria-controls="au41" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexander R. Pico">Alexander R. Pico</a><div data-db-target-of="au41-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexander R. Pico</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute of Data Science and Biotechnology, Gladstone Institutes, San Francisco, CA 94158, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Pico%2C+Alexander R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au42-authorcard" aria-expanded="false" aria-controls="au42" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pei Wang">Pei Wang</a><div data-db-target-of="au42-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pei Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences and Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Pei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au43-authorcard" aria-expanded="false" aria-controls="au43" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Arul M. Chinnaiyan">Arul M. Chinnaiyan</a><div data-db-target-of="au43-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Arul M. Chinnaiyan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chinnaiyan%2C+Arul M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au44-authorcard" aria-expanded="false" aria-controls="au44" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karen A. Ketchum">Karen A. Ketchum</a><div data-db-target-of="au44-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">ESAC, Inc., Rockville, MD 20850, United States</div></div><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au45-authorcard" aria-expanded="false" aria-controls="au45" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au45-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au46-authorcard" aria-expanded="false" aria-controls="au46" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au46-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au47-authorcard" aria-expanded="false" aria-controls="au47" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Eunkyung An">Eunkyung An</a><div data-db-target-of="au47-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Eunkyung An</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=An%2C+Eunkyung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au48-authorcard" aria-expanded="false" aria-controls="au48" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tara Hiltke">Tara Hiltke</a><div data-db-target-of="au48-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au49-authorcard" aria-expanded="false" aria-controls="au49" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mehdi Mesri">Mehdi Mesri</a><div data-db-target-of="au49-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au50-authorcard" aria-expanded="false" aria-controls="au50" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au50-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research Inc., Frederick NaVonal Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au51-authorcard" aria-expanded="false" aria-controls="au51" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alissa M. Weaver">Alissa M. Weaver</a><div data-db-target-of="au51-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alissa M. Weaver</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Cell and Developmental Biology, Vanderbilt University School of Medicine, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Weaver%2C+Alissa M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au52-authorcard" aria-expanded="false" aria-controls="au52" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Andrew G. Sikora">Andrew G. Sikora</a><div data-db-target-of="au52-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Andrew G. Sikora</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Head and Neck Surgery, University of Texas M.D. Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Sikora%2C+Andrew G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au53-authorcard" aria-expanded="false" aria-controls="au53" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jan Lubiński">Jan Lubiński</a><div data-db-target-of="au53-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jan Lubiński</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Pathology, International Hereditary Cancer Center, Pomeranian Medical University, 71-252 Szczecin, Poland<p></p>International Institute for Molecular Oncology, 60-203 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Lubiński%2C+Jan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au54-authorcard" aria-expanded="false" aria-controls="au54" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Małgorzata Wierzbicka">Małgorzata Wierzbicka</a><div data-db-target-of="au54-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Małgorzata Wierzbicka</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Poznań University of Medical Sciences, 61-701 Poznań, Poland<p></p>Institute of Human Genetics Polish Academy of Sciences, 60-479 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Wierzbicka%2C+Małgorzata" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au55-authorcard" aria-expanded="false" aria-controls="au55" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Maciej Wiznerowicz">Maciej Wiznerowicz</a><div data-db-target-of="au55-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Maciej Wiznerowicz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">International Institute for Molecular Oncology, 60-203 Poznań, Poland<p></p>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Wiznerowicz%2C+Maciej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au56-authorcard" aria-expanded="false" aria-controls="au56" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shankha Satpathy">Shankha Satpathy</a><div data-db-target-of="au56-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shankha Satpathy</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Satpathy%2C+Shankha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au57-authorcard" aria-expanded="false" aria-controls="au57" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael A. Gillette">Michael A. Gillette</a><div data-db-target-of="au57-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael A. Gillette</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02114, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au58-authorcard" aria-expanded="false" aria-controls="au58" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author George Miles">George Miles</a><div data-db-target-of="au58-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">George Miles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Miles%2C+George" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au59-authorcard" aria-expanded="false" aria-controls="au59" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew J. Ellis">Matthew J. Ellis</a><div data-db-target-of="au59-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew J. Ellis</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au60-authorcard" aria-expanded="false" aria-controls="au60" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gilbert S. Omenn">Gilbert S. Omenn</a><div data-db-target-of="au60-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gilbert S. Omenn</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Omenn%2C+Gilbert S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au61-authorcard" aria-expanded="false" aria-controls="au61" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au61-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au62-authorcard" aria-expanded="false" aria-controls="au62" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily S. Boja">Emily S. Boja</a><div data-db-target-of="au62-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au63-authorcard" aria-expanded="false" aria-controls="au63" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Saravana M. Dhanasekaran">Saravana M. Dhanasekaran</a><div data-db-target-of="au63-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Saravana M. Dhanasekaran</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dhanasekaran%2C+Saravana M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au64-authorcard" aria-expanded="false" aria-controls="au64" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au64-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63110, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au65-authorcard" aria-expanded="false" aria-controls="au65" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexey I. Nesvizhskii">Alexey I. Nesvizhskii</a><div data-db-target-of="au65-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexey I. Nesvizhskii</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Michigan Center for Translational Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Nesvizhskii%2C+Alexey I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au66-authorcard" aria-expanded="false" aria-controls="au66" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Adel K. El-Naggar">Adel K. El-Naggar</a><div data-db-target-of="au66-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Adel K. El-Naggar</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Division of Pathology and Laboratory Medicine, MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=El-Naggar%2C+Adel K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au67-authorcard" aria-expanded="false" aria-controls="au67" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel W. Chan">Daniel W. Chan<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au67-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel W. Chan</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#d7b3b4bfb6b997bdbfbabef9b2b3a2" title="Send email to Daniel W. Chan"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Daniel W. Chan" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#8ce8efe4ede2cce6e4e1e5a2e9e8f9"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au68-authorcard" aria-expanded="false" aria-controls="au68" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Hui Zhang">Hui Zhang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au68-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Hui Zhang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#402835293a28212e27002a28356e252435" title="Send email to Hui Zhang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology and Oncology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Hui Zhang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#ed85988497858c838aad878598c3888998"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au69-authorcard" aria-expanded="false" aria-controls="au69" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0550a" aria-expanded="false" aria-controls="crosref0550a" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>29</sup></span></a><div data-db-target-of="crosref0550a" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">29 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au69-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#5d3f34333a7327353c333a1d3f3e3073383928" title="Send email to Bing Zhang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">29 Lead Contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Bing Zhang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#17757e7970396d7f7679705775747a39727362"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Anupriya Agarwal" class="dropBlock__slider__ctrl">Anupriya Agarwal</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anupriya back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anupriya Agarwal</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Agarwal%2C+Anupriya" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew L. Anderson" class="dropBlock__slider__ctrl">Matthew L. Anderson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew L. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew L. Anderson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Anderson%2C+Matthew L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shayan C. Avanessian" class="dropBlock__slider__ctrl">Shayan C. Avanessian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shayan C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shayan C. Avanessian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Avanessian%2C+Shayan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dmitry Avtonomov" class="dropBlock__slider__ctrl">Dmitry Avtonomov</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dmitry back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dmitry Avtonomov</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Avtonomov%2C+Dmitry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Oliver F. Bathe" class="dropBlock__slider__ctrl">Oliver F. Bathe</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Oliver F. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Oliver F. Bathe</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bathe%2C+Oliver F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chet Birger" class="dropBlock__slider__ctrl">Chet Birger</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chet back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chet Birger</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birger%2C+Chet" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael J. Birrer" class="dropBlock__slider__ctrl">Michael J. Birrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael J. Birrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birrer%2C+Michael J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lili Blumenberg" class="dropBlock__slider__ctrl">Lili Blumenberg</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lili back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lili Blumenberg</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Blumenberg%2C+Lili" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="William E. Bocik" class="dropBlock__slider__ctrl">William E. Bocik</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="William E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">William E. Bocik</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bocik%2C+William E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Borate" class="dropBlock__slider__ctrl">Uma Borate</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Borate</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borate%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melissa Borucki" class="dropBlock__slider__ctrl">Melissa Borucki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melissa Borucki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borucki%2C+Melissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Meghan C. Burke" class="dropBlock__slider__ctrl">Meghan C. Burke</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Meghan C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Meghan C. Burke</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Burke%2C+Meghan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anna Pamela Calinawan" class="dropBlock__slider__ctrl">Anna Pamela Calinawan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anna Pamela back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anna Pamela Calinawan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Calinawan%2C+Anna Pamela" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sandra Cerda" class="dropBlock__slider__ctrl">Sandra Cerda</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sandra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sandra Cerda</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cerda%2C+Sandra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alyssa Charamut" class="dropBlock__slider__ctrl">Alyssa Charamut</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alyssa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alyssa Charamut</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Charamut%2C+Alyssa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lin S. Chen" class="dropBlock__slider__ctrl">Lin S. Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lin S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lin S. Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Lin S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shrabanti Chowdhury" class="dropBlock__slider__ctrl">Shrabanti Chowdhury</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shrabanti back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shrabanti Chowdhury</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chowdhury%2C+Shrabanti" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karl R. Clauser" class="dropBlock__slider__ctrl">Karl R. Clauser</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karl R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karl R. Clauser</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Clauser%2C+Karl R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Houston Culpepper" class="dropBlock__slider__ctrl">Houston Culpepper</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Houston back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Houston Culpepper</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Culpepper%2C+Houston" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tomasz Czernicki" class="dropBlock__slider__ctrl">Tomasz Czernicki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tomasz back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tomasz Czernicki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Czernicki%2C+Tomasz" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Fulvio D'Angelo" class="dropBlock__slider__ctrl">Fulvio D'Angelo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Fulvio back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Fulvio D'Angelo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=D'Angelo%2C+Fulvio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jacob Day" class="dropBlock__slider__ctrl">Jacob Day</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jacob back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jacob Day</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Day%2C+Jacob" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephanie De Young" class="dropBlock__slider__ctrl">Stephanie De Young</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephanie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephanie De Young</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=De Young%2C+Stephanie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Emek Demir" class="dropBlock__slider__ctrl">Emek Demir</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Emek back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Emek Demir</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Demir%2C+Emek" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Fei Ding" class="dropBlock__slider__ctrl">Fei Ding</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Fei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Fei Ding</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Fei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marcin J. Domagalski" class="dropBlock__slider__ctrl">Marcin J. Domagalski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marcin J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marcin J. Domagalski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Domagalski%2C+Marcin J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Joseph C. Dort" class="dropBlock__slider__ctrl">Joseph C. Dort</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Joseph C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Joseph C. Dort</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dort%2C+Joseph C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Brian Druker" class="dropBlock__slider__ctrl">Brian Druker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Brian back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Brian Druker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Druker%2C+Brian" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elizabeth Duffy" class="dropBlock__slider__ctrl">Elizabeth Duffy</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elizabeth back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elizabeth Duffy</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Duffy%2C+Elizabeth" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Maureen Dyer" class="dropBlock__slider__ctrl">Maureen Dyer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Maureen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Maureen Dyer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dyer%2C+Maureen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kimberly Elburn" class="dropBlock__slider__ctrl">Kimberly Elburn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kimberly back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kimberly Elburn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Elburn%2C+Kimberly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tatiana S. Ermakova" class="dropBlock__slider__ctrl">Tatiana S. Ermakova</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tatiana S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tatiana S. Ermakova</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ermakova%2C+Tatiana S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Fenyo" class="dropBlock__slider__ctrl">David Fenyo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Fenyo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Fenyo%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Renata Ferrarotto" class="dropBlock__slider__ctrl">Renata Ferrarotto</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Renata back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Renata Ferrarotto</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ferrarotto%2C+Renata" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alicia Francis" class="dropBlock__slider__ctrl">Alicia Francis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alicia back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alicia Francis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Francis%2C+Alicia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stacey Gabriel" class="dropBlock__slider__ctrl">Stacey Gabriel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stacey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stacey Gabriel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabriel%2C+Stacey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Luciano Garofano" class="dropBlock__slider__ctrl">Luciano Garofano</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Luciano back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Luciano Garofano</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Garofano%2C+Luciano" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yifat Geffen" class="dropBlock__slider__ctrl">Yifat Geffen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yifat back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yifat Geffen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Geffen%2C+Yifat" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gad Getz" class="dropBlock__slider__ctrl">Gad Getz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gad back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gad Getz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Getz%2C+Gad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Charles A. Goldthwaite" class="dropBlock__slider__ctrl">Charles A. Goldthwaite</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Charles A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Charles A. Goldthwaite</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Goldthwaite%2C+Charles A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Linda I. Hannick" class="dropBlock__slider__ctrl">Linda I. Hannick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Linda I. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Linda I. Hannick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hannick%2C+Linda I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pushpa Hariharan" class="dropBlock__slider__ctrl">Pushpa Hariharan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pushpa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pushpa Hariharan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hariharan%2C+Pushpa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David N. Hayes" class="dropBlock__slider__ctrl">David N. Hayes</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David N. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David N. Hayes</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hayes%2C+David N" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Heiman" class="dropBlock__slider__ctrl">David Heiman</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Heiman</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Heiman%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Hindenach" class="dropBlock__slider__ctrl">Barbara Hindenach</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Hindenach</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hindenach%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Katherine A. Hoadley" class="dropBlock__slider__ctrl">Katherine A. Hoadley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Katherine A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Katherine A. Hoadley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoadley%2C+Katherine A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Galen Hostetter" class="dropBlock__slider__ctrl">Galen Hostetter</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Galen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Galen Hostetter</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hostetter%2C+Galen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Martin Hyrcza" class="dropBlock__slider__ctrl">Martin Hyrcza</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Martin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Martin Hyrcza</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hyrcza%2C+Martin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Scott D. Jewell" class="dropBlock__slider__ctrl">Scott D. Jewell</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Scott D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Scott D. Jewell</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jewell%2C+Scott D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Corbin D. Jones" class="dropBlock__slider__ctrl">Corbin D. Jones</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Corbin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Corbin D. Jones</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jones%2C+Corbin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="M. Harry Kane" class="dropBlock__slider__ctrl">M. Harry Kane</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="M. Harry back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">M. Harry Kane</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kane%2C+M. Harry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alicia Karz" class="dropBlock__slider__ctrl">Alicia Karz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alicia back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alicia Karz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Karz%2C+Alicia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ramani B. Kothadia" class="dropBlock__slider__ctrl">Ramani B. Kothadia</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ramani B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ramani B. Kothadia</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kothadia%2C+Ramani B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Azra Krek" class="dropBlock__slider__ctrl">Azra Krek</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Azra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Azra Krek</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Krek%2C+Azra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chandan Kumar-Sinha" class="dropBlock__slider__ctrl">Chandan Kumar-Sinha</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chandan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chandan Kumar-Sinha</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kumar-Sinha%2C+Chandan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tao Liu" class="dropBlock__slider__ctrl">Tao Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tao Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hongwei Liu" class="dropBlock__slider__ctrl">Hongwei Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hongwei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hongwei Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Hongwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Weiping Ma" class="dropBlock__slider__ctrl">Weiping Ma</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Weiping back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Weiping Ma</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ma%2C+Weiping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ewa Malc" class="dropBlock__slider__ctrl">Ewa Malc</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ewa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ewa Malc</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malc%2C+Ewa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anna Malovannaya" class="dropBlock__slider__ctrl">Anna Malovannaya</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anna Malovannaya</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malovannaya%2C+Anna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sailaja Mareedu" class="dropBlock__slider__ctrl">Sailaja Mareedu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sailaja back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sailaja Mareedu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mareedu%2C+Sailaja" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sanford P. Markey" class="dropBlock__slider__ctrl">Sanford P. Markey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sanford P. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sanford P. Markey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Markey%2C+Sanford P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Annette Marrero-Oliveras" class="dropBlock__slider__ctrl">Annette Marrero-Oliveras</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Annette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Annette Marrero-Oliveras</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Marrero-Oliveras%2C+Annette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nicollette Maunganidze" class="dropBlock__slider__ctrl">Nicollette Maunganidze</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nicollette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nicollette Maunganidze</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maunganidze%2C+Nicollette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jason E. McDermott" class="dropBlock__slider__ctrl">Jason E. McDermott</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jason E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jason E. McDermott</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="John McGee" class="dropBlock__slider__ctrl">John McGee</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="John back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">John McGee</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGee%2C+John" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Piotr Mieczkowski" class="dropBlock__slider__ctrl">Piotr Mieczkowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Piotr back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Piotr Mieczkowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mieczkowski%2C+Piotr" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Simona Migliozzi" class="dropBlock__slider__ctrl">Simona Migliozzi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Simona back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Simona Migliozzi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Migliozzi%2C+Simona" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rebecca Montgomery" class="dropBlock__slider__ctrl">Rebecca Montgomery</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rebecca back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rebecca Montgomery</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Montgomery%2C+Rebecca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chelsea J. Newton" class="dropBlock__slider__ctrl">Chelsea J. Newton</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chelsea J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chelsea J. Newton</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Newton%2C+Chelsea J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Umut Ozbek" class="dropBlock__slider__ctrl">Umut Ozbek</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Umut back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Umut Ozbek</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ozbek%2C+Umut" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amanda G. Paulovich" class="dropBlock__slider__ctrl">Amanda G. Paulovich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amanda G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amanda G. Paulovich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Samuel H. Payne" class="dropBlock__slider__ctrl">Samuel H. Payne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Samuel H. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dimitar Dimitrov Pazardzhikliev" class="dropBlock__slider__ctrl">Dimitar Dimitrov Pazardzhikliev</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dimitar Dimitrov back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dimitar Dimitrov Pazardzhikliev</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pazardzhikliev%2C+Dimitar Dimitrov" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amy M. Perou" class="dropBlock__slider__ctrl">Amy M. Perou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amy M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amy M. Perou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Perou%2C+Amy M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Francesca Petralia" class="dropBlock__slider__ctrl">Francesca Petralia</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Francesca back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Francesca Petralia</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Petralia%2C+Francesca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lyudmila Petrenko" class="dropBlock__slider__ctrl">Lyudmila Petrenko</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lyudmila back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lyudmila Petrenko</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Petrenko%2C+Lyudmila" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Paul D. Piehowski" class="dropBlock__slider__ctrl">Paul D. Piehowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Paul D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Paul D. Piehowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Piehowski%2C+Paul D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dmitris Placantonakis" class="dropBlock__slider__ctrl">Dmitris Placantonakis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dmitris back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dmitris Placantonakis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Placantonakis%2C+Dmitris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Larisa Polonskaya" class="dropBlock__slider__ctrl">Larisa Polonskaya</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Larisa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Larisa Polonskaya</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Polonskaya%2C+Larisa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elena V. Ponomareva" class="dropBlock__slider__ctrl">Elena V. Ponomareva</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elena V. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elena V. Ponomareva</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ponomareva%2C+Elena V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Olga Potapova" class="dropBlock__slider__ctrl">Olga Potapova</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Olga back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Olga Potapova</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Potapova%2C+Olga" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liqun Qi" class="dropBlock__slider__ctrl">Liqun Qi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liqun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liqun Qi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Qi%2C+Liqun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ning Qu" class="dropBlock__slider__ctrl">Ning Qu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ning back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ning Qu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Qu%2C+Ning" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shakti Ramkissoon" class="dropBlock__slider__ctrl">Shakti Ramkissoon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shakti back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shakti Ramkissoon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ramkissoon%2C+Shakti" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Boris Reva" class="dropBlock__slider__ctrl">Boris Reva</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Boris back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Boris Reva</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Reva%2C+Boris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shannon Richey" class="dropBlock__slider__ctrl">Shannon Richey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shannon back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shannon Richey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Richey%2C+Shannon" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karna Robinson" class="dropBlock__slider__ctrl">Karna Robinson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karna Robinson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Robinson%2C+Karna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nancy Roche" class="dropBlock__slider__ctrl">Nancy Roche</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nancy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nancy Roche</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Roche%2C+Nancy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karin Rodland" class="dropBlock__slider__ctrl">Karin Rodland</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karin Rodland</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel C. Rohrer" class="dropBlock__slider__ctrl">Daniel C. Rohrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel C. Rohrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rohrer%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dmitry Rykunov" class="dropBlock__slider__ctrl">Dmitry Rykunov</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dmitry back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dmitry Rykunov</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rykunov%2C+Dmitry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric E. Schadt" class="dropBlock__slider__ctrl">Eric E. Schadt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric E. Schadt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Schadt%2C+Eric E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yan Shi" class="dropBlock__slider__ctrl">Yan Shi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yan Shi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Yan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yvonne Shutack" class="dropBlock__slider__ctrl">Yvonne Shutack</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yvonne back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yvonne Shutack</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shutack%2C+Yvonne" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shilpi Singh" class="dropBlock__slider__ctrl">Shilpi Singh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shilpi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shilpi Singh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Singh%2C+Shilpi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tara Skelly" class="dropBlock__slider__ctrl">Tara Skelly</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tara Skelly</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Skelly%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Richard Smith" class="dropBlock__slider__ctrl">Richard Smith</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Richard back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Richard Smith</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lori J. Sokoll" class="dropBlock__slider__ctrl">Lori J. Sokoll</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lori J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lori J. Sokoll</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sokoll%2C+Lori J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jakub Stawicki" class="dropBlock__slider__ctrl">Jakub Stawicki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jakub back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jakub Stawicki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stawicki%2C+Jakub" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephen E. Stein" class="dropBlock__slider__ctrl">Stephen E. Stein</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephen E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephen E. Stein</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stein%2C+Stephen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="James Suh" class="dropBlock__slider__ctrl">James Suh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="James back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">James Suh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Suh%2C+James" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Wojciech Szopa" class="dropBlock__slider__ctrl">Wojciech Szopa</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Wojciech back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Wojciech Szopa</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Szopa%2C+Wojciech" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dave Tabor" class="dropBlock__slider__ctrl">Dave Tabor</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dave back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dave Tabor</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tabor%2C+Dave" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Donghui Tan" class="dropBlock__slider__ctrl">Donghui Tan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Donghui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Donghui Tan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tan%2C+Donghui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Darlene Tansil" class="dropBlock__slider__ctrl">Darlene Tansil</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Darlene back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Darlene Tansil</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tansil%2C+Darlene" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Guo Ci Teo" class="dropBlock__slider__ctrl">Guo Ci Teo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Guo Ci back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Guo Ci Teo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Teo%2C+Guo Ci" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Cristina Tognon" class="dropBlock__slider__ctrl">Cristina Tognon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Cristina back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Cristina Tognon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tognon%2C+Cristina" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elie Traer" class="dropBlock__slider__ctrl">Elie Traer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elie Traer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Traer%2C+Elie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shirley Tsang" class="dropBlock__slider__ctrl">Shirley Tsang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shirley back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shirley Tsang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tsang%2C+Shirley" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey Tyner" class="dropBlock__slider__ctrl">Jeffrey Tyner</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey Tyner</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tyner%2C+Jeffrey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ki Sung Um" class="dropBlock__slider__ctrl">Ki Sung Um</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ki Sung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ki Sung Um</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Um%2C+Ki Sung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dana R. Valley" class="dropBlock__slider__ctrl">Dana R. Valley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dana R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dana R. Valley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Valley%2C+Dana R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lyubomir Valkov Vasilev" class="dropBlock__slider__ctrl">Lyubomir Valkov Vasilev</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lyubomir Valkov back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lyubomir Valkov Vasilev</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vasilev%2C+Lyubomir Valkov" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Negin Vatanian" class="dropBlock__slider__ctrl">Negin Vatanian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Negin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Negin Vatanian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vatanian%2C+Negin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Velvulou" class="dropBlock__slider__ctrl">Uma Velvulou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Velvulou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Velvulou%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Vernon" class="dropBlock__slider__ctrl">Michael Vernon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Vernon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vernon%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Thomas F. Westbrook" class="dropBlock__slider__ctrl">Thomas F. Westbrook</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Thomas F. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Thomas F. Westbrook</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Westbrook%2C+Thomas F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey R. Whiteaker" class="dropBlock__slider__ctrl">Jeffrey R. Whiteaker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey R. Whiteaker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteaker%2C+Jeffrey R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yige Wu" class="dropBlock__slider__ctrl">Yige Wu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yige back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yige Wu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wu%2C+Yige" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Midie Xu" class="dropBlock__slider__ctrl">Midie Xu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Midie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Midie Xu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Xu%2C+Midie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lijun Yao" class="dropBlock__slider__ctrl">Lijun Yao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lijun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lijun Yao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Yao%2C+Lijun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Xinpei Yi" class="dropBlock__slider__ctrl">Xinpei Yi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Xinpei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Xinpei Yi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Yi%2C+Xinpei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Fengchao Yu" class="dropBlock__slider__ctrl">Fengchao Yu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Fengchao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Fengchao Yu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Yu%2C+Fengchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kakhaber Zaalishvili" class="dropBlock__slider__ctrl">Kakhaber Zaalishvili</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kakhaber back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kakhaber Zaalishvili</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zaalishvili%2C+Kakhaber" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yuriy Zakhartsev" class="dropBlock__slider__ctrl">Yuriy Zakhartsev</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yuriy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yuriy Zakhartsev</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zakhartsev%2C+Yuriy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert Zelt" class="dropBlock__slider__ctrl">Robert Zelt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert Zelt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zelt%2C+Robert" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Grace Zhao" class="dropBlock__slider__ctrl">Grace Zhao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Grace back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Grace Zhao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Grace" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jun Zhu" class="dropBlock__slider__ctrl">Jun Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jun Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Jun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">28 The authors contributed equally<br/>29 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">January 07, 2021</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.ccell.2020.12.007">https://doi.org/10.1016/j.ccell.2020.12.007</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.ccell.2020.12.007"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.ccell.2020.12.007" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cancer-cell/fulltext/S1535-6108(20)30662-0" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Patterns of transcription factor programs and immune pathway activation define four major subtypes of SCLC with distinct therapeutic vulnerabilities</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="/cancer-cell/fulltext/S1535-6108(21)00105-7" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">A hotspot mutation in transcription factor IKZF3 drives B cell neoplasia via transcriptional dysregulation</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

        



    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">A systematic inventory of HNSCC-associated proteins, phosphosites, and pathways</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Three multi-omic subtypes linked to targeted treatment approaches and immunotherapy</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Widespread deletion of immune modulatory genes accounts for loss of immunogenicity</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Two modes of EGFR activation inform response to anti-EGFR monoclonal antibodies</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">We present a proteogenomic study of 108 human papilloma virus (HPV)-negative head and neck squamous cell carcinomas (HNSCCs). Proteomic analysis systematically catalogs HNSCC-associated proteins and phosphosites, prioritizes copy number drivers, and highlights an oncogenic role for RNA processing genes. Proteomic investigation of mutual exclusivity between <em>FAT1</em> truncating mutations and 11q13.3 amplifications reveals dysregulated actin dynamics as a common functional consequence. Phosphoproteomics characterizes two modes of EGFR activation, suggesting a new strategy to stratify HNSCCs based on EGFR ligand abundance for effective treatment with inhibitory EGFR monoclonal antibodies. Widespread deletion of immune modulatory genes accounts for low immune infiltration in immune-cold tumors, whereas concordant upregulation of multiple immune checkpoint proteins may underlie resistance to anti-programmed cell death protein 1 monotherapy in immune-hot tumors. Multi-omic analysis identifies three molecular subtypes with high potential for treatment with CDK inhibitors, anti-EGFR antibody therapy, and immunotherapy, respectively. Altogether, proteogenomics provides a systematic framework to inform HNSCC biology and treatment.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical Abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical Abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/05ac5f87-c959-4dbb-9dd3-d653665bb980/fx1_lrg.jpg" data-large-image="/cms/attachment/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/02f72d70-ddaf-49a0-a619-b002a8aa7fd1/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/05ac5f87-c959-4dbb-9dd3-d653665bb980/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;HNSCC&#34;&amp;ISSN=1535-6108">HNSCC</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=1535-6108">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=1535-6108">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;immunotherapy&#34;&amp;ISSN=1535-6108">immunotherapy</a></li><li><a href="/action/doSearch?AllField=&#34;EGFR mAb&#34;&amp;ISSN=1535-6108">EGFR mAb</a></li><li><a href="/action/doSearch?AllField=&#34;immune evasion&#34;&amp;ISSN=1535-6108">immune evasion</a></li><li><a href="/action/doSearch?AllField=&#34;combination immune checkpoint blockade&#34;&amp;ISSN=1535-6108">combination immune checkpoint blockade</a></li><li><a href="/action/doSearch?AllField=&#34;CPTAC&#34;&amp;ISSN=1535-6108">CPTAC</a></li><li><a href="/action/doSearch?AllField=&#34;CDK inhibitor&#34;&amp;ISSN=1535-6108">CDK inhibitor</a></li><li><a href="/action/doSearch?AllField=&#34;predictive biomarker&#34;&amp;ISSN=1535-6108">predictive biomarker</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">Head and neck squamous cell carcinoma (HNSCC) is the sixth most common epithelial malignancy worldwide (<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d52364298e1725" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Bray et al., 2018</a><div data-db-target-of="bib16-d52364298e1725" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bray F. </li><li class="loa__item">Ferlay J. </li><li class="loa__item">Soerjomataram I. </li><li class="loa__item">Siegel R.L. </li><li class="loa__item">Torre L.A. </li><li class="loa__item">Jemal A. </li></ul><div class="reference-citations__detials">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries.<div class="popup-ref"><span class="ref__series"><i>Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">394-424</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3322%2Fcaac.21492&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30207593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85053395052&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23716)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&journal=Cancer+J.+Clin.&author=Bray+F.&author=Ferlay+J.&author=Soerjomataram+I.&author=Siegel+R.L.&author=Torre+L.A.&author=Jemal+A.&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and can be broadly classified into human papillomavirus (HPV)-associated (HPV<sup>pos</sup>) and HPV-negative (HPV<sup>neg</sup>) subtypes. Most HNSCC patients are treated with surgery, chemotherapy, and radiotherapy. Targeted agents, including an EGFR monoclonal antibody (mAb) inhibitor and two programmed cell death protein 1 (PD-1) inhibitors, have been approved by the US Food and Drug Administration (FDA) for HNSCC treatment, but overall response rates have been moderate (<div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d52364298e1734" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Baselga et al., 2005</a><div data-db-target-of="bib9-d52364298e1734" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Baselga J. </li><li class="loa__item">Trigo J.M. </li><li class="loa__item">Bourhis J. </li><li class="loa__item">Tortochaux J. </li><li class="loa__item">Cortés-Funes H. </li><li class="loa__item">Hitt R. </li><li class="loa__item">Gascón P. </li><li class="loa__item">Amellal N. </li><li class="loa__item">Harstrick A. </li><li class="loa__item">Eckardt A. </li></ul><div class="reference-citations__detials">Phase II multicenter study of the antiepidermal growth factor receptor monoclonal antibody cetuximab in combination with platinum-based chemotherapy in patients with platinum-refractory metastatic and/or recurrent squamous cell carcinoma of the head and neck.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5568-5577</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.07.119&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16009950&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-23844523253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (432)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5568-5577&journal=J.%C2%A0Clin.+Oncol.&author=Baselga+J.&author=Trigo+J.M.&author=Bourhis+J.&author=Tortochaux+J.&author=Cort%C3%A9s-Funes+H.&author=Hitt+R.&author=Gasc%C3%B3n+P.&author=Amellal+N.&author=Harstrick+A.&author=Eckardt+A.&title=Phase+II+multicenter+study+of+the+antiepidermal+growth+factor+receptor+monoclonal+antibody+cetuximab+in+combination+with+platinum-based+chemotherapy+in+patients+with+platinum-refractory+metastatic+and%2For+recurrent+squamous+cell+carcinoma+of+the+head+and+neck">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d52364298e1737" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Burtness et al., 2005</a><div data-db-target-of="bib19-d52364298e1737" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Burtness B. </li><li class="loa__item">Goldwasser M.A. </li><li class="loa__item">Flood W. </li><li class="loa__item">Mattar B. </li><li class="loa__item">Forastiere A.A. </li><li class="loa__item">Eastern Cooperative Oncology G. </li></ul><div class="reference-citations__detials">Phase III randomized trial of cisplatin plus placebo compared with cisplatin plus cetuximab in metastatic/recurrent head and neck cancer: an Eastern Cooperative Oncology Group study.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">8646-8654</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.02.4646&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16314626&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33144461661&amp;cf=" onclick="newWindow(this.href);return false">Scopus (704)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=8646-8654&journal=J.%C2%A0Clin.+Oncol.&author=Burtness+B.&author=Goldwasser+M.A.&author=Flood+W.&author=Mattar+B.&author=Forastiere+A.A.&author=Eastern+Cooperative+Oncology+G.&title=Phase+III+randomized+trial+of+cisplatin+plus+placebo+compared+with+cisplatin+plus+cetuximab+in+metastatic%2Frecurrent+head+and+neck+cancer%3A+an+Eastern+Cooperative+Oncology+Group+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d52364298e1741" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Herbst et al., 2005</a><div data-db-target-of="bib56-d52364298e1741" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Herbst R.S. </li><li class="loa__item">Arquette M. </li><li class="loa__item">Shin D.M. </li><li class="loa__item">Dicke K. </li><li class="loa__item">Vokes E.E. </li><li class="loa__item">Azarnia N. </li><li class="loa__item">Hong W.K. </li><li class="loa__item">Kies M.S. </li></ul><div class="reference-citations__detials">Phase II multicenter study of the epidermal growth factor receptor antibody cetuximab and cisplatin for recurrent and refractory squamous cell carcinoma of the head and neck.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5578-5587</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.07.120&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16009949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-24944490558&amp;cf=" onclick="newWindow(this.href);return false">Scopus (350)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5578-5587&journal=J.%C2%A0Clin.+Oncol.&author=Herbst+R.S.&author=Arquette+M.&author=Shin+D.M.&author=Dicke+K.&author=Vokes+E.E.&author=Azarnia+N.&author=Hong+W.K.&author=Kies+M.S.&title=Phase+II+multicenter+study+of+the+epidermal+growth+factor+receptor+antibody+cetuximab+and+cisplatin+for+recurrent+and+refractory+squamous+cell+carcinoma+of+the+head+and+neck">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d52364298e1744" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Seiwert et al., 2016</a><div data-db-target-of="bib117-d52364298e1744" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Seiwert T.Y. </li><li class="loa__item">Burtness B. </li><li class="loa__item">Mehra R. </li><li class="loa__item">Weiss J. </li><li class="loa__item">Berger R. </li><li class="loa__item">Eder J.P. </li><li class="loa__item">Heath K. </li><li class="loa__item">McClanahan T. </li><li class="loa__item">Lunceford J. </li><li class="loa__item">Gause C. </li><li>et al.</li></ul><div class="reference-citations__detials">Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">956-965</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27247226&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84969988424&amp;cf=" onclick="newWindow(this.href);return false">Scopus (740)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=956-965&journal=Lancet+Oncol.&author=Seiwert+T.Y.&author=Burtness+B.&author=Mehra+R.&author=Weiss+J.&author=Berger+R.&author=Eder+J.P.&author=Heath+K.&author=McClanahan+T.&author=Lunceford+J.&author=Gause+C.&title=Safety+and+clinical+activity+of+pembrolizumab+for+treatment+of+recurrent+or+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+%28KEYNOTE-012%29%3A+an+open-label%2C+multicentre%2C+phase+1b+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib136" id="back-bib136" data-db-target-for="bib136-d52364298e1747" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib136">Vermorken et al., 2007</a><div data-db-target-of="bib136-d52364298e1747" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vermorken J.B. </li><li class="loa__item">Trigo J. </li><li class="loa__item">Hitt R. </li><li class="loa__item">Koralewski P. </li><li class="loa__item">Diaz-Rubio E. </li><li class="loa__item">Rolland F. </li><li class="loa__item">Knecht R. </li><li class="loa__item">Amellal N. </li><li class="loa__item">Schueler A. </li><li class="loa__item">Baselga J. </li></ul><div class="reference-citations__detials">Open-label, uncontrolled, multicenter phase II study to evaluate the efficacy and toxicity of cetuximab as a single agent in patients with recurrent and/or metastatic squamous cell carcinoma of the head and neck who failed to respond to platinum-based therapy.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2171-2177</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2006.06.7447&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17538161&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34250180582&amp;cf=" onclick="newWindow(this.href);return false">Scopus (694)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=2171-2177&journal=J.%C2%A0Clin.+Oncol.&author=Vermorken+J.B.&author=Trigo+J.&author=Hitt+R.&author=Koralewski+P.&author=Diaz-Rubio+E.&author=Rolland+F.&author=Knecht+R.&author=Amellal+N.&author=Schueler+A.&author=Baselga+J.&title=Open-label%2C+uncontrolled%2C+multicenter+phase+II+study+to+evaluate+the+efficacy+and+toxicity+of+cetuximab+as+a+single+agent+in+patients+with+recurrent+and%2For+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+who+failed+to+respond+to+platinum-based+therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d52364298e1750" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Vermorken et al., 2008</a><div data-db-target-of="bib135-d52364298e1750" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vermorken J.B. </li><li class="loa__item">Mesia R. </li><li class="loa__item">Rivera F. </li><li class="loa__item">Remenar E. </li><li class="loa__item">Kawecki A. </li><li class="loa__item">Rottey S. </li><li class="loa__item">Erfan J. </li><li class="loa__item">Zabolotnyy D. </li><li class="loa__item">Kienzer H.-R. </li><li class="loa__item">Cupissol D. </li><li>et al.</li></ul><div class="reference-citations__detials">Platinum-based chemotherapy plus cetuximab in head and neck cancer.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">359</span></span>: <span class="ref__seriesPages">1116-1127</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1056%2FNEJMoa0802656&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18784101&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-51649085865&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2087)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=1116-1127&journal=N.%C2%A0Engl.+J.+Med.&author=Vermorken+J.B.&author=Mesia+R.&author=Rivera+F.&author=Remenar+E.&author=Kawecki+A.&author=Rottey+S.&author=Erfan+J.&author=Zabolotnyy+D.&author=Kienzer+H.-R.&author=Cupissol+D.&title=Platinum-based+chemotherapy+plus+cetuximab+in+head+and+neck+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Recently, The Cancer Genome Atlas (TCGA) and other studies have defined the genomic landscape and transcriptomic subtypes of HNSCC (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d52364298e1753" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Cancer Genome Atlas, 2015</a><div data-db-target-of="bib22-d52364298e1753" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cancer Genome Atlas N. </li></ul><div class="reference-citations__detials">Comprehensive genomic characterization of head and neck squamous cell carcinomas.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">517</span></span>: <span class="ref__seriesPages">576-582</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnature14129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25631445&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923197605&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1706)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=576-582&journal=Nature&author=Cancer+Genome+Atlas+N.&title=Comprehensive+genomic+characterization+of+head+and+neck+squamous+cell+carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d52364298e1756" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Chung et al., 2004</a><div data-db-target-of="bib28-d52364298e1756" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chung C.H. </li><li class="loa__item">Parker J.S. </li><li class="loa__item">Karaca G. </li><li class="loa__item">Wu J. </li><li class="loa__item">Funkhouser W.K. </li><li class="loa__item">Moore D. </li><li class="loa__item">Butterfoss D. </li><li class="loa__item">Xiang D. </li><li class="loa__item">Zanation A. </li><li class="loa__item">Yin X. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular classification of head and neck squamous cell carcinomas using patterns of gene expression.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">489-500</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15144956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2342492313&amp;cf=" onclick="newWindow(this.href);return false">Scopus (459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=489-500&journal=Cancer+Cell&author=Chung+C.H.&author=Parker+J.S.&author=Karaca+G.&author=Wu+J.&author=Funkhouser+W.K.&author=Moore+D.&author=Butterfoss+D.&author=Xiang+D.&author=Zanation+A.&author=Yin+X.&title=Molecular+classification+of+head+and+neck+squamous+cell+carcinomas+using+patterns+of+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d52364298e1760" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Keck et al., 2015</a><div data-db-target-of="bib62-d52364298e1760" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keck M.K. </li><li class="loa__item">Zuo Z. </li><li class="loa__item">Khattri A. </li><li class="loa__item">Stricker T.P. </li><li class="loa__item">Brown C.D. </li><li class="loa__item">Imanguli M. </li><li class="loa__item">Rieke D. </li><li class="loa__item">Endhardt K. </li><li class="loa__item">Fang P. </li><li class="loa__item">Brägelmann J. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative analysis of head and neck cancer identifies two biologically distinct HPV and three non-HPV subtypes.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">870-881</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-2481&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25492084&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923172078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (191)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=870-881&journal=Clin.+Cancer+Res.&author=Keck+M.K.&author=Zuo+Z.&author=Khattri+A.&author=Stricker+T.P.&author=Brown+C.D.&author=Imanguli+M.&author=Rieke+D.&author=Endhardt+K.&author=Fang+P.&author=Br%C3%A4gelmann+J.&title=Integrative+analysis+of+head+and+neck+cancer+identifies+two+biologically+distinct+HPV+and+three+non-HPV+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d52364298e1763" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Walter et al., 2013</a><div data-db-target-of="bib137-d52364298e1763" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Walter V. </li><li class="loa__item">Yin X. </li><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Zhao N. </li><li class="loa__item">Du Y. </li><li class="loa__item">Ang M.K. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Salazar A.H. </li><li class="loa__item">Hoadley K.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes.<div class="popup-ref"><span class="ref__series"><i>PLoS One.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">e56823</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pone.0056823&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23451093&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874327940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e56823&journal=PLoS+One&author=Walter+V.&author=Yin+X.&author=Wilkerson+M.D.&author=Cabanski+C.R.&author=Zhao+N.&author=Du+Y.&author=Ang+M.K.&author=Hayward+M.C.&author=Salazar+A.H.&author=Hoadley+K.A.&title=Molecular+subtypes+in+head+and+neck+cancer+exhibit+distinct+patterns+of+chromosomal+gain+and+loss+of+canonical+cancer+genes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, a complete understanding of how genetic aberrations drive tumor phenotypes remains elusive, and translation of genomic and transcriptomic findings into improved HNSCC treatment has been limited.</div><div class="section-paragraph">By integrating mass spectrometry (MS)-based proteomics with genomics and transcriptomics, we performed an integrated proteogenomic characterization of 108 HPV<sup>neg</sup> HNSCCs. We focused on HPV<sup>neg</sup> HNSCCs because they account for 75% of all HNSCCs and have distinct molecular profiles and significantly worse prognosis compared with HPV<sup>pos</sup> tumors (<div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d52364298e1777" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Kreimer et al., 2005</a><div data-db-target-of="bib72-d52364298e1777" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kreimer A.R. </li><li class="loa__item">Clifford G.M. </li><li class="loa__item">Boyle P. </li><li class="loa__item">Franceschi S. </li></ul><div class="reference-citations__detials">Human papillomavirus types in head and neck squamous cell carcinomas worldwide: a systematic review.<div class="popup-ref"><span class="ref__series"><i>Cancer Epidemiol. Biomarkers Prev.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">467-475</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1055-9965.EPI-04-0551&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15734974&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-13944263935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1506)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=467-475&journal=Cancer+Epidemiol.+Biomarkers+Prev.&author=Kreimer+A.R.&author=Clifford+G.M.&author=Boyle+P.&author=Franceschi+S.&title=Human+papillomavirus+types+in+head+and+neck+squamous+cell+carcinomas+worldwide%3A+a+systematic+review">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Our study systematically catalogs HPV<sup>neg</sup> HNSCC-associated proteins, phosphosites, and signaling pathways. Proteogenomic integration provides functional insights into genomic aberrations, with practical implications for precision treatment of patients with HPV<sup>neg</sup> HNSCC.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic profiling</span></h3><div class="section-paragraph">We prospectively collected 110 treatment-naive primary HNSCC tumors and matched blood samples (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/c694c779-4ad1-4577-bdbb-5df01b62ad6d/mmc2" class="scroll-into-link">Table S1</a>), and 66 tumors had matched normal adjacent tissues (NATs). Homogenized samples were aliquoted for molecular profiling using whole-exome sequencing (WES), whole-genome sequencing (WGS), methylation array, RNA sequencing (RNA-seq), microRNA sequencing (miRNA-seq), and isobaric tandem mass tag (TMT) labeling-based global proteomics and phosphoproteomics (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A). One sample with evidence of HPV infection by RNA-seq was removed from downstream analysis (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S1</a>A). The cohort was 87% male and tumor sites were predominantly from the oral cavity and larynx (44.5% each). Consistent with self-reporting, genomics-based smoking inference associated 70% of the patients with strong evidence of smoking (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S1</a>B and S1C).<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/e16c7f25-3283-4c7e-a7e9-a85c2e9c05c1/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/5eb25179-ab9b-414e-9598-e692ecf96870/gr1_lrg.jpg" data-large-image="/cms/attachment/e16c7f25-3283-4c7e-a7e9-a85c2e9c05c1/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Proteogenomic profiling and impact of genetic aberrations on proteins</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Cohort clinical features and omic data generation.</div><div class="section-paragraph">(B and C) (B) Global proteomics and (C) peptide-level phosphoproteomics PCA plots.</div><div class="section-paragraph">(D) Gene-wise mRNA-protein correlation and pathway enrichment.</div><div class="section-paragraph">(E) Area under the receiver operating characteristic curve (AUROC) for KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway membership prediction using RNA and protein data. Red and blue indicate pathways with &gt;10% difference between the two.</div><div class="section-paragraph">(F) Arm-level SCNAs.</div><div class="section-paragraph">(G) Focal-level SCNAs with known drivers and RNA processing genes (red) annotated. (H) Prioritization of genes in focal amplification peaks.</div><div class="section-paragraph">(I) Gene Ontology (GO) terms enriched for prioritized SCNA drivers (Fisher's exact test).</div><div class="section-paragraph">(J) Protein abundance of RNA processing genes in tumors and NATs, annotated with amplification rate, copy number-protein correlation (Pearson's correlation), and presence (green) in the COSMIC (Catalogue of Somatic Mutations in Cancer) Cancer Gene Census.</div><div class="section-paragraph">(K) Mutation frequency and type for the most frequently mutated genes.</div><div class="section-paragraph">(L) Comparisons of RNA and protein levels for <em>AJUBA</em> and <em>KMT2D</em> between samples with truncating mutations and WT samples.</div><div class="section-paragraph"><sup>∗∗</sup>p &lt; 0.01, Student's t test. n.s., not significant. Numbers in parentheses represent the sample sizes for the involved groups. See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2375" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S1</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2380" href="/cms/10.1016/j.ccell.2020.12.007/attachment/c694c779-4ad1-4577-bdbb-5df01b62ad6d/mmc2" target="_blank">Tables S1</a> and <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2385" href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3" target="_blank">S2</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/e16c7f25-3283-4c7e-a7e9-a85c2e9c05c1/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/5eb25179-ab9b-414e-9598-e692ecf96870/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Proteomic analysis identified 11,744 proteins. Phosphoproteomic analysis identified 97,210 phosphopeptides, covering 56,959 confidently localized phosphosites from 8,133 genes, including 81% on serine, 16% on threonine, and 3% on tyrosine. Replicate samples showed high measurement reproducibility across the TMT plexes, and there were no observable batch effects by TMT plex (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S1</a>D–S1G). Unsupervised principal component analysis (PCA) of both proteomic and phosphoproteomic data separated tumor samples from NATs (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>B and 1C). One tumor and three NAT samples with questionable tumor/NAT identity in data quality control (labeled in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>B and 1C) were confirmed by pathological inspection and removed from downstream analysis.</div><div class="section-paragraph">For the 108 tumors, the median gene-wise Spearman's correlation between protein and RNA abundance was 0.52, and the median sample-wise correlation was 0.43. Genes involved in forming large protein complexes, such as those related to complement activation, oxidative phosphorylation, and transcription initiation, showed lower protein-RNA correlation (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>D). Protein data substantially outperformed RNA data in co-expression-based gene function prediction (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>E), suggesting a critical role for protein-level regulation in determining gene functions.</div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Impact of genetic aberrations on cognate proteins</span></h3><div class="section-paragraph">Somatic copy number alteration (SCNA) analysis identified arm-level amplifications and deletions (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>F). Focal peaks included amplifications of 3q26.33, 7p11.2, 7q22.3, 8p11.23, and 11q13.3 and deletions of 8p23.2 and 9p21.3, among others (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>G). The strongest focal alteration was observed at 11q13.3. Some of these focal SCNA hotspots were also associated with structural variation events (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S1</a>H and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3" class="scroll-into-link">Table S2</a>).</div><div class="section-paragraph">By filtering for correlated copy number (CN), mRNA, and protein levels across tumor samples and concordant protein-level changes between paired tumor and NAT samples, we prioritized 202 putative SCNA drivers, including well-established <em>PIK3CA</em>, <em>EGFR</em>, <em>CCND1</em>, and <em>CTTN</em>, from a total of 759 quantifiable genes in the focal amplicons (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>H and 1G, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3" class="scroll-into-link">Table S2</a>). The prioritized genes showed higher essentiality in HPV<sup>neg</sup> HNSCC cell lines in a genome-wide genetic perturbation screen (<div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d52364298e1864" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Tsherniak et al., 2017</a><div data-db-target-of="bib130-d52364298e1864" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tsherniak A. </li><li class="loa__item">Vazquez F. </li><li class="loa__item">Montgomery P.G. </li><li class="loa__item">Weir B.A. </li><li class="loa__item">Kryukov G. </li><li class="loa__item">Cowley G.S. </li><li class="loa__item">Gill S. </li><li class="loa__item">Harrington W.F. </li><li class="loa__item">Pantel S. </li><li class="loa__item">Krill-Burger J.M. </li><li>et al.</li></ul><div class="reference-citations__detials">Defining a cancer dependency map.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">170</span></span>: <span class="ref__seriesPages">564-576.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28753430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85026422523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (435)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=564-576.e16&journal=Cell&author=Tsherniak+A.&author=Vazquez+F.&author=Montgomery+P.G.&author=Weir+B.A.&author=Kryukov+G.&author=Cowley+G.S.&author=Gill+S.&author=Harrington+W.F.&author=Pantel+S.&author=Krill-Burger+J.M.&title=Defining+a+cancer+dependency+map">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S1</a>I). Enrichment analysis associated the prioritized genes with cell cycle, DNA replication, chromosome organization, mRNA processing, and RNA splicing (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>I). Amplification of the RNA processing genes (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>G) has not been linked to HNSCC tumorigenesis previously, but some (<em>WWTR1</em>, <em>TBL1XR1</em>, <em>CUX1</em>, <em>TRRAP</em>, <em>NSD3</em>) are known cancer genes (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>J).</div><div class="section-paragraph">Frequently mutated genes in this cohort (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>K) were consistent with those in TCGA HPV<sup>neg</sup> HNSCCs, and we assessed the impact of mutations on cognate gene products. Missense mutations in <em>TP53</em> were associated with increased p53 mRNA and protein abundance (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S1</a>J), suggesting that certain <em>TP53</em> mutations might endow oncogenic gain of function to this protein. Several frequently mutated genes had a substantial proportion of truncating mutations (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>K), which typically induce nonsense-mediated decay (NMD) of cognate transcripts. Interestingly, truncating mutations in <em>KMT2D</em> and <em>AJUBA</em> were associated with reduced protein but not mRNA abundance (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>L). Most of the truncating variants in these two genes may escape NMD according to the NMD rules (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d52364298e1916" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Lindeboom et al., 2016</a><div data-db-target-of="bib85-d52364298e1916" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lindeboom R.G.H. </li><li class="loa__item">Supek F. </li><li class="loa__item">Lehner B. </li></ul><div class="reference-citations__detials">The rules and impact of nonsense-mediated mRNA decay in human cancers.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">1112-1118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3664&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27618451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84987641578&amp;cf=" onclick="newWindow(this.href);return false">Scopus (137)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=1112-1118&journal=Nat.+Genet.&author=Lindeboom+R.G.H.&author=Supek+F.&author=Lehner+B.&title=The+rules+and+impact+of+nonsense-mediated+mRNA+decay+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3" class="scroll-into-link">Table S2</a>). Thus, proteomic data were crucial to support the tumor suppressor role of these genes.</div><div class="section-paragraph">DNA methylation of 91 genes was associated with both reduced mRNA and protein abundance in tumors (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3" class="scroll-into-link">Table S2</a>). These genes included several putative tumor suppressor genes whose expression was reported to be regulated by promoter methylation in other cancer types, such as <em>NEFM</em> (<div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d52364298e1929" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Calmon et al., 2015</a><div data-db-target-of="bib21-d52364298e1929" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Calmon M.F. </li><li class="loa__item">Jeschke J. </li><li class="loa__item">Zhang W. </li><li class="loa__item">Dhir M. </li><li class="loa__item">Siebenkäs C. </li><li class="loa__item">Herrera A. </li><li class="loa__item">Tsai H.-C. </li><li class="loa__item">O'Hagan H.M. </li><li class="loa__item">Pappou E.P. </li><li class="loa__item">Hooker C.M. </li><li>et al.</li></ul><div class="reference-citations__detials">Epigenetic silencing of neurofilament genes promotes an aggressive phenotype in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Epigenetics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">622-632</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1080%2F15592294.2015.1050173&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25985363&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=622-632&journal=Epigenetics&author=Calmon+M.F.&author=Jeschke+J.&author=Zhang+W.&author=Dhir+M.&author=Siebenk%C3%A4s+C.&author=Herrera+A.&author=Tsai+H.-C.&author=O%27Hagan+H.M.&author=Pappou+E.P.&author=Hooker+C.M.&title=Epigenetic+silencing+of+neurofilament+genes+promotes+an+aggressive+phenotype+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), <em>MGMT</em> (<div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d52364298e1934" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Rivera et al., 2010</a><div data-db-target-of="bib110-d52364298e1934" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rivera A.L. </li><li class="loa__item">Pelloski C.E. </li><li class="loa__item">Gilbert M.R. </li><li class="loa__item">Colman H. </li><li class="loa__item">De La Cruz C. </li><li class="loa__item">Sulman E.P. </li><li class="loa__item">Bekele B.N. </li><li class="loa__item">Aldape K.D. </li></ul><div class="reference-citations__detials">MGMT promoter methylation is predictive of response to radiotherapy and prognostic in the absence of adjuvant alkylating chemotherapy for glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Neuro Oncol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">116-121</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fneuonc%2Fnop020&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20150378&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-75849146316&amp;cf=" onclick="newWindow(this.href);return false">Scopus (235)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=116-121&journal=Neuro+Oncol.&author=Rivera+A.L.&author=Pelloski+C.E.&author=Gilbert+M.R.&author=Colman+H.&author=De+La+Cruz+C.&author=Sulman+E.P.&author=Bekele+B.N.&author=Aldape+K.D.&title=MGMT+promoter+methylation+is+predictive+of+response+to+radiotherapy+and+prognostic+in+the+absence+of+adjuvant+alkylating+chemotherapy+for+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), <em>GLDC</em> (<div class="dropBlock reference-citations"><a href="#bib99" id="back-bib99" data-db-target-for="bib99-d52364298e1940" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Min et al., 2016</a><div data-db-target-of="bib99-d52364298e1940" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Min H.L. </li><li class="loa__item">Kim J. </li><li class="loa__item">Kim W.H. </li><li class="loa__item">Jang B.G. </li><li class="loa__item">Kim M.A. </li></ul><div class="reference-citations__detials">Epigenetic silencing of the putative tumor suppressor gene GLDC (glycine dehydrogenase) in gastric carcinoma.<div class="popup-ref"><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">179-187</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26722042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=179-187&journal=Anticancer+Res.&author=Min+H.L.&author=Kim+J.&author=Kim+W.H.&author=Jang+B.G.&author=Kim+M.A.&title=Epigenetic+silencing+of+the+putative+tumor+suppressor+gene+GLDC+%28glycine+dehydrogenase%29+in+gastric+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and <em>CHFR</em> (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d52364298e1945" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Brandes et al., 2005</a><div data-db-target-of="bib15-d52364298e1945" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brandes J.C. </li><li class="loa__item">van Engeland M. </li><li class="loa__item">Wouters K.A.D. </li><li class="loa__item">Weijenberg M.P. </li><li class="loa__item">Herman J.G. </li></ul><div class="reference-citations__detials">CHFR promoter hypermethylation in colon cancer correlates with the microsatellite instability phenotype.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1152-1156</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fcarcin%2Fbgi058&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15760919&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-22344454699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (69)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1152-1156&journal=Carcinogenesis&author=Brandes+J.C.&author=van+Engeland+M.&author=Wouters+K.A.D.&author=Weijenberg+M.P.&author=Herman+J.G.&title=CHFR+promoter+hypermethylation+in+colon+cancer+correlates+with+the+microsatellite+instability+phenotype">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic alterations associated with tumorigenesis and prognosis</span></h3><div class="section-paragraph">We compared 63 tumors vs paired NATs to identify HNSCC-associated alterations in transcripts (mRNA, miRNA, and circular RNA [circRNA]), proteins, and phosphosites (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4" class="scroll-into-link">Table S3</a>). Here we focus on results from the analysis of proteins and phosphosites quantified in ≥50% of the pairs. For proteins, 3,355 (35%) were significantly increased and 3,163 (33%) were significantly decreased in tumors (adjusted p &lt; 0.01, Wilcoxon signed-rank test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A). The 104 proteins increased by &gt;2-fold were enriched in biological processes such as protein hydroxylation, leukocyte migration, cell chemotaxis, and angiogenesis, whereas the 488 decreased proteins were enriched in acute inflammatory response, platelet degranulation, muscle system process, and fatty acid metabolic process (adjusted p &lt; 0.01, Fisher's exact test, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4" class="scroll-into-link">Table S3</a>). After controlling for epithelium content in a multivariate model, 63 out of the 104 remained significantly elevated by &gt;2-fold (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S2</a>A and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4" class="scroll-into-link">Table S3</a>), and over two-thirds showed above-average abundance of all proteins (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S2</a>B). Among the 63, 22 are secretable and could serve as putative salivary biomarkers, seven can be targeted by FDA-approved drugs, and one is a cancer/testis (C/T) antigen (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Table S3</a>). Notably, KIT, CAMP, and other highly increased proteins such as DEFA3, DEFA1B, CRTAP, and CLCNKA had decreased mRNA in tumors (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>C and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S2</a>C). Elevated tumor expression was supported by data-independent acquisition (DIA) proteomics (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S2</a>D. I). Immunohistochemistry (IHC) data from the Human Protein Atlas (HPA) provided further independent validation for a subset of proteins (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S2</a>E).<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/ed23ca90-b6ef-44b3-8bf9-70c7c8aacb6e/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/285b8284-029d-459a-9a6f-7562299702cb/gr2_lrg.jpg" data-large-image="/cms/attachment/ed23ca90-b6ef-44b3-8bf9-70c7c8aacb6e/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">Proteomic alterations associated with tumorigenesis and prognosis</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Protein abundance differences between tumors and NATs (Wilcoxon signed-rank test). Representative GO terms for 2-fold increased and decreased proteins are listed.</div><div class="section-paragraph">(B) Abundance fold changes (FCs) for selected highly elevated proteins annotated with potential clinical utilities.</div><div class="section-paragraph">(C) Comparisons of RNA and protein levels for KIT and CAMP between tumors and NATs, Student's t test.</div><div class="section-paragraph">(D) Comparison of protein changes in two anatomic sites. Dot colors indicate shared or site-specific elevations, and font colors indicate different types of clinical utilities.</div><div class="section-paragraph">(E) Comparison of protein changes in tumors with strong and weak smoking evidence, colored as panel (D).</div><div class="section-paragraph">(F) Phosphosite abundance differences between tumors and NATs (Wilcoxon signed-rank test). Representative GO terms for proteins with 2-fold increased or decreased phosphosites are listed.</div><div class="section-paragraph">(G) Comparison of abundance changes between phosphosites and their corresponding proteins.</div><div class="section-paragraph">(H) Kinases with increased activity inferred from phosphorylation of its substrates (normalized enrichment score) or increased phosphorylation of its activating site.</div><div class="section-paragraph">(I) Increased phosphorylation (circle) on transcription factor substrates (rectangle) of kinases (hexagon) with increased activity. All transcription factors had increased inferred activity from the RNA targets.</div><div class="section-paragraph">(J) The common pathways enriched with proteins or phosphoproteins associated with OS or PFS (Fisher's exact test).</div><div class="section-paragraph">(K) Kaplan-Meier plot comparing OS for patients stratified by the median Chrldx score, log rank test. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2390" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S2</a> and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2395" href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4" target="_blank">Table S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/ed23ca90-b6ef-44b3-8bf9-70c7c8aacb6e/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/285b8284-029d-459a-9a6f-7562299702cb/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Most proteins behaved similarly in the tumor vs NAT comparison whether derived from larynx or oral cavity (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D). However, 261 proteins, many muscle related, were decreased specifically in oral cavity tumors, likely due to higher levels of muscle tissue in oral cavity NATs compared with larynx NATs. Several proteins with potential clinical utility were increased in a particular site (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D), including MAGEB2, PTGS2, matrix metalloproteinase (MMP) 7, COL10A1, and IL36G in larynx (blue dots), and MMP3, MMP10, and CRELD2 in oral cavity (red dots). When grouped based on smoking evidence, tumors with strong smoking evidence showed specific increase of four secretable proteins (CXCL8, SFRP4, IL36G, COL22A1) and two targets of approved drugs (KIT, SLC7A11) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>E). The highly specific overexpression of KIT in tumors with strong smoking evidence explained the large variation of protein fold change observed for KIT across all tumors (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B).</div><div class="section-paragraph">For phosphosites, 7,265 (35%) were significantly increased and 6,320 (31%) were significantly decreased in tumors vs NATs (adjusted p &lt; 0.01, Wilcoxon signed-rank test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F). Proteins with a phosphosite change &gt;2-fold were enriched in biological processes such as DNA replication and cell cycle checkpoint, whereas proteins with a phosphosite change decreased by &gt; 2-fold were enriched in muscle system process and actin filament organization. Among the 559 phosphosites increased by &gt;2-fold with stronger changes than in the corresponding protein (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>G), only 8% had known functional annotations. Of these, 30% were involved in cell cycle regulation, including a site on the essential mitotic regulator CDC20 (<div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d52364298e2014" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Hein et al., 2017</a><div data-db-target-of="bib55-d52364298e2014" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hein J.B. </li><li class="loa__item">Hertz E.P.T. </li><li class="loa__item">Garvanska D.H. </li><li class="loa__item">Kruse T. </li><li class="loa__item">Nilsson J. </li></ul><div class="reference-citations__detials">Distinct kinetics of serine and threonine dephosphorylation are essential for mitosis.<div class="popup-ref"><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">1433-1440</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncb3634&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29084198&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85036508836&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1433-1440&journal=Nat.+Cell+Biol.&author=Hein+J.B.&author=Hertz+E.P.T.&author=Garvanska+D.H.&author=Kruse+T.&author=Nilsson+J.&title=Distinct+kinetics+of+serine+and+threonine+dephosphorylation+are+essential+for+mitosis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib141" id="back-bib141" data-db-target-for="bib141-d52364298e2017" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Wang et al., 2015</a><div data-db-target-of="bib141-d52364298e2017" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang L. </li><li class="loa__item">Zhang J. </li><li class="loa__item">Wan L. </li><li class="loa__item">Zhou X. </li><li class="loa__item">Wang Z. </li><li class="loa__item">Wei W. </li></ul><div class="reference-citations__detials">Targeting Cdc20 as a novel cancer therapeutic strategy.<div class="popup-ref"><span class="ref__series"><i>Pharmacol. Ther.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">151</span></span>: <span class="ref__seriesPages">141-151</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.pharmthera.2015.04.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25850036&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84930088030&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2015&pages=141-151&journal=Pharmacol.+Ther.&author=Wang+L.&author=Zhang+J.&author=Wan+L.&author=Zhou+X.&author=Wang+Z.&author=Wei+W.&title=Targeting+Cdc20+as+a+novel+cancer+therapeutic+strategy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). An additional 12% were involved in cytoskeleton reorganization, including ECT2 T359, which contributes to tumor cell invasion (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d52364298e2020" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">de Cárcer et al., 2017</a><div data-db-target-of="bib37-d52364298e2020" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">de Cárcer G. </li><li class="loa__item">Wachowicz P. </li><li class="loa__item">Martínez-Martínez S. </li><li class="loa__item">Oller J. </li><li class="loa__item">Méndez-Barbero N. </li><li class="loa__item">Escobar B. </li><li class="loa__item">González-Loyola A. </li><li class="loa__item">Takaki T. </li><li class="loa__item">El Bakkali A. </li><li class="loa__item">Cámara J.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Plk1 regulates contraction of postmitotic smooth muscle cells and is required for vascular homeostasis.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">964-974</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnm.4364&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28692064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85029285746&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=964-974&journal=Nat.+Med.&author=de+C%C3%A1rcer+G.&author=Wachowicz+P.&author=Mart%C3%ADnez-Mart%C3%ADnez+S.&author=Oller+J.&author=M%C3%A9ndez-Barbero+N.&author=Escobar+B.&author=Gonz%C3%A1lez-Loyola+A.&author=Takaki+T.&author=El+Bakkali+A.&author=C%C3%A1mara+J.A.&title=Plk1+regulates+contraction+of+postmitotic+smooth+muscle+cells+and+is+required+for+vascular+homeostasis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d52364298e2024" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Justilien et al., 2011</a><div data-db-target-of="bib59-d52364298e2024" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Justilien V. </li><li class="loa__item">Jameison L. </li><li class="loa__item">Der C.J. </li><li class="loa__item">Rossman K.L. </li><li class="loa__item">Fields A.P. </li></ul><div class="reference-citations__detials">Oncogenic activity of Ect2 is regulated through protein kinase C iota-mediated phosphorylation.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">286</span></span>: <span class="ref__seriesPages">8149-8157</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21189248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79953154962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=8149-8157&journal=J.%C2%A0Biol.+Chem.&author=Justilien+V.&author=Jameison+L.&author=Der+C.J.&author=Rossman+K.L.&author=Fields+A.P.&title=Oncogenic+activity+of+Ect2+is+regulated+through+protein+kinase+C+iota-mediated+phosphorylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). TP53BP1 S1623, which inhibits DNA repair, was also highly phosphorylated (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d52364298e2027" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Benada et al., 2015</a><div data-db-target-of="bib12-d52364298e2027" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benada J. </li><li class="loa__item">Burdová K. </li><li class="loa__item">Lidak T. </li><li class="loa__item">von Morgen P. </li><li class="loa__item">Macurek L. </li></ul><div class="reference-citations__detials">Polo-like kinase 1 inhibits DNA damage response during mitosis.<div class="popup-ref"><span class="ref__series"><i>Cell Cycle.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">219-231</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.4161%2F15384101.2014.977067&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25607646&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84926624707&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=219-231&journal=Cell+Cycle&author=Benada+J.&author=Burdov%C3%A1+K.&author=Lidak+T.&author=von+Morgen+P.&author=Macurek+L.&title=Polo-like+kinase+1+inhibits+DNA+damage+response+during+mitosis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Differential phosphosite analysis identified only one hyperphosphorylated activating site on a kinase, CHEK2 (<div class="dropBlock reference-citations"><a href="#bib88" id="back-bib88" data-db-target-for="bib88-d52364298e2030" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib88">Lovly et al., 2008</a><div data-db-target-of="bib88-d52364298e2030" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lovly C.M. </li><li class="loa__item">Yan L. </li><li class="loa__item">Ryan C.E. </li><li class="loa__item">Takada S. </li><li class="loa__item">Piwnica-Worms H. </li></ul><div class="reference-citations__detials">Regulation of Chk2 ubiquitination and signaling through autophosphorylation of serine 379.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">5874-5885</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1128%2FMCB.00821-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18644861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-52649098505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=5874-5885&journal=Mol.+Cell+Biol.&author=Lovly+C.M.&author=Yan+L.&author=Ryan+C.E.&author=Takada+S.&author=Piwnica-Worms+H.&title=Regulation+of+Chk2+ubiquitination+and+signaling+through+autophosphorylation+of+serine+379">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Kinase activities were also inferred based on the levels of substrate phosphorylation. A total of nine kinases had significantly increased activity, including targets of approved inhibitors (CDK4 and CDK6) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>H). Using predicted substrates for kinases (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d52364298e2036" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Linding et al., 2007</a><div data-db-target-of="bib86-d52364298e2036" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linding R. </li><li class="loa__item">Jensen L.J. </li><li class="loa__item">Ostheimer G.J. </li><li class="loa__item">van Vugt M.A.T.M. </li><li class="loa__item">Jørgensen C. </li><li class="loa__item">Miron I.M. </li><li class="loa__item">Diella F. </li><li class="loa__item">Colwill K. </li><li class="loa__item">Taylor L. </li><li class="loa__item">Elder K. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic discovery of in vivo phosphorylation networks.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">129</span></span>: <span class="ref__seriesPages">1415-1426</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17570479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34250743173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (553)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=1415-1426&journal=Cell&author=Linding+R.&author=Jensen+L.J.&author=Ostheimer+G.J.&author=van+Vugt+M.A.T.M.&author=J%C3%B8rgensen+C.&author=Miron+I.M.&author=Diella+F.&author=Colwill+K.&author=Taylor+L.&author=Elder+K.&title=Systematic+discovery+of+in%C2%A0vivo+phosphorylation+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we identified 12 more kinases with increased activity in tumors, including CHUK and IKBKB, which are targets of approved inhibitors (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S2</a>F). Increased kinase activity was supported by phosphorylation of corresponding transcription factor substrates and, in turn, increased transcription factor activity inferred from their mRNA targets (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>I, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Table S3</a>).</div><div class="section-paragraph">Survival analysis identified 263 proteins and 173 phosphoproteins significantly associated with overall survival (OS), and 162 proteins and 164 phosphoproteins significantly associated with progression-free survival (PFS) (nominal p &lt; 0.01, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4" class="scroll-into-link">Table S3</a>). OXSR1 and GPALPP1 remained significant for both after multiple test adjustment, and eight phosphoproteins remained significant for OS (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S2</a>G and S2H, adjusted p &lt; 0.2). Strikingly, poor-prognosis-associated proteins/phosphoproteins were enriched in pathways reported in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>I for SCNA drivers, including DNA replication, cell cycle, and RNA processing (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>J), suggesting a potential relationship between SCNA and adverse clinical outcome. Indeed, tumors with higher chromosome instability scores (ChrIdx score) tended to have shorter OS (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>K) and PFS (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S2</a>I).</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutually exclusive <em>FAT1</em> truncating mutations and 11q13.3 amplification converge to protein-level actin dysregulation</span></h3><div class="section-paragraph"><em>FAT1</em> encodes an atypical cadherin and is one of the most frequently mutated genes in this cohort (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>K) and the TCGA cohort. Truncating mutations account for &gt;70% of all <em>FAT1</em> mutations in both cohorts, in sharp contrast to other cancer types (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S3</a>A). By integrating CN data, we further divided <em>FAT1</em> mutations into four groups (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S3</a>B and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/68ecf9b0-b448-4bd6-a8c3-a16895560d68/mmc5" class="scroll-into-link">Table S4</a>). Compared with wild-type (WT) <em>FAT1</em>, samples with <em>FAT1</em> biallelic truncations showed reduced FAT1 protein and mRNA levels (p = 3.4 × 10<sup>−6</sup> and p = 1.5 × 10<sup>−3</sup>, Student's t test, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S3</a>C and S3D). Since samples with other types of <em>FAT1</em> mutations showed intermediate and more variable mRNA and protein levels, we excluded them from downstream analyses to focus on the most frequent mutation type with the strongest <em>cis</em> effects.</div><div class="section-paragraph">Mutual exclusivity was observed (p = 6.0 × 10<sup>−3</sup>, Fisher's exact test) between <em>FAT1</em> truncating mutation and 11q13.3 amplification (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A), the strongest focal SCNA in our cohort (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>G). This was confirmed in TCGA HPV<sup>neg</sup> HNSCCs (p &lt; 0.001, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S3</a>E). For all nine protein coding genes in 11q13.3, amplification resulted in concordantly increased mRNA and protein abundance (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A).<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/2d2b4b0e-74a2-40f8-949f-190cc846f614/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/14341aab-e8b0-4e87-a6fd-56400a1e91ff/gr3_lrg.jpg" data-large-image="/cms/attachment/2d2b4b0e-74a2-40f8-949f-190cc846f614/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Mutually exclusive <em>FAT1</em> truncating mutations and 11q13.3 amplification converge to protein-level actin dysregulation</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmap visualizing multi-omic profiles of <em>FAT1</em> and the nine coding genes in 11q13.3.</div><div class="section-paragraph">(B) GSEA plots for actin-related pathways in <em>FAT1</em> truncation or 11q13.3 amplification vs WT comparisons.</div><div class="section-paragraph">(C) Relative mRNA and protein abundance in the <em>FAT1</em> truncation, 11q13.3 amplification, and WT groups for five actin isoforms. <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, Student's t test. n.s., not significant.</div><div class="section-paragraph">(D) CTTN phosphosite abundance differences between the 11q13.3 amplification and WT groups (Student's t test).</div><div class="section-paragraph">(E) Relapse-free survival in HPV<sup>neg</sup> HNSCC TCGA patients with <em>FAT1</em> truncation or 11q13.3 amplification compared with WT (log rank test).</div><div class="section-paragraph">(F) Proposed model explaining the mutual exclusivity between <em>FAT1</em> truncating mutations and 11q13.3 amplification. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2400" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S3</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2405" href="/cms/10.1016/j.ccell.2020.12.007/attachment/68ecf9b0-b448-4bd6-a8c3-a16895560d68/mmc5" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/2d2b4b0e-74a2-40f8-949f-190cc846f614/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/14341aab-e8b0-4e87-a6fd-56400a1e91ff/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Mutual exclusivity may arise when two aberrations are functionally equivalent (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d52364298e2139" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Ciriello et al., 2012</a><div data-db-target-of="bib30-d52364298e2139" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ciriello G. </li><li class="loa__item">Cerami E. </li><li class="loa__item">Sander C. </li><li class="loa__item">Schultz N. </li></ul><div class="reference-citations__detials">Mutual exclusivity analysis identifies oncogenic network modules.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">398-406</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.125567.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21908773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84856535081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (385)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=398-406&journal=Genome+Res.&author=Ciriello+G.&author=Cerami+E.&author=Sander+C.&author=Schultz+N.&title=Mutual+exclusivity+analysis+identifies+oncogenic+network+modules">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Pathway enrichment analysis using proteomic data showed downregulation of proteins involved in actin dynamics in both <em>FAT1</em> truncated and 11q13.3 amplified groups compared with the WT group (adjusted p &lt; 0.05, gene set enrichment analysis [GSEA], <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B). Despite varying mRNA abundance of actin genes between groups, protein abundance for five actin genes was higher in the WT group (p &lt; 0.05, Student's t test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). In particular, beta-actin (encoded by <em>ACTB</em>), a non-muscle actin implicated in cell motility, structure, and integrity (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d52364298e2153" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Drazic et al., 2018</a><div data-db-target-of="bib39-d52364298e2153" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Drazic A. </li><li class="loa__item">Aksnes H. </li><li class="loa__item">Marie M. </li><li class="loa__item">Boczkowska M. </li><li class="loa__item">Varland S. </li><li class="loa__item">Timmerman E. </li><li class="loa__item">Foyn H. </li><li class="loa__item">Glomnes N. </li><li class="loa__item">Rebowski G. </li><li class="loa__item">Impens F. </li><li>et al.</li></ul><div class="reference-citations__detials">NAA80 is actin's N-terminal acetyltransferase and regulates cytoskeleton assembly and cell motility.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">115</span></span>: <span class="ref__seriesPages">4399-4404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1073%2Fpnas.1718336115&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29581253&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045948499&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4399-4404&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Drazic+A.&author=Aksnes+H.&author=Marie+M.&author=Boczkowska+M.&author=Varland+S.&author=Timmerman+E.&author=Foyn+H.&author=Glomnes+N.&author=Rebowski+G.&author=Impens+F.&title=NAA80+is+actin%27s+N-terminal+acetyltransferase+and+regulates+cytoskeleton+assembly+and+cell+motility">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was significantly downregulated at the protein level in both <em>FAT1</em> truncated and 11q13.3 amplified groups despite upregulated mRNA. This finding was verified using DIA proteomic data (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S3</a>F). These data suggest that <em>FAT1</em> truncation and 11q13.3 amplification converge on regulating actin dynamics at the protein level.</div><div class="section-paragraph">Depletion of FAT1 has been causally linked to dysregulated actin organization at the cell periphery, looser cell association, and abrogated cell polarity (<div class="dropBlock reference-citations"><a href="#bib127" id="back-bib127" data-db-target-for="bib127-d52364298e2165" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib127">Tanoue and Takeichi, 2004</a><div data-db-target-of="bib127-d52364298e2165" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tanoue T. </li><li class="loa__item">Takeichi M. </li></ul><div class="reference-citations__detials">Mammalian Fat1 cadherin regulates actin dynamics and cell-cell contact.<div class="popup-ref"><span class="ref__series"><i>J. Cell Biol.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">517-528</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1083%2Fjcb.200403006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15148305&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2542438768&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2004&pages=517-528&journal=J.%C2%A0Cell+Biol.&author=Tanoue+T.&author=Takeichi+M.&title=Mammalian+Fat1+cadherin+regulates+actin+dynamics+and+cell-cell+contact">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphoproteomic data showed significantly elevated levels of CTTN phosphosites in 11q13.3 amplified samples compared with WT (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>D). The most elevated phosphosite was CTTN S418, reported to alter cell motility and cytoskeletal rearrangement to promote tumor progression (<div class="dropBlock reference-citations"><a href="#bib89" id="back-bib89" data-db-target-for="bib89-d52364298e2171" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">MacGrath and Koleske, 2012</a><div data-db-target-of="bib89-d52364298e2171" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">MacGrath S.M. </li><li class="loa__item">Koleske A.J. </li></ul><div class="reference-citations__detials">Cortactin in cell migration and cancer at a glance.<div class="popup-ref"><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">125</span></span>: <span class="ref__seriesPages">1621-1626</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1242%2Fjcs.093781&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22566665&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863084923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (118)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=1621-1626&journal=J.%C2%A0Cell+Sci&author=MacGrath+S.M.&author=Koleske+A.J.&title=Cortactin+in+cell+migration+and+cancer+at+a+glance">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Although small sample size and short follow-up time precluded prognostic association in our cohort, patients harboring <em>FAT1</em> truncation or 11q13.3 amplification had worse survival than those without either alteration in TCGA HPV<sup>neg</sup> HNSCCs (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>E). These results suggest that both <em>FAT1</em> truncation and 11q13.3 amplification drive poor prognosis, possibly through a common mechanism of modulating actin dynamics (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>F), which provides an explanation for the mutual exclusivity between these frequent genomic aberrations in HPV<sup>neg</sup> HNSCC.</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic delineation of the Rb pathway</span></h3><div class="section-paragraph">The most common genetic aberrations affected the cyclin D-CDK4/6-Rb pathway, including <em>CDKN2A</em> deletions (57%) and mutations (27%) and <em>CCND1</em> amplifications (32%) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A), all of which had comparable frequencies in the TCGA HPV<sup>neg</sup> HNSCCs. <em>CDKN2A</em> was hypermethylated in 13 tumors (12%), and hypermethylation was mutually exclusive with other loss of function alterations (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S4</a>A, and S4B and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/68ecf9b0-b448-4bd6-a8c3-a16895560d68/mmc5" class="scroll-into-link">Table S4</a>). Mutations in <em>CDK6</em> (5%) and <em>RB1</em> (2%) were also observed (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A).<div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/38608314-4878-405b-9d2e-3b5593af359f/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/c2931c66-f824-4858-9e08-e247defd66a3/gr4_lrg.jpg" data-large-image="/cms/attachment/38608314-4878-405b-9d2e-3b5593af359f/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Proteogenomic delineation of the cyclin D-CDK4/6-Rb pathway</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Genetic and epigenetic aberrations in pathway genes. Impact of <em>CDKN2A</em> aberrations for two major isoforms, <em>p16INK4a</em> (<em>p16</em>) and <em>p14ARF</em> (<em>p14</em>), on respective transcript mRNA levels are shown separately.</div><div class="section-paragraph">(B and C) <em>Cis</em> effects of <em>CCND1</em> amplification on RNA (B) and protein abundance (C). <sup>∗∗∗</sup>p &lt; 1 × 10<sup>−4</sup>, Wilcoxon rank-sum test, n = 108.</div><div class="section-paragraph">(D) Comparison of Rb phosphorylation levels (average of all CDK4/6 target sites) among three tumor groups. <sup>∗</sup>p &lt; 0.05. <sup>∗∗</sup>p &lt; 0.001, Wilcoxon rank-sum test.</div><div class="section-paragraph">(E) Heatmap comparing Rb phosphorylation, E2F activity, and the mean of cell cycle-regulated genes (MGPS), with genomic aberrations annotated. <sup>∗∗∗</sup>p &lt; 1 × 10<sup>−4</sup>, Pearson's correlation with Rb phosphorylation.</div><div class="section-paragraph">(F) Comparison of phospho-Rb-S807/811 in non-responsive and responsive HPV<sup>neg</sup> HNSCC PDX models with abemaciclib, Student's t test.</div><div class="section-paragraph">(G and H) Associations between MS-based Rb abundance and CDK6 essentiality scores derived from shRNA: DEMETER2 (G) or CRISPR (CERES)-based (H) genetic perturbations, respectively, in seven HPV<sup>neg</sup> HNSCC cell lines. R, Pearson's correlation coefficient. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2410" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S4</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2415" href="/cms/10.1016/j.ccell.2020.12.007/attachment/68ecf9b0-b448-4bd6-a8c3-a16895560d68/mmc5" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/38608314-4878-405b-9d2e-3b5593af359f/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/c2931c66-f824-4858-9e08-e247defd66a3/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Homozygous deletion of <em>CDKN2A</em> led to loss of mRNA expression of both major isoforms, p16INK4a (p16) and p14ARF (p14), but other aberrations, such as promoter hypermethylation, primarily affected p16 (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S4</a>B). Five <em>CDKN2A</em> mutations associated with loss of heterozygosity (LOH) altered p16 but not p14, and an additional six mutations resulted in truncation of p16 but only missense or in-frame indel changes in p14 (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S4</a>A). Altogether, 68 tumors (63%) had genetic or epigenetic events predicted to disrupt p16 expression (tumors with homozygous deletion, p16 hypermethylation, or p16 truncation LOH in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A), which may explain widespread missing proteomic measurements for p16. <em>CCND1</em> amplification was associated with increased levels of <em>CCND1</em> RNA and protein in general, but not in all tumors, as shown by overlap between the distributions of the amplified and WT groups (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>B and 4C). To assess the impact of <em>CDKN2A</em> aberrations and <em>CCND1</em> amplifications on CDK4/6-Rb signaling, we defined three groups: samples that were WT for pathway genes, including those with <em>CDKN2A</em> heterozygous deletion (n = 13), samples with p16 aberration affecting RNA expression but no <em>CCND1</em> amplification (n = 36), and samples with both p16 aberration and <em>CCND1</em> amplification (n = 26). Mean phosphorylation levels of the CDK4/6 target sites on Rb protein (Rb phosphosite score) were significantly higher in the second and third groups than in the first group (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>D). However, many of the samples from the second and third groups had Rb phosphosite scores that were well within the range of the first group. The observation that <em>CDKN2A</em> and <em>CCND1</em> aberrations did not always result in increased CCND1 protein and CDK4/6 activity was also seen in data from TCGA HPV<sup>neg</sup> HNSCCs (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S4</a>C–S4E).</div><div class="section-paragraph">Rb phosphosite scores were significantly correlated with both E2F activity scores and multi-gene proliferation scores (MGPS) inferred from the RNA data (Pearson's correlation = 0.50 and 0.47, p = 4 × 10<sup>−8</sup> and 4 × 10<sup>−7</sup>, respectively, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>E). However, several samples with low Rb phosphosite scores had high E2F activity and MGPS, indicating cell cycle activation through other mechanisms besides CDK4/6-mediated phosphorylation. For example, the two samples with the lowest Rb phosphosite scores had high E2F activity and MGPS. These samples harbored <em>RB1</em> mutations, potentially bypassing the need for CDK4/6 to phosphorylate Rb. While nearly all samples with high cell cycle activity had <em>CDKN2A</em> or <em>CCND1</em> aberrations, a number of samples with these aberrations had low cell cycle activity. These observations suggest that Rb status is an effective and necessary indicator of CDK4/6-dependent cell cycle activity, which cannot be accurately predicted using genomic or transcriptomic markers.</div><div class="section-paragraph">To test the clinical relevance of Rb for CDK4/6 targeting, we analyzed data from HPV<sup>neg</sup> HNSCC patient-derived xenograft (PDX) models treated with abemaciclib (<div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d52364298e2300" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Karamboulas et al., 2018</a><div data-db-target-of="bib60-d52364298e2300" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karamboulas C. </li><li class="loa__item">Bruce J.P. </li><li class="loa__item">Hope A.J. </li><li class="loa__item">Meens J. </li><li class="loa__item">Huang S.H. </li><li class="loa__item">Erdmann N. </li><li class="loa__item">Hyatt E. </li><li class="loa__item">Pereira K. </li><li class="loa__item">Goldstein D.P. </li><li class="loa__item">Weinreb I. </li><li>et al.</li></ul><div class="reference-citations__detials">Patient-derived xenografts for prognostication and personalized treatment for head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1318-1331.e1314</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30380421&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85055137026&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1318-1331.e1314&journal=Cell+Rep.&author=Karamboulas+C.&author=Bruce+J.P.&author=Hope+A.J.&author=Meens+J.&author=Huang+S.H.&author=Erdmann+N.&author=Hyatt+E.&author=Pereira+K.&author=Goldstein+D.P.&author=Weinreb+I.&title=Patient-derived+xenografts+for+prognostication+and+personalized+treatment+for+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), a CDK4/6 inhibitor in phase II clinical trials for HNSCC. While <em>CCND1</em> and/or <em>CDKN2A</em> status did not separate response to abemaciclib, treatment-responsive PDXs had elevated phospho-Rb-S807/811 signal (p = 0.05, Student's t test) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>F). Moreover, CDK6 dependency was examined in HPV<sup>neg</sup> HNSCC cell lines from two independent genetic perturbation screens with associated molecular profiles from the Cancer Cell Line Encyclopedia (CCLE) (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d52364298e2314" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Behan et al., 2019</a><div data-db-target-of="bib10-d52364298e2314" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Behan F.M. </li><li class="loa__item">Iorio F. </li><li class="loa__item">Picco G. </li><li class="loa__item">Gonçalves E. </li><li class="loa__item">Beaver C.M. </li><li class="loa__item">Migliardi G. </li><li class="loa__item">Santos R. </li><li class="loa__item">Rao Y. </li><li class="loa__item">Sassi F. </li><li class="loa__item">Pinnelli M. </li><li>et al.</li></ul><div class="reference-citations__detials">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">568</span></span>: <span class="ref__seriesPages">511-516</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41586-019-1103-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30971826&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85064233743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (198)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&journal=Nature&author=Behan+F.M.&author=Iorio+F.&author=Picco+G.&author=Gon%C3%A7alves+E.&author=Beaver+C.M.&author=Migliardi+G.&author=Santos+R.&author=Rao+Y.&author=Sassi+F.&author=Pinnelli+M.&title=Prioritization+of+cancer+therapeutic+targets+using+CRISPR-Cas9+screens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d52364298e2317" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">McFarland et al., 2018</a><div data-db-target-of="bib93-d52364298e2317" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McFarland J.M. </li><li class="loa__item">Ho Z.V. </li><li class="loa__item">Kugener G. </li><li class="loa__item">Dempster J.M. </li><li class="loa__item">Montgomery P.G. </li><li class="loa__item">Bryan J.G. </li><li class="loa__item">Krill-Burger J.M. </li><li class="loa__item">Green T.M. </li><li class="loa__item">Vazquez F. </li><li class="loa__item">Boehm J.S. </li><li>et al.</li></ul><div class="reference-citations__detials">Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">4610</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-018-06916-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30389920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85056025716&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=4610&journal=Nat.+Commun.&author=McFarland+J.M.&author=Ho+Z.V.&author=Kugener+G.&author=Dempster+J.M.&author=Montgomery+P.G.&author=Bryan+J.G.&author=Krill-Burger+J.M.&author=Green+T.M.&author=Vazquez+F.&author=Boehm+J.S.&title=Improved+estimation+of+cancer+dependencies+from+large-scale+RNAi+screens+using+model-based+normalization+and+data+integration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d52364298e2320" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Nusinow et al., 2020</a><div data-db-target-of="bib102-d52364298e2320" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nusinow D.P. </li><li class="loa__item">Szpyt J. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Rose C.M. </li><li class="loa__item">McDonald E.R. </li><li class="loa__item">Kalocsay M. </li><li class="loa__item">Jané-Valbuena J. </li><li class="loa__item">Gelfand E. </li><li class="loa__item">Schweppe D.K. </li><li class="loa__item">Jedrychowski M. </li><li>et al.</li></ul><div class="reference-citations__detials">Quantitative proteomics of the cancer cell line encyclopedia.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">387-402.e316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31978347&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85077916308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=387-402.e316&journal=Cell&author=Nusinow+D.P.&author=Szpyt+J.&author=Ghandi+M.&author=Rose+C.M.&author=McDonald+E.R.&author=Kalocsay+M.&author=Jan%C3%A9-Valbuena+J.&author=Gelfand+E.&author=Schweppe+D.K.&author=Jedrychowski+M.&title=Quantitative+proteomics+of+the+cancer+cell+line+encyclopedia">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Cell lines with higher levels of Rb protein, which was highly correlated with the Rb phosphosite score in our dataset (Spearman rho = 0.89, p &lt; 0.0001), were more sensitive to genetic depletion of <em>CDK6</em> (p &lt; 0.05, Pearson's correlation, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>G–4H). Taken together, these results support the hypothesis that phospho- or total Rb may serve as markers for CDK4/6 inhibitors in HPV<sup>neg</sup> HNSCC.</div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Two modes of EGFR activation</span></h3><div class="section-paragraph">We analyzed our data to gain insights into the poor response of HNSCC patients to EGFR inhibition. <em>EGFR</em> mutations were identified in only three tumors, and none were hotspot mutations. Moreover, no samples harbored the EGFR VIII fusion variant. However, 49 samples showed <em>EGFR</em> amplification (CN log2 ratio &gt;0.1), and six had high amplification (CN log2 ratio &gt;1) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A). <em>EGFR</em> CN was significantly associated with mRNA and protein abundance, overall phosphorylation level of EGFR, and phosphorylation levels of activation sites Y1110, Y1172, and Y1197 (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and 5B). Thus, <em>EGFR</em> amplification is associated with EGFR activation.<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/bbd3f0e0-e07f-4d41-9dc1-7507e8366639/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/ef73a97c-6b08-4a9d-ba1d-88e37e0f722c/gr5_lrg.jpg" data-large-image="/cms/attachment/bbd3f0e0-e07f-4d41-9dc1-7507e8366639/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Proteogenomic characterization of EGFR ligand-dependent and -independent pathways</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A and B) (A) Heatmap comparing EGFR multi-omics profiles and the inferred PROGENy EGFR pathway activity and (B) their Pearson's correlation coefficients. <sup>∗</sup>p &lt; 0.01.</div><div class="section-paragraph">(C) EGFR ligand mRNA abundance in tumors and NATs. <sup>∗∗</sup>p &lt; 0.001, Student's t test.</div><div class="section-paragraph">(D) Pearson's correlation between EGFR pathway activity and mRNA abundance of individual ligands.</div><div class="section-paragraph">(E) For genes in the ligand-dependent pathways downstream of EGFR, the Pearson's correlations between each omics feature and average ligand abundance (correlation to ligands) or EGFR abundance (correlation to receptor) are shown. <sup>∗</sup>p &lt; 0.01. Reported functional sites are colored green.</div><div class="section-paragraph">(F) Relationship between PROGENy EGFR pathway activity (color gradients) and average ligand abundance or EGFR phosphorylation level. The six triangles represent samples with the high <em>EGFR</em> amplification.</div><div class="section-paragraph">(G) Abundance comparisons between amplified samples and controls for 11 tyrosine phosphosites and cognate mRNA and proteins. Green box indicates known regulation by EGFR, black and gray indicate known and predicted EGFR substrates, respectively. Numbers on the side indicate fold changes. <sup>∗</sup>p &lt; 0.01, Student's t test.</div><div class="section-paragraph">(H) GO biological processes enriched with proteins with <em>EGFR</em> CN-associated phosphorylation (Fisher's exact test).</div><div class="section-paragraph">(I) Diagram depicting two modes of EGFR activation with implications for EGFR mAb therapies.</div><div class="section-paragraph">(J) Comparisons between non-responsive (NR) and responsive (R) HPV<sup>neg</sup> HNSCC PDX models to cetuximab treatment for average ligand (ligand_ave), individual ligands, and EGFR mRNA abundance. <sup>∗</sup>p &lt; 0.05; <sup>∗∗</sup>p &lt; 0.01 Student's t test.</div><div class="section-paragraph">(K) Spearman's correlations between mRNA abundance and PFS using data from a clinical trial testing panitumumab in HNSCC patients. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2420" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S5</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2425" href="/cms/10.1016/j.ccell.2020.12.007/attachment/6ed86b8c-c3f6-450d-a012-1157f4bc1cd1/mmc6" target="_blank">Table S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/bbd3f0e0-e07f-4d41-9dc1-7507e8366639/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/ef73a97c-6b08-4a9d-ba1d-88e37e0f722c/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We inferred EGFR pathway activity based on mRNA expression data using PROGENy (<div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d52364298e2355" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">Schubert et al., 2018</a><div data-db-target-of="bib116-d52364298e2355" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Schubert M. </li><li class="loa__item">Klinger B. </li><li class="loa__item">Klünemann M. </li><li class="loa__item">Sieber A. </li><li class="loa__item">Uhlitz F. </li><li class="loa__item">Sauer S. </li><li class="loa__item">Garnett M.J. </li><li class="loa__item">Blüthgen N. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Perturbation-response genes reveal signaling footprints in cancer gene expression.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Unexpectedly, the inferred pathway activity showed no or weak correlations with <em>EGFR</em> alterations (Pearson's correlation = 0.03–0.23, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>A and 5B). In contrast, except for the two with very low mRNA abundance in tumors (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C), all EGFR ligands (<div class="dropBlock reference-citations"><a href="#bib153" id="back-bib153" data-db-target-for="bib153-d52364298e2366" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib153">Singh et al., 2016</a><div data-db-target-of="bib153-d52364298e2366" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Singh B. </li><li class="loa__item">Carpenter G. </li><li class="loa__item">Coffey R.J. </li></ul><div class="reference-citations__detials">EGF receptor ligands: recent advances.<div class="popup-ref"><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.12688%2Ff1000research.9025.1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&journal=F1000Res&author=Singh+B.&author=Carpenter+G.&author=Coffey+R.J.&title=EGF+receptor+ligands%3A+recent+advances">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) showed strong correlations with inferred pathway activity (Pearson's correlation = 0.51–0.72, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>D). These observations were fully recapitulated in data from TCGA HPV<sup>neg</sup> HNSCCs (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S5</a>A–S5D). Moreover, phosphoproteomics quantified several phosphosites on proteins involved in the PI3K/Akt/mTOR and the RAF/MEK/ERK pathways, two primary downstream pathways of EGFR (<div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d52364298e2379" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Wee and Wang, 2017</a><div data-db-target-of="bib144-d52364298e2379" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wee P. </li><li class="loa__item">Wang Z. </li></ul><div class="reference-citations__detials">Epidermal growth factor receptor cell proliferation signaling pathways.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">52</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3390%2Fcancers9050052&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28513565&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85019993185&amp;cf=" onclick="newWindow(this.href);return false">Scopus (377)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=52&journal=Cancers+%28Basel%29&author=Wee+P.&author=Wang+Z.&title=Epidermal+growth+factor+receptor+cell+proliferation+signaling+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These phosphosites, including several key functional sites such as PIK3C2A S259 (<div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d52364298e2382" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Margaria et al., 2019</a><div data-db-target-of="bib91-d52364298e2382" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Margaria J.P. </li><li class="loa__item">Ratto E. </li><li class="loa__item">Gozzelino L. </li><li class="loa__item">Li H. </li><li class="loa__item">Hirsch E. </li></ul><div class="reference-citations__detials">Class II PI3Ks at the intersection between signal transduction and membrane trafficking.<div class="popup-ref"><span class="ref__series"><i>Biomolecules.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">104</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3390%2Fbiom9030104&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85063273855&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=104&journal=Biomolecules&author=Margaria+J.P.&author=Ratto+E.&author=Gozzelino+L.&author=Li+H.&author=Hirsch+E.&title=Class+II+PI3Ks+at+the+intersection+between+signal+transduction+and+membrane+trafficking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), RPTOR S859 (<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d52364298e2385" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Wang et al., 2009</a><div data-db-target-of="bib140-d52364298e2385" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang L. </li><li class="loa__item">Lawrence J.C. </li><li class="loa__item">Sturgill T.W. </li><li class="loa__item">Harris T.E. </li></ul><div class="reference-citations__detials">Mammalian target of rapamycin complex 1 (mTORC1) activity is associated with phosphorylation of raptor by mTOR.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">284</span></span>: <span class="ref__seriesPages">14693-14697</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19346248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-67649344456&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=14693-14697&journal=J.%C2%A0Biol.+Chem.&author=Wang+L.&author=Lawrence+J.C.&author=Sturgill+T.W.&author=Harris+T.E.&title=Mammalian+target+of+rapamycin+complex+1+%28mTORC1%29+activity+is+associated+with+phosphorylation+of+raptor+by+mTOR">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and EIF4B S422 (<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d52364298e2389" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Shahbazian et al., 2006</a><div data-db-target-of="bib118-d52364298e2389" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shahbazian D. </li><li class="loa__item">Roux P.P. </li><li class="loa__item">Mieulet V. </li><li class="loa__item">Cohen M.S. </li><li class="loa__item">Raught B. </li><li class="loa__item">Taunton J. </li><li class="loa__item">Hershey J.W.B. </li><li class="loa__item">Blenis J. </li><li class="loa__item">Pende M. </li><li class="loa__item">Sonenberg N. </li></ul><div class="reference-citations__detials">The mTOR/PI3K and MAPK pathways converge on eIF4B to control its phosphorylation and activity.<div class="popup-ref"><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2781-2791</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsj.emboj.7601166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16763566&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33745570504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (356)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=2781-2791&journal=EMBO+J.&author=Shahbazian+D.&author=Roux+P.P.&author=Mieulet+V.&author=Cohen+M.S.&author=Raught+B.&author=Taunton+J.&author=Hershey+J.W.B.&author=Blenis+J.&author=Pende+M.&author=Sonenberg+N.&title=The+mTOR%2FPI3K+and+MAPK+pathways+converge+on+eIF4B+to+control+its+phosphorylation+and+activity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), showed strong correlations with EGFR ligands, independent of the mRNA and protein expression of the host genes (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E). Conversely, none were significantly correlated with EGFR protein abundance. Thus, both transcriptomic and phosphoproteomic data suggest that the EGFR ligands, instead of the receptor, are the rate-limiting factors for EGFR pathway activity.</div><div class="section-paragraph">To identify signaling changes associated with <em>EGFR</em> amplification, which did not result in increased EGFR pathway activity but seemed to enhance EGFR phosphorylation in a ligand-independent manner (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>F), we compared phosphoproteomic profiles between the six samples with high <em>EGFR</em> amplification and 38 other samples with similar chromosomal instability (ChrIdx score &gt;3) (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S5</a>E). This analysis identified 297 phosphosites with significantly higher phosphorylation in the <em>EGFR</em> amplification group (p &lt; 0.01, Student's t test), and 212 phosphosites showed stronger changes than at the RNA or protein level, suggesting these are <em>bona fide</em> phosphorylation changes and not due to differential gene expression or cell type composition (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/6ed86b8c-c3f6-450d-a012-1157f4bc1cd1/mmc6" class="scroll-into-link">Table S5</a>). The 11 tyrosine sites with significantly increased phosphorylation (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>G) included five known or predicted EGFR substrates (EGFR Y1197, ANXA1 Y21, PTPN11 Y546, PTPN11 Y62, and ABI1 Y213). Additionally, PTPN11 Y546, PTPN11 Y62, and CSTB Y97 are reported to be regulated by EGFR in the PhosphoSitePlus database. Proteins harboring the 212 sites were enriched in cytoskeleton organization, actin filament, and intermediate filament junction-related pathways (adjusted p &lt; 0.01, Fisher's exact test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>H), suggesting a role for EGFR in modulating intercellular junctions and cell motility, as previously reported (<div class="dropBlock reference-citations"><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d52364298e2421" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Klymkowsky and Parr, 1995</a><div data-db-target-of="bib68-d52364298e2421" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Klymkowsky M.W. </li><li class="loa__item">Parr B. </li></ul><div class="reference-citations__detials">The body language of cells: the intimate connection between cell adhesion and behavior.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">83</span></span>: <span class="ref__seriesPages">5-8</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2F0092-8674%2895%2990226-0&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2F0092-8674%2895%2990226-0&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=7553873&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1995&pages=5-8&journal=Cell&author=Klymkowsky+M.W.&author=Parr+B.&title=The+body+language+of+cells%3A+the+intimate+connection+between+cell+adhesion+and+behavior">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d52364298e2424" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Stallaert et al., 2018</a><div data-db-target-of="bib124-d52364298e2424" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stallaert W. </li><li class="loa__item">Brüggemann Y. </li><li class="loa__item">Sabet O. </li><li class="loa__item">Baak L. </li><li class="loa__item">Gattiglio M. </li><li class="loa__item">Bastiaens P.I.H. </li></ul><div class="reference-citations__detials">Contact inhibitory Eph signaling suppresses EGF-promoted cell migration by decoupling EGFR activity from vesicular recycling.<div class="popup-ref"><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">eaat0114</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscisignal.aat0114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30065026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85050958000&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=eaat0114&journal=Sci.+Signal.&author=Stallaert+W.&author=Br%C3%BCggemann+Y.&author=Sabet+O.&author=Baak+L.&author=Gattiglio+M.&author=Bastiaens+P.I.H.&title=Contact+inhibitory+Eph+signaling+suppresses+EGF-promoted+cell+migration+by+decoupling+EGFR+activity+from+vesicular+recycling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Since our data suggest that <em>EGFR</em> amplification activates EGFR in a ligand-independent manner, and EGFR mAbs function primarily by binding to the EGFR extracellular domain to prevent ligand-induced pathway activity (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d52364298e2431" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Harding and Burtness, 2005</a><div data-db-target-of="bib54-d52364298e2431" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Harding J. </li><li class="loa__item">Burtness B. </li></ul><div class="reference-citations__detials">Cetuximab: an epidermal growth factor receptor chimeric human-murine monoclonal antibody.<div class="popup-ref"><span class="ref__series"><i>Drugs Today.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">107-127</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1358%2Fdot.2005.41.2.882662&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15821783&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-16244400108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (209)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=107-127&journal=Drugs+Today&author=Harding+J.&author=Burtness+B.&title=Cetuximab%3A+an+epidermal+growth+factor+receptor+chimeric+human-murine+monoclonal+antibody">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib98" id="back-bib98" data-db-target-for="bib98-d52364298e2434" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Messersmith and Hidalgo, 2007</a><div data-db-target-of="bib98-d52364298e2434" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Messersmith W.A. </li><li class="loa__item">Hidalgo M. </li></ul><div class="reference-citations__detials">Panitumumab, a monoclonal anti epidermal growth factor receptor antibody in colorectal cancer: another one or the one?.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">4664-4666</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-07-0065&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17699842&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34548056116&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=4664-4666&journal=Clin.+Cancer+Res.&author=Messersmith+W.A.&author=Hidalgo+M.&title=Panitumumab%2C+a+monoclonal+anti+epidermal+growth+factor+receptor+antibody+in+colorectal+cancer%3A+another+one+or+the+one%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), EGFR ligand abundance, rather than <em>EGFR</em> amplification or overexpression, should be used to select HNSCC patients for treatment with anti-EGFR mAbs (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>I). Indeed, utilizing data from an HNSCC PDX study with treatment response to an EGFR mAb, cetuximab (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d52364298e2443" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Klinghammer et al., 2017</a><div data-db-target-of="bib67-d52364298e2443" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Klinghammer K. </li><li class="loa__item">Otto R. </li><li class="loa__item">Raguse J.-D. </li><li class="loa__item">Albers A.E. </li><li class="loa__item">Tinhofer I. </li><li class="loa__item">Fichtner I. </li><li class="loa__item">Leser U. </li><li class="loa__item">Keilholz U. </li><li class="loa__item">Hoffmann J. </li></ul><div class="reference-citations__detials">Basal subtype is predictive for response to cetuximab treatment in patient-derived xenografts of squamous cell head and neck cancer.<div class="popup-ref"><span class="ref__series"><i>Int. J. Cancer.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">141</span></span>: <span class="ref__seriesPages">1215-1221</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fijc.30808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28560858&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85021184298&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=1215-1221&journal=Int.+J.+Cancer&author=Klinghammer+K.&author=Otto+R.&author=Raguse+J.-D.&author=Albers+A.E.&author=Tinhofer+I.&author=Fichtner+I.&author=Leser+U.&author=Keilholz+U.&author=Hoffmann+J.&title=Basal+subtype+is+predictive+for+response+to+cetuximab+treatment+in+patient-derived+xenografts+of+squamous+cell+head+and+neck+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we found that EGFR ligands, not the receptor, showed significantly higher expression in responders than non-responders (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>J). Furthermore, in a clinical trial testing another EGFR mAb, panitumumab, in HNSCC patients (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d52364298e2449" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Siano et al., 2018</a><div data-db-target-of="bib120-d52364298e2449" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Siano M. </li><li class="loa__item">Espeli V. </li><li class="loa__item">Mach N. </li><li class="loa__item">Bossi P. </li><li class="loa__item">Licitra L. </li><li class="loa__item">Ghielmini M. </li><li class="loa__item">Frattini M. </li><li class="loa__item">Canevari S. </li><li class="loa__item">De Cecco L. </li></ul><div class="reference-citations__detials">Gene signatures and expression of miRNAs associated with efficacy of panitumumab in a head and neck cancer phase II trial.<div class="popup-ref"><span class="ref__series"><i>Oral Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">144-151</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.oraloncology.2018.05.013&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29909889&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85047610082&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2018&pages=144-151&journal=Oral+Oncol.&author=Siano+M.&author=Espeli+V.&author=Mach+N.&author=Bossi+P.&author=Licitra+L.&author=Ghielmini+M.&author=Frattini+M.&author=Canevari+S.&author=De+Cecco+L.&title=Gene+signatures+and+expression+of+miRNAs+associated+with+efficacy+of+panitumumab+in+a+head+and+neck+cancer+phase+II+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we found that EGFR ligand abundance, but not receptor abundance, significantly correlated with PFS (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>K).</div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immuno-proteogenomic analysis reveals immunosuppressive SCNAs</span></h3><div class="section-paragraph">To gain a deeper understanding of immune evasion and resistance to PD-1 inhibitors in HNSCC, we performed an immuno-proteogenomic analysis. Tumors showed a wide range of immune cell infiltration levels as consistently quantified by ESTIMATE immune score (<div class="dropBlock reference-citations"><a href="#bib149" id="back-bib149" data-db-target-for="bib149-d52364298e2460" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Yoshihara et al., 2013</a><div data-db-target-of="bib149-d52364298e2460" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1032)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), CD3 IHC staining, and CD3 proteomic data (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). Higher immune cell infiltration was not linked to any anatomic sites but was associated with lower clinical stage, less smoking, and better prognosis (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>B, <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S6</a>A, and S6B). <em>In silico</em> deconvolution using xCell (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d52364298e2476" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d52364298e2476" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) showed that both cytotoxic immune cells (e.g., CD8 T cells and M1 macrophages) and immunosuppressive cells (e.g., regulatory T cells [Treg cells] and M2 macrophages) were enriched in tumors with high levels of immune cell infiltration. In these immune-hot tumors, both cytotoxic immune enzymes and immunosuppressive proteins were overexpressed at the protein and/or mRNA levels (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A), with high correlations observed across immune inhibitory genes (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>C). These data may explain the moderate response rate to single-agent pembrolizumab treatment in programmed death-ligand 1 (PD-L1)-positive HNSCCs (<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d52364298e2485" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Seiwert et al., 2016</a><div data-db-target-of="bib117-d52364298e2485" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Seiwert T.Y. </li><li class="loa__item">Burtness B. </li><li class="loa__item">Mehra R. </li><li class="loa__item">Weiss J. </li><li class="loa__item">Berger R. </li><li class="loa__item">Eder J.P. </li><li class="loa__item">Heath K. </li><li class="loa__item">McClanahan T. </li><li class="loa__item">Lunceford J. </li><li class="loa__item">Gause C. </li><li>et al.</li></ul><div class="reference-citations__detials">Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">956-965</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27247226&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84969988424&amp;cf=" onclick="newWindow(this.href);return false">Scopus (740)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=956-965&journal=Lancet+Oncol.&author=Seiwert+T.Y.&author=Burtness+B.&author=Mehra+R.&author=Weiss+J.&author=Berger+R.&author=Eder+J.P.&author=Heath+K.&author=McClanahan+T.&author=Lunceford+J.&author=Gause+C.&title=Safety+and+clinical+activity+of+pembrolizumab+for+treatment+of+recurrent+or+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+%28KEYNOTE-012%29%3A+an+open-label%2C+multicentre%2C+phase+1b+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and suggest combinatorial checkpoint inhibition as a logical proposition to increase treatment efficacy.<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/045edebb-67e8-4801-8354-cd3b05ec0335/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/0074fca2-c59e-49ff-aeef-f286d78e99f8/gr6_lrg.jpg" data-large-image="/cms/attachment/045edebb-67e8-4801-8354-cd3b05ec0335/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Immuno-proteogenomic analysis reveals immunosuppressive SCNA drivers</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Pearson's correlations between ESTIMATE immune score and proteogenomic profiles of immune infiltration, cytotoxic factors, and immune inhibitors.</div><div class="section-paragraph">(B) Comparisons of the immune score across clinical attributes (<sup>∗</sup>p &lt; 0.01, Student's t test).</div><div class="section-paragraph">(C) Correlations among immune checkpoints and suppressors.</div><div class="section-paragraph">(D) CN, mRNA abundance, and protein abundance of three SCNA driver genes.</div><div class="section-paragraph">(E) Diagram showing the information flow from antigen processing and presenting machinery (APM) regulators to APM components. The top row for each gene shows the <em>cis</em> effect of CN on RNA and protein abundance, and the bottom row shows the correlation between immune score and each omics type.</div><div class="section-paragraph">(F) Pathways enriched for immune-associated genes whose expression was suppressed by SCNA (i.e., immunosuppressive SCNA drivers).</div><div class="section-paragraph">(G) Pathways enriched for immune-associated genes whose expression was not associated with SCNA (i.e., effectors of the immune-suppressive CN deletions).</div><div class="section-paragraph">(H) The distribution of immunosuppressive SCNAs across the genome. Selected immune genes are highlighted. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2430" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S6</a> and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2435" href="/cms/10.1016/j.ccell.2020.12.007/attachment/a2aa4e57-81c8-4a6d-b520-a2063ce725f3/mmc7" target="_blank">Table S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/045edebb-67e8-4801-8354-cd3b05ec0335/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/0074fca2-c59e-49ff-aeef-f286d78e99f8/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Next, we sought to identify tumor intrinsic determinants of low immune infiltration in immune-cold tumors. We observed negative correlations between immune cell infiltration and either tumor mutational burden or protein abundance of quantified C/T antigens (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S6</a>C). Moreover, proteomics-supported neoantigens (<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d52364298e2493" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Wen et al., 2020</a><div data-db-target-of="bib145-d52364298e2493" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/a2aa4e57-81c8-4a6d-b520-a2063ce725f3/mmc7" class="scroll-into-link">Table S6</a>) showed little correlation to immune cell infiltration (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S6</a>C). Thus, the low immune infiltration was not driven by a lack of tumor antigen sources. Instead, we found significantly reduced expression of multiple components and regulators of the antigen presentation machinery (APM) pathway at both mRNA and protein levels in immune-cold tumors (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S6</a>D and S6E). Few mutations in APM genes and their regulators were identified in our cohort (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S6</a>F), but frequent somatic CN deletions (&gt;25%) were found in APM regulators IFNGR2, JAK2, and IRF1 (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S6</a>G). CN levels of these genes correlated strongly with mRNA and protein levels when data were available, and these genes further showed strong correlation with immune infiltration at all molecular levels (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>D and 6E), suggesting a causal contribution of these gene deletions to APM deficiency and low immune infiltration (i.e., SCNA drivers). In comparison, APM components showed significant correlation with immune infiltration at the gene expression, but not CN level, indicating that the changes occurred <em>in trans</em> (i.e., SCNA effectors). Consistent with the model depicted in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E, gene regulatory network analysis using VIPER (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d52364298e2520" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Alvarez et al., 2016</a><div data-db-target-of="bib3-d52364298e2520" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alvarez M.J. </li><li class="loa__item">Shen Y. </li><li class="loa__item">Giorgi F.M. </li><li class="loa__item">Lachmann A. </li><li class="loa__item">Ding B.B. </li><li class="loa__item">Ye B.H. </li><li class="loa__item">Califano A. </li></ul><div class="reference-citations__detials">Functional characterization of somatic mutations in cancer using network-based inference of protein activity.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">838-847</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3593&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27322546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975246805&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=838-847&journal=Nat.+Genet.&author=Alvarez+M.J.&author=Shen+Y.&author=Giorgi+F.M.&author=Lachmann+A.&author=Ding+B.B.&author=Ye+B.H.&author=Califano+A.&title=Functional+characterization+of+somatic+mutations+in+cancer+using+network-based+inference+of+protein+activity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) identified STAT1 as the central transcription factor regulating immune activity.</div><div class="section-paragraph">Expanding the APM-focused analysis to a genome-wide search identified 294 putative SCNA drivers (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/a2aa4e57-81c8-4a6d-b520-a2063ce725f3/mmc7" class="scroll-into-link">Table S6</a>) and 2,058 putative SCNA effectors. SCNA drivers were enriched in cytokine/chemokine receptor, JAK-STAT, and TLR pathways, all of which regulate immunogenicity or immune response within tumor cells (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>F and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S6</a>H). By contrast, SCNA effectors were mostly involved in immune cytotoxicity, especially adaptive immune cell activation and function (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>G). SCNA drivers were distributed widely across the genome (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>H), including the most frequently deleted 3p region, which encodes chemokine/cytokine receptors and TLRs, and 9p24.1, which encodes JAK2, a key component of the JAK-STAT pathway. Genes deleted in 9p24.1 also include <em>CD274</em>, which encodes PD-L1, suggesting that PD-L1-mediated immune checkpoint is not needed in immune-cold HNSCCs. Our observations for both immune-cold and immune-hot tumors were supported by our reanalysis of transcriptomic data from TCGA HPV<sup>neg</sup> HNSCCs (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S6</a>I–S6L), albeit at a lower sensitivity for detecting the driver CN pathway signals, suggesting that integrating proteomic data helped prioritize driver CN events involved in immunogenicity.</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omics subtypes and targeted therapies</span></h3><div class="section-paragraph">By integrating CN, RNA, miRNA, protein, and phosphopeptide data, an unsupervised clustering analysis grouped tumors into three clusters. Clusters I, II, and III were significantly associated with previously established classical, basal, and mesenchymal RNA subtypes (<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d52364298e2554" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Walter et al., 2013</a><div data-db-target-of="bib137-d52364298e2554" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Walter V. </li><li class="loa__item">Yin X. </li><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Zhao N. </li><li class="loa__item">Du Y. </li><li class="loa__item">Ang M.K. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Salazar A.H. </li><li class="loa__item">Hoadley K.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes.<div class="popup-ref"><span class="ref__series"><i>PLoS One.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">e56823</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pone.0056823&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23451093&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874327940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e56823&journal=PLoS+One&author=Walter+V.&author=Yin+X.&author=Wilkerson+M.D.&author=Cabanski+C.R.&author=Zhao+N.&author=Du+Y.&author=Ang+M.K.&author=Hayward+M.C.&author=Salazar+A.H.&author=Hoadley+K.A.&title=Molecular+subtypes+in+head+and+neck+cancer+exhibit+distinct+patterns+of+chromosomal+gain+and+loss+of+canonical+cancer+genes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), respectively (p &lt; 0.05, Fisher's exact test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A and 7B ).<div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/f8b1347e-0e50-4866-852a-74ebc8830639/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/83464528-98cf-4b81-b841-90b7434245b3/gr7_lrg.jpg" data-large-image="/cms/attachment/f8b1347e-0e50-4866-852a-74ebc8830639/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Integrated multi-omics subtypes and subtype-specific targeted therapies</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Proteomic and phosphoproteomic profiles of the signature proteins and the enriched biological processes of the three integrated subtypes.</div><div class="section-paragraph">(B) Sample distribution across different clinical attributes.</div><div class="section-paragraph">(C) Comparisons of the three subtypes for four molecular phenotypes. <sup>∗</sup>p &lt; 0.01, <sup>∗∗</sup>p &lt; 0.001, Student's t test.</div><div class="section-paragraph">(D) mRNA and protein levels of protein-specific gene signatures related to epigenetic, basal, and translation initiation factors for different subtypes. Each feature was tested for its differential abundance between the given subtype and the other two subtypes. <sup>∗</sup>Adjusted p &lt; 0.01 for both comparisons, Student's t test.</div><div class="section-paragraph">(E) Heatmap visualizing proteogenomic measurements of the suggested biomarkers for targeted therapies and candidacy for treatment with CDK inhibitors (upper), EGFR mAb (middle), and immune checkpoint blockade (bottom).</div><div class="section-paragraph">(F) Comparisons of the proposed biomarkers between high-potential and low-potential tumors, and between each group of tumors and NATs. Numbers at the top denote fold changes. <sup>∗</sup>p &lt; 0.01, <sup>∗∗</sup>p &lt; 0.001. Student's t test.</div><div class="section-paragraph">(G) The proportions of high-potential candidates for each target therapy in the three subtypes. Numbers in parentheses represent the sample sizes for the involved groups.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2440" href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" target="_blank">Figure S7</a> and <a data-locator="mmc8" aria-expanded="false" aria-controls="crosref2445" href="/cms/10.1016/j.ccell.2020.12.007/attachment/253a90e3-f1d5-4228-baa2-d71e3f72e180/mmc8" target="_blank">Table S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/f8b1347e-0e50-4866-852a-74ebc8830639/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/83464528-98cf-4b81-b841-90b7434245b3/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(20)30655-3&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The classical, basal, and mesenchymal RNA subtypes have been characterized by overexpression of genes related to cell proliferation, epidermal development, and stromal infiltration, respectively (<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d52364298e2564" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Walter et al., 2013</a><div data-db-target-of="bib137-d52364298e2564" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Walter V. </li><li class="loa__item">Yin X. </li><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Zhao N. </li><li class="loa__item">Du Y. </li><li class="loa__item">Ang M.K. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Salazar A.H. </li><li class="loa__item">Hoadley K.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes.<div class="popup-ref"><span class="ref__series"><i>PLoS One.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">e56823</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pone.0056823&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23451093&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874327940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e56823&journal=PLoS+One&author=Walter+V.&author=Yin+X.&author=Wilkerson+M.D.&author=Cabanski+C.R.&author=Zhao+N.&author=Du+Y.&author=Ang+M.K.&author=Hayward+M.C.&author=Salazar+A.H.&author=Hoadley+K.A.&title=Molecular+subtypes+in+head+and+neck+cancer+exhibit+distinct+patterns+of+chromosomal+gain+and+loss+of+canonical+cancer+genes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Proteomic and phosphoproteomic data not only confirmed these features but also provided new insights (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>A and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S7</a>A–S7C and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/253a90e3-f1d5-4228-baa2-d71e3f72e180/mmc8" class="scroll-into-link">Table S7</a>). Cluster I was associated with the larynx, strong smoking (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B, p &lt; 0.01, Fisher's exact test), and high chromosome instability (CIN) (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>C and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S7</a>D). Proteomic data, but not RNA-seq data, showed increased levels of multiple epigenetic regulators in this cluster (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>D), supporting a suggested linkage between aberrant epigenetic activities and smoking and CIN in HNSCC (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d52364298e2589" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Ghantous et al., 2018</a><div data-db-target-of="bib51-d52364298e2589" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ghantous Y. </li><li class="loa__item">Schussel J.L. </li><li class="loa__item">Brait M. </li></ul><div class="reference-citations__detials">Tobacco and alcohol-induced epigenetic changes in oral carcinoma.<div class="popup-ref"><span class="ref__series"><i>Curr. Opin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">152-158</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1097%2FCCO.0000000000000444&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29538041&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045508925&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2018&pages=152-158&journal=Curr.+Opin.+Oncol.&author=Ghantous+Y.&author=Schussel+J.L.&author=Brait+M.&title=Tobacco+and+alcohol-induced+epigenetic+changes+in+oral+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d52364298e2592" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Papillon-Cavanagh et al., 2017</a><div data-db-target-of="bib104-d52364298e2592" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Papillon-Cavanagh S. </li><li class="loa__item">Lu C. </li><li class="loa__item">Gayden T. </li><li class="loa__item">Mikael L.G. </li><li class="loa__item">Bechet D. </li><li class="loa__item">Karamboulas C. </li><li class="loa__item">Ailles L. </li><li class="loa__item">Karamchandani J. </li><li class="loa__item">Marchione D.M. </li><li class="loa__item">Garcia B.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Impaired H3K36 methylation defines a subset of head and neck squamous cell carcinomas.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">180-185</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3757&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28067913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85008661446&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=180-185&journal=Nat.+Genet.&author=Papillon-Cavanagh+S.&author=Lu+C.&author=Gayden+T.&author=Mikael+L.G.&author=Bechet+D.&author=Karamboulas+C.&author=Ailles+L.&author=Karamchandani+J.&author=Marchione+D.M.&author=Garcia+B.A.&title=Impaired+H3K36+methylation+defines+a+subset+of+head+and+neck+squamous+cell+carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This cluster showed the worst prognosis (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figures S7</a>E and S7F), which is consistent with the observation for the classical RNA subtype of HNSCC (<div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d52364298e2599" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Keck et al., 2015</a><div data-db-target-of="bib62-d52364298e2599" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keck M.K. </li><li class="loa__item">Zuo Z. </li><li class="loa__item">Khattri A. </li><li class="loa__item">Stricker T.P. </li><li class="loa__item">Brown C.D. </li><li class="loa__item">Imanguli M. </li><li class="loa__item">Rieke D. </li><li class="loa__item">Endhardt K. </li><li class="loa__item">Fang P. </li><li class="loa__item">Brägelmann J. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative analysis of head and neck cancer identifies two biologically distinct HPV and three non-HPV subtypes.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">870-881</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-2481&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25492084&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923172078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (191)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=870-881&journal=Clin.+Cancer+Res.&author=Keck+M.K.&author=Zuo+Z.&author=Khattri+A.&author=Stricker+T.P.&author=Brown+C.D.&author=Imanguli+M.&author=Rieke+D.&author=Endhardt+K.&author=Fang+P.&author=Br%C3%A4gelmann+J.&title=Integrative+analysis+of+head+and+neck+cancer+identifies+two+biologically+distinct+HPV+and+three+non-HPV+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Cluster II showed protein-level elevation of several basal factors (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>D). Moreover, protein- and phosphorylation-specific elevation of eukaryotic translation initiation (EIF) complex members indicated higher translational activity in these tumors (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>D and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S7</a>G). Both clusters I and II were associated with higher stemness (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C), likely due to aberrant epigenetic activity and basal-like factor activation, respectively (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d52364298e2614" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Malta et al., 2018</a><div data-db-target-of="bib90-d52364298e2614" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kamińska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials">Machine learning identifies stemness features associated with oncogenic dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e315</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (245)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e315&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kami%C5%84ska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+learning+identifies+stemness+features+associated+with+oncogenic+dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Cluster III was enriched with tumors with weak smoking evidence (p &lt; 0.01, Fisher's exact test, <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B). It was also associated with higher immune scores and, to a lesser degree, higher stromal scores (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C). Consistent with our observation, a more recent HNSCC subtyping study annotated the mesenchymal RNA subtype as inflamed (<div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d52364298e2624" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Keck et al., 2015</a><div data-db-target-of="bib62-d52364298e2624" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keck M.K. </li><li class="loa__item">Zuo Z. </li><li class="loa__item">Khattri A. </li><li class="loa__item">Stricker T.P. </li><li class="loa__item">Brown C.D. </li><li class="loa__item">Imanguli M. </li><li class="loa__item">Rieke D. </li><li class="loa__item">Endhardt K. </li><li class="loa__item">Fang P. </li><li class="loa__item">Brägelmann J. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative analysis of head and neck cancer identifies two biologically distinct HPV and three non-HPV subtypes.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">870-881</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-2481&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25492084&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923172078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (191)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=870-881&journal=Clin.+Cancer+Res.&author=Keck+M.K.&author=Zuo+Z.&author=Khattri+A.&author=Stricker+T.P.&author=Brown+C.D.&author=Imanguli+M.&author=Rieke+D.&author=Endhardt+K.&author=Fang+P.&author=Br%C3%A4gelmann+J.&title=Integrative+analysis+of+head+and+neck+cancer+identifies+two+biologically+distinct+HPV+and+three+non-HPV+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Notably, the atypical RNA subtype, which was enriched with HPV<sup>pos</sup> samples in the original study, collapsed to clusters I and III (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B), which was supported by their associations with higher stemness cores and higher immune/stromal scores, respectively (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S7</a>H). Taken together, our multi-omics subtyping identified three subtypes of HPV<sup>neg</sup> HNSCCs, which we named CIN, Basal, and Immune, respectively.</div><div class="section-paragraph">To examine the utility of these subtypes in guiding treatment selection, we evaluated their associations with our proposed biomarkers for targeted therapies in HNSCC (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>E). The CIN subtype was associated with frequent genetic aberrations of <em>CCND1</em> and <em>CDKN2A</em> and high CDK4/6 activity as indicated by Rb hyperphosphorylation, suggesting potential response to CDK4/6 inhibitors. The Basal subtype was characterized by high EGFR ligand expression (e.g., AREG and TGFA) and high EGFR pathway activity, suggesting potential response to EGFR mAb. The Immune subtype showed high expression of multiple immune checkpoint proteins, and thus may benefit from checkpoint blockade. For each treatment option, the high-potential tumors showed significantly higher levels of the biomarkers than the low-potential tumors and the matched NATs (p &lt; 0.01, Student's t test), whereas the latter two showed no or a less significant difference (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>F). In total, 32% of the CIN tumors, 62% of the Basal tumors, and 83% of the Immune tumors had high potential for treatment with CDK inhibitors, EGFR mAb, and immunotherapy, respectively (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>G).</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0080" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">With several targeted therapies approved for the treatment of HNSCC and many more in development, the identification of accurate biomarkers to guide treatment selection is a major research priority (<div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d52364298e2660" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Santuray et al., 2018</a><div data-db-target-of="bib112-d52364298e2660" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Santuray R.T. </li><li class="loa__item">Johnson D.E. </li><li class="loa__item">Grandis J.R. </li></ul><div class="reference-citations__detials">New therapies in head and neck cancer.<div class="popup-ref"><span class="ref__series"><i>Trends Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">385-396</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29709262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045920059&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=385-396&journal=Trends+Cancer&author=Santuray+R.T.&author=Johnson+D.E.&author=Grandis+J.R.&title=New+therapies+in+head+and+neck+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Our study demonstrates the promise of proteogenomics in addressing this challenge. For EGFR-targeted therapy, it is well acknowledged that EGFR amplification or overexpression cannot be used to predict response to EGFR mAbs in HNSCC (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d52364298e2663" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Ang et al., 2014</a><div data-db-target-of="bib4-d52364298e2663" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ang K.K. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Rosenthal D.I. </li><li class="loa__item">Nguyen-Tan P.F. </li><li class="loa__item">Sherman E.J. </li><li class="loa__item">Weber R.S. </li><li class="loa__item">Galvin J.M. </li><li class="loa__item">Bonner J.A. </li><li class="loa__item">Harris J. </li><li class="loa__item">El-Naggar A.K. </li><li>et al.</li></ul><div class="reference-citations__detials">Randomized phase III trial of concurrent accelerated radiation plus cisplatin with or without cetuximab for stage III to IV head and neck carcinoma: RTOG 0522.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2940-2950</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2013.53.5633&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25154822&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84907200175&amp;cf=" onclick="newWindow(this.href);return false">Scopus (437)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=2940-2950&journal=J.%C2%A0Clin.+Oncol.&author=Ang+K.K.&author=Zhang+Q.&author=Rosenthal+D.I.&author=Nguyen-Tan+P.F.&author=Sherman+E.J.&author=Weber+R.S.&author=Galvin+J.M.&author=Bonner+J.A.&author=Harris+J.&author=El-Naggar+A.K.&title=Randomized+phase+III+trial+of+concurrent+accelerated+radiation+plus+cisplatin+with+or+without+cetuximab+for+stage+III+to+IV+head+and+neck+carcinoma%3A+RTOG+0522">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d52364298e2666" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Burtness et al., 2005</a><div data-db-target-of="bib19-d52364298e2666" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Burtness B. </li><li class="loa__item">Goldwasser M.A. </li><li class="loa__item">Flood W. </li><li class="loa__item">Mattar B. </li><li class="loa__item">Forastiere A.A. </li><li class="loa__item">Eastern Cooperative Oncology G. </li></ul><div class="reference-citations__detials">Phase III randomized trial of cisplatin plus placebo compared with cisplatin plus cetuximab in metastatic/recurrent head and neck cancer: an Eastern Cooperative Oncology Group study.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">8646-8654</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.02.4646&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16314626&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33144461661&amp;cf=" onclick="newWindow(this.href);return false">Scopus (704)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=8646-8654&journal=J.%C2%A0Clin.+Oncol.&author=Burtness+B.&author=Goldwasser+M.A.&author=Flood+W.&author=Mattar+B.&author=Forastiere+A.A.&author=Eastern+Cooperative+Oncology+G.&title=Phase+III+randomized+trial+of+cisplatin+plus+placebo+compared+with+cisplatin+plus+cetuximab+in+metastatic%2Frecurrent+head+and+neck+cancer%3A+an+Eastern+Cooperative+Oncology+Group+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d52364298e2669" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Crombet et al., 2004</a><div data-db-target-of="bib35-d52364298e2669" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Crombet T. </li><li class="loa__item">Osorio M. </li><li class="loa__item">Cruz T. </li><li class="loa__item">Roca C. </li><li class="loa__item">del Castillo R. </li><li class="loa__item">Mon R. </li><li class="loa__item">Iznaga-Escobar N. </li><li class="loa__item">Figueredo R. </li><li class="loa__item">Koropatnick J. </li><li class="loa__item">Renginfo E. </li><li>et al.</li></ul><div class="reference-citations__detials">Use of the humanized anti-epidermal growth factor receptor monoclonal antibody h-R3 in combination with radiotherapy in the treatment of locally advanced head and neck cancer patients.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">1646-1654</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2004.03.089&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15117987&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2442704412&amp;cf=" onclick="newWindow(this.href);return false">Scopus (252)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2004&pages=1646-1654&journal=J.%C2%A0Clin.+Oncol.&author=Crombet+T.&author=Osorio+M.&author=Cruz+T.&author=Roca+C.&author=del+Castillo+R.&author=Mon+R.&author=Iznaga-Escobar+N.&author=Figueredo+R.&author=Koropatnick+J.&author=Renginfo+E.&title=Use+of+the+humanized+anti-epidermal+growth+factor+receptor+monoclonal+antibody+h-R3+in+combination+with+radiotherapy+in+the+treatment+of+locally+advanced+head+and+neck+cancer+patients">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d52364298e2672" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Psyrri et al., 2014</a><div data-db-target-of="bib106-d52364298e2672" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Psyrri A. </li><li class="loa__item">Lee J.-W. </li><li class="loa__item">Pectasides E. </li><li class="loa__item">Vassilakopoulou M. </li><li class="loa__item">Kosmidis E.K. </li><li class="loa__item">Burtness B.A. </li><li class="loa__item">Rimm D.L. </li><li class="loa__item">Wanebo H.J. </li><li class="loa__item">Forastiere A.A. </li></ul><div class="reference-citations__detials">Prognostic biomarkers in phase II trial of cetuximab-containing induction and chemoradiation in resectable HNSCC: Eastern Cooperative Oncology Group E2303.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">3023-3032</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-0113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24700741&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84901811387&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=3023-3032&journal=Clin.+Cancer+Res.&author=Psyrri+A.&author=Lee+J.-W.&author=Pectasides+E.&author=Vassilakopoulou+M.&author=Kosmidis+E.K.&author=Burtness+B.A.&author=Rimm+D.L.&author=Wanebo+H.J.&author=Forastiere+A.A.&title=Prognostic+biomarkers+in+phase+II+trial+of+cetuximab-containing+induction+and+chemoradiation+in+resectable+HNSCC%3A+Eastern+Cooperative+Oncology+Group+E2303">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Our data suggest a new strategy of using EGFR ligand abundance to stratify patients for effective treatment with EGFR mAb. In addition, some tumors with EGFR ligand overexpression also harbor <em>CCND1</em> and <em>CDKN2A</em> aberrations, which may render them resistant to anti-EGFR mAb monotherapy. Tumors with high <em>EGFR</em> amplification do not necessarily have high levels of EGFR ligands and may not respond to EGFR mAbs. However, these tumors show strong EGFR phosphorylation and thus could respond to small-molecule EGFR tyrosine kinase inhibitors (TKIs). Consistent with this hypothesis, the combination of p16-negativity and EGFR amplification identified HNSCC patients that achieved a clinically meaningful benefit from afatinib, an EGFR TKI, in a phase III trial (<div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d52364298e2682" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Santuray et al., 2018</a><div data-db-target-of="bib112-d52364298e2682" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Santuray R.T. </li><li class="loa__item">Johnson D.E. </li><li class="loa__item">Grandis J.R. </li></ul><div class="reference-citations__detials">New therapies in head and neck cancer.<div class="popup-ref"><span class="ref__series"><i>Trends Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">385-396</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29709262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045920059&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=385-396&journal=Trends+Cancer&author=Santuray+R.T.&author=Johnson+D.E.&author=Grandis+J.R.&title=New+therapies+in+head+and+neck+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">For immunotherapy, immune-hot tumors concordantly overexpress multiple checkpoints and other immunosuppressive genes, which may partially explain the moderate response rate in PD-L1 positive HNSCCs to a single-agent pembrolizumab treatment (<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d52364298e2687" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Seiwert et al., 2016</a><div data-db-target-of="bib117-d52364298e2687" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Seiwert T.Y. </li><li class="loa__item">Burtness B. </li><li class="loa__item">Mehra R. </li><li class="loa__item">Weiss J. </li><li class="loa__item">Berger R. </li><li class="loa__item">Eder J.P. </li><li class="loa__item">Heath K. </li><li class="loa__item">McClanahan T. </li><li class="loa__item">Lunceford J. </li><li class="loa__item">Gause C. </li><li>et al.</li></ul><div class="reference-citations__detials">Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">956-965</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27247226&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84969988424&amp;cf=" onclick="newWindow(this.href);return false">Scopus (740)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=956-965&journal=Lancet+Oncol.&author=Seiwert+T.Y.&author=Burtness+B.&author=Mehra+R.&author=Weiss+J.&author=Berger+R.&author=Eder+J.P.&author=Heath+K.&author=McClanahan+T.&author=Lunceford+J.&author=Gause+C.&title=Safety+and+clinical+activity+of+pembrolizumab+for+treatment+of+recurrent+or+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+%28KEYNOTE-012%29%3A+an+open-label%2C+multicentre%2C+phase+1b+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Moreover, there was no clinical improvement from combining durvalumab (PD-L1 antibody) and tremelimumab (CTLA-4 antibody) in unselected patients with relapsed/metastatic HNSCC in a phase III trial (<div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d52364298e2690" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">Ferris et al., 2020</a><div data-db-target-of="bib41-d52364298e2690" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ferris R.L. </li><li class="loa__item">Haddad R. </li><li class="loa__item">Even C. </li><li class="loa__item">Tahara M. </li><li class="loa__item">Dvorkin M. </li><li class="loa__item">Ciuleanu T.E. </li><li class="loa__item">Clement P.M. </li><li class="loa__item">Mesia R. </li><li class="loa__item">Kutukova S. </li><li class="loa__item">Zholudeva L. </li><li>et al.</li></ul><div class="reference-citations__detials">Durvalumab with or without tremelimumab in patients with recurrent or metastatic head and neck squamous cell carcinoma: EAGLE, a randomized, open-label phase III study.<div class="popup-ref"><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">942-950</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32294530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85084237800&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=942-950&journal=Ann.+Oncol.&author=Ferris+R.L.&author=Haddad+R.&author=Even+C.&author=Tahara+M.&author=Dvorkin+M.&author=Ciuleanu+T.E.&author=Clement+P.M.&author=Mesia+R.&author=Kutukova+S.&author=Zholudeva+L.&title=Durvalumab+with+or+without+tremelimumab+in+patients+with+recurrent+or+metastatic+head+and+neck+squamous+cell+carcinoma%3A+EAGLE%2C+a+randomized%2C+open-label+phase+III+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Profiling of multiple immune checkpoint proteins may allow more precise personalization of combination immunotherapy regimens, potentially leading to improved outcomes through accurate patient selection.</div><div class="section-paragraph">Multiple clinical trials are evaluating CDK4/6 inhibitors in HNSCC, but there are no established biomarkers to guide patient selection (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d52364298e2695" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">Adkins et al., 2019</a><div data-db-target-of="bib1-d52364298e2695" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Adkins D. </li><li class="loa__item">Ley J. </li><li class="loa__item">Neupane P. </li><li class="loa__item">Worden F. </li><li class="loa__item">Sacco A.G. </li><li class="loa__item">Palka K. </li><li class="loa__item">Grilley-Olson J.E. </li><li class="loa__item">Maggiore R. </li><li class="loa__item">Salama N.N. </li><li class="loa__item">Trinkaus K. </li><li>et al.</li></ul><div class="reference-citations__detials">Palbociclib and cetuximab in platinum-resistant and in cetuximab-resistant human papillomavirus-unrelated head and neck cancer: a multicentre, multigroup, phase 2 trial.<div class="popup-ref"><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1295-1305</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31351869&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071533195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=1295-1305&journal=Lancet+Oncol.&author=Adkins+D.&author=Ley+J.&author=Neupane+P.&author=Worden+F.&author=Sacco+A.G.&author=Palka+K.&author=Grilley-Olson+J.E.&author=Maggiore+R.&author=Salama+N.N.&author=Trinkaus+K.&title=Palbociclib+and+cetuximab+in+platinum-resistant+and+in+cetuximab-resistant+human+papillomavirus-unrelated+head+and+neck+cancer%3A+a+multicentre%2C+multigroup%2C+phase+2+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Rb phosphorylation status could be considered as a biomarker together with <em>CCND1/CDKN2A</em> genomic aberrations for future clinical trials of CDK4/6 inhibitors, whereas these genomic markers alone, or transcriptomic markers of E2F activity, may not accurately reflect CDK4/6 activity.</div><div class="section-paragraph">We identified new targets for therapeutic development, such as KIT, FCER1G, PLAU, SERPINE1, TOP2A, several MMPs, and several cell cycle and DNA damage-related kinases. In addition, multiple C/T antigens are recurrently overexpressed in tumors compared with NATs, including IGF2BP3, MAGEB2, KIF2C, CEP55, and NUF2 (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a> and <a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">S6</a>), and proteomics-supported neoantigens were predicted for 20.4% of the patients (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/a2aa4e57-81c8-4a6d-b520-a2063ce725f3/mmc7" class="scroll-into-link">Table S6</a>). Both C/T antigens and neoantigens are promising immunotherapy targets.</div><div class="section-paragraph">We also generated new knowledge concerning HNSCC biology. Proteomics data prioritized CN drivers and highlighted an oncogenic role for RNA processing factors in HNSCC tumorigenesis. Widespread deletion of immune modulatory genes may account for loss of immunogenicity and low immune infiltration in HNSCC. <em>FAT1</em> was among the most frequently mutated genes in HNSCC. Previous studies have linked <em>FAT1</em> mutations to the WNT and HIPPO pathways (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d52364298e2717" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Ciriello et al., 2012</a><div data-db-target-of="bib30-d52364298e2717" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ciriello G. </li><li class="loa__item">Cerami E. </li><li class="loa__item">Sander C. </li><li class="loa__item">Schultz N. </li></ul><div class="reference-citations__detials">Mutual exclusivity analysis identifies oncogenic network modules.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">398-406</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.125567.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21908773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84856535081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (385)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=398-406&journal=Genome+Res.&author=Ciriello+G.&author=Cerami+E.&author=Sander+C.&author=Schultz+N.&title=Mutual+exclusivity+analysis+identifies+oncogenic+network+modules">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d52364298e2720" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Martin et al., 2018</a><div data-db-target-of="bib92-d52364298e2720" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Martin D. </li><li class="loa__item">Degese M.S. </li><li class="loa__item">Vitale-Cross L. </li><li class="loa__item">Iglesias-Bartolome R. </li><li class="loa__item">Valera J.L.C. </li><li class="loa__item">Wang Z. </li><li class="loa__item">Feng X. </li><li class="loa__item">Yeerna H. </li><li class="loa__item">Vadmal V. </li><li class="loa__item">Moroishi T. </li><li>et al.</li></ul><div class="reference-citations__detials">Assembly and activation of the Hippo signalome by FAT1 tumor suppressor.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">2372</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-018-04590-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29985391&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049751886&amp;cf=" onclick="newWindow(this.href);return false">Scopus (36)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2372&journal=Nat.+Commun.&author=Martin+D.&author=Degese+M.S.&author=Vitale-Cross+L.&author=Iglesias-Bartolome+R.&author=Valera+J.L.C.&author=Wang+Z.&author=Feng+X.&author=Yeerna+H.&author=Vadmal+V.&author=Moroishi+T.&title=Assembly+and+activation+of+the+Hippo+signalome+by+FAT1+tumor+suppressor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) or apoptosis (<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d52364298e2723" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Kranz and Boutros, 2014</a><div data-db-target-of="bib71-d52364298e2723" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kranz D. </li><li class="loa__item">Boutros M. </li></ul><div class="reference-citations__detials">A synthetic lethal screen identifies FAT1 as an antagonist of caspase-8 in extrinsic apoptosis.<div class="popup-ref"><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">181-197</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24442637&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=181-197&journal=EMBO+J.&author=Kranz+D.&author=Boutros+M.&title=A+synthetic+lethal+screen+identifies+FAT1+as+an+antagonist+of+caspase-8+in+extrinsic+apoptosis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), but none of these theories were supported by our data (<a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1" class="scroll-into-link">Figure S3</a>G). Instead, proteomic investigation of the mutually exclusive relationship between <em>FAT1</em> truncating mutations and 11q13.3 amplifications revealed their functional convergence on dysregulated actin dynamics, which may underlie poor prognosis of tumors with these genetic aberrations.</div><div class="section-paragraph">In summary, this study extends our biological understanding of HPV<sup>neg</sup> HNSCC and generates therapeutic hypotheses that may serve as the basis for future preclinical studies and clinical trials toward molecularly guided precision treatment of this aggressive cancer type. Meanwhile, we have made the primary and processed datasets available in publicly accessible data repositories and portals, which will allow full investigation of this extensively characterized cohort by both the HNSCC and broader scientific communities. We also expect wide application of the demonstrated proteogenomics framework to future studies of HNSCC and other cancer types.</div></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec5.1" data-locator="sec5.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key resources table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Chemicals, Peptides, and Recombinant Proteins</strong></td></tr><tr><td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</td><td>Alfa Aesar</td><td>Catalog: J63218</td></tr><tr><td>Acetonitrile, Optima LC/MS</td><td>Fisher Chemical</td><td>Catalog: A955-4</td></tr><tr><td>Ammonium Hydroxide solution</td><td>Sigma</td><td>Catalog: 338818</td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>CD3 antibody (polyclonal)</td><td>Agilent - Dako</td><td>Cat# A0452; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2335677" target="_blank">AB_2335677</a></td></tr><tr><td>DAB</td><td>Agilent - Dako</td><td>Catalog: K3468</td></tr><tr><td>Dithiothreitol</td><td>Thermo Fisher Scientific</td><td>Catalog: 20291</td></tr><tr><td>Envision+ System HRP labelled polymer, anti-Rabbit</td><td>Agilent - Dako</td><td>Cat# K4002; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2630375" target="_blank">AB_2630375</a></td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Formic acid</td><td>Fisher Chemical</td><td>Catalog: A117-50</td></tr><tr><td>Hydroxylamine solution</td><td>Aldrich</td><td>Catalog: 467804</td></tr><tr><td>Iodoacetamide</td><td>Thermo Fisher Scientific</td><td>Catalog: A3221</td></tr><tr><td>Iron (III) chloride</td><td>Sigma</td><td>Catalog: 451649</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Lysyl endopeptidase, aass spectrometry grade</td><td>Wako Chemicals</td><td>Catalog: 125-05061</td></tr><tr><td>Ni-NTA agarose beads</td><td>QIAGEN</td><td>Catalog: 30410</td></tr><tr><td>PUGNAc</td><td>Sigma</td><td>Catalog: A7229</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog: 93482</td></tr><tr><td>Phosphatase Inhibitor Cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase Inhibitor Cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Reversed-phase C18 SepPak</td><td>Waters</td><td>Catalog: WAT054925</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Catalog: V511X</td></tr><tr><td>Sodium chloride</td><td>Santa Cruz Biotechnology</td><td>Catalog: sc-295833</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>TMT11-131C label reagent</td><td>Thermo Fisher Scientific</td><td>Catalog: A34807</td></tr><tr><td>Tandem mass tags – 10plex</td><td>Thermo Fisher Scientific</td><td>Catalog: 90406</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>Catalog: 302031</td></tr><tr><td>Tris(hydroxymethyl)aminomethane</td><td>Invitrogen</td><td>Catalog: AM9855G</td></tr><tr><td>Urea</td><td>Sigma</td><td>Catalog: U0631</td></tr><tr><td class="rowsep">Water, Optima LC/MS</td><td class="rowsep">Fisher Chemical</td><td class="rowsep">W6-4</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical Commercial Assays</strong></td></tr><tr><td>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold</td><td>Illumina</td><td>Catalog: RS-122-2301</td></tr><tr><td>Infinium MethylationEPIC Kit</td><td>Illumina</td><td>Catalog: WG-317-1003</td></tr><tr><td>Nextera DNA Exosome Kit</td><td>Illumina</td><td>Catalog: 20020617</td></tr><tr><td>KAPA Hyper Prep Kit, PCR-free</td><td>Roche</td><td>Catalog: 07962371001</td></tr><tr><td class="rowsep">BCA Protein Assay Kit</td><td class="rowsep">ThermoFisher Scientific</td><td class="rowsep">Catalog: 23225</td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>Software</td><td>Source</td><td>Identifier (i.e. links)</td></tr><tr><td>Bowtie (v1.1.1)</td><td>(<div class="dropBlock reference-citations"><a href="#bib76" id="back-bib76" data-db-target-for="bib76-d52364298e3032" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Langmead et al., 2009</a><div data-db-target-of="bib76-d52364298e3032" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Langmead B. </li><li class="loa__item">Trapnell C. </li><li class="loa__item">Pop M. </li><li class="loa__item">Salzberg S.L. </li></ul><div class="reference-citations__detials">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">R25</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2009-10-3-r25&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19261174&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-62349130698&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12570)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&journal=Genome+Biol.&author=Langmead+B.&author=Trapnell+C.&author=Pop+M.&author=Salzberg+S.L.&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/bowtie-bio/files/bowtie/" target="_blank">https://sourceforge.net/projects/bowtie-bio/files/bowtie/</a></td></tr><tr><td>Bowtie2 (v2.3.3)</td><td>(<div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d52364298e3043" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Langmead and Salzberg, 2012</a><div data-db-target-of="bib75-d52364298e3043" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Langmead B. </li><li class="loa__item">Salzberg S.L. </li></ul><div class="reference-citations__detials">Fast gapped-read alignment with Bowtie 2.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">357-359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.1923&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22388286&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84859210032&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16937)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=357-359&journal=Nat.+Methods&author=Langmead+B.&author=Salzberg+S.L.&title=Fast+gapped-read+alignment+with+Bowtie+2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>BWA (v0.7.17-r1188)</td><td>(<div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d52364298e3054" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Li and Durbin, 2009</a><div data-db-target-of="bib79-d52364298e3054" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Fast and accurate short read alignment with Burrows-Wheeler transform.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1754-1760</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp324&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19451168&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-67649884743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19246)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1754-1760&journal=Bioinformatics&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td>cBioPortal</td><td>(<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d52364298e3065" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Cerami et al., 2012</a><div data-db-target-of="bib24-d52364298e3065" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cerami E. </li><li class="loa__item">Gao J. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Gross B.E. </li><li class="loa__item">Sumer S.O. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Jacobsen A. </li><li class="loa__item">Byrne C.J. </li><li class="loa__item">Heuer M.L. </li><li class="loa__item">Larsson E. </li><li>et al.</li></ul><div class="reference-citations__detials">The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">401-404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F2159-8290.CD-12-0095&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22588877&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84866002291&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6173)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=401-404&journal=Cancer+Discov.&author=Cerami+E.&author=Gao+J.&author=Dogrusoz+U.&author=Gross+B.E.&author=Sumer+S.O.&author=Aksoy+B.A.&author=Jacobsen+A.&author=Byrne+C.J.&author=Heuer+M.L.&author=Larsson+E.&title=The+cBio+cancer+genomics+portal%3A+an+open+platform+for+exploring+multidimensional+cancer+genomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d52364298e3068" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Gao et al., 2013</a><div data-db-target-of="bib45-d52364298e3068" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao J. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Dresdner G. </li><li class="loa__item">Gross B. </li><li class="loa__item">Sumer S.O. </li><li class="loa__item">Sun Y. </li><li class="loa__item">Jacobsen A. </li><li class="loa__item">Sinha R. </li><li class="loa__item">Larsson E. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal.<div class="popup-ref"><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">pl1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscisignal.2004088&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23550210&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84875740314&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5610)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=pl1&journal=Sci.+Signal.&author=Gao+J.&author=Aksoy+B.A.&author=Dogrusoz+U.&author=Dresdner+G.&author=Gross+B.&author=Sumer+S.O.&author=Sun+Y.&author=Jacobsen+A.&author=Sinha+R.&author=Larsson+E.&title=Integrative+analysis+of+complex+cancer+genomics+and+clinical+profiles+using+the+cBioPortal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.cbioportal.org" target="_blank">https://www.cbioportal.org</a></td></tr><tr><td>CIRI (v2.0.6)</td><td>(<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d52364298e3079" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Gao et al., 2015</a><div data-db-target-of="bib47-d52364298e3079" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhao F. </li></ul><div class="reference-citations__detials">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-014-0571-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25583365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84937578024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (401)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=4&journal=Genome+Biol.&author=Gao+Y.&author=Wang+J.&author=Zhao+F.&title=CIRI%3A+an+efficient+and+unbiased+algorithm+for+de+novo+circular+RNA+identification">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/ciri/" target="_blank">https://sourceforge.net/projects/ciri/</a></td></tr><tr><td>CNVEX</td><td>Marcin Cieslik Lab</td><td><a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a></td></tr><tr><td>Customprodbj</td><td>(<div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d52364298e3099" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Wang and Zhang, 2013</a><div data-db-target-of="bib143-d52364298e3099" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">3235-3237</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtt543&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24058055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84894110748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (87)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=3235-3237&journal=Bioinformatics&author=Wang+X.&author=Zhang+B.&title=customProDB%3A+an+R+package+to+generate+customized+protein+databases+from+RNA-Seq+data+for+proteomics+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a></td></tr><tr><td>EricScript v0.5.5</td><td>(<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d52364298e3110" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benelli et al., 2012</a><div data-db-target-of="bib13-d52364298e3110" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benelli M. </li><li class="loa__item">Pescucci C. </li><li class="loa__item">Marseglia G. </li><li class="loa__item">Severgnini M. </li><li class="loa__item">Torricelli F. </li><li class="loa__item">Magi A. </li></ul><div class="reference-citations__detials">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sites.google.com/site/bioericscript/" target="_blank">https://sites.google.com/site/bioericscript/</a></td></tr><tr><td>ESTIMATE</td><td>(<div class="dropBlock reference-citations"><a href="#bib149" id="back-bib149" data-db-target-for="bib149-d52364298e3121" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Yoshihara et al., 2013</a><div data-db-target-of="bib149-d52364298e3121" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1032)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioinformatics.mdanderson.org/public-software/estimate/" target="_blank">https://bioinformatics.mdanderson.org/public-software/estimate/</a></td></tr><tr><td>germlinewrapper v1.1</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/germlinewrapper" target="_blank">https://github.com/ding-lab/germlinewrapper</a></td></tr><tr><td>GISTIC2</td><td>(<div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d52364298e3140" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">Mermel et al., 2011</a><div data-db-target-of="bib95-d52364298e3140" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1119)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.genepattern.org/modules/docs/GISTIC_2.0" target="_blank">https://www.genepattern.org/modules/docs/GISTIC_2.0</a></td></tr><tr><td>GSVA</td><td>(<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d52364298e3151" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Hänzelmann et al., 2013</a><div data-db-target-of="bib53-d52364298e3151" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1108)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/GSVA.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/GSVA.html</a></td></tr><tr><td>lmerTest</td><td>(<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d52364298e3162" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Kuznetsova et al., 2017</a><div data-db-target-of="bib73-d52364298e3162" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuznetsova A. </li><li class="loa__item">Brockhoff P.B. </li><li class="loa__item">Christensen R.H.B. </li></ul><div class="reference-citations__detials">lmerTest package: tests in linear mixed effects models.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Softw.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">1-26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.18637%2Fjss.v082.i13&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2017&pages=1-26&journal=J.%C2%A0Stat.+Softw.&author=Kuznetsova+A.&author=Brockhoff+P.B.&author=Christensen+R.H.B.&title=lmerTest+package%3A+tests+in+linear+mixed+effects+models">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/lmerTest/index.html" target="_blank">https://cran.r-project.org/web/packages/lmerTest/index.html</a></td></tr><tr><td>INTEGRATE v0.2.6</td><td>(<div class="dropBlock reference-citations"><a href="#bib151" id="back-bib151" data-db-target-for="bib151-d52364298e3173" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib151">Zhang et al., 2016</a><div data-db-target-of="bib151-d52364298e3173" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang J. </li><li class="loa__item">White N.M. </li><li class="loa__item">Schmidt H.K. </li><li class="loa__item">Fulton R.S. </li><li class="loa__item">Tomlinson C. </li><li class="loa__item">Warren W.C. </li><li class="loa__item">Wilson R.K. </li><li class="loa__item">Maher C.A. </li></ul><div class="reference-citations__detials">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/integrate-fusion/" target="_blank">https://sourceforge.net/projects/integrate-fusion/</a></td></tr><tr><td>iProFUN</td><td>(<div class="dropBlock reference-citations"><a href="#bib123" id="back-bib123" data-db-target-for="bib123-d52364298e3184" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Song et al., 2019</a><div data-db-target-of="bib123-d52364298e3184" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/iProFun" target="_blank">https://github.com/WangLab-MSSM/iProFun</a></td></tr><tr><td>LinkedOmics</td><td>(<div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d52364298e3195" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Vasaikar et al., 2018</a><div data-db-target-of="bib134-d52364298e3195" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (310)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://linkedomics.org" target="_blank">http://linkedomics.org</a></td></tr><tr><td>Manta v1.6.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d52364298e3207" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Chen et al., 2016</a><div data-db-target-of="bib26-d52364298e3207" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Schulz-Trieglaff O. </li><li class="loa__item">Shaw R. </li><li class="loa__item">Barnes B. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Cox A.J. </li><li class="loa__item">Kruglyak S. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (366)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/Illumina/manta" target="_blank">https://github.com/Illumina/manta</a></td></tr><tr><td>MoonlightR</td><td>(<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d52364298e3218" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Colaprico et al., 2020</a><div data-db-target-of="bib33-d52364298e3218" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/MoonlightR/" target="_blank">http://bioconductor.org/packages/MoonlightR/</a></td></tr><tr><td>MSFragger-20190628</td><td>(<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d52364298e3229" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Kong et al., 2017</a><div data-db-target-of="bib70-d52364298e3229" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://msfragger.nesvilab.org/" target="_blank">http://msfragger.nesvilab.org/</a></td></tr><tr><td>MS-GF+</td><td>(<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d52364298e3240" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Kim and Pevzner, 2014</a><div data-db-target-of="bib65-d52364298e3240" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (467)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/MSGFPlus/msgfplus/" target="_blank">https://github.com/MSGFPlus/msgfplus/</a></td></tr><tr><td>MuTect v1.1.7</td><td>(<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d52364298e3251" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Cibulskis et al., 2013</a><div data-db-target-of="bib29-d52364298e3251" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2136)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/mutect" target="_blank">https://github.com/broadinstitute/mutect</a></td></tr><tr><td>NeoFlow</td><td>(<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d52364298e3262" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Wen et al., 2020</a><div data-db-target-of="bib145-d52364298e3262" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a></td></tr><tr><td>NetworKIN</td><td>(<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d52364298e3273" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Linding et al., 2008</a><div data-db-target-of="bib87-d52364298e3273" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linding R. </li><li class="loa__item">Jensen L.J. </li><li class="loa__item">Pasculescu A. </li><li class="loa__item">Olhovsky M. </li><li class="loa__item">Colwill K. </li><li class="loa__item">Bork P. </li><li class="loa__item">Yaffe M.B. </li><li class="loa__item">Pawson T. </li></ul><div class="reference-citations__detials">NetworKIN: a resource for exploring cellular phosphorylation networks.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">D695-D699</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkm902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17981841&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-38549109487&amp;cf=" onclick="newWindow(this.href);return false">Scopus (223)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D695-D699&journal=Nucleic+Acids+Res.&author=Linding+R.&author=Jensen+L.J.&author=Pasculescu+A.&author=Olhovsky+M.&author=Colwill+K.&author=Bork+P.&author=Yaffe+M.B.&author=Pawson+T.&title=NetworKIN%3A+a+resource+for+exploring+cellular+phosphorylation+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://networkin.info" target="_blank">https://networkin.info</a></td></tr><tr><td>OmicsEV</td><td>Bing Zhang lab</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/OmicsEV" target="_blank">https://github.com/bzhanglab/OmicsEV</a></td></tr><tr><td>OmicsOne</td><td>Hui Zhang Lab</td><td><a class="inter-ref-link" href="https://github.com/huizhanglab-jhu/OmicsOne" target="_blank">https://github.com/huizhanglab-jhu/OmicsOne</a></td></tr><tr><td>PepQuery</td><td>(<div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d52364298e3300" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Wen et al., 2019</a><div data-db-target-of="bib146-d52364298e3300" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://pepquery.org/" target="_blank">http://pepquery.org/</a></td></tr><tr><td>PeptideProphet</td><td>(<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d52364298e3311" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Keller et al., 2002</a><div data-db-target-of="bib63-d52364298e3311" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keller A. </li><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.<div class="popup-ref"><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3577)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://tools.proteomecenter.org/wiki/index.php?title=Main_Page" target="_blank">http://tools.proteomecenter.org/wiki/index.php?title=Main_Page</a></td></tr><tr><td>PDV</td><td>(<div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d52364298e3323" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">Li et al., 2019</a><div data-db-target-of="bib81-d52364298e3323" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/wenbostar/PDV" target="_blank">https://github.com/wenbostar/PDV</a></td></tr><tr><td>Philosopher-v1.6.0</td><td>Alexey Nesvizhskii lab</td><td><a class="inter-ref-link" href="https://philosopher.nesvilab.org/" target="_blank">https://philosopher.nesvilab.org/</a></td></tr><tr><td>Pindel v0.2.5</td><td>(<div class="dropBlock reference-citations"><a href="#bib148" id="back-bib148" data-db-target-for="bib148-d52364298e3342" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib148">Ye et al., 2009</a><div data-db-target-of="bib148-d52364298e3342" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1151)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/genome/pindel" target="_blank">https://github.com/genome/pindel</a></td></tr><tr><td>ProteinProphet</td><td>(<div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d52364298e3353" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">Nesvizhskii et al., 2003</a><div data-db-target-of="bib101-d52364298e3353" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Keller A. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">A statistical model for identifying proteins by tandem mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3267)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://tools.proteomecenter.org/wiki/index.php?title=Main_Page" target="_blank">http://tools.proteomecenter.org/wiki/index.php?title=Main_Page</a></td></tr><tr><td>PTMProphet</td><td>(<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d52364298e3364" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Shteynberg et al., 2019</a><div data-db-target-of="bib119-d52364298e3364" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shteynberg D.D. </li><li class="loa__item">Deutsch E.W. </li><li class="loa__item">Campbell D.S. </li><li class="loa__item">Hoopmann M.R. </li><li class="loa__item">Kusebauch U. </li><li class="loa__item">Lee D. </li><li class="loa__item">Mendoza L. </li><li class="loa__item">Midha M.K. </li><li class="loa__item">Sun Z. </li><li class="loa__item">Whetton A.D. </li><li class="loa__item">Moritz R.L. </li></ul><div class="reference-citations__detials">PTMProphet: fast and accurate mass modification localization for the trans-proteomic pipeline.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">4262-4272</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Facs.jproteome.9b00205&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31290668&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85070371700&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=4262-4272&journal=J.%C2%A0Proteome+Res.&author=Shteynberg+D.D.&author=Deutsch+E.W.&author=Campbell+D.S.&author=Hoopmann+M.R.&author=Kusebauch+U.&author=Lee+D.&author=Mendoza+L.&author=Midha+M.K.&author=Sun+Z.&author=Whetton+A.D.&author=Moritz+R.L.&title=PTMProphet%3A+fast+and+accurate+mass+modification+localization+for+the+trans-proteomic+pipeline">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://tools.proteomecenter.org/wiki/index.php?title=Main_Page" target="_blank">http://tools.proteomecenter.org/wiki/index.php?title=Main_Page</a></td></tr><tr><td>QuickMIRSeq</td><td>(<div class="dropBlock reference-citations"><a href="#bib152" id="back-bib152" data-db-target-for="bib152-d52364298e3375" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib152">Zhao et al., 2017</a><div data-db-target-of="bib152-d52364298e3375" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhao S. </li><li class="loa__item">Gordon W. </li><li class="loa__item">Du S. </li><li class="loa__item">Zhang C. </li><li class="loa__item">He W. </li><li class="loa__item">Xi L. </li><li class="loa__item">Mathur S. </li><li class="loa__item">Agostino M. </li><li class="loa__item">Paradis T. </li><li class="loa__item">von Schack D. </li><li>et al.</li></ul><div class="reference-citations__detials">QuickMIRSeq: a pipeline for quick and accurate quantification of both known miRNAs and isomiRs by jointly processing multiple samples from microRNA sequencing.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">180</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs12859-017-1601-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28320324&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85015748083&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=180&journal=BMC+Bioinformatics&author=Zhao+S.&author=Gordon+W.&author=Du+S.&author=Zhang+C.&author=He+W.&author=Xi+L.&author=Mathur+S.&author=Agostino+M.&author=Paradis+T.&author=von+Schack+D.&title=QuickMIRSeq%3A+a+pipeline+for+quick+and+accurate+quantification+of+both+known+miRNAs+and+isomiRs+by+jointly+processing+multiple+samples+from+microRNA+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/quickmirseq/" target="_blank">https://sourceforge.net/projects/quickmirseq/</a></td></tr><tr><td>RSEM (v1.3.1)</td><td>(<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d52364298e3386" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Li and Dewey, 2011</a><div data-db-target-of="bib78-d52364298e3386" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li B. </li><li class="loa__item">Dewey C.N. </li></ul><div class="reference-citations__detials">RSEM: accurate transcript quantification from RNA-seq data with or without a reference genome.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6871)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://deweylab.github.io/RSEM/" target="_blank">https://deweylab.github.io/RSEM/</a></td></tr><tr><td>Samtools (V1.1.0)</td><td>(<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d52364298e3397" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Li et al., 2009</a><div data-db-target-of="bib80-d52364298e3397" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Handsaker B. </li><li class="loa__item">Wysoker A. </li><li class="loa__item">Fennell T. </li><li class="loa__item">Ruan J. </li><li class="loa__item">Homer N. </li><li class="loa__item">Marth G. </li><li class="loa__item">Abecasis G. </li><li class="loa__item">Durbin R. </li><li class="loa__item">Genome Project Data Processing S. </li></ul><div class="reference-citations__detials">The sequence alignment/map format and SAMtools.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21754)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinformatics&author=Li+H.&author=Handsaker+B.&author=Wysoker+A.&author=Fennell+T.&author=Ruan+J.&author=Homer+N.&author=Marth+G.&author=Abecasis+G.&author=Durbin+R.&author=Genome+Project+Data+Processing+S.&title=The+sequence+alignment%2Fmap+format+and+SAMtools">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/samtools/samtools" target="_blank">https://github.com/samtools/samtools</a></td></tr><tr><td>SignatureAnalyzer</td><td>(<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d52364298e3408" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Kim et al., 2016</a><div data-db-target-of="bib64-d52364298e3408" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Polak P. </li><li class="loa__item">Braunstein L.Z. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Rosenberg J.E. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">D'Andrea A. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (154)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%27Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/msp" target="_blank">https://software.broadinstitute.org/cancer/cga/msp</a></td></tr><tr><td>somaticwrapper v1.5</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/somaticwrapper" target="_blank">https://github.com/ding-lab/somaticwrapper</a></td></tr><tr><td>STAR-Fusion v1.5.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d52364298e3427" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Haas et al., 2019</a><div data-db-target-of="bib52-d52364298e3427" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haas B.J. </li><li class="loa__item">Dobin A. </li><li class="loa__item">Li B. </li><li class="loa__item">Stransky N. </li><li class="loa__item">Pochet N. </li><li class="loa__item">Regev A. </li></ul><div class="reference-citations__detials">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (43)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/STAR-Fusion/STAR-Fusion" target="_blank">https://github.com/STAR-Fusion/STAR-Fusion</a></td></tr><tr><td>Strelka v2.9.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d52364298e3439" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Kim et al., 2018</a><div data-db-target-of="bib66-d52364298e3439" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (170)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&author=Saunders+C.T.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/Illumina/strelka" target="_blank">https://github.com/Illumina/strelka</a></td></tr><tr><td>Sumer</td><td>(<div class="dropBlock reference-citations"><a href="#bib114" id="back-bib114" data-db-target-for="bib114-d52364298e3450" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Savage et al., 2019</a><div data-db-target-of="bib114-d52364298e3450" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Savage S.R. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Liao Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Graph algorithms for condensing and consolidating gene set analysis results.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S141-S152</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31142576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071350100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S141-S152&journal=Mol.+Cell.+Proteomics&author=Savage+S.R.&author=Shi+Z.&author=Liao+Y.&author=Zhang+B.&title=Graph+algorithms+for+condensing+and+consolidating+gene+set+analysis+results">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/sumer" target="_blank">https://github.com/bzhanglab/sumer</a></td></tr><tr><td>TCGAbiolinks</td><td>(<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d52364298e3461" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Colaprico et al., 2016</a><div data-db-target-of="bib34-d52364298e3461" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (578)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/TCGAbiolinks/" target="_blank">http://bioconductor.org/packages/TCGAbiolinks/</a></td></tr><tr><td>TMT-Integrator-v1.0.9</td><td>Alexey Nesvizhskii lab</td><td><a class="inter-ref-link" href="http://tmt-integrator.nesvilab.org/" target="_blank">http://tmt-integrator.nesvilab.org/</a></td></tr><tr><td>VarScan v2.3.8</td><td>(<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d52364298e3480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Koboldt et al., 2012</a><div data-db-target-of="bib69-d52364298e3480" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2185)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://dkoboldt.github.io/varscan/" target="_blank">https://dkoboldt.github.io/varscan/</a></td></tr><tr><td>VIPER</td><td>(<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d52364298e3491" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Alvarez et al., 2016</a><div data-db-target-of="bib3-d52364298e3491" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alvarez M.J. </li><li class="loa__item">Shen Y. </li><li class="loa__item">Giorgi F.M. </li><li class="loa__item">Lachmann A. </li><li class="loa__item">Ding B.B. </li><li class="loa__item">Ye B.H. </li><li class="loa__item">Califano A. </li></ul><div class="reference-citations__detials">Functional characterization of somatic mutations in cancer using network-based inference of protein activity.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">838-847</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3593&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27322546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975246805&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=838-847&journal=Nat.+Genet.&author=Alvarez+M.J.&author=Shen+Y.&author=Giorgi+F.M.&author=Lachmann+A.&author=Ding+B.B.&author=Ye+B.H.&author=Califano+A.&title=Functional+characterization+of+somatic+mutations+in+cancer+using+network-based+inference+of+protein+activity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://califano.c2b2.columbia.edu/viper" target="_blank">http://califano.c2b2.columbia.edu/viper</a></td></tr><tr><td>VirusScan</td><td>(<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d52364298e3502" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Cao et al., 2016</a><div data-db-target-of="bib23-d52364298e3502" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cao S. </li><li class="loa__item">Wendl M.C. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Wylie K. </li><li class="loa__item">Ye K. </li><li class="loa__item">Jayasinghe R. </li><li class="loa__item">Xie M. </li><li class="loa__item">Wu S. </li><li class="loa__item">Niu B. </li><li class="loa__item">Grubb R. </li><li>et al.</li></ul><div class="reference-citations__detials">Divergent viral presentation among human tumors and adjacent normal tissues.<div class="popup-ref"><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">28294</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsrep28294&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27339696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975700575&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=28294&journal=Sci.+Rep.&author=Cao+S.&author=Wendl+M.C.&author=Wyczalkowski+M.A.&author=Wylie+K.&author=Ye+K.&author=Jayasinghe+R.&author=Xie+M.&author=Wu+S.&author=Niu+B.&author=Grubb+R.&title=Divergent+viral+presentation+among+human+tumors+and+adjacent+normal+tissues">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/VirusScan/tree/simplified" target="_blank">https://github.com/ding-lab/VirusScan/tree/simplified</a></td></tr><tr><td>WebgestaltR</td><td>(<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d52364298e3513" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Liao et al., 2019</a><div data-db-target-of="bib83-d52364298e3513" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://www.webgestalt.org/" target="_blank">http://www.webgestalt.org/</a></td></tr><tr><td class="rowsep">xCell</td><td class="rowsep">(<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d52364298e3524" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d52364298e3524" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td class="rowsep"><a class="inter-ref-link" href="http://xcell.ucsf.edu/" target="_blank">http://xcell.ucsf.edu/</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited Data</strong></td></tr><tr><td>cBioPortal</td><td>(<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d52364298e3538" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Cerami et al., 2012</a><div data-db-target-of="bib24-d52364298e3538" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cerami E. </li><li class="loa__item">Gao J. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Gross B.E. </li><li class="loa__item">Sumer S.O. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Jacobsen A. </li><li class="loa__item">Byrne C.J. </li><li class="loa__item">Heuer M.L. </li><li class="loa__item">Larsson E. </li><li>et al.</li></ul><div class="reference-citations__detials">The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">401-404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F2159-8290.CD-12-0095&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22588877&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84866002291&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6173)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=401-404&journal=Cancer+Discov.&author=Cerami+E.&author=Gao+J.&author=Dogrusoz+U.&author=Gross+B.E.&author=Sumer+S.O.&author=Aksoy+B.A.&author=Jacobsen+A.&author=Byrne+C.J.&author=Heuer+M.L.&author=Larsson+E.&title=The+cBio+cancer+genomics+portal%3A+an+open+platform+for+exploring+multidimensional+cancer+genomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.cbioportal.org/" target="_blank">https://www.cbioportal.org/</a></td></tr><tr><td>CTDatabase</td><td>(<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d52364298e3550" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Almeida et al., 2009</a><div data-db-target-of="bib2-d52364298e3550" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.-T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (213)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://www.cta.lncc.br" target="_blank">http://www.cta.lncc.br</a></td></tr><tr><td>DoRothEA</td><td>(<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d52364298e3561" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Garcia-Alonso et al., 2019</a><div data-db-target-of="bib48-d52364298e3561" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Garcia-Alonso L. </li><li class="loa__item">Holland C.H. </li><li class="loa__item">Ibrahim M.M. </li><li class="loa__item">Turei D. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Benchmark and integration of resources for the estimation of human transcription factor activities.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">1363-1375</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.240663.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31340985&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071059943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1363-1375&journal=Genome+Res.&author=Garcia-Alonso+L.&author=Holland+C.H.&author=Ibrahim+M.M.&author=Turei+D.&author=Saez-Rodriguez+J.&title=Benchmark+and+integration+of+resources+for+the+estimation+of+human+transcription+factor+activities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/saezlab/DoRothEA" target="_blank">https://github.com/saezlab/DoRothEA</a></td></tr><tr><td>Human Protein Atlas</td><td>(<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d52364298e3572" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Uhlén et al., 2015</a><div data-db-target-of="bib132-d52364298e3572" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Uhlén M. </li><li class="loa__item">Fagerberg L. </li><li class="loa__item">Hallström B.M. </li><li class="loa__item">Lindskog C. </li><li class="loa__item">Oksvold P. </li><li class="loa__item">Mardinoglu A. </li><li class="loa__item">Sivertsson Å. </li><li class="loa__item">Kampf C. </li><li class="loa__item">Sjöstedt E. </li><li class="loa__item">Asplund A. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteomics. Tissue-based map of the human proteome.<div class="popup-ref"><span class="ref__series"><i>Science (New York, NY).</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">347</span></span>: <span class="ref__seriesPages">1260419</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscience.1260419&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84920269464&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3889)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1260419&journal=Science+%28New+York%2C+NY%29&author=Uhl%C3%A9n+M.&author=Fagerberg+L.&author=Hallstr%C3%B6m+B.M.&author=Lindskog+C.&author=Oksvold+P.&author=Mardinoglu+A.&author=Sivertsson+%C3%85.&author=Kampf+C.&author=Sj%C3%B6stedt+E.&author=Asplund+A.&title=Proteomics.+Tissue-based+map+of+the+human+proteome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.proteinatlas.org" target="_blank">https://www.proteinatlas.org</a></td></tr><tr><td>MSigDBv7.0 Hallmark gene sets</td><td>(<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d52364298e3583" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Liberzon et al., 2015</a><div data-db-target-of="bib84-d52364298e3583" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1573)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.gsea-msigdb.org" target="_blank">https://www.gsea-msigdb.org</a></td></tr><tr><td>PhosphoSitePlus</td><td>(<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d52364298e3594" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Hornbeck et al., 2015</a><div data-db-target-of="bib57-d52364298e3594" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1086)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.phosphosite.org" target="_blank">https://www.phosphosite.org</a></td></tr><tr><td>CPTAC HNSCC proteomics data</td><td>this study</td><td><a class="inter-ref-link" href="https://proteomics.cancer.gov/data-portal" target="_blank">https://proteomics.cancer.gov/data-portal</a>; <a class="inter-ref-link" href="https://pdc.cancer.gov" target="_blank">https://pdc.cancer.gov</a></td></tr><tr><td>CPTAC HNSCC genomic and transcriptomic data</td><td>this study</td><td><a class="inter-ref-link" href="https://portal.gdc.cancer.gov/" target="_blank">https://portal.gdc.cancer.gov/</a></td></tr><tr><td>CPTAC HNSCC processed data matrices</td><td>this study</td><td><a class="inter-ref-link" href="http://linkedomics.org/data_download/CPTAC-HNSCC" target="_blank">http://linkedomics.org/data_download/CPTAC-HNSCC</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S1535-6108%2820%2930655-3">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec5.2" data-locator="sec5.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Resource availability</span></h3><section id="sec5.2.1" data-locator="sec5.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead contact</span></h4><div class="section-paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Bing Zhang (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#9ffdf6f1f8b1e5f7fef1f8dffdfcf2b1fafbea" target="_blank"><span class="__cf_email__" data-cfemail="6c0e05020b4216040d020b2c0e0f0142090819">[email&#160;protected]</span></a>).</div></section><section id="sec5.2.2" data-locator="sec5.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Materials availability</span></h4><div class="section-paragraph">This study did not generate new unique reagents.</div></section><section id="sec5.2.3" data-locator="sec5.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and code availability</span></h4><div class="section-paragraph">Raw proteomics data files are hosted by the CPTAC Data Portal and can be accessed at: <a class="inter-ref-link" href="https://proteomics.cancer.gov/data-portal" target="_blank">https://proteomics.cancer.gov/data-portal</a> and can also be accessed at the Proteomic Data Commons: <a class="inter-ref-link" href="https://pdc.cancer.gov" target="_blank">https://pdc.cancer.gov</a>. Genomic and transcriptomic data files can be accessed via the Genomic Data Commons (GDC) Data Portal: <a class="inter-ref-link" href="https://portal.gdc.cancer.gov" target="_blank">https://portal.gdc.cancer.gov</a>. Processed data utilized for this publication can be accessed via LinkedOmics: <a class="inter-ref-link" href="http://www.linkedomics.org" target="_blank">http://www.linkedomics.org</a>.</div><div class="section-paragraph">Several customized coding software packages were generated as part of this study and have been referenced in the corresponding <a href="#sec5" class="scroll-into-link">STAR Methods</a> section and listed with links to the coding script in the <a href="#sec5.1" class="scroll-into-link">Key Resources Table</a>: software codes generated by the Cieslik laboratory for genomic analyses (CNVEX), by the Nesvizhskii laboratory for proteomic data processing (Philosopher and TMT-Integrator), and by the Zhang lab for data processing and neoantigen detection (NeoFlow and PepQuery).</div></section></section><section id="sec5.3" data-locator="sec5.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental model and subject details</span></h3><section id="sec5.3.1" data-locator="sec5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimens and clinical data</span></h4><div class="section-paragraph">Tumor and germline blood samples from 110 qualified cases were collected from 6 tissue source sites in strict accordance to the CPTAC-3 protocol. All patients provided written informed consent. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the CPTAC guidelines. Normal adjacent tissues were collected from 83 cases. This study contained predominantly males (87%) and the cases were collected from 7 different countries. Histopathologically-defined squamous cell carcinomas were considered for analysis, with an age range of 23-81. A complete set of clinical data were obtained from the tissue source sites and reviewed for correctness and completeness.</div></section><section id="sec5.3.2" data-locator="sec5.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample processing</span></h4><div class="section-paragraph">The CPTAC Biospecimen Core Resource (BCR) at the Pathology and Biorepository Core of the Van Andel Research Institute in Grand Rapids, Michigan manufactured and distributed biospecimen kits to the Tissue Source Sites (TSS) located in the US, Europe, and Asia. Each kit contains a set of pre-manufactured labels unique for each specimen respective to TSS location, disease, and sample type that is used to track the specimens through the BCR to the CPTAC proteomic and genomic characterization centers.</div><div class="section-paragraph">Tissue specimens averaging 200 mg were snap-frozen by the TSS within a 30-minute cold ischemic time (CIT) (CIT average = 13 minutes) and an adjacent segment was formalin-fixed paraffin-embedded (FFPE) and H&amp;E stained by the TSS for quality assessment to meet the CPTAC HNSCC requirements. Routinely, several tissue segments for each case were collected. Tissues were flash frozen in liquid nitrogen (LN<sub>2</sub>) then transferred to a LN<sub>2</sub> freezer for storage until approval for shipment to the BCR.</div><div class="section-paragraph">Specimens were shipped using a cryoport that maintained an average temperature of under -140°C to the BCR with a time and temperature tracker to monitor the shipment. Receipt of specimens at the BCR included a physical inspection and review of the time and temperature tracker data for specimen integrity, followed by barcode entry into a biospecimen tracking database. Specimens were again placed in LN<sub>2</sub> storage until further processing. Acceptable HNSCC tumor tissue segments were determined by TSS pathologists based on the percent viable tumor nuclei (&gt; 80%), total cellularity (&gt; 50%), and necrosis (&lt; 20%). Segments received at the BCR were verified by BCR and Leidos Biomedical Research (LBR) pathologists and the percent of total area of tumor in the segment was also documented. Additionally, disease-specific working group pathology experts reviewed the morphology to clarify or standardize specific disease classifications and correlation to the proteomic and genomic data.</div><div class="section-paragraph">Specimens selected for this study were determined on the maximal percent in the pathology criteria and best weight. Specimens were pulled from the biorepository using an LN<sub>2</sub> cryocart to maintain specimen integrity and then cryopulverized. The cryopulverized specimen was divided into aliquots for DNA (30 mg) and RNA (30 mg) isolation and proteomics (50 mg) for molecular characterization. Nucleic acids were isolated and stored at -80°C until further processing and distribution; cryopulverized protein material was returned to the LN<sub>2</sub> freezer until distribution. Shipment of the cryopulverized segments used cryoports for distribution to the proteomic characterization centers and shipment of the nucleic acids used dry ice shippers for distribution to the genomic characterization centers; a shipment manifest accompanied all distributions for the receipt and integrity inspection of the specimens at the destination. The DNA sequencing was performed at the Broad Institute, Cambridge, MA and RNA sequencing was performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses was sent to the Proteomic Characterization Center at Johns Hopkins University, Maryland, USA.</div></section></section><section id="sec5.4" data-locator="sec5.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method details</span></h3><section id="sec5.4.1" data-locator="sec5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomics and transcriptomics profiling experiments</span></h4><section id="sec5.4.1.1" data-locator="sec5.4.1.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample processing for genomic DNA and total RNA extraction</span></h4><div class="section-paragraph">Our study sampled a single site of the primary tumor from surgical resections, due to the internal requirement to process a minimum of 125 mg of tumor issue and 50 mg of adjacent normal tissue. DNA and RNA were extracted from tumor and normal specimens in a co-isolation protocol using Qiagen’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3-5 mL) to serve as matched germline reference material. The Qubit™ dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorometer to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through various genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality was assessed using the Agilent Bioanalyzer. A sample that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity match for germline, normal adjacent tissue, and tumor tissue was assayed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification.</div></section></section><section id="sec5.4.2" data-locator="sec5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole exome sequencing (WES)</span></h4><section id="sec5.4.2.1" data-locator="sec5.4.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Library construction</span></h4><div class="section-paragraph">Library construction was performed as described in (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d52364298e3721" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Fisher et al., 2011</a><div data-db-target-of="bib42-d52364298e3721" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fisher S. </li><li class="loa__item">Barry A. </li><li class="loa__item">Abreu J. </li><li class="loa__item">Minie B. </li><li class="loa__item">Nolan J. </li><li class="loa__item">Delorey T.M. </li><li class="loa__item">Young G. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Allen A. </li><li class="loa__item">Ambrogio L. </li><li>et al.</li></ul><div class="reference-citations__detials">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with the following modifications: initial genomic DNA input into shearing was reduced from 3 μg to 20-250 ng in 50 μL of solution. For adapter ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies, with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30 μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted.</div></section><section id="sec5.4.2.2" data-locator="sec5.4.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">In-solution hybrid selection</span></h4><div class="section-paragraph">After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina's Nextera Exome Kit and followed the manufacturer’s suggested protocol, with the following exceptions. First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system.</div></section><section id="sec5.4.2.3" data-locator="sec5.4.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of libraries for cluster amplification and sequencing</span></h4><div class="section-paragraph">After post-capture enrichment, library pools were quantified using qPCR (automated assay on the Agilent Bravo) using a kit purchased from KAPA Biosystems with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2 nM.</div></section><section id="sec5.4.2.4" data-locator="sec5.4.2.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster amplification and sequencing</span></h4><div class="section-paragraph">Cluster amplification of DNA libraries was performed according to the manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flowcells were sequenced utilizing sequencing-by-synthesis chemistry. The flowcells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76 cycle runs with two 8 cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq 4000 paired-end runs to achieve a minimum of 150x on target coverage per sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling.</div></section></section><section id="sec5.4.3" data-locator="sec5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PCR-free whole genome sequencing</span></h4><section id="sec5.4.3.1" data-locator="sec5.4.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of libraries for cluster amplification and sequencing</span></h4><div class="section-paragraph">An aliquot of genomic DNA (350 ng in 50 μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using a SPRI cleanup. Library preparation was performed using a commercially available kit provided by KAPA Biosystems (KAPA Hyper Prep without amplification module) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (purchased from IDT). Following sample preparation, libraries were quantified using quantitative PCR (kit purchased from KAPA Biosystems), with probes specific to the ends of the adapters. This assay was automated using Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7 nM and pooled into 24-plexes.</div></section><section id="sec5.4.3.2" data-locator="sec5.4.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster amplification and sequencing (HiSeq X)</span></h4><div class="section-paragraph">Sample pools were combined with HiSeq X Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flowcells were sequenced to a minimum of 15x on HiSeq X utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAMs containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging.</div></section><section id="sec5.4.3.3" data-locator="sec5.4.3.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Illumina infinium methylationEPIC beadchip array</span></h4><div class="section-paragraph">The MethylationEPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt; 850,000 DNA methylation sites per sample. 250 ng of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit. The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data were processed through an automated genotype calling pipeline. Data generated consisted of raw IDAT files and a sample sheet.</div></section></section><section id="sec5.4.4" data-locator="sec5.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA sequencing</span></h4><section id="sec5.4.4.1" data-locator="sec5.4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quality assurance and quality control of RNA analytes</span></h4><div class="section-paragraph">All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt; 8.0 were considered high quality.</div></section><section id="sec5.4.4.2" data-locator="sec5.4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA-seq library construction</span></h4><div class="section-paragraph">Total RNA-seq library construction was performed from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags following the manufacturer’s instructions (Illumina, Inc. San Diego, CA). Libraries were prepared on an Agilent Bravo Automated Liquid Handling System. Quality control was performed at every step and the libraries were quantified using the TapeStation system.</div></section><section id="sec5.4.4.3" data-locator="sec5.4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were prepared and run on HiSeq 4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. Typically, these were pools of four samples. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were quantified. Samples were then assessed for quality by mapping reads to the hg38 human genome reference, estimating the total number of reads that mapped, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository.</div></section><section id="sec5.4.4.4" data-locator="sec5.4.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-seq library construction</span></h4><div class="section-paragraph">miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and bar-coded with individual tags following the manufacturer’s instructions. Libraries were prepared on the Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex Kit specifications using a Pippin Prep system (Sage Science, Beverly, MA).</div></section><section id="sec5.4.4.5" data-locator="sec5.4.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were loaded on the Hiseq 4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (<div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d52364298e3785" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Chu et al., 2016</a><div data-db-target-of="bib27-d52364298e3785" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chu A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Brooks D. </li><li class="loa__item">Mungall A.J. </li><li class="loa__item">Birol I. </li><li class="loa__item">Coope R. </li><li class="loa__item">Ma Y. </li><li class="loa__item">Jones S. </li><li class="loa__item">Marra M.A. </li></ul><div class="reference-citations__detials">Large-scale profiling of microRNAs for the cancer genome Atlas.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res.&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+the+cancer+genome+Atlas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Samples were assessed for the number of miRNAs called, species diversity, and total abundance. Samples passing quality control were uploaded to the GDC repository.</div></section></section><section id="sec5.4.5" data-locator="sec5.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomics and transcriptomics data processing</span></h4><section id="sec5.4.5.1" data-locator="sec5.4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic mutation calling</span></h4><div class="section-paragraph">Somatic variants were called by the Somaticwrapper pipeline, which includes four different callers, i.e., Strelka v.2 (<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d52364298e3796" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Saunders et al., 2012</a><div data-db-target-of="bib113-d52364298e3796" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Saunders C.T. </li><li class="loa__item">Wong W.S.W. </li><li class="loa__item">Swamy S. </li><li class="loa__item">Becq J. </li><li class="loa__item">Murray L.J. </li><li class="loa__item">Cheetham R.K. </li></ul><div class="reference-citations__detials">Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1811-1817</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbts271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22581179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84864153492&amp;cf=" onclick="newWindow(this.href);return false">Scopus (752)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1811-1817&journal=Bioinformatics&author=Saunders+C.T.&author=Wong+W.S.W.&author=Swamy+S.&author=Becq+J.&author=Murray+L.J.&author=Cheetham+R.K.&title=Strelka%3A+accurate+somatic+small-variant+calling+from+sequenced+tumor-normal+sample+pairs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), MUTECT v1.7 (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d52364298e3799" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Cibulskis et al., 2013</a><div data-db-target-of="bib29-d52364298e3799" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2136)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), VarScan v.2.3.8 (<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d52364298e3802" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Koboldt et al., 2012</a><div data-db-target-of="bib69-d52364298e3802" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2185)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and Pindel v.0.2.5 (<div class="dropBlock reference-citations"><a href="#bib148" id="back-bib148" data-db-target-for="bib148-d52364298e3805" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib148">Ye et al., 2009</a><div data-db-target-of="bib148-d52364298e3805" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1151)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from WES. We kept the exonic SNVs called by any 2 callers among MUTECT v1.7, VarScan v.2.3.8, and Strelka v.2 and indels called by any 2 callers among VarScan v.2.3.8, Strelka v.2, and Pindel v.0.2.5. For the merged SNVs and indels, we applied a 14X and 8X coverage cutoff for tumor and normal, separately. We also filtered SNVs and indels by a minimal variant allele frequency (VAF) of 0.05 in tumors and a maximal VAF of 0.02 in normal samples. Finally, we filtered any SNV which was within 10bp of an indel found in the same tumor sample.</div></section><section id="sec5.4.5.2" data-locator="sec5.4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline variant calling</span></h4><div class="section-paragraph">Germline Variant Calling was performed using germlinewrapper v1.1, which implements multiple tools for the detection of germline INDELs and SNVs. Germline SNVs were identified using VarScan v2.3.8 (with parameters: --min-var-freq 0.10 --p-value 0.10, --min-coverage 3 --strand-filter 1) operating on a mpileup stream produced by samtools v1.2 (with parameters: -q 1 -Q 13) and GATK v4.0.0.0 (<div class="dropBlock reference-citations"><a href="#bib94" id="back-bib94" data-db-target-for="bib94-d52364298e3813" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib94">McKenna et al., 2010</a><div data-db-target-of="bib94-d52364298e3813" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li class="loa__item">DePristo M.A. </li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10664)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using its haplotype caller in single-sample mode with duplicate and unmapped reads removed and retaining calls with a minimum quality threshold of 10. All resulting variants were limited to the coding region of the full-length transcripts obtained from Ensembl release 95 plus additional two base pairs flanking each exon to cover splice donor/acceptor sites. We required variants to have allelic depth ≥ 5 reads for the alternative allele. We filtered large INDELs that were longer than 100 bps.</div></section><section id="sec5.4.5.3" data-locator="sec5.4.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">DNA methylation array</span></h4><div class="section-paragraph">The raw data from Illumina's EPIC methylation arrays were available as IDAT files from the CPTAC consortium. The methylation analysis was performed using the cross-package workflow methylationArrayAnalysis available on Bioconductor. In brief, the raw data IDAT files were processed to obtain the methylated (M) and unmethylated (U) signal intensities for each locus. The processing step included an unsupervised normalization step called functional normalization that has been previously implemented for Illumina 450K methylation arrays (<div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d52364298e3821" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Fortin et al., 2014</a><div data-db-target-of="bib43-d52364298e3821" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fortin J.-P. </li><li class="loa__item">Labbe A. </li><li class="loa__item">Lemire M. </li><li class="loa__item">Zanke B.W. </li><li class="loa__item">Hudson T.J. </li><li class="loa__item">Fertig E.J. </li><li class="loa__item">Greenwood C.M. </li><li class="loa__item">Hansen K.D. </li></ul><div class="reference-citations__detials">Functional normalization of 450k methylation array data improves replication in large cancer studies.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (284)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.-P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A detection p value was also calculated for each locus, and this p value captured the quality of detection at the locus with respect to negative control background probes included in the array. Loci having common SNPs (with MAF &gt; 0.01), as per dbSNP build 132 through 147 via the UCSC “snp132common” track through “snp147common” track, were removed from further analysis. Beta values were calculated as M/(M+U), that is equal to the fraction methylated for each locus. Beta values of loci whose detection p values were &gt; 0.01 were assigned NA in the output file. All loci were annotated with the EPIC Manifest from MethylationEPIC_v-1-0_B2.csv from the zip archive infinium-methylationepic-v1-0-b2-manifest-file-csv.zip from Illumina through the “IlluminaHumanMethylationEPICanno.ilm10b2.hg19” package on Bioconductor. To map EPIC arrays to the GRCh38 assembly, all probes were reannotated by annotation information from InfiniumAnnotation.</div></section><section id="sec5.4.5.4" data-locator="sec5.4.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Copy-number analysis</span></h4><div class="section-paragraph">Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing data of the tumor and germline DNA, using CNVEX (<a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a>). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3-0.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by a custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) (<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d52364298e3832" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Gey and Lebarbier, 2008</a><div data-db-target-of="bib50-d52364298e3832" class="dropBlock__holder"><div class="dropBlock__body"><p>Gey, S., and Lebarbier, E. (2008). Using CART to Detect Multiple Change Points in the Mean for Large Sample, <a class="inter-ref-link" href="https://hal.archives-ouvertes.fr/hal-00327146/" target="_blank">https://hal.archives-ouvertes.fr/hal-00327146/</a>.</p><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Gey%2C+S.%2C+and+Lebarbier%2C+E.+%282008%29.+Using+CART+to+Detect+Multiple+Change+Points+in+the+Mean+for+Large+Sample%2C+https%3A%2F%2Fhal.archives-ouvertes.fr%2Fhal-00327146%2F.">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and dynamic programming (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d52364298e3835" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Bellman, 1961</a><div data-db-target-of="bib11-d52364298e3835" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bellman R. </li></ul><div class="reference-citations__detials">On the approximation of curves by line segments using dynamic programming.<div class="popup-ref"><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), as implemented in the R package jointseg (<div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d52364298e3838" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Pierre-Jean et al., 2015</a><div data-db-target-of="bib105-d52364298e3838" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pierre-Jean M. </li><li class="loa__item">Rigaill G. </li><li class="loa__item">Neuvial P. </li></ul><div class="reference-citations__detials">Performance evaluation of DNA copy number segmentation methods.<div class="popup-ref"><span class="ref__series"><i>Brief Bioinform.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">600-615</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbib%2Fbbu026&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25202135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84925419952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=600-615&journal=Brief+Bioinform&author=Pierre-Jean+M.&author=Rigaill+G.&author=Neuvial+P.&title=Performance+evaluation+of+DNA+copy+number+segmentation+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs, and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d52364298e3841" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Lebarbier, 2005</a><div data-db-target-of="bib77-d52364298e3841" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lebarbier E. </li></ul><div class="reference-citations__detials">Detecting multiple change-points in the mean of Gaussian process by model selection.<div class="popup-ref"><span class="ref__series"><i>Signal Process.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Process.&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the final set of CNV segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data.</div></section><section id="sec5.4.5.5" data-locator="sec5.4.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA quantification and circular RNA prediction</span></h4><div class="section-paragraph">The hg38 reference genome and RefSeq annotations were used for the RNA-seq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression (<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d52364298e3849" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">Li et al., 2017</a><div data-db-target-of="bib82-d52364298e3849" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li M. </li><li class="loa__item">Xie X. </li><li class="loa__item">Zhou J. </li><li class="loa__item">Sheng M. </li><li class="loa__item">Yin X. </li><li class="loa__item">Ko E.-A. </li><li class="loa__item">Zhou T. </li><li class="loa__item">Gu W. </li></ul><div class="reference-citations__detials">Quantifying circular RNA expression from RNA-seq data using model-based framework.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinformatics&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene and circular RNA expression based on the mixed transcripts. After quantification, the upper quantile method was applied for normalization. The normalized matrix was log2-transformed and separated into gene and circular RNA expression matrices.</div></section><section id="sec5.4.5.6" data-locator="sec5.4.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-Seq data analysis</span></h4><div class="section-paragraph">Processed miRNA bam files were downloaded from GDC and transferred to fastq format using samtools (version 1.10). QuickMIRSeq (<div class="dropBlock reference-citations"><a href="#bib152" id="back-bib152" data-db-target-for="bib152-d52364298e3857" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib152">Zhao et al., 2017</a><div data-db-target-of="bib152-d52364298e3857" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhao S. </li><li class="loa__item">Gordon W. </li><li class="loa__item">Du S. </li><li class="loa__item">Zhang C. </li><li class="loa__item">He W. </li><li class="loa__item">Xi L. </li><li class="loa__item">Mathur S. </li><li class="loa__item">Agostino M. </li><li class="loa__item">Paradis T. </li><li class="loa__item">von Schack D. </li><li>et al.</li></ul><div class="reference-citations__detials">QuickMIRSeq: a pipeline for quick and accurate quantification of both known miRNAs and isomiRs by jointly processing multiple samples from microRNA sequencing.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">180</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs12859-017-1601-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28320324&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85015748083&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=180&journal=BMC+Bioinformatics&author=Zhao+S.&author=Gordon+W.&author=Du+S.&author=Zhang+C.&author=He+W.&author=Xi+L.&author=Mathur+S.&author=Agostino+M.&author=Paradis+T.&author=von+Schack+D.&title=QuickMIRSeq%3A+a+pipeline+for+quick+and+accurate+quantification+of+both+known+miRNAs+and+isomiRs+by+jointly+processing+multiple+samples+from+microRNA+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with bowtie as a mapping tool (version 1.1.1) was used to quantify miRNA expression. The following parameters were used for QuickMIRSeq: 1) 2 bp extension / shorten were allowed in both upstream and downstream regions of mature miRNA; 2) The minimum and maximum length of miRNA reads were set to 16 and 28, respectively. Then RPM (reads per million) values were used to quantify miRNA expression levels.</div></section><section id="sec5.4.5.7" data-locator="sec5.4.5.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">HPV virus identification</span></h4><div class="section-paragraph">The unmapped RNA-seq reads to the human reference genome were extracted and mapped to the virus reference by VirusScan (<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d52364298e3865" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Cao et al., 2016</a><div data-db-target-of="bib23-d52364298e3865" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cao S. </li><li class="loa__item">Wendl M.C. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Wylie K. </li><li class="loa__item">Ye K. </li><li class="loa__item">Jayasinghe R. </li><li class="loa__item">Xie M. </li><li class="loa__item">Wu S. </li><li class="loa__item">Niu B. </li><li class="loa__item">Grubb R. </li><li>et al.</li></ul><div class="reference-citations__detials">Divergent viral presentation among human tumors and adjacent normal tissues.<div class="popup-ref"><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">28294</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsrep28294&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27339696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975700575&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=28294&journal=Sci.+Rep.&author=Cao+S.&author=Wendl+M.C.&author=Wyczalkowski+M.A.&author=Wylie+K.&author=Ye+K.&author=Jayasinghe+R.&author=Xie+M.&author=Wu+S.&author=Niu+B.&author=Grubb+R.&title=Divergent+viral+presentation+among+human+tumors+and+adjacent+normal+tissues">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The reference contained the known HPV genotypes including the "high-risk" genotypes such as HPV 16 and HPV 18. Samples were classified as HPV positive using an empiric definition of detection of &gt; 1,000 mapped RNA-seq reads.</div></section><section id="sec5.4.5.8" data-locator="sec5.4.5.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutational signature analysis</span></h4><div class="section-paragraph">Non-negative matrix factorization algorithm (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used the Somaticwrapper pipeline to call mutations from WES and WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization) (<div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d52364298e3873" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Kasar et al., 2015</a><div data-db-target-of="bib61-d52364298e3873" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kasar S. </li><li class="loa__item">Kim J. </li><li class="loa__item">Improgo R. </li><li class="loa__item">Tiao G. </li><li class="loa__item">Polak P. </li><li class="loa__item">Haradhvala N. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Kiezun A. </li><li class="loa__item">Fernandes S.M. </li><li class="loa__item">Bahl S. </li><li>et al.</li></ul><div class="reference-citations__detials">Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">8866</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms9866&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26638776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8866&journal=Nat.+Commun.&author=Kasar+S.&author=Kim+J.&author=Improgo+R.&author=Tiao+G.&author=Polak+P.&author=Haradhvala+N.&author=Lawrence+M.S.&author=Kiezun+A.&author=Fernandes+S.M.&author=Bahl+S.&title=Whole-genome+sequencing+reveals+activation-induced+cytidine+deaminase+signatures+during+indolent+chronic+lymphocytic+leukaemia+evolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d52364298e3876" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Kim et al., 2016</a><div data-db-target-of="bib64-d52364298e3876" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Polak P. </li><li class="loa__item">Braunstein L.Z. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Rosenberg J.E. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">D'Andrea A. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (154)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%27Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d52364298e3879" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Tan and Févotte, 2013</a><div data-db-target-of="bib126-d52364298e3879" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tan V.Y.F. </li><li class="loa__item">Févotte C. </li></ul><div class="reference-citations__detials">Automatic relevance determination in nonnegative matrix factorization with the β-divergence.<div class="popup-ref"><span class="ref__series"><i>IEEE Trans. Pattern Anal. Mach. Intell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1592-1605</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1109%2FTPAMI.2012.240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23681989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84878158173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (140)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2013&pages=1592-1605&journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&author=Tan+V.Y.F.&author=F%C3%A9votte+C.&title=Automatic+relevance+determination+in+nonnegative+matrix+factorization+with+the+%CE%B2-divergence">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Signatures were compared against known signatures derived from COSMIC (<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d52364298e3882" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Tate et al., 2019</a><div data-db-target-of="bib128-d52364298e3882" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tate J.G. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Jubb H.C. </li><li class="loa__item">Sondka Z. </li><li class="loa__item">Beare D.M. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Creatore C. </li><li class="loa__item">Dawson E. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: the catalogue of somatic mutations in cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (622)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+catalogue+of+somatic+mutations+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and cosine similarity was calculated to identify the best match.</div></section><section id="sec5.4.5.9" data-locator="sec5.4.5.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Gene fusion detection</span></h4><div class="section-paragraph">Fusions in RNA-seq samples were called using three callers: STAR-Fusion v1.5.0 (<div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d52364298e3890" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Haas et al., 2019</a><div data-db-target-of="bib52-d52364298e3890" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haas B.J. </li><li class="loa__item">Dobin A. </li><li class="loa__item">Li B. </li><li class="loa__item">Stransky N. </li><li class="loa__item">Pochet N. </li><li class="loa__item">Regev A. </li></ul><div class="reference-citations__detials">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (43)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), EricScript v0.5.5 (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d52364298e3893" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benelli et al., 2012</a><div data-db-target-of="bib13-d52364298e3893" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benelli M. </li><li class="loa__item">Pescucci C. </li><li class="loa__item">Marseglia G. </li><li class="loa__item">Severgnini M. </li><li class="loa__item">Torricelli F. </li><li class="loa__item">Magi A. </li></ul><div class="reference-citations__detials">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and INTEGRATE v0.2.6 (<div class="dropBlock reference-citations"><a href="#bib151" id="back-bib151" data-db-target-for="bib151-d52364298e3896" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib151">Zhang et al., 2016</a><div data-db-target-of="bib151-d52364298e3896" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang J. </li><li class="loa__item">White N.M. </li><li class="loa__item">Schmidt H.K. </li><li class="loa__item">Fulton R.S. </li><li class="loa__item">Tomlinson C. </li><li class="loa__item">Warren W.C. </li><li class="loa__item">Wilson R.K. </li><li class="loa__item">Maher C.A. </li></ul><div class="reference-citations__detials">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). As STAR-Fusion has higher sensitivity, calls made by this tool with higher supporting evidence (defined by fusion fragments per million total reads, or FFPM &gt; 0.1) is required, or a given fusion must be reported by at least 2 callers being retained. Fusions present in the following databases were then excluded: 1) uncharacterized genes, immunoglobulin genes, mitochondrial genes, etc.; 2) fusions from the same gene or paralog genes; and 3) fusions reported in TCGA normal samples (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d52364298e3899" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Gao et al., 2018</a><div data-db-target-of="bib46-d52364298e3899" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao Q. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Foltz S.M. </li><li class="loa__item">Mutharasu G. </li><li class="loa__item">Jayasinghe R.G. </li><li class="loa__item">Cao S. </li><li class="loa__item">Liao W.-W. </li><li class="loa__item">Reynolds S.M. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Yao L. </li><li>et al.</li></ul><div class="reference-citations__detials">Driver fusions and their implications in the development and treatment of human cancers.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">227-238.e223</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29617662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85044744284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (142)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=227-238.e223&journal=Cell+Rep.&author=Gao+Q.&author=Liang+W.-W.&author=Foltz+S.M.&author=Mutharasu+G.&author=Jayasinghe+R.G.&author=Cao+S.&author=Liao+W.-W.&author=Reynolds+S.M.&author=Wyczalkowski+M.A.&author=Yao+L.&title=Driver+fusions+and+their+implications+in+the+development+and+treatment+of+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), GTEx tissues (reported in the STAR-Fusion output), and non-cancer cell studies (<div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d52364298e3902" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Babiceanu et al., 2016</a><div data-db-target-of="bib7-d52364298e3902" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Babiceanu M. </li><li class="loa__item">Qin F. </li><li class="loa__item">Xie Z. </li><li class="loa__item">Jia Y. </li><li class="loa__item">Lopez K. </li><li class="loa__item">Janus N. </li><li class="loa__item">Facemire L. </li><li class="loa__item">Kumar S. </li><li class="loa__item">Pang Y. </li><li class="loa__item">Qi Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Recurrent chimeric fusion RNAs in non-cancer tissues and cells.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">2859-2872</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkw032&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26837576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84963985214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=2859-2872&journal=Nucleic+Acids+Res.&author=Babiceanu+M.&author=Qin+F.&author=Xie+Z.&author=Jia+Y.&author=Lopez+K.&author=Janus+N.&author=Facemire+L.&author=Kumar+S.&author=Pang+Y.&author=Qi+Y.&title=Recurrent+chimeric+fusion+RNAs+in+non-cancer+tissues+and+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Finally, normal fusions were filtered out from the tumor fusions.</div></section><section id="sec5.4.5.10" data-locator="sec5.4.5.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Structural variant analysis</span></h4><div class="section-paragraph">Structural variants in WGS samples were called with Manta 1.3.2 (<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d52364298e3910" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Chen et al., 2016</a><div data-db-target-of="bib26-d52364298e3910" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Schulz-Trieglaff O. </li><li class="loa__item">Shaw R. </li><li class="loa__item">Barnes B. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Cox A.J. </li><li class="loa__item">Kruglyak S. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (366)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), retaining variants where sample site depth is less than 3x the median chromosome depth near one or both variant breakends, somatic score is greater than 30, and for small variants (&lt;1000 bases) in the normal sample, the fraction of reads with MAPQ0 around either breakend does not exceed 0.4. It is optimized for the analysis of somatic variation in tumor/normal sample pairs. The paired and split-read evidence were combined during the SV discovery and scoring to improve accuracy. We prioritized the variants by the number of spanning read pairs which strongly (Q30) support the variants (&gt;5 as high confidence level).</div></section><section id="sec5.4.5.11" data-locator="sec5.4.5.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">SCNA arm and focal significance</span></h4><div class="section-paragraph">From the segment-level SCNA data, we used GISTIC2 (<div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d52364298e3919" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">Mermel et al., 2011</a><div data-db-target-of="bib95-d52364298e3919" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1119)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to assess the arm- and focal-level SCNA significance using the default parameters except for increased threshold of significance (i.e., -ta and -td parameters of GISTIC2) to 0.3 based on the distribution of germline copy number variants.</div></section></section><section id="sec5.4.6" data-locator="sec5.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic and phosphoproteomic profiling experiments</span></h4><section id="sec5.4.6.1" data-locator="sec5.4.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample processing for protein extraction and tryptic digestion</span></h4><div class="section-paragraph">All samples for the current study were prospectively collected for the Clinical Proteomic Tumor Analysis Consortium (CPTAC) project as described above and processed for mass spectrometric (MS) analysis at Johns Hopkins University (JHU). Tissue lysis and downstream sample preparation for global proteomic and phosphoproteomic analysis was carried out as previously described (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d52364298e3930" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Clark et al., 2018</a><div data-db-target-of="bib32-d52364298e3930" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Bocik W. </li><li class="loa__item">Chen L. </li><li class="loa__item">Schnaubelt M. </li><li class="loa__item">Roberts R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Whiteley G. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Evaluation of NCI-7 cell line panel as a reference material for clinical proteomics.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">2205-2215</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Facs.jproteome.8b00165&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29718670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85046635566&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2205-2215&journal=J.%C2%A0Proteome+Res.&author=Clark+D.J.&author=Hu+Y.&author=Bocik+W.&author=Chen+L.&author=Schnaubelt+M.&author=Roberts+R.&author=Shah+P.&author=Whiteley+G.&author=Zhang+H.&title=Evaluation+of+NCI-7+cell+line+panel+as+a+reference+material+for+clinical+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib97" id="back-bib97" data-db-target-for="bib97-d52364298e3933" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Mertins et al., 2018</a><div data-db-target-of="bib97-d52364298e3933" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Approximately 25-120 mg of each cryo-pulverized head and neck squamous cell carcinoma (HNSCC) tissue or normal adjacent tissue (NAT) was resuspended in lysis buffer (8 M urea, 75 mM NaCl, 50 mM Tris, pH 8.0, 1 mM EDTA, 2 μg/mL aprotinin, 10 μg/mL leupeptin, 1 mM PMSF, 10 mM NaF, Phosphatase Inhibitor Cocktail 2 and Phosphatase Inhibitor Cocktail 3 [1:100 dilution], and 20 μM PUGNAc) by repeated vortexing. Lysates were clarified by centrifugation at 20,000 x g for 10 min at 4°C, and protein concentrations were determined by BCA assay (Pierce). Proteins were diluted to a final concentration of 8 mg/mL with lysis buffer, and 800 mg of protein was reduced with 5 mM dithiothreitol (DTT, ThermoFisher) for 1h at 37°C, and subsequently alkylated with 10 mM iodoacetamide (Sigma) for 45 min at room temperature (RT) in the dark. Samples were diluted 1:4 with 50 mM Tris-HCl (pH 8.0) and subjected to proteolytic digestion with LysC (Wako Chemicals) at 1 mAU:50 mg enzyme-to-substrate ratio for 2h at RT, followed by the addition of sequencing grade modified trypsin (Promega) at 1:50 enzyme-to-substrate ratio and overnight incubation at RT. The digested samples were then acidified with 50% Formic acid (FA, Fisher Chemicals) to pH 2. Tryptic peptides were desalted on reversed phase C18 SPE columns (Waters) and dried using Speed-Vac (Thermo Scientific).</div></section><section id="sec5.4.6.2" data-locator="sec5.4.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT labeling of peptides</span></h4><div class="section-paragraph">Dried peptides from each sample were labeled with 11-plex TMT (Tandem Mass Tag) reagents (Thermo Fisher Scientific). Peptides (300 μg) from each of the HNSCC and NAT samples were dissolved in 60 μL of 50 mM HEPES, pH 8.5 solution. An internal quality control (QC) sample, NCI-7 Cell Line (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d52364298e3941" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Clark et al., 2018</a><div data-db-target-of="bib32-d52364298e3941" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Bocik W. </li><li class="loa__item">Chen L. </li><li class="loa__item">Schnaubelt M. </li><li class="loa__item">Roberts R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Whiteley G. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Evaluation of NCI-7 cell line panel as a reference material for clinical proteomics.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">2205-2215</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Facs.jproteome.8b00165&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29718670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85046635566&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2205-2215&journal=J.%C2%A0Proteome+Res.&author=Clark+D.J.&author=Hu+Y.&author=Bocik+W.&author=Chen+L.&author=Schnaubelt+M.&author=Roberts+R.&author=Shah+P.&author=Whiteley+G.&author=Zhang+H.&title=Evaluation+of+NCI-7+cell+line+panel+as+a+reference+material+for+clinical+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was interspersed among all TMT 11-plex sets. HNSCC and NAT samples with NCI-7 QC aliquots were co-randomized to 19 TMT sets. A reference sample was created by pooling an aliquot from 87 HNSCC tissues and 50 NAT tissues (representing ∼80% of the sample cohort), and included in all TMT 11-plex sets as a pooled reference channel. Five mg of TMT reagent was dissolved in 250 μL of anhydrous acetonitrile, and then 20 μL of each TMT reagent was added to the corresponding aliquot of peptides. After 1h incubation at RT, the reaction was quenched by incubation with 5% NH2OH for 15 min at RT. Following labeling, peptides were desalted on reversed phase C18 SPE columns (Waters) and dried using Speed-Vac (Thermo Scientific).</div></section><section id="sec5.4.6.3" data-locator="sec5.4.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide fractionation by basic reversed-phase liquid chromatography (bRPLC)</span></h4><div class="section-paragraph">To reduce the likelihood of peptides co-isolating and co-fragmenting due to high sample complexity, we employed extensive, high-resolution fractionation via basic reversed phase liquid chromatography (bRPLC). Previous reports have indicated this approach can reduce the incidence of isobaric reporter ion ratio distortion effects, which would impact downstream quantitation (<div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d52364298e3949" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Ow et al., 2011</a><div data-db-target-of="bib103-d52364298e3949" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ow S.Y. </li><li class="loa__item">Salim M. </li><li class="loa__item">Noirel J. </li><li class="loa__item">Evans C. </li><li class="loa__item">Wright P.C. </li></ul><div class="reference-citations__detials">Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2341-2346</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fpmic.201000752&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21548092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79956270174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=2341-2346&journal=Proteomics&author=Ow+S.Y.&author=Salim+M.&author=Noirel+J.&author=Evans+C.&author=Wright+P.C.&title=Minimising+iTRAQ+ratio+compression+through+understanding+LC-MS+elution+dependence+and+high-resolution+HILIC+fractionation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib107" id="back-bib107" data-db-target-for="bib107-d52364298e3952" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib107">Rauniyar and Yates, 2014</a><div data-db-target-of="bib107-d52364298e3952" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rauniyar N. </li><li class="loa__item">Yates J.R. </li></ul><div class="reference-citations__detials">Isobaric labeling-based relative quantification in shotgun proteomics.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">5293-5309</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fpr500880b&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25337643&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84915751035&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=5293-5309&journal=J.%C2%A0Proteome+Res.&author=Rauniyar+N.&author=Yates+J.R.&title=Isobaric+labeling-based+relative+quantification+in+shotgun+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For each TMT set, about 3.3 mg desalted peptides were reconstituted in 900 μL of 5 mM ammonium formate (pH 10) and 2% acetonitrile (ACN) and loaded onto a 4.6 mm x 250 mm RP Zorbax 300 A Extend-C18 column with 3.5 mm size beads (Agilent). Peptides were separated using an Agilent 1200 Series HPLC instrument using basic reversed-phase chromatography with Solvent A (2% ACN, 5 mM ammonium formate, pH 10) and a non-linear gradient of Solvent B (90% ACN, 5 mM ammonium formate, pH 10) at 1 mL/min as follows: 0% Solvent B (7 min), 0% to 16% Solvent B, (6 min), 16% to 40% Solvent B 60 min), 40% to 44% Solvent B (4 min), 44% to 60% Solvent B (5 min) and then held at 60% Solvent B for 14 min. Collected fractions were concatenated into 24 fractions as described previously (<div class="dropBlock reference-citations"><a href="#bib97" id="back-bib97" data-db-target-for="bib97-d52364298e3955" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Mertins et al., 2018</a><div data-db-target-of="bib97-d52364298e3955" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); 5% of each of the 24 fractions was aliquoted for global proteomic analysis, dried down in a Speed-Vac, and resuspended in 3% ACN, 0.1% formic acid prior to ESI-LC-MS/MS analysis. The remaining sample was utilized for phosphopeptide enrichment.</div></section><section id="sec5.4.6.4" data-locator="sec5.4.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Enrichment of phosphopeptides by Fe-IMAC</span></h4><div class="section-paragraph">The remaining 95% of the sample were further concatenated into 12 fractions prior to phosphopeptide enrichment using immobilized metal affinity chromatography (IMAC) as previously described (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d52364298e3963" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Mertins et al., 2013</a><div data-db-target-of="bib96-d52364298e3963" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Patel J. </li><li class="loa__item">Udeshi N.D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Burgess M.W. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Jaffe J.D. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Integrated proteomic analysis of post-translational modifications by serial enrichment.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">634-637</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.2518&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23749302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84879637572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (333)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=634-637&journal=Nat.+Methods&author=Mertins+P.&author=Qiao+J.W.&author=Patel+J.&author=Udeshi+N.D.&author=Clauser+K.R.&author=Mani+D.R.&author=Burgess+M.W.&author=Gillette+M.A.&author=Jaffe+J.D.&author=Carr+S.A.&title=Integrated+proteomic+analysis+of+post-translational+modifications+by+serial+enrichment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In brief, Ni-NTA agarose beads were utilized to prepare Fe<sup>3+</sup>-NTA agarose beads, and then about 250 μg peptides of each fraction reconstituted in 80% ACN/0.1% trifluoroacetic acid were incubated with 10 μL of the Fe<sup>3+</sup>-IMAC beads for 30 mins. Samples were then spun down and the supernatant containing unbound peptides was removed. The beads were brought up in 80% ACN, 0.1% trifluoroacetic acid and then loaded onto equilibrated C-18 Stage Tips, and washed by 80% ACN, 0.1% trifluoroacetic acid. Tips were rinsed twice with 1% formic acid, followed by sample elution off the Fe<sup>3+</sup>-IMAC beads and onto the C-18 State Tips with 70 μL of 500 mM dibasic potassium phosphate, pH 7.0 three times. C-18 Stage Tips were washed twice with 1% formic acid, followed by elution of the phosphopeptides from the C-18 Stage Tips with 50% ACN, 0.1% formic acid twice. Samples were dried down and resuspended in 3% ACN, 0.1% formic acid prior to ESI-LC-MS/MS analysis.</div></section><section id="sec5.4.6.5" data-locator="sec5.4.6.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">ESI-LC-MS/MS for global proteome and phosphoproteome analysis</span></h4><div class="section-paragraph">The global proteome and phosphoproteome fractions were analyzed using the same instrumentation and methodology as described in a previous study (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d52364298e3980" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Clark et al., 2019</a><div data-db-target-of="bib31-d52364298e3980" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Pan J. </li><li class="loa__item">Song X. </li><li class="loa__item">Hu Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Reva B. </li><li class="loa__item">Lih T.-S.M. </li><li class="loa__item">Chang H.-Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated proteogenomic characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e931</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e931&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Peptides (∼ 0.8 μg) were separated on an Easy nLC 1200 UHPLC system (Thermo Scientific) on an in-house packed 20 cm x 75 mm diameter C18 column (1.9 mm Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10 mm opening (New Objective). The column was heated to 50°C using a column heater (Phoenix-ST). The flow rate was 0.200 μl/min with 0.1% formic acid and 2% acetonitrile in water (A) and 0.1% formic acid, 90% acetonitrile (B). The peptides were separated with a 6–30% B gradient in 84 mins and analyzed using the Thermo Fusion Lumos mass spectrometer (Thermo Scientific). Parameters were as follows MS1: resolution – 60,000, mass range – 350 to 1800 m/z, RF Lens – 30%, AGC Target 4.0e<sup>5</sup>, Max IT – 50 ms, charge state include - 2-6, dynamic exclusion – 45 s, top 20 ions selected for MS2; MS2: resolution – 50,000, high-energy collision dissociation activation energy (HCD) – 37, isolation width (m/z) – 0.7, AGC Target – 2.0e<sup>5</sup>, Max IT – 105 ms.</div></section></section><section id="sec5.4.7" data-locator="sec5.4.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomics and phosphoproteomics data processing</span></h4><section id="sec5.4.7.1" data-locator="sec5.4.7.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein database searching and quantification of global and phosphoproteomic data</span></h4><div class="section-paragraph">MS/MS spectra were searched using the MSFragger version 20190628 (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d52364298e3997" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Kong et al., 2017</a><div data-db-target-of="bib70-d52364298e3997" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kong A.T. </li><li class="loa__item">Leprevost F.V. </li><li class="loa__item">Avtonomov D.M. </li><li class="loa__item">Mellacheruvu D. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) against a CPTAC harmonized RefSeq protein sequence database appended with an equal number of decoy sequences. For the analysis of whole proteome data, MS/MS spectra were searched using a precursor-ion mass tolerance of 20 ppm, fragment mass tolerance of 20 ppm, and allowing C12/C13 isotope errors (1/0/1/2/3). Cysteine carbamidomethylation (+57.0215) and lysine TMT labeling (+229.1629) were specified as fixed modifications, and methionine oxidation (+15.9949), N-terminal protein acetylation (+42.0106), and TMT labeling of peptide N terminus and serine residues (to account for any over-labeling) were specified as variable modifications (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d52364298e4000" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Clark et al., 2019</a><div data-db-target-of="bib31-d52364298e4000" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Pan J. </li><li class="loa__item">Song X. </li><li class="loa__item">Hu Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Reva B. </li><li class="loa__item">Lih T.-S.M. </li><li class="loa__item">Chang H.-Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated proteogenomic characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e931</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e931&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib150" id="back-bib150" data-db-target-for="bib150-d52364298e4003" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib150">Zecha et al., 2019</a><div data-db-target-of="bib150-d52364298e4003" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zecha J. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Kanashova T. </li><li class="loa__item">Avanessian S.C. </li><li class="loa__item">Kane M.H. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Kuster B. </li></ul><div class="reference-citations__detials">TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (41)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell.+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+a+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The search was restricted to fully tryptic peptides, allowing up to two missed cleavage sites. For the analysis of phosphopeptide enriched data, the set of variable modifications also included phosphorylation (+79.9663) of serine, threonine, and tyrosine residues. The post-processing of the search results was done using the Philosopher toolkit version v1.6.0 (<a class="inter-ref-link" href="https://philosopher.nesvilab.org" target="_blank">https://philosopher.nesvilab.org</a>). MSFragger output files (in pepXML format) were processed using PeptideProphet (<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d52364298e4009" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Keller et al., 2002</a><div data-db-target-of="bib63-d52364298e4009" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keller A. </li><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.<div class="popup-ref"><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3577)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (with the high–mass accuracy binning and semi-parametric mixture modeling options) to compute the posterior probability of correct identification for each peptide to spectrum match (PSM). In the case of the phosphopeptide-enriched dataset, PeptideProphet files were additionally processed using PTMProphet (<div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d52364298e4013" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Deutsch et al., 2015</a><div data-db-target-of="bib38-d52364298e4013" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Deutsch E.W. </li><li class="loa__item">Mendoza L. </li><li class="loa__item">Shteynberg D. </li><li class="loa__item">Slagel J. </li><li class="loa__item">Sun Z. </li><li class="loa__item">Moritz R.L. </li></ul><div class="reference-citations__detials">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics.<div class="popup-ref"><span class="ref__series"><i>Proteomics Clin. Appl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">745-754</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fprca.201400164&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25631240&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84948987628&amp;cf=" onclick="newWindow(this.href);return false">Scopus (148)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=745-754&journal=Proteomics+Clin.+Appl.&author=Deutsch+E.W.&author=Mendoza+L.&author=Shteynberg+D.&author=Slagel+J.&author=Sun+Z.&author=Moritz+R.L.&title=Trans-Proteomic+Pipeline%2C+a+standardized+data+processing+pipeline+for+large-scale+reproducible+proteomics+informatics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to localize the phosphorylation sites. The resulting pepXML files from PeptideProphet (or PTMProphet) from all 20 TMT 11-plex experiments were then processed together to assemble peptides into proteins (protein inference) and to create a combined file (in protXML format) of high confidence protein groups. The combined protXML file and the individual PSM lists for each TMT 11-plex were further processed using the Philosopher filter command as follows. Each peptide was assigned either as a unique peptide to a particular protein group or assigned as a razor peptide to a single protein group that had the most peptide evidence. The protein groups assembled by ProteinProphet (<div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d52364298e4016" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">Nesvizhskii et al., 2003</a><div data-db-target-of="bib101-d52364298e4016" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nesvizhskii A.I. </li><li class="loa__item">Keller A. </li><li class="loa__item">Kolker E. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">A statistical model for identifying proteins by tandem mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3267)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were filtered to 1% protein-level False Discovery Rate (FDR) using the chosen FDR target-decoy strategy and the best peptide approach (allowing both unique and razor peptides) and applying the picked FDR strategy (<div class="dropBlock reference-citations"><a href="#bib115" id="back-bib115" data-db-target-for="bib115-d52364298e4019" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib115">Savitski et al., 2015</a><div data-db-target-of="bib115-d52364298e4019" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Savitski M.M. </li><li class="loa__item">Wilhelm M. </li><li class="loa__item">Hahne H. </li><li class="loa__item">Kuster B. </li><li class="loa__item">Bantscheff M. </li></ul><div class="reference-citations__detials">A scalable approach for protein false discovery rate estimation in large proteomic data sets.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (147)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+scalable+approach+for+protein+false+discovery+rate+estimation+in+large+proteomic+data+sets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In each TMT 11-plex, the PSM lists were filtered using a stringent, sequential FDR strategy, retaining only those PSMs with PeptideProphet probability of 0.9 or higher (which in these data corresponded to less than 1% PSM-level FDR) and mapped to proteins that also passed the global 1% protein-level FDR filter.</div><div class="section-paragraph">For each PSM that passed these filters, MS1 intensity of the corresponding precursor-ion was extracted using the Philosopher label-free quantification module based on the moFF method (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d52364298e4024" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Argentini et al., 2016</a><div data-db-target-of="bib6-d52364298e4024" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Argentini A. </li><li class="loa__item">Goeminne L.J.E. </li><li class="loa__item">Verheggen K. </li><li class="loa__item">Hulstaert N. </li><li class="loa__item">Staes A. </li><li class="loa__item">Clement L. </li><li class="loa__item">Martens L. </li></ul><div class="reference-citations__detials">moFF: a robust and automated approach to extract peptide ion intensities.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">964-966</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.4075&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27898063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84999277470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=964-966&journal=Nat.+Methods&author=Argentini+A.&author=Goeminne+L.J.E.&author=Verheggen+K.&author=Hulstaert+N.&author=Staes+A.&author=Clement+L.&author=Martens+L.&title=moFF%3A+a+robust+and+automated+approach+to+extract+peptide+ion+intensities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (using 10 p.p.m mass tolerance and 0.4 min retention time window for extracted ion chromatogram peak tracing). In addition, for all PSMs corresponding to a TMT-labeled peptide, eleven TMT reporter ion intensities were extracted from the MS/MS scans (using 0.002 Da window) and the precursor ion purity scores were calculated using the intensity of the sequenced precursor ion and that of other interfering ions observed in MS1 data (within a 0.7 Da isolation window). All supporting information for each PSM, including the accession numbers and names of the protein/gene selected based on the protein inference approach with razor peptide assignment and quantification information (MS1 precursor-ion intensity and the TMT reporter ion intensities) was summarized in the output PSM.tsv files, one file for each TMT 11-plex experiment.</div><div class="section-paragraph">To generate summary reports in different levels (gene, peptide, and protein for global and phosphopeptide enriched data; additional modification site report for phosphopeptide data), we processed the PSM.tsv files using TMT-Integrator. Each PSM in a PSM.tsv file passing the following criteria were kept for creating integrated reports, including (1) having TMT label, (2) having intensity in the reference channel, (2) precursor-ion purity above 50%, (3) summed reported ion intensity (across all channels) not in the lower 5% of all PSMs (2.5% for phosphopeptide enriched data), (4) peptide with phosphorylation (for phosphopeptide enriched data). For a peptide with redundant PSMs, only the PSM with the highest summed TMT intensity was kept for later analysis. PSMs mapping to common external contaminant proteins were excluded, and unique and razor peptides were both used for quantification. Next, the report ion intensities of each PSM were log2 transformed and normalized by the reference channel intensity (i.e., subtracted log2 reference intensity from those log2 report ion intensities), therefore the intensities were converted into log2-based ratio (denoted as ‘ratios’ in the following paragraphs). After converting the intensities to ratios, the PSMs were grouped based on the predefined level (i.e., gene, protein, peptide, and phosphopeptide-level). The interquartile range (IQR) algorithm was then applied to remove the outliers in each PSM group and the remaining ratios were normalized using the median centering. Finally, the normalized ratios were converted back to abundances using the weighted sum of the MS1 intensities of the top three most intense peptide ions. The details of the IQR, median centering, and abundance conversion algorithms are described in detail in (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d52364298e4029" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Clark et al., 2019</a><div data-db-target-of="bib31-d52364298e4029" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Pan J. </li><li class="loa__item">Song X. </li><li class="loa__item">Hu Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Reva B. </li><li class="loa__item">Lih T.-S.M. </li><li class="loa__item">Chang H.-Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated proteogenomic characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e931</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e931&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For phosphosite quantification, a phosphopeptide was assigned to a single protein based on razor assignment. Specifically, each phosphopeptide was assigned to the protein with most evidence when it could be mapped to multiple proteins. If all mapped proteins for a given phosphopeptide are indistinguishable (i.e. the same level of evidence), then one is selected alphabetically. The ratio for each site was calculated from the ratios of all phosphopeptides (including those with multiple phosphorylations) containing this site by median. Then the ratios of sites were converted to intensities using the way similar to that for protein quantification.</div></section></section><section id="sec5.4.8" data-locator="sec5.4.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data harmonization</span></h4><section id="sec5.4.8.1" data-locator="sec5.4.8.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA data</span></h4><div class="section-paragraph">Gene-level mRNA data (RSEM) were upper-quantile normalized and log2 transformed. Genes with missing values (i.e., raw RSEM equal to zero) in less than half of the samples were regarded as quantifiable (N = 29020) and used for downstream analyses (e.g., protein-RNA correlation). Another version of the mRNA data, which were represented as FPKM, was used in certain analyses that benefit from normalization by gene length (specified in the Integrated Analysis section).</div></section><section id="sec5.4.8.2" data-locator="sec5.4.8.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Methylation data</span></h4><div class="section-paragraph">The probe-level methylation data (represented as beta-values) were aggregated to gene-level by extracting probes located in the CpG island of the promoter region of a given gene. The mean beta-value of these probes was used to represent gene-level methylation.</div></section><section id="sec5.4.8.3" data-locator="sec5.4.8.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomics data</span></h4><div class="section-paragraph">Gene-level proteomics data, which were represented as normalized MS intensity, were log2 transformed. Proteins with missing values in less than half of the samples were regarded as quantifiable (N = 9657) and used for downstream analyses.</div></section><section id="sec5.4.8.4" data-locator="sec5.4.8.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomics data</span></h4><div class="section-paragraph">Peptide-level, single site-level, and gene-level phosphoproteomics data, which were represented as normalized MS intensity, were log2 transformed. The peptide-level data contained both localized and non-localized phosphorylation events and the peptides could harbor single, double, or triple phosphorylations. This was utilized by the quality control analyses and a localized version was used for multi-omics subtyping (further described in <em>Unsupervised Subtyping Using Non-negative Matrix Factorization (NMF)</em>). Site-level phosphorylation data consisted of single confidently localized sites and were used for the majority of analyses. Gene-level data were used for survival association. Phosphopeptides, phosphosites, and genes with missing values in less than half of the samples were regarded as quantifiable unless otherwise described.</div></section><section id="sec5.4.8.5" data-locator="sec5.4.8.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Handling of missing values in proteomics and phosphoproteomics data</span></h4><div class="section-paragraph">The missing values were handled in a few different ways for different analyses. 1) For PCA analysis, because it does not allow missing values, we removed any proteins and phosphopeptides missed in more than 50% of the samples and then did missing value imputation using the K nearest neighbors (KNN) algorithm for the remaining proteins and phosphopeptides. 2) For differential abundance analysis across tumors, non-missing values were required for 50% of the samples for each protein and phosphopeptide. 3) In the tumor versus normal analysis, multiplex bias is mitigated by the requirement that the protein or phosphopeptide needed to be quantified in both the tumor and paired normal sample for a patient in the paired statistical test. The paired samples for each patient were assigned to the same TMT plex by experimental design. 4) For the multi-omics subtyping analysis, all features with missing values were excluded.</div></section></section><section id="sec5.4.9" data-locator="sec5.4.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data quality control</span></h4><section id="sec5.4.9.1" data-locator="sec5.4.9.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quality control of proteomic and phosphoproteomic data generation</span></h4><div class="section-paragraph">During the mass spectrometric analysis of global peptides and phosphopeptides, we established a quality control procedure to assess the data quality and instrument performance. For quality control of the HNSCC proteomic and phosphoproteomic raw data, we processed the raw data files through the MS-PyCloud proteomics pipeline (<div class="dropBlock reference-citations"><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d52364298e4070" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Chen et al., 2018</a><div data-db-target-of="bib25-d52364298e4070" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen L. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Schnaubelt M. </li><li class="loa__item">Shah P. </li><li class="loa__item">Aiyetan P. </li><li class="loa__item">Chan D. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Zhang Z. </li></ul><div class="reference-citations__detials">MS-PyCloud: an open-source, cloud computing-based pipeline for LC-MS/MS data analysis.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">320887</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=320887&journal=bioRxiv&author=Chen+L.&author=Zhang+B.&author=Schnaubelt+M.&author=Shah+P.&author=Aiyetan+P.&author=Chan+D.&author=Zhang+H.&author=Zhang+Z.&title=MS-PyCloud%3A+an+open-source%2C+cloud+computing-based+pipeline+for+LC-MS%2FMS+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to generate a quality control table. The quality control table includes the number of PSMs, peptides, and proteins identified in each LC-MS/MS analysis and in each TMT set. Other reported quality control parameters include the total MS2 count; percent of peptides identified in 1, 2, 3, or 4+ charge states; and the minimum and maximum TMT channel intensity ratios relative to the median channel intensity for each TMT set, as well as the injection time for MS/MS analysis.</div><div class="section-paragraph">We calculated the number of unique and shared peptides/proteins in each fraction of a TMT set. A peptide/protein was considered shared if it was identified in more than one fraction. Each fraction contributed unique peptides/proteins relative to the other fractions. This ensured that no fraction was a duplicate of another fraction and each fraction generated quality data.</div></section><section id="sec5.4.9.2" data-locator="sec5.4.9.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Correlation-based checking for potential sample mislabeling</span></h4><div class="section-paragraph">In a large-scale multi-omics study, it is critical to validate the sample labels in experiments as well as in data analysis to avoid mislabeling. The cross-omics correlation (e.g., RNA and protein) of the same sample is usually highest among the correlation values of different samples. Thus, after calculating all pairs of sample correlation values among different omics levels, we could determine whether the highest correlated pairs between two levels have the same sample labels. Here, we applied the sample correlation method implemented in OmicsOne (<a class="inter-ref-link" href="https://github.com/huizhanglab-jhu/OmicsOne" target="_blank">https://github.com/huizhanglab-jhu/OmicsOne</a>) to the abundance matrices of RNA-seq, proteomics, and phosphoproteomics data to check the sample labels of all 110 samples that passed pathological quality control. The three normalized gene level expression matrices of RNA-seq, proteomics, and phosphoproteomics data were firstly z-score transformed for each gene across all samples. In correlating RNA-seq with proteomics data, only the top 500 most correlated genes were considered for sample correlation based on the gene-wise correlation of the two levels. All pair-wise correlation values were calculated using Spearman’s rank correlation. All the samples were highly correlated with samples having the identical label, and 96% of them were the best ranked correlations. We also did the sample correlation tests on “RNA-seq and phosphoproteomics” and “proteomics and phosphoproteomics” comparisons. Based on all the correlation results, there were no mislabeled samples observed in the RNA-seq, proteomics, and phosphoproteomics data sets.</div></section><section id="sec5.4.9.3" data-locator="sec5.4.9.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Machine learning-based checking for potential sample mislabeling</span></h4><div class="section-paragraph">We use our omics data to build machine learning models to detect possible sample mislabeling. Specifically, we built Random Forest classification models using molecular data (RNA-seq, protein) as features to predict patient gender and disease status (tumor vs. normal). When predicting the gender, we only used known sex genes as input features. Leave-one-out cross-validation was used to predict the labels (gender, disease status respectively) for each sample. Samples with low predicted probability (&lt;0.15) for the provided class for either gender or disease status are flagged for manual check. Following this procedure, two samples were identified for potential gender mislabeling. These two samples are a pair of tumor and NAT from the same patient. This was later confirmed as a data entry error for the patient and was corrected in the released data. Additionally, three NAT samples (C3L-01138-N, C3N-03612-N, and C3N-02275-N) and one tumor sample (C3N-01643-T) were predicted as tumor and normal samples, respectively, by both mRNA and protein models. These results were consistent with the unsupervised PCA analysis (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>B and 1C). These samples with questionable tumor/NAT identity were confirmed by follow-up pathological inspection and then removed from all downstream analyses.</div></section><section id="sec5.4.9.4" data-locator="sec5.4.9.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Function prediction based on gene co-expression</span></h4><div class="section-paragraph">Co-expression network construction using mRNA and protein expression data and network-based gene function prediction for KEGG pathways were performed as previously described (<div class="dropBlock reference-citations"><a href="#bib139" id="back-bib139" data-db-target-for="bib139-d52364298e4096" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib139">Wang et al., 2017</a><div data-db-target-of="bib139-d52364298e4096" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Ma Z. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Ellis M.J.C. </li><li class="loa__item">Townsend R.R. </li><li class="loa__item">Smith R.D. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteome profiling outperforms transcriptome profiling for coexpression based gene function prediction.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">121-134</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27836980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85008490068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=121-134&journal=Mol.+Cell+Proteomics&author=Wang+J.&author=Ma+Z.&author=Carr+S.A.&author=Mertins+P.&author=Zhang+H.&author=Zhang+Z.&author=Chan+D.W.&author=Ellis+M.J.C.&author=Townsend+R.R.&author=Smith+R.D.&title=Proteome+profiling+outperforms+transcriptome+profiling+for+coexpression+based+gene+function+prediction">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using OmicsEV (<a class="inter-ref-link" href="https://github.com/bzhanglab/OmicsEV" target="_blank">https://github.com/bzhanglab/OmicsEV</a>).</div></section><section id="sec5.4.9.5" data-locator="sec5.4.9.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data quality evaluation using quality control samples</span></h4><div class="section-paragraph">As one of our quality control (QC) strategies, we included the reference sample (i.e., pooled HNSCC samples) in a non-reference channel in three different TMTs. For these QC samples, the R-squared value between each pair of replicate samples was calculated on log2 transformed protein expression data at gene level or phosphopeptide data. Then the R-squared values were used to evaluate the quality (reproducibility) of the data. In addition, a new quantification strategy was used to process both global proteomics and phosphoproteomics data, in which a virtual reference channel was built for each TMT experiment to serve as “common reference” rather than using the reference sample in the reference channel. The virtual reference was computed by taking the average of all channels in each TMT experiment. In this case, all the reference samples from the reference channels could be used as QC samples and they were treated equally as the samples in non-reference channels during the quantification. Then the R-squared values between each pair of the reference samples were calculated using the same way to evaluate the quality (reproducibility) of the data.</div></section><section id="sec5.4.9.6" data-locator="sec5.4.9.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Batch effect evaluation</span></h4><div class="section-paragraph">The batch effect was evaluated for both global proteome and phosphoproteome data using principal component regression analysis (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d52364298e4112" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Büttner et al., 2019</a><div data-db-target-of="bib20-d52364298e4112" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Büttner M. </li><li class="loa__item">Miao Z. </li><li class="loa__item">Wolf F.A. </li><li class="loa__item">Teichmann S.A. </li><li class="loa__item">Theis F.J. </li></ul><div class="reference-citations__detials">A test metric for assessing single-cell RNA-seq batch correction.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">43-49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41592-018-0254-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30573817&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85058910761&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=43-49&journal=Nat.+Methods&author=B%C3%BCttner+M.&author=Miao+Z.&author=Wolf+F.A.&author=Teichmann+S.A.&author=Theis+F.J.&title=A+test+metric+for+assessing+single-cell+RNA-seq+batch+correction">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, we first removed any proteins missing in more than 50% samples followed by missing value imputation using KNN before performing PCA analysis. Next, for each PC, Pearson’s correlation coefficient with batch covariate was calculated and the significance of the correlation coefficient was estimated using one-way ANOVA. A p value less than 0.05 is considered significant. The analysis was performed using OmicsEV (<a class="inter-ref-link" href="https://github.com/bzhanglab/OmicsEV" target="_blank">https://github.com/bzhanglab/OmicsEV</a>).</div></section></section><section id="sec5.4.10" data-locator="sec5.4.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immunohistochemistry (IHC)</span></h4><section id="sec5.4.10.1" data-locator="sec5.4.10.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">IHC analysis of CD3 protein</span></h4><div class="section-paragraph">Cut tissue sections (5 μm) on charged glass slides were baked for 10-12 hours at 58°C in a dry slide incubator, deparaffinized in xylene and rehydrated via an ethanol step gradient. Heat-induced antigen retrieval steps were performed at pH 9.0, with the primary antibody incubated at room temperature for 1 hour (CD3, polyclonal, Dako, 1:100) followed by standard chromogenic staining protocol with the Envision Polymer-HRP/3,3’diaminobenzidine (DAB, Dako) process. Slides were counterstained in Harris hematoxylin. Immunohistochemistry scoring was performed using the percentage of stromal CD3-positive tumor infiltrating lymphocytes (TILS). All IHC results were evaluated against positive and negative tissue controls.</div></section></section><section id="sec5.4.11" data-locator="sec5.4.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data-independent acquisition (DIA) analysis</span></h4><section id="sec5.4.11.1" data-locator="sec5.4.11.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">ESI-LC-MS/MS</span></h4><div class="section-paragraph">Unlabeled, digested peptide material from individual tissue samples (HNSCC and NAT) was spiked with index Retention Time (iRT) peptides (Biognosys) and subjected to data-independent acquisition (DIA) analysis. Peptides (∼ 0.8 μg) were separated on an Easy nLC 1200 UHPLC system (Thermo Scientific) on an in-house packed 20 cm x 75 μm diameter C18 column (1.9 μm Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10 μm opening (New Objective)). The column was heated to 50°C using a column heater (Phoenix-ST). The flow rate was 0.200 μl/min with 0.1% formic acid and 3% acetonitrile in water (A) and 0.1% formic acid, 90% acetonitrile (B). The peptides were separated with a 7–30% B gradient in 84 mins and analyzed using the Thermo Fusion Lumos mass spectrometer (Thermo Scientific). The DIA segment consisted of one MS1 scan (350-1650 m/z range, 120K resolution) followed by 30 MS2 scans (variable m/z range, 30K resolution). Additional parameters were as follows: MS1: RF Lens – 30%, AGC Target 4.0e<sup>5</sup>, Max IT – 50 ms, charge state include - 2-6; MS2: isolation width (m/z) – 0.7, AGC Target – 3.0e<sup>6</sup>, Max IT – 120 ms.</div></section><section id="sec5.4.11.2" data-locator="sec5.4.11.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Spectral library generation</span></h4><div class="section-paragraph">For spectral library generation, an aliquot (2 μg) of unlabeled, digested peptide material from individual tissue samples (HNSCC and NAT) was pooled and subjected to bRPLC as previously described. Collected fractions were concatenated into eight fractions by combining twelve fractions that are eight fractions apart (i.e., combining fractions #1, #9, #17, #25, #33, #41, #49, #57, #65, #73, #81, and #89; #2, #10, #18, #26, #34, #42, #50, #58, #66, #74, #82, and #90; and so on); dried down in a Speed-Vac, resuspended in 3% ACN, 0.1% formic acid, and was spiked with index Retention Time (iRT) peptides (Biognosys) prior to ESI-LC-MS/MS analysis. Parameters were the same as previously described for ESI-LC-MS/MS with a high-energy collision dissociation activation energy (HCD) – 34.</div></section><section id="sec5.4.11.3" data-locator="sec5.4.11.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein database searching and quantification</span></h4><div class="section-paragraph">The spectral library file was generated based on PulSar search engine (SpectroNaut 13, Biognosys) against combined search results derived from DDA (n = 8) and DIA (n=203). In brief, the DDA files were searched with BGS factory search setting (default) against human database (uniprot released; 20,380 entries). MS1 and MS2 tolerance was set as “dynamic” and only tryptic peptides were allowed with two missed cleavages. Allowable modification on amino acid set to: oxidation (M, variable), acetylation (protein N-terminus, variable), and carbamidomethylation (C, fixed). The DIA files were searched using the PulSar search engine based on DIA umpire algorithm (<div class="dropBlock reference-citations"><a href="#bib131" id="back-bib131" data-db-target-for="bib131-d52364298e4151" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib131">Tsou et al., 2015</a><div data-db-target-of="bib131-d52364298e4151" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tsou C.-C. </li><li class="loa__item">Avtonomov D. </li><li class="loa__item">Larsen B. </li><li class="loa__item">Tucholska M. </li><li class="loa__item">Choi H. </li><li class="loa__item">Gingras A.-C. </li><li class="loa__item">Nesvizhskii A.I. </li></ul><div class="reference-citations__detials">DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span> (<span class="refComment">257 p following 264</span>)</span>: <span class="ref__seriesPages">258-264</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.3255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25599550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923838580&amp;cf=" onclick="newWindow(this.href);return false">Scopus (265)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=258-264&journal=Nat.+Methods&author=Tsou+C.-C.&author=Avtonomov+D.&author=Larsen+B.&author=Tucholska+M.&author=Choi+H.&author=Gingras+A.-C.&author=Nesvizhskii+A.I.&title=DIA-Umpire%3A+comprehensive+computational+framework+for+data-independent+acquisition+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with default parameter as like DDA. To get homogenous protein inference and protein/peptide FDR of 1% in the integrated library, “generate library from search achieve” option was used on SpectroNaut. The spectral library precursor filter was set as follows: amino acid length &gt; 3, relative intensity &gt; 5%, m/z range between 300-1800, and best N fragment ions per peptide = 3∼6. The retention times of these filtered PSMs were further transformed to indexed retention time (iRT) scale based on the standard peptides (iRT peptide kit, Biognosys) spiked into samples.</div><div class="section-paragraph">The integrated spectral library was loaded onto SpectroNaut (version13), and then targeted quantification was performed using default settings, as described in the previous report (<div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d52364298e4156" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Bruderer et al., 2017</a><div data-db-target-of="bib17-d52364298e4156" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bruderer R. </li><li class="loa__item">Bernhardt O.M. </li><li class="loa__item">Gandhi T. </li><li class="loa__item">Xuan Y. </li><li class="loa__item">Sondermann J. </li><li class="loa__item">Schmidt M. </li><li class="loa__item">Gomez-Varela D. </li><li class="loa__item">Reiter L. </li></ul><div class="reference-citations__detials">Optimization of experimental parameters in data-independent mass spectrometry significantly increases depth and reproducibility of results.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">2296-2309</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29070702&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85038848913&amp;cf=" onclick="newWindow(this.href);return false">Scopus (102)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=2296-2309&journal=Mol.+Cell.+Proteomics&author=Bruderer+R.&author=Bernhardt+O.M.&author=Gandhi+T.&author=Xuan+Y.&author=Sondermann+J.&author=Schmidt+M.&author=Gomez-Varela+D.&author=Reiter+L.&title=Optimization+of+experimental+parameters+in+data-independent+mass+spectrometry+significantly+increases+depth+and+reproducibility+of+results">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The retention time of a peptide feature was transformed into iRT scale using the “local regression” option and the extracted ion chromatogram (XIC) retention time window was set to “dynamic” with 1 for the correction factor. The identified precursors and proteins were filtered at 1% of q-value was considered for the identification of precursor and protein (<div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d52364298e4159" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Reiter et al., 2011</a><div data-db-target-of="bib108-d52364298e4159" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reiter L. </li><li class="loa__item">Rinner O. </li><li class="loa__item">Picotti P. </li><li class="loa__item">Hüttenhain R. </li><li class="loa__item">Beck M. </li><li class="loa__item">Brusniak M.-Y. </li><li class="loa__item">Hengartner M.O. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">mProphet: automated data processing and statistical validation for large-scale SRM experiments.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">430-435</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.1584&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21423193&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79955595074&amp;cf=" onclick="newWindow(this.href);return false">Scopus (293)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=430-435&journal=Nat.+Methods&author=Reiter+L.&author=Rinner+O.&author=Picotti+P.&author=H%C3%BCttenhain+R.&author=Beck+M.&author=Brusniak+M.-Y.&author=Hengartner+M.O.&author=Aebersold+R.&title=mProphet%3A+automated+data+processing+and+statistical+validation+for+large-scale+SRM+experiments">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Interference correction on corrections at MS1 and MS2 level was levels were enabled, removing fragments/isotopes from quantification based on the presence of interfering signals but keeping at least three for quantification. All abundances were calculated based on the area under the extracted ion chromatogram (XIC) of all assigned fragments that passed filtering.</div></section></section><section id="sec5.4.12" data-locator="sec5.4.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Integrated analysis</span></h4><section id="sec5.4.12.1" data-locator="sec5.4.12.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein-RNA correlation</span></h4><div class="section-paragraph">The gene-wise protein-RNA correlation for all genes quantifiable in both omics data types (N = 9579) was computed using Spearman’s correlation. The correlation significance was set at Benjamini-Hochberg adjusted p value &lt; 0.01. Signed -log10 p value was used as the ranking metric for GSEA analysis using Webgestalt (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d52364298e4170" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Liao et al., 2019</a><div data-db-target-of="bib83-d52364298e4170" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to identify GO biological processes and KEGG pathways enriched for genes with low and high protein-RNA correlations, respectively.</div></section><section id="sec5.4.12.2" data-locator="sec5.4.12.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Estimate score</span></h4><div class="section-paragraph">The ESTIMATE scores reflecting the overall immune and stromal infiltration were calculated by the R package ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib149" id="back-bib149" data-db-target-for="bib149-d52364298e4178" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Yoshihara et al., 2013</a><div data-db-target-of="bib149-d52364298e4178" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1032)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using the normalized RNA expression data (RSEM).</div></section><section id="sec5.4.12.3" data-locator="sec5.4.12.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differential abundance analysis</span></h4><div class="section-paragraph">Differential analysis was performed for paired tumor and NAT samples using the Wilcoxon signed-rank test. Each feature was required to be non-missing in at least 50% of the paired samples. P values were adjusted using the Benjamini-Hochberg method and features were considered significant with an adjusted p value &lt; 0.01. Proteomic and transcriptomic features with at least a median 2-fold increase in tumors were considered to be tumor-associated proteins. Over-representation analysis with the GO Biological Process (BP) gene sets was performed separately for proteins either increased or decreased &gt;2-fold using WebGestaltR (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d52364298e4186" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Liao et al., 2019</a><div data-db-target-of="bib83-d52364298e4186" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The reference set was the proteins identified in at least 50% of the paired tumor and NAT samples. Over-representation analysis with the GO Biological Process (BP) gene sets was also performed separately for proteins with at least one phosphosite either increased or decreased &gt;2-fold. The reference set was the proteins with an identified phosphosite in at least 50% of the paired tumor and NAT samples. GO BP terms were considered significant with a Benjamini-Hochberg (BH) adjusted p value &lt; 0.01.</div><div class="section-paragraph">A linear mixed model (<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d52364298e4191" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Kuznetsova et al., 2017</a><div data-db-target-of="bib73-d52364298e4191" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuznetsova A. </li><li class="loa__item">Brockhoff P.B. </li><li class="loa__item">Christensen R.H.B. </li></ul><div class="reference-citations__detials">lmerTest package: tests in linear mixed effects models.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Softw.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">1-26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.18637%2Fjss.v082.i13&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2017&pages=1-26&journal=J.%C2%A0Stat.+Softw.&author=Kuznetsova+A.&author=Brockhoff+P.B.&author=Christensen+R.H.B.&title=lmerTest+package%3A+tests+in+linear+mixed+effects+models">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to identify differential proteins while controlling for non-epithelial content difference. Specifically, the z score-transformed ESTIMATE score (i.e., the sum of the ESTIMATE immune score and stromal score) was used as a fixed effect. Proteins with a significant (Benjamini-Hochberg adjusted p &lt; 0.01) coefficient for tumor vs NAT expression change that remained greater than 1 after accounting for non-epithelial content were retained as tumor-associated proteins. Proteins were annotated as C/T Antigens (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d52364298e4194" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Almeida et al., 2009</a><div data-db-target-of="bib2-d52364298e4194" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.-T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (213)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), secretable (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d52364298e4197" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Uhlén et al., 2015</a><div data-db-target-of="bib132-d52364298e4197" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Uhlén M. </li><li class="loa__item">Fagerberg L. </li><li class="loa__item">Hallström B.M. </li><li class="loa__item">Lindskog C. </li><li class="loa__item">Oksvold P. </li><li class="loa__item">Mardinoglu A. </li><li class="loa__item">Sivertsson Å. </li><li class="loa__item">Kampf C. </li><li class="loa__item">Sjöstedt E. </li><li class="loa__item">Asplund A. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteomics. Tissue-based map of the human proteome.<div class="popup-ref"><span class="ref__series"><i>Science (New York, NY).</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">347</span></span>: <span class="ref__seriesPages">1260419</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscience.1260419&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84920269464&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3889)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1260419&journal=Science+%28New+York%2C+NY%29&author=Uhl%C3%A9n+M.&author=Fagerberg+L.&author=Hallstr%C3%B6m+B.M.&author=Lindskog+C.&author=Oksvold+P.&author=Mardinoglu+A.&author=Sivertsson+%C3%85.&author=Kampf+C.&author=Sj%C3%B6stedt+E.&author=Asplund+A.&title=Proteomics.+Tissue-based+map+of+the+human+proteome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and as the targets of FDA approved drugs (Human Protein Atlas, DrugBank). Phosphosites were annotated with the “ON_FUNCTION” and “ON_PROCESS” from PhosphoSitePlus.</div><div class="section-paragraph">Differential analysis was repeated for samples within a subgroup (e.g. larynx tumors, oral cavity tumors). Proteins were required to be non-missing in at least 50% of the paired tumor and NAT samples within the subgroup. Proteins increased 2-fold in one group but not the other were considered specific markers for that group.</div></section><section id="sec5.4.12.4" data-locator="sec5.4.12.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kinase activity inference</span></h4><div class="section-paragraph">Activated kinases were first inferred from significantly increased phosphorylation of their sites annotated as ‘enzymatic activity, induced’ from PhosphoSitePlus. We also required these sites to have a median phosphorylation fold change greater than the median protein fold change in tumor compared to paired NAT. Kinase activity was additionally inferred from the phosphorylation of its substrates. Pre-ranked GSEA in WebGestaltR was performed on the signed -log10 p values from the differential abundance analysis using site-level substrates annotated in PhosphoSitePlus and UniProt. At least 10 substrates were required for each kinase. Kinases were considered significantly activated with a Benjamini-Hochberg adjusted p value &lt; 0.05.</div></section><section id="sec5.4.12.5" data-locator="sec5.4.12.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Prediction of kinase substrates</span></h4><div class="section-paragraph">NetworKIN (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d52364298e4212" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Linding et al., 2007</a><div data-db-target-of="bib86-d52364298e4212" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linding R. </li><li class="loa__item">Jensen L.J. </li><li class="loa__item">Ostheimer G.J. </li><li class="loa__item">van Vugt M.A.T.M. </li><li class="loa__item">Jørgensen C. </li><li class="loa__item">Miron I.M. </li><li class="loa__item">Diella F. </li><li class="loa__item">Colwill K. </li><li class="loa__item">Taylor L. </li><li class="loa__item">Elder K. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic discovery of in vivo phosphorylation networks.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">129</span></span>: <span class="ref__seriesPages">1415-1426</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17570479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34250743173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (553)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=1415-1426&journal=Cell&author=Linding+R.&author=Jensen+L.J.&author=Ostheimer+G.J.&author=van+Vugt+M.A.T.M.&author=J%C3%B8rgensen+C.&author=Miron+I.M.&author=Diella+F.&author=Colwill+K.&author=Taylor+L.&author=Elder+K.&title=Systematic+discovery+of+in%C2%A0vivo+phosphorylation+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to predict kinases for every identified phosphosite in the phosphoproteomics data. Substrate sets were generated using the combined set of known substrates from PhosphoSitePlus and UniProt used in Kinase Activity Inference and the predicted substrates from NetworKIN with a NetworKIN score ≥ 5. Kinase Activity Inference was performed as above with the new combined set of substrates.</div></section><section id="sec5.4.12.6" data-locator="sec5.4.12.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Transcription factor activity inference</span></h4><div class="section-paragraph">Transcription factor activity for each sample was inferred using the VIPER package (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d52364298e4220" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Alvarez et al., 2016</a><div data-db-target-of="bib3-d52364298e4220" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alvarez M.J. </li><li class="loa__item">Shen Y. </li><li class="loa__item">Giorgi F.M. </li><li class="loa__item">Lachmann A. </li><li class="loa__item">Ding B.B. </li><li class="loa__item">Ye B.H. </li><li class="loa__item">Califano A. </li></ul><div class="reference-citations__detials">Functional characterization of somatic mutations in cancer using network-based inference of protein activity.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">838-847</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3593&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27322546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975246805&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=838-847&journal=Nat.+Genet.&author=Alvarez+M.J.&author=Shen+Y.&author=Giorgi+F.M.&author=Lachmann+A.&author=Ding+B.B.&author=Ye+B.H.&author=Califano+A.&title=Functional+characterization+of+somatic+mutations+in+cancer+using+network-based+inference+of+protein+activity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on z-score transformed RNA data. The transcription factor targets were collected from DoRothEA (<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d52364298e4223" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Garcia-Alonso et al., 2019</a><div data-db-target-of="bib48-d52364298e4223" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Garcia-Alonso L. </li><li class="loa__item">Holland C.H. </li><li class="loa__item">Ibrahim M.M. </li><li class="loa__item">Turei D. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Benchmark and integration of resources for the estimation of human transcription factor activities.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">1363-1375</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.240663.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31340985&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071059943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1363-1375&journal=Genome+Res.&author=Garcia-Alonso+L.&author=Holland+C.H.&author=Ibrahim+M.M.&author=Turei+D.&author=Saez-Rodriguez+J.&title=Benchmark+and+integration+of+resources+for+the+estimation+of+human+transcription+factor+activities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and the medium confidence targets were used for analysis. Activity scores for tumor and normal samples were compared using Student’s <em>t</em>-test and the p values were adjusted using the Benjamini-Hochberg method. Transcription factors with an adjusted p value &lt; 0.05 were considered significant.</div><div class="section-paragraph">We also used the ARACNe algorithm (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d52364298e4230" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Lachmann et al., 2016</a><div data-db-target-of="bib74-d52364298e4230" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lachmann A. </li><li class="loa__item">Giorgi F.M. </li><li class="loa__item">Lopez G. </li><li class="loa__item">Califano A. </li></ul><div class="reference-citations__detials">ARACNe-AP: gene network reverse engineering through adaptive partitioning inference of mutual information.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2233</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtw216&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27153652&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84992345426&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2233&journal=Bioinformatics&author=Lachmann+A.&author=Giorgi+F.M.&author=Lopez+G.&author=Califano+A.&title=ARACNe-AP%3A+gene+network+reverse+engineering+through+adaptive+partitioning+inference+of+mutual+information">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) in VIPER to construct gene regulatory networks and infer transcription factor targets based on correlation to the transcription factor protein abundance. This allows for cancer-specific transcription factor gene regulation. We correlated the normalized enrichment protein activity scores with immune scores.</div></section><section id="sec5.4.12.7" data-locator="sec5.4.12.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Stemness score inference</span></h4><div class="section-paragraph">Stemness scores were calculated as previously described (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d52364298e4238" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Malta et al., 2018</a><div data-db-target-of="bib90-d52364298e4238" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kamińska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials">Machine learning identifies stemness features associated with oncogenic dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e315</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (245)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e315&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kami%C5%84ska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+learning+identifies+stemness+features+associated+with+oncogenic+dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We used MoonlightR (<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d52364298e4241" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Colaprico et al., 2020</a><div data-db-target-of="bib33-d52364298e4241" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to query, download, and preprocess the pluripotent stem cell samples (ESC and iPSC) from the Progenitor Cell Biology Consortium (PCBC) dataset (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d52364298e4244" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Daily et al., 2017</a><div data-db-target-of="bib36-d52364298e4244" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Daily K. </li><li class="loa__item">Ho Sui S.J. </li><li class="loa__item">Schriml L.M. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Salomonis N. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li class="loa__item">Lotia S. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.<div class="popup-ref"><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib111" id="back-bib111" data-db-target-for="bib111-d52364298e4247" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Salomonis et al., 2016</a><div data-db-target-of="bib111-d52364298e4247" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Salomonis N. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Huo J. </li><li class="loa__item">Schriml L. </li><li class="loa__item">Ho Sui S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated genomic analysis of diverse induced pluripotent stem cells from the Progenitor Cell Biology Consortium.<div class="popup-ref"><span class="ref__series"><i>Stem Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Rep.&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+genomic+analysis+of+diverse+induced+pluripotent+stem+cells+from+the+Progenitor+Cell+Biology+Consortium">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To calculate the stemness scores based on mRNA expression, we built a predictive model using one-class logistic regression (OCLR) (<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d52364298e4250" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Sokolov et al., 2016</a><div data-db-target-of="bib122-d52364298e4250" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sokolov A. </li><li class="loa__item">Paull E.O. </li><li class="loa__item">Stuart J.M. </li></ul><div class="reference-citations__detials">One-class detection of cell states in tumor subtypes.<div class="popup-ref"><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-class+detection+of+cell+states+in+tumor+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on the PCBC dataset.</div><div class="section-paragraph">For mRNA expression-based signatures, to ensure compatibility with the CPTAC HNSCC cohort, we first mapped the gene names from Ensembl IDs to Human Genome Organization (HUGO), dropping any genes that had no such mapping. The resulting training matrix contained 12,955 mRNA expression values measured across all available PCBC samples. To calculate mRNA based stemness index (mRNASi) we used the FPKM (Fragments Per Kilobase Million) mRNA expression values for all the 161 HNSCC samples (108 tumor samples and 53 NAT samples).</div><div class="section-paragraph">We used the function TCGAanalyze_Stemness from the package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d52364298e4257" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Colaprico et al., 2016</a><div data-db-target-of="bib34-d52364298e4257" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (578)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and followed our previously-described workflow (<div class="dropBlock reference-citations"><a href="#bib100" id="back-bib100" data-db-target-for="bib100-d52364298e4260" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">Mounir et al., 2019</a><div data-db-target-of="bib100-d52364298e4260" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mounir M. </li><li class="loa__item">Lucchetta M. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Chen X. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Papaleo E. </li></ul><div class="reference-citations__detials">New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">e1006701</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pcbi.1006701&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30835723&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85062948513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=e1006701&journal=PLoS+Comput.+Biol.&author=Mounir+M.&author=Lucchetta+M.&author=Silva+T.C.&author=Olsen+C.&author=Bontempi+G.&author=Chen+X.&author=Noushmehr+H.&author=Colaprico+A.&author=Papaleo+E.&title=New+functionalities+in+the+TCGAbiolinks+package+for+the+study+and+integration+of+cancer+data+from+GDC+and+GTEx">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)) with “stemSig” argument set to PCBC_stemSig.</div></section><section id="sec5.4.12.8" data-locator="sec5.4.12.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">MSigDB hallmark pathway single sample gene set enrichment analysis (ssGSEA)</span></h4><div class="section-paragraph">ssGSEA was performed for each sample using gene-wise Z-scores of log2 upper-quartile normalized RNA-seq data for the MSigDB Hallmark gene sets (<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d52364298e4268" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Liberzon et al., 2015</a><div data-db-target-of="bib84-d52364298e4268" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1573)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) via WebGestaltR (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d52364298e4271" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Liao et al., 2019</a><div data-db-target-of="bib83-d52364298e4271" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For this analysis, read counts of 0 were treated as NAs prior to gene-wise normalization. Pathway activity scores are enrichment scores from ssGSEA.</div></section><section id="sec5.4.12.9" data-locator="sec5.4.12.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-gene proliferation scores (MGPS)</span></h4><div class="section-paragraph">MGPS were calculated as described previously (<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d52364298e4279" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Ellis et al., 2017</a><div data-db-target-of="bib40-d52364298e4279" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 proliferation index as a tool for chemotherapy decisions during and after neoadjuvant aromatase inhibitor treatment of breast cancer: results from the American College of Surgeons Oncology Group Z1031 trial (alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+proliferation+index+as+a+tool+for+chemotherapy+decisions+during+and+after+neoadjuvant+aromatase+inhibitor+treatment+of+breast+cancer%3A+results+from+the+American+College+of+Surgeons+Oncology+Group+Z1031+trial+%28alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, the scores are the mean of gene-wise Z-scores for log2 upper-quartile normalized RSEM data for all cell cycle-regulated genes identified by Whitfield et al. in each sample (<div class="dropBlock reference-citations"><a href="#bib147" id="back-bib147" data-db-target-for="bib147-d52364298e4282" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib147">Whitfield et al., 2002</a><div data-db-target-of="bib147-d52364298e4282" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Whitfield M.L. </li><li class="loa__item">Sherlock G. </li><li class="loa__item">Saldanha A.J. </li><li class="loa__item">Murray J.I. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Alexander K.E. </li><li class="loa__item">Matese J.C. </li><li class="loa__item">Perou C.M. </li><li class="loa__item">Hurt M.M. </li><li class="loa__item">Brown P.O. </li><li class="loa__item">Botstein D. </li></ul><div class="reference-citations__detials">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.<div class="popup-ref"><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1095)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.4.12.10" data-locator="sec5.4.12.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Curation of <em>FAT1</em> and <em>CDKN2A</em> genetic aberration</span></h4><div class="section-paragraph">In order to facilitate a granular analysis between tumor groups with distinct types of molecular loss of the tumor suppressor genes <em>FAT1</em> and <em>CDKN2A</em>, a comprehensive tumor annotation was carried out using multiple molecular features for each patient. For example, for annotating <em>CDKN2A</em>, a gene largely affected by copy number events in HNSCC, we considered the following: 1) mutation types (Missense (including in-frame insertion and deletions (indels)), Truncation (stop gain, frameshift indels) and Splice site mutations) as separate categories. For cases with mutation, we next looked at both variant allele frequency of the mutation as adjudged from whole exome sequencing, estimated tumor purity, and copy number data (log ratio, absolute copy number, and B-allele frequency). Mutated samples with either lower tumor purity (as estimated from WGS) or with subclonal events based on mutation VAF, were excluded. Subsequently, cases with Truncation mutation + one copy loss were annotated as Bi-allelic truncation loss or truncation loss of heterozygosity (Biallelic. Trunc or truncation LOH). Cases with Splicing mutation with associated one copy loss were classified as Bi-allelic splicing or splicing LOH, and finally tumors with Missense mutations with associated one copy loss were annotated as missense LOH. While these tumors were classified as LOH mutants, the remaining mutant copy of <em>CDKN2A</em> frequently showed evidence for copy number amplification of the mutant alleles. 2) Based on <em>CDKN2A</em> copy number data, we next annotated tumors as Homozygous deletion (Homozyg. del.), one copy loss (Heterozyg. del) or no loss (Wild-type). Mutations affecting <em>CDKN2A</em> were assessed separately for both splice isoforms of the gene, p16INK4a and p14ARF.</div></section><section id="sec5.4.12.11" data-locator="sec5.4.12.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Prioritizing putative SCNA drivers</span></h4><div class="section-paragraph">The workflow to prioritize putative SCNA drivers was shown in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>H. First, all the genes with quantifiable copy number, RNA expression, and proteomics (N = 9507) were filtered to keep the focal amplified genes, which were located in the segments identified by GISTIC2 with Q value &lt; 0.25 (N = 759). These focal amplified genes were further filtered by their CN-mRNA correlation and next CN-protein correlation to keep the genes with significant CN <em>cis</em>-effect (BH adjusted p value &lt; 0.01, Spearman’s correlation). Finally, the remaining genes (N = 356) were further filtered to keep the ones with significant higher protein levels in tumors than NATs (BH adjusted p value &lt; 0.01, Student’s <em>t</em>-test), which generated a list of 202 putative SCNA driver genes. These genes were used to perform over-representation analysis against all quantifiable genes in the three omics to identify enriched GO biological processes.</div></section><section id="sec5.4.12.12" data-locator="sec5.4.12.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text"><em>cis</em>-effect of DNA methylations</span></h4><div class="section-paragraph">To study the <em>cis</em>-effect of DNA methylation on mRNA and protein expression, we performed a multivariate correlation analysis that included SCNA and mutation effects as confounding variables using the software iProFun (<div class="dropBlock reference-citations"><a href="#bib123" id="back-bib123" data-db-target-for="bib123-d52364298e4328" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Song et al., 2019</a><div data-db-target-of="bib123-d52364298e4328" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The DNA methylation levels were averaged from the CpG islands located in the upstream and nearby transcription start site (TSS) regions, including 5’UTR, 1st exon, and upstream TSS. Somatic mutations were considered if their mutation rate was &gt;5%. In the analysis, we controlled age, gender, immune score, stromal score, inferred smoking status, and tumor location as covariates.</div><div class="section-paragraph">A significant association is identified if it passes three criteria: (1) biological filtering procedure highlighting significant methylations that are hyper- or hypo-methylated in tumor vs NAT, (2) posterior probabilities of associating to an outcome &gt; 75%, and (3) empirical false discovery rate (eFDR) &lt;10% from 100 permutations.</div></section><section id="sec5.4.12.13" data-locator="sec5.4.12.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text"><em>FAT1</em> mutation and 11q13.3 amplification analysis</span></h4><div class="section-paragraph">TCGA <em>FAT1</em> mutation, CNV, and patient survival data were downloaded from cBioPortal (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d52364298e4341" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Gao et al., 2013</a><div data-db-target-of="bib45-d52364298e4341" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao J. </li><li class="loa__item">Aksoy B.A. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Dresdner G. </li><li class="loa__item">Gross B. </li><li class="loa__item">Sumer S.O. </li><li class="loa__item">Sun Y. </li><li class="loa__item">Jacobsen A. </li><li class="loa__item">Sinha R. </li><li class="loa__item">Larsson E. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal.<div class="popup-ref"><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">pl1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscisignal.2004088&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23550210&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84875740314&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5610)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=pl1&journal=Sci.+Signal.&author=Gao+J.&author=Aksoy+B.A.&author=Dogrusoz+U.&author=Dresdner+G.&author=Gross+B.&author=Sumer+S.O.&author=Sun+Y.&author=Jacobsen+A.&author=Sinha+R.&author=Larsson+E.&title=Integrative+analysis+of+complex+cancer+genomics+and+clinical+profiles+using+the+cBioPortal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). HPV+ samples were excluded from downstream analysis. 11q13.3 amplification was defined by the mean CNV values of the 9 genes in the focal region (mean CNVs &gt; 1). These genes include <em>FAT1</em>, <em>CCND1</em>, <em>LTO1</em>, <em>FGF19</em>, <em>FGF4</em>, <em>FGF3</em>, <em>ANO1</em>, <em>FADD</em>, <em>PPFIA1</em>, and <em>CTTN</em>. Samples with both <em>FAT1</em> truncation mutation and 11q13.3 amplification were randomly assigned to <em>FAT1</em> truncation or 11q13.3 amplification groups. The R package “<em>survival</em>” (<div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d52364298e4372" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Therneau, 2020</a><div data-db-target-of="bib129-d52364298e4372" class="dropBlock__holder"><div class="dropBlock__body"><p>Therneau, T.M. (2020). A Package for Survival Analysis in R. <a class="inter-ref-link" href="https://cran.r-project.org/web/packages/survival/index.html" target="_blank">https://cran.r-project.org/web/packages/survival/index.html</a></p><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Therneau%2C+T.M.+%282020%29.+A+Package+for+Survival+Analysis+in+R.+https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2Fsurvival%2Findex.html">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to compare patient survival between groups.</div></section><section id="sec5.4.12.14" data-locator="sec5.4.12.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Inferred HIPPO, WNT, and apoptotic activity scores</span></h4><div class="section-paragraph">All scores were inferred using protein abundance by the ssGSEA method from the GSVA R package (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d52364298e4380" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Barbie et al., 2009</a><div data-db-target-of="bib8-d52364298e4380" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1148)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d52364298e4383" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Hänzelmann et al., 2013</a><div data-db-target-of="bib53-d52364298e4383" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1108)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The gene sets HIPPO pathway, WNT pathway, and apoptotic were from the KEGG pathway, Reactome pathway, and the MSigDB hallmark gene sets, respectively.</div></section><section id="sec5.4.12.15" data-locator="sec5.4.12.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CDK4/6 pathway analysis</span></h4><div class="section-paragraph"><em>CCND1</em> amplified samples are those with copy-number log2 ratios &gt;= 1 for the <em>CCND1</em> gene. <em>CDKN2A</em> groups were defined as described above (Curation of <em>FAT1</em> and <em>CDKN2A</em> Genetic Aberration). For <em>cis</em> analysis, wild-type samples are those without mutations, amplifications, or deletions of each respective gene, while wild-type samples for downstream analysis are those that are wild-type for both genes. For <em>cis</em> analysis of the effects of <em>CDKN2A</em> aberrations, transcript level log2 RSEM for the p16INK4a and p14ARF isoforms were evaluated separately, considering aberrations that affect the respective isoform. Scores for Hallmark E2F targets (ssGSEA) and multi-gene proliferation (MGPS) were calculated as described above. Rb phosphorylation is the mean of the 5 sites with complete data for all tumors that were annotated as CDK4 or CDK6 target sites in PhosphoSitePlus (pT252, pT356, pT373, pS780, and pS795). For the analysis presented in <a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>D, the first group included tumors that were wild-type for all pathway genes (<em>CDKN2A</em>, <em>CCND1</em>, <em>CDK6</em>, and <em>RB1</em>; heterozygous deletions of <em>CDKN2A</em> were included if the retained allele was WT), while the second group included tumors with loss of expression aberrations affecting only <em>CDKN2A/p16INK4a</em> (<em>CDKN2A</em> homozygous deletion, p16 promoter hypermethylation, and <em>p16</em> truncation LOH mutations), and the third group included tumors with <em>CDKN2A/p16INK4a</em> aberrations in combination with <em>CCND1</em> amplifications.</div></section><section id="sec5.4.12.16" data-locator="sec5.4.12.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">EGFR pathway analysis</span></h4><div class="section-paragraph">We inferred signaling activity of the EGFR pathway based on tumor mRNA expression data using PROGENy (<div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d52364298e4437" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">Schubert et al., 2018</a><div data-db-target-of="bib116-d52364298e4437" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Schubert M. </li><li class="loa__item">Klinger B. </li><li class="loa__item">Klünemann M. </li><li class="loa__item">Sieber A. </li><li class="loa__item">Uhlitz F. </li><li class="loa__item">Sauer S. </li><li class="loa__item">Garnett M.J. </li><li class="loa__item">Blüthgen N. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Perturbation-response genes reveal signaling footprints in cancer gene expression.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which computes pathway activity on the basis of mRNA abundance of a responsive gene set identified from a large compendium of publicly available cell line perturbation experiments. To identify ligand-depend EGFR pathway phosphorylation cascade, three omics measurements (RNA, protein, and site-level phosphorylation) for the curated pathway component genes (<a class="inter-ref-link" href="https://www.wikipathways.org/index.php/Pathway:WP437" target="_blank">https://www.wikipathways.org/index.php/Pathway:WP437</a> and also from relevant literatures (<div class="dropBlock reference-citations"><a href="#bib121" id="back-bib121" data-db-target-for="bib121-d52364298e4443" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib121">Sigismund et al., 2018</a><div data-db-target-of="bib121-d52364298e4443" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sigismund S. </li><li class="loa__item">Avanzato D. </li><li class="loa__item">Lanzetti L. </li></ul><div class="reference-citations__detials">Emerging functions of the EGFR in cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">3-20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2F1878-0261.12155&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29124875&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85035196496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=3-20&journal=Mol.+Oncol.&author=Sigismund+S.&author=Avanzato+D.&author=Lanzetti+L.&title=Emerging+functions+of+the+EGFR+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d52364298e4446" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Wee and Wang, 2017</a><div data-db-target-of="bib144-d52364298e4446" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wee P. </li><li class="loa__item">Wang Z. </li></ul><div class="reference-citations__detials">Epidermal growth factor receptor cell proliferation signaling pathways.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">52</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3390%2Fcancers9050052&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28513565&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85019993185&amp;cf=" onclick="newWindow(this.href);return false">Scopus (377)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=52&journal=Cancers+%28Basel%29&author=Wee+P.&author=Wang+Z.&title=Epidermal+growth+factor+receptor+cell+proliferation+signaling+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were correlated to the average RNA level of five EGFR ligands (i.e., AREG, TGFA, EREG, EPGN, and HBEGF) using Pearson’s correlation. Only the phosphosites with both significant correlation (p &lt; 0.01) and higher correlation coefficient than those from RNA and protein at the gene level were considered as evidence for phosphorylation-level regulation. Similar correlation analysis was performed to examine whether these pathway components were associated with the EGFR receptor protein level.</div><div class="section-paragraph">The pathway activity primarily driven by EGFR-amplification was investigated by comparing the highest six EGFR-amplified samples to the rest of the samples with high CN instability (ChrIdx &gt; 3). Three omics measurements for (RNA, protein, and site-level phosphorylation) of all quantifiable genes/sites were used to perform Student’s <em>t</em>-test between the two groups. Only the phosphosites with both significant difference (p &lt; 0.01) and higher fold-change than those from RNA and protein at gene level were considered as evidence for phosphorylation-level regulation. Over-representation analysis was performed on genes containing these qualified sites against all genes quantifiable in RNA, protein and phosphorylation to identify function enrichment of GO biological processes (BH adjusted p &lt;0.05).</div></section><section id="sec5.4.12.17" data-locator="sec5.4.12.17" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Driver copy number deletions associated with low immune infiltration</span></h4><div class="section-paragraph">The CN log2 ratio, RNA expression, and protein expression of quantifiable genes were correlated to the immune score, which was inferred by ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib149" id="back-bib149" data-db-target-for="bib149-d52364298e4458" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Yoshihara et al., 2013</a><div data-db-target-of="bib149-d52364298e4458" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1032)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For putative driver copy number deletions contributing to immune suppression, we required significant correlation for all three measurements to the immune score (Spearman’s correlation, BH adjusted p &lt; 0.01). Meanwhile, the copy number was also required to have significant <em>cis</em>-effect (i.e., significant correlation from CN to both protein and RNA). SCNA effector genes were defined as those with significant correlation to the immune score at gene expression (both mRNA and protein) but not copy number level, indicating their changes occurred in <em>trans</em>. Over-representation analysis was performed using these qualified genes to identify pathway enrichment (adjusted p &lt; 0.05).</div></section><section id="sec5.4.12.18" data-locator="sec5.4.12.18" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune cell type composition</span></h4><div class="section-paragraph">The abundances of 64 different cell types in 162 HNSCC samples (108 tumor samples and 53 NAT samples) were computed via xCell (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d52364298e4470" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Aran et al., 2017</a><div data-db-target-of="bib5-d52364298e4470" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For this analysis, FPKM (Fragments Per Kilobase Million) mRNA expression values were utilized.</div></section><section id="sec5.4.12.19" data-locator="sec5.4.12.19" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Neoantigen identification</span></h4><div class="section-paragraph">We used NeoFlow (<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d52364298e4478" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Wen et al., 2020</a><div data-db-target-of="bib145-d52364298e4478" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)(<a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a>) for neoantigen prediction. Specifically, Optitype (<div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d52364298e4484" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Szolek et al., 2014</a><div data-db-target-of="bib125-d52364298e4484" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to find human leukocyte antigens (HLA) in the WES data. Then we used netMHCpan 4.0 (<div class="dropBlock reference-citations"><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d52364298e4487" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Jurtz et al., 2017</a><div data-db-target-of="bib58-d52364298e4487" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: improved peptide-MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (338)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+improved+peptide-MHC+class+I+interaction+predictions+integrating+eluted+ligand+and+peptide+binding+affinity+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to predict HLA peptide binding affinity for somatic mutation-derived variant peptides with a length between 8-11 amino acids. The IC50 binding affinity cutoff was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-seq data and MS/MS data, respectively. To identify variant peptides, we used a customized protein sequence database approach (<div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d52364298e4490" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Wang et al., 2012</a><div data-db-target-of="bib142-d52364298e4490" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang X. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Wang D. </li><li class="loa__item">Halvey P.J. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Liebler D.C. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Protein identification using customized protein sequence databases derived from RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (117)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.C.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases+derived+from+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We derived customized protein sequence databases from matched WES data and then performed database searching using the customized databases for individual TMT experiments. We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (<a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a>) for customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at the PSM level. Remaining variant peptides were further filtered using PepQuery (<a class="inter-ref-link" href="http://www.pepquery.org" target="_blank">http://www.pepquery.org</a>) (<div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d52364298e4500" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Wen et al., 2019</a><div data-db-target-of="bib146-d52364298e4500" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with the p-value cutoff &lt;= 0.01. The spectra of variant peptides were annotated using PDV (<a class="inter-ref-link" href="https://github.com/wenbostar/PDV" target="_blank">https://github.com/wenbostar/PDV</a>) (<div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d52364298e4506" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">Li et al., 2019</a><div data-db-target-of="bib81-d52364298e4506" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.4.12.20" data-locator="sec5.4.12.20" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Transcriptomics subtypes</span></h4><div class="section-paragraph">The transcriptomics-based subtyping was performed using the centroid-based method with previously established signature genes (<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d52364298e4514" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Walter et al., 2013</a><div data-db-target-of="bib137-d52364298e4514" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Walter V. </li><li class="loa__item">Yin X. </li><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Zhao N. </li><li class="loa__item">Du Y. </li><li class="loa__item">Ang M.K. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Salazar A.H. </li><li class="loa__item">Hoadley K.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes.<div class="popup-ref"><span class="ref__series"><i>PLoS One.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">e56823</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pone.0056823&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23451093&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874327940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e56823&journal=PLoS+One&author=Walter+V.&author=Yin+X.&author=Wilkerson+M.D.&author=Cabanski+C.R.&author=Zhao+N.&author=Du+Y.&author=Ang+M.K.&author=Hayward+M.C.&author=Salazar+A.H.&author=Hoadley+K.A.&title=Molecular+subtypes+in+head+and+neck+cancer+exhibit+distinct+patterns+of+chromosomal+gain+and+loss+of+canonical+cancer+genes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, the RNA-seq matrix, represented as RSEM, were aggregated together with the TCGA RSEM matrix (downloaded from Broad GDAC Firehose data portal: <a class="inter-ref-link" href="https://gdac.broadinstitute.org/" target="_blank">https://gdac.broadinstitute.org/</a>) and upper-quartile normalized. The RNA expression matrix was median-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to different transcriptomics subtypes according to the highest correlations. Samples with an insignificant correlation to all subtypes (p &gt; 0.01) were marked as ‘undecided’.</div></section><section id="sec5.4.12.21" data-locator="sec5.4.12.21" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unsupervised subtyping using non-negative matrix factorization (NMF)</span></h4><div class="section-paragraph">Non-negative matrix factorization (NMF) implemented in the NMF R package (<div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d52364298e4525" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib49-d52364298e4525" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (437)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to perform unsupervised clustering of tumor samples and to identify proteogenomic features (proteins, phosphopeptide, mRNA transcripts, miRNAs, and somatic copy number alterations) that showed characteristic abundance patterns for each cluster. Briefly, given a factorization rank <em>k</em> (where <em>k</em> is the number of clusters), NMF decomposes a <em>p</em> x <em>n</em> data matrix <em>V</em> into two matrices <em>W</em> and <em>H</em> such that multiplication of <em>W</em> and <em>H</em> approximates <em>V</em>. Matrix <em>H</em> is a <em>k</em> x <em>n</em> matrix whose entries represent weights for each sample (1 to <em>N</em>) to contribute to each cluster (1 to <em>k</em>), whereas matrix <em>W</em> is a <em>p</em> x <em>k</em> matrix representing weights for each feature (1 to <em>p</em>) to contribute to each cluster (1 to <em>k</em>). Matrix <em>H</em> was used to assign samples to clusters by choosing the <em>k</em> with maximum score in each column of <em>H</em>. For each sample, we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em>H</em>. We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5. Matrix <em>W</em> containing the weights of each feature in a certain cluster was used to derive a list of representative features separating the clusters using the method proposed in (Kim and Park, 2007). Cluster-specific features were further subjected to a 2-sample moderated t-test (<div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d52364298e4582" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Ritchie et al., 2015</a><div data-db-target-of="bib109-d52364298e4582" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">Limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7814)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=Limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) comparing the feature abundance between the respective cluster and all other clusters. Derived p-values were adjusted for multiple hypothesis testing using the methods proposed in (<div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d52364298e4585" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib14-d52364298e4585" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the false discovery rate: a practical and powerful approach to multiple testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Ser. B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Ser.+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.4.12.22" data-locator="sec5.4.12.22" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preprocessing of data tables</span></h4><div class="section-paragraph">To enable integrative multi-omics clustering, we required all data types (and converted if necessary) to represent ratios to either a common reference measured in each TMT plex (proteome, phosphoproteome) or an <em>in-silico</em> common reference calculated as the median abundance across all samples (mRNA, mi-RNA). The phosphoproteome data consisted of phosphopeptides containing confidently localized sites. All data tables were then concatenated and only features non-missing in all tumors were used for subsequent analysis. Features with the lowest standard deviation (bottom 5<sup>th</sup> percentile) across all samples were deemed uninformative and were removed from the dataset. Each row in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores.</div><div class="section-paragraph">Since NMF requires a non-negative input matrix, the data matrix of z-scores was further converted into a non-negative matrix as follows:<ul class="ce-list--remove-bullets" id="olist0010"><li class="ce-list--remove-bullets__list-item" id="o0010"><span class="label">1)</span><div class="ce-list--remove-bullets__list-item__text">Create one data matrix with all negative numbers zeroed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0015"><span class="label">2)</span><div class="ce-list--remove-bullets__list-item__text">Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0020"><span class="label">3)</span><div class="ce-list--remove-bullets__list-item__text">Concatenate both matrices resulting in a data matrix twice as large as the original, but with positive values only and zeros and hence appropriate for NMF.</div></li></ul></div></section><section id="sec5.4.12.23" data-locator="sec5.4.12.23" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Determination of factorization rank</span></h4><div class="section-paragraph">The resulting matrix was then subjected to NMF analysis leveraging the NMF R package (<div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d52364298e4619" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib49-d52364298e4619" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (437)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and using the factorization method described in (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d52364298e4622" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brunet et al., 2004</a><div data-db-target-of="bib18-d52364298e4622" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Metagenes and molecular pattern discovery using matrix factorization.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1038)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To determine the optimal factorization rank <em>k</em> (number of clusters) for the multi-omic data matrix, a range of clusters between <em>k</em>=2 and 8 was tested. For each <em>k</em> we factorized matrix <em>V</em> using 50 iterations with random initializations of <em>W</em> and <em>H</em>. To determine the optimal factorization rank we calculated cophenetic correlation coefficients measuring how well the intrinsic structure of the data is recapitulated after clustering and chose the <em>k</em> with maximal cophenetic correlation for cluster numbers between <em>k</em>=3 and 8.</div></section><section id="sec5.4.12.24" data-locator="sec5.4.12.24" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">NMF clustering</span></h4><div class="section-paragraph">Having determined the optimal factorization rank <em>k</em>, and in order to achieve robust factorization of the multi-omics data matrix <em>V</em>, the NMF analysis was repeated using 500 iterations with random initializations of <em>W</em> and <em>H</em> and partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em>W</em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to reverse the non-negative transformation and to derive a single signed weight for each feature, each row in matrix <em>W</em> was first normalized by dividing by the sum of feature weights in each row. Weights per feature and cluster were then aggregated by keeping the maximal normalized weight and multiplying with the sign of the z-score from the initial data matrix. Thus, the resulting transformed version of matrix <em>W</em><sub>signed</sub> contained signed cluster weights for each feature present in the input matrix.</div></section><section id="sec5.4.12.25" data-locator="sec5.4.12.25" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Subtype signature gene and phosphosite identifications</span></h4><div class="section-paragraph">For each of the three subtypes, the subtype signature genes and phosphosites were defined as those significantly more abundant in that subtype compared to both of the other two subtypes (BH adjusted p value &lt; 0.01, Student’s <em>t</em>-test). The subtype signature genes were identified at the RNA, protein, and phosphosite levels. All the subtype-significant genes and phosphosites (collapsed into gene-level) were used to perform over-representation analysis to identify enriched GO biological processes.</div></section><section id="sec5.4.12.26" data-locator="sec5.4.12.26" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omics gene set enrichment analysis</span></h4><div class="section-paragraph">GO biological processes enriched for each data type for a subtype were ranked by the signed -log10 p value and submitted to sumer (<div class="dropBlock reference-citations"><a href="#bib114" id="back-bib114" data-db-target-for="bib114-d52364298e4676" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Savage et al., 2019</a><div data-db-target-of="bib114-d52364298e4676" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Savage S.R. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Liao Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Graph algorithms for condensing and consolidating gene set analysis results.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S141-S152</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31142576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071350100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S141-S152&journal=Mol.+Cell.+Proteomics&author=Savage+S.R.&author=Shi+Z.&author=Liao+Y.&author=Zhang+B.&title=Graph+algorithms+for+condensing+and+consolidating+gene+set+analysis+results">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, sumer uses weighted set cover to remove redundant processes within a data type and clusters remaining pathways using affinity propagation (<div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d52364298e4679" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Frey and Dueck, 2007</a><div data-db-target-of="bib44-d52364298e4679" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Frey B.J. </li><li class="loa__item">Dueck D. </li></ul><div class="reference-citations__detials">Clustering by passing messages between data points.<div class="popup-ref"><span class="ref__series"><i>Science (New York, NY).</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">315</span></span>: <span class="ref__seriesPages">972-976</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscience.1136800&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33847172327&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4235)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2007&pages=972-976&journal=Science+%28New+York%2C+NY%29&author=Frey+B.J.&author=Dueck+D.&title=Clustering+by+passing+messages+between+data+points">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) based on the Simpson similarity index.</div></section><section id="sec5.4.12.27" data-locator="sec5.4.12.27" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Chromosome instability (CIN) score</span></h4><div class="section-paragraph">The CIN score (ChrIdx) reflects the overall copy number aberration across the whole genome. From the segmentation result, we used a weighted-sum approach to summarize the chromosome instability for each sample (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d52364298e4687" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Vasaikar et al., 2019</a><div data-db-target-of="bib133-d52364298e4687" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.A.&author=Savage+S.R.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.A.&title=Proteogenomic+analysis+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The absolute segment-level log2 ratios of all segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes.</div></section><section id="sec5.4.12.28" data-locator="sec5.4.12.28" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Identification of high-potential candidate samples for targeted therapies</span></h4><div class="section-paragraph">For each of the three targeted therapies, i.e., CDK4/6 inhibitors, EGFR mAb, and combinatorial immunotherapy, we proposed samples to be high-potential candidates for response to these three therapeutic strategies if they have both high level of the pathway activity targetable by the drug and high level of candidates reflecting the pathway activity. For CDK4/6 inhibitors, the pathway and candidates are CDK4/6-Rb1 phosphorylation-cell cycle (represented by phosphorylation of CDK4/6 substrates on Rb) and any of <em>CCND1</em>/<em>CDKN2A</em> genetic aberrations (including high <em>CCND1</em> CN amplification, with CN log2 ratio &gt; 1, <em>CDKN2A</em> bi-allelic loss or mutation), respectively. For EGFR mAb, the pathway and candidates are EGFR PROGENy pathway score and AREG or TGFA RNA expression, respectively. For combinatorial immunotherapy, the pathway and molecular signatures are ESTIMATE immune score and protein or RNA expression of immune checkpoint or suppressor molecules (e.g., PDL1). We required that the level of both pathways and molecular signatures from high-potential samples to be significantly higher than the rest of tumor samples (i.e., low-potential candidates) and also higher than the matched NAT samples. For each feature in the molecular signatures and pathways mentioned above, we performed univariate k-means clustering to cluster samples into low, medium, and high groups (k=3) using the Ckmeans.1d.dp R package (<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d52364298e4703" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Wang and Song, 2011</a><div data-db-target-of="bib138-d52364298e4703" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang H. </li><li class="loa__item">Song M. </li></ul><div class="reference-citations__detials">Ckmeans.1d.dp: optimal k-means clustering in one dimension by dynamic programming.<div class="popup-ref"><span class="ref__series"><i>R. J.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">29-33</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.32614%2FRJ-2011-015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27942416&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=29-33&journal=R.%C2%A0J.&author=Wang+H.&author=Song+M.&title=Ckmeans.1d.dp%3A+optimal+k-means+clustering+in+one+dimension+by+dynamic+programming">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Samples were assigned to the ‘high potential’ group if they belonged to the ‘high’ group based on the pathway clustering and simultaneously belonged to the ‘high’ group based on the clustering of at least one of the molecule signatures. The enrichment of each target therapies to the integrated subtypes was performed using Fisher’s exact test.</div></section><section id="sec5.4.12.29" data-locator="sec5.4.12.29" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">External data collection</span></h4><div class="section-paragraph">For analyses using datasets from previously published, publicly available studies, we collected high throughput transcriptomic profiling and associated response data from NCBI GEO: <a class="inter-ref-link" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102995" target="_blank">GSE102995</a> (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d52364298e4714" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Siano et al., 2018</a><div data-db-target-of="bib120-d52364298e4714" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Siano M. </li><li class="loa__item">Espeli V. </li><li class="loa__item">Mach N. </li><li class="loa__item">Bossi P. </li><li class="loa__item">Licitra L. </li><li class="loa__item">Ghielmini M. </li><li class="loa__item">Frattini M. </li><li class="loa__item">Canevari S. </li><li class="loa__item">De Cecco L. </li></ul><div class="reference-citations__detials">Gene signatures and expression of miRNAs associated with efficacy of panitumumab in a head and neck cancer phase II trial.<div class="popup-ref"><span class="ref__series"><i>Oral Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">144-151</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.oraloncology.2018.05.013&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29909889&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85047610082&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2018&pages=144-151&journal=Oral+Oncol.&author=Siano+M.&author=Espeli+V.&author=Mach+N.&author=Bossi+P.&author=Licitra+L.&author=Ghielmini+M.&author=Frattini+M.&author=Canevari+S.&author=De+Cecco+L.&title=Gene+signatures+and+expression+of+miRNAs+associated+with+efficacy+of+panitumumab+in+a+head+and+neck+cancer+phase+II+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and <a class="inter-ref-link" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84713" target="_blank">GSE84713</a> (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d52364298e4720" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Klinghammer et al., 2017</a><div data-db-target-of="bib67-d52364298e4720" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Klinghammer K. </li><li class="loa__item">Otto R. </li><li class="loa__item">Raguse J.-D. </li><li class="loa__item">Albers A.E. </li><li class="loa__item">Tinhofer I. </li><li class="loa__item">Fichtner I. </li><li class="loa__item">Leser U. </li><li class="loa__item">Keilholz U. </li><li class="loa__item">Hoffmann J. </li></ul><div class="reference-citations__detials">Basal subtype is predictive for response to cetuximab treatment in patient-derived xenografts of squamous cell head and neck cancer.<div class="popup-ref"><span class="ref__series"><i>Int. J. Cancer.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">141</span></span>: <span class="ref__seriesPages">1215-1221</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fijc.30808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28560858&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85021184298&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=1215-1221&journal=Int.+J.+Cancer&author=Klinghammer+K.&author=Otto+R.&author=Raguse+J.-D.&author=Albers+A.E.&author=Tinhofer+I.&author=Fichtner+I.&author=Leser+U.&author=Keilholz+U.&author=Hoffmann+J.&title=Basal+subtype+is+predictive+for+response+to+cetuximab+treatment+in+patient-derived+xenografts+of+squamous+cell+head+and+neck+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). 26 HPV-negative HNSCC PDX models were analyzed from <a class="inter-ref-link" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE102995" target="_blank">GSE102995</a>. For <a class="inter-ref-link" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84713" target="_blank">GSE84713</a>, 1 out of 25 tumors was HPV-positive but not annotated in the sequencing files, therefore all 25 tumors were analyzed. <em>In vitro</em> genome-wide perturbation datasets of HNSCC cell lines using shRNA (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d52364298e4732" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">McFarland et al., 2018</a><div data-db-target-of="bib93-d52364298e4732" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McFarland J.M. </li><li class="loa__item">Ho Z.V. </li><li class="loa__item">Kugener G. </li><li class="loa__item">Dempster J.M. </li><li class="loa__item">Montgomery P.G. </li><li class="loa__item">Bryan J.G. </li><li class="loa__item">Krill-Burger J.M. </li><li class="loa__item">Green T.M. </li><li class="loa__item">Vazquez F. </li><li class="loa__item">Boehm J.S. </li><li>et al.</li></ul><div class="reference-citations__detials">Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">4610</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-018-06916-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30389920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85056025716&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=4610&journal=Nat.+Commun.&author=McFarland+J.M.&author=Ho+Z.V.&author=Kugener+G.&author=Dempster+J.M.&author=Montgomery+P.G.&author=Bryan+J.G.&author=Krill-Burger+J.M.&author=Green+T.M.&author=Vazquez+F.&author=Boehm+J.S.&title=Improved+estimation+of+cancer+dependencies+from+large-scale+RNAi+screens+using+model-based+normalization+and+data+integration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and CRISPR (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d52364298e4735" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Behan et al., 2019</a><div data-db-target-of="bib10-d52364298e4735" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Behan F.M. </li><li class="loa__item">Iorio F. </li><li class="loa__item">Picco G. </li><li class="loa__item">Gonçalves E. </li><li class="loa__item">Beaver C.M. </li><li class="loa__item">Migliardi G. </li><li class="loa__item">Santos R. </li><li class="loa__item">Rao Y. </li><li class="loa__item">Sassi F. </li><li class="loa__item">Pinnelli M. </li><li>et al.</li></ul><div class="reference-citations__detials">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">568</span></span>: <span class="ref__seriesPages">511-516</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41586-019-1103-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30971826&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85064233743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (198)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&journal=Nature&author=Behan+F.M.&author=Iorio+F.&author=Picco+G.&author=Gon%C3%A7alves+E.&author=Beaver+C.M.&author=Migliardi+G.&author=Santos+R.&author=Rao+Y.&author=Sassi+F.&author=Pinnelli+M.&title=Prioritization+of+cancer+therapeutic+targets+using+CRISPR-Cas9+screens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d52364298e4738" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Wang and Song, 2011</a><div data-db-target-of="bib138-d52364298e4738" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang H. </li><li class="loa__item">Song M. </li></ul><div class="reference-citations__detials">Ckmeans.1d.dp: optimal k-means clustering in one dimension by dynamic programming.<div class="popup-ref"><span class="ref__series"><i>R. J.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">29-33</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.32614%2FRJ-2011-015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27942416&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=29-33&journal=R.%C2%A0J.&author=Wang+H.&author=Song+M.&title=Ckmeans.1d.dp%3A+optimal+k-means+clustering+in+one+dimension+by+dynamic+programming">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were downloaded from DepMap (<a class="inter-ref-link" href="https://depmap.org/portal/" target="_blank">https://depmap.org/portal/</a>). Of note, all of the HNSCC cell lines included here have <em>CCND1</em> amplification and/or <em>CDKN2A</em> deletion, so genomic aberrations could not be used to predict response (<a data-link="modal" data-target="#image-S1535-6108(20)30655-3gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>G and 4H). Phospho-Rb (Ser-807/811) and GAPDH loading control signals from untreated (control) HPV-negative HNSCC PDXs were quantified by densitometry analysis from western blotting data (<div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d52364298e4752" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Karamboulas et al., 2018</a><div data-db-target-of="bib60-d52364298e4752" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karamboulas C. </li><li class="loa__item">Bruce J.P. </li><li class="loa__item">Hope A.J. </li><li class="loa__item">Meens J. </li><li class="loa__item">Huang S.H. </li><li class="loa__item">Erdmann N. </li><li class="loa__item">Hyatt E. </li><li class="loa__item">Pereira K. </li><li class="loa__item">Goldstein D.P. </li><li class="loa__item">Weinreb I. </li><li>et al.</li></ul><div class="reference-citations__detials">Patient-derived xenografts for prognostication and personalized treatment for head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1318-1331.e1314</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30380421&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85055137026&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1318-1331.e1314&journal=Cell+Rep.&author=Karamboulas+C.&author=Bruce+J.P.&author=Hope+A.J.&author=Meens+J.&author=Huang+S.H.&author=Erdmann+N.&author=Hyatt+E.&author=Pereira+K.&author=Goldstein+D.P.&author=Weinreb+I.&title=Patient-derived+xenografts+for+prognostication+and+personalized+treatment+for+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using Image Studio Lite Software (LI-COR). Normalized phospho-Rb abundance from each PDX was calculated by dividing phospho-Rb signals by their corresponding GAPDH signals. The TCGA data were downloaded from the Broad Firehose data portal (<a class="inter-ref-link" href="https://gdac.broadinstitute.org/" target="_blank">https://gdac.broadinstitute.org/</a>). <em>CDKN2A</em> mutated gene matrix, <em>CCND1</em> and <em>CDKN2A</em> copy number (GISTIC), CCND1 RSEM data, and CCND1 and Rb pS807/811 RPPA data for HPV<sup>neg</sup> TCGA tumors were downloaded from LinkedOmics (<a class="inter-ref-link" href="http://linkedomics.org" target="_blank">http://linkedomics.org</a>).</div></section></section></section><section id="sec5.5" data-locator="sec5.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and statistical analysis</span></h3><div class="section-paragraph">Statistical analyses were performed using R unless explained otherwise. Multiple comparisons were adjusted by the Benjamini-Hochberg correction (<div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d52364298e4776" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib14-d52364298e4776" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the false discovery rate: a practical and powerful approach to multiple testing.<div class="popup-ref"><span class="ref__series"><i>J. R. Stat. Soc. Ser. B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Ser.+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Details can be found in Results and figure legends.</div></section></section><section><h2 class="top" id="secsectitle0575" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by grants U24 CA210954, U24 CA210985, U24 CA210972, U24 CA210979, U24 CA210986, U24 CA214125, U24 CA210967, and U24 CA210993 from the National Cancer Institute (NCI) Clinical Proteomic Tumor Analysis Consortium (CPTAC), by a Cancer Prevention Institute of Texas (CPRIT) award RR160027, by grant T32 CA203690 from the Translational Breast Cancer Research Training Program, and by funding from the McNair Foundation. B.Z. and M.J.E. are CPRIT Scholar in Cancer Research and McNair Medical Institute Scholar. M.J.E. is a Susan G. Komen Foundation Scholar. We thank Christopher J. Rickets (NCI) for helpful discussions.</div><h3>Author contributions</h3><div class="section-paragraph">Conceptualization: D.W.C., H.Z., B.Z.; Methodology: C.H., L.C., S.R.S., R.V.E., Y.D., Y.L., E.J.J., M.S., K.K., M.C., H.C., B.W., K.L., D.J.C., Y.H., L.C., Y.W., Z.S., M.A., M.W., D.R.M., S.S., M.A.G., L.D., A.I.N., H.Z., B.Z.; Software: F.d.V.L., M.S., K.K., H.C., B.W., K.L., Y.H., J.P., K.C., Z.S., Y.L., J.Q., A.I.N.; Validation: L.C., R.V.E., M.S., D.J.C., Y.H., L.C., Y.W., G.M., H.Z.; Formal Analysis: C.H., S.R.S., Y.D., Y.L., F.d.V.L., E.J.J., K.K., X.S., M.C., H.C., M.A.W., B.W., J.T.L., K.L., A.C., Y.H., J.P., Z.S., W.J., M.A., J.J., D.C.Z., W.L., P.V., D.R.M., J.Q., S.S., M.A.G., S.M.D., B.Z.; Investigation: C.H., L.C., S.R.S., R.V.E., Y.D., Y.L., F.d.V.L., E.J.J., M.S., K.K., J.T.L., Q.K.L., D.J.C., Y.H., L.C., J.P., Y.W., W.J., M.A., S.Y., P.V., Z.Z., J.Q., M.J.E., G.S.O., S.M.D., L.D., A.I.N., A.K.E., D.W.C., H.Z., B.Z.; Resources: L.C., M.S., B.W., K.L., Q.K.L., Y.H., J.P., Y.W., Z.S., Y.L., D.R.M., J.Q., X.S.C., G.M., H.Z., B.Z.; Data curation: C.H., L.C., R.V.E., Q.K.L., D.J.C., L.C., Y.W., K.A.K., S.M.D., H.Z.; Writing – Original Drafts: C.H., L.C., S.R.S., Y.D., Y.L., F.d.V.L., E.J.J., M.S., B.W., J.T.L., K.L., Y.H., Z.S., W.J., L.D., A.K.E., H.Z., B.Z.; Writing – Review &amp; Editing: C.H., S.R.S., Y.D., Y.L., E.J.J., K.K., J.T.L., A.C., D.R.M., A.R.P., A.M.C., E.A., A.M.W., J.L., M.W., M.W., A.S., S.S., M.A.G., G.S.O., E.S.B., S.M.D., L.D., A.I.N., A.K.E., H.Z., B.Z.; Visualization: C.H., S.R.S., Y.D., Y.L., E.J.J., K.K., X.S., J.T.L., Q.K.L., Y.H., J.P., J.J., B.Z.; Supervision: S.A.C., D.R.M., Z.Z., J.Q., X.S.C., P.W., A.M.C., S.M.D., L.D., A.I.N., D.W.C., H.Z., B.Z.; Project administration: Z.Z., J.Q., C.R.K., A.I.R., E.A., T.H., M.M., M.T., H.R., E.S.B.., D.W.C., H.Z., B.Z.; Funding acquisition: S.A.C., D.R.M., Z.Z., P.W., A.M.C., A.I.N., D.W.C., H.Z., B.Z.</div><h3>Declaration of interests</h3><div class="section-paragraph">S.A.C. is a member of the scientific advisory boards of Kymera, PTM BioLabs, and Seer and a scientific advisor to Pfizer and Biogen. The other authors have no conflicts of interest to declare. M.J.E. reports ownership and royalties associated with Bioclassifier LLC through sales by Nanostring LLC and Veracyte for the "Prosigna" breast cancer prognostic test. M.J.E. also reports ad hoc consulting for AstraZeneca, Foundation Medicine, G1 Therapeutics, Novartis, Sermonix, Abbvie, Lilly and Pfizer.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0085" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">Anupriya Agarwal, Eunkyung An, Matthew L. Anderson, Meenakshi Anurag, Shayan C. Avanessian, Dmitry Avtonomov, Oliver F. Bathe, Chet Birger, Michael J. Birrer, Lili Blumenberg, William E. Bocik, Emily S. Boja, Uma Borate, Melissa Borucki, Meghan C. Burke, Shuang Cai, Anna Pamela Calinawan, Liwei Cao, Steven A. Carr, Sandra Cerda, Daniel W. Chan, Hui-Yin Chang, Alyssa Charamut, Lijun Chen, Lin S. Chen, Xi S. Chen, Arul M. Chinnaiyan, Kyung-Cho Cho, Shrabanti Chowdhury, Marcin Cieślik, David J. Clark, Karl R. Clauser, Antonio Colaprico, Daniel Cui Zhou, Houston Culpepper, Tomasz Czernicki, Fulvio D'Angelo, Felipe da Veiga Leprevost, Jacob Day, Stephanie De Young, Emek Demir, Saravana M. Dhanasekaran, Fei Ding, Li Ding, Marcin J. Domagalski, Joseph C. Dort, Yongchao Dou, Brian Druker, Elizabeth Duffy, Maureen Dyer, Nathan J. Edwards, Adel K. El-Naggar, Kimberly Elburn, Matthew J. Ellis, Tatiana S. Ermakova, David Fenyo, Renata Ferrarotto, Alicia Francis, Stacey Gabriel, Luciano Garofano, Yifat Geffen, Gad Getz, Michael A. Gillette, Charles A. Goldthwaite, Linda I. Hannick, Pushpa Hariharan, David N. Hayes, David Heiman, Tara Hiltke, Barbara Hindenach, Katherine A. Hoadley, Galen Hostetter, Yingwei Hu, Chen Huang, Martin Hyrcza, Eric J. Jaehnig, Scott D. Jewell, Jiayi Ji, Wen Jiang, Corbin D. Jones, M. Harry Kane, Alicia Karz, Karen A. Ketchum, Christopher R. Kinsinger, Ramani B. Kothadia, Azra Krek, Karsten Krug, Chandan Kumar-Sinha, Jonathan T. Lei, Kai Li, Qing Kay Li, Yize Li, Wen-Wei Liang, Yuxing Liao, Hongwei Liu, Tao Liu, Jan Lubiński, Weiping Ma, Ewa Malc, Anna Malovannaya, D. R. Mani, Sailaja Mareedu, Sanford P. Markey, Annette Marrero-Oliveras, Nicollette Maunganidze, Jason E. McDermott, Peter B. McGarvey, John McGee, Mehdi Mesri, Piotr Mieczkowski, Simona Migliozzi, George Miles, Rebecca Montgomery, Alexey I. Nesvizhskii, Chelsea J. Newton, Gilbert S. Omenn, Umut Ozbek, Jianbo Pan, Amanda G. Paulovich, Samuel H. Payne, Dimitar Dimitrov Pazardzhikliev, Amy M. Perou, Francesca Petralia, Lyudmila Petrenko, Alexander R. Pico, Paul D. Piehowski, Dmitris Placantonakis, Larisa Polonskaya, Elena V. Ponomareva, Olga Potapova, Liqun Qi, Jiang Qian, Ning Qu, Shakti Ramkissoon, Boris Reva, Shannon Richey, Karna Robinson, Ana I. Robles, Nancy Roche, Karin Rodland, Henry Rodriguez, Daniel C. Rohrer, Dmitry Rykunov, Shankha Satpathy, Sara R. Savage, Eric E. Schadt, Michael Schnaubelt, Yan Shi, Zhiao Shi, Yvonne Shutack, Andrew G. Sikora, Shilpi Singh, Tara Skelly, Richard Smith, Lori J. Sokoll, Xiaoyu Song, Jakub Stawicki, Stephen E. Stein, James Suh, Wojciech Szopa, Dave Tabor, Donghui Tan, Darlene Tansil, Guo Ci Teo, Ratna R. Thangudu, Mathangi Thiagarajan, Cristina Tognon, Elie Traer, Shirley Tsang, Jeffrey Tyner, Ki Sung Um, Dana R. Valley, Rodrigo Vargas Eguez, Lyubomir Valkov Vasilev, Negin Vatanian, Pankaj Vats, Uma Velvulou, Michael Vernon, Pei Wang, Yuefan Wang, Alissa M. Weaver, Bo Wen, Michael Wendl, Thomas F Westbrook, Jeffrey R. Whiteaker, Małgorzata Wierzbicka, Maciej Wiznerowicz, Yige Wu, Matthew A. Wyczalkowski, Midie Xu, Lijun Yao, Xinpei Yi, Seungyeul Yoo, Fengchao Yu, Kakhaber Zaalishvili, Yuriy Zakhartsev, Robert Zelt, Bing Zhang, Hui Zhang, Zhen Zhang, Grace Zhao, Jun Zhu</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-pdf-file2"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/96282112-1f8e-4651-a811-478cc09cc282/mmc1.pdf" class="supplemental-information__download " target="_blank">
                                            Download .pdf (3.42
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    pdf files
                                </a></div></div><div class="supplemental-information__title">Document S1. Figures S1–S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/c694c779-4ad1-4577-bdbb-5df01b62ad6d/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.05
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. Clinical information and molecular phenotypes of the samples analyzed in this study, related to Figure 1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8cc397bf-9344-4c45-bb6d-45c8a144d83c/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.08
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Summary of significant genomic aberrations, related to Figure 1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/8d7bb559-1e8f-4e01-a503-27d0d756e6c2/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (4.15
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. Tumor-associated molecular features and protein changes and putative drug targets, related to Figure 2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/68ecf9b0-b448-4bd6-a8c3-a16895560d68/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.02
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. Manually curated FAT1 and CDKN2A genotypes, related to Figures 3 and 4</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/6ed86b8c-c3f6-450d-a012-1157f4bc1cd1/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.03
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Phosphosites associated with EGFR amplification, related to Figure 5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/a2aa4e57-81c8-4a6d-b520-a2063ce725f3/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (3.71
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Proteomics-supported putative neoantigens and putative immunosuppressive SCNAs, related to Figure 6</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc8"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2020.12.007/attachment/253a90e3-f1d5-4228-baa2-d71e3f72e180/mmc8.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.03
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. Subtype-specific molecular features and treatment opportunities, related to Figure 7</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0600" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Adkins D. </li><li class="ref__authors__name">Ley J. </li><li class="ref__authors__name">Neupane P. </li><li class="ref__authors__name">Worden F. </li><li class="ref__authors__name">Sacco A.G. </li><li class="ref__authors__name">Palka K. </li><li class="ref__authors__name">Grilley-Olson J.E. </li><li class="ref__authors__name">Maggiore R. </li><li class="ref__authors__name">Salama N.N. </li><li class="ref__authors__name">Trinkaus K. </li><li>et al.</li></ul><div class="ref__title">Palbociclib and cetuximab in platinum-resistant and in cetuximab-resistant human papillomavirus-unrelated head and neck cancer: a multicentre, multigroup, phase 2 trial.</div><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1295-1305</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071533195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31351869&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2819%2930405-X&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=1295-1305&journal=Lancet+Oncol.&author=Adkins+D.&author=Ley+J.&author=Neupane+P.&author=Worden+F.&author=Sacco+A.G.&author=Palka+K.&author=Grilley-Olson+J.E.&author=Maggiore+R.&author=Salama+N.N.&author=Trinkaus+K.&title=Palbociclib+and+cetuximab+in+platinum-resistant+and+in+cetuximab-resistant+human+papillomavirus-unrelated+head+and+neck+cancer%3A+a+multicentre%2C+multigroup%2C+phase+2+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Almeida L.G. </li><li class="ref__authors__name">Sakabe N.J. </li><li class="ref__authors__name">deOliveira A.R. </li><li class="ref__authors__name">Silva M.C.C. </li><li class="ref__authors__name">Mundstein A.S. </li><li class="ref__authors__name">Cohen T. </li><li class="ref__authors__name">Chen Y.-T. </li><li class="ref__authors__name">Chua R. </li><li class="ref__authors__name">Gurung S. </li><li class="ref__authors__name">Gnjatic S. </li><li>et al.</li></ul><div class="ref__title">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (213)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Alvarez M.J. </li><li class="ref__authors__name">Shen Y. </li><li class="ref__authors__name">Giorgi F.M. </li><li class="ref__authors__name">Lachmann A. </li><li class="ref__authors__name">Ding B.B. </li><li class="ref__authors__name">Ye B.H. </li><li class="ref__authors__name">Califano A. </li></ul><div class="ref__title">Functional characterization of somatic mutations in cancer using network-based inference of protein activity.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">838-847</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975246805&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27322546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3593&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=838-847&journal=Nat.+Genet.&author=Alvarez+M.J.&author=Shen+Y.&author=Giorgi+F.M.&author=Lachmann+A.&author=Ding+B.B.&author=Ye+B.H.&author=Califano+A.&title=Functional+characterization+of+somatic+mutations+in+cancer+using+network-based+inference+of+protein+activity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ang K.K. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Rosenthal D.I. </li><li class="ref__authors__name">Nguyen-Tan P.F. </li><li class="ref__authors__name">Sherman E.J. </li><li class="ref__authors__name">Weber R.S. </li><li class="ref__authors__name">Galvin J.M. </li><li class="ref__authors__name">Bonner J.A. </li><li class="ref__authors__name">Harris J. </li><li class="ref__authors__name">El-Naggar A.K. </li><li>et al.</li></ul><div class="ref__title">Randomized phase III trial of concurrent accelerated radiation plus cisplatin with or without cetuximab for stage III to IV head and neck carcinoma: RTOG 0522.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2940-2950</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84907200175&amp;cf=" onclick="newWindow(this.href);return false">Scopus (437)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25154822&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2013.53.5633&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=2940-2950&journal=J.%C2%A0Clin.+Oncol.&author=Ang+K.K.&author=Zhang+Q.&author=Rosenthal+D.I.&author=Nguyen-Tan+P.F.&author=Sherman+E.J.&author=Weber+R.S.&author=Galvin+J.M.&author=Bonner+J.A.&author=Harris+J.&author=El-Naggar+A.K.&title=Randomized+phase+III+trial+of+concurrent+accelerated+radiation+plus+cisplatin+with+or+without+cetuximab+for+stage+III+to+IV+head+and+neck+carcinoma%3A+RTOG+0522">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Aran D. </li><li class="ref__authors__name">Hu Z. </li><li class="ref__authors__name">Butte A.J. </li></ul><div class="ref__title">xCell: digitally portraying the tissue cellular heterogeneity landscape.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Argentini A. </li><li class="ref__authors__name">Goeminne L.J.E. </li><li class="ref__authors__name">Verheggen K. </li><li class="ref__authors__name">Hulstaert N. </li><li class="ref__authors__name">Staes A. </li><li class="ref__authors__name">Clement L. </li><li class="ref__authors__name">Martens L. </li></ul><div class="ref__title">moFF: a robust and automated approach to extract peptide ion intensities.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">964-966</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84999277470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27898063&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.4075&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=964-966&journal=Nat.+Methods&author=Argentini+A.&author=Goeminne+L.J.E.&author=Verheggen+K.&author=Hulstaert+N.&author=Staes+A.&author=Clement+L.&author=Martens+L.&title=moFF%3A+a+robust+and+automated+approach+to+extract+peptide+ion+intensities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Babiceanu M. </li><li class="ref__authors__name">Qin F. </li><li class="ref__authors__name">Xie Z. </li><li class="ref__authors__name">Jia Y. </li><li class="ref__authors__name">Lopez K. </li><li class="ref__authors__name">Janus N. </li><li class="ref__authors__name">Facemire L. </li><li class="ref__authors__name">Kumar S. </li><li class="ref__authors__name">Pang Y. </li><li class="ref__authors__name">Qi Y. </li><li>et al.</li></ul><div class="ref__title">Recurrent chimeric fusion RNAs in non-cancer tissues and cells.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">2859-2872</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84963985214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26837576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkw032&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=2859-2872&journal=Nucleic+Acids+Res.&author=Babiceanu+M.&author=Qin+F.&author=Xie+Z.&author=Jia+Y.&author=Lopez+K.&author=Janus+N.&author=Facemire+L.&author=Kumar+S.&author=Pang+Y.&author=Qi+Y.&title=Recurrent+chimeric+fusion+RNAs+in+non-cancer+tissues+and+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barbie D.A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Boehm J.S. </li><li class="ref__authors__name">Kim S.Y. </li><li class="ref__authors__name">Moody S.E. </li><li class="ref__authors__name">Dunn I.F. </li><li class="ref__authors__name">Schinzel A.C. </li><li class="ref__authors__name">Sandy P. </li><li class="ref__authors__name">Meylan E. </li><li class="ref__authors__name">Scholl C. </li><li>et al.</li></ul><div class="ref__title">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1148)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Baselga J. </li><li class="ref__authors__name">Trigo J.M. </li><li class="ref__authors__name">Bourhis J. </li><li class="ref__authors__name">Tortochaux J. </li><li class="ref__authors__name">Cortés-Funes H. </li><li class="ref__authors__name">Hitt R. </li><li class="ref__authors__name">Gascón P. </li><li class="ref__authors__name">Amellal N. </li><li class="ref__authors__name">Harstrick A. </li><li class="ref__authors__name">Eckardt A. </li></ul><div class="ref__title">Phase II multicenter study of the antiepidermal growth factor receptor monoclonal antibody cetuximab in combination with platinum-based chemotherapy in patients with platinum-refractory metastatic and/or recurrent squamous cell carcinoma of the head and neck.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5568-5577</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-23844523253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (432)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16009950&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.07.119&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5568-5577&journal=J.%C2%A0Clin.+Oncol.&author=Baselga+J.&author=Trigo+J.M.&author=Bourhis+J.&author=Tortochaux+J.&author=Cort%C3%A9s-Funes+H.&author=Hitt+R.&author=Gasc%C3%B3n+P.&author=Amellal+N.&author=Harstrick+A.&author=Eckardt+A.&title=Phase+II+multicenter+study+of+the+antiepidermal+growth+factor+receptor+monoclonal+antibody+cetuximab+in+combination+with+platinum-based+chemotherapy+in+patients+with+platinum-refractory+metastatic+and%2For+recurrent+squamous+cell+carcinoma+of+the+head+and+neck">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Behan F.M. </li><li class="ref__authors__name">Iorio F. </li><li class="ref__authors__name">Picco G. </li><li class="ref__authors__name">Gonçalves E. </li><li class="ref__authors__name">Beaver C.M. </li><li class="ref__authors__name">Migliardi G. </li><li class="ref__authors__name">Santos R. </li><li class="ref__authors__name">Rao Y. </li><li class="ref__authors__name">Sassi F. </li><li class="ref__authors__name">Pinnelli M. </li><li>et al.</li></ul><div class="ref__title">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">568</span></span>: <span class="ref__seriesPages">511-516</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85064233743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (198)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30971826&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41586-019-1103-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&journal=Nature&author=Behan+F.M.&author=Iorio+F.&author=Picco+G.&author=Gon%C3%A7alves+E.&author=Beaver+C.M.&author=Migliardi+G.&author=Santos+R.&author=Rao+Y.&author=Sassi+F.&author=Pinnelli+M.&title=Prioritization+of+cancer+therapeutic+targets+using+CRISPR-Cas9+screens">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bellman R. </li></ul><div class="ref__title">On the approximation of curves by line segments using dynamic programming.</div><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benada J. </li><li class="ref__authors__name">Burdová K. </li><li class="ref__authors__name">Lidak T. </li><li class="ref__authors__name">von Morgen P. </li><li class="ref__authors__name">Macurek L. </li></ul><div class="ref__title">Polo-like kinase 1 inhibits DNA damage response during mitosis.</div><span class="ref__series"><i>Cell Cycle.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">219-231</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84926624707&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25607646&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.4161%2F15384101.2014.977067&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=219-231&journal=Cell+Cycle&author=Benada+J.&author=Burdov%C3%A1+K.&author=Lidak+T.&author=von+Morgen+P.&author=Macurek+L.&title=Polo-like+kinase+1+inhibits+DNA+damage+response+during+mitosis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benelli M. </li><li class="ref__authors__name">Pescucci C. </li><li class="ref__authors__name">Marseglia G. </li><li class="ref__authors__name">Severgnini M. </li><li class="ref__authors__name">Torricelli F. </li><li class="ref__authors__name">Magi A. </li></ul><div class="ref__title">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benjamini Y. </li><li class="ref__authors__name">Hochberg Y. </li></ul><div class="ref__title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</div><span class="ref__series"><i>J. R. Stat. Soc. Ser. B Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.%C2%A0R.+Stat.+Soc.+Ser.+B+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brandes J.C. </li><li class="ref__authors__name">van Engeland M. </li><li class="ref__authors__name">Wouters K.A.D. </li><li class="ref__authors__name">Weijenberg M.P. </li><li class="ref__authors__name">Herman J.G. </li></ul><div class="ref__title">CHFR promoter hypermethylation in colon cancer correlates with the microsatellite instability phenotype.</div><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1152-1156</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-22344454699&amp;cf=" onclick="newWindow(this.href);return false">Scopus (69)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15760919&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fcarcin%2Fbgi058&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1152-1156&journal=Carcinogenesis&author=Brandes+J.C.&author=van+Engeland+M.&author=Wouters+K.A.D.&author=Weijenberg+M.P.&author=Herman+J.G.&title=CHFR+promoter+hypermethylation+in+colon+cancer+correlates+with+the+microsatellite+instability+phenotype">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bray F. </li><li class="ref__authors__name">Ferlay J. </li><li class="ref__authors__name">Soerjomataram I. </li><li class="ref__authors__name">Siegel R.L. </li><li class="ref__authors__name">Torre L.A. </li><li class="ref__authors__name">Jemal A. </li></ul><div class="ref__title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries.</div><span class="ref__series"><i>Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">394-424</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85053395052&amp;cf=" onclick="newWindow(this.href);return false">Scopus (23716)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30207593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3322%2Fcaac.21492&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&journal=Cancer+J.+Clin.&author=Bray+F.&author=Ferlay+J.&author=Soerjomataram+I.&author=Siegel+R.L.&author=Torre+L.A.&author=Jemal+A.&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bruderer R. </li><li class="ref__authors__name">Bernhardt O.M. </li><li class="ref__authors__name">Gandhi T. </li><li class="ref__authors__name">Xuan Y. </li><li class="ref__authors__name">Sondermann J. </li><li class="ref__authors__name">Schmidt M. </li><li class="ref__authors__name">Gomez-Varela D. </li><li class="ref__authors__name">Reiter L. </li></ul><div class="ref__title">Optimization of experimental parameters in data-independent mass spectrometry significantly increases depth and reproducibility of results.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">2296-2309</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85038848913&amp;cf=" onclick="newWindow(this.href);return false">Scopus (102)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29070702&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA117.000314&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=2296-2309&journal=Mol.+Cell.+Proteomics&author=Bruderer+R.&author=Bernhardt+O.M.&author=Gandhi+T.&author=Xuan+Y.&author=Sondermann+J.&author=Schmidt+M.&author=Gomez-Varela+D.&author=Reiter+L.&title=Optimization+of+experimental+parameters+in+data-independent+mass+spectrometry+significantly+increases+depth+and+reproducibility+of+results">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Metagenes and molecular pattern discovery using matrix factorization.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1038)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Burtness B. </li><li class="ref__authors__name">Goldwasser M.A. </li><li class="ref__authors__name">Flood W. </li><li class="ref__authors__name">Mattar B. </li><li class="ref__authors__name">Forastiere A.A. </li><li class="ref__authors__name">Eastern Cooperative Oncology G. </li></ul><div class="ref__title">Phase III randomized trial of cisplatin plus placebo compared with cisplatin plus cetuximab in metastatic/recurrent head and neck cancer: an Eastern Cooperative Oncology Group study.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">8646-8654</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33144461661&amp;cf=" onclick="newWindow(this.href);return false">Scopus (704)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16314626&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.02.4646&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=8646-8654&journal=J.%C2%A0Clin.+Oncol.&author=Burtness+B.&author=Goldwasser+M.A.&author=Flood+W.&author=Mattar+B.&author=Forastiere+A.A.&author=Eastern+Cooperative+Oncology+G.&title=Phase+III+randomized+trial+of+cisplatin+plus+placebo+compared+with+cisplatin+plus+cetuximab+in+metastatic%2Frecurrent+head+and+neck+cancer%3A+an+Eastern+Cooperative+Oncology+Group+study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Büttner M. </li><li class="ref__authors__name">Miao Z. </li><li class="ref__authors__name">Wolf F.A. </li><li class="ref__authors__name">Teichmann S.A. </li><li class="ref__authors__name">Theis F.J. </li></ul><div class="ref__title">A test metric for assessing single-cell RNA-seq batch correction.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">43-49</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85058910761&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30573817&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41592-018-0254-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=43-49&journal=Nat.+Methods&author=B%C3%BCttner+M.&author=Miao+Z.&author=Wolf+F.A.&author=Teichmann+S.A.&author=Theis+F.J.&title=A+test+metric+for+assessing+single-cell+RNA-seq+batch+correction">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Calmon M.F. </li><li class="ref__authors__name">Jeschke J. </li><li class="ref__authors__name">Zhang W. </li><li class="ref__authors__name">Dhir M. </li><li class="ref__authors__name">Siebenkäs C. </li><li class="ref__authors__name">Herrera A. </li><li class="ref__authors__name">Tsai H.-C. </li><li class="ref__authors__name">O'Hagan H.M. </li><li class="ref__authors__name">Pappou E.P. </li><li class="ref__authors__name">Hooker C.M. </li><li>et al.</li></ul><div class="ref__title">Epigenetic silencing of neurofilament genes promotes an aggressive phenotype in breast cancer.</div><span class="ref__series"><i>Epigenetics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">622-632</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25985363&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1080%2F15592294.2015.1050173&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=622-632&journal=Epigenetics&author=Calmon+M.F.&author=Jeschke+J.&author=Zhang+W.&author=Dhir+M.&author=Siebenk%C3%A4s+C.&author=Herrera+A.&author=Tsai+H.-C.&author=O%27Hagan+H.M.&author=Pappou+E.P.&author=Hooker+C.M.&title=Epigenetic+silencing+of+neurofilament+genes+promotes+an+aggressive+phenotype+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cancer Genome Atlas N. </li></ul><div class="ref__title">Comprehensive genomic characterization of head and neck squamous cell carcinomas.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">517</span></span>: <span class="ref__seriesPages">576-582</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923197605&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1706)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25631445&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnature14129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=576-582&journal=Nature&author=Cancer+Genome+Atlas+N.&title=Comprehensive+genomic+characterization+of+head+and+neck+squamous+cell+carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Wendl M.C. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Wylie K. </li><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Jayasinghe R. </li><li class="ref__authors__name">Xie M. </li><li class="ref__authors__name">Wu S. </li><li class="ref__authors__name">Niu B. </li><li class="ref__authors__name">Grubb R. </li><li>et al.</li></ul><div class="ref__title">Divergent viral presentation among human tumors and adjacent normal tissues.</div><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">28294</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84975700575&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27339696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsrep28294&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=28294&journal=Sci.+Rep.&author=Cao+S.&author=Wendl+M.C.&author=Wyczalkowski+M.A.&author=Wylie+K.&author=Ye+K.&author=Jayasinghe+R.&author=Xie+M.&author=Wu+S.&author=Niu+B.&author=Grubb+R.&title=Divergent+viral+presentation+among+human+tumors+and+adjacent+normal+tissues">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cerami E. </li><li class="ref__authors__name">Gao J. </li><li class="ref__authors__name">Dogrusoz U. </li><li class="ref__authors__name">Gross B.E. </li><li class="ref__authors__name">Sumer S.O. </li><li class="ref__authors__name">Aksoy B.A. </li><li class="ref__authors__name">Jacobsen A. </li><li class="ref__authors__name">Byrne C.J. </li><li class="ref__authors__name">Heuer M.L. </li><li class="ref__authors__name">Larsson E. </li><li>et al.</li></ul><div class="ref__title">The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">401-404</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84866002291&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6173)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22588877&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F2159-8290.CD-12-0095&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2012&pages=401-404&journal=Cancer+Discov.&author=Cerami+E.&author=Gao+J.&author=Dogrusoz+U.&author=Gross+B.E.&author=Sumer+S.O.&author=Aksoy+B.A.&author=Jacobsen+A.&author=Byrne+C.J.&author=Heuer+M.L.&author=Larsson+E.&title=The+cBio+cancer+genomics+portal%3A+an+open+platform+for+exploring+multidimensional+cancer+genomics+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Schnaubelt M. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Aiyetan P. </li><li class="ref__authors__name">Chan D. </li><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Zhang Z. </li></ul><div class="ref__title">MS-PyCloud: an open-source, cloud computing-based pipeline for LC-MS/MS data analysis.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">320887</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=320887&journal=bioRxiv&author=Chen+L.&author=Zhang+B.&author=Schnaubelt+M.&author=Shah+P.&author=Aiyetan+P.&author=Chan+D.&author=Zhang+H.&author=Zhang+Z.&title=MS-PyCloud%3A+an+open-source%2C+cloud+computing-based+pipeline+for+LC-MS%2FMS+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Schulz-Trieglaff O. </li><li class="ref__authors__name">Shaw R. </li><li class="ref__authors__name">Barnes B. </li><li class="ref__authors__name">Schlesinger F. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Cox A.J. </li><li class="ref__authors__name">Kruglyak S. </li><li class="ref__authors__name">Saunders C.T. </li></ul><div class="ref__title">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (366)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chu A. </li><li class="ref__authors__name">Robertson G. </li><li class="ref__authors__name">Brooks D. </li><li class="ref__authors__name">Mungall A.J. </li><li class="ref__authors__name">Birol I. </li><li class="ref__authors__name">Coope R. </li><li class="ref__authors__name">Ma Y. </li><li class="ref__authors__name">Jones S. </li><li class="ref__authors__name">Marra M.A. </li></ul><div class="ref__title">Large-scale profiling of microRNAs for the cancer genome Atlas.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res.&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+the+cancer+genome+Atlas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chung C.H. </li><li class="ref__authors__name">Parker J.S. </li><li class="ref__authors__name">Karaca G. </li><li class="ref__authors__name">Wu J. </li><li class="ref__authors__name">Funkhouser W.K. </li><li class="ref__authors__name">Moore D. </li><li class="ref__authors__name">Butterfoss D. </li><li class="ref__authors__name">Xiang D. </li><li class="ref__authors__name">Zanation A. </li><li class="ref__authors__name">Yin X. </li><li>et al.</li></ul><div class="ref__title">Molecular classification of head and neck squamous cell carcinomas using patterns of gene expression.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">489-500</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2342492313&amp;cf=" onclick="newWindow(this.href);return false">Scopus (459)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15144956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1535-6108%2804%2900112-6&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=489-500&journal=Cancer+Cell&author=Chung+C.H.&author=Parker+J.S.&author=Karaca+G.&author=Wu+J.&author=Funkhouser+W.K.&author=Moore+D.&author=Butterfoss+D.&author=Xiang+D.&author=Zanation+A.&author=Yin+X.&title=Molecular+classification+of+head+and+neck+squamous+cell+carcinomas+using+patterns+of+gene+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Jaffe D. </li><li class="ref__authors__name">Sougnez C. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2136)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ciriello G. </li><li class="ref__authors__name">Cerami E. </li><li class="ref__authors__name">Sander C. </li><li class="ref__authors__name">Schultz N. </li></ul><div class="ref__title">Mutual exclusivity analysis identifies oncogenic network modules.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">398-406</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84856535081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (385)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21908773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.125567.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=398-406&journal=Genome+Res.&author=Ciriello+G.&author=Cerami+E.&author=Sander+C.&author=Schultz+N.&title=Mutual+exclusivity+analysis+identifies+oncogenic+network+modules">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Pan J. </li><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">da Veiga Leprevost F. </li><li class="ref__authors__name">Reva B. </li><li class="ref__authors__name">Lih T.-S.M. </li><li class="ref__authors__name">Chang H.-Y. </li><li>et al.</li></ul><div class="ref__title">Integrated proteogenomic characterization of clear cell renal cell carcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e931</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e931&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Bocik W. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Schnaubelt M. </li><li class="ref__authors__name">Roberts R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Whiteley G. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">Evaluation of NCI-7 cell line panel as a reference material for clinical proteomics.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">2205-2215</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85046635566&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29718670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Facs.jproteome.8b00165&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=2205-2215&journal=J.%C2%A0Proteome+Res.&author=Clark+D.J.&author=Hu+Y.&author=Bocik+W.&author=Chen+L.&author=Schnaubelt+M.&author=Roberts+R.&author=Shah+P.&author=Whiteley+G.&author=Zhang+H.&title=Evaluation+of+NCI-7+cell+line+panel+as+a+reference+material+for+clinical+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Bailey M.H. </li><li class="ref__authors__name">Odom G.J. </li><li class="ref__authors__name">Terkelsen T. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen A.V. </li><li class="ref__authors__name">Cantini L. </li><li class="ref__authors__name">Zinovyev A. </li><li class="ref__authors__name">Barillot E. </li><li>et al.</li></ul><div class="ref__title">Interpreting pathways to discover cancer driver genes with Moonlight.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Garofano L. </li><li class="ref__authors__name">Cava C. </li><li class="ref__authors__name">Garolini D. </li><li class="ref__authors__name">Sabedot T.S. </li><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Pagnotta S.M. </li><li class="ref__authors__name">Castiglioni I. </li><li>et al.</li></ul><div class="ref__title">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (578)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Crombet T. </li><li class="ref__authors__name">Osorio M. </li><li class="ref__authors__name">Cruz T. </li><li class="ref__authors__name">Roca C. </li><li class="ref__authors__name">del Castillo R. </li><li class="ref__authors__name">Mon R. </li><li class="ref__authors__name">Iznaga-Escobar N. </li><li class="ref__authors__name">Figueredo R. </li><li class="ref__authors__name">Koropatnick J. </li><li class="ref__authors__name">Renginfo E. </li><li>et al.</li></ul><div class="ref__title">Use of the humanized anti-epidermal growth factor receptor monoclonal antibody h-R3 in combination with radiotherapy in the treatment of locally advanced head and neck cancer patients.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">1646-1654</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2442704412&amp;cf=" onclick="newWindow(this.href);return false">Scopus (252)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15117987&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2004.03.089&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2004&pages=1646-1654&journal=J.%C2%A0Clin.+Oncol.&author=Crombet+T.&author=Osorio+M.&author=Cruz+T.&author=Roca+C.&author=del+Castillo+R.&author=Mon+R.&author=Iznaga-Escobar+N.&author=Figueredo+R.&author=Koropatnick+J.&author=Renginfo+E.&title=Use+of+the+humanized+anti-epidermal+growth+factor+receptor+monoclonal+antibody+h-R3+in+combination+with+radiotherapy+in+the+treatment+of+locally+advanced+head+and+neck+cancer+patients">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Daily K. </li><li class="ref__authors__name">Ho Sui S.J. </li><li class="ref__authors__name">Schriml L.M. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li class="ref__authors__name">Lotia S. </li><li>et al.</li></ul><div class="ref__title">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.</div><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">de Cárcer G. </li><li class="ref__authors__name">Wachowicz P. </li><li class="ref__authors__name">Martínez-Martínez S. </li><li class="ref__authors__name">Oller J. </li><li class="ref__authors__name">Méndez-Barbero N. </li><li class="ref__authors__name">Escobar B. </li><li class="ref__authors__name">González-Loyola A. </li><li class="ref__authors__name">Takaki T. </li><li class="ref__authors__name">El Bakkali A. </li><li class="ref__authors__name">Cámara J.A. </li><li>et al.</li></ul><div class="ref__title">Plk1 regulates contraction of postmitotic smooth muscle cells and is required for vascular homeostasis.</div><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">964-974</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85029285746&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28692064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnm.4364&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=964-974&journal=Nat.+Med.&author=de+C%C3%A1rcer+G.&author=Wachowicz+P.&author=Mart%C3%ADnez-Mart%C3%ADnez+S.&author=Oller+J.&author=M%C3%A9ndez-Barbero+N.&author=Escobar+B.&author=Gonz%C3%A1lez-Loyola+A.&author=Takaki+T.&author=El+Bakkali+A.&author=C%C3%A1mara+J.A.&title=Plk1+regulates+contraction+of+postmitotic+smooth+muscle+cells+and+is+required+for+vascular+homeostasis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Deutsch E.W. </li><li class="ref__authors__name">Mendoza L. </li><li class="ref__authors__name">Shteynberg D. </li><li class="ref__authors__name">Slagel J. </li><li class="ref__authors__name">Sun Z. </li><li class="ref__authors__name">Moritz R.L. </li></ul><div class="ref__title">Trans-Proteomic Pipeline, a standardized data processing pipeline for large-scale reproducible proteomics informatics.</div><span class="ref__series"><i>Proteomics Clin. Appl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">745-754</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84948987628&amp;cf=" onclick="newWindow(this.href);return false">Scopus (148)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25631240&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fprca.201400164&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=745-754&journal=Proteomics+Clin.+Appl.&author=Deutsch+E.W.&author=Mendoza+L.&author=Shteynberg+D.&author=Slagel+J.&author=Sun+Z.&author=Moritz+R.L.&title=Trans-Proteomic+Pipeline%2C+a+standardized+data+processing+pipeline+for+large-scale+reproducible+proteomics+informatics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Drazic A. </li><li class="ref__authors__name">Aksnes H. </li><li class="ref__authors__name">Marie M. </li><li class="ref__authors__name">Boczkowska M. </li><li class="ref__authors__name">Varland S. </li><li class="ref__authors__name">Timmerman E. </li><li class="ref__authors__name">Foyn H. </li><li class="ref__authors__name">Glomnes N. </li><li class="ref__authors__name">Rebowski G. </li><li class="ref__authors__name">Impens F. </li><li>et al.</li></ul><div class="ref__title">NAA80 is actin's N-terminal acetyltransferase and regulates cytoskeleton assembly and cell motility.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">115</span></span>: <span class="ref__seriesPages">4399-4404</span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045948499&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29581253&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1073%2Fpnas.1718336115&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4399-4404&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Drazic+A.&author=Aksnes+H.&author=Marie+M.&author=Boczkowska+M.&author=Varland+S.&author=Timmerman+E.&author=Foyn+H.&author=Glomnes+N.&author=Rebowski+G.&author=Impens+F.&title=NAA80+is+actin%27s+N-terminal+acetyltransferase+and+regulates+cytoskeleton+assembly+and+cell+motility">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Suman V.J. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Goncalves R. </li><li class="ref__authors__name">Sanati S. </li><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">DeSchryver K. </li><li class="ref__authors__name">Crouch E. </li><li class="ref__authors__name">Brink A. </li><li class="ref__authors__name">Watson M. </li><li>et al.</li></ul><div class="ref__title">Ki67 proliferation index as a tool for chemotherapy decisions during and after neoadjuvant aromatase inhibitor treatment of breast cancer: results from the American College of Surgeons Oncology Group Z1031 trial (alliance).</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+proliferation+index+as+a+tool+for+chemotherapy+decisions+during+and+after+neoadjuvant+aromatase+inhibitor+treatment+of+breast+cancer%3A+results+from+the+American+College+of+Surgeons+Oncology+Group+Z1031+trial+%28alliance%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ferris R.L. </li><li class="ref__authors__name">Haddad R. </li><li class="ref__authors__name">Even C. </li><li class="ref__authors__name">Tahara M. </li><li class="ref__authors__name">Dvorkin M. </li><li class="ref__authors__name">Ciuleanu T.E. </li><li class="ref__authors__name">Clement P.M. </li><li class="ref__authors__name">Mesia R. </li><li class="ref__authors__name">Kutukova S. </li><li class="ref__authors__name">Zholudeva L. </li><li>et al.</li></ul><div class="ref__title">Durvalumab with or without tremelimumab in patients with recurrent or metastatic head and neck squamous cell carcinoma: EAGLE, a randomized, open-label phase III study.</div><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">942-950</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85084237800&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32294530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.annonc.2020.04.001&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=942-950&journal=Ann.+Oncol.&author=Ferris+R.L.&author=Haddad+R.&author=Even+C.&author=Tahara+M.&author=Dvorkin+M.&author=Ciuleanu+T.E.&author=Clement+P.M.&author=Mesia+R.&author=Kutukova+S.&author=Zholudeva+L.&title=Durvalumab+with+or+without+tremelimumab+in+patients+with+recurrent+or+metastatic+head+and+neck+squamous+cell+carcinoma%3A+EAGLE%2C+a+randomized%2C+open-label+phase+III+study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fisher S. </li><li class="ref__authors__name">Barry A. </li><li class="ref__authors__name">Abreu J. </li><li class="ref__authors__name">Minie B. </li><li class="ref__authors__name">Nolan J. </li><li class="ref__authors__name">Delorey T.M. </li><li class="ref__authors__name">Young G. </li><li class="ref__authors__name">Fennell T.J. </li><li class="ref__authors__name">Allen A. </li><li class="ref__authors__name">Ambrogio L. </li><li>et al.</li></ul><div class="ref__title">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fortin J.-P. </li><li class="ref__authors__name">Labbe A. </li><li class="ref__authors__name">Lemire M. </li><li class="ref__authors__name">Zanke B.W. </li><li class="ref__authors__name">Hudson T.J. </li><li class="ref__authors__name">Fertig E.J. </li><li class="ref__authors__name">Greenwood C.M. </li><li class="ref__authors__name">Hansen K.D. </li></ul><div class="ref__title">Functional normalization of 450k methylation array data improves replication in large cancer studies.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (284)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.-P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Frey B.J. </li><li class="ref__authors__name">Dueck D. </li></ul><div class="ref__title">Clustering by passing messages between data points.</div><span class="ref__series"><i>Science (New York, NY).</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">315</span></span>: <span class="ref__seriesPages">972-976</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33847172327&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4235)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscience.1136800&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=315&publication_year=2007&pages=972-976&journal=Science+%28New+York%2C+NY%29&author=Frey+B.J.&author=Dueck+D.&title=Clustering+by+passing+messages+between+data+points">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao J. </li><li class="ref__authors__name">Aksoy B.A. </li><li class="ref__authors__name">Dogrusoz U. </li><li class="ref__authors__name">Dresdner G. </li><li class="ref__authors__name">Gross B. </li><li class="ref__authors__name">Sumer S.O. </li><li class="ref__authors__name">Sun Y. </li><li class="ref__authors__name">Jacobsen A. </li><li class="ref__authors__name">Sinha R. </li><li class="ref__authors__name">Larsson E. </li><li>et al.</li></ul><div class="ref__title">Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal.</div><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">pl1</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84875740314&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5610)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23550210&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscisignal.2004088&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=pl1&journal=Sci.+Signal.&author=Gao+J.&author=Aksoy+B.A.&author=Dogrusoz+U.&author=Dresdner+G.&author=Gross+B.&author=Sumer+S.O.&author=Sun+Y.&author=Jacobsen+A.&author=Sinha+R.&author=Larsson+E.&title=Integrative+analysis+of+complex+cancer+genomics+and+clinical+profiles+using+the+cBioPortal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao Q. </li><li class="ref__authors__name">Liang W.-W. </li><li class="ref__authors__name">Foltz S.M. </li><li class="ref__authors__name">Mutharasu G. </li><li class="ref__authors__name">Jayasinghe R.G. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Liao W.-W. </li><li class="ref__authors__name">Reynolds S.M. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Yao L. </li><li>et al.</li></ul><div class="ref__title">Driver fusions and their implications in the development and treatment of human cancers.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">227-238.e223</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85044744284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (142)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29617662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=227-238.e223&journal=Cell+Rep.&author=Gao+Q.&author=Liang+W.-W.&author=Foltz+S.M.&author=Mutharasu+G.&author=Jayasinghe+R.G.&author=Cao+S.&author=Liao+W.-W.&author=Reynolds+S.M.&author=Wyczalkowski+M.A.&author=Yao+L.&title=Driver+fusions+and+their+implications+in+the+development+and+treatment+of+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao Y. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhao F. </li></ul><div class="ref__title">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84937578024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (401)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25583365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-014-0571-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=4&journal=Genome+Biol.&author=Gao+Y.&author=Wang+J.&author=Zhao+F.&title=CIRI%3A+an+efficient+and+unbiased+algorithm+for+de+novo+circular+RNA+identification">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Garcia-Alonso L. </li><li class="ref__authors__name">Holland C.H. </li><li class="ref__authors__name">Ibrahim M.M. </li><li class="ref__authors__name">Turei D. </li><li class="ref__authors__name">Saez-Rodriguez J. </li></ul><div class="ref__title">Benchmark and integration of resources for the estimation of human transcription factor activities.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">1363-1375</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071059943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31340985&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.240663.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1363-1375&journal=Genome+Res.&author=Garcia-Alonso+L.&author=Holland+C.H.&author=Ibrahim+M.M.&author=Turei+D.&author=Saez-Rodriguez+J.&title=Benchmark+and+integration+of+resources+for+the+estimation+of+human+transcription+factor+activities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gaujoux R. </li><li class="ref__authors__name">Seoighe C. </li></ul><div class="ref__title">A flexible R package for nonnegative matrix factorization.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (437)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><p>Gey, S., and Lebarbier, E. (2008). Using CART to Detect Multiple Change Points in the Mean for Large Sample, <a class="inter-ref-link" href="https://hal.archives-ouvertes.fr/hal-00327146/" target="_blank">https://hal.archives-ouvertes.fr/hal-00327146/</a>.</p><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Gey%2C+S.%2C+and+Lebarbier%2C+E.+%282008%29.+Using+CART+to+Detect+Multiple+Change+Points+in+the+Mean+for+Large+Sample%2C+https%3A%2F%2Fhal.archives-ouvertes.fr%2Fhal-00327146%2F.">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ghantous Y. </li><li class="ref__authors__name">Schussel J.L. </li><li class="ref__authors__name">Brait M. </li></ul><div class="ref__title">Tobacco and alcohol-induced epigenetic changes in oral carcinoma.</div><span class="ref__series"><i>Curr. Opin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">152-158</span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045508925&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29538041&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1097%2FCCO.0000000000000444&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2018&pages=152-158&journal=Curr.+Opin.+Oncol.&author=Ghantous+Y.&author=Schussel+J.L.&author=Brait+M.&title=Tobacco+and+alcohol-induced+epigenetic+changes+in+oral+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Haas B.J. </li><li class="ref__authors__name">Dobin A. </li><li class="ref__authors__name">Li B. </li><li class="ref__authors__name">Stransky N. </li><li class="ref__authors__name">Pochet N. </li><li class="ref__authors__name">Regev A. </li></ul><div class="ref__title">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (43)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hänzelmann S. </li><li class="ref__authors__name">Castelo R. </li><li class="ref__authors__name">Guinney J. </li></ul><div class="ref__title">GSVA: gene set variation analysis for microarray and RNA-seq data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1108)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Harding J. </li><li class="ref__authors__name">Burtness B. </li></ul><div class="ref__title">Cetuximab: an epidermal growth factor receptor chimeric human-murine monoclonal antibody.</div><span class="ref__series"><i>Drugs Today.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">41</span></span>: <span class="ref__seriesPages">107-127</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-16244400108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (209)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15821783&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1358%2Fdot.2005.41.2.882662&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=107-127&journal=Drugs+Today&author=Harding+J.&author=Burtness+B.&title=Cetuximab%3A+an+epidermal+growth+factor+receptor+chimeric+human-murine+monoclonal+antibody">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hein J.B. </li><li class="ref__authors__name">Hertz E.P.T. </li><li class="ref__authors__name">Garvanska D.H. </li><li class="ref__authors__name">Kruse T. </li><li class="ref__authors__name">Nilsson J. </li></ul><div class="ref__title">Distinct kinetics of serine and threonine dephosphorylation are essential for mitosis.</div><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">1433-1440</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85036508836&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29084198&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncb3634&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1433-1440&journal=Nat.+Cell+Biol.&author=Hein+J.B.&author=Hertz+E.P.T.&author=Garvanska+D.H.&author=Kruse+T.&author=Nilsson+J.&title=Distinct+kinetics+of+serine+and+threonine+dephosphorylation+are+essential+for+mitosis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Herbst R.S. </li><li class="ref__authors__name">Arquette M. </li><li class="ref__authors__name">Shin D.M. </li><li class="ref__authors__name">Dicke K. </li><li class="ref__authors__name">Vokes E.E. </li><li class="ref__authors__name">Azarnia N. </li><li class="ref__authors__name">Hong W.K. </li><li class="ref__authors__name">Kies M.S. </li></ul><div class="ref__title">Phase II multicenter study of the epidermal growth factor receptor antibody cetuximab and cisplatin for recurrent and refractory squamous cell carcinoma of the head and neck.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">5578-5587</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-24944490558&amp;cf=" onclick="newWindow(this.href);return false">Scopus (350)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16009949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2005.07.120&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=5578-5587&journal=J.%C2%A0Clin.+Oncol.&author=Herbst+R.S.&author=Arquette+M.&author=Shin+D.M.&author=Dicke+K.&author=Vokes+E.E.&author=Azarnia+N.&author=Hong+W.K.&author=Kies+M.S.&title=Phase+II+multicenter+study+of+the+epidermal+growth+factor+receptor+antibody+cetuximab+and+cisplatin+for+recurrent+and+refractory+squamous+cell+carcinoma+of+the+head+and+neck">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Skrzypek E. </li></ul><div class="ref__title">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1086)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jurtz V. </li><li class="ref__authors__name">Paul S. </li><li class="ref__authors__name">Andreatta M. </li><li class="ref__authors__name">Marcatili P. </li><li class="ref__authors__name">Peters B. </li><li class="ref__authors__name">Nielsen M. </li></ul><div class="ref__title">NetMHCpan-4.0: improved peptide-MHC class I interaction predictions integrating eluted ligand and peptide binding affinity data.</div><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (338)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+improved+peptide-MHC+class+I+interaction+predictions+integrating+eluted+ligand+and+peptide+binding+affinity+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Justilien V. </li><li class="ref__authors__name">Jameison L. </li><li class="ref__authors__name">Der C.J. </li><li class="ref__authors__name">Rossman K.L. </li><li class="ref__authors__name">Fields A.P. </li></ul><div class="ref__title">Oncogenic activity of Ect2 is regulated through protein kinase C iota-mediated phosphorylation.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">286</span></span>: <span class="ref__seriesPages">8149-8157</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79953154962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21189248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.M110.196113&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=8149-8157&journal=J.%C2%A0Biol.+Chem.&author=Justilien+V.&author=Jameison+L.&author=Der+C.J.&author=Rossman+K.L.&author=Fields+A.P.&title=Oncogenic+activity+of+Ect2+is+regulated+through+protein+kinase+C+iota-mediated+phosphorylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Karamboulas C. </li><li class="ref__authors__name">Bruce J.P. </li><li class="ref__authors__name">Hope A.J. </li><li class="ref__authors__name">Meens J. </li><li class="ref__authors__name">Huang S.H. </li><li class="ref__authors__name">Erdmann N. </li><li class="ref__authors__name">Hyatt E. </li><li class="ref__authors__name">Pereira K. </li><li class="ref__authors__name">Goldstein D.P. </li><li class="ref__authors__name">Weinreb I. </li><li>et al.</li></ul><div class="ref__title">Patient-derived xenografts for prognostication and personalized treatment for head and neck squamous cell carcinoma.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1318-1331.e1314</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85055137026&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30380421&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.celrep.2018.10.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1318-1331.e1314&journal=Cell+Rep.&author=Karamboulas+C.&author=Bruce+J.P.&author=Hope+A.J.&author=Meens+J.&author=Huang+S.H.&author=Erdmann+N.&author=Hyatt+E.&author=Pereira+K.&author=Goldstein+D.P.&author=Weinreb+I.&title=Patient-derived+xenografts+for+prognostication+and+personalized+treatment+for+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kasar S. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Improgo R. </li><li class="ref__authors__name">Tiao G. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Haradhvala N. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Kiezun A. </li><li class="ref__authors__name">Fernandes S.M. </li><li class="ref__authors__name">Bahl S. </li><li>et al.</li></ul><div class="ref__title">Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">8866</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26638776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms9866&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8866&journal=Nat.+Commun.&author=Kasar+S.&author=Kim+J.&author=Improgo+R.&author=Tiao+G.&author=Polak+P.&author=Haradhvala+N.&author=Lawrence+M.S.&author=Kiezun+A.&author=Fernandes+S.M.&author=Bahl+S.&title=Whole-genome+sequencing+reveals+activation-induced+cytidine+deaminase+signatures+during+indolent+chronic+lymphocytic+leukaemia+evolution">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Keck M.K. </li><li class="ref__authors__name">Zuo Z. </li><li class="ref__authors__name">Khattri A. </li><li class="ref__authors__name">Stricker T.P. </li><li class="ref__authors__name">Brown C.D. </li><li class="ref__authors__name">Imanguli M. </li><li class="ref__authors__name">Rieke D. </li><li class="ref__authors__name">Endhardt K. </li><li class="ref__authors__name">Fang P. </li><li class="ref__authors__name">Brägelmann J. </li><li>et al.</li></ul><div class="ref__title">Integrative analysis of head and neck cancer identifies two biologically distinct HPV and three non-HPV subtypes.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">870-881</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923172078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (191)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25492084&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-2481&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=870-881&journal=Clin.+Cancer+Res.&author=Keck+M.K.&author=Zuo+Z.&author=Khattri+A.&author=Stricker+T.P.&author=Brown+C.D.&author=Imanguli+M.&author=Rieke+D.&author=Endhardt+K.&author=Fang+P.&author=Br%C3%A4gelmann+J.&title=Integrative+analysis+of+head+and+neck+cancer+identifies+two+biologically+distinct+HPV+and+three+non-HPV+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Keller A. </li><li class="ref__authors__name">Nesvizhskii A.I. </li><li class="ref__authors__name">Kolker E. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search.</div><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">5383-5392</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0037108887&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3577)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=12403597&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac025747h&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=5383-5392&journal=Anal+Chem.&author=Keller+A.&author=Nesvizhskii+A.I.&author=Kolker+E.&author=Aebersold+R.&title=Empirical+statistical+model+to+estimate+the+accuracy+of+peptide+identifications+made+by+MS%2FMS+and+database+search">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Mouw K.W. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Braunstein L.Z. </li><li class="ref__authors__name">Kamburov A. </li><li class="ref__authors__name">Kwiatkowski D.J. </li><li class="ref__authors__name">Rosenberg J.E. </li><li class="ref__authors__name">Van Allen E.M. </li><li class="ref__authors__name">D'Andrea A. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (154)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%27Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (467)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Scheffler K. </li><li class="ref__authors__name">Halpern A.L. </li><li class="ref__authors__name">Bekritsky M.A. </li><li class="ref__authors__name">Noh E. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Kim Y. </li><li class="ref__authors__name">Beyter D. </li><li class="ref__authors__name">Krusche P. </li><li class="ref__authors__name">Saunders C.T. </li></ul><div class="ref__title">Strelka2: fast and accurate calling of germline and somatic variants.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (170)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&author=Saunders+C.T.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Klinghammer K. </li><li class="ref__authors__name">Otto R. </li><li class="ref__authors__name">Raguse J.-D. </li><li class="ref__authors__name">Albers A.E. </li><li class="ref__authors__name">Tinhofer I. </li><li class="ref__authors__name">Fichtner I. </li><li class="ref__authors__name">Leser U. </li><li class="ref__authors__name">Keilholz U. </li><li class="ref__authors__name">Hoffmann J. </li></ul><div class="ref__title">Basal subtype is predictive for response to cetuximab treatment in patient-derived xenografts of squamous cell head and neck cancer.</div><span class="ref__series"><i>Int. J. Cancer.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">141</span></span>: <span class="ref__seriesPages">1215-1221</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85021184298&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28560858&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fijc.30808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=1215-1221&journal=Int.+J.+Cancer&author=Klinghammer+K.&author=Otto+R.&author=Raguse+J.-D.&author=Albers+A.E.&author=Tinhofer+I.&author=Fichtner+I.&author=Leser+U.&author=Keilholz+U.&author=Hoffmann+J.&title=Basal+subtype+is+predictive+for+response+to+cetuximab+treatment+in+patient-derived+xenografts+of+squamous+cell+head+and+neck+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Klymkowsky M.W. </li><li class="ref__authors__name">Parr B. </li></ul><div class="ref__title">The body language of cells: the intimate connection between cell adhesion and behavior.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">83</span></span>: <span class="ref__seriesPages">5-8</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=7553873&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2F0092-8674%2895%2990226-0&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2F0092-8674%2895%2990226-0&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1995&pages=5-8&journal=Cell&author=Klymkowsky+M.W.&author=Parr+B.&title=The+body+language+of+cells%3A+the+intimate+connection+between+cell+adhesion+and+behavior">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Koboldt D.C. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Larson D.E. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Lin L. </li><li class="ref__authors__name">Miller C.A. </li><li class="ref__authors__name">Mardis E.R. </li><li class="ref__authors__name">Ding L. </li><li class="ref__authors__name">Wilson R.K. </li></ul><div class="ref__title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2185)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kong A.T. </li><li class="ref__authors__name">Leprevost F.V. </li><li class="ref__authors__name">Avtonomov D.M. </li><li class="ref__authors__name">Mellacheruvu D. </li><li class="ref__authors__name">Nesvizhskii A.I. </li></ul><div class="ref__title">MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry-based proteomics.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">513-520</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85017247712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28394336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.4256&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=513-520&journal=Nat.+Methods&author=Kong+A.T.&author=Leprevost+F.V.&author=Avtonomov+D.M.&author=Mellacheruvu+D.&author=Nesvizhskii+A.I.&title=MSFragger%3A+ultrafast+and+comprehensive+peptide+identification+in+mass+spectrometry-based+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kranz D. </li><li class="ref__authors__name">Boutros M. </li></ul><div class="ref__title">A synthetic lethal screen identifies FAT1 as an antagonist of caspase-8 in extrinsic apoptosis.</div><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">181-197</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24442637&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=181-197&journal=EMBO+J.&author=Kranz+D.&author=Boutros+M.&title=A+synthetic+lethal+screen+identifies+FAT1+as+an+antagonist+of+caspase-8+in+extrinsic+apoptosis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kreimer A.R. </li><li class="ref__authors__name">Clifford G.M. </li><li class="ref__authors__name">Boyle P. </li><li class="ref__authors__name">Franceschi S. </li></ul><div class="ref__title">Human papillomavirus types in head and neck squamous cell carcinomas worldwide: a systematic review.</div><span class="ref__series"><i>Cancer Epidemiol. Biomarkers Prev.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">467-475</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-13944263935&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1506)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15734974&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1055-9965.EPI-04-0551&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2005&pages=467-475&journal=Cancer+Epidemiol.+Biomarkers+Prev.&author=Kreimer+A.R.&author=Clifford+G.M.&author=Boyle+P.&author=Franceschi+S.&title=Human+papillomavirus+types+in+head+and+neck+squamous+cell+carcinomas+worldwide%3A+a+systematic+review">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kuznetsova A. </li><li class="ref__authors__name">Brockhoff P.B. </li><li class="ref__authors__name">Christensen R.H.B. </li></ul><div class="ref__title">lmerTest package: tests in linear mixed effects models.</div><span class="ref__series"><i>J. Stat. Softw.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">1-26</span><div class="ref__controls"><a href="#back-bib73" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.18637%2Fjss.v082.i13&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2017&pages=1-26&journal=J.%C2%A0Stat.+Softw.&author=Kuznetsova+A.&author=Brockhoff+P.B.&author=Christensen+R.H.B.&title=lmerTest+package%3A+tests+in+linear+mixed+effects+models">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lachmann A. </li><li class="ref__authors__name">Giorgi F.M. </li><li class="ref__authors__name">Lopez G. </li><li class="ref__authors__name">Califano A. </li></ul><div class="ref__title">ARACNe-AP: gene network reverse engineering through adaptive partitioning inference of mutual information.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2233</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84992345426&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27153652&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtw216&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2233&journal=Bioinformatics&author=Lachmann+A.&author=Giorgi+F.M.&author=Lopez+G.&author=Califano+A.&title=ARACNe-AP%3A+gene+network+reverse+engineering+through+adaptive+partitioning+inference+of+mutual+information">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Langmead B. </li><li class="ref__authors__name">Salzberg S.L. </li></ul><div class="ref__title">Fast gapped-read alignment with Bowtie 2.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">357-359</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84859210032&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16937)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22388286&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.1923&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=357-359&journal=Nat.+Methods&author=Langmead+B.&author=Salzberg+S.L.&title=Fast+gapped-read+alignment+with+Bowtie+2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Langmead B. </li><li class="ref__authors__name">Trapnell C. </li><li class="ref__authors__name">Pop M. </li><li class="ref__authors__name">Salzberg S.L. </li></ul><div class="ref__title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">R25</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-62349130698&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12570)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19261174&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2009-10-3-r25&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&journal=Genome+Biol.&author=Langmead+B.&author=Trapnell+C.&author=Pop+M.&author=Salzberg+S.L.&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lebarbier E. </li></ul><div class="ref__title">Detecting multiple change-points in the mean of Gaussian process by model selection.</div><span class="ref__series"><i>Signal Process.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Process.&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li B. </li><li class="ref__authors__name">Dewey C.N. </li></ul><div class="ref__title">RSEM: accurate transcript quantification from RNA-seq data with or without a reference genome.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6871)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Durbin R. </li></ul><div class="ref__title">Fast and accurate short read alignment with Burrows-Wheeler transform.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1754-1760</span><div class="ref__controls"><a href="#back-bib79" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-67649884743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19246)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19451168&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp324&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1754-1760&journal=Bioinformatics&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Handsaker B. </li><li class="ref__authors__name">Wysoker A. </li><li class="ref__authors__name">Fennell T. </li><li class="ref__authors__name">Ruan J. </li><li class="ref__authors__name">Homer N. </li><li class="ref__authors__name">Marth G. </li><li class="ref__authors__name">Abecasis G. </li><li class="ref__authors__name">Durbin R. </li><li class="ref__authors__name">Genome Project Data Processing S. </li></ul><div class="ref__title">The sequence alignment/map format and SAMtools.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span><div class="ref__controls"><a href="#back-bib80" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21754)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinformatics&author=Li+H.&author=Handsaker+B.&author=Wysoker+A.&author=Fennell+T.&author=Ruan+J.&author=Homer+N.&author=Marth+G.&author=Abecasis+G.&author=Durbin+R.&author=Genome+Project+Data+Processing+S.&title=The+sequence+alignment%2Fmap+format+and+SAMtools">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li K. </li><li class="ref__authors__name">Vaudel M. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Ren Y. </li><li class="ref__authors__name">Wen B. </li></ul><div class="ref__title">PDV: an integrative proteomics data viewer.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span><div class="ref__controls"><a href="#back-bib81" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li M. </li><li class="ref__authors__name">Xie X. </li><li class="ref__authors__name">Zhou J. </li><li class="ref__authors__name">Sheng M. </li><li class="ref__authors__name">Yin X. </li><li class="ref__authors__name">Ko E.-A. </li><li class="ref__authors__name">Zhou T. </li><li class="ref__authors__name">Gu W. </li></ul><div class="ref__title">Quantifying circular RNA expression from RNA-seq data using model-based framework.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span><div class="ref__controls"><a href="#back-bib82" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinformatics&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liao Y. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span><div class="ref__controls"><a href="#back-bib83" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Mesirov J.P. </li><li class="ref__authors__name">Tamayo P. </li></ul><div class="ref__title">The Molecular Signatures Database (MSigDB) hallmark gene set collection.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span><div class="ref__controls"><a href="#back-bib84" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1573)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lindeboom R.G.H. </li><li class="ref__authors__name">Supek F. </li><li class="ref__authors__name">Lehner B. </li></ul><div class="ref__title">The rules and impact of nonsense-mediated mRNA decay in human cancers.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">1112-1118</span><div class="ref__controls"><a href="#back-bib85" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84987641578&amp;cf=" onclick="newWindow(this.href);return false">Scopus (137)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27618451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3664&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=1112-1118&journal=Nat.+Genet.&author=Lindeboom+R.G.H.&author=Supek+F.&author=Lehner+B.&title=The+rules+and+impact+of+nonsense-mediated+mRNA+decay+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Linding R. </li><li class="ref__authors__name">Jensen L.J. </li><li class="ref__authors__name">Ostheimer G.J. </li><li class="ref__authors__name">van Vugt M.A.T.M. </li><li class="ref__authors__name">Jørgensen C. </li><li class="ref__authors__name">Miron I.M. </li><li class="ref__authors__name">Diella F. </li><li class="ref__authors__name">Colwill K. </li><li class="ref__authors__name">Taylor L. </li><li class="ref__authors__name">Elder K. </li><li>et al.</li></ul><div class="ref__title">Systematic discovery of in vivo phosphorylation networks.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">129</span></span>: <span class="ref__seriesPages">1415-1426</span><div class="ref__controls"><a href="#back-bib86" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34250743173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (553)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17570479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2007.05.052&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=1415-1426&journal=Cell&author=Linding+R.&author=Jensen+L.J.&author=Ostheimer+G.J.&author=van+Vugt+M.A.T.M.&author=J%C3%B8rgensen+C.&author=Miron+I.M.&author=Diella+F.&author=Colwill+K.&author=Taylor+L.&author=Elder+K.&title=Systematic+discovery+of+in%C2%A0vivo+phosphorylation+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Linding R. </li><li class="ref__authors__name">Jensen L.J. </li><li class="ref__authors__name">Pasculescu A. </li><li class="ref__authors__name">Olhovsky M. </li><li class="ref__authors__name">Colwill K. </li><li class="ref__authors__name">Bork P. </li><li class="ref__authors__name">Yaffe M.B. </li><li class="ref__authors__name">Pawson T. </li></ul><div class="ref__title">NetworKIN: a resource for exploring cellular phosphorylation networks.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">D695-D699</span><div class="ref__controls"><a href="#back-bib87" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-38549109487&amp;cf=" onclick="newWindow(this.href);return false">Scopus (223)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17981841&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkm902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D695-D699&journal=Nucleic+Acids+Res.&author=Linding+R.&author=Jensen+L.J.&author=Pasculescu+A.&author=Olhovsky+M.&author=Colwill+K.&author=Bork+P.&author=Yaffe+M.B.&author=Pawson+T.&title=NetworKIN%3A+a+resource+for+exploring+cellular+phosphorylation+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lovly C.M. </li><li class="ref__authors__name">Yan L. </li><li class="ref__authors__name">Ryan C.E. </li><li class="ref__authors__name">Takada S. </li><li class="ref__authors__name">Piwnica-Worms H. </li></ul><div class="ref__title">Regulation of Chk2 ubiquitination and signaling through autophosphorylation of serine 379.</div><span class="ref__series"><i>Mol. Cell Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">5874-5885</span><div class="ref__controls"><a href="#back-bib88" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-52649098505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18644861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1128%2FMCB.00821-08&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=5874-5885&journal=Mol.+Cell+Biol.&author=Lovly+C.M.&author=Yan+L.&author=Ryan+C.E.&author=Takada+S.&author=Piwnica-Worms+H.&title=Regulation+of+Chk2+ubiquitination+and+signaling+through+autophosphorylation+of+serine+379">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">MacGrath S.M. </li><li class="ref__authors__name">Koleske A.J. </li></ul><div class="ref__title">Cortactin in cell migration and cancer at a glance.</div><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">125</span></span>: <span class="ref__seriesPages">1621-1626</span><div class="ref__controls"><a href="#back-bib89" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863084923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (118)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22566665&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1242%2Fjcs.093781&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=1621-1626&journal=J.%C2%A0Cell+Sci&author=MacGrath+S.M.&author=Koleske+A.J.&title=Cortactin+in+cell+migration+and+cancer+at+a+glance">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Burzykowski T. </li><li class="ref__authors__name">Poisson L. </li><li class="ref__authors__name">Weinstein J.N. </li><li class="ref__authors__name">Kamińska B. </li><li class="ref__authors__name">Huelsken J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Gevaert O. </li><li>et al.</li></ul><div class="ref__title">Machine learning identifies stemness features associated with oncogenic dedifferentiation.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e315</span><div class="ref__controls"><a href="#back-bib90" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (245)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e315&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kami%C5%84ska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+learning+identifies+stemness+features+associated+with+oncogenic+dedifferentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Margaria J.P. </li><li class="ref__authors__name">Ratto E. </li><li class="ref__authors__name">Gozzelino L. </li><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Hirsch E. </li></ul><div class="ref__title">Class II PI3Ks at the intersection between signal transduction and membrane trafficking.</div><span class="ref__series"><i>Biomolecules.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">104</span><div class="ref__controls"><a href="#back-bib91" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85063273855&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3390%2Fbiom9030104&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=104&journal=Biomolecules&author=Margaria+J.P.&author=Ratto+E.&author=Gozzelino+L.&author=Li+H.&author=Hirsch+E.&title=Class+II+PI3Ks+at+the+intersection+between+signal+transduction+and+membrane+trafficking">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Martin D. </li><li class="ref__authors__name">Degese M.S. </li><li class="ref__authors__name">Vitale-Cross L. </li><li class="ref__authors__name">Iglesias-Bartolome R. </li><li class="ref__authors__name">Valera J.L.C. </li><li class="ref__authors__name">Wang Z. </li><li class="ref__authors__name">Feng X. </li><li class="ref__authors__name">Yeerna H. </li><li class="ref__authors__name">Vadmal V. </li><li class="ref__authors__name">Moroishi T. </li><li>et al.</li></ul><div class="ref__title">Assembly and activation of the Hippo signalome by FAT1 tumor suppressor.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">2372</span><div class="ref__controls"><a href="#back-bib92" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049751886&amp;cf=" onclick="newWindow(this.href);return false">Scopus (36)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29985391&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-018-04590-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=2372&journal=Nat.+Commun.&author=Martin+D.&author=Degese+M.S.&author=Vitale-Cross+L.&author=Iglesias-Bartolome+R.&author=Valera+J.L.C.&author=Wang+Z.&author=Feng+X.&author=Yeerna+H.&author=Vadmal+V.&author=Moroishi+T.&title=Assembly+and+activation+of+the+Hippo+signalome+by+FAT1+tumor+suppressor">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McFarland J.M. </li><li class="ref__authors__name">Ho Z.V. </li><li class="ref__authors__name">Kugener G. </li><li class="ref__authors__name">Dempster J.M. </li><li class="ref__authors__name">Montgomery P.G. </li><li class="ref__authors__name">Bryan J.G. </li><li class="ref__authors__name">Krill-Burger J.M. </li><li class="ref__authors__name">Green T.M. </li><li class="ref__authors__name">Vazquez F. </li><li class="ref__authors__name">Boehm J.S. </li><li>et al.</li></ul><div class="ref__title">Improved estimation of cancer dependencies from large-scale RNAi screens using model-based normalization and data integration.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">4610</span><div class="ref__controls"><a href="#back-bib93" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85056025716&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30389920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-018-06916-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=4610&journal=Nat.+Commun.&author=McFarland+J.M.&author=Ho+Z.V.&author=Kugener+G.&author=Dempster+J.M.&author=Montgomery+P.G.&author=Bryan+J.G.&author=Krill-Burger+J.M.&author=Green+T.M.&author=Vazquez+F.&author=Boehm+J.S.&title=Improved+estimation+of+cancer+dependencies+from+large-scale+RNAi+screens+using+model-based+normalization+and+data+integration">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Hanna M. </li><li class="ref__authors__name">Banks E. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Kernytsky A. </li><li class="ref__authors__name">Garimella K. </li><li class="ref__authors__name">Altshuler D. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Daly M. </li><li class="ref__authors__name">DePristo M.A. </li></ul><div class="ref__title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span><div class="ref__controls"><a href="#back-bib94" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10664)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib95" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1119)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Patel J. </li><li class="ref__authors__name">Udeshi N.D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Burgess M.W. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Jaffe J.D. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Integrated proteomic analysis of post-translational modifications by serial enrichment.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">634-637</span><div class="ref__controls"><a href="#back-bib96" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84879637572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (333)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23749302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.2518&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=634-637&journal=Nat.+Methods&author=Mertins+P.&author=Qiao+J.W.&author=Patel+J.&author=Udeshi+N.D.&author=Clauser+K.R.&author=Mani+D.R.&author=Burgess+M.W.&author=Gillette+M.A.&author=Jaffe+J.D.&author=Carr+S.A.&title=Integrated+proteomic+analysis+of+post-translational+modifications+by+serial+enrichment">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Tang L.C. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Clauss T.R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Gillette M.A. </li><li>et al.</li></ul><div class="ref__title">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.</div><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span><div class="ref__controls"><a href="#back-bib97" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Messersmith W.A. </li><li class="ref__authors__name">Hidalgo M. </li></ul><div class="ref__title">Panitumumab, a monoclonal anti epidermal growth factor receptor antibody in colorectal cancer: another one or the one?.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">4664-4666</span><div class="ref__controls"><a href="#back-bib98" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34548056116&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17699842&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-07-0065&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=4664-4666&journal=Clin.+Cancer+Res.&author=Messersmith+W.A.&author=Hidalgo+M.&title=Panitumumab%2C+a+monoclonal+anti+epidermal+growth+factor+receptor+antibody+in+colorectal+cancer%3A+another+one+or+the+one%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Min H.L. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Kim W.H. </li><li class="ref__authors__name">Jang B.G. </li><li class="ref__authors__name">Kim M.A. </li></ul><div class="ref__title">Epigenetic silencing of the putative tumor suppressor gene GLDC (glycine dehydrogenase) in gastric carcinoma.</div><span class="ref__series"><i>Anticancer Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">179-187</span><div class="ref__controls"><a href="#back-bib99" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26722042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=179-187&journal=Anticancer+Res.&author=Min+H.L.&author=Kim+J.&author=Kim+W.H.&author=Jang+B.G.&author=Kim+M.A.&title=Epigenetic+silencing+of+the+putative+tumor+suppressor+gene+GLDC+%28glycine+dehydrogenase%29+in+gastric+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mounir M. </li><li class="ref__authors__name">Lucchetta M. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Bontempi G. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Noushmehr H. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Papaleo E. </li></ul><div class="ref__title">New functionalities in the TCGAbiolinks package for the study and integration of cancer data from GDC and GTEx.</div><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">e1006701</span><div class="ref__controls"><a href="#back-bib100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85062948513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30835723&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pcbi.1006701&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=e1006701&journal=PLoS+Comput.+Biol.&author=Mounir+M.&author=Lucchetta+M.&author=Silva+T.C.&author=Olsen+C.&author=Bontempi+G.&author=Chen+X.&author=Noushmehr+H.&author=Colaprico+A.&author=Papaleo+E.&title=New+functionalities+in+the+TCGAbiolinks+package+for+the+study+and+integration+of+cancer+data+from+GDC+and+GTEx">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nesvizhskii A.I. </li><li class="ref__authors__name">Keller A. </li><li class="ref__authors__name">Kolker E. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">A statistical model for identifying proteins by tandem mass spectrometry.</div><span class="ref__series"><i>Anal Chem.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">75</span></span>: <span class="ref__seriesPages">4646-4658</span><div class="ref__controls"><a href="#back-bib101" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0042338362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3267)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=14632076&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fac0341261&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=4646-4658&journal=Anal+Chem.&author=Nesvizhskii+A.I.&author=Keller+A.&author=Kolker+E.&author=Aebersold+R.&title=A+statistical+model+for+identifying+proteins+by+tandem+mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nusinow D.P. </li><li class="ref__authors__name">Szpyt J. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Rose C.M. </li><li class="ref__authors__name">McDonald E.R. </li><li class="ref__authors__name">Kalocsay M. </li><li class="ref__authors__name">Jané-Valbuena J. </li><li class="ref__authors__name">Gelfand E. </li><li class="ref__authors__name">Schweppe D.K. </li><li class="ref__authors__name">Jedrychowski M. </li><li>et al.</li></ul><div class="ref__title">Quantitative proteomics of the cancer cell line encyclopedia.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">387-402.e316</span><div class="ref__controls"><a href="#back-bib102" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85077916308&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31978347&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.12.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=387-402.e316&journal=Cell&author=Nusinow+D.P.&author=Szpyt+J.&author=Ghandi+M.&author=Rose+C.M.&author=McDonald+E.R.&author=Kalocsay+M.&author=Jan%C3%A9-Valbuena+J.&author=Gelfand+E.&author=Schweppe+D.K.&author=Jedrychowski+M.&title=Quantitative+proteomics+of+the+cancer+cell+line+encyclopedia">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ow S.Y. </li><li class="ref__authors__name">Salim M. </li><li class="ref__authors__name">Noirel J. </li><li class="ref__authors__name">Evans C. </li><li class="ref__authors__name">Wright P.C. </li></ul><div class="ref__title">Minimising iTRAQ ratio compression through understanding LC-MS elution dependence and high-resolution HILIC fractionation.</div><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2341-2346</span><div class="ref__controls"><a href="#back-bib103" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79956270174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (82)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21548092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2Fpmic.201000752&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=2341-2346&journal=Proteomics&author=Ow+S.Y.&author=Salim+M.&author=Noirel+J.&author=Evans+C.&author=Wright+P.C.&title=Minimising+iTRAQ+ratio+compression+through+understanding+LC-MS+elution+dependence+and+high-resolution+HILIC+fractionation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Papillon-Cavanagh S. </li><li class="ref__authors__name">Lu C. </li><li class="ref__authors__name">Gayden T. </li><li class="ref__authors__name">Mikael L.G. </li><li class="ref__authors__name">Bechet D. </li><li class="ref__authors__name">Karamboulas C. </li><li class="ref__authors__name">Ailles L. </li><li class="ref__authors__name">Karamchandani J. </li><li class="ref__authors__name">Marchione D.M. </li><li class="ref__authors__name">Garcia B.A. </li><li>et al.</li></ul><div class="ref__title">Impaired H3K36 methylation defines a subset of head and neck squamous cell carcinomas.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">180-185</span><div class="ref__controls"><a href="#back-bib104" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85008661446&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28067913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fng.3757&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=180-185&journal=Nat.+Genet.&author=Papillon-Cavanagh+S.&author=Lu+C.&author=Gayden+T.&author=Mikael+L.G.&author=Bechet+D.&author=Karamboulas+C.&author=Ailles+L.&author=Karamchandani+J.&author=Marchione+D.M.&author=Garcia+B.A.&title=Impaired+H3K36+methylation+defines+a+subset+of+head+and+neck+squamous+cell+carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pierre-Jean M. </li><li class="ref__authors__name">Rigaill G. </li><li class="ref__authors__name">Neuvial P. </li></ul><div class="ref__title">Performance evaluation of DNA copy number segmentation methods.</div><span class="ref__series"><i>Brief Bioinform.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">600-615</span><div class="ref__controls"><a href="#back-bib105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84925419952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25202135&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbib%2Fbbu026&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=600-615&journal=Brief+Bioinform&author=Pierre-Jean+M.&author=Rigaill+G.&author=Neuvial+P.&title=Performance+evaluation+of+DNA+copy+number+segmentation+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Psyrri A. </li><li class="ref__authors__name">Lee J.-W. </li><li class="ref__authors__name">Pectasides E. </li><li class="ref__authors__name">Vassilakopoulou M. </li><li class="ref__authors__name">Kosmidis E.K. </li><li class="ref__authors__name">Burtness B.A. </li><li class="ref__authors__name">Rimm D.L. </li><li class="ref__authors__name">Wanebo H.J. </li><li class="ref__authors__name">Forastiere A.A. </li></ul><div class="ref__title">Prognostic biomarkers in phase II trial of cetuximab-containing induction and chemoradiation in resectable HNSCC: Eastern Cooperative Oncology Group E2303.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">3023-3032</span><div class="ref__controls"><a href="#back-bib106" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84901811387&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24700741&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1158%2F1078-0432.CCR-14-0113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=3023-3032&journal=Clin.+Cancer+Res.&author=Psyrri+A.&author=Lee+J.-W.&author=Pectasides+E.&author=Vassilakopoulou+M.&author=Kosmidis+E.K.&author=Burtness+B.A.&author=Rimm+D.L.&author=Wanebo+H.J.&author=Forastiere+A.A.&title=Prognostic+biomarkers+in+phase+II+trial+of+cetuximab-containing+induction+and+chemoradiation+in+resectable+HNSCC%3A+Eastern+Cooperative+Oncology+Group+E2303">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rauniyar N. </li><li class="ref__authors__name">Yates J.R. </li></ul><div class="ref__title">Isobaric labeling-based relative quantification in shotgun proteomics.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">5293-5309</span><div class="ref__controls"><a href="#back-bib107" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84915751035&amp;cf=" onclick="newWindow(this.href);return false">Scopus (278)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25337643&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fpr500880b&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=5293-5309&journal=J.%C2%A0Proteome+Res.&author=Rauniyar+N.&author=Yates+J.R.&title=Isobaric+labeling-based+relative+quantification+in+shotgun+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Reiter L. </li><li class="ref__authors__name">Rinner O. </li><li class="ref__authors__name">Picotti P. </li><li class="ref__authors__name">Hüttenhain R. </li><li class="ref__authors__name">Beck M. </li><li class="ref__authors__name">Brusniak M.-Y. </li><li class="ref__authors__name">Hengartner M.O. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">mProphet: automated data processing and statistical validation for large-scale SRM experiments.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">430-435</span><div class="ref__controls"><a href="#back-bib108" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-79955595074&amp;cf=" onclick="newWindow(this.href);return false">Scopus (293)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=21423193&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.1584&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2011&pages=430-435&journal=Nat.+Methods&author=Reiter+L.&author=Rinner+O.&author=Picotti+P.&author=H%C3%BCttenhain+R.&author=Beck+M.&author=Brusniak+M.-Y.&author=Hengartner+M.O.&author=Aebersold+R.&title=mProphet%3A+automated+data+processing+and+statistical+validation+for+large-scale+SRM+experiments">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ritchie M.E. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Wu D. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Law C.W. </li><li class="ref__authors__name">Shi W. </li><li class="ref__authors__name">Smyth G.K. </li></ul><div class="ref__title">Limma powers differential expression analyses for RNA-sequencing and microarray studies.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span><div class="ref__controls"><a href="#back-bib109" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7814)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=Limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rivera A.L. </li><li class="ref__authors__name">Pelloski C.E. </li><li class="ref__authors__name">Gilbert M.R. </li><li class="ref__authors__name">Colman H. </li><li class="ref__authors__name">De La Cruz C. </li><li class="ref__authors__name">Sulman E.P. </li><li class="ref__authors__name">Bekele B.N. </li><li class="ref__authors__name">Aldape K.D. </li></ul><div class="ref__title">MGMT promoter methylation is predictive of response to radiotherapy and prognostic in the absence of adjuvant alkylating chemotherapy for glioblastoma.</div><span class="ref__series"><i>Neuro Oncol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">116-121</span><div class="ref__controls"><a href="#back-bib110" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-75849146316&amp;cf=" onclick="newWindow(this.href);return false">Scopus (235)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=20150378&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fneuonc%2Fnop020&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=116-121&journal=Neuro+Oncol.&author=Rivera+A.L.&author=Pelloski+C.E.&author=Gilbert+M.R.&author=Colman+H.&author=De+La+Cruz+C.&author=Sulman+E.P.&author=Bekele+B.N.&author=Aldape+K.D.&title=MGMT+promoter+methylation+is+predictive+of+response+to+radiotherapy+and+prognostic+in+the+absence+of+adjuvant+alkylating+chemotherapy+for+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Huo J. </li><li class="ref__authors__name">Schriml L. </li><li class="ref__authors__name">Ho Sui S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li>et al.</li></ul><div class="ref__title">Integrated genomic analysis of diverse induced pluripotent stem cells from the Progenitor Cell Biology Consortium.</div><span class="ref__series"><i>Stem Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span><div class="ref__controls"><a href="#back-bib111" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Rep.&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+genomic+analysis+of+diverse+induced+pluripotent+stem+cells+from+the+Progenitor+Cell+Biology+Consortium">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Santuray R.T. </li><li class="ref__authors__name">Johnson D.E. </li><li class="ref__authors__name">Grandis J.R. </li></ul><div class="ref__title">New therapies in head and neck cancer.</div><span class="ref__series"><i>Trends Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">385-396</span><div class="ref__controls"><a href="#back-bib112" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85045920059&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29709262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.trecan.2018.03.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=385-396&journal=Trends+Cancer&author=Santuray+R.T.&author=Johnson+D.E.&author=Grandis+J.R.&title=New+therapies+in+head+and+neck+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Saunders C.T. </li><li class="ref__authors__name">Wong W.S.W. </li><li class="ref__authors__name">Swamy S. </li><li class="ref__authors__name">Becq J. </li><li class="ref__authors__name">Murray L.J. </li><li class="ref__authors__name">Cheetham R.K. </li></ul><div class="ref__title">Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1811-1817</span><div class="ref__controls"><a href="#back-bib113" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84864153492&amp;cf=" onclick="newWindow(this.href);return false">Scopus (752)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22581179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbts271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1811-1817&journal=Bioinformatics&author=Saunders+C.T.&author=Wong+W.S.W.&author=Swamy+S.&author=Becq+J.&author=Murray+L.J.&author=Cheetham+R.K.&title=Strelka%3A+accurate+somatic+small-variant+calling+from+sequenced+tumor-normal+sample+pairs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Liao Y. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Graph algorithms for condensing and consolidating gene set analysis results.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S141-S152</span><div class="ref__controls"><a href="#back-bib114" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071350100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31142576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR118.001263&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S141-S152&journal=Mol.+Cell.+Proteomics&author=Savage+S.R.&author=Shi+Z.&author=Liao+Y.&author=Zhang+B.&title=Graph+algorithms+for+condensing+and+consolidating+gene+set+analysis+results">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Savitski M.M. </li><li class="ref__authors__name">Wilhelm M. </li><li class="ref__authors__name">Hahne H. </li><li class="ref__authors__name">Kuster B. </li><li class="ref__authors__name">Bantscheff M. </li></ul><div class="ref__title">A scalable approach for protein false discovery rate estimation in large proteomic data sets.</div><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span><div class="ref__controls"><a href="#back-bib115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (147)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+scalable+approach+for+protein+false+discovery+rate+estimation+in+large+proteomic+data+sets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Schubert M. </li><li class="ref__authors__name">Klinger B. </li><li class="ref__authors__name">Klünemann M. </li><li class="ref__authors__name">Sieber A. </li><li class="ref__authors__name">Uhlitz F. </li><li class="ref__authors__name">Sauer S. </li><li class="ref__authors__name">Garnett M.J. </li><li class="ref__authors__name">Blüthgen N. </li><li class="ref__authors__name">Saez-Rodriguez J. </li></ul><div class="ref__title">Perturbation-response genes reveal signaling footprints in cancer gene expression.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span><div class="ref__controls"><a href="#back-bib116" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Seiwert T.Y. </li><li class="ref__authors__name">Burtness B. </li><li class="ref__authors__name">Mehra R. </li><li class="ref__authors__name">Weiss J. </li><li class="ref__authors__name">Berger R. </li><li class="ref__authors__name">Eder J.P. </li><li class="ref__authors__name">Heath K. </li><li class="ref__authors__name">McClanahan T. </li><li class="ref__authors__name">Lunceford J. </li><li class="ref__authors__name">Gause C. </li><li>et al.</li></ul><div class="ref__title">Safety and clinical activity of pembrolizumab for treatment of recurrent or metastatic squamous cell carcinoma of the head and neck (KEYNOTE-012): an open-label, multicentre, phase 1b trial.</div><span class="ref__series"><i>Lancet Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">956-965</span><div class="ref__controls"><a href="#back-bib117" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84969988424&amp;cf=" onclick="newWindow(this.href);return false">Scopus (740)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27247226&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=abstract&amp;site=lancet-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=fulltext&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2FS1470-2045%2816%2930066-3&amp;cf=pdf&amp;site=lancet-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=956-965&journal=Lancet+Oncol.&author=Seiwert+T.Y.&author=Burtness+B.&author=Mehra+R.&author=Weiss+J.&author=Berger+R.&author=Eder+J.P.&author=Heath+K.&author=McClanahan+T.&author=Lunceford+J.&author=Gause+C.&title=Safety+and+clinical+activity+of+pembrolizumab+for+treatment+of+recurrent+or+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+%28KEYNOTE-012%29%3A+an+open-label%2C+multicentre%2C+phase+1b+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shahbazian D. </li><li class="ref__authors__name">Roux P.P. </li><li class="ref__authors__name">Mieulet V. </li><li class="ref__authors__name">Cohen M.S. </li><li class="ref__authors__name">Raught B. </li><li class="ref__authors__name">Taunton J. </li><li class="ref__authors__name">Hershey J.W.B. </li><li class="ref__authors__name">Blenis J. </li><li class="ref__authors__name">Pende M. </li><li class="ref__authors__name">Sonenberg N. </li></ul><div class="ref__title">The mTOR/PI3K and MAPK pathways converge on eIF4B to control its phosphorylation and activity.</div><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2781-2791</span><div class="ref__controls"><a href="#back-bib118" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-33745570504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (356)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=16763566&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fsj.emboj.7601166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=2781-2791&journal=EMBO+J.&author=Shahbazian+D.&author=Roux+P.P.&author=Mieulet+V.&author=Cohen+M.S.&author=Raught+B.&author=Taunton+J.&author=Hershey+J.W.B.&author=Blenis+J.&author=Pende+M.&author=Sonenberg+N.&title=The+mTOR%2FPI3K+and+MAPK+pathways+converge+on+eIF4B+to+control+its+phosphorylation+and+activity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shteynberg D.D. </li><li class="ref__authors__name">Deutsch E.W. </li><li class="ref__authors__name">Campbell D.S. </li><li class="ref__authors__name">Hoopmann M.R. </li><li class="ref__authors__name">Kusebauch U. </li><li class="ref__authors__name">Lee D. </li><li class="ref__authors__name">Mendoza L. </li><li class="ref__authors__name">Midha M.K. </li><li class="ref__authors__name">Sun Z. </li><li class="ref__authors__name">Whetton A.D. </li><li class="ref__authors__name">Moritz R.L. </li></ul><div class="ref__title">PTMProphet: fast and accurate mass modification localization for the trans-proteomic pipeline.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">4262-4272</span><div class="ref__controls"><a href="#back-bib119" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85070371700&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31290668&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Facs.jproteome.9b00205&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=4262-4272&journal=J.%C2%A0Proteome+Res.&author=Shteynberg+D.D.&author=Deutsch+E.W.&author=Campbell+D.S.&author=Hoopmann+M.R.&author=Kusebauch+U.&author=Lee+D.&author=Mendoza+L.&author=Midha+M.K.&author=Sun+Z.&author=Whetton+A.D.&author=Moritz+R.L.&title=PTMProphet%3A+fast+and+accurate+mass+modification+localization+for+the+trans-proteomic+pipeline">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Siano M. </li><li class="ref__authors__name">Espeli V. </li><li class="ref__authors__name">Mach N. </li><li class="ref__authors__name">Bossi P. </li><li class="ref__authors__name">Licitra L. </li><li class="ref__authors__name">Ghielmini M. </li><li class="ref__authors__name">Frattini M. </li><li class="ref__authors__name">Canevari S. </li><li class="ref__authors__name">De Cecco L. </li></ul><div class="ref__title">Gene signatures and expression of miRNAs associated with efficacy of panitumumab in a head and neck cancer phase II trial.</div><span class="ref__series"><i>Oral Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">82</span></span>: <span class="ref__seriesPages">144-151</span><div class="ref__controls"><a href="#back-bib120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85047610082&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29909889&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.oraloncology.2018.05.013&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2018&pages=144-151&journal=Oral+Oncol.&author=Siano+M.&author=Espeli+V.&author=Mach+N.&author=Bossi+P.&author=Licitra+L.&author=Ghielmini+M.&author=Frattini+M.&author=Canevari+S.&author=De+Cecco+L.&title=Gene+signatures+and+expression+of+miRNAs+associated+with+efficacy+of+panitumumab+in+a+head+and+neck+cancer+phase+II+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sigismund S. </li><li class="ref__authors__name">Avanzato D. </li><li class="ref__authors__name">Lanzetti L. </li></ul><div class="ref__title">Emerging functions of the EGFR in cancer.</div><span class="ref__series"><i>Mol. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">3-20</span><div class="ref__controls"><a href="#back-bib121" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85035196496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29124875&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1002%2F1878-0261.12155&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=3-20&journal=Mol.+Oncol.&author=Sigismund+S.&author=Avanzato+D.&author=Lanzetti+L.&title=Emerging+functions+of+the+EGFR+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Singh B. </li><li class="ref__authors__name">Carpenter G. </li><li class="ref__authors__name">Coffey R.J. </li></ul><div class="ref__title">EGF receptor ligands: recent advances.</div><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span><div class="ref__controls"><a href="#back-bib153" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.12688%2Ff1000research.9025.1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&journal=F1000Res&author=Singh+B.&author=Carpenter+G.&author=Coffey+R.J.&title=EGF+receptor+ligands%3A+recent+advances">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Paull E.O. </li><li class="ref__authors__name">Stuart J.M. </li></ul><div class="ref__title">One-class detection of cell states in tumor subtypes.</div><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span><div class="ref__controls"><a href="#back-bib122" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-class+detection+of+cell+states+in+tumor+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Ji J. </li><li class="ref__authors__name">Gleason K.J. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Martignetti J.A. </li><li class="ref__authors__name">Chen L.S. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span><div class="ref__controls"><a href="#back-bib123" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Stallaert W. </li><li class="ref__authors__name">Brüggemann Y. </li><li class="ref__authors__name">Sabet O. </li><li class="ref__authors__name">Baak L. </li><li class="ref__authors__name">Gattiglio M. </li><li class="ref__authors__name">Bastiaens P.I.H. </li></ul><div class="ref__title">Contact inhibitory Eph signaling suppresses EGF-promoted cell migration by decoupling EGFR activity from vesicular recycling.</div><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">eaat0114</span><div class="ref__controls"><a href="#back-bib124" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85050958000&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30065026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscisignal.aat0114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=eaat0114&journal=Sci.+Signal.&author=Stallaert+W.&author=Br%C3%BCggemann+Y.&author=Sabet+O.&author=Baak+L.&author=Gattiglio+M.&author=Bastiaens+P.I.H.&title=Contact+inhibitory+Eph+signaling+suppresses+EGF-promoted+cell+migration+by+decoupling+EGFR+activity+from+vesicular+recycling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Szolek A. </li><li class="ref__authors__name">Schubert B. </li><li class="ref__authors__name">Mohr C. </li><li class="ref__authors__name">Sturm M. </li><li class="ref__authors__name">Feldhahn M. </li><li class="ref__authors__name">Kohlbacher O. </li></ul><div class="ref__title">OptiType: precision HLA typing from next-generation sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span><div class="ref__controls"><a href="#back-bib125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (228)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tan V.Y.F. </li><li class="ref__authors__name">Févotte C. </li></ul><div class="ref__title">Automatic relevance determination in nonnegative matrix factorization with the β-divergence.</div><span class="ref__series"><i>IEEE Trans. Pattern Anal. Mach. Intell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1592-1605</span><div class="ref__controls"><a href="#back-bib126" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84878158173&amp;cf=" onclick="newWindow(this.href);return false">Scopus (140)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23681989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1109%2FTPAMI.2012.240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2013&pages=1592-1605&journal=IEEE+Trans.+Pattern+Anal.+Mach.+Intell.&author=Tan+V.Y.F.&author=F%C3%A9votte+C.&title=Automatic+relevance+determination+in+nonnegative+matrix+factorization+with+the+%CE%B2-divergence">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tanoue T. </li><li class="ref__authors__name">Takeichi M. </li></ul><div class="ref__title">Mammalian Fat1 cadherin regulates actin dynamics and cell-cell contact.</div><span class="ref__series"><i>J. Cell Biol.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">517-528</span><div class="ref__controls"><a href="#back-bib127" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-2542438768&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=15148305&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1083%2Fjcb.200403006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2004&pages=517-528&journal=J.%C2%A0Cell+Biol.&author=Tanoue+T.&author=Takeichi+M.&title=Mammalian+Fat1+cadherin+regulates+actin+dynamics+and+cell-cell+contact">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tate J.G. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Jubb H.C. </li><li class="ref__authors__name">Sondka Z. </li><li class="ref__authors__name">Beare D.M. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Boutselakis H. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Creatore C. </li><li class="ref__authors__name">Dawson E. </li><li>et al.</li></ul><div class="ref__title">COSMIC: the catalogue of somatic mutations in cancer.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span><div class="ref__controls"><a href="#back-bib128" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (622)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+catalogue+of+somatic+mutations+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><p>Therneau, T.M. (2020). A Package for Survival Analysis in R. <a class="inter-ref-link" href="https://cran.r-project.org/web/packages/survival/index.html" target="_blank">https://cran.r-project.org/web/packages/survival/index.html</a></p><div class="ref__controls"><a href="#back-bib129" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Therneau%2C+T.M.+%282020%29.+A+Package+for+Survival+Analysis+in+R.+https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2Fsurvival%2Findex.html">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tsherniak A. </li><li class="ref__authors__name">Vazquez F. </li><li class="ref__authors__name">Montgomery P.G. </li><li class="ref__authors__name">Weir B.A. </li><li class="ref__authors__name">Kryukov G. </li><li class="ref__authors__name">Cowley G.S. </li><li class="ref__authors__name">Gill S. </li><li class="ref__authors__name">Harrington W.F. </li><li class="ref__authors__name">Pantel S. </li><li class="ref__authors__name">Krill-Burger J.M. </li><li>et al.</li></ul><div class="ref__title">Defining a cancer dependency map.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">170</span></span>: <span class="ref__seriesPages">564-576.e16</span><div class="ref__controls"><a href="#back-bib130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85026422523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (435)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28753430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=564-576.e16&journal=Cell&author=Tsherniak+A.&author=Vazquez+F.&author=Montgomery+P.G.&author=Weir+B.A.&author=Kryukov+G.&author=Cowley+G.S.&author=Gill+S.&author=Harrington+W.F.&author=Pantel+S.&author=Krill-Burger+J.M.&title=Defining+a+cancer+dependency+map">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tsou C.-C. </li><li class="ref__authors__name">Avtonomov D. </li><li class="ref__authors__name">Larsen B. </li><li class="ref__authors__name">Tucholska M. </li><li class="ref__authors__name">Choi H. </li><li class="ref__authors__name">Gingras A.-C. </li><li class="ref__authors__name">Nesvizhskii A.I. </li></ul><div class="ref__title">DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span> (<span class="refComment">257 p following 264</span>)</span>: <span class="ref__seriesPages">258-264</span><div class="ref__controls"><a href="#back-bib131" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84923838580&amp;cf=" onclick="newWindow(this.href);return false">Scopus (265)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25599550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fnmeth.3255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=258-264&journal=Nat.+Methods&author=Tsou+C.-C.&author=Avtonomov+D.&author=Larsen+B.&author=Tucholska+M.&author=Choi+H.&author=Gingras+A.-C.&author=Nesvizhskii+A.I.&title=DIA-Umpire%3A+comprehensive+computational+framework+for+data-independent+acquisition+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Uhlén M. </li><li class="ref__authors__name">Fagerberg L. </li><li class="ref__authors__name">Hallström B.M. </li><li class="ref__authors__name">Lindskog C. </li><li class="ref__authors__name">Oksvold P. </li><li class="ref__authors__name">Mardinoglu A. </li><li class="ref__authors__name">Sivertsson Å. </li><li class="ref__authors__name">Kampf C. </li><li class="ref__authors__name">Sjöstedt E. </li><li class="ref__authors__name">Asplund A. </li><li>et al.</li></ul><div class="ref__title">Proteomics. Tissue-based map of the human proteome.</div><span class="ref__series"><i>Science (New York, NY).</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">347</span></span>: <span class="ref__seriesPages">1260419</span><div class="ref__controls"><a href="#back-bib132" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84920269464&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3889)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1126%2Fscience.1260419&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1260419&journal=Science+%28New+York%2C+NY%29&author=Uhl%C3%A9n+M.&author=Fagerberg+L.&author=Hallstr%C3%B6m+B.M.&author=Lindskog+C.&author=Oksvold+P.&author=Mardinoglu+A.&author=Sivertsson+%C3%85.&author=Kampf+C.&author=Sj%C3%B6stedt+E.&author=Asplund+A.&title=Proteomics.+Tissue-based+map+of+the+human+proteome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O.A. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib133" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.A.&author=Savage+S.R.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.A.&title=Proteogenomic+analysis+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Straub P. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span><div class="ref__controls"><a href="#back-bib134" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (310)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vermorken J.B. </li><li class="ref__authors__name">Mesia R. </li><li class="ref__authors__name">Rivera F. </li><li class="ref__authors__name">Remenar E. </li><li class="ref__authors__name">Kawecki A. </li><li class="ref__authors__name">Rottey S. </li><li class="ref__authors__name">Erfan J. </li><li class="ref__authors__name">Zabolotnyy D. </li><li class="ref__authors__name">Kienzer H.-R. </li><li class="ref__authors__name">Cupissol D. </li><li>et al.</li></ul><div class="ref__title">Platinum-based chemotherapy plus cetuximab in head and neck cancer.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">359</span></span>: <span class="ref__seriesPages">1116-1127</span><div class="ref__controls"><a href="#back-bib135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-51649085865&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2087)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=18784101&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1056%2FNEJMoa0802656&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=1116-1127&journal=N.%C2%A0Engl.+J.+Med.&author=Vermorken+J.B.&author=Mesia+R.&author=Rivera+F.&author=Remenar+E.&author=Kawecki+A.&author=Rottey+S.&author=Erfan+J.&author=Zabolotnyy+D.&author=Kienzer+H.-R.&author=Cupissol+D.&title=Platinum-based+chemotherapy+plus+cetuximab+in+head+and+neck+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vermorken J.B. </li><li class="ref__authors__name">Trigo J. </li><li class="ref__authors__name">Hitt R. </li><li class="ref__authors__name">Koralewski P. </li><li class="ref__authors__name">Diaz-Rubio E. </li><li class="ref__authors__name">Rolland F. </li><li class="ref__authors__name">Knecht R. </li><li class="ref__authors__name">Amellal N. </li><li class="ref__authors__name">Schueler A. </li><li class="ref__authors__name">Baselga J. </li></ul><div class="ref__title">Open-label, uncontrolled, multicenter phase II study to evaluate the efficacy and toxicity of cetuximab as a single agent in patients with recurrent and/or metastatic squamous cell carcinoma of the head and neck who failed to respond to platinum-based therapy.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2171-2177</span><div class="ref__controls"><a href="#back-bib136" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-34250180582&amp;cf=" onclick="newWindow(this.href);return false">Scopus (694)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=17538161&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1200%2FJCO.2006.06.7447&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=2171-2177&journal=J.%C2%A0Clin.+Oncol.&author=Vermorken+J.B.&author=Trigo+J.&author=Hitt+R.&author=Koralewski+P.&author=Diaz-Rubio+E.&author=Rolland+F.&author=Knecht+R.&author=Amellal+N.&author=Schueler+A.&author=Baselga+J.&title=Open-label%2C+uncontrolled%2C+multicenter+phase+II+study+to+evaluate+the+efficacy+and+toxicity+of+cetuximab+as+a+single+agent+in+patients+with+recurrent+and%2For+metastatic+squamous+cell+carcinoma+of+the+head+and+neck+who+failed+to+respond+to+platinum-based+therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Walter V. </li><li class="ref__authors__name">Yin X. </li><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Cabanski C.R. </li><li class="ref__authors__name">Zhao N. </li><li class="ref__authors__name">Du Y. </li><li class="ref__authors__name">Ang M.K. </li><li class="ref__authors__name">Hayward M.C. </li><li class="ref__authors__name">Salazar A.H. </li><li class="ref__authors__name">Hoadley K.A. </li><li>et al.</li></ul><div class="ref__title">Molecular subtypes in head and neck cancer exhibit distinct patterns of chromosomal gain and loss of canonical cancer genes.</div><span class="ref__series"><i>PLoS One.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">e56823</span><div class="ref__controls"><a href="#back-bib137" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84874327940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=23451093&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1371%2Fjournal.pone.0056823&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e56823&journal=PLoS+One&author=Walter+V.&author=Yin+X.&author=Wilkerson+M.D.&author=Cabanski+C.R.&author=Zhao+N.&author=Du+Y.&author=Ang+M.K.&author=Hayward+M.C.&author=Salazar+A.H.&author=Hoadley+K.A.&title=Molecular+subtypes+in+head+and+neck+cancer+exhibit+distinct+patterns+of+chromosomal+gain+and+loss+of+canonical+cancer+genes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Song M. </li></ul><div class="ref__title">Ckmeans.1d.dp: optimal k-means clustering in one dimension by dynamic programming.</div><span class="ref__series"><i>R. J.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">29-33</span><div class="ref__controls"><a href="#back-bib138" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27942416&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.32614%2FRJ-2011-015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=29-33&journal=R.%C2%A0J.&author=Wang+H.&author=Song+M.&title=Ckmeans.1d.dp%3A+optimal+k-means+clustering+in+one+dimension+by+dynamic+programming">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Ma Z. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Chan D.W. </li><li class="ref__authors__name">Ellis M.J.C. </li><li class="ref__authors__name">Townsend R.R. </li><li class="ref__authors__name">Smith R.D. </li><li>et al.</li></ul><div class="ref__title">Proteome profiling outperforms transcriptome profiling for coexpression based gene function prediction.</div><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">121-134</span><div class="ref__controls"><a href="#back-bib139" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85008490068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=27836980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=121-134&journal=Mol.+Cell+Proteomics&author=Wang+J.&author=Ma+Z.&author=Carr+S.A.&author=Mertins+P.&author=Zhang+H.&author=Zhang+Z.&author=Chan+D.W.&author=Ellis+M.J.C.&author=Townsend+R.R.&author=Smith+R.D.&title=Proteome+profiling+outperforms+transcriptome+profiling+for+coexpression+based+gene+function+prediction">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Lawrence J.C. </li><li class="ref__authors__name">Sturgill T.W. </li><li class="ref__authors__name">Harris T.E. </li></ul><div class="ref__title">Mammalian target of rapamycin complex 1 (mTORC1) activity is associated with phosphorylation of raptor by mTOR.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">284</span></span>: <span class="ref__seriesPages">14693-14697</span><div class="ref__controls"><a href="#back-bib140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-67649344456&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19346248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fjbc.C109.002907&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=14693-14697&journal=J.%C2%A0Biol.+Chem.&author=Wang+L.&author=Lawrence+J.C.&author=Sturgill+T.W.&author=Harris+T.E.&title=Mammalian+target+of+rapamycin+complex+1+%28mTORC1%29+activity+is+associated+with+phosphorylation+of+raptor+by+mTOR">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Zhang J. </li><li class="ref__authors__name">Wan L. </li><li class="ref__authors__name">Zhou X. </li><li class="ref__authors__name">Wang Z. </li><li class="ref__authors__name">Wei W. </li></ul><div class="ref__title">Targeting Cdc20 as a novel cancer therapeutic strategy.</div><span class="ref__series"><i>Pharmacol. Ther.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">151</span></span>: <span class="ref__seriesPages">141-151</span><div class="ref__controls"><a href="#back-bib141" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84930088030&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=25850036&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1016%2Fj.pharmthera.2015.04.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2015&pages=141-151&journal=Pharmacol.+Ther.&author=Wang+L.&author=Zhang+J.&author=Wan+L.&author=Zhou+X.&author=Wang+Z.&author=Wei+W.&title=Targeting+Cdc20+as+a+novel+cancer+therapeutic+strategy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Slebos R.J.C. </li><li class="ref__authors__name">Wang D. </li><li class="ref__authors__name">Halvey P.J. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Liebler D.C. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Protein identification using customized protein sequence databases derived from RNA-seq data.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span><div class="ref__controls"><a href="#back-bib142" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (117)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.C.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases+derived+from+RNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">3235-3237</span><div class="ref__controls"><a href="#back-bib143" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84894110748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (87)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24058055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtt543&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=3235-3237&journal=Bioinformatics&author=Wang+X.&author=Zhang+B.&title=customProDB%3A+an+R+package+to+generate+customized+protein+databases+from+RNA-Seq+data+for+proteomics+search">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wee P. </li><li class="ref__authors__name">Wang Z. </li></ul><div class="ref__title">Epidermal growth factor receptor cell proliferation signaling pathways.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">52</span><div class="ref__controls"><a href="#back-bib144" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85019993185&amp;cf=" onclick="newWindow(this.href);return false">Scopus (377)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28513565&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.3390%2Fcancers9050052&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=52&journal=Cancers+%28Basel%29&author=Wee+P.&author=Wang+Z.&title=Epidermal+growth+factor+receptor+cell+proliferation+signaling+pathways">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Li K. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span><div class="ref__controls"><a href="#back-bib145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span><div class="ref__controls"><a href="#back-bib146" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Whitfield M.L. </li><li class="ref__authors__name">Sherlock G. </li><li class="ref__authors__name">Saldanha A.J. </li><li class="ref__authors__name">Murray J.I. </li><li class="ref__authors__name">Ball C.A. </li><li class="ref__authors__name">Alexander K.E. </li><li class="ref__authors__name">Matese J.C. </li><li class="ref__authors__name">Perou C.M. </li><li class="ref__authors__name">Hurt M.M. </li><li class="ref__authors__name">Brown P.O. </li><li class="ref__authors__name">Botstein D. </li></ul><div class="ref__title">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.</div><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span><div class="ref__controls"><a href="#back-bib147" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1095)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Schulz M.H. </li><li class="ref__authors__name">Long Q. </li><li class="ref__authors__name">Apweiler R. </li><li class="ref__authors__name">Ning Z. </li></ul><div class="ref__title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span><div class="ref__controls"><a href="#back-bib148" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1151)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoshihara K. </li><li class="ref__authors__name">Shahmoradgoli M. </li><li class="ref__authors__name">Martínez E. </li><li class="ref__authors__name">Vegesna R. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Torres-Garcia W. </li><li class="ref__authors__name">Treviño V. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Levine D.A. </li><li>et al.</li></ul><div class="ref__title">Inferring tumour purity and stromal and immune cell admixture from expression data.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span><div class="ref__controls"><a href="#back-bib149" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1032)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zecha J. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Kanashova T. </li><li class="ref__authors__name">Avanessian S.C. </li><li class="ref__authors__name">Kane M.H. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Kuster B. </li></ul><div class="ref__title">TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span><div class="ref__controls"><a href="#back-bib150" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (41)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell.+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+a+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang J. </li><li class="ref__authors__name">White N.M. </li><li class="ref__authors__name">Schmidt H.K. </li><li class="ref__authors__name">Fulton R.S. </li><li class="ref__authors__name">Tomlinson C. </li><li class="ref__authors__name">Warren W.C. </li><li class="ref__authors__name">Wilson R.K. </li><li class="ref__authors__name">Maher C.A. </li></ul><div class="ref__title">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span><div class="ref__controls"><a href="#back-bib151" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhao S. </li><li class="ref__authors__name">Gordon W. </li><li class="ref__authors__name">Du S. </li><li class="ref__authors__name">Zhang C. </li><li class="ref__authors__name">He W. </li><li class="ref__authors__name">Xi L. </li><li class="ref__authors__name">Mathur S. </li><li class="ref__authors__name">Agostino M. </li><li class="ref__authors__name">Paradis T. </li><li class="ref__authors__name">von Schack D. </li><li>et al.</li></ul><div class="ref__title">QuickMIRSeq: a pipeline for quick and accurate quantification of both known miRNAs and isomiRs by jointly processing multiple samples from microRNA sequencing.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">180</span><div class="ref__controls"><a href="#back-bib152" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=2-s2.0-85015748083&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=28320324&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2020.12.007&amp;key=10.1186%2Fs12859-017-1601-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=180&journal=BMC+Bioinformatics&author=Zhao+S.&author=Gordon+W.&author=Du+S.&author=Zhang+C.&author=He+W.&author=Xi+L.&author=Mathur+S.&author=Agostino+M.&author=Paradis+T.&author=von+Schack+D.&title=QuickMIRSeq%3A+a+pipeline+for+quick+and+accurate+quantification+of+both+known+miRNAs+and+isomiRs+by+jointly+processing+multiple+samples+from+microRNA+sequencing">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: January 7, 2021</div><div class="article-info__date">Accepted:
            December 7,
            2020</div><div class="article-info__date">Received in revised form:
            September 13,
            2020</div><div class="article-info__date">Received:
            May 12,
            2020</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.ccell.2020.12.007">https://doi.org/10.1016/j.ccell.2020.12.007</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2021 Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S1535610820306553">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S1535610820306553" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cancer-cell/fulltext/S1535-6108(20)30655-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610820306553%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cancer-cell/fulltext/S1535-6108(20)30655-3?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610820306553%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"1535-6108","name":"Cancer Cell"},"page":{"loadTimestamp":"1723231681275","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"39","issueNumber":"3","viewState":"fulltext","format":"MIME-HTML","id":"pii:S1535610820306553","coverDate":"202103","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Proteogenomic insights into the biology and treatment of HPVnegative head and neck squamous cell carcinoma","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
