<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="5a84b42a-4430-4954-b651-d489825ae8f7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:xcrm;pageGroup:string:Article View;requestedJournal:journal:xcrm;page:string:Show Full Text;wgroup:string:Default Website Group;article:article:pii\:S2666379120300045;subPage:string:Full Text;issue:issue:pii\:S2666379120X00024"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cell-reports-medicine/abstract/S2666-3791(20)30004-5" />
    



        <meta property="og:site_name" content="Cell Reports Medicine" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg" />
    


        <meta property="og:description" content="McDermott et&nbsp;al. present the proteogenomic analysis of prospectively collected ovarian
high-grade serous cancer samples and appropriate normal precursor samples under tight
ischemic control. They identify tumor-associated signaling pathways and mitotic and
cyclin-dependent kinases as key oncogenic drivers potentially related to chromosomal
instability." />
    











        <meta name="twitter:title" content="Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg" />
    


        <meta name="twitter:description" content="McDermott et&nbsp;al. present the proteogenomic analysis of prospectively collected ovarian
high-grade serous cancer samples and appropriate normal precursor samples under tight
ischemic control. They identify tumor-associated signaling pathways and mitotic and
cyclin-dependent kinases as key oncogenic drivers potentially related to chromosomal
instability." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(20)30004-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666379120300045%3Fshowall%3Dtrue">
    


<title>Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability: Cell Reports Medicine</title><meta name="citation_journal_title" content="Cell Reports Medicine"/> <meta name="citation_journal_abbrev" content="CR Med"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability"/> <meta name="citation_date" content="2020/04/21"/> <meta name="citation_online_date" content="2020/04/09"/> <meta name="citation_volume" content="1"/> <meta name="citation_issue" content="1"/> <meta name="citation_pmid" content="32529193"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S2666379120300045/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S2666379120300045/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S2666379120300045/pdf"/> <meta name="citation_issn" content="2666-3791"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.xcrm.2020.100004"/> <meta name="citation_pmid" content="32529193"/> <meta name="citation_author" content="Jason E. McDermott"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author_institution" content="Department of Molecular Microbiology and Immunology, Oregon Health & Science University, Portland, OR 97201, USA"></meta><meta name="citation_author" content="Osama A. Arshad"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Vladislav A. Petyuk"></meta><meta name="citation_author_orcid" content="0000-0003-4076-151X"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Yi Fu"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA"></meta><meta name="citation_author" content="Marina A. Gritsenko"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Therese R. Clauss"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Ronald J. Moore"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Athena A. Schepmoes"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Rui Zhao"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Matthew E. Monroe"></meta><meta name="citation_author_orcid" content="0000-0002-1210-7169"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Michael Schnaubelt"></meta><meta name="citation_author_orcid" content="0000-0001-8617-894X"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA"></meta><meta name="citation_author" content="Chia-Feng Tsai"></meta><meta name="citation_author_orcid" content="0000-0002-6514-6911"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author_orcid" content="0000-0002-8351-1994"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Liang-Bo Wang"></meta><meta name="citation_author_orcid" content="0000-0001-6977-9348"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA"></meta><meta name="citation_author" content="Steven Foltz"></meta><meta name="citation_author_orcid" content="0000-0002-9526-8194"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA"></meta><meta name="citation_author" content="Matthew Wyczalkowski"></meta><meta name="citation_author_orcid" content="0000-0003-4197-551X"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA"></meta><meta name="citation_author" content="Yige Wu"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA"></meta><meta name="citation_author" content="Ehwang Song"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Molly A. Brewer"></meta><meta name="citation_author_institution" content="Department of Obstetrics and Gynecology, University of Connecticut, Farmington, CT 06030, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research, Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_orcid" content="0000-0001-5019-4374"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_orcid" content="0000-0002-4593-4232"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author_email" content="tao.liu@pnnl.gov"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author_email" content="karin.rodland@pnnl.gov"></meta><meta name="citation_author_orcid" content="0000-0001-7070-6541"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author_institution" content="Department of Cell, Developmental, and Cancer Biology, Oregon Health & Science University, Portland, OR 97201, USA"></meta><meta name="citation_keywords" content="proteogenomics; ovarian cancer; phosphoproteomics; proteomics; fallopian tube; homologous repair deficiency; replication stress"></meta><meta name="citation_reference" content="citation_title=Cancer statistics, 2018; citation_author=R.L. Siegel; citation_author=K.D. Miller; citation_author=A. Jemal; citation_journal_title=CA Cancer J. Clin.; citation_volume=68; citation_firstpage=7; citation_lastpage=30; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Key Statistics for Ovarian Cancer; citation_author=American Cancer Society; "></meta><meta name="citation_reference" content="citation_title=Integrated genomic analyses of ovarian carcinoma; citation_author=Cancer Genome Atlas Research Network; citation_journal_title=Nature; citation_volume=474; citation_firstpage=609; citation_lastpage=615; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=SEER Cancer Statistics Review, 1975-2015; citation_author=National Cancer Institute; "></meta><meta name="citation_reference" content="citation_title=Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer; citation_author=H. Zhang; citation_author=T. Liu; citation_author=Z. Zhang; citation_author=S.H. Payne; citation_author=B. Zhang; citation_author=J.E. McDermott; citation_author=J.Y. Zhou; citation_author=V.A. Petyuk; citation_author=L. Chen; citation_author=D. Ray; citation_author=CPTAC Investigators; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=755; citation_lastpage=765; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels; citation_author=P. Mertins; citation_author=F. Yang; citation_author=T. Liu; citation_author=D.R. Mani; citation_author=V.A. Petyuk; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=J.W. Qiao; citation_author=M.A. Gritsenko; citation_author=R.J. Moore; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=13; citation_firstpage=1690; citation_lastpage=1704; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=FLAGS, frequently mutated genes in public exomes; citation_author=C. Shyr; citation_author=M. Tarailo-Graovac; citation_author=M. Gottlieb; citation_author=J.J. Lee; citation_author=C. van Karnebeek; citation_author=W.W. Wasserman; citation_journal_title=BMC Med. Genomics; citation_volume=7; citation_firstpage=64; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Integrated genomic analyses of ovarian carcinoma; citation_author=Cancer Genome Atlas Research Network; citation_journal_title=Nature; citation_volume=474; citation_firstpage=609; citation_lastpage=615; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles; citation_author=A. Subramanian; citation_author=P. Tamayo; citation_author=V.K. Mootha; citation_author=S. Mukherjee; citation_author=B.L. Ebert; citation_author=M.A. Gillette; citation_author=A. Paulovich; citation_author=S.L. Pomeroy; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=15545; citation_lastpage=15550; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Targeted Proteomic Analyses of Histone H4 Acetylation Changes Associated with Homologous-Recombination-Deficient High-Grade Serous Ovarian Carcinomas; citation_author=S.N. Thomas; citation_author=L. Chen; citation_author=Y. Liu; citation_author=N. Höti; citation_author=H. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=16; citation_firstpage=3704; citation_lastpage=3710; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=On the Dependency of Cellular Protein Levels on mRNA Abundance; citation_author=Y. Liu; citation_author=A. Beyer; citation_author=R. Aebersold; citation_journal_title=Cell; citation_volume=165; citation_firstpage=535; citation_lastpage=550; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of human colon and rectal cancer; citation_author=B. Zhang; citation_author=J. Wang; citation_author=X. Wang; citation_author=J. Zhu; citation_author=Q. Liu; citation_author=Z. Shi; citation_author=M.C. Chambers; citation_author=L.J. Zimmerman; citation_author=K.F. Shaddox; citation_author=S. Kim; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=382; citation_lastpage=387; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of human colon cancer reveals new therapeutic opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V. Petyuk; citation_author=S. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O. Arshad; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Molecular analysis of high-grade serous ovarian carcinoma with and without associated serous tubal intra-epithelial carcinoma; citation_author=J. Ducie; citation_author=F. Dao; citation_author=M. Considine; citation_author=N. Olvera; citation_author=P.A. Shaw; citation_author=R.J. Kurman; citation_author=I.M. Shih; citation_author=R.A. Soslow; citation_author=L. Cope; citation_author=D.A. Levine; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=990; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=High grade serous ovarian carcinomas originate in the fallopian tube; citation_author=S.I. Labidi-Galy; citation_author=E. Papp; citation_author=D. Hallberg; citation_author=N. Niknafs; citation_author=V. Adleff; citation_author=M. Noe; citation_author=R. Bhattacharya; citation_author=M. Novak; citation_author=S. Jones; citation_author=J. Phallen; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=8; citation_firstpage=1093; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Kinase-substrate enrichment analysis provides insights into the heterogeneity of signaling pathway activation in leukemia cells; citation_author=P. Casado; citation_author=J.C. Rodriguez-Prados; citation_author=S.C. Cosulich; citation_author=S. Guichard; citation_author=B. Vanhaesebroeck; citation_author=S. Joel; citation_author=P.R. Cutillas; citation_journal_title=Sci. Signal.; citation_volume=6; citation_firstpage=rs6; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics; citation_author=D.D. Wiredja; citation_author=M. Koyutürk; citation_author=M.R. Chance; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=3489; citation_lastpage=3491; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse; citation_author=P.V. Hornbeck; citation_author=J.M. Kornhauser; citation_author=S. Tkachev; citation_author=B. Zhang; citation_author=E. Skrzypek; citation_author=B. Murray; citation_author=V. Latham; citation_author=M. Sullivan; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=D261; citation_lastpage=D270; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Death-associated protein kinase 1 mediates interleukin-1β production through regulating inflammasome activation in Bv2 microglial cells and mice; citation_author=L. Song; citation_author=L. Pei; citation_author=L. Hu; citation_author=S. Pan; citation_author=W. Xiong; citation_author=M. Liu; citation_author=Y. Wu; citation_author=Y. Shang; citation_author=S. Yao; citation_journal_title=Sci. Rep.; citation_volume=8; citation_firstpage=9930; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The tumor suppressor DAP kinase is a target of RSK-mediated survival signaling; citation_author=R. Anjum; citation_author=P.P. Roux; citation_author=B.A. Ballif; citation_author=S.P. Gygi; citation_author=J. Blenis; citation_journal_title=Curr. Biol.; citation_volume=15; citation_firstpage=1762; citation_lastpage=1767; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=The retinoblastoma tumor-suppressor gene, the exception that proves the rule; citation_author=D.W. Goodrich; citation_journal_title=Oncogene; citation_volume=25; citation_firstpage=5233; citation_lastpage=5243; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=RB1 dual role in proliferation and apoptosis: cell fate control and implications for cancer therapy; citation_author=P. Indovina; citation_author=F. Pentimalli; citation_author=N. Casini; citation_author=I. Vocca; citation_author=A. Giordano; citation_journal_title=Oncotarget; citation_volume=6; citation_firstpage=17873; citation_lastpage=17890; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Coordinated regulation of life and death by RB; citation_author=B.N. Chau; citation_author=J.Y.J. Wang; citation_journal_title=Nat. Rev. Cancer; citation_volume=3; citation_firstpage=130; citation_lastpage=138; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=Differential phosphorylation of the retinoblastoma protein by G1/S cyclin-dependent kinases; citation_author=T. Zarkowska; citation_author=S. Mittnacht; citation_journal_title=J. Biol. Chem.; citation_volume=272; citation_firstpage=12738; citation_lastpage=12746; citation_publication_date=1997; "></meta><meta name="citation_reference" content="citation_title=Cell cycle regulation of CDK2 activity by phosphorylation of Thr160 and Tyr15; citation_author=Y. Gu; citation_author=J. Rosenblatt; citation_author=D.O. Morgan; citation_journal_title=EMBO J.; citation_volume=11; citation_firstpage=3995; citation_lastpage=4005; citation_publication_date=1992; "></meta><meta name="citation_reference" content="citation_title=Essential role for Cdk2 inhibitory phosphorylation during replication stress revealed by a human Cdk2 knockin mutation; citation_author=B.T. Hughes; citation_author=J. Sidorova; citation_author=J. Swanger; citation_author=R.J. Monnat; citation_author=B.E. Clurman; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=8954; citation_lastpage=8959; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Cyclin E deregulation promotes loss of specific genomic regions; citation_author=L.K. Teixeira; citation_author=X. Wang; citation_author=Y. Li; citation_author=S. Ekholm-Reed; citation_author=X. Wu; citation_author=P. Wang; citation_author=S.I. Reed; citation_journal_title=Curr. Biol.; citation_volume=25; citation_firstpage=1327; citation_lastpage=1333; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=DNA replication stress: causes, resolution and disease; citation_author=A. Mazouzi; citation_author=G. Velimezi; citation_author=J.I. Loizou; citation_journal_title=Exp. Cell Res.; citation_volume=329; citation_firstpage=85; citation_lastpage=93; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=DNA damage response and cancer therapeutics through the lens of the Fanconi Anemia DNA repair pathway; citation_author=S. Bhattacharjee; citation_author=S. Nandi; citation_journal_title=Cell Commun. Signal.; citation_volume=15; citation_firstpage=41; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Activation of the FA pathway mediated by phosphorylation and ubiquitination; citation_author=M. Ishiai; citation_author=K. Sato; citation_author=J. Tomida; citation_author=H. Kitao; citation_author=H. Kurumizaka; citation_author=M. Takata; citation_journal_title=Mutat. Res.; citation_volume=803-805; citation_firstpage=89; citation_lastpage=95; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Modulation of p53β and p53γ expression by regulating the alternative splicing of TP53 gene modifies cellular response; citation_author=V. Marcel; citation_author=K. Fernandes; citation_author=O. Terrier; citation_author=D.P. Lane; citation_author=J.C. Bourdon; citation_journal_title=Cell Death Differ.; citation_volume=21; citation_firstpage=1377; citation_lastpage=1387; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Protein kinases that phosphorylate splicing factors: roles in cancer development, progression and possible therapeutic options; citation_author=A. Czubaty; citation_author=A. Piekiełko-Witkowska; citation_journal_title=Int. J. Biochem. Cell Biol.; citation_volume=91; citation_number=Pt B; citation_firstpage=102; citation_lastpage=115; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Synthetic Lethal Strategy Identifies a Potent and Selective TTK and CLK1/2 Inhibitor for Treatment of Triple-Negative Breast Cancer with a Compromised G1-S Checkpoint; citation_author=D. Zhu; citation_author=S. Xu; citation_author=G. Deyanat-Yazdi; citation_author=S.X. Peng; citation_author=L.A. Barnes; citation_author=R.K. Narla; citation_author=T. Tran; citation_author=D. Mikolon; citation_author=Y. Ning; citation_author=T. Shi; citation_author=et al.; citation_journal_title=Mol. Cancer Ther.; citation_volume=17; citation_firstpage=1727; citation_lastpage=1738; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Pathogenesis and heterogeneity of ovarian cancer; citation_author=P.T. Kroeger; citation_author=R. Drapkin; citation_journal_title=Curr. Opin. Obstet. Gynecol.; citation_volume=29; citation_firstpage=26; citation_lastpage=34; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Cell Origins of High-Grade Serous Ovarian Cancer; citation_author=J. Kim; citation_author=E.Y. Park; citation_author=O. Kim; citation_author=J.M. Schilder; citation_author=D.M. Coffey; citation_author=C.H. Cho; citation_author=R.C. Bast; citation_journal_title=Cancers (Basel); citation_volume=10; citation_firstpage=E433; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma; citation_author=D.J. Clark; citation_author=S.M. Dhanasekaran; citation_author=F. Petralia; citation_author=J. Pan; citation_author=X. Song; citation_author=Y. Hu; citation_author=F. da Veiga Leprevost; citation_author=B. Reva; citation_author=T.M. Lih; citation_author=H.Y. Chang; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=179; citation_firstpage=964; citation_lastpage=983.e31; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Multi-Omics Profiling Reveals Distinct Microenvironment Characterization and Suggests Immune Escape Mechanisms of Triple-Negative Breast Cancer; citation_author=Y. Xiao; citation_author=D. Ma; citation_author=S. Zhao; citation_author=C. Suo; citation_author=J. Shi; citation_author=M.Z. Xue; citation_author=M. Ruan; citation_author=H. Wang; citation_author=J. Zhao; citation_author=Q. Li; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_doi=10.1158/1078-0432.CCR-18-3524; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=CD8+ and Regulatory T cells Differentiate Tumor Immune Phenotypes and Predict Survival in Locally Advanced Head and Neck Cancer; citation_author=A. Echarti; citation_author=M. Hecht; citation_author=M. Büttner-Herold; citation_author=M. Haderlein; citation_author=A. Hartmann; citation_author=R. Fietkau; citation_author=L. Distel; citation_journal_title=Cancers (Basel); citation_volume=11; citation_firstpage=E1398; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Translating the role of PARP inhibitors in triple-negative breast cancer; citation_author=M. Beniey; citation_author=T. Haque; citation_author=S. Hassan; citation_journal_title=Oncoscience; citation_volume=6; citation_firstpage=287; citation_lastpage=288; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=DNA replication stress and cancer: cause or cure?; citation_author=E.M. Taylor; citation_author=H.D. Lindsay; citation_journal_title=Future Oncol.; citation_volume=12; citation_firstpage=221; citation_lastpage=237; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Correcting systematic bias and instrument measurement drift with mzRefinery; citation_author=B.C. Gibbons; citation_author=M.C. Chambers; citation_author=M.E. Monroe; citation_author=D.L. Tabb; citation_author=S.H. Payne; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=3838; citation_lastpage=3840; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features; citation_author=M.E. Monroe; citation_author=J.L. Shaw; citation_author=D.S. Daly; citation_author=J.N. Adkins; citation_author=R.D. Smith; citation_journal_title=Comput. Biol. Chem.; citation_volume=32; citation_firstpage=215; citation_lastpage=217; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=ProteoWizard: open source software for rapid proteomics tools development; citation_author=D. Kessner; citation_author=M. Chambers; citation_author=R. Burke; citation_author=D. Agus; citation_author=P. Mallick; citation_journal_title=Bioinformatics; citation_volume=24; citation_firstpage=2534; citation_lastpage=2536; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=COSMIC: somatic cancer genetics at high-resolution; citation_author=S.A. Forbes; citation_author=D. Beare; citation_author=H. Boutselakis; citation_author=S. Bamford; citation_author=N. Bindal; citation_author=J. Tate; citation_author=C.G. Cole; citation_author=S. Ward; citation_author=E. Dawson; citation_author=L. Ponting; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D777; citation_lastpage=D783; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs; citation_author=C.T. Saunders; citation_author=W.S. Wong; citation_author=S. Swamy; citation_author=J. Becq; citation_author=L.J. Murray; citation_author=R.K. Cheetham; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=1811; citation_lastpage=1817; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=D.C. Koboldt; citation_author=Q. Zhang; citation_author=D.E. Larson; citation_author=D. Shen; citation_author=M.D. McLellan; citation_author=L. Lin; citation_author=C.A. Miller; citation_author=E.R. Mardis; citation_author=L. Ding; citation_author=R.K. Wilson; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=568; citation_lastpage=576; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads; citation_author=K. Ye; citation_author=M.H. Schulz; citation_author=Q. Long; citation_author=R. Apweiler; citation_author=Z. Ning; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2865; citation_lastpage=2871; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; citation_author=S.T. Sherry; citation_author=M.H. Ward; citation_author=M. Kholodov; citation_author=J. Baker; citation_author=L. Phan; citation_author=E.M. Smigielski; citation_author=K. Sirotkin; citation_journal_title=Nucleic Acids Res.; citation_volume=29; citation_firstpage=308; citation_lastpage=311; citation_publication_date=2001; "></meta><meta name="citation_reference" content="citation_title=CrossMap: a versatile tool for coordinate conversion between genome assemblies; citation_author=H. Zhao; citation_author=Z. Sun; citation_author=J. Wang; citation_author=H. Huang; citation_author=J.P. Kocher; citation_author=L. Wang; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=1006; citation_lastpage=1007; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=M.A. DePristo; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=The Ensembl Variant Effect Predictor; citation_author=W. McLaren; citation_author=L. Gil; citation_author=S.E. Hunt; citation_author=H.S. Riat; citation_author=G.R. Ritchie; citation_author=A. Thormann; citation_author=P. Flicek; citation_author=F. Cunningham; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=122; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=CharGer: Clinical Characterization of Germline Variants; citation_author=A.D. Scott; citation_author=K.L. Huang; citation_author=A. Weerasinghe; citation_author=R.J. Mashl; citation_author=Q. Gao; citation_author=F. Martins Rodrigues; citation_author=M.A. Wyczalkowski; citation_author=L. Ding; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=865; citation_lastpage=867; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The ExAC browser: displaying reference data information from over 60 000 exomes; citation_author=K.J. Karczewski; citation_author=B. Weisburd; citation_author=B. Thomas; citation_author=M. Solomonson; citation_author=D.M. Ruderfer; citation_author=D. Kavanagh; citation_author=T. Hamamsy; citation_author=M. Lek; citation_author=K.E. Samocha; citation_author=B.B. Cummings; citation_author=The Exome Aggregation Consortium; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D840; citation_lastpage=D845; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=ClinVar: public archive of interpretations of clinically relevant variants; citation_author=M.J. Landrum; citation_author=J.M. Lee; citation_author=M. Benson; citation_author=G. Brown; citation_author=C. Chao; citation_author=S. Chitipiralla; citation_author=B. Gu; citation_author=J. Hart; citation_author=D. Hoffman; citation_author=J. Hoover; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=D1; citation_firstpage=D862; citation_lastpage=D868; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm; citation_author=P. Kumar; citation_author=S. Henikoff; citation_author=P.C. Ng; citation_journal_title=Nat. Protoc.; citation_volume=4; citation_firstpage=1073; citation_lastpage=1081; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Predicting functional effect of human missense mutations using PolyPhen-2; citation_author=I. Adzhubei; citation_author=D.M. Jordan; citation_author=S.R. Sunyaev; citation_journal_title=Curr. Protoc. Hum. Genet.; citation_volume=Chapter 7; citation_number=January; citation_firstpage=Unit 7.20; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Integrative genomics viewer; citation_author=J.T. Robinson; citation_author=H. Thorvaldsdóttir; citation_author=W. Winckler; citation_author=M. Guttman; citation_author=E.S. Lander; citation_author=G. Getz; citation_author=J.P. Mesirov; citation_journal_title=Nat. Biotechnol.; citation_volume=29; citation_firstpage=24; citation_lastpage=26; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=multiOmicsViz: plot the effect of one omics data on other omics data along the chromosome; citation_author=J. Wang; "></meta><meta name="citation_reference" content="citation_title=clusterProfiler: an R package for comparing biological themes among gene clusters; citation_author=G. Yu; citation_author=L.G. Wang; citation_author=Y. Han; citation_author=Q.Y. He; citation_journal_title=OMICS; citation_volume=16; citation_firstpage=284; citation_lastpage=287; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=M.E. Ritchie; citation_author=B. Phipson; citation_author=D. Wu; citation_author=Y. Hu; citation_author=C.W. Law; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=e47; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Reproducibility of Differential Proteomic Technologies in CPTAC Fractionated Xenografts; citation_author=D.L. Tabb; citation_author=X. Wang; citation_author=S.A. Carr; citation_author=K.R. Clauser; citation_author=P. Mertins; citation_author=M.C. Chambers; citation_author=J.D. Holman; citation_author=J. Wang; citation_author=B. Zhang; citation_author=L.J. Zimmerman; citation_author=et al.; citation_journal_title=J. Proteome Res.; citation_volume=15; citation_firstpage=691; citation_lastpage=706; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts; citation_author=S. Li; citation_author=D. Shen; citation_author=J. Shao; citation_author=R. Crowder; citation_author=W. Liu; citation_author=A. Prat; citation_author=X. He; citation_author=S. Liu; citation_author=J. Hoog; citation_author=C. Lu; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=4; citation_firstpage=1116; citation_lastpage=1130; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology; citation_author=S. Richards; citation_author=N. Aziz; citation_author=S. Bale; citation_author=D. Bick; citation_author=S. Das; citation_author=J. Gastier-Foster; citation_author=W.W. Grody; citation_author=M. Hegde; citation_author=E. Lyon; citation_author=E. Spector; citation_author=ACMG Laboratory Quality Assurance Committee; citation_author=et al.; citation_journal_title=Genet. Med.; citation_volume=17; citation_firstpage=405; citation_lastpage=424; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Pathogenic Germline Variants in 10,389 Adult Cancers; citation_author=K.L. Huang; citation_author=R.J. Mashl; citation_author=Y. Wu; citation_author=D.I. Ritter; citation_author=J. Wang; citation_author=C. Oh; citation_author=M. Paczkowska; citation_author=S. Reynolds; citation_author=M.A. Wyczalkowski; citation_author=N. Oak; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=355; citation_lastpage=370.e1–e6; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases; citation_author=S. Kim; citation_author=N. Gupta; citation_author=P.A. Pevzner; citation_journal_title=J. Proteome Res.; citation_volume=7; citation_firstpage=3354; citation_lastpage=3363; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=A probability-based approach for high-throughput protein phosphorylation analysis and site localization; citation_author=S.A. Beausoleil; citation_author=J. Villén; citation_author=S.A. Gerber; citation_author=J. Rush; citation_author=S.P. Gygi; citation_journal_title=Nat. Biotechnol.; citation_volume=24; citation_firstpage=1285; citation_lastpage=1292; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O.A. Arshad; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=NetworKIN: a resource for exploring cellular phosphorylation networks; citation_author=R. Linding; citation_author=L.J. Jensen; citation_author=A. Pasculescu; citation_author=M. Olhovsky; citation_author=K. Colwill; citation_author=P. Bork; citation_author=M.B. Yaffe; citation_author=T. Pawson; citation_journal_title=Nucleic Acids Res.; citation_volume=36; citation_firstpage=D695; citation_lastpage=D699; citation_publication_date=2008; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;In the absence of a dominant driving mutation other than uniformly present &lt;i&gt;TP53&lt;/i&gt; mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-xcrm.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell-reports-medicine%2Ffulltext%2FS2666-3791%2820%2930004-5%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS2666379120300045%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(20)30004-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666379120300045%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cell-reports-medicine/fulltext/S2666-3791(20)30004-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666379120300045%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell-reports-medicine%2Ffulltext%2FS2666-3791%2820%2930004-5%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS2666379120300045%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="xcrm">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos descender">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell-reports-medicine/home" title="Cell Reports Medicine "><img alt="Cell Reports Medicine " src="/pb-assets/ux3/logos/cell/CP21491_CRMedicine_40pxH_1LineLogo_WO-1632424456153.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos descender">
            

            <span data-widget-id="44c1ca1f-d146-4b99-b00f-af08b03feabc" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.8.1644587109507&amp;__hsfp=3253153019" title="All content is freely available to readers and supported through open access"><img alt="All content is freely available to readers and supported through open access" src="/pb-assets/ux3/logos/cell/Open_access_border-1657136680560.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="74a132e6-760d-4f9e-9deb-118a977ec63a" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cr-medicine" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell-reports-medicine%2Ffulltext%2FS2666-3791%2820%2930004-5%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS2666379120300045%253Fshowall%253Dtrue&amp;pii=S2666-3791%2820%2930004-5" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=xcrm&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell-reports-medicine%2Ffulltext%2FS2666-3791%2820%2930004-5%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS2666379120300045%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cr-medicine" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell-reports-medicine%2Ffulltext%2FS2666-3791%2820%2930004-5%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS2666379120300045%253Fshowall%253Dtrue&amp;pii=S2666-3791%2820%2930004-5" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.xcrm.2020.100004">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S2666-3791(23)X0008-1"><img src="/cms/asset/atypon:cms:attachment:img:d100e6:rev:1720834361070-3656:pii:S2666379123X00081/cover.tif.jpg" alt="Cover Image - Cell Reports Medicine, Volume 5, Issue 7"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cr-medicine?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694797673397.1694802293413.149&amp;__hssc=25856994.3.1694802293413&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="6860b84d-aff3-4022-bf58-787d0f6794d3" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=xcrm" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=xcrm&amp;ISSN=2666-3791&amp;journalCode=xcrm" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="xcrm"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-6860b84d-aff3-4022-bf58-787d0f6794d3" title="This journal" value="xcrm" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-6860b84d-aff3-4022-bf58-787d0f6794d3" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=xcrm&amp;ISSN=2666-3791&amp;journalCode=xcrm" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f47373e1-34e3-4b89-82d5-e9ff3efe5438" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S2666-3791(20)30004-5"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Article</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cell-reports-medicine/issue?pii=S2666-3791(20)X0002-4" class="article-header__vol faded">
                                                    Volume 1, ISSUE 1</a><span>, </span><span>100004</span><span>, </span><span class="article-header__date faded">April 21, 2020</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=xcrm&amp;volume=1&amp;issue=1&amp;pii=S2666-3791%2820%29X0002-4" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__pdf"><a href="#" title="open PDF menu" data-db-target-for="article-tools-pdf" aria-expanded="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF</span><span class="alternative">PDF</span></a><div data-db-target-of="article-tools-pdf" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="/action/showPdf?pii=S2666-3791%2820%2930004-5" title="Standard PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayStandardPdf ">Standard PDF [3 MB]</a></li><li class="article-tools__sub-item"><a href="/cell-reports-medicine/pdfExtended/S2666-3791(20)30004-5" title="Extended PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayExtendedPdf ">Extended PDF [7 MB]</a></li></ul></div></div></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cell-reports-medicine/ppt/S2666-3791(20)30004-5.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.xcrm.2020.100004" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.xcrm.2020.100004&amp;pii=S2666-3791%2820%2930004-5" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.xcrm.2020.100004" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-533" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-533">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791%2820%2930004-5"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S2666379120300045&amp;issn=26663791&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S2666379120300045&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jason E. McDermott">Jason E. McDermott</a> <a href="#" data-db-target-for="crosref0020" aria-expanded="false" aria-controls="crosref0020" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>12</sup></span></a><div data-db-target-of="crosref0020" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">12 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jason E. McDermott</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">12 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA<p></p>Department of Molecular Microbiology and Immunology, Oregon Health &amp; Science University, Portland, OR 97201, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Osama A. Arshad">Osama A. Arshad</a> <a href="#" data-db-target-for="crosref0030" aria-expanded="false" aria-controls="crosref0030" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>12</sup></span></a><div data-db-target-of="crosref0030" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">12 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Osama A. Arshad</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">12 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Arshad%2C+Osama A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Vladislav A. Petyuk">Vladislav A. Petyuk</a><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Vladislav A. Petyuk</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petyuk%2C+Vladislav A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yi Fu">Yi Fu</a><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yi Fu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Fu%2C+Yi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marina A. Gritsenko">Marina A. Gritsenko</a><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marina A. Gritsenko</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gritsenko%2C+Marina A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Therese R. Clauss">Therese R. Clauss</a><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Therese R. Clauss</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clauss%2C+Therese R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ronald J. Moore">Ronald J. Moore</a><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ronald J. Moore</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Moore%2C+Ronald J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Athena A. Schepmoes">Athena A. Schepmoes</a><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Athena A. Schepmoes</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schepmoes%2C+Athena A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rui Zhao">Rui Zhao</a><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rui Zhao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Rui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew E. Monroe">Matthew E. Monroe</a><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew E. Monroe</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Monroe%2C+Matthew E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael Schnaubelt">Michael Schnaubelt</a><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael Schnaubelt</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schnaubelt%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chia-Feng Tsai">Chia-Feng Tsai</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chia-Feng Tsai</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Tsai%2C+Chia-Feng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Samuel H. Payne">Samuel H. Payne</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Liang-Bo Wang">Liang-Bo Wang</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Liang-Bo Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Liang-Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Steven Foltz">Steven Foltz</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Steven Foltz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Foltz%2C+Steven" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew Wyczalkowski">Matthew Wyczalkowski</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yige Wu">Yige Wu</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yige Wu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wu%2C+Yige" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ehwang Song">Ehwang Song</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ehwang Song</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Song%2C+Ehwang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Molly A. Brewer">Molly A. Brewer</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Molly A. Brewer</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Obstetrics and Gynecology, University of Connecticut, Farmington, CT 06030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Brewer%2C+Molly A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research, Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily S. Boja">Emily S. Boja</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel W. Chan">Daniel W. Chan</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel W. Chan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhen Zhang">Zhen Zhang</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins Medical Institutions, Baltimore, MD 21205, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, St Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Richard D. Smith">Richard D. Smith</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tao Liu">Tao Liu<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tao Liu</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#1064717f3e7c796550607e7e7c3e777f66" title="Send email to Tao Liu"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Tao Liu" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#c0b4a1afeeaca9b580b0aeaeaceea7afb6"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karin D. Rodland">Karin D. Rodland<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0205" aria-expanded="false" aria-controls="crosref0205" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>13</sup></span></a><div data-db-target-of="crosref0205" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">13 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karin D. Rodland</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#2f444e5d4641015d404b434e414b6f5f41414301484059" title="Send email to Karin D. Rodland"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">13 Lead Contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA<p></p>Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97201, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Karin D. Rodland" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#701b1102191e5e021f141c111e1430001e1e1c5e171f06"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span>Clinical Proteomic Tumor Analysis Consortium<div class="dropBlock article-header__info"></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">12 These authors contributed equally<br/>13 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">April 09, 2020</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.xcrm.2020.100004">https://doi.org/10.1016/j.xcrm.2020.100004</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.xcrm.2020.100004"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.xcrm.2020.100004" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cell-reports-medicine/fulltext/S2666-3791(20)30006-9" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">GLP-1 Receptor Agonist Treatment in Morbid Obesity and Type 2 Diabetes Due to Pathogenic Homozygous Melanocortin-4 Receptor Mutation: A Case Report</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="/cell-reports-medicine/fulltext/S2666-3791(20)30007-0" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">Clonal Evolution and Heterogeneity of Osimertinib Acquired Resistance Mechanisms in EGFR Mutant Lung Cancer</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><div class="ce-list--remove-bullets__list-item__text">Comparison of ovarian cancer and normal precursors identifies key signaling pathways</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><div class="ce-list--remove-bullets__list-item__text">Mitotic and cyclin-dependent kinases emerge as potential therapeutic targets</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><div class="ce-list--remove-bullets__list-item__text">Previously identified hallmarks of homologous repair status and survival are confirmed</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><div class="ce-list--remove-bullets__list-item__text">Replication stress appears to drive increased chromosomal instability</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">In the absence of a dominant driving mutation other than uniformly present <em>TP53</em> mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical Abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical Abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/77d2cc65-65b6-414d-a688-7cf229c61f51/fx1_lrg.jpg" data-large-image="/cms/attachment/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/f190076c-f6fc-48ff-bc70-46c63d6409da/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/77d2cc65-65b6-414d-a688-7cf229c61f51/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=2666-3791">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;ovarian cancer&#34;&amp;ISSN=2666-3791">ovarian cancer</a></li><li><a href="/action/doSearch?AllField=&#34;phosphoproteomics&#34;&amp;ISSN=2666-3791">phosphoproteomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=2666-3791">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;fallopian tube&#34;&amp;ISSN=2666-3791">fallopian tube</a></li><li><a href="/action/doSearch?AllField=&#34;homologous repair deficiency&#34;&amp;ISSN=2666-3791">homologous repair deficiency</a></li><li><a href="/action/doSearch?AllField=&#34;replication stress&#34;&amp;ISSN=2666-3791">replication stress</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">High-grade serous cancer (HGSC), the most prevalent histotype of ovarian cancer, has the lowest survival rates<div class="dropBlock reference-citations"><sup><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d15348236e577" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">1</a></sup><div data-db-target-of="bib1-d15348236e577" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Siegel R.L. </li><li class="loa__item">Miller K.D. </li><li class="loa__item">Jemal A. </li></ul><div class="reference-citations__detials">Cancer statistics, 2018.<div class="popup-ref"><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3322%2Fcaac.21442&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29313949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85040049759&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7396)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2018">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and is the leading cause of gynecological cancer-related deaths in the developed world.<div class="dropBlock reference-citations"><sup><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d15348236e581" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">2</a></sup><div data-db-target-of="bib2-d15348236e581" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>American Cancer Society</span><br />Key Statistics for Ovarian Cancer.<div class="popup-ref"><div class="citation-host"><a class="inter-ref-link" href="https://www.cancer.org/cancer/ovarian-cancer/about/key-statistics.html" target="_blank">https://www.cancer.org/cancer/ovarian-cancer/about/key-statistics.html</a></div></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=American+Cancer+SocietyKey+Statistics+for+Ovarian+Cancer.https%3A%2F%2Fwww.cancer.org%2Fcancer%2Fovarian-cancer%2Fabout%2Fkey-statistics.html">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> HGSC is characterized by the presence of nearly universal <em>TP53</em> mutations, diverse and widespread chromosomal instability, and a general shortage of targetable driving mutations.<div class="dropBlock reference-citations"><sup><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d15348236e587" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">3</a></sup><div data-db-target-of="bib3-d15348236e587" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Integrated genomic analyses of ovarian carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">474</span></span>: <span class="ref__seriesPages">609-615</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature10166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21720365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79959838081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5913)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2011&pages=609-615&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Integrated+genomic+analyses+of+ovarian+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Given this genomic heterogeneity, the standard of care for HGSC is surgical debulking, followed by combination chemotherapy with platinum-based agents and microtubule inhibitors such as cisplatin-paclitaxel. Despite an initial clinical response in most patients, recurrence is frequent, with current 5-year survival rates for stage IV HGSC of &lt;20%.<div class="dropBlock reference-citations"><sup><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d15348236e591" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">4</a></sup><div data-db-target-of="bib4-d15348236e591" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>National Cancer Institute</span><br />SEER Cancer Statistics Review, 1975-2015.<div class="popup-ref"><div class="citation-host"><a class="inter-ref-link" href="https://seer.cancer.gov/archive/csr/1975_2015/" target="_blank">https://seer.cancer.gov/archive/csr/1975_2015/</a></div></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=National+Cancer+InstituteSEER+Cancer+Statistics+Review%2C+1975-2015.https%3A%2F%2Fseer.cancer.gov%2Farchive%2Fcsr%2F1975_2015%2F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Thus, the major clinical needs in HGSC include the identification of alternative therapeutic targets and an improved understanding of the mechanisms driving chromosomal heterogeneity.</div><div class="section-paragraph">HGSC has been the focus of numerous genomic investigations and was among the first cancers studied by The Cancer Genome Atlas (TCGA).<div class="dropBlock reference-citations"><sup><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d15348236e597" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">3</a></sup><div data-db-target-of="bib3-d15348236e597" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Integrated genomic analyses of ovarian carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">474</span></span>: <span class="ref__seriesPages">609-615</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature10166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21720365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79959838081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5913)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2011&pages=609-615&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Integrated+genomic+analyses+of+ovarian+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Recently, we reported a comprehensive proteogenomic and phosphoproteomic characterization of 174 ovarian cancer samples previously analyzed by TCGA.<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e601" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e601" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> While the availability of outcomes data allowed us to stratify TCGA patients into short-term survivors (&lt;2 years) and long-term survivors (&gt;3 years) and identify proteomic features associated with survival, the absence of appropriate normal control tissues for proteomic analysis precluded the identification of pathways specifically associated with carcinogenesis.</div><div class="section-paragraph">In this study, we prospectively collected 83 new ovarian HGSC samples and 20 normal precursor tissue samples (fallopian tube, FT) for deep characterization at the genomic, transcriptomic, proteomic, and phosphoproteomic levels; 10 of the 20 normal FT samples were matched with tumor samples from the same patient. We were also able to obtain cytobrush samples of FT epithelium (FTE) and provide an extensive comparison of HGSC and FT proteomes and phosphoproteomes, using surgical specimens rather than cell lines. The ability to compare tumor and normal precursor tissues provided biological insights regarding the role of increased proliferation in promoting replication stress in a DNA repair-deficient background, potentially explaining the high degree of chromosomal instability associated with HGSC. Our collection protocols were specifically designed to minimize the effect of ischemia on protein phosphorylation, which has been identified as a significant confounding variable.<div class="dropBlock reference-citations"><sup><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d15348236e607" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">6</a></sup><div data-db-target-of="bib6-d15348236e607" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Yang F. </li><li class="loa__item">Liu T. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Moore R.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Thus, we were able to identify signaling interactions that would have been lost in the background of ischemic stress, most notably the activation of cyclin-dependent kinase 4 (CDK4) and CDK7. Widespread increases in protein phosphorylation and pathway activation in tumors compared to normal tissues, particularly in the proliferation-associated CDK-RB (retinoblastoma protein) and aurora kinase A (AURKA) pathways that are targetable by US Food and Drug Administration (FDA)-approved inhibitors, provide a rational basis for the use of these therapeutics in ovarian cancer. Phosphosite-specific analysis of CDKs and Fanconi anemia complementation group D (FANCD) was consistent with an elevated response to replication stress, potentially induced by proliferative signals. In addition, pathway-level observations from the retrospective TCGA cohort were conserved in the prospective cohort, including histone acetylation marks associated with homologous repair deficiency (HRD). The survival signature observed in the retrospective TCGA cohort was found to be lowest (consistent with short-term survival) in tumors from patients who were deceased at the 1-year follow-on time point and highest in the normal tissues (consistent with long-term survival), providing preliminary support for the survival signature in an independent cohort.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic Profiling</span></h3><div class="section-paragraph">We prospectively collected tumor specimens, appropriate non-tumor tissues (FT), and blood samples from 83 patients with ovarian HGSC. All of the tissue samples were subjected to mass spectrometry (MS)-based integrated global proteomics and phosphoproteomics analysis using 10-plexed isobaric tandem mass tag (TMT) labeling with extensive fractionation and concatenation (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>A). The clinical and pathological characteristics of the cohort are shown in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/27ca9135-f87e-493a-87b3-c5db2d0d8005/mmc2" class="scroll-into-link">Table S1</a>. The relative abundances of the proteins and phosphopeptides across the tumor and normal tissue samples were quantified using a universal reference strategy as reported recently,<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e625" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e625" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d15348236e630" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">7</a></sup><div data-db-target-of="bib7-d15348236e630" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> applying a stringent 1% false discovery rate (FDR) cutoff at the protein level. Inference of the parsimonious protein set resulted in a total of 10,706 proteins and a total of 49,616 phosphopeptides identified. The processed proteomics data tables are available in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>; subsets of data used in the different statistical analyses were described in the respective sections or <a href="#sec4" class="scroll-into-link">STAR Methods</a>. The raw proteomic data are available via the Clinical Proteomic Tumor Analysis Consortium (CPTAC) Data Portal (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer;jsessionid=3725434E55F3889301A71D9F76AAE3C6?accNum=S038" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer;jsessionid=3725434E55F3889301A71D9F76AAE3C6?accNum=S038</a>). In addition, HGSC tissue samples were analyzed by whole-exome sequencing (WXS) and mRNA sequencing (RNA-seq). Raw genomic data associated with this study (harmonized to the GRCh38 reference genome) have been released at the Genomic Data Commons (<a class="inter-ref-link" href="https://gdc.cancer.gov" target="_blank">https://gdc.cancer.gov</a>) and are accessible via the database of Genotypes and Phenotypes (dbGaP) Study Accession: phs000892; the processed RNA-seq data are available in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/a962c084-c058-4152-89e4-bb7599a4ae40/mmc4" class="scroll-into-link">Table S3</a>.</div><div class="section-paragraph">The identification and quantification results and normalization methods were carefully evaluated to confirm the data quality (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figures S1</a>B–S1D; <a href="#sec4" class="scroll-into-link">STAR Methods</a>). The consistency of the proteomics pipeline was evident in both the protein sequence coverage (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>B) and the quantitation reproducibility in the external quality controls comparing two breast cancer patient-derived xenograft samples analyzed alongside the ovarian tumors and normal tissue samples (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>C; <a href="#sec4" class="scroll-into-link">STAR Methods</a>). The proteomics and phosphoproteomics data also readily separated the tumors from the normal FT tissues (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>D). Principal-component analysis (PCA) on both the proteomic and phosphoproteomic data indicated that there was no clear separation of the tumor phosphopeptide data by either anatomic site of the tumor sample (ovary versus omentum, right panel in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>E) or tumor site of origin (FT versus peritoneum, right panel in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>F). However, the global proteomics data showed a clear separation between the ovary and all of the other anatomic sites of tumor samples (left panel in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>E).</div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutational Landscape of HGSC</span></h3><div class="section-paragraph">The 83 HGSC tumors in our prospective cohort displayed a wide range of somatic mutations, from a high of 7,195 to a low of 10; 93% of tumors had &gt;50 mutations (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A). A modest increase in average mutational load compared to TCGA was observed, which may relate to advances in WXS technologies. Specifically, missense mutations were 61.0% in our cohort versus 48.6% in TCGA, frameshift indel 22.1% versus 24.7%, splice site 6.5% versus 11.1%, nonsense 6.5% versus 10.7%, and inframe indel 3.9% versus 4.9%. As reported by TCGA, &gt;92% of the HGSC samples had somatic mutations in <em>TP53</em>; 47 samples had missense mutations, while 30 had various deletions, insertions, and splice variants (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). When frequently mutated genes (FLAG), which are known to be frequently mutated in many cancers simply as a function of size,<div class="dropBlock reference-citations"><sup><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d15348236e694" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">8</a></sup><div data-db-target-of="bib8-d15348236e694" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shyr C. </li><li class="loa__item">Tarailo-Graovac M. </li><li class="loa__item">Gottlieb M. </li><li class="loa__item">Lee J.J. </li><li class="loa__item">van Karnebeek C. </li><li class="loa__item">Wasserman W.W. </li></ul><div class="reference-citations__detials">FLAGS, frequently mutated genes in public exomes.<div class="popup-ref"><span class="ref__series"><i>BMC Med. Genomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs12920-014-0064-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25466818&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84946548774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=64&journal=BMC+Med.+Genomics&author=Shyr+C.&author=Tarailo-Graovac+M.&author=Gottlieb+M.&author=Lee+J.J.&author=van+Karnebeek+C.&author=Wasserman+W.W.&title=FLAGS%2C+frequently+mutated+genes+in+public+exomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> were eliminated, we observed 5 other genes that were mutated in 4%–7% of samples (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B).<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/0bd2e698-052a-4b0a-9350-d3276fc24a7f/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/738c00ef-57ee-46b8-989b-522f0832b69f/gr1_lrg.jpg" data-large-image="/cms/attachment/0bd2e698-052a-4b0a-9350-d3276fc24a7f/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Mutational Profiles of Prospective HGSC Samples</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Distribution of the total number of variants in each of the 83 tumor samples across the prospective cohort. Each bar depicts the number of mutations in that sample. Samples are ordered by the number of variants. Missense mutations are in blue, whereas all other mutations are in gray. The first four samples with substantially more variants are shown separately.</div><div class="section-paragraph">(B) Mutational landscape of the prospective HGSC cohort, including mutation frequency and type.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1205" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">Figure S2</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/0bd2e698-052a-4b0a-9350-d3276fc24a7f/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/738c00ef-57ee-46b8-989b-522f0832b69f/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Numerous arm-level and focal somatic copy number alterations (SCNAs) were identified from high-resolution exome sequence data using the Genomic Identification of Significant Targets in Cancer (GISTIC) 2.0 algorithm<div class="dropBlock reference-citations"><sup><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d15348236e703" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">9</a></sup><div data-db-target-of="bib9-d15348236e703" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, including 18 arm-level events at a false discovery rate (FDR) &lt; 0.05 (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S2</a>A; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e52c3284-ff78-4dfa-b9f6-288aad8b7ae7/mmc5" class="scroll-into-link">Table S4</a>). The significant arm-level amplifications and deletions identified in the prospective HGSC cohort substantially replicated those previously reported by TCGA.<div class="dropBlock reference-citations"><sup><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d15348236e713" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">10</a></sup><div data-db-target-of="bib10-d15348236e713" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Integrated genomic analyses of ovarian carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">474</span></span>: <span class="ref__seriesPages">609-615</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature10166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21720365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79959838081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4655)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2011&pages=609-615&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Integrated+genomic+analyses+of+ovarian+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> In addition, GISTIC detected regions of significant focal SCNAs across the genome, including 50 focal amplifications and 63 focal deletions (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S2</a>B; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e52c3284-ff78-4dfa-b9f6-288aad8b7ae7/mmc5" class="scroll-into-link">Table S4</a>). Direct comparisons of focal SCNAs between the retrospective TCGA cohort and the present cohort are complicated by the use of different platforms (Affymetrix Genome-Wide Human SNP Array 6.0 in TCGA versus WXS in the prospective cohort) to define SCNAs, resulting in an increased number of focal SCNAs in the prospective cohort; focal SCNAs from TCGA were generally detectable in the prospective samples.</div><div class="section-paragraph">The high number of SCNAs observed in both the retrospective TCGA cohort and our prospective cohort confirms the status of HGSC as predominantly a disease of “broken chromosomes” with high rates of faulty DNA repair. Calculations of the Chromosomal Instability Index (CIN) in the prospective cohort using the same algorithms described in the retrospective study<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e726" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e726" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> indicated that the instability was widely dispersed over all chromosomes (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S2</a>C). Statistical analysis of proteins whose abundance correlated highly with the CIN score demonstrated substantial overlap with a similar analysis performed on the retrospective TCGA samples (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A). Among the proteins most significantly associated with the CIN index were PARP1 (polyadenosine diphosphate [ADP]-ribose polymerase I), GTF2I (general transcription factor II-I, involved in cell growth and division), GMPS (guanine monophosphate synthase, involved in nucleotide metabolism), and XPO1 (exportin 1-chromosomal maintenance factor 1, which regulates the nuclear transport of cyclin B, mitogen-activated protein kinase [MAPK], and nuclear factor of activated T cells [NFATs]).<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/04c1fa5a-ef73-4f2a-9dbe-1dd2fe8a54a1/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/75103a75-4e6d-4df1-8097-f18b64abb43d/gr2_lrg.jpg" data-large-image="/cms/attachment/04c1fa5a-ef73-4f2a-9dbe-1dd2fe8a54a1/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">Correlates of Homologous Repair Deficiency in Prospective HGSC Samples</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Proteins associated with chromosomal instability in prospective HGSC samples.</div><div class="section-paragraph">(B and C) Boxplots of acetylation on histone H4 lysine K12 (B) and dual acetylation on K12 and K16 (C) as a function of HRD status (16 of 83 patients were HRD<sup>+</sup>, as indicated by <em>BRCA1/2</em> mutation or <em>PTEN</em> deletion).</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1210" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">Figure S2</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/04c1fa5a-ef73-4f2a-9dbe-1dd2fe8a54a1/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/75103a75-4e6d-4df1-8097-f18b64abb43d/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The impact of SCNAs on mRNA and protein expression (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figures S3</a>A and S3B) was explored by computing Spearman rank correlation across all of the tumors for each gene with SCNA measurements and available mRNA and/or protein abundances (<a href="#sec5.4" class="scroll-into-link">Method Details</a>); 475,448 of ∼102 million CNA-protein pairs (0.46%) and 742,678 of 102 million equivalent CNA-mRNA pairs (0.73%) were significantly correlated at an adjusted p &lt; 0.05. Positive correlations along the diagonal indicate strong <em>cis</em> effects of SCNAs on the abundance of mRNAs (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S3</a>A) and proteins (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S3</a>B) for cognate genes within the altered locus, as previously reported in the retrospective HGSC cohort. We also identified multiple chromosomal loci where changes in copy number affect the abundance of mRNA and proteins encoded by genes located elsewhere in the genome (i.e., effects in <em>trans</em>), including SCNA hotspots on chromosomes 1, 16, and 19, with strong effects on protein abundance in <em>trans</em> (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S3</a>B). Although the hotspots on chromosomes 2, 20, and 22 observed in the retrospective cohort<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e761" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e761" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> were not prominent in the prospective cohort at either the mRNA or protein levels, gene set enrichment analysis (GSEA)<div class="dropBlock reference-citations"><sup><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d15348236e765" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">11</a></sup><div data-db-target-of="bib11-d15348236e765" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Mootha V.K. </li><li class="loa__item">Mukherjee S. </li><li class="loa__item">Ebert B.L. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Paulovich A. </li><li class="loa__item">Pomeroy S.L. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32409)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> of the <em>trans</em>-affected proteins in the prospective cohort showed substantial enrichment of proteins associated with immune regulation (e.g., acute immune response, regulation of humoral immune response, regulation of immune effector process; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S3</a>C), confirming the apparent selection for proteins associated with immune function observed in the retrospective cohort. In general, however, the increased resolution of focal areas provided by the use of WXS to identify CNAs resulted in a dilution of the <em>trans</em> effect, while a positive correlation between copy number and <em>cis</em> effects on mRNA and protein abundance was consistently observed.</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Histone H4 Acetylation as an Indicator of HRD Status</span></h3><div class="section-paragraph">In our retrospective analysis, HGSC samples were deliberately chosen to represent HRD<sup>+</sup> and homologous repair proficient (HRD<sup>−</sup>) genotypes, based on mutations in <em>BRCA1</em>, <em>BRCA2</em>, and/or deletion of <em>PTEN</em>, to identify proteomic signatures associated with HRD status that may be used to stratify women for treatment with PARP inhibitors. In the retrospective study, we reported that the acetylation of two specific lysine residues of histone H4 (K12 and K16) was statistically significantly decreased in patients who were HRD<sup>+</sup>,<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e800" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e800" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, and these acetylation events were orthogonally validated by immunoblot and parallel reaction monitoring MS in a subsequent publication.<div class="dropBlock reference-citations"><sup><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d15348236e804" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">12</a></sup><div data-db-target-of="bib12-d15348236e804" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thomas S.N. </li><li class="loa__item">Chen L. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Höti N. </li><li class="loa__item">Zhang H. </li></ul><div class="reference-citations__detials">Targeted Proteomic Analyses of Histone H4 Acetylation Changes Associated with Homologous-Recombination-Deficient High-Grade Serous Ovarian Carcinomas.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">3704-3710</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Facs.jproteome.7b00405&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28866885&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85045317477&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=3704-3710&journal=J.%C2%A0Proteome+Res.&author=Thomas+S.N.&author=Chen+L.&author=Liu+Y.&author=H%C3%B6ti+N.&author=Zhang+H.&title=Targeted+Proteomic+Analyses+of+Histone+H4+Acetylation+Changes+Associated+with+Homologous-Recombination-Deficient+High-Grade+Serous+Ovarian+Carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Genomic analysis of the prospective HGSC samples for either somatic or germline mutations of <em>BRCA1</em> or <em>BRCA2</em> indicated that 16 of 83 patients were HRD<sup>+</sup>. Comparison of histone H4 K12 and K16 acetylation in these samples showed a statistically significant decrease in the acetylation of K12 in histone H4 (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B) and a similar but less significant decrease in dual acetylation of K12 and K16 (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>C). These results suggest that histone acetylation marks may have potential utility as an indicator of HRD status.</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Consistency of mRNA-Protein Correlations by Gene Family</span></h3><div class="section-paragraph">While within-sample correlations of mRNA and protein measurements are fairly consistent, across-sample mRNA-protein correlations for each gene-protein pair are known to be more variable, potentially reflecting gene-specific post-transcriptional regulatory mechanisms.<div class="dropBlock reference-citations"><sup><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d15348236e827" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">13</a></sup><div data-db-target-of="bib13-d15348236e827" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu Y. </li><li class="loa__item">Beyer A. </li><li class="loa__item">Aebersold R. </li></ul><div class="reference-citations__detials">On the Dependency of Cellular Protein Levels on mRNA Abundance.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">535-550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27104977&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84964321113&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1798)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=535-550&journal=Cell&author=Liu+Y.&author=Beyer+A.&author=Aebersold+R.&title=On+the+Dependency+of+Cellular+Protein+Levels+on+mRNA+Abundance">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> When we calculated the correlation between each protein and its cognate mRNA across all 83 tumors (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S4</a>A), the results were consistent with our previous report on HGSC<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e834" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e834" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and with proteogenomic analyses of breast and colorectal cancers,<div class="dropBlock reference-citations"><sup><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d15348236e838" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">7</a></sup><div data-db-target-of="bib7-d15348236e838" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d15348236e843" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">14</a></sup><div data-db-target-of="bib14-d15348236e843" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d15348236e849" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">15</a></sup><div data-db-target-of="bib15-d15348236e849" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V. </li><li class="loa__item">Savage S. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic characterization of human colon cancer reveals new therapeutic opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.&author=Savage+S.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.&title=Proteogenomic+characterization+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> suggesting that there is functional significance to the degree of correlation between cognate mRNA and proteins across multiple cancer types. Briefly, we observed a range of correlations between paired mRNA and protein values, with a median correlation of 0.35 across all 83 samples (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S4</a>A). A comparison of the genes representing the top and bottom 10% of correlation values from this study and previous studies is shown in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S4</a>B. As observed previously, functional enrichment analysis indicated that genes for nucleotide and amino acid metabolism and interferon response showed good correlation between mRNA and protein, while genes for such housekeeping functions as ribosomes, oxidative phosphorylation, mRNA splicing, and the complement and coagulation cascade were relatively poorly correlated.</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Comparison of Protein Expression in Ovarian HGSC and Normal Precursor Tissues</span></h3><div class="section-paragraph">A substantial limitation of the prior proteogenomic analysis of HGSC samples from the TCGA cohort was the absence of an appropriate control for tumor versus normal comparisons at the protein level. In this prospective study, phenotypically normal FT was collected when feasible at the time of surgery, as there is substantial evidence that serous tubal intra-epithelial carcinoma (STIC) lesions in the FT are the dominant precursor lesions for HGSC.<div class="dropBlock reference-citations"><sup><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d15348236e864" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">16</a></sup><div data-db-target-of="bib16-d15348236e864" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ducie J. </li><li class="loa__item">Dao F. </li><li class="loa__item">Considine M. </li><li class="loa__item">Olvera N. </li><li class="loa__item">Shaw P.A. </li><li class="loa__item">Kurman R.J. </li><li class="loa__item">Shih I.M. </li><li class="loa__item">Soslow R.A. </li><li class="loa__item">Cope L. </li><li class="loa__item">Levine D.A. </li></ul><div class="reference-citations__detials">Molecular analysis of high-grade serous ovarian carcinoma with and without associated serous tubal intra-epithelial carcinoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">990</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41467-017-01217-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29042553&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85031805521&amp;cf=" onclick="newWindow(this.href);return false">Scopus (132)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=990&journal=Nat.+Commun.&author=Ducie+J.&author=Dao+F.&author=Considine+M.&author=Olvera+N.&author=Shaw+P.A.&author=Kurman+R.J.&author=Shih+I.M.&author=Soslow+R.A.&author=Cope+L.&author=Levine+D.A.&title=Molecular+analysis+of+high-grade+serous+ovarian+carcinoma+with+and+without+associated+serous+tubal+intra-epithelial+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d15348236e869" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">17</a></sup><div data-db-target-of="bib17-d15348236e869" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Labidi-Galy S.I. </li><li class="loa__item">Papp E. </li><li class="loa__item">Hallberg D. </li><li class="loa__item">Niknafs N. </li><li class="loa__item">Adleff V. </li><li class="loa__item">Noe M. </li><li class="loa__item">Bhattacharya R. </li><li class="loa__item">Novak M. </li><li class="loa__item">Jones S. </li><li class="loa__item">Phallen J. </li><li>et al.</li></ul><div class="reference-citations__detials">High grade serous ovarian carcinomas originate in the fallopian tube.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1093</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41467-017-00962-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29061967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85032176462&amp;cf=" onclick="newWindow(this.href);return false">Scopus (457)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1093&journal=Nat.+Commun.&author=Labidi-Galy+S.I.&author=Papp+E.&author=Hallberg+D.&author=Niknafs+N.&author=Adleff+V.&author=Noe+M.&author=Bhattacharya+R.&author=Novak+M.&author=Jones+S.&author=Phallen+J.&title=High+grade+serous+ovarian+carcinomas+originate+in+the+fallopian+tube">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Twenty normal FT samples were characterized, 10 of which were matched with tumor samples from the same patient. Distal fimbrae from uninvolved, phenotypically normal FTs were collected under tight ischemia control of ≤30 min in recognition of the observed effects of ischemia times on the phosphorylation of stress-related kinases.<div class="dropBlock reference-citations"><sup><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d15348236e873" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">6</a></sup><div data-db-target-of="bib6-d15348236e873" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Yang F. </li><li class="loa__item">Liu T. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Moore R.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> As shown in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>A, this tightened control of ischemia time substantially reduced the number of patients with artifactual phosphorylation of p38/MAPK and other stress-responsive pathways. <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>B demonstrates that there was no significant difference in the levels of phosphorylation as a function of ischemia time up to 30 min, as determined by PCA.</div><div class="section-paragraph">The availability of prospectively collected FT samples allowed us to interrogate the proteomic dataset for differences in protein expression patterns in HGSC compared to the normal precursor tissue. We identified 509 proteins differentially abundant (log2 fold-change &gt; 2, FDR &lt; 0.05) in HGSC compared to normal tissue (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e47f102a-2542-4245-bfe8-45ad04ff40dd/mmc6" class="scroll-into-link">Table S5</a>). The major difference observed was in proteins that were under-expressed in HGSC compared to FT; 489 proteins were statistically more abundant in FT, compared to 20 that were statistically more abundant in HGSC. As a quality control, we compared protein identifications in HGSC samples, normal FT samples harvested surgically within the prospective cohort, and FTE cell-enriched samples prepared by scraping the mucosal surface of healthy FTs at the time of surgery for benign conditions (cytobrush FTE). Overlap between the surgical FT and cytobrush FTE samples was &gt;50% (3,894 of 7,599 proteins observed in surgical FT were also observed in the cytobrushed samples; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>C), and both samples contained known markers of secretory epithelial cells, most notably oviductal glycoprotein-1 (OGP-1). GSEA analysis indicated that processes associated with DNA repair and DNA replication (e.g., interstrand crosslink repair, regulation of DNA damage response, negative regulation of telomere maintenance, DNA-dependent DNA replication maintenance of fidelity, homologous recombination) were modestly increased in HGSC tissues, while FT appeared to be enriched in proteins associated with muscle processes, immune function, and signaling pathways associated with reproductive and neurological processes (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/6730ffc7-15b5-4ee8-a850-8f76779f6539/mmc7" class="scroll-into-link">Table S6</a>), consistent with the histological composition of the FT. To control for proteins associated with non-epithelial components of the FT, we repeated the analysis using only those proteins also observed in the cytobrush FTE preparations (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>D) and still observed an enrichment for immune functions and developmental processes in normal FT compared to tumors.<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/a25878c5-6f1b-418a-9ef9-34b1be2909ff/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/5b772dcf-bb38-4185-a68a-e47f774b5163/gr3_lrg.jpg" data-large-image="/cms/attachment/a25878c5-6f1b-418a-9ef9-34b1be2909ff/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Comparison of Protein Abundance in HGSC and Normal FT</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Volcano plot representation of differential protein expression in tumor versus normal FT. Proteins with significantly increased or decreased (±2-fold change, Benjamini-Hochberg adjusted p &lt; 0.05) expression in 83 tumors compared to 18 normals are colored red and blue, respectively. The lighter colors are the significantly differential proteins at a lower fold-change threshold of ±1.5. Differentially expressed proteins with the largest fold-changes are labeled. An interactive version of this plot is available at <a class="inter-ref-link" href="https://doi.org/10.25584/cptac/1601822" target="_blank">https://doi.org/10.25584/cptac/1601822</a>.</div><div class="section-paragraph">(B) GSEA of Gene Ontology biological process gene sets for cancer versus normal protein differential expression. The clusterProfiler dot plot visualization shows enriched terms as dots, in which the size of the dot represents the gene count and the color corresponds to the statistical significance of the enrichment. The x axis is the proportion of genes that are annotated to a term.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1215" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">Figures S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1220" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/a25878c5-6f1b-418a-9ef9-34b1be2909ff/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/5b772dcf-bb38-4185-a68a-e47f774b5163/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Enrichment of Proliferation-Associated Kinase Activity in HGSC</span></h3><div class="section-paragraph">We analyzed and compared the phosphoproteomes of the tumor and normal FT samples by assessing the differences in phosphopeptide abundance between the two groups, with and without normalization to changes in protein abundance. Differential phosphoproteomic analysis identified 593 phosphopeptides with higher abundance (2-fold change, Benjamini-Hochberg adjusted p &lt; 0.05) in the HGSC samples, whereas 383 phosphopeptides were found to be more highly abundant in FT samples, indicating that the tumor samples displayed a generalized increase in phosphorylation, and thus increased pathway activation, compared to normal tissues (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e47f102a-2542-4245-bfe8-45ad04ff40dd/mmc6" class="scroll-into-link">Table S5</a>). We examined the differences in pathway representation between tumor and normal FT in both the proteomics and phosphoproteomics data (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>B) and found, similar to our previous observations,<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e920" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e920" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> that phosphoproteomics provided much more robust pathway enrichment than proteomics. However, many identified pathways were significant at both the protein and phosphopeptide levels, including the multifunctional RhoA regulatory pathway. Many of the pathways that were differentially active in tumor versus normal were also previously observed to be more active in short-term survivors compared to long-term survivors (e.g., RhoA, Notch, integrin-like kinase, Rac1).<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e924" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e924" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/a44c062d-5750-4e74-b9eb-a94c471cd24a/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/099e8efe-4a46-4be8-80c7-db1efe42b773/gr4_lrg.jpg" data-large-image="/cms/attachment/a44c062d-5750-4e74-b9eb-a94c471cd24a/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Comparison of Protein Phosphorylation and Kinase Activity in HGSC and Normal FT</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Volcano plot of tumor-normal differences in peptide phosphorylation. Phosphopeptides with significantly increased or decreased (±2-fold change, Benjamini-Hochberg adjusted p &lt; 0.05) abundance in tumors compared to normal FT are colored red and blue, respectively. The lighter colors are the significantly differential phosphopeptides at a lower fold-change threshold of ±1.5. An interactive version of this plot is available at <a class="inter-ref-link" href="https://doi.org/10.25584/cptac/1601822" target="_blank">https://doi.org/10.25584/cptac/1601822</a>.</div><div class="section-paragraph">(B) Pathway components were compared (2-sided t test) between tumor and normal FT samples for differences in protein and phosphorylation abundance. The top 10 most significant pathways in terms of Benjamini-Hochberg adjusted p value for proteomics and phosphoproteomics data are shown. These pathways exhibit increased phosphorylation in tumors. The notch, integrin-like kinase, and RhoA regulatory pathways emerged as the most activated in this analysis.</div><div class="section-paragraph">(C) Inference of kinase activity in tumors compared to normal using KSEA on phosphopeptides for known kinase substrates. The bar plot shows the relative activity of kinases in tumors versus normal FT. The <em>Z</em> scores are a measure of the enrichment for phosphorylated substrates of a given kinase, with higher scores corresponding to greater activity in the tumors. Kinases with significant (FDR &lt; 0.05) increases in tumors are in red, whereas those with significantly lower activity in tumors are in blue. Darker colors are used for kinases for which phosphorylation changes are independent of abundance changes.</div><div class="section-paragraph">(D) Comparison of DAPK1 phosphorylation at residue S289 in tumor and normal samples.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1225" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">Figures S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1230" href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" target="_blank">S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/a44c062d-5750-4e74-b9eb-a94c471cd24a/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/099e8efe-4a46-4be8-80c7-db1efe42b773/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We used kinase substrate enrichment analysis (KSEA) to characterize differences in the relative phosphoproteomic profiles of the tumors with respect to the normal tissues (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). KSEA is a computational framework for inferring the activation status of signaling kinases from quantitative phosphoproteomic datasets through the changes in the phosphorylation levels of their respective substrates.<div class="dropBlock reference-citations"><sup><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d15348236e932" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">18</a></sup><div data-db-target-of="bib18-d15348236e932" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Casado P. </li><li class="loa__item">Rodriguez-Prados J.C. </li><li class="loa__item">Cosulich S.C. </li><li class="loa__item">Guichard S. </li><li class="loa__item">Vanhaesebroeck B. </li><li class="loa__item">Joel S. </li><li class="loa__item">Cutillas P.R. </li></ul><div class="reference-citations__detials">Kinase-substrate enrichment analysis provides insights into the heterogeneity of signaling pathway activation in leukemia cells.<div class="popup-ref"><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">rs6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1126%2Fscisignal.2003573&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23532336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84875699641&amp;cf=" onclick="newWindow(this.href);return false">Scopus (242)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=rs6&journal=Sci.+Signal.&author=Casado+P.&author=Rodriguez-Prados+J.C.&author=Cosulich+S.C.&author=Guichard+S.&author=Vanhaesebroeck+B.&author=Joel+S.&author=Cutillas+P.R.&title=Kinase-substrate+enrichment+analysis+provides+insights+into+the+heterogeneity+of+signaling+pathway+activation+in+leukemia+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d15348236e937" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">19</a></sup><div data-db-target-of="bib19-d15348236e937" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wiredja D.D. </li><li class="loa__item">Koyutürk M. </li><li class="loa__item">Chance M.R. </li></ul><div class="reference-citations__detials">The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3489-3491</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtx415&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28655153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044408024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (135)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3489-3491&journal=Bioinformatics&author=Wiredja+D.D.&author=Koyut%C3%BCrk+M.&author=Chance+M.R.&title=The+KSEA+App%3A+a+web-based+tool+for+kinase+activity+inference+from+quantitative+phosphoproteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Analysis of substrate phosphorylation using KSEA revealed significantly increased activity of CDK1, CDK2, CDK7, and CDK4; the mitotic kinase AURKA; and the mRNA processing-associated kinase CLK1 in tumors. Given the existence of FDA-approved drugs targeting Aurora kinases and CDKs, these results suggest possible therapeutic strategies for HGSC. Normal FT showed increased activation of multiple kinases associated with stress responses and homeostatic regulation. The inferred activation in normal FT of JNK1 kinase (MAPK8) and adrenergic β receptor kinase (ADRBK), a G-protein coupled receptor kinase associated with homeostatic regulation, was significant both as phosphopeptides and as phosphopeptides normalized to protein abundance (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/c8d22e84-84a4-42c7-8f62-9f428a9e87ee/mmc8" class="scroll-into-link">Table S7</a>).</div><div class="section-paragraph">To evaluate the impact of the more tightly controlled ischemia times in the prospective cohort, we repeated KSEA on phosphopeptides that had been filtered conservatively to remove phosphosites known to be affected by ischemia (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>E) using a procedure previously described.<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e952" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e952" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d15348236e957" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">6</a></sup><div data-db-target-of="bib6-d15348236e957" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Yang F. </li><li class="loa__item">Liu T. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Moore R.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> The differences observed between <a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>C and <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">S5</a>E indicate that improved ischemia control increases the biological insights from the analysis of the prospectively collected samples, specifically in regard to processes that are enriched in the normal FT samples. Of the 22 kinases identified as activated in FT without using the ischemia filter, only 2 kinases, calcium/calmodulin-dependent protein kinase II α (CAMK2A) and protein kinase cyclic adenosine monophosphate-activated catalytic subunit α (PRKACA), passed the ischemia filter. While many of the kinases lost from the analysis are related to the p38-MAPK stress-activated pathway, information regarding the activation of p21-activated protein kinase 2 (PAK2) and rho-associated protein kinase 1 (ROCK1), kinases associated with organization of the actin cytoskeleton and cell motility, would have been lost.</div><div class="section-paragraph">While KSEA identifies high-level pathway alterations, additional biology can be discerned by focusing the analysis of phosphoproteomics data on known phosphorylation sites linked to kinase activity. Significantly differential phosphopeptides were filtered using the PhosphoSitePlus database,<div class="dropBlock reference-citations"><sup><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d15348236e969" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">20</a></sup><div data-db-target-of="bib20-d15348236e969" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Tkachev S. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Skrzypek E. </li><li class="loa__item">Murray B. </li><li class="loa__item">Latham V. </li><li class="loa__item">Sullivan M. </li></ul><div class="reference-citations__detials">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">40</span></span>: <span class="ref__seriesPages">D261-D270</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkr1122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22135298&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84857047339&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1237)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D261-D270&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Kornhauser+J.M.&author=Tkachev+S.&author=Zhang+B.&author=Skrzypek+E.&author=Murray+B.&author=Latham+V.&author=Sullivan+M.&title=PhosphoSitePlus%3A+a+comprehensive+resource+for+investigating+the+structure+and+function+of+experimentally+determined+post-translational+modifications+in+man+and+mouse">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> which highlights modifications known to affect protein function (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/2340a46e-8336-4ce0-ae98-e74faf0d6336/mmc9" class="scroll-into-link">Table S8</a>). Multiple differential phosphorylation sites with relevant functional consequences were identified in this analysis. For example, increased phosphorylation of death-associated protein kinase 1 (DAPK1) in tumors is relevant to the consistent observation of suppressed apoptosis in cancer (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>D). DAPK1 is a serine-threonine kinase involved in multiple pathways related to cell survival, apoptosis, and autophagy.<div class="dropBlock reference-citations"><sup><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d15348236e979" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">21</a></sup><div data-db-target-of="bib21-d15348236e979" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song L. </li><li class="loa__item">Pei L. </li><li class="loa__item">Hu L. </li><li class="loa__item">Pan S. </li><li class="loa__item">Xiong W. </li><li class="loa__item">Liu M. </li><li class="loa__item">Wu Y. </li><li class="loa__item">Shang Y. </li><li class="loa__item">Yao S. </li></ul><div class="reference-citations__detials">Death-associated protein kinase 1 mediates interleukin-1β production through regulating inflammasome activation in Bv2 microglial cells and mice.<div class="popup-ref"><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">9930</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41598-018-27842-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29967321&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85049359757&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9930&journal=Sci.+Rep.&author=Song+L.&author=Pei+L.&author=Hu+L.&author=Pan+S.&author=Xiong+W.&author=Liu+M.&author=Wu+Y.&author=Shang+Y.&author=Yao+S.&title=Death-associated+protein+kinase+1+mediates+interleukin-1%CE%B2+production+through+regulating+inflammasome+activation+in+Bv2+microglial+cells+and+mice">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> We observed significantly increased phosphorylation in tumors relative to FT at S289 of DAPK1, a site known to suppress apoptosis induced by DAPK1.<div class="dropBlock reference-citations"><sup><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d15348236e983" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">22</a></sup><div data-db-target-of="bib22-d15348236e983" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anjum R. </li><li class="loa__item">Roux P.P. </li><li class="loa__item">Ballif B.A. </li><li class="loa__item">Gygi S.P. </li><li class="loa__item">Blenis J. </li></ul><div class="reference-citations__detials">The tumor suppressor DAP kinase is a target of RSK-mediated survival signaling.<div class="popup-ref"><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1762-1767</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16213824&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-26244449244&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1762-1767&journal=Curr.+Biol.&author=Anjum+R.&author=Roux+P.P.&author=Ballif+B.A.&author=Gygi+S.P.&author=Blenis+J.&title=The+tumor+suppressor+DAP+kinase+is+a+target+of+RSK-mediated+survival+signaling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cell-Cycle Control and Replication Stress in HGSC</span></h3><div class="section-paragraph">KSEA revealed substantial increases in the inferred activity of multiple kinases associated with cell-cycle control, including CDK1, CDK2, AURKA, CDK7, and CDK4 (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). This observation prompted a closer look at the possible role of cell-cycle dysregulation and replication stress in HGSC.</div><div class="section-paragraph">RB1 is a tumor suppressor that is frequently deleted in cancer<div class="dropBlock reference-citations"><sup><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d15348236e996" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">23</a></sup><div data-db-target-of="bib23-d15348236e996" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Goodrich D.W. </li></ul><div class="reference-citations__detials">The retinoblastoma tumor-suppressor gene, the exception that proves the rule.<div class="popup-ref"><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">5233-5243</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fsj.onc.1209616&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16936742&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-33748075785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=5233-5243&journal=Oncogene&author=Goodrich+D.W.&title=The+retinoblastoma+tumor-suppressor+gene%2C+the+exception+that+proves+the+rule">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; unphosphorylated RB1 suppresses proliferation by sequestering the E2F transcription factor, which is released upon phosphorylation.<div class="dropBlock reference-citations"><sup><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d15348236e1000" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">24</a></sup><div data-db-target-of="bib24-d15348236e1000" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Indovina P. </li><li class="loa__item">Pentimalli F. </li><li class="loa__item">Casini N. </li><li class="loa__item">Vocca I. </li><li class="loa__item">Giordano A. </li></ul><div class="reference-citations__detials">RB1 dual role in proliferation and apoptosis: cell fate control and implications for cancer therapy.<div class="popup-ref"><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">17873-17890</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.18632%2Foncotarget.4286&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26160835&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84938632165&amp;cf=" onclick="newWindow(this.href);return false">Scopus (118)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17873-17890&journal=Oncotarget&author=Indovina+P.&author=Pentimalli+F.&author=Casini+N.&author=Vocca+I.&author=Giordano+A.&title=RB1+dual+role+in+proliferation+and+apoptosis%3A+cell+fate+control+and+implications+for+cancer+therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Phosphorylated RB1 also promotes apoptosis in response to replication stress and DNA damage.<div class="dropBlock reference-citations"><sup><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d15348236e1004" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">25</a></sup><div data-db-target-of="bib25-d15348236e1004" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chau B.N. </li><li class="loa__item">Wang J.Y.J. </li></ul><div class="reference-citations__detials">Coordinated regulation of life and death by RB.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">130-138</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnrc993&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=12563312&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0037313618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (246)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=130-138&journal=Nat.+Rev.+Cancer&author=Chau+B.N.&author=Wang+J.Y.J.&title=Coordinated+regulation+of+life+and+death+by+RB">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Although 70% of SCNAs in HGSC involving <em>RB1</em> were associated with chromosomal losses, we found only a modest positive correlation of <em>RB1</em> SCNAs and RB1 protein levels (Pearson correlation r = 0.51); RB1 hyperphosphorylation correlated positively with protein abundance (Pearson correlation r = 0.74), but was largely independent of SCNAs (Pearson correlation r = 0.27; <a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A), indicating a general downregulation of the tumor suppressor activities of RB1 in HGSC, mediated at the protein phosphorylation level (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B; mean average fold-change 1.5; p value 3.7e−27). This observation highlights the power of phosphoproteomics to identify functional changes independent of protein or mRNA abundance.<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/896dd084-50a0-47e0-b566-942f268d6e71/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/72ebe23e-d1c2-495b-8704-f7cd7d5d98a7/gr5_lrg.jpg" data-large-image="/cms/attachment/896dd084-50a0-47e0-b566-942f268d6e71/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Assessment of Relation between SCNA and Protein Levels and RB1 Hyperphosphorylation in HGSC</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) RB1 was consistently hyperphosphorylated independent of SCNAs. RB1 phosphorylation (boxplots) in all of the samples is shown ordered by the relative abundance of RB1 protein (red dots), and SCNA levels (purple dots). RB1 phosphorylation is correlated with protein abundance (r = 0.74) but is not well correlated with the RB1 SCNA level (r = 0.27).</div><div class="section-paragraph">(B) RB1 phosphorylation level was higher in tumor (n = 83) relative to normal FT tissue (n = 18).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/896dd084-50a0-47e0-b566-942f268d6e71/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/72ebe23e-d1c2-495b-8704-f7cd7d5d98a7/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">One well-known driver of increased RB1 hyperphosphorylation is the cyclin E-CDK2 complex.<div class="dropBlock reference-citations"><sup><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d15348236e1022" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">26</a></sup><div data-db-target-of="bib26-d15348236e1022" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zarkowska T. </li><li class="loa__item">Mittnacht S. </li></ul><div class="reference-citations__detials">Differential phosphorylation of the retinoblastoma protein by G1/S cyclin-dependent kinases.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">1997; </span><span class="ref__seriesVolume">272</span></span>: <span class="ref__seriesPages">12738-12746</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=9139732&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0030918714&amp;cf=" onclick="newWindow(this.href);return false">Scopus (439)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=12738-12746&journal=J.%C2%A0Biol.+Chem.&author=Zarkowska+T.&author=Mittnacht+S.&title=Differential+phosphorylation+of+the+retinoblastoma+protein+by+G1%2FS+cyclin-dependent+kinases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> In our analysis of HGSC, mean CDK2 activity inferred from known substrate phosphorylation levels after removing the contribution from RB1 phosphosites was significantly correlated with mean RB1 phosphorylation (Pearson correlation r = 0.53, p value 7e−7; <a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A), similar to observations in colorectal cancer.<div class="dropBlock reference-citations"><sup><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d15348236e1030" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">15</a></sup><div data-db-target-of="bib15-d15348236e1030" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V. </li><li class="loa__item">Savage S. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic characterization of human colon cancer reveals new therapeutic opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.&author=Savage+S.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.&title=Proteogenomic+characterization+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> However, examination of individual CDK2 phosphosites in HGSC compared to normal FT showed significantly increased phosphorylation of the two major phosphorylation sites CDK2-Y15 and CDK2-T160 in HGSC (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B). These two sites have opposing roles in CDK2 function, with Y15 being inhibitory and T160 activating the kinase.<div class="dropBlock reference-citations"><sup><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d15348236e1037" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">27</a></sup><div data-db-target-of="bib27-d15348236e1037" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gu Y. </li><li class="loa__item">Rosenblatt J. </li><li class="loa__item">Morgan D.O. </li></ul><div class="reference-citations__detials">Cell cycle regulation of CDK2 activity by phosphorylation of Thr160 and Tyr15.<div class="popup-ref"><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">1992; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">3995-4005</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1002%2Fj.1460-2075.1992.tb05493.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=1396589&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0026713875&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=3995-4005&journal=EMBO+J.&author=Gu+Y.&author=Rosenblatt+J.&author=Morgan+D.O.&title=Cell+cycle+regulation+of+CDK2+activity+by+phosphorylation+of+Thr160+and+Tyr15">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Although increased phosphorylation at Y15 in tumors would appear to be counterintuitive, there is evidence that distinct subpopulations of CDK2 could be inhibited by Y15 phosphorylation even if the overall CDK2 activity is increased.<div class="dropBlock reference-citations"><sup><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d15348236e1042" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">27</a></sup><div data-db-target-of="bib27-d15348236e1042" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gu Y. </li><li class="loa__item">Rosenblatt J. </li><li class="loa__item">Morgan D.O. </li></ul><div class="reference-citations__detials">Cell cycle regulation of CDK2 activity by phosphorylation of Thr160 and Tyr15.<div class="popup-ref"><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">1992; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">3995-4005</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1002%2Fj.1460-2075.1992.tb05493.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=1396589&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0026713875&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=3995-4005&journal=EMBO+J.&author=Gu+Y.&author=Rosenblatt+J.&author=Morgan+D.O.&title=Cell+cycle+regulation+of+CDK2+activity+by+phosphorylation+of+Thr160+and+Tyr15">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> In addition, increased phosphorylation of CDK2 on Y15 has been associated with cell-cycle exit in response to replication stress<div class="dropBlock reference-citations"><sup><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d15348236e1046" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">28</a></sup><div data-db-target-of="bib28-d15348236e1046" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hughes B.T. </li><li class="loa__item">Sidorova J. </li><li class="loa__item">Swanger J. </li><li class="loa__item">Monnat Jr., R.J. </li><li class="loa__item">Clurman B.E. </li></ul><div class="reference-citations__detials">Essential role for Cdk2 inhibitory phosphorylation during replication stress revealed by a human Cdk2 knockin mutation.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">110</span></span>: <span class="ref__seriesPages">8954-8959</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1073%2Fpnas.1302927110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23671119&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84878460802&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8954-8959&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Hughes+B.T.&author=Sidorova+J.&author=Swanger+J.&author=Monnat+R.J.&author=Clurman+B.E.&title=Essential+role+for+Cdk2+inhibitory+phosphorylation+during+replication+stress+revealed+by+a+human+Cdk2+knockin+mutation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d15348236e1051" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">29</a></sup><div data-db-target-of="bib29-d15348236e1051" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Teixeira L.K. </li><li class="loa__item">Wang X. </li><li class="loa__item">Li Y. </li><li class="loa__item">Ekholm-Reed S. </li><li class="loa__item">Wu X. </li><li class="loa__item">Wang P. </li><li class="loa__item">Reed S.I. </li></ul><div class="reference-citations__detials">Cyclin E deregulation promotes loss of specific genomic regions.<div class="popup-ref"><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1327-1333</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25959964&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84929607476&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1327-1333&journal=Curr.+Biol.&author=Teixeira+L.K.&author=Wang+X.&author=Li+Y.&author=Ekholm-Reed+S.&author=Wu+X.&author=Wang+P.&author=Reed+S.I.&title=Cyclin+E+deregulation+promotes+loss+of+specific+genomic+regions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and could conceivably constitute a response to mutant TP53-associated replication stress in HGSC. We observed a highly statistically significant correlation between the phosphorylation of CDK2-Y15 and the phosphorylation of RB1-S249 (r = 0.56, p = 0.00054) and RB1-T356 (r = 0.60, p = 0.0002). While these RB1 sites are known to be activating, they have not been previously linked to CDK2 activity.<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/fe642555-2582-47ec-8784-c97976edac32/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/94b3d4bb-a18d-4348-aa83-5a525a47c706/gr6_lrg.jpg" data-large-image="/cms/attachment/fe642555-2582-47ec-8784-c97976edac32/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Phosphosite Analysis Provides Evidence of Increased Proliferation and Replication Stress in HGSC Compared to Normal FT Tissues</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Correlation of mean CDK2 activity inferred from known substrate phosphorylation levels after removing the contribution from RB1 phosphosites, with mean RB1 phosphorylation level.</div><div class="section-paragraph">(B) Comparison of phosphorylation at specific CDK2 phosphosites in tumors (teal; n = 83) versus normal FT (red; n = 18).</div><div class="section-paragraph">(C) Assessment of abundance of FANCD2 phosphosites. Distribution of abundance of phosphosites of FANCD2 in tumors versus normal FT.</div><div class="section-paragraph">(D) Diagram of pathways linking DNA damage, cell-cycle regulation, and replication stress found to be activated in HGSC. As indicated in the figure, proteins colored in salmon are significantly upregulated in tumors, and phosphosites indicated in red are also significantly increased in tumors. The red arrows indicate that the known substrates of the identified kinase show significant increases in phosphorylation.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/fe642555-2582-47ec-8784-c97976edac32/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/94b3d4bb-a18d-4348-aa83-5a525a47c706/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Examining other cellular components associated with replication stress, we found statistically significant increases in the abundance of FANCD, PARP, CHK1, CDK1, and CDK2; we also observed modest (but not statistically significant) increases in the kinase activity of both ataxia telangiectasia mutated (ATM) and ATM and RAD3-related (ATR) (<em>Z</em> scores are 0.56 and 0.40, respectively), key regulators of DNA repair in response to replication stress.<div class="dropBlock reference-citations"><sup><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d15348236e1059" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">30</a></sup><div data-db-target-of="bib30-d15348236e1059" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mazouzi A. </li><li class="loa__item">Velimezi G. </li><li class="loa__item">Loizou J.I. </li></ul><div class="reference-citations__detials">DNA replication stress: causes, resolution and disease.<div class="popup-ref"><span class="ref__series"><i>Exp. Cell Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">329</span></span>: <span class="ref__seriesPages">85-93</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.yexcr.2014.09.030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25281304&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84919413650&amp;cf=" onclick="newWindow(this.href);return false">Scopus (154)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2014&pages=85-93&journal=Exp.+Cell+Res.&author=Mazouzi+A.&author=Velimezi+G.&author=Loizou+J.I.&title=DNA+replication+stress%3A+causes%2C+resolution+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> FANCD2 is a key component of the Fanconi anemia homologous DNA repair complex,<div class="dropBlock reference-citations"><sup><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d15348236e1063" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">31</a></sup><div data-db-target-of="bib31-d15348236e1063" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bhattacharjee S. </li><li class="loa__item">Nandi S. </li></ul><div class="reference-citations__detials">DNA damage response and cancer therapeutics through the lens of the Fanconi Anemia DNA repair pathway.<div class="popup-ref"><span class="ref__series"><i>Cell Commun. Signal.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs12964-017-0195-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29017571&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85031099180&amp;cf=" onclick="newWindow(this.href);return false">Scopus (64)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=41&journal=Cell+Commun.+Signal.&author=Bhattacharjee+S.&author=Nandi+S.&title=DNA+damage+response+and+cancer+therapeutics+through+the+lens+of+the+Fanconi+Anemia+DNA+repair+pathway">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and an examination of FANCD2 revealed both increased overall abundance and concomitant phosphorylation at S590 and S592 in HGSC tumors compared to normal FT samples (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>C). At present, very little is known about the functional consequences of specific phosphorylation sites on FANCD2, although its partner FANC1 is known to be activated by phosphorylation.<div class="dropBlock reference-citations"><sup><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d15348236e1070" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">32</a></sup><div data-db-target-of="bib32-d15348236e1070" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ishiai M. </li><li class="loa__item">Sato K. </li><li class="loa__item">Tomida J. </li><li class="loa__item">Kitao H. </li><li class="loa__item">Kurumizaka H. </li><li class="loa__item">Takata M. </li></ul><div class="reference-citations__detials">Activation of the FA pathway mediated by phosphorylation and ubiquitination.<div class="popup-ref"><span class="ref__series"><i>Mutat. Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">803-805</span></span>: <span class="ref__seriesPages">89-95</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.mrfmmm.2017.05.003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28552166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85019867355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=803-805&publication_year=2017&pages=89-95&journal=Mutat.+Res.&author=Ishiai+M.&author=Sato+K.&author=Tomida+J.&author=Kitao+H.&author=Kurumizaka+H.&author=Takata+M.&title=Activation+of+the+FA+pathway+mediated+by+phosphorylation+and+ubiquitination">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> We also observed previously undescribed associations between the phosphorylation of CDK2-T160 and the phosphorylation of PARP-S257 (r = 0.79, p = 0.039), and between CDK2-Y15 and 2 sites on nucleophosmin 1 (NPM1) (S243, r = 0.57, p = 0.002; S254, r = 0.58, p = 0.0009). These phosphoproteomics results suggest a possible regulatory mechanism for homologous repair that deserves further investigation.</div><div class="section-paragraph">The activity of the mRNA processing-associated kinase CLK1 was also significantly higher in HGSC compared to that in the normal FT tissues (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). In breast cancer, CLK1/4 kinases are involved in the regulation of the alternative splicing of <em>TP53</em>.<div class="dropBlock reference-citations"><sup><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d15348236e1081" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">33</a></sup><div data-db-target-of="bib33-d15348236e1081" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Marcel V. </li><li class="loa__item">Fernandes K. </li><li class="loa__item">Terrier O. </li><li class="loa__item">Lane D.P. </li><li class="loa__item">Bourdon J.C. </li></ul><div class="reference-citations__detials">Modulation of p53β and p53γ expression by regulating the alternative splicing of TP53 gene modifies cellular response.<div class="popup-ref"><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1377-1387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fcdd.2014.73&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24926616&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84905909213&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1377-1387&journal=Cell+Death+Differ.&author=Marcel+V.&author=Fernandes+K.&author=Terrier+O.&author=Lane+D.P.&author=Bourdon+J.C.&title=Modulation+of+p53%CE%B2+and+p53%CE%B3+expression+by+regulating+the+alternative+splicing+of+TP53+gene+modifies+cellular+response">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> CLK1 also plays a role in the response to replication stress,<div class="dropBlock reference-citations"><sup><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d15348236e1085" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">34</a></sup><div data-db-target-of="bib34-d15348236e1085" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Czubaty A. </li><li class="loa__item">Piekiełko-Witkowska A. </li></ul><div class="reference-citations__detials">Protein kinases that phosphorylate splicing factors: roles in cancer development, progression and possible therapeutic options.<div class="popup-ref"><span class="ref__series"><i>Int. J. Biochem. Cell Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">91</span></span>: <span class="ref__seriesPages">102-115</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.biocel.2017.05.024&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28552434&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85020007962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&pages=102-115&journal=Int.+J.+Biochem.+Cell+Biol.&author=Czubaty+A.&author=Piekie%C5%82ko-Witkowska+A.&title=Protein+kinases+that+phosphorylate+splicing+factors%3A+roles+in+cancer+development%2C+progression+and+possible+therapeutic+options">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and CLK1 and CLK2 inhibitors have been screened for use in triple-negative breast cancer (TNBC) through compromising the G1-S checkpoint,<div class="dropBlock reference-citations"><sup><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d15348236e1089" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">35</a></sup><div data-db-target-of="bib35-d15348236e1089" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhu D. </li><li class="loa__item">Xu S. </li><li class="loa__item">Deyanat-Yazdi G. </li><li class="loa__item">Peng S.X. </li><li class="loa__item">Barnes L.A. </li><li class="loa__item">Narla R.K. </li><li class="loa__item">Tran T. </li><li class="loa__item">Mikolon D. </li><li class="loa__item">Ning Y. </li><li class="loa__item">Shi T. </li><li>et al.</li></ul><div class="reference-citations__detials">Synthetic Lethal Strategy Identifies a Potent and Selective TTK and CLK1/2 Inhibitor for Treatment of Triple-Negative Breast Cancer with a Compromised G<sub>1</sub>-S Checkpoint.<div class="popup-ref"><span class="ref__series"><i>Mol. Cancer Ther.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">1727-1738</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1158%2F1535-7163.MCT-17-1084&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29866747&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85052403479&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1727-1738&journal=Mol.+Cancer+Ther.&author=Zhu+D.&author=Xu+S.&author=Deyanat-Yazdi+G.&author=Peng+S.X.&author=Barnes+L.A.&author=Narla+R.K.&author=Tran+T.&author=Mikolon+D.&author=Ning+Y.&author=Shi+T.&title=Synthetic+Lethal+Strategy+Identifies+a+Potent+and+Selective+TTK+and+CLK1%2F2+Inhibitor+for+Treatment+of+Triple-Negative+Breast+Cancer+with+a+Compromised+G1-S+Checkpoint">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> suggesting the potential utility of CLK1 inhibitors in HGSC. <a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D illustrates the connections between these activated kinases and the regulation of cell-cycle transit, DNA repair, and replication stress.</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Survival Signature from Retrospective HGSC Samples Correlates with Preliminary Survival Data in Prospective Cohort</span></h3><div class="section-paragraph">A key finding of our previous study of retrospective HGSC samples was a regression model-based minimal protein signature capable of predicting overall survival in a held-out test set of patients; “low” scores correlated with decreased survival time.<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e1102" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e1102" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> The utility of such signatures can only be established by validation in an independent sample set such as the prospective cohort in this study. During the first year of clinical follow-up, there were seven deaths reported within the cohort. In the present study, we applied our 90-protein signature to produce scores for each tumor.</div><div class="section-paragraph">We found that the mean score was lowest (indicative of short survival) in the seven deceased patients, intermediate in the currently surviving patients with tumors, and highest in the normal FT tissues (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>). The difference between the deceased group and the living group was statistically significant (p = 0.03), despite the small number of samples. These results support the ability of our signature to discriminate between short- and long-surviving patients with HGSC at the time of tumor resection, although further mortality data from this cohort will shed more light on the efficacy of the signature.<div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/dd8906ae-c517-426d-af30-7d84fe92300a/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/b9b544a3-26c5-4211-8baa-5445fc396720/gr7_lrg.jpg" data-large-image="/cms/attachment/dd8906ae-c517-426d-af30-7d84fe92300a/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Survival Signature Predicts Tumor Status</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">The survival signature identified in our previous study using retrospective HGSC samples was applied to proteomics data from the present study showing that it significantly discriminated between deceased and surviving patients and had a significantly different profile in normal tissues.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/dd8906ae-c517-426d-af30-7d84fe92300a/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/b9b544a3-26c5-4211-8baa-5445fc396720/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S2666-3791(20)30004-5&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We also examined the possibility that other clinical covariates may be associated with survival status in our current cohort. In our retrospective study, we analyzed the contribution of these clinical variables in a Kaplan-Meier survival model. However, as described previously, the survival analysis for the present study was precluded due to the limited follow-up data available (1 year). Accordingly, we tested for association using a t test for continuous variables (age) and Fisher’s exact test for categorical variables (site of disease, anatomic site of tumor, tumor stage, tumor grade, tumor residual disease, race, weight of tissue sample) in the seven deceased patients versus the other living patients (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/039539f0-07ea-4afc-85cf-5c40748795f0/mmc10" class="scroll-into-link">Table S9</a>), but we found that these variables did not statistically affect mortality (p &lt; 0.05) in this small subset of patients.</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0080" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">The most significant opportunity presented in this study of prospectively collected HGSC is the ability to compare the tumor proteome and phosphoproteome with an appropriate normal control for the tissue of origin, in this case FT, based on the current consensus that fallopian STIC lesions are the precursor of HGSC.<div class="dropBlock reference-citations"><sup><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d15348236e1122" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">36</a></sup><div data-db-target-of="bib36-d15348236e1122" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kroeger Jr., P.T. </li><li class="loa__item">Drapkin R. </li></ul><div class="reference-citations__detials">Pathogenesis and heterogeneity of ovarian cancer.<div class="popup-ref"><span class="ref__series"><i>Curr. Opin. Obstet. Gynecol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">26-34</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1097%2FGCO.0000000000000340&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27898521&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85000399534&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=26-34&journal=Curr.+Opin.+Obstet.+Gynecol.&author=Kroeger+P.T.&author=Drapkin+R.&title=Pathogenesis+and+heterogeneity+of+ovarian+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d15348236e1127" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">37</a></sup><div data-db-target-of="bib37-d15348236e1127" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Park E.Y. </li><li class="loa__item">Kim O. </li><li class="loa__item">Schilder J.M. </li><li class="loa__item">Coffey D.M. </li><li class="loa__item">Cho C.H. </li><li class="loa__item">Bast Jr., R.C. </li></ul><div class="reference-citations__detials">Cell Origins of High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">E433</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3390%2Fcancers10110433&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30424539&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85057199902&amp;cf=" onclick="newWindow(this.href);return false">Scopus (158)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=E433&journal=Cancers+%28Basel%29&author=Kim+J.&author=Park+E.Y.&author=Kim+O.&author=Schilder+J.M.&author=Coffey+D.M.&author=Cho+C.H.&author=Bast+R.C.&title=Cell+Origins+of+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Here, we provide an extensive comparison of HGSC and FT at the proteome level and the phosphoproteome level using surgical specimens rather than cell lines. Because TCGA tumor samples were collected with a focus on genomic analyses, DNA from peripheral blood monocytic cells (PBMCs) was used as the “normal control” and somatic mutations in the tumor were identified by comparison to the patient’s own germline DNA from PBMCs. There was no attempt to collect normal adjacent tissue from the same organ as the tumor in most TCGA studies. In addition to germline DNA for the assessment of somatic mutations, we compared tumor tissue to two normal tissue controls, FT and FTE. HGSC showed reduced diversity of the global proteome compared to normal FT; specifically, normal FT featured a higher abundance of proteins associated with immune regulation, including both innate and adaptive immune responses, as well as neutrophil- and leukocyte-mediated immunity. This enrichment was preserved when the comparison was limited to proteins observed in FTE samples obtained by cytobrushing to control for proteins associated with non-epithelial tissue layers (e.g., muscularis, serosa). Although it may seem counterintuitive to observe an increase in immune-associated protein expression in the normal FT compared to the HGSC tumor, several recent proteomic studies have identified “immune desert” subtypes, specifically renal clear cell carcinoma,<div class="dropBlock reference-citations"><sup><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d15348236e1131" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">38</a></sup><div data-db-target-of="bib38-d15348236e1131" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Pan J. </li><li class="loa__item">Song X. </li><li class="loa__item">Hu Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Reva B. </li><li class="loa__item">Lih T.M. </li><li class="loa__item">Chang H.Y. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e31</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e31&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Integrated+Proteogenomic+Characterization+of+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> breast cancer,<div class="dropBlock reference-citations"><sup><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d15348236e1135" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">39</a></sup><div data-db-target-of="bib39-d15348236e1135" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xiao Y. </li><li class="loa__item">Ma D. </li><li class="loa__item">Zhao S. </li><li class="loa__item">Suo C. </li><li class="loa__item">Shi J. </li><li class="loa__item">Xue M.Z. </li><li class="loa__item">Ruan M. </li><li class="loa__item">Wang H. </li><li class="loa__item">Zhao J. </li><li class="loa__item">Li Q. </li><li>et al.</li></ul><div class="reference-citations__detials">Multi-Omics Profiling Reveals Distinct Microenvironment Characterization and Suggests Immune Escape Mechanisms of Triple-Negative Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1158/1078-0432.CCR-18-3524">https://doi.org/10.1158/1078-0432.CCR-18-3524</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1158%2F1078-0432.CCR-18-3524&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85069636726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (237)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=Clin.+Cancer+Res.&author=Xiao+Y.&author=Ma+D.&author=Zhao+S.&author=Suo+C.&author=Shi+J.&author=Xue+M.Z.&author=Ruan+M.&author=Wang+H.&author=Zhao+J.&author=Li+Q.&title=Multi-Omics+Profiling+Reveals+Distinct+Microenvironment+Characterization+and+Suggests+Immune+Escape+Mechanisms+of+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and head and neck cancer.<div class="dropBlock reference-citations"><sup><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d15348236e1139" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">40</a></sup><div data-db-target-of="bib40-d15348236e1139" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Echarti A. </li><li class="loa__item">Hecht M. </li><li class="loa__item">Büttner-Herold M. </li><li class="loa__item">Haderlein M. </li><li class="loa__item">Hartmann A. </li><li class="loa__item">Fietkau R. </li><li class="loa__item">Distel L. </li></ul><div class="reference-citations__detials">CD8+ and Regulatory T cells Differentiate Tumor Immune Phenotypes and Predict Survival in Locally Advanced Head and Neck Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">E1398</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3390%2Fcancers11091398&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31546872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85078775668&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=E1398&journal=Cancers+%28Basel%29&author=Echarti+A.&author=Hecht+M.&author=B%C3%BCttner-Herold+M.&author=Haderlein+M.&author=Hartmann+A.&author=Fietkau+R.&author=Distel+L.&title=CD8%2B+and+Regulatory+T%C2%A0cells+Differentiate+Tumor+Immune+Phenotypes+and+Predict+Survival+in+Locally+Advanced+Head+and+Neck+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> It is possible that HGSC is dominated by an immune desert phenotype, potentially contributing to the poor prognosis for HGSC patients.</div><div class="section-paragraph">Gain-of-function events in the tumors were most evident in the phosphoproteome and were characterized by a marked increase in relative phosphorylation across multiple phosphoproteins, particularly CDKs and mitotic kinases. Many of the pathways upregulated in association with short survival in retrospective HGSC samples were also enriched in HGSC compared to FT, specifically RhoA, Notch, integrin-like kinase, and Rac. These observations suggest a continuum of pathway activations from normal FT through HGSC, associated with long-term survival, to the most aggressive forms of HGSC, associated with short-term survival. The observed changes are consistent with the upregulation of proteins associated with the cancer hallmarks of proliferation (CDKs and AURKA) and motility and invasion (RhoA and Rac) as well as cell-cell communication (Notch and integrin-like kinase).</div><div class="section-paragraph">In our prior retrospective study of HGSC,<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e1147" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e1147" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> we derived a survival-associated signature based on differential <em>trans</em>-regulated protein abundance in short-term versus long-term survivors. Using survival data for the first year of clinical follow-up available for patients in the prospective cohort, we determined that tumors from the seven deceased patients had significantly worse survival scores than those from surviving patients, and the normal tissues from all of the patients had better survival scores than from either set of tumors. Further development of these models could promote their use in a precision medicine context to guide the use of targeted therapies either in conjunction with initial platinum treatment or after recurrence. A second clinically relevant observation was the association of decreased histone H4 acetylation, specifically on K12 and K16, with positive HRD status; this observation was confirmed in the prospective cohort. Given the preliminary success of PARP inhibitor therapies in TNBC,<div class="dropBlock reference-citations"><sup><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d15348236e1153" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">41</a></sup><div data-db-target-of="bib41-d15348236e1153" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beniey M. </li><li class="loa__item">Haque T. </li><li class="loa__item">Hassan S. </li></ul><div class="reference-citations__detials">Translating the role of PARP inhibitors in triple-negative breast cancer.<div class="popup-ref"><span class="ref__series"><i>Oncoscience.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">287-288</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.18632%2Foncoscience.474&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30800714&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=287-288&journal=Oncoscience&author=Beniey+M.&author=Haque+T.&author=Hassan+S.&title=Translating+the+role+of+PARP+inhibitors+in+triple-negative+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> a robust biomarker for HRD status independent of known pathogenomic mutations in <em>BRCA1</em> and <em>BRCA2</em> could be a useful adjunct to guide therapy choices in women with HGSC.</div><div class="section-paragraph">The control of ischemia time in the prospective sample collection process enabled a deeper analysis of the phosphoproteomic data. This stringent criterion allowed us to include the ischemia-responsive p38 and other stress kinases in the analysis of the phosphoproteome. Use of KSEA to infer kinase activity from the phosphorylation status of known kinase substrates indicated significantly increased activation in HGSC of the mitotic kinase AURKA, CDK1, CDK2, CDK4, and CDK7, and the mRNA processing-associated kinase CLK1. All of these kinases are potential targets of therapeutic intervention in HGSC, particularly given the availability of FDA-approved inhibitors for CDKs and Aurora kinases.</div><div class="section-paragraph">In addition to opening the prospect of additional therapeutic strategies for HGSC, characterizing the phosphorylation status of specific, well-annotated phosphosites has the potential to provide insight into HGSC biology. This work shows that RB1 is consistently hyperphosphorylated in HGSC, largely independent of changes in RB1 copy number, thus facilitating a general increase in proliferation that is consistent with the increased phosphorylation of the AURKA mitotic kinase. Comparison of specific phosphorylation sites on CDK2 shows increased phosphorylation of CDK2-T160, which is consistent with increased proliferation and increased phosphorylation of CDK2-Y15. Phosphorylation at this site has been associated with cell-cycle exit in response to replication stress. The latter observation, together with the observed increase in CLK1 activity, suggest a general response to increased replication stress in HGSC that is borne out through the phosphoproteome, as evidenced by increased phosphorylation of PARP, FANCD, and NPM.</div><div class="section-paragraph">Finally, we hypothesize that phosphoproteomics reveals a potential mechanism to explain the high degree of chromosomal instability characteristic of HGSC (<a data-link="modal" data-target="#image-S2666-3791(20)30004-5gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D). Increased proliferation driven by mitotic kinase activation and RB1 hyperphosphorylation induces a state of replication stress, evidenced by CDK2-pY15, increased CLK1 activity, and increased activity of ATR and ATM. In the presence of impaired capabilities for DNA repair resulting from breast cancer gene 1 and 2 (BRCA1 and 2) mutations or dysregulation of PARP and FANCD, this could result in increased non-homologous end-joining and chromosomal translocations. This pattern of oncogene-driven increased proliferation leading to replicative stress and genomic instability is increasingly recognized as a novel hallmark of cancer.<div class="dropBlock reference-citations"><sup><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d15348236e1170" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">40</a></sup><div data-db-target-of="bib40-d15348236e1170" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Echarti A. </li><li class="loa__item">Hecht M. </li><li class="loa__item">Büttner-Herold M. </li><li class="loa__item">Haderlein M. </li><li class="loa__item">Hartmann A. </li><li class="loa__item">Fietkau R. </li><li class="loa__item">Distel L. </li></ul><div class="reference-citations__detials">CD8+ and Regulatory T cells Differentiate Tumor Immune Phenotypes and Predict Survival in Locally Advanced Head and Neck Cancer.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">E1398</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3390%2Fcancers11091398&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31546872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85078775668&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=E1398&journal=Cancers+%28Basel%29&author=Echarti+A.&author=Hecht+M.&author=B%C3%BCttner-Herold+M.&author=Haderlein+M.&author=Hartmann+A.&author=Fietkau+R.&author=Distel+L.&title=CD8%2B+and+Regulatory+T%C2%A0cells+Differentiate+Tumor+Immune+Phenotypes+and+Predict+Survival+in+Locally+Advanced+Head+and+Neck+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d15348236e1175" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">42</a></sup><div data-db-target-of="bib42-d15348236e1175" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Taylor E.M. </li><li class="loa__item">Lindsay H.D. </li></ul><div class="reference-citations__detials">DNA replication stress and cancer: cause or cure?.<div class="popup-ref"><span class="ref__series"><i>Future Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">221-237</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.2217%2Ffon.15.292&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26616915&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84962627069&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=221-237&journal=Future+Oncol.&author=Taylor+E.M.&author=Lindsay+H.D.&title=DNA+replication+stress+and+cancer%3A+cause+or+cure%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Therapies that break this cycle by targeting cyclin-dependent kinases or that leverage damaged DNA repair to induce apoptosis (e.g., use of PARP inhibitors) may prove particularly effective in HGSC when used in conjunction with DNA-damaging platinum-based therapies. Although additional research will be required to build a prognostic marker, it is possible that proteomic indicators of replication stress and mitotic kinase activation could eventually be applied to HGSC tissues at the time of cytoreduction, providing guidance on the adjuvant use of targeted therapies with traditional platinum-based therapies.</div><div class="section-paragraph">In conclusion, analysis of prospectively collected HGSC samples from 83 patients provided significant insights stemming from direct comparisons to normal precursor tissues. Key elements of the prior retrospective study were confirmed, including the variation in mRNA-protein correlation by functional group, the consistent enrichment of immune regulatory pathways despite heterogeneity in the focal sites of SCNA, and the performance of a protein abundance-based survival signature to discriminate between short-term survivors (currently deceased) and longer-term survivors (currently alive). Histone H4 acetylation remains a promising indicator of HRD status. Comprehensive analysis of kinase activity, inferred from the phosphorylation of known kinase substrates, suggests a key role for cell-cycle control and replication stress in HGSC, and the identification of multiple CDKs with increased activity in tumors compared to normal tissue suggests a potential application for FDA-approved CDK inhibitors in ovarian cancer.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0085" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the National Cancer Institute Clinical Proteomic Tumor Analysis Consortium are: Jasmin Bavarva, Emily Boja, Melissa Borucki, Shuang Cai, Steven A. Carr, Daniel Chan, Xian Chen, Karl R. Clauser, Sherri R. Davies, Li Ding, Nathan J. Edwards, Kimberly Elburn, Matthew J. C. Ellis, David Fenyo, Michael A. Gillette, Linda Hannick, Tara Hiltke, Andrew Hoofnagle, Karen A. Ketchum, Christopher R. Kinsinger, Eric Kuhn, Douglas A. Levine, Daniel C. Liebler, Tao Liu, Subha Madhavan, D. R. Mani, Jason E. McDermott, Peter B. McGarvey, Philip Mertins, Mehdi Mesri, Mauricio Oberti, Akhilesh Pandey, Amanda Paulovich, Samuel H. Payne, David F. Ransohoff, Robert C. Rivers, Karin D. Rodland, Henry Rodriguez, Paul A. Rudnick, Kelly Ruggles, Melinda E. Sanders, Zhiao Shi, Ie-Ming Shih, Steven J. Skates, Robert J. C. Slebos, Richard D. Smith, Michael Snyder, Stephen E. Stein, David L. Tabb, Ratna R. Thangudu, Mathangi Thiagarajan, R. Reid Townsend, Negin Vatanian, Yue Wang, Mark Watson, Forest M. White, Jeffery Whiteaker, Gordon A. Whiteley, Bing Zhang, Hui Zhang, Zhen Zhang, Yingming Zhao, Heng Zhu, and Lisa J. Zimmerman.</div></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec5.1" data-locator="sec5.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key Resources Table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Biological Samples</strong></td></tr><tr><td>Primary tumor and normal tissue samples</td><td>CPTAC Biospecimen Core Resource</td><td>See <a href="#sec5.3" class="scroll-into-link">Experimental Model and Subject Details</a></td></tr><tr><td class="rowsep">Patient-derived xenograft tissue samples</td><td class="rowsep">Washington University in St. Louis</td><td class="rowsep">See <a href="#sec5.4" class="scroll-into-link">Method Details</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Critical Commercial Assays</strong></td></tr><tr><td>Illumina Barcoded Paired-End Library Preparation Kit</td><td>Illumina</td><td>Cat#PE-410-1001</td></tr><tr><td>Illumina’s Infinium LCG Quad Assay</td><td>Illumina</td><td>Cat#15025908</td></tr><tr><td>TruSeq Stranded RNA Sample Preparation Kit</td><td>Illumina</td><td>Cat#RS-122-2103</td></tr><tr><td>TruSeq Small Total RNA Sample Prep Kit</td><td>Illumina</td><td>Cat#RS-200-0048</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Cat#V517</td></tr><tr><td>Tandem mass tags - 10plex</td><td>Thermo Scientific</td><td>Cat#90110</td></tr><tr><td class="rowsep">Ni-NTA Superflow agarose beads</td><td class="rowsep">QIAGEN</td><td class="rowsep">Cat#30410</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited Data</strong></td></tr><tr><td>CPTAC Ovarian raw genomic data</td><td>database of Genotypes and Phenotypes (dbGaP)</td><td><a class="inter-ref-link" href="https://www.ncbi.nlm.nih.gov/gap/" target="_blank">https://www.ncbi.nlm.nih.gov/gap/</a>, phs000892</td></tr><tr><td class="rowsep">Proteomic data</td><td class="rowsep">CPTAC Data Portal</td><td class="rowsep"><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>mzRefinery</td><td><div class="dropBlock reference-citations"><sup><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d15348236e1309" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">43</a></sup><div data-db-target-of="bib43-d15348236e1309" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbons B.C. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Payne S.H. </li></ul><div class="reference-citations__detials">Correcting systematic bias and instrument measurement drift with mzRefinery.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtv437&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84950239084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinformatics&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://omics.pnl.gov/software/mzrefinery" target="_blank">https://omics.pnl.gov/software/mzrefinery</a></td></tr><tr><td>MASIC</td><td><div class="dropBlock reference-citations"><sup><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d15348236e1319" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">44</a></sup><div data-db-target-of="bib44-d15348236e1319" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/PNNL-Comp-Mass-Spec/MASIC/releases" target="_blank">https://github.com/PNNL-Comp-Mass-Spec/MASIC/releases</a></td></tr><tr><td>MS-GF+</td><td><div class="dropBlock reference-citations"><sup><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d15348236e1329" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">45</a></sup><div data-db-target-of="bib45-d15348236e1329" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://omics.pnl.gov/software/ms-gf" target="_blank">https://omics.pnl.gov/software/ms-gf</a></td></tr><tr><td>ProteoWizard</td><td><div class="dropBlock reference-citations"><sup><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d15348236e1339" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">46</a></sup><div data-db-target-of="bib46-d15348236e1339" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kessner D. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Burke R. </li><li class="loa__item">Agus D. </li><li class="loa__item">Mallick P. </li></ul><div class="reference-citations__detials">ProteoWizard: open source software for rapid proteomics tools development.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1395)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://proteowizard.sourceforge.net" target="_blank">http://proteowizard.sourceforge.net</a></td></tr><tr><td>COSMIC</td><td><div class="dropBlock reference-citations"><sup><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d15348236e1349" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">47</a></sup><div data-db-target-of="bib47-d15348236e1349" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Forbes S.A. </li><li class="loa__item">Beare D. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Tate J. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Ward S. </li><li class="loa__item">Dawson E. </li><li class="loa__item">Ponting L. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: somatic cancer genetics at high-resolution.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D777-D783</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkw1121&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27899578&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85016154270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1390)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D777-D783&journal=Nucleic+Acids+Res.&author=Forbes+S.A.&author=Beare+D.&author=Boutselakis+H.&author=Bamford+S.&author=Bindal+N.&author=Tate+J.&author=Cole+C.G.&author=Ward+S.&author=Dawson+E.&author=Ponting+L.&title=COSMIC%3A+somatic+cancer+genetics+at+high-resolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cancer.sanger.ac.uk/cosmic" target="_blank">https://cancer.sanger.ac.uk/cosmic</a></td></tr><tr><td>SomaticWrapper</td><td>N/A</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/somaticwrapper" target="_blank">https://github.com/ding-lab/somaticwrapper</a></td></tr><tr><td>Mutect1</td><td><div class="dropBlock reference-citations"><sup><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d15348236e1367" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">48</a></sup><div data-db-target-of="bib48-d15348236e1367" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3325)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/mutect" target="_blank">https://software.broadinstitute.org/cancer/cga/mutect</a></td></tr><tr><td>Strelka1</td><td><div class="dropBlock reference-citations"><sup><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d15348236e1378" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">49</a></sup><div data-db-target-of="bib49-d15348236e1378" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Saunders C.T. </li><li class="loa__item">Wong W.S. </li><li class="loa__item">Swamy S. </li><li class="loa__item">Becq J. </li><li class="loa__item">Murray L.J. </li><li class="loa__item">Cheetham R.K. </li></ul><div class="reference-citations__detials">Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1811-1817</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbts271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22581179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84864153492&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1125)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1811-1817&journal=Bioinformatics&author=Saunders+C.T.&author=Wong+W.S.&author=Swamy+S.&author=Becq+J.&author=Murray+L.J.&author=Cheetham+R.K.&title=Strelka%3A+accurate+somatic+small-variant+calling+from+sequenced+tumor-normal+sample+pairs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/Illumina/strelka" target="_blank">https://github.com/Illumina/strelka</a></td></tr><tr><td>VarScan2</td><td><div class="dropBlock reference-citations"><sup><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d15348236e1388" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">50</a></sup><div data-db-target-of="bib50-d15348236e1388" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3368)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://dkoboldt.github.io/varscan/" target="_blank">https://dkoboldt.github.io/varscan/</a></td></tr><tr><td>Pindel</td><td><div class="dropBlock reference-citations"><sup><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d15348236e1398" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">51</a></sup><div data-db-target-of="bib51-d15348236e1398" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1501)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/genome/pindel" target="_blank">https://github.com/genome/pindel</a></td></tr><tr><td>dbSNP</td><td><div class="dropBlock reference-citations"><sup><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d15348236e1408" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">52</a></sup><div data-db-target-of="bib52-d15348236e1408" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sherry S.T. </li><li class="loa__item">Ward M.H. </li><li class="loa__item">Kholodov M. </li><li class="loa__item">Baker J. </li><li class="loa__item">Phan L. </li><li class="loa__item">Smigielski E.M. </li><li class="loa__item">Sirotkin K. </li></ul><div class="reference-citations__detials">dbSNP: the NCBI database of genetic variation.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">308-311</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2F29.1.308&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=11125122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0035173378&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5316)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=308-311&journal=Nucleic+Acids+Res.&author=Sherry+S.T.&author=Ward+M.H.&author=Kholodov+M.&author=Baker+J.&author=Phan+L.&author=Smigielski+E.M.&author=Sirotkin+K.&title=dbSNP%3A+the+NCBI+database+of+genetic+variation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.ncbi.nlm.nih.gov/snp" target="_blank">https://www.ncbi.nlm.nih.gov/snp</a></td></tr><tr><td>CrossMap</td><td><div class="dropBlock reference-citations"><sup><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d15348236e1418" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">53</a></sup><div data-db-target-of="bib53-d15348236e1418" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhao H. </li><li class="loa__item">Sun Z. </li><li class="loa__item">Wang J. </li><li class="loa__item">Huang H. </li><li class="loa__item">Kocher J.P. </li><li class="loa__item">Wang L. </li></ul><div class="reference-citations__detials">CrossMap: a versatile tool for coordinate conversion between genome assemblies.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">1006-1007</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtt730&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24351709&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84897373336&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=1006-1007&journal=Bioinformatics&author=Zhao+H.&author=Sun+Z.&author=Wang+J.&author=Huang+H.&author=Kocher+J.P.&author=Wang+L.&title=CrossMap%3A+a+versatile+tool+for+coordinate+conversion+between+genome+assemblies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://crossmap.sourceforge.net/" target="_blank">http://crossmap.sourceforge.net/</a></td></tr><tr><td>GermlineWrapper</td><td>N/A</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/germlinewrapper" target="_blank">https://github.com/ding-lab/germlinewrapper</a></td></tr><tr><td>GATK</td><td><div class="dropBlock reference-citations"><sup><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d15348236e1436" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">54</a></sup><div data-db-target-of="bib54-d15348236e1436" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li class="loa__item">DePristo M.A. </li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17509)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://gatk.broadinstitute.org" target="_blank">https://gatk.broadinstitute.org</a></td></tr><tr><td>Variant Effect Predictor (VEP)</td><td><div class="dropBlock reference-citations"><sup><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d15348236e1446" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">55</a></sup><div data-db-target-of="bib55-d15348236e1446" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McLaren W. </li><li class="loa__item">Gil L. </li><li class="loa__item">Hunt S.E. </li><li class="loa__item">Riat H.S. </li><li class="loa__item">Ritchie G.R. </li><li class="loa__item">Thormann A. </li><li class="loa__item">Flicek P. </li><li class="loa__item">Cunningham F. </li></ul><div class="reference-citations__detials">The Ensembl Variant Effect Predictor.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3975)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://uswestembl.org/info/docs/tools/vep/index.html" target="_blank">https://uswestembl.org/info/docs/tools/vep/index.html</a></td></tr><tr><td>CharGer</td><td><div class="dropBlock reference-citations"><sup><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d15348236e1456" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">56</a></sup><div data-db-target-of="bib56-d15348236e1456" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Scott A.D. </li><li class="loa__item">Huang K.L. </li><li class="loa__item">Weerasinghe A. </li><li class="loa__item">Mashl R.J. </li><li class="loa__item">Gao Q. </li><li class="loa__item">Martins Rodrigues F. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Ding L. </li></ul><div class="reference-citations__detials">CharGer: Clinical Characterization of Germline Variants.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">865-867</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbty649&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30102335&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85062278783&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=865-867&journal=Bioinformatics&author=Scott+A.D.&author=Huang+K.L.&author=Weerasinghe+A.&author=Mashl+R.J.&author=Gao+Q.&author=Martins+Rodrigues+F.&author=Wyczalkowski+M.A.&author=Ding+L.&title=CharGer%3A+Clinical+Characterization+of+Germline+Variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/ding-lab/CharGer" target="_blank">https://github.com/ding-lab/CharGer</a></td></tr><tr><td>ExAC</td><td><div class="dropBlock reference-citations"><sup><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d15348236e1466" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">57</a></sup><div data-db-target-of="bib57-d15348236e1466" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karczewski K.J. </li><li class="loa__item">Weisburd B. </li><li class="loa__item">Thomas B. </li><li class="loa__item">Solomonson M. </li><li class="loa__item">Ruderfer D.M. </li><li class="loa__item">Kavanagh D. </li><li class="loa__item">Hamamsy T. </li><li class="loa__item">Lek M. </li><li class="loa__item">Samocha K.E. </li><li class="loa__item">Cummings B.B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>The Exome Aggregation Consortium</span><br />The ExAC browser: displaying reference data information from over 60 000 exomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D840-D845</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkw971&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27899611&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85011303767&amp;cf=" onclick="newWindow(this.href);return false">Scopus (493)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D840-D845&journal=Nucleic+Acids+Res.&author=The+Exome+Aggregation+Consortium&title=The+ExAC+browser%3A+displaying+reference+data+information+from+over+60+000+exomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://exac.broadinstitute.org/" target="_blank">http://exac.broadinstitute.org/</a></td></tr><tr><td>ClinVar</td><td><div class="dropBlock reference-citations"><sup><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d15348236e1476" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">58</a></sup><div data-db-target-of="bib58-d15348236e1476" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Landrum M.J. </li><li class="loa__item">Lee J.M. </li><li class="loa__item">Benson M. </li><li class="loa__item">Brown G. </li><li class="loa__item">Chao C. </li><li class="loa__item">Chitipiralla S. </li><li class="loa__item">Gu B. </li><li class="loa__item">Hart J. </li><li class="loa__item">Hoffman D. </li><li class="loa__item">Hoover J. </li><li>et al.</li></ul><div class="reference-citations__detials">ClinVar: public archive of interpretations of clinically relevant variants.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D862-D868</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkv1222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26582918&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84976904305&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1797)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D862-D868&journal=Nucleic+Acids+Res.&author=Landrum+M.J.&author=Lee+J.M.&author=Benson+M.&author=Brown+G.&author=Chao+C.&author=Chitipiralla+S.&author=Gu+B.&author=Hart+J.&author=Hoffman+D.&author=Hoover+J.&title=ClinVar%3A+public+archive+of+interpretations+of+clinically+relevant+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">https://www.ncbi.nlm.nih.gov/clinvar/</a></td></tr><tr><td>gnomAD</td><td><div class="dropBlock reference-citations"><sup><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d15348236e1487" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">57</a></sup><div data-db-target-of="bib57-d15348236e1487" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karczewski K.J. </li><li class="loa__item">Weisburd B. </li><li class="loa__item">Thomas B. </li><li class="loa__item">Solomonson M. </li><li class="loa__item">Ruderfer D.M. </li><li class="loa__item">Kavanagh D. </li><li class="loa__item">Hamamsy T. </li><li class="loa__item">Lek M. </li><li class="loa__item">Samocha K.E. </li><li class="loa__item">Cummings B.B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>The Exome Aggregation Consortium</span><br />The ExAC browser: displaying reference data information from over 60 000 exomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D840-D845</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkw971&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27899611&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85011303767&amp;cf=" onclick="newWindow(this.href);return false">Scopus (493)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D840-D845&journal=Nucleic+Acids+Res.&author=The+Exome+Aggregation+Consortium&title=The+ExAC+browser%3A+displaying+reference+data+information+from+over+60+000+exomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://gnomad.broadinstitute.org/" target="_blank">https://gnomad.broadinstitute.org/</a></td></tr><tr><td>SIFT</td><td><div class="dropBlock reference-citations"><sup><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d15348236e1497" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">59</a></sup><div data-db-target-of="bib59-d15348236e1497" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kumar P. </li><li class="loa__item">Henikoff S. </li><li class="loa__item">Ng P.C. </li></ul><div class="reference-citations__detials">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1073-1081</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnprot.2009.86&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=19561590&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-68149165614&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5290)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1073-1081&journal=Nat.+Protoc.&author=Kumar+P.&author=Henikoff+S.&author=Ng+P.C.&title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>N/A</td></tr><tr><td>PolyPhen</td><td><div class="dropBlock reference-citations"><sup><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d15348236e1506" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">60</a></sup><div data-db-target-of="bib60-d15348236e1506" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Adzhubei I. </li><li class="loa__item">Jordan D.M. </li><li class="loa__item">Sunyaev S.R. </li></ul><div class="reference-citations__detials">Predicting functional effect of human missense mutations using PolyPhen-2.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Hum. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">Chapter 7</span></span>: <span class="ref__seriesPages">Unit 7.20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=Chapter+7&publication_year=2013&pages=Unit+7.20&journal=Curr.+Protoc.+Hum.+Genet.&author=Adzhubei+I.&author=Jordan+D.M.&author=Sunyaev+S.R.&title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>N/A</td></tr><tr><td>Integrative Genomics Viewer (IGV)</td><td><div class="dropBlock reference-citations"><sup><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d15348236e1515" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">61</a></sup><div data-db-target-of="bib61-d15348236e1515" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson J.T. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Winckler W. </li><li class="loa__item">Guttman M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Integrative genomics viewer.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">24-26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt.1754&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21221095&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-78651271733&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9132)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=24-26&journal=Nat.+Biotechnol.&author=Robinson+J.T.&author=Thorvaldsd%C3%B3ttir+H.&author=Winckler+W.&author=Guttman+M.&author=Lander+E.S.&author=Getz+G.&author=Mesirov+J.P.&title=Integrative+genomics+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/software/igv/" target="_blank">https://software.broadinstitute.org/software/igv/</a></td></tr><tr><td>GISTIC 2.0</td><td><div class="dropBlock reference-citations"><sup><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d15348236e1525" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">9</a></sup><div data-db-target-of="bib9-d15348236e1525" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>N/A</td></tr><tr><td>MultiOmicsViz</td><td><div class="dropBlock reference-citations"><sup><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d15348236e1534" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">62</a></sup><div data-db-target-of="bib62-d15348236e1534" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li></ul><div class="reference-citations__detials">multiOmicsViz: plot the effect of one omics data on other omics data along the chromosome.<div class="popup-ref"><div class="citation-host"><a class="inter-ref-link" href="https://rdrr.io/bioc/multiOmicsViz/" target="_blank">https://rdrr.io/bioc/multiOmicsViz/</a></div></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Wang+J.+multiOmicsViz%3A+plot+the+effect+of+one+omics+data+on+other+omics+data+along+the+chromosome.https%3A%2F%2Frdrr.io%2Fbioc%2FmultiOmicsViz%2F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>10.18129/B9.bioc.multiOmicsViz</td></tr><tr><td>clusterProfiler</td><td><div class="dropBlock reference-citations"><sup><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d15348236e1543" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">63</a></sup><div data-db-target-of="bib63-d15348236e1543" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yu G. </li><li class="loa__item">Wang L.G. </li><li class="loa__item">Han Y. </li><li class="loa__item">He Q.Y. </li></ul><div class="reference-citations__detials">clusterProfiler: an R package for comparing biological themes among gene clusters.<div class="popup-ref"><span class="ref__series"><i>OMICS.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">284-287</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1089%2Fomi.2011.0118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22455463&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84860718683&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17419)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=284-287&journal=OMICS&author=Yu+G.&author=Wang+L.G.&author=Han+Y.&author=He+Q.Y.&title=clusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clusters">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>10.18129/B9.bioc.clusterProfiler</td></tr><tr><td>limma</td><td><div class="dropBlock reference-citations"><sup><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d15348236e1552" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">64</a></sup><div data-db-target-of="bib64-d15348236e1552" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20650)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>10.18129/B9.bioc.limma</td></tr><tr><td>KSEAapp</td><td><div class="dropBlock reference-citations"><sup><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d15348236e1561" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">19</a></sup><div data-db-target-of="bib19-d15348236e1561" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wiredja D.D. </li><li class="loa__item">Koyutürk M. </li><li class="loa__item">Chance M.R. </li></ul><div class="reference-citations__detials">The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3489-3491</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtx415&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28655153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044408024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (135)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3489-3491&journal=Bioinformatics&author=Wiredja+D.D.&author=Koyut%C3%BCrk+M.&author=Chance+M.R.&title=The+KSEA+App%3A+a+web-based+tool+for+kinase+activity+inference+from+quantitative+phosphoproteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/KSEAapp/index.html" target="_blank">https://cran.r-project.org/web/packages/KSEAapp/index.html</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S2666-3791%2820%2930004-5">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec5.2" data-locator="sec5.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead Contact and Materials Availability</span></h3><div class="section-paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Karin Rodland (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#f39892819a9ddd819c979f929d97b3839d9d9fdd949c85" target="_blank"><span class="__cf_email__" data-cfemail="177c76657e79396578737b767973576779797b39707861">[email&#160;protected]</span></a>). This study did not generate new unique reagents.</div></section><section id="sec5.3" data-locator="sec5.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental Model and Subject Details</span></h3><section id="sec5.3.1" data-locator="sec5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimens and Clinical Data</span></h4><div class="section-paragraph">Ovarian tumor, normal Fallopian tube (FT), and blood samples were collected from 100 women between the ages of 36 and 86. Tissues were collected at twelve Tissue Source Sites (TSS) in strict accordance to the CPTAC-2 ovarian cancer procurement protocol, and institutional review boards at each TSS reviewed protocols and consent documentation, in adherence to Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Of the 100 HGSC cases collected, 83 had both tumor and FT that passed molecular qualification, while ten patients provided FT only, as the tumor failed molecular qualification, due to insufficient yield, poor gel scores, or RIN values below 7, as described in more detail below. The inclusion criteria encompassed newly diagnosed, untreated patients undergoing primary surgery for high grade serous adenocarcinoma of the ovary. Patients with prior history of other malignancies within twelve months, any systematic chemotherapy, endocrine or biological therapy as well as prior radiation therapy to the abdomen or pelvis for any cancer type were excluded from the study. Required clinical information regarding patient history and status of surgery along with relevant diagnostic information were collected using case reports forms. One-year follow-up information and updated history after completion of the initial treatment regimen were also collected through follow-up forms. De-identified pathology reports and representative diagnostic slide images were utilized to review and qualify cases for this study. Peripheral venous blood was collected from each patient prior to administration of anesthesia. Segments from qualified tumor specimens were greater than 300 mg in mass with at least 60% tumor cell nuclei and less than 20% necrosis. Tissue and the adjacent normal specimens were collected with less than 30 min total ischemic time and embedded in optimal cutting temperature (OCT) compound for processing at a common CPTAC-2 Biospecimen Core Resource (BCR) center. Pathologically-qualified cases underwent further molecular qualification for extraction and co-isolation of nucleic acids. Tissue segments that passed pathology and molecular qualifications were shipped to proteomic characterization centers. DNA and RNA from the same tumor segment and DNA from germline blood were further aliquoted and quantified per protocol. DNA quality was confirmed using gel electrophoresis and Nano drop methods. RNA quality was confirmed using Nano drop and Agilent bioanalyzer. Sufficient yield, a good gel score and a RIN value of 7 or greater qualified the DNA and RNA for sequencing, respectively. Analytes were then shipped to the sequencing center. Corresponding clinical data were formatted and distributed through the CPTAC data coordinating center (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038</a>) and are summarized in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/27ca9135-f87e-493a-87b3-c5db2d0d8005/mmc2" class="scroll-into-link">Table S1</a>.</div></section></section><section id="sec5.4" data-locator="sec5.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method Details</span></h3><section id="sec5.4.1" data-locator="sec5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein Extraction and Tryptic Digestion</span></h4><div class="section-paragraph">The tumor and normal Fallopian tube tissue samples were obtained as pulverized OCT-embedded tissues through the CPTAC BCR. Approximately 50 mg of each of the samples were homogenized separately in 600 μL of lysis buffer (8 M urea, 100 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8, 0.1% NP-40, 0.5% sodium deoxycholate, 10 mM NaF, phosphatase inhibitor cocktail 2, phosphatase inhibitor cocktail 3, 20 μM PUGNAc). Lysates were precleared by centrifugation at 16,500 g for 5 min at 4°C and protein concentrations were determined by BCA assay (Pierce). Proteins were reduced with 5 mM dithiothreitol for 1 h at 37°C and subsequently alkylated with 10 mM iodoacetamide for 1 h at 25°C in the dark. Samples were diluted 1:2 with 100 mM NH<sub>4</sub>HCO<sub>3</sub>, 1 mM CaCl<sub>2</sub> and digested with sequencing-grade modified trypsin (Promega) at 1:50 enzyme-to-substrate ratio. After 4 h of digestion at 37°C, samples were diluted 1:4 with the same buffers and another aliquot of the same amount of trypsin was added to the samples and further incubated at 25°C overnight (∼16 h). The digested samples were then acidified with 10% trifluoroacetic acid to ∼pH 2. Tryptic peptides were desalted on strong cation exchange SPE (Supelco) and reversed phase C18 SPE columns (Supelco) and dried using Speed-Vac.</div></section><section id="sec5.4.2" data-locator="sec5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT-10 Labeling of Peptides</span></h4><div class="section-paragraph">Desalted peptides from each sample were labeled with 10-plex Tandem Mass Tag (TMT) reagents according to the manufacturer’s instructions (ThermoScientific). Peptides (300 μg) from each of the samples were dissolved in 300 μL of 50 mM HEPES, pH 8.5, and mixed with 3 units of TMT reagent that was dissolved freshly in 123 μL of anhydrous acetonitrile. Channel 131 was used for labeling the internal reference sample (pooled from all tumor and normal samples with equal contribution) throughout the sample analysis. After 1 h incubation at RT, 24 μL of 5% hydroxylamine was added and incubated for 15 min at RT to quench the reaction. Peptides labeled by different TMT reagents were then mixed, dried using Speed-Vac, reconstituted with 3% acetonitrile, 0.1% formic acid, and desalted on C18 SPE columns.</div></section><section id="sec5.4.3" data-locator="sec5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide Fractionation by Basic Reversed-phase Liquid Chromatography</span></h4><div class="section-paragraph">Approximately 2.5 mg of 10-plex TMT-labeled sample was separated on a Waters reversed-phase XBridge C18 column (250 mm × 4.6 mm column containing 5-μm particles, and a 4.6 mm × 20 mm guard column) using an Agilent 1200 HPLC System. After sample loading, the C18 column was washed for 35 min with solvent A (5 mM ammonium formate, pH 10.0), before applying a 100-min LC gradient with solvent B (5 mM ammonium formate, pH 10, 90% acetonitrile). The LC gradient began with a linear increase of solvent A to 10% B in 2 min, then linearly increased to 45% B in 78 min, 10 min to 42.5% B, 10 min to 100% B. The gradient then resolved to 100% solvent A in ten minutes. The flow rate was 0.5 mL/min. A total of 96 fractions were collected into a 96-well plate throughout the LC gradient. These fractions were concatenated into 24 fractions by combining four fractions that are 24 fractions apart (i.e., combining fractions #1, #25, #49, and #73; #2, #26, #50, and #74; and so on). For proteome analysis, 5% of each concatenated fraction was dried and resuspended in 2% acetonitrile, 0.1% formic acid to a peptide concentration of 0.1 μg/μL for LC-MS/MS analysis. The remainder of the concatenated fractions (95%) were further concatenated into twelve fractions by combining two concatenated fractions (i.e., combining concatenated fractions #1 and #13; #3 and #15; and so on), dried, and subjected to immobilized metal affinity chromatography (IMAC) for phosphopeptide enrichment.</div></section><section id="sec5.4.4" data-locator="sec5.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide Enrichment Using IMAC</span></h4><div class="section-paragraph">Fe<sup>3+</sup>-NTA-agarose beads were freshly prepared using Ni-NTA Superflow agarose beads (QIAGEN) for phosphopeptide enrichment. For each of the twelve fractions, peptides were reconstituted in 400 μL IMAC binding/wash buffer (80% acetonitrile, 0.1% formic acid) and incubated with 20 μL of the 50% bead suspension for 30 min at RT. After incubation, the beads were washed two times each with 100 μL of wash buffer on the stage tip packed with two discs of Empore C18 material. Phosphopeptides were eluted from the beads on C18 using 70 μL of Elution Buffer (500 mM K<sub>2</sub>HPO<sub>4</sub>, pH 7.0). 50% acetonitrile, 0.1% formic acid was used to elute phosphopeptides from the C18 stage tips. Samples were dried using Speed-Vac and later reconstituted with 10 μL of 3% acetonitrile, 0.1% formic acid for LC-MS/MS analysis.</div></section><section id="sec5.4.5" data-locator="sec5.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for Global Proteome Analysis</span></h4><div class="section-paragraph">The global proteome fractions were separated using a nanoAquity UPLC system (Waters Corporation) by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase (Dr. Maisch GmbH) and slurry packed into a 30-cm length of 360 μm o.d. x 75 μm i.d. fused silica containing a 3-mm sol-gel frit. The trapping column was manufactured in-house using Jupiter 300 C18 5-um stationary phase (Phenomenex) and slurry packed into a 4-cm length of 360 μm o.d. x 150 μm i.d. fused silica with the final column being sol-gel fritted on both ends. The analytical column was heated to 50°C using an AgileSLEEVE column heater (Analytical Sales and Services, Inc.). The analytical column was equilibrated to 95% Mobile Phase A (MP A, 0.1% formic acid in water) and 5% Mobile Phase B (MP B, 0.1% formic acid in acetonitrile) and maintained at a constant column flow of 200 nL/min. The injected sample (5 μL) was trapped using 100% MP A for 10 min at flow rate of 3 μL/min before being placed in-line with the analytical column and subjected to the gradient profile (min:%MP B): 0:5, 1:8, 44:15, 85:30, 94:55, 102:70, 105:95, 108:95, 115:5, 150:5.</div><div class="section-paragraph">MS analysis was performed using a Q-Exactive Plus mass spectrometer (Thermo Scientific, San Jose, CA). Electrospray voltage (2.2 kV) was applied at a carbon composite union (Valco Instruments Co. Inc.) between the analytical column and electrospray emitter (chemically etched 360-um o.d. x 20-um i.d.). The ion transfer tube was set at 250°C. Following a 15-min delay from the end of sample trapping, Orbitrap precursor spectra (AGC 1x10<sup>6</sup>) were collected from 300-1800 m/z for 120 min at a resolution of 70K along with the top twelve data-dependent Orbitrap HCD MS/MS spectra at a resolution of 35K (AGC 1x10<sup>5</sup>) and max ion time of 100 msec. Masses selected for MS/MS were isolated at a width of 0.7 m/z and fragmented using a normalized collision energy of 32%. Peptide match was set to ‘”preferred,” exclude isotopes was set to ”on,” and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 20 s to discriminate against previously analyzed ions.</div></section><section id="sec5.4.6" data-locator="sec5.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for Phosphoproteome Analysis</span></h4><div class="section-paragraph">Phosphoproteome fractions were separated using a nanoAquity UPLC system by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase and slurry packed into a 35-cm length of 360 μm o.d. x 50 μm i.d. fused silica picofrit capillary tubing (New Objective, Inc.). The trapping column was manufactured in-house using Jupiter 300 C18 5-μm stationary phase (Phenomenex) and slurry packed into a 4-cm length of 360 μm o.d. x 150 μm i.d. fused silica, with the final column being sol-gel fritted on both ends. The analytical column was heated to 50°C using an AgileSLEEVE column heater. The analytical column was equilibrated to 98% MP A and 2% MP B and maintained at a constant column flow of 120 nL/min. The injected sample (5 μL) was trapped using 100% MP A for 5 min at flow rate of 3 μL/min before being placed in-line with the analytical column and subjected to the gradient profile (min:%MP B): 0:2, 8:4, 50:15, 85:35, 94:60, 95:95, 105:95, 115:2, 170:2.</div><div class="section-paragraph">MS analysis was performed using an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). Electrospray voltage (1.8 kV) was applied at a Valco carbon composite union coupling a 360 μm o.d. x 20 μm i.d. fused silica extension from the LC gradient pump to the analytical column and the ion transfer tube was set at 250°C. Following a 40-min delay from the end of sample trapping, Orbitrap precursor spectra (AGC 4x10<sup>5</sup>) were collected from 350-1800 m/z for 120 min at a resolution of 60K along with data-dependent Orbitrap HCD MS/MS spectra (centroided) at a resolution of 50K (AGC 1x10<sup>5</sup>) and max ion time of 105 msec for a total duty cycle of 2 s. Masses selected for MS/MS were isolated (quadrupole) at a width of 0.7 m/z and fragmented using a collision energy of 30%. Peptide mode was selected for monoisotopic precursor scan and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 45 s to discriminate against previously analyzed ions between ± 10 ppm.</div></section><section id="sec5.4.7" data-locator="sec5.4.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Construction and Utilization of the Comparative Reference Samples</span></h4><div class="section-paragraph">As a quality control measure, two “Comparative Reference” (“CompRef”) samples were generated as previously described<div class="dropBlock reference-citations"><sup><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d15348236e1667" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">65</a></sup><div data-db-target-of="bib65-d15348236e1667" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tabb D.L. </li><li class="loa__item">Wang X. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Holman J.D. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Zimmerman L.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducibility of Differential Proteomic Technologies in CPTAC Fractionated Xenografts.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">691-706</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Facs.jproteome.5b00859&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26653538&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84960343160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=691-706&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Wang+X.&author=Carr+S.A.&author=Clauser+K.R.&author=Mertins+P.&author=Chambers+M.C.&author=Holman+J.D.&author=Wang+J.&author=Zhang+B.&author=Zimmerman+L.J.&title=Reproducibility+of+Differential+Proteomic+Technologies+in+CPTAC+Fractionated+Xenografts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d15348236e1672" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">66</a></sup><div data-db-target-of="bib66-d15348236e1672" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li S. </li><li class="loa__item">Shen D. </li><li class="loa__item">Shao J. </li><li class="loa__item">Crowder R. </li><li class="loa__item">Liu W. </li><li class="loa__item">Prat A. </li><li class="loa__item">He X. </li><li class="loa__item">Liu S. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Lu C. </li><li>et al.</li></ul><div class="reference-citations__detials">Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1116-1130</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24055055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84884559238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (498)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1116-1130&journal=Cell+Rep.&author=Li+S.&author=Shen+D.&author=Shao+J.&author=Crowder+R.&author=Liu+W.&author=Prat+A.&author=He+X.&author=Liu+S.&author=Hoog+J.&author=Lu+C.&title=Endocrine-therapy-resistant+ESR1+variants+revealed+by+genomic+characterization+of+breast-cancer-derived+xenografts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and used to monitor the longitudinal performance of the proteomics workflow throughout the course of this study. Briefly, patient-derived xenograft (PDX) tumors from established basal and luminal breast cancer intrinsic subtypes were raised subcutaneously in 8-week old NOD.Cg-Prkdc<sup>scid</sup> Il2rg<sup>tm1Wjl</sup>/SzJ mice (Jackson Laboratories, Bar Harbor, ME) using procedures reviewed and approved by the Institutional Animal Care and Use Committee at Washington University in St. Louis. Xenografts were grown in multiple mice, pooled, and cryopulverized to provide a sufficient amount of uniform material for the duration of the study. Full proteome and phosphoproteome process replicates of each of the two CompRef samples were prepared and analyzed as standalone 10-plex TMT experiments alongside every four TMT-10 experiments of the study samples, using the same analysis protocol as the patient samples. These interstitially analyzed CompRef samples were evaluated for depth of proteome and phosphoproteome coverage and for consistency in quantitative comparison between the basal and luminal models (i.e., basal/luminal ratios across different replicates and batches; <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>C).</div></section><section id="sec5.4.8" data-locator="sec5.4.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Global Proteome Analysis of Fallopian Tube Epithelial Cells</span></h4><div class="section-paragraph">Ten Fallopian tube epithelial (FTE) cell samples were freshly collected using cytobrush techniques at University of Connecticut Health Center (Dr. Molly Brewer) from patients undergoing surgery for benign gynecological conditions. The cell pellets were lysed in 50 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8 with intermittent sonication on ice. Trifluoroethanol (TFE) was then added to the samples at 50% (v/v) concentration and incubated at 60°C for 2 h. After being reduced with 2 mM dithiothreitol for 1 h at 37°C, the samples were diluted 1:5 with 50 mM NH<sub>4</sub>HCO<sub>3</sub> (i.e., 10% TFE) and digested with sequencing grade modified trypsin (Promega) at 1:50 enzyme-to-substrate ratio for 3 h at 37°C. The resulting tryptic peptides were concentrated using Speed-Vac, and the peptide concentration was measured using the BCA assay. Two pooled samples, each with 100 μg of total peptides, were created by equally mixing five samples and subjected to fractionation via bRPLC as described above, generating 24 fractions for each pooled sample. Each fraction was analyzed by LC-MS/MS using conditions similar to those in the ovarian cancer tissue global proteome analysis (described above), except for a resolution of 35,000 at the MS level and 17,500 at the MS/MS level and an isolation window of 2.0 m/z that were used in the label-free analysis of the FTE cell samples. The resulting data were searched against the same protein sequence database and filtered in the same fashion for assembling the final list of protein identifications.</div></section><section id="sec5.4.9" data-locator="sec5.4.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genotyping Array Analysis</span></h4><div class="section-paragraph">Genomic DNA samples were prepared according to the Illumina Infinium LCG Quad Assay manual protocol. Processed samples were loaded on HumanOmni5-Quad BeadChips and run on the HiScan platform. SNP and SCNA genotyping were performed with the Genome Studio Genotyping Module. Although these data were not used in the final analysis due to limited dynamic range, the data are publicly accessible through the Data Coordinating Center.</div></section><section id="sec5.4.10" data-locator="sec5.4.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole Exome Sequencing</span></h4><div class="section-paragraph">Genomic DNA samples were used to prepare indexed libraries using the Nextera Rapid Capture Exome kit from Illumina. Library preparation was performed using a semi-automated 96-well plate method, with washing and clean-up/concentration steps performed on the Beckman Coulter Biomek NXP platform and with ZR-96 DNA Clean &amp; Concentrator-5 plates, respectively. Libraries were quantified using the Agilent 2100 Bioanalyzer. Pooled libraries were run on HiSeq4000 (2x150 paired end runs) to achieve a minimum of 150x on target coverage per each sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files, and adaptor and low-quality sequences were trimmed. Whole exome sequencing (WXS) data were used for somatic mutation detection, germline variant detection, pathogenicity annotation of germline variants, and somatic copy number alteration (SCNA) analysis as described below.</div></section><section id="sec5.4.11" data-locator="sec5.4.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic Mutation Detection</span></h4><div class="section-paragraph">We used SomaticWrapper (available on GitHub at <a class="inter-ref-link" href="https://github.com/ding-lab/somaticwrapper" target="_blank">https://github.com/ding-lab/somaticwrapper</a>) to perform somatic mutation calling. In SomaticWrapper, we incorporated four mutation callers, Mutect1 (3.1-0-g72492bb), Strelka1 (1.0.14), VarScan2 (2.3.8) and Pindel (0.2.5b9-20160729). The raw calls were subjected to individual somatic filtering steps. We removed germline variants by filtering variants that exist in dbSNP but not in the COSMIC database. We used GATK (3.7-0-gcfedb67) to combine variants from different callers. For single nucleotide variants (SNVs), only the calls called by more than two out of three callers (Strelka1, VarScan2, and Pindel) were retained. For insertions and deletions (INDELs), only those called by more than two out of three callers (Strelka1, VarScan2, and Pindel) were retained. We later applied tumor variant allele frequency (VAF) filtering (&gt; = 5%) and normal VAF filtering (≤2%) on the merged mutation calls. To obtain high-quality somatic indel calls, we restricted the indel size to less than 100 nts. Finally, the genomic coordinates of the mutation calls were lifted over from hg19 to hg38 by CrossMap v0.3.3 using UCSC chain file hg19ToHg38.over.chain.gz.</div></section><section id="sec5.4.12" data-locator="sec5.4.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline Variant Detection</span></h4><div class="section-paragraph">We used the GermlineWrapper pipeline (available on GitHub at <a class="inter-ref-link" href="https://github.com/ding-lab/germlinewrapper" target="_blank">https://github.com/ding-lab/germlinewrapper</a>) to call germline variants. In GermlineWrapper, we used GATK (3.7-0-gcfedb67), VarScan2 (2.3.8), and Pindel (0.2.5b9-20160729) to call germline variants from normal samples. For single nucleotide polymorphism (SNPs), we took the union of calls from both GATK and VarScan2. Finally, we retained on those indels called by both GATK and VarScan or by Pindel. Cutoffs of a minimum of 10X coverage and 20% VAF were used in the final step to report the high-quality germline variants.</div></section><section id="sec5.4.13" data-locator="sec5.4.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Pathogenicity Annotation of Germline Variants</span></h4><div class="section-paragraph">Germline variants were annotated using the Ensembl Variant Effect Predictor (VEP)<div class="dropBlock reference-citations"><sup><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d15348236e1734" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">55</a></sup><div data-db-target-of="bib55-d15348236e1734" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McLaren W. </li><li class="loa__item">Gil L. </li><li class="loa__item">Hunt S.E. </li><li class="loa__item">Riat H.S. </li><li class="loa__item">Ritchie G.R. </li><li class="loa__item">Thormann A. </li><li class="loa__item">Flicek P. </li><li class="loa__item">Cunningham F. </li></ul><div class="reference-citations__detials">The Ensembl Variant Effect Predictor.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3975)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (version 85 with default parameters, except where–everything). Pathogenicity of germline variants was determined with our automatic pipeline CharGer<div class="dropBlock reference-citations"><sup><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d15348236e1738" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">56</a></sup><div data-db-target-of="bib56-d15348236e1738" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Scott A.D. </li><li class="loa__item">Huang K.L. </li><li class="loa__item">Weerasinghe A. </li><li class="loa__item">Mashl R.J. </li><li class="loa__item">Gao Q. </li><li class="loa__item">Martins Rodrigues F. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Ding L. </li></ul><div class="reference-citations__detials">CharGer: Clinical Characterization of Germline Variants.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">865-867</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbty649&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30102335&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85062278783&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=865-867&journal=Bioinformatics&author=Scott+A.D.&author=Huang+K.L.&author=Weerasinghe+A.&author=Mashl+R.J.&author=Gao+Q.&author=Martins+Rodrigues+F.&author=Wyczalkowski+M.A.&author=Ding+L.&title=CharGer%3A+Clinical+Characterization+of+Germline+Variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (<a class="inter-ref-link" href="https://github.com/ding-lab/CharGer" target="_blank">https://github.com/ding-lab/CharGer</a>), which annotates and prioritizes variants based on AMP-ACMG guidelines.<div class="dropBlock reference-citations"><sup><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d15348236e1745" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">67</a></sup><div data-db-target-of="bib67-d15348236e1745" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Richards S. </li><li class="loa__item">Aziz N. </li><li class="loa__item">Bale S. </li><li class="loa__item">Bick D. </li><li class="loa__item">Das S. </li><li class="loa__item">Gastier-Foster J. </li><li class="loa__item">Grody W.W. </li><li class="loa__item">Hegde M. </li><li class="loa__item">Lyon E. </li><li class="loa__item">Spector E. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>ACMG Laboratory Quality Assurance Committee</span><br />Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology.<div class="popup-ref"><span class="ref__series"><i>Genet. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">405-424</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=abstract&amp;site=gim-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=fulltext&amp;site=gim-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=pdf&amp;site=gim-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25741868&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84928209346&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19710)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=405-424&journal=Genet.+Med.&author=ACMG+Laboratory+Quality+Assurance+Committee&title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> This pipeline implements twelve pathogenic and four benign evidence levels using several datasets, including ExAC and ClinVar (parsed through MacArthur lab ClinVar; <a class="inter-ref-link" href="https://github.com/macarthur-lab/clinvar" target="_blank">https://github.com/macarthur-lab/clinvar</a>), and computational tools including SIFT<div class="dropBlock reference-citations"><sup><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d15348236e1753" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">59</a></sup><div data-db-target-of="bib59-d15348236e1753" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kumar P. </li><li class="loa__item">Henikoff S. </li><li class="loa__item">Ng P.C. </li></ul><div class="reference-citations__detials">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1073-1081</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnprot.2009.86&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=19561590&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-68149165614&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5290)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1073-1081&journal=Nat.+Protoc.&author=Kumar+P.&author=Henikoff+S.&author=Ng+P.C.&title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and PolyPhen.<div class="dropBlock reference-citations"><sup><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d15348236e1757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">60</a></sup><div data-db-target-of="bib60-d15348236e1757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Adzhubei I. </li><li class="loa__item">Jordan D.M. </li><li class="loa__item">Sunyaev S.R. </li></ul><div class="reference-citations__detials">Predicting functional effect of human missense mutations using PolyPhen-2.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Hum. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">Chapter 7</span></span>: <span class="ref__seriesPages">Unit 7.20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=Chapter+7&publication_year=2013&pages=Unit+7.20&journal=Curr.+Protoc.+Hum.+Genet.&author=Adzhubei+I.&author=Jordan+D.M.&author=Sunyaev+S.R.&title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> The detailed implementation and score of each evidence level are as previously described.<div class="dropBlock reference-citations"><sup><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d15348236e1761" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">68</a></sup><div data-db-target-of="bib68-d15348236e1761" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang K.L. </li><li class="loa__item">Mashl R.J. </li><li class="loa__item">Wu Y. </li><li class="loa__item">Ritter D.I. </li><li class="loa__item">Wang J. </li><li class="loa__item">Oh C. </li><li class="loa__item">Paczkowska M. </li><li class="loa__item">Reynolds S. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Oak N. </li><li>et al.</li></ul><div class="reference-citations__detials">Pathogenic Germline Variants in 10,389 Adult Cancers.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">355-370.e1–e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29625052&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044578706&amp;cf=" onclick="newWindow(this.href);return false">Scopus (517)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=355-370.e1%E2%80%93e6&journal=Cell&author=Huang+K.L.&author=Mashl+R.J.&author=Wu+Y.&author=Ritter+D.I.&author=Wang+J.&author=Oh+C.&author=Paczkowska+M.&author=Reynolds+S.&author=Wyczalkowski+M.A.&author=Oak+N.&title=Pathogenic+Germline+Variants+in+10%2C389+Adult+Cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div><div class="section-paragraph">Variants prioritized by CharGer were further filtered for rare variants with ≤ 0.05% allele frequency in ExAC (release r0.3.1) and gnomAD (release 2.1). We then selected for cancer-relevant pathogenic and likely pathogenic variants, based on whether they were found in the curated cancer variant database or in the curated list of 152 cancer predisposition genes compiled as previously described.<div class="dropBlock reference-citations"><sup><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d15348236e1766" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">68</a></sup><div data-db-target-of="bib68-d15348236e1766" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang K.L. </li><li class="loa__item">Mashl R.J. </li><li class="loa__item">Wu Y. </li><li class="loa__item">Ritter D.I. </li><li class="loa__item">Wang J. </li><li class="loa__item">Oh C. </li><li class="loa__item">Paczkowska M. </li><li class="loa__item">Reynolds S. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Oak N. </li><li>et al.</li></ul><div class="reference-citations__detials">Pathogenic Germline Variants in 10,389 Adult Cancers.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">355-370.e1–e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29625052&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044578706&amp;cf=" onclick="newWindow(this.href);return false">Scopus (517)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=355-370.e1%E2%80%93e6&journal=Cell&author=Huang+K.L.&author=Mashl+R.J.&author=Wu+Y.&author=Ritter+D.I.&author=Wang+J.&author=Oh+C.&author=Paczkowska+M.&author=Reynolds+S.&author=Wyczalkowski+M.A.&author=Oak+N.&title=Pathogenic+Germline+Variants+in+10%2C389+Adult+Cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Special focus was given to variants in genes of interest to ovarian cancer, such as <em>BRCA1</em> and <em>BRCA2</em>, as well as Fanconi Anemia-related genes. Resulting variants were manually reviewed using Integrative Genomics Viewer (IGV) software<div class="dropBlock reference-citations"><sup><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d15348236e1774" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">61</a></sup><div data-db-target-of="bib61-d15348236e1774" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson J.T. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Winckler W. </li><li class="loa__item">Guttman M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Integrative genomics viewer.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">24-26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt.1754&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21221095&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-78651271733&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9132)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=24-26&journal=Nat.+Biotechnol.&author=Robinson+J.T.&author=Thorvaldsd%C3%B3ttir+H.&author=Winckler+W.&author=Guttman+M.&author=Lander+E.S.&author=Getz+G.&author=Mesirov+J.P.&title=Integrative+genomics+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (version 2.4.18) using both normal and tumor data. Variants that passed manual review with low allele frequencies (MAF &lt; 0.05%) were further evaluated for loss-of-heterozygosity (LOH) by comparing VAF in tumor and normal samples using a Fisher’s exact test. LOH was considered significant at p value ≤ 0.05.</div></section><section id="sec5.4.14" data-locator="sec5.4.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic Copy Number Alteration Analysis</span></h4><div class="section-paragraph">We used the GATK (version 4. beta.5) CNV workflow to detect somatic copy number variations in WXS data. This platform initially collects proportional coverage using target intervals and WXS BAM files. GC-correction was applied to the derived coverage using the CorrectGCBias module, and a CNV panel of normals was created to remove the batch effect from the tumor samples. The resulting coverage data were then normalized and segmented. Finally, our workflow called copy number variants after filtering out copy number neutral regions and outlier coverage regions (using default parameters).</div><div class="section-paragraph">For quality control, we compared germline and somatic copy number variation. We found one group of cases with evidence of switched tumor and normal samples, shown by the germline CNV profiles resembling somatic CNV profiles, and vice-versa. This switch was further indicated by those cases having significantly fewer somatic mutations than the rest of the cohort. Swapping the tumor and normal sequencing files (BAMs) before re-processing CNV and somatic mutations eliminated the issue and rescued many important somatic mutations, including <em>TP53</em>. The sample swap issue was limited to exome sequencing data.</div><div class="section-paragraph">Significantly altered regions of amplification and deletion across the tumor population were identified using the Genomic Identification of Significant Targets in Cancer (GISTIC) 2.0 algorithm<div class="dropBlock reference-citations"><sup><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d15348236e1789" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">9</a></sup><div data-db-target-of="bib9-d15348236e1789" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> on the segmented copy number profiles of the tumor samples. The GISTIC algorithm uses relative copy number data to determine significantly aberrant regions of somatic copy number alteration across a set of tumors. The algorithm calculates a G-score for each locus that accounts for both the frequency and the amplitude of observed alterations across the set of samples. Separate scores are computed for gains and losses. Only SCNAs that met relatively high thresholds of copy number changes (in log2-space) &gt; 0.3 (amplification) or &lt; −0.3 (deletion) were included in the computations. Statistical significance of the aberrations is determined by comparing the observed statistics against a random distribution obtained by a permutation of the data in each dataset. The p values are corrected for multiple hypotheses testing. A q-value cutoff of 0.05 was used to identify significant alterations. SCNA profiles are divided into arm-level and focal alterations. Significant aberrant regions that span more than half of the chromosome arm are characterized as arm-level events whereas regions smaller than this length are labeled focal events.</div></section><section id="sec5.4.15" data-locator="sec5.4.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA Sequencing</span></h4><div class="section-paragraph">Indexed cDNA sequencing libraries were prepared from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags. Library preparation was performed similarly to the method used for WXS. Quality control was performed at every step, and the libraries were quantified using the Agilent 2100 Bioanalyzer. Indexed libraries were prepared as equimolar pools and run on the HiSeq4000 platform (2x150 paired end runs) to generate a minimum of 30 million paired-end reads per sample library. The raw Illumina sequence data were demultiplexed and converted to. fastq files, and adaptor and low-quality sequences were trimmed.</div></section><section id="sec5.4.16" data-locator="sec5.4.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA-Seq Data Analysis</span></h4><div class="section-paragraph">mRNA sequencing reads were mapped to human genome hg19 by STAR (version 2.5.3a) using the one-pass model. Hg19 sequence and genome annotation were downloaded from NCBI RefSeq (03/29/2017). RSEM (version 1.2.31) was used to quantify gene and transcript expression levels. Gene read counts were calculated by HTseq (version 0.7.2) based on the above annotation. The RSEM output, including the mRNA RSEM and FPKM (Fragments Per Kilobase of transcript per Million mapped reads) results, were reported in table format (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/a962c084-c058-4152-89e4-bb7599a4ae40/mmc4" class="scroll-into-link">Table S3</a>).</div></section></section><section id="sec5.5" data-locator="sec5.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and Statistical Analysis</span></h3><section id="sec5.5.1" data-locator="sec5.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of Global Proteomics Data</span></h4><div class="section-paragraph">LC-MS/MS analysis of the TMT10-labeled, bRPLC fractionated samples generated a total of 288 global proteomics data files. The Thermo RAW files were processed with mzRefinery<div class="dropBlock reference-citations"><sup><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d15348236e1814" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">43</a></sup><div data-db-target-of="bib43-d15348236e1814" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbons B.C. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Payne S.H. </li></ul><div class="reference-citations__detials">Correcting systematic bias and instrument measurement drift with mzRefinery.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtv437&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84950239084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinformatics&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> to characterize and correct for any instrument calibration errors and then with MS-GF+<div class="dropBlock reference-citations"><sup><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d15348236e1818" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">45</a></sup><div data-db-target-of="bib45-d15348236e1818" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d15348236e1823" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">69</a></sup><div data-db-target-of="bib69-d15348236e1823" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Gupta N. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (v9881) to match against the RefSeq human protein sequence database released on September 14, 2016 (37,593 proteins) combined with 149 contaminants (e.g., trypsin, keratin). The partially tryptic search used a ± 10 ppm parent ion tolerance, allowed for isotopic error in precursor ion selection, and searched a decoy database composed of the forward and reversed protein sequences. MS-GF+ settings included static carbamidomethylation (+57.0215 Da) on Cys residues, TMT modification (+229.1629 Da) on the peptide N terminus and Lys residues, and dynamic oxidation (+15.9949 Da) on Met residues for searching the global proteome data.</div><div class="section-paragraph">Peptide identification stringency was tuned to a PepQValue &lt; 0.005 and a parent ion mass deviation &lt; 7 ppm criteria such that it did not exceed false discovery rate (FDR) of 1% at the unique peptide sequence level. The minimum number of unique peptides per 1,000 amino acids of protein length was also explicitly evaluated, and a minimum of six peptides per 1000 AA was finally applied to achieve 1% FDR at the protein level within the full dataset. Inference of the parsimonious protein set resulted in 10,706 protein identifications (an average of 9,769 proteins per sample), with 8,184 proteins common among all 103 samples (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>).</div><div class="section-paragraph">The intensities of all ten TMT reporter ions were extracted using MASIC software.<div class="dropBlock reference-citations"><sup><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d15348236e1834" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">44</a></sup><div data-db-target-of="bib44-d15348236e1834" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> Next, PSMs were linked to the extracted reporter ion intensities by scan number. The reporter ion intensities from different scans and different bRPLC fractions corresponding to the same protein were summed. Relative protein abundance was calculated as the ratio of abundance in a given sample to the reference abundance. The pooled reference sample was labeled with TMT 131 reagent, allowing comparison of relative protein abundances across different TMT-10 plexes. The relative abundances were log2 transformed and zero-centered for each protein to obtain final, relative abundance values.</div><div class="section-paragraph">Small differences in sample handling and BCA assay measurement errors can result in detectable systematic, sample-specific bias in the quantification of protein levels. To mitigate these effects, we computed the median, log2 relative protein abundance for each sample and re-centered to achieve a common median of 0. The final processed global proteome data table is available in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>; raw proteomic data is available via the CPTAC Data Portal (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/study/disclaimer?accNum=S038</a>).</div></section><section id="sec5.5.2" data-locator="sec5.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of Phosphopeptides</span></h4><div class="section-paragraph">Phosphopeptides were identified from the 144 phosphoproteomics data files as described above (e.g., peptide level FDR &lt; 1%), with an additional dynamic phosphorylation (+79.9663 Da) on Ser, Thr, or Tyr residues. A total of 49,616 phosphopeptides were identified. The phosphoproteome data were further processed by the Ascore algorithm<div class="dropBlock reference-citations"><sup><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d15348236e1851" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">70</a></sup><div data-db-target-of="bib70-d15348236e1851" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beausoleil S.A. </li><li class="loa__item">Villén J. </li><li class="loa__item">Gerber S.A. </li><li class="loa__item">Rush J. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1297)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> for phosphorylation site localization, and the top-scoring sequences were reported. For phosphoproteomic datasets, the TMT-10 quantitative data were not summarized by protein but remained at the phosphopeptide level (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>).</div><div class="section-paragraph">All the peptides (phosphopeptides and global peptides) were labeled with TMT-10 reagent simultaneously, and the enrichment of phosphopeptides using IMAC was performed after the labeling. Thus, all the biases upstream of labeling are assumed to be identical between global and phosphoproteomics datasets. Therefore, to account for sample-specific biases in the phosphoproteome analysis we applied the correction factors derived from mean-centering the global proteomic dataset.</div><div class="section-paragraph">In addition to the original phosphorylation abundances, phosphorylation site occupancy ratios (i.e., the relative extent of phosphorylation) were calculated by subtracting the log2 relative abundance of the parent protein (reflecting the overall abundance) from the phosphopeptide (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>). The resulting phosphopeptide values reflect the changes in relative phosphorylation rates of the protein and are not confounded by changes in expression of the protein itself.</div><div class="section-paragraph">The proteomics data quality has been carefully evaluated in terms of both proteome coverage and quantitation. The relative abundances of the proteins and phosphopeptides across the tumor and normal tissue samples were quantified using a universal reference strategy as reported recently,<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e1867" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e1867" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d15348236e1872" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">7</a></sup><div data-db-target-of="bib7-d15348236e1872" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> applying a stringent 1% FDR cutoff at the protein level. The consistency of the proteomics pipeline was demonstrated in both the protein sequence coverage in the proteome data (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>B) and the quantitation reproducibility in the external quality controls comparing two breast cancer patient-derived xenograft samples (i.e., CompRef samples, see “<a href="#sec5.4.7" class="scroll-into-link">Construction and Utilization of the Comparative Reference Samples</a>” above) that were analyzed alongside the ovarian tumors and normal tissue samples using the same proteomics pipeline (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>C). Note that it was the ratio between two different xenograft samples, not the quantitation values of just one sample, that was compared in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>C for correlation. We have successfully used such external QCs to effectively monitor the performance of the proteomics pipeline over time in previous analyses.<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e1889" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e1889" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d15348236e1894" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">7</a></sup><div data-db-target-of="bib7-d15348236e1894" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>,</sup><div class="dropBlock reference-citations"><sup><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d15348236e1899" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">71</a></sup><div data-db-target-of="bib71-d15348236e1899" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (425)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div><div class="section-paragraph">Isobaric labeling based quantitation could have interference-based compression compared to the label-free methods; however, we have performed extensive fractionation to significantly reduce the sample complexity in each individually analyzed fraction, thus effectively alleviating such compression issues. The consistency and robustness of the quantitation results were also demonstrated in the PCA analysis of the entire global proteome and phosphoproteome datasets: the tumors were readily separated from the normal FT tissues (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S1</a>D).</div></section><section id="sec5.5.3" data-locator="sec5.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Identification and Quantification of Acetylated Histone H4</span></h4><div class="section-paragraph">In order to identify specific sites of acetylation using TMT global proteomic data, we performed the search similarly yet with the addition of a dynamic acetylation to lysine (+42 Da). The PSMs were filtered by (1) FDR &lt; 1%, (2) number of missed cleavage sites no greater than 2, (3) at least 3 PSMs for an identified peptide, and (4) at least 3 peptides for an identified protein. Each identified PSM was quantified using TMT reporter ion intensities.</div></section><section id="sec5.5.4" data-locator="sec5.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Gene-wise Correlation between Different Platforms</span></h4><div class="section-paragraph">We calculated gene-wise correlations between RNA-seq and TMT proteomics that were quantifiable in both platforms. The Spearman correlation between the two platforms was calculated for each gene across all the samples.</div></section><section id="sec5.5.5" data-locator="sec5.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Correlation of Somatic Copy Number Alterations with mRNA/protein Abundance</span></h4><div class="section-paragraph">To determine the consequences of copy number variation on mRNA and protein expression, we carried out a correlation analysis as previously described<div class="dropBlock reference-citations"><sup><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d15348236e1922" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">14</a></sup><div data-db-target-of="bib14-d15348236e1922" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> and implemented in the R package MultiomicsViz.<div class="dropBlock reference-citations"><sup><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d15348236e1926" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">62</a></sup><div data-db-target-of="bib62-d15348236e1926" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li></ul><div class="reference-citations__detials">multiOmicsViz: plot the effect of one omics data on other omics data along the chromosome.<div class="popup-ref"><div class="citation-host"><a class="inter-ref-link" href="https://rdrr.io/bioc/multiOmicsViz/" target="_blank">https://rdrr.io/bioc/multiOmicsViz/</a></div></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Wang+J.+multiOmicsViz%3A+plot+the+effect+of+one+omics+data+on+other+omics+data+along+the+chromosome.https%3A%2F%2Frdrr.io%2Fbioc%2FmultiOmicsViz%2F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> The datasets were initially ordered in terms of their chromosomal location. mRNA expression was transformed to log2 fold change versus mean expression of each gene. The Spearman rank correlation was then calculated for the genes quantified in all samples in the SCNA dataset for each gene with complete data in both the mRNA and proteomic datasets using the corresponding measurements for the 81 overlapping samples between the SCNA, mRNA, and proteomic matrices. The p values for the correlation were also determined to assess statistical significance. The p values were adjusted for multiple-testing correction by applying the Benjamini-Hochberg procedure across each set of p values of the correlations of each SCNA with the proteins. SCNA-protein correlations (positive or negative) with an adjusted p value below 0.05 were deemed to be significant. Functional enrichment analysis for gene sets represented in the group of proteins in the SCNA loci with the strongest trans effects was carried out using clusterProfiler<div class="dropBlock reference-citations"><sup><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d15348236e1930" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">63</a></sup><div data-db-target-of="bib63-d15348236e1930" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yu G. </li><li class="loa__item">Wang L.G. </li><li class="loa__item">Han Y. </li><li class="loa__item">He Q.Y. </li></ul><div class="reference-citations__detials">clusterProfiler: an R package for comparing biological themes among gene clusters.<div class="popup-ref"><span class="ref__series"><i>OMICS.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">284-287</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1089%2Fomi.2011.0118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22455463&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84860718683&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17419)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=284-287&journal=OMICS&author=Yu+G.&author=Wang+L.G.&author=Han+Y.&author=He+Q.Y.&title=clusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clusters">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> at an FDR of 0.05 to determine enriched gene ontology biological processes.</div></section><section id="sec5.5.6" data-locator="sec5.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor versus Normal Differential Proteomic Analysis</span></h4><div class="section-paragraph">For the proteomic dataset, peptides were aggregated to the gene level by computing their mean abundance. Two normal samples, N02OV001 and N17OV001, were removed from all tumor-normal comparisons based on quality control analysis. Proteins with missing values in any of the samples were dropped, yielding 7,599 complete proteins that were tested for statistically significant differential abundance between tumor and normal samples. Differential abundance analyses were conducted using empirical Bayes moderated t tests via the Bioconductor package limma.<div class="dropBlock reference-citations"><sup><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d15348236e1939" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">64</a></sup><div data-db-target-of="bib64-d15348236e1939" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20650)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> To correct for multiple testing, the Benjamini-Hochberg procedure was used to control the FDR. Gene-set enrichment analysis was conducted using the R package clusterProfiler<div class="dropBlock reference-citations"><sup><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d15348236e1943" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">63</a></sup><div data-db-target-of="bib63-d15348236e1943" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yu G. </li><li class="loa__item">Wang L.G. </li><li class="loa__item">Han Y. </li><li class="loa__item">He Q.Y. </li></ul><div class="reference-citations__detials">clusterProfiler: an R package for comparing biological themes among gene clusters.<div class="popup-ref"><span class="ref__series"><i>OMICS.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">284-287</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1089%2Fomi.2011.0118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22455463&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84860718683&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17419)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=284-287&journal=OMICS&author=Yu+G.&author=Wang+L.G.&author=Han+Y.&author=He+Q.Y.&title=clusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clusters">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> with the list of proteins tested for differential expression and their corresponding log fold-changes as determined by limma as input. Gene symbols were mapped to their Entrez ids and ranked according to their log fold-change values. ClusterProfiler calculated normalized enrichment scores (NES) and FDR values for gene sets in GO BP. An FDR cutoff of 0.05 was used to designate significant enrichment of pathways. This differential expression analysis was repeated and restricted to the set of proteins that overlapped with cytobrush FTE samples (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>D).</div></section><section id="sec5.5.7" data-locator="sec5.5.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor versus Normal Differential Phosphoproteomic Analysis</span></h4><div class="section-paragraph">Non-normalized phosphoproteomics data were reduced to the phosphosite level by averaging phosphopeptides that mapped to the same site. As with the corresponding proteomic analysis, samples N02OV001 and N17OV001 were discarded. Phosphosites with missing values in at least half of either the tumor or the normal samples were filtered out, resulting in 7,215 phosphosites for further downstream analysis. Differential expression analysis to characterize changes in phosphoproteomic levels between tumors and the FT samples were conducted as described in the preceding subsection using limma to calculate the fold changes and associated adjusted p values for each phosphosite. The differential expression results were then used to determine kinase activity using KSEA<div class="dropBlock reference-citations"><sup><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d15348236e1955" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">18</a></sup><div data-db-target-of="bib18-d15348236e1955" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Casado P. </li><li class="loa__item">Rodriguez-Prados J.C. </li><li class="loa__item">Cosulich S.C. </li><li class="loa__item">Guichard S. </li><li class="loa__item">Vanhaesebroeck B. </li><li class="loa__item">Joel S. </li><li class="loa__item">Cutillas P.R. </li></ul><div class="reference-citations__detials">Kinase-substrate enrichment analysis provides insights into the heterogeneity of signaling pathway activation in leukemia cells.<div class="popup-ref"><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">rs6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1126%2Fscisignal.2003573&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23532336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84875699641&amp;cf=" onclick="newWindow(this.href);return false">Scopus (242)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=rs6&journal=Sci.+Signal.&author=Casado+P.&author=Rodriguez-Prados+J.C.&author=Cosulich+S.C.&author=Guichard+S.&author=Vanhaesebroeck+B.&author=Joel+S.&author=Cutillas+P.R.&title=Kinase-substrate+enrichment+analysis+provides+insights+into+the+heterogeneity+of+signaling+pathway+activation+in+leukemia+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> implemented in the R package KSEAapp.<div class="dropBlock reference-citations"><sup><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d15348236e1959" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">19</a></sup><div data-db-target-of="bib19-d15348236e1959" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wiredja D.D. </li><li class="loa__item">Koyutürk M. </li><li class="loa__item">Chance M.R. </li></ul><div class="reference-citations__detials">The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3489-3491</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtx415&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28655153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044408024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (135)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3489-3491&journal=Bioinformatics&author=Wiredja+D.D.&author=Koyut%C3%BCrk+M.&author=Chance+M.R.&title=The+KSEA+App%3A+a+web-based+tool+for+kinase+activity+inference+from+quantitative+phosphoproteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> KSEA computes a normalized score to measure relative kinase activity in test versus control using the difference in the mean log fold-changes of predefined substrate groups of the kinase, as referenced from the PhosphoSitePlus<div class="dropBlock reference-citations"><sup><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d15348236e1963" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">20</a></sup><div data-db-target-of="bib20-d15348236e1963" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Tkachev S. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Skrzypek E. </li><li class="loa__item">Murray B. </li><li class="loa__item">Latham V. </li><li class="loa__item">Sullivan M. </li></ul><div class="reference-citations__detials">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">40</span></span>: <span class="ref__seriesPages">D261-D270</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkr1122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22135298&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84857047339&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1237)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D261-D270&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Kornhauser+J.M.&author=Tkachev+S.&author=Zhang+B.&author=Skrzypek+E.&author=Murray+B.&author=Latham+V.&author=Sullivan+M.&title=PhosphoSitePlus%3A+a+comprehensive+resource+for+investigating+the+structure+and+function+of+experimentally+determined+post-translational+modifications+in+man+and+mouse">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (July 2016) and NetworKIN<div class="dropBlock reference-citations"><sup><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d15348236e1967" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">72</a></sup><div data-db-target-of="bib72-d15348236e1967" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Linding R. </li><li class="loa__item">Jensen L.J. </li><li class="loa__item">Pasculescu A. </li><li class="loa__item">Olhovsky M. </li><li class="loa__item">Colwill K. </li><li class="loa__item">Bork P. </li><li class="loa__item">Yaffe M.B. </li><li class="loa__item">Pawson T. </li></ul><div class="reference-citations__detials">NetworKIN: a resource for exploring cellular phosphorylation networks.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">D695-D699</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkm902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=17981841&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-38549109487&amp;cf=" onclick="newWindow(this.href);return false">Scopus (272)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D695-D699&journal=Nucleic+Acids+Res.&author=Linding+R.&author=Jensen+L.J.&author=Pasculescu+A.&author=Olhovsky+M.&author=Colwill+K.&author=Bork+P.&author=Yaffe+M.B.&author=Pawson+T.&title=NetworKIN%3A+a+resource+for+exploring+cellular+phosphorylation+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> databases. In the latter database, kinase-substrate annotations with a NetworKIN score below five are discarded. The mean log fold-changes of all the substrates are divided by the standard deviation of log fold-changes across all phosphosites, followed by an adjustment to the score for the number of substrates in the data that annotate to the kinase. The statistical significance of the scores is calculated using a one-tailed z-test followed by BH multiple testing correction. Kinases with z-scores with corresponding p values below 0.05 were considered to be significant. The KSEA analysis was repeated after filtering out phosphorylation sites known to be altered by ischemia<div class="dropBlock reference-citations"><sup><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d15348236e1971" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">6</a></sup><div data-db-target-of="bib6-d15348236e1971" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Yang F. </li><li class="loa__item">Liu T. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Moore R.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div> (<a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>E).</div><div class="section-paragraph">Pathway activity was compared between the tumor and normal samples using the method described in our previous study<div class="dropBlock reference-citations"><sup><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d15348236e1981" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">5</a></sup><div data-db-target-of="bib5-d15348236e1981" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Phosphopeptides from the stoichiometrically corrected phosphoproteomics data were mapped to pathways in NCI PID using gene symbols. For each pathway, the relative abundances of all phosphopeptides mapping to the pathway were compared between tumor and normal samples using a two-tailed t test followed by FDR correction using the Benjamini-Hochberg procedure. A similar analysis was performed with the global proteomics data.</div></section><section id="sec5.5.8" data-locator="sec5.5.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteomics Pathway Analysis Across Cohorts</span></h4><div class="section-paragraph">To identify potential improvement in phosphoproteome quality resulting from improved ischemia control, we compared the pathway activity of the retrospective and prospective cohorts using KEGG, Reactome and BioCarta pathway collections. For each patient in a particular cohort, we tested for enrichment for a specific pathway by comparing the abundance of phosphosites on proteins in the pathway against that of all other phosphosites in the dataset using a Kolmogorov-Smirnov test, adjusted for multiple hypothesis testing using the Benjamini-Hochberg approach. An adjusted p value threshold of 0.05 was used to determine significant enrichment. The number of patients for which the pathway was enriched was used as a measure of overall activity of the pathway in the cohort for purposes of comparing across the retrospective and prospective cohorts. <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1" class="scroll-into-link">Figure S5</a>A depicts a scatterplot of the number of patients for which a pathway was determined to be significantly enriched in the two datasets.</div></section><section id="sec5.5.9" data-locator="sec5.5.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">NetFill Analysis Linking CIN Score Associated Proteins Across Cohorts</span></h4><div class="section-paragraph">Proteogenomic analysis of HGSC tumor tissues from the TCGA cohort and the current prospectively collected cohort both identified a relatively small number of proteins that are highly associated with CIN score. A common issue when comparing discoveries across multiple studies is that the results may not overlap on the surface. To compare and interpret the results beyond simple Venn diagrams, we applied an in-house software package, NetFill, to identify a parsimonious set of molecular mediators to connect the results from the current and the TCGA cohorts. NetFill inputs information from multiple interactome databases and formulates the solution as a linear programming problem under constraints.</div></section><section id="sec5.5.10" data-locator="sec5.5.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Survival Analysis</span></h4><div class="section-paragraph">This was performed by first applying the four individual models (a set of proteins and weights) to the proteomics data reported in the current study to get scores, then averaging the resulting scores to get an overall predictive score for each sample. We note that in the previous study we used a voting method to combine results from the different models, but that was because the survival analysis (which we could not reasonably perform on the current cohort) allowed this. The proteins in our models were all observed and quantified in the current study with the exception of KIAA1715, out of 142 unique proteins considered in the models.</div></section></section><section id="sec5.6" data-locator="sec5.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and Code Availability</span></h3><div class="section-paragraph">Processed proteomic data tables are available in <a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3" class="scroll-into-link">Table S2</a>. Raw genomic data is available from the Genomic Data Commons (<a class="inter-ref-link" href="https://gdc.cancer.gov/" target="_blank">https://gdc.cancer.gov/</a>) and proteomic data is available via the CPTAC Data Portal (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptacPublic/" target="_blank">https://cptac-data-portal.georgetown.edu/cptacPublic/</a>). Interactive plots for proteomics and phosphoproteomics data are available at <a class="inter-ref-link" href="https://doi.org/10.25584/cptac/1601822" target="_blank">https://doi.org/10.25584/cptac/1601822</a>.</div></section></section><section><h2 class="top" id="secsectitle0260" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by grants U24CA160019, U24CA160036, and U24CA210955 from the National Institute of General Medical Sciences (NCI) CPTAC, and P41GM103493 from the National Institutes of Health. The proteomics work described herein was performed in the Environmental Molecular Sciences Laboratory (grid.436923.9), a US Department of Energy (DOE) national scientific user facility located at the Pacific Northwest National Laboratory (PNNL) in Richland, Washington. PNNL is a multi-program national laboratory operated by Battelle Memorial Institute for the DOE under Contract DE-AC05-76RL01830.</div><h3>Author Contributions</h3><div class="section-paragraph">Study Conception &amp; Design, R.D.S., M.A.B., T.L., and K.D.R. Investigation, Performed Experiment or Data Collection, M.A.G., T.R.C., R.J.M., A.A.S., M.A.B., and R.Z. Computation &amp; Statistical Analysis, J.E.M., O.A.A., V.A.P., Y.F., M.E.M., S.H.P., C.H., L.-B.W., S.F., M.W., Y.W., B.Z., Z.Z., and L.D. Data Interpretation &amp; Biological Analysis, J.E.M., O.A.A., V.A.P., Y.F., C.-F.T., E.S., D.W.C., B.Z., Z.Z., L.D., R.D.S., T.L., and K.D.R. Manuscript Preparation &amp; Revision, J.E.M., O.A.A., C.-F.T., M.T., A.I.R., Z.Z., L.D., R.D.S., T.L., and K.D.R. Supervision &amp; Administration, C.R.K., E.S.B., H.R., D.W.C., R.D.S., T.L., and K.D.R.</div><h3>Declaration of Interests</h3><div class="section-paragraph">The authors declare no competing interests.</div></section><section id="appsec2" data-locator="appsec2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental Information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental Information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-pdf-file2"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/fe84a7d5-139a-4117-b53b-7b4b3c624184/mmc1.pdf" class="supplemental-information__download " target="_blank">
                                            Download .pdf (3.69
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    pdf files
                                </a></div></div><div class="supplemental-information__title">Document S1. Figures S1–S5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/27ca9135-f87e-493a-87b3-c5db2d0d8005/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.08
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. Patient Cohort Clinical Data, Related to STAR Methods</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e513d6bf-d624-4680-911b-7daf7d15b7cf/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (49.74
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Proteomic and Phosphoproteomic Data, Related to STAR Methods</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/a962c084-c058-4152-89e4-bb7599a4ae40/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (16.39
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. RNA-Seq Data, Related to STAR Methods</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e52c3284-ff78-4dfa-b9f6-288aad8b7ae7/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.04
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. GISTIC Analysis of Arm-Level and Focal Alterations, Related to STAR Methods</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/e47f102a-2542-4245-bfe8-45ad04ff40dd/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.47
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Proteome and Phosphoproteome Tumor versus Normal Differential Analysis, Related to Figures 3 and 4</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/6730ffc7-15b5-4ee8-a850-8f76779f6539/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.13
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Tumor versus Normal Proteome Gene Set Enrichment Analysis, Related to Figure 3</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc8"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/c8d22e84-84a4-42c7-8f62-9f428a9e87ee/mmc8.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.03
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. Phosphoproteome Kinase Substrate Enrichment Analysis, Related to Figure 4</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc9"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/2340a46e-8336-4ce0-ae98-e74faf0d6336/mmc9.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.01
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S8. Differential Phosphosites Known to Be Involved in Kinase Activity, Related to Figure 6</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc10"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.xcrm.2020.100004/attachment/039539f0-07ea-4afc-85cf-5c40748795f0/mmc10.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.01
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S9. Analysis of Covariates Potentially Affecting Survival, Related to Figure 7</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0285" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Siegel R.L. </li><li class="ref__authors__name">Miller K.D. </li><li class="ref__authors__name">Jemal A. </li></ul><div class="ref__title">Cancer statistics, 2018.</div><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7-30</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85040049759&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7396)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29313949&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3322%2Fcaac.21442&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2018">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>American Cancer Society</li></ul><div class="ref__title">Key Statistics for Ovarian Cancer.</div><div class="citation-host"><a class="inter-ref-link" href="https://www.cancer.org/cancer/ovarian-cancer/about/key-statistics.html" target="_blank">https://www.cancer.org/cancer/ovarian-cancer/about/key-statistics.html</a></div><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=American+Cancer+SocietyKey+Statistics+for+Ovarian+Cancer.https%3A%2F%2Fwww.cancer.org%2Fcancer%2Fovarian-cancer%2Fabout%2Fkey-statistics.html">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Integrated genomic analyses of ovarian carcinoma.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">474</span></span>: <span class="ref__seriesPages">609-615</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79959838081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5913)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21720365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature10166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2011&pages=609-615&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Integrated+genomic+analyses+of+ovarian+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>National Cancer Institute</li></ul><div class="ref__title">SEER Cancer Statistics Review, 1975-2015.</div><div class="citation-host"><a class="inter-ref-link" href="https://seer.cancer.gov/archive/csr/1975_2015/" target="_blank">https://seer.cancer.gov/archive/csr/1975_2015/</a></div><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=National+Cancer+InstituteSEER+Cancer+Statistics+Review%2C+1975-2015.https%3A%2F%2Fseer.cancer.gov%2Farchive%2Fcsr%2F1975_2015%2F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Liu T. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">McDermott J.E. </li><li class="ref__authors__name">Zhou J.Y. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Ray D. </li><li>et al.</li><li>CPTAC Investigators</li></ul><div class="ref__title">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Liu T. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Moore R.J. </li><li>et al.</li></ul><div class="ref__title">Ischemia in tumors induces early and sustained phosphorylation changes in stress kinase pathways but does not affect global protein levels.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1690-1704</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84904111890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24719451&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fmcp.M113.036392&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=1690-1704&journal=Mol.+Cell.+Proteomics&author=Mertins+P.&author=Yang+F.&author=Liu+T.&author=Mani+D.R.&author=Petyuk+V.A.&author=Gillette+M.A.&author=Clauser+K.R.&author=Qiao+J.W.&author=Gritsenko+M.A.&author=Moore+R.J.&title=Ischemia+in+tumors+induces+early+and+sustained+phosphorylation+changes+in+stress+kinase+pathways+but+does+not+affect+global+protein+levels">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shyr C. </li><li class="ref__authors__name">Tarailo-Graovac M. </li><li class="ref__authors__name">Gottlieb M. </li><li class="ref__authors__name">Lee J.J. </li><li class="ref__authors__name">van Karnebeek C. </li><li class="ref__authors__name">Wasserman W.W. </li></ul><div class="ref__title">FLAGS, frequently mutated genes in public exomes.</div><span class="ref__series"><i>BMC Med. Genomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">64</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84946548774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (90)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25466818&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs12920-014-0064-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=64&journal=BMC+Med.+Genomics&author=Shyr+C.&author=Tarailo-Graovac+M.&author=Gottlieb+M.&author=Lee+J.J.&author=van+Karnebeek+C.&author=Wasserman+W.W.&title=FLAGS%2C+frequently+mutated+genes+in+public+exomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Integrated genomic analyses of ovarian carcinoma.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">474</span></span>: <span class="ref__seriesPages">609-615</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-79959838081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4655)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21720365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature10166&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2011&pages=609-615&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Integrated+genomic+analyses+of+ovarian+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Mootha V.K. </li><li class="ref__authors__name">Mukherjee S. </li><li class="ref__authors__name">Ebert B.L. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Paulovich A. </li><li class="ref__authors__name">Pomeroy S.L. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">15545-15550</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-27344435774&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32409)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16199517&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1073%2Fpnas.0506580102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Subramanian+A.&author=Tamayo+P.&author=Mootha+V.K.&author=Mukherjee+S.&author=Ebert+B.L.&author=Gillette+M.A.&author=Paulovich+A.&author=Pomeroy+S.L.&author=Golub+T.R.&author=Lander+E.S.&author=Mesirov+J.P.&title=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Thomas S.N. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Höti N. </li><li class="ref__authors__name">Zhang H. </li></ul><div class="ref__title">Targeted Proteomic Analyses of Histone H4 Acetylation Changes Associated with Homologous-Recombination-Deficient High-Grade Serous Ovarian Carcinomas.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">3704-3710</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85045317477&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28866885&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Facs.jproteome.7b00405&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=3704-3710&journal=J.%C2%A0Proteome+Res.&author=Thomas+S.N.&author=Chen+L.&author=Liu+Y.&author=H%C3%B6ti+N.&author=Zhang+H.&title=Targeted+Proteomic+Analyses+of+Histone+H4+Acetylation+Changes+Associated+with+Homologous-Recombination-Deficient+High-Grade+Serous+Ovarian+Carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Beyer A. </li><li class="ref__authors__name">Aebersold R. </li></ul><div class="ref__title">On the Dependency of Cellular Protein Levels on mRNA Abundance.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">535-550</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84964321113&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1798)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27104977&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2016.03.014&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=535-550&journal=Cell&author=Liu+Y.&author=Beyer+A.&author=Aebersold+R.&title=On+the+Dependency+of+Cellular+Protein+Levels+on+mRNA+Abundance">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Zimmerman L.J. </li><li class="ref__authors__name">Shaddox K.F. </li><li class="ref__authors__name">Kim S. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomic characterization of human colon and rectal cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V. </li><li class="ref__authors__name">Savage S. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic characterization of human colon cancer reveals new therapeutic opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.&author=Savage+S.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.&title=Proteogenomic+characterization+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ducie J. </li><li class="ref__authors__name">Dao F. </li><li class="ref__authors__name">Considine M. </li><li class="ref__authors__name">Olvera N. </li><li class="ref__authors__name">Shaw P.A. </li><li class="ref__authors__name">Kurman R.J. </li><li class="ref__authors__name">Shih I.M. </li><li class="ref__authors__name">Soslow R.A. </li><li class="ref__authors__name">Cope L. </li><li class="ref__authors__name">Levine D.A. </li></ul><div class="ref__title">Molecular analysis of high-grade serous ovarian carcinoma with and without associated serous tubal intra-epithelial carcinoma.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">990</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85031805521&amp;cf=" onclick="newWindow(this.href);return false">Scopus (132)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29042553&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41467-017-01217-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=990&journal=Nat.+Commun.&author=Ducie+J.&author=Dao+F.&author=Considine+M.&author=Olvera+N.&author=Shaw+P.A.&author=Kurman+R.J.&author=Shih+I.M.&author=Soslow+R.A.&author=Cope+L.&author=Levine+D.A.&title=Molecular+analysis+of+high-grade+serous+ovarian+carcinoma+with+and+without+associated+serous+tubal+intra-epithelial+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Labidi-Galy S.I. </li><li class="ref__authors__name">Papp E. </li><li class="ref__authors__name">Hallberg D. </li><li class="ref__authors__name">Niknafs N. </li><li class="ref__authors__name">Adleff V. </li><li class="ref__authors__name">Noe M. </li><li class="ref__authors__name">Bhattacharya R. </li><li class="ref__authors__name">Novak M. </li><li class="ref__authors__name">Jones S. </li><li class="ref__authors__name">Phallen J. </li><li>et al.</li></ul><div class="ref__title">High grade serous ovarian carcinomas originate in the fallopian tube.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">1093</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85032176462&amp;cf=" onclick="newWindow(this.href);return false">Scopus (457)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29061967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41467-017-00962-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1093&journal=Nat.+Commun.&author=Labidi-Galy+S.I.&author=Papp+E.&author=Hallberg+D.&author=Niknafs+N.&author=Adleff+V.&author=Noe+M.&author=Bhattacharya+R.&author=Novak+M.&author=Jones+S.&author=Phallen+J.&title=High+grade+serous+ovarian+carcinomas+originate+in+the+fallopian+tube">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Casado P. </li><li class="ref__authors__name">Rodriguez-Prados J.C. </li><li class="ref__authors__name">Cosulich S.C. </li><li class="ref__authors__name">Guichard S. </li><li class="ref__authors__name">Vanhaesebroeck B. </li><li class="ref__authors__name">Joel S. </li><li class="ref__authors__name">Cutillas P.R. </li></ul><div class="ref__title">Kinase-substrate enrichment analysis provides insights into the heterogeneity of signaling pathway activation in leukemia cells.</div><span class="ref__series"><i>Sci. Signal.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">rs6</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84875699641&amp;cf=" onclick="newWindow(this.href);return false">Scopus (242)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23532336&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1126%2Fscisignal.2003573&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=rs6&journal=Sci.+Signal.&author=Casado+P.&author=Rodriguez-Prados+J.C.&author=Cosulich+S.C.&author=Guichard+S.&author=Vanhaesebroeck+B.&author=Joel+S.&author=Cutillas+P.R.&title=Kinase-substrate+enrichment+analysis+provides+insights+into+the+heterogeneity+of+signaling+pathway+activation+in+leukemia+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wiredja D.D. </li><li class="ref__authors__name">Koyutürk M. </li><li class="ref__authors__name">Chance M.R. </li></ul><div class="ref__title">The KSEA App: a web-based tool for kinase activity inference from quantitative phosphoproteomics.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">3489-3491</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044408024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (135)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28655153&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtx415&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3489-3491&journal=Bioinformatics&author=Wiredja+D.D.&author=Koyut%C3%BCrk+M.&author=Chance+M.R.&title=The+KSEA+App%3A+a+web-based+tool+for+kinase+activity+inference+from+quantitative+phosphoproteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Tkachev S. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Skrzypek E. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Sullivan M. </li></ul><div class="ref__title">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">40</span></span>: <span class="ref__seriesPages">D261-D270</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84857047339&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1237)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22135298&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkr1122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D261-D270&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Kornhauser+J.M.&author=Tkachev+S.&author=Zhang+B.&author=Skrzypek+E.&author=Murray+B.&author=Latham+V.&author=Sullivan+M.&title=PhosphoSitePlus%3A+a+comprehensive+resource+for+investigating+the+structure+and+function+of+experimentally+determined+post-translational+modifications+in+man+and+mouse">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Song L. </li><li class="ref__authors__name">Pei L. </li><li class="ref__authors__name">Hu L. </li><li class="ref__authors__name">Pan S. </li><li class="ref__authors__name">Xiong W. </li><li class="ref__authors__name">Liu M. </li><li class="ref__authors__name">Wu Y. </li><li class="ref__authors__name">Shang Y. </li><li class="ref__authors__name">Yao S. </li></ul><div class="ref__title">Death-associated protein kinase 1 mediates interleukin-1β production through regulating inflammasome activation in Bv2 microglial cells and mice.</div><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">9930</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85049359757&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29967321&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fs41598-018-27842-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9930&journal=Sci.+Rep.&author=Song+L.&author=Pei+L.&author=Hu+L.&author=Pan+S.&author=Xiong+W.&author=Liu+M.&author=Wu+Y.&author=Shang+Y.&author=Yao+S.&title=Death-associated+protein+kinase+1+mediates+interleukin-1%CE%B2+production+through+regulating+inflammasome+activation+in+Bv2+microglial+cells+and+mice">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anjum R. </li><li class="ref__authors__name">Roux P.P. </li><li class="ref__authors__name">Ballif B.A. </li><li class="ref__authors__name">Gygi S.P. </li><li class="ref__authors__name">Blenis J. </li></ul><div class="ref__title">The tumor suppressor DAP kinase is a target of RSK-mediated survival signaling.</div><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">1762-1767</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-26244449244&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16213824&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2005.08.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=1762-1767&journal=Curr.+Biol.&author=Anjum+R.&author=Roux+P.P.&author=Ballif+B.A.&author=Gygi+S.P.&author=Blenis+J.&title=The+tumor+suppressor+DAP+kinase+is+a+target+of+RSK-mediated+survival+signaling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Goodrich D.W. </li></ul><div class="ref__title">The retinoblastoma tumor-suppressor gene, the exception that proves the rule.</div><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">5233-5243</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-33748075785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16936742&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fsj.onc.1209616&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=5233-5243&journal=Oncogene&author=Goodrich+D.W.&title=The+retinoblastoma+tumor-suppressor+gene%2C+the+exception+that+proves+the+rule">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Indovina P. </li><li class="ref__authors__name">Pentimalli F. </li><li class="ref__authors__name">Casini N. </li><li class="ref__authors__name">Vocca I. </li><li class="ref__authors__name">Giordano A. </li></ul><div class="ref__title">RB1 dual role in proliferation and apoptosis: cell fate control and implications for cancer therapy.</div><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">17873-17890</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84938632165&amp;cf=" onclick="newWindow(this.href);return false">Scopus (118)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26160835&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.18632%2Foncotarget.4286&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17873-17890&journal=Oncotarget&author=Indovina+P.&author=Pentimalli+F.&author=Casini+N.&author=Vocca+I.&author=Giordano+A.&title=RB1+dual+role+in+proliferation+and+apoptosis%3A+cell+fate+control+and+implications+for+cancer+therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chau B.N. </li><li class="ref__authors__name">Wang J.Y.J. </li></ul><div class="ref__title">Coordinated regulation of life and death by RB.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">130-138</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0037313618&amp;cf=" onclick="newWindow(this.href);return false">Scopus (246)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=12563312&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnrc993&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=130-138&journal=Nat.+Rev.+Cancer&author=Chau+B.N.&author=Wang+J.Y.J.&title=Coordinated+regulation+of+life+and+death+by+RB">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zarkowska T. </li><li class="ref__authors__name">Mittnacht S. </li></ul><div class="ref__title">Differential phosphorylation of the retinoblastoma protein by G1/S cyclin-dependent kinases.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">1997; </span><span class="ref__seriesVolume">272</span></span>: <span class="ref__seriesPages">12738-12746</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0030918714&amp;cf=" onclick="newWindow(this.href);return false">Scopus (439)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=9139732&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1074%2Fjbc.272.19.12738&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=12738-12746&journal=J.%C2%A0Biol.+Chem.&author=Zarkowska+T.&author=Mittnacht+S.&title=Differential+phosphorylation+of+the+retinoblastoma+protein+by+G1%2FS+cyclin-dependent+kinases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gu Y. </li><li class="ref__authors__name">Rosenblatt J. </li><li class="ref__authors__name">Morgan D.O. </li></ul><div class="ref__title">Cell cycle regulation of CDK2 activity by phosphorylation of Thr160 and Tyr15.</div><span class="ref__series"><i>EMBO J.</i> <span class="ref__seriesDate">1992; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">3995-4005</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0026713875&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=1396589&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1002%2Fj.1460-2075.1992.tb05493.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1992&pages=3995-4005&journal=EMBO+J.&author=Gu+Y.&author=Rosenblatt+J.&author=Morgan+D.O.&title=Cell+cycle+regulation+of+CDK2+activity+by+phosphorylation+of+Thr160+and+Tyr15">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hughes B.T. </li><li class="ref__authors__name">Sidorova J. </li><li class="ref__authors__name">Swanger J. </li><li class="ref__authors__name">Monnat Jr., R.J. </li><li class="ref__authors__name">Clurman B.E. </li></ul><div class="ref__title">Essential role for Cdk2 inhibitory phosphorylation during replication stress revealed by a human Cdk2 knockin mutation.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">110</span></span>: <span class="ref__seriesPages">8954-8959</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84878460802&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23671119&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1073%2Fpnas.1302927110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8954-8959&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Hughes+B.T.&author=Sidorova+J.&author=Swanger+J.&author=Monnat+R.J.&author=Clurman+B.E.&title=Essential+role+for+Cdk2+inhibitory+phosphorylation+during+replication+stress+revealed+by+a+human+Cdk2+knockin+mutation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Teixeira L.K. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Ekholm-Reed S. </li><li class="ref__authors__name">Wu X. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Reed S.I. </li></ul><div class="ref__title">Cyclin E deregulation promotes loss of specific genomic regions.</div><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1327-1333</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84929607476&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25959964&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cub.2015.03.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1327-1333&journal=Curr.+Biol.&author=Teixeira+L.K.&author=Wang+X.&author=Li+Y.&author=Ekholm-Reed+S.&author=Wu+X.&author=Wang+P.&author=Reed+S.I.&title=Cyclin+E+deregulation+promotes+loss+of+specific+genomic+regions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mazouzi A. </li><li class="ref__authors__name">Velimezi G. </li><li class="ref__authors__name">Loizou J.I. </li></ul><div class="ref__title">DNA replication stress: causes, resolution and disease.</div><span class="ref__series"><i>Exp. Cell Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">329</span></span>: <span class="ref__seriesPages">85-93</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84919413650&amp;cf=" onclick="newWindow(this.href);return false">Scopus (154)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25281304&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.yexcr.2014.09.030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=329&publication_year=2014&pages=85-93&journal=Exp.+Cell+Res.&author=Mazouzi+A.&author=Velimezi+G.&author=Loizou+J.I.&title=DNA+replication+stress%3A+causes%2C+resolution+and+disease">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bhattacharjee S. </li><li class="ref__authors__name">Nandi S. </li></ul><div class="ref__title">DNA damage response and cancer therapeutics through the lens of the Fanconi Anemia DNA repair pathway.</div><span class="ref__series"><i>Cell Commun. Signal.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">41</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85031099180&amp;cf=" onclick="newWindow(this.href);return false">Scopus (64)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29017571&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs12964-017-0195-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=41&journal=Cell+Commun.+Signal.&author=Bhattacharjee+S.&author=Nandi+S.&title=DNA+damage+response+and+cancer+therapeutics+through+the+lens+of+the+Fanconi+Anemia+DNA+repair+pathway">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ishiai M. </li><li class="ref__authors__name">Sato K. </li><li class="ref__authors__name">Tomida J. </li><li class="ref__authors__name">Kitao H. </li><li class="ref__authors__name">Kurumizaka H. </li><li class="ref__authors__name">Takata M. </li></ul><div class="ref__title">Activation of the FA pathway mediated by phosphorylation and ubiquitination.</div><span class="ref__series"><i>Mutat. Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">803-805</span></span>: <span class="ref__seriesPages">89-95</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85019867355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28552166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.mrfmmm.2017.05.003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=803-805&publication_year=2017&pages=89-95&journal=Mutat.+Res.&author=Ishiai+M.&author=Sato+K.&author=Tomida+J.&author=Kitao+H.&author=Kurumizaka+H.&author=Takata+M.&title=Activation+of+the+FA+pathway+mediated+by+phosphorylation+and+ubiquitination">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Marcel V. </li><li class="ref__authors__name">Fernandes K. </li><li class="ref__authors__name">Terrier O. </li><li class="ref__authors__name">Lane D.P. </li><li class="ref__authors__name">Bourdon J.C. </li></ul><div class="ref__title">Modulation of p53β and p53γ expression by regulating the alternative splicing of TP53 gene modifies cellular response.</div><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1377-1387</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84905909213&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24926616&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fcdd.2014.73&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1377-1387&journal=Cell+Death+Differ.&author=Marcel+V.&author=Fernandes+K.&author=Terrier+O.&author=Lane+D.P.&author=Bourdon+J.C.&title=Modulation+of+p53%CE%B2+and+p53%CE%B3+expression+by+regulating+the+alternative+splicing+of+TP53+gene+modifies+cellular+response">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Czubaty A. </li><li class="ref__authors__name">Piekiełko-Witkowska A. </li></ul><div class="ref__title">Protein kinases that phosphorylate splicing factors: roles in cancer development, progression and possible therapeutic options.</div><span class="ref__series"><i>Int. J. Biochem. Cell Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">91</span></span>: <span class="ref__seriesPages">102-115</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85020007962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=28552434&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.biocel.2017.05.024&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&pages=102-115&journal=Int.+J.+Biochem.+Cell+Biol.&author=Czubaty+A.&author=Piekie%C5%82ko-Witkowska+A.&title=Protein+kinases+that+phosphorylate+splicing+factors%3A+roles+in+cancer+development%2C+progression+and+possible+therapeutic+options">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhu D. </li><li class="ref__authors__name">Xu S. </li><li class="ref__authors__name">Deyanat-Yazdi G. </li><li class="ref__authors__name">Peng S.X. </li><li class="ref__authors__name">Barnes L.A. </li><li class="ref__authors__name">Narla R.K. </li><li class="ref__authors__name">Tran T. </li><li class="ref__authors__name">Mikolon D. </li><li class="ref__authors__name">Ning Y. </li><li class="ref__authors__name">Shi T. </li><li>et al.</li></ul><div class="ref__title">Synthetic Lethal Strategy Identifies a Potent and Selective TTK and CLK1/2 Inhibitor for Treatment of Triple-Negative Breast Cancer with a Compromised G<sub>1</sub>-S Checkpoint.</div><span class="ref__series"><i>Mol. Cancer Ther.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">1727-1738</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85052403479&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29866747&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1158%2F1535-7163.MCT-17-1084&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=1727-1738&journal=Mol.+Cancer+Ther.&author=Zhu+D.&author=Xu+S.&author=Deyanat-Yazdi+G.&author=Peng+S.X.&author=Barnes+L.A.&author=Narla+R.K.&author=Tran+T.&author=Mikolon+D.&author=Ning+Y.&author=Shi+T.&title=Synthetic+Lethal+Strategy+Identifies+a+Potent+and+Selective+TTK+and+CLK1%2F2+Inhibitor+for+Treatment+of+Triple-Negative+Breast+Cancer+with+a+Compromised+G1-S+Checkpoint">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kroeger Jr., P.T. </li><li class="ref__authors__name">Drapkin R. </li></ul><div class="ref__title">Pathogenesis and heterogeneity of ovarian cancer.</div><span class="ref__series"><i>Curr. Opin. Obstet. Gynecol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">26-34</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85000399534&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27898521&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1097%2FGCO.0000000000000340&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=26-34&journal=Curr.+Opin.+Obstet.+Gynecol.&author=Kroeger+P.T.&author=Drapkin+R.&title=Pathogenesis+and+heterogeneity+of+ovarian+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Park E.Y. </li><li class="ref__authors__name">Kim O. </li><li class="ref__authors__name">Schilder J.M. </li><li class="ref__authors__name">Coffey D.M. </li><li class="ref__authors__name">Cho C.H. </li><li class="ref__authors__name">Bast Jr., R.C. </li></ul><div class="ref__title">Cell Origins of High-Grade Serous Ovarian Cancer.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">E433</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85057199902&amp;cf=" onclick="newWindow(this.href);return false">Scopus (158)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30424539&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3390%2Fcancers10110433&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=E433&journal=Cancers+%28Basel%29&author=Kim+J.&author=Park+E.Y.&author=Kim+O.&author=Schilder+J.M.&author=Coffey+D.M.&author=Cho+C.H.&author=Bast+R.C.&title=Cell+Origins+of+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Pan J. </li><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">da Veiga Leprevost F. </li><li class="ref__authors__name">Reva B. </li><li class="ref__authors__name">Lih T.M. </li><li class="ref__authors__name">Chang H.Y. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e31</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (360)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e31&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Integrated+Proteogenomic+Characterization+of+Clear+Cell+Renal+Cell+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Xiao Y. </li><li class="ref__authors__name">Ma D. </li><li class="ref__authors__name">Zhao S. </li><li class="ref__authors__name">Suo C. </li><li class="ref__authors__name">Shi J. </li><li class="ref__authors__name">Xue M.Z. </li><li class="ref__authors__name">Ruan M. </li><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Zhao J. </li><li class="ref__authors__name">Li Q. </li><li>et al.</li></ul><div class="ref__title">Multi-Omics Profiling Reveals Distinct Microenvironment Characterization and Suggests Immune Escape Mechanisms of Triple-Negative Breast Cancer.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1158/1078-0432.CCR-18-3524">https://doi.org/10.1158/1078-0432.CCR-18-3524</a></span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85069636726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (237)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1158%2F1078-0432.CCR-18-3524&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&journal=Clin.+Cancer+Res.&author=Xiao+Y.&author=Ma+D.&author=Zhao+S.&author=Suo+C.&author=Shi+J.&author=Xue+M.Z.&author=Ruan+M.&author=Wang+H.&author=Zhao+J.&author=Li+Q.&title=Multi-Omics+Profiling+Reveals+Distinct+Microenvironment+Characterization+and+Suggests+Immune+Escape+Mechanisms+of+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Echarti A. </li><li class="ref__authors__name">Hecht M. </li><li class="ref__authors__name">Büttner-Herold M. </li><li class="ref__authors__name">Haderlein M. </li><li class="ref__authors__name">Hartmann A. </li><li class="ref__authors__name">Fietkau R. </li><li class="ref__authors__name">Distel L. </li></ul><div class="ref__title">CD8+ and Regulatory T cells Differentiate Tumor Immune Phenotypes and Predict Survival in Locally Advanced Head and Neck Cancer.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">E1398</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85078775668&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31546872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.3390%2Fcancers11091398&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=E1398&journal=Cancers+%28Basel%29&author=Echarti+A.&author=Hecht+M.&author=B%C3%BCttner-Herold+M.&author=Haderlein+M.&author=Hartmann+A.&author=Fietkau+R.&author=Distel+L.&title=CD8%2B+and+Regulatory+T%C2%A0cells+Differentiate+Tumor+Immune+Phenotypes+and+Predict+Survival+in+Locally+Advanced+Head+and+Neck+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Beniey M. </li><li class="ref__authors__name">Haque T. </li><li class="ref__authors__name">Hassan S. </li></ul><div class="ref__title">Translating the role of PARP inhibitors in triple-negative breast cancer.</div><span class="ref__series"><i>Oncoscience.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">287-288</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30800714&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.18632%2Foncoscience.474&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=287-288&journal=Oncoscience&author=Beniey+M.&author=Haque+T.&author=Hassan+S.&title=Translating+the+role+of+PARP+inhibitors+in+triple-negative+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Taylor E.M. </li><li class="ref__authors__name">Lindsay H.D. </li></ul><div class="ref__title">DNA replication stress and cancer: cause or cure?.</div><span class="ref__series"><i>Future Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">221-237</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84962627069&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26616915&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.2217%2Ffon.15.292&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=221-237&journal=Future+Oncol.&author=Taylor+E.M.&author=Lindsay+H.D.&title=DNA+replication+stress+and+cancer%3A+cause+or+cure%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gibbons B.C. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Payne S.H. </li></ul><div class="ref__title">Correcting systematic bias and instrument measurement drift with mzRefinery.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84950239084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtv437&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinformatics&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Shaw J.L. </li><li class="ref__authors__name">Daly D.S. </li><li class="ref__authors__name">Adkins J.N. </li><li class="ref__authors__name">Smith R.D. </li></ul><div class="ref__title">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.</div><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kessner D. </li><li class="ref__authors__name">Chambers M. </li><li class="ref__authors__name">Burke R. </li><li class="ref__authors__name">Agus D. </li><li class="ref__authors__name">Mallick P. </li></ul><div class="ref__title">ProteoWizard: open source software for rapid proteomics tools development.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1395)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Forbes S.A. </li><li class="ref__authors__name">Beare D. </li><li class="ref__authors__name">Boutselakis H. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Tate J. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Ward S. </li><li class="ref__authors__name">Dawson E. </li><li class="ref__authors__name">Ponting L. </li><li>et al.</li></ul><div class="ref__title">COSMIC: somatic cancer genetics at high-resolution.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D777-D783</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85016154270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1390)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27899578&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkw1121&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D777-D783&journal=Nucleic+Acids+Res.&author=Forbes+S.A.&author=Beare+D.&author=Boutselakis+H.&author=Bamford+S.&author=Bindal+N.&author=Tate+J.&author=Cole+C.G.&author=Ward+S.&author=Dawson+E.&author=Ponting+L.&title=COSMIC%3A+somatic+cancer+genetics+at+high-resolution">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Jaffe D. </li><li class="ref__authors__name">Sougnez C. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3325)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Saunders C.T. </li><li class="ref__authors__name">Wong W.S. </li><li class="ref__authors__name">Swamy S. </li><li class="ref__authors__name">Becq J. </li><li class="ref__authors__name">Murray L.J. </li><li class="ref__authors__name">Cheetham R.K. </li></ul><div class="ref__title">Strelka: accurate somatic small-variant calling from sequenced tumor-normal sample pairs.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">1811-1817</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84864153492&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1125)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22581179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbts271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1811-1817&journal=Bioinformatics&author=Saunders+C.T.&author=Wong+W.S.&author=Swamy+S.&author=Becq+J.&author=Murray+L.J.&author=Cheetham+R.K.&title=Strelka%3A+accurate+somatic+small-variant+calling+from+sequenced+tumor-normal+sample+pairs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Koboldt D.C. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Larson D.E. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Lin L. </li><li class="ref__authors__name">Miller C.A. </li><li class="ref__authors__name">Mardis E.R. </li><li class="ref__authors__name">Ding L. </li><li class="ref__authors__name">Wilson R.K. </li></ul><div class="ref__title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3368)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Schulz M.H. </li><li class="ref__authors__name">Long Q. </li><li class="ref__authors__name">Apweiler R. </li><li class="ref__authors__name">Ning Z. </li></ul><div class="ref__title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1501)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sherry S.T. </li><li class="ref__authors__name">Ward M.H. </li><li class="ref__authors__name">Kholodov M. </li><li class="ref__authors__name">Baker J. </li><li class="ref__authors__name">Phan L. </li><li class="ref__authors__name">Smigielski E.M. </li><li class="ref__authors__name">Sirotkin K. </li></ul><div class="ref__title">dbSNP: the NCBI database of genetic variation.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">308-311</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-0035173378&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5316)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=11125122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2F29.1.308&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=308-311&journal=Nucleic+Acids+Res.&author=Sherry+S.T.&author=Ward+M.H.&author=Kholodov+M.&author=Baker+J.&author=Phan+L.&author=Smigielski+E.M.&author=Sirotkin+K.&title=dbSNP%3A+the+NCBI+database+of+genetic+variation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhao H. </li><li class="ref__authors__name">Sun Z. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Huang H. </li><li class="ref__authors__name">Kocher J.P. </li><li class="ref__authors__name">Wang L. </li></ul><div class="ref__title">CrossMap: a versatile tool for coordinate conversion between genome assemblies.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">1006-1007</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84897373336&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24351709&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbtt730&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=1006-1007&journal=Bioinformatics&author=Zhao+H.&author=Sun+Z.&author=Wang+J.&author=Huang+H.&author=Kocher+J.P.&author=Wang+L.&title=CrossMap%3A+a+versatile+tool+for+coordinate+conversion+between+genome+assemblies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Hanna M. </li><li class="ref__authors__name">Banks E. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Kernytsky A. </li><li class="ref__authors__name">Garimella K. </li><li class="ref__authors__name">Altshuler D. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Daly M. </li><li class="ref__authors__name">DePristo M.A. </li></ul><div class="ref__title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17509)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&author=DePristo+M.A.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McLaren W. </li><li class="ref__authors__name">Gil L. </li><li class="ref__authors__name">Hunt S.E. </li><li class="ref__authors__name">Riat H.S. </li><li class="ref__authors__name">Ritchie G.R. </li><li class="ref__authors__name">Thormann A. </li><li class="ref__authors__name">Flicek P. </li><li class="ref__authors__name">Cunningham F. </li></ul><div class="ref__title">The Ensembl Variant Effect Predictor.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3975)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Scott A.D. </li><li class="ref__authors__name">Huang K.L. </li><li class="ref__authors__name">Weerasinghe A. </li><li class="ref__authors__name">Mashl R.J. </li><li class="ref__authors__name">Gao Q. </li><li class="ref__authors__name">Martins Rodrigues F. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Ding L. </li></ul><div class="ref__title">CharGer: Clinical Characterization of Germline Variants.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">865-867</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85062278783&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=30102335&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fbioinformatics%2Fbty649&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=865-867&journal=Bioinformatics&author=Scott+A.D.&author=Huang+K.L.&author=Weerasinghe+A.&author=Mashl+R.J.&author=Gao+Q.&author=Martins+Rodrigues+F.&author=Wyczalkowski+M.A.&author=Ding+L.&title=CharGer%3A+Clinical+Characterization+of+Germline+Variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Karczewski K.J. </li><li class="ref__authors__name">Weisburd B. </li><li class="ref__authors__name">Thomas B. </li><li class="ref__authors__name">Solomonson M. </li><li class="ref__authors__name">Ruderfer D.M. </li><li class="ref__authors__name">Kavanagh D. </li><li class="ref__authors__name">Hamamsy T. </li><li class="ref__authors__name">Lek M. </li><li class="ref__authors__name">Samocha K.E. </li><li class="ref__authors__name">Cummings B.B. </li><li>et al.</li><li>The Exome Aggregation Consortium</li></ul><div class="ref__title">The ExAC browser: displaying reference data information from over 60 000 exomes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D840-D845</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85011303767&amp;cf=" onclick="newWindow(this.href);return false">Scopus (493)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=27899611&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkw971&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D840-D845&journal=Nucleic+Acids+Res.&author=The+Exome+Aggregation+Consortium&title=The+ExAC+browser%3A+displaying+reference+data+information+from+over+60+000+exomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Landrum M.J. </li><li class="ref__authors__name">Lee J.M. </li><li class="ref__authors__name">Benson M. </li><li class="ref__authors__name">Brown G. </li><li class="ref__authors__name">Chao C. </li><li class="ref__authors__name">Chitipiralla S. </li><li class="ref__authors__name">Gu B. </li><li class="ref__authors__name">Hart J. </li><li class="ref__authors__name">Hoffman D. </li><li class="ref__authors__name">Hoover J. </li><li>et al.</li></ul><div class="ref__title">ClinVar: public archive of interpretations of clinically relevant variants.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D862-D868</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84976904305&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1797)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26582918&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkv1222&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D862-D868&journal=Nucleic+Acids+Res.&author=Landrum+M.J.&author=Lee+J.M.&author=Benson+M.&author=Brown+G.&author=Chao+C.&author=Chitipiralla+S.&author=Gu+B.&author=Hart+J.&author=Hoffman+D.&author=Hoover+J.&title=ClinVar%3A+public+archive+of+interpretations+of+clinically+relevant+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kumar P. </li><li class="ref__authors__name">Henikoff S. </li><li class="ref__authors__name">Ng P.C. </li></ul><div class="ref__title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</div><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1073-1081</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-68149165614&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5290)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=19561590&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnprot.2009.86&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=1073-1081&journal=Nat.+Protoc.&author=Kumar+P.&author=Henikoff+S.&author=Ng+P.C.&title=Predicting+the+effects+of+coding+non-synonymous+variants+on+protein+function+using+the+SIFT+algorithm">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Adzhubei I. </li><li class="ref__authors__name">Jordan D.M. </li><li class="ref__authors__name">Sunyaev S.R. </li></ul><div class="ref__title">Predicting functional effect of human missense mutations using PolyPhen-2.</div><span class="ref__series"><i>Curr. Protoc. Hum. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">Chapter 7</span></span>: <span class="ref__seriesPages">Unit 7.20</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=Chapter+7&publication_year=2013&pages=Unit+7.20&journal=Curr.+Protoc.+Hum.+Genet.&author=Adzhubei+I.&author=Jordan+D.M.&author=Sunyaev+S.R.&title=Predicting+functional+effect+of+human+missense+mutations+using+PolyPhen-2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Robinson J.T. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Winckler W. </li><li class="ref__authors__name">Guttman M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Integrative genomics viewer.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">24-26</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-78651271733&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9132)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=21221095&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt.1754&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=24-26&journal=Nat.+Biotechnol.&author=Robinson+J.T.&author=Thorvaldsd%C3%B3ttir+H.&author=Winckler+W.&author=Guttman+M.&author=Lander+E.S.&author=Getz+G.&author=Mesirov+J.P.&title=Integrative+genomics+viewer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li></ul><div class="ref__title">multiOmicsViz: plot the effect of one omics data on other omics data along the chromosome.</div><div class="citation-host"><a class="inter-ref-link" href="https://rdrr.io/bioc/multiOmicsViz/" target="_blank">https://rdrr.io/bioc/multiOmicsViz/</a></div><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Wang+J.+multiOmicsViz%3A+plot+the+effect+of+one+omics+data+on+other+omics+data+along+the+chromosome.https%3A%2F%2Frdrr.io%2Fbioc%2FmultiOmicsViz%2F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yu G. </li><li class="ref__authors__name">Wang L.G. </li><li class="ref__authors__name">Han Y. </li><li class="ref__authors__name">He Q.Y. </li></ul><div class="ref__title">clusterProfiler: an R package for comparing biological themes among gene clusters.</div><span class="ref__series"><i>OMICS.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">284-287</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84860718683&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17419)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=22455463&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1089%2Fomi.2011.0118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=284-287&journal=OMICS&author=Yu+G.&author=Wang+L.G.&author=Han+Y.&author=He+Q.Y.&title=clusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clusters">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ritchie M.E. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Wu D. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Law C.W. </li><li class="ref__authors__name">Shi W. </li><li class="ref__authors__name">Smyth G.K. </li></ul><div class="ref__title">limma powers differential expression analyses for RNA-sequencing and microarray studies.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20650)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Holman J.D. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Zimmerman L.J. </li><li>et al.</li></ul><div class="ref__title">Reproducibility of Differential Proteomic Technologies in CPTAC Fractionated Xenografts.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">691-706</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84960343160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=26653538&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Facs.jproteome.5b00859&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=691-706&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Wang+X.&author=Carr+S.A.&author=Clauser+K.R.&author=Mertins+P.&author=Chambers+M.C.&author=Holman+J.D.&author=Wang+J.&author=Zhang+B.&author=Zimmerman+L.J.&title=Reproducibility+of+Differential+Proteomic+Technologies+in+CPTAC+Fractionated+Xenografts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li S. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">Shao J. </li><li class="ref__authors__name">Crowder R. </li><li class="ref__authors__name">Liu W. </li><li class="ref__authors__name">Prat A. </li><li class="ref__authors__name">He X. </li><li class="ref__authors__name">Liu S. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Lu C. </li><li>et al.</li></ul><div class="ref__title">Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1116-1130</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84884559238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (498)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=24055055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1116-1130&journal=Cell+Rep.&author=Li+S.&author=Shen+D.&author=Shao+J.&author=Crowder+R.&author=Liu+W.&author=Prat+A.&author=He+X.&author=Liu+S.&author=Hoog+J.&author=Lu+C.&title=Endocrine-therapy-resistant+ESR1+variants+revealed+by+genomic+characterization+of+breast-cancer-derived+xenografts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Richards S. </li><li class="ref__authors__name">Aziz N. </li><li class="ref__authors__name">Bale S. </li><li class="ref__authors__name">Bick D. </li><li class="ref__authors__name">Das S. </li><li class="ref__authors__name">Gastier-Foster J. </li><li class="ref__authors__name">Grody W.W. </li><li class="ref__authors__name">Hegde M. </li><li class="ref__authors__name">Lyon E. </li><li class="ref__authors__name">Spector E. </li><li>et al.</li><li>ACMG Laboratory Quality Assurance Committee</li></ul><div class="ref__title">Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology.</div><span class="ref__series"><i>Genet. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">405-424</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-84928209346&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19710)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=25741868&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=abstract&amp;site=gim-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=fulltext&amp;site=gim-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fgim.2015.30&amp;cf=pdf&amp;site=gim-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=405-424&journal=Genet.+Med.&author=ACMG+Laboratory+Quality+Assurance+Committee&title=Standards+and+guidelines+for+the+interpretation+of+sequence+variants%3A+a+joint+consensus+recommendation+of+the+American+College+of+Medical+Genetics+and+Genomics+and+the+Association+for+Molecular+Pathology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huang K.L. </li><li class="ref__authors__name">Mashl R.J. </li><li class="ref__authors__name">Wu Y. </li><li class="ref__authors__name">Ritter D.I. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Oh C. </li><li class="ref__authors__name">Paczkowska M. </li><li class="ref__authors__name">Reynolds S. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Oak N. </li><li>et al.</li></ul><div class="ref__title">Pathogenic Germline Variants in 10,389 Adult Cancers.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">355-370.e1–e6</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85044578706&amp;cf=" onclick="newWindow(this.href);return false">Scopus (517)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=29625052&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2018.03.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=355-370.e1%E2%80%93e6&journal=Cell&author=Huang+K.L.&author=Mashl+R.J.&author=Wu+Y.&author=Ritter+D.I.&author=Wang+J.&author=Oh+C.&author=Paczkowska+M.&author=Reynolds+S.&author=Wyczalkowski+M.A.&author=Oak+N.&title=Pathogenic+Germline+Variants+in+10%2C389+Adult+Cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Gupta N. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Beausoleil S.A. </li><li class="ref__authors__name">Villén J. </li><li class="ref__authors__name">Gerber S.A. </li><li class="ref__authors__name">Rush J. </li><li class="ref__authors__name">Gygi S.P. </li></ul><div class="ref__title">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1297)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O.A. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (425)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Linding R. </li><li class="ref__authors__name">Jensen L.J. </li><li class="ref__authors__name">Pasculescu A. </li><li class="ref__authors__name">Olhovsky M. </li><li class="ref__authors__name">Colwill K. </li><li class="ref__authors__name">Bork P. </li><li class="ref__authors__name">Yaffe M.B. </li><li class="ref__authors__name">Pawson T. </li></ul><div class="ref__title">NetworKIN: a resource for exploring cellular phosphorylation networks.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">D695-D699</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=2-s2.0-38549109487&amp;cf=" onclick="newWindow(this.href);return false">Scopus (272)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=17981841&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.xcrm.2020.100004&amp;key=10.1093%2Fnar%2Fgkm902&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D695-D699&journal=Nucleic+Acids+Res.&author=Linding+R.&author=Jensen+L.J.&author=Pasculescu+A.&author=Olhovsky+M.&author=Colwill+K.&author=Bork+P.&author=Yaffe+M.B.&author=Pawson+T.&title=NetworKIN%3A+a+resource+for+exploring+cellular+phosphorylation+networks">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: April 21, 2020; corrected online August 13, 2020</div><div class="article-info__date">Accepted:
            March 17,
            2020</div><div class="article-info__date">Received in revised form:
            December 5,
            2019</div><div class="article-info__date">Received:
            July 11,
            2019</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.xcrm.2020.100004">https://doi.org/10.1016/j.xcrm.2020.100004</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2020 The Author(s).</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S2666379120300045">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

            

            

            <section data-widget-id="e587233e-22aa-4221-8ae1-22df489d46f7" data-widget-name="UX3linkedContentList"><h2 id="section-e587233e-22aa-4221-8ae1-22df489d46f7" class="top">Linked Article</h2><div class="section-paragraph linked-article"><ul class="rlist"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(20)30096-3" title="Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability">Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability</a><div><ul class="meta__authors rlist--inline comma" title="list of authors">McDermott et al.</ul></div><div class="linked-article__details"><span><i>Cell Reports Medicine</i></span><span id="item_date">August 25, 2020</span></div><ul class="rlist rlist--inline linked-article__links clearfix"><li class="bulleted preview-label"><a href="#" data-db-target-for="related-article-in-brief-S2666-3791(20)30096-3" class="link--underlined" title="Preview Abstract">In Brief</a><div data-db-target-of="related-article-in-brief-S2666-3791(20)30096-3" class="dropBlock__holder__article-preview dropBlock__holder"><div class="dropBlock__body"><span>(Cell Reports Medicine <em>1</em>, 100004-1–14.e1–e8; April 21, 2020)</span></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></li><li class="bulleted"><a title="Full-Text" class="faded link--underlined" href="/cell-reports-medicine/fulltext/S2666-3791(20)30096-3">
                Full-Text
            </a></li><li class="bulleted"><a title="PDF" class="link--underlined" target="_blank" href="https://www.cell.com/cell-reports-medicine/pdf/S2666-3791(20)30096-3.pdf">
                    PDF
                </a></li></ul><span class="article-header__access">Open Access</span></li></ul></div></section>

            

        

    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S2666379120300045" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(20)30004-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666379120300045%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(20)30004-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2666379120300045%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"2666-3791","name":"Cell Reports Medicine"},"page":{"loadTimestamp":"1723231594446","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"1","issueNumber":"1","viewState":"fulltext","format":"MIME-HTML","id":"pii:S2666379120300045","coverDate":"202004","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases Replication Stress in Observed Chromosomal Instability","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
