<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="40c8f549-9466-4031-a528-84ea3d29bae8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text;article:article:pii\:S0092867421008576;issue:issue:pii\:S0092867420X00171"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="A proteogenomic portrait of lung squamous cell carcinoma" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00857-6" />
    



        <meta property="og:site_name" content="Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg" />
    


        <meta property="og:description" content="Comprehensive proteogenomic characterization of lung squamous cell carcinomas and
paired normal adjacent tissues identifies taxonomic subclasses, alternative driver
events, and insights into immune modulation, as well as putative biomarkers and potential
therapeutic targets." />
    











        <meta name="twitter:title" content="A proteogenomic portrait of lung squamous cell carcinoma" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg" />
    


        <meta name="twitter:description" content="Comprehensive proteogenomic characterization of lung squamous cell carcinomas and
paired normal adjacent tissues identifies taxonomic subclasses, alternative driver
events, and insights into immune modulation, as well as putative biomarkers and potential
therapeutic targets." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867421008576%3Fshowall%3Dtrue">
    


<title>A proteogenomic portrait of lung squamous cell carcinoma: Cell</title><meta name="citation_journal_title" content="Cell"/> <meta name="citation_journal_abbrev" content="Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="A proteogenomic portrait of lung squamous cell carcinoma"/> <meta name="citation_date" content="2021/08/05"/> <meta name="citation_volume" content="184"/> <meta name="citation_issue" content="16"/> <meta name="citation_firstpage" content="4348"/> <meta name="citation_lastpage" content="4371.e40"/> <meta name="citation_pmid" content="34358469"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S0092867421008576/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S0092867421008576/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S0092867421008576/pdf"/> <meta name="citation_issn" content="0092-8674"/> <meta name="citation_issn" content="1097-4172"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.cell.2021.07.016"/> <meta name="citation_pmid" content="34358469"/> <meta name="citation_author" content="Shankha Satpathy"></meta><meta name="citation_author_email" content="shankha@broadinstitute.org"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Karsten Krug"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Pierre M. Jean Beltran"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Sara R. Savage"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Francesca Petralia"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Chandan Kumar-Sinha"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Yongchao Dou"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Boris Reva"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="M. Harry Kane"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Shayan C. Avanessian"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Suhas V. Vasaikar"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Azra Krek"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Jonathan T. Lei"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Eric J. Jaehnig"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Tatiana Omelchenko"></meta><meta name="citation_author_institution" content="Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"></meta><meta name="citation_author" content="Yifat Geffen"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Erik J. Bergstrom"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Vasileios Stathias"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Karen E. Christianson"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="David I. Heiman"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Marcin P. Cieslik"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Song Cao"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Xiaoyu Song"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Jiayi Ji"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Wenke Liu"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Kai Li"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yize Li"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Zeynep H. Gümüş"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Myvizhi Esai Selvan"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Rama Soundararajan"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Tanvi H. Visal"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Maria G. Raso"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Edwin Roger Parra"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Özgün Babur"></meta><meta name="citation_author_institution" content="Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA"></meta><meta name="citation_author" content="Pankaj Vats"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Shankara Anand"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Tobias Schraink"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="MacIntosh Cornwell"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Fernanda Martins Rodrigues"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Houxiang Zhu"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Chia-Kuei Mo"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Yuping Zhang"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Felipe da Veiga Leprevost"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Arul M. Chinnaiyan"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Matthew A. Wyczalkowski"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Gilbert S. Omenn"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Chelsea J. Newton"></meta><meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, MI 49503, USA"></meta><meta name="citation_author" content="Stephan Schurer"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA"></meta><meta name="citation_author" content="Kelly V. Ruggles"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="David Fenyö"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Scott D. Jewell"></meta><meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, MI 49503, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Sendurai A. Mani"></meta><meta name="citation_author_institution" content="Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Namrata D. Udeshi"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Gad Getz"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="James Suh"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Qing Kay Li"></meta><meta name="citation_author_institution" content="Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD 21224, USA"></meta><meta name="citation_author" content="Galen Hostetter"></meta><meta name="citation_author_institution" content="Van Andel Research Institute, Grand Rapids, MI 49503, USA"></meta><meta name="citation_author" content="Paul K. Paik"></meta><meta name="citation_author_institution" content="Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA"></meta><meta name="citation_author" content="Saravana M. Dhanasekaran"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Ramaswamy Govindan"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Karl R. Clauser"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Alexey I. Nesvizhskii"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Pei Wang"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author_email" content="scarr@broad.mit.edu"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_email" content="bing.zhang@bcm.edu"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author_email" content="manidr@broadinstitute.org"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author_email" content="gillette@broadinstitute.org"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author_institution" content="Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02115, USA"></meta><meta name="citation_author" content="Alex Green"></meta><meta name="citation_author" content="Alfredo Molinolo"></meta><meta name="citation_author" content="Alicia Francis"></meta><meta name="citation_author" content="Amanda G. Paulovich"></meta><meta name="citation_author" content="Andrii Karnuta"></meta><meta name="citation_author" content="Antonio Colaprico"></meta><meta name="citation_author" content="Barbara Hindenach"></meta><meta name="citation_author" content="Barbara L. Pruetz"></meta><meta name="citation_author" content="Bartosz Kubisa"></meta><meta name="citation_author" content="Brian J. Druker"></meta><meta name="citation_author" content="Carissa Huynh"></meta><meta name="citation_author" content="Charles A. Goldthwaite"></meta><meta name="citation_author" content="Chet Birger"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author" content="Corbin D. Jones"></meta><meta name="citation_author" content="Dan Rohrer"></meta><meta name="citation_author" content="Dana R. Valley"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author" content="David Chesla"></meta><meta name="citation_author" content="Donna Hansel"></meta><meta name="citation_author" content="Elena V. Ponomareva"></meta><meta name="citation_author" content="Elizabeth Duffy"></meta><meta name="citation_author" content="Eric Burks"></meta><meta name="citation_author" content="Eric E. Schadt"></meta><meta name="citation_author" content="Eugene S. Fedorov"></meta><meta name="citation_author" content="Eunkyung An"></meta><meta name="citation_author" content="Fei Ding"></meta><meta name="citation_author" content="George D. Wilson"></meta><meta name="citation_author" content="Harsh Batra"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author" content="Jennifer E. Maas"></meta><meta name="citation_author" content="Jennifer Eschbacher"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author" content="Katherine A. Hoadley"></meta><meta name="citation_author" content="Kei Suzuki"></meta><meta name="citation_author" content="Ki Sung Um"></meta><meta name="citation_author" content="Liqun Qi"></meta><meta name="citation_author" content="Lori Bernard"></meta><meta name="citation_author" content="Maciej Wiznerowicz"></meta><meta name="citation_author" content="Małgorzata Wojtyś"></meta><meta name="citation_author" content="Marcin J. Domagalski"></meta><meta name="citation_author" content="Matthew J. Ellis"></meta><meta name="citation_author" content="Maureen A. Dyer"></meta><meta name="citation_author" content="Melissa Borucki"></meta><meta name="citation_author" content="Meenakshi Anurag"></meta><meta name="citation_author" content="Michael J. Birrer"></meta><meta name="citation_author" content="Midie Xu"></meta><meta name="citation_author" content="Mikhail Krotevich"></meta><meta name="citation_author" content="Nancy Roche"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Negin Vatanian"></meta><meta name="citation_author" content="Neil R. Mucci"></meta><meta name="citation_author" content="Nicollette Maunganidze"></meta><meta name="citation_author" content="Nikolay Gabrovski"></meta><meta name="citation_author" content="Olga Potapova"></meta><meta name="citation_author" content="Oluwole Fadare"></meta><meta name="citation_author" content="Pamela Grady"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="Pushpa Hariharan"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Rebecca Montgomery"></meta><meta name="citation_author" content="Renganayaki Pandurengan"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author" content="Robert J. Welsh"></meta><meta name="citation_author" content="Sailaja Mareedu"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author" content="Sandra Cottingham"></meta><meta name="citation_author" content="Shilpi Singh"></meta><meta name="citation_author" content="Shirley X. Tsang"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Stacey Gabriel"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author" content="Tanmayi Vashist"></meta><meta name="citation_author" content="Thomas Bauer"></meta><meta name="citation_author" content="Volodymyr Sovenko"></meta><meta name="citation_author" content="Warren G. Tourtellotte"></meta><meta name="citation_author" content="Weiping Ma"></meta><meta name="citation_author" content="William Bocik"></meta><meta name="citation_author" content="Wohaib Hasan"></meta><meta name="citation_author" content="Xiaojun Jing"></meta><meta name="citation_author" content="Ximing Tang"></meta><meta name="citation_author" content="Yuxing Liao"></meta><meta name="citation_author" content=" Yvonne"></meta><meta name="citation_author" content=" Shutack"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author" content="Ziad Hanhan"></meta><meta name="citation_keywords" content="lung cancer; squamous; proteogenomics; proteomics; genomics; protein; phosphorylation; acetylation; ubiquitylation; CPTAC"></meta><meta name="citation_reference" content="citation_title=SnapShot: Immune Checkpoint Inhibitors; citation_author=G. Abril-Rodriguez; citation_author=A. Ribas; citation_journal_title=Cancer Cell; citation_volume=31; citation_firstpage=848; citation_lastpage=848.e1; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study; citation_author=E.R. Ahn; citation_author=P.K. Mangat; citation_author=E. Garrett-Mayer; citation_author=S. Halabi; citation_author=E.G. Dib; citation_author=D.E. Haggstrom; citation_author=K.B. Alguire; citation_author=C.J. Calfa; citation_author=T.L. Cannon; citation_author=P.A. Crilley; citation_author=et al.; citation_journal_title=JCO Precision Oncology; citation_doi=10.1200/PO.20.00037; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis; citation_author=M. Albert; citation_author=M. Bécares; citation_author=M. Falqui; citation_author=C. Fernández-Lozano; citation_author=S. Guerra; citation_journal_title=Viruses; citation_volume=10; citation_firstpage=629; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity; citation_author=S. Alcalá; citation_author=P. Sancho; citation_author=P. Martinelli; citation_author=D. Navarro; citation_author=C. Pedrero; citation_author=L. Martín-Hijano; citation_author=S. Valle; citation_author=J. Earl; citation_author=M. Rodríguez-Serrano; citation_author=L. Ruiz-Cañas; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=2682; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens; citation_author=L.G. Almeida; citation_author=N.J. Sakabe; citation_author=A.R. deOliveira; citation_author=M.C.C. Silva; citation_author=A.S. Mundstein; citation_author=T. Cohen; citation_author=Y.-T. Chen; citation_author=R. Chua; citation_author=S. Gurung; citation_author=S. Gnjatic; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=37; citation_firstpage=D816; citation_lastpage=D819; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=xCell: digitally portraying the tissue cellular heterogeneity landscape; citation_author=D. Aran; citation_author=Z. Hu; citation_author=A.J. Butte; citation_journal_title=Genome Biol.; citation_volume=18; citation_firstpage=220; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=LungMAP: The Molecular Atlas of Lung Development Program; citation_author=M.E. Ardini-Poleske; citation_author=R.F. Clark; citation_author=C. Ansong; citation_author=J.P. Carson; citation_author=R.A. Corley; citation_author=G.H. Deutsch; citation_author=J.S. Hagood; citation_author=N. Kaminski; citation_author=T.J. Mariani; citation_author=S.S. Potter; citation_author=LungMAP Consortium; citation_author=et al.; citation_journal_title=Am. J. Physiol. Lung Cell. Mol. Physiol.; citation_volume=313; citation_firstpage=L733; citation_lastpage=L740; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Causal interactions from proteomic profiles: Molecular data meet pathway knowledge; citation_author=Ö. Babur; citation_author=A. Luna; citation_author=A. Korkut; citation_author=F. Durupinar; citation_author=M.C. Siper; citation_author=U. Dogrusoz; citation_author=A.S. Vaca Jacome; citation_author=R. Peckner; citation_author=K.E. Christianson; citation_author=J.D. Jaffe; citation_author=et al.; citation_journal_title=Patterns; citation_volume=2; citation_firstpage=100257; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Site-specific reactivity of nonenzymatic lysine acetylation; citation_author=J. Baeza; citation_author=M.J. Smallegan; citation_author=J.M. Denu; citation_journal_title=ACS Chem. Biol.; citation_volume=10; citation_firstpage=122; citation_lastpage=128; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Comprehensive Characterization of Cancer Driver Genes and Mutations; citation_author=M.H. Bailey; citation_author=C. Tokheim; citation_author=E. Porta-Pardo; citation_author=S. Sengupta; citation_author=D. Bertrand; citation_author=A. Weerasinghe; citation_author=A. Colaprico; citation_author=M.C. Wendl; citation_author=J. Kim; citation_author=B. Reardon; citation_author=MC3 Working Group; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=371; citation_lastpage=385.e18; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer; citation_author=H. Bando; citation_author=T. Atsumi; citation_author=T. Nishio; citation_author=H. Niwa; citation_author=S. Mishima; citation_author=C. Shimizu; citation_author=N. Yoshioka; citation_author=R. Bucala; citation_author=T. Koike; citation_journal_title=Clin. Cancer Res.; citation_volume=11; citation_firstpage=5784; citation_lastpage=5792; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1; citation_author=D.A. Barbie; citation_author=P. Tamayo; citation_author=J.S. Boehm; citation_author=S.Y. Kim; citation_author=S.E. Moody; citation_author=I.F. Dunn; citation_author=A.C. Schinzel; citation_author=P. Sandy; citation_author=E. Meylan; citation_author=C. Scholl; citation_author=et al.; citation_journal_title=Nature; citation_volume=462; citation_firstpage=108; citation_lastpage=112; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5; citation_author=L. Barker; citation_journal_title=Am. Stat.; citation_volume=56; citation_firstpage=85; citation_lastpage=89; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=On the approximation of curves by line segments using dynamic programming; citation_author=R. Bellman; citation_journal_title=Commun. ACM; citation_volume=4; citation_firstpage=284; citation_publication_date=1961; "></meta><meta name="citation_reference" content="citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J.R. Stat. Soc. Series B Stat. Methodol.; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "></meta><meta name="citation_reference" content="citation_title=Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries; citation_author=F. Bray; citation_author=J. Ferlay; citation_author=I. Soerjomataram; citation_author=R.L. Siegel; citation_author=L.A. Torre; citation_author=A. Jemal; citation_journal_title=CA Cancer J. Clin.; citation_volume=68; citation_firstpage=394; citation_lastpage=424; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Metagenes and molecular pattern discovery using matrix factorization; citation_author=J.-P. Brunet; citation_author=P. Tamayo; citation_author=T.R. Golub; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=4164; citation_lastpage=4169; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer; citation_author=C. Cadenas; citation_author=D. Franckenstein; citation_author=M. Schmidt; citation_author=M. Gehrmann; citation_author=M. Hermes; citation_author=B. Geppert; citation_author=W. Schormann; citation_author=L.J. Maccoux; citation_author=M. Schug; citation_author=A. Schumann; citation_author=et al.; citation_journal_title=Breast Cancer Res.; citation_volume=12; citation_firstpage=R44; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer; citation_author=J. Cai; citation_author=L. Fang; citation_author=Y. Huang; citation_author=R. Li; citation_author=J. Yuan; citation_author=Y. Yang; citation_author=X. Zhu; citation_author=B. Chen; citation_author=J. Wu; citation_author=M. Li; citation_journal_title=Cancer Res.; citation_volume=73; citation_firstpage=5402; citation_lastpage=5415; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42; citation_author=M. Cammer; citation_author=J.-C. Gevrey; citation_author=M. Lorenz; citation_author=A. Dovas; citation_author=J. Condeelis; citation_author=D. Cox; citation_journal_title=J. Biol. Chem.; citation_volume=284; citation_firstpage=23302; citation_lastpage=23311; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas; citation_author=J.D. Campbell; citation_author=C. Yau; citation_author=R. Bowlby; citation_author=Y. Liu; citation_author=K. Brennan; citation_author=H. Fan; citation_author=A.M. Taylor; citation_author=C. Wang; citation_author=V. Walter; citation_author=R. Akbani; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=23; citation_firstpage=194; citation_lastpage=212.e6; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy; citation_author=M.A. Cannarile; citation_author=M. Weisser; citation_author=W. Jacob; citation_author=A.-M. Jegg; citation_author=C.H. Ries; citation_author=D. Rüttinger; citation_journal_title=J. Immunother. Cancer; citation_volume=5; citation_firstpage=53; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells; citation_author=S. Cannito; citation_author=E. Novo; citation_author=A. Compagnone; citation_author=L. Valfrè di Bonzo; citation_author=C. Busletta; citation_author=E. Zamara; citation_author=C. Paternostro; citation_author=D. Povero; citation_author=A. Bandino; citation_author=F. Bozzo; citation_author=et al.; citation_journal_title=Carcinogenesis; citation_volume=29; citation_firstpage=2267; citation_lastpage=2278; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Targeting macrophages: therapeutic approaches in cancer; citation_author=L. Cassetta; citation_author=J.W. Pollard; citation_journal_title=Nat. Rev. Drug Discov.; citation_volume=17; citation_firstpage=887; citation_lastpage=904; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer; citation_author=S. Chida; citation_author=H. Okayama; citation_author=M. Noda; citation_author=K. Saito; citation_author=T. Nakajima; citation_author=K. Aoto; citation_author=S. Hayase; citation_author=T. Momma; citation_author=S. Ohki; citation_author=K. Kono; citation_author=S. Takenoshita; citation_journal_title=Carcinogenesis; citation_volume=37; citation_firstpage=878; citation_lastpage=887; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=ordinal-Regression Models for Ordinal Data. R package version 2019.2-10. (The R Project for Statistical Computing); citation_author=R.H.B. Christensen; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Large-scale profiling of microRNAs for The Cancer Genome Atlas; citation_author=A. Chu; citation_author=G. Robertson; citation_author=D. Brooks; citation_author=A.J. Mungall; citation_author=I. Birol; citation_author=R. Coope; citation_author=Y. Ma; citation_author=S. Jones; citation_author=M.A. Marra; citation_journal_title=Nucleic Acids Res; citation_volume=44; citation_firstpage=e3; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Ror2 as a therapeutic target in cancer; citation_author=Z. Debebe; citation_author=W.K. Rathmell; citation_journal_title=Pharmacol. Ther.; citation_volume=150; citation_firstpage=143; citation_lastpage=148; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Fundamentals of cancer metabolism; citation_author=R.J. DeBerardinis; citation_author=N.S. Chandel; citation_journal_title=Sci. Adv.; citation_volume=2; citation_firstpage=e1600200; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration; citation_author=P. Deelen; citation_author=M.J. Bonder; citation_author=K.J. van der Velde; citation_author=H.-J. Westra; citation_author=E. Winder; citation_author=D. Hendriksen; citation_author=L. Franke; citation_author=M.A. Swertz; citation_journal_title=BMC Res. Notes; citation_volume=7; citation_firstpage=901; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma; citation_author=M.P. DeYoung; citation_author=C.M. Johannessen; citation_author=C.-O. Leong; citation_author=W. Faquin; citation_author=J.W. Rocco; citation_author=L.W. Ellisen; citation_journal_title=Cancer Res.; citation_volume=66; citation_firstpage=9362; citation_lastpage=9368; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling; citation_author=D. Diviani; citation_author=F. Raimondi; citation_author=C.D. Del Vescovo; citation_author=E. Dreyer; citation_author=E. Reggi; citation_author=H. Osman; citation_author=L. Ruggieri; citation_author=C. Gonano; citation_author=S. Cavin; citation_author=C.L. Box; citation_author=et al.; citation_journal_title=Cell Chem. Biol.; citation_volume=23; citation_firstpage=1135; citation_lastpage=1146; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages; citation_author=A. Dovas; citation_author=J.-C. Gevrey; citation_author=A. Grossi; citation_author=H. Park; citation_author=W. Abou-Kheir; citation_author=D. Cox; citation_journal_title=J. Cell Sci.; citation_volume=122; citation_firstpage=3873; citation_lastpage=3882; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=SWOG S1400C (NCT02154490)-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy); citation_author=M.J. Edelman; citation_author=M.W. Redman; citation_author=K.S. Albain; citation_author=E.C. McGary; citation_author=N.M. Rafique; citation_author=D. Petro; citation_author=S.N. Waqar; citation_author=K. Minichiello; citation_author=J. Miao; citation_author=V.A. Papadimitrakopoulou; citation_author=et al.; citation_journal_title=J. Thorac. Oncol.; citation_volume=14; citation_firstpage=1853; citation_lastpage=1859; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance); citation_author=M.J. Ellis; citation_author=V.J. Suman; citation_author=J. Hoog; citation_author=R. Goncalves; citation_author=S. Sanati; citation_author=C.J. Creighton; citation_author=K. DeSchryver; citation_author=E. Crouch; citation_author=A. Brink; citation_author=M. Watson; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=35; citation_firstpage=1061; citation_lastpage=1069; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The Reactome Pathway Knowledgebase; citation_author=A. Fabregat; citation_author=S. Jupe; citation_author=L. Matthews; citation_author=K. Sidiropoulos; citation_author=M. Gillespie; citation_author=P. Garapati; citation_author=R. Haw; citation_author=B. Jassal; citation_author=F. Korninger; citation_author=B. May; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D649; citation_lastpage=D655; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment; citation_author=T. Fan; citation_author=G. Sun; citation_author=X. Sun; citation_author=L. Zhao; citation_author=R. Zhong; citation_author=Y. Peng; citation_journal_title=Cancers (Basel); citation_volume=11; citation_firstpage=317; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Lactate Metabolism in Human Lung Tumors; citation_author=B. Faubert; citation_author=K.Y. Li; citation_author=L. Cai; citation_author=C.T. Hensley; citation_author=J. Kim; citation_author=L.G. Zacharias; citation_author=C. Yang; citation_author=Q.N. Do; citation_author=S. Doucette; citation_author=D. Burguete; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=358; citation_lastpage=371.e9; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation; citation_author=A.P. Fernandes; citation_author=A. Capitanio; citation_author=M. Selenius; citation_author=O. Brodin; citation_author=A.-K. Rundlöf; citation_author=M. Björnstedt; citation_journal_title=Histopathology; citation_volume=55; citation_firstpage=313; citation_lastpage=320; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries; citation_author=S. Fisher; citation_author=A. Barry; citation_author=J. Abreu; citation_author=B. Minie; citation_author=J. Nolan; citation_author=T.M. Delorey; citation_author=G. Young; citation_author=T.J. Fennell; citation_author=A. Allen; citation_author=L. Ambrogio; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R1; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Glycogen synthase kinase-3beta modulates notch signaling and stability; citation_author=D.R. Foltz; citation_author=M.C. Santiago; citation_author=B.E. Berechid; citation_author=J.S. Nye; citation_journal_title=Curr. Biol.; citation_volume=12; citation_firstpage=1006; citation_lastpage=1011; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Functional normalization of 450k methylation array data improves replication in large cancer studies; citation_author=J.-P. Fortin; citation_author=A. Labbe; citation_author=M. Lemire; citation_author=B.W. Zanke; citation_author=T.J. Hudson; citation_author=E.J. Fertig; citation_author=C.M. Greenwood; citation_author=K.D. Hansen; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=503; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=mclust 5: clustering, classification and density estimation using Gaussian finite mixture models; citation_author=C. Fraley; citation_author=A.E. Raftery; citation_author=L. Scrucca; citation_author=T.B. Murphy; citation_author=M. Fop; citation_journal_title=The R Journal; citation_volume=8; citation_firstpage=289; citation_lastpage=317; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering; citation_author=D. Freed; citation_author=R. Pan; citation_author=R. Aldana; citation_journal_title=bioRxiv; citation_doi=10.1101/250647; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma; citation_author=B. Fu; citation_author=W. Meng; citation_author=X. Zeng; citation_author=H. Zhao; citation_author=W. Liu; citation_author=T. Zhang; citation_journal_title=BioMed Res. Int.; citation_volume=2017; citation_firstpage=4698167; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications; citation_author=N. Gandhi; citation_author=G.M. Das; citation_journal_title=Cells; citation_volume=8; citation_firstpage=89; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma; citation_author=C. Gao; citation_author=J. Zhuang; citation_author=H. Li; citation_author=C. Liu; citation_author=C. Zhou; citation_author=L. Liu; citation_author=C. Sun; citation_journal_title=Cancer Cell Int.; citation_volume=18; citation_firstpage=194; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The ‘invisible hand’: regulation of RHO GTPases by RHOGDIs; citation_author=R. Garcia-Mata; citation_author=E. Boulter; citation_author=K. Burridge; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=12; citation_firstpage=493; citation_lastpage=504; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=A flexible R package for nonnegative matrix factorization; citation_author=R. Gaujoux; citation_author=C. Seoighe; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=367; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Using CART to detect multiple change points in the mean for large sample; citation_author=S. Gey; citation_author=E. Lebarbier; citation_journal_title=HAL; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer; citation_author=G. Giaccone; citation_author=P. Zatloukal; citation_author=J. Roubec; citation_author=K. Floor; citation_author=J. Musil; citation_author=M. Kuta; citation_author=R.J. van Klaveren; citation_author=S. Chaudhary; citation_author=A. Gunther; citation_author=S. Shamsili; citation_journal_title=J. Clin. Oncol.; citation_volume=27; citation_firstpage=4481; citation_lastpage=4486; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Proteomic identification of proteins conjugated to ISG15 in mouse and human cells; citation_author=N.V. Giannakopoulos; citation_author=J.-K. Luo; citation_author=V. Papov; citation_author=W. Zou; citation_author=D.J. Lenschow; citation_author=B.S. Jacobs; citation_author=E.C. Borden; citation_author=J. Li; citation_author=H.W. Virgin; citation_author=D.-E. Zhang; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=336; citation_firstpage=496; citation_lastpage=506; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Robust classification of Immune Subtypes in Cancer; citation_author=D.L. Gibbs; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.01.17.910950; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma; citation_author=M.A. Gillette; citation_author=S. Satpathy; citation_author=S. Cao; citation_author=S.M. Dhanasekaran; citation_author=S.V. Vasaikar; citation_author=K. Krug; citation_author=F. Petralia; citation_author=Y. Li; citation_author=W.-W. Liang; citation_author=B. Reva; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=182; citation_firstpage=200; citation_lastpage=225.e35; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity; citation_author=J.P. Gorvel; citation_author=T.C. Chang; citation_author=J. Boretto; citation_author=T. Azuma; citation_author=P. Chavrier; citation_journal_title=FEBS Lett.; citation_volume=422; citation_firstpage=269; citation_lastpage=273; citation_publication_date=1998; "></meta><meta name="citation_reference" content="citation_title=The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1; citation_author=P.A. Gregory; citation_author=A.G. Bert; citation_author=E.L. Paterson; citation_author=S.C. Barry; citation_author=A. Tsykin; citation_author=G. Farshid; citation_author=M.A. Vadas; citation_author=Y. Khew-Goodall; citation_author=G.J. Goodall; citation_journal_title=Nat. Cell Biol.; citation_volume=10; citation_firstpage=593; citation_lastpage=601; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Comprehensive genomic characterization of squamous cell lung cancers; citation_author=P.S. Hammerman; citation_author=M.S. Lawrence; citation_author=D. Voet; citation_author=R. Jing; citation_author=K. Cibulskis; citation_author=A. Sivachenko; citation_author=P. Stojanov; citation_author=A. McKenna; citation_author=E.S. Lander; citation_author=S. Gabriel; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Nature; citation_volume=489; citation_firstpage=519; citation_lastpage=525; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=IlluminaHumanMethylationEPICanno.ilm10b2.hg19 (Bioconductor); citation_author=K.D. Hansen; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=GSVA: gene set variation analysis for microarray and RNA-seq data; citation_author=S. Hänzelmann; citation_author=R. Castelo; citation_author=J. Guinney; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=7; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3; citation_author=N. Happel; citation_author=S. Stoldt; citation_author=B. Schmidt; citation_author=D. Doenecke; citation_journal_title=J. Mol. Biol.; citation_volume=386; citation_firstpage=339; citation_lastpage=350; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs; citation_author=A. Haslam; citation_author=V. Prasad; citation_journal_title=JAMA Netw. Open; citation_volume=2; citation_firstpage=e192535; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome; citation_author=A.S. Hebert; citation_author=K.E. Dittenhafer-Reed; citation_author=W. Yu; citation_author=D.J. Bailey; citation_author=E.S. Selen; citation_author=M.D. Boersma; citation_author=J.J. Carson; citation_author=M. Tonelli; citation_author=A.J. Balloon; citation_author=A.J. Higbee; citation_author=et al.; citation_journal_title=Mol. Cell; citation_volume=49; citation_firstpage=186; citation_lastpage=199; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=The biology and management of non-small cell lung cancer; citation_author=R.S. Herbst; citation_author=D. Morgensztern; citation_author=C. Boshoff; citation_journal_title=Nature; citation_volume=553; citation_firstpage=446; citation_lastpage=454; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter; citation_author=A.M. Herreño; citation_author=A.C. Ramírez; citation_author=V.P. Chaparro; citation_author=M.J. Fernandez; citation_author=A. Cañas; citation_author=C.F. Morantes; citation_author=O.M. Moreno; citation_author=R.E. Brugés; citation_author=J.A. Mejía; citation_author=F.J. Bustos; citation_author=et al.; citation_journal_title=Tumour Biol.; citation_volume=41; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Post-translational modifications and the Warburg effect; citation_author=T. Hitosugi; citation_author=J. Chen; citation_journal_title=Oncogene; citation_volume=33; citation_firstpage=4279; citation_lastpage=4285; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Upregulation of Wnt signaling under hypoxia promotes lung cancer progression; citation_author=C.-F. Hong; citation_author=W.-Y. Chen; citation_author=C.-W. Wu; citation_journal_title=Oncol. Rep.; citation_volume=38; citation_firstpage=1706; citation_lastpage=1714; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse; citation_author=P.V. Hornbeck; citation_author=J.M. Kornhauser; citation_author=S. Tkachev; citation_author=B. Zhang; citation_author=E. Skrzypek; citation_author=B. Murray; citation_author=V. Latham; citation_author=M. Sullivan; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=D261; citation_lastpage=D270; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1; citation_author=E.-C. Hsu; citation_author=M.A. Rice; citation_author=A. Bermudez; citation_author=F.J.G. Marques; citation_author=M. Aslan; citation_author=S. Liu; citation_author=A. Ghoochani; citation_author=C.A. Zhang; citation_author=Y.-S. Chen; citation_author=A. Zlitni; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=117; citation_firstpage=2032; citation_lastpage=2042; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database; citation_author=H.-Y. Huang; citation_author=Y.-C.-D. Lin; citation_author=J. Li; citation_author=K.-Y. Huang; citation_author=S. Shrestha; citation_author=H.-C. Hong; citation_author=Y. Tang; citation_author=Y.-G. Chen; citation_author=C.-N. Jin; citation_author=Y. Yu; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=48; citation_number=D1; citation_firstpage=D148; citation_lastpage=D154; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma; citation_author=C. Huang; citation_author=L. Chen; citation_author=S.R. Savage; citation_author=R.V. Eguez; citation_author=Y. Dou; citation_author=Y. Li; citation_author=F. da Veiga Leprevost; citation_author=E.J. Jaehnig; citation_author=J.T. Lei; citation_author=B. Wen; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=361; citation_lastpage=379.e16; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer; citation_author=R. Hubaux; citation_author=K.L. Thu; citation_author=E.A. Vucic; citation_author=L.A. Pikor; citation_author=S.H.Y. Kung; citation_author=V.D. Martinez; citation_author=M. Mosslemi; citation_author=D.D. Becker-Santos; citation_author=A.F. Gazdar; citation_author=S. Lam; citation_author=W.L. Lam; citation_journal_title=Int. J. Cancer; citation_volume=137; citation_firstpage=2072; citation_lastpage=2082; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis; citation_author=Y.P. Hung; citation_author=A. Redig; citation_author=J.L. Hornick; citation_author=L.M. Sholl; citation_journal_title=Mod. Pathol.; citation_volume=33; citation_firstpage=2256; citation_lastpage=2268; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex; citation_author=T. Ishitani; citation_author=T. Hirao; citation_author=M. Suzuki; citation_author=M. Isoda; citation_author=S. Ishitani; citation_author=K. Harigaya; citation_author=M. Kitagawa; citation_author=K. Matsumoto; citation_author=M. Itoh; citation_journal_title=Nat. Cell Biol.; citation_volume=12; citation_firstpage=278; citation_lastpage=285; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway; citation_author=D. Jimenez-Blasco; citation_author=P. Santofimia-Castaño; citation_author=A. Gonzalez; citation_author=A. Almeida; citation_author=J.P. Bolaños; citation_journal_title=Cell Death Differ.; citation_volume=22; citation_firstpage=1877; citation_lastpage=1889; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta; citation_author=Y.H. Jin; citation_author=H. Kim; citation_author=M. Oh; citation_author=H. Ki; citation_author=K. Kim; citation_journal_title=Mol. Cells; citation_volume=27; citation_firstpage=15; citation_lastpage=19; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Adjusting batch effects in microarray expression data using empirical Bayes methods; citation_author=W.E. Johnson; citation_author=C. Li; citation_author=A. Rabinovic; citation_journal_title=Biostatistics; citation_volume=8; citation_firstpage=118; citation_lastpage=127; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data; citation_author=V. Jurtz; citation_author=S. Paul; citation_author=M. Andreatta; citation_author=P. Marcatili; citation_author=B. Peters; citation_author=M. Nielsen; citation_journal_title=J. Immunol.; citation_volume=199; citation_firstpage=3360; citation_lastpage=3368; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Comprehensive assessment of cancer missense mutation clustering in protein structures; citation_author=A. Kamburov; citation_author=M.S. Lawrence; citation_author=P. Polak; citation_author=I. Leshchiner; citation_author=K. Lage; citation_author=T.R. Golub; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E5486; citation_lastpage=E5495; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=KEGG: new perspectives on genomes, pathways, diseases and drugs; citation_author=M. Kanehisa; citation_author=M. Furumichi; citation_author=M. Tanabe; citation_author=Y. Sato; citation_author=K. Morishima; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D353; citation_lastpage=D361; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer?; citation_author=N. Karachaliou; citation_author=M. Fernandez-Bruno; citation_author=R. Rosell; citation_journal_title=Transl. Lung Cancer Res.; citation_volume=7; citation_number=Suppl 3; citation_firstpage=S198; citation_lastpage=S201; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer; citation_author=R.J. Kelly; citation_author=A. Thomas; citation_author=A. Rajan; citation_author=G. Chun; citation_author=A. Lopez-Chavez; citation_author=E. Szabo; citation_author=S. Spencer; citation_author=C.A. Carter; citation_author=U. Guha; citation_author=S. Khozin; citation_author=et al.; citation_journal_title=Ann. Oncol.; citation_volume=24; citation_firstpage=2601; citation_lastpage=2606; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2; citation_author=S.B. Keysar; citation_author=P.N. Le; citation_author=B. Miller; citation_author=B.C. Jackson; citation_author=J.R. Eagles; citation_author=C. Nieto; citation_author=J. Kim; citation_author=B. Tang; citation_author=M.J. Glogowska; citation_author=J.J. Morton; citation_author=et al.; citation_journal_title=J. Natl. Cancer Inst.; citation_volume=109; citation_firstpage=djw189; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers; citation_author=T. Kikuchi; citation_author=M. Hassanein; citation_author=J.M. Amann; citation_author=Q. Liu; citation_author=R.J.C. Slebos; citation_author=S.M.J. Rahman; citation_author=J.M. Kaufman; citation_author=X. Zhang; citation_author=M.D. Hoeksema; citation_author=B.K. Harris; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=11; citation_firstpage=916; citation_lastpage=932; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis; citation_author=H. Kim; citation_author=H. Park; citation_journal_title=Bioinformatics; citation_volume=23; citation_firstpage=1495; citation_lastpage=1502; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors; citation_author=J. Kim; citation_author=K.W. Mouw; citation_author=P. Polak; citation_author=L.Z. Braunstein; citation_author=A. Kamburov; citation_author=D.J. Kwiatkowski; citation_author=J.E. Rosenberg; citation_author=E.M. Van Allen; citation_author=A. D’Andrea; citation_author=G. Getz; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=600; citation_lastpage=606; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade; citation_author=V. Klepsch; citation_author=N. Hermann-Kleiter; citation_author=P. Do-Dinh; citation_author=B. Jakic; citation_author=A. Offermann; citation_author=M. Efremova; citation_author=S. Sopper; citation_author=D. Rieder; citation_author=A. Krogsdam; citation_author=G. Gamerith; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=1538; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Pan-cancer molecular analysis of the RB tumor suppressor pathway; citation_author=E.S. Knudsen; citation_author=R. Nambiar; citation_author=S.R. Rosario; citation_author=D.J. Smiraglia; citation_author=D.W. Goodrich; citation_author=A.K. Witkiewicz; citation_journal_title=Commun. Biol; citation_volume=3; citation_firstpage=158; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Causal analysis approaches in Ingenuity Pathway Analysis; citation_author=A. Krämer; citation_author=J. Green; citation_author=J. Pollard; citation_author=S. Tugendreich; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=523; citation_lastpage=530; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells; citation_author=C. Kröger; citation_author=A. Afeyan; citation_author=J. Mraz; citation_author=E.N. Eaton; citation_author=F. Reinhardt; citation_author=Y.L. Khodor; citation_author=P. Thiru; citation_author=B. Bierie; citation_author=X. Ye; citation_author=C.B. Burge; citation_author=R.A. Weinberg; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=7353; citation_lastpage=7362; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=A curated resource for phosphosite-specific signature analysis; citation_author=K. Krug; citation_author=P. Mertins; citation_author=B. Zhang; citation_author=P. Hornbeck; citation_author=R. Raju; citation_author=R. Ahmad; citation_author=M. Szucs; citation_author=F. Mundt; citation_author=D. Forestier; citation_author=J. Jane-Valbuena; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=576; citation_lastpage=593; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy; citation_author=K. Krug; citation_author=E.J. Jaehnig; citation_author=S. Satpathy; citation_author=L. Blumenberg; citation_author=A. Karpova; citation_author=M. Anurag; citation_author=G. Miles; citation_author=P. Mertins; citation_author=Y. Geffen; citation_author=L.C. Tang; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=183; citation_firstpage=1436; citation_lastpage=1456.e31; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation; citation_author=N. Kuhlmann; citation_author=S. Wroblowski; citation_author=P. Knyphausen; citation_author=S. de Boor; citation_author=J. Brenig; citation_author=A.Y. Zienert; citation_author=K. Meyer-Teschendorf; citation_author=G.J.K. Praefcke; citation_author=H. Nolte; citation_author=M. Krüger; citation_author=et al.; citation_journal_title=J. Biol. Chem.; citation_volume=291; citation_firstpage=5484; citation_lastpage=5499; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA; citation_author=N. Kuhlmann; citation_author=S. Wroblowski; citation_author=L. Scislowski; citation_author=M. Lammers; citation_journal_title=Biochemistry; citation_volume=55; citation_firstpage=304; citation_lastpage=312; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Gene regulatory effects of disease-associated variation in the NRF2 network; citation_author=S.E. Lacher; citation_author=M. Slattery; citation_journal_title=Curr. Opin. Toxicol.; citation_volume=1; citation_firstpage=71; citation_lastpage=79; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease; citation_author=J. Lamb; citation_author=E.D. Crawford; citation_author=D. Peck; citation_author=J.W. Modell; citation_author=I.C. Blat; citation_author=M.J. Wrobel; citation_author=J. Lerner; citation_author=J.-P. Brunet; citation_author=A. Subramanian; citation_author=K.N. Ross; citation_author=et al.; citation_journal_title=Science; citation_volume=313; citation_firstpage=1929; citation_lastpage=1935; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer; citation_author=S. Lamprecht; citation_author=M. Kaller; citation_author=E.M. Schmidt; citation_author=C. Blaj; citation_author=T.S. Schiergens; citation_author=J. Engel; citation_author=A. Jung; citation_author=H. Hermeking; citation_author=T.G.P. Grünewald; citation_author=T. Kirchner; citation_author=D. Horst; citation_journal_title=Clin. Cancer Res.; citation_volume=24; citation_firstpage=1974; citation_lastpage=1986; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Discovery and saturation analysis of cancer genes across 21 tumour types; citation_author=M.S. Lawrence; citation_author=P. Stojanov; citation_author=C.H. Mermel; citation_author=J.T. Robinson; citation_author=L.A. Garraway; citation_author=T.R. Golub; citation_author=M. Meyerson; citation_author=S.B. Gabriel; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nature; citation_volume=505; citation_firstpage=495; citation_lastpage=501; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=FactoMineR: AnRPackage for Multivariate Analysis; citation_author=S. Lê; citation_author=J. Josse; citation_author=F. Husson; citation_journal_title=J. Stat. Softw.; citation_doi=10.18637/jss.v025.i01; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets; citation_author=T.Y. Le Large; citation_author=G. Mantini; citation_author=L.L. Meijer; citation_author=T.V. Pham; citation_author=N. Funel; citation_author=N.C. van Grieken; citation_author=B. Kok; citation_author=J. Knol; citation_author=H.W. van Laarhoven; citation_author=S.R. Piersma; citation_author=et al.; citation_journal_title=JCI Insight; citation_volume=5; citation_firstpage=e138290; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma; citation_author=D. Lebanony; citation_author=H. Benjamin; citation_author=S. Gilad; citation_author=M. Ezagouri; citation_author=A. Dov; citation_author=K. Ashkenazi; citation_author=N. Gefen; citation_author=S. Izraeli; citation_author=G. Rechavi; citation_author=H. Pass; citation_author=et al.; citation_journal_title=J. Clin. Oncol.; citation_volume=27; citation_firstpage=2030; citation_lastpage=2037; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Detecting multiple change-points in the mean of Gaussian process by model selection; citation_author=E. Lebarbier; citation_journal_title=Signal Processing; citation_volume=85; citation_firstpage=717; citation_lastpage=736; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity; citation_author=K.T. Lewandowski; citation_author=H. Piwnica-Worms; citation_journal_title=J. Cell Sci.; citation_volume=127; citation_firstpage=315; citation_lastpage=327; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Quantifying circular RNA expression from RNA-seq data using model-based framework; citation_author=M. Li; citation_author=X. Xie; citation_author=J. Zhou; citation_author=M. Sheng; citation_author=X. Yin; citation_author=E.-A. Ko; citation_author=T. Zhou; citation_author=W. Gu; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=2131; citation_lastpage=2139; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study?; citation_author=F. Li; citation_author=I. Aljahdali; citation_author=X. Ling; citation_journal_title=J. Exp. Clin. Cancer Res.; citation_volume=38; citation_firstpage=368; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=PDV: an integrative proteomics data viewer; citation_author=K. Li; citation_author=M. Vaudel; citation_author=B. Zhang; citation_author=Y. Ren; citation_author=B. Wen; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=1249; citation_lastpage=1251; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors; citation_author=P. Li; citation_author=G. Cao; citation_author=Y. Zhang; citation_author=J. Shi; citation_author=K. Cai; citation_author=L. Zhen; citation_author=X. He; citation_author=Y. Zhou; citation_author=Y. Li; citation_author=Y. Zhu; citation_author=et al.; citation_journal_title=Aging (Albany N.Y.); citation_volume=12; citation_firstpage=53; citation_lastpage=69; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer; citation_author=Y. Liang; citation_author=M. Liu; citation_author=P. Wang; citation_author=X. Ding; citation_author=Y. Cao; citation_journal_title=Genes Chromosomes Cancer; citation_volume=52; citation_firstpage=305; citation_lastpage=315; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs; citation_author=Y. Liao; citation_author=J. Wang; citation_author=E.J. Jaehnig; citation_author=Z. Shi; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=W1; citation_firstpage=W199; citation_lastpage=W205; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst.; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study; citation_author=S.H. Lim; citation_author=J.-M. Sun; citation_author=Y.-L. Choi; citation_author=H.R. Kim; citation_author=S. Ahn; citation_author=J.Y. Lee; citation_author=S.-H. Lee; citation_author=J.S. Ahn; citation_author=K. Park; citation_author=J.H. Kim; citation_author=et al.; citation_journal_title=Cancer; citation_volume=122; citation_firstpage=3024; citation_lastpage=3031; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration; citation_author=Y.W. Lim; citation_author=H. Chen-Harris; citation_author=O. Mayba; citation_author=S. Lianoglou; citation_author=A. Wuster; citation_author=T. Bhangale; citation_author=Z. Khan; citation_author=S. Mariathasan; citation_author=A. Daemen; citation_author=J. Reeder; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E11701; citation_lastpage=E11710; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Role of DeltaNp63gamma in epithelial to mesenchymal transition; citation_author=J. Lindsay; citation_author=S.S. McDade; citation_author=A. Pickard; citation_author=K.D. McCloskey; citation_author=D.J. McCance; citation_journal_title=J. Biol. Chem.; citation_volume=286; citation_firstpage=3915; citation_lastpage=3924; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells; citation_author=X. Liu; citation_author=L. Wang; citation_author=W. Cui; citation_author=X. Yuan; citation_author=L. Lin; citation_author=Q. Cao; citation_author=N. Wang; citation_author=Y. Li; citation_author=W. Guo; citation_author=X. Zhang; citation_author=et al.; citation_journal_title=Oncotarget; citation_volume=7; citation_firstpage=58516; citation_lastpage=58530; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis; citation_author=W. Liu; citation_author=S.H. Payne; citation_author=S. Ma; citation_author=D. Fenyö; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_number=8, suppl 1; citation_firstpage=S169; citation_lastpage=S182; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue; citation_author=M. Lo Iacono; citation_author=V. Monica; citation_author=S. Saviozzi; citation_author=P. Ceppi; citation_author=E. Bracco; citation_author=M. Papotti; citation_author=G.V. Scagliotti; citation_journal_title=J. Thorac. Oncol.; citation_volume=6; citation_firstpage=473; citation_lastpage=481; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=A cross-package Bioconductor workflow for analysing methylation array data; citation_author=J. Maksimovic; citation_author=B. Phipson; citation_author=A. Oshlack; citation_journal_title=F1000Res.; citation_volume=5; citation_firstpage=1281; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis; citation_author=D. Malhotra; citation_author=E. Portales-Casamar; citation_author=A. Singh; citation_author=S. Srivastava; citation_author=D. Arenillas; citation_author=C. Happel; citation_author=C. Shyr; citation_author=N. Wakabayashi; citation_author=T.W. Kensler; citation_author=W.W. Wasserman; citation_author=S. Biswal; citation_journal_title=Nucleic Acids Res.; citation_volume=38; citation_firstpage=5718; citation_lastpage=5734; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis; citation_author=D.R. Mani; citation_author=M. Maynard; citation_author=R. Kothadia; citation_author=K. Krug; citation_author=K.E. Christianson; citation_author=D. Heiman; citation_author=K.R. Clauser; citation_author=C. Birger; citation_author=G. Getz; citation_author=S.A. Carr; citation_journal_title=Nature; citation_volume=18; citation_firstpage=580; citation_lastpage=582; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=PANOPLY: a cloud-based platform for automated and reproducible proteogenomic data analysis; citation_author=D.R. Mani; citation_author=M. Maynard; citation_author=R. Kothadia; citation_author=K. Krug; citation_author=K.E. Christianson; citation_author=D. Heiman; citation_author=K.R. Clauser; citation_author=C. Birger; citation_author=G. Getz; citation_author=S.A. Carr; citation_journal_title=Nat. Methods; citation_volume=18; citation_firstpage=580; citation_lastpage=582; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=hdbscan: Hierarchical density based clustering; citation_author=L. McInnes; citation_author=J. Healy; citation_author=S. Astels; citation_journal_title=Journal of Open Source Software; citation_volume=2; citation_firstpage=205; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The Ensembl Variant Effect Predictor; citation_author=W. McLaren; citation_author=L. Gil; citation_author=S.E. Hunt; citation_author=H.S. Riat; citation_author=G.R.S. Ritchie; citation_author=A. Thormann; citation_author=P. Flicek; citation_author=F. Cunningham; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=122; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation; citation_author=B. Medvar; citation_author=V. Raghuram; citation_author=T. Pisitkun; citation_author=A. Sarkar; citation_author=M.A. Knepper; citation_journal_title=Physiol. Genomics; citation_volume=48; citation_firstpage=502; citation_lastpage=512; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Enrichment map: a network-based method for gene-set enrichment visualization and interpretation; citation_author=D. Merico; citation_author=R. Isserlin; citation_author=O. Stueker; citation_author=A. Emili; citation_author=G.D. Bader; citation_journal_title=PLoS ONE; citation_volume=5; citation_firstpage=e13984; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry; citation_author=P. Mertins; citation_author=L.C. Tang; citation_author=K. Krug; citation_author=D.J. Clark; citation_author=M.A. Gritsenko; citation_author=L. Chen; citation_author=K.R. Clauser; citation_author=T.R. Clauss; citation_author=P. Shah; citation_author=M.A. Gillette; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=13; citation_firstpage=1632; citation_lastpage=1661; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The National Lung Matrix Trial of personalized therapy in lung cancer; citation_author=G. Middleton; citation_author=P. Fletcher; citation_author=S. Popat; citation_author=J. Savage; citation_author=Y. Summers; citation_author=A. Greystoke; citation_author=D. Gilligan; citation_author=J. Cave; citation_author=N. O’Rourke; citation_author=A. Brewster; citation_author=et al.; citation_journal_title=Nature; citation_volume=583; citation_firstpage=807; citation_lastpage=812; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases; citation_author=K. Moissoglu; citation_author=K.S. McRoberts; citation_author=J.A. Meier; citation_author=D. Theodorescu; citation_author=M.A. Schwartz; citation_journal_title=Cancer Res.; citation_volume=69; citation_firstpage=2838; citation_lastpage=2844; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer; citation_author=J.A. Morrison; citation_author=L.A. Pike; citation_author=S.B. Sams; citation_author=V. Sharma; citation_author=Q. Zhou; citation_author=J.J. Severson; citation_author=A.-C. Tan; citation_author=W.M. Wood; citation_author=B.R. Haugen; citation_journal_title=Mol. Cancer; citation_volume=13; citation_firstpage=62; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity; citation_author=K.A. Mouchemore; citation_author=N.G. Sampaio; citation_author=M.W. Murrey; citation_author=E.R. Stanley; citation_author=B.J. Lannutti; citation_author=F.J. Pixley; citation_journal_title=FEBS J.; citation_volume=280; citation_firstpage=5228; citation_lastpage=5236; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Evaluation of Advanced Precursor Determination for Tandem Mass Tag (TMT)-Based Quantitative Proteomics across Instrument Platforms; citation_author=S.A. Myers; citation_author=S. Klaeger; citation_author=S. Satpathy; citation_author=R. Viner; citation_author=J. Choi; citation_author=J. Rogers; citation_author=K. Clauser; citation_author=N.D. Udeshi; citation_author=S.A. Carr; citation_journal_title=J. Proteome Res.; citation_volume=18; citation_firstpage=542; citation_lastpage=547; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Structure and kinetic stability of the p63 tetramerization domain; citation_author=E. Natan; citation_author=A.C. Joerger; citation_journal_title=J. Mol. Biol.; citation_volume=415; citation_firstpage=503; citation_lastpage=513; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Robust enumeration of cell subsets from tissue expression profiles; citation_author=A.M. Newman; citation_author=C.L. Liu; citation_author=M.R. Green; citation_author=A.J. Gentles; citation_author=W. Feng; citation_author=Y. Xu; citation_author=C.D. Hoang; citation_author=M. Diehn; citation_author=A.A. Alizadeh; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=453; citation_lastpage=457; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data; citation_author=D. Nicorici; citation_author=M. Satalan; citation_author=H. Edgren; citation_author=S. Kangaspeska; citation_author=A. Murumagi; citation_author=O. Kallioniemi; citation_author=S. Virtanen; citation_author=O. Kilkku; citation_journal_title=bioRxiv; citation_doi=10.1101/011650; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=A genomic and functional inventory of deubiquitinating enzymes; citation_author=S.M.B. Nijman; citation_author=M.P.A. Luna-Vargas; citation_author=A. Velds; citation_author=T.R. Brummelkamp; citation_author=A.M.G. Dirac; citation_author=T.K. Sixma; citation_author=R. Bernards; citation_journal_title=Cell; citation_volume=123; citation_firstpage=773; citation_lastpage=786; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas; citation_author=D.-F. Niu; citation_author=T. Kondo; citation_author=T. Nakazawa; citation_author=N. Oishi; citation_author=T. Kawasaki; citation_author=K. Mochizuki; citation_author=T. Yamane; citation_author=R. Katoh; citation_journal_title=Lab. Invest.; citation_volume=92; citation_firstpage=1181; citation_lastpage=1190; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants; citation_author=V. Obenchain; citation_author=M. Lawrence; citation_author=V. Carey; citation_author=S. Gogarten; citation_author=P. Shannon; citation_author=M. Morgan; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=2076; citation_lastpage=2078; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=New Treatment Options in Advanced Squamous Cell Lung Cancer; citation_author=P.K. Paik; citation_author=R.N. Pillai; citation_author=C.S. Lathan; citation_author=S.A. Velasco; citation_author=V. Papadimitrakopoulou; citation_journal_title=Am. Soc. Clin. Oncol. Educ. Book; citation_volume=39; citation_firstpage=e198; citation_lastpage=e206; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies; citation_author=E.R. Parra; citation_author=M. Jiang; citation_author=L. Solis; citation_author=B. Mino; citation_author=C. Laberiano; citation_author=S. Hernandez; citation_author=S. Gite; citation_author=A. Verma; citation_author=M. Tetzlaff; citation_author=C. Haymaker; citation_author=et al.; citation_journal_title=Cancers (Basel); citation_volume=12; citation_firstpage=255; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors; citation_author=M. Patel; citation_author=L. Lu; citation_author=D.S. Zander; citation_author=L. Sreerama; citation_author=D. Coco; citation_author=J.S. Moreb; citation_journal_title=Lung Cancer; citation_volume=59; citation_firstpage=340; citation_lastpage=349; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors; citation_author=A. Patnaik; citation_author=L.S. Rosen; citation_author=S.M. Tolaney; citation_author=A.W. Tolcher; citation_author=J.W. Goldman; citation_author=L. Gandhi; citation_author=K.P. Papadopoulos; citation_author=M. Beeram; citation_author=D.W. Rasco; citation_author=J.F. Hilton; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=6; citation_firstpage=740; citation_lastpage=753; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Monocarboxylate transporters in cancer; citation_author=V.L. Payen; citation_author=E. Mina; citation_author=V.F. Van Hée; citation_author=P.E. Porporato; citation_author=P. Sonveaux; citation_journal_title=Mol. Metab.; citation_volume=33; citation_firstpage=48; citation_lastpage=66; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer; citation_author=L. Paz-Ares; citation_author=A. Luft; citation_author=D. Vicente; citation_author=A. Tafreshi; citation_author=M. Gümüş; citation_author=J. Mazières; citation_author=B. Hermes; citation_author=F. Çay Şenler; citation_author=T. Csőszi; citation_author=A. Fülöp; citation_author=KEYNOTE-407 Investigators; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=379; citation_firstpage=2040; citation_lastpage=2051; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Vcfanno: fast, flexible annotation of genetic variants; citation_author=B.S. Pedersen; citation_author=R.M. Layer; citation_author=A.R. Quinlan; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=118; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology; citation_author=A. Peng; citation_author=X. Mao; citation_author=J. Zhong; citation_author=S. Fan; citation_author=Y. Hu; citation_journal_title=Proteomics; citation_volume=20; citation_firstpage=e1900271; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=A functional genomics predictive network model identifies regulators of inflammatory bowel disease; citation_author=L.A. Peters; citation_author=J. Perrigoue; citation_author=A. Mortha; citation_author=A. Iuga; citation_author=W.-M. Song; citation_author=E.M. Neiman; citation_author=S.R. Llewellyn; citation_author=A. Di Narzo; citation_author=B.A. Kidd; citation_author=S.E. Telesco; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1437; citation_lastpage=1449; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity; citation_author=F. Petralia; citation_author=L. Wang; citation_author=J. Peng; citation_author=A. Yan; citation_author=J. Zhu; citation_author=P. Wang; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=i528; citation_lastpage=i536; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S. Purcell; citation_author=B. Neale; citation_author=K. Todd-Brown; citation_author=L. Thomas; citation_author=M.A.R. Ferreira; citation_author=D. Bender; citation_author=J. Maller; citation_author=P. Sklar; citation_author=P.I.W. de Bakker; citation_author=M.J. Daly; citation_author=P.C. Sham; citation_journal_title=Am. J. Hum. Genet.; citation_volume=81; citation_firstpage=559; citation_lastpage=575; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Role of chromosome 3q amplification in lung cancer; citation_author=J. Qian; citation_author=P.P. Massion; citation_journal_title=J. Thorac. Oncol.; citation_volume=3; citation_firstpage=212; citation_lastpage=215; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance; citation_author=S. Ramadoss; citation_author=S. Sen; citation_author=I. Ramachandran; citation_author=S. Roy; citation_author=G. Chaudhuri; citation_author=R. Farias-Eisner; citation_journal_title=Oncogene; citation_volume=36; citation_firstpage=6508; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update); citation_author=U. Raudvere; citation_author=L. Kolberg; citation_author=I. Kuzmin; citation_author=T. Arak; citation_author=P. Adler; citation_author=H. Peterson; citation_author=J. Vilo; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=W1; citation_firstpage=W191; citation_lastpage=W198; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis; citation_author=S.E. Reese; citation_author=K.J. Archer; citation_author=T.M. Therneau; citation_author=E.J. Atkinson; citation_author=C.M. Vachon; citation_author=M. de Andrade; citation_author=J.-P.A. Kocher; citation_author=J.E. Eckel-Passow; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=2877; citation_lastpage=2883; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-g axis and identification of immune resistance mechanisms; citation_author=B. Reva; citation_author=T. Omelchenko; citation_author=A. Calinawan; citation_author=S. Nair; citation_author=E. Schadt; citation_author=A. Tewari; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.10.19.345629; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=M.E. Ritchie; citation_author=B. Phipson; citation_author=D. Wu; citation_author=Y. Hu; citation_author=C.W. Law; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=e47; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Integrative clinical genomics of metastatic cancer; citation_author=D.R. Robinson; citation_author=Y.-M. Wu; citation_author=R.J. Lonigro; citation_author=P. Vats; citation_author=E. Cobain; citation_author=J. Everett; citation_author=X. Cao; citation_author=E. Rabban; citation_author=C. Kumar-Sinha; citation_author=V. Raymond; citation_author=et al.; citation_journal_title=Nature; citation_volume=548; citation_firstpage=297; citation_lastpage=303; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders; citation_author=R. Roskoski; citation_journal_title=Pharmacol. Res.; citation_volume=129; citation_firstpage=65; citation_lastpage=83; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Microscaled proteogenomic methods for precision oncology; citation_author=S. Satpathy; citation_author=E.J. Jaehnig; citation_author=K. Krug; citation_author=B.-J. Kim; citation_author=A.B. Saltzman; citation_author=D.W. Chan; citation_author=K.R. Holloway; citation_author=M. Anurag; citation_author=C. Huang; citation_author=P. Singh; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=532; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Perturbation-response genes reveal signaling footprints in cancer gene expression; citation_author=M. Schubert; citation_author=B. Klinger; citation_author=M. Klünemann; citation_author=A. Sieber; citation_author=F. Uhlitz; citation_author=S. Sauer; citation_author=M.J. Garnett; citation_author=N. Blüthgen; citation_author=J. Saez-Rodriguez; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=20; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models; citation_author=L. Scrucca; citation_author=M. Fop; citation_author=T.B. Murphy; citation_author=A.E. Raftery; citation_journal_title=The R Journal; citation_volume=8; citation_firstpage=289; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=i-Tracker: for quantitative proteomics using iTRAQ; citation_author=I.P. Shadforth; citation_author=T.P.J. Dunkley; citation_author=K.S. Lilley; citation_author=C. Bessant; citation_journal_title=BMC Genomics; citation_volume=6; citation_firstpage=145; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Germline Features Associated with Immune Infiltration in Solid Tumors; citation_author=S. Shahamatdar; citation_author=M.X. He; citation_author=M.A. Reyna; citation_author=A. Gusev; citation_author=S.H. AlDubayan; citation_author=E.M. Van Allen; citation_author=S. Ramachandran; citation_journal_title=Cell Rep.; citation_volume=30; citation_firstpage=2900; citation_lastpage=2908.e4; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; citation_author=P. Shannon; citation_author=A. Markiel; citation_author=O. Ozier; citation_author=N.S. Baliga; citation_author=J.T. Wang; citation_author=D. Ramage; citation_author=N. Amin; citation_author=B. Schwikowski; citation_author=T. Ideker; citation_journal_title=Genome Res.; citation_volume=13; citation_firstpage=2498; citation_lastpage=2504; citation_publication_date=2003; "></meta><meta name="citation_reference" content="citation_title=ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade; citation_author=J. Shen; citation_author=Z. Ju; citation_author=W. Zhao; citation_author=L. Wang; citation_author=Y. Peng; citation_author=Z. Ge; citation_author=Z.D. Nagel; citation_author=J. Zou; citation_author=C. Wang; citation_author=P. Kapoor; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=24; citation_firstpage=556; citation_lastpage=562; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease; citation_author=M.K. Showe; citation_author=A. Vachani; citation_author=A.V. Kossenkov; citation_author=M. Yousef; citation_author=C. Nichols; citation_author=E.V. Nikonova; citation_author=C. Chang; citation_author=J. Kucharczuk; citation_author=B. Tran; citation_author=E. Wakeam; citation_author=et al.; citation_journal_title=Cancer Res.; citation_volume=69; citation_firstpage=9202; citation_lastpage=9210; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Cancer statistics, 2020; citation_author=R.L. Siegel; citation_author=K.D. Miller; citation_author=A. Jemal; citation_journal_title=CA Cancer J. Clin.; citation_volume=70; citation_firstpage=7; citation_lastpage=30; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy; citation_author=A. Singh; citation_author=S. Boldin-Adamsky; citation_author=R.K. Thimmulappa; citation_author=S.K. Rath; citation_author=H. Ashush; citation_author=J. Coulter; citation_author=A. Blackford; citation_author=S.N. Goodman; citation_author=F. Bunz; citation_author=W.H. Watson; citation_author=et al.; citation_journal_title=Cancer Res.; citation_volume=68; citation_firstpage=7975; citation_lastpage=7984; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC; citation_author=S.L. Sinicropi-Yao; citation_author=J.M. Amann; citation_author=D.L.Y. Lopez; citation_author=F. Cerciello; citation_author=K.R. Coombes; citation_author=D.P. Carbone; citation_journal_title=J. Thorac. Oncol.; citation_volume=14; citation_firstpage=223; citation_lastpage=236; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers; citation_author=Z. Sondka; citation_author=S. Bamford; citation_author=C.G. Cole; citation_author=S.A. Ward; citation_author=I. Dunham; citation_author=S.A. Forbes; citation_journal_title=Nat. Rev. Cancer; citation_volume=18; citation_firstpage=696; citation_lastpage=705; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis; citation_author=X. Song; citation_author=J. Ji; citation_author=K.J. Gleason; citation_author=F. Yang; citation_author=J.A. Martignetti; citation_author=L.S. Chen; citation_author=P. Wang; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_number=8, suppl 1; citation_firstpage=S52; citation_lastpage=S65; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Mass Cytometry: Single Cells, Many Features; citation_author=M.H. Spitzer; citation_author=G.P. Nolan; citation_journal_title=Cell; citation_volume=165; citation_firstpage=780; citation_lastpage=791; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers; citation_author=K. Srivastava; citation_author=A. Pickard; citation_author=S.G. Craig; citation_author=G.P. Quinn; citation_author=S.M. Lambe; citation_author=J.A. James; citation_author=S.S. McDade; citation_author=D.J. McCance; citation_journal_title=Clin. Cancer Res.; citation_volume=24; citation_firstpage=3917; citation_lastpage=3927; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic landscape of squamous cell lung cancer; citation_author=P.A. Stewart; citation_author=E.A. Welsh; citation_author=R.J.C. Slebos; citation_author=B. Fang; citation_author=V. Izumi; citation_author=M. Chambers; citation_author=G. Zhang; citation_author=L. Cen; citation_author=F. Pettersson; citation_author=Y. Zhang; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=3578; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1; citation_author=J. Su; citation_author=S.M. Morgani; citation_author=C.J. David; citation_author=Q. Wang; citation_author=E.E. Er; citation_author=Y.-H. Huang; citation_author=H. Basnet; citation_author=Y. Zou; citation_author=W. Shu; citation_author=R.K. Soni; citation_author=et al.; citation_journal_title=Nature; citation_volume=577; citation_firstpage=566; citation_lastpage=571; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles; citation_author=A. Subramanian; citation_author=R. Narayan; citation_author=S.M. Corsello; citation_author=D.D. Peck; citation_author=T.E. Natoli; citation_author=X. Lu; citation_author=J. Gould; citation_author=J.F. Davis; citation_author=A.A. Tubelli; citation_author=J.K. Asiedu; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1437; citation_lastpage=1452.e17; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=OptiType: precision HLA typing from next-generation sequencing data; citation_author=A. Szolek; citation_author=B. Schubert; citation_author=C. Mohr; citation_author=M. Sturm; citation_author=M. Feldhahn; citation_author=O. Kohlbacher; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=3310; citation_lastpage=3316; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells; citation_author=W.L. Tam; citation_author=H. Lu; citation_author=J. Buikhuisen; citation_author=B.S. Soh; citation_author=E. Lim; citation_author=F. Reinhardt; citation_author=Z.J. Wu; citation_author=J.A. Krall; citation_author=B. Bierie; citation_author=W. Guo; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=24; citation_firstpage=347; citation_lastpage=364; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=COSMIC: the Catalogue Of Somatic Mutations In Cancer; citation_author=J.G. Tate; citation_author=S. Bamford; citation_author=H.C. Jubb; citation_author=Z. Sondka; citation_author=D.M. Beare; citation_author=N. Bindal; citation_author=H. Boutselakis; citation_author=C.G. Cole; citation_author=C. Creatore; citation_author=E. Dawson; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D941; citation_lastpage=D947; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells; citation_author=A.C. Tecalco-Cruz; citation_author=E. Cruz-Ramos; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=499; citation_firstpage=973; citation_lastpage=978; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells; citation_author=C.S. Tellez; citation_author=D.E. Juri; citation_author=K. Do; citation_author=A.M. Bernauer; citation_author=C.L. Thomas; citation_author=L.A. Damiani; citation_author=M. Tessema; citation_author=S. Leng; citation_author=S.A. Belinsky; citation_journal_title=Cancer Res.; citation_volume=71; citation_firstpage=3087; citation_lastpage=3097; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=The Immune Landscape of Cancer; citation_author=V. Thorsson; citation_author=D.L. Gibbs; citation_author=S.D. Brown; citation_author=D. Wolf; citation_author=D.S. Bortone; citation_author=T.-H. Ou Yang; citation_author=E. Porta-Pardo; citation_author=G.F. Gao; citation_author=C.L. Plaisier; citation_author=J.A. Eddy; citation_author=Cancer Genome Atlas Research Network; citation_author=et al.; citation_journal_title=Immunity; citation_volume=48; citation_firstpage=812; citation_lastpage=830.e14; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer; citation_author=J. Tian; citation_author=Y. Cai; citation_author=Y. Li; citation_author=Z. Lu; citation_author=J. Huang; citation_author=Y. Deng; citation_author=N. Yang; citation_author=X. Wang; citation_author=P. Ying; citation_author=S. Zhang; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=49; citation_number=D1; citation_firstpage=D1065; citation_lastpage=D1073; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=IL-18 receptor marks functional CD8+ T cells in non-small cell lung cancer; citation_author=E. Timperi; citation_author=C. Focaccetti; citation_author=D. Gallerano; citation_author=M. Panetta; citation_author=S. Spada; citation_author=E. Gallo; citation_author=P. Visca; citation_author=F. Venuta; citation_author=D. Diso; citation_author=A. Prelaj; citation_author=et al.; citation_journal_title=OncoImmunology; citation_volume=6; citation_firstpage=e1328337; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205; citation_author=M.N. Tran; citation_author=W. Choi; citation_author=M.F. Wszolek; citation_author=N. Navai; citation_author=I.-L.C. Lee; citation_author=G. Nitti; citation_author=S. Wen; citation_author=E.R. Flores; citation_author=A. Siefker-Radtke; citation_author=B. Czerniak; citation_author=et al.; citation_journal_title=J. Biol. Chem.; citation_volume=288; citation_firstpage=3275; citation_lastpage=3288; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Defining a Cancer Dependency Map; citation_author=A. Tsherniak; citation_author=F. Vazquez; citation_author=P.G. Montgomery; citation_author=B.A. Weir; citation_author=G. Kryukov; citation_author=G.S. Cowley; citation_author=S. Gill; citation_author=W.F. Harrington; citation_author=S. Pantel; citation_author=J.M. Krill-Burger; citation_author=et al.; citation_journal_title=Cell; citation_volume=170; citation_firstpage=564; citation_lastpage=576.e16; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Rapid and deep-scale ubiquitylation profiling for biology and translational research; citation_author=N.D. Udeshi; citation_author=D.C. Mani; citation_author=S. Satpathy; citation_author=S. Fereshetian; citation_author=J.A. Gasser; citation_author=T. Svinkina; citation_author=M.E. Olive; citation_author=B.L. Ebert; citation_author=P. Mertins; citation_author=S.A. Carr; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=359; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=A human protein atlas for normal and cancer tissues based on antibody proteomics; citation_author=M. Uhlén; citation_author=E. Björling; citation_author=C. Agaton; citation_author=C.A.-K. Szigyarto; citation_author=B. Amini; citation_author=E. Andersen; citation_author=A.-C. Andersson; citation_author=P. Angelidou; citation_author=A. Asplund; citation_author=C. Asplund; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=4; citation_firstpage=1920; citation_lastpage=1932; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Accurate and efficient detection of gene fusions from RNA sequencing data; citation_author=S. Uhrig; citation_author=J. Ellermann; citation_author=T. Walther; citation_author=P. Burkhardt; citation_author=M. Fröhlich; citation_author=B. Hutter; citation_author=U.H. Toprak; citation_author=O. Neumann; citation_author=A. Stenzinger; citation_author=C. Scholl; citation_author=et al.; citation_journal_title=Genome Res; citation_volume=31; citation_firstpage=448; citation_lastpage=460; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=LinkedOmics: analyzing multi-omics data within and across 32 cancer types; citation_author=S.V. Vasaikar; citation_author=P. Straub; citation_author=J. Wang; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D956; citation_lastpage=D963; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O.A. Arshad; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit; citation_author=P. Vats; citation_author=A.M. Chinnaiyan; citation_author=C. Kumar-Sinha; citation_journal_title=Methods Mol. Biol.; citation_volume=2079; citation_firstpage=69; citation_lastpage=79; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Rho GDP dissociation inhibitor-β in renal cell carcinoma; citation_author=C.-A. von Klot; citation_author=N. Dubrowinskaja; citation_author=I. Peters; citation_author=J. Hennenlotter; citation_author=A.S. Merseburger; citation_author=A. Stenzl; citation_author=M.A. Kuczyk; citation_author=J. Serth; citation_journal_title=Oncol. Lett.; citation_volume=14; citation_firstpage=8190; citation_lastpage=8196; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=miR-205 targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma; citation_author=H. Vosgha; citation_author=A. Ariana; citation_author=R.A. Smith; citation_author=A.K.-Y. Lam; citation_journal_title=Endocr. Relat. Cancer; citation_volume=25; citation_firstpage=323; citation_lastpage=337; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Smoking and lung cancer: the role of inflammation; citation_author=T. Walser; citation_author=X. Cui; citation_author=J. Yanagawa; citation_author=J.M. Lee; citation_author=E. Heinrich; citation_author=G. Lee; citation_author=S. Sharma; citation_author=S.M. Dubinett; citation_journal_title=Proc. Am. Thorac. Soc.; citation_volume=5; citation_firstpage=811; citation_lastpage=815; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers; citation_author=I.-M. Wang; citation_author=B. Zhang; citation_author=X. Yang; citation_author=J. Zhu; citation_author=S. Stepaniants; citation_author=C. Zhang; citation_author=Q. Meng; citation_author=M. Peters; citation_author=Y. He; citation_author=C. Ni; citation_author=et al.; citation_journal_title=Mol. Syst. Biol.; citation_volume=8; citation_firstpage=594; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1; citation_author=S. Wang; citation_author=B. Jiang; citation_author=T. Zhang; citation_author=L. Liu; citation_author=Y. Wang; citation_author=Y. Wang; citation_author=X. Chen; citation_author=H. Lin; citation_author=L. Zhou; citation_author=Y. Xia; citation_author=et al.; citation_journal_title=PLoS Biol.; citation_volume=13; citation_firstpage=e1002243; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit; citation_author=J. Wang; citation_author=S. Vasaikar; citation_author=Z. Shi; citation_author=M. Greer; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=W1; citation_firstpage=W130; citation_lastpage=W137; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic and metabolomic characterization of human glioblastoma; citation_author=L.-B. Wang; citation_author=A. Karpova; citation_author=M.A. Gritsenko; citation_author=J.E. Kyle; citation_author=S. Cao; citation_author=Y. Li; citation_author=D. Rykunov; citation_author=A. Colaprico; citation_author=J.H. Rothstein; citation_author=R. Hong; citation_author=Clinical Proteomic Tumor Analysis Consortium; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=39; citation_firstpage=509; citation_lastpage=528.e20; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies; citation_author=C.E. Weeden; citation_author=B. Solomon; citation_author=M.-L. Asselin-Labat; citation_journal_title=Cell Death Discov.; citation_volume=1; citation_firstpage=15049; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations; citation_author=B. Wen; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=485; citation_lastpage=493; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis; citation_author=B. Wen; citation_author=K. Li; citation_author=Y. Zhang; citation_author=B. Zhang; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=1759; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Systematic identification of trans eQTLs as putative drivers of known disease associations; citation_author=H.-J. Westra; citation_author=M.J. Peters; citation_author=T. Esko; citation_author=H. Yaghootkar; citation_author=C. Schurmann; citation_author=J. Kettunen; citation_author=M.W. Christiansen; citation_author=B.P. Fairfax; citation_author=K. Schramm; citation_author=J.E. Powell; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=45; citation_firstpage=1238; citation_lastpage=1243; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Survivin at a glance; citation_author=S.P. Wheatley; citation_author=D.C. Altieri; citation_journal_title=J. Cell Sci.; citation_volume=132; citation_firstpage=jcs223826; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Identification of genes periodically expressed in the human cell cycle and their expression in tumors; citation_author=M.L. Whitfield; citation_author=G. Sherlock; citation_author=A.J. Saldanha; citation_author=J.I. Murray; citation_author=C.A. Ball; citation_author=K.E. Alexander; citation_author=J.C. Matese; citation_author=C.M. Perou; citation_author=M.M. Hurt; citation_author=P.O. Brown; citation_author=D. Botstein; citation_journal_title=Mol. Biol. Cell; citation_volume=13; citation_firstpage=1977; citation_lastpage=2000; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking; citation_author=M.D. Wilkerson; citation_author=D.N. Hayes; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=1572; citation_lastpage=1573; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types; citation_author=M.D. Wilkerson; citation_author=X. Yin; citation_author=K.A. Hoadley; citation_author=Y. Liu; citation_author=M.C. Hayward; citation_author=C.R. Cabanski; citation_author=K. Muldrew; citation_author=C.R. Miller; citation_author=S.H. Randell; citation_author=M.A. Socinski; citation_author=et al.; citation_journal_title=Clin. Cancer Res.; citation_volume=16; citation_firstpage=4864; citation_lastpage=4875; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets; citation_author=J.J.Y. Wong; citation_author=Y.F. Pung; citation_author=N.S.-K. Sze; citation_author=K.-C. Chin; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=103; citation_firstpage=10735; citation_lastpage=10740; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer; citation_author=Y.-M. Wu; citation_author=M. Cieślik; citation_author=R.J. Lonigro; citation_author=P. Vats; citation_author=M.A. Reimers; citation_author=X. Cao; citation_author=Y. Ning; citation_author=L. Wang; citation_author=L.P. Kunju; citation_author=N. de Sarkar; citation_author=PCF/SU2C International Prostate Cancer Dream Team; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=1770; citation_lastpage=1782.e14; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis; citation_author=Y. Wu; citation_author=Y. Zhang; citation_author=X. Zheng; citation_author=F. Dai; citation_author=Y. Lu; citation_author=L. Dai; citation_author=M. Niu; citation_author=H. Guo; citation_author=W. Li; citation_author=X. Xue; citation_author=et al.; citation_journal_title=Mol. Cancer; citation_volume=19; citation_firstpage=99; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK; citation_author=Y. Xiong; citation_author=Q.-Y. Lei; citation_author=S. Zhao; citation_author=K.-L. Guan; citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.; citation_volume=76; citation_firstpage=285; citation_lastpage=289; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Nuclear PKM2 regulates the Warburg effect; citation_author=W. Yang; citation_author=Z. Lu; citation_journal_title=Cell Cycle; citation_volume=12; citation_firstpage=3154; citation_lastpage=3158; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads; citation_author=K. Ye; citation_author=M.H. Schulz; citation_author=Q. Long; citation_author=R. Apweiler; citation_author=Z. Ning; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2865; citation_lastpage=2871; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer; citation_author=S. Yomtoubian; citation_author=S.B. Lee; citation_author=A. Verma; citation_author=F. Izzo; citation_author=G. Markowitz; citation_author=H. Choi; citation_author=L. Cerchietti; citation_author=L. Vahdat; citation_author=K.A. Brown; citation_author=E. Andreopoulou; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=30; citation_firstpage=755; citation_lastpage=770.e6; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=Inferring tumour purity and stromal and immune cell admixture from expression data; citation_author=K. Yoshihara; citation_author=M. Shahmoradgoli; citation_author=E. Martínez; citation_author=R. Vegesna; citation_author=H. Kim; citation_author=W. Torres-Garcia; citation_author=V. Treviño; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=D.A. Levine; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2612; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Elevated NSD3 histone methylation activity drives squamous cell lung cancer; citation_author=G. Yuan; citation_author=N.M. Flores; citation_author=S. Hausmann; citation_author=S.M. Lofgren; citation_author=V. Kharchenko; citation_author=M. Angulo-Ibanez; citation_author=D. Sengupta; citation_author=X. Lu; citation_author=I. Czaban; citation_author=D. Azhibek; citation_author=et al.; citation_journal_title=Nature; citation_volume=590; citation_firstpage=504; citation_lastpage=508; citation_publication_date=2021; "></meta><meta name="citation_reference" content="citation_title=TMT labeling for the masses: A robust and cost-efficient, in-solution labeling approach; citation_author=J. Zecha; citation_author=S. Satpathy; citation_author=T. Kanashova; citation_author=S.C. Avanessian; citation_author=M.H. Kane; citation_author=K.R. Clauser; citation_author=P. Mertins; citation_author=S.A. Carr; citation_author=B. Kuster; citation_journal_title=Mol. Cell Proteomics; citation_volume=18; citation_firstpage=1468; citation_lastpage=1478; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells; citation_author=X. Zhang; citation_author=F. Lu; citation_author=J. Wang; citation_author=F. Yin; citation_author=Z. Xu; citation_author=D. Qi; citation_author=X. Wu; citation_author=Y. Cao; citation_author=W. Liang; citation_author=Y. Liu; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=5; citation_firstpage=445; citation_lastpage=457; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis; citation_author=Y. Zhang; citation_author=X. Zou; citation_author=W. Qian; citation_author=X. Weng; citation_author=L. Zhang; citation_author=L. Zhang; citation_author=S. Wang; citation_author=X. Cao; citation_author=L. Ma; citation_author=G. Wei; citation_author=et al.; citation_journal_title=Cell Death Differ.; citation_volume=26; citation_firstpage=565; citation_lastpage=579; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection; citation_author=Y. Zhang; citation_author=F. Thery; citation_author=N.C. Wu; citation_author=E.K. Luhmann; citation_author=O. Dussurget; citation_author=M. Foecke; citation_author=C. Bredow; citation_author=D. Jiménez-Fernández; citation_author=K. Leandro; citation_author=A. Beling; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=5383; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Impact of acetylation on tumor metabolism; citation_author=D. Zhao; citation_author=F.-L. Li; citation_author=Z.-L. Cheng; citation_author=Q.-Y. Lei; citation_journal_title=Mol. Cell. Oncol.; citation_volume=1; citation_firstpage=e963452; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Glycogen synthase kinase 3β inhibition enhances Notch1 recycling; citation_author=L. Zheng; citation_author=S.D. Conner; citation_journal_title=Mol. Biol. Cell; citation_volume=29; citation_firstpage=389; citation_lastpage=395; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Exploring genomic alteration in pediatric cancer using ProteinPaint; citation_author=X. Zhou; citation_author=M.N. Edmonson; citation_author=M.R. Wilkinson; citation_author=A. Patel; citation_author=G. Wu; citation_author=Y. Liu; citation_author=Y. Li; citation_author=Z. Zhang; citation_author=M.C. Rusch; citation_author=M. Parker; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=4; citation_lastpage=6; citation_publication_date=2016; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900857-6%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867421008576%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867421008576%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00857-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867421008576%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900857-6%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867421008576%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900857-6%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867421008576%253Fshowall%253Dtrue&amp;pii=S0092-8674%2821%2900857-6" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=cell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900857-6%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867421008576%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900857-6%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867421008576%253Fshowall%253Dtrue&amp;pii=S0092-8674%2821%2900857-6" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.07.016">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(23)X0017-8"><img src="/cms/asset/atypon:cms:attachment:img:d18e6:rev:1722883367683-3936:pii:S0092867423X00178/cover.tif.jpg" alt="Cover Image - Cell, Volume 187, Issue 16"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S0092-8674(21)00857-6"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Resource</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0017-1" class="article-header__vol faded">
                                                    Volume 184, ISSUE 16</a><span></span><span>, </span><span class="article-header__pages faded">P4348-4371.e40, </span><span class="article-header__date faded">August 05, 2021</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=16&amp;pii=S0092-8674%2820%29X0017-1" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900857-6" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF [20 MB]</span><span class="alternative">PDF [20 MB]</span></a></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cell/ppt/S0092-8674(21)00857-6.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.cell.2021.07.016" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.07.016&amp;pii=S0092-8674%2821%2900857-6" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.07.016" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-740" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-740">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900857-6"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S0092867421008576&amp;issn=00928674&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421008576&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">A proteogenomic portrait of lung squamous cell carcinoma</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shankha Satpathy">Shankha Satpathy<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0015" aria-expanded="false" aria-controls="crosref0015" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>19</sup></span></a><div data-db-target-of="crosref0015" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">19 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shankha Satpathy</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#fd8e959c9396959cbd9f8f929c9994938e899489888998d3928f9a" title="Send email to Shankha Satpathy"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">19 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Satpathy%2C+Shankha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Shankha Satpathy" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#e5968d848b8e8d84a587978a84818c8b96918c91909180cb8a9782"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karsten Krug">Karsten Krug</a> <a href="#" data-db-target-for="crosref0030" aria-expanded="false" aria-controls="crosref0030" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>19</sup></span></a><div data-db-target-of="crosref0030" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">19 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karsten Krug</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">19 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krug%2C+Karsten" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pierre M. Jean Beltran">Pierre M. Jean Beltran</a> <a href="#" data-db-target-for="crosref0040" aria-expanded="false" aria-controls="crosref0040" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>19</sup></span></a><div data-db-target-of="crosref0040" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">19 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pierre M. Jean Beltran</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">19 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jean Beltran%2C+Pierre M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sara R. Savage">Sara R. Savage</a> <a href="#" data-db-target-for="crosref0050" aria-expanded="false" aria-controls="crosref0050" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>19</sup></span></a><div data-db-target-of="crosref0050" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">19 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sara R. Savage</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">19 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Savage%2C+Sara R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Francesca Petralia">Francesca Petralia</a> <a href="#" data-db-target-for="crosref0060" aria-expanded="false" aria-controls="crosref0060" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0060" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Francesca Petralia</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petralia%2C+Francesca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chandan Kumar-Sinha">Chandan Kumar-Sinha</a> <a href="#" data-db-target-for="crosref0070" aria-expanded="false" aria-controls="crosref0070" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0070" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chandan Kumar-Sinha</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kumar-Sinha%2C+Chandan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yongchao Dou">Yongchao Dou</a> <a href="#" data-db-target-for="crosref0080" aria-expanded="false" aria-controls="crosref0080" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0080" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yongchao Dou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dou%2C+Yongchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Boris Reva">Boris Reva</a> <a href="#" data-db-target-for="crosref0090" aria-expanded="false" aria-controls="crosref0090" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0090" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Boris Reva</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Reva%2C+Boris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author M. Harry Kane">M. Harry Kane</a> <a href="#" data-db-target-for="crosref0100" aria-expanded="false" aria-controls="crosref0100" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0100" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">M. Harry Kane</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kane%2C+M. Harry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shayan C. Avanessian">Shayan C. Avanessian</a> <a href="#" data-db-target-for="crosref0110" aria-expanded="false" aria-controls="crosref0110" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0110" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shayan C. Avanessian</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Avanessian%2C+Shayan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Suhas V. Vasaikar">Suhas V. Vasaikar</a> <a href="#" data-db-target-for="crosref0120" aria-expanded="false" aria-controls="crosref0120" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>20</sup></span></a><div data-db-target-of="crosref0120" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">20 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Suhas V. Vasaikar</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">20 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Vasaikar%2C+Suhas V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Azra Krek">Azra Krek</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Azra Krek</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krek%2C+Azra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jonathan T. Lei">Jonathan T. Lei</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jonathan T. Lei</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lei%2C+Jonathan T" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Eric J. Jaehnig">Eric J. Jaehnig</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Eric J. Jaehnig</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jaehnig%2C+Eric J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tatiana Omelchenko">Tatiana Omelchenko</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tatiana Omelchenko</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Omelchenko%2C+Tatiana" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yifat Geffen">Yifat Geffen</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yifat Geffen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Geffen%2C+Yifat" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Erik J. Bergstrom">Erik J. Bergstrom</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Erik J. Bergstrom</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Bergstrom%2C+Erik J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Vasileios Stathias">Vasileios Stathias</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Vasileios Stathias</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Stathias%2C+Vasileios" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karen E. Christianson">Karen E. Christianson</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karen E. Christianson</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Christianson%2C+Karen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David I. Heiman">David I. Heiman</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David I. Heiman</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Heiman%2C+David I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marcin P. Cieslik">Marcin P. Cieslik</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marcin P. Cieslik</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cieslik%2C+Marcin P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Song Cao">Song Cao</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Song Cao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cao%2C+Song" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaoyu Song">Xiaoyu Song</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaoyu Song</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Population Health Science and Policy, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Song%2C+Xiaoyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiayi Ji">Jiayi Ji</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiayi Ji</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Population Health Science and Policy, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ji%2C+Jiayi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wenke Liu">Wenke Liu</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wenke Liu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Wenke" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kai Li">Kai Li</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kai Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Kai" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bo Wen">Bo Wen</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yize Li">Yize Li</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yize Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Yize" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zeynep H. Gümüş">Zeynep H. Gümüş</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zeynep H. Gümüş</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gümüş%2C+Zeynep H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Myvizhi Esai Selvan">Myvizhi Esai Selvan</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Myvizhi Esai Selvan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Selvan%2C+Myvizhi Esai" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rama Soundararajan">Rama Soundararajan</a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rama Soundararajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Soundararajan%2C+Rama" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tanvi H. Visal">Tanvi H. Visal</a><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tanvi H. Visal</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Visal%2C+Tanvi H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au33-authorcard" aria-expanded="false" aria-controls="au33" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Maria G. Raso">Maria G. Raso</a><div data-db-target-of="au33-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Maria G. Raso</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Raso%2C+Maria G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au34-authorcard" aria-expanded="false" aria-controls="au34" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Edwin Roger Parra">Edwin Roger Parra</a><div data-db-target-of="au34-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Edwin Roger Parra</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Parra%2C+Edwin Roger" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au35-authorcard" aria-expanded="false" aria-controls="au35" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Özgün Babur">Özgün Babur</a><div data-db-target-of="au35-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Özgün Babur</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Computer Science Department, University of Massachusetts Boston, Boston, MA 02125, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Babur%2C+Özgün" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au36-authorcard" aria-expanded="false" aria-controls="au36" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pankaj Vats">Pankaj Vats</a><div data-db-target-of="au36-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pankaj Vats</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Vats%2C+Pankaj" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au37-authorcard" aria-expanded="false" aria-controls="au37" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shankara Anand">Shankara Anand</a><div data-db-target-of="au37-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shankara Anand</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Anand%2C+Shankara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au38-authorcard" aria-expanded="false" aria-controls="au38" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tobias Schraink">Tobias Schraink</a><div data-db-target-of="au38-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tobias Schraink</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schraink%2C+Tobias" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au39-authorcard" aria-expanded="false" aria-controls="au39" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author MacIntosh Cornwell">MacIntosh Cornwell</a><div data-db-target-of="au39-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">MacIntosh Cornwell</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cornwell%2C+MacIntosh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au40-authorcard" aria-expanded="false" aria-controls="au40" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Fernanda Martins Rodrigues">Fernanda Martins Rodrigues</a><div data-db-target-of="au40-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Fernanda Martins Rodrigues</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodrigues%2C+Fernanda Martins" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au41-authorcard" aria-expanded="false" aria-controls="au41" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Houxiang Zhu">Houxiang Zhu</a><div data-db-target-of="au41-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Houxiang Zhu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Houxiang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au42-authorcard" aria-expanded="false" aria-controls="au42" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chia-Kuei Mo">Chia-Kuei Mo</a><div data-db-target-of="au42-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chia-Kuei Mo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mo%2C+Chia-Kuei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au43-authorcard" aria-expanded="false" aria-controls="au43" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yuping Zhang">Yuping Zhang</a><div data-db-target-of="au43-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yuping Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Yuping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au44-authorcard" aria-expanded="false" aria-controls="au44" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Felipe da Veiga Leprevost">Felipe da Veiga Leprevost</a><div data-db-target-of="au44-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Felipe da Veiga Leprevost</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=da Veiga Leprevost%2C+Felipe" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au45-authorcard" aria-expanded="false" aria-controls="au45" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a><div data-db-target-of="au45-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au46-authorcard" aria-expanded="false" aria-controls="au46" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Arul M. Chinnaiyan">Arul M. Chinnaiyan</a><div data-db-target-of="au46-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Arul M. Chinnaiyan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chinnaiyan%2C+Arul M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au47-authorcard" aria-expanded="false" aria-controls="au47" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew A. Wyczalkowski">Matthew A. Wyczalkowski</a><div data-db-target-of="au47-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew A. Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au48-authorcard" aria-expanded="false" aria-controls="au48" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gilbert S. Omenn">Gilbert S. Omenn</a><div data-db-target-of="au48-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gilbert S. Omenn</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Omenn%2C+Gilbert S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au49-authorcard" aria-expanded="false" aria-controls="au49" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chelsea J. Newton">Chelsea J. Newton</a><div data-db-target-of="au49-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chelsea J. Newton</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Van Andel Research Institute, Grand Rapids, MI 49503, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Newton%2C+Chelsea J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au50-authorcard" aria-expanded="false" aria-controls="au50" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Stephan Schurer">Stephan Schurer</a><div data-db-target-of="au50-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Stephan Schurer</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center and Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schurer%2C+Stephan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au51-authorcard" aria-expanded="false" aria-controls="au51" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kelly V. Ruggles">Kelly V. Ruggles</a><div data-db-target-of="au51-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kelly V. Ruggles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ruggles%2C+Kelly V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au52-authorcard" aria-expanded="false" aria-controls="au52" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David Fenyö">David Fenyö</a><div data-db-target-of="au52-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David Fenyö</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics and Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Fenyö%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au53-authorcard" aria-expanded="false" aria-controls="au53" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Scott D. Jewell">Scott D. Jewell</a><div data-db-target-of="au53-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Scott D. Jewell</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Van Andel Research Institute, Grand Rapids, MI 49503, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Jewell%2C+Scott D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au54-authorcard" aria-expanded="false" aria-controls="au54" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au54-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au55-authorcard" aria-expanded="false" aria-controls="au55" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mehdi Mesri">Mehdi Mesri</a><div data-db-target-of="au55-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au56-authorcard" aria-expanded="false" aria-controls="au56" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au56-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au57-authorcard" aria-expanded="false" aria-controls="au57" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sendurai A. Mani">Sendurai A. Mani</a><div data-db-target-of="au57-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sendurai A. Mani</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Translational Molecular Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+Sendurai A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au58-authorcard" aria-expanded="false" aria-controls="au58" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Namrata D. Udeshi">Namrata D. Udeshi</a><div data-db-target-of="au58-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Namrata D. Udeshi</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Udeshi%2C+Namrata D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au59-authorcard" aria-expanded="false" aria-controls="au59" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Gad Getz">Gad Getz</a><div data-db-target-of="au59-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Gad Getz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Getz%2C+Gad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au60-authorcard" aria-expanded="false" aria-controls="au60" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author James Suh">James Suh</a><div data-db-target-of="au60-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">James Suh</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Suh%2C+James" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au61-authorcard" aria-expanded="false" aria-controls="au61" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Qing Kay Li">Qing Kay Li</a><div data-db-target-of="au61-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Qing Kay Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sidney Kimmel Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD 21224, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Qing Kay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au62-authorcard" aria-expanded="false" aria-controls="au62" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Galen Hostetter">Galen Hostetter</a><div data-db-target-of="au62-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Galen Hostetter</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Van Andel Research Institute, Grand Rapids, MI 49503, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hostetter%2C+Galen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au63-authorcard" aria-expanded="false" aria-controls="au63" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Paul K. Paik">Paul K. Paik</a><div data-db-target-of="au63-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Paul K. Paik</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Memorial Sloan Kettering Cancer Center, New York, NY 10065, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Paik%2C+Paul K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au64-authorcard" aria-expanded="false" aria-controls="au64" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Saravana M. Dhanasekaran">Saravana M. Dhanasekaran</a><div data-db-target-of="au64-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Saravana M. Dhanasekaran</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dhanasekaran%2C+Saravana M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au65-authorcard" aria-expanded="false" aria-controls="au65" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ramaswamy Govindan">Ramaswamy Govindan</a><div data-db-target-of="au65-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ramaswamy Govindan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Govindan%2C+Ramaswamy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au66-authorcard" aria-expanded="false" aria-controls="au66" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au66-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63110, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au67-authorcard" aria-expanded="false" aria-controls="au67" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au67-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au68-authorcard" aria-expanded="false" aria-controls="au68" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karl R. Clauser">Karl R. Clauser</a><div data-db-target-of="au68-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karl R. Clauser</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clauser%2C+Karl R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au69-authorcard" aria-expanded="false" aria-controls="au69" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexey I. Nesvizhskii">Alexey I. Nesvizhskii</a><div data-db-target-of="au69-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexey I. Nesvizhskii</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Nesvizhskii%2C+Alexey I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au70-authorcard" aria-expanded="false" aria-controls="au70" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pei Wang">Pei Wang</a><div data-db-target-of="au70-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pei Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute for Data Science and Genomic Technology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Pei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au71-authorcard" aria-expanded="false" aria-controls="au71" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Steven A. Carr">Steven A. Carr<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au71-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Steven A. Carr</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#1c6f7f7d6e6e5c7e6e737d783271756832797869" title="Send email to Steven A. Carr"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Steven A. Carr" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#c8bbaba9baba88aabaa7a9ace6a5a1bce6adacbd"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au72-authorcard" aria-expanded="false" aria-controls="au72" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au72-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#a6c4cfc8c188dccec7c8c1e6c4c5cb88c3c2d3" title="Send email to Bing Zhang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Bing Zhang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#a3c1cacdc48dd9cbc2cdc4e3c1c0ce8dc6c7d6"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au73-authorcard" aria-expanded="false" aria-controls="au73" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author D.R. Mani">D.R. Mani<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au73-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">D.R. Mani</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#cfa2aea1a6abbd8fadbda0aeaba6a1bcbba6bbbabbaae1a0bda8" title="Send email to D.R. Mani"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to D.R. Mani" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#1d707c7374796f5d7f6f727c7974736e69746968697833726f7a"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au74-authorcard" aria-expanded="false" aria-controls="au74" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael A. Gillette">Michael A. Gillette<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0470" aria-expanded="false" aria-controls="crosref0470" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>21</sup></span></a><div data-db-target-of="crosref0470" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">21 Lead contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au74-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael A. Gillette</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#781f1114141d0c0c1d381a0a17191c11160b0c110c0d0c1d56170a1f" title="Send email to Michael A. Gillette"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">21 Lead contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA<p></p>Division of Pulmonary and Critical Care Medicine, Massachusetts General Hospital, Boston, MA 02115, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Michael A. Gillette" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#e5828c898980919180a587978a84818c8b96918c91909180cb8a9782"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Alex Green" class="dropBlock__slider__ctrl">Alex Green</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alex back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alex Green</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Green%2C+Alex" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alfredo Molinolo" class="dropBlock__slider__ctrl">Alfredo Molinolo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alfredo back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alfredo Molinolo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Molinolo%2C+Alfredo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alicia Francis" class="dropBlock__slider__ctrl">Alicia Francis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alicia back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alicia Francis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Francis%2C+Alicia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amanda G. Paulovich" class="dropBlock__slider__ctrl">Amanda G. Paulovich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amanda G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amanda G. Paulovich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Andrii Karnuta" class="dropBlock__slider__ctrl">Andrii Karnuta</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Andrii back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Andrii Karnuta</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Karnuta%2C+Andrii" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Antonio Colaprico" class="dropBlock__slider__ctrl">Antonio Colaprico</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Antonio back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Antonio Colaprico</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Colaprico%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Hindenach" class="dropBlock__slider__ctrl">Barbara Hindenach</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Hindenach</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hindenach%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara L. Pruetz" class="dropBlock__slider__ctrl">Barbara L. Pruetz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara L. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara L. Pruetz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pruetz%2C+Barbara L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Bartosz Kubisa" class="dropBlock__slider__ctrl">Bartosz Kubisa</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Bartosz back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Bartosz Kubisa</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kubisa%2C+Bartosz" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Brian J. Druker" class="dropBlock__slider__ctrl">Brian J. Druker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Brian J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Brian J. Druker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Druker%2C+Brian J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Carissa Huynh" class="dropBlock__slider__ctrl">Carissa Huynh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Carissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Carissa Huynh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Huynh%2C+Carissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Charles A. Goldthwaite Jr." class="dropBlock__slider__ctrl">Charles A. Goldthwaite Jr.</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Charles A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Charles A. Goldthwaite Jr.</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Goldthwaite%2C+Charles A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chet Birger" class="dropBlock__slider__ctrl">Chet Birger</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chet back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chet Birger</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birger%2C+Chet" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Christopher R. Kinsinger" class="dropBlock__slider__ctrl">Christopher R. Kinsinger</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Christopher R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Corbin D. Jones" class="dropBlock__slider__ctrl">Corbin D. Jones</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Corbin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Corbin D. Jones</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jones%2C+Corbin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dan Rohrer" class="dropBlock__slider__ctrl">Dan Rohrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dan Rohrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rohrer%2C+Dan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dana R. Valley" class="dropBlock__slider__ctrl">Dana R. Valley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dana R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dana R. Valley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Valley%2C+Dana R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel W. Chan" class="dropBlock__slider__ctrl">Daniel W. Chan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel W. Chan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Chesla" class="dropBlock__slider__ctrl">David Chesla</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Chesla</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chesla%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Donna Hansel" class="dropBlock__slider__ctrl">Donna Hansel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Donna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Donna Hansel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hansel%2C+Donna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elena V. Ponomareva" class="dropBlock__slider__ctrl">Elena V. Ponomareva</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elena V. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elena V. Ponomareva</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ponomareva%2C+Elena V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elizabeth Duffy" class="dropBlock__slider__ctrl">Elizabeth Duffy</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elizabeth back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elizabeth Duffy</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Duffy%2C+Elizabeth" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric Burks" class="dropBlock__slider__ctrl">Eric Burks</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric Burks</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Burks%2C+Eric" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric E. Schadt" class="dropBlock__slider__ctrl">Eric E. Schadt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric E. Schadt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Schadt%2C+Eric E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eugene S. Fedorov" class="dropBlock__slider__ctrl">Eugene S. Fedorov</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eugene S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eugene S. Fedorov</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Fedorov%2C+Eugene S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eunkyung An" class="dropBlock__slider__ctrl">Eunkyung An</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eunkyung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eunkyung An</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=An%2C+Eunkyung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Fei Ding" class="dropBlock__slider__ctrl">Fei Ding</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Fei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Fei Ding</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Fei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="George D. Wilson" class="dropBlock__slider__ctrl">George D. Wilson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="George D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">George D. Wilson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wilson%2C+George D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Harsh Batra" class="dropBlock__slider__ctrl">Harsh Batra</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Harsh back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Harsh Batra</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Batra%2C+Harsh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hui Zhang" class="dropBlock__slider__ctrl">Hui Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hui Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jennifer E. Maas" class="dropBlock__slider__ctrl">Jennifer E. Maas</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jennifer E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jennifer E. Maas</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maas%2C+Jennifer E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jennifer Eschbacher" class="dropBlock__slider__ctrl">Jennifer Eschbacher</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jennifer back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jennifer Eschbacher</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Eschbacher%2C+Jennifer" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karen A. Ketchum" class="dropBlock__slider__ctrl">Karen A. Ketchum</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karen A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karin D. Rodland" class="dropBlock__slider__ctrl">Karin D. Rodland</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karin D. Rodland</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Katherine A. Hoadley" class="dropBlock__slider__ctrl">Katherine A. Hoadley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Katherine A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Katherine A. Hoadley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoadley%2C+Katherine A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kei Suzuki" class="dropBlock__slider__ctrl">Kei Suzuki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kei Suzuki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Suzuki%2C+Kei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ki Sung Um" class="dropBlock__slider__ctrl">Ki Sung Um</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ki Sung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ki Sung Um</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Um%2C+Ki Sung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liqun Qi" class="dropBlock__slider__ctrl">Liqun Qi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liqun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liqun Qi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Qi%2C+Liqun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lori Bernard" class="dropBlock__slider__ctrl">Lori Bernard</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lori back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lori Bernard</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bernard%2C+Lori" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Maciej Wiznerowicz" class="dropBlock__slider__ctrl">Maciej Wiznerowicz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Maciej back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Maciej Wiznerowicz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wiznerowicz%2C+Maciej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Małgorzata Wojtyś" class="dropBlock__slider__ctrl">Małgorzata Wojtyś</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Małgorzata back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Małgorzata Wojtyś</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wojtyś%2C+Małgorzata" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marcin J. Domagalski" class="dropBlock__slider__ctrl">Marcin J. Domagalski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marcin J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marcin J. Domagalski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Domagalski%2C+Marcin J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew J. Ellis" class="dropBlock__slider__ctrl">Matthew J. Ellis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew J. Ellis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Maureen A. Dyer" class="dropBlock__slider__ctrl">Maureen A. Dyer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Maureen A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Maureen A. Dyer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dyer%2C+Maureen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melissa Borucki" class="dropBlock__slider__ctrl">Melissa Borucki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melissa Borucki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borucki%2C+Melissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Meenakshi Anurag" class="dropBlock__slider__ctrl">Meenakshi Anurag</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Meenakshi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Meenakshi Anurag</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Anurag%2C+Meenakshi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael J. Birrer" class="dropBlock__slider__ctrl">Michael J. Birrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael J. Birrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birrer%2C+Michael J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Midie Xu" class="dropBlock__slider__ctrl">Midie Xu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Midie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Midie Xu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Xu%2C+Midie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mikhail Krotevich" class="dropBlock__slider__ctrl">Mikhail Krotevich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mikhail back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mikhail Krotevich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Krotevich%2C+Mikhail" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nancy Roche" class="dropBlock__slider__ctrl">Nancy Roche</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nancy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nancy Roche</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Roche%2C+Nancy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Negin Vatanian" class="dropBlock__slider__ctrl">Negin Vatanian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Negin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Negin Vatanian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vatanian%2C+Negin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Neil R. Mucci" class="dropBlock__slider__ctrl">Neil R. Mucci</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Neil R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Neil R. Mucci</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mucci%2C+Neil R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nicollette Maunganidze" class="dropBlock__slider__ctrl">Nicollette Maunganidze</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nicollette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nicollette Maunganidze</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maunganidze%2C+Nicollette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nikolay Gabrovski" class="dropBlock__slider__ctrl">Nikolay Gabrovski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nikolay back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nikolay Gabrovski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabrovski%2C+Nikolay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Olga Potapova" class="dropBlock__slider__ctrl">Olga Potapova</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Olga back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Olga Potapova</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Potapova%2C+Olga" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Oluwole Fadare" class="dropBlock__slider__ctrl">Oluwole Fadare</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Oluwole back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Oluwole Fadare</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Fadare%2C+Oluwole" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pamela Grady" class="dropBlock__slider__ctrl">Pamela Grady</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pamela back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pamela Grady</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Grady%2C+Pamela" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pushpa Hariharan" class="dropBlock__slider__ctrl">Pushpa Hariharan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pushpa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pushpa Hariharan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hariharan%2C+Pushpa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rebecca Montgomery" class="dropBlock__slider__ctrl">Rebecca Montgomery</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rebecca back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rebecca Montgomery</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Montgomery%2C+Rebecca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Renganayaki Pandurengan" class="dropBlock__slider__ctrl">Renganayaki Pandurengan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Renganayaki back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Renganayaki Pandurengan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pandurengan%2C+Renganayaki" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Richard D. Smith" class="dropBlock__slider__ctrl">Richard D. Smith</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Richard D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert J. Welsh" class="dropBlock__slider__ctrl">Robert J. Welsh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert J. Welsh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Welsh%2C+Robert J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sailaja Mareedu" class="dropBlock__slider__ctrl">Sailaja Mareedu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sailaja back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sailaja Mareedu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mareedu%2C+Sailaja" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Samuel H. Payne" class="dropBlock__slider__ctrl">Samuel H. Payne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Samuel H. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sandra Cottingham" class="dropBlock__slider__ctrl">Sandra Cottingham</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sandra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sandra Cottingham</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cottingham%2C+Sandra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shilpi Singh" class="dropBlock__slider__ctrl">Shilpi Singh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shilpi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shilpi Singh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Singh%2C+Shilpi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shirley X. Tsang" class="dropBlock__slider__ctrl">Shirley X. Tsang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shirley X. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shirley X. Tsang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tsang%2C+Shirley X" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stacey Gabriel" class="dropBlock__slider__ctrl">Stacey Gabriel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stacey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stacey Gabriel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabriel%2C+Stacey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tao Liu" class="dropBlock__slider__ctrl">Tao Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tao Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tara Hiltke" class="dropBlock__slider__ctrl">Tara Hiltke</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tanmayi Vashist" class="dropBlock__slider__ctrl">Tanmayi Vashist</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tanmayi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tanmayi Vashist</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vashist%2C+Tanmayi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Thomas Bauer" class="dropBlock__slider__ctrl">Thomas Bauer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Thomas back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Thomas Bauer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bauer%2C+Thomas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Volodymyr Sovenko" class="dropBlock__slider__ctrl">Volodymyr Sovenko</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Volodymyr back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Volodymyr Sovenko</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sovenko%2C+Volodymyr" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Warren G. Tourtellotte" class="dropBlock__slider__ctrl">Warren G. Tourtellotte</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Warren G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Warren G. Tourtellotte</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tourtellotte%2C+Warren G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Weiping Ma" class="dropBlock__slider__ctrl">Weiping Ma</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Weiping back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Weiping Ma</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ma%2C+Weiping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="William Bocik" class="dropBlock__slider__ctrl">William Bocik</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="William back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">William Bocik</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bocik%2C+William" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Wohaib Hasan" class="dropBlock__slider__ctrl">Wohaib Hasan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Wohaib back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Wohaib Hasan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hasan%2C+Wohaib" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Xiaojun Jing" class="dropBlock__slider__ctrl">Xiaojun Jing</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Xiaojun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Xiaojun Jing</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jing%2C+Xiaojun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ximing Tang" class="dropBlock__slider__ctrl">Ximing Tang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ximing back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ximing Tang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tang%2C+Ximing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yuxing Liao" class="dropBlock__slider__ctrl">Yuxing Liao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yuxing back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yuxing Liao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liao%2C+Yuxing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title=" Yvonne" class="dropBlock__slider__ctrl"> Yvonne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title=" back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label"> Yvonne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Yvonne%2C+" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title=" Shutack" class="dropBlock__slider__ctrl"> Shutack</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title=" back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label"> Shutack</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shutack%2C+" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhen Zhang" class="dropBlock__slider__ctrl">Zhen Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ziad Hanhan" class="dropBlock__slider__ctrl">Ziad Hanhan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ziad back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ziad Hanhan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hanhan%2C+Ziad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">19 These authors contributed equally<br/>20 These authors contributed equally<br/>21 Lead contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.cell.2021.07.016">https://doi.org/10.1016/j.cell.2021.07.016</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.cell.2021.07.016"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.07.016" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">A proteogenomic portrait of lung squamous cell carcinoma</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cell/fulltext/S0092-8674(21)00758-3" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Molecular topography of an entire nervous system</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        

    

        


    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Unsupervised clustering revealed subtype with EMT and phosphoprotein signatures</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Potential therapeutic vulnerabilities included survivin, NSD3, LSD1, and EZH2</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Rb phosphorylation nominated as a biomarker for trials with CDK4/6 inhibitors</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Detailed immune landscape analysis highlighted targetable points of immuneregulation</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/d72c481d-4999-423a-a262-14ecf2c4d607/fx1_lrg.jpg" data-large-image="/cms/attachment/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/08ac60e8-0368-4842-b2c3-062600337bf1/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/d72c481d-4999-423a-a262-14ecf2c4d607/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;lung cancer&#34;&amp;ISSN=0092-8674">lung cancer</a></li><li><a href="/action/doSearch?AllField=&#34;squamous&#34;&amp;ISSN=0092-8674">squamous</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=0092-8674">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=0092-8674">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;genomics&#34;&amp;ISSN=0092-8674">genomics</a></li><li><a href="/action/doSearch?AllField=&#34;protein&#34;&amp;ISSN=0092-8674">protein</a></li><li><a href="/action/doSearch?AllField=&#34;phosphorylation&#34;&amp;ISSN=0092-8674">phosphorylation</a></li><li><a href="/action/doSearch?AllField=&#34;acetylation&#34;&amp;ISSN=0092-8674">acetylation</a></li><li><a href="/action/doSearch?AllField=&#34;ubiquitylation&#34;&amp;ISSN=0092-8674">ubiquitylation</a></li><li><a href="/action/doSearch?AllField=&#34;CPTAC&#34;&amp;ISSN=0092-8674">CPTAC</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">Lung cancer is the leading cause of cancer-associated mortality (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d10391928e1845" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Bray et al., 2018</a><div data-db-target-of="bib15-d10391928e1845" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bray F. </li><li class="loa__item">Ferlay J. </li><li class="loa__item">Soerjomataram I. </li><li class="loa__item">Siegel R.L. </li><li class="loa__item">Torre L.A. </li><li class="loa__item">Jemal A. </li></ul><div class="reference-citations__detials">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries.<div class="popup-ref"><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">394-424</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3322%2Fcaac.21492&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30207593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85053395052&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30524)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&journal=CA+Cancer+J.+Clin.&author=Bray+F.&author=Ferlay+J.&author=Soerjomataram+I.&author=Siegel+R.L.&author=Torre+L.A.&author=Jemal+A.&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib157" id="back-bib157" data-db-target-for="bib157-d10391928e1848" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib157">Siegel et al., 2020</a><div data-db-target-of="bib157-d10391928e1848" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Siegel R.L. </li><li class="loa__item">Miller K.D. </li><li class="loa__item">Jemal A. </li></ul><div class="reference-citations__detials">Cancer statistics, 2020.<div class="popup-ref"><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">7-30</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3322%2Fcaac.21590&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31912902&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077522581&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5586)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2020&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2020">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d10391928e1851" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Hammerman et al., 2012</a><div data-db-target-of="bib54-d10391928e1851" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hammerman P.S. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Voet D. </li><li class="loa__item">Jing R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Gabriel S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib131" id="back-bib131" data-db-target-for="bib131-d10391928e1854" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib131">Paik et al., 2019</a><div data-db-target-of="bib131-d10391928e1854" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Paik P.K. </li><li class="loa__item">Pillai R.N. </li><li class="loa__item">Lathan C.S. </li><li class="loa__item">Velasco S.A. </li><li class="loa__item">Papadimitrakopoulou V. </li></ul><div class="reference-citations__detials">New Treatment Options in Advanced Squamous Cell Lung Cancer.<div class="popup-ref"><span class="ref__series"><i>Am. Soc. Clin. Oncol. Educ. Book.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">e198-e206</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FEDBK_237829&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31099625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=e198-e206&journal=Am.+Soc.+Clin.+Oncol.+Educ.+Book&author=Paik+P.K.&author=Pillai+R.N.&author=Lathan+C.S.&author=Velasco+S.A.&author=Papadimitrakopoulou+V.&title=New+Treatment+Options+in+Advanced+Squamous+Cell+Lung+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as <em>CDKN2A</em>) are found in upward of 60% but targeting these clinically has largely failed (<div class="dropBlock reference-citations"><a href="#bib131" id="back-bib131" data-db-target-for="bib131-d10391928e1860" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib131">Paik et al., 2019</a><div data-db-target-of="bib131-d10391928e1860" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Paik P.K. </li><li class="loa__item">Pillai R.N. </li><li class="loa__item">Lathan C.S. </li><li class="loa__item">Velasco S.A. </li><li class="loa__item">Papadimitrakopoulou V. </li></ul><div class="reference-citations__detials">New Treatment Options in Advanced Squamous Cell Lung Cancer.<div class="popup-ref"><span class="ref__series"><i>Am. Soc. Clin. Oncol. Educ. Book.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">e198-e206</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FEDBK_237829&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31099625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=e198-e206&journal=Am.+Soc.+Clin.+Oncol.+Educ.+Book&author=Paik+P.K.&author=Pillai+R.N.&author=Lathan+C.S.&author=Velasco+S.A.&author=Papadimitrakopoulou+V.&title=New+Treatment+Options+in+Advanced+Squamous+Cell+Lung+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). While other therapeutic regimens are being explored as part of the LungMAP consortium (<div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d10391928e1863" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Ardini-Poleske et al., 2017</a><div data-db-target-of="bib7-d10391928e1863" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ardini-Poleske M.E. </li><li class="loa__item">Clark R.F. </li><li class="loa__item">Ansong C. </li><li class="loa__item">Carson J.P. </li><li class="loa__item">Corley R.A. </li><li class="loa__item">Deutsch G.H. </li><li class="loa__item">Hagood J.S. </li><li class="loa__item">Kaminski N. </li><li class="loa__item">Mariani T.J. </li><li class="loa__item">Potter S.S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>LungMAP Consortium</span><br />LungMAP: The Molecular Atlas of Lung Development Program.<div class="popup-ref"><span class="ref__series"><i>Am. J. Physiol. Lung Cell. Mol. Physiol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">L733-L740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1152%2Fajplung.00139.2017&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28798251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85032899579&amp;cf=" onclick="newWindow(this.href);return false">Scopus (70)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2017&pages=L733-L740&journal=Am.+J.+Physiol.+Lung+Cell.+Mol.+Physiol.&author=LungMAP+Consortium&title=LungMAP%3A+The+Molecular+Atlas+of+Lung+Development+Program">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC (<div class="dropBlock reference-citations"><a href="#bib76" id="back-bib76" data-db-target-for="bib76-d10391928e1866" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Karachaliou et al., 2018</a><div data-db-target-of="bib76-d10391928e1866" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karachaliou N. </li><li class="loa__item">Fernandez-Bruno M. </li><li class="loa__item">Rosell R. </li></ul><div class="reference-citations__detials">Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer?.<div class="popup-ref"><span class="ref__series"><i>Transl. Lung Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">S198-S201</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.21037%2Ftlcr.2018.07.02&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30393600&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054551350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=S198-S201&journal=Transl.+Lung+Cancer+Res.&author=Karachaliou+N.&author=Fernandez-Bruno+M.&author=Rosell+R.&title=Strategies+for+first-line+immunotherapy+in+squamous+cell+lung+cancer%3A+are+combinations+a+game+changer%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib136" id="back-bib136" data-db-target-for="bib136-d10391928e1869" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib136">Paz-Ares et al., 2018</a><div data-db-target-of="bib136-d10391928e1869" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Paz-Ares L. </li><li class="loa__item">Luft A. </li><li class="loa__item">Vicente D. </li><li class="loa__item">Tafreshi A. </li><li class="loa__item">Gümüş M. </li><li class="loa__item">Mazières J. </li><li class="loa__item">Hermes B. </li><li class="loa__item">Çay Şenler F. </li><li class="loa__item">Csőszi T. </li><li class="loa__item">Fülöp A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>KEYNOTE-407 Investigators</span><br />Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">379</span></span>: <span class="ref__seriesPages">2040-2051</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1056%2FNEJMoa1810865&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30280635&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054684704&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1136)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=379&publication_year=2018&pages=2040-2051&journal=N.%C2%A0Engl.+J.+Med.&author=KEYNOTE-407+Investigators&title=Pembrolizumab+plus+Chemotherapy+for+Squamous+Non-Small-Cell+Lung+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification.</div><div class="section-paragraph">We characterized 108 prospectively collected, treatment-naive, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0040" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic landscape of LSCC</span></h3><div class="section-paragraph">We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A, <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">S1</a>A, <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>-3). Proteomics data were generated using TMT11-based multiplexing linked by a common reference (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>A) and demonstrated high technical quality (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>B and S1C). The cohort demographics are summarized in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B. Most self-reported never-smokers displayed high genomic smoking scores (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>D), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e1906" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e1906" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The genomic landscape reaffirmed previously reported somatic alterations in LSCC (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d10391928e1909" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Hammerman et al., 2012</a><div data-db-target-of="bib54-d10391928e1909" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hammerman P.S. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Voet D. </li><li class="loa__item">Jing R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Gabriel S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>E).<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/6e1bb903-9db1-4a42-b84d-24bcf1e623fb/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/917afd4d-4da1-4fe5-87e0-9027d467b78f/gr1_lrg.jpg" data-large-image="/cms/attachment/6e1bb903-9db1-4a42-b84d-24bcf1e623fb/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Proteogenomic landscape of LSCC</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS, whole-genome sequencing; WES, whole-exome sequencing; CNA, copy number alteration; DNAme, DNA methylation; pSTY, phosphoproteome; Ac, acetylproteome; Ub, ubiquitylproteome.</div><div class="section-paragraph">(B) Stacked histograms indicating the distribution of patient phenotypes. Smoking history reflects self-report.</div><div class="section-paragraph">(C) Co-occurring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q value &lt; 0.1) in this dataset are highlighted in red font.</div><div class="section-paragraph">(D) Heatmaps showing correlation between copy number alterations (CNAs) and RNA (left) or proteomics (right). Red and green events represent significant (FDR &lt; 0.01) positive and negative correlations, respectively.</div><div class="section-paragraph">(E) Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q &lt; 0.25) and <em>cis</em>-effects in both mRNA and protein (FDR &lt; 0.05).</div><div class="section-paragraph">(F) Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the <em>FGFR1</em> gene (GISTIC thresholded value = 2).</div><div class="section-paragraph">(G) Genes whose DNA methylation was significantly associated with cascading <em>cis</em> regulation of their cognate mRNA expression, protein level, and phosphopeptide and acetylpeptide abundance. Shapes indicate the <em>cis</em>-effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs.</div><div class="section-paragraph">See also <a data-locator="figs1" aria-expanded="false" aria-controls="crosref2820" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs1">Figure S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2825" href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" target="_blank">Table S1</a>, <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2830" href="/cms/10.1016/j.cell.2021.07.016/attachment/b531dd09-5d15-4a8a-92a4-8cf82c7b7778/mmc2" target="_blank">S2</a>, and <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2835" href="/cms/10.1016/j.cell.2021.07.016/attachment/21d6959b-79c4-4371-84f1-ade0942b9013/mmc3" target="_blank">S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/6e1bb903-9db1-4a42-b84d-24bcf1e623fb/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/917afd4d-4da1-4fe5-87e0-9027d467b78f/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/dbc1ee13-acb8-4fe3-b335-2f46b7475677/figs1.jpg" alt="Figure thumbnail figs1" data-locator="figs1" data-high-res="/cms/attachment/20223a57-c3c7-4e76-8f05-6b027dde0a1a/figs1_lrg.jpg" data-large-image="/cms/attachment/dbc1ee13-acb8-4fe3-b335-2f46b7475677/figs1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S1" class="label figure__label">Figure S1</span><span class="figure__title__text">Experimental design and proteogenomic data quality metrics, related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2925" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Experimental workflow of LSCC tumor and NAT proteogenomic profiling. Samples were cryopulverized, aliquoted for genomic DNA, total RNA and protein extraction, and utilized for DNA sequencing (WGS and WES) and methylation analysis, RNA and miRNA sequencing and proteomics / PTM analysis as indicated. TMT11-based isobaric labeling was used to multiplex up to five tumors (blue filled circles) and paired NATs (pink filled circles) in a single TMT11-plex. The last TMT channel (131C) was the LSCC common reference (CR), comprising peptides from all LSCC tumor and NAT samples included in this study and replicated in every plex to serve as the common denominator for calculating peptide ratios. A total of 300ug peptides/sample was used for proteome, phosphoproteome and acetylproteome analyses via serial enrichment (STAR methods). For the ubiquitylproteome, a different TMT11 experimental design was used (<a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2930" href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" target="_blank">Table S1</a>) for the subset of tumors and NATs with ≥750ug residual peptide amounts after mainstream proteomic processing (85 tumors and 58 NATs). 750ug/sample was used for ubiquitylproteome analysis, with a ubiquitylproteome CR composed of peptides from the 143 available samples. The CR used for proteome, phosphoproteome and acetylproteome and that for ubiquitylproteome showed median Pearson correlations &gt; 0.85 across analyses.</div><div class="section-paragraph">(B) Numbers of quantified proteins, phosphosites, acetylsites, and ubiquitylsites across all TMT11-plexes. The values above each bar indicate percentage deviation from the median.</div><div class="section-paragraph">(C) Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes were consistently higher for CR than for sample channels, enabling precise relative quantification of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as the common denominator. The CRs of three TMT plexes in the ubiquitylproteome data showed lower reproducibility compared to the CRs of other ubiquitylproteome TMT plexes, which was computationally corrected as described in STAR methods (“<span class="underline">Quality control and Batch effect assessment</span>”). Top panel: Boxplots show distributions of delta CR-sample correlations derived from all pairwise TMT plex comparisons across the different proteomic data types. Positive delta correlations indicate higher correlations of the CR channel compared to sample channels. Lower panel: Bar plots represent pairwise TMT plex correlations for proteomics, phosphoproteomics, acetylproteomics and ubiquitylproteomics using TMT plexes 1 and 2 as representative examples. Blue bars depict correlations of sample channels; red bars show the correlation of the CR.</div><div class="section-paragraph">(D) Smoking scores calculated by WES stratified by self-reported smoking status. While some reformed and never-smokers had low scores, many self-reported never-smokers from Slavic countries had scores comparable to those of smokers.</div><div class="section-paragraph">(E) PCA plots showing the lack of batch effect across all plexes in the different data types, with samples from each batch represented by different shapes. A clear separation in the PC1 component is observed between tumors and NATs in each data type.</div><div class="section-paragraph">(F) To identify specific events associated with CNA <em>trans</em>-effects (vertical stripes in <a data-locator="fig1" aria-expanded="false" aria-controls="crosref2935" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a>D), we leveraged CMAP data (<a class="inter-ref-link" href="https://clue.io/cmap" target="_blank">https://clue.io/cmap</a>), which includes over 1.3 million gene expression profiles from ∼20,000 small-molecule compounds and ∼4,300 genetic reagents tested in multiple cell types. Patterns of significant <em>trans</em>-effects associated with CNAs that also had significant <em>cis</em>-effects were queried against CMAP profiles to identify knockout perturbations that significantly resembled those CNA-associated aberrations. Shown for two of the six genes (NRF2F6 and IL18) are correlations between GISTIC log2 CNA ratio or xCell-derived immune score and relative protein expression (Log2 TMT ratio). Pearson correlations and p values are indicated.</div><div class="section-paragraph">(G) Unsupervised consensus clustering results of promoter-level DNA methylation data from tumors and NATs. LSCC tumor DNA showed overall hypermethylation at gene promoters relative to NATs (Wilcoxon p &lt; 2x10<sup>−16</sup>).</div><div class="section-paragraph">(H) Unsupervised consensus clustering of tumor methylation revealed CIMP (CpG island methylator phenotype) -high, -intermediate and -low clusters.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/dbc1ee13-acb8-4fe3-b335-2f46b7475677/figs1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/20223a57-c3c7-4e76-8f05-6b027dde0a1a/figs1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Proteogenomic data helped annotate the impact of copy number alteration (CNA) events (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>D and 1E). Of 5,523 significant (false discovery rate [FDR] &lt; 0.01) <em>cis</em> CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 “cancer associated genes” (CAGs) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>E; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>). We identified six amplified (including <em>WHSC1L1</em>, <em>CCND1</em>, and <em>SOX2</em>) and 29 deleted (including <em>NCOR1</em>, <em>SETD2</em>, and <em>CBL</em>) CAGs in significant focal events (Q &lt; 0.25) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>E; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>) (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d10391928e1952" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Mermel et al., 2011</a><div data-db-target-of="bib120-d10391928e1952" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1222)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Intriguingly, <em>WHSC1L1</em> (<em>NSD3</em>) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses <em>FGFR1</em>. Therapies targeted against FGFR1 have been unsuccessful (<div class="dropBlock reference-citations"><a href="#bib107" id="back-bib107" data-db-target-for="bib107-d10391928e1961" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib107">Lim et al., 2016</a><div data-db-target-of="bib107-d10391928e1961" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lim S.H. </li><li class="loa__item">Sun J.-M. </li><li class="loa__item">Choi Y.-L. </li><li class="loa__item">Kim H.R. </li><li class="loa__item">Ahn S. </li><li class="loa__item">Lee J.Y. </li><li class="loa__item">Lee S.-H. </li><li class="loa__item">Ahn J.S. </li><li class="loa__item">Park K. </li><li class="loa__item">Kim J.H. </li><li>et al.</li></ul><div class="reference-citations__detials">Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study.<div class="popup-ref"><span class="ref__series"><i>Cancer.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">122</span></span>: <span class="ref__seriesPages">3024-3031</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fcncr.30135&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27315356&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=3024-3031&journal=Cancer&author=Lim+S.H.&author=Sun+J.-M.&author=Choi+Y.-L.&author=Kim+H.R.&author=Ahn+S.&author=Lee+J.Y.&author=Lee+S.-H.&author=Ahn+J.S.&author=Park+K.&author=Kim+J.H.&title=Efficacy+and+safety+of+dovitinib+in+pretreated+patients+with+advanced+squamous+non-small+cell+lung+cancer+with+FGFR1+amplification%3A+A+single-arm%2C+phase+2+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib189" id="back-bib189" data-db-target-for="bib189-d10391928e1965" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib189">Weeden et al., 2015</a><div data-db-target-of="bib189-d10391928e1965" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Weeden C.E. </li><li class="loa__item">Solomon B. </li><li class="loa__item">Asselin-Labat M.-L. </li></ul><div class="reference-citations__detials">FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies.<div class="popup-ref"><span class="ref__series"><i>Cell Death Discov.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">15049</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_199_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fcddiscovery.2015.49&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_199_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27551478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=15049&journal=Cell+Death+Discov.&author=Weeden+C.E.&author=Solomon+B.&author=Asselin-Labat+M.-L.&title=FGFR1+inhibition+in+lung+squamous+cell+carcinoma%3A+questions+and+controversies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>F), nominating it as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes (<div class="dropBlock reference-citations"><a href="#bib205" id="back-bib205" data-db-target-for="bib205-d10391928e1971" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib205">Yuan et al., 2021</a><div data-db-target-of="bib205-d10391928e1971" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yuan G. </li><li class="loa__item">Flores N.M. </li><li class="loa__item">Hausmann S. </li><li class="loa__item">Lofgren S.M. </li><li class="loa__item">Kharchenko V. </li><li class="loa__item">Angulo-Ibanez M. </li><li class="loa__item">Sengupta D. </li><li class="loa__item">Lu X. </li><li class="loa__item">Czaban I. </li><li class="loa__item">Azhibek D. </li><li>et al.</li></ul><div class="reference-citations__detials">Elevated NSD3 histone methylation activity drives squamous cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">590</span></span>: <span class="ref__seriesPages">504-508</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-020-03170-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33536620&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85100414615&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=590&publication_year=2021&pages=504-508&journal=Nature&author=Yuan+G.&author=Flores+N.M.&author=Hausmann+S.&author=Lofgren+S.M.&author=Kharchenko+V.&author=Angulo-Ibanez+M.&author=Sengupta+D.&author=Lu+X.&author=Czaban+I.&author=Azhibek+D.&title=Elevated+NSD3+histone+methylation+activity+drives+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant <em>trans</em>-effects (vertical stripes in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>D) to perturbation profiles from the Connectivity Map (CMap) (<a class="inter-ref-link" href="https://clue.io/cmap" target="_blank">https://clue.io/cmap</a>). Six CNAs (<em>IKBKAP</em>, <em>PIN1</em>, <em>DNAJB1</em>, <em>IL18</em>, <em>NR2F6</em>, <em>AKAP</em>) (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>) were enriched for both amplification and deletion (FDR = 0.06) and associated (Fisher’s exact p &lt; 0.01) with clinical metadata. <em>IL-18</em> deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>F, upper panel), consistent with its playing a key modulatory role in the tumor microenvironment of non-small cell lung cancer (NSCLC) (<div class="dropBlock reference-citations"><a href="#bib174" id="back-bib174" data-db-target-for="bib174-d10391928e2006" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib174">Timperi et al., 2017</a><div data-db-target-of="bib174-d10391928e2006" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Timperi E. </li><li class="loa__item">Focaccetti C. </li><li class="loa__item">Gallerano D. </li><li class="loa__item">Panetta M. </li><li class="loa__item">Spada S. </li><li class="loa__item">Gallo E. </li><li class="loa__item">Visca P. </li><li class="loa__item">Venuta F. </li><li class="loa__item">Diso D. </li><li class="loa__item">Prelaj A. </li><li>et al.</li></ul><div class="reference-citations__detials">IL-18 receptor marks functional CD8<sup>+</sup> T cells in non-small cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>OncoImmunology.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">e1328337</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1080%2F2162402X.2017.1328337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28811967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85021447409&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=e1328337&journal=OncoImmunology&author=Timperi+E.&author=Focaccetti+C.&author=Gallerano+D.&author=Panetta+M.&author=Spada+S.&author=Gallo+E.&author=Visca+P.&author=Venuta+F.&author=Diso+D.&author=Prelaj+A.&title=IL-18+receptor+marks+functional+CD8%2B+T%C2%A0cells+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). <em>NR2F6</em> amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>F, lower panel). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade (<div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d10391928e2014" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Klepsch et al., 2018</a><div data-db-target-of="bib83-d10391928e2014" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Klepsch V. </li><li class="loa__item">Hermann-Kleiter N. </li><li class="loa__item">Do-Dinh P. </li><li class="loa__item">Jakic B. </li><li class="loa__item">Offermann A. </li><li class="loa__item">Efremova M. </li><li class="loa__item">Sopper S. </li><li class="loa__item">Rieder D. </li><li class="loa__item">Krogsdam A. </li><li class="loa__item">Gamerith G. </li><li>et al.</li></ul><div class="reference-citations__detials">Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1538</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-018-04004-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29670099&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85045763585&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1538&journal=Nat.+Commun.&author=Klepsch+V.&author=Hermann-Kleiter+N.&author=Do-Dinh+P.&author=Jakic+B.&author=Offermann+A.&author=Efremova+M.&author=Sopper+S.&author=Rieder+D.&author=Krogsdam+A.&author=Gamerith+G.&title=Nuclear+receptor+NR2F6+inhibition+potentiates+responses+to+PD-L1%2FPD-1+cancer+immune+checkpoint+blockade">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">LSCC tumor DNA showed overall hypermethylation relative to NATs (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>G), with tumors separating into CIMP (CpG island methylator phenotype) clusters (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>H). Multi-omic data allowed the identification of “cascading” promoter methylation <em>cis</em>-effects across cognate mRNA, protein, and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR &lt; 0.1) <em>cis</em>-effects in tumors (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>), 20 were also differentially expressed (FDR &lt; 0.01) between tumors and NATs (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>G). While most were newly associated with LSCC, methylation-driven expression of <em>FAM110A</em>, <em>PTGES3</em>, <em>PLAU</em>, and <em>SLC16A3</em> (<div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d10391928e2044" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Faubert et al., 2017</a><div data-db-target-of="bib35-d10391928e2044" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Faubert B. </li><li class="loa__item">Li K.Y. </li><li class="loa__item">Cai L. </li><li class="loa__item">Hensley C.T. </li><li class="loa__item">Kim J. </li><li class="loa__item">Zacharias L.G. </li><li class="loa__item">Yang C. </li><li class="loa__item">Do Q.N. </li><li class="loa__item">Doucette S. </li><li class="loa__item">Burguete D. </li><li>et al.</li></ul><div class="reference-citations__detials">Lactate Metabolism in Human Lung Tumors.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">358-371.e9</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28985563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85030560166&amp;cf=" onclick="newWindow(this.href);return false">Scopus (414)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=358-371.e9&journal=Cell&author=Faubert+B.&author=Li+K.Y.&author=Cai+L.&author=Hensley+C.T.&author=Kim+J.&author=Zacharias+L.G.&author=Yang+C.&author=Do+Q.N.&author=Doucette+S.&author=Burguete+D.&title=Lactate+Metabolism+in+Human+Lung+Tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d10391928e2048" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Gao et al., 2018</a><div data-db-target-of="bib44-d10391928e2048" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao C. </li><li class="loa__item">Zhuang J. </li><li class="loa__item">Li H. </li><li class="loa__item">Liu C. </li><li class="loa__item">Zhou C. </li><li class="loa__item">Liu L. </li><li class="loa__item">Sun C. </li></ul><div class="reference-citations__detials">Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell Int.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">194</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs12935-018-0691-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30498398&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85057211952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=194&journal=Cancer+Cell+Int.&author=Gao+C.&author=Zhuang+J.&author=Li+H.&author=Liu+C.&author=Zhou+C.&author=Liu+L.&author=Sun+C.&title=Exploration+of+methylation-driven+genes+for+monitoring+and+prognosis+of+patients+with+lung+adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d10391928e2051" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Kikuchi et al., 2012</a><div data-db-target-of="bib79-d10391928e2051" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kikuchi T. </li><li class="loa__item">Hassanein M. </li><li class="loa__item">Amann J.M. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Rahman S.M.J. </li><li class="loa__item">Kaufman J.M. </li><li class="loa__item">Zhang X. </li><li class="loa__item">Hoeksema M.D. </li><li class="loa__item">Harris B.K. </li><li>et al.</li></ul><div class="reference-citations__detials">In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">916-932</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22761400&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84867148065&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=916-932&journal=Mol.+Cell.+Proteomics&author=Kikuchi+T.&author=Hassanein+M.&author=Amann+J.M.&author=Liu+Q.&author=Slebos+R.J.C.&author=Rahman+S.M.J.&author=Kaufman+J.M.&author=Zhang+X.&author=Hoeksema+M.D.&author=Harris+B.K.&title=In-depth+proteomic+analysis+of+nonsmall+cell+lung+cancer+to+discover+molecular+targets+and+candidate+biomarkers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d10391928e2054" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Liang et al., 2013</a><div data-db-target-of="bib104-d10391928e2054" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liang Y. </li><li class="loa__item">Liu M. </li><li class="loa__item">Wang P. </li><li class="loa__item">Ding X. </li><li class="loa__item">Cao Y. </li></ul><div class="reference-citations__detials">Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Genes Chromosomes Cancer.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">305-315</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fgcc.22030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23225332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872424747&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=305-315&journal=Genes+Chromosomes+Cancer&author=Liang+Y.&author=Liu+M.&author=Wang+P.&author=Ding+X.&author=Cao+Y.&title=Analysis+of+20+genes+at+chromosome+band+12q13%3A+RACGAP1+and+MCRS1+overexpression+in+nonsmall-cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d10391928e2057" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Payen et al., 2020</a><div data-db-target-of="bib135-d10391928e2057" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Payen V.L. </li><li class="loa__item">Mina E. </li><li class="loa__item">Van Hée V.F. </li><li class="loa__item">Porporato P.E. </li><li class="loa__item">Sonveaux P. </li></ul><div class="reference-citations__detials">Monocarboxylate transporters in cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Metab.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">48-66</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molmet.2019.07.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395464&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077249276&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=48-66&journal=Mol.+Metab.&author=Payen+V.L.&author=Mina+E.&author=Van+H%C3%A9e+V.F.&author=Porporato+P.E.&author=Sonveaux+P.&title=Monocarboxylate+transporters+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib156" id="back-bib156" data-db-target-for="bib156-d10391928e2060" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib156">Showe et al., 2009</a><div data-db-target-of="bib156-d10391928e2060" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Showe M.K. </li><li class="loa__item">Vachani A. </li><li class="loa__item">Kossenkov A.V. </li><li class="loa__item">Yousef M. </li><li class="loa__item">Nichols C. </li><li class="loa__item">Nikonova E.V. </li><li class="loa__item">Chang C. </li><li class="loa__item">Kucharczuk J. </li><li class="loa__item">Tran B. </li><li class="loa__item">Wakeam E. </li><li>et al.</li></ul><div class="reference-citations__detials">Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">9202-9210</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-09-1378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19951989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-73649083403&amp;cf=" onclick="newWindow(this.href);return false">Scopus (120)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=9202-9210&journal=Cancer+Res.&author=Showe+M.K.&author=Vachani+A.&author=Kossenkov+A.V.&author=Yousef+M.&author=Nichols+C.&author=Nikonova+E.V.&author=Chang+C.&author=Kucharczuk+J.&author=Tran+B.&author=Wakeam+E.&title=Gene+expression+profiles+in+peripheral+blood+mononuclear+cells+can+distinguish+patients+with+non-small+cell+lung+cancer+from+patients+with+nonmalignant+lung+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) has been implicated in lung and other cancers.</div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-enriched</span></h3><div class="section-paragraph">We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein, and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the entire cohort (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A). The five resulting multi-omic subtypes (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B ; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>A) were named based on their predominant pathway associations and similarities to previously defined RNA clusters (<div class="dropBlock reference-citations"><a href="#bib196" id="back-bib196" data-db-target-for="bib196-d10391928e2080" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib196">Wilkerson et al., 2010</a><div data-db-target-of="bib196-d10391928e2080" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Yin X. </li><li class="loa__item">Hoadley K.A. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Muldrew K. </li><li class="loa__item">Miller C.R. </li><li class="loa__item">Randell S.H. </li><li class="loa__item">Socinski M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4864-4875</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-10-0199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20643781&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77957570386&amp;cf=" onclick="newWindow(this.href);return false">Scopus (163)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4864-4875&journal=Clin.+Cancer+Res.&author=Wilkerson+M.D.&author=Yin+X.&author=Hoadley+K.A.&author=Liu+Y.&author=Hayward+M.C.&author=Cabanski+C.R.&author=Muldrew+K.&author=Miller+C.R.&author=Randell+S.H.&author=Socinski+M.A.&title=Lung+squamous+cell+carcinoma+mRNA+expression+subtypes+are+reproducible%2C+clinically+important%2C+and+correspond+to+normal+cell+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Significant associations between NMF subtypes and cohort metadata are presented in <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>. Two NMF subtypes, “basal-inclusive” (B-I) and “epithelial to mesenchymal transition-enriched” (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA basal cluster, although only B-I retained significant association after Bonferroni correction (Fisher’s p &lt; 0.01; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>). B-I tumors showed basaloid histology and upregulated metabolic, immune, and estrogen receptor signaling (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B). EMT-E showed upregulation of EMT, angiogenesis, and myogenesis (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in <em>KEAP1</em>, <em>CUL3</em>, and <em>NFE2L2</em> genes and high-level amplification (GISTIC threshold value = 2) of <em>SOX2</em> and <em>TP63</em> (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/b531dd09-5d15-4a8a-92a4-8cf82c7b7778/mmc2" class="scroll-into-link">Table S2</a>) was labeled the “classical” subtype, consistent with TCGA nomenclature and with a previous publication (<div class="dropBlock reference-citations"><a href="#bib164" id="back-bib164" data-db-target-for="bib164-d10391928e2113" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib164">Stewart et al., 2019</a><div data-db-target-of="bib164-d10391928e2113" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stewart P.A. </li><li class="loa__item">Welsh E.A. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Fang B. </li><li class="loa__item">Izumi V. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Zhang G. </li><li class="loa__item">Cen L. </li><li class="loa__item">Pettersson F. </li><li class="loa__item">Zhang Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic landscape of squamous cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">3578</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-11452-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395880&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85070354617&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3578&journal=Nat.+Commun.&author=Stewart+P.A.&author=Welsh+E.A.&author=Slebos+R.J.C.&author=Fang+B.&author=Izumi+V.&author=Chambers+M.&author=Zhang+G.&author=Cen+L.&author=Pettersson+F.&author=Zhang+Y.&title=Proteogenomic+landscape+of+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B). A fourth NMF subtype was designated “inflamed-secretory” (I-S) due to its alignment with the RNA-based secretory cluster and strong upregulation of immune-related pathways. The fifth, “proliferative-primitive” (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a “mixed” subgroup, since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>C), an observation also made in a recent proteogenomic analysis of glioblastoma (<div class="dropBlock reference-citations"><a href="#bib188" id="back-bib188" data-db-target-for="bib188-d10391928e2123" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib188">Wang et al., 2021</a><div data-db-target-of="bib188-d10391928e2123" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang L.-B. </li><li class="loa__item">Karpova A. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Kyle J.E. </li><li class="loa__item">Cao S. </li><li class="loa__item">Li Y. </li><li class="loa__item">Rykunov D. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Rothstein J.H. </li><li class="loa__item">Hong R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic and metabolomic characterization of human glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">509-528.e20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33577785&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85101666510&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=509-528.e20&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+and+metabolomic+characterization+of+human+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The mixed class was associated in the current cohort with increased frequency of <em>SOX2</em> amplifications (p = 0.0038) but not grade, mutation burden, stemness, or other tested variables.<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/35bb8996-5a84-48a5-a5c9-aacdb938b2f8/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/614ac304-de46-4041-a8f2-13113fa21e4d/gr2_lrg.jpg" data-large-image="/cms/attachment/35bb8996-5a84-48a5-a5c9-aacdb938b2f8/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">LSCC molecular subtypes and associations</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) NMF-based clustering of tumor CNA, RNA, protein, phosphosite, and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row).</div><div class="section-paragraph">(B) Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes.</div><div class="section-paragraph">(C) Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed).</div><div class="section-paragraph">(D) Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others.</div><div class="section-paragraph">(E) Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO, Gene Ontology) genesets (Pearson correlation = 0.65, p = 2.8x10<sup>−14</sup>).</div><div class="section-paragraph">(F) Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E versus other subtypes are 1.5x10<sup>−6</sup> for PDGFRB and 2.7x10<sup>−7</sup> for ROR2.</div><div class="section-paragraph">(G) Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles.</div><div class="section-paragraph">See also <a data-locator="figs2" aria-expanded="false" aria-controls="crosref2840" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref3080" href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" target="_blank">Tables S1</a>, <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2850" href="/cms/10.1016/j.cell.2021.07.016/attachment/b531dd09-5d15-4a8a-92a4-8cf82c7b7778/mmc2" target="_blank">S2</a>, <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2855" href="/cms/10.1016/j.cell.2021.07.016/attachment/21d6959b-79c4-4371-84f1-ade0942b9013/mmc3" target="_blank">S3</a>, and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2860" href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" target="_blank">S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/35bb8996-5a84-48a5-a5c9-aacdb938b2f8/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/614ac304-de46-4041-a8f2-13113fa21e4d/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/89e3c668-cf42-4914-9ff1-7a1feebe97a1/figs2.jpg" alt="Figure thumbnail figs2" data-locator="figs2" data-high-res="/cms/attachment/edf630c5-521c-4c2b-8ef1-991bcd545593/figs2_lrg.jpg" data-large-image="/cms/attachment/89e3c668-cf42-4914-9ff1-7a1feebe97a1/figs2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S2" class="label figure__label">Figure S2</span><span class="figure__title__text">Characteristics of multi-omic NMF subtypes, related to <a data-locator="fig2" aria-expanded="false" aria-controls="crosref2940" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Upper panel: Line plot representing cophenetic correlation coefficients (y axis) calculated for a range of factorization ranks (x axis). The maximal cophenetic correlation coefficient was observed for rank K = 5 as highlighted by the red box. Lower panel: silhouette plot for K = 5. This plot indicates the quality of cluster separation.</div><div class="section-paragraph">(B) Directionality and significance (signed Wilcoxon p value) of abundance changes in RNA, protein (green), and PTMs (phosphosites in purple, acetylsites in blue, ubiquitylsites in yellow) between the Basal-inclusive and other NMF subtypes. Identities of the most extreme outliers are designated and the modification site (K- lysine, S- Serine, T- Threonine, Y- Tyrosine) is separated by an underscore. PBX3 is highlighted by a red box.</div><div class="section-paragraph">(C) Same as B but in the EMT-enriched subtype.</div><div class="section-paragraph">(D) Pathway analysis of proteins with high correlation with PBX3 protein expression (FDR &lt; 0.05 and Pearson correlation &gt; 0.5) revealed neutrophil activation and neutrophil degranulation as the top protein-driven pathways.</div><div class="section-paragraph">(E) Scatterplot showing Pearson correlations between PBX3 protein and other transcripts and PBX3 protein and other proteins. Points represent the signed -log10 (p value). Neutrophil degranulation-associated genes show very high correlation with PBX3 protein only at the level of protein expression, whereas neutrophil chemotactic factors such as CXCL1 and 8 show high correlation at both RNA and protein levels.</div><div class="section-paragraph">(F) Comparison of xCell-derived Immune, Microenvironment, Neutrophil and Stroma scores across five NMF subtypes. Z-score values are used for the boxplots. P values are derived from the Wilcoxon test. B-I - Basal-inclusive, C - Classical, E-E - EMT-enriched, I-S - Inflamed-Secretory, P-P - Proliferative-Primitive.</div><div class="section-paragraph">(G) Representative dual immunohistochemical staining of α-smooth muscle actin (α-SMA) and pan-Cytokeratin (CK) of two samples corresponding to EMT-E and Classical subtypes.</div><div class="section-paragraph">(H) Normalized connectivity scores (NCS) between the Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and protein level are represented in the heatmap. Each NMF signature was used to query the CMAP LINCS L1000 resource (2020 release), a collection of more than 1M response signatures of cancer cell lines to different types of chemical (e.g., drugs, tool compounds) and genetic perturbations (e.g shRNAs, sgRNAs, cDNAs). The 20 most negatively connected compounds to each NMF signature (with the most negative NCS) were then selected for visualization. Rows were ordered based on the EMT-E RNA signature indicating an enrichment for TGF-b inhibitors in the negatively-connected compounds of the EMT-E cluster.</div><div class="section-paragraph">(I) Integrative analysis of the cumulative 216 treatment-naive LSCC tumors between this and the Stewart study (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib164" id="back-bib164" data-db-target-for="bib164-d10391928e378" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib164">Stewart et al., 2019</a><div data-db-target-of="bib164-d10391928e378" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stewart P.A. </li><li class="loa__item">Welsh E.A. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Fang B. </li><li class="loa__item">Izumi V. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Zhang G. </li><li class="loa__item">Cen L. </li><li class="loa__item">Pettersson F. </li><li class="loa__item">Zhang Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic landscape of squamous cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">3578</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-11452-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395880&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85070354617&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3578&journal=Nat.+Commun.&author=Stewart+P.A.&author=Welsh+E.A.&author=Slebos+R.J.C.&author=Fang+B.&author=Izumi+V.&author=Chambers+M.&author=Zhang+G.&author=Cen+L.&author=Pettersson+F.&author=Zhang+Y.&title=Proteogenomic+landscape+of+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>), focused on the global proteome in the absence of global gene copy number and post-translational modification data in that study. Our NMF-based pipeline applied to the integrated proteomics dataset (<a data-locator="sec4" aria-expanded="false" aria-controls="crosref2950" class=" caption__link scroll-into-link" href="&#xA;                                #sec4">STAR Methods</a>) identified six clusters that partially aligned with RNA-based clusters as well as with proteomic and multi-omic subtypes defined in this study (<a data-locator="figs2" aria-expanded="false" aria-controls="crosref2955" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a>A). As expected, the Stewart et al., Redox B proteomics cluster samples joined with the NMF Classical subtype, enriched for NRF2 pathway mutations and redox biology. Notably, their Inflamed A and B and our Inflamed-Secretory tumors were each enriched in a separate cluster, their Inflamed A joining our Basal-Inclusive and their Inflamed B defining a cluster that was enriched for samples specifically from their cohort and for <em>TP53</em> WT status, and that shared enrichment for secretory RNA status with our separately clustered Inflamed-Secretory tumors.</div><div class="section-paragraph">(J) Significantly enriched pathways (MSigDB Hallmark) in one or more of the six clusters shown in <a data-locator="figs2" aria-expanded="false" aria-controls="crosref2960" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a>I. Yellow-to-blue color scale depicts positive to negative normalized enrichment scores (NES), with asterisks denoting significant pathways (FDR &lt; 0.05) among the subtypes. At the pathway level, their Inflamed B had signals of coagulation and complement activity, but lacked the broader evidence of immune signaling associated with the Inflamed A and Inflamed-Secretory-enriched clusters (<a data-locator="figs2" aria-expanded="false" aria-controls="crosref2965" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a>I). The Proliferative-Primitive subset we defined retained its integrity and association with Primitive RNA status in the combined analysis. Finally, the EMT-E cluster defined in our study was clearly enriched with the mixed subtype samples from the Stewart et al., study. In addition to very strong EMT signaling, that cluster had evidence of hypoxemia and strong downregulation of oxidative phosphorylation (<a data-locator="figs2" aria-expanded="false" aria-controls="crosref2970" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a>I), known drivers of EMT (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d10391928e400" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Cannito et al., 2008</a><div data-db-target-of="bib22-d10391928e400" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cannito S. </li><li class="loa__item">Novo E. </li><li class="loa__item">Compagnone A. </li><li class="loa__item">Valfrè di Bonzo L. </li><li class="loa__item">Busletta C. </li><li class="loa__item">Zamara E. </li><li class="loa__item">Paternostro C. </li><li class="loa__item">Povero D. </li><li class="loa__item">Bandino A. </li><li class="loa__item">Bozzo F. </li><li>et al.</li></ul><div class="reference-citations__detials">Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2267-2278</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fcarcin%2Fbgn216&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18791199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-57349164238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (242)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=2267-2278&journal=Carcinogenesis&author=Cannito+S.&author=Novo+E.&author=Compagnone+A.&author=Valfr%C3%A8+di+Bonzo+L.&author=Busletta+C.&author=Zamara+E.&author=Paternostro+C.&author=Povero+D.&author=Bandino+A.&author=Bozzo+F.&title=Redox+mechanisms+switch+on+hypoxia-dependent+epithelial-mesenchymal+transition+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) and of Wnt signaling (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d10391928e403" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Hong et al., 2017</a><div data-db-target-of="bib62-d10391928e403" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hong C.-F. </li><li class="loa__item">Chen W.-Y. </li><li class="loa__item">Wu C.-W. </li></ul><div class="reference-citations__detials">Upregulation of Wnt signaling under hypoxia promotes lung cancer progression.<div class="popup-ref"><span class="ref__series"><i>Oncol. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">1706-1714</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3892%2For.2017.5807&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28713928&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85026356039&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=1706-1714&journal=Oncol.+Rep.&author=Hong+C.-F.&author=Chen+W.-Y.&author=Wu+C.-W.&title=Upregulation+of+Wnt+signaling+under+hypoxia+promotes+lung+cancer+progression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>).</div><div class="section-paragraph">(K) A schematic representation of the proposed crosstalk between non-canonical WNT5A/ROR2 and PDGFRB signaling axes. The binding of WNT5A to ROR2 activates the planar cell polarity pathway, shown in yellow ovals (DVL/RAC/RHO/JNK), and the Ca<sup>2+</sup> pathway, shown in blue ovals (PKCa) involving cytoskeleton rearrangements and cell migration. Upregulation of RAC requires clathrin-mediated ROR2 receptor internalization of the Wnt signalosome (AP2, DVL, ROR2). WNT5A/ROR2 also activates PI3K/AKT signaling (green ovals) leading to RUNX2-mediated transcription of EMT (ZEB1, MMP2) and cell invasion (WNT5A, FLNa, ITG5A) programs (red ovals). Targeted therapeutics against WNT5A/ROR2 upregulation are shown in red, with some of them in clinical trials as indicated. Red arrows represent upregulation in the EMT-E proteome and phosphoproteome. Genes directly observed as upregulated at protein/phosphoprotein levels are in thick oval dark blue outlines.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/89e3c668-cf42-4914-9ff1-7a1feebe97a1/figs2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/edf630c5-521c-4c2b-8ef1-991bcd545593/figs2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We compared B-I (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>B) and EMT-E (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 (<div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d10391928e2136" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Happel et al., 2009</a><div data-db-target-of="bib56-d10391928e2136" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Happel N. </li><li class="loa__item">Stoldt S. </li><li class="loa__item">Schmidt B. </li><li class="loa__item">Doenecke D. </li></ul><div class="reference-citations__detials">M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3.<div class="popup-ref"><span class="ref__series"><i>J. Mol. Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">386</span></span>: <span class="ref__seriesPages">339-350</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jmb.2008.12.047&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19136008&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-59149103350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=339-350&journal=J.%C2%A0Mol.+Biol.&author=Happel+N.&author=Stoldt+S.&author=Schmidt+B.&author=Doenecke+D.&title=M+phase-specific+phosphorylation+of+histone+H1.5+at+threonine+10+by+GSK-3">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis (<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d10391928e2139" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Hsu et al., 2020</a><div data-db-target-of="bib64-d10391928e2139" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hsu E.-C. </li><li class="loa__item">Rice M.A. </li><li class="loa__item">Bermudez A. </li><li class="loa__item">Marques F.J.G. </li><li class="loa__item">Aslan M. </li><li class="loa__item">Liu S. </li><li class="loa__item">Ghoochani A. </li><li class="loa__item">Zhang C.A. </li><li class="loa__item">Chen Y.-S. </li><li class="loa__item">Zlitni A. </li><li>et al.</li></ul><div class="reference-citations__detials">Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">117</span></span>: <span class="ref__seriesPages">2032-2042</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1905384117&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31932422&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078684513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2032-2042&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Hsu+E.-C.&author=Rice+M.A.&author=Bermudez+A.&author=Marques+F.J.G.&author=Aslan+M.&author=Liu+S.&author=Ghoochani+A.&author=Zhang+C.A.&author=Chen+Y.-S.&author=Zlitni+A.&title=Trop2+is+a+driver+of+metastatic+prostate+cancer+with+neuroendocrine+phenotype+via+PARP1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d10391928e2142" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Hubaux et al., 2015</a><div data-db-target-of="bib67-d10391928e2142" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hubaux R. </li><li class="loa__item">Thu K.L. </li><li class="loa__item">Vucic E.A. </li><li class="loa__item">Pikor L.A. </li><li class="loa__item">Kung S.H.Y. </li><li class="loa__item">Martinez V.D. </li><li class="loa__item">Mosslemi M. </li><li class="loa__item">Becker-Santos D.D. </li><li class="loa__item">Gazdar A.F. </li><li class="loa__item">Lam S. </li><li class="loa__item">Lam W.L. </li></ul><div class="reference-citations__detials">Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Int. J. Cancer.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">137</span></span>: <span class="ref__seriesPages">2072-2082</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fijc.29577&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25907283&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84939588821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2072-2082&journal=Int.+J.+Cancer&author=Hubaux+R.&author=Thu+K.L.&author=Vucic+E.A.&author=Pikor+L.A.&author=Kung+S.H.Y.&author=Martinez+V.D.&author=Mosslemi+M.&author=Becker-Santos+D.D.&author=Gazdar+A.F.&author=Lam+S.&author=Lam+W.L.&title=Microtubule+affinity-regulating+kinase+2+is+associated+with+DNA+damage+response+and+cisplatin+resistance+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib99" id="back-bib99" data-db-target-for="bib99-d10391928e2146" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Lewandowski and Piwnica-Worms, 2014</a><div data-db-target-of="bib99-d10391928e2146" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lewandowski K.T. </li><li class="loa__item">Piwnica-Worms H. </li></ul><div class="reference-citations__detials">Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity.<div class="popup-ref"><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">127</span></span>: <span class="ref__seriesPages">315-327</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.129148&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24259665&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84892555914&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2014&pages=315-327&journal=J.%C2%A0Cell+Sci.&author=Lewandowski+K.T.&author=Piwnica-Worms+H.&title=Phosphorylation+of+the+E3+ubiquitin+ligase+RNF41+by+the+kinase+Par-1b+is+required+for+epithelial+cell+polarity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Transcription activator PBX3, which is upregulated only at the protein level (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>B), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d10391928e2152" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Lamprecht et al., 2018</a><div data-db-target-of="bib93-d10391928e2152" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamprecht S. </li><li class="loa__item">Kaller M. </li><li class="loa__item">Schmidt E.M. </li><li class="loa__item">Blaj C. </li><li class="loa__item">Schiergens T.S. </li><li class="loa__item">Engel J. </li><li class="loa__item">Jung A. </li><li class="loa__item">Hermeking H. </li><li class="loa__item">Grünewald T.G.P. </li><li class="loa__item">Kirchner T. </li><li class="loa__item">Horst D. </li></ul><div class="reference-citations__detials">PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1974-1986</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-17-2572&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29391352&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85047860831&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1974-1986&journal=Clin.+Cancer+Res.&author=Lamprecht+S.&author=Kaller+M.&author=Schmidt+E.M.&author=Blaj+C.&author=Schiergens+T.S.&author=Engel+J.&author=Jung+A.&author=Hermeking+H.&author=Gr%C3%BCnewald+T.G.P.&author=Kirchner+T.&author=Horst+D.&title=PBX3+Is+Part+of+an+EMT+Regulatory+Network+and+Indicates+Poor+Outcome+in+Colorectal+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) including in laryngeal squamous cell carcinoma (<div class="dropBlock reference-citations"><a href="#bib199" id="back-bib199" data-db-target-for="bib199-d10391928e2155" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib199">Wu et al., 2020</a><div data-db-target-of="bib199-d10391928e2155" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wu Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zheng X. </li><li class="loa__item">Dai F. </li><li class="loa__item">Lu Y. </li><li class="loa__item">Dai L. </li><li class="loa__item">Niu M. </li><li class="loa__item">Guo H. </li><li class="loa__item">Li W. </li><li class="loa__item">Xue X. </li><li>et al.</li></ul><div class="reference-citations__detials">Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">99</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs12943-020-01215-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32487167&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85085909077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=99&journal=Mol.+Cancer&author=Wu+Y.&author=Zhang+Y.&author=Zheng+X.&author=Dai+F.&author=Lu+Y.&author=Dai+L.&author=Niu+M.&author=Guo+H.&author=Li+W.&author=Xue+X.&title=Circular+RNA+circCORO1C+promotes+laryngeal+squamous+cell+carcinoma+progression+by+modulating+the+let-7c-5p%2FPBX3+axis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>D), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>E) and with the xCell-based deconvolution signature (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>F). This correlation was not observed in paired NATs, suggesting a tumor-specific role.</div><div class="section-paragraph">In addition to a strong EMT signature (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B), the EMT-E subtype showed upregulated VCAN (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C), a tumor-promoting target of SNAIL (<div class="dropBlock reference-citations"><a href="#bib207" id="back-bib207" data-db-target-for="bib207-d10391928e2176" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib207">Zhang et al., 2019a</a><div data-db-target-of="bib207-d10391928e2176" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang Y. </li><li class="loa__item">Zou X. </li><li class="loa__item">Qian W. </li><li class="loa__item">Weng X. </li><li class="loa__item">Zhang L. </li><li class="loa__item">Zhang L. </li><li class="loa__item">Wang S. </li><li class="loa__item">Cao X. </li><li class="loa__item">Ma L. </li><li class="loa__item">Wei G. </li><li>et al.</li></ul><div class="reference-citations__detials">Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis.<div class="popup-ref"><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">565-579</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41418-018-0147-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29955124&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85049124890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=565-579&journal=Cell+Death+Differ.&author=Zhang+Y.&author=Zou+X.&author=Qian+W.&author=Weng+X.&author=Zhang+L.&author=Zhang+L.&author=Wang+S.&author=Cao+X.&author=Ma+L.&author=Wei+G.&title=Enhanced+PAPSS2%2FVCAN+sulfation+axis+is+essential+for+Snail-mediated+breast+cancer+cell+migration+and+metastasis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) expressed in cancer-associated fibroblasts (CAFs) (<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d10391928e2179" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Chida et al., 2016</a><div data-db-target-of="bib24-d10391928e2179" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chida S. </li><li class="loa__item">Okayama H. </li><li class="loa__item">Noda M. </li><li class="loa__item">Saito K. </li><li class="loa__item">Nakajima T. </li><li class="loa__item">Aoto K. </li><li class="loa__item">Hayase S. </li><li class="loa__item">Momma T. </li><li class="loa__item">Ohki S. </li><li class="loa__item">Kono K. </li><li class="loa__item">Takenoshita S. </li></ul><div class="reference-citations__detials">Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer.<div class="popup-ref"><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">878-887</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fcarcin%2Fbgw069&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27287872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84991037068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=878-887&journal=Carcinogenesis&author=Chida+S.&author=Okayama+H.&author=Noda+M.&author=Saito+K.&author=Nakajima+T.&author=Aoto+K.&author=Hayase+S.&author=Momma+T.&author=Ohki+S.&author=Kono+K.&author=Takenoshita+S.&title=Stromal+VCAN+expression+as+a+potential+prognostic+biomarker+for+disease+recurrence+in+stage+II-III+colon+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and FHL3, a protein that stabilizes EMT-associated transcription factors (<div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d10391928e2182" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Li et al., 2020</a><div data-db-target-of="bib103-d10391928e2182" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li P. </li><li class="loa__item">Cao G. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi J. </li><li class="loa__item">Cai K. </li><li class="loa__item">Zhen L. </li><li class="loa__item">He X. </li><li class="loa__item">Zhou Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">Zhu Y. </li><li>et al.</li></ul><div class="reference-citations__detials">FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors.<div class="popup-ref"><span class="ref__series"><i>Aging (Albany N.Y.).</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">53-69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31935687&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=53-69&journal=Aging+%28Albany+N.Y.%29&author=Li+P.&author=Cao+G.&author=Zhang+Y.&author=Shi+J.&author=Cai+K.&author=Zhen+L.&author=He+X.&author=Zhou+Y.&author=Li+Y.&author=Zhu+Y.&title=FHL3+promotes+pancreatic+cancer+invasion+and+metastasis+through+preventing+the+ubiquitination+degradation+of+EMT+associated+transcription+factors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C). DVL3, FN1, and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly (FDR = 0.04) enriched pathways in this subtype. As shown in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>D and 2E, coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a “fibroblast proliferation” signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker α-smooth muscle actin (α-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>G), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p = 0.025), but not of cells showing single staining of either CK or α-SMA alone. TGFb is a well-established and potent inducer of EMT (<div class="dropBlock reference-citations"><a href="#bib165" id="back-bib165" data-db-target-for="bib165-d10391928e2195" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib165">Su et al., 2020</a><div data-db-target-of="bib165-d10391928e2195" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Su J. </li><li class="loa__item">Morgani S.M. </li><li class="loa__item">David C.J. </li><li class="loa__item">Wang Q. </li><li class="loa__item">Er E.E. </li><li class="loa__item">Huang Y.-H. </li><li class="loa__item">Basnet H. </li><li class="loa__item">Zou Y. </li><li class="loa__item">Shu W. </li><li class="loa__item">Soni R.K. </li><li>et al.</li></ul><div class="reference-citations__detials">TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">577</span></span>: <span class="ref__seriesPages">566-571</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-019-1897-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31915377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077595486&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=577&publication_year=2020&pages=566-571&journal=Nature&author=Su+J.&author=Morgani+S.M.&author=David+C.J.&author=Wang+Q.&author=Er+E.E.&author=Huang+Y.-H.&author=Basnet+H.&author=Zou+Y.&author=Shu+W.&author=Soni+R.K.&title=TGF-%CE%B2+orchestrates+fibrogenic+and+developmental+EMTs+via+the+RAS+effector+RREB1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGFb inhibitors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>H; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>H).</div><div class="section-paragraph">Integrating our global proteome data with a prior LSCC dataset (<div class="dropBlock reference-citations"><a href="#bib164" id="back-bib164" data-db-target-for="bib164-d10391928e2206" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib164">Stewart et al., 2019</a><div data-db-target-of="bib164-d10391928e2206" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stewart P.A. </li><li class="loa__item">Welsh E.A. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Fang B. </li><li class="loa__item">Izumi V. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Zhang G. </li><li class="loa__item">Cen L. </li><li class="loa__item">Pettersson F. </li><li class="loa__item">Zhang Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic landscape of squamous cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">3578</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-11452-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395880&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85070354617&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3578&journal=Nat.+Commun.&author=Stewart+P.A.&author=Welsh+E.A.&author=Slebos+R.J.C.&author=Fang+B.&author=Izumi+V.&author=Chambers+M.&author=Zhang+G.&author=Cen+L.&author=Pettersson+F.&author=Zhang+Y.&title=Proteogenomic+landscape+of+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) resulted in six clusters largely recapitulating RNA, proteomic, and multi-omic clusters defined in the current study (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>I and S2J).</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling</span></h3><div class="section-paragraph">Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p &lt; 0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>G) (<div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d10391928e2223" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Herreño et al., 2019</a><div data-db-target-of="bib60-d10391928e2223" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Herreño A.M. </li><li class="loa__item">Ramírez A.C. </li><li class="loa__item">Chaparro V.P. </li><li class="loa__item">Fernandez M.J. </li><li class="loa__item">Cañas A. </li><li class="loa__item">Morantes C.F. </li><li class="loa__item">Moreno O.M. </li><li class="loa__item">Brugés R.E. </li><li class="loa__item">Mejía J.A. </li><li class="loa__item">Bustos F.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter.<div class="popup-ref"><span class="ref__series"><i>Tumour Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">41</span> (<span class="refComment">1010428319851014</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1177%2F1010428319851014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31109257&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85066436930&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&journal=Tumour+Biol.&author=Herre%C3%B1o+A.M.&author=Ram%C3%ADrez+A.C.&author=Chaparro+V.P.&author=Fernandez+M.J.&author=Ca%C3%B1as+A.&author=Morantes+C.F.&author=Moreno+O.M.&author=Brug%C3%A9s+R.E.&author=Mej%C3%ADa+J.A.&author=Bustos+F.J.&title=Role+of+RUNX2+transcription+factor+in+epithelial+mesenchymal+transition+in+non-small+cell+lung+cancer+lung+cancer%3A+Epigenetic+control+of+the+RUNX2+P1+promoter">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d10391928e2226" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Niu et al., 2012</a><div data-db-target-of="bib129-d10391928e2226" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Niu D.-F. </li><li class="loa__item">Kondo T. </li><li class="loa__item">Nakazawa T. </li><li class="loa__item">Oishi N. </li><li class="loa__item">Kawasaki T. </li><li class="loa__item">Mochizuki K. </li><li class="loa__item">Yamane T. </li><li class="loa__item">Katoh R. </li></ul><div class="reference-citations__detials">Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas.<div class="popup-ref"><span class="ref__series"><i>Lab. Invest.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">92</span></span>: <span class="ref__seriesPages">1181-1190</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Flabinvest.2012.84&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22641097&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84864428355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (74)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1181-1190&journal=Lab.+Invest.&author=Niu+D.-F.&author=Kondo+T.&author=Nakazawa+T.&author=Oishi+N.&author=Kawasaki+T.&author=Mochizuki+K.&author=Yamane+T.&author=Katoh+R.&title=Transcription+factor+Runx2+is+a+regulator+of+epithelial-mesenchymal+transition+and+invasion+in+thyroid+carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). As summarized in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>K, we found phosphosite-based evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR (<div class="dropBlock reference-citations"><a href="#bib168" id="back-bib168" data-db-target-for="bib168-d10391928e2233" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib168">Tam et al., 2013</a><div data-db-target-of="bib168-d10391928e2233" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tam W.L. </li><li class="loa__item">Lu H. </li><li class="loa__item">Buikhuisen J. </li><li class="loa__item">Soh B.S. </li><li class="loa__item">Lim E. </li><li class="loa__item">Reinhardt F. </li><li class="loa__item">Wu Z.J. </li><li class="loa__item">Krall J.A. </li><li class="loa__item">Bierie B. </li><li class="loa__item">Guo W. </li><li>et al.</li></ul><div class="reference-citations__detials">Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">347-364</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24029232&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84883621632&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=347-364&journal=Cancer+Cell&author=Tam+W.L.&author=Lu+H.&author=Buikhuisen+J.&author=Soh+B.S.&author=Lim+E.&author=Reinhardt+F.&author=Wu+Z.J.&author=Krall+J.A.&author=Bierie+B.&author=Guo+W.&title=Protein+kinase+C+%CE%B1+is+a+central+signaling+node+and+therapeutic+target+for+breast+cancer+stem+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), both targetable (<div class="dropBlock reference-citations"><a href="#bib149" id="back-bib149" data-db-target-for="bib149-d10391928e2236" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib149">Roskoski, 2018</a><div data-db-target-of="bib149-d10391928e2236" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Roskoski Jr., R. </li></ul><div class="reference-citations__detials">The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders.<div class="popup-ref"><span class="ref__series"><i>Pharmacol. Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">129</span></span>: <span class="ref__seriesPages">65-83</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.phrs.2018.01.021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29408302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85041552287&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2018&pages=65-83&journal=Pharmacol.+Res.&author=Roskoski+R.&title=The+role+of+small+molecule+platelet-derived+growth+factor+receptor+%28PDGFR%29+inhibitors+in+the+treatment+of+neoplastic+disorders">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d10391928e2239" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Debebe and Rathmell, 2015</a><div data-db-target-of="bib25-d10391928e2239" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Debebe Z. </li><li class="loa__item">Rathmell W.K. </li></ul><div class="reference-citations__detials">Ror2 as a therapeutic target in cancer.<div class="popup-ref"><span class="ref__series"><i>Pharmacol. Ther.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">150</span></span>: <span class="ref__seriesPages">143-148</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.pharmthera.2015.01.010&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25614331&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928380373&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=143-148&journal=Pharmacol.+Ther.&author=Debebe+Z.&author=Rathmell+W.K.&title=Ror2+as+a+therapeutic+target+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and recently implicated in the progression from a hybrid epithelial/mesenchymal to a highly mesenchymal state (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d10391928e2242" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Kröger et al., 2019</a><div data-db-target-of="bib86-d10391928e2242" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kröger C. </li><li class="loa__item">Afeyan A. </li><li class="loa__item">Mraz J. </li><li class="loa__item">Eaton E.N. </li><li class="loa__item">Reinhardt F. </li><li class="loa__item">Khodor Y.L. </li><li class="loa__item">Thiru P. </li><li class="loa__item">Bierie B. </li><li class="loa__item">Ye X. </li><li class="loa__item">Burge C.B. </li><li class="loa__item">Weinberg R.A. </li></ul><div class="reference-citations__detials">Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">116</span></span>: <span class="ref__seriesPages">7353-7362</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1812876116&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30910979&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064109821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (110)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7353-7362&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Kr%C3%B6ger+C.&author=Afeyan+A.&author=Mraz+J.&author=Eaton+E.N.&author=Reinhardt+F.&author=Khodor+Y.L.&author=Thiru+P.&author=Bierie+B.&author=Ye+X.&author=Burge+C.B.&author=Weinberg+R.A.&title=Acquisition+of+a+hybrid+E%2FM+state+is+essential+for+tumorigenicity+of+basal+breast+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation.</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable</span></h3><div class="section-paragraph">We investigated the impact of recurrent mutations on cognate RNA, proteins, and PTMs (<em>cis</em>-effects) and on a set of CAGs (<em>trans</em>-effects) (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d10391928e2254" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Bailey et al., 2018</a><div data-db-target-of="bib10-d10391928e2254" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bailey M.H. </li><li class="loa__item">Tokheim C. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Sengupta S. </li><li class="loa__item">Bertrand D. </li><li class="loa__item">Weerasinghe A. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Wendl M.C. </li><li class="loa__item">Kim J. </li><li class="loa__item">Reardon B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>MC3 Working Group</span><span>Cancer Genome Atlas Research Network</span><br />Comprehensive Characterization of Cancer Driver Genes and Mutations.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">371-385.e18</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29625053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044623848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (581)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=371-385.e18&journal=Cell&author=MC3+Working+GroupCancer+Genome+Atlas+Research+Network&title=Comprehensive+Characterization+of+Cancer+Driver+Genes+and+Mutations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Significant (FDR &lt; 0.05) <em>cis/trans</em> pairs are shown in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A–3D. <em>RB1</em> mutated tumors had upregulated cell cycle-related protein expression. <em>NOTCH1</em> mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21), and upregulation of NLK. NLK (<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d10391928e2268" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Ishitani et al., 2010</a><div data-db-target-of="bib69-d10391928e2268" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ishitani T. </li><li class="loa__item">Hirao T. </li><li class="loa__item">Suzuki M. </li><li class="loa__item">Isoda M. </li><li class="loa__item">Ishitani S. </li><li class="loa__item">Harigaya K. </li><li class="loa__item">Kitagawa M. </li><li class="loa__item">Matsumoto K. </li><li class="loa__item">Itoh M. </li></ul><div class="reference-citations__detials">Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex.<div class="popup-ref"><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">278-285</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncb2028&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20118921&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77649273214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (84)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=278-285&journal=Nat.+Cell+Biol.&author=Ishitani+T.&author=Hirao+T.&author=Suzuki+M.&author=Isoda+M.&author=Ishitani+S.&author=Harigaya+K.&author=Kitagawa+M.&author=Matsumoto+K.&author=Itoh+M.&title=Nemo-like+kinase+suppresses+Notch+signalling+by+interfering+with+formation+of+the+Notch+active+transcriptional+complex">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and GSK3B (<div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d10391928e2271" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Foltz et al., 2002</a><div data-db-target-of="bib38-d10391928e2271" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Foltz D.R. </li><li class="loa__item">Santiago M.C. </li><li class="loa__item">Berechid B.E. </li><li class="loa__item">Nye J.S. </li></ul><div class="reference-citations__detials">Glycogen synthase kinase-3beta modulates notch signaling and stability.<div class="popup-ref"><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">1006-1011</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=12123574&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0037172661&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1006-1011&journal=Curr.+Biol.&author=Foltz+D.R.&author=Santiago+M.C.&author=Berechid+B.E.&author=Nye+J.S.&title=Glycogen+synthase+kinase-3beta+modulates+notch+signaling+and+stability">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d10391928e2274" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Jin et al., 2009</a><div data-db-target-of="bib71-d10391928e2274" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jin Y.H. </li><li class="loa__item">Kim H. </li><li class="loa__item">Oh M. </li><li class="loa__item">Ki H. </li><li class="loa__item">Kim K. </li></ul><div class="reference-citations__detials">Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta.<div class="popup-ref"><span class="ref__series"><i>Mol. Cells.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">15-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1007%2Fs10059-009-0001-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19214430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-62849118806&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=15-19&journal=Mol.+Cells&author=Jin+Y.H.&author=Kim+H.&author=Oh+M.&author=Ki+H.&author=Kim+K.&title=Regulation+of+Notch1%2FNICD+and+Hes1+expressions+by+GSK-3alpha%2Fbeta">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib210" id="back-bib210" data-db-target-for="bib210-d10391928e2277" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib210">Zheng and Conner, 2018</a><div data-db-target-of="bib210-d10391928e2277" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zheng L. </li><li class="loa__item">Conner S.D. </li></ul><div class="reference-citations__detials">Glycogen synthase kinase 3β inhibition enhances Notch1 recycling.<div class="popup-ref"><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">389-395</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1091%2Fmbc.E17-07-0474&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29237816&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85042268410&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=389-395&journal=Mol.+Biol.+Cell&author=Zheng+L.&author=Conner+S.D.&title=Glycogen+synthase+kinase+3%CE%B2+inhibition+enhances+Notch1+recycling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) inhibit NOTCH1, recently described as a tumor suppressor in LSCC (<div class="dropBlock reference-citations"><a href="#bib159" id="back-bib159" data-db-target-for="bib159-d10391928e2281" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib159">Sinicropi-Yao et al., 2019</a><div data-db-target-of="bib159-d10391928e2281" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sinicropi-Yao S.L. </li><li class="loa__item">Amann J.M. </li><li class="loa__item">Lopez D.L.Y. </li><li class="loa__item">Cerciello F. </li><li class="loa__item">Coombes K.R. </li><li class="loa__item">Carbone D.P. </li></ul><div class="reference-citations__detials">Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC.<div class="popup-ref"><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">223-236</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30408569&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=223-236&journal=J.%C2%A0Thorac.+Oncol.&author=Sinicropi-Yao+S.L.&author=Amann+J.M.&author=Lopez+D.L.Y.&author=Cerciello+F.&author=Coombes+K.R.&author=Carbone+D.P.&title=Co-Expression+Analysis+Reveals+Mechanisms+Underlying+the+Varied+Roles+of+NOTCH1+in+NSCLC">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggesting that both might inhibit NOTCH1 downstream signaling in tumors with <em>NOTCH1</em> mutations. <em>ARID1A</em> is one of the most frequently mutated switch sucrose non-fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden (<div class="dropBlock reference-citations"><a href="#bib155" id="back-bib155" data-db-target-for="bib155-d10391928e2288" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib155">Shen et al., 2018</a><div data-db-target-of="bib155-d10391928e2288" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shen J. </li><li class="loa__item">Ju Z. </li><li class="loa__item">Zhao W. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng Y. </li><li class="loa__item">Ge Z. </li><li class="loa__item">Nagel Z.D. </li><li class="loa__item">Zou J. </li><li class="loa__item">Wang C. </li><li class="loa__item">Kapoor P. </li><li>et al.</li></ul><div class="reference-citations__detials">ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">556-562</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41591-018-0012-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29736026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85046552398&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=556-562&journal=Nat.+Med.&author=Shen+J.&author=Ju+Z.&author=Zhao+W.&author=Wang+L.&author=Peng+Y.&author=Ge+Z.&author=Nagel+Z.D.&author=Zou+J.&author=Wang+C.&author=Kapoor+P.&title=ARID1A+deficiency+promotes+mutability+and+potentiates+therapeutic+antitumor+immunity+unleashed+by+immune+checkpoint+blockade">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>A) and led to protein downregulation, associated with worse overall survival in NSCLC (<div class="dropBlock reference-citations"><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d10391928e2296" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Hung et al., 2020</a><div data-db-target-of="bib68-d10391928e2296" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hung Y.P. </li><li class="loa__item">Redig A. </li><li class="loa__item">Hornick J.L. </li><li class="loa__item">Sholl L.M. </li></ul><div class="reference-citations__detials">ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis.<div class="popup-ref"><span class="ref__series"><i>Mod. Pathol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2256-2268</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41379-020-0592-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32572156&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85086777439&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=2256-2268&journal=Mod.+Pathol.&author=Hung+Y.P.&author=Redig+A.&author=Hornick+J.L.&author=Sholl+L.M.&title=ARID1A+mutations+and+expression+loss+in+non-small+cell+lung+carcinomas%3A+clinicopathologic+and+molecular+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/2b24b7c6-8abe-464a-80be-907789cddbd4/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/05a9e198-32f0-43cd-80de-dddcdab21b0c/gr3_lrg.jpg" data-large-image="/cms/attachment/2b24b7c6-8abe-464a-80be-907789cddbd4/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Impact of somatic mutations on proteogenomic features</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Significant (Wilcoxon FDR &lt; 0.05) <em>cis</em>- (circles) and <em>trans</em>-effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green.</div><div class="section-paragraph">(B) Similar to (A) but showing phosphosites.</div><div class="section-paragraph">(C) Similar to (A) but showing acetylsites.</div><div class="section-paragraph">(D) Similar to (A) but showing ubiquitylsites.</div><div class="section-paragraph">(E) CNA data for <em>CDKN2A</em> and <em>RB1</em> was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2865" href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" target="_blank">Table S4</a>). <em>CDKN2A</em> genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these <em>CDKN2A/p16</em> aberrations on both major isoforms (p16INK4a and p14ARF [p14]) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of <em>CCND1-3</em>, <em>CDK4</em>, and <em>CDK6</em> were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post-translational modifications-signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of <em>CDKN2A</em> are not included.</div><div class="section-paragraph">(F) Correlation between differential regulation of protein abundance (Log2 fold change [FC]) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in <em>KEAP1</em>, <em>CUL3</em>, or <em>NFE2L2</em>) versus NRF2 WT tumors (no NRF2 pathway aberration).</div><div class="section-paragraph">(G) NRF2 pathway score and RNA, protein, and phosphoprotein expression of key NRF pathway members according to NMF subtype.</div><div class="section-paragraph">(H) CDK5 protein expression (Log2 TMT ratio) by NMF subtype. p values are from the Anova test.</div><div class="section-paragraph">(I) PTM-SEA-derived normalized enrichment scores (NESs) for pathways enriched in NRF2 pathway-mutated (Mutant) versus wild-type (WT) samples plotted against NES for pathways enriched in NMF classical versus other subtypes. Significantly upregulated (FDR &lt; 0.05) PTM-SEA terms in the classical subtype are indicated by red dots and labeled.</div><div class="section-paragraph">See also <a data-locator="figs3" aria-expanded="false" aria-controls="crosref2870" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs3">Figure S3</a> and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2875" href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/2b24b7c6-8abe-464a-80be-907789cddbd4/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/05a9e198-32f0-43cd-80de-dddcdab21b0c/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/1bfa2388-fb4b-4422-9986-e260cfdbb48c/figs3.jpg" alt="Figure thumbnail figs3" data-locator="figs3" data-high-res="/cms/attachment/2ca3e1b4-934d-4599-9700-58d25f5c37e3/figs3_lrg.jpg" data-large-image="/cms/attachment/1bfa2388-fb4b-4422-9986-e260cfdbb48c/figs3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S3" class="label figure__label">Figure S3</span><span class="figure__title__text">Proteogenomic consequences of CDKN2A and NRF2 pathway aberrations, related to <a data-locator="fig3" aria-expanded="false" aria-controls="crosref2985" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig3">Figure 3</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Tumor Mutational Burden per megabase (MB) as estimated from whole exome sequencing among samples with and without <em>ARID1A</em> gene somatic mutation.</div><div class="section-paragraph">(B) Schematic diagram showing amino acid alterations resulting from <em>CDKN2A/p16INK4a</em> nonsense and frameshift mutations in the Clinical Proteomic Tumor Analysis Consortium (CPTAC) LSCC and head and neck squamous cell cancer (HNSCC) datasets. Not shown are seven mutant tumors with missense or in-frame indels that would not be predicted to trigger nonsense mediated decay (NMD). Unlike the CPTAC HNSCC cohort, where <em>p16INK4a (p16)</em> nonsense and frameshift mutations always resulted in NMD (tumors with low p16 RNA expression are indicated by blue outline), five of the nine nonsense/frameshift mutations in LSCC appeared to have escaped NMD. These mutations either resulted in truncation at the very 5′ end (E10<sup>∗</sup>) or were frameshift mutations that did not actually result in a significantly truncated predicted protein product (all predicted to generate protein products of 142 or more amino acids compared to the full length p16 protein of 160 amino acids), while all of the mutations triggering NMD resulted in production of a protein product that was 40-120 amino acids shorter than the wild-type protein.</div><div class="section-paragraph">(C) Boxplots showing Rb protein and phosphoprotein (mean of all Rb phosphosites) (Log2 TMT ratio) expression in samples with and without <em>CCND1</em> amplification. Amplified samples are those with GISTIC threshold = 2, while all other samples were considered not amplified. P values are from Welch’s two-sample t tests.</div><div class="section-paragraph">(D) Phospho-Rb levels correlate with response to CDK4/6 inhibitors in various LSCC cell lines. Scatterplots show correlation between phospho-Rb S807/811 by RPPA (CCLE 2019, Proteomics (RPPA) from DepMap) and abemaciclib response (PRISM Repurposing Secondary Screen 19Q4 from DepMap) (Left panel) or palbociclib response (Sanger GDSC1 from DepMap) for LSCC cell lines (right panel). <em>CCND1</em>, <em>CDK2NA</em>, and <em>RB1</em> mutation status (DepMapPublic 21Q1, Mutation) depicted as filled circles. Outline depicts presence of <em>CCND1</em> amplification (GISTIC threshold ≥ 1, derived from segmented copy number, DepMapPublic 21Q1) and/or <em>CDKN2A</em> deletion (GISTIC threshold ≤ -1). Cell lines with higher levels of phospho-Rb S807/811 were more sensitive to CDK4/6 inhibitors abemaciclib and palbociclib. R, Pearson correlation coefficient. DepMap (Dependency map) is a compendium of genomics, proteomics, shRNA and CRISPR based gene dependency datasets of hundreds of cell lines. PRISM (Profiling Relative Inhibition Simultaneously in Mixtures) and GDSC (Genomics of Drug Sensitivity) are databases containing sensitivity data to hundreds of compounds in hundreds of cell lines.</div><div class="section-paragraph">(E) Boxplots showing NFE2L2 S215 and S433 abundance in samples with <em>CUL3</em>, <em>KEAP1</em> and <em>NFE2L2</em> mutations. <sup>∗∗∗∗</sup> &lt; 0.0001, <sup>∗∗∗</sup> &lt; 0.01, <sup>∗∗</sup>&lt; 0.01, <sup>∗</sup>&lt; 0.05, ns = not significant. P values are from the Wilcoxon test.</div><div class="section-paragraph">(F) Correlation between differential mRNA expression (log2 FC) and protein abundance (log2 FC) levels among tumors with NRF2 pathway mutation (one and two hits) versus NRF2 pathway WT tumors. Significant events (&gt; 2 fold and p value &lt; 0.05) are represented as circles with black borders. NRF2 targets in human cancer tissues (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d10391928e472" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Lacher and Slattery, 2016</a><div data-db-target-of="bib91-d10391928e472" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lacher S.E. </li><li class="loa__item">Slattery M. </li></ul><div class="reference-citations__detials">Gene regulatory effects of disease-associated variation in the NRF2 network.<div class="popup-ref"><span class="ref__series"><i>Curr. Opin. Toxicol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">71-79</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cotox.2016.09.001&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28203648&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016740108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=71-79&journal=Curr.+Opin.+Toxicol.&author=Lacher+S.E.&author=Slattery+M.&title=Gene+regulatory+effects+of+disease-associated+variation+in+the+NRF2+network">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) are represented by solid circles in green, while solid circles in black indicate novel targets. Candidates that also show phosphosite modification are represented by pink-filled gene name boxes.</div><div class="section-paragraph">(G) Overlap between NRF2-related genesets (upper left), and up- (upper right) and downregulated (bottom) events in NRF2 pathway mutated samples across mRNA, proteomics and phosphoproteomics.</div><div class="section-paragraph">(H) Relative mRNA, protein and phosphosite abundance of NRF2 pathway-related and other cancer-associated genes in tumor samples with two NRF2 pathway (<em>NF2L2, KEAP1, CUL3</em>) mutations, one mutation, or WT status, as well as in NATs. A dosage effect is observed. <sup>#</sup> no phosphosite levels changes were detected for NR0B1 between the groups.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/1bfa2388-fb4b-4422-9986-e260cfdbb48c/figs3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/2ca3e1b4-934d-4599-9700-58d25f5c37e3/figs3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Paradoxically, mutations in <em>CDKN2A</em>, proposed as a potential biomarker for CDK4/6 inhibitors (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d10391928e2303" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Ahn et al., 2020</a><div data-db-target-of="bib2-d10391928e2303" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ahn E.R. </li><li class="loa__item">Mangat P.K. </li><li class="loa__item">Garrett-Mayer E. </li><li class="loa__item">Halabi S. </li><li class="loa__item">Dib E.G. </li><li class="loa__item">Haggstrom D.E. </li><li class="loa__item">Alguire K.B. </li><li class="loa__item">Calfa C.J. </li><li class="loa__item">Cannon T.L. </li><li class="loa__item">Crilley P.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study.<div class="popup-ref"><span class="ref__series"><i>JCO Precision Oncology.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1200/PO.20.00037">https://doi.org/10.1200/PO.20.00037</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FPO.20.00037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33015524&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85090749277&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=JCO+Precision+Oncology&author=Ahn+E.R.&author=Mangat+P.K.&author=Garrett-Mayer+E.&author=Halabi+S.&author=Dib+E.G.&author=Haggstrom+D.E.&author=Alguire+K.B.&author=Calfa+C.J.&author=Cannon+T.L.&author=Crilley+P.A.&title=Palbociclib+in+Patients+With+Non%E2%80%93Small-Cell+Lung+Cancer+With+CDKN2A+Alterations%3A+Results+From+the+Targeted+Agent+and+Profiling+Utilization+Registry+Study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib121" id="back-bib121" data-db-target-for="bib121-d10391928e2306" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib121">Middleton et al., 2020</a><div data-db-target-of="bib121-d10391928e2306" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Middleton G. </li><li class="loa__item">Fletcher P. </li><li class="loa__item">Popat S. </li><li class="loa__item">Savage J. </li><li class="loa__item">Summers Y. </li><li class="loa__item">Greystoke A. </li><li class="loa__item">Gilligan D. </li><li class="loa__item">Cave J. </li><li class="loa__item">O’Rourke N. </li><li class="loa__item">Brewster A. </li><li>et al.</li></ul><div class="reference-citations__detials">The National Lung Matrix Trial of personalized therapy in lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">583</span></span>: <span class="ref__seriesPages">807-812</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-020-2481-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32669708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087956964&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2020&pages=807-812&journal=Nature&author=Middleton+G.&author=Fletcher+P.&author=Popat+S.&author=Savage+J.&author=Summers+Y.&author=Greystoke+A.&author=Gilligan+D.&author=Cave+J.&author=O%E2%80%99Rourke+N.&author=Brewster+A.&title=The+National+Lung+Matrix+Trial+of+personalized+therapy+in+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), resulted in increased RNA expression (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A). We analyzed the effects of genetic and epigenetic alterations of <em>CDKN2A</em> on both its major isoforms, p16INK4a (p16) and p14ARF (p14) (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>). As expected, its homozygous deletion resulted in the loss of expression of both isoforms (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>E). However, only eight of the 58 <em>CDKN2A</em> wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with <em>CDKN2A</em>/p16 missense mutations (log2(FPKM+1) &gt; 2). Many of those <em>CDKN2A</em>/p16 mutations were unlikely to result in nonsense-mediated decay (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>B), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had <em>RB1</em> mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes (<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d10391928e2333" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Knudsen et al., 2020</a><div data-db-target-of="bib84-d10391928e2333" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Knudsen E.S. </li><li class="loa__item">Nambiar R. </li><li class="loa__item">Rosario S.R. </li><li class="loa__item">Smiraglia D.J. </li><li class="loa__item">Goodrich D.W. </li><li class="loa__item">Witkiewicz A.K. </li></ul><div class="reference-citations__detials">Pan-cancer molecular analysis of the RB tumor suppressor pathway.<div class="popup-ref"><span class="ref__series"><i>Commun. Biol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">158</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs42003-020-0873-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32242058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85082980228&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=158&journal=Commun.+Biol&author=Knudsen+E.S.&author=Nambiar+R.&author=Rosario+S.R.&author=Smiraglia+D.J.&author=Goodrich+D.W.&author=Witkiewicz+A.K.&title=Pan-cancer+molecular+analysis+of+the+RB+tumor+suppressor+pathway">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of Cyclin D-CDK4/6 complex genes is frequent in LSCC (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d10391928e2336" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Hammerman et al., 2012</a><div data-db-target-of="bib54-d10391928e2336" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hammerman P.S. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Voet D. </li><li class="loa__item">Jing R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Gabriel S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), there was no association between the amplification of <em>CCND2</em>, <em>CCND3</em>, <em>CDK4</em>, or <em>CDK6</em> and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>). <em>CCND1</em> amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions were overlapping (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>C). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in <em>CDKN2A</em> and copy number alterations in <em>CCND1</em> and <em>CDKN2A</em> (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>D). LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib (<div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d10391928e2366" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Patnaik et al., 2016</a><div data-db-target-of="bib134-d10391928e2366" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Patnaik A. </li><li class="loa__item">Rosen L.S. </li><li class="loa__item">Tolaney S.M. </li><li class="loa__item">Tolcher A.W. </li><li class="loa__item">Goldman J.W. </li><li class="loa__item">Gandhi L. </li><li class="loa__item">Papadopoulos K.P. </li><li class="loa__item">Beeram M. </li><li class="loa__item">Rasco D.W. </li><li class="loa__item">Hilton J.F. </li><li>et al.</li></ul><div class="reference-citations__detials">Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">740-753</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F2159-8290.CD-16-0095&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27217383&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84977598561&amp;cf=" onclick="newWindow(this.href);return false">Scopus (335)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=740-753&journal=Cancer+Discov.&author=Patnaik+A.&author=Rosen+L.S.&author=Tolaney+S.M.&author=Tolcher+A.W.&author=Goldman+J.W.&author=Gandhi+L.&author=Papadopoulos+K.P.&author=Beeram+M.&author=Rasco+D.W.&author=Hilton+J.F.&title=Efficacy+and+Safety+of+Abemaciclib%2C+an+Inhibitor+of+CDK4+and+CDK6%2C+for+Patients+with+Breast+Cancer%2C+Non-Small+Cell+Lung+Cancer%2C+and+Other+Solid+Tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and palbociclib (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d10391928e2369" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Ahn et al., 2020</a><div data-db-target-of="bib2-d10391928e2369" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ahn E.R. </li><li class="loa__item">Mangat P.K. </li><li class="loa__item">Garrett-Mayer E. </li><li class="loa__item">Halabi S. </li><li class="loa__item">Dib E.G. </li><li class="loa__item">Haggstrom D.E. </li><li class="loa__item">Alguire K.B. </li><li class="loa__item">Calfa C.J. </li><li class="loa__item">Cannon T.L. </li><li class="loa__item">Crilley P.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study.<div class="popup-ref"><span class="ref__series"><i>JCO Precision Oncology.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1200/PO.20.00037">https://doi.org/10.1200/PO.20.00037</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FPO.20.00037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33015524&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85090749277&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=JCO+Precision+Oncology&author=Ahn+E.R.&author=Mangat+P.K.&author=Garrett-Mayer+E.&author=Halabi+S.&author=Dib+E.G.&author=Haggstrom+D.E.&author=Alguire+K.B.&author=Calfa+C.J.&author=Cannon+T.L.&author=Crilley+P.A.&title=Palbociclib+in+Patients+With+Non%E2%80%93Small-Cell+Lung+Cancer+With+CDKN2A+Alterations%3A+Results+From+the+Targeted+Agent+and+Profiling+Utilization+Registry+Study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d10391928e2372" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Edelman et al., 2019</a><div data-db-target-of="bib31-d10391928e2372" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Edelman M.J. </li><li class="loa__item">Redman M.W. </li><li class="loa__item">Albain K.S. </li><li class="loa__item">McGary E.C. </li><li class="loa__item">Rafique N.M. </li><li class="loa__item">Petro D. </li><li class="loa__item">Waqar S.N. </li><li class="loa__item">Minichiello K. </li><li class="loa__item">Miao J. </li><li class="loa__item">Papadimitrakopoulou V.A. </li><li>et al.</li></ul><div class="reference-citations__detials">SWOG S1400C (NCT02154490)-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy).<div class="popup-ref"><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">1853-1859</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31302234&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1853-1859&journal=J.%C2%A0Thorac.+Oncol.&author=Edelman+M.J.&author=Redman+M.W.&author=Albain+K.S.&author=McGary+E.C.&author=Rafique+N.M.&author=Petro+D.&author=Waqar+S.N.&author=Minichiello+K.&author=Miao+J.&author=Papadimitrakopoulou+V.A.&title=SWOG+S1400C+%28NCT02154490%29-A+Phase+II+Study+of+Palbociclib+for+Previously+Treated+Cell+Cycle+Gene+Alteration-Positive+Patients+with+Stage+IV+Squamous+Cell+Lung+Cancer+%28Lung-MAP+Substudy%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib121" id="back-bib121" data-db-target-for="bib121-d10391928e2375" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib121">Middleton et al., 2020</a><div data-db-target-of="bib121-d10391928e2375" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Middleton G. </li><li class="loa__item">Fletcher P. </li><li class="loa__item">Popat S. </li><li class="loa__item">Savage J. </li><li class="loa__item">Summers Y. </li><li class="loa__item">Greystoke A. </li><li class="loa__item">Gilligan D. </li><li class="loa__item">Cave J. </li><li class="loa__item">O’Rourke N. </li><li class="loa__item">Brewster A. </li><li>et al.</li></ul><div class="reference-citations__detials">The National Lung Matrix Trial of personalized therapy in lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">583</span></span>: <span class="ref__seriesPages">807-812</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-020-2481-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32669708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087956964&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2020&pages=807-812&journal=Nature&author=Middleton+G.&author=Fletcher+P.&author=Popat+S.&author=Savage+J.&author=Summers+Y.&author=Greystoke+A.&author=Gilligan+D.&author=Cave+J.&author=O%E2%80%99Rourke+N.&author=Brewster+A.&title=The+National+Lung+Matrix+Trial+of+personalized+therapy+in+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). While we found that <em>CCND1</em> amplification resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with <em>CCND1</em> amplification provides a potential explanation for some patients’ lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e., RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 inhibition.</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">NRF2 pathway activation in tumors with and without NRF2 pathway mutations</span></h3><div class="section-paragraph">LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, <em>NFE2L2</em> (<em>NRF2</em>), <em>CUL3</em>, and <em>KEAP1</em>. NFE2L2 phosphorylation was observed in samples with any of these mutations (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>E). LSCC tumors were annotated as “NRF2 pathway mutated” (n = 33; with either one hit [n = 28] or two hits [n = 5] in <em>NFE2L2</em>, <em>KEAP1</em>, and <em>CUL3</em>), or “NRF2 WT” (n = 68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression of mRNA, protein, and phosphoprotein levels of NRF2 pathway genes (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>F; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figures S3</a>F and S3G; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>) demonstrating increased up-/downregulation of NRF2 pathway gene products relative to one-hit cases (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>H). Using the proteogenomic signature (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>) that defined <em>NFE2L2</em> mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF classical subtype (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>G). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>H) and PTM-SEA-derived activity (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d10391928e2434" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Krug et al., 2019</a><div data-db-target-of="bib87-d10391928e2434" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A curated resource for phosphosite-specific signature analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>I) were significantly upregulated (FDR &lt; 0.05) in the classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d10391928e2440" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Jimenez-Blasco et al., 2015</a><div data-db-target-of="bib70-d10391928e2440" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jimenez-Blasco D. </li><li class="loa__item">Santofimia-Castaño P. </li><li class="loa__item">Gonzalez A. </li><li class="loa__item">Almeida A. </li><li class="loa__item">Bolaños J.P. </li></ul><div class="reference-citations__detials">Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway.<div class="popup-ref"><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">1877-1889</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fcdd.2015.49&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25909891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943197836&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1877-1889&journal=Cell+Death+Differ.&author=Jimenez-Blasco+D.&author=Santofimia-Casta%C3%B1o+P.&author=Gonzalez+A.&author=Almeida+A.&author=Bola%C3%B1os+J.P.&title=Astrocyte+NMDA+receptors%E2%80%99+activity+sustains+neuronal+survival+through+a+Cdk5-Nrf2+pathway">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e2443" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e2443" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), <em>KEAP1</em> mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes.</div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC</span></h3><div class="section-paragraph">Chromosome 3q, which harbors key squamous differentiation markers <em>SOX2</em> and <em>TP63</em> (<div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d10391928e2457" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Qian and Massion, 2008</a><div data-db-target-of="bib142-d10391928e2457" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Qian J. </li><li class="loa__item">Massion P.P. </li></ul><div class="reference-citations__detials">Role of chromosome 3q amplification in lung cancer.<div class="popup-ref"><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">212-215</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18317062&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-43249114355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=212-215&journal=J.%C2%A0Thorac.+Oncol.&author=Qian+J.&author=Massion+P.P.&title=Role+of+chromosome+3q+amplification+in+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), showed the most dramatic arm-level amplification in this cohort (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>A). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A). LSCC cell lines amplified for <em>TP63</em> and <em>SOX2</em> were highly dependent upon them, supporting oncogene addiction (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>B).<div class="floatDisplay"><figure id="figs4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/6e6261b7-26fa-407e-a035-c223907af00c/figs4.jpg" alt="Figure thumbnail figs4" data-locator="figs4" data-high-res="/cms/attachment/9104a1e0-6b4b-44e8-a1fd-3ad8ea475a61/figs4_lrg.jpg" data-large-image="/cms/attachment/6e6261b7-26fa-407e-a035-c223907af00c/figs4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S4" class="label figure__label">Figure S4</span><span class="figure__title__text">Association between TP63 and miR205 expression in LSCC, related to <a data-locator="fig4" aria-expanded="false" aria-controls="crosref2995" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig4">Figure 4</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Focal amplification and deletion landscape of the CPTAC LSCC cohort. The top five significant focal peaks are labeled in the plot</div><div class="section-paragraph">(B) Copy number segmentation file (DepMapPublic 21Q1) for LSCC cell lines from DepMap, processed with GISTIC2.0 (same parameters used to process LSCC cohort as described in STAR METHODS). Genes were considered amplified if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Each gene upregulated at the protein level with recurrent copy number gain in tumors versus NATs (from <a data-locator="fig4" aria-expanded="false" aria-controls="crosref3000" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig4">Figure 4</a>A with Log2FC &gt; 0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores (Combined CRISPR KO screen, DepMapPublic 21Q1) by t test of a given gene in amplified versus non-amplified LSCC cell lines. X axis depicts the mean change in dependency scores for a queried gene in cell lines with amplification versus without amplification of that gene. Y axis depicts the -log10(p value) from t test. Genes passing FDR &lt; 0.25 are depicted by red and blue colored dots.</div><div class="section-paragraph">(C) mRNA expression of all detected TP63 Ensembl transcript (ENST) isoforms for tumors. The transcript corresponding to ΔNp63α was the dominant isoform observed in this cohort.</div><div class="section-paragraph">(D) Correlations between ΔNp63α transcript and gene-level TP63 mRNA expression (upper panel) or TP63 relative protein expression (log2 TMT ratio) and gene-level TP63 mRNA expression (lower panel).</div><div class="section-paragraph">(E) Kaplan-Meier plot of overall survival in ΔNp63α transcript-low versus -high samples from the TCGA LUSC (LSCC) cohort. Samples with low ΔNp63α expression have significantly worse outcomes.</div><div class="section-paragraph">(F) Np63 mRNA transcript isoform expression analysis (RNA-Seq (Transcript isoform), DepMapPublic 21Q1) shows LSCC cell lines with high and low levels of Np63.</div><div class="section-paragraph">(G) Response to survivin (BIRC5) inhibitor YM-155 (PRISM Repurposing Secondary Screen 19Q4 from DepMap) in LSCC cell lines with high and low expression of Np63. Np63-low lines are more sensitive to YM-155 than Np63-high cell lines. P value derived from t test.</div><div class="section-paragraph">(H) Rank plots showing enrichment scores for proliferation-related pathways from pre-ranked GSEA. The inputs are t-values of gene expression between TP63-low and TP63-high samples.</div><div class="section-paragraph">(I) Volcano plot showing differential expression of miRNA (Log2 FC) between tumors and NATs. miR-205 shows the highest relative overexpression in tumors.</div><div class="section-paragraph">(J) Differential promoter level methylation of miR-205 in tumors compared to NATs. miR-205 is hypomethylated in tumors.</div><div class="section-paragraph">(K) Pearson correlations between miR-205 expression and all quantified proteins in tumors. TP63 is highlighted and shows the highest correlation with miR-205.</div><div class="section-paragraph">(L) Gene-level RNA (left) and protein (right) expression correlations between TP63 and TP73.</div><div class="section-paragraph">(M) Correlation between EZH2 dependency by shRNA (shRNA KD screen (Combined), DEMETER2 Data v6 from DepMap) and <em>SOX2</em> Log2 copy number (Copy number (gene level), DepMapPublic 21Q1) in LSCC cell lines. R, Pearson correlation coefficient.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/6e6261b7-26fa-407e-a035-c223907af00c/figs4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/9104a1e0-6b4b-44e8-a1fd-3ad8ea475a61/figs4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/89ec2b13-86aa-4dc1-b10b-4732db269d39/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/942fa692-64c8-4592-bcc8-db15b875c066/gr4_lrg.jpg" data-large-image="/cms/attachment/89ec2b13-86aa-4dc1-b10b-4732db269d39/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Proteogenomic impact of chromosome 3q amplification</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q.</div><div class="section-paragraph">(B) Frequency distribution of ΔNp63α RNA expression in tumors and NATs.</div><div class="section-paragraph">(C) Differential protein expression in samples classified as ΔNp63α-low versus -high based on ΔNp63α transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin.</div><div class="section-paragraph">(D) Differential expression of microRNAs in ΔNp63α-low versus -high samples.</div><div class="section-paragraph">(E) Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets.</div><div class="section-paragraph">(F) Pearson correlation between expression of miR-205 and protein abundance of its cognate targets.</div><div class="section-paragraph">(G) Relationship between miR-205 expression (log2 TPM) and EMT (top, p = 2.3x10<sup>−08</sup>, correlation = −0.53) and DNA replication (bottom, p = 1.1x10<sup>−06</sup>, correlation = 0.47) scores.</div><div class="section-paragraph">(H) Heatmap showing relative protein expression (log2 TMT ratio) of selected proteins with significant (FDR &lt; 0.01) Pearson correlation values (positive or negative) with SOX2 protein expression.</div><div class="section-paragraph">(I) Pearson correlation (p = 5.2x10<sup>−07</sup>, Correlation = −0.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling normalized enrichment score (NES).</div><div class="section-paragraph">See also <a data-locator="figs4" aria-expanded="false" aria-controls="crosref2880" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs4">Figure S4</a> and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2885" href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/89ec2b13-86aa-4dc1-b10b-4732db269d39/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/942fa692-64c8-4592-bcc8-db15b875c066/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">As previously reported (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d10391928e2478" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Campbell et al., 2018</a><div data-db-target-of="bib20-d10391928e2478" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Campbell J.D. </li><li class="loa__item">Yau C. </li><li class="loa__item">Bowlby R. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Brennan K. </li><li class="loa__item">Fan H. </li><li class="loa__item">Taylor A.M. </li><li class="loa__item">Wang C. </li><li class="loa__item">Walter V. </li><li class="loa__item">Akbani R. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">194-212.e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29617660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044610513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (100)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=194-212.e6&journal=Cell+Rep.&author=Cancer+Genome+Atlas+Research+Network&title=Genomic%2C+Pathway+Network%2C+and+Immunologic+Features+Distinguishing+Squamous+Carcinomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d10391928e2481" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Lo Iacono et al., 2011</a><div data-db-target-of="bib112-d10391928e2481" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lo Iacono M. </li><li class="loa__item">Monica V. </li><li class="loa__item">Saviozzi S. </li><li class="loa__item">Ceppi P. </li><li class="loa__item">Bracco E. </li><li class="loa__item">Papotti M. </li><li class="loa__item">Scagliotti G.V. </li></ul><div class="reference-citations__detials">p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue.<div class="popup-ref"><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">473-481</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21289519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79951770197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=473-481&journal=J.%C2%A0Thorac.+Oncol.&author=Lo+Iacono+M.&author=Monica+V.&author=Saviozzi+S.&author=Ceppi+P.&author=Bracco+E.&author=Papotti+M.&author=Scagliotti+G.V.&title=p63+and+p73+isoform+expression+in+non-small+cell+lung+cancer+and+corresponding+morphological+normal+lung+tissue">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), ΔNp63α was the dominant TP63 isoform in this dataset (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>C) and was highly correlated with TP63 expression (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>D). Its amplification was associated with improved survival in the TCGA dataset (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>E). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 “Np63-low” samples histologically confirmed to be LSCC showed no elevation at RNA or protein levels (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>B). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). Notably, a substantial number of LSCC cell lines also had low Np63 expression (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>F), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>G). Survivin promotes cell proliferation and G2M checkpoints (<div class="dropBlock reference-citations"><a href="#bib193" id="back-bib193" data-db-target-for="bib193-d10391928e2506" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib193">Wheatley and Altieri, 2019</a><div data-db-target-of="bib193-d10391928e2506" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wheatley S.P. </li><li class="loa__item">Altieri D.C. </li></ul><div class="reference-citations__detials">Survivin at a glance.<div class="popup-ref"><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">132</span></span>: <span class="ref__seriesPages">jcs223826</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.223826&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30948431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064325670&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2019&pages=jcs223826&journal=J.%C2%A0Cell+Sci.&author=Wheatley+S.P.&author=Altieri+D.C.&title=Survivin+at+a+glance">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), evident in Np63-low samples (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>H). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in NSCLC (<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d10391928e2513" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Giaccone et al., 2009</a><div data-db-target-of="bib48-d10391928e2513" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Giaccone G. </li><li class="loa__item">Zatloukal P. </li><li class="loa__item">Roubec J. </li><li class="loa__item">Floor K. </li><li class="loa__item">Musil J. </li><li class="loa__item">Kuta M. </li><li class="loa__item">van Klaveren R.J. </li><li class="loa__item">Chaudhary S. </li><li class="loa__item">Gunther A. </li><li class="loa__item">Shamsili S. </li></ul><div class="reference-citations__detials">Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">4481-4486</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2008.21.1862&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19687333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-76749118103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (186)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=4481-4486&journal=J.%C2%A0Clin.+Oncol.&author=Giaccone+G.&author=Zatloukal+P.&author=Roubec+J.&author=Floor+K.&author=Musil+J.&author=Kuta+M.&author=van+Klaveren+R.J.&author=Chaudhary+S.&author=Gunther+A.&author=Shamsili+S.&title=Multicenter+phase+II+trial+of+YM155%2C+a+small-molecule+suppressor+of+survivin%2C+in+patients+with+advanced%2C+refractory%2C+non-small-cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d10391928e2516" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Kelly et al., 2013</a><div data-db-target-of="bib77-d10391928e2516" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kelly R.J. </li><li class="loa__item">Thomas A. </li><li class="loa__item">Rajan A. </li><li class="loa__item">Chun G. </li><li class="loa__item">Lopez-Chavez A. </li><li class="loa__item">Szabo E. </li><li class="loa__item">Spencer S. </li><li class="loa__item">Carter C.A. </li><li class="loa__item">Guha U. </li><li class="loa__item">Khozin S. </li><li>et al.</li></ul><div class="reference-citations__detials">A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2601-2606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23857959&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84884717482&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=2601-2606&journal=Ann.+Oncol.&author=Kelly+R.J.&author=Thomas+A.&author=Rajan+A.&author=Chun+G.&author=Lopez-Chavez+A.&author=Szabo+E.&author=Spencer+S.&author=Carter+C.A.&author=Guha+U.&author=Khozin+S.&title=A+phase+I%2FII+study+of+sepantronium+bromide+%28YM155%2C+survivin+suppressor%29+with+paclitaxel+and+carboplatin+in+patients+with+advanced+non-small-cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d10391928e2519" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">Li et al., 2019a</a><div data-db-target-of="bib101-d10391928e2519" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li F. </li><li class="loa__item">Aljahdali I. </li><li class="loa__item">Ling X. </li></ul><div class="reference-citations__detials">Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study?.<div class="popup-ref"><span class="ref__series"><i>J. Exp. Clin. Cancer Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13046-019-1362-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31439015&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071240357&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=368&journal=J.%C2%A0Exp.+Clin.+Cancer+Res.&author=Li+F.&author=Aljahdali+I.&author=Ling+X.&title=Cancer+therapeutics+using+survivin+BIRC5+as+a+target%3A+what+can+we+do+after+over+two+decades+of+study%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) but have shown modest or no improvement (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d10391928e2522" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Kelly et al., 2013</a><div data-db-target-of="bib77-d10391928e2522" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kelly R.J. </li><li class="loa__item">Thomas A. </li><li class="loa__item">Rajan A. </li><li class="loa__item">Chun G. </li><li class="loa__item">Lopez-Chavez A. </li><li class="loa__item">Szabo E. </li><li class="loa__item">Spencer S. </li><li class="loa__item">Carter C.A. </li><li class="loa__item">Guha U. </li><li class="loa__item">Khozin S. </li><li>et al.</li></ul><div class="reference-citations__detials">A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2601-2606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23857959&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84884717482&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=2601-2606&journal=Ann.+Oncol.&author=Kelly+R.J.&author=Thomas+A.&author=Rajan+A.&author=Chun+G.&author=Lopez-Chavez+A.&author=Szabo+E.&author=Spencer+S.&author=Carter+C.A.&author=Guha+U.&author=Khozin+S.&title=A+phase+I%2FII+study+of+sepantronium+bromide+%28YM155%2C+survivin+suppressor%29+with+paclitaxel+and+carboplatin+in+patients+with+advanced+non-small-cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin inhibition.</div><div class="section-paragraph">Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC (<div class="dropBlock reference-citations"><a href="#bib97" id="back-bib97" data-db-target-for="bib97-d10391928e2527" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Lebanony et al., 2009</a><div data-db-target-of="bib97-d10391928e2527" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lebanony D. </li><li class="loa__item">Benjamin H. </li><li class="loa__item">Gilad S. </li><li class="loa__item">Ezagouri M. </li><li class="loa__item">Dov A. </li><li class="loa__item">Ashkenazi K. </li><li class="loa__item">Gefen N. </li><li class="loa__item">Izraeli S. </li><li class="loa__item">Rechavi G. </li><li class="loa__item">Pass H. </li><li>et al.</li></ul><div class="reference-citations__detials">Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma.<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2030-2037</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2008.19.4134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19273703&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-65349166324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (318)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=2030-2037&journal=J.%C2%A0Clin.+Oncol.&author=Lebanony+D.&author=Benjamin+H.&author=Gilad+S.&author=Ezagouri+M.&author=Dov+A.&author=Ashkenazi+K.&author=Gefen+N.&author=Izraeli+S.&author=Rechavi+G.&author=Pass+H.&title=Diagnostic+assay+based+on+hsa-miR-205+expression+distinguishes+squamous+from+nonsquamous+non-small-cell+lung+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>I), was associated with decreased promoter DNA methylation (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>J) and elevation of ΔNp63α, a transcriptional regulator of this miRNA (<div class="dropBlock reference-citations"><a href="#bib175" id="back-bib175" data-db-target-for="bib175-d10391928e2536" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib175">Tran et al., 2013</a><div data-db-target-of="bib175-d10391928e2536" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tran M.N. </li><li class="loa__item">Choi W. </li><li class="loa__item">Wszolek M.F. </li><li class="loa__item">Navai N. </li><li class="loa__item">Lee I.-L.C. </li><li class="loa__item">Nitti G. </li><li class="loa__item">Wen S. </li><li class="loa__item">Flores E.R. </li><li class="loa__item">Siefker-Radtke A. </li><li class="loa__item">Czerniak B. </li><li>et al.</li></ul><div class="reference-citations__detials">The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">288</span></span>: <span class="ref__seriesPages">3275-3288</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23239884&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84873304587&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=3275-3288&journal=J.%C2%A0Biol.+Chem.&author=Tran+M.N.&author=Choi+W.&author=Wszolek+M.F.&author=Navai+N.&author=Lee+I.-L.C.&author=Nitti+G.&author=Wen+S.&author=Flores+E.R.&author=Siefker-Radtke+A.&author=Czerniak+B.&title=The+p63+protein+isoform+%CE%94Np63%CE%B1+inhibits+epithelial-mesenchymal+transition+in+human+bladder+cancer+cells%3A+role+of+MIR-205">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>D, <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>K). As expected, the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d10391928e2546" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Cai et al., 2013</a><div data-db-target-of="bib18-d10391928e2546" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cai J. </li><li class="loa__item">Fang L. </li><li class="loa__item">Huang Y. </li><li class="loa__item">Li R. </li><li class="loa__item">Yuan J. </li><li class="loa__item">Yang Y. </li><li class="loa__item">Zhu X. </li><li class="loa__item">Chen B. </li><li class="loa__item">Wu J. </li><li class="loa__item">Li M. </li></ul><div class="reference-citations__detials">miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">5402-5415</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-13-0297&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23856247&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84883478160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (145)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5402-5415&journal=Cancer+Res.&author=Cai+J.&author=Fang+L.&author=Huang+Y.&author=Li+R.&author=Yuan+J.&author=Yang+Y.&author=Zhu+X.&author=Chen+B.&author=Wu+J.&author=Li+M.&title=miR-205+targets+PTEN+and+PHLPP2+to+augment+AKT+signaling+and+drive+malignant+phenotypes+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d10391928e2549" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Gregory et al., 2008</a><div data-db-target-of="bib53-d10391928e2549" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gregory P.A. </li><li class="loa__item">Bert A.G. </li><li class="loa__item">Paterson E.L. </li><li class="loa__item">Barry S.C. </li><li class="loa__item">Tsykin A. </li><li class="loa__item">Farshid G. </li><li class="loa__item">Vadas M.A. </li><li class="loa__item">Khew-Goodall Y. </li><li class="loa__item">Goodall G.J. </li></ul><div class="reference-citations__detials">The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1.<div class="popup-ref"><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">593-601</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncb1722&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18376396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-43049103824&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2912)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=593-601&journal=Nat.+Cell+Biol.&author=Gregory+P.A.&author=Bert+A.G.&author=Paterson+E.L.&author=Barry+S.C.&author=Tsykin+A.&author=Farshid+G.&author=Vadas+M.A.&author=Khew-Goodall+Y.&author=Goodall+G.J.&title=The+miR-200+family+and+miR-205+regulate+epithelial+to+mesenchymal+transition+by+targeting+ZEB1+and+SIP1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib171" id="back-bib171" data-db-target-for="bib171-d10391928e2552" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib171">Tellez et al., 2011</a><div data-db-target-of="bib171-d10391928e2552" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tellez C.S. </li><li class="loa__item">Juri D.E. </li><li class="loa__item">Do K. </li><li class="loa__item">Bernauer A.M. </li><li class="loa__item">Thomas C.L. </li><li class="loa__item">Damiani L.A. </li><li class="loa__item">Tessema M. </li><li class="loa__item">Leng S. </li><li class="loa__item">Belinsky S.A. </li></ul><div class="reference-citations__detials">EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">71</span></span>: <span class="ref__seriesPages">3087-3097</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-10-3035&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21363915&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79954576652&amp;cf=" onclick="newWindow(this.href);return false">Scopus (238)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3087-3097&journal=Cancer+Res.&author=Tellez+C.S.&author=Juri+D.E.&author=Do+K.&author=Bernauer+A.M.&author=Thomas+C.L.&author=Damiani+L.A.&author=Tessema+M.&author=Leng+S.&author=Belinsky+S.A.&title=EMT+and+stem+cell-like+properties+associated+with+miR-205+and+miR-200+epigenetic+silencing+are+early+manifestations+during+carcinogen-induced+transformation+of+human+lung+epithelial+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib183" id="back-bib183" data-db-target-for="bib183-d10391928e2555" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib183">Vosgha et al., 2018</a><div data-db-target-of="bib183-d10391928e2555" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vosgha H. </li><li class="loa__item">Ariana A. </li><li class="loa__item">Smith R.A. </li><li class="loa__item">Lam A.K.-Y. </li></ul><div class="reference-citations__detials"><em>miR-205</em> targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma.<div class="popup-ref"><span class="ref__series"><i>Endocr. Relat. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">323-337</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1530%2FERC-17-0497&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29317480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85042625022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=323-337&journal=Endocr.+Relat.+Cancer&author=Vosgha+H.&author=Ariana+A.&author=Smith+R.A.&author=Lam+A.K.-Y.&title=miR-205+targets+angiogenesis+and+EMT+concurrently+in+anaplastic+thyroid+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>E). TP73 positively correlated with miR-205 at the transcript (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>E) and still more strongly at the protein level (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>F) and with TP63 protein but not RNA (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>L). TP73 overexpression mediates ΔNp63α-dependent cell survival in squamous cell carcinoma (<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d10391928e2571" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">DeYoung et al., 2006</a><div data-db-target-of="bib28-d10391928e2571" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">DeYoung M.P. </li><li class="loa__item">Johannessen C.M. </li><li class="loa__item">Leong C.-O. </li><li class="loa__item">Faquin W. </li><li class="loa__item">Rocco J.W. </li><li class="loa__item">Ellisen L.W. </li></ul><div class="reference-citations__detials">Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">66</span></span>: <span class="ref__seriesPages">9362-9368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-06-1619&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17018588&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33750378033&amp;cf=" onclick="newWindow(this.href);return false">Scopus (91)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=9362-9368&journal=Cancer+Res.&author=DeYoung+M.P.&author=Johannessen+C.M.&author=Leong+C.-O.&author=Faquin+W.&author=Rocco+J.W.&author=Ellisen+L.W.&title=Tumor-specific+p73+up-regulation+mediates+p63+dependence+in+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC (<div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d10391928e2574" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">Natan and Joerger, 2012</a><div data-db-target-of="bib125-d10391928e2574" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Natan E. </li><li class="loa__item">Joerger A.C. </li></ul><div class="reference-citations__detials">Structure and kinetic stability of the p63 tetramerization domain.<div class="popup-ref"><span class="ref__series"><i>J. Mol. Biol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">415</span></span>: <span class="ref__seriesPages">503-513</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jmb.2011.11.007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22100306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84855823856&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=415&publication_year=2012&pages=503-513&journal=J.%C2%A0Mol.+Biol.&author=Natan+E.&author=Joerger+A.C.&title=Structure+and+kinetic+stability+of+the+p63+tetramerization+domain">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>G). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in previous studies (<div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d10391928e2581" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Lindsay et al., 2011</a><div data-db-target-of="bib109-d10391928e2581" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lindsay J. </li><li class="loa__item">McDade S.S. </li><li class="loa__item">Pickard A. </li><li class="loa__item">McCloskey K.D. </li><li class="loa__item">McCance D.J. </li></ul><div class="reference-citations__detials">Role of DeltaNp63gamma in epithelial to mesenchymal transition.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">286</span></span>: <span class="ref__seriesPages">3915-3924</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21127042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79952811643&amp;cf=" onclick="newWindow(this.href);return false">Scopus (46)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=3915-3924&journal=J.%C2%A0Biol.+Chem.&author=Lindsay+J.&author=McDade+S.S.&author=Pickard+A.&author=McCloskey+K.D.&author=McCance+D.J.&title=Role+of+DeltaNp63gamma+in+epithelial+to+mesenchymal+transition">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib163" id="back-bib163" data-db-target-for="bib163-d10391928e2584" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib163">Srivastava et al., 2018</a><div data-db-target-of="bib163-d10391928e2584" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Srivastava K. </li><li class="loa__item">Pickard A. </li><li class="loa__item">Craig S.G. </li><li class="loa__item">Quinn G.P. </li><li class="loa__item">Lambe S.M. </li><li class="loa__item">James J.A. </li><li class="loa__item">McDade S.S. </li><li class="loa__item">McCance D.J. </li></ul><div class="reference-citations__detials">ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">3917-3927</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-17-3775&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29739791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85051740562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=3917-3927&journal=Clin.+Cancer+Res.&author=Srivastava+K.&author=Pickard+A.&author=Craig+S.G.&author=Quinn+G.P.&author=Lambe+S.M.&author=James+J.A.&author=McDade+S.S.&author=McCance+D.J.&title=%CE%94Np63%CE%B3%2FSRC%2FSlug+Signaling+Axis+Promotes+Epithelial-to-Mesenchymal+Transition+in+Squamous+Cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib175" id="back-bib175" data-db-target-for="bib175-d10391928e2587" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib175">Tran et al., 2013</a><div data-db-target-of="bib175-d10391928e2587" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tran M.N. </li><li class="loa__item">Choi W. </li><li class="loa__item">Wszolek M.F. </li><li class="loa__item">Navai N. </li><li class="loa__item">Lee I.-L.C. </li><li class="loa__item">Nitti G. </li><li class="loa__item">Wen S. </li><li class="loa__item">Flores E.R. </li><li class="loa__item">Siefker-Radtke A. </li><li class="loa__item">Czerniak B. </li><li>et al.</li></ul><div class="reference-citations__detials">The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">288</span></span>: <span class="ref__seriesPages">3275-3288</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23239884&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84873304587&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=3275-3288&journal=J.%C2%A0Biol.+Chem.&author=Tran+M.N.&author=Choi+W.&author=Wszolek+M.F.&author=Navai+N.&author=Lee+I.-L.C.&author=Nitti+G.&author=Wen+S.&author=Flores+E.R.&author=Siefker-Radtke+A.&author=Czerniak+B.&title=The+p63+protein+isoform+%CE%94Np63%CE%B1+inhibits+epithelial-mesenchymal+transition+in+human+bladder+cancer+cells%3A+role+of+MIR-205">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Of note, although ΔNp63α and miR-205 were consistently highly overexpressed in the B-I and classical subtypes compared to NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, ΔNp63α, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and classical tumors, with additional mechanisms leading to de-repression and a metastatic phenotype (<div class="dropBlock reference-citations"><a href="#bib183" id="back-bib183" data-db-target-for="bib183-d10391928e2590" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib183">Vosgha et al., 2018</a><div data-db-target-of="bib183-d10391928e2590" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vosgha H. </li><li class="loa__item">Ariana A. </li><li class="loa__item">Smith R.A. </li><li class="loa__item">Lam A.K.-Y. </li></ul><div class="reference-citations__detials"><em>miR-205</em> targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma.<div class="popup-ref"><span class="ref__series"><i>Endocr. Relat. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">323-337</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1530%2FERC-17-0497&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29317480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85042625022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=323-337&journal=Endocr.+Relat.+Cancer&author=Vosgha+H.&author=Ariana+A.&author=Smith+R.A.&author=Lam+A.K.-Y.&title=miR-205+targets+angiogenesis+and+EMT+concurrently+in+anaplastic+thyroid+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples.</div><div class="section-paragraph"><em>SOX2</em>, a recurrently amplified squamous cell marker often co-amplified with <em>TP63</em>, was also overexpressed in tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A). <em>SOX2</em> is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>H) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression (<div class="dropBlock reference-citations"><a href="#bib206" id="back-bib206" data-db-target-for="bib206-d10391928e2606" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib206">Zhang et al., 2013</a><div data-db-target-of="bib206-d10391928e2606" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang X. </li><li class="loa__item">Lu F. </li><li class="loa__item">Wang J. </li><li class="loa__item">Yin F. </li><li class="loa__item">Xu Z. </li><li class="loa__item">Qi D. </li><li class="loa__item">Wu X. </li><li class="loa__item">Cao Y. </li><li class="loa__item">Liang W. </li><li class="loa__item">Liu Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">445-457</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24139802&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84887021821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (79)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=445-457&journal=Cell+Rep.&author=Zhang+X.&author=Lu+F.&author=Wang+J.&author=Yin+F.&author=Xu+Z.&author=Qi+D.&author=Wu+X.&author=Cao+Y.&author=Liang+W.&author=Liu+Y.&title=Pluripotent+stem+cell+protein+Sox2+confers+sensitivity+to+LSD1+inhibition+in+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and its inhibition in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy (NCT04350463). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer (<div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d10391928e2610" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Ramadoss et al., 2017</a><div data-db-target-of="bib143-d10391928e2610" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ramadoss S. </li><li class="loa__item">Sen S. </li><li class="loa__item">Ramachandran I. </li><li class="loa__item">Roy S. </li><li class="loa__item">Chaudhuri G. </li><li class="loa__item">Farias-Eisner R. </li></ul><div class="reference-citations__detials">Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance.<div class="popup-ref"><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">6508</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fonc.2017.331&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28925393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034566788&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=6508&journal=Oncogene&author=Ramadoss+S.&author=Sen+S.&author=Ramachandran+I.&author=Roy+S.&author=Chaudhuri+G.&author=Farias-Eisner+R.&title=Lysine-specific+demethylase+KDM3A+regulates+ovarian+cancer+stemness+and+chemoresistance">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression (<div class="dropBlock reference-citations"><a href="#bib203" id="back-bib203" data-db-target-for="bib203-d10391928e2613" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib203">Yomtoubian et al., 2020</a><div data-db-target-of="bib203-d10391928e2613" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yomtoubian S. </li><li class="loa__item">Lee S.B. </li><li class="loa__item">Verma A. </li><li class="loa__item">Izzo F. </li><li class="loa__item">Markowitz G. </li><li class="loa__item">Choi H. </li><li class="loa__item">Cerchietti L. </li><li class="loa__item">Vahdat L. </li><li class="loa__item">Brown K.A. </li><li class="loa__item">Andreopoulou E. </li><li>et al.</li></ul><div class="reference-citations__detials">Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">755-770.e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31968251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078319103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=755-770.e6&journal=Cell+Rep.&author=Yomtoubian+S.&author=Lee+S.B.&author=Verma+A.&author=Izzo+F.&author=Markowitz+G.&author=Choi+H.&author=Cerchietti+L.&author=Vahdat+L.&author=Brown+K.A.&author=Andreopoulou+E.&title=Inhibition+of+EZH2+Catalytic+Activity+Selectively+Targets+a+Metastatic+Subpopulation+in+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). EZH2 dependency was observed in SOX2 amplified LSCC cell lines (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>M). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>H) suggest stemness-like features in these tumors (<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d10391928e2622" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Keysar et al., 2016</a><div data-db-target-of="bib78-d10391928e2622" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keysar S.B. </li><li class="loa__item">Le P.N. </li><li class="loa__item">Miller B. </li><li class="loa__item">Jackson B.C. </li><li class="loa__item">Eagles J.R. </li><li class="loa__item">Nieto C. </li><li class="loa__item">Kim J. </li><li class="loa__item">Tang B. </li><li class="loa__item">Glogowska M.J. </li><li class="loa__item">Morton J.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2.<div class="popup-ref"><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">djw189</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fjnci%2Fdjw189&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85014859275&amp;cf=" onclick="newWindow(this.href);return false">Scopus (46)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2016&pages=djw189&journal=J.%C2%A0Natl.+Cancer+Inst.&author=Keysar+S.B.&author=Le+P.N.&author=Miller+B.&author=Jackson+B.C.&author=Eagles+J.R.&author=Nieto+C.&author=Kim+J.&author=Tang+B.&author=Glogowska+M.J.&author=Morton+J.J.&title=Regulation+of+Head+and+Neck+Squamous+Cancer+Stem+Cells+by+PI3K+and+SOX2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d10391928e2625" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Liu et al., 2016</a><div data-db-target-of="bib110-d10391928e2625" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu X. </li><li class="loa__item">Wang L. </li><li class="loa__item">Cui W. </li><li class="loa__item">Yuan X. </li><li class="loa__item">Lin L. </li><li class="loa__item">Cao Q. </li><li class="loa__item">Wang N. </li><li class="loa__item">Li Y. </li><li class="loa__item">Guo W. </li><li class="loa__item">Zhang X. </li><li>et al.</li></ul><div class="reference-citations__detials">Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells.<div class="popup-ref"><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">58516-58530</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.18632%2Foncotarget.11305&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27542268&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84988355081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=58516-58530&journal=Oncotarget&author=Liu+X.&author=Wang+L.&author=Cui+W.&author=Yuan+X.&author=Lin+L.&author=Cao+Q.&author=Wang+N.&author=Li+Y.&author=Guo+W.&author=Zhang+X.&title=Targeting+ALDH1A1+by+disulfiram%2Fcopper+complex+inhibits+non-small+cell+lung+cancer+recurrence+driven+by+ALDH-positive+cancer+stem+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d10391928e2629" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Patel et al., 2008</a><div data-db-target-of="bib133-d10391928e2629" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Patel M. </li><li class="loa__item">Lu L. </li><li class="loa__item">Zander D.S. </li><li class="loa__item">Sreerama L. </li><li class="loa__item">Coco D. </li><li class="loa__item">Moreb J.S. </li></ul><div class="reference-citations__detials">ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors.<div class="popup-ref"><span class="ref__series"><i>Lung Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">59</span></span>: <span class="ref__seriesPages">340-349</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=abstract&amp;site=lung-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=fulltext&amp;site=lung-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=pdf&amp;site=lung-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17920722&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-40249108437&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2008&pages=340-349&journal=Lung+Cancer&author=Patel+M.&author=Lu+L.&author=Zander+D.S.&author=Sreerama+L.&author=Coco+D.&author=Moreb+J.S.&title=ALDH1A1+and+ALDH3A1+expression+in+lung+cancers%3A+correlation+with+histologic+type+and+potential+precursors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>H), and negative correlation was seen between SOX2 and JAK-STAT signaling (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>I).</div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism</span></h3><div class="section-paragraph">Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" class="scroll-into-link">Table S5</a>) associated with country of origin and ethnicity (Fisher’s exact p &lt; 0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>A and S5B; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" class="scroll-into-link">Table S5</a>).<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/8d326ce7-e5f5-4015-91bf-86ae68ee20f4/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/5a9dca8f-ef45-4929-bf17-c8e427d2c1a8/gr5_lrg.jpg" data-large-image="/cms/attachment/8d326ce7-e5f5-4015-91bf-86ae68ee20f4/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Ubiquitylation landscape in LSCC</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR &lt; 0.01). Enriched pathways and molecular and clinical annotations are indicated.</div><div class="section-paragraph">(B) Number of K-GG sites showing significant correlations (FDR &lt; 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations.</div><div class="section-paragraph">(C) Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1.</div><div class="section-paragraph">(D) HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal-Wallis, p = 2.8x10<sup>−05</sup>) association is seen between HERC5 abundance and immune subtypes.</div><div class="section-paragraph">(E) Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:<a class="inter-ref-link" href="http://firstglance.jmol.org/fg.htm?mol=3ZOZ" target="_blank">3ZOZ</a>. ACADVL structure = PDB ID:3B9).</div><div class="section-paragraph">(F) TXN protein levels in the NMF classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal-Wallis p values are indicated in the respective plots.</div><div class="section-paragraph">(G) Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted.</div><div class="section-paragraph">(H) Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)&gt;1 or Hypo: &lt; −1, FDR &lt; 0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypo-acetylated in tumors; the lower panel shows specific sites that were hyper-acetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. “k” represents modified lysine.</div><div class="section-paragraph">(I) Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG), and RAN protein levels across NMF subtypes and NATs. Wilcoxon p values are indicated above; ns represents p &gt; 0.05.</div><div class="section-paragraph">See also <a data-locator="figs5" aria-expanded="false" aria-controls="crosref2890" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs5">Figure S5</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2895" href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" target="_blank">Table S5</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/8d326ce7-e5f5-4015-91bf-86ae68ee20f4/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/5a9dca8f-ef45-4929-bf17-c8e427d2c1a8/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/e72a031e-1ef9-4a99-8b3c-9b0840212e13/figs5.jpg" alt="Figure thumbnail figs5" data-locator="figs5" data-high-res="/cms/attachment/8da12953-7963-4f03-afc4-762e4db95e07/figs5_lrg.jpg" data-large-image="/cms/attachment/e72a031e-1ef9-4a99-8b3c-9b0840212e13/figs5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S5" class="label figure__label">Figure S5</span><span class="figure__title__text">Pathway analysis and differential ubiquitylation and acetylation, related to <a data-locator="fig5" aria-expanded="false" aria-controls="crosref3005" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Network representation of pathways significantly enriched in modulated protein-corrected K-GG sites across Ub clusters (<a data-locator="fig5" aria-expanded="false" aria-controls="crosref3010" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a>A). Two patterns of K-GG site abundance were observed across Ub clusters (top). The red and blue arrows indicate up- and downregulation across the clusters, respectively. Significant pathways are indicated as nodes and edges indicate shared genes. Network clusters were summarized to reduce pathway redundancies and the common functional themes were manually annotated as text.</div><div class="section-paragraph">(B) STRING network showing the genes driving the enrichment of the “Immune signaling, MAPK targets, and JAK-STAT signaling” cluster shown in panel A. These genes are core members of many signaling pathways, resulting in enrichment of multiple pathways.</div><div class="section-paragraph">(C) Number of K-GG sites showing significant correlations with deubiquitylases (DUBs). Shown are the five DUBs with the highest proportion of negative correlations.</div><div class="section-paragraph">(D) HERC5 and ISG15 protein expression (Log2 TMT ratios) show significant positive correlation.</div><div class="section-paragraph">(E) Pathway enrichment analysis results of K-GG sites that significantly correlate with HERC5 protein expression.</div><div class="section-paragraph">(F) HERC5 protein expression (Log2 TMT ratio) and example K-GG site abundances (protein-corrected Log2 TMT ratio) from PKM, ENO1, and PGK1 show strong positive correlations in tumors.</div><div class="section-paragraph">(G) PKM, ENO1, and PGK1 plotted against sample K-GG site abundance of the respective proteins. K-GG site abundance is shown without protein correction (Log2 TMT ratio) to highlight the lack of negative correlation prior to protein correction.</div><div class="section-paragraph">(H) HERC5 protein expression (Log2 TMT ratio) plotted against PKM, ENO1, and PGK1 protein expression (Log2 TMT ratio), which do not show positive correlation in tumors.</div><div class="section-paragraph">(I) HERC5 protein expression (Log2 TMT ratio) and interferon-gamma (IFNG) score show significant positive correlation.</div><div class="section-paragraph">(J) ISG15 protein expression (Log2 TMT ratio) and IFNG score show significant positive correlation.</div><div class="section-paragraph">(K) PGK1 acetylated sites marked on the protein structure in <a data-locator="fig5" aria-expanded="false" aria-controls="crosref3015" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a>E (top left) are decreased in tumors compared to NATs; Ub sites are increased (top right). PKM acetylated sites (lower left) are decreased in tumors compared to NATs; Ub sites are increased (lower center). ACADVL acetylated sites marked on the protein structure in <a data-locator="fig5" aria-expanded="false" aria-controls="crosref3020" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a>E (lower right) are decreased in tumors compared to NATs. Wilcoxon p values are indicated.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/e72a031e-1ef9-4a99-8b3c-9b0840212e13/figs5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/8da12953-7963-4f03-afc4-762e4db95e07/figs5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" class="scroll-into-link">Table S5</a>). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B and <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>C. HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in humans (<div class="dropBlock reference-citations"><a href="#bib197" id="back-bib197" data-db-target-for="bib197-d10391928e2672" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib197">Wong et al., 2006</a><div data-db-target-of="bib197-d10391928e2672" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wong J.J.Y. </li><li class="loa__item">Pung Y.F. </li><li class="loa__item">Sze N.S.-K. </li><li class="loa__item">Chin K.-C. </li></ul><div class="reference-citations__detials">HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">103</span></span>: <span class="ref__seriesPages">10735-10740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0600397103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16815975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33746089547&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=10735-10740&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Wong+J.J.Y.&author=Pung+Y.F.&author=Sze+N.S.-K.&author=Chin+K.-C.&title=HERC5+is+an+IFN-induced+HECT-type+E3+protein+ligase+that+mediates+type+I+IFN-induced+ISGylation+of+protein+targets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); hence the numerous positively correlated Ub/UbL sites (n = 474; FDR &lt; 0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 abundance (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>D). Positively correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>E). We observed several positively correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>F). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>G, Figure S5H). ISG15 modifications have been described in these glycolytic enzymes (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d10391928e2691" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Albert et al., 2018</a><div data-db-target-of="bib3-d10391928e2691" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Albert M. </li><li class="loa__item">Bécares M. </li><li class="loa__item">Falqui M. </li><li class="loa__item">Fernández-Lozano C. </li><li class="loa__item">Guerra S. </li></ul><div class="reference-citations__detials">ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis.<div class="popup-ref"><span class="ref__series"><i>Viruses.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">629</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fv10110629&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30428561&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85056635216&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=629&journal=Viruses&author=Albert+M.&author=B%C3%A9cares+M.&author=Falqui+M.&author=Fern%C3%A1ndez-Lozano+C.&author=Guerra+S.&title=ISG15%2C+a+Small+Molecule+with+Huge+Implications%3A+Regulation+of+Mitochondrial+Homeostasis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d10391928e2694" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Giannakopoulos et al., 2005</a><div data-db-target-of="bib49-d10391928e2694" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Giannakopoulos N.V. </li><li class="loa__item">Luo J.-K. </li><li class="loa__item">Papov V. </li><li class="loa__item">Zou W. </li><li class="loa__item">Lenschow D.J. </li><li class="loa__item">Jacobs B.S. </li><li class="loa__item">Borden E.C. </li><li class="loa__item">Li J. </li><li class="loa__item">Virgin H.W. </li><li class="loa__item">Zhang D.-E. </li></ul><div class="reference-citations__detials">Proteomic identification of proteins conjugated to ISG15 in mouse and human cells.<div class="popup-ref"><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">336</span></span>: <span class="ref__seriesPages">496-506</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.bbrc.2005.08.132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16139798&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-24644474832&amp;cf=" onclick="newWindow(this.href);return false">Scopus (164)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2005&pages=496-506&journal=Biochem.+Biophys.+Res.+Commun.&author=Giannakopoulos+N.V.&author=Luo+J.-K.&author=Papov+V.&author=Zou+W.&author=Lenschow+D.J.&author=Jacobs+B.S.&author=Borden+E.C.&author=Li+J.&author=Virgin+H.W.&author=Zhang+D.-E.&title=Proteomic+identification+of+proteins+conjugated+to+ISG15+in+mouse+and+human+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib197" id="back-bib197" data-db-target-for="bib197-d10391928e2698" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib197">Wong et al., 2006</a><div data-db-target-of="bib197-d10391928e2698" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wong J.J.Y. </li><li class="loa__item">Pung Y.F. </li><li class="loa__item">Sze N.S.-K. </li><li class="loa__item">Chin K.-C. </li></ul><div class="reference-citations__detials">HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">103</span></span>: <span class="ref__seriesPages">10735-10740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0600397103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16815975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33746089547&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=10735-10740&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Wong+J.J.Y.&author=Pung+Y.F.&author=Sze+N.S.-K.&author=Chin+K.-C.&title=HERC5+is+an+IFN-induced+HECT-type+E3+protein+ligase+that+mediates+type+I+IFN-induced+ISGylation+of+protein+targets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d10391928e2701" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Alcalá et al., 2020</a><div data-db-target-of="bib4-d10391928e2701" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Alcalá S. </li><li class="loa__item">Sancho P. </li><li class="loa__item">Martinelli P. </li><li class="loa__item">Navarro D. </li><li class="loa__item">Pedrero C. </li><li class="loa__item">Martín-Hijano L. </li><li class="loa__item">Valle S. </li><li class="loa__item">Earl J. </li><li class="loa__item">Rodríguez-Serrano M. </li><li class="loa__item">Ruiz-Cañas L. </li><li>et al.</li></ul><div class="reference-citations__detials">ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2682</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-16395-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32472071&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85085688337&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2682&journal=Nat.+Commun.&author=Alcal%C3%A1+S.&author=Sancho+P.&author=Martinelli+P.&author=Navarro+D.&author=Pedrero+C.&author=Mart%C3%ADn-Hijano+L.&author=Valle+S.&author=Earl+J.&author=Rodr%C3%ADguez-Serrano+M.&author=Ruiz-Ca%C3%B1as+L.&title=ISG15+and+ISGylation+is+required+for+pancreatic+cancer+stem+cell+mitophagy+and+metabolic+plasticity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). ISGylation can be induced by interferon in cancer (<div class="dropBlock reference-citations"><a href="#bib170" id="back-bib170" data-db-target-for="bib170-d10391928e2704" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib170">Tecalco-Cruz and Cruz-Ramos, 2018</a><div data-db-target-of="bib170-d10391928e2704" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tecalco-Cruz A.C. </li><li class="loa__item">Cruz-Ramos E. </li></ul><div class="reference-citations__detials">Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells.<div class="popup-ref"><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">973-978</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.bbrc.2018.04.030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29626479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85045093507&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2018&pages=973-978&journal=Biochem.+Biophys.+Res.+Commun.&author=Tecalco-Cruz+A.C.&author=Cruz-Ramos+E.&title=Protein+ISGylation+and+free+ISG15+levels+are+increased+by+interferon+gamma+in+breast+cancer+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib197" id="back-bib197" data-db-target-for="bib197-d10391928e2707" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib197">Wong et al., 2006</a><div data-db-target-of="bib197-d10391928e2707" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wong J.J.Y. </li><li class="loa__item">Pung Y.F. </li><li class="loa__item">Sze N.S.-K. </li><li class="loa__item">Chin K.-C. </li></ul><div class="reference-citations__detials">HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">103</span></span>: <span class="ref__seriesPages">10735-10740</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0600397103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16815975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33746089547&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=10735-10740&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Wong+J.J.Y.&author=Pung+Y.F.&author=Sze+N.S.-K.&author=Chin+K.-C.&title=HERC5+is+an+IFN-induced+HECT-type+E3+protein+ligase+that+mediates+type+I+IFN-induced+ISGylation+of+protein+targets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figures S5</a>I and S5J), and HERC5 was increased in tumors of the IFN-gamma dominant subtype (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>D). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion (<div class="dropBlock reference-citations"><a href="#bib208" id="back-bib208" data-db-target-for="bib208-d10391928e2717" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib208">Zhang et al., 2019b</a><div data-db-target-of="bib208-d10391928e2717" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang Y. </li><li class="loa__item">Thery F. </li><li class="loa__item">Wu N.C. </li><li class="loa__item">Luhmann E.K. </li><li class="loa__item">Dussurget O. </li><li class="loa__item">Foecke M. </li><li class="loa__item">Bredow C. </li><li class="loa__item">Jiménez-Fernández D. </li><li class="loa__item">Leandro K. </li><li class="loa__item">Beling A. </li><li>et al.</li></ul><div class="reference-citations__detials">The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">5383</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-13393-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31772204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85075610966&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=5383&journal=Nat.+Commun.&author=Zhang+Y.&author=Thery+F.&author=Wu+N.C.&author=Luhmann+E.K.&author=Dussurget+O.&author=Foecke+M.&author=Bredow+C.&author=Jim%C3%A9nez-Fern%C3%A1ndez+D.&author=Leandro+K.&author=Beling+A.&title=The+in%C2%A0vivo+ISGylome+links+ISG15+to+metabolic+pathways+and+autophagy+upon+Listeria+monocytogenes+infection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Collectively, our data support a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC.</div><div class="section-paragraph">This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d10391928e2722" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Kamburov et al., 2015</a><div data-db-target-of="bib74-d10391928e2722" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kamburov A. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Polak P. </li><li class="loa__item">Leshchiner I. </li><li class="loa__item">Lage K. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Comprehensive assessment of cancer missense mutation clustering in protein structures.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">E5486-E5495</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1516373112&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26392535&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943411478&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E5486-E5495&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Kamburov+A.&author=Lawrence+M.S.&author=Polak+P.&author=Leshchiner+I.&author=Lage+K.&author=Golub+T.R.&author=Lander+E.S.&author=Getz+G.&title=Comprehensive+assessment+of+cancer+missense+mutation+clustering+in+protein+structures">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p &lt; 0.05) (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" class="scroll-into-link">Table S5</a>). Top hits revealed enrichment of glycolysis, fatty acid metabolism, and OxPhos (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5" class="scroll-into-link">Table S5</a>), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket (<div class="dropBlock reference-citations"><a href="#bib186" id="back-bib186" data-db-target-for="bib186-d10391928e2731" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib186">Wang et al., 2015</a><div data-db-target-of="bib186-d10391928e2731" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang S. </li><li class="loa__item">Jiang B. </li><li class="loa__item">Zhang T. </li><li class="loa__item">Liu L. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Wang Y. </li><li class="loa__item">Chen X. </li><li class="loa__item">Lin H. </li><li class="loa__item">Zhou L. </li><li class="loa__item">Xia Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1.<div class="popup-ref"><span class="ref__series"><i>PLoS Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">e1002243</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1371%2Fjournal.pbio.1002243&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26356530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943165788&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=e1002243&journal=PLoS+Biol.&author=Wang+S.&author=Jiang+B.&author=Zhang+T.&author=Liu+L.&author=Wang+Y.&author=Wang+Y.&author=Chen+X.&author=Lin+H.&author=Zhou+L.&author=Xia+Y.&title=Insulin+and+mTOR+Pathway+Regulate+HDAC3-Mediated+Deacetylation+and+Activation+of+PGK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib200" id="back-bib200" data-db-target-for="bib200-d10391928e2734" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib200">Xiong et al., 2011</a><div data-db-target-of="bib200-d10391928e2734" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xiong Y. </li><li class="loa__item">Lei Q.-Y. </li><li class="loa__item">Zhao S. </li><li class="loa__item">Guan K.-L. </li></ul><div class="reference-citations__detials">Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK.<div class="popup-ref"><span class="ref__series"><i>Cold Spring Harb. Symp. Quant. Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">76</span></span>: <span class="ref__seriesPages">285-289</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fsqb.2011.76.010942&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22096030&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84867414358&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=285-289&journal=Cold+Spring+Harb.+Symp.+Quant.+Biol.&author=Xiong+Y.&author=Lei+Q.-Y.&author=Zhao+S.&author=Guan+K.-L.&title=Regulation+of+glycolysis+and+gluconeogenesis+by+acetylation+of+PKM+and+PEPCK">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib209" id="back-bib209" data-db-target-for="bib209-d10391928e2738" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib209">Zhao et al., 2014</a><div data-db-target-of="bib209-d10391928e2738" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhao D. </li><li class="loa__item">Li F.-L. </li><li class="loa__item">Cheng Z.-L. </li><li class="loa__item">Lei Q.-Y. </li></ul><div class="reference-citations__detials">Impact of acetylation on tumor metabolism.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">e963452</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4161%2F23723548.2014.963452&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27308346&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85029407726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=e963452&journal=Mol.+Cell.+Oncol.&author=Zhao+D.&author=Li+F.-L.&author=Cheng+Z.-L.&author=Lei+Q.-Y.&title=Impact+of+acetylation+on+tumor+metabolism">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>K). Tumors also had decreased ubiquitylation at closely positioned K-GG-sites for TXN1, an important redox regulator (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>F). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37) (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d10391928e2750" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Bando et al., 2005</a><div data-db-target-of="bib11-d10391928e2750" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bando H. </li><li class="loa__item">Atsumi T. </li><li class="loa__item">Nishio T. </li><li class="loa__item">Niwa H. </li><li class="loa__item">Mishima S. </li><li class="loa__item">Shimizu C. </li><li class="loa__item">Yoshioka N. </li><li class="loa__item">Bucala R. </li><li class="loa__item">Koike T. </li></ul><div class="reference-citations__detials">Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">5784-5792</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-05-0149&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16115917&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-23844517036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=5784-5792&journal=Clin.+Cancer+Res.&author=Bando+H.&author=Atsumi+T.&author=Nishio+T.&author=Niwa+H.&author=Mishima+S.&author=Shimizu+C.&author=Yoshioka+N.&author=Bucala+R.&author=Koike+T.&title=Phosphorylation+of+the+6-phosphofructo-2-kinase%2Ffructose+2%2C6-bisphosphatase%2FPFKFB3+family+of+glycolytic+regulators+in+human+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib201" id="back-bib201" data-db-target-for="bib201-d10391928e2753" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib201">Yang and Lu, 2013</a><div data-db-target-of="bib201-d10391928e2753" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yang W. </li><li class="loa__item">Lu Z. </li></ul><div class="reference-citations__detials">Nuclear PKM2 regulates the Warburg effect.<div class="popup-ref"><span class="ref__series"><i>Cell Cycle.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">3154-3158</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4161%2Fcc.26182&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24013426&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84886704519&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=3154-3158&journal=Cell+Cycle&author=Yang+W.&author=Lu+Z.&title=Nuclear+PKM2+regulates+the+Warburg+effect">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Upregulation of a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs (<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d10391928e2757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">DeBerardinis and Chandel, 2016</a><div data-db-target-of="bib26-d10391928e2757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">DeBerardinis R.J. </li><li class="loa__item">Chandel N.S. </li></ul><div class="reference-citations__detials">Fundamentals of cancer metabolism.<div class="popup-ref"><span class="ref__series"><i>Sci. Adv.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">e1600200</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fsciadv.1600200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27386546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600200&journal=Sci.+Adv.&author=DeBerardinis+R.J.&author=Chandel+N.S.&title=Fundamentals+of+cancer+metabolism">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d10391928e2760" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Hitosugi and Chen, 2014</a><div data-db-target-of="bib61-d10391928e2760" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hitosugi T. </li><li class="loa__item">Chen J. </li></ul><div class="reference-citations__detials">Post-translational modifications and the Warburg effect.<div class="popup-ref"><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">4279-4285</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fonc.2013.406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24096483&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84906937144&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=4279-4285&journal=Oncogene&author=Hitosugi+T.&author=Chen+J.&title=Post-translational+modifications+and+the+Warburg+effect">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark.</div><div class="section-paragraph">Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF classical subtype (Kruskal Wallis FDR &lt; 0.01). The TXN1 activator TXNRD1 (<div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d10391928e2765" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Cadenas et al., 2010</a><div data-db-target-of="bib17-d10391928e2765" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cadenas C. </li><li class="loa__item">Franckenstein D. </li><li class="loa__item">Schmidt M. </li><li class="loa__item">Gehrmann M. </li><li class="loa__item">Hermes M. </li><li class="loa__item">Geppert B. </li><li class="loa__item">Schormann W. </li><li class="loa__item">Maccoux L.J. </li><li class="loa__item">Schug M. </li><li class="loa__item">Schumann A. </li><li>et al.</li></ul><div class="reference-citations__detials">Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer.<div class="popup-ref"><span class="ref__series"><i>Breast Cancer Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R44</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fbcr2599&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20584310&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78049394036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (139)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=R44&journal=Breast+Cancer+Res.&author=Cadenas+C.&author=Franckenstein+D.&author=Schmidt+M.&author=Gehrmann+M.&author=Hermes+M.&author=Geppert+B.&author=Schormann+W.&author=Maccoux+L.J.&author=Schug+M.&author=Schumann+A.&title=Role+of+thioredoxin+reductase+1+and+thioredoxin+interacting+protein+in+prognosis+of+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was also increased in the classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1 (<div class="dropBlock reference-citations"><a href="#bib123" id="back-bib123" data-db-target-for="bib123-d10391928e2768" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Morrison et al., 2014</a><div data-db-target-of="bib123-d10391928e2768" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Morrison J.A. </li><li class="loa__item">Pike L.A. </li><li class="loa__item">Sams S.B. </li><li class="loa__item">Sharma V. </li><li class="loa__item">Zhou Q. </li><li class="loa__item">Severson J.J. </li><li class="loa__item">Tan A.-C. </li><li class="loa__item">Wood W.M. </li><li class="loa__item">Haugen B.R. </li></ul><div class="reference-citations__detials">Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer.<div class="popup-ref"><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1476-4598-13-62&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24645981&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84899506046&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=62&journal=Mol.+Cancer&author=Morrison+J.A.&author=Pike+L.A.&author=Sams+S.B.&author=Sharma+V.&author=Zhou+Q.&author=Severson+J.J.&author=Tan+A.-C.&author=Wood+W.M.&author=Haugen+B.R.&title=Thioredoxin+interacting+protein+%28TXNIP%29+is+a+novel+tumor+suppressor+in+thyroid+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was decreased (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>F). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R = 0.85; TXNRD1: R = 0.8, both p &lt; 2.2x10<sup>−16</sup>), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation, and prognosis in multiple epithelial cancers (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d10391928e2777" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Fu et al., 2017</a><div data-db-target-of="bib42-d10391928e2777" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fu B. </li><li class="loa__item">Meng W. </li><li class="loa__item">Zeng X. </li><li class="loa__item">Zhao H. </li><li class="loa__item">Liu W. </li><li class="loa__item">Zhang T. </li></ul><div class="reference-citations__detials">TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma.<div class="popup-ref"><span class="ref__series"><i>BioMed Res. Int.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2017</span></span>: <span class="ref__seriesPages">4698167</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1155%2F2017%2F4698167&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28536696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85018902496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=4698167&journal=BioMed+Res.+Int.&author=Fu+B.&author=Meng+W.&author=Zeng+X.&author=Zhao+H.&author=Liu+W.&author=Zhang+T.&title=TXNRD1+Is+an+Unfavorable+Prognostic+Factor+for+Patients+with+Hepatocellular+Carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), including NSCLC (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d10391928e2781" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Fernandes et al., 2009</a><div data-db-target-of="bib36-d10391928e2781" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fernandes A.P. </li><li class="loa__item">Capitanio A. </li><li class="loa__item">Selenius M. </li><li class="loa__item">Brodin O. </li><li class="loa__item">Rundlöf A.-K. </li><li class="loa__item">Björnstedt M. </li></ul><div class="reference-citations__detials">Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation.<div class="popup-ref"><span class="ref__series"><i>Histopathology.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">55</span></span>: <span class="ref__seriesPages">313-320</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Fj.1365-2559.2009.03381.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19723146&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-69549116697&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2009&pages=313-320&journal=Histopathology&author=Fernandes+A.P.&author=Capitanio+A.&author=Selenius+M.&author=Brodin+O.&author=Rundl%C3%B6f+A.-K.&author=Bj%C3%B6rnstedt+M.&title=Expression+profiles+of+thioredoxin+family+proteins+in+human+lung+cancer+tissue%3A+correlation+with+proliferation+and+differentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The small proportion of glucose directed to the tumor mitochondria (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d10391928e2784" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Fan et al., 2019</a><div data-db-target-of="bib34-d10391928e2784" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fan T. </li><li class="loa__item">Sun G. </li><li class="loa__item">Sun X. </li><li class="loa__item">Zhao L. </li><li class="loa__item">Zhong R. </li><li class="loa__item">Peng Y. </li></ul><div class="reference-citations__detials">Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">317</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcancers11030317&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064332549&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=317&journal=Cancers+%28Basel%29&author=Fan+T.&author=Sun+G.&author=Sun+X.&author=Zhao+L.&author=Zhong+R.&author=Peng+Y.&title=Tumor+Energy+Metabolism+and+Potential+of+3-Bromopyruvate+as+an+Inhibitor+of+Aerobic+Glycolysis%3A+Implications+in+Tumor+Treatment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis) (<div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d10391928e2787" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Baeza et al., 2015</a><div data-db-target-of="bib9-d10391928e2787" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Baeza J. </li><li class="loa__item">Smallegan M.J. </li><li class="loa__item">Denu J.M. </li></ul><div class="reference-citations__detials">Site-specific reactivity of nonenzymatic lysine acetylation.<div class="popup-ref"><span class="ref__series"><i>ACS Chem. Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">122-128</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1021%2Fcb500848p&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25555129&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84921405747&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=122-128&journal=ACS+Chem.+Biol.&author=Baeza+J.&author=Smallegan+M.J.&author=Denu+J.M.&title=Site-specific+reactivity+of+nonenzymatic+lysine+acetylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d10391928e2790" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Gandhi and Das, 2019</a><div data-db-target-of="bib43-d10391928e2790" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gandhi N. </li><li class="loa__item">Das G.M. </li></ul><div class="reference-citations__detials">Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications.<div class="popup-ref"><span class="ref__series"><i>Cells.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">89</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcells8020089&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=89&journal=Cells&author=Gandhi+N.&author=Das+G.M.&title=Metabolic+Reprogramming+in+Breast+Cancer+and+Its+Therapeutic+Implications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d10391928e2793" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Hebert et al., 2013</a><div data-db-target-of="bib58-d10391928e2793" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hebert A.S. </li><li class="loa__item">Dittenhafer-Reed K.E. </li><li class="loa__item">Yu W. </li><li class="loa__item">Bailey D.J. </li><li class="loa__item">Selen E.S. </li><li class="loa__item">Boersma M.D. </li><li class="loa__item">Carson J.J. </li><li class="loa__item">Tonelli M. </li><li class="loa__item">Balloon A.J. </li><li class="loa__item">Higbee A.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">186-199</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23201123&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872276165&amp;cf=" onclick="newWindow(this.href);return false">Scopus (400)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=186-199&journal=Mol.+Cell&author=Hebert+A.S.&author=Dittenhafer-Reed+K.E.&author=Yu+W.&author=Bailey+D.J.&author=Selen+E.S.&author=Boersma+M.D.&author=Carson+J.J.&author=Tonelli+M.&author=Balloon+A.J.&author=Higbee+A.J.&title=Calorie+restriction+and+SIRT3+trigger+global+reprogramming+of+the+mitochondrial+protein+acetylome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>K). PTM-based modulation of these key metabolic and ROS pathways is summarized in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>G.</div><div class="section-paragraph">As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>H shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression, and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import/export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>I), named in part for its generally high proliferation (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious.</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune landscape and regulation in LSCC</span></h3><div class="section-paragraph">Consensus clustering based on the xCell (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d10391928e2822" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Aran et al., 2017</a><div data-db-target-of="bib6-d10391928e2822" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) signatures (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A–6C), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher’s exact p = 6.371x10<sup>−09</sup>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6C; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>C and S6D) and showed upregulation of immune-related pathways (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>A; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274), and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, mitochondrial complex, and TCA cycle pathways, consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D). Notably, NATs did not display consonant immune signaling (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>B). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, interferon gamma (IFN-G) response, and allograft rejection relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>A). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>E; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>)<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/efb4134c-bea9-45f5-9aa6-76002cdcd062/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/2c603eae-1342-482b-a84c-b172ae25094b/gr6_lrg.jpg" data-large-image="/cms/attachment/efb4134c-bea9-45f5-9aa6-76002cdcd062/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Immune landscape of LSCC</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA, and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA and global protein abundance (common) or on global protein abundance alone (proteomics only).</div><div class="section-paragraph">(B) Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p values for the individual comparisons are provided on top.</div><div class="section-paragraph">(C) Contour plot of two-dimensional density based on macrophage (x axis) and CD8 T cell scores (y axis) showing the variation in these cell types’ distributions observed across the different immune clusters.</div><div class="section-paragraph">(D) Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation (OxPhos) pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR &lt; 0.1).</div><div class="section-paragraph">(E) Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB.</div><div class="section-paragraph">(F) Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p values are reported.</div><div class="section-paragraph">(G) RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d10391928e258" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Aran et al., 2017</a><div data-db-target-of="bib6-d10391928e258" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) and the third pathway scores based on global protein abundance.</div><div class="section-paragraph">(H) Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots.</div><div class="section-paragraph">(I) Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also <a data-locator="figs6" aria-expanded="false" aria-controls="crosref2905" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs6">Figure S6</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2910" href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" target="_blank">Table S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/efb4134c-bea9-45f5-9aa6-76002cdcd062/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/2c603eae-1342-482b-a84c-b172ae25094b/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/c05878b3-0349-4c53-a6db-c57192520f0c/figs6.jpg" alt="Figure thumbnail figs6" data-locator="figs6" data-high-res="/cms/attachment/f938c101-da6a-424d-b314-05daff0b2c4f/figs6_lrg.jpg" data-large-image="/cms/attachment/c05878b3-0349-4c53-a6db-c57192520f0c/figs6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S6" class="label figure__label">Figure S6</span><span class="figure__title__text">Differential pathway, CBPE scores, and impact of germline mutations on immune signaling, related to <a data-locator="fig6" aria-expanded="false" aria-controls="crosref3025" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig6">Figure 6</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Key pathways differentially expressed between Hot, Warm and Cold tumors based on their gene expression and global protein abundance. Wilcoxon p values are reported.</div><div class="section-paragraph">(B) Key pathways differentially expressed between Hot, Warm and Cold tumors based on the gene expression and global proteomic abundance of corresponding NATs. Wilcoxon p values are reported.</div><div class="section-paragraph">(C) Averages of CD163, CD4, and CD8 IHC scores derived from three independent pathologists on four cores/sample from 17 samples representing Hot (5), Warm (5) and Cold (7) clusters. P values are based on ordinal regression analysis of immune cluster scores, using random effects to account for multiple cores/samples and independent pathologists.</div><div class="section-paragraph">(D) Representative immunohistochemical staining pattern for CD163, CD8, and CD4 proteins on three samples drawn from Hot, Warm, and Cold immune clusters. Protein expression levels in different cell types are highlighted with colored arrowheads as indicated.</div><div class="section-paragraph">(E) Overlap between differentially regulated mRNA and proteins between Hot versus Cold and Warm versus Cold tumors, and eGenes downstream of <em>cis</em>-eQTLs and <em>cis</em>-pQTLs. Recent studies have suggested the importance of germline variants in the immune profiles of solid tumors (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d10391928e587" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Lim et al., 2018</a><div data-db-target-of="bib108-d10391928e587" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lim Y.W. </li><li class="loa__item">Chen-Harris H. </li><li class="loa__item">Mayba O. </li><li class="loa__item">Lianoglou S. </li><li class="loa__item">Wuster A. </li><li class="loa__item">Bhangale T. </li><li class="loa__item">Khan Z. </li><li class="loa__item">Mariathasan S. </li><li class="loa__item">Daemen A. </li><li class="loa__item">Reeder J. </li><li>et al.</li></ul><div class="reference-citations__detials">Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">115</span></span>: <span class="ref__seriesPages">E11701-E11710</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1804506115&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30463956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85058338986&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E11701-E11710&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Lim+Y.W.&author=Chen-Harris+H.&author=Mayba+O.&author=Lianoglou+S.&author=Wuster+A.&author=Bhangale+T.&author=Khan+Z.&author=Mariathasan+S.&author=Daemen+A.&author=Reeder+J.&title=Germline+genetic+polymorphisms+influence+tumor+gene+expression+and+immune+cell+infiltration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib153" id="back-bib153" data-db-target-for="bib153-d10391928e590" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib153">Shahamatdar et al., 2020</a><div data-db-target-of="bib153-d10391928e590" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shahamatdar S. </li><li class="loa__item">He M.X. </li><li class="loa__item">Reyna M.A. </li><li class="loa__item">Gusev A. </li><li class="loa__item">AlDubayan S.H. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">Ramachandran S. </li></ul><div class="reference-citations__detials">Germline Features Associated with Immune Infiltration in Solid Tumors.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">2900-2908.e4</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32130895&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85080113093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=2900-2908.e4&journal=Cell+Rep.&author=Shahamatdar+S.&author=He+M.X.&author=Reyna+M.A.&author=Gusev+A.&author=AlDubayan+S.H.&author=Van+Allen+E.M.&author=Ramachandran+S.&title=Germline+Features+Associated+with+Immune+Infiltration+in+Solid+Tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>; <div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib173" id="back-bib173" data-db-target-for="bib173-d10391928e593" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib173">Tian et al., 2021</a><div data-db-target-of="bib173-d10391928e593" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tian J. </li><li class="loa__item">Cai Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">Lu Z. </li><li class="loa__item">Huang J. </li><li class="loa__item">Deng Y. </li><li class="loa__item">Yang N. </li><li class="loa__item">Wang X. </li><li class="loa__item">Ying P. </li><li class="loa__item">Zhang S. </li><li>et al.</li></ul><div class="reference-citations__detials">CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">D1065-D1073</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkaa805&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33010176&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099428264&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2021&pages=D1065-D1073&journal=Nucleic+Acids+Res.&author=Tian+J.&author=Cai+Y.&author=Li+Y.&author=Lu+Z.&author=Huang+J.&author=Deng+Y.&author=Yang+N.&author=Wang+X.&author=Ying+P.&author=Zhang+S.&title=CancerImmunityQTL%3A+a+database+to+systematically+evaluate+the+impact+of+genetic+variants+on+immune+infiltration+in+human+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). Among 319 <em>cis</em>-pQTLs (<em>cis</em>-regulating the cognate protein) identified in this study, only 38 were also <em>cis</em>-eQTLs (<em>cis</em>-regulating cognate RNA)(<a data-locator="mmc6" aria-expanded="false" aria-controls="crosref3045" href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" target="_blank">Table S6</a>). Interestingly, the set of all proteins with pQTLs (but not the set of all genes with eQTLs) was enriched in pathways involved in communication between innate and adaptive immune cells. The 15 pQTLs in proteins differentially abundant between Hot and Cold tumors included proteins related to immune response (TRIM22), antigen processing (TAPBPL and HLA-A) and IL-6 signaling pathway (LRPPRC); eight were previously reported (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d10391928e608" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Lim et al., 2018</a><div data-db-target-of="bib108-d10391928e608" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lim Y.W. </li><li class="loa__item">Chen-Harris H. </li><li class="loa__item">Mayba O. </li><li class="loa__item">Lianoglou S. </li><li class="loa__item">Wuster A. </li><li class="loa__item">Bhangale T. </li><li class="loa__item">Khan Z. </li><li class="loa__item">Mariathasan S. </li><li class="loa__item">Daemen A. </li><li class="loa__item">Reeder J. </li><li>et al.</li></ul><div class="reference-citations__detials">Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">115</span></span>: <span class="ref__seriesPages">E11701-E11710</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1804506115&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30463956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85058338986&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E11701-E11710&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Lim+Y.W.&author=Chen-Harris+H.&author=Mayba+O.&author=Lianoglou+S.&author=Wuster+A.&author=Bhangale+T.&author=Khan+Z.&author=Mariathasan+S.&author=Daemen+A.&author=Reeder+J.&title=Germline+genetic+polymorphisms+influence+tumor+gene+expression+and+immune+cell+infiltration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>). These results imply a potentially important role of germline variants in shaping immune processes in LSCC.</div><div class="section-paragraph">(F) Protein abundance and/or phosphorylation abundance (Log2 TMT ratio) of key markers involved in Rho GTPase signaling stratified by immune clusters. Wilcoxon p values are reported.</div><div class="section-paragraph">(G) Immunohistochemical staining images of ARHGDIB (RhoGDIB) protein in two tumors. Its expression is predominantly observed in tumor microenvironments with different types of cells indicated with differently colored arrowheads.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/c05878b3-0349-4c53-a6db-c57192520f0c/figs6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/f938c101-da6a-424d-b314-05daff0b2c4f/figs6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Rho GTPase signaling was upregulated in Hot tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d10391928e2886" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Garcia-Mata et al., 2011</a><div data-db-target-of="bib45-d10391928e2886" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Garcia-Mata R. </li><li class="loa__item">Boulter E. </li><li class="loa__item">Burridge K. </li></ul><div class="reference-citations__detials">The ‘invisible hand’: regulation of RHO GTPases by RHOGDIs.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">493-504</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnrm3153&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21779026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79960706295&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=493-504&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Garcia-Mata+R.&author=Boulter+E.&author=Burridge+K.&title=The+%E2%80%98invisible+hand%E2%80%99%3A+regulation+of+RHO+GTPases+by+RHOGDIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial enzymes (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly (<div class="dropBlock reference-citations"><a href="#bib89" id="back-bib89" data-db-target-for="bib89-d10391928e2892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">Kuhlmann et al., 2016a</a><div data-db-target-of="bib89-d10391928e2892" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuhlmann N. </li><li class="loa__item">Wroblowski S. </li><li class="loa__item">Knyphausen P. </li><li class="loa__item">de Boor S. </li><li class="loa__item">Brenig J. </li><li class="loa__item">Zienert A.Y. </li><li class="loa__item">Meyer-Teschendorf K. </li><li class="loa__item">Praefcke G.J.K. </li><li class="loa__item">Nolte H. </li><li class="loa__item">Krüger M. </li><li>et al.</li></ul><div class="reference-citations__detials">Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">291</span></span>: <span class="ref__seriesPages">5484-5499</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26719334&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84974850910&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=5484-5499&journal=J.%C2%A0Biol.+Chem.&author=Kuhlmann+N.&author=Wroblowski+S.&author=Knyphausen+P.&author=de+Boor+S.&author=Brenig+J.&author=Zienert+A.Y.&author=Meyer-Teschendorf+K.&author=Praefcke+G.J.K.&author=Nolte+H.&author=Kr%C3%BCger+M.&title=Structural+and+Mechanistic+Insights+into+the+Regulation+of+the+Fundamental+Rho+Regulator+RhoGDI%CE%B1+by+Lysine+Acetylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) by activating RhoA (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d10391928e2896" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">Kuhlmann et al., 2016b</a><div data-db-target-of="bib90-d10391928e2896" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuhlmann N. </li><li class="loa__item">Wroblowski S. </li><li class="loa__item">Scislowski L. </li><li class="loa__item">Lammers M. </li></ul><div class="reference-citations__detials">RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA.<div class="popup-ref"><span class="ref__series"><i>Biochemistry.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">55</span></span>: <span class="ref__seriesPages">304-312</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1021%2Facs.biochem.5b01242&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26695096&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84955559442&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=304-312&journal=Biochemistry&author=Kuhlmann+N.&author=Wroblowski+S.&author=Scislowski+L.&author=Lammers+M.&title=RhoGDI%CE%B1+Acetylation+at+K127+and+K141+Affects+Binding+toward+Nonprenylated+RhoA">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low (<div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d10391928e2899" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Gorvel et al., 1998</a><div data-db-target-of="bib52-d10391928e2899" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gorvel J.P. </li><li class="loa__item">Chang T.C. </li><li class="loa__item">Boretto J. </li><li class="loa__item">Azuma T. </li><li class="loa__item">Chavrier P. </li></ul><div class="reference-citations__detials">Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity.<div class="popup-ref"><span class="ref__series"><i>FEBS Lett.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">422</span></span>: <span class="ref__seriesPages">269-273</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0014-5793%2898%2900020-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=9490022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0032579315&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=422&publication_year=1998&pages=269-273&journal=FEBS+Lett.&author=Gorvel+J.P.&author=Chang+T.C.&author=Boretto+J.&author=Azuma+T.&author=Chavrier+P.&title=Differential+properties+of+D4%2FLyGDI+versus+RhoGDI%3A+phosphorylation+and+rho+GTPase+selectivity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead (<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d10391928e2902" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Moissoglu et al., 2009</a><div data-db-target-of="bib122-d10391928e2902" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Moissoglu K. </li><li class="loa__item">McRoberts K.S. </li><li class="loa__item">Meier J.A. </li><li class="loa__item">Theodorescu D. </li><li class="loa__item">Schwartz M.A. </li></ul><div class="reference-citations__detials">Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">2838-2844</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-08-1397&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19276387&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-66149117890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2838-2844&journal=Cancer+Res.&author=Moissoglu+K.&author=McRoberts+K.S.&author=Meier+J.A.&author=Theodorescu+D.&author=Schwartz+M.A.&title=Rho+GDP+dissociation+inhibitor+2+suppresses+metastasis+via+unconventional+regulation+of+RhoGTPases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>F; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>F; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>). Notably, these proteins are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>G), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas (<div class="dropBlock reference-citations"><a href="#bib182" id="back-bib182" data-db-target-for="bib182-d10391928e2918" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib182">von Klot et al., 2017</a><div data-db-target-of="bib182-d10391928e2918" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">von Klot C.-A. </li><li class="loa__item">Dubrowinskaja N. </li><li class="loa__item">Peters I. </li><li class="loa__item">Hennenlotter J. </li><li class="loa__item">Merseburger A.S. </li><li class="loa__item">Stenzl A. </li><li class="loa__item">Kuczyk M.A. </li><li class="loa__item">Serth J. </li></ul><div class="reference-citations__detials">Rho GDP dissociation inhibitor-β in renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Oncol. Lett.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">8190-8196</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_192_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29250194&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=8190-8196&journal=Oncol.+Lett.&author=von+Klot+C.-A.&author=Dubrowinskaja+N.&author=Peters+I.&author=Hennenlotter+J.&author=Merseburger+A.S.&author=Stenzl+A.&author=Kuczyk+M.A.&author=Serth+J.&title=Rho+GDP+dissociation+inhibitor-%CE%B2+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature (<div class="dropBlock reference-citations"><a href="#bib185" id="back-bib185" data-db-target-for="bib185-d10391928e2921" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib185">Wang et al., 2012</a><div data-db-target-of="bib185-d10391928e2921" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang I.-M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Yang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Stepaniants S. </li><li class="loa__item">Zhang C. </li><li class="loa__item">Meng Q. </li><li class="loa__item">Peters M. </li><li class="loa__item">He Y. </li><li class="loa__item">Ni C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers.<div class="popup-ref"><span class="ref__series"><i>Mol. Syst. Biol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fmsb.2012.24&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22806142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84864208657&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=594&journal=Mol.+Syst.+Biol.&author=Wang+I.-M.&author=Zhang+B.&author=Yang+X.&author=Zhu+J.&author=Stepaniants+S.&author=Zhang+C.&author=Meng+Q.&author=Peters+M.&author=He+Y.&author=Ni+C.&title=Systems+analysis+of+eleven+rodent+disease+models+reveals+an+inflammatome+signature+and+key+drivers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and immune-enriched networks (<div class="dropBlock reference-citations"><a href="#bib139" id="back-bib139" data-db-target-for="bib139-d10391928e2924" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib139">Peters et al., 2017</a><div data-db-target-of="bib139-d10391928e2924" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Peters L.A. </li><li class="loa__item">Perrigoue J. </li><li class="loa__item">Mortha A. </li><li class="loa__item">Iuga A. </li><li class="loa__item">Song W.-M. </li><li class="loa__item">Neiman E.M. </li><li class="loa__item">Llewellyn S.R. </li><li class="loa__item">Di Narzo A. </li><li class="loa__item">Kidd B.A. </li><li class="loa__item">Telesco S.E. </li><li>et al.</li></ul><div class="reference-citations__detials">A functional genomics predictive network model identifies regulators of inflammatory bowel disease.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">1437-1449</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3947&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28892060&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85030166718&amp;cf=" onclick="newWindow(this.href);return false">Scopus (69)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1437-1449&journal=Nat.+Genet.&author=Peters+L.A.&author=Perrigoue+J.&author=Mortha+A.&author=Iuga+A.&author=Song+W.-M.&author=Neiman+E.M.&author=Llewellyn+S.R.&author=Di+Narzo+A.&author=Kidd+B.A.&author=Telesco+S.E.&title=A+functional+genomics+predictive+network+model+identifies+regulators+of+inflammatory+bowel+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">We explored the immunological associations of the RTK CBPE scores described above (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>I and 2J) in LSCC tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>G). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>H) and consequently with immune-related pathways and cell types (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>G). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103 (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d10391928e2941" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Cammer et al., 2009</a><div data-db-target-of="bib19-d10391928e2941" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cammer M. </li><li class="loa__item">Gevrey J.-C. </li><li class="loa__item">Lorenz M. </li><li class="loa__item">Dovas A. </li><li class="loa__item">Condeelis J. </li><li class="loa__item">Cox D. </li></ul><div class="reference-citations__detials">The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">284</span></span>: <span class="ref__seriesPages">23302-23311</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19561083&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-69949155572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23302-23311&journal=J.%C2%A0Biol.+Chem.&author=Cammer+M.&author=Gevrey+J.-C.&author=Lorenz+M.&author=Dovas+A.&author=Condeelis+J.&author=Cox+D.&title=The+mechanism+of+CSF-1-induced+Wiskott-Aldrich+syndrome+protein+activation+in%C2%A0vivo%3A+a+role+for+phosphatidylinositol+3-kinase+and+Cdc42">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d10391928e2945" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Dovas et al., 2009</a><div data-db-target-of="bib30-d10391928e2945" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dovas A. </li><li class="loa__item">Gevrey J.-C. </li><li class="loa__item">Grossi A. </li><li class="loa__item">Park H. </li><li class="loa__item">Abou-Kheir W. </li><li class="loa__item">Cox D. </li></ul><div class="reference-citations__detials">Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages.<div class="popup-ref"><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">122</span></span>: <span class="ref__seriesPages">3873-3882</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.051755&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19808890&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70450236976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=3873-3882&journal=J.%C2%A0Cell+Sci.&author=Dovas+A.&author=Gevrey+J.-C.&author=Grossi+A.&author=Park+H.&author=Abou-Kheir+W.&author=Cox+D.&title=Regulation+of+podosome+dynamics+by+WASp+phosphorylation%3A+implication+in+matrix+degradation+and+chemotaxis+in+macrophages">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d10391928e2948" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Mouchemore et al., 2013</a><div data-db-target-of="bib124-d10391928e2948" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mouchemore K.A. </li><li class="loa__item">Sampaio N.G. </li><li class="loa__item">Murrey M.W. </li><li class="loa__item">Stanley E.R. </li><li class="loa__item">Lannutti B.J. </li><li class="loa__item">Pixley F.J. </li></ul><div class="reference-citations__detials">Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity.<div class="popup-ref"><span class="ref__series"><i>FEBS J.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">280</span></span>: <span class="ref__seriesPages">5228-5236</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Ffebs.12316&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23648053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84886101666&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=5228-5236&journal=FEBS+J.&author=Mouchemore+K.A.&author=Sampaio+N.G.&author=Murrey+M.W.&author=Stanley+E.R.&author=Lannutti+B.J.&author=Pixley+F.J.&title=Specific+inhibition+of+PI3K+p110%CE%B4+inhibits+CSF-1-induced+macrophage+spreading+and+invasive+capacity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and upregulation of AKAP13, among other immune-related proteins (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>I). Tumor-associated macrophages (<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d10391928e2954" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Cassetta and Pollard, 2018</a><div data-db-target-of="bib23-d10391928e2954" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cassetta L. </li><li class="loa__item">Pollard J.W. </li></ul><div class="reference-citations__detials">Targeting macrophages: therapeutic approaches in cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Drug Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">887-904</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnrd.2018.169&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30361552&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85058944664&amp;cf=" onclick="newWindow(this.href);return false">Scopus (362)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=887-904&journal=Nat.+Rev.+Drug+Discov.&author=Cassetta+L.&author=Pollard+J.W.&title=Targeting+macrophages%3A+therapeutic+approaches+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), CSF1R signaling (<div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d10391928e2957" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Cannarile et al., 2017</a><div data-db-target-of="bib21-d10391928e2957" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cannarile M.A. </li><li class="loa__item">Weisser M. </li><li class="loa__item">Jacob W. </li><li class="loa__item">Jegg A.-M. </li><li class="loa__item">Ries C.H. </li><li class="loa__item">Rüttinger D. </li></ul><div class="reference-citations__detials">Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy.<div class="popup-ref"><span class="ref__series"><i>J. Immunother. Cancer.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">53</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs40425-017-0257-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28716061&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85024385914&amp;cf=" onclick="newWindow(this.href);return false">Scopus (315)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=53&journal=J.%C2%A0Immunother.+Cancer&author=Cannarile+M.A.&author=Weisser+M.&author=Jacob+W.&author=Jegg+A.-M.&author=Ries+C.H.&author=R%C3%BCttinger+D.&title=Colony-stimulating+factor+1+receptor+%28CSF1R%29+inhibitors+in+cancer+therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and AKAP13 (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d10391928e2960" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Diviani et al., 2016</a><div data-db-target-of="bib29-d10391928e2960" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Diviani D. </li><li class="loa__item">Raimondi F. </li><li class="loa__item">Del Vescovo C.D. </li><li class="loa__item">Dreyer E. </li><li class="loa__item">Reggi E. </li><li class="loa__item">Osman H. </li><li class="loa__item">Ruggieri L. </li><li class="loa__item">Gonano C. </li><li class="loa__item">Cavin S. </li><li class="loa__item">Box C.L. </li><li>et al.</li></ul><div class="reference-citations__detials">Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling.<div class="popup-ref"><span class="ref__series"><i>Cell Chem. Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1135-1146</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27593112&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84988585445&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1135-1146&journal=Cell+Chem.+Biol.&author=Diviani+D.&author=Raimondi+F.&author=Del+Vescovo+C.D.&author=Dreyer+E.&author=Reggi+E.&author=Osman+H.&author=Ruggieri+L.&author=Gonano+C.&author=Cavin+S.&author=Box+C.L.&title=Small-Molecule+Protein-Protein+Interaction+Inhibitor+of+Oncogenic+Rho+Signaling">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) have each been proposed as anti-cancer targets.</div></section><section id="sec2.9" data-locator="sec2.9" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomic biomarker candidates for prognosis, diagnosis, and treatment</span></h3><div class="section-paragraph">Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>A; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/f54f2d41-2c47-48e5-8152-94e5558b1a23/mmc7" class="scroll-into-link">Table S7</a>). The top 50 of 502 differential proteins are shown in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>A. In addition to 206 upregulated oncogenic proteins (OncoKB, <a href="/cms/10.1016/j.cell.2021.07.016/attachment/f54f2d41-2c47-48e5-8152-94e5558b1a23/mmc7" class="scroll-into-link">Table S7</a>), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>B), with the I-S samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d10391928e2986" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Hammerman et al., 2012</a><div data-db-target-of="bib54-d10391928e2986" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hammerman P.S. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Voet D. </li><li class="loa__item">Jing R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Gabriel S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B; <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>C). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines (<a class="inter-ref-link" href="https://depmap.org/" target="_blank">https://depmap.org/</a>), suggesting critical roles in key cellular transformation and proliferation processes (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C; <a href="/cms/10.1016/j.cell.2021.07.016/attachment/f54f2d41-2c47-48e5-8152-94e5558b1a23/mmc7" class="scroll-into-link">Table S7</a>).<div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/d2c694a4-5902-4925-842c-a94feb810e60/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/55833d6c-4154-46a0-838c-052ce5edc9ed/gr7_lrg.jpg" data-large-image="/cms/attachment/d2c694a4-5902-4925-842c-a94feb810e60/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Proteomic features related to diagnosis, prognosis, or treatment</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Differentially expressed proteins between tumors and NATs.</div><div class="section-paragraph">(B) Significantly increased proteins (larger font indicates &gt; 4-fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC (referred to as LUSC) cohort mRNA.</div><div class="section-paragraph">(C) Genetic dependencies of 502 proteins (log2 FC &gt; 2, FDR &lt; 0.01 and NAs &lt; 50%) in LSCC cell lines (n = 16) profiled as part of the Achilles Dependency-Map project.</div><div class="section-paragraph">(D) Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted.</div><div class="section-paragraph">(E) EGFR protein and tyrosine phosphorylation levels compared to <em>EGFR</em> copy number and an EGFR activity score (PROGENy).</div><div class="section-paragraph">(F) Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. <sup>∗</sup>p &lt; 0.05</div><div class="section-paragraph">(G) GO biological process enrichment for proteins with increased phosphorylation in <em>EGFR</em> amplified samples compared to non-amplified samples.</div><div class="section-paragraph">(H) Summary roadmap figure partitioned into five major categories, indicated by different colors.</div><div class="section-paragraph">See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref2915" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a> and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2920" href="/cms/10.1016/j.cell.2021.07.016/attachment/f54f2d41-2c47-48e5-8152-94e5558b1a23/mmc7" target="_blank">Table S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/d2c694a4-5902-4925-842c-a94feb810e60/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/55833d6c-4154-46a0-838c-052ce5edc9ed/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/10355d5b-b6de-440d-b624-d932ae1cbe7c/figs7.jpg" alt="Figure thumbnail figs7" data-locator="figs7" data-high-res="/cms/attachment/e33347f2-8a47-4197-ad87-21c0944fdfa1/figs7_lrg.jpg" data-large-image="/cms/attachment/10355d5b-b6de-440d-b624-d932ae1cbe7c/figs7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S7" class="label figure__label">Figure S7</span><span class="figure__title__text">Differential tumor and NAT analysis, related to <a data-locator="fig7" aria-expanded="false" aria-controls="crosref3055" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig7">Figure 7</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Log2 fold change of proteins increased in tumors relative to paired NATs, together with candidate representation in the Human Proteome Atlas (HPA) and various annotations. Shown are the top 50 of 502 differential proteins with coherent expression in all omics that were significantly upregulated in ≥90% of T/NAT pairs (log2(FC)&gt;2, FDR &lt; 0.01). The blue box indicates the protein that was upregulated in 100% of T/NAT pairs.</div><div class="section-paragraph">(B) Fold changes of CT antigens and their relationship to NMF subtype and immune clusters.</div><div class="section-paragraph">(C) Tumor-specific proteins from the current study showing significant association with disease-free survival in the TCGA LSCC cohort.</div><div class="section-paragraph">(D) Focal amplification (red) and deletion (blue) events in the copy number data from three CPTAC tumor cohorts.</div><div class="section-paragraph">(E) To identify copy number drivers of the immune score, correlations between CNA, RNA, protein, and the immune score were assessed for all genes present in both CNA and RNA in a cohort. To be considered drivers, genes had to have positive pairwise correlations between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, positive correlations were also required between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH-adjusted p &lt; 0.01 were considered significant. The figure shows genes deleted in at least 25% of the cohorts that were significantly correlated with RNA and protein expression as well as ESTIMATE immune score. LSCC-specific genes are indicated in black, genes in both LSCC and HNSCC are in red, genes in both LSCC and LUAD are in blue, and genes shared by all three are in purple.</div><div class="section-paragraph">(F) Significantly increased activating sites on kinases in the CPTAC LSCC, LUAD (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e635" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e635" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) and the HNSCC (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d10391928e638" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Huang et al., 2021</a><div data-db-target-of="bib66-d10391928e638" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang C. </li><li class="loa__item">Chen L. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Eguez R.V. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Wen B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">361-379.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33417831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099708609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=361-379.e16&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+insights+into+the+biology+and+treatment+of+HPV-negative+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>) cohorts.</div><div class="section-paragraph">(G) EGFR log2 tumor/NAT protein fold change in three CPTAC cancer cohorts (LSCC, LUAD, HNSCC).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/10355d5b-b6de-440d-b624-d932ae1cbe7c/figs7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e33347f2-8a47-4197-ad87-21c0944fdfa1/figs7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(21)00857-6&amp;id=figs7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell type of origin (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>D). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>D). Immune-related deletions were less prominent in HNSCC and especially LUAD (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>E), except for chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>F), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not in LUAD (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>G), although LUAD had many activating <em>EGFR</em> mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High <em>EGFR</em> amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy scoring (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>E) (<div class="dropBlock reference-citations"><a href="#bib151" id="back-bib151" data-db-target-for="bib151-d10391928e3030" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib151">Schubert et al., 2018</a><div data-db-target-of="bib151-d10391928e3030" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Schubert M. </li><li class="loa__item">Klinger B. </li><li class="loa__item">Klünemann M. </li><li class="loa__item">Sieber A. </li><li class="loa__item">Uhlitz F. </li><li class="loa__item">Sauer S. </li><li class="loa__item">Garnett M.J. </li><li class="loa__item">Blüthgen N. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Perturbation-response genes reveal signaling footprints in cancer gene expression.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>F). Furthermore, <em>EGFR</em> amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>G). These findings and similar results recently reported in HNSCC (<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d10391928e3042" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Huang et al., 2021</a><div data-db-target-of="bib66-d10391928e3042" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang C. </li><li class="loa__item">Chen L. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Eguez R.V. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Wen B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">361-379.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33417831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099708609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=361-379.e16&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+insights+into+the+biology+and+treatment+of+HPV-negative+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>H graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications.</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">In this study, we provide detailed proteogenomic profiles of treatment-naive, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation, and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into basal-inclusive and EMT-E subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs.</div><div class="section-paragraph">Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified/overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation, and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d10391928e3055" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Haslam and Prasad, 2019</a><div data-db-target-of="bib57-d10391928e3055" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haslam A. </li><li class="loa__item">Prasad V. </li></ul><div class="reference-citations__detials">Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs.<div class="popup-ref"><span class="ref__series"><i>JAMA Netw. Open.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">e192535</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1001%2Fjamanetworkopen.2019.2535&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31050774&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85065557338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2019&pages=e192535&journal=JAMA+Netw.+Open&author=Haslam+A.&author=Prasad+V.&title=Estimation+of+the+Percentage+of+US+Patients+With+Cancer+Who+Are+Eligible+for+and+Respond+to+Checkpoint+Inhibitor+Immunotherapy+Drugs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d10391928e3058" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Herbst et al., 2018</a><div data-db-target-of="bib59-d10391928e3058" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Herbst R.S. </li><li class="loa__item">Morgensztern D. </li><li class="loa__item">Boshoff C. </li></ul><div class="reference-citations__detials">The biology and management of non-small cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">553</span></span>: <span class="ref__seriesPages">446-454</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature25183&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29364287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85041105043&amp;cf=" onclick="newWindow(this.href);return false">Scopus (891)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=553&publication_year=2018&pages=446-454&journal=Nature&author=Herbst+R.S.&author=Morgensztern+D.&author=Boshoff+C.&title=The+biology+and+management+of+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>); numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the downstream effects of <em>CDKN2A</em> mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types.</div><section id="sec3.1" data-locator="sec3.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Limitations of the study</span></h3><div class="section-paragraph">Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity (<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d10391928e3068" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Peng et al., 2020</a><div data-db-target-of="bib138-d10391928e3068" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Peng A. </li><li class="loa__item">Mao X. </li><li class="loa__item">Zhong J. </li><li class="loa__item">Fan S. </li><li class="loa__item">Hu Y. </li></ul><div class="reference-citations__detials">Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">e1900271</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fpmic.201900271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32223079&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083998470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=e1900271&journal=Proteomics&author=Peng+A.&author=Mao+X.&author=Zhong+J.&author=Fan+S.&author=Hu+Y.&title=Single-Cell+Multi-Omics+and+Its+Prospective+Application+in+Cancer+Biology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib150" id="back-bib150" data-db-target-for="bib150-d10391928e3071" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib150">Satpathy et al., 2020</a><div data-db-target-of="bib150-d10391928e3071" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Satpathy S. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Krug K. </li><li class="loa__item">Kim B.-J. </li><li class="loa__item">Saltzman A.B. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Holloway K.R. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Huang C. </li><li class="loa__item">Singh P. </li><li>et al.</li></ul><div class="reference-citations__detials">Microscaled proteogenomic methods for precision oncology.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), such as pairing microdissection with mass spectrometry (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d10391928e3074" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Le Large et al., 2020</a><div data-db-target-of="bib96-d10391928e3074" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Le Large T.Y. </li><li class="loa__item">Mantini G. </li><li class="loa__item">Meijer L.L. </li><li class="loa__item">Pham T.V. </li><li class="loa__item">Funel N. </li><li class="loa__item">van Grieken N.C. </li><li class="loa__item">Kok B. </li><li class="loa__item">Knol J. </li><li class="loa__item">van Laarhoven H.W. </li><li class="loa__item">Piersma S.R. </li><li>et al.</li></ul><div class="reference-citations__detials">Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets.<div class="popup-ref"><span class="ref__series"><i>JCI Insight.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e138290</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1172%2Fjci.insight.138290&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32634123&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85089205740&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=e138290&journal=JCI+Insight&author=Le+Large+T.Y.&author=Mantini+G.&author=Meijer+L.L.&author=Pham+T.V.&author=Funel+N.&author=van+Grieken+N.C.&author=Kok+B.&author=Knol+J.&author=van+Laarhoven+H.W.&author=Piersma+S.R.&title=Microdissected+pancreatic+cancer+proteomes+reveal+tumor+heterogeneity+and+therapeutic+targets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) or using mass cytometry (<div class="dropBlock reference-citations"><a href="#bib162" id="back-bib162" data-db-target-for="bib162-d10391928e3077" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib162">Spitzer and Nolan, 2016</a><div data-db-target-of="bib162-d10391928e3077" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Spitzer M.H. </li><li class="loa__item">Nolan G.P. </li></ul><div class="reference-citations__detials">Mass Cytometry: Single Cells, Many Features.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">780-791</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27153492&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=780-791&journal=Cell&author=Spitzer+M.H.&author=Nolan+G.P.&title=Mass+Cytometry%3A+Single+Cells%2C+Many+Features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities and advance the understanding and treatment of LSCC.</div></section></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0100" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec4.1" data-locator="sec4.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key resources table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Antibodies</strong></td></tr><tr><td>PTMScan® Ubiquitin Remnant Motif (K-e-GG) Kit</td><td>Cell Signaling Technology</td><td>Catalog 5562</td></tr><tr><td>PTMScan Acetyl-lysine Kit</td><td>Cell Signaling Technology</td><td>Catalog: 13416</td></tr><tr><td>CD4</td><td>Dako</td><td>Clone 4B12, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_2728838" target="_blank">AB_2728838</a></td></tr><tr><td>CD8</td><td>Bio-Rad</td><td>Clone 4B11, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_322868" target="_blank">AB_322868</a></td></tr><tr><td>CD163</td><td>Abcam</td><td>Ab182422, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_2753196" target="_blank">AB_2753196</a></td></tr><tr><td>ARHGDIB</td><td>Sigma-Aldrich</td><td>HPA051235, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_2681398" target="_blank">AB_2681398</a></td></tr><tr><td>CK (pan-Cytokeratin)</td><td>Dako</td><td>clone AE1/AE3, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_2132885" target="_blank">AB_2132885</a></td></tr><tr><td class="rowsep">alpha-SMA</td><td class="rowsep">Abcam</td><td class="rowsep">ab5694, RRID:<a class="inter-ref-link" href="http://antibodyregistry.org/AB_2223021" target="_blank">AB_2223021</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Biological samples</strong></td></tr><tr><td class="rowsep">Primary tumor samples</td><td class="rowsep">See <a href="#sec4.3" class="scroll-into-link">experimental model and subject details</a></td><td class="rowsep">N/A</td></tr><tr><td class="rowsep" colspan="3"><strong>Chemicals and Reagents</strong></td></tr><tr><td>HPLC-grade water</td><td>J.T. Baker</td><td>Catalog: 4218-03</td></tr><tr><td>Urea</td><td>Sigma</td><td>Catalog: U0631</td></tr><tr><td>Sodium chloride</td><td>Sigma</td><td>Catalog: 71376</td></tr><tr><td>1M Tris, pH 8.0</td><td>Invitrogen</td><td>Catalog: AM9855G</td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog: 78830</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>Phosphatase inhibitor cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase inhibitor cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Dithiothretiol, No-Weigh Format</td><td>ThermoScientific</td><td>Catalog: 20291</td></tr><tr><td>Iodoacetamide</td><td>Sigma</td><td>Catalog: A3221</td></tr><tr><td>Lysyl endopeptidase</td><td>Wako Chemicals</td><td>Catalog: 129-02541</td></tr><tr><td>Sequencing-grade modified trypsin</td><td>Promega</td><td>Catalog: V511X</td></tr><tr><td>Formic acid</td><td>Sigma</td><td>Catalog: F0507</td></tr><tr><td>Acetonitrile, LC-MS grade</td><td>Honeywell</td><td>Catalog: 34967</td></tr><tr><td>Acetonitrile, anhydrous</td><td>Sigma</td><td>Catalog: 271004</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>Catalog: 302031</td></tr><tr><td>Tandem Mass Tag reagent kit – 11plex</td><td>ThermoFisher</td><td>Catalog: A34808</td></tr><tr><td>0.5M HEPES, pH 8.5</td><td>Alfa Aesar</td><td>Catalog: J63218</td></tr><tr><td>Hydroxylamine solution, 50% (vol/vol) in H<sub>2</sub>O</td><td>Aldrich</td><td>Catalog: 467804</td></tr><tr><td>Methanol</td><td>Honeywell</td><td>Catalog: 34966</td></tr><tr><td>Ammonium hydroxide solution, 28% (wt/vol) in H<sub>2</sub>O</td><td>Sigma</td><td>Catalog: 338818</td></tr><tr><td>Ni-NTA agarose beads</td><td>QIAGEN</td><td>Catalog: 30410</td></tr><tr><td>Iron (III) chloride</td><td>Sigma</td><td>Catalog: 451649</td></tr><tr><td>Acetic acid, glacial</td><td>Sigma</td><td>Catalog: AX0073</td></tr><tr><td>Potassium phosphate, monobasic</td><td>Sigma</td><td>Catalog: P0662</td></tr><tr><td>Potassium phosphate, dibasic</td><td>Sigma</td><td>Catalog: P3786</td></tr><tr><td>MOPS</td><td>Sigma</td><td>Catalog: M5162</td></tr><tr><td>Sodium hydroxide</td><td>VWR</td><td>Catalog: BDH7225</td></tr><tr><td>Sodium phosphate, dibasic</td><td>Sigma</td><td>Catalog: S9763</td></tr><tr><td>Phosphate-buffered saline</td><td>Fisher Scientific</td><td>Catalog: 10010023</td></tr><tr><td class="rowsep">iVIEW DAB Detection Kit</td><td class="rowsep">Roche</td><td class="rowsep">Catalog: 760-091</td></tr><tr><td class="rowsep" colspan="3"><strong>Equipment</strong></td></tr><tr><td>Reversed-phase tC18 SepPak, 3cc 200mg</td><td>Waters</td><td>Catalog: WAT054925</td></tr><tr><td>Solid-phase C18 disk, for Stage-tips</td><td>Empore</td><td>Catalog: 66883-U</td></tr><tr><td>Stage-tip needle</td><td>Cadence</td><td>Catalog: 7928</td></tr><tr><td>Stage-tip puncher, PEEK tubing</td><td>Idex Health &amp; Science</td><td>Catalog: 1581</td></tr><tr><td>PicoFrit LC-MS column</td><td>New Objective</td><td>Catalog: PF360-75-10-N-5</td></tr><tr><td>ReproSil-Pur, 120 Å, C18-AQ, 1.9-μm resin</td><td>Dr. Maisch</td><td>Catalog: r119.aq</td></tr><tr><td>Nanospray column heater</td><td>Phoenix S&amp;T</td><td>Catalog: PST-CH-20U</td></tr><tr><td>Column heater controller</td><td>Phoenix S&amp;T</td><td>Catalog: PST-CHC</td></tr><tr><td>300 μL LC-MS autosampler vial and cap</td><td>Waters</td><td>Catalog: 186002639</td></tr><tr><td>Offline HPLC column, 3.5-μm particle size, 4.6 um × 250 mm</td><td>Agilent</td><td>Catalog: Custom order</td></tr><tr><td>Offline 96-well fractionation plate</td><td>Whatman</td><td>Catalog: 77015200</td></tr><tr><td>700 μL bRP fractionation autosampler vial</td><td>ThermoFisher</td><td>Catalog: C4010-14</td></tr><tr><td>700 μL bRP fractionation autosampler cap</td><td>ThermoFisher</td><td>Catalog: C4010-55A</td></tr><tr><td>96-well microplate for BCA</td><td>Greiner</td><td>Catalog: 655101</td></tr><tr><td>Microplate foil cover</td><td>Corning</td><td>Catalog: PCR-AS-200</td></tr><tr><td>Vacuum centrifuge</td><td>ThermoFisher</td><td>Catalog: SPD121P-115</td></tr><tr><td>Centrifuge</td><td>Eppendorf</td><td>Catalog: 5427 R</td></tr><tr><td>Benchtop mini centrifuge</td><td>Corning</td><td>Catalog: 6765</td></tr><tr><td>Benchtop vortex</td><td>Scientific Industries</td><td>Catalog: SI-0236</td></tr><tr><td>Incubating shaker</td><td>VWR</td><td>Catalog: 12620-942</td></tr><tr><td>15 mL centrifuge tube</td><td>Corning</td><td>Catalog: 352097</td></tr><tr><td>50 mL centrifuge tube</td><td>Corning</td><td>Catalog: 352070</td></tr><tr><td>1.5 mL microtube w/o cap</td><td>Sarstedt</td><td>Catalog: 72.607</td></tr><tr><td>2.0 mL microtube w/o cap</td><td>Sarstedt</td><td>Catalog: 72.608</td></tr><tr><td>Microtube caps</td><td>Sarstedt</td><td>Catalog: 72.692</td></tr><tr><td>1.5 mL snapcap tube</td><td>ThermoFisher</td><td>Catalog: AM12450</td></tr><tr><td class="rowsep">2.0 mL snapcap tube</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: AM12475</td></tr><tr><td class="rowsep" colspan="3"><strong>Instrumentation</strong></td></tr><tr><td>Microplate Reader</td><td>Molecular Devices</td><td>Catalog: M2</td></tr><tr><td>Offline HPLC System for bRP fractionation</td><td>Agilent 1260</td><td>Catalog: G1380-90000</td></tr><tr><td>Online LC for LC-MS</td><td>ThermoFisher</td><td>Catalog: LC140</td></tr><tr><td>Q Exactive Plus Mass Spectrometer</td><td>ThermoFisher</td><td>Catalog: IQLAAEGAAPFALGMBDK</td></tr><tr><td>Q Exactive HF-X Mass Spectrometer</td><td>ThermoFisher</td><td>Catalog: 0726042</td></tr><tr><td class="rowsep">Orbitrap Fusion Lumos Tribrid Mass Spectrometer</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: IQLAAEGAAPFADBMBHQ</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical commercial assays</strong></td></tr><tr><td>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold</td><td>Illumina</td><td>Catalog: RS-122-2301</td></tr><tr><td>Infinium MethylationEPIC Kit</td><td>Illumina</td><td>Catalog: WG-317-1003</td></tr><tr><td>Nextera DNA Exosome Kit</td><td>Illumina</td><td>Catalog: 20020617</td></tr><tr><td>KAPA Hyper Prep Kit, PCR-free</td><td>Roche</td><td>Catalog: 07962371001</td></tr><tr><td class="rowsep">BCA Protein Assay Kit</td><td class="rowsep">ThermoFisher</td><td class="rowsep">Catalog: 23225</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited data</strong></td></tr><tr><td>DepMap: Mutation</td><td>DepMapPublic 21Q1 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/31068700" target="_blank">31068700</a>; Dataset <a class="inter-ref-link" href="https://doi.org/10.6084/m9.figshare.13681534.v1" target="_blank">https://doi.org/10.6084/m9.figshare.13681534.v1</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: Segmented copy number</td><td>DepMapPublic 21Q1 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/31068700" target="_blank">31068700</a>; Dataset <a class="inter-ref-link" href="https://doi.org/10.6084/m9.figshare.13681534.v1" target="_blank">https://doi.org/10.6084/m9.figshare.13681534.v1</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: Gene level copy number</td><td>DepMapPublic 21Q1 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/31068700" target="_blank">31068700</a>; Dataset <a class="inter-ref-link" href="https://doi.org/10.6084/m9.figshare.13681534.v1" target="_blank">https://doi.org/10.6084/m9.figshare.13681534.v1</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: RNaseq (transcript isoform)</td><td>DepMapPublic 21Q1 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/31068700" target="_blank">31068700</a>; Dataset <a class="inter-ref-link" href="https://doi.org/10.6084/m9.figshare.13681534.v1" target="_blank">https://doi.org/10.6084/m9.figshare.13681534.v1</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: Proteomics (RPPA)</td><td>CCLE 2019 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/31068700" target="_blank">31068700</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: CRISPR KO screen (combined)</td><td>DepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: <a class="inter-ref-link" href="https://doi.org/10.1101/2020.05.22.110247" target="_blank">https://doi.org/10.1101/2020.05.22.110247</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: shRNA screen (combined)</td><td>DEMETER2 Data v6 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/30389920" target="_blank">30389920</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: GDSC drug screen</td><td>Sanger GDSC 1 (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/27397505" target="_blank">27397505</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>DepMap: PRISM drug screen</td><td>PRISM Repurposing 19Q4 Secondary Screen (PMID: <a class="inter-ref-link" href="https://pubmed.ncbi.nlm.nih.gov/32613204" target="_blank">32613204</a>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a></td></tr><tr><td>PhosphoSitePlus</td><td>(<div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d10391928e3834" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Hornbeck et al., 2012</a><div data-db-target-of="bib63-d10391928e3834" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Tkachev S. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Skrzypek E. </li><li class="loa__item">Murray B. </li><li class="loa__item">Latham V. </li><li class="loa__item">Sullivan M. </li></ul><div class="reference-citations__detials">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">40</span></span>: <span class="ref__seriesPages">D261-D270</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkr1122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22135298&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84857047339&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1019)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D261-D270&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Kornhauser+J.M.&author=Tkachev+S.&author=Zhang+B.&author=Skrzypek+E.&author=Murray+B.&author=Latham+V.&author=Sullivan+M.&title=PhosphoSitePlus%3A+a+comprehensive+resource+for+investigating+the+structure+and+function+of+experimentally+determined+post-translational+modifications+in+man+and+mouse">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.phosphosite.org" target="_blank">https://www.phosphosite.org</a></td></tr><tr><td>Connectivity Map (CMAP)</td><td>(<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d10391928e3845" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Lamb et al., 2006</a><div data-db-target-of="bib92-d10391928e3845" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2951)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e3848" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e3848" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.broadinstitute.org/connectivity-map-cmap" target="_blank">https://www.broadinstitute.org/connectivity-map-cmap</a></td></tr><tr><td>Human Protein Atlas (HPA)</td><td>(<div class="dropBlock reference-citations"><a href="#bib178" id="back-bib178" data-db-target-for="bib178-d10391928e3860" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib178">Uhlén et al., 2005</a><div data-db-target-of="bib178-d10391928e3860" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Uhlén M. </li><li class="loa__item">Björling E. </li><li class="loa__item">Agaton C. </li><li class="loa__item">Szigyarto C.A.-K. </li><li class="loa__item">Amini B. </li><li class="loa__item">Andersen E. </li><li class="loa__item">Andersson A.-C. </li><li class="loa__item">Angelidou P. </li><li class="loa__item">Asplund A. </li><li class="loa__item">Asplund C. </li><li>et al.</li></ul><div class="reference-citations__detials">A human protein atlas for normal and cancer tissues based on antibody proteomics.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1920-1932</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16127175&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-29244477844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (875)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1920-1932&journal=Mol.+Cell.+Proteomics&author=Uhl%C3%A9n+M.&author=Bj%C3%B6rling+E.&author=Agaton+C.&author=Szigyarto+C.A.-K.&author=Amini+B.&author=Andersen+E.&author=Andersson+A.-C.&author=Angelidou+P.&author=Asplund+A.&author=Asplund+C.&title=A+human+protein+atlas+for+normal+and+cancer+tissues+based+on+antibody+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.proteinatlas.org" target="_blank">https://www.proteinatlas.org</a></td></tr><tr><td>CT Antigen database</td><td>(<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d10391928e3871" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Almeida et al., 2009</a><div data-db-target-of="bib5-d10391928e3871" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.-T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://www.cta.lncc.br" target="_blank">http://www.cta.lncc.br</a></td></tr><tr><td>Dependency map (DepMap)</td><td>(<div class="dropBlock reference-citations"><a href="#bib176" id="back-bib176" data-db-target-for="bib176-d10391928e3882" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib176">Tsherniak et al., 2017</a><div data-db-target-of="bib176-d10391928e3882" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tsherniak A. </li><li class="loa__item">Vazquez F. </li><li class="loa__item">Montgomery P.G. </li><li class="loa__item">Weir B.A. </li><li class="loa__item">Kryukov G. </li><li class="loa__item">Cowley G.S. </li><li class="loa__item">Gill S. </li><li class="loa__item">Harrington W.F. </li><li class="loa__item">Pantel S. </li><li class="loa__item">Krill-Burger J.M. </li><li>et al.</li></ul><div class="reference-citations__detials">Defining a Cancer Dependency Map.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">170</span></span>: <span class="ref__seriesPages">564-576.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28753430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85026422523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (541)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=564-576.e16&journal=Cell&author=Tsherniak+A.&author=Vazquez+F.&author=Montgomery+P.G.&author=Weir+B.A.&author=Kryukov+G.&author=Cowley+G.S.&author=Gill+S.&author=Harrington+W.F.&author=Pantel+S.&author=Krill-Burger+J.M.&title=Defining+a+Cancer+Dependency+Map">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://depmap.org/portal/" target="_blank">https://depmap.org/portal/</a><br></br>v3.3.8 is a GECKOv2 Achilles dataset</td></tr><tr><td>Library of Integrated Network-based Cellular Signatures (LINCS)</td><td>(<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d10391928e3895" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Lamb et al., 2006</a><div data-db-target-of="bib92-d10391928e3895" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2951)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e3898" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e3898" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://clue.io/data" target="_blank">https://clue.io/data</a><br></br>Expanded CMap LINCS Resource 2020 (1/28/2021 update)</td></tr><tr><td>CPTAC HNSCC cohort</td><td>(<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d10391928e3911" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Huang et al., 2021</a><div data-db-target-of="bib66-d10391928e3911" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang C. </li><li class="loa__item">Chen L. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Eguez R.V. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Wen B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">361-379.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33417831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099708609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=361-379.e16&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+insights+into+the+biology+and+treatment+of+HPV-negative+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S054" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S054</a></td></tr><tr><td class="rowsep">CPTAC LSCC cohort</td><td class="rowsep">This study</td><td class="rowsep"><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S063" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S063</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>methylationArrayAnalysis (version 3.9)</td><td>(<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d10391928e3933" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Maksimovic et al., 2016</a><div data-db-target-of="bib113-d10391928e3933" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maksimovic J. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Oshlack A. </li></ul><div class="reference-citations__detials">A cross-package Bioconductor workflow for analysing methylation array data.<div class="popup-ref"><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1281</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.12688%2Ff1000research.8839.1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27347385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1281&journal=F1000Res.&author=Maksimovic+J.&author=Phipson+B.&author=Oshlack+A.&title=A+cross-package+Bioconductor+workflow+for+analysing+methylation+array+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html" target="_blank">https://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html</a></td></tr><tr><td>Illumina EPIC methylation array (3.9)</td><td>(<div class="dropBlock reference-citations"><a href="#bib215" id="back-bib215" data-db-target-for="bib215-d10391928e3944" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib215">Hansen, 2017</a><div data-db-target-of="bib215-d10391928e3944" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hansen K.D. </li></ul><div class="reference-citations__detials">IlluminaHumanMethylationEPICanno.ilm10b2.hg19 (Bioconductor).<div class="popup-ref"><span class="refDate">2017</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Hansen+K.D.&title=IlluminaHumanMethylationEPICanno.ilm10b2.hg19+%28Bioconductor%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html" target="_blank">https://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html</a></td></tr><tr><td>Methylation array analysis pipeline for CPTAC</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/cptac_methylation" target="_blank">https://github.com/ding-lab/cptac_methylation</a></td></tr><tr><td>miRNA-Seq analysis pipeline for CPTAC</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/CPTAC_miRNA" target="_blank">https://github.com/ding-lab/CPTAC_miRNA</a></td></tr><tr><td>VEP</td><td>(<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d10391928e3972" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">McLaren et al., 2016</a><div data-db-target-of="bib117-d10391928e3972" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McLaren W. </li><li class="loa__item">Gil L. </li><li class="loa__item">Hunt S.E. </li><li class="loa__item">Riat H.S. </li><li class="loa__item">Ritchie G.R.S. </li><li class="loa__item">Thormann A. </li><li class="loa__item">Flicek P. </li><li class="loa__item">Cunningham F. </li></ul><div class="reference-citations__detials">The Ensembl Variant Effect Predictor.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1715)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.S.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/Ensembl/ensembl-vep/tags" target="_blank">https://github.com/Ensembl/ensembl-vep/tags</a></td></tr><tr><td>TNScope / DNAScope (Sentieon)</td><td>(<div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d10391928e3983" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">Freed et al., 2018</a><div data-db-target-of="bib41-d10391928e3983" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Freed D. </li><li class="loa__item">Pan R. </li><li class="loa__item">Aldana R. </li></ul><div class="reference-citations__detials">TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/250647">https://doi.org/10.1101/250647</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F250647&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85095625543&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&journal=bioRxiv&author=Freed+D.&author=Pan+R.&author=Aldana+R.&title=TNscope%3A+Accurate+Detection+of+Somatic+Mutations+with+Haplotype-based+Variant+Candidate+Detection+and+Machine+Learning+Filtering">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://sentieon.com" target="_blank">sentieon.com</a></td></tr><tr><td>vcfAnno</td><td>(<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d10391928e3994" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Pedersen et al., 2016</a><div data-db-target-of="bib137-d10391928e3994" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pedersen B.S. </li><li class="loa__item">Layer R.M. </li><li class="loa__item">Quinlan A.R. </li></ul><div class="reference-citations__detials">Vcfanno: fast, flexible annotation of genetic variants.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-016-0973-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27250555&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84975311524&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=118&journal=Genome+Biol.&author=Pedersen+B.S.&author=Layer+R.M.&author=Quinlan+A.R.&title=Vcfanno%3A+fast%2C+flexible+annotation+of+genetic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/brentp/vcfanno" target="_blank">https://github.com/brentp/vcfanno</a></td></tr><tr><td>VariantAnnotation (Bioconductor)</td><td>(<div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d10391928e4005" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Obenchain et al., 2014</a><div data-db-target-of="bib130-d10391928e4005" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Obenchain V. </li><li class="loa__item">Lawrence M. </li><li class="loa__item">Carey V. </li><li class="loa__item">Gogarten S. </li><li class="loa__item">Shannon P. </li><li class="loa__item">Morgan M. </li></ul><div class="reference-citations__detials">VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">2076-2078</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtu168&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24681907&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84901835522&amp;cf=" onclick="newWindow(this.href);return false">Scopus (128)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=2076-2078&journal=Bioinformatics&author=Obenchain+V.&author=Lawrence+M.&author=Carey+V.&author=Gogarten+S.&author=Shannon+P.&author=Morgan+M.&title=VariantAnnotation%3A+a+Bioconductor+package+for+exploration+and+annotation+of+genetic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html</a></td></tr><tr><td>arriba_v1.1.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib221" id="back-bib221" data-db-target-for="bib221-d10391928e4016" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib221">Uhrig et al., 2021</a><div data-db-target-of="bib221-d10391928e4016" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Uhrig S. </li><li class="loa__item">Ellermann J. </li><li class="loa__item">Walther T. </li><li class="loa__item">Burkhardt P. </li><li class="loa__item">Fröhlich M. </li><li class="loa__item">Hutter B. </li><li class="loa__item">Toprak U.H. </li><li class="loa__item">Neumann O. </li><li class="loa__item">Stenzinger A. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Accurate and efficient detection of gene fusions from RNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">448-460</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.257246.119&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33441414&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2021&pages=448-460&journal=Genome+Res&author=Uhrig+S.&author=Ellermann+J.&author=Walther+T.&author=Burkhardt+P.&author=Fr%C3%B6hlich+M.&author=Hutter+B.&author=Toprak+U.H.&author=Neumann+O.&author=Stenzinger+A.&author=Scholl+C.&title=Accurate+and+efficient+detection+of+gene+fusions+from+RNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/suhrig/arriba/" target="_blank">https://github.com/suhrig/arriba/</a></td></tr><tr><td>fusioncatcher_v1.10</td><td>(<div class="dropBlock reference-citations"><a href="#bib127" id="back-bib127" data-db-target-for="bib127-d10391928e4027" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib127">Nicorici et al., 2014</a><div data-db-target-of="bib127-d10391928e4027" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nicorici D. </li><li class="loa__item">Satalan M. </li><li class="loa__item">Edgren H. </li><li class="loa__item">Kangaspeska S. </li><li class="loa__item">Murumagi A. </li><li class="loa__item">Kallioniemi O. </li><li class="loa__item">Virtanen S. </li><li class="loa__item">Kilkku O. </li></ul><div class="reference-citations__detials">FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/011650">https://doi.org/10.1101/011650</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F011650&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&journal=bioRxiv&author=Nicorici+D.&author=Satalan+M.&author=Edgren+H.&author=Kangaspeska+S.&author=Murumagi+A.&author=Kallioniemi+O.&author=Virtanen+S.&author=Kilkku+O.&title=FusionCatcher+-+a+tool+for+finding+somatic+fusion+genes+in+paired-end+RNA-sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md" target="_blank">https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md</a></td></tr><tr><td>eQTLGen</td><td>(<div class="dropBlock reference-citations"><a href="#bib192" id="back-bib192" data-db-target-for="bib192-d10391928e4038" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib192">Westra et al., 2013</a><div data-db-target-of="bib192-d10391928e4038" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Westra H.-J. </li><li class="loa__item">Peters M.J. </li><li class="loa__item">Esko T. </li><li class="loa__item">Yaghootkar H. </li><li class="loa__item">Schurmann C. </li><li class="loa__item">Kettunen J. </li><li class="loa__item">Christiansen M.W. </li><li class="loa__item">Fairfax B.P. </li><li class="loa__item">Schramm K. </li><li class="loa__item">Powell J.E. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic identification of trans eQTLs as putative drivers of known disease associations.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">1238-1243</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.2756&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24013639&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84885018609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1038)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1238-1243&journal=Nat.+Genet.&author=Westra+H.-J.&author=Peters+M.J.&author=Esko+T.&author=Yaghootkar+H.&author=Schurmann+C.&author=Kettunen+J.&author=Christiansen+M.W.&author=Fairfax+B.P.&author=Schramm+K.&author=Powell+J.E.&title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)" target="_blank">https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen)</a></td></tr><tr><td>Pindel0.2.5</td><td>(<div class="dropBlock reference-citations"><a href="#bib202" id="back-bib202" data-db-target-for="bib202-d10391928e4049" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib202">Ye et al., 2009</a><div data-db-target-of="bib202-d10391928e4049" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1209)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://gmt.genome.wustl.edu/packages/pindel/" target="_blank">http://gmt.genome.wustl.edu/packages/pindel/</a></td></tr><tr><td>SignatureAnalyzer</td><td>(<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d10391928e4060" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">Kim et al., 2016</a><div data-db-target-of="bib82-d10391928e4060" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim J. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Polak P. </li><li class="loa__item">Braunstein L.Z. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Kwiatkowski D.J. </li><li class="loa__item">Rosenberg J.E. </li><li class="loa__item">Van Allen E.M. </li><li class="loa__item">D’Andrea A. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (171)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%E2%80%99Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/cancer/cga/msp" target="_blank">https://software.broadinstitute.org/cancer/cga/msp</a></td></tr><tr><td>CNVEX</td><td>Marcin Cieslik Lab</td><td><a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a></td></tr><tr><td>CRISP</td><td>Marcin Cieslik Lab</td><td><a class="inter-ref-link" href="https://github.com/mcieslik-mctp/crisp-build" target="_blank">https://github.com/mcieslik-mctp/crisp-build</a></td></tr><tr><td>Spectrum Mill</td><td>Karl R. Clauser, Broad Institute Proteomics Platform</td><td><a class="inter-ref-link" href="https://proteomics.broadinstitute.org/" target="_blank">https://proteomics.broadinstitute.org/</a></td></tr><tr><td>ComBat (v3.20.0)</td><td>(<div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d10391928e4096" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Johnson et al., 2007</a><div data-db-target-of="bib72-d10391928e4096" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Johnson W.E. </li><li class="loa__item">Li C. </li><li class="loa__item">Rabinovic A. </li></ul><div class="reference-citations__detials">Adjusting batch effects in microarray expression data using empirical Bayes methods.<div class="popup-ref"><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">118-127</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbiostatistics%2Fkxj037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16632515&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33845432928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3072)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=118-127&journal=Biostatistics&author=Johnson+W.E.&author=Li+C.&author=Rabinovic+A.&title=Adjusting+batch+effects+in+microarray+expression+data+using+empirical+Bayes+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/sva.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/sva.html</a></td></tr><tr><td>gPCA</td><td>(<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d10391928e4107" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Reese et al., 2013</a><div data-db-target-of="bib145-d10391928e4107" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reese S.E. </li><li class="loa__item">Archer K.J. </li><li class="loa__item">Therneau T.M. </li><li class="loa__item">Atkinson E.J. </li><li class="loa__item">Vachon C.M. </li><li class="loa__item">de Andrade M. </li><li class="loa__item">Kocher J.-P.A. </li><li class="loa__item">Eckel-Passow J.E. </li></ul><div class="reference-citations__detials">A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2877-2883</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtt480&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23958724&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84890087087&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2877-2883&journal=Bioinformatics&author=Reese+S.E.&author=Archer+K.J.&author=Therneau+T.M.&author=Atkinson+E.J.&author=Vachon+C.M.&author=de+Andrade+M.&author=Kocher+J.-P.A.&author=Eckel-Passow+J.E.&title=A+new+statistic+for+identifying+batch+effects+in+high-throughput+genomic+data+that+uses+guided+principal+component+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/gPCA/index.html" target="_blank">https://cran.r-project.org/web/packages/gPCA/index.html</a></td></tr><tr><td>GISTIC2.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d10391928e4118" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Mermel et al., 2011</a><div data-db-target-of="bib120-d10391928e4118" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1222)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz" target="_blank">ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz</a></td></tr><tr><td>iProFun</td><td>(<div class="dropBlock reference-citations"><a href="#bib161" id="back-bib161" data-db-target-for="bib161-d10391928e4129" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib161">Song et al., 2019</a><div data-db-target-of="bib161-d10391928e4129" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+Impact+of+DNA+Copy+Number+Alteration+and+Methylation+on+the+Proteogenomic+Landscape+of+Human+Ovarian+Cancer+via+a+Multi-omics+Integrative+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/iProFun" target="_blank">https://github.com/WangLab-MSSM/iProFun</a></td></tr><tr><td>ESTIMATE</td><td>(<div class="dropBlock reference-citations"><a href="#bib204" id="back-bib204" data-db-target-for="bib204-d10391928e4140" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib204">Yoshihara et al., 2013</a><div data-db-target-of="bib204-d10391928e4140" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1382)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioinformatics.mdanderson.org/public-software/estimate/" target="_blank">https://bioinformatics.mdanderson.org/public-software/estimate/</a></td></tr><tr><td>WebGestaltR</td><td>(<div class="dropBlock reference-citations"><a href="#bib187" id="back-bib187" data-db-target-for="bib187-d10391928e4151" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib187">Wang et al., 2017</a><div data-db-target-of="bib187-d10391928e4151" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Greer M. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (470)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt+2017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://www.webgestalt.org/" target="_blank">http://www.webgestalt.org/</a></td></tr><tr><td>GSVA</td><td>(<div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d10391928e4162" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Hänzelmann et al., 2013</a><div data-db-target-of="bib55-d10391928e4162" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1428)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/GSVA.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/GSVA.html</a></td></tr><tr><td>TSNet</td><td>(<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d10391928e4173" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Petralia et al., 2018</a><div data-db-target-of="bib140-d10391928e4173" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng J. </li><li class="loa__item">Yan A. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/WangLab-MSSM/TSNet" target="_blank">https://github.com/WangLab-MSSM/TSNet</a></td></tr><tr><td>xCell</td><td>(<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d10391928e4184" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Aran et al., 2017</a><div data-db-target-of="bib6-d10391928e4184" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://xcell.ucsf.edu/" target="_blank">https://xcell.ucsf.edu/</a></td></tr><tr><td>CPTAC LSCC Data Viewer</td><td>Broad Institute Proteomics Platform</td><td><a class="inter-ref-link" href="https://proteomics.broadapps.org/CPTAC-LSCC2021" target="_blank">https://proteomics.broadapps.org/CPTAC-LSCC2021</a></td></tr><tr><td>ConsensusClusterPlus</td><td>(<div class="dropBlock reference-citations"><a href="#bib195" id="back-bib195" data-db-target-for="bib195-d10391928e4204" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib195">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib195-d10391928e4204" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (710)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html</a></td></tr><tr><td>MS-GF+</td><td>(<div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d10391928e4215" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">Kim and Pevzner, 2014</a><div data-db-target-of="bib81-d10391928e4215" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (502)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/MSGFPlus/msgfplus" target="_blank">https://github.com/MSGFPlus/msgfplus</a></td></tr><tr><td>NeoFlow</td><td>(<div class="dropBlock reference-citations"><a href="#bib191" id="back-bib191" data-db-target-for="bib191-d10391928e4226" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib191">Wen et al., 2020</a><div data-db-target-of="bib191-d10391928e4226" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a></td></tr><tr><td>netMHCpan</td><td>(<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d10391928e4237" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Jurtz et al., 2017</a><div data-db-target-of="bib73-d10391928e4237" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (413)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://www.cbs.dtu.dk/services/NetMHCpan/" target="_blank">http://www.cbs.dtu.dk/services/NetMHCpan/</a></td></tr><tr><td>Optitype</td><td>(<div class="dropBlock reference-citations"><a href="#bib167" id="back-bib167" data-db-target-for="bib167-d10391928e4248" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib167">Szolek et al., 2014</a><div data-db-target-of="bib167-d10391928e4248" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/FRED-2/OptiType" target="_blank">https://github.com/FRED-2/OptiType</a></td></tr><tr><td>Customprodbj</td><td>(<div class="dropBlock reference-citations"><a href="#bib191" id="back-bib191" data-db-target-for="bib191-d10391928e4259" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib191">Wen et al., 2020</a><div data-db-target-of="bib191-d10391928e4259" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a></td></tr><tr><td>PDV</td><td>(<div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d10391928e4270" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Li et al., 2019b</a><div data-db-target-of="bib102-d10391928e4270" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/wenbostar/PDV" target="_blank">https://github.com/wenbostar/PDV</a></td></tr><tr><td>PepQuery</td><td>(<div class="dropBlock reference-citations"><a href="#bib190" id="back-bib190" data-db-target-for="bib190-d10391928e4281" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib190">Wen et al., 2019</a><div data-db-target-of="bib190-d10391928e4281" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://pepquery.org" target="_blank">http://pepquery.org</a></td></tr><tr><td>PTM-SEA</td><td>(<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d10391928e4292" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Krug et al., 2019</a><div data-db-target-of="bib87-d10391928e4292" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A curated resource for phosphosite-specific signature analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a></td></tr><tr><td>PTMsigDB</td><td>(<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d10391928e4303" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Krug et al., 2019</a><div data-db-target-of="bib87-d10391928e4303" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A curated resource for phosphosite-specific signature analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/</a></td></tr><tr><td>Terra</td><td>Broad Institute data science platform.</td><td><a class="inter-ref-link" href="https://terra.bio/" target="_blank">https://terra.bio/</a></td></tr><tr><td>Panoply</td><td>(<div class="dropBlock reference-citations"><a href="#bib216" id="back-bib216" data-db-target-for="bib216-d10391928e4323" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib216">Mani et al., 2021</a><div data-db-target-of="bib216-d10391928e4323" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mani D.R. </li><li class="loa__item">Maynard M. </li><li class="loa__item">Kothadia R. </li><li class="loa__item">Krug K. </li><li class="loa__item">Christianson K.E. </li><li class="loa__item">Heiman D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Birger C. </li><li class="loa__item">Getz G. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">PANOPLY: a cloud-based platform for automated and reproducible proteogenomic data analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">580-582</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41592-021-01176-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=34040252&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85106486253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2021&pages=580-582&journal=Nat.+Methods&author=Mani+D.R.&author=Maynard+M.&author=Kothadia+R.&author=Krug+K.&author=Christianson+K.E.&author=Heiman+D.&author=Clauser+K.R.&author=Birger+C.&author=Getz+G.&author=Carr+S.A.&title=PANOPLY%3A+a+cloud-based+platform+for+automated+and+reproducible+proteogenomic+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/PANOPLY" target="_blank">https://github.com/broadinstitute/PANOPLY</a></td></tr><tr><td>CMap</td><td>(<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d10391928e4334" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Lamb et al., 2006</a><div data-db-target-of="bib92-d10391928e4334" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2951)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e4337" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e4337" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://clue.io/cmap" target="_blank">https://clue.io/cmap</a></td></tr><tr><td>LIMMA v3.36 (R Package)</td><td>(<div class="dropBlock reference-citations"><a href="#bib147" id="back-bib147" data-db-target-for="bib147-d10391928e4348" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib147">Ritchie et al., 2015</a><div data-db-target-of="bib147-d10391928e4348" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9097)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/limma.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/limma.html</a></td></tr><tr><td>FactoMineR v1.41NMF (R -package)</td><td>(<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d10391928e4359" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib46-d10391928e4359" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (488)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d10391928e4362" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">Lê et al., 2008</a><div data-db-target-of="bib95-d10391928e4362" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lê S. </li><li class="loa__item">Josse J. </li><li class="loa__item">Husson F. </li></ul><div class="reference-citations__detials">FactoMineR: AnRPackage for Multivariate Analysis.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Softw.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.18637/jss.v025.i01">https://doi.org/10.18637/jss.v025.i01</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.18637%2Fjss.v025.i01&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-46749110034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3306)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&journal=J.%C2%A0Stat.+Softw.&author=L%C3%AA+S.&author=Josse+J.&author=Husson+F.&title=FactoMineR%3A+AnRPackage+for+Multivariate+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/FactoMineR/index.html" target="_blank">https://cran.r-project.org/web/packages/FactoMineR/index.html</a></td></tr><tr><td>MClust v5.4 (R package)</td><td>(<div class="dropBlock reference-citations"><a href="#bib220" id="back-bib220" data-db-target-for="bib220-d10391928e4373" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib220">Scrucca et al., 2016</a><div data-db-target-of="bib220-d10391928e4373" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Scrucca L. </li><li class="loa__item">Fop M. </li><li class="loa__item">Murphy T.B. </li><li class="loa__item">Raftery A.E. </li></ul><div class="reference-citations__detials">mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models.<div class="popup-ref"><span class="ref__series"><i>The R Journal.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">289</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.32614%2FRJ-2016-021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27818791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=289&journal=The+R+Journal&author=Scrucca+L.&author=Fop+M.&author=Murphy+T.B.&author=Raftery+A.E.&title=mclust+5%3A+Clustering%2C+Classification+and+Density+Estimation+Using+Gaussian+Finite+Mixture+Models">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/mclust/index.html" target="_blank">https://cran.r-project.org/web/packages/mclust/index.html</a></td></tr><tr><td>g:Profiler</td><td>(<div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d10391928e4384" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Raudvere et al., 2019</a><div data-db-target-of="bib144-d10391928e4384" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Raudvere U. </li><li class="loa__item">Kolberg L. </li><li class="loa__item">Kuzmin I. </li><li class="loa__item">Arak T. </li><li class="loa__item">Adler P. </li><li class="loa__item">Peterson H. </li><li class="loa__item">Vilo J. </li></ul><div class="reference-citations__detials">g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update).<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W191-W198</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkz369&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31066453&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85068995138&amp;cf=" onclick="newWindow(this.href);return false">Scopus (585)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W191-W198&journal=Nucleic+Acids+Res.&author=Raudvere+U.&author=Kolberg+L.&author=Kuzmin+I.&author=Arak+T.&author=Adler+P.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler%3A+a+web+server+for+functional+enrichment+analysis+and+conversions+of+gene+lists+%282019+update%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://biit.cs.ut.ee/gprofiler/gost" target="_blank">https://biit.cs.ut.ee/gprofiler/gost</a></td></tr><tr><td>Cytoscape</td><td>(<div class="dropBlock reference-citations"><a href="#bib154" id="back-bib154" data-db-target-for="bib154-d10391928e4395" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib154">Shannon et al., 2003</a><div data-db-target-of="bib154-d10391928e4395" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shannon P. </li><li class="loa__item">Markiel A. </li><li class="loa__item">Ozier O. </li><li class="loa__item">Baliga N.S. </li><li class="loa__item">Wang J.T. </li><li class="loa__item">Ramage D. </li><li class="loa__item">Amin N. </li><li class="loa__item">Schwikowski B. </li><li class="loa__item">Ideker T. </li></ul><div class="reference-citations__detials">Cytoscape: a software environment for integrated models of biomolecular interaction networks.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17796)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cytoscape.org/" target="_blank">https://cytoscape.org/</a></td></tr><tr><td>ImmuneSubtypeClassifier</td><td>(<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d10391928e4406" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Gibbs, 2020</a><div data-db-target-of="bib50-d10391928e4406" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbs D.L. </li></ul><div class="reference-citations__detials">Robust classification of Immune Subtypes in Cancer.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.01.17.910950">https://doi.org/10.1101/2020.01.17.910950</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.01.17.910950&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85095499730&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Gibbs+D.L.&title=Robust+classification+of+Immune+Subtypes+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier" target="_blank">https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier</a></td></tr><tr><td>ProteinPaint</td><td>(<div class="dropBlock reference-citations"><a href="#bib211" id="back-bib211" data-db-target-for="bib211-d10391928e4417" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib211">Zhou et al., 2016</a><div data-db-target-of="bib211-d10391928e4417" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhou X. </li><li class="loa__item">Edmonson M.N. </li><li class="loa__item">Wilkinson M.R. </li><li class="loa__item">Patel A. </li><li class="loa__item">Wu G. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Rusch M.C. </li><li class="loa__item">Parker M. </li><li>et al.</li></ul><div class="reference-citations__detials">Exploring genomic alteration in pediatric cancer using ProteinPaint.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">4-6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3466&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26711108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84971602748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=4-6&journal=Nat.+Genet.&author=Zhou+X.&author=Edmonson+M.N.&author=Wilkinson+M.R.&author=Patel+A.&author=Wu+G.&author=Liu+Y.&author=Li+Y.&author=Zhang+Z.&author=Rusch+M.C.&author=Parker+M.&title=Exploring+genomic+alteration+in+pediatric+cancer+using+ProteinPaint">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://pecan.stjude.cloud/proteinpaint/" target="_blank">https://pecan.stjude.cloud/proteinpaint/</a></td></tr><tr><td>Ordinal</td><td>(<div class="dropBlock reference-citations"><a href="#bib213" id="back-bib213" data-db-target-for="bib213-d10391928e4428" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib213">Christensen, 2019</a><div data-db-target-of="bib213-d10391928e4428" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Christensen R.H.B. </li></ul><div class="reference-citations__detials">ordinal-Regression Models for Ordinal Data. R package version 2019.2-10. (The R Project for Statistical Computing).<div class="popup-ref"><span class="refDate">2019</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Christensen+R.H.B.&title=ordinal-Regression+Models+for+Ordinal+Data.+R+package+version+2019.2-10.+%28The+R+Project+for+Statistical+Computing%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://CRAN.R-project.org/package=ordinal" target="_blank">https://CRAN.R-project.org/package=ordinal</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900857-6">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec4.2" data-locator="sec4.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Resource availability</span></h3><section id="sec4.2.1" data-locator="sec4.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead contact</span></h4><div class="section-paragraph">Further information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#c8afa1a4a4adbcbcad88aabaa7a9aca1a6bbbca1bcbdbcade6a7baaf" target="_blank"><span class="__cf_email__" data-cfemail="9ff8f6f3f3faebebfadffdedf0fefbf6f1ecebf6ebeaebfab1f0edf8">[email&#160;protected]</span></a>)</div></section><section id="sec4.2.2" data-locator="sec4.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Material availability</span></h4><div class="section-paragraph">This study did not generate new unique reagents.</div></section></section><section id="sec4.3" data-locator="sec4.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental model and subject details</span></h3><section id="sec4.3.1" data-locator="sec4.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Human subjects</span></h4><div class="section-paragraph">A total of 113 participants (90 males, 23 females, 40-88 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40-88. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/" target="_blank">https://cptac-data-portal.georgetown.edu/</a>). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&amp;E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>.</div></section></section><section id="sec4.4" data-locator="sec4.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method details</span></h3><section id="sec4.4.1" data-locator="sec4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimen acquisition</span></h4><div class="section-paragraph">The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 mg. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 min prior to freezing in liquid nitrogen; the average ischemic time was 16 min from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than −140°C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA.</div></section><section id="sec4.4.2" data-locator="sec4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sequencing sample preparation</span></h4><div class="section-paragraph">Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol using QIAGEN’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3-5mL) to serve as matched normal reference material. The Qubit dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification.</div><section id="sec4.4.2.1" data-locator="sec4.4.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole exome sequencing (WES)</span></h4><div class="section-paragraph"><em>Library construction and Hybrid Selection</em>. Library construction was performed as described in (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d10391928e4483" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">Fisher et al., 2011</a><div data-db-target-of="bib37-d10391928e4483" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fisher S. </li><li class="loa__item">Barry A. </li><li class="loa__item">Abreu J. </li><li class="loa__item">Minie B. </li><li class="loa__item">Nolan J. </li><li class="loa__item">Delorey T.M. </li><li class="loa__item">Young G. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Allen A. </li><li class="loa__item">Ambrogio L. </li><li>et al.</li></ul><div class="reference-citations__detials">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (256)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with the following modifications: initial genomic DNA input into shearing was reduced from 3μg to 20-250ng in 50μL of solution. For adaptor ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adaptor ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina’s Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system.</div><div class="section-paragraph"><em>Cluster Amplification and Sequencing</em>. After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adaptor and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling.</div></section><section id="sec4.4.2.2" data-locator="sec4.4.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole genome sequencing (WGS)</span></h4><div class="section-paragraph"><em>Cluster Amplification and Sequencing</em>. An aliquot of genomic DNA (350ng in 50μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adaptor (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into 24-plexes.</div><div class="section-paragraph">Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging.</div></section></section><section id="sec4.4.3" data-locator="sec4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Array based methylation analysis</span></h4><div class="section-paragraph">The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt; 850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet.</div></section><section id="sec4.4.4" data-locator="sec4.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA and miRNA sequencing</span></h4><section id="sec4.4.4.1" data-locator="sec4.4.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quality assurance and control of RNA analytes</span></h4><div class="section-paragraph">All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt; 7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer’s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system.</div></section><section id="sec4.4.4.2" data-locator="sec4.4.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adaptor and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository.</div></section><section id="sec4.4.4.3" data-locator="sec4.4.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-seq library construction</span></h4><div class="section-paragraph">miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer’s instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA).</div></section><section id="sec4.4.4.4" data-locator="sec4.4.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (<div class="dropBlock reference-citations"><a href="#bib214" id="back-bib214" data-db-target-for="bib214-d10391928e4525" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib214">Chu et al., 2016</a><div data-db-target-of="bib214-d10391928e4525" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chu A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Brooks D. </li><li class="loa__item">Mungall A.J. </li><li class="loa__item">Birol I. </li><li class="loa__item">Coope R. </li><li class="loa__item">Ma Y. </li><li class="loa__item">Jones S. </li><li class="loa__item">Marra M.A. </li></ul><div class="reference-citations__detials">Large-scale profiling of microRNAs for The Cancer Genome Atlas.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+The+Cancer+Genome+Atlas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Data from samples were assessed for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository.</div></section></section><section id="sec4.4.5" data-locator="sec4.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mass spectrometry methods</span></h4><div class="section-paragraph">The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e4533" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e4533" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Acetyl-enrichment was performed as described before (<div class="dropBlock reference-citations"><a href="#bib88" id="back-bib88" data-db-target-for="bib88-d10391928e4536" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib88">Krug et al., 2020</a><div data-db-target-of="bib88-d10391928e4536" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Blumenberg L. </li><li class="loa__item">Karpova A. </li><li class="loa__item">Anurag M. </li><li class="loa__item">Miles G. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Geffen Y. </li><li class="loa__item">Tang L.C. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">183</span></span>: <span class="ref__seriesPages">1436-1456.e31</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33212010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85096494429&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2020&pages=1436-1456.e31&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Landscape+of+Breast+Cancer+Tumorigenesis+and+Targeted+Therapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with modifications as indicated below.</div><section id="sec4.4.5.1" data-locator="sec4.4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein extraction and tryptic digestion</span></h4><div class="section-paragraph">Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 min at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 μg/mL aprotinin (Sigma, A6103), 10 μg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 min and protein concentrations of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an h at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 min in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129-02541) for 2 h and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ∼3), and centrifuged at 1,500 g for 15 min to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus.</div></section><section id="sec4.4.5.2" data-locator="sec4.4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Common reference (CR) pool and plex layout</span></h4><div class="section-paragraph"><em>Proteome, Phosphoproteome, and Acetyprotome:</em> The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. A common physical, rather than <em>in silico</em> reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data analyses.</div><div class="section-paragraph">110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5<sup>th</sup> tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set.</div><div class="section-paragraph">To ensure capacity for additional samples or experiments given a target input of 300 μg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 μg peptide / sample for individual sample analysis, an additional amount of 120 μg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 μg aliquots, dried down, and frozen at −80°C until use.</div><div class="section-paragraph"><span class="underline">Ubiquitylproteome:</span> 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types:<ul class="ce-list--remove-bullets" id="olist0010"><li class="ce-list--remove-bullets__list-item" id="o0010"><span class="label">1.</span><div class="ce-list--remove-bullets__list-item__text">104/147 samples included in ubiquitylproteome study and made full CR contribution, &gt; 1000 ug available.</div></li><li class="ce-list--remove-bullets__list-item" id="o0015"><span class="label">2.</span><div class="ce-list--remove-bullets__list-item__text">42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750 ug −1000 ug available.</div></li><li class="ce-list--remove-bullets__list-item" id="o0020"><span class="label">3.</span><div class="ce-list--remove-bullets__list-item__text">63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, &lt; 750 ug available.</div></li></ul></div><div class="section-paragraph">CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR’s proportion of 50:50), and aliquoted into 750 ug aliquots.</div><div class="section-paragraph">In the ubiquitylproteome experimental design, 15 11-plex experiments contained <strong>87</strong> unique tumor samples in the C channels and <strong>60</strong> normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11<sup>th</sup> channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished).</div></section><section id="sec4.4.5.3" data-locator="sec4.4.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT-11 labeling of peptides</span></h4><div class="section-paragraph"><em>Proteome, Phosphoproteome, Acetylproteome:</em> 300 μg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a “reduced labeling” approach (<div class="dropBlock reference-citations"><a href="#bib222" id="back-bib222" data-db-target-for="bib222-d10391928e4592" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib222">Zecha et al., 2019</a><div data-db-target-of="bib222-d10391928e4592" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zecha J. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Kanashova T. </li><li class="loa__item">Avanessian S.C. </li><li class="loa__item">Kane M.H. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Kuster B. </li></ul><div class="reference-citations__detials">TMT labeling for the masses: A robust and cost-efficient, in-solution labeling approach.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+A+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 μg peptide aliquot of an individual sample, peptides were reconstituted to 5 mg/mL with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 μL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 μg aliquot from each sample and analyzing 0.25 μg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples were taken out, an additional 4 μg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of ± 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample’s contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1…Differentially labeled peptides were then mixed (11 × 300 μg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399).</div></section><section id="sec4.4.5.4" data-locator="sec4.4.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide fractionation</span></h4><div class="section-paragraph"><em>For Proteome, Phosphoproteome, Acetylproteome:</em> To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (<div class="dropBlock reference-citations"><a href="#bib218" id="back-bib218" data-db-target-for="bib218-d10391928e4601" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib218">Mertins et al., 2018</a><div data-db-target-of="bib218-d10391928e4601" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Tang L.C. </li><li class="loa__item">Krug K. </li><li class="loa__item">Clark D.J. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Clauss T.R. </li><li class="loa__item">Shah P. </li><li class="loa__item">Gillette M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was reconstituted in 935 μL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20,000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 μm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were made by 40–fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1 L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/mL, Sigma-Aldrich) to ∼850 ml of HPLC grade water, then add ∼35 mL of 10% (vol/vol) formic acid (&gt; 95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 L with HPLC grade water. The 96 min separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701-5200), with fractions combined in a stepwise concatenation strategy and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 μg/ μL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below.</div></section><section id="sec4.4.5.5" data-locator="sec4.4.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide enrichment</span></h4><div class="section-paragraph">Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ∼261.25 μg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 μL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 μL of the IMAC beads for 30 min with end-over-end rotation at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 μL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 μL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 × 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 μL 1% FA and phosphopeptides were eluted from the stage tips with 60 μL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 μL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 μL was injected per run.</div></section><section id="sec4.4.5.6" data-locator="sec4.4.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Acetylpeptide enrichment</span></h4><div class="section-paragraph">Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (∼783.75 μg peptides per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 h at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 × 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run.</div></section><section id="sec4.4.5.7" data-locator="sec4.4.5.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Ubiquitylpeptide enrichment with on-bead TMT labeling</span></h4><div class="section-paragraph">Ubiquitin enrichment was performed based on the recently published UbiFast protocol (<div class="dropBlock reference-citations"><a href="#bib177" id="back-bib177" data-db-target-for="bib177-d10391928e4619" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib177">Udeshi et al., 2020</a><div data-db-target-of="bib177-d10391928e4619" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Udeshi N.D. </li><li class="loa__item">Mani D.C. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Fereshetian S. </li><li class="loa__item">Gasser J.A. </li><li class="loa__item">Svinkina T. </li><li class="loa__item">Olive M.E. </li><li class="loa__item">Ebert B.L. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Rapid and deep-scale ubiquitylation profiling for biology and translational research.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-14175-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31953384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078023220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=359&journal=Nat.+Commun.&author=Udeshi+N.D.&author=Mani+D.C.&author=Satpathy+S.&author=Fereshetian+S.&author=Gasser+J.A.&author=Svinkina+T.&author=Olive+M.E.&author=Ebert+B.L.&author=Mertins+P.&author=Carr+S.A.&title=Rapid+and+deep-scale+ubiquitylation+profiling+for+biology+and+translational+research">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Before enrichment, anti-K-e-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4°C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4°C at a concentration of 0.5 ug/uL.</div><div class="section-paragraph">For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000 g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4°C for 1 h. Following enrichment, samples were spun at 2000 rcf for ∼1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ∼5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30-40 s, left to settle on ice for 10-20 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were re-constituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20°C for 10 min, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 min. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted K-GG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of K-GG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples.</div></section><section id="sec4.4.5.8" data-locator="sec4.4.5.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for proteomics analyses</span></h4><div class="section-paragraph">Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross (360 mm, IDEX Health &amp; Science, UH-906x). The column was heated to 50°C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide fraction, ∼1 ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22 cm x 75um internal diameter C18 silica picofrit capillary column (1.9 mm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10 um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-min LC-MS/MS method consisted of a 10 min column-equilibration procedure; a 20 min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 min with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154 min with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate).</div><div class="section-paragraph">For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 ms, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 s for proteome or 30 s for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2-6, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (<div class="dropBlock reference-citations"><a href="#bib219" id="back-bib219" data-db-target-for="bib219-d10391928e4631" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib219">Myers et al., 2018</a><div data-db-target-of="bib219-d10391928e4631" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Myers S.A. </li><li class="loa__item">Klaeger S. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Viner R. </li><li class="loa__item">Choi J. </li><li class="loa__item">Rogers J. </li><li class="loa__item">Clauser K. </li><li class="loa__item">Udeshi N.D. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">Evaluation of Advanced Precursor Determination for Tandem Mass Tag (TMT)-Based Quantitative Proteomics across Instrument Platforms.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">542-547</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30351145&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=542-547&journal=J.+Proteome+Res.&author=Myers+S.A.&author=Klaeger+S.&author=Satpathy+S.&author=Viner+R.&author=Choi+J.&author=Rogers+J.&author=Clauser+K.&author=Udeshi+N.D.&author=Carr+S.A.&title=Evaluation+of+Advanced+Precursor+Determination+for+Tandem+Mass+Tag+%28TMT%29-Based+Quantitative+Proteomics+across+Instrument+Platforms">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard).</div><div class="section-paragraph">For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 s with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 ms for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge states 2-6, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 s (20 s for ubiquitylproteome), with a ±10 ppm mass tolerance. “Perform dependent scan on single charge state per precursor only” was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis.</div></section></section><section id="sec4.4.6" data-locator="sec4.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immunohistochemistry (IHC) analysis</span></h4><section id="sec4.4.6.1" data-locator="sec4.4.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMA construction</span></h4><div class="section-paragraph">Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases.</div></section><section id="sec4.4.6.2" data-locator="sec4.4.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune cell marker, Immunohistochemistry (IHC)</span></h4><div class="section-paragraph">Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0-3 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the “ordinal” package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists.</div></section><section id="sec4.4.6.3" data-locator="sec4.4.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multiplex Immunofluorescence (mIF) staining</span></h4><div class="section-paragraph">Development and optimization of the mIF platform has been previously described (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d10391928e4654" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Parra et al., 2020</a><div data-db-target-of="bib132-d10391928e4654" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Parra E.R. </li><li class="loa__item">Jiang M. </li><li class="loa__item">Solis L. </li><li class="loa__item">Mino B. </li><li class="loa__item">Laberiano C. </li><li class="loa__item">Hernandez S. </li><li class="loa__item">Gite S. </li><li class="loa__item">Verma A. </li><li class="loa__item">Tetzlaff M. </li><li class="loa__item">Haymaker C. </li><li>et al.</li></ul><div class="reference-citations__detials">Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies.<div class="popup-ref"><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">255</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcancers12020255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078416940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=255&journal=Cancers+%28Basel%29&author=Parra+E.R.&author=Jiang+M.&author=Solis+L.&author=Mino+B.&author=Laberiano+C.&author=Hernandez+S.&author=Gite+S.&author=Verma+A.&author=Tetzlaff+M.&author=Haymaker+C.&title=Procedural+Requirements+and+Recommendations+for+Multiplex+Immunofluorescence+Tyramide+Signal+Amplification+Assays+to+Support+Translational+Oncology+Studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). mIF staining was performed in a 4-μm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences).</div></section><section id="sec4.4.6.4" data-locator="sec4.4.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">IHC based assessment of TP63 low samples (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>)</span></h4><div class="section-paragraph">For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66-2754) deemed LSCC by TCGA pathology.</div></section></section></section><section id="sec4.5" data-locator="sec4.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and statistical analysis</span></h3><section id="sec4.5.1" data-locator="sec4.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic data analysis</span></h4><section id="sec4.5.1.1" data-locator="sec4.5.1.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Copy number calling</span></h4><div class="section-paragraph">Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX (<a class="inter-ref-link" href="https://github.com/mctp/cnvex" target="_blank">https://github.com/mctp/cnvex</a>). CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3-0.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary segmentation (RBS) (<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d10391928e4679" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Gey and Lebarbier, 2008</a><div data-db-target-of="bib47-d10391928e4679" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gey S. </li><li class="loa__item">Lebarbier E. </li></ul><div class="reference-citations__detials">Using CART to detect multiple change points in the mean for large sample.<div class="popup-ref"><span class="ref__series"><i>HAL.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume"></span> (<span class="refComment">ffhal-00327146</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&journal=HAL&author=Gey+S.&author=Lebarbier+E.&title=Using+CART+to+detect+multiple+change+points+in+the+mean+for+large+sample">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and dynamic programming (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d10391928e4682" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Bellman, 1961</a><div data-db-target-of="bib13-d10391928e4682" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bellman R. </li></ul><div class="reference-citations__detials">On the approximation of curves by line segments using dynamic programming.<div class="popup-ref"><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters alpha = 0.01, trim = 0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively “pruned” by merging segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were “pruned” using a statistical model selection method (<div class="dropBlock reference-citations"><a href="#bib98" id="back-bib98" data-db-target-for="bib98-d10391928e4685" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Lebarbier, 2005</a><div data-db-target-of="bib98-d10391928e4685" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lebarbier E. </li></ul><div class="reference-citations__detials">Detecting multiple change-points in the mean of Gaussian process by model selection.<div class="popup-ref"><span class="ref__series"><i>Signal Processing.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (103)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Processing&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the final set of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the underlying data.</div></section><section id="sec4.5.1.2" data-locator="sec4.5.1.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic variant calling</span></h4><div class="section-paragraph">Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels “GATK bundle known” and “Mills 1000G” somatic variants were called using Sentieon TNScope with the following settings “–max_fisher_pv_active 0.05–min_tumor_allele_frac 0.0075–min_init_tumor_lod 2.5–assemble_mode 4–SVIntrusionThres 5,” with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings “–assembly GRCh38–species homo_sapiens–cache_version 97–format vcf–gene_phenotype–symbol–canonical–ccds–hgvs–biotype–tsl–uniprot–domains–appris–protein–variant_class–sift b,–polyphen b–no_stats–total_length–allele_number–no_escape–flag_pick_allele–pick_order canonical,tsl,biotype,rank,ccds,length–buffer_size 20000.” Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most ‘severe’ consequence occurring in a ‘reliable’ transcript was chosen. To accomplish this the computed consequences were ranked by ‘severity’ and ‘priority’ reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked ‘CANONICAL’ or ‘PICK’ by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD &gt; 5, NLOD &gt; 6, Fisher’s test p value &lt; 0.05, NFLOD &gt; 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD.</div></section><section id="sec4.5.1.3" data-locator="sec4.5.1.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline variant calling</span></h4><div class="section-paragraph">Germline variant calling and annotation was carried out analogously using DNAScope using the following settings “–emit_conf 10–call_conf 10” to generate a VCF file and “–emit_mode GVCF” to generate a gVCF file. The variants were custom-filtered based on the specific applications.</div></section><section id="sec4.5.1.4" data-locator="sec4.5.1.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">GISTIC and mutSig analysis</span></h4><div class="section-paragraph">The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d10391928e4703" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Mermel et al., 2011</a><div data-db-target-of="bib120-d10391928e4703" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1222)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value &lt; 0.25 considered significant. The following parameters were used:<ul class="ce-list--remove-bullets" id="ulist0030"><li class="ce-list--remove-bullets__list-item" id="u0055"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Amplification Threshold = 0.1</div></li><li class="ce-list--remove-bullets__list-item" id="u0060"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Deletion Threshold = 0.1</div></li><li class="ce-list--remove-bullets__list-item" id="u0065"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Cap Values = 1.5</div></li><li class="ce-list--remove-bullets__list-item" id="u0070"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Broad Length Cutoff = 0.98</div></li><li class="ce-list--remove-bullets__list-item" id="u0075"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Remove X-Chromosome = 0</div></li><li class="ce-list--remove-bullets__list-item" id="u0080"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Confidence Level = 0.99</div></li><li class="ce-list--remove-bullets__list-item" id="u0085"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Join Segment Size = 4</div></li><li class="ce-list--remove-bullets__list-item" id="u0090"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Arm Level Peel Off = 1</div></li><li class="ce-list--remove-bullets__list-item" id="u0095"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Maximum Sample Segments = 2000</div></li><li class="ce-list--remove-bullets__list-item" id="u0100"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Gene GISTIC = 1</div></li></ul></div><div class="section-paragraph">Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from −2 to 2, where 0 means no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: −1 means deletion beyond the threshold; −2 means deletions greater than the minimum arm-level copy number observed in the sample.</div><div class="section-paragraph">The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV (<div class="dropBlock reference-citations"><a href="#bib94" id="back-bib94" data-db-target-for="bib94-d10391928e4751" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib94">Lawrence et al., 2014</a><div data-db-target-of="bib94-d10391928e4751" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">Mermel C.H. </li><li class="loa__item">Robinson J.T. </li><li class="loa__item">Garraway L.A. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Gabriel S.B. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Discovery and saturation analysis of cancer genes across 21 tumour types.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">505</span></span>: <span class="ref__seriesPages">495-501</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature12912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24390350&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84892833777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1758)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=495-501&journal=Nature&author=Lawrence+M.S.&author=Stojanov+P.&author=Mermel+C.H.&author=Robinson+J.T.&author=Garraway+L.A.&author=Golub+T.R.&author=Meyerson+M.&author=Gabriel+S.B.&author=Lander+E.S.&author=Getz+G.&title=Discovery+and+saturation+analysis+of+cancer+genes+across+21+tumour+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was run on these filtered results to evaluate the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census (<div class="dropBlock reference-citations"><a href="#bib160" id="back-bib160" data-db-target-for="bib160-d10391928e4754" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib160">Sondka et al., 2018</a><div data-db-target-of="bib160-d10391928e4754" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sondka Z. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Ward S.A. </li><li class="loa__item">Dunham I. </li><li class="loa__item">Forbes S.A. </li></ul><div class="reference-citations__detials">The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">696-705</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41568-018-0060-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30293088&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054518074&amp;cf=" onclick="newWindow(this.href);return false">Scopus (268)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=696-705&journal=Nat.+Rev.+Cancer&author=Sondka+Z.&author=Bamford+S.&author=Cole+C.G.&author=Ward+S.A.&author=Dunham+I.&author=Forbes+S.A.&title=The+COSMIC+Cancer+Gene+Census%3A+describing+genetic+dysfunction+across+all+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with false discovery rates (q values) recalculated. Genes of q value &lt; 0.1 were declared significant.</div></section></section><section id="sec4.5.2" data-locator="sec4.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA-seq and miRNaseq quantification</span></h4><section id="sec4.5.2.1" data-locator="sec4.5.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA quantification</span></h4><div class="section-paragraph">The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page (<a class="inter-ref-link" href="https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files" target="_blank">https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files</a>). The readcount is later converted to FPKM and FPKM-UQ using GDC’s formula (<a class="inter-ref-link" href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/" target="_blank">https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/</a>).</div></section><section id="sec4.5.2.2" data-locator="sec4.5.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Isoform specific RNA quantification</span></h4><div class="section-paragraph">For TP63 isoform centric analysis presented in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>, the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression (<div class="dropBlock reference-citations"><a href="#bib100" id="back-bib100" data-db-target-for="bib100-d10391928e4779" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">Li et al., 2017</a><div data-db-target-of="bib100-d10391928e4779" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li M. </li><li class="loa__item">Xie X. </li><li class="loa__item">Zhou J. </li><li class="loa__item">Sheng M. </li><li class="loa__item">Yin X. </li><li class="loa__item">Ko E.-A. </li><li class="loa__item">Zhou T. </li><li class="loa__item">Gu W. </li></ul><div class="reference-citations__detials">Quantifying circular RNA expression from RNA-seq data using model-based framework.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinformatics&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3′ end was copied to the 5′ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts.</div></section><section id="sec4.5.2.3" data-locator="sec4.5.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">DNA methylation data preprocessing</span></h4><div class="section-paragraph">Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d10391928e4787" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Fortin et al., 2014</a><div data-db-target-of="bib39-d10391928e4787" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fortin J.-P. </li><li class="loa__item">Labbe A. </li><li class="loa__item">Lemire M. </li><li class="loa__item">Zanke B.W. </li><li class="loa__item">Hudson T.J. </li><li class="loa__item">Fertig E.J. </li><li class="loa__item">Greenwood C.M. </li><li class="loa__item">Hansen K.D. </li></ul><div class="reference-citations__detials">Functional normalization of 450k methylation array data improves replication in large cancer studies.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (313)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.-P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We flagged locus as NA where probes did not meet a detection p value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent analysis.</div><div class="section-paragraph">Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5′ UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package (<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d10391928e4792" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Fraley et al., 2016</a><div data-db-target-of="bib40-d10391928e4792" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fraley C. </li><li class="loa__item">Raftery A.E. </li><li class="loa__item">Scrucca L. </li><li class="loa__item">Murphy T.B. </li><li class="loa__item">Fop M. </li></ul><div class="reference-citations__detials">mclust 5: clustering, classification and density estimation using Gaussian finite mixture models.<div class="popup-ref"><span class="ref__series"><i>The R Journal.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">289-317</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.32614%2FRJ-2016-021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27818791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=289-317&journal=The+R+Journal&author=Fraley+C.&author=Raftery+A.E.&author=Scrucca+L.&author=Murphy+T.B.&author=Fop+M.&title=mclust+5%3A+clustering%2C+classification+and+density+estimation+using+Gaussian+finite+mixture+models">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty &lt; 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis.</div></section><section id="sec4.5.2.4" data-locator="sec4.5.2.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-seq data analysis</span></h4><div class="section-paragraph">miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API (<a class="inter-ref-link" href="https://docs.gdc.cancer.gov" target="_blank">https://docs.gdc.cancer.gov</a>). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting (<a class="inter-ref-link" href="https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md" target="_blank">https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md</a>). The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression.</div></section></section><section id="sec4.5.3" data-locator="sec4.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomics data analysis</span></h4><section id="sec4.5.3.1" data-locator="sec4.5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Spectrum quality filtering and searching</span></h4><div class="section-paragraph">All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (<a class="inter-ref-link" href="https://www.broadinstitute.org/proteomics" target="_blank">https://www.broadinstitute.org/proteomics</a>). Similar MS/MS spectra acquired on the same precursor m/z within ± 45 s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length &gt; 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800-6000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser (<a class="inter-ref-link" href="https://genome.ucsc.edu/cgi-bin/hgTables" target="_blank">https://genome.ucsc.edu/cgi-bin/hgTables</a>) on June 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed ± 20 ppm mass tolerance for precursor and product ions, 30% minimum matched peak intensity, and “trypsin allow P” enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of −18 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −18 to 272 Da. For the acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of −690 to 70 Da.</div></section><section id="sec4.5.3.2" data-locator="sec4.5.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein grouping, and localization of PTMs</span></h4><div class="section-paragraph">Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels.</div><div class="section-paragraph">In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 – Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of &lt; 0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5-6, thresholds were optimized to yield a PSM-level FDR estimate of &lt; 0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher charge states.</div><div class="section-paragraph">In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMT-plex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called “one-hit wonders.” After assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method “expand subgroups, top uses shared” to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g., as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide sequence of &gt; 8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted toward the total number of proteins, and given related protein subgroup numbers (e.g., 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of &lt; 0.6% at the peptide-spectrum match level and &lt; 0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and &lt; 0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs.</div><div class="section-paragraph">In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g., different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, &gt; 1.1, essentially corresponded to at least 1 b or y ion located between two candidate sites that has a peak height &gt; 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, &gt; 99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue.</div></section><section id="sec4.5.3.3" data-locator="sec4.5.3.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantitation using TMT ratios</span></h4><div class="section-paragraph">Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method “expand subgroups, top uses shared” (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method “unexpand subgroups.” Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer’s Rule (<div class="dropBlock reference-citations"><a href="#bib152" id="back-bib152" data-db-target-for="bib152-d10391928e4836" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib152">Shadforth et al., 2005</a><div data-db-target-of="bib152-d10391928e4836" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shadforth I.P. </li><li class="loa__item">Dunkley T.P.J. </li><li class="loa__item">Lilley K.S. </li><li class="loa__item">Bessant C. </li></ul><div class="reference-citations__detials">i-Tracker: for quantitative proteomics using iTRAQ.<div class="popup-ref"><span class="ref__series"><i>BMC Genomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">145</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2164-6-145&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16242023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-27744450286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (218)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=145&journal=BMC+Genomics&author=Shadforth+I.P.&author=Dunkley+T.P.J.&author=Lilley+K.S.&author=Bessant+C.&title=i-Tracker%3A+for+quantitative+proteomics+using+iTRAQ">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and correction factors obtained from the reagent manufacturer’s certificate of analysis (<a class="inter-ref-link" href="https://www.thermofisher.com/order/catalog/product/90406" target="_blank">https://www.thermofisher.com/order/catalog/product/90406</a>) for TMT11 lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity &lt; 50% (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets.</div></section><section id="sec4.5.3.4" data-locator="sec4.5.3.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Two-component normalization of TMT ratios</span></h4><div class="section-paragraph">It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or downregulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians <em>N</em>(<em>mi</em>1,<em>si</em>1) and <em>N</em>(<em>mi</em>2,<em>si</em>2) for a sample <em>i</em> were fitted and used in the normalization process as follows: the mode <em>mi</em> of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians constrained to be <em>mi</em>, i.e., <em>mi</em>1 = <em>mi</em>2 = <em>mi</em>. The Gaussian with the smaller estimated standard deviation <em>si</em> = min(<!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif"><mi>s</mi></math><em>i</em>1,<!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif"><mi>s</mi></math><em>i</em>2) was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using (<em>mi</em>,<em>si</em>) by subtracting the mean <em>mi</em> from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation <em>si</em>.</div></section><section id="sec4.5.3.5" data-locator="sec4.5.3.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Normalization of acetyl- and ubiquitylproteome</span></h4><div class="section-paragraph">To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site’s main accession number did not yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function lm, PTM ∼protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances.</div></section></section><section id="sec4.5.4" data-locator="sec4.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic analysis</span></h4><section id="sec4.5.4.1" data-locator="sec4.5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quality control and batch effect assessment</span></h4><div class="section-paragraph">We used guided principal component analysis (gPCA) (<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d10391928e4907" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Reese et al., 2013</a><div data-db-target-of="bib145-d10391928e4907" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reese S.E. </li><li class="loa__item">Archer K.J. </li><li class="loa__item">Therneau T.M. </li><li class="loa__item">Atkinson E.J. </li><li class="loa__item">Vachon C.M. </li><li class="loa__item">de Andrade M. </li><li class="loa__item">Kocher J.-P.A. </li><li class="loa__item">Eckel-Passow J.E. </li></ul><div class="reference-citations__detials">A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2877-2883</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtt480&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23958724&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84890087087&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2877-2883&journal=Bioinformatics&author=Reese+S.E.&author=Archer+K.J.&author=Therneau+T.M.&author=Atkinson+E.J.&author=Vachon+C.M.&author=de+Andrade+M.&author=Kocher+J.-P.A.&author=Eckel-Passow+J.E.&title=A+new+statistic+for+identifying+batch+effects+in+high-throughput+genomic+data+that+uses+guided+principal+component+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p value = 1) in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets.</div><div class="section-paragraph">To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median pearson correlation &gt; 0.85.</div><div class="section-paragraph">In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR channels as common denominator (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figure S1</a>C). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p value &lt; 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using <em>limma’s</em> removeBatchEffect (<a class="inter-ref-link" href="https://rdrr.io/bioc/limma/man/removeBatchEffect.html" target="_blank">https://rdrr.io/bioc/limma/man/removeBatchEffect.html</a>) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate.</div></section><section id="sec4.5.4.2" data-locator="sec4.5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Dataset filtering</span></h4><div class="section-paragraph">Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in &gt; 70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at least two observed TMT ratios in &gt; 25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio in &gt; 25% of samples.</div><div class="section-paragraph">Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods.</div></section><section id="sec4.5.4.3" data-locator="sec4.5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unsupervised multi-omic clustering using NMF</span></h4><div class="section-paragraph">We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e4934" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e4934" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, given a factorization rank <em>k</em> (where <em>k</em> is the number of clusters), NMF decomposes a <em>p x n</em> data matrix <em>V</em> into two matrices <em>W</em> and <em>H</em> such that multiplication of <em>W</em> and <em>H</em> approximates <em>V</em>. Matrix <em>H</em> is a <em>k x n</em> matrix whose entries represent weights for each sample (1 to <em>N</em>) to contribute to each cluster (1 to <em>k</em>), whereas matrix <em>W</em> is a <em>p x k</em> matrix representing weights for each feature (1 to <em>p</em>) to contribute to each cluster (1 to <em>k</em>). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em>H</em>. We defined a ”cluster core” as the set of samples with cluster membership score &gt; 0.5.</div><div class="section-paragraph">To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an <em>in silico</em> common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or −2) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by NMF as follows:<ul class="ce-list--remove-bullets" id="olist0015"><li class="ce-list--remove-bullets__list-item" id="o0025"><span class="label">1.</span><div class="ce-list--remove-bullets__list-item__text">Create one data matrix with all negative numbers zeroed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0030"><span class="label">2.</span><div class="ce-list--remove-bullets__list-item__text">Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed.</div></li><li class="ce-list--remove-bullets__list-item" id="o0035"><span class="label">3.</span><div class="ce-list--remove-bullets__list-item__text">Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF.</div></li></ul></div><div class="section-paragraph">The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d10391928e4995" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib46-d10391928e4995" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (488)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and using the factorization method described in (<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d10391928e4998" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Brunet et al., 2004</a><div data-db-target-of="bib16-d10391928e4998" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Metagenes and molecular pattern discovery using matrix factorization.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1082)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters between <em>k</em> = 2 and 8. For each k we factorized matrix <em>V</em> using 50 iterations with random initializations of <em>W</em> and <em>H</em>. To determine the optimal factorization rank we calculated two metrics for each <em>k</em>: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix as defined in (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d10391928e5012" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Kim and Park, 2007</a><div data-db-target-of="bib80-d10391928e5012" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim H. </li><li class="loa__item">Park H. </li></ul><div class="reference-citations__detials">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (518)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinformatics&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) measuring the reproducibility of the clustering across 50 iterations. The optimal <em>k</em> was defined as the maximum of the product of both metrics for cluster numbers between <em>k</em> = 3 and 8</div><div class="section-paragraph">Having determined the optimal factorization rank <em>k</em>, in order to achieve robust factorization of the multi-omic data matrix <em>V</em>, we repeated the NMF analysis using 500 iterations with random initializations of <em>W</em> and <em>H</em> and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix <em>W</em> of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first normalized each row in matrix <em>W</em> by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix W<sub>signed</sub> contained signed cluster weights for each feature in the input matrix.</div><div class="section-paragraph">For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights (W<sub>signed</sub>) onto Hallmark pathway gene sets (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d10391928e5042" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Liberzon et al., 2015</a><div data-db-target-of="bib106-d10391928e5042" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1936)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using ssGSEA (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d10391928e5045" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Barbie et al., 2009</a><div data-db-target-of="bib12-d10391928e5045" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a> using the following parameters:<ul class="ce-list--remove-bullets" id="ulist0035"><li class="ce-list--remove-bullets__list-item" id="u0105"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">gene.set.database = “h.all.v6.2.symbols.gmt”</div></li><li class="ce-list--remove-bullets__list-item" id="u0110"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">sample.norm.type = “rank”</div></li><li class="ce-list--remove-bullets__list-item" id="u0115"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">weight = 1</div></li><li class="ce-list--remove-bullets__list-item" id="u0120"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">statistic = ”area.under.RES”</div></li><li class="ce-list--remove-bullets__list-item" id="u0125"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">output.score.type = “NES”</div></li><li class="ce-list--remove-bullets__list-item" id="u0130"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">nperm = 1000</div></li><li class="ce-list--remove-bullets__list-item" id="u0135"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">global.fdr = TRUE</div></li><li class="ce-list--remove-bullets__list-item" id="u0140"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">min.overlap = 5</div></li><li class="ce-list--remove-bullets__list-item" id="u0145"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">correl.type = ”z.score”</div></li></ul></div><div class="section-paragraph">To test the association of the resulting clusters to clinical variables we used Fisher’s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above.</div><div class="section-paragraph">The entire workflow described above has been implemented as a module for PANOPLY (<div class="dropBlock reference-citations"><a href="#bib115" id="back-bib115" data-db-target-for="bib115-d10391928e5092" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib115">Mani et al., 2020</a><div data-db-target-of="bib115-d10391928e5092" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mani D.R. </li><li class="loa__item">Maynard M. </li><li class="loa__item">Kothadia R. </li><li class="loa__item">Krug K. </li><li class="loa__item">Christianson K.E. </li><li class="loa__item">Heiman D. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Birger C. </li><li class="loa__item">Getz G. </li><li class="loa__item">Carr S.A. </li></ul><div class="reference-citations__detials">PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">580-582</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=580-582&journal=Nature&author=Mani+D.R.&author=Maynard+M.&author=Kothadia+R.&author=Krug+K.&author=Christianson+K.E.&author=Heiman+D.&author=Clauser+K.R.&author=Birger+C.&author=Getz+G.&author=Carr+S.A.&title=PANOPLY%3A+A+cloud-based+platform+for+automated+and+reproducible+proteogenomic+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) which runs on Broad’s Cloud platform Terra (<a class="inter-ref-link" href="https://app.terra.bio/" target="_blank">https://app.terra.bio/</a>). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a>.</div></section><section id="sec4.5.4.4" data-locator="sec4.5.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Integrative analysis with Stewart et al</span></h4><div class="section-paragraph">To enable integrative analysis of our protein data with the protein data in Stewart et al.(<div class="dropBlock reference-citations"><a href="#bib164" id="back-bib164" data-db-target-for="bib164-d10391928e5106" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib164">Stewart et al., 2019</a><div data-db-target-of="bib164-d10391928e5106" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stewart P.A. </li><li class="loa__item">Welsh E.A. </li><li class="loa__item">Slebos R.J.C. </li><li class="loa__item">Fang B. </li><li class="loa__item">Izumi V. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Zhang G. </li><li class="loa__item">Cen L. </li><li class="loa__item">Pettersson F. </li><li class="loa__item">Zhang Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic landscape of squamous cell lung cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">3578</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-11452-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395880&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85070354617&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3578&journal=Nat.+Commun.&author=Stewart+P.A.&author=Welsh+E.A.&author=Slebos+R.J.C.&author=Fang+B.&author=Izumi+V.&author=Chambers+M.&author=Zhang+G.&author=Cen+L.&author=Pettersson+F.&author=Zhang+Y.&title=Proteogenomic+landscape+of+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors.</div></section><section id="sec4.5.4.5" data-locator="sec4.5.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA subtyping</span></h4><div class="section-paragraph">Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication (<div class="dropBlock reference-citations"><a href="#bib196" id="back-bib196" data-db-target-for="bib196-d10391928e5114" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib196">Wilkerson et al., 2010</a><div data-db-target-of="bib196-d10391928e5114" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Yin X. </li><li class="loa__item">Hoadley K.A. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Hayward M.C. </li><li class="loa__item">Cabanski C.R. </li><li class="loa__item">Muldrew K. </li><li class="loa__item">Miller C.R. </li><li class="loa__item">Randell S.H. </li><li class="loa__item">Socinski M.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types.<div class="popup-ref"><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4864-4875</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-10-0199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20643781&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77957570386&amp;cf=" onclick="newWindow(this.href);return false">Scopus (163)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4864-4875&journal=Clin.+Cancer+Res.&author=Wilkerson+M.D.&author=Yin+X.&author=Hoadley+K.A.&author=Liu+Y.&author=Hayward+M.C.&author=Cabanski+C.R.&author=Muldrew+K.&author=Miller+C.R.&author=Randell+S.H.&author=Socinski+M.A.&title=Lung+squamous+cell+carcinoma+mRNA+expression+subtypes+are+reproducible%2C+clinically+important%2C+and+correspond+to+normal+cell+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d10391928e5117" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Hammerman et al., 2012</a><div data-db-target-of="bib54-d10391928e5117" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hammerman P.S. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Voet D. </li><li class="loa__item">Jing R. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Stojanov P. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Gabriel S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p &gt; 0.01) were marked to have ‘undecided’ subtype.</div></section><section id="sec4.5.4.6" data-locator="sec4.5.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Chromosomal instability index (CIN)</span></h4><div class="section-paragraph">The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample (<div class="dropBlock reference-citations"><a href="#bib180" id="back-bib180" data-db-target-for="bib180-d10391928e5125" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib180">Vasaikar et al., 2019</a><div data-db-target-of="bib180-d10391928e5125" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (136)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes.</div></section><section id="sec4.5.4.7" data-locator="sec4.5.4.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Fusion detection and analysis</span></h4><div class="section-paragraph">Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline (<div class="dropBlock reference-citations"><a href="#bib148" id="back-bib148" data-db-target-for="bib148-d10391928e5133" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib148">Robinson et al., 2017</a><div data-db-target-of="bib148-d10391928e5133" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson D.R. </li><li class="loa__item">Wu Y.-M. </li><li class="loa__item">Lonigro R.J. </li><li class="loa__item">Vats P. </li><li class="loa__item">Cobain E. </li><li class="loa__item">Everett J. </li><li class="loa__item">Cao X. </li><li class="loa__item">Rabban E. </li><li class="loa__item">Kumar-Sinha C. </li><li class="loa__item">Raymond V. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrative clinical genomics of metastatic cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">548</span></span>: <span class="ref__seriesPages">297-303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature23306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28783718&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85027893400&amp;cf=" onclick="newWindow(this.href);return false">Scopus (322)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2017&pages=297-303&journal=Nature&author=Robinson+D.R.&author=Wu+Y.-M.&author=Lonigro+R.J.&author=Vats+P.&author=Cobain+E.&author=Everett+J.&author=Cao+X.&author=Rabban+E.&author=Kumar-Sinha+C.&author=Raymond+V.&title=Integrative+clinical+genomics+of+metastatic+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib198" id="back-bib198" data-db-target-for="bib198-d10391928e5136" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib198">Wu et al., 2018</a><div data-db-target-of="bib198-d10391928e5136" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wu Y.-M. </li><li class="loa__item">Cieślik M. </li><li class="loa__item">Lonigro R.J. </li><li class="loa__item">Vats P. </li><li class="loa__item">Reimers M.A. </li><li class="loa__item">Cao X. </li><li class="loa__item">Ning Y. </li><li class="loa__item">Wang L. </li><li class="loa__item">Kunju L.P. </li><li class="loa__item">de Sarkar N. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>PCF/SU2C International Prostate Cancer Dream Team</span><br />Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">1770-1782.e14</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29906450&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85047481697&amp;cf=" onclick="newWindow(this.href);return false">Scopus (196)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=1770-1782.e14&journal=Cell&author=PCF%2FSU2C+International+Prostate+Cancer+Dream+Team&title=Inactivation+of+CDK12+Delineates+a+Distinct+Immunogenic+Class+of+Advanced+Prostate+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), fusioncatcher_v1.10 (Nicorici et al.) and arriba_v1.1.0 (<a class="inter-ref-link" href="https://github.com/suhrig/arriba/" target="_blank">https://github.com/suhrig/arriba/</a>). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described (<div class="dropBlock reference-citations"><a href="#bib181" id="back-bib181" data-db-target-for="bib181-d10391928e5142" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib181">Vats et al., 2020</a><div data-db-target-of="bib181-d10391928e5142" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vats P. </li><li class="loa__item">Chinnaiyan A.M. </li><li class="loa__item">Kumar-Sinha C. </li></ul><div class="reference-citations__detials">Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit.<div class="popup-ref"><span class="ref__series"><i>Methods Mol. Biol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">2079</span></span>: <span class="ref__seriesPages">69-79</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1007%2F978-1-4939-9904-0_5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31728962&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85075114327&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2079&publication_year=2020&pages=69-79&journal=Methods+Mol.+Biol.&author=Vats+P.&author=Chinnaiyan+A.M.&author=Kumar-Sinha+C.&title=Case+Study%3A+Systematic+Detection+and+Prioritization+of+Gene+Fusions+in+Cancer+by+RNA-Seq%3A+A+DIY+Toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.4.8" data-locator="sec4.5.4.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA and Protein correlation</span></h4><div class="section-paragraph">To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient and p value were recorded. Further, adjusted p values were calculated using the Benjamini–Hochberg procedure.</div></section><section id="sec4.5.4.9" data-locator="sec4.5.4.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CNA-driven <em>Cis</em> and Trans effects</span></h4><div class="section-paragraph">Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR &lt; 0.05) positive or negative correlations.</div></section><section id="sec4.5.4.10" data-locator="sec4.5.4.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CMAP analysis</span></h4><div class="section-paragraph">Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP (<div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d10391928e5162" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Lamb et al., 2006</a><div data-db-target-of="bib92-d10391928e5162" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lamb J. </li><li class="loa__item">Crawford E.D. </li><li class="loa__item">Peck D. </li><li class="loa__item">Modell J.W. </li><li class="loa__item">Blat I.C. </li><li class="loa__item">Wrobel M.J. </li><li class="loa__item">Lerner J. </li><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Subramanian A. </li><li class="loa__item">Ross K.N. </li><li>et al.</li></ul><div class="reference-citations__detials">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2951)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e5165" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e5165" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules (∼20,000 drug perturbagens), shRNA gene knockdowns (∼4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See <a class="inter-ref-link" href="https://clue.io/GEO-guide" target="_blank">https://clue.io/GEO-guide</a> for more information.</div><div class="section-paragraph">To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy number values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA deletion), the <em>trans</em> genes (identified by significant correlation in ‘‘CNA driven <em>cis</em> and <em>trans</em> effects’’ above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d10391928e5179" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Barbie et al., 2009</a><div data-db-target-of="bib12-d10391928e5179" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See (<div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e5182" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e5182" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically <em>cis</em>-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive <em>cis</em>-correlation.</div><div class="section-paragraph">For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant <em>trans</em> genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were identified with Fisher’s test FDR &lt; 0.25, using this process.</div><div class="section-paragraph">In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (<div class="dropBlock reference-citations"><a href="#bib217" id="back-bib217" data-db-target-for="bib217-d10391928e5196" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib217">Mertins et al., 2016</a><div data-db-target-of="bib217-d10391928e5196" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (717)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=Mertins+P.&author=Mani+D.R.&author=Ruggles+K.V.&author=Gillette+M.A.&author=Clauser+K.R.&author=Wang+P.&author=Wang+X.&author=Qiao+J.W.&author=Cao+S.&author=Petralia+F.&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with rate l, counting the number of identified candidate driver genes. Given the small n ( = 10) and l, a Score confidence interval was calculated (<div class="dropBlock reference-citations"><a href="#bib212" id="back-bib212" data-db-target-for="bib212-d10391928e5199" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib212">Barker, 2002</a><div data-db-target-of="bib212-d10391928e5199" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barker L. </li></ul><div class="reference-citations__detials">A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5.<div class="popup-ref"><span class="ref__series"><i>Am. Stat.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">56</span></span>: <span class="ref__seriesPages">85-89</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1198%2F000313002317572736&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0035995688&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2002&pages=85-89&journal=Am.+Stat.&author=Barker+L.&title=A+comparison+of+nine+confidence+intervals+for+a+Poisson+parameter+when+the+expected+number+of+events+is+%E2%89%A4+5">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR = 0.06 (95% CI: 0.01, 0.12) for six (6) genes with <em>both</em> CNA amplification and deletion.</div><div class="section-paragraph">To identify how many <em>trans</em>-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with a = 0.05, followed by counting the number of trans-genes in this list of regulated genes.</div><div class="section-paragraph">To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on <em>cis</em>/<em>trans</em>-correlation values to find enriched pathways.</div></section><section id="sec4.5.4.11" data-locator="sec4.5.4.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LINCS analysis</span></h4><div class="section-paragraph">Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p value &lt; 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis.</div><div class="section-paragraph">Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p values were adjusted using the Benjamini &amp; Hochberg method. Subtype-specific differentially expressed proteins were identified based on their differential expression (adjusted p value &lt; 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline (<a class="inter-ref-link" href="https://hub.docker.com/u/cmap" target="_blank">https://hub.docker.com/u/cmap</a>) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (<a class="inter-ref-link" href="https://clue.io" target="_blank">https://clue.io</a>, “Expanded CMap LINCS Resource 2020 Release”). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described (<div class="dropBlock reference-citations"><a href="#bib166" id="back-bib166" data-db-target-for="bib166-d10391928e5228" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib166">Subramanian et al., 2017</a><div data-db-target-of="bib166-d10391928e5228" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The corresponding compound metadata were obtained from CLUE (clue.io, “Expanded CMap LINCS Resource 2020 Release”) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization.</div></section><section id="sec4.5.4.12" data-locator="sec4.5.4.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Defining cancer-associated genes</span></h4><div class="section-paragraph">Cancer-associated genes (CAG) were compiled using the Census website <a class="inter-ref-link" href="https://cancer.sanger.ac.uk/census" target="_blank">https://cancer.sanger.ac.uk/census</a>. The list of genes is provided in <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Table S1</a>.</div></section><section id="sec4.5.4.13" data-locator="sec4.5.4.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CpG island methylator phenotype</span></h4><div class="section-paragraph">The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5' UTR regions of the gene. Then, we preprocessed the data by filtering genes that were hypermethylated, i.e., the gene-level methylation score &gt; 0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value −0.041 and −0.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups.</div></section><section id="sec4.5.4.14" data-locator="sec4.5.4.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">iProFun based <em>cis</em> association analysis</span></h4><div class="section-paragraph">We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their <em>cis</em> regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2ˆ5 = 32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR)&lt;10%. As DNA methylation at the promoter regions of a gene downregulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Second, a significance was called only if the posterior probabilities &gt; 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) &lt; 10% by selecting a minimal cutoff value of alpha such that 75%&lt;alpha &lt; 100%. The eFDR is calculated by: eFDR = (Averaged No. of genes with posterior probabilities &gt; alpha in permuted data) / (Averaged No. of genes with posterior probabilities &gt; alpha in original data). <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>G annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated.</div></section><section id="sec4.5.4.15" data-locator="sec4.5.4.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">EMT-derived cluster and fibroblast enrichment</span></h4><div class="section-paragraph">NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigDB (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigDB hallmark genesets v6.1).</div></section><section id="sec4.5.4.16" data-locator="sec4.5.4.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differential marker analysis</span></h4><div class="section-paragraph">A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having &lt; 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR (<div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d10391928e5269" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Liao et al., 2019</a><div data-db-target-of="bib105-d10391928e5269" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (429)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The selected gene list was either significant proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted p value &lt; 0.01.</div><div class="section-paragraph">Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, <a class="inter-ref-link" href="https://www.proteinatlas.org" target="_blank">https://www.proteinatlas.org</a>), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)&gt; 2 in tumors and Benjamini-Hochberg FDR &lt; 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot.</div></section><section id="sec4.5.4.17" data-locator="sec4.5.4.17" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Overall and disease-free survival in TCGA lung squamous carcinoma</span></h4><div class="section-paragraph">Survival for lung squamous tumor patients in TCGA LUSC (refered to as LSCC in our study) cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group.</div></section><section id="sec4.5.4.18" data-locator="sec4.5.4.18" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kaplan-Meier survival analysis</span></h4><div class="section-paragraph">We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy.</div><div class="section-paragraph">We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors.</div></section><section id="sec4.5.4.19" data-locator="sec4.5.4.19" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Continuous smoking score</span></h4><div class="section-paragraph">Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC (<div class="dropBlock reference-citations"><a href="#bib169" id="back-bib169" data-db-target-for="bib169-d10391928e5294" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib169">Tate et al., 2019</a><div data-db-target-of="bib169-d10391928e5294" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tate J.G. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Jubb H.C. </li><li class="loa__item">Sondka Z. </li><li class="loa__item">Beare D.M. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Creatore C. </li><li class="loa__item">Dawson E. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: the Catalogue Of Somatic Mutations In Cancer.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (858)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and cosine similarity was calculated to identify the best match.</div><div class="section-paragraph">We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 &lt; S &lt; 1, to quantify the degree of confidence that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C Ç T Ç N, and used A and A’ to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A’ and D to the joint probability of A and D. For example, the latter can be written P(A)・P(C|A)・P(T|A)・P(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e., P(A) = 0.9 (<div class="dropBlock reference-citations"><a href="#bib184" id="back-bib184" data-db-target-for="bib184-d10391928e5299" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib184">Walser et al., 2008</a><div data-db-target-of="bib184-d10391928e5299" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Walser T. </li><li class="loa__item">Cui X. </li><li class="loa__item">Yanagawa J. </li><li class="loa__item">Lee J.M. </li><li class="loa__item">Heinrich E. </li><li class="loa__item">Lee G. </li><li class="loa__item">Sharma S. </li><li class="loa__item">Dubinett S.M. </li></ul><div class="reference-citations__detials">Smoking and lung cancer: the role of inflammation.<div class="popup-ref"><span class="ref__series"><i>Proc. Am. Thorac. Soc.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">811-815</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1513%2Fpats.200809-100TH&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19017734&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-57349193494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (129)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=811-815&journal=Proc.+Am.+Thorac.+Soc.&author=Walser+T.&author=Cui+X.&author=Yanagawa+J.&author=Lee+J.M.&author=Heinrich+E.&author=Lee+G.&author=Sharma+S.&author=Dubinett+S.M.&title=Smoking+and+lung+cancer%3A+the+role+of+inflammation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">We consider <em>S</em> to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type <em>G</em> is P(<em>G</em>|<em>A</em>) = [<em>x</em>・<em>erf</em> (<em>g/y</em>) + 1] <em>/</em> (<em>x</em> + 2) and P(<em>G</em>|<em>A</em>’) = 1 - P(<em>G</em>|<em>A</em>), where <em>erf</em> is the Gaussian error function, <em>g</em> is the observed value for the given data type, and <em>x</em> and <em>y</em> are empirically-determined weights.</div><div class="section-paragraph">The shape function for total mutations, <em>T</em>, has two modifiers. First, it includes a simple expected-value correction for purity, <em>u</em>. Namely, assuming mutation-calling does not capture all mutations because of impurities, <em>t</em> is taken as the observed number of mutations divided by a purity shape function, <em>f</em>, where <em>f</em> ≤ 1. Although one might model <em>f</em> according to common characteristics of mutation callers, e.g., close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we use a line function, <em>f</em> = 0.2・<em>u</em> + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce’s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score &gt; 0.6; samples with weak smoking evidence if smoking score &lt; 0.15.</div></section><section id="sec4.5.4.20" data-locator="sec4.5.4.20" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Identification of differentially regulated events in NRF2 mutant tumors</span></h4><div class="section-paragraph">For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and p value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for multi-test, p values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4" class="scroll-into-link">Table S4</a>) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA (<a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a>). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p value &lt; 0.05 and logFC &gt; 1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 (<div class="dropBlock reference-citations"><a href="#bib114" id="back-bib114" data-db-target-for="bib114-d10391928e5370" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Malhotra et al., 2010</a><div data-db-target-of="bib114-d10391928e5370" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malhotra D. </li><li class="loa__item">Portales-Casamar E. </li><li class="loa__item">Singh A. </li><li class="loa__item">Srivastava S. </li><li class="loa__item">Arenillas D. </li><li class="loa__item">Happel C. </li><li class="loa__item">Shyr C. </li><li class="loa__item">Wakabayashi N. </li><li class="loa__item">Kensler T.W. </li><li class="loa__item">Wasserman W.W. </li><li class="loa__item">Biswal S. </li></ul><div class="reference-citations__detials">Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">5718-5734</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkq212&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20460467&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77957237159&amp;cf=" onclick="newWindow(this.href);return false">Scopus (470)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=5718-5734&journal=Nucleic+Acids+Res.&author=Malhotra+D.&author=Portales-Casamar+E.&author=Singh+A.&author=Srivastava+S.&author=Arenillas+D.&author=Happel+C.&author=Shyr+C.&author=Wakabayashi+N.&author=Kensler+T.W.&author=Wasserman+W.W.&author=Biswal+S.&title=Global+mapping+of+binding+sites+for+Nrf2+identifies+novel+targets+in+cell+survival+response+through+ChIP-Seq+profiling+and+network+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and SINGH_NFE2L2_TARGETS ((<div class="dropBlock reference-citations"><a href="#bib158" id="back-bib158" data-db-target-for="bib158-d10391928e5373" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib158">Singh et al., 2008</a><div data-db-target-of="bib158-d10391928e5373" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Singh A. </li><li class="loa__item">Boldin-Adamsky S. </li><li class="loa__item">Thimmulappa R.K. </li><li class="loa__item">Rath S.K. </li><li class="loa__item">Ashush H. </li><li class="loa__item">Coulter J. </li><li class="loa__item">Blackford A. </li><li class="loa__item">Goodman S.N. </li><li class="loa__item">Bunz F. </li><li class="loa__item">Watson W.H. </li><li>et al.</li></ul><div class="reference-citations__detials">RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7975-7984</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-08-1401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18829555&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-54249087596&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=7975-7984&journal=Cancer+Res.&author=Singh+A.&author=Boldin-Adamsky+S.&author=Thimmulappa+R.K.&author=Rath+S.K.&author=Ashush+H.&author=Coulter+J.&author=Blackford+A.&author=Goodman+S.N.&author=Bunz+F.&author=Watson+W.H.&title=RNAi-mediated+silencing+of+nuclear+factor+erythroid-2-related+factor+2+gene+expression+in+non-small+cell+lung+cancer+inhibits+tumor+growth+and+increases+efficacy+of+chemotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al. (<div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d10391928e5376" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Lacher and Slattery, 2016</a><div data-db-target-of="bib91-d10391928e5376" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lacher S.E. </li><li class="loa__item">Slattery M. </li></ul><div class="reference-citations__detials">Gene regulatory effects of disease-associated variation in the NRF2 network.<div class="popup-ref"><span class="ref__series"><i>Curr. Opin. Toxicol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">71-79</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cotox.2016.09.001&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28203648&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016740108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=71-79&journal=Curr.+Opin.+Toxicol.&author=Lacher+S.E.&author=Slattery+M.&title=Gene+regulatory+effects+of+disease-associated+variation+in+the+NRF2+network">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Ten of the 54 genes were novel targets identified in this study.</div></section><section id="sec4.5.4.21" data-locator="sec4.5.4.21" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune cluster identification based on cell type composition</span></h4><div class="section-paragraph">The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d10391928e5384" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Aran et al., 2017</a><div data-db-target-of="bib6-d10391928e5384" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) based on 108 tumors and 94 NAT samples. <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a> contains the final score computed by xCell of different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR &lt; 5%). This filtering resulted in 25 cell types annotated in <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>. Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R package ConsensusClusterPlus (<div class="dropBlock reference-citations"><a href="#bib195" id="back-bib195" data-db-target-for="bib195-d10391928e5393" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib195">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib195-d10391928e5393" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (710)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times (<div class="dropBlock reference-citations"><a href="#bib195" id="back-bib195" data-db-target-for="bib195-d10391928e5396" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib195">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib195-d10391928e5396" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (710)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A, <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>).</div></section><section id="sec4.5.4.22" data-locator="sec4.5.4.22" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TCGA pan-cancer immune subtyping</span></h4><div class="section-paragraph">Tumors were classified into the six distinct immune subtypes — wound healing, IFN-g dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-b dominant — identified by a TCGA pan-cancer analysis presented in (<div class="dropBlock reference-citations"><a href="#bib172" id="back-bib172" data-db-target-for="bib172-d10391928e5412" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib172">Thorsson et al., 2018</a><div data-db-target-of="bib172-d10391928e5412" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Thorsson V. </li><li class="loa__item">Gibbs D.L. </li><li class="loa__item">Brown S.D. </li><li class="loa__item">Wolf D. </li><li class="loa__item">Bortone D.S. </li><li class="loa__item">Ou Yang T.-H. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Gao G.F. </li><li class="loa__item">Plaisier C.L. </li><li class="loa__item">Eddy J.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />The Immune Landscape of Cancer.<div class="popup-ref"><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">812-830.e14</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29628290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044934017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1025)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=812-830.e14&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)<span class="underline">.</span> Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package (<a class="inter-ref-link" href="https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier" target="_blank">https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier</a>) (<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d10391928e5420" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Gibbs, 2020</a><div data-db-target-of="bib50-d10391928e5420" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbs D.L. </li></ul><div class="reference-citations__detials">Robust classification of Immune Subtypes in Cancer.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.01.17.910950">https://doi.org/10.1101/2020.01.17.910950</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.01.17.910950&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85095499730&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Gibbs+D.L.&title=Robust+classification+of+Immune+Subtypes+in+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to assign an immune subtype to each of the tumors.</div></section><section id="sec4.5.4.23" data-locator="sec4.5.4.23" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Ranking tumors by inferred activity of IFN-g pathway</span></h4><div class="section-paragraph">We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top (<div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d10391928e5428" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Reva et al., 2020</a><div data-db-target-of="bib146-d10391928e5428" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reva B. </li><li class="loa__item">Omelchenko T. </li><li class="loa__item">Calinawan A. </li><li class="loa__item">Nair S. </li><li class="loa__item">Schadt E. </li><li class="loa__item">Tewari A. </li></ul><div class="reference-citations__detials">Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-g axis and identification of immune resistance mechanisms.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.10.19.345629">https://doi.org/10.1101/2020.10.19.345629</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.10.19.345629&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85098898555&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Reva+B.&author=Omelchenko+T.&author=Calinawan+A.&author=Nair+S.&author=Schadt+E.&author=Tewari+A.&title=Prioritization+prostate+cancer+to+immune+checkpoint+therapy+by+ranking+tumors+along+IFN-g+axis+and+identification+of+immune+resistance+mechanisms">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above “top” and “bottom” probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the “top” and “bottom” probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher’s exact test, given protein’s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor (<div class="dropBlock reference-citations"><a href="#bib146" id="back-bib146" data-db-target-for="bib146-d10391928e5431" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib146">Reva et al., 2020</a><div data-db-target-of="bib146-d10391928e5431" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Reva B. </li><li class="loa__item">Omelchenko T. </li><li class="loa__item">Calinawan A. </li><li class="loa__item">Nair S. </li><li class="loa__item">Schadt E. </li><li class="loa__item">Tewari A. </li></ul><div class="reference-citations__detials">Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-g axis and identification of immune resistance mechanisms.<div class="popup-ref"><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.10.19.345629">https://doi.org/10.1101/2020.10.19.345629</a></span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.10.19.345629&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85098898555&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Reva+B.&author=Omelchenko+T.&author=Calinawan+A.&author=Nair+S.&author=Schadt+E.&author=Tewari+A.&title=Prioritization+prostate+cancer+to+immune+checkpoint+therapy+by+ranking+tumors+along+IFN-g+axis+and+identification+of+immune+resistance+mechanisms">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The thermodynamic interpretation of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway’s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway’s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted toward top or bottom.</div><div class="section-paragraph">In our analysis, we used a gene list of IFN-g signaling pathway from (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d10391928e5436" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">Abril-Rodriguez and Ribas, 2017</a><div data-db-target-of="bib1-d10391928e5436" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Abril-Rodriguez G. </li><li class="loa__item">Ribas A. </li></ul><div class="reference-citations__detials">SnapShot: Immune Checkpoint Inhibitors.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">848-848.e1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2017.05.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2017.05.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28609660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85020645133&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=848-848.e1&journal=Cancer+Cell&author=Abril-Rodriguez+G.&author=Ribas+A.&title=SnapShot%3A+Immune+Checkpoint+Inhibitors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We settled with this 15 genes signaling pathway because it has all necessary components – a cytokine mediator, receptors, signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells.</div></section><section id="sec4.5.4.24" data-locator="sec4.5.4.24" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Estimation of tumor purity, stromal and immune scores</span></h4><div class="section-paragraph">Besides xCell, we utilized ESTIMATE (<div class="dropBlock reference-citations"><a href="#bib204" id="back-bib204" data-db-target-for="bib204-d10391928e5444" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib204">Yoshihara et al., 2013</a><div data-db-target-of="bib204-d10391928e5444" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1382)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to infer immune and stromal scores based on gene expression data (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app (<a class="inter-ref-link" href="https://cibersort.stanford.edu" target="_blank">https://cibersort.stanford.edu</a>; (<div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d10391928e5453" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">Newman et al., 2015</a><div data-db-target-of="bib126-d10391928e5453" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Green M.R. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Feng W. </li><li class="loa__item">Xu Y. </li><li class="loa__item">Hoang C.D. </li><li class="loa__item">Diehn M. </li><li class="loa__item">Alizadeh A.A. </li></ul><div class="reference-citations__detials">Robust enumeration of cell subsets from tissue expression profiles.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2201)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)) in absolute mode. To infer tumor purity, TSNet was utilized (<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d10391928e5456" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Petralia et al., 2018</a><div data-db-target-of="bib140-d10391928e5456" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng J. </li><li class="loa__item">Yan A. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>).</div></section><section id="sec4.5.4.25" data-locator="sec4.5.4.25" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differentially expressed genes and pathway analysis</span></h4><div class="section-paragraph">Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p value lower than 10%, a Fisher's exact test was implemented to derive enriched pathways. ssGSEA (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d10391928e5468" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Barbie et al., 2009</a><div data-db-target-of="bib12-d10391928e5468" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA (<div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d10391928e5471" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Hänzelmann et al., 2013</a><div data-db-target-of="bib55-d10391928e5471" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1428)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For this analysis, pathways from the Reactome (<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d10391928e5474" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Fabregat et al., 2018</a><div data-db-target-of="bib33-d10391928e5474" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fabregat A. </li><li class="loa__item">Jupe S. </li><li class="loa__item">Matthews L. </li><li class="loa__item">Sidiropoulos K. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Garapati P. </li><li class="loa__item">Haw R. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Korninger F. </li><li class="loa__item">May B. </li><li>et al.</li></ul><div class="reference-citations__detials">The Reactome Pathway Knowledgebase.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1055)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+Reactome+Pathway+Knowledgebase">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), KEGG (<div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d10391928e5477" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Kanehisa et al., 2017</a><div data-db-target-of="bib75-d10391928e5477" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kanehisa M. </li><li class="loa__item">Furumichi M. </li><li class="loa__item">Tanabe M. </li><li class="loa__item">Sato Y. </li><li class="loa__item">Morishima K. </li></ul><div class="reference-citations__detials">KEGG: new perspectives on genomes, pathways, diseases and drugs.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D353-D361</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkw1092&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27899662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016149190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2897)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D353-D361&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Furumichi+M.&author=Tanabe+M.&author=Sato+Y.&author=Morishima+K.&title=KEGG%3A+new+perspectives+on+genomes%2C+pathways%2C+diseases+and+drugs">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and Hallmark (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d10391928e5480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Liberzon et al., 2015</a><div data-db-target-of="bib106-d10391928e5480" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1936)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in <a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6" class="scroll-into-link">Table S6</a>.</div></section><section id="sec4.5.4.26" data-locator="sec4.5.4.26" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Deriving RTK CBPE scores</span></h4><div class="section-paragraph">A total of 42 human RTKs were present in our proteomics dataset. For each phosphosite in our dataset we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNaseq data via TSNet (<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d10391928e5492" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Petralia et al., 2018</a><div data-db-target-of="bib140-d10391928e5492" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petralia F. </li><li class="loa__item">Wang L. </li><li class="loa__item">Peng J. </li><li class="loa__item">Yan A. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Each association computation was carried out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor dataset, and seven in NAT.</div></section><section id="sec4.5.4.27" data-locator="sec4.5.4.27" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Independent component analysis</span></h4><div class="section-paragraph">ICA was performed with a workflow modified from previously described (<div class="dropBlock reference-citations"><a href="#bib111" id="back-bib111" data-db-target-for="bib111-d10391928e5500" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Liu et al., 2019</a><div data-db-target-of="bib111-d10391928e5500" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu W. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Ma S. </li><li class="loa__item">Fenyö D. </li></ul><div class="reference-citations__detials">Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S169-S182</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31213479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071350104&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S169-S182&journal=Mol.+Cell.+Proteomics&author=Liu+W.&author=Payne+S.H.&author=Ma+S.&author=Feny%C3%B6+D.&title=Extracting+Pathway-level+Signatures+from+Proteogenomic+Data+in+Breast+Cancer+Using+Independent+Component+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n = 99). Independent components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm (<div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d10391928e5503" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">McInnes et al., 2017</a><div data-db-target-of="bib116-d10391928e5503" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McInnes L. </li><li class="loa__item">Healy J. </li><li class="loa__item">Astels S. </li></ul><div class="reference-citations__detials">hdbscan: Hierarchical density based clustering.<div class="popup-ref"><span class="ref__series"><i>Journal of Open Source Software.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">205</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.21105%2Fjoss.00205&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=205&journal=Journal+of+Open+Source+Software&author=McInnes+L.&author=Healy+J.&author=Astels+S.&title=hdbscan%3A+Hierarchical+density+based+clustering">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with cosine distance as dissimilarity metric, min_cluster_size = 50 and min_samples = 20. The centroids of clusters (n = 37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 p values. Significance was controlled for multiple testing at 0.01 level (log10 (<em>p value)</em> = −5.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations.</div></section><section id="sec4.5.4.28" data-locator="sec4.5.4.28" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutation-based cis- and trans-effects</span></h4><div class="section-paragraph">We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d10391928e5513" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Bailey et al., 2018</a><div data-db-target-of="bib10-d10391928e5513" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bailey M.H. </li><li class="loa__item">Tokheim C. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Sengupta S. </li><li class="loa__item">Bertrand D. </li><li class="loa__item">Weerasinghe A. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Wendl M.C. </li><li class="loa__item">Kim J. </li><li class="loa__item">Reardon B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>MC3 Working Group</span><span>Cancer Genome Atlas Research Network</span><br />Comprehensive Characterization of Cancer Driver Genes and Mutations.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">371-385.e18</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29625053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044623848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (581)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=371-385.e18&journal=Cell&author=MC3+Working+GroupCancer+Genome+Atlas+Research+Network&title=Comprehensive+Characterization+of+Cancer+Driver+Genes+and+Mutations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR &lt; 0.05 cut-off were separated into two categories based on <em>cis</em>- and <em>trans</em>- effects.</div></section><section id="sec4.5.4.29" data-locator="sec4.5.4.29" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline quantitative trait loci (QTL) analysis</span></h4><div class="section-paragraph">To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at <a class="inter-ref-link" href="https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen" target="_blank">https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen</a>). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count <sup>3</sup> 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio aggregated to gene-level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9 (<div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d10391928e5531" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Deelen et al., 2014</a><div data-db-target-of="bib27-d10391928e5531" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Deelen P. </li><li class="loa__item">Bonder M.J. </li><li class="loa__item">van der Velde K.J. </li><li class="loa__item">Westra H.-J. </li><li class="loa__item">Winder E. </li><li class="loa__item">Hendriksen D. </li><li class="loa__item">Franke L. </li><li class="loa__item">Swertz M.A. </li></ul><div class="reference-citations__detials">Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration.<div class="popup-ref"><span class="ref__series"><i>BMC Res. Notes.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">901</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1756-0500-7-901&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25495213&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928813777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (40)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=901&journal=BMC+Res.+Notes&author=Deelen+P.&author=Bonder+M.J.&author=van+der+Velde+K.J.&author=Westra+H.-J.&author=Winder+E.&author=Hendriksen+D.&author=Franke+L.&author=Swertz+M.A.&title=Genotype+harmonizer%3A+automatic+strand+alignment+and+format+conversion+for+genotype+data+integration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To adjust for population stratification, we identified the multi-dimensional scaling (MDS) components using the genotype data using PLINK v1.07 (<div class="dropBlock reference-citations"><a href="#bib141" id="back-bib141" data-db-target-for="bib141-d10391928e5534" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Purcell et al., 2007</a><div data-db-target-of="bib141-d10391928e5534" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Purcell S. </li><li class="loa__item">Neale B. </li><li class="loa__item">Todd-Brown K. </li><li class="loa__item">Thomas L. </li><li class="loa__item">Ferreira M.A.R. </li><li class="loa__item">Bender D. </li><li class="loa__item">Maller J. </li><li class="loa__item">Sklar P. </li><li class="loa__item">de Bakker P.I.W. </li><li class="loa__item">Daly M.J. </li><li class="loa__item">Sham P.C. </li></ul><div class="reference-citations__detials">PLINK: a tool set for whole-genome association and population-based linkage analyses.<div class="popup-ref"><span class="ref__series"><i>Am. J. Hum. Genet.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">559-575</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17701901&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-34548292504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16449)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=559-575&journal=Am.+J.+Hum.+Genet.&author=Purcell+S.&author=Neale+B.&author=Todd-Brown+K.&author=Thomas+L.&author=Ferreira+M.A.R.&author=Bender+D.&author=Maller+J.&author=Sklar+P.&author=de+Bakker+P.I.W.&author=Daly+M.J.&author=Sham+P.C.&title=PLINK%3A+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE &gt; 0.0001, MAF &gt; 0.01, and call rate &gt; 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at £ FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis (<div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d10391928e5537" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Krämer et al., 2014</a><div data-db-target-of="bib85-d10391928e5537" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krämer A. </li><li class="loa__item">Green J. </li><li class="loa__item">Pollard Jr., J. </li><li class="loa__item">Tugendreich S. </li></ul><div class="reference-citations__detials">Causal analysis approaches in Ingenuity Pathway Analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">523-530</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtt703&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24336805&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84894571860&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1880)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=523-530&journal=Bioinformatics&author=Kr%C3%A4mer+A.&author=Green+J.&author=Pollard+J.&author=Tugendreich+S.&title=Causal+analysis+approaches+in+Ingenuity+Pathway+Analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</div></section><section id="sec4.5.4.30" data-locator="sec4.5.4.30" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA analysis presented in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a></span></h4><div class="section-paragraph">Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained (<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d10391928e5547" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Huang et al., 2020</a><div data-db-target-of="bib65-d10391928e5547" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang H.-Y. </li><li class="loa__item">Lin Y.-C.-D. </li><li class="loa__item">Li J. </li><li class="loa__item">Huang K.-Y. </li><li class="loa__item">Shrestha S. </li><li class="loa__item">Hong H.-C. </li><li class="loa__item">Tang Y. </li><li class="loa__item">Chen Y.-G. </li><li class="loa__item">Jin C.-N. </li><li class="loa__item">Yu Y. </li><li>et al.</li></ul><div class="reference-citations__detials">miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D148-D154</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31647101&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D148-D154&journal=Nucleic+Acids+Res.&author=Huang+H.-Y.&author=Lin+Y.-C.-D.&author=Li+J.&author=Huang+K.-Y.&author=Shrestha+S.&author=Hong+H.-C.&author=Tang+Y.&author=Chen+Y.-G.&author=Jin+C.-N.&author=Yu+Y.&title=miRTarBase+2020%3A+updates+to+the+experimentally+validated+microRNA-target+interaction+database">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Spearman correlation was used to calculate the correlations between miRNA and its target genes.</div></section><section id="sec4.5.4.31" data-locator="sec4.5.4.31" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Pathway projection using ssGSEA</span></h4><div class="section-paragraph">The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on</div><div class="section-paragraph"><a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a> was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters:<ul class="ce-list--remove-bullets" id="ulist0040"><li class="ce-list--remove-bullets__list-item" id="u0150"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">gene.set.database = “h.all.v6.2.symbols.gmt”</div></li><li class="ce-list--remove-bullets__list-item" id="u0155"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">sample.norm.type = “rank”</div></li><li class="ce-list--remove-bullets__list-item" id="u0160"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">weight = 0.75</div></li><li class="ce-list--remove-bullets__list-item" id="u0165"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">statistic = ”area.under.RES”</div></li><li class="ce-list--remove-bullets__list-item" id="u0170"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">output.score.type = “NES”</div></li><li class="ce-list--remove-bullets__list-item" id="u0175"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">nperm = 1000</div></li><li class="ce-list--remove-bullets__list-item" id="u0180"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">global.fdr = TRUE</div></li><li class="ce-list--remove-bullets__list-item" id="u0185"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">min.overlap = 10</div></li><li class="ce-list--remove-bullets__list-item" id="u0190"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">correl.type = ”z.score”</div></li></ul></div></section><section id="sec4.5.4.32" data-locator="sec4.5.4.32" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphorylation-driven signature analysis</span></h4><div class="section-paragraph">We performed phosphosite-specific signature enrichment analysis (PTM-SEA) (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d10391928e5601" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Krug et al., 2019</a><div data-db-target-of="bib87-d10391928e5601" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A curated resource for phosphosite-specific signature analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p values were log-transformed and signed according to the fold change (signed -log10 (p value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>I)</div><div class="section-paragraph">logPsite = -log10(p value)<sup>∗</sup>sign(log2(fold change))</div><div class="section-paragraph">PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software.</div><div class="section-paragraph">We employed the heuristic method introduced by Krug et al. (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d10391928e5616" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Krug et al., 2019</a><div data-db-target-of="bib87-d10391928e5616" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Krug K. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Hornbeck P. </li><li class="loa__item">Raju R. </li><li class="loa__item">Ahmad R. </li><li class="loa__item">Szucs M. </li><li class="loa__item">Mundt F. </li><li class="loa__item">Forestier D. </li><li class="loa__item">Jane-Valbuena J. </li><li>et al.</li></ul><div class="reference-citations__detials">A curated resource for phosphosite-specific signature analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved by using the p value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p value derived from the mono-phosphorylated peptide proteoform to T4, and the p value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p value to both sites T4 and S8.</div><div class="section-paragraph">We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb" target="_blank">https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb</a> using the flanking amino acid sequence (+/− 7 aa) as primary identifier. We used the implementation of PTM-SEA available on GitHub (<a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a>) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA:</div><div class="section-paragraph">weight: 1</div><div class="section-paragraph">statistic: “area.under.RES”</div><div class="section-paragraph">output.score.type: “NES”</div><div class="section-paragraph">nperm: 1000</div><div class="section-paragraph">min.overlap: 5</div><div class="section-paragraph">correl.type: “z-score”</div><div class="section-paragraph">The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini &amp; and Hochberg (<div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d10391928e5642" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Benjamini and Hochberg, 1995</a><div data-db-target-of="bib14-d10391928e5642" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benjamini Y. </li><li class="loa__item">Hochberg Y. </li></ul><div class="reference-citations__detials">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.<div class="popup-ref"><span class="ref__series"><i>J.R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Signatures with FDR-corrected p values &lt; 0.05 were considered to be differential between tumor and NAT.</div></section><section id="sec4.5.4.33" data-locator="sec4.5.4.33" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text"><em>CDKN2A</em> and <em>RB1</em> annotations and pathway analysis</span></h4><div class="section-paragraph">Comprehensive tumor annotation for <em>CDKN2A</em> and <em>RB1</em> genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values &gt; = 0.2. Multi-gene proliferation scores were calculated as described previously (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d10391928e5658" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Ellis et al., 2017</a><div data-db-target-of="bib32-d10391928e5658" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ellis M.J. </li><li class="loa__item">Suman V.J. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Goncalves R. </li><li class="loa__item">Sanati S. </li><li class="loa__item">Creighton C.J. </li><li class="loa__item">DeSchryver K. </li><li class="loa__item">Crouch E. </li><li class="loa__item">Brink A. </li><li class="loa__item">Watson M. </li><li>et al.</li></ul><div class="reference-citations__detials">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).<div class="popup-ref"><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al. (<div class="dropBlock reference-citations"><a href="#bib194" id="back-bib194" data-db-target-for="bib194-d10391928e5661" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib194">Whitfield et al., 2002</a><div data-db-target-of="bib194-d10391928e5661" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Whitfield M.L. </li><li class="loa__item">Sherlock G. </li><li class="loa__item">Saldanha A.J. </li><li class="loa__item">Murray J.I. </li><li class="loa__item">Ball C.A. </li><li class="loa__item">Alexander K.E. </li><li class="loa__item">Matese J.C. </li><li class="loa__item">Perou C.M. </li><li class="loa__item">Hurt M.M. </li><li class="loa__item">Brown P.O. </li><li class="loa__item">Botstein D. </li></ul><div class="reference-citations__detials">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.<div class="popup-ref"><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1117)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq NP_000068.1 for p16INK4a and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations (<div class="dropBlock reference-citations"><a href="#bib211" id="back-bib211" data-db-target-for="bib211-d10391928e5664" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib211">Zhou et al., 2016</a><div data-db-target-of="bib211-d10391928e5664" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhou X. </li><li class="loa__item">Edmonson M.N. </li><li class="loa__item">Wilkinson M.R. </li><li class="loa__item">Patel A. </li><li class="loa__item">Wu G. </li><li class="loa__item">Liu Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Rusch M.C. </li><li class="loa__item">Parker M. </li><li>et al.</li></ul><div class="reference-citations__detials">Exploring genomic alteration in pediatric cancer using ProteinPaint.<div class="popup-ref"><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">4-6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3466&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26711108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84971602748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=4-6&journal=Nat.+Genet.&author=Zhou+X.&author=Edmonson+M.N.&author=Wilkinson+M.R.&author=Patel+A.&author=Wu+G.&author=Liu+Y.&author=Li+Y.&author=Zhang+Z.&author=Rusch+M.C.&author=Parker+M.&title=Exploring+genomic+alteration+in+pediatric+cancer+using+ProteinPaint">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations.</div></section><section id="sec4.5.4.34" data-locator="sec4.5.4.34" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Association analysis between KGG-site abundances and E3 ligases and DUBs</span></h4><div class="section-paragraph">A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from (<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d10391928e5672" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Medvar et al., 2016</a><div data-db-target-of="bib118-d10391928e5672" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Medvar B. </li><li class="loa__item">Raghuram V. </li><li class="loa__item">Pisitkun T. </li><li class="loa__item">Sarkar A. </li><li class="loa__item">Knepper M.A. </li></ul><div class="reference-citations__detials">Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation.<div class="popup-ref"><span class="ref__series"><i>Physiol. Genomics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">502-512</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1152%2Fphysiolgenomics.00031.2016&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27199454&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84983739108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=502-512&journal=Physiol.+Genomics&author=Medvar+B.&author=Raghuram+V.&author=Pisitkun+T.&author=Sarkar+A.&author=Knepper+M.A.&title=Comprehensive+database+of+human+E3+ubiquitin+ligases%3A+application+to+aquaporin-2+regulation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d10391928e5675" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Nijman et al., 2005</a><div data-db-target-of="bib128-d10391928e5675" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nijman S.M.B. </li><li class="loa__item">Luna-Vargas M.P.A. </li><li class="loa__item">Velds A. </li><li class="loa__item">Brummelkamp T.R. </li><li class="loa__item">Dirac A.M.G. </li><li class="loa__item">Sixma T.K. </li><li class="loa__item">Bernards R. </li></ul><div class="reference-citations__detials">A genomic and functional inventory of deubiquitinating enzymes.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">123</span></span>: <span class="ref__seriesPages">773-786</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16325574&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-28344456279&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1262)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=773-786&journal=Cell&author=Nijman+S.M.B.&author=Luna-Vargas+M.P.A.&author=Velds+A.&author=Brummelkamp+T.R.&author=Dirac+A.M.G.&author=Sixma+T.K.&author=Bernards+R.&title=A+genomic+and+functional+inventory+of+deubiquitinating+enzymes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We then fit a linear model using limma in R with the formula <em>kgg_site_abundance ∼protein_abundance</em>, followed by empirical bayes shrinkage. The coefficient p values were used to determine significant associations after FDR correction (FDR &lt; 0.01).</div></section><section id="sec4.5.4.35" data-locator="sec4.5.4.35" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster and pathway analysis of significantly modulated K-GG sites in tumors</span></h4><div class="section-paragraph">Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters.</div><div class="section-paragraph">Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler (<div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d10391928e5687" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Raudvere et al., 2019</a><div data-db-target-of="bib144-d10391928e5687" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Raudvere U. </li><li class="loa__item">Kolberg L. </li><li class="loa__item">Kuzmin I. </li><li class="loa__item">Arak T. </li><li class="loa__item">Adler P. </li><li class="loa__item">Peterson H. </li><li class="loa__item">Vilo J. </li></ul><div class="reference-citations__detials">g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update).<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W191-W198</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkz369&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31066453&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85068995138&amp;cf=" onclick="newWindow(this.href);return false">Scopus (585)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W191-W198&journal=Nucleic+Acids+Res.&author=Raudvere+U.&author=Kolberg+L.&author=Kuzmin+I.&author=Arak+T.&author=Adler+P.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler%3A+a+web+server+for+functional+enrichment+analysis+and+conversions+of+gene+lists+%282019+update%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Pathway enrichment was performed using a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape (<div class="dropBlock reference-citations"><a href="#bib154" id="back-bib154" data-db-target-for="bib154-d10391928e5690" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib154">Shannon et al., 2003</a><div data-db-target-of="bib154-d10391928e5690" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shannon P. </li><li class="loa__item">Markiel A. </li><li class="loa__item">Ozier O. </li><li class="loa__item">Baliga N.S. </li><li class="loa__item">Wang J.T. </li><li class="loa__item">Ramage D. </li><li class="loa__item">Amin N. </li><li class="loa__item">Schwikowski B. </li><li class="loa__item">Ideker T. </li></ul><div class="reference-citations__detials">Cytoscape: a software environment for integrated models of biomolecular interaction networks.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17796)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using the Enrichment Map app (<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d10391928e5693" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Merico et al., 2010</a><div data-db-target-of="bib119-d10391928e5693" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Merico D. </li><li class="loa__item">Isserlin R. </li><li class="loa__item">Stueker O. </li><li class="loa__item">Emili A. </li><li class="loa__item">Bader G.D. </li></ul><div class="reference-citations__detials">Enrichment map: a network-based method for gene-set enrichment visualization and interpretation.<div class="popup-ref"><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e13984</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1371%2Fjournal.pone.0013984&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21085593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78649775562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1020)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e13984&journal=PLoS+ONE&author=Merico+D.&author=Isserlin+R.&author=Stueker+O.&author=Emili+A.&author=Bader+G.D.&title=Enrichment+map%3A+a+network-based+method+for+gene-set+enrichment+visualization+and+interpretation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for network analysis of pathways. Pathways were connected using the gene overlap and clustered (pathway cutoff q-val &lt; 0.1; jaccard overlap &gt; 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation.</div></section><section id="sec4.5.4.36" data-locator="sec4.5.4.36" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PTM CLUMPS analysis</span></h4><div class="section-paragraph">We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative versus positive normalized signal. These are considered as individual “events” as done with mutations in CLUMPS v1 (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d10391928e5701" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Kamburov et al., 2015</a><div data-db-target-of="bib74-d10391928e5701" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kamburov A. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Polak P. </li><li class="loa__item">Leshchiner I. </li><li class="loa__item">Lage K. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Comprehensive assessment of cancer missense mutation clustering in protein structures.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">E5486-E5495</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1516373112&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26392535&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943411478&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E5486-E5495&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Kamburov+A.&author=Lawrence+M.S.&author=Polak+P.&author=Leshchiner+I.&author=Lage+K.&author=Golub+T.R.&author=Lander+E.S.&author=Getz+G.&title=Comprehensive+assessment+of+cancer+missense+mutation+clustering+in+protein+structures">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and Tumor samples (<div class="dropBlock reference-citations"><a href="#bib147" id="back-bib147" data-db-target-for="bib147-d10391928e5704" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib147">Ritchie et al., 2015</a><div data-db-target-of="bib147-d10391928e5704" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ritchie M.E. </li><li class="loa__item">Phipson B. </li><li class="loa__item">Wu D. </li><li class="loa__item">Hu Y. </li><li class="loa__item">Law C.W. </li><li class="loa__item">Shi W. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">limma powers differential expression analyses for RNA-sequencing and microarray studies.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9097)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Sites are selected with an FDR &lt; 0.1 (Benjamini-Hochberg) and LFC &gt; 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p value.</div></section><section id="sec4.5.4.37" data-locator="sec4.5.4.37" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">DepMap genetic dependency and drug response analysis</span></h4><div class="section-paragraph">Cell lines annotated as “NSCLC_squamous” from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used.</div><div class="section-paragraph">To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study’s LSCC cohort as described in STAR METHODS). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = −2, −1, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from DepMap.</div><div class="section-paragraph"><a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>: For identifying if top protein biomarkers (502 proteins significantly overexpressed (log<sub>2</sub>(FC) &gt; 2, FDR &lt; 0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines (<a class="inter-ref-link" href="https://depmap.org/portal/download/" target="_blank">https://depmap.org/portal/download/</a> file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes.</div><div class="section-paragraph"><a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>B: To investigate copy number and dependency associations in LSCC cell lines, each gene upregulated at the protein level with recurrent copy number gain in tumors versus NATs (from <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A with Log2FC &gt; 0 &amp; FDR &lt; 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified versus not-amplified. A lower mean change in dependency scores for a given gene in amplified versus not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified.</div><div class="section-paragraph">Continuous log2 copy number data from DepMap was used in <a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>M to correlate <em>SOX2</em> copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap).</div><div class="section-paragraph">Drug responses to YM-155 (PRISM secondary screen) were compared by t test in LSCC cell lines with low versus high levels of Np63 by transcript isoform expression (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">Figure S4</a>G). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) (<a data-link="modal" data-target="#image-S0092-8674(21)00857-6figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>D).</div></section><section id="sec4.5.4.38" data-locator="sec4.5.4.38" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">CausalPath analysis</span></h4><div class="section-paragraph">CausalPath (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d10391928e5754" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Babur et al., 2018</a><div data-db-target-of="bib8-d10391928e5754" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Babur Ö. </li><li class="loa__item">Luna A. </li><li class="loa__item">Korkut A. </li><li class="loa__item">Durupinar F. </li><li class="loa__item">Siper M.C. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Vaca Jacome A.S. </li><li class="loa__item">Peckner R. </li><li class="loa__item">Christianson K.E. </li><li class="loa__item">Jaffe J.D. </li><li>et al.</li></ul><div class="reference-citations__detials">Causal interactions from proteomic profiles: Molecular data meet pathway knowledge.<div class="popup-ref"><span class="ref__series"><i>Patterns.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">100257</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85106566488&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2018&pages=100257&journal=Patterns&author=Babur+%C3%96.&author=Luna+A.&author=Korkut+A.&author=Durupinar+F.&author=Siper+M.C.&author=Dogrusoz+U.&author=Vaca+Jacome+A.S.&author=Peckner+R.&author=Christianson+K.E.&author=Jaffe+J.D.&title=Causal+interactions+from+proteomic+profiles%3A+Molecular+data+meet+pathway+knowledge">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of <a data-link="modal" data-target="#image-S0092-8674(21)00857-6gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C, identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins).</div></section><section id="sec4.5.4.39" data-locator="sec4.5.4.39" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Variant peptide identification</span></h4><div class="section-paragraph">We used NeoFlow (<a class="inter-ref-link" href="https://github.com/bzhanglab/neoflow" target="_blank">https://github.com/bzhanglab/neoflow</a>) for neoantigen prediction (<div class="dropBlock reference-citations"><a href="#bib191" id="back-bib191" data-db-target-for="bib191-d10391928e5768" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib191">Wen et al., 2020</a><div data-db-target-of="bib191-d10391928e5768" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Specifically, Optitype (<div class="dropBlock reference-citations"><a href="#bib167" id="back-bib167" data-db-target-for="bib167-d10391928e5771" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib167">Szolek et al., 2014</a><div data-db-target-of="bib167-d10391928e5771" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan (<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d10391928e5774" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Jurtz et al., 2017</a><div data-db-target-of="bib73-d10391928e5774" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (413)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to predict HLA peptide binding affinity for somatic mutation–derived variant peptides with a length between 8-11 amino acids. The cutoff of IC<sub>50</sub> binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study (<div class="dropBlock reference-citations"><a href="#bib180" id="back-bib180" data-db-target-for="bib180-d10391928e5781" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib180">Vasaikar et al., 2019</a><div data-db-target-of="bib180-d10391928e5781" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (136)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To identify variant peptides, we used a customized protein sequence database approach (<div class="dropBlock reference-citations"><a href="#bib185" id="back-bib185" data-db-target-for="bib185-d10391928e5784" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib185">Wang et al., 2012</a><div data-db-target-of="bib185-d10391928e5784" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang I.-M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Yang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Stepaniants S. </li><li class="loa__item">Zhang C. </li><li class="loa__item">Meng Q. </li><li class="loa__item">Peters M. </li><li class="loa__item">He Y. </li><li class="loa__item">Ni C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers.<div class="popup-ref"><span class="ref__series"><i>Mol. Syst. Biol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fmsb.2012.24&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22806142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84864208657&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=594&journal=Mol.+Syst.+Biol.&author=Wang+I.-M.&author=Zhang+B.&author=Yang+X.&author=Zhu+J.&author=Stepaniants+S.&author=Zhang+C.&author=Meng+Q.&author=Peters+M.&author=He+Y.&author=Ni+C.&title=Systems+analysis+of+eleven+rodent+disease+models+reveals+an+inflammatome+signature+and+key+drivers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (<div class="dropBlock reference-citations"><a href="#bib191" id="back-bib191" data-db-target-for="bib191-d10391928e5787" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib191">Wen et al., 2020</a><div data-db-target-of="bib191-d10391928e5787" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li K. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a>) for customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using PepQuery (<a class="inter-ref-link" href="http://www.pepquery.org" target="_blank">http://www.pepquery.org</a>) (<div class="dropBlock reference-citations"><a href="#bib190" id="back-bib190" data-db-target-for="bib190-d10391928e5796" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib190">Wen et al., 2019</a><div data-db-target-of="bib190-d10391928e5796" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with the p value cutoff of 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV (<a class="inter-ref-link" href="http://pdv.zhang-lab.org" target="_blank">http://pdv.zhang-lab.org</a>) (<div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d10391928e5803" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Li et al., 2019b</a><div data-db-target-of="bib102-d10391928e5803" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li K. </li><li class="loa__item">Vaudel M. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Ren Y. </li><li class="loa__item">Wen B. </li></ul><div class="reference-citations__detials">PDV: an integrative proteomics data viewer.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec4.5.4.40" data-locator="sec4.5.4.40" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cancer/testis antigen prediction</span></h4><div class="section-paragraph">Cancer/testis (CT) antigens were downloaded from the CTdatabase (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d10391928e5811" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Almeida et al., 2009</a><div data-db-target-of="bib5-d10391928e5811" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.-T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). CT antigens with a median &gt; 4-fold increase in tumor from NAT were highlighted.</div></section><section id="sec4.5.4.41" data-locator="sec4.5.4.41" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PROGENy Scores</span></h4><div class="section-paragraph">PROGENy (<div class="dropBlock reference-citations"><a href="#bib151" id="back-bib151" data-db-target-for="bib151-d10391928e5819" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib151">Schubert et al., 2018</a><div data-db-target-of="bib151-d10391928e5819" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Schubert M. </li><li class="loa__item">Klinger B. </li><li class="loa__item">Klünemann M. </li><li class="loa__item">Sieber A. </li><li class="loa__item">Uhlitz F. </li><li class="loa__item">Sauer S. </li><li class="loa__item">Garnett M.J. </li><li class="loa__item">Blüthgen N. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">Perturbation-response genes reveal signaling footprints in cancer gene expression.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy.</div></section><section id="sec4.5.4.42" data-locator="sec4.5.4.42" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LSCC, HNSCC and LUAD integrative analysis</span></h4><div class="section-paragraph">LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript (<div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d10391928e5827" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Gillette et al., 2020</a><div data-db-target-of="bib51-d10391928e5827" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gillette M.A. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Cao S. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Krug K. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Li Y. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Reva B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript (<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d10391928e5830" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Huang et al., 2021</a><div data-db-target-of="bib66-d10391928e5830" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huang C. </li><li class="loa__item">Chen L. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Eguez R.V. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Li Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Jaehnig E.J. </li><li class="loa__item">Lei J.T. </li><li class="loa__item">Wen B. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>Clinical Proteomic Tumor Analysis Consortium</span><br />Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">361-379.e16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33417831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099708609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=361-379.e16&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+insights+into+the+biology+and+treatment+of+HPV-negative+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in the <span class="underline">Differential marker analysis</span> method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded.</div><div class="section-paragraph">Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value &lt; 0.25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value &lt; 0.01). Proteins were also required to be significantly increased in tumor versus paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p value &lt; 0.01).</div><div class="section-paragraph">To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method <span class="underline">Estimation of Tumor Purity, Stromal and Immune Scores</span>. To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA &amp; RNA, CNA &amp; immune score, and RNA &amp; immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA &amp; protein and protein &amp; immune score. Spearman correlations with a BH adjusted p value &lt; 0.01 were considered significant.</div></section></section></section><section id="sec4.6" data-locator="sec4.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Additional resources</span></h3><div class="section-paragraph">The CPTAC program website, detailing program initiatives, investigators, and datasets, is found at <a class="inter-ref-link" href="https://proteomics.cancer.gov/programs/cptac" target="_blank">https://proteomics.cancer.gov/programs/cptac</a>.</div><div class="section-paragraph">A website for interactive visualization of the multi-omic dataset is available at: <a class="inter-ref-link" href="https://proteomics.broadapps.org/CPTAC-LSCC2021/" target="_blank">https://proteomics.broadapps.org/CPTAC-LSCC2021/</a></div><div class="section-paragraph">All processed data matrices are available at LinkedOmics (<div class="dropBlock reference-citations"><a href="#bib179" id="back-bib179" data-db-target-for="bib179-d10391928e5855" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib179">Vasaikar et al., 2018</a><div data-db-target-of="bib179-d10391928e5855" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (415)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a class="inter-ref-link" href="http://www.linkedomics.org" target="_blank">http://www.linkedomics.org</a>), where computational tools are available for further exploration of this dataset.</div></section></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0565" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the National Cancer Institute Clinical Proteomic Tumor Analysis Consortium are Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Fenyö, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Małgorzata Wojtyś, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, Özgün Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, and Ziad Hanhan.</div></section><section class="data-availability"><h2 class="top" id="sectitle0030" data-left-hand-nav="Data and code availability" tabindex="0">Data and code availability</h2><div class="section-paragraph">Clinical Proteomic Tumor Analysis Consortium (CPTAC) LSCC proteomics data:<ul class="ce-list--remove-bullets" id="ulist0015"><li class="ce-list--remove-bullets__list-item" id="u0030"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Proteomic Data Commons (PDC); <a class="inter-ref-link" href="https://pdc.cancer.gov/pdc/" target="_blank">https://pdc.cancer.gov/pdc/</a> with identifiers PDC000232, PDC000233, PDC000234 and PDC000237</div></li><li class="ce-list--remove-bullets__list-item" id="u0035"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">CPTAC data portal LSCC: <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S063" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S063</a></div></li></ul></div><div class="section-paragraph">CPTAC head and neck squamous cell cancer (HNSCC) proteomics data:<ul class="ce-list--remove-bullets" id="ulist0020"><li class="ce-list--remove-bullets__list-item" id="u0040"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">PDC: <a class="inter-ref-link" href="https://pdc.cancer.gov/pdc/" target="_blank">https://pdc.cancer.gov/pdc/</a> with identifiers PDC000221 and PDC000222</div></li><li class="ce-list--remove-bullets__list-item" id="u0045"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text"><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S054%20" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S054</a></div></li></ul></div><div class="section-paragraph">CPTAC lung adenocarcinoma (LUAD) proteomics data:<ul class="ce-list--remove-bullets" id="ulist0025"><li class="ce-list--remove-bullets__list-item" id="u0050"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">PDC: <a class="inter-ref-link" href="https://pdc.cancer.gov/pdc/" target="_blank">https://pdc.cancer.gov/pdc/</a> with identifiers PDC000153, PDC000149, PDC000224)<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S056" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S056</a></div></li></ul></div><div class="section-paragraph">Genomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); <a class="inter-ref-link" href="https://portal.gdc.cancer.gov/" target="_blank">https://portal.gdc.cancer.gov/</a>, via dbGaP Study Accession: phs001287.v10.p5</div><div class="section-paragraph">All histologic <a class="inter-ref-link" href="https://www.cancerimagingarchive.net/datascope/cptac/home/" target="_blank">https://www.cancerimagingarchive.net/datascope/cptac/home/</a> and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access</div><div class="section-paragraph"><a class="inter-ref-link" href="https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC" target="_blank">https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC</a>.</div><div class="section-paragraph">Sample annotation, processed and normalized data files are provided as <a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1" class="scroll-into-link">Tables S1</a>, <a href="/cms/10.1016/j.cell.2021.07.016/attachment/b531dd09-5d15-4a8a-92a4-8cf82c7b7778/mmc2" class="scroll-into-link">S2</a>, and <a href="/cms/10.1016/j.cell.2021.07.016/attachment/21d6959b-79c4-4371-84f1-ade0942b9013/mmc3" class="scroll-into-link">S3</a>.</div><div class="section-paragraph">Software and code used in this study are referenced in their corresponding <a href="#sec4" class="scroll-into-link">STAR Methods</a> sections and also the <a href="#sec4.1" class="scroll-into-link">key resources table</a>.</div></section><section><h2 class="top" id="secsectitle0570" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by the National Cancer Institute (NCI) Clinical Proteomic Tumor Analysis Consortium (CPTAC) grants U24CA210955, U24CA210985, U24CA210986, U24CA210954, U24CA210967, U24CA210972, U24CA210979, U24CA210993, U01CA214114, U01CA214116, and U01CA214125 and NIH grants U54HL127624, P30CA240139, 1F32HL154711-01, and T32CA203690. We thank Dr. Lata Adnani for helping with graphics.</div><h3>Author contributions</h3><div class="section-paragraph">Conception, S.S., S.A.C., and M.A.G.; data collection, S.S., M.H.K., S.C.A., and E.J.B.; data analysis, S.S., K.K., P.M.J.B., S.R.S., F.P., C.K.-S., Y.D., B.R., M.H.K., S.V.V., A.K., J.T.L., E.J.J., T.O., Y.G., V.S., K.E.C., D.I.H., M.P.C., S.C., X.S., J.J., W.L., K.L., B.W., Y.L., Z.H.G., M.E.S., O.B., P.V., S.A., T.S., M.C., C.-K.M., Y.Z., F.d.V.L., C.H., M.A.W., S.M.D., K.R.C., D.R.M.; QC, C.J.N., G.H., J.S., Q.K.L., and M.T.; IHC and analysis, R.S., T.H.V., M.G.R., E.R.P.C., S.A.M., C.J.N., G.H., J.S., and K.L.; manuscript writing and editing, S.S. and M.A.G. with help from all authors; supervision, S.S., L.D., D.F., K.R., M.M., H.R., M.T., A.I.R., P.W., A.I.N., S.A.C., B.Z., D.R.M., and M.A.G.</div><h3>Declaration of interests</h3><div class="section-paragraph">The authors declare no competing interests.</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/480adb63-7291-4412-af89-9cf71c653f0d/mmc1.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.32
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. Metadata associated with LSCC patients and tumors, related to Figures 1, 2, 3, 4, 5, 6, 7, S1, S2, S3, S4, S5, S6, and S7</div><div class="supplemental-information__description"><p>(A) Metadata associated with this study (B) Cancer Associated Genes (CAGs) (C) Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. NMF core members were considered for this analysis. (D) Table containing results from association analyses between key metadata and technical parameters for NMF multi-omic subtyping. All samples; tumors and NATs were considered for this association. (E) Table containing results from association analyses between key metadata and technical parameters for immune clusters. All samples; tumors and NATs were considered for this association.</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/b531dd09-5d15-4a8a-92a4-8cf82c7b7778/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (218.85
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Genomic and methylation datasets associated with this study, related to Figures 1, 2, 3, 4, 5, 6, 7, S1, S2, S3, S4, S5, S6, and S7</div><div class="supplemental-information__description"><p>(A) Gene-level data matrix of somatic copy number alterations (CNA) obtained from GISTIC2 analysis. CNAs are represented as absolute gene copy numbers. Refseq GRCh38/hg38 database was used as a reference. (B) Same as Table S2A but CNAs are represented as log2(CN)-1. (C) Same as Table S2A but CNAs are represented as thresholded copy number level calls. (D) RNA expression data matrix reporting Log2-transformed upper-quartile (UQ)-normalized FPKM values. (E) Same as Table S2D but with FPKM values median-centered by gene. (F) miRNA expression data matrix reporting log2 TPM (Transcripts Per Kilobase Million) values. (G) Same as Table S2F but with TPM values median-centered by gene. (H) Gene level DNA methylation data matrix. Beta values were obtained by averaging the beta values of individual probes in the islands of promoter or 5′ UTR regions of the gene. (I) Somatic mutations associated with the CPTAC LSCC cohort in Mutation Annotation Format (MAF) format.</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/21d6959b-79c4-4371-84f1-ade0942b9013/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (372.89
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. Global proteomics, phosphoproteomics, acetylproteomics, and ubiquitylproteomics data associated with this study, related to Figures 1, 2, 3, 4, 5, 6, 7, S1, S2, S3, S4, S5, S6, and S7</div><div class="supplemental-information__description"><p>(A) Two-component-normalized log2-transformed protein data matrix (sample/CR ratios) before application of the filtering strategy described in the STAR Methods. (B) Same as (A) but after application of the filtering strategy described in the STAR Methods (C) Gene-centric global proteome data matrix derived from Table S3B. Protein-level data were aggregated to gene-level using the sample/CR ratio assigned to the dominant isoform per gene symbol (similar to the razor-peptide approach). (D) Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the STAR methods. (E) Two-component-normalized log2-transformed phosphoproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the STAR methods. (F) Gene-centric phosphoproteomic data matrix derived from Table S3E. Phosphosite-level sample/CR ratios were aggregated to gene-level using mean ratio. (G) Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the STAR methods. (H) Two-component-normalized log2-transformed acetylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the STAR methods. (I) Gene-centric acetylproteomics data matrix derived from Table S3H. Acetylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. (J) Acetylproteomic data matrix based on Table S3G corrected for global protein expression (Table S3B) as described in the STAR methods. (K) Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) before application of the filtering strategy described in the STAR methods. See STAR methods for details on batch correction. (L) Two-component-normalized log2-transformed, batch corrected ubiquitylproteomic data matrix (sample/CR ratios) after application of the filtering strategy described in the STAR methods. See STAR methods for details on batch correction. (M) Gene-centric ubiquitylproteomic data matrix derived from Table S3L. Ubiquitylsite-level sample/CR ratios were aggregated to gene-level using mean ratio. (N) Ubiquitylproteomic data matrix based on Table S3L corrected for global protein expression (Table S3B) as described in the STAR methods.</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/4b5095d7-3912-42e0-bafe-7603da9b3d8c/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (7.92
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. CNA cis and trans effects, iProFun, CMAP, NRF2 differential expression, NRF2 signature, cis-eQTL from RNA and proteomics datasets, eGenes, related to Figures 1, 2, 3, and 4</div><div class="supplemental-information__description"><p>(A) Copy number amplified cytobands (B) Copy number deleted cytobands (C) CNA <em>cis-</em> and <em>trans-</em>effects (D) Significant focal amplicons that show significant <em>cis</em>-effect and are cancer associated genes (E) iProFun results with mRNA, protein, phosphoprotein, acetylprotein as outcomes and methylation as the predictor, adjusted for CNV(lr), CNV(baf) and tumor purity. Results include identified genes (Yes = 1/No = 0) with FDR&lt;10% and identified phenotype association analyses (Yes = 1/No = 0). (F) CMAP tab- Connectivity Map (CMAP) results (G) Association between genes enriched in the CMAP analysis and sample metadata (H) Normalized connectivity scores (NCS) between Library of Integrated Network-based Cellular Signatures (LINCS) compound signatures and the five NMF multi-omic subtype signatures at the RNA and Protein level. (I) Summary of CDKN2A and RB1 annotations (J) t test p values of differential proteomic features CCND1, CCND2, CCND3, and CDK6 (K) Differential proteogenomic features in samples with and without <em>NFE2L2</em> (<em>NRF2</em>), <em>CUL3</em> and <em>KEAP1</em> mutations (L) NRF2 signature (M) Differential expression of RNA and proteins between TP63-low vsTP63-high samples.</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/00860ac6-4c62-49af-802c-e198ebc9b4c8/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.44
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Tables associated with ubiquitylproteome analysis, related to Figure 5</div><div class="supplemental-information__description"><p>(A) Clustering of tumor samples using protein-corrected ubiquitin data (B) Ub-sites that show differential abundance across Ub clusters (C) Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 1 (D) Pathway enrichment results for Ub-sites with higher abundance in Ub Cluster 2 (E) Ub-sites and E3 ligase pairs with significant abundance association (FDR &lt; 0.01) (F) Ub-sites and DUB pairs with significant abundance association (FDR &lt; 0.01) (G) Pathway enrichment results for Ub-sites positively associated with HERC5 abundance (H) Genes with significant spatial differential acetylation clustering within a protein structure as detected by CLUMPS (I) Genes with significant spatial differential ubiquitylation clustering within a protein structure as detected by CLUMPS (J) Genes with significant spatial differential NAT ubiquitylation clustering within a protein structure as detected by CLUMPS</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/6857848e-0bbc-4c66-80c7-06cb6cbfc199/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (3.44
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Tables associated with immune-landscape analysis, related to Figure 6</div><div class="supplemental-information__description"><p>(A) Immune composition of LSSC tumors based on deconvolution of RNA expression using xCell. (B) Immune clusters based on xCell signatures (C) Pathways upregulated in immune groups based on RNA-Seq data (C1), global proteomics (C2), phosphoproteomics (C3), acetylproteomics (C4) (D) Genes upregulated in immune groups based on RNA-Seq data (D1), global proteomics (D2), phosphoproteomics (D3), acetylproteomics (D4) (E) RTK Correlation-Based Enrichment (CBPE) Scores (F) Pathway scores in Tumor samples based on global protein abundance via ssGSEA (G) <em>cis</em>-eQTLS identified based on germline variants and tumor RNA-seq data (at 5% FDR) (H) <em>cis</em>-pQTLS identified based on germline variants and tumor proteomics data (at 5% FDR) (I) List of genes/proteins whose expression is associated with a SNP (J) Ingenuity pathway analysis of the eGenes from QTL analysis (K) <em>cis</em>-QTLs identified in genes/proteins differentially expressed in immune groups (K1-3) and list of eGenes (K4)</p></div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2021.07.016/attachment/f54f2d41-2c47-48e5-8152-94e5558b1a23/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (9.76
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. Tables associated with tumor-associated proteins, related to Figure 7</div><div class="supplemental-information__description"><p>(A) Differential abundance of proteins in tumor versus paired NAT in LSCC, HNSCC, and LUAD (B) 502 proteins significantly overexpressed (log<sub>2</sub>(FC) &gt;2, FDR &lt;0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis (C) Tumor suppressors or oncogenes regulated at the level of protein expression or phosphoproteome. Activating phosphosite: regulated at the level of phosphorylation, annotated as activating site, Inhibiting phosphosite: regulated at the level of phosphorylation, annotated as inhibitory site. CausalPath was used for this analysis. (D) DepMap dependency scores (CRISPR (Avana) Public 20Q3) across 16 LSCC cell lines (E) Genes that correlate to immune score at the copy number, RNA, and protein (if available) levels in LSCC, HNSCC, and LUAD (F) Predicted tumor neoantigens identified in the CPTAC LSCC cohort at the level of RNA transcripts and peptides containing evidence of somatic mutations.</p></div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0595" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Abril-Rodriguez G. </li><li class="ref__authors__name">Ribas A. </li></ul><div class="ref__title">SnapShot: Immune Checkpoint Inhibitors.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">848-848.e1</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85020645133&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28609660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2017.05.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2017.05.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=848-848.e1&journal=Cancer+Cell&author=Abril-Rodriguez+G.&author=Ribas+A.&title=SnapShot%3A+Immune+Checkpoint+Inhibitors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ahn E.R. </li><li class="ref__authors__name">Mangat P.K. </li><li class="ref__authors__name">Garrett-Mayer E. </li><li class="ref__authors__name">Halabi S. </li><li class="ref__authors__name">Dib E.G. </li><li class="ref__authors__name">Haggstrom D.E. </li><li class="ref__authors__name">Alguire K.B. </li><li class="ref__authors__name">Calfa C.J. </li><li class="ref__authors__name">Cannon T.L. </li><li class="ref__authors__name">Crilley P.A. </li><li>et al.</li></ul><div class="ref__title">Palbociclib in Patients With Non–Small-Cell Lung Cancer With CDKN2A Alterations: Results From the Targeted Agent and Profiling Utilization Registry Study.</div><span class="ref__series"><i>JCO Precision Oncology.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1200/PO.20.00037">https://doi.org/10.1200/PO.20.00037</a></span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85090749277&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33015524&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FPO.20.00037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=JCO+Precision+Oncology&author=Ahn+E.R.&author=Mangat+P.K.&author=Garrett-Mayer+E.&author=Halabi+S.&author=Dib+E.G.&author=Haggstrom+D.E.&author=Alguire+K.B.&author=Calfa+C.J.&author=Cannon+T.L.&author=Crilley+P.A.&title=Palbociclib+in+Patients+With+Non%E2%80%93Small-Cell+Lung+Cancer+With+CDKN2A+Alterations%3A+Results+From+the+Targeted+Agent+and+Profiling+Utilization+Registry+Study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Albert M. </li><li class="ref__authors__name">Bécares M. </li><li class="ref__authors__name">Falqui M. </li><li class="ref__authors__name">Fernández-Lozano C. </li><li class="ref__authors__name">Guerra S. </li></ul><div class="ref__title">ISG15, a Small Molecule with Huge Implications: Regulation of Mitochondrial Homeostasis.</div><span class="ref__series"><i>Viruses.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">629</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85056635216&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30428561&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fv10110629&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=629&journal=Viruses&author=Albert+M.&author=B%C3%A9cares+M.&author=Falqui+M.&author=Fern%C3%A1ndez-Lozano+C.&author=Guerra+S.&title=ISG15%2C+a+Small+Molecule+with+Huge+Implications%3A+Regulation+of+Mitochondrial+Homeostasis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Alcalá S. </li><li class="ref__authors__name">Sancho P. </li><li class="ref__authors__name">Martinelli P. </li><li class="ref__authors__name">Navarro D. </li><li class="ref__authors__name">Pedrero C. </li><li class="ref__authors__name">Martín-Hijano L. </li><li class="ref__authors__name">Valle S. </li><li class="ref__authors__name">Earl J. </li><li class="ref__authors__name">Rodríguez-Serrano M. </li><li class="ref__authors__name">Ruiz-Cañas L. </li><li>et al.</li></ul><div class="ref__title">ISG15 and ISGylation is required for pancreatic cancer stem cell mitophagy and metabolic plasticity.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">2682</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85085688337&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32472071&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-16395-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2682&journal=Nat.+Commun.&author=Alcal%C3%A1+S.&author=Sancho+P.&author=Martinelli+P.&author=Navarro+D.&author=Pedrero+C.&author=Mart%C3%ADn-Hijano+L.&author=Valle+S.&author=Earl+J.&author=Rodr%C3%ADguez-Serrano+M.&author=Ruiz-Ca%C3%B1as+L.&title=ISG15+and+ISGylation+is+required+for+pancreatic+cancer+stem+cell+mitophagy+and+metabolic+plasticity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Almeida L.G. </li><li class="ref__authors__name">Sakabe N.J. </li><li class="ref__authors__name">deOliveira A.R. </li><li class="ref__authors__name">Silva M.C.C. </li><li class="ref__authors__name">Mundstein A.S. </li><li class="ref__authors__name">Cohen T. </li><li class="ref__authors__name">Chen Y.-T. </li><li class="ref__authors__name">Chua R. </li><li class="ref__authors__name">Gurung S. </li><li class="ref__authors__name">Gnjatic S. </li><li>et al.</li></ul><div class="ref__title">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (225)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.-T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Aran D. </li><li class="ref__authors__name">Hu Z. </li><li class="ref__authors__name">Butte A.J. </li></ul><div class="ref__title">xCell: digitally portraying the tissue cellular heterogeneity landscape.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (490)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ardini-Poleske M.E. </li><li class="ref__authors__name">Clark R.F. </li><li class="ref__authors__name">Ansong C. </li><li class="ref__authors__name">Carson J.P. </li><li class="ref__authors__name">Corley R.A. </li><li class="ref__authors__name">Deutsch G.H. </li><li class="ref__authors__name">Hagood J.S. </li><li class="ref__authors__name">Kaminski N. </li><li class="ref__authors__name">Mariani T.J. </li><li class="ref__authors__name">Potter S.S. </li><li>et al.</li><li>LungMAP Consortium</li></ul><div class="ref__title">LungMAP: The Molecular Atlas of Lung Development Program.</div><span class="ref__series"><i>Am. J. Physiol. Lung Cell. Mol. Physiol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">L733-L740</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85032899579&amp;cf=" onclick="newWindow(this.href);return false">Scopus (70)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28798251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1152%2Fajplung.00139.2017&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2017&pages=L733-L740&journal=Am.+J.+Physiol.+Lung+Cell.+Mol.+Physiol.&author=LungMAP+Consortium&title=LungMAP%3A+The+Molecular+Atlas+of+Lung+Development+Program">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Babur Ö. </li><li class="ref__authors__name">Luna A. </li><li class="ref__authors__name">Korkut A. </li><li class="ref__authors__name">Durupinar F. </li><li class="ref__authors__name">Siper M.C. </li><li class="ref__authors__name">Dogrusoz U. </li><li class="ref__authors__name">Vaca Jacome A.S. </li><li class="ref__authors__name">Peckner R. </li><li class="ref__authors__name">Christianson K.E. </li><li class="ref__authors__name">Jaffe J.D. </li><li>et al.</li></ul><div class="ref__title">Causal interactions from proteomic profiles: Molecular data meet pathway knowledge.</div><span class="ref__series"><i>Patterns.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">100257</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85106566488&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.patter.2021.100257&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2018&pages=100257&journal=Patterns&author=Babur+%C3%96.&author=Luna+A.&author=Korkut+A.&author=Durupinar+F.&author=Siper+M.C.&author=Dogrusoz+U.&author=Vaca+Jacome+A.S.&author=Peckner+R.&author=Christianson+K.E.&author=Jaffe+J.D.&title=Causal+interactions+from+proteomic+profiles%3A+Molecular+data+meet+pathway+knowledge">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Baeza J. </li><li class="ref__authors__name">Smallegan M.J. </li><li class="ref__authors__name">Denu J.M. </li></ul><div class="ref__title">Site-specific reactivity of nonenzymatic lysine acetylation.</div><span class="ref__series"><i>ACS Chem. Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">122-128</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84921405747&amp;cf=" onclick="newWindow(this.href);return false">Scopus (92)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25555129&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1021%2Fcb500848p&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=122-128&journal=ACS+Chem.+Biol.&author=Baeza+J.&author=Smallegan+M.J.&author=Denu+J.M.&title=Site-specific+reactivity+of+nonenzymatic+lysine+acetylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bailey M.H. </li><li class="ref__authors__name">Tokheim C. </li><li class="ref__authors__name">Porta-Pardo E. </li><li class="ref__authors__name">Sengupta S. </li><li class="ref__authors__name">Bertrand D. </li><li class="ref__authors__name">Weerasinghe A. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Wendl M.C. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Reardon B. </li><li>et al.</li><li>MC3 Working Group</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive Characterization of Cancer Driver Genes and Mutations.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">371-385.e18</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044623848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (581)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29625053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=371-385.e18&journal=Cell&author=MC3+Working+GroupCancer+Genome+Atlas+Research+Network&title=Comprehensive+Characterization+of+Cancer+Driver+Genes+and+Mutations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bando H. </li><li class="ref__authors__name">Atsumi T. </li><li class="ref__authors__name">Nishio T. </li><li class="ref__authors__name">Niwa H. </li><li class="ref__authors__name">Mishima S. </li><li class="ref__authors__name">Shimizu C. </li><li class="ref__authors__name">Yoshioka N. </li><li class="ref__authors__name">Bucala R. </li><li class="ref__authors__name">Koike T. </li></ul><div class="ref__title">Phosphorylation of the 6-phosphofructo-2-kinase/fructose 2,6-bisphosphatase/PFKFB3 family of glycolytic regulators in human cancer.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">5784-5792</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-23844517036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16115917&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-05-0149&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=5784-5792&journal=Clin.+Cancer+Res.&author=Bando+H.&author=Atsumi+T.&author=Nishio+T.&author=Niwa+H.&author=Mishima+S.&author=Shimizu+C.&author=Yoshioka+N.&author=Bucala+R.&author=Koike+T.&title=Phosphorylation+of+the+6-phosphofructo-2-kinase%2Ffructose+2%2C6-bisphosphatase%2FPFKFB3+family+of+glycolytic+regulators+in+human+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barbie D.A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Boehm J.S. </li><li class="ref__authors__name">Kim S.Y. </li><li class="ref__authors__name">Moody S.E. </li><li class="ref__authors__name">Dunn I.F. </li><li class="ref__authors__name">Schinzel A.C. </li><li class="ref__authors__name">Sandy P. </li><li class="ref__authors__name">Meylan E. </li><li class="ref__authors__name">Scholl C. </li><li>et al.</li></ul><div class="ref__title">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1243)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barker L. </li></ul><div class="ref__title">A comparison of nine confidence intervals for a Poisson parameter when the expected number of events is ≤ 5.</div><span class="ref__series"><i>Am. Stat.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">56</span></span>: <span class="ref__seriesPages">85-89</span><div class="ref__controls"><a href="#back-bib212" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0035995688&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1198%2F000313002317572736&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2002&pages=85-89&journal=Am.+Stat.&author=Barker+L.&title=A+comparison+of+nine+confidence+intervals+for+a+Poisson+parameter+when+the+expected+number+of+events+is+%E2%89%A4+5">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bellman R. </li></ul><div class="ref__title">On the approximation of curves by line segments using dynamic programming.</div><span class="ref__series"><i>Commun. ACM.</i> <span class="ref__seriesDate">1961; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">284</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84945709924&amp;cf=" onclick="newWindow(this.href);return false">Scopus (248)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1145%2F366573.366611&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1961&pages=284&journal=Commun.+ACM&author=Bellman+R.&title=On+the+approximation+of+curves+by+line+segments+using+dynamic+programming">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benjamini Y. </li><li class="ref__authors__name">Hochberg Y. </li></ul><div class="ref__title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.</div><span class="ref__series"><i>J.R. Stat. Soc. Series B Stat. Methodol.</i> <span class="ref__seriesDate">1995; </span><span class="ref__seriesVolume">57</span></span>: <span class="ref__seriesPages">289-300</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Fj.2517-6161.1995.tb02031.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1995&pages=289-300&journal=J.R.+Stat.+Soc.+Series+B+Stat.+Methodol.&author=Benjamini+Y.&author=Hochberg+Y.&title=Controlling+the+False+Discovery+Rate%3A+A+Practical+and+Powerful+Approach+to+Multiple+Testing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bray F. </li><li class="ref__authors__name">Ferlay J. </li><li class="ref__authors__name">Soerjomataram I. </li><li class="ref__authors__name">Siegel R.L. </li><li class="ref__authors__name">Torre L.A. </li><li class="ref__authors__name">Jemal A. </li></ul><div class="ref__title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries.</div><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">394-424</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85053395052&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30524)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30207593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3322%2Fcaac.21492&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&journal=CA+Cancer+J.+Clin.&author=Bray+F.&author=Ferlay+J.&author=Soerjomataram+I.&author=Siegel+R.L.&author=Torre+L.A.&author=Jemal+A.&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Metagenes and molecular pattern discovery using matrix factorization.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1082)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cadenas C. </li><li class="ref__authors__name">Franckenstein D. </li><li class="ref__authors__name">Schmidt M. </li><li class="ref__authors__name">Gehrmann M. </li><li class="ref__authors__name">Hermes M. </li><li class="ref__authors__name">Geppert B. </li><li class="ref__authors__name">Schormann W. </li><li class="ref__authors__name">Maccoux L.J. </li><li class="ref__authors__name">Schug M. </li><li class="ref__authors__name">Schumann A. </li><li>et al.</li></ul><div class="ref__title">Role of thioredoxin reductase 1 and thioredoxin interacting protein in prognosis of breast cancer.</div><span class="ref__series"><i>Breast Cancer Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R44</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78049394036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (139)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20584310&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fbcr2599&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=R44&journal=Breast+Cancer+Res.&author=Cadenas+C.&author=Franckenstein+D.&author=Schmidt+M.&author=Gehrmann+M.&author=Hermes+M.&author=Geppert+B.&author=Schormann+W.&author=Maccoux+L.J.&author=Schug+M.&author=Schumann+A.&title=Role+of+thioredoxin+reductase+1+and+thioredoxin+interacting+protein+in+prognosis+of+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cai J. </li><li class="ref__authors__name">Fang L. </li><li class="ref__authors__name">Huang Y. </li><li class="ref__authors__name">Li R. </li><li class="ref__authors__name">Yuan J. </li><li class="ref__authors__name">Yang Y. </li><li class="ref__authors__name">Zhu X. </li><li class="ref__authors__name">Chen B. </li><li class="ref__authors__name">Wu J. </li><li class="ref__authors__name">Li M. </li></ul><div class="ref__title">miR-205 targets PTEN and PHLPP2 to augment AKT signaling and drive malignant phenotypes in non-small cell lung cancer.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">73</span></span>: <span class="ref__seriesPages">5402-5415</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84883478160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (145)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23856247&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-13-0297&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=5402-5415&journal=Cancer+Res.&author=Cai+J.&author=Fang+L.&author=Huang+Y.&author=Li+R.&author=Yuan+J.&author=Yang+Y.&author=Zhu+X.&author=Chen+B.&author=Wu+J.&author=Li+M.&title=miR-205+targets+PTEN+and+PHLPP2+to+augment+AKT+signaling+and+drive+malignant+phenotypes+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cammer M. </li><li class="ref__authors__name">Gevrey J.-C. </li><li class="ref__authors__name">Lorenz M. </li><li class="ref__authors__name">Dovas A. </li><li class="ref__authors__name">Condeelis J. </li><li class="ref__authors__name">Cox D. </li></ul><div class="ref__title">The mechanism of CSF-1-induced Wiskott-Aldrich syndrome protein activation in vivo: a role for phosphatidylinositol 3-kinase and Cdc42.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">284</span></span>: <span class="ref__seriesPages">23302-23311</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-69949155572&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19561083&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M109.036384&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23302-23311&journal=J.%C2%A0Biol.+Chem.&author=Cammer+M.&author=Gevrey+J.-C.&author=Lorenz+M.&author=Dovas+A.&author=Condeelis+J.&author=Cox+D.&title=The+mechanism+of+CSF-1-induced+Wiskott-Aldrich+syndrome+protein+activation+in%C2%A0vivo%3A+a+role+for+phosphatidylinositol+3-kinase+and+Cdc42">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Campbell J.D. </li><li class="ref__authors__name">Yau C. </li><li class="ref__authors__name">Bowlby R. </li><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Brennan K. </li><li class="ref__authors__name">Fan H. </li><li class="ref__authors__name">Taylor A.M. </li><li class="ref__authors__name">Wang C. </li><li class="ref__authors__name">Walter V. </li><li class="ref__authors__name">Akbani R. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Genomic, Pathway Network, and Immunologic Features Distinguishing Squamous Carcinomas.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">194-212.e6</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044610513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (100)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29617660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2018.03.063&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=194-212.e6&journal=Cell+Rep.&author=Cancer+Genome+Atlas+Research+Network&title=Genomic%2C+Pathway+Network%2C+and+Immunologic+Features+Distinguishing+Squamous+Carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cannarile M.A. </li><li class="ref__authors__name">Weisser M. </li><li class="ref__authors__name">Jacob W. </li><li class="ref__authors__name">Jegg A.-M. </li><li class="ref__authors__name">Ries C.H. </li><li class="ref__authors__name">Rüttinger D. </li></ul><div class="ref__title">Colony-stimulating factor 1 receptor (CSF1R) inhibitors in cancer therapy.</div><span class="ref__series"><i>J. Immunother. Cancer.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">53</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85024385914&amp;cf=" onclick="newWindow(this.href);return false">Scopus (315)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28716061&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs40425-017-0257-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=53&journal=J.%C2%A0Immunother.+Cancer&author=Cannarile+M.A.&author=Weisser+M.&author=Jacob+W.&author=Jegg+A.-M.&author=Ries+C.H.&author=R%C3%BCttinger+D.&title=Colony-stimulating+factor+1+receptor+%28CSF1R%29+inhibitors+in+cancer+therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cannito S. </li><li class="ref__authors__name">Novo E. </li><li class="ref__authors__name">Compagnone A. </li><li class="ref__authors__name">Valfrè di Bonzo L. </li><li class="ref__authors__name">Busletta C. </li><li class="ref__authors__name">Zamara E. </li><li class="ref__authors__name">Paternostro C. </li><li class="ref__authors__name">Povero D. </li><li class="ref__authors__name">Bandino A. </li><li class="ref__authors__name">Bozzo F. </li><li>et al.</li></ul><div class="ref__title">Redox mechanisms switch on hypoxia-dependent epithelial-mesenchymal transition in cancer cells.</div><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2267-2278</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-57349164238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (242)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18791199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fcarcin%2Fbgn216&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=2267-2278&journal=Carcinogenesis&author=Cannito+S.&author=Novo+E.&author=Compagnone+A.&author=Valfr%C3%A8+di+Bonzo+L.&author=Busletta+C.&author=Zamara+E.&author=Paternostro+C.&author=Povero+D.&author=Bandino+A.&author=Bozzo+F.&title=Redox+mechanisms+switch+on+hypoxia-dependent+epithelial-mesenchymal+transition+in+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cassetta L. </li><li class="ref__authors__name">Pollard J.W. </li></ul><div class="ref__title">Targeting macrophages: therapeutic approaches in cancer.</div><span class="ref__series"><i>Nat. Rev. Drug Discov.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">887-904</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85058944664&amp;cf=" onclick="newWindow(this.href);return false">Scopus (362)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30361552&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnrd.2018.169&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=887-904&journal=Nat.+Rev.+Drug+Discov.&author=Cassetta+L.&author=Pollard+J.W.&title=Targeting+macrophages%3A+therapeutic+approaches+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chida S. </li><li class="ref__authors__name">Okayama H. </li><li class="ref__authors__name">Noda M. </li><li class="ref__authors__name">Saito K. </li><li class="ref__authors__name">Nakajima T. </li><li class="ref__authors__name">Aoto K. </li><li class="ref__authors__name">Hayase S. </li><li class="ref__authors__name">Momma T. </li><li class="ref__authors__name">Ohki S. </li><li class="ref__authors__name">Kono K. </li><li class="ref__authors__name">Takenoshita S. </li></ul><div class="ref__title">Stromal VCAN expression as a potential prognostic biomarker for disease recurrence in stage II-III colon cancer.</div><span class="ref__series"><i>Carcinogenesis.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">878-887</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84991037068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27287872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fcarcin%2Fbgw069&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=878-887&journal=Carcinogenesis&author=Chida+S.&author=Okayama+H.&author=Noda+M.&author=Saito+K.&author=Nakajima+T.&author=Aoto+K.&author=Hayase+S.&author=Momma+T.&author=Ohki+S.&author=Kono+K.&author=Takenoshita+S.&title=Stromal+VCAN+expression+as+a+potential+prognostic+biomarker+for+disease+recurrence+in+stage+II-III+colon+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Christensen R.H.B. </li></ul><div class="ref__title">ordinal-Regression Models for Ordinal Data. R package version 2019.2-10. (The R Project for Statistical Computing).</div><span class="refDate">2019</span><div class="ref__controls"><a href="#back-bib213" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Christensen+R.H.B.&title=ordinal-Regression+Models+for+Ordinal+Data.+R+package+version+2019.2-10.+%28The+R+Project+for+Statistical+Computing%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chu A. </li><li class="ref__authors__name">Robertson G. </li><li class="ref__authors__name">Brooks D. </li><li class="ref__authors__name">Mungall A.J. </li><li class="ref__authors__name">Birol I. </li><li class="ref__authors__name">Coope R. </li><li class="ref__authors__name">Ma Y. </li><li class="ref__authors__name">Jones S. </li><li class="ref__authors__name">Marra M.A. </li></ul><div class="ref__title">Large-scale profiling of microRNAs for The Cancer Genome Atlas.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span><div class="ref__controls"><a href="#back-bib214" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+The+Cancer+Genome+Atlas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Debebe Z. </li><li class="ref__authors__name">Rathmell W.K. </li></ul><div class="ref__title">Ror2 as a therapeutic target in cancer.</div><span class="ref__series"><i>Pharmacol. Ther.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">150</span></span>: <span class="ref__seriesPages">143-148</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928380373&amp;cf=" onclick="newWindow(this.href);return false">Scopus (52)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25614331&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.pharmthera.2015.01.010&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=143-148&journal=Pharmacol.+Ther.&author=Debebe+Z.&author=Rathmell+W.K.&title=Ror2+as+a+therapeutic+target+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">DeBerardinis R.J. </li><li class="ref__authors__name">Chandel N.S. </li></ul><div class="ref__title">Fundamentals of cancer metabolism.</div><span class="ref__series"><i>Sci. Adv.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">e1600200</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27386546&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fsciadv.1600200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=e1600200&journal=Sci.+Adv.&author=DeBerardinis+R.J.&author=Chandel+N.S.&title=Fundamentals+of+cancer+metabolism">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Deelen P. </li><li class="ref__authors__name">Bonder M.J. </li><li class="ref__authors__name">van der Velde K.J. </li><li class="ref__authors__name">Westra H.-J. </li><li class="ref__authors__name">Winder E. </li><li class="ref__authors__name">Hendriksen D. </li><li class="ref__authors__name">Franke L. </li><li class="ref__authors__name">Swertz M.A. </li></ul><div class="ref__title">Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration.</div><span class="ref__series"><i>BMC Res. Notes.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">901</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928813777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (40)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25495213&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1756-0500-7-901&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=901&journal=BMC+Res.+Notes&author=Deelen+P.&author=Bonder+M.J.&author=van+der+Velde+K.J.&author=Westra+H.-J.&author=Winder+E.&author=Hendriksen+D.&author=Franke+L.&author=Swertz+M.A.&title=Genotype+harmonizer%3A+automatic+strand+alignment+and+format+conversion+for+genotype+data+integration">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">DeYoung M.P. </li><li class="ref__authors__name">Johannessen C.M. </li><li class="ref__authors__name">Leong C.-O. </li><li class="ref__authors__name">Faquin W. </li><li class="ref__authors__name">Rocco J.W. </li><li class="ref__authors__name">Ellisen L.W. </li></ul><div class="ref__title">Tumor-specific p73 up-regulation mediates p63 dependence in squamous cell carcinoma.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">66</span></span>: <span class="ref__seriesPages">9362-9368</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33750378033&amp;cf=" onclick="newWindow(this.href);return false">Scopus (91)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17018588&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-06-1619&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=9362-9368&journal=Cancer+Res.&author=DeYoung+M.P.&author=Johannessen+C.M.&author=Leong+C.-O.&author=Faquin+W.&author=Rocco+J.W.&author=Ellisen+L.W.&title=Tumor-specific+p73+up-regulation+mediates+p63+dependence+in+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Diviani D. </li><li class="ref__authors__name">Raimondi F. </li><li class="ref__authors__name">Del Vescovo C.D. </li><li class="ref__authors__name">Dreyer E. </li><li class="ref__authors__name">Reggi E. </li><li class="ref__authors__name">Osman H. </li><li class="ref__authors__name">Ruggieri L. </li><li class="ref__authors__name">Gonano C. </li><li class="ref__authors__name">Cavin S. </li><li class="ref__authors__name">Box C.L. </li><li>et al.</li></ul><div class="ref__title">Small-Molecule Protein-Protein Interaction Inhibitor of Oncogenic Rho Signaling.</div><span class="ref__series"><i>Cell Chem. Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1135-1146</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84988585445&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27593112&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.chembiol.2016.07.015&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1135-1146&journal=Cell+Chem.+Biol.&author=Diviani+D.&author=Raimondi+F.&author=Del+Vescovo+C.D.&author=Dreyer+E.&author=Reggi+E.&author=Osman+H.&author=Ruggieri+L.&author=Gonano+C.&author=Cavin+S.&author=Box+C.L.&title=Small-Molecule+Protein-Protein+Interaction+Inhibitor+of+Oncogenic+Rho+Signaling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dovas A. </li><li class="ref__authors__name">Gevrey J.-C. </li><li class="ref__authors__name">Grossi A. </li><li class="ref__authors__name">Park H. </li><li class="ref__authors__name">Abou-Kheir W. </li><li class="ref__authors__name">Cox D. </li></ul><div class="ref__title">Regulation of podosome dynamics by WASp phosphorylation: implication in matrix degradation and chemotaxis in macrophages.</div><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">122</span></span>: <span class="ref__seriesPages">3873-3882</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70450236976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (76)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19808890&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.051755&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=3873-3882&journal=J.%C2%A0Cell+Sci.&author=Dovas+A.&author=Gevrey+J.-C.&author=Grossi+A.&author=Park+H.&author=Abou-Kheir+W.&author=Cox+D.&title=Regulation+of+podosome+dynamics+by+WASp+phosphorylation%3A+implication+in+matrix+degradation+and+chemotaxis+in+macrophages">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Edelman M.J. </li><li class="ref__authors__name">Redman M.W. </li><li class="ref__authors__name">Albain K.S. </li><li class="ref__authors__name">McGary E.C. </li><li class="ref__authors__name">Rafique N.M. </li><li class="ref__authors__name">Petro D. </li><li class="ref__authors__name">Waqar S.N. </li><li class="ref__authors__name">Minichiello K. </li><li class="ref__authors__name">Miao J. </li><li class="ref__authors__name">Papadimitrakopoulou V.A. </li><li>et al.</li></ul><div class="ref__title">SWOG S1400C (NCT02154490)-A Phase II Study of Palbociclib for Previously Treated Cell Cycle Gene Alteration-Positive Patients with Stage IV Squamous Cell Lung Cancer (Lung-MAP Substudy).</div><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">1853-1859</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31302234&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2019.06.027&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=1853-1859&journal=J.%C2%A0Thorac.+Oncol.&author=Edelman+M.J.&author=Redman+M.W.&author=Albain+K.S.&author=McGary+E.C.&author=Rafique+N.M.&author=Petro+D.&author=Waqar+S.N.&author=Minichiello+K.&author=Miao+J.&author=Papadimitrakopoulou+V.A.&title=SWOG+S1400C+%28NCT02154490%29-A+Phase+II+Study+of+Palbociclib+for+Previously+Treated+Cell+Cycle+Gene+Alteration-Positive+Patients+with+Stage+IV+Squamous+Cell+Lung+Cancer+%28Lung-MAP+Substudy%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Suman V.J. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Goncalves R. </li><li class="ref__authors__name">Sanati S. </li><li class="ref__authors__name">Creighton C.J. </li><li class="ref__authors__name">DeSchryver K. </li><li class="ref__authors__name">Crouch E. </li><li class="ref__authors__name">Brink A. </li><li class="ref__authors__name">Watson M. </li><li>et al.</li></ul><div class="ref__title">Ki67 Proliferation Index as a Tool for Chemotherapy Decisions During and After Neoadjuvant Aromatase Inhibitor Treatment of Breast Cancer: Results From the American College of Surgeons Oncology Group Z1031 Trial (Alliance).</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1061-1069</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016622416&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28045625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2016.69.4406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=1061-1069&journal=J.%C2%A0Clin.+Oncol.&author=Ellis+M.J.&author=Suman+V.J.&author=Hoog+J.&author=Goncalves+R.&author=Sanati+S.&author=Creighton+C.J.&author=DeSchryver+K.&author=Crouch+E.&author=Brink+A.&author=Watson+M.&title=Ki67+Proliferation+Index+as+a+Tool+for+Chemotherapy+Decisions+During+and+After+Neoadjuvant+Aromatase+Inhibitor+Treatment+of+Breast+Cancer%3A+Results+From+the+American+College+of+Surgeons+Oncology+Group+Z1031+Trial+%28Alliance%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fabregat A. </li><li class="ref__authors__name">Jupe S. </li><li class="ref__authors__name">Matthews L. </li><li class="ref__authors__name">Sidiropoulos K. </li><li class="ref__authors__name">Gillespie M. </li><li class="ref__authors__name">Garapati P. </li><li class="ref__authors__name">Haw R. </li><li class="ref__authors__name">Jassal B. </li><li class="ref__authors__name">Korninger F. </li><li class="ref__authors__name">May B. </li><li>et al.</li></ul><div class="ref__title">The Reactome Pathway Knowledgebase.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1055)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+Reactome+Pathway+Knowledgebase">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fan T. </li><li class="ref__authors__name">Sun G. </li><li class="ref__authors__name">Sun X. </li><li class="ref__authors__name">Zhao L. </li><li class="ref__authors__name">Zhong R. </li><li class="ref__authors__name">Peng Y. </li></ul><div class="ref__title">Tumor Energy Metabolism and Potential of 3-Bromopyruvate as an Inhibitor of Aerobic Glycolysis: Implications in Tumor Treatment.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">317</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064332549&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcancers11030317&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=317&journal=Cancers+%28Basel%29&author=Fan+T.&author=Sun+G.&author=Sun+X.&author=Zhao+L.&author=Zhong+R.&author=Peng+Y.&title=Tumor+Energy+Metabolism+and+Potential+of+3-Bromopyruvate+as+an+Inhibitor+of+Aerobic+Glycolysis%3A+Implications+in+Tumor+Treatment">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Faubert B. </li><li class="ref__authors__name">Li K.Y. </li><li class="ref__authors__name">Cai L. </li><li class="ref__authors__name">Hensley C.T. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Zacharias L.G. </li><li class="ref__authors__name">Yang C. </li><li class="ref__authors__name">Do Q.N. </li><li class="ref__authors__name">Doucette S. </li><li class="ref__authors__name">Burguete D. </li><li>et al.</li></ul><div class="ref__title">Lactate Metabolism in Human Lung Tumors.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">358-371.e9</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85030560166&amp;cf=" onclick="newWindow(this.href);return false">Scopus (414)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28985563&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.09.019&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=358-371.e9&journal=Cell&author=Faubert+B.&author=Li+K.Y.&author=Cai+L.&author=Hensley+C.T.&author=Kim+J.&author=Zacharias+L.G.&author=Yang+C.&author=Do+Q.N.&author=Doucette+S.&author=Burguete+D.&title=Lactate+Metabolism+in+Human+Lung+Tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fernandes A.P. </li><li class="ref__authors__name">Capitanio A. </li><li class="ref__authors__name">Selenius M. </li><li class="ref__authors__name">Brodin O. </li><li class="ref__authors__name">Rundlöf A.-K. </li><li class="ref__authors__name">Björnstedt M. </li></ul><div class="ref__title">Expression profiles of thioredoxin family proteins in human lung cancer tissue: correlation with proliferation and differentiation.</div><span class="ref__series"><i>Histopathology.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">55</span></span>: <span class="ref__seriesPages">313-320</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-69549116697&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19723146&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Fj.1365-2559.2009.03381.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2009&pages=313-320&journal=Histopathology&author=Fernandes+A.P.&author=Capitanio+A.&author=Selenius+M.&author=Brodin+O.&author=Rundl%C3%B6f+A.-K.&author=Bj%C3%B6rnstedt+M.&title=Expression+profiles+of+thioredoxin+family+proteins+in+human+lung+cancer+tissue%3A+correlation+with+proliferation+and+differentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fisher S. </li><li class="ref__authors__name">Barry A. </li><li class="ref__authors__name">Abreu J. </li><li class="ref__authors__name">Minie B. </li><li class="ref__authors__name">Nolan J. </li><li class="ref__authors__name">Delorey T.M. </li><li class="ref__authors__name">Young G. </li><li class="ref__authors__name">Fennell T.J. </li><li class="ref__authors__name">Allen A. </li><li class="ref__authors__name">Ambrogio L. </li><li>et al.</li></ul><div class="ref__title">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (256)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Foltz D.R. </li><li class="ref__authors__name">Santiago M.C. </li><li class="ref__authors__name">Berechid B.E. </li><li class="ref__authors__name">Nye J.S. </li></ul><div class="ref__title">Glycogen synthase kinase-3beta modulates notch signaling and stability.</div><span class="ref__series"><i>Curr. Biol.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">1006-1011</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0037172661&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=12123574&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0960-9822%2802%2900888-6&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=1006-1011&journal=Curr.+Biol.&author=Foltz+D.R.&author=Santiago+M.C.&author=Berechid+B.E.&author=Nye+J.S.&title=Glycogen+synthase+kinase-3beta+modulates+notch+signaling+and+stability">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fortin J.-P. </li><li class="ref__authors__name">Labbe A. </li><li class="ref__authors__name">Lemire M. </li><li class="ref__authors__name">Zanke B.W. </li><li class="ref__authors__name">Hudson T.J. </li><li class="ref__authors__name">Fertig E.J. </li><li class="ref__authors__name">Greenwood C.M. </li><li class="ref__authors__name">Hansen K.D. </li></ul><div class="ref__title">Functional normalization of 450k methylation array data improves replication in large cancer studies.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">503</span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84923699348&amp;cf=" onclick="newWindow(this.href);return false">Scopus (313)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25599564&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-014-0503-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=503&journal=Genome+Biol.&author=Fortin+J.-P.&author=Labbe+A.&author=Lemire+M.&author=Zanke+B.W.&author=Hudson+T.J.&author=Fertig+E.J.&author=Greenwood+C.M.&author=Hansen+K.D.&title=Functional+normalization+of+450k+methylation+array+data+improves+replication+in+large+cancer+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fraley C. </li><li class="ref__authors__name">Raftery A.E. </li><li class="ref__authors__name">Scrucca L. </li><li class="ref__authors__name">Murphy T.B. </li><li class="ref__authors__name">Fop M. </li></ul><div class="ref__title">mclust 5: clustering, classification and density estimation using Gaussian finite mixture models.</div><span class="ref__series"><i>The R Journal.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">289-317</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27818791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.32614%2FRJ-2016-021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=289-317&journal=The+R+Journal&author=Fraley+C.&author=Raftery+A.E.&author=Scrucca+L.&author=Murphy+T.B.&author=Fop+M.&title=mclust+5%3A+clustering%2C+classification+and+density+estimation+using+Gaussian+finite+mixture+models">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Freed D. </li><li class="ref__authors__name">Pan R. </li><li class="ref__authors__name">Aldana R. </li></ul><div class="ref__title">TNscope: Accurate Detection of Somatic Mutations with Haplotype-based Variant Candidate Detection and Machine Learning Filtering.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/250647">https://doi.org/10.1101/250647</a></span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85095625543&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F250647&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&journal=bioRxiv&author=Freed+D.&author=Pan+R.&author=Aldana+R.&title=TNscope%3A+Accurate+Detection+of+Somatic+Mutations+with+Haplotype-based+Variant+Candidate+Detection+and+Machine+Learning+Filtering">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fu B. </li><li class="ref__authors__name">Meng W. </li><li class="ref__authors__name">Zeng X. </li><li class="ref__authors__name">Zhao H. </li><li class="ref__authors__name">Liu W. </li><li class="ref__authors__name">Zhang T. </li></ul><div class="ref__title">TXNRD1 Is an Unfavorable Prognostic Factor for Patients with Hepatocellular Carcinoma.</div><span class="ref__series"><i>BioMed Res. Int.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2017</span></span>: <span class="ref__seriesPages">4698167</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85018902496&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28536696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1155%2F2017%2F4698167&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=4698167&journal=BioMed+Res.+Int.&author=Fu+B.&author=Meng+W.&author=Zeng+X.&author=Zhao+H.&author=Liu+W.&author=Zhang+T.&title=TXNRD1+Is+an+Unfavorable+Prognostic+Factor+for+Patients+with+Hepatocellular+Carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gandhi N. </li><li class="ref__authors__name">Das G.M. </li></ul><div class="ref__title">Metabolic Reprogramming in Breast Cancer and Its Therapeutic Implications.</div><span class="ref__series"><i>Cells.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">89</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcells8020089&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=89&journal=Cells&author=Gandhi+N.&author=Das+G.M.&title=Metabolic+Reprogramming+in+Breast+Cancer+and+Its+Therapeutic+Implications">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao C. </li><li class="ref__authors__name">Zhuang J. </li><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Liu C. </li><li class="ref__authors__name">Zhou C. </li><li class="ref__authors__name">Liu L. </li><li class="ref__authors__name">Sun C. </li></ul><div class="ref__title">Exploration of methylation-driven genes for monitoring and prognosis of patients with lung adenocarcinoma.</div><span class="ref__series"><i>Cancer Cell Int.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">194</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85057211952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30498398&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs12935-018-0691-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=194&journal=Cancer+Cell+Int.&author=Gao+C.&author=Zhuang+J.&author=Li+H.&author=Liu+C.&author=Zhou+C.&author=Liu+L.&author=Sun+C.&title=Exploration+of+methylation-driven+genes+for+monitoring+and+prognosis+of+patients+with+lung+adenocarcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Garcia-Mata R. </li><li class="ref__authors__name">Boulter E. </li><li class="ref__authors__name">Burridge K. </li></ul><div class="ref__title">The ‘invisible hand’: regulation of RHO GTPases by RHOGDIs.</div><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">493-504</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79960706295&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21779026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnrm3153&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=493-504&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Garcia-Mata+R.&author=Boulter+E.&author=Burridge+K.&title=The+%E2%80%98invisible+hand%E2%80%99%3A+regulation+of+RHO+GTPases+by+RHOGDIs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gaujoux R. </li><li class="ref__authors__name">Seoighe C. </li></ul><div class="ref__title">A flexible R package for nonnegative matrix factorization.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (488)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gey S. </li><li class="ref__authors__name">Lebarbier E. </li></ul><div class="ref__title">Using CART to detect multiple change points in the mean for large sample.</div><span class="ref__series"><i>HAL.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume"></span> (<span class="refComment">ffhal-00327146</span>)</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&journal=HAL&author=Gey+S.&author=Lebarbier+E.&title=Using+CART+to+detect+multiple+change+points+in+the+mean+for+large+sample">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Giaccone G. </li><li class="ref__authors__name">Zatloukal P. </li><li class="ref__authors__name">Roubec J. </li><li class="ref__authors__name">Floor K. </li><li class="ref__authors__name">Musil J. </li><li class="ref__authors__name">Kuta M. </li><li class="ref__authors__name">van Klaveren R.J. </li><li class="ref__authors__name">Chaudhary S. </li><li class="ref__authors__name">Gunther A. </li><li class="ref__authors__name">Shamsili S. </li></ul><div class="ref__title">Multicenter phase II trial of YM155, a small-molecule suppressor of survivin, in patients with advanced, refractory, non-small-cell lung cancer.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">4481-4486</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-76749118103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (186)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19687333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2008.21.1862&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=4481-4486&journal=J.%C2%A0Clin.+Oncol.&author=Giaccone+G.&author=Zatloukal+P.&author=Roubec+J.&author=Floor+K.&author=Musil+J.&author=Kuta+M.&author=van+Klaveren+R.J.&author=Chaudhary+S.&author=Gunther+A.&author=Shamsili+S.&title=Multicenter+phase+II+trial+of+YM155%2C+a+small-molecule+suppressor+of+survivin%2C+in+patients+with+advanced%2C+refractory%2C+non-small-cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Giannakopoulos N.V. </li><li class="ref__authors__name">Luo J.-K. </li><li class="ref__authors__name">Papov V. </li><li class="ref__authors__name">Zou W. </li><li class="ref__authors__name">Lenschow D.J. </li><li class="ref__authors__name">Jacobs B.S. </li><li class="ref__authors__name">Borden E.C. </li><li class="ref__authors__name">Li J. </li><li class="ref__authors__name">Virgin H.W. </li><li class="ref__authors__name">Zhang D.-E. </li></ul><div class="ref__title">Proteomic identification of proteins conjugated to ISG15 in mouse and human cells.</div><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">336</span></span>: <span class="ref__seriesPages">496-506</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-24644474832&amp;cf=" onclick="newWindow(this.href);return false">Scopus (164)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16139798&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.bbrc.2005.08.132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2005&pages=496-506&journal=Biochem.+Biophys.+Res.+Commun.&author=Giannakopoulos+N.V.&author=Luo+J.-K.&author=Papov+V.&author=Zou+W.&author=Lenschow+D.J.&author=Jacobs+B.S.&author=Borden+E.C.&author=Li+J.&author=Virgin+H.W.&author=Zhang+D.-E.&title=Proteomic+identification+of+proteins+conjugated+to+ISG15+in+mouse+and+human+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gibbs D.L. </li></ul><div class="ref__title">Robust classification of Immune Subtypes in Cancer.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.01.17.910950">https://doi.org/10.1101/2020.01.17.910950</a></span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85095499730&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.01.17.910950&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Gibbs+D.L.&title=Robust+classification+of+Immune+Subtypes+in+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Liang W.-W. </li><li class="ref__authors__name">Reva B. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">182</span></span>: <span class="ref__seriesPages">200-225.e35</span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087400742&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32649874&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.06.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2020&pages=200-225.e35&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Characterization+Reveals+Therapeutic+Vulnerabilities+in+Lung+Adenocarcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gorvel J.P. </li><li class="ref__authors__name">Chang T.C. </li><li class="ref__authors__name">Boretto J. </li><li class="ref__authors__name">Azuma T. </li><li class="ref__authors__name">Chavrier P. </li></ul><div class="ref__title">Differential properties of D4/LyGDI versus RhoGDI: phosphorylation and rho GTPase selectivity.</div><span class="ref__series"><i>FEBS Lett.</i> <span class="ref__seriesDate">1998; </span><span class="ref__seriesVolume">422</span></span>: <span class="ref__seriesPages">269-273</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0032579315&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=9490022&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2FS0014-5793%2898%2900020-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=422&publication_year=1998&pages=269-273&journal=FEBS+Lett.&author=Gorvel+J.P.&author=Chang+T.C.&author=Boretto+J.&author=Azuma+T.&author=Chavrier+P.&title=Differential+properties+of+D4%2FLyGDI+versus+RhoGDI%3A+phosphorylation+and+rho+GTPase+selectivity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gregory P.A. </li><li class="ref__authors__name">Bert A.G. </li><li class="ref__authors__name">Paterson E.L. </li><li class="ref__authors__name">Barry S.C. </li><li class="ref__authors__name">Tsykin A. </li><li class="ref__authors__name">Farshid G. </li><li class="ref__authors__name">Vadas M.A. </li><li class="ref__authors__name">Khew-Goodall Y. </li><li class="ref__authors__name">Goodall G.J. </li></ul><div class="ref__title">The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1.</div><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">593-601</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-43049103824&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2912)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18376396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncb1722&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2008&pages=593-601&journal=Nat.+Cell+Biol.&author=Gregory+P.A.&author=Bert+A.G.&author=Paterson+E.L.&author=Barry+S.C.&author=Tsykin+A.&author=Farshid+G.&author=Vadas+M.A.&author=Khew-Goodall+Y.&author=Goodall+G.J.&title=The+miR-200+family+and+miR-205+regulate+epithelial+to+mesenchymal+transition+by+targeting+ZEB1+and+SIP1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hammerman P.S. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Voet D. </li><li class="ref__authors__name">Jing R. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Stojanov P. </li><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Gabriel S. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive genomic characterization of squamous cell lung cancers.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2459)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hansen K.D. </li></ul><div class="ref__title">IlluminaHumanMethylationEPICanno.ilm10b2.hg19 (Bioconductor).</div><span class="refDate">2017</span><div class="ref__controls"><a href="#back-bib215" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Hansen+K.D.&title=IlluminaHumanMethylationEPICanno.ilm10b2.hg19+%28Bioconductor%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hänzelmann S. </li><li class="ref__authors__name">Castelo R. </li><li class="ref__authors__name">Guinney J. </li></ul><div class="ref__title">GSVA: gene set variation analysis for microarray and RNA-seq data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1428)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Happel N. </li><li class="ref__authors__name">Stoldt S. </li><li class="ref__authors__name">Schmidt B. </li><li class="ref__authors__name">Doenecke D. </li></ul><div class="ref__title">M phase-specific phosphorylation of histone H1.5 at threonine 10 by GSK-3.</div><span class="ref__series"><i>J. Mol. Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">386</span></span>: <span class="ref__seriesPages">339-350</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-59149103350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19136008&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jmb.2008.12.047&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=386&publication_year=2009&pages=339-350&journal=J.%C2%A0Mol.+Biol.&author=Happel+N.&author=Stoldt+S.&author=Schmidt+B.&author=Doenecke+D.&title=M+phase-specific+phosphorylation+of+histone+H1.5+at+threonine+10+by+GSK-3">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Haslam A. </li><li class="ref__authors__name">Prasad V. </li></ul><div class="ref__title">Estimation of the Percentage of US Patients With Cancer Who Are Eligible for and Respond to Checkpoint Inhibitor Immunotherapy Drugs.</div><span class="ref__series"><i>JAMA Netw. Open.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">e192535</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85065557338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31050774&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1001%2Fjamanetworkopen.2019.2535&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2019&pages=e192535&journal=JAMA+Netw.+Open&author=Haslam+A.&author=Prasad+V.&title=Estimation+of+the+Percentage+of+US+Patients+With+Cancer+Who+Are+Eligible+for+and+Respond+to+Checkpoint+Inhibitor+Immunotherapy+Drugs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hebert A.S. </li><li class="ref__authors__name">Dittenhafer-Reed K.E. </li><li class="ref__authors__name">Yu W. </li><li class="ref__authors__name">Bailey D.J. </li><li class="ref__authors__name">Selen E.S. </li><li class="ref__authors__name">Boersma M.D. </li><li class="ref__authors__name">Carson J.J. </li><li class="ref__authors__name">Tonelli M. </li><li class="ref__authors__name">Balloon A.J. </li><li class="ref__authors__name">Higbee A.J. </li><li>et al.</li></ul><div class="ref__title">Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome.</div><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">186-199</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872276165&amp;cf=" onclick="newWindow(this.href);return false">Scopus (400)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23201123&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molcel.2012.10.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=186-199&journal=Mol.+Cell&author=Hebert+A.S.&author=Dittenhafer-Reed+K.E.&author=Yu+W.&author=Bailey+D.J.&author=Selen+E.S.&author=Boersma+M.D.&author=Carson+J.J.&author=Tonelli+M.&author=Balloon+A.J.&author=Higbee+A.J.&title=Calorie+restriction+and+SIRT3+trigger+global+reprogramming+of+the+mitochondrial+protein+acetylome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Herbst R.S. </li><li class="ref__authors__name">Morgensztern D. </li><li class="ref__authors__name">Boshoff C. </li></ul><div class="ref__title">The biology and management of non-small cell lung cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">553</span></span>: <span class="ref__seriesPages">446-454</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85041105043&amp;cf=" onclick="newWindow(this.href);return false">Scopus (891)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29364287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature25183&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=553&publication_year=2018&pages=446-454&journal=Nature&author=Herbst+R.S.&author=Morgensztern+D.&author=Boshoff+C.&title=The+biology+and+management+of+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Herreño A.M. </li><li class="ref__authors__name">Ramírez A.C. </li><li class="ref__authors__name">Chaparro V.P. </li><li class="ref__authors__name">Fernandez M.J. </li><li class="ref__authors__name">Cañas A. </li><li class="ref__authors__name">Morantes C.F. </li><li class="ref__authors__name">Moreno O.M. </li><li class="ref__authors__name">Brugés R.E. </li><li class="ref__authors__name">Mejía J.A. </li><li class="ref__authors__name">Bustos F.J. </li><li>et al.</li></ul><div class="ref__title">Role of RUNX2 transcription factor in epithelial mesenchymal transition in non-small cell lung cancer lung cancer: Epigenetic control of the RUNX2 P1 promoter.</div><span class="ref__series"><i>Tumour Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">41</span> (<span class="refComment">1010428319851014</span>)</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85066436930&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31109257&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1177%2F1010428319851014&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&journal=Tumour+Biol.&author=Herre%C3%B1o+A.M.&author=Ram%C3%ADrez+A.C.&author=Chaparro+V.P.&author=Fernandez+M.J.&author=Ca%C3%B1as+A.&author=Morantes+C.F.&author=Moreno+O.M.&author=Brug%C3%A9s+R.E.&author=Mej%C3%ADa+J.A.&author=Bustos+F.J.&title=Role+of+RUNX2+transcription+factor+in+epithelial+mesenchymal+transition+in+non-small+cell+lung+cancer+lung+cancer%3A+Epigenetic+control+of+the+RUNX2+P1+promoter">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hitosugi T. </li><li class="ref__authors__name">Chen J. </li></ul><div class="ref__title">Post-translational modifications and the Warburg effect.</div><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">4279-4285</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84906937144&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24096483&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fonc.2013.406&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=4279-4285&journal=Oncogene&author=Hitosugi+T.&author=Chen+J.&title=Post-translational+modifications+and+the+Warburg+effect">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hong C.-F. </li><li class="ref__authors__name">Chen W.-Y. </li><li class="ref__authors__name">Wu C.-W. </li></ul><div class="ref__title">Upregulation of Wnt signaling under hypoxia promotes lung cancer progression.</div><span class="ref__series"><i>Oncol. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">1706-1714</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85026356039&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28713928&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3892%2For.2017.5807&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=1706-1714&journal=Oncol.+Rep.&author=Hong+C.-F.&author=Chen+W.-Y.&author=Wu+C.-W.&title=Upregulation+of+Wnt+signaling+under+hypoxia+promotes+lung+cancer+progression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Tkachev S. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Skrzypek E. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Sullivan M. </li></ul><div class="ref__title">PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">40</span></span>: <span class="ref__seriesPages">D261-D270</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84857047339&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1019)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22135298&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkr1122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D261-D270&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Kornhauser+J.M.&author=Tkachev+S.&author=Zhang+B.&author=Skrzypek+E.&author=Murray+B.&author=Latham+V.&author=Sullivan+M.&title=PhosphoSitePlus%3A+a+comprehensive+resource+for+investigating+the+structure+and+function+of+experimentally+determined+post-translational+modifications+in+man+and+mouse">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hsu E.-C. </li><li class="ref__authors__name">Rice M.A. </li><li class="ref__authors__name">Bermudez A. </li><li class="ref__authors__name">Marques F.J.G. </li><li class="ref__authors__name">Aslan M. </li><li class="ref__authors__name">Liu S. </li><li class="ref__authors__name">Ghoochani A. </li><li class="ref__authors__name">Zhang C.A. </li><li class="ref__authors__name">Chen Y.-S. </li><li class="ref__authors__name">Zlitni A. </li><li>et al.</li></ul><div class="ref__title">Trop2 is a driver of metastatic prostate cancer with neuroendocrine phenotype via PARP1.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">117</span></span>: <span class="ref__seriesPages">2032-2042</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078684513&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31932422&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1905384117&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2032-2042&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Hsu+E.-C.&author=Rice+M.A.&author=Bermudez+A.&author=Marques+F.J.G.&author=Aslan+M.&author=Liu+S.&author=Ghoochani+A.&author=Zhang+C.A.&author=Chen+Y.-S.&author=Zlitni+A.&title=Trop2+is+a+driver+of+metastatic+prostate+cancer+with+neuroendocrine+phenotype+via+PARP1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huang H.-Y. </li><li class="ref__authors__name">Lin Y.-C.-D. </li><li class="ref__authors__name">Li J. </li><li class="ref__authors__name">Huang K.-Y. </li><li class="ref__authors__name">Shrestha S. </li><li class="ref__authors__name">Hong H.-C. </li><li class="ref__authors__name">Tang Y. </li><li class="ref__authors__name">Chen Y.-G. </li><li class="ref__authors__name">Jin C.-N. </li><li class="ref__authors__name">Yu Y. </li><li>et al.</li></ul><div class="ref__title">miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D148-D154</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31647101&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D148-D154&journal=Nucleic+Acids+Res.&author=Huang+H.-Y.&author=Lin+Y.-C.-D.&author=Li+J.&author=Huang+K.-Y.&author=Shrestha+S.&author=Hong+H.-C.&author=Tang+Y.&author=Chen+Y.-G.&author=Jin+C.-N.&author=Yu+Y.&title=miRTarBase+2020%3A+updates+to+the+experimentally+validated+microRNA-target+interaction+database">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Eguez R.V. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">da Veiga Leprevost F. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Lei J.T. </li><li class="ref__authors__name">Wen B. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">361-379.e16</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099708609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33417831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2020.12.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=361-379.e16&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+insights+into+the+biology+and+treatment+of+HPV-negative+head+and+neck+squamous+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hubaux R. </li><li class="ref__authors__name">Thu K.L. </li><li class="ref__authors__name">Vucic E.A. </li><li class="ref__authors__name">Pikor L.A. </li><li class="ref__authors__name">Kung S.H.Y. </li><li class="ref__authors__name">Martinez V.D. </li><li class="ref__authors__name">Mosslemi M. </li><li class="ref__authors__name">Becker-Santos D.D. </li><li class="ref__authors__name">Gazdar A.F. </li><li class="ref__authors__name">Lam S. </li><li class="ref__authors__name">Lam W.L. </li></ul><div class="ref__title">Microtubule affinity-regulating kinase 2 is associated with DNA damage response and cisplatin resistance in non-small cell lung cancer.</div><span class="ref__series"><i>Int. J. Cancer.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">137</span></span>: <span class="ref__seriesPages">2072-2082</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84939588821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25907283&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fijc.29577&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=2072-2082&journal=Int.+J.+Cancer&author=Hubaux+R.&author=Thu+K.L.&author=Vucic+E.A.&author=Pikor+L.A.&author=Kung+S.H.Y.&author=Martinez+V.D.&author=Mosslemi+M.&author=Becker-Santos+D.D.&author=Gazdar+A.F.&author=Lam+S.&author=Lam+W.L.&title=Microtubule+affinity-regulating+kinase+2+is+associated+with+DNA+damage+response+and+cisplatin+resistance+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hung Y.P. </li><li class="ref__authors__name">Redig A. </li><li class="ref__authors__name">Hornick J.L. </li><li class="ref__authors__name">Sholl L.M. </li></ul><div class="ref__title">ARID1A mutations and expression loss in non-small cell lung carcinomas: clinicopathologic and molecular analysis.</div><span class="ref__series"><i>Mod. Pathol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2256-2268</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85086777439&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32572156&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41379-020-0592-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=2256-2268&journal=Mod.+Pathol.&author=Hung+Y.P.&author=Redig+A.&author=Hornick+J.L.&author=Sholl+L.M.&title=ARID1A+mutations+and+expression+loss+in+non-small+cell+lung+carcinomas%3A+clinicopathologic+and+molecular+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ishitani T. </li><li class="ref__authors__name">Hirao T. </li><li class="ref__authors__name">Suzuki M. </li><li class="ref__authors__name">Isoda M. </li><li class="ref__authors__name">Ishitani S. </li><li class="ref__authors__name">Harigaya K. </li><li class="ref__authors__name">Kitagawa M. </li><li class="ref__authors__name">Matsumoto K. </li><li class="ref__authors__name">Itoh M. </li></ul><div class="ref__title">Nemo-like kinase suppresses Notch signalling by interfering with formation of the Notch active transcriptional complex.</div><span class="ref__series"><i>Nat. Cell Biol.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">278-285</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77649273214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (84)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20118921&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncb2028&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2010&pages=278-285&journal=Nat.+Cell+Biol.&author=Ishitani+T.&author=Hirao+T.&author=Suzuki+M.&author=Isoda+M.&author=Ishitani+S.&author=Harigaya+K.&author=Kitagawa+M.&author=Matsumoto+K.&author=Itoh+M.&title=Nemo-like+kinase+suppresses+Notch+signalling+by+interfering+with+formation+of+the+Notch+active+transcriptional+complex">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jimenez-Blasco D. </li><li class="ref__authors__name">Santofimia-Castaño P. </li><li class="ref__authors__name">Gonzalez A. </li><li class="ref__authors__name">Almeida A. </li><li class="ref__authors__name">Bolaños J.P. </li></ul><div class="ref__title">Astrocyte NMDA receptors’ activity sustains neuronal survival through a Cdk5-Nrf2 pathway.</div><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">1877-1889</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943197836&amp;cf=" onclick="newWindow(this.href);return false">Scopus (73)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25909891&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fcdd.2015.49&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1877-1889&journal=Cell+Death+Differ.&author=Jimenez-Blasco+D.&author=Santofimia-Casta%C3%B1o+P.&author=Gonzalez+A.&author=Almeida+A.&author=Bola%C3%B1os+J.P.&title=Astrocyte+NMDA+receptors%E2%80%99+activity+sustains+neuronal+survival+through+a+Cdk5-Nrf2+pathway">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jin Y.H. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Oh M. </li><li class="ref__authors__name">Ki H. </li><li class="ref__authors__name">Kim K. </li></ul><div class="ref__title">Regulation of Notch1/NICD and Hes1 expressions by GSK-3alpha/beta.</div><span class="ref__series"><i>Mol. Cells.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">15-19</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-62849118806&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19214430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1007%2Fs10059-009-0001-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=15-19&journal=Mol.+Cells&author=Jin+Y.H.&author=Kim+H.&author=Oh+M.&author=Ki+H.&author=Kim+K.&title=Regulation+of+Notch1%2FNICD+and+Hes1+expressions+by+GSK-3alpha%2Fbeta">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Johnson W.E. </li><li class="ref__authors__name">Li C. </li><li class="ref__authors__name">Rabinovic A. </li></ul><div class="ref__title">Adjusting batch effects in microarray expression data using empirical Bayes methods.</div><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">118-127</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33845432928&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3072)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16632515&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbiostatistics%2Fkxj037&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=118-127&journal=Biostatistics&author=Johnson+W.E.&author=Li+C.&author=Rabinovic+A.&title=Adjusting+batch+effects+in+microarray+expression+data+using+empirical+Bayes+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jurtz V. </li><li class="ref__authors__name">Paul S. </li><li class="ref__authors__name">Andreatta M. </li><li class="ref__authors__name">Marcatili P. </li><li class="ref__authors__name">Peters B. </li><li class="ref__authors__name">Nielsen M. </li></ul><div class="ref__title">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.</div><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span><div class="ref__controls"><a href="#back-bib73" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (413)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kamburov A. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Leshchiner I. </li><li class="ref__authors__name">Lage K. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Comprehensive assessment of cancer missense mutation clustering in protein structures.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">112</span></span>: <span class="ref__seriesPages">E5486-E5495</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943411478&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26392535&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1516373112&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E5486-E5495&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Kamburov+A.&author=Lawrence+M.S.&author=Polak+P.&author=Leshchiner+I.&author=Lage+K.&author=Golub+T.R.&author=Lander+E.S.&author=Getz+G.&title=Comprehensive+assessment+of+cancer+missense+mutation+clustering+in+protein+structures">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kanehisa M. </li><li class="ref__authors__name">Furumichi M. </li><li class="ref__authors__name">Tanabe M. </li><li class="ref__authors__name">Sato Y. </li><li class="ref__authors__name">Morishima K. </li></ul><div class="ref__title">KEGG: new perspectives on genomes, pathways, diseases and drugs.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D353-D361</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016149190&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2897)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27899662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkw1092&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D353-D361&journal=Nucleic+Acids+Res.&author=Kanehisa+M.&author=Furumichi+M.&author=Tanabe+M.&author=Sato+Y.&author=Morishima+K.&title=KEGG%3A+new+perspectives+on+genomes%2C+pathways%2C+diseases+and+drugs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Karachaliou N. </li><li class="ref__authors__name">Fernandez-Bruno M. </li><li class="ref__authors__name">Rosell R. </li></ul><div class="ref__title">Strategies for first-line immunotherapy in squamous cell lung cancer: are combinations a game changer?.</div><span class="ref__series"><i>Transl. Lung Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">S198-S201</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054551350&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30393600&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.21037%2Ftlcr.2018.07.02&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=S198-S201&journal=Transl.+Lung+Cancer+Res.&author=Karachaliou+N.&author=Fernandez-Bruno+M.&author=Rosell+R.&title=Strategies+for+first-line+immunotherapy+in+squamous+cell+lung+cancer%3A+are+combinations+a+game+changer%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kelly R.J. </li><li class="ref__authors__name">Thomas A. </li><li class="ref__authors__name">Rajan A. </li><li class="ref__authors__name">Chun G. </li><li class="ref__authors__name">Lopez-Chavez A. </li><li class="ref__authors__name">Szabo E. </li><li class="ref__authors__name">Spencer S. </li><li class="ref__authors__name">Carter C.A. </li><li class="ref__authors__name">Guha U. </li><li class="ref__authors__name">Khozin S. </li><li>et al.</li></ul><div class="ref__title">A phase I/II study of sepantronium bromide (YM155, survivin suppressor) with paclitaxel and carboplatin in patients with advanced non-small-cell lung cancer.</div><span class="ref__series"><i>Ann. Oncol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2601-2606</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84884717482&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23857959&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=abstract&amp;site=annonc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=fulltext&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fannonc%2Fmdt249&amp;cf=pdf&amp;site=annonc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=2601-2606&journal=Ann.+Oncol.&author=Kelly+R.J.&author=Thomas+A.&author=Rajan+A.&author=Chun+G.&author=Lopez-Chavez+A.&author=Szabo+E.&author=Spencer+S.&author=Carter+C.A.&author=Guha+U.&author=Khozin+S.&title=A+phase+I%2FII+study+of+sepantronium+bromide+%28YM155%2C+survivin+suppressor%29+with+paclitaxel+and+carboplatin+in+patients+with+advanced+non-small-cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Keysar S.B. </li><li class="ref__authors__name">Le P.N. </li><li class="ref__authors__name">Miller B. </li><li class="ref__authors__name">Jackson B.C. </li><li class="ref__authors__name">Eagles J.R. </li><li class="ref__authors__name">Nieto C. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Tang B. </li><li class="ref__authors__name">Glogowska M.J. </li><li class="ref__authors__name">Morton J.J. </li><li>et al.</li></ul><div class="ref__title">Regulation of Head and Neck Squamous Cancer Stem Cells by PI3K and SOX2.</div><span class="ref__series"><i>J. Natl. Cancer Inst.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">djw189</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85014859275&amp;cf=" onclick="newWindow(this.href);return false">Scopus (46)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fjnci%2Fdjw189&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2016&pages=djw189&journal=J.%C2%A0Natl.+Cancer+Inst.&author=Keysar+S.B.&author=Le+P.N.&author=Miller+B.&author=Jackson+B.C.&author=Eagles+J.R.&author=Nieto+C.&author=Kim+J.&author=Tang+B.&author=Glogowska+M.J.&author=Morton+J.J.&title=Regulation+of+Head+and+Neck+Squamous+Cancer+Stem+Cells+by+PI3K+and+SOX2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kikuchi T. </li><li class="ref__authors__name">Hassanein M. </li><li class="ref__authors__name">Amann J.M. </li><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Slebos R.J.C. </li><li class="ref__authors__name">Rahman S.M.J. </li><li class="ref__authors__name">Kaufman J.M. </li><li class="ref__authors__name">Zhang X. </li><li class="ref__authors__name">Hoeksema M.D. </li><li class="ref__authors__name">Harris B.K. </li><li>et al.</li></ul><div class="ref__title">In-depth proteomic analysis of nonsmall cell lung cancer to discover molecular targets and candidate biomarkers.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">916-932</span><div class="ref__controls"><a href="#back-bib79" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84867148065&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22761400&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M111.015370&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=916-932&journal=Mol.+Cell.+Proteomics&author=Kikuchi+T.&author=Hassanein+M.&author=Amann+J.M.&author=Liu+Q.&author=Slebos+R.J.C.&author=Rahman+S.M.J.&author=Kaufman+J.M.&author=Zhang+X.&author=Hoeksema+M.D.&author=Harris+B.K.&title=In-depth+proteomic+analysis+of+nonsmall+cell+lung+cancer+to+discover+molecular+targets+and+candidate+biomarkers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Park H. </li></ul><div class="ref__title">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span><div class="ref__controls"><a href="#back-bib80" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (518)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinformatics&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib81" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (502)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Mouw K.W. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Braunstein L.Z. </li><li class="ref__authors__name">Kamburov A. </li><li class="ref__authors__name">Kwiatkowski D.J. </li><li class="ref__authors__name">Rosenberg J.E. </li><li class="ref__authors__name">Van Allen E.M. </li><li class="ref__authors__name">D’Andrea A. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">600-606</span><div class="ref__controls"><a href="#back-bib82" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84964308124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (171)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27111033&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3557&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=600-606&journal=Nat.+Genet.&author=Kim+J.&author=Mouw+K.W.&author=Polak+P.&author=Braunstein+L.Z.&author=Kamburov+A.&author=Kwiatkowski+D.J.&author=Rosenberg+J.E.&author=Van+Allen+E.M.&author=D%E2%80%99Andrea+A.&author=Getz+G.&title=Somatic+ERCC2+mutations+are+associated+with+a+distinct+genomic+signature+in+urothelial+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Klepsch V. </li><li class="ref__authors__name">Hermann-Kleiter N. </li><li class="ref__authors__name">Do-Dinh P. </li><li class="ref__authors__name">Jakic B. </li><li class="ref__authors__name">Offermann A. </li><li class="ref__authors__name">Efremova M. </li><li class="ref__authors__name">Sopper S. </li><li class="ref__authors__name">Rieder D. </li><li class="ref__authors__name">Krogsdam A. </li><li class="ref__authors__name">Gamerith G. </li><li>et al.</li></ul><div class="ref__title">Nuclear receptor NR2F6 inhibition potentiates responses to PD-L1/PD-1 cancer immune checkpoint blockade.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1538</span><div class="ref__controls"><a href="#back-bib83" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85045763585&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29670099&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-018-04004-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1538&journal=Nat.+Commun.&author=Klepsch+V.&author=Hermann-Kleiter+N.&author=Do-Dinh+P.&author=Jakic+B.&author=Offermann+A.&author=Efremova+M.&author=Sopper+S.&author=Rieder+D.&author=Krogsdam+A.&author=Gamerith+G.&title=Nuclear+receptor+NR2F6+inhibition+potentiates+responses+to+PD-L1%2FPD-1+cancer+immune+checkpoint+blockade">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Knudsen E.S. </li><li class="ref__authors__name">Nambiar R. </li><li class="ref__authors__name">Rosario S.R. </li><li class="ref__authors__name">Smiraglia D.J. </li><li class="ref__authors__name">Goodrich D.W. </li><li class="ref__authors__name">Witkiewicz A.K. </li></ul><div class="ref__title">Pan-cancer molecular analysis of the RB tumor suppressor pathway.</div><span class="ref__series"><i>Commun. Biol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">158</span><div class="ref__controls"><a href="#back-bib84" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85082980228&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32242058&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs42003-020-0873-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=158&journal=Commun.+Biol&author=Knudsen+E.S.&author=Nambiar+R.&author=Rosario+S.R.&author=Smiraglia+D.J.&author=Goodrich+D.W.&author=Witkiewicz+A.K.&title=Pan-cancer+molecular+analysis+of+the+RB+tumor+suppressor+pathway">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Krämer A. </li><li class="ref__authors__name">Green J. </li><li class="ref__authors__name">Pollard Jr., J. </li><li class="ref__authors__name">Tugendreich S. </li></ul><div class="ref__title">Causal analysis approaches in Ingenuity Pathway Analysis.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">523-530</span><div class="ref__controls"><a href="#back-bib85" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84894571860&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1880)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24336805&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtt703&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=523-530&journal=Bioinformatics&author=Kr%C3%A4mer+A.&author=Green+J.&author=Pollard+J.&author=Tugendreich+S.&title=Causal+analysis+approaches+in+Ingenuity+Pathway+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kröger C. </li><li class="ref__authors__name">Afeyan A. </li><li class="ref__authors__name">Mraz J. </li><li class="ref__authors__name">Eaton E.N. </li><li class="ref__authors__name">Reinhardt F. </li><li class="ref__authors__name">Khodor Y.L. </li><li class="ref__authors__name">Thiru P. </li><li class="ref__authors__name">Bierie B. </li><li class="ref__authors__name">Ye X. </li><li class="ref__authors__name">Burge C.B. </li><li class="ref__authors__name">Weinberg R.A. </li></ul><div class="ref__title">Acquisition of a hybrid E/M state is essential for tumorigenicity of basal breast cancer cells.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">116</span></span>: <span class="ref__seriesPages">7353-7362</span><div class="ref__controls"><a href="#back-bib86" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064109821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (110)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30910979&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1812876116&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7353-7362&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Kr%C3%B6ger+C.&author=Afeyan+A.&author=Mraz+J.&author=Eaton+E.N.&author=Reinhardt+F.&author=Khodor+Y.L.&author=Thiru+P.&author=Bierie+B.&author=Ye+X.&author=Burge+C.B.&author=Weinberg+R.A.&title=Acquisition+of+a+hybrid+E%2FM+state+is+essential+for+tumorigenicity+of+basal+breast+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Hornbeck P. </li><li class="ref__authors__name">Raju R. </li><li class="ref__authors__name">Ahmad R. </li><li class="ref__authors__name">Szucs M. </li><li class="ref__authors__name">Mundt F. </li><li class="ref__authors__name">Forestier D. </li><li class="ref__authors__name">Jane-Valbuena J. </li><li>et al.</li></ul><div class="ref__title">A curated resource for phosphosite-specific signature analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">576-593</span><div class="ref__controls"><a href="#back-bib87" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062992827&amp;cf=" onclick="newWindow(this.href);return false">Scopus (34)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30563849&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR118.000943&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=576-593&journal=Mol.+Cell.+Proteomics&author=Krug+K.&author=Mertins+P.&author=Zhang+B.&author=Hornbeck+P.&author=Raju+R.&author=Ahmad+R.&author=Szucs+M.&author=Mundt+F.&author=Forestier+D.&author=Jane-Valbuena+J.&title=A+curated+resource+for+phosphosite-specific+signature+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Blumenberg L. </li><li class="ref__authors__name">Karpova A. </li><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Miles G. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Geffen Y. </li><li class="ref__authors__name">Tang L.C. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">183</span></span>: <span class="ref__seriesPages">1436-1456.e31</span><div class="ref__controls"><a href="#back-bib88" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85096494429&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33212010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2020.10.036&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2020&pages=1436-1456.e31&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Landscape+of+Breast+Cancer+Tumorigenesis+and+Targeted+Therapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kuhlmann N. </li><li class="ref__authors__name">Wroblowski S. </li><li class="ref__authors__name">Knyphausen P. </li><li class="ref__authors__name">de Boor S. </li><li class="ref__authors__name">Brenig J. </li><li class="ref__authors__name">Zienert A.Y. </li><li class="ref__authors__name">Meyer-Teschendorf K. </li><li class="ref__authors__name">Praefcke G.J.K. </li><li class="ref__authors__name">Nolte H. </li><li class="ref__authors__name">Krüger M. </li><li>et al.</li></ul><div class="ref__title">Structural and Mechanistic Insights into the Regulation of the Fundamental Rho Regulator RhoGDIα by Lysine Acetylation.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">291</span></span>: <span class="ref__seriesPages">5484-5499</span><div class="ref__controls"><a href="#back-bib89" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84974850910&amp;cf=" onclick="newWindow(this.href);return false">Scopus (28)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26719334&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M115.707091&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=5484-5499&journal=J.%C2%A0Biol.+Chem.&author=Kuhlmann+N.&author=Wroblowski+S.&author=Knyphausen+P.&author=de+Boor+S.&author=Brenig+J.&author=Zienert+A.Y.&author=Meyer-Teschendorf+K.&author=Praefcke+G.J.K.&author=Nolte+H.&author=Kr%C3%BCger+M.&title=Structural+and+Mechanistic+Insights+into+the+Regulation+of+the+Fundamental+Rho+Regulator+RhoGDI%CE%B1+by+Lysine+Acetylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kuhlmann N. </li><li class="ref__authors__name">Wroblowski S. </li><li class="ref__authors__name">Scislowski L. </li><li class="ref__authors__name">Lammers M. </li></ul><div class="ref__title">RhoGDIα Acetylation at K127 and K141 Affects Binding toward Nonprenylated RhoA.</div><span class="ref__series"><i>Biochemistry.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">55</span></span>: <span class="ref__seriesPages">304-312</span><div class="ref__controls"><a href="#back-bib90" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84955559442&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26695096&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1021%2Facs.biochem.5b01242&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=304-312&journal=Biochemistry&author=Kuhlmann+N.&author=Wroblowski+S.&author=Scislowski+L.&author=Lammers+M.&title=RhoGDI%CE%B1+Acetylation+at+K127+and+K141+Affects+Binding+toward+Nonprenylated+RhoA">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lacher S.E. </li><li class="ref__authors__name">Slattery M. </li></ul><div class="ref__title">Gene regulatory effects of disease-associated variation in the NRF2 network.</div><span class="ref__series"><i>Curr. Opin. Toxicol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">71-79</span><div class="ref__controls"><a href="#back-bib91" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85016740108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28203648&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cotox.2016.09.001&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=71-79&journal=Curr.+Opin.+Toxicol.&author=Lacher+S.E.&author=Slattery+M.&title=Gene+regulatory+effects+of+disease-associated+variation+in+the+NRF2+network">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lamb J. </li><li class="ref__authors__name">Crawford E.D. </li><li class="ref__authors__name">Peck D. </li><li class="ref__authors__name">Modell J.W. </li><li class="ref__authors__name">Blat I.C. </li><li class="ref__authors__name">Wrobel M.J. </li><li class="ref__authors__name">Lerner J. </li><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Ross K.N. </li><li>et al.</li></ul><div class="ref__title">The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">313</span></span>: <span class="ref__seriesPages">1929-1935</span><div class="ref__controls"><a href="#back-bib92" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33749335282&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2951)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17008526&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1126%2Fscience.1132939&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&journal=Science&author=Lamb+J.&author=Crawford+E.D.&author=Peck+D.&author=Modell+J.W.&author=Blat+I.C.&author=Wrobel+M.J.&author=Lerner+J.&author=Brunet+J.-P.&author=Subramanian+A.&author=Ross+K.N.&title=The+Connectivity+Map%3A+using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lamprecht S. </li><li class="ref__authors__name">Kaller M. </li><li class="ref__authors__name">Schmidt E.M. </li><li class="ref__authors__name">Blaj C. </li><li class="ref__authors__name">Schiergens T.S. </li><li class="ref__authors__name">Engel J. </li><li class="ref__authors__name">Jung A. </li><li class="ref__authors__name">Hermeking H. </li><li class="ref__authors__name">Grünewald T.G.P. </li><li class="ref__authors__name">Kirchner T. </li><li class="ref__authors__name">Horst D. </li></ul><div class="ref__title">PBX3 Is Part of an EMT Regulatory Network and Indicates Poor Outcome in Colorectal Cancer.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1974-1986</span><div class="ref__controls"><a href="#back-bib93" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85047860831&amp;cf=" onclick="newWindow(this.href);return false">Scopus (22)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29391352&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-17-2572&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1974-1986&journal=Clin.+Cancer+Res.&author=Lamprecht+S.&author=Kaller+M.&author=Schmidt+E.M.&author=Blaj+C.&author=Schiergens+T.S.&author=Engel+J.&author=Jung+A.&author=Hermeking+H.&author=Gr%C3%BCnewald+T.G.P.&author=Kirchner+T.&author=Horst+D.&title=PBX3+Is+Part+of+an+EMT+Regulatory+Network+and+Indicates+Poor+Outcome+in+Colorectal+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Stojanov P. </li><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Robinson J.T. </li><li class="ref__authors__name">Garraway L.A. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Gabriel S.B. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Discovery and saturation analysis of cancer genes across 21 tumour types.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">505</span></span>: <span class="ref__seriesPages">495-501</span><div class="ref__controls"><a href="#back-bib94" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84892833777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1758)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24390350&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature12912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=505&publication_year=2014&pages=495-501&journal=Nature&author=Lawrence+M.S.&author=Stojanov+P.&author=Mermel+C.H.&author=Robinson+J.T.&author=Garraway+L.A.&author=Golub+T.R.&author=Meyerson+M.&author=Gabriel+S.B.&author=Lander+E.S.&author=Getz+G.&title=Discovery+and+saturation+analysis+of+cancer+genes+across+21+tumour+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lê S. </li><li class="ref__authors__name">Josse J. </li><li class="ref__authors__name">Husson F. </li></ul><div class="ref__title">FactoMineR: AnRPackage for Multivariate Analysis.</div><span class="ref__series"><i>J. Stat. Softw.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.18637/jss.v025.i01">https://doi.org/10.18637/jss.v025.i01</a></span><div class="ref__controls"><a href="#back-bib95" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-46749110034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3306)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.18637%2Fjss.v025.i01&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&journal=J.%C2%A0Stat.+Softw.&author=L%C3%AA+S.&author=Josse+J.&author=Husson+F.&title=FactoMineR%3A+AnRPackage+for+Multivariate+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Le Large T.Y. </li><li class="ref__authors__name">Mantini G. </li><li class="ref__authors__name">Meijer L.L. </li><li class="ref__authors__name">Pham T.V. </li><li class="ref__authors__name">Funel N. </li><li class="ref__authors__name">van Grieken N.C. </li><li class="ref__authors__name">Kok B. </li><li class="ref__authors__name">Knol J. </li><li class="ref__authors__name">van Laarhoven H.W. </li><li class="ref__authors__name">Piersma S.R. </li><li>et al.</li></ul><div class="ref__title">Microdissected pancreatic cancer proteomes reveal tumor heterogeneity and therapeutic targets.</div><span class="ref__series"><i>JCI Insight.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e138290</span><div class="ref__controls"><a href="#back-bib96" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85089205740&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32634123&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1172%2Fjci.insight.138290&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=e138290&journal=JCI+Insight&author=Le+Large+T.Y.&author=Mantini+G.&author=Meijer+L.L.&author=Pham+T.V.&author=Funel+N.&author=van+Grieken+N.C.&author=Kok+B.&author=Knol+J.&author=van+Laarhoven+H.W.&author=Piersma+S.R.&title=Microdissected+pancreatic+cancer+proteomes+reveal+tumor+heterogeneity+and+therapeutic+targets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lebanony D. </li><li class="ref__authors__name">Benjamin H. </li><li class="ref__authors__name">Gilad S. </li><li class="ref__authors__name">Ezagouri M. </li><li class="ref__authors__name">Dov A. </li><li class="ref__authors__name">Ashkenazi K. </li><li class="ref__authors__name">Gefen N. </li><li class="ref__authors__name">Izraeli S. </li><li class="ref__authors__name">Rechavi G. </li><li class="ref__authors__name">Pass H. </li><li>et al.</li></ul><div class="ref__title">Diagnostic assay based on hsa-miR-205 expression distinguishes squamous from nonsquamous non-small-cell lung carcinoma.</div><span class="ref__series"><i>J. Clin. Oncol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2030-2037</span><div class="ref__controls"><a href="#back-bib97" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-65349166324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (318)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19273703&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FJCO.2008.19.4134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2009&pages=2030-2037&journal=J.%C2%A0Clin.+Oncol.&author=Lebanony+D.&author=Benjamin+H.&author=Gilad+S.&author=Ezagouri+M.&author=Dov+A.&author=Ashkenazi+K.&author=Gefen+N.&author=Izraeli+S.&author=Rechavi+G.&author=Pass+H.&title=Diagnostic+assay+based+on+hsa-miR-205+expression+distinguishes+squamous+from+nonsquamous+non-small-cell+lung+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lebarbier E. </li></ul><div class="ref__title">Detecting multiple change-points in the mean of Gaussian process by model selection.</div><span class="ref__series"><i>Signal Processing.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">85</span></span>: <span class="ref__seriesPages">717-736</span><div class="ref__controls"><a href="#back-bib98" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-13844315857&amp;cf=" onclick="newWindow(this.href);return false">Scopus (103)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.sigpro.2004.11.012&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2005&pages=717-736&journal=Signal+Processing&author=Lebarbier+E.&title=Detecting+multiple+change-points+in+the+mean+of+Gaussian+process+by+model+selection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lewandowski K.T. </li><li class="ref__authors__name">Piwnica-Worms H. </li></ul><div class="ref__title">Phosphorylation of the E3 ubiquitin ligase RNF41 by the kinase Par-1b is required for epithelial cell polarity.</div><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">127</span></span>: <span class="ref__seriesPages">315-327</span><div class="ref__controls"><a href="#back-bib99" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84892555914&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24259665&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.129148&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2014&pages=315-327&journal=J.%C2%A0Cell+Sci.&author=Lewandowski+K.T.&author=Piwnica-Worms+H.&title=Phosphorylation+of+the+E3+ubiquitin+ligase+RNF41+by+the+kinase+Par-1b+is+required+for+epithelial+cell+polarity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li M. </li><li class="ref__authors__name">Xie X. </li><li class="ref__authors__name">Zhou J. </li><li class="ref__authors__name">Sheng M. </li><li class="ref__authors__name">Yin X. </li><li class="ref__authors__name">Ko E.-A. </li><li class="ref__authors__name">Zhou T. </li><li class="ref__authors__name">Gu W. </li></ul><div class="ref__title">Quantifying circular RNA expression from RNA-seq data using model-based framework.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span><div class="ref__controls"><a href="#back-bib100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (27)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinformatics&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li F. </li><li class="ref__authors__name">Aljahdali I. </li><li class="ref__authors__name">Ling X. </li></ul><div class="ref__title">Cancer therapeutics using survivin BIRC5 as a target: what can we do after over two decades of study?.</div><span class="ref__series"><i>J. Exp. Clin. Cancer Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">368</span><div class="ref__controls"><a href="#back-bib101" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071240357&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31439015&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13046-019-1362-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=368&journal=J.%C2%A0Exp.+Clin.+Cancer+Res.&author=Li+F.&author=Aljahdali+I.&author=Ling+X.&title=Cancer+therapeutics+using+survivin+BIRC5+as+a+target%3A+what+can+we+do+after+over+two+decades+of+study%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li K. </li><li class="ref__authors__name">Vaudel M. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Ren Y. </li><li class="ref__authors__name">Wen B. </li></ul><div class="ref__title">PDV: an integrative proteomics data viewer.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">1249-1251</span><div class="ref__controls"><a href="#back-bib102" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85059503259&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30169737&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty770&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2019&pages=1249-1251&journal=Bioinformatics&author=Li+K.&author=Vaudel+M.&author=Zhang+B.&author=Ren+Y.&author=Wen+B.&title=PDV%3A+an+integrative+proteomics+data+viewer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li P. </li><li class="ref__authors__name">Cao G. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi J. </li><li class="ref__authors__name">Cai K. </li><li class="ref__authors__name">Zhen L. </li><li class="ref__authors__name">He X. </li><li class="ref__authors__name">Zhou Y. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Zhu Y. </li><li>et al.</li></ul><div class="ref__title">FHL3 promotes pancreatic cancer invasion and metastasis through preventing the ubiquitination degradation of EMT associated transcription factors.</div><span class="ref__series"><i>Aging (Albany N.Y.).</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">53-69</span><div class="ref__controls"><a href="#back-bib103" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31935687&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=53-69&journal=Aging+%28Albany+N.Y.%29&author=Li+P.&author=Cao+G.&author=Zhang+Y.&author=Shi+J.&author=Cai+K.&author=Zhen+L.&author=He+X.&author=Zhou+Y.&author=Li+Y.&author=Zhu+Y.&title=FHL3+promotes+pancreatic+cancer+invasion+and+metastasis+through+preventing+the+ubiquitination+degradation+of+EMT+associated+transcription+factors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liang Y. </li><li class="ref__authors__name">Liu M. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Ding X. </li><li class="ref__authors__name">Cao Y. </li></ul><div class="ref__title">Analysis of 20 genes at chromosome band 12q13: RACGAP1 and MCRS1 overexpression in nonsmall-cell lung cancer.</div><span class="ref__series"><i>Genes Chromosomes Cancer.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">52</span></span>: <span class="ref__seriesPages">305-315</span><div class="ref__controls"><a href="#back-bib104" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84872424747&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23225332&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fgcc.22030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=305-315&journal=Genes+Chromosomes+Cancer&author=Liang+Y.&author=Liu+M.&author=Wang+P.&author=Ding+X.&author=Cao+Y.&title=Analysis+of+20+genes+at+chromosome+band+12q13%3A+RACGAP1+and+MCRS1+overexpression+in+nonsmall-cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liao Y. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">WebGestalt 2019: gene set analysis toolkit with revamped UIs and APIs.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W199-W205</span><div class="ref__controls"><a href="#back-bib105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85069234777&amp;cf=" onclick="newWindow(this.href);return false">Scopus (429)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31114916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkz401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W199-W205&journal=Nucleic+Acids+Res.&author=Liao+Y.&author=Wang+J.&author=Jaehnig+E.J.&author=Shi+Z.&author=Zhang+B.&title=WebGestalt+2019%3A+gene+set+analysis+toolkit+with+revamped+UIs+and+APIs">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Mesirov J.P. </li><li class="ref__authors__name">Tamayo P. </li></ul><div class="ref__title">The Molecular Signatures Database (MSigDB) hallmark gene set collection.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span><div class="ref__controls"><a href="#back-bib106" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1936)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lim S.H. </li><li class="ref__authors__name">Sun J.-M. </li><li class="ref__authors__name">Choi Y.-L. </li><li class="ref__authors__name">Kim H.R. </li><li class="ref__authors__name">Ahn S. </li><li class="ref__authors__name">Lee J.Y. </li><li class="ref__authors__name">Lee S.-H. </li><li class="ref__authors__name">Ahn J.S. </li><li class="ref__authors__name">Park K. </li><li class="ref__authors__name">Kim J.H. </li><li>et al.</li></ul><div class="ref__title">Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study.</div><span class="ref__series"><i>Cancer.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">122</span></span>: <span class="ref__seriesPages">3024-3031</span><div class="ref__controls"><a href="#back-bib107" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27315356&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fcncr.30135&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2016&pages=3024-3031&journal=Cancer&author=Lim+S.H.&author=Sun+J.-M.&author=Choi+Y.-L.&author=Kim+H.R.&author=Ahn+S.&author=Lee+J.Y.&author=Lee+S.-H.&author=Ahn+J.S.&author=Park+K.&author=Kim+J.H.&title=Efficacy+and+safety+of+dovitinib+in+pretreated+patients+with+advanced+squamous+non-small+cell+lung+cancer+with+FGFR1+amplification%3A+A+single-arm%2C+phase+2+study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lim Y.W. </li><li class="ref__authors__name">Chen-Harris H. </li><li class="ref__authors__name">Mayba O. </li><li class="ref__authors__name">Lianoglou S. </li><li class="ref__authors__name">Wuster A. </li><li class="ref__authors__name">Bhangale T. </li><li class="ref__authors__name">Khan Z. </li><li class="ref__authors__name">Mariathasan S. </li><li class="ref__authors__name">Daemen A. </li><li class="ref__authors__name">Reeder J. </li><li>et al.</li></ul><div class="ref__title">Germline genetic polymorphisms influence tumor gene expression and immune cell infiltration.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">115</span></span>: <span class="ref__seriesPages">E11701-E11710</span><div class="ref__controls"><a href="#back-bib108" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85058338986&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30463956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.1804506115&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E11701-E11710&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Lim+Y.W.&author=Chen-Harris+H.&author=Mayba+O.&author=Lianoglou+S.&author=Wuster+A.&author=Bhangale+T.&author=Khan+Z.&author=Mariathasan+S.&author=Daemen+A.&author=Reeder+J.&title=Germline+genetic+polymorphisms+influence+tumor+gene+expression+and+immune+cell+infiltration">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lindsay J. </li><li class="ref__authors__name">McDade S.S. </li><li class="ref__authors__name">Pickard A. </li><li class="ref__authors__name">McCloskey K.D. </li><li class="ref__authors__name">McCance D.J. </li></ul><div class="ref__title">Role of DeltaNp63gamma in epithelial to mesenchymal transition.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">286</span></span>: <span class="ref__seriesPages">3915-3924</span><div class="ref__controls"><a href="#back-bib109" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79952811643&amp;cf=" onclick="newWindow(this.href);return false">Scopus (46)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21127042&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M110.162511&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=3915-3924&journal=J.%C2%A0Biol.+Chem.&author=Lindsay+J.&author=McDade+S.S.&author=Pickard+A.&author=McCloskey+K.D.&author=McCance+D.J.&title=Role+of+DeltaNp63gamma+in+epithelial+to+mesenchymal+transition">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu X. </li><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Cui W. </li><li class="ref__authors__name">Yuan X. </li><li class="ref__authors__name">Lin L. </li><li class="ref__authors__name">Cao Q. </li><li class="ref__authors__name">Wang N. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Guo W. </li><li class="ref__authors__name">Zhang X. </li><li>et al.</li></ul><div class="ref__title">Targeting ALDH1A1 by disulfiram/copper complex inhibits non-small cell lung cancer recurrence driven by ALDH-positive cancer stem cells.</div><span class="ref__series"><i>Oncotarget.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">58516-58530</span><div class="ref__controls"><a href="#back-bib110" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84988355081&amp;cf=" onclick="newWindow(this.href);return false">Scopus (45)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27542268&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.18632%2Foncotarget.11305&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=58516-58530&journal=Oncotarget&author=Liu+X.&author=Wang+L.&author=Cui+W.&author=Yuan+X.&author=Lin+L.&author=Cao+Q.&author=Wang+N.&author=Li+Y.&author=Guo+W.&author=Zhang+X.&title=Targeting+ALDH1A1+by+disulfiram%2Fcopper+complex+inhibits+non-small+cell+lung+cancer+recurrence+driven+by+ALDH-positive+cancer+stem+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu W. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Ma S. </li><li class="ref__authors__name">Fenyö D. </li></ul><div class="ref__title">Extracting Pathway-level Signatures from Proteogenomic Data in Breast Cancer Using Independent Component Analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S169-S182</span><div class="ref__controls"><a href="#back-bib111" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071350104&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31213479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001442&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S169-S182&journal=Mol.+Cell.+Proteomics&author=Liu+W.&author=Payne+S.H.&author=Ma+S.&author=Feny%C3%B6+D.&title=Extracting+Pathway-level+Signatures+from+Proteogenomic+Data+in+Breast+Cancer+Using+Independent+Component+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lo Iacono M. </li><li class="ref__authors__name">Monica V. </li><li class="ref__authors__name">Saviozzi S. </li><li class="ref__authors__name">Ceppi P. </li><li class="ref__authors__name">Bracco E. </li><li class="ref__authors__name">Papotti M. </li><li class="ref__authors__name">Scagliotti G.V. </li></ul><div class="ref__title">p63 and p73 isoform expression in non-small cell lung cancer and corresponding morphological normal lung tissue.</div><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">473-481</span><div class="ref__controls"><a href="#back-bib112" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79951770197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21289519&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e31820b86b0&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=473-481&journal=J.%C2%A0Thorac.+Oncol.&author=Lo+Iacono+M.&author=Monica+V.&author=Saviozzi+S.&author=Ceppi+P.&author=Bracco+E.&author=Papotti+M.&author=Scagliotti+G.V.&title=p63+and+p73+isoform+expression+in+non-small+cell+lung+cancer+and+corresponding+morphological+normal+lung+tissue">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Maksimovic J. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Oshlack A. </li></ul><div class="ref__title">A cross-package Bioconductor workflow for analysing methylation array data.</div><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1281</span><div class="ref__controls"><a href="#back-bib113" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27347385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.12688%2Ff1000research.8839.1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1281&journal=F1000Res.&author=Maksimovic+J.&author=Phipson+B.&author=Oshlack+A.&title=A+cross-package+Bioconductor+workflow+for+analysing+methylation+array+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Malhotra D. </li><li class="ref__authors__name">Portales-Casamar E. </li><li class="ref__authors__name">Singh A. </li><li class="ref__authors__name">Srivastava S. </li><li class="ref__authors__name">Arenillas D. </li><li class="ref__authors__name">Happel C. </li><li class="ref__authors__name">Shyr C. </li><li class="ref__authors__name">Wakabayashi N. </li><li class="ref__authors__name">Kensler T.W. </li><li class="ref__authors__name">Wasserman W.W. </li><li class="ref__authors__name">Biswal S. </li></ul><div class="ref__title">Global mapping of binding sites for Nrf2 identifies novel targets in cell survival response through ChIP-Seq profiling and network analysis.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">5718-5734</span><div class="ref__controls"><a href="#back-bib114" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77957237159&amp;cf=" onclick="newWindow(this.href);return false">Scopus (470)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20460467&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkq212&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=5718-5734&journal=Nucleic+Acids+Res.&author=Malhotra+D.&author=Portales-Casamar+E.&author=Singh+A.&author=Srivastava+S.&author=Arenillas+D.&author=Happel+C.&author=Shyr+C.&author=Wakabayashi+N.&author=Kensler+T.W.&author=Wasserman+W.W.&author=Biswal+S.&title=Global+mapping+of+binding+sites+for+Nrf2+identifies+novel+targets+in+cell+survival+response+through+ChIP-Seq+profiling+and+network+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Maynard M. </li><li class="ref__authors__name">Kothadia R. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Christianson K.E. </li><li class="ref__authors__name">Heiman D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Getz G. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">PANOPLY: A cloud-based platform for automated and reproducible proteogenomic data analysis.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">580-582</span><div class="ref__controls"><a href="#back-bib115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=580-582&journal=Nature&author=Mani+D.R.&author=Maynard+M.&author=Kothadia+R.&author=Krug+K.&author=Christianson+K.E.&author=Heiman+D.&author=Clauser+K.R.&author=Birger+C.&author=Getz+G.&author=Carr+S.A.&title=PANOPLY%3A+A+cloud-based+platform+for+automated+and+reproducible+proteogenomic+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Maynard M. </li><li class="ref__authors__name">Kothadia R. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Christianson K.E. </li><li class="ref__authors__name">Heiman D. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Getz G. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">PANOPLY: a cloud-based platform for automated and reproducible proteogenomic data analysis.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">580-582</span><div class="ref__controls"><a href="#back-bib216" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85106486253&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=34040252&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41592-021-01176-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2021&pages=580-582&journal=Nat.+Methods&author=Mani+D.R.&author=Maynard+M.&author=Kothadia+R.&author=Krug+K.&author=Christianson+K.E.&author=Heiman+D.&author=Clauser+K.R.&author=Birger+C.&author=Getz+G.&author=Carr+S.A.&title=PANOPLY%3A+a+cloud-based+platform+for+automated+and+reproducible+proteogenomic+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McInnes L. </li><li class="ref__authors__name">Healy J. </li><li class="ref__authors__name">Astels S. </li></ul><div class="ref__title">hdbscan: Hierarchical density based clustering.</div><span class="ref__series"><i>Journal of Open Source Software.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">205</span><div class="ref__controls"><a href="#back-bib116" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.21105%2Fjoss.00205&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=205&journal=Journal+of+Open+Source+Software&author=McInnes+L.&author=Healy+J.&author=Astels+S.&title=hdbscan%3A+Hierarchical+density+based+clustering">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McLaren W. </li><li class="ref__authors__name">Gil L. </li><li class="ref__authors__name">Hunt S.E. </li><li class="ref__authors__name">Riat H.S. </li><li class="ref__authors__name">Ritchie G.R.S. </li><li class="ref__authors__name">Thormann A. </li><li class="ref__authors__name">Flicek P. </li><li class="ref__authors__name">Cunningham F. </li></ul><div class="ref__title">The Ensembl Variant Effect Predictor.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">122</span><div class="ref__controls"><a href="#back-bib117" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84973338712&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1715)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27268795&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-016-0974-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=122&journal=Genome+Biol.&author=McLaren+W.&author=Gil+L.&author=Hunt+S.E.&author=Riat+H.S.&author=Ritchie+G.R.S.&author=Thormann+A.&author=Flicek+P.&author=Cunningham+F.&title=The+Ensembl+Variant+Effect+Predictor">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Medvar B. </li><li class="ref__authors__name">Raghuram V. </li><li class="ref__authors__name">Pisitkun T. </li><li class="ref__authors__name">Sarkar A. </li><li class="ref__authors__name">Knepper M.A. </li></ul><div class="ref__title">Comprehensive database of human E3 ubiquitin ligases: application to aquaporin-2 regulation.</div><span class="ref__series"><i>Physiol. Genomics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">502-512</span><div class="ref__controls"><a href="#back-bib118" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84983739108&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27199454&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1152%2Fphysiolgenomics.00031.2016&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=502-512&journal=Physiol.+Genomics&author=Medvar+B.&author=Raghuram+V.&author=Pisitkun+T.&author=Sarkar+A.&author=Knepper+M.A.&title=Comprehensive+database+of+human+E3+ubiquitin+ligases%3A+application+to+aquaporin-2+regulation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Merico D. </li><li class="ref__authors__name">Isserlin R. </li><li class="ref__authors__name">Stueker O. </li><li class="ref__authors__name">Emili A. </li><li class="ref__authors__name">Bader G.D. </li></ul><div class="ref__title">Enrichment map: a network-based method for gene-set enrichment visualization and interpretation.</div><span class="ref__series"><i>PLoS ONE.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">e13984</span><div class="ref__controls"><a href="#back-bib119" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-78649775562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1020)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21085593&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1371%2Fjournal.pone.0013984&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e13984&journal=PLoS+ONE&author=Merico+D.&author=Isserlin+R.&author=Stueker+O.&author=Emili+A.&author=Bader+G.D.&title=Enrichment+map%3A+a+network-based+method+for+gene-set+enrichment+visualization+and+interpretation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1222)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib217" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (717)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=Mertins+P.&author=Mani+D.R.&author=Ruggles+K.V.&author=Gillette+M.A.&author=Clauser+K.R.&author=Wang+P.&author=Wang+X.&author=Qiao+J.W.&author=Cao+S.&author=Petralia+F.&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Tang L.C. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Clauss T.R. </li><li class="ref__authors__name">Shah P. </li><li class="ref__authors__name">Gillette M.A. </li><li>et al.</li></ul><div class="ref__title">Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry.</div><span class="ref__series"><i>Nat. Protoc.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1632-1661</span><div class="ref__controls"><a href="#back-bib218" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85049660247&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29988108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41596-018-0006-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=1632-1661&journal=Nat.+Protoc.&author=Mertins+P.&author=Tang+L.C.&author=Krug+K.&author=Clark+D.J.&author=Gritsenko+M.A.&author=Chen+L.&author=Clauser+K.R.&author=Clauss+T.R.&author=Shah+P.&author=Gillette+M.A.&title=Reproducible+workflow+for+multiplexed+deep-scale+proteome+and+phosphoproteome+analysis+of+tumor+tissues+by+liquid+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Middleton G. </li><li class="ref__authors__name">Fletcher P. </li><li class="ref__authors__name">Popat S. </li><li class="ref__authors__name">Savage J. </li><li class="ref__authors__name">Summers Y. </li><li class="ref__authors__name">Greystoke A. </li><li class="ref__authors__name">Gilligan D. </li><li class="ref__authors__name">Cave J. </li><li class="ref__authors__name">O’Rourke N. </li><li class="ref__authors__name">Brewster A. </li><li>et al.</li></ul><div class="ref__title">The National Lung Matrix Trial of personalized therapy in lung cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">583</span></span>: <span class="ref__seriesPages">807-812</span><div class="ref__controls"><a href="#back-bib121" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85087956964&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32669708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-020-2481-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2020&pages=807-812&journal=Nature&author=Middleton+G.&author=Fletcher+P.&author=Popat+S.&author=Savage+J.&author=Summers+Y.&author=Greystoke+A.&author=Gilligan+D.&author=Cave+J.&author=O%E2%80%99Rourke+N.&author=Brewster+A.&title=The+National+Lung+Matrix+Trial+of+personalized+therapy+in+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Moissoglu K. </li><li class="ref__authors__name">McRoberts K.S. </li><li class="ref__authors__name">Meier J.A. </li><li class="ref__authors__name">Theodorescu D. </li><li class="ref__authors__name">Schwartz M.A. </li></ul><div class="ref__title">Rho GDP dissociation inhibitor 2 suppresses metastasis via unconventional regulation of RhoGTPases.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">2838-2844</span><div class="ref__controls"><a href="#back-bib122" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-66149117890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (57)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19276387&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-08-1397&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2838-2844&journal=Cancer+Res.&author=Moissoglu+K.&author=McRoberts+K.S.&author=Meier+J.A.&author=Theodorescu+D.&author=Schwartz+M.A.&title=Rho+GDP+dissociation+inhibitor+2+suppresses+metastasis+via+unconventional+regulation+of+RhoGTPases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Morrison J.A. </li><li class="ref__authors__name">Pike L.A. </li><li class="ref__authors__name">Sams S.B. </li><li class="ref__authors__name">Sharma V. </li><li class="ref__authors__name">Zhou Q. </li><li class="ref__authors__name">Severson J.J. </li><li class="ref__authors__name">Tan A.-C. </li><li class="ref__authors__name">Wood W.M. </li><li class="ref__authors__name">Haugen B.R. </li></ul><div class="ref__title">Thioredoxin interacting protein (TXNIP) is a novel tumor suppressor in thyroid cancer.</div><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">62</span><div class="ref__controls"><a href="#back-bib123" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84899506046&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24645981&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1476-4598-13-62&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=62&journal=Mol.+Cancer&author=Morrison+J.A.&author=Pike+L.A.&author=Sams+S.B.&author=Sharma+V.&author=Zhou+Q.&author=Severson+J.J.&author=Tan+A.-C.&author=Wood+W.M.&author=Haugen+B.R.&title=Thioredoxin+interacting+protein+%28TXNIP%29+is+a+novel+tumor+suppressor+in+thyroid+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mouchemore K.A. </li><li class="ref__authors__name">Sampaio N.G. </li><li class="ref__authors__name">Murrey M.W. </li><li class="ref__authors__name">Stanley E.R. </li><li class="ref__authors__name">Lannutti B.J. </li><li class="ref__authors__name">Pixley F.J. </li></ul><div class="ref__title">Specific inhibition of PI3K p110δ inhibits CSF-1-induced macrophage spreading and invasive capacity.</div><span class="ref__series"><i>FEBS J.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">280</span></span>: <span class="ref__seriesPages">5228-5236</span><div class="ref__controls"><a href="#back-bib124" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84886101666&amp;cf=" onclick="newWindow(this.href);return false">Scopus (24)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23648053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1111%2Ffebs.12316&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=5228-5236&journal=FEBS+J.&author=Mouchemore+K.A.&author=Sampaio+N.G.&author=Murrey+M.W.&author=Stanley+E.R.&author=Lannutti+B.J.&author=Pixley+F.J.&title=Specific+inhibition+of+PI3K+p110%CE%B4+inhibits+CSF-1-induced+macrophage+spreading+and+invasive+capacity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Myers S.A. </li><li class="ref__authors__name">Klaeger S. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Viner R. </li><li class="ref__authors__name">Choi J. </li><li class="ref__authors__name">Rogers J. </li><li class="ref__authors__name">Clauser K. </li><li class="ref__authors__name">Udeshi N.D. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Evaluation of Advanced Precursor Determination for Tandem Mass Tag (TMT)-Based Quantitative Proteomics across Instrument Platforms.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">542-547</span><div class="ref__controls"><a href="#back-bib219" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30351145&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=542-547&journal=J.+Proteome+Res.&author=Myers+S.A.&author=Klaeger+S.&author=Satpathy+S.&author=Viner+R.&author=Choi+J.&author=Rogers+J.&author=Clauser+K.&author=Udeshi+N.D.&author=Carr+S.A.&title=Evaluation+of+Advanced+Precursor+Determination+for+Tandem+Mass+Tag+%28TMT%29-Based+Quantitative+Proteomics+across+Instrument+Platforms">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Natan E. </li><li class="ref__authors__name">Joerger A.C. </li></ul><div class="ref__title">Structure and kinetic stability of the p63 tetramerization domain.</div><span class="ref__series"><i>J. Mol. Biol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">415</span></span>: <span class="ref__seriesPages">503-513</span><div class="ref__controls"><a href="#back-bib125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84855823856&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22100306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jmb.2011.11.007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=415&publication_year=2012&pages=503-513&journal=J.%C2%A0Mol.+Biol.&author=Natan+E.&author=Joerger+A.C.&title=Structure+and+kinetic+stability+of+the+p63+tetramerization+domain">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Newman A.M. </li><li class="ref__authors__name">Liu C.L. </li><li class="ref__authors__name">Green M.R. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Feng W. </li><li class="ref__authors__name">Xu Y. </li><li class="ref__authors__name">Hoang C.D. </li><li class="ref__authors__name">Diehn M. </li><li class="ref__authors__name">Alizadeh A.A. </li></ul><div class="ref__title">Robust enumeration of cell subsets from tissue expression profiles.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">453-457</span><div class="ref__controls"><a href="#back-bib126" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84928927858&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2201)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25822800&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnmeth.3337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=453-457&journal=Nat.+Methods&author=Newman+A.M.&author=Liu+C.L.&author=Green+M.R.&author=Gentles+A.J.&author=Feng+W.&author=Xu+Y.&author=Hoang+C.D.&author=Diehn+M.&author=Alizadeh+A.A.&title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nicorici D. </li><li class="ref__authors__name">Satalan M. </li><li class="ref__authors__name">Edgren H. </li><li class="ref__authors__name">Kangaspeska S. </li><li class="ref__authors__name">Murumagi A. </li><li class="ref__authors__name">Kallioniemi O. </li><li class="ref__authors__name">Virtanen S. </li><li class="ref__authors__name">Kilkku O. </li></ul><div class="ref__title">FusionCatcher - a tool for finding somatic fusion genes in paired-end RNA-sequencing data.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/011650">https://doi.org/10.1101/011650</a></span><div class="ref__controls"><a href="#back-bib127" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F011650&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&journal=bioRxiv&author=Nicorici+D.&author=Satalan+M.&author=Edgren+H.&author=Kangaspeska+S.&author=Murumagi+A.&author=Kallioniemi+O.&author=Virtanen+S.&author=Kilkku+O.&title=FusionCatcher+-+a+tool+for+finding+somatic+fusion+genes+in+paired-end+RNA-sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nijman S.M.B. </li><li class="ref__authors__name">Luna-Vargas M.P.A. </li><li class="ref__authors__name">Velds A. </li><li class="ref__authors__name">Brummelkamp T.R. </li><li class="ref__authors__name">Dirac A.M.G. </li><li class="ref__authors__name">Sixma T.K. </li><li class="ref__authors__name">Bernards R. </li></ul><div class="ref__title">A genomic and functional inventory of deubiquitinating enzymes.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">123</span></span>: <span class="ref__seriesPages">773-786</span><div class="ref__controls"><a href="#back-bib128" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-28344456279&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1262)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16325574&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2005.11.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2005&pages=773-786&journal=Cell&author=Nijman+S.M.B.&author=Luna-Vargas+M.P.A.&author=Velds+A.&author=Brummelkamp+T.R.&author=Dirac+A.M.G.&author=Sixma+T.K.&author=Bernards+R.&title=A+genomic+and+functional+inventory+of+deubiquitinating+enzymes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Niu D.-F. </li><li class="ref__authors__name">Kondo T. </li><li class="ref__authors__name">Nakazawa T. </li><li class="ref__authors__name">Oishi N. </li><li class="ref__authors__name">Kawasaki T. </li><li class="ref__authors__name">Mochizuki K. </li><li class="ref__authors__name">Yamane T. </li><li class="ref__authors__name">Katoh R. </li></ul><div class="ref__title">Transcription factor Runx2 is a regulator of epithelial-mesenchymal transition and invasion in thyroid carcinomas.</div><span class="ref__series"><i>Lab. Invest.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">92</span></span>: <span class="ref__seriesPages">1181-1190</span><div class="ref__controls"><a href="#back-bib129" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84864428355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (74)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22641097&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Flabinvest.2012.84&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1181-1190&journal=Lab.+Invest.&author=Niu+D.-F.&author=Kondo+T.&author=Nakazawa+T.&author=Oishi+N.&author=Kawasaki+T.&author=Mochizuki+K.&author=Yamane+T.&author=Katoh+R.&title=Transcription+factor+Runx2+is+a+regulator+of+epithelial-mesenchymal+transition+and+invasion+in+thyroid+carcinomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Obenchain V. </li><li class="ref__authors__name">Lawrence M. </li><li class="ref__authors__name">Carey V. </li><li class="ref__authors__name">Gogarten S. </li><li class="ref__authors__name">Shannon P. </li><li class="ref__authors__name">Morgan M. </li></ul><div class="ref__title">VariantAnnotation: a Bioconductor package for exploration and annotation of genetic variants.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">2076-2078</span><div class="ref__controls"><a href="#back-bib130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84901835522&amp;cf=" onclick="newWindow(this.href);return false">Scopus (128)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24681907&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtu168&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=2076-2078&journal=Bioinformatics&author=Obenchain+V.&author=Lawrence+M.&author=Carey+V.&author=Gogarten+S.&author=Shannon+P.&author=Morgan+M.&title=VariantAnnotation%3A+a+Bioconductor+package+for+exploration+and+annotation+of+genetic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Paik P.K. </li><li class="ref__authors__name">Pillai R.N. </li><li class="ref__authors__name">Lathan C.S. </li><li class="ref__authors__name">Velasco S.A. </li><li class="ref__authors__name">Papadimitrakopoulou V. </li></ul><div class="ref__title">New Treatment Options in Advanced Squamous Cell Lung Cancer.</div><span class="ref__series"><i>Am. Soc. Clin. Oncol. Educ. Book.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">e198-e206</span><div class="ref__controls"><a href="#back-bib131" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31099625&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1200%2FEDBK_237829&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=e198-e206&journal=Am.+Soc.+Clin.+Oncol.+Educ.+Book&author=Paik+P.K.&author=Pillai+R.N.&author=Lathan+C.S.&author=Velasco+S.A.&author=Papadimitrakopoulou+V.&title=New+Treatment+Options+in+Advanced+Squamous+Cell+Lung+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Parra E.R. </li><li class="ref__authors__name">Jiang M. </li><li class="ref__authors__name">Solis L. </li><li class="ref__authors__name">Mino B. </li><li class="ref__authors__name">Laberiano C. </li><li class="ref__authors__name">Hernandez S. </li><li class="ref__authors__name">Gite S. </li><li class="ref__authors__name">Verma A. </li><li class="ref__authors__name">Tetzlaff M. </li><li class="ref__authors__name">Haymaker C. </li><li>et al.</li></ul><div class="ref__title">Procedural Requirements and Recommendations for Multiplex Immunofluorescence Tyramide Signal Amplification Assays to Support Translational Oncology Studies.</div><span class="ref__series"><i>Cancers (Basel).</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">255</span><div class="ref__controls"><a href="#back-bib132" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078416940&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3390%2Fcancers12020255&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=255&journal=Cancers+%28Basel%29&author=Parra+E.R.&author=Jiang+M.&author=Solis+L.&author=Mino+B.&author=Laberiano+C.&author=Hernandez+S.&author=Gite+S.&author=Verma+A.&author=Tetzlaff+M.&author=Haymaker+C.&title=Procedural+Requirements+and+Recommendations+for+Multiplex+Immunofluorescence+Tyramide+Signal+Amplification+Assays+to+Support+Translational+Oncology+Studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Patel M. </li><li class="ref__authors__name">Lu L. </li><li class="ref__authors__name">Zander D.S. </li><li class="ref__authors__name">Sreerama L. </li><li class="ref__authors__name">Coco D. </li><li class="ref__authors__name">Moreb J.S. </li></ul><div class="ref__title">ALDH1A1 and ALDH3A1 expression in lung cancers: correlation with histologic type and potential precursors.</div><span class="ref__series"><i>Lung Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">59</span></span>: <span class="ref__seriesPages">340-349</span><div class="ref__controls"><a href="#back-bib133" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-40249108437&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17920722&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=abstract&amp;site=lung-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=fulltext&amp;site=lung-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.lungcan.2007.08.033&amp;cf=pdf&amp;site=lung-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2008&pages=340-349&journal=Lung+Cancer&author=Patel+M.&author=Lu+L.&author=Zander+D.S.&author=Sreerama+L.&author=Coco+D.&author=Moreb+J.S.&title=ALDH1A1+and+ALDH3A1+expression+in+lung+cancers%3A+correlation+with+histologic+type+and+potential+precursors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Patnaik A. </li><li class="ref__authors__name">Rosen L.S. </li><li class="ref__authors__name">Tolaney S.M. </li><li class="ref__authors__name">Tolcher A.W. </li><li class="ref__authors__name">Goldman J.W. </li><li class="ref__authors__name">Gandhi L. </li><li class="ref__authors__name">Papadopoulos K.P. </li><li class="ref__authors__name">Beeram M. </li><li class="ref__authors__name">Rasco D.W. </li><li class="ref__authors__name">Hilton J.F. </li><li>et al.</li></ul><div class="ref__title">Efficacy and Safety of Abemaciclib, an Inhibitor of CDK4 and CDK6, for Patients with Breast Cancer, Non-Small Cell Lung Cancer, and Other Solid Tumors.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">740-753</span><div class="ref__controls"><a href="#back-bib134" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84977598561&amp;cf=" onclick="newWindow(this.href);return false">Scopus (335)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27217383&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F2159-8290.CD-16-0095&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=740-753&journal=Cancer+Discov.&author=Patnaik+A.&author=Rosen+L.S.&author=Tolaney+S.M.&author=Tolcher+A.W.&author=Goldman+J.W.&author=Gandhi+L.&author=Papadopoulos+K.P.&author=Beeram+M.&author=Rasco+D.W.&author=Hilton+J.F.&title=Efficacy+and+Safety+of+Abemaciclib%2C+an+Inhibitor+of+CDK4+and+CDK6%2C+for+Patients+with+Breast+Cancer%2C+Non-Small+Cell+Lung+Cancer%2C+and+Other+Solid+Tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Payen V.L. </li><li class="ref__authors__name">Mina E. </li><li class="ref__authors__name">Van Hée V.F. </li><li class="ref__authors__name">Porporato P.E. </li><li class="ref__authors__name">Sonveaux P. </li></ul><div class="ref__title">Monocarboxylate transporters in cancer.</div><span class="ref__series"><i>Mol. Metab.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">48-66</span><div class="ref__controls"><a href="#back-bib135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077249276&amp;cf=" onclick="newWindow(this.href);return false">Scopus (88)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395464&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.molmet.2019.07.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2020&pages=48-66&journal=Mol.+Metab.&author=Payen+V.L.&author=Mina+E.&author=Van+H%C3%A9e+V.F.&author=Porporato+P.E.&author=Sonveaux+P.&title=Monocarboxylate+transporters+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Paz-Ares L. </li><li class="ref__authors__name">Luft A. </li><li class="ref__authors__name">Vicente D. </li><li class="ref__authors__name">Tafreshi A. </li><li class="ref__authors__name">Gümüş M. </li><li class="ref__authors__name">Mazières J. </li><li class="ref__authors__name">Hermes B. </li><li class="ref__authors__name">Çay Şenler F. </li><li class="ref__authors__name">Csőszi T. </li><li class="ref__authors__name">Fülöp A. </li><li>et al.</li><li>KEYNOTE-407 Investigators</li></ul><div class="ref__title">Pembrolizumab plus Chemotherapy for Squamous Non-Small-Cell Lung Cancer.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">379</span></span>: <span class="ref__seriesPages">2040-2051</span><div class="ref__controls"><a href="#back-bib136" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054684704&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1136)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30280635&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1056%2FNEJMoa1810865&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=379&publication_year=2018&pages=2040-2051&journal=N.%C2%A0Engl.+J.+Med.&author=KEYNOTE-407+Investigators&title=Pembrolizumab+plus+Chemotherapy+for+Squamous+Non-Small-Cell+Lung+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pedersen B.S. </li><li class="ref__authors__name">Layer R.M. </li><li class="ref__authors__name">Quinlan A.R. </li></ul><div class="ref__title">Vcfanno: fast, flexible annotation of genetic variants.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">118</span><div class="ref__controls"><a href="#back-bib137" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84975311524&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27250555&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs13059-016-0973-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=118&journal=Genome+Biol.&author=Pedersen+B.S.&author=Layer+R.M.&author=Quinlan+A.R.&title=Vcfanno%3A+fast%2C+flexible+annotation+of+genetic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Peng A. </li><li class="ref__authors__name">Mao X. </li><li class="ref__authors__name">Zhong J. </li><li class="ref__authors__name">Fan S. </li><li class="ref__authors__name">Hu Y. </li></ul><div class="ref__title">Single-Cell Multi-Omics and Its Prospective Application in Cancer Biology.</div><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">e1900271</span><div class="ref__controls"><a href="#back-bib138" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083998470&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32223079&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1002%2Fpmic.201900271&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=e1900271&journal=Proteomics&author=Peng+A.&author=Mao+X.&author=Zhong+J.&author=Fan+S.&author=Hu+Y.&title=Single-Cell+Multi-Omics+and+Its+Prospective+Application+in+Cancer+Biology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Peters L.A. </li><li class="ref__authors__name">Perrigoue J. </li><li class="ref__authors__name">Mortha A. </li><li class="ref__authors__name">Iuga A. </li><li class="ref__authors__name">Song W.-M. </li><li class="ref__authors__name">Neiman E.M. </li><li class="ref__authors__name">Llewellyn S.R. </li><li class="ref__authors__name">Di Narzo A. </li><li class="ref__authors__name">Kidd B.A. </li><li class="ref__authors__name">Telesco S.E. </li><li>et al.</li></ul><div class="ref__title">A functional genomics predictive network model identifies regulators of inflammatory bowel disease.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">1437-1449</span><div class="ref__controls"><a href="#back-bib139" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85030166718&amp;cf=" onclick="newWindow(this.href);return false">Scopus (69)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28892060&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_147_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3947&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1437-1449&journal=Nat.+Genet.&author=Peters+L.A.&author=Perrigoue+J.&author=Mortha+A.&author=Iuga+A.&author=Song+W.-M.&author=Neiman+E.M.&author=Llewellyn+S.R.&author=Di+Narzo+A.&author=Kidd+B.A.&author=Telesco+S.E.&title=A+functional+genomics+predictive+network+model+identifies+regulators+of+inflammatory+bowel+disease">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Peng J. </li><li class="ref__authors__name">Yan A. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">A new method for constructing tumor specific gene co-expression networks based on samples with tumor purity heterogeneity.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">i528-i536</span><div class="ref__controls"><a href="#back-bib140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85050807280&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29949994&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbty280&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=i528-i536&journal=Bioinformatics&author=Petralia+F.&author=Wang+L.&author=Peng+J.&author=Yan+A.&author=Zhu+J.&author=Wang+P.&title=A+new+method+for+constructing+tumor+specific+gene+co-expression+networks+based+on+samples+with+tumor+purity+heterogeneity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Purcell S. </li><li class="ref__authors__name">Neale B. </li><li class="ref__authors__name">Todd-Brown K. </li><li class="ref__authors__name">Thomas L. </li><li class="ref__authors__name">Ferreira M.A.R. </li><li class="ref__authors__name">Bender D. </li><li class="ref__authors__name">Maller J. </li><li class="ref__authors__name">Sklar P. </li><li class="ref__authors__name">de Bakker P.I.W. </li><li class="ref__authors__name">Daly M.J. </li><li class="ref__authors__name">Sham P.C. </li></ul><div class="ref__title">PLINK: a tool set for whole-genome association and population-based linkage analyses.</div><span class="ref__series"><i>Am. J. Hum. Genet.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">559-575</span><div class="ref__controls"><a href="#back-bib141" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-34548292504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16449)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=17701901&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=559-575&journal=Am.+J.+Hum.+Genet.&author=Purcell+S.&author=Neale+B.&author=Todd-Brown+K.&author=Thomas+L.&author=Ferreira+M.A.R.&author=Bender+D.&author=Maller+J.&author=Sklar+P.&author=de+Bakker+P.I.W.&author=Daly+M.J.&author=Sham+P.C.&title=PLINK%3A+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Qian J. </li><li class="ref__authors__name">Massion P.P. </li></ul><div class="ref__title">Role of chromosome 3q amplification in lung cancer.</div><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">212-215</span><div class="ref__controls"><a href="#back-bib142" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-43249114355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18317062&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_150_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1097%2FJTO.0b013e3181663544&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=212-215&journal=J.%C2%A0Thorac.+Oncol.&author=Qian+J.&author=Massion+P.P.&title=Role+of+chromosome+3q+amplification+in+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ramadoss S. </li><li class="ref__authors__name">Sen S. </li><li class="ref__authors__name">Ramachandran I. </li><li class="ref__authors__name">Roy S. </li><li class="ref__authors__name">Chaudhuri G. </li><li class="ref__authors__name">Farias-Eisner R. </li></ul><div class="ref__title">Lysine-specific demethylase KDM3A regulates ovarian cancer stemness and chemoresistance.</div><span class="ref__series"><i>Oncogene.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">6508</span><div class="ref__controls"><a href="#back-bib143" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85034566788&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28925393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fonc.2017.331&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=6508&journal=Oncogene&author=Ramadoss+S.&author=Sen+S.&author=Ramachandran+I.&author=Roy+S.&author=Chaudhuri+G.&author=Farias-Eisner+R.&title=Lysine-specific+demethylase+KDM3A+regulates+ovarian+cancer+stemness+and+chemoresistance">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Raudvere U. </li><li class="ref__authors__name">Kolberg L. </li><li class="ref__authors__name">Kuzmin I. </li><li class="ref__authors__name">Arak T. </li><li class="ref__authors__name">Adler P. </li><li class="ref__authors__name">Peterson H. </li><li class="ref__authors__name">Vilo J. </li></ul><div class="ref__title">g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update).</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">W191-W198</span><div class="ref__controls"><a href="#back-bib144" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85068995138&amp;cf=" onclick="newWindow(this.href);return false">Scopus (585)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31066453&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkz369&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=W191-W198&journal=Nucleic+Acids+Res.&author=Raudvere+U.&author=Kolberg+L.&author=Kuzmin+I.&author=Arak+T.&author=Adler+P.&author=Peterson+H.&author=Vilo+J.&title=g%3AProfiler%3A+a+web+server+for+functional+enrichment+analysis+and+conversions+of+gene+lists+%282019+update%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Reese S.E. </li><li class="ref__authors__name">Archer K.J. </li><li class="ref__authors__name">Therneau T.M. </li><li class="ref__authors__name">Atkinson E.J. </li><li class="ref__authors__name">Vachon C.M. </li><li class="ref__authors__name">de Andrade M. </li><li class="ref__authors__name">Kocher J.-P.A. </li><li class="ref__authors__name">Eckel-Passow J.E. </li></ul><div class="ref__title">A new statistic for identifying batch effects in high-throughput genomic data that uses guided principal component analysis.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">2877-2883</span><div class="ref__controls"><a href="#back-bib145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84890087087&amp;cf=" onclick="newWindow(this.href);return false">Scopus (67)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23958724&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtt480&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2877-2883&journal=Bioinformatics&author=Reese+S.E.&author=Archer+K.J.&author=Therneau+T.M.&author=Atkinson+E.J.&author=Vachon+C.M.&author=de+Andrade+M.&author=Kocher+J.-P.A.&author=Eckel-Passow+J.E.&title=A+new+statistic+for+identifying+batch+effects+in+high-throughput+genomic+data+that+uses+guided+principal+component+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Reva B. </li><li class="ref__authors__name">Omelchenko T. </li><li class="ref__authors__name">Calinawan A. </li><li class="ref__authors__name">Nair S. </li><li class="ref__authors__name">Schadt E. </li><li class="ref__authors__name">Tewari A. </li></ul><div class="ref__title">Prioritization prostate cancer to immune checkpoint therapy by ranking tumors along IFN-g axis and identification of immune resistance mechanisms.</div><span class="ref__series"><i>bioRxiv.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume"></span></span><span class="doi"><a href="https://doi.org/10.1101/2020.10.19.345629">https://doi.org/10.1101/2020.10.19.345629</a></span><div class="ref__controls"><a href="#back-bib146" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85098898555&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_154_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2F2020.10.19.345629&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&journal=bioRxiv&author=Reva+B.&author=Omelchenko+T.&author=Calinawan+A.&author=Nair+S.&author=Schadt+E.&author=Tewari+A.&title=Prioritization+prostate+cancer+to+immune+checkpoint+therapy+by+ranking+tumors+along+IFN-g+axis+and+identification+of+immune+resistance+mechanisms">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ritchie M.E. </li><li class="ref__authors__name">Phipson B. </li><li class="ref__authors__name">Wu D. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">Law C.W. </li><li class="ref__authors__name">Shi W. </li><li class="ref__authors__name">Smyth G.K. </li></ul><div class="ref__title">limma powers differential expression analyses for RNA-sequencing and microarray studies.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">e47</span><div class="ref__controls"><a href="#back-bib147" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84926507971&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9097)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25605792&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkv007&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=e47&journal=Nucleic+Acids+Res.&author=Ritchie+M.E.&author=Phipson+B.&author=Wu+D.&author=Hu+Y.&author=Law+C.W.&author=Shi+W.&author=Smyth+G.K.&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Robinson D.R. </li><li class="ref__authors__name">Wu Y.-M. </li><li class="ref__authors__name">Lonigro R.J. </li><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Cobain E. </li><li class="ref__authors__name">Everett J. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Rabban E. </li><li class="ref__authors__name">Kumar-Sinha C. </li><li class="ref__authors__name">Raymond V. </li><li>et al.</li></ul><div class="ref__title">Integrative clinical genomics of metastatic cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">548</span></span>: <span class="ref__seriesPages">297-303</span><div class="ref__controls"><a href="#back-bib148" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85027893400&amp;cf=" onclick="newWindow(this.href);return false">Scopus (322)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28783718&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fnature23306&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=548&publication_year=2017&pages=297-303&journal=Nature&author=Robinson+D.R.&author=Wu+Y.-M.&author=Lonigro+R.J.&author=Vats+P.&author=Cobain+E.&author=Everett+J.&author=Cao+X.&author=Rabban+E.&author=Kumar-Sinha+C.&author=Raymond+V.&title=Integrative+clinical+genomics+of+metastatic+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Roskoski Jr., R. </li></ul><div class="ref__title">The role of small molecule platelet-derived growth factor receptor (PDGFR) inhibitors in the treatment of neoplastic disorders.</div><span class="ref__series"><i>Pharmacol. Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">129</span></span>: <span class="ref__seriesPages">65-83</span><div class="ref__controls"><a href="#back-bib149" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85041552287&amp;cf=" onclick="newWindow(this.href);return false">Scopus (47)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29408302&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.phrs.2018.01.021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2018&pages=65-83&journal=Pharmacol.+Res.&author=Roskoski+R.&title=The+role+of+small+molecule+platelet-derived+growth+factor+receptor+%28PDGFR%29+inhibitors+in+the+treatment+of+neoplastic+disorders">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Jaehnig E.J. </li><li class="ref__authors__name">Krug K. </li><li class="ref__authors__name">Kim B.-J. </li><li class="ref__authors__name">Saltzman A.B. </li><li class="ref__authors__name">Chan D.W. </li><li class="ref__authors__name">Holloway K.R. </li><li class="ref__authors__name">Anurag M. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Singh P. </li><li>et al.</li></ul><div class="ref__title">Microscaled proteogenomic methods for precision oncology.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">532</span><div class="ref__controls"><a href="#back-bib150" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078365239&amp;cf=" onclick="newWindow(this.href);return false">Scopus (20)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31988290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-14381-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=532&journal=Nat.+Commun.&author=Satpathy+S.&author=Jaehnig+E.J.&author=Krug+K.&author=Kim+B.-J.&author=Saltzman+A.B.&author=Chan+D.W.&author=Holloway+K.R.&author=Anurag+M.&author=Huang+C.&author=Singh+P.&title=Microscaled+proteogenomic+methods+for+precision+oncology">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Schubert M. </li><li class="ref__authors__name">Klinger B. </li><li class="ref__authors__name">Klünemann M. </li><li class="ref__authors__name">Sieber A. </li><li class="ref__authors__name">Uhlitz F. </li><li class="ref__authors__name">Sauer S. </li><li class="ref__authors__name">Garnett M.J. </li><li class="ref__authors__name">Blüthgen N. </li><li class="ref__authors__name">Saez-Rodriguez J. </li></ul><div class="ref__title">Perturbation-response genes reveal signaling footprints in cancer gene expression.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">20</span><div class="ref__controls"><a href="#back-bib151" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85039954686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29295995&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-017-02391-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=20&journal=Nat.+Commun.&author=Schubert+M.&author=Klinger+B.&author=Kl%C3%BCnemann+M.&author=Sieber+A.&author=Uhlitz+F.&author=Sauer+S.&author=Garnett+M.J.&author=Bl%C3%BCthgen+N.&author=Saez-Rodriguez+J.&title=Perturbation-response+genes+reveal+signaling+footprints+in+cancer+gene+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Scrucca L. </li><li class="ref__authors__name">Fop M. </li><li class="ref__authors__name">Murphy T.B. </li><li class="ref__authors__name">Raftery A.E. </li></ul><div class="ref__title">mclust 5: Clustering, Classification and Density Estimation Using Gaussian Finite Mixture Models.</div><span class="ref__series"><i>The R Journal.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">289</span><div class="ref__controls"><a href="#back-bib220" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27818791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.32614%2FRJ-2016-021&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=289&journal=The+R+Journal&author=Scrucca+L.&author=Fop+M.&author=Murphy+T.B.&author=Raftery+A.E.&title=mclust+5%3A+Clustering%2C+Classification+and+Density+Estimation+Using+Gaussian+Finite+Mixture+Models">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shadforth I.P. </li><li class="ref__authors__name">Dunkley T.P.J. </li><li class="ref__authors__name">Lilley K.S. </li><li class="ref__authors__name">Bessant C. </li></ul><div class="ref__title">i-Tracker: for quantitative proteomics using iTRAQ.</div><span class="ref__series"><i>BMC Genomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">145</span><div class="ref__controls"><a href="#back-bib152" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-27744450286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (218)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16242023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2F1471-2164-6-145&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=145&journal=BMC+Genomics&author=Shadforth+I.P.&author=Dunkley+T.P.J.&author=Lilley+K.S.&author=Bessant+C.&title=i-Tracker%3A+for+quantitative+proteomics+using+iTRAQ">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shahamatdar S. </li><li class="ref__authors__name">He M.X. </li><li class="ref__authors__name">Reyna M.A. </li><li class="ref__authors__name">Gusev A. </li><li class="ref__authors__name">AlDubayan S.H. </li><li class="ref__authors__name">Van Allen E.M. </li><li class="ref__authors__name">Ramachandran S. </li></ul><div class="ref__title">Germline Features Associated with Immune Infiltration in Solid Tumors.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">2900-2908.e4</span><div class="ref__controls"><a href="#back-bib153" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85080113093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32130895&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2020.02.039&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=2900-2908.e4&journal=Cell+Rep.&author=Shahamatdar+S.&author=He+M.X.&author=Reyna+M.A.&author=Gusev+A.&author=AlDubayan+S.H.&author=Van+Allen+E.M.&author=Ramachandran+S.&title=Germline+Features+Associated+with+Immune+Infiltration+in+Solid+Tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shannon P. </li><li class="ref__authors__name">Markiel A. </li><li class="ref__authors__name">Ozier O. </li><li class="ref__authors__name">Baliga N.S. </li><li class="ref__authors__name">Wang J.T. </li><li class="ref__authors__name">Ramage D. </li><li class="ref__authors__name">Amin N. </li><li class="ref__authors__name">Schwikowski B. </li><li class="ref__authors__name">Ideker T. </li></ul><div class="ref__title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2003; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">2498-2504</span><div class="ref__controls"><a href="#back-bib154" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0242490780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17796)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=14597658&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_163_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.1239303&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&journal=Genome+Res.&author=Shannon+P.&author=Markiel+A.&author=Ozier+O.&author=Baliga+N.S.&author=Wang+J.T.&author=Ramage+D.&author=Amin+N.&author=Schwikowski+B.&author=Ideker+T.&title=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shen J. </li><li class="ref__authors__name">Ju Z. </li><li class="ref__authors__name">Zhao W. </li><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Peng Y. </li><li class="ref__authors__name">Ge Z. </li><li class="ref__authors__name">Nagel Z.D. </li><li class="ref__authors__name">Zou J. </li><li class="ref__authors__name">Wang C. </li><li class="ref__authors__name">Kapoor P. </li><li>et al.</li></ul><div class="ref__title">ARID1A deficiency promotes mutability and potentiates therapeutic antitumor immunity unleashed by immune checkpoint blockade.</div><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">556-562</span><div class="ref__controls"><a href="#back-bib155" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85046552398&amp;cf=" onclick="newWindow(this.href);return false">Scopus (153)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29736026&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41591-018-0012-z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=556-562&journal=Nat.+Med.&author=Shen+J.&author=Ju+Z.&author=Zhao+W.&author=Wang+L.&author=Peng+Y.&author=Ge+Z.&author=Nagel+Z.D.&author=Zou+J.&author=Wang+C.&author=Kapoor+P.&title=ARID1A+deficiency+promotes+mutability+and+potentiates+therapeutic+antitumor+immunity+unleashed+by+immune+checkpoint+blockade">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Showe M.K. </li><li class="ref__authors__name">Vachani A. </li><li class="ref__authors__name">Kossenkov A.V. </li><li class="ref__authors__name">Yousef M. </li><li class="ref__authors__name">Nichols C. </li><li class="ref__authors__name">Nikonova E.V. </li><li class="ref__authors__name">Chang C. </li><li class="ref__authors__name">Kucharczuk J. </li><li class="ref__authors__name">Tran B. </li><li class="ref__authors__name">Wakeam E. </li><li>et al.</li></ul><div class="ref__title">Gene expression profiles in peripheral blood mononuclear cells can distinguish patients with non-small cell lung cancer from patients with nonmalignant lung disease.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">69</span></span>: <span class="ref__seriesPages">9202-9210</span><div class="ref__controls"><a href="#back-bib156" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-73649083403&amp;cf=" onclick="newWindow(this.href);return false">Scopus (120)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19951989&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-09-1378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=9202-9210&journal=Cancer+Res.&author=Showe+M.K.&author=Vachani+A.&author=Kossenkov+A.V.&author=Yousef+M.&author=Nichols+C.&author=Nikonova+E.V.&author=Chang+C.&author=Kucharczuk+J.&author=Tran+B.&author=Wakeam+E.&title=Gene+expression+profiles+in+peripheral+blood+mononuclear+cells+can+distinguish+patients+with+non-small+cell+lung+cancer+from+patients+with+nonmalignant+lung+disease">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Siegel R.L. </li><li class="ref__authors__name">Miller K.D. </li><li class="ref__authors__name">Jemal A. </li></ul><div class="ref__title">Cancer statistics, 2020.</div><span class="ref__series"><i>CA Cancer J. Clin.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">70</span></span>: <span class="ref__seriesPages">7-30</span><div class="ref__controls"><a href="#back-bib157" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077522581&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5586)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31912902&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.3322%2Fcaac.21590&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2020&pages=7-30&journal=CA+Cancer+J.+Clin.&author=Siegel+R.L.&author=Miller+K.D.&author=Jemal+A.&title=Cancer+statistics%2C+2020">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Singh A. </li><li class="ref__authors__name">Boldin-Adamsky S. </li><li class="ref__authors__name">Thimmulappa R.K. </li><li class="ref__authors__name">Rath S.K. </li><li class="ref__authors__name">Ashush H. </li><li class="ref__authors__name">Coulter J. </li><li class="ref__authors__name">Blackford A. </li><li class="ref__authors__name">Goodman S.N. </li><li class="ref__authors__name">Bunz F. </li><li class="ref__authors__name">Watson W.H. </li><li>et al.</li></ul><div class="ref__title">RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">68</span></span>: <span class="ref__seriesPages">7975-7984</span><div class="ref__controls"><a href="#back-bib158" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-54249087596&amp;cf=" onclick="newWindow(this.href);return false">Scopus (283)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=18829555&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-08-1401&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=7975-7984&journal=Cancer+Res.&author=Singh+A.&author=Boldin-Adamsky+S.&author=Thimmulappa+R.K.&author=Rath+S.K.&author=Ashush+H.&author=Coulter+J.&author=Blackford+A.&author=Goodman+S.N.&author=Bunz+F.&author=Watson+W.H.&title=RNAi-mediated+silencing+of+nuclear+factor+erythroid-2-related+factor+2+gene+expression+in+non-small+cell+lung+cancer+inhibits+tumor+growth+and+increases+efficacy+of+chemotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sinicropi-Yao S.L. </li><li class="ref__authors__name">Amann J.M. </li><li class="ref__authors__name">Lopez D.L.Y. </li><li class="ref__authors__name">Cerciello F. </li><li class="ref__authors__name">Coombes K.R. </li><li class="ref__authors__name">Carbone D.P. </li></ul><div class="ref__title">Co-Expression Analysis Reveals Mechanisms Underlying the Varied Roles of NOTCH1 in NSCLC.</div><span class="ref__series"><i>J. Thorac. Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">223-236</span><div class="ref__controls"><a href="#back-bib159" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30408569&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=abstract&amp;site=jtho-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=fulltext&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.jtho.2018.10.162&amp;cf=pdf&amp;site=jtho-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=223-236&journal=J.%C2%A0Thorac.+Oncol.&author=Sinicropi-Yao+S.L.&author=Amann+J.M.&author=Lopez+D.L.Y.&author=Cerciello+F.&author=Coombes+K.R.&author=Carbone+D.P.&title=Co-Expression+Analysis+Reveals+Mechanisms+Underlying+the+Varied+Roles+of+NOTCH1+in+NSCLC">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sondka Z. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Ward S.A. </li><li class="ref__authors__name">Dunham I. </li><li class="ref__authors__name">Forbes S.A. </li></ul><div class="ref__title">The COSMIC Cancer Gene Census: describing genetic dysfunction across all human cancers.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">696-705</span><div class="ref__controls"><a href="#back-bib160" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85054518074&amp;cf=" onclick="newWindow(this.href);return false">Scopus (268)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30293088&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41568-018-0060-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=696-705&journal=Nat.+Rev.+Cancer&author=Sondka+Z.&author=Bamford+S.&author=Cole+C.G.&author=Ward+S.A.&author=Dunham+I.&author=Forbes+S.A.&title=The+COSMIC+Cancer+Gene+Census%3A+describing+genetic+dysfunction+across+all+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Ji J. </li><li class="ref__authors__name">Gleason K.J. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Martignetti J.A. </li><li class="ref__authors__name">Chen L.S. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">Insights into Impact of DNA Copy Number Alteration and Methylation on the Proteogenomic Landscape of Human Ovarian Cancer via a Multi-omics Integrative Analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span><div class="ref__controls"><a href="#back-bib161" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+Impact+of+DNA+Copy+Number+Alteration+and+Methylation+on+the+Proteogenomic+Landscape+of+Human+Ovarian+Cancer+via+a+Multi-omics+Integrative+Analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Spitzer M.H. </li><li class="ref__authors__name">Nolan G.P. </li></ul><div class="ref__title">Mass Cytometry: Single Cells, Many Features.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">165</span></span>: <span class="ref__seriesPages">780-791</span><div class="ref__controls"><a href="#back-bib162" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27153492&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2016.04.019&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=780-791&journal=Cell&author=Spitzer+M.H.&author=Nolan+G.P.&title=Mass+Cytometry%3A+Single+Cells%2C+Many+Features">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Srivastava K. </li><li class="ref__authors__name">Pickard A. </li><li class="ref__authors__name">Craig S.G. </li><li class="ref__authors__name">Quinn G.P. </li><li class="ref__authors__name">Lambe S.M. </li><li class="ref__authors__name">James J.A. </li><li class="ref__authors__name">McDade S.S. </li><li class="ref__authors__name">McCance D.J. </li></ul><div class="ref__title">ΔNp63γ/SRC/Slug Signaling Axis Promotes Epithelial-to-Mesenchymal Transition in Squamous Cancers.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">3917-3927</span><div class="ref__controls"><a href="#back-bib163" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85051740562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29739791&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-17-3775&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=3917-3927&journal=Clin.+Cancer+Res.&author=Srivastava+K.&author=Pickard+A.&author=Craig+S.G.&author=Quinn+G.P.&author=Lambe+S.M.&author=James+J.A.&author=McDade+S.S.&author=McCance+D.J.&title=%CE%94Np63%CE%B3%2FSRC%2FSlug+Signaling+Axis+Promotes+Epithelial-to-Mesenchymal+Transition+in+Squamous+Cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Stewart P.A. </li><li class="ref__authors__name">Welsh E.A. </li><li class="ref__authors__name">Slebos R.J.C. </li><li class="ref__authors__name">Fang B. </li><li class="ref__authors__name">Izumi V. </li><li class="ref__authors__name">Chambers M. </li><li class="ref__authors__name">Zhang G. </li><li class="ref__authors__name">Cen L. </li><li class="ref__authors__name">Pettersson F. </li><li class="ref__authors__name">Zhang Y. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic landscape of squamous cell lung cancer.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">3578</span><div class="ref__controls"><a href="#back-bib164" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85070354617&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31395880&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-11452-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3578&journal=Nat.+Commun.&author=Stewart+P.A.&author=Welsh+E.A.&author=Slebos+R.J.C.&author=Fang+B.&author=Izumi+V.&author=Chambers+M.&author=Zhang+G.&author=Cen+L.&author=Pettersson+F.&author=Zhang+Y.&title=Proteogenomic+landscape+of+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Su J. </li><li class="ref__authors__name">Morgani S.M. </li><li class="ref__authors__name">David C.J. </li><li class="ref__authors__name">Wang Q. </li><li class="ref__authors__name">Er E.E. </li><li class="ref__authors__name">Huang Y.-H. </li><li class="ref__authors__name">Basnet H. </li><li class="ref__authors__name">Zou Y. </li><li class="ref__authors__name">Shu W. </li><li class="ref__authors__name">Soni R.K. </li><li>et al.</li></ul><div class="ref__title">TGF-β orchestrates fibrogenic and developmental EMTs via the RAS effector RREB1.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">577</span></span>: <span class="ref__seriesPages">566-571</span><div class="ref__controls"><a href="#back-bib165" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85077595486&amp;cf=" onclick="newWindow(this.href);return false">Scopus (61)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31915377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-019-1897-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=577&publication_year=2020&pages=566-571&journal=Nature&author=Su+J.&author=Morgani+S.M.&author=David+C.J.&author=Wang+Q.&author=Er+E.E.&author=Huang+Y.-H.&author=Basnet+H.&author=Zou+Y.&author=Shu+W.&author=Soni+R.K.&title=TGF-%CE%B2+orchestrates+fibrogenic+and+developmental+EMTs+via+the+RAS+effector+RREB1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Narayan R. </li><li class="ref__authors__name">Corsello S.M. </li><li class="ref__authors__name">Peck D.D. </li><li class="ref__authors__name">Natoli T.E. </li><li class="ref__authors__name">Lu X. </li><li class="ref__authors__name">Gould J. </li><li class="ref__authors__name">Davis J.F. </li><li class="ref__authors__name">Tubelli A.A. </li><li class="ref__authors__name">Asiedu J.K. </li><li>et al.</li></ul><div class="ref__title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span><div class="ref__controls"><a href="#back-bib166" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (669)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Szolek A. </li><li class="ref__authors__name">Schubert B. </li><li class="ref__authors__name">Mohr C. </li><li class="ref__authors__name">Sturm M. </li><li class="ref__authors__name">Feldhahn M. </li><li class="ref__authors__name">Kohlbacher O. </li></ul><div class="ref__title">OptiType: precision HLA typing from next-generation sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span><div class="ref__controls"><a href="#back-bib167" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tam W.L. </li><li class="ref__authors__name">Lu H. </li><li class="ref__authors__name">Buikhuisen J. </li><li class="ref__authors__name">Soh B.S. </li><li class="ref__authors__name">Lim E. </li><li class="ref__authors__name">Reinhardt F. </li><li class="ref__authors__name">Wu Z.J. </li><li class="ref__authors__name">Krall J.A. </li><li class="ref__authors__name">Bierie B. </li><li class="ref__authors__name">Guo W. </li><li>et al.</li></ul><div class="ref__title">Protein kinase C α is a central signaling node and therapeutic target for breast cancer stem cells.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">347-364</span><div class="ref__controls"><a href="#back-bib168" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84883621632&amp;cf=" onclick="newWindow(this.href);return false">Scopus (208)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24029232&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccr.2013.08.005&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=347-364&journal=Cancer+Cell&author=Tam+W.L.&author=Lu+H.&author=Buikhuisen+J.&author=Soh+B.S.&author=Lim+E.&author=Reinhardt+F.&author=Wu+Z.J.&author=Krall+J.A.&author=Bierie+B.&author=Guo+W.&title=Protein+kinase+C+%CE%B1+is+a+central+signaling+node+and+therapeutic+target+for+breast+cancer+stem+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tate J.G. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Jubb H.C. </li><li class="ref__authors__name">Sondka Z. </li><li class="ref__authors__name">Beare D.M. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Boutselakis H. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Creatore C. </li><li class="ref__authors__name">Dawson E. </li><li>et al.</li></ul><div class="ref__title">COSMIC: the Catalogue Of Somatic Mutations In Cancer.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D941-D947</span><div class="ref__controls"><a href="#back-bib169" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85057093998&amp;cf=" onclick="newWindow(this.href);return false">Scopus (858)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30371878&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgky1015&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D941-D947&journal=Nucleic+Acids+Res.&author=Tate+J.G.&author=Bamford+S.&author=Jubb+H.C.&author=Sondka+Z.&author=Beare+D.M.&author=Bindal+N.&author=Boutselakis+H.&author=Cole+C.G.&author=Creatore+C.&author=Dawson+E.&title=COSMIC%3A+the+Catalogue+Of+Somatic+Mutations+In+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tecalco-Cruz A.C. </li><li class="ref__authors__name">Cruz-Ramos E. </li></ul><div class="ref__title">Protein ISGylation and free ISG15 levels are increased by interferon gamma in breast cancer cells.</div><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">499</span></span>: <span class="ref__seriesPages">973-978</span><div class="ref__controls"><a href="#back-bib170" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85045093507&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29626479&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.bbrc.2018.04.030&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2018&pages=973-978&journal=Biochem.+Biophys.+Res.+Commun.&author=Tecalco-Cruz+A.C.&author=Cruz-Ramos+E.&title=Protein+ISGylation+and+free+ISG15+levels+are+increased+by+interferon+gamma+in+breast+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tellez C.S. </li><li class="ref__authors__name">Juri D.E. </li><li class="ref__authors__name">Do K. </li><li class="ref__authors__name">Bernauer A.M. </li><li class="ref__authors__name">Thomas C.L. </li><li class="ref__authors__name">Damiani L.A. </li><li class="ref__authors__name">Tessema M. </li><li class="ref__authors__name">Leng S. </li><li class="ref__authors__name">Belinsky S.A. </li></ul><div class="ref__title">EMT and stem cell-like properties associated with miR-205 and miR-200 epigenetic silencing are early manifestations during carcinogen-induced transformation of human lung epithelial cells.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">71</span></span>: <span class="ref__seriesPages">3087-3097</span><div class="ref__controls"><a href="#back-bib171" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-79954576652&amp;cf=" onclick="newWindow(this.href);return false">Scopus (238)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=21363915&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F0008-5472.CAN-10-3035&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3087-3097&journal=Cancer+Res.&author=Tellez+C.S.&author=Juri+D.E.&author=Do+K.&author=Bernauer+A.M.&author=Thomas+C.L.&author=Damiani+L.A.&author=Tessema+M.&author=Leng+S.&author=Belinsky+S.A.&title=EMT+and+stem+cell-like+properties+associated+with+miR-205+and+miR-200+epigenetic+silencing+are+early+manifestations+during+carcinogen-induced+transformation+of+human+lung+epithelial+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Thorsson V. </li><li class="ref__authors__name">Gibbs D.L. </li><li class="ref__authors__name">Brown S.D. </li><li class="ref__authors__name">Wolf D. </li><li class="ref__authors__name">Bortone D.S. </li><li class="ref__authors__name">Ou Yang T.-H. </li><li class="ref__authors__name">Porta-Pardo E. </li><li class="ref__authors__name">Gao G.F. </li><li class="ref__authors__name">Plaisier C.L. </li><li class="ref__authors__name">Eddy J.A. </li><li>et al.</li><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">The Immune Landscape of Cancer.</div><span class="ref__series"><i>Immunity.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">812-830.e14</span><div class="ref__controls"><a href="#back-bib172" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85044934017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1025)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29628290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.immuni.2018.03.023&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=812-830.e14&journal=Immunity&author=Cancer+Genome+Atlas+Research+Network&title=The+Immune+Landscape+of+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tian J. </li><li class="ref__authors__name">Cai Y. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Lu Z. </li><li class="ref__authors__name">Huang J. </li><li class="ref__authors__name">Deng Y. </li><li class="ref__authors__name">Yang N. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Ying P. </li><li class="ref__authors__name">Zhang S. </li><li>et al.</li></ul><div class="ref__title">CancerImmunityQTL: a database to systematically evaluate the impact of genetic variants on immune infiltration in human cancer.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">49</span></span>: <span class="ref__seriesPages">D1065-D1073</span><div class="ref__controls"><a href="#back-bib173" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85099428264&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33010176&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkaa805&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2021&pages=D1065-D1073&journal=Nucleic+Acids+Res.&author=Tian+J.&author=Cai+Y.&author=Li+Y.&author=Lu+Z.&author=Huang+J.&author=Deng+Y.&author=Yang+N.&author=Wang+X.&author=Ying+P.&author=Zhang+S.&title=CancerImmunityQTL%3A+a+database+to+systematically+evaluate+the+impact+of+genetic+variants+on+immune+infiltration+in+human+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Timperi E. </li><li class="ref__authors__name">Focaccetti C. </li><li class="ref__authors__name">Gallerano D. </li><li class="ref__authors__name">Panetta M. </li><li class="ref__authors__name">Spada S. </li><li class="ref__authors__name">Gallo E. </li><li class="ref__authors__name">Visca P. </li><li class="ref__authors__name">Venuta F. </li><li class="ref__authors__name">Diso D. </li><li class="ref__authors__name">Prelaj A. </li><li>et al.</li></ul><div class="ref__title">IL-18 receptor marks functional CD8<sup>+</sup> T cells in non-small cell lung cancer.</div><span class="ref__series"><i>OncoImmunology.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">e1328337</span><div class="ref__controls"><a href="#back-bib174" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85021447409&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28811967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1080%2F2162402X.2017.1328337&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=e1328337&journal=OncoImmunology&author=Timperi+E.&author=Focaccetti+C.&author=Gallerano+D.&author=Panetta+M.&author=Spada+S.&author=Gallo+E.&author=Visca+P.&author=Venuta+F.&author=Diso+D.&author=Prelaj+A.&title=IL-18+receptor+marks+functional+CD8%2B+T%C2%A0cells+in+non-small+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tran M.N. </li><li class="ref__authors__name">Choi W. </li><li class="ref__authors__name">Wszolek M.F. </li><li class="ref__authors__name">Navai N. </li><li class="ref__authors__name">Lee I.-L.C. </li><li class="ref__authors__name">Nitti G. </li><li class="ref__authors__name">Wen S. </li><li class="ref__authors__name">Flores E.R. </li><li class="ref__authors__name">Siefker-Radtke A. </li><li class="ref__authors__name">Czerniak B. </li><li>et al.</li></ul><div class="ref__title">The p63 protein isoform ΔNp63α inhibits epithelial-mesenchymal transition in human bladder cancer cells: role of MIR-205.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">288</span></span>: <span class="ref__seriesPages">3275-3288</span><div class="ref__controls"><a href="#back-bib175" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84873304587&amp;cf=" onclick="newWindow(this.href);return false">Scopus (94)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=23239884&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_184_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fjbc.M112.408104&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=3275-3288&journal=J.%C2%A0Biol.+Chem.&author=Tran+M.N.&author=Choi+W.&author=Wszolek+M.F.&author=Navai+N.&author=Lee+I.-L.C.&author=Nitti+G.&author=Wen+S.&author=Flores+E.R.&author=Siefker-Radtke+A.&author=Czerniak+B.&title=The+p63+protein+isoform+%CE%94Np63%CE%B1+inhibits+epithelial-mesenchymal+transition+in+human+bladder+cancer+cells%3A+role+of+MIR-205">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tsherniak A. </li><li class="ref__authors__name">Vazquez F. </li><li class="ref__authors__name">Montgomery P.G. </li><li class="ref__authors__name">Weir B.A. </li><li class="ref__authors__name">Kryukov G. </li><li class="ref__authors__name">Cowley G.S. </li><li class="ref__authors__name">Gill S. </li><li class="ref__authors__name">Harrington W.F. </li><li class="ref__authors__name">Pantel S. </li><li class="ref__authors__name">Krill-Burger J.M. </li><li>et al.</li></ul><div class="ref__title">Defining a Cancer Dependency Map.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">170</span></span>: <span class="ref__seriesPages">564-576.e16</span><div class="ref__controls"><a href="#back-bib176" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85026422523&amp;cf=" onclick="newWindow(this.href);return false">Scopus (541)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28753430&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_185_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2017.06.010&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=564-576.e16&journal=Cell&author=Tsherniak+A.&author=Vazquez+F.&author=Montgomery+P.G.&author=Weir+B.A.&author=Kryukov+G.&author=Cowley+G.S.&author=Gill+S.&author=Harrington+W.F.&author=Pantel+S.&author=Krill-Burger+J.M.&title=Defining+a+Cancer+Dependency+Map">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Udeshi N.D. </li><li class="ref__authors__name">Mani D.C. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Fereshetian S. </li><li class="ref__authors__name">Gasser J.A. </li><li class="ref__authors__name">Svinkina T. </li><li class="ref__authors__name">Olive M.E. </li><li class="ref__authors__name">Ebert B.L. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Carr S.A. </li></ul><div class="ref__title">Rapid and deep-scale ubiquitylation profiling for biology and translational research.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">359</span><div class="ref__controls"><a href="#back-bib177" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078023220&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31953384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_186_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-14175-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=359&journal=Nat.+Commun.&author=Udeshi+N.D.&author=Mani+D.C.&author=Satpathy+S.&author=Fereshetian+S.&author=Gasser+J.A.&author=Svinkina+T.&author=Olive+M.E.&author=Ebert+B.L.&author=Mertins+P.&author=Carr+S.A.&title=Rapid+and+deep-scale+ubiquitylation+profiling+for+biology+and+translational+research">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Uhlén M. </li><li class="ref__authors__name">Björling E. </li><li class="ref__authors__name">Agaton C. </li><li class="ref__authors__name">Szigyarto C.A.-K. </li><li class="ref__authors__name">Amini B. </li><li class="ref__authors__name">Andersen E. </li><li class="ref__authors__name">Andersson A.-C. </li><li class="ref__authors__name">Angelidou P. </li><li class="ref__authors__name">Asplund A. </li><li class="ref__authors__name">Asplund C. </li><li>et al.</li></ul><div class="ref__title">A human protein atlas for normal and cancer tissues based on antibody proteomics.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1920-1932</span><div class="ref__controls"><a href="#back-bib178" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-29244477844&amp;cf=" onclick="newWindow(this.href);return false">Scopus (875)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16127175&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_187_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.M500279-MCP200&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1920-1932&journal=Mol.+Cell.+Proteomics&author=Uhl%C3%A9n+M.&author=Bj%C3%B6rling+E.&author=Agaton+C.&author=Szigyarto+C.A.-K.&author=Amini+B.&author=Andersen+E.&author=Andersson+A.-C.&author=Angelidou+P.&author=Asplund+A.&author=Asplund+C.&title=A+human+protein+atlas+for+normal+and+cancer+tissues+based+on+antibody+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Uhrig S. </li><li class="ref__authors__name">Ellermann J. </li><li class="ref__authors__name">Walther T. </li><li class="ref__authors__name">Burkhardt P. </li><li class="ref__authors__name">Fröhlich M. </li><li class="ref__authors__name">Hutter B. </li><li class="ref__authors__name">Toprak U.H. </li><li class="ref__authors__name">Neumann O. </li><li class="ref__authors__name">Stenzinger A. </li><li class="ref__authors__name">Scholl C. </li><li>et al.</li></ul><div class="ref__title">Accurate and efficient detection of gene fusions from RNA sequencing data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">448-460</span><div class="ref__controls"><a href="#back-bib221" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33441414&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_188_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.257246.119&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2021&pages=448-460&journal=Genome+Res&author=Uhrig+S.&author=Ellermann+J.&author=Walther+T.&author=Burkhardt+P.&author=Fr%C3%B6hlich+M.&author=Hutter+B.&author=Toprak+U.H.&author=Neumann+O.&author=Stenzinger+A.&author=Scholl+C.&title=Accurate+and+efficient+detection+of+gene+fusions+from+RNA+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Straub P. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span><div class="ref__controls"><a href="#back-bib179" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (415)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_189_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O.A. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib180" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (136)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_190_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+Analysis+of+Human+Colon+Cancer+Reveals+New+Therapeutic+Opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Chinnaiyan A.M. </li><li class="ref__authors__name">Kumar-Sinha C. </li></ul><div class="ref__title">Case Study: Systematic Detection and Prioritization of Gene Fusions in Cancer by RNA-Seq: A DIY Toolkit.</div><span class="ref__series"><i>Methods Mol. Biol.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">2079</span></span>: <span class="ref__seriesPages">69-79</span><div class="ref__controls"><a href="#back-bib181" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85075114327&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31728962&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_191_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1007%2F978-1-4939-9904-0_5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2079&publication_year=2020&pages=69-79&journal=Methods+Mol.+Biol.&author=Vats+P.&author=Chinnaiyan+A.M.&author=Kumar-Sinha+C.&title=Case+Study%3A+Systematic+Detection+and+Prioritization+of+Gene+Fusions+in+Cancer+by+RNA-Seq%3A+A+DIY+Toolkit">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">von Klot C.-A. </li><li class="ref__authors__name">Dubrowinskaja N. </li><li class="ref__authors__name">Peters I. </li><li class="ref__authors__name">Hennenlotter J. </li><li class="ref__authors__name">Merseburger A.S. </li><li class="ref__authors__name">Stenzl A. </li><li class="ref__authors__name">Kuczyk M.A. </li><li class="ref__authors__name">Serth J. </li></ul><div class="ref__title">Rho GDP dissociation inhibitor-β in renal cell carcinoma.</div><span class="ref__series"><i>Oncol. Lett.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">8190-8196</span><div class="ref__controls"><a href="#back-bib182" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_192_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29250194&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=8190-8196&journal=Oncol.+Lett.&author=von+Klot+C.-A.&author=Dubrowinskaja+N.&author=Peters+I.&author=Hennenlotter+J.&author=Merseburger+A.S.&author=Stenzl+A.&author=Kuczyk+M.A.&author=Serth+J.&title=Rho+GDP+dissociation+inhibitor-%CE%B2+in+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vosgha H. </li><li class="ref__authors__name">Ariana A. </li><li class="ref__authors__name">Smith R.A. </li><li class="ref__authors__name">Lam A.K.-Y. </li></ul><div class="ref__title"><em>miR-205</em> targets angiogenesis and EMT concurrently in anaplastic thyroid carcinoma.</div><span class="ref__series"><i>Endocr. Relat. Cancer.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">323-337</span><div class="ref__controls"><a href="#back-bib183" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85042625022&amp;cf=" onclick="newWindow(this.href);return false">Scopus (32)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29317480&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_193_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1530%2FERC-17-0497&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=323-337&journal=Endocr.+Relat.+Cancer&author=Vosgha+H.&author=Ariana+A.&author=Smith+R.A.&author=Lam+A.K.-Y.&title=miR-205+targets+angiogenesis+and+EMT+concurrently+in+anaplastic+thyroid+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Walser T. </li><li class="ref__authors__name">Cui X. </li><li class="ref__authors__name">Yanagawa J. </li><li class="ref__authors__name">Lee J.M. </li><li class="ref__authors__name">Heinrich E. </li><li class="ref__authors__name">Lee G. </li><li class="ref__authors__name">Sharma S. </li><li class="ref__authors__name">Dubinett S.M. </li></ul><div class="ref__title">Smoking and lung cancer: the role of inflammation.</div><span class="ref__series"><i>Proc. Am. Thorac. Soc.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">811-815</span><div class="ref__controls"><a href="#back-bib184" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-57349193494&amp;cf=" onclick="newWindow(this.href);return false">Scopus (129)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19017734&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_194_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1513%2Fpats.200809-100TH&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=811-815&journal=Proc.+Am.+Thorac.+Soc.&author=Walser+T.&author=Cui+X.&author=Yanagawa+J.&author=Lee+J.M.&author=Heinrich+E.&author=Lee+G.&author=Sharma+S.&author=Dubinett+S.M.&title=Smoking+and+lung+cancer%3A+the+role+of+inflammation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang I.-M. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Yang X. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Stepaniants S. </li><li class="ref__authors__name">Zhang C. </li><li class="ref__authors__name">Meng Q. </li><li class="ref__authors__name">Peters M. </li><li class="ref__authors__name">He Y. </li><li class="ref__authors__name">Ni C. </li><li>et al.</li></ul><div class="ref__title">Systems analysis of eleven rodent disease models reveals an inflammatome signature and key drivers.</div><span class="ref__series"><i>Mol. Syst. Biol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">594</span><div class="ref__controls"><a href="#back-bib185" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84864208657&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22806142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_195_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fmsb.2012.24&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=594&journal=Mol.+Syst.+Biol.&author=Wang+I.-M.&author=Zhang+B.&author=Yang+X.&author=Zhu+J.&author=Stepaniants+S.&author=Zhang+C.&author=Meng+Q.&author=Peters+M.&author=He+Y.&author=Ni+C.&title=Systems+analysis+of+eleven+rodent+disease+models+reveals+an+inflammatome+signature+and+key+drivers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang S. </li><li class="ref__authors__name">Jiang B. </li><li class="ref__authors__name">Zhang T. </li><li class="ref__authors__name">Liu L. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Wang Y. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Lin H. </li><li class="ref__authors__name">Zhou L. </li><li class="ref__authors__name">Xia Y. </li><li>et al.</li></ul><div class="ref__title">Insulin and mTOR Pathway Regulate HDAC3-Mediated Deacetylation and Activation of PGK1.</div><span class="ref__series"><i>PLoS Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">e1002243</span><div class="ref__controls"><a href="#back-bib186" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84943165788&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26356530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_196_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1371%2Fjournal.pbio.1002243&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=e1002243&journal=PLoS+Biol.&author=Wang+S.&author=Jiang+B.&author=Zhang+T.&author=Liu+L.&author=Wang+Y.&author=Wang+Y.&author=Chen+X.&author=Lin+H.&author=Zhou+L.&author=Xia+Y.&title=Insulin+and+mTOR+Pathway+Regulate+HDAC3-Mediated+Deacetylation+and+Activation+of+PGK1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Greer M. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span><div class="ref__controls"><a href="#back-bib187" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (470)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_197_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt+2017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang L.-B. </li><li class="ref__authors__name">Karpova A. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Kyle J.E. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Rykunov D. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Rothstein J.H. </li><li class="ref__authors__name">Hong R. </li><li>et al.</li><li>Clinical Proteomic Tumor Analysis Consortium</li></ul><div class="ref__title">Proteogenomic and metabolomic characterization of human glioblastoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">39</span></span>: <span class="ref__seriesPages">509-528.e20</span><div class="ref__controls"><a href="#back-bib188" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85101666510&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33577785&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_198_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.ccell.2021.01.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2021&pages=509-528.e20&journal=Cancer+Cell&author=Clinical+Proteomic+Tumor+Analysis+Consortium&title=Proteogenomic+and+metabolomic+characterization+of+human+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Weeden C.E. </li><li class="ref__authors__name">Solomon B. </li><li class="ref__authors__name">Asselin-Labat M.-L. </li></ul><div class="ref__title">FGFR1 inhibition in lung squamous cell carcinoma: questions and controversies.</div><span class="ref__series"><i>Cell Death Discov.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">15049</span><div class="ref__controls"><a href="#back-bib189" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_199_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27551478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_199_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fcddiscovery.2015.49&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=15049&journal=Cell+Death+Discov.&author=Weeden+C.E.&author=Solomon+B.&author=Asselin-Labat+M.-L.&title=FGFR1+inhibition+in+lung+squamous+cell+carcinoma%3A+questions+and+controversies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span><div class="ref__controls"><a href="#back-bib190" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_200_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Li K. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Cancer neoantigen prioritization through sensitive and reliable proteogenomics analysis.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1759</span><div class="ref__controls"><a href="#back-bib191" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85083223977&amp;cf=" onclick="newWindow(this.href);return false">Scopus (15)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32273506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_201_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-020-15456-w&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1759&journal=Nat.+Commun.&author=Wen+B.&author=Li+K.&author=Zhang+Y.&author=Zhang+B.&title=Cancer+neoantigen+prioritization+through+sensitive+and+reliable+proteogenomics+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Westra H.-J. </li><li class="ref__authors__name">Peters M.J. </li><li class="ref__authors__name">Esko T. </li><li class="ref__authors__name">Yaghootkar H. </li><li class="ref__authors__name">Schurmann C. </li><li class="ref__authors__name">Kettunen J. </li><li class="ref__authors__name">Christiansen M.W. </li><li class="ref__authors__name">Fairfax B.P. </li><li class="ref__authors__name">Schramm K. </li><li class="ref__authors__name">Powell J.E. </li><li>et al.</li></ul><div class="ref__title">Systematic identification of trans eQTLs as putative drivers of known disease associations.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">1238-1243</span><div class="ref__controls"><a href="#back-bib192" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84885018609&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1038)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24013639&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_202_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.2756&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1238-1243&journal=Nat.+Genet.&author=Westra+H.-J.&author=Peters+M.J.&author=Esko+T.&author=Yaghootkar+H.&author=Schurmann+C.&author=Kettunen+J.&author=Christiansen+M.W.&author=Fairfax+B.P.&author=Schramm+K.&author=Powell+J.E.&title=Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wheatley S.P. </li><li class="ref__authors__name">Altieri D.C. </li></ul><div class="ref__title">Survivin at a glance.</div><span class="ref__series"><i>J. Cell Sci.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">132</span></span>: <span class="ref__seriesPages">jcs223826</span><div class="ref__controls"><a href="#back-bib193" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85064325670&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30948431&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_203_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1242%2Fjcs.223826&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2019&pages=jcs223826&journal=J.%C2%A0Cell+Sci.&author=Wheatley+S.P.&author=Altieri+D.C.&title=Survivin+at+a+glance">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Whitfield M.L. </li><li class="ref__authors__name">Sherlock G. </li><li class="ref__authors__name">Saldanha A.J. </li><li class="ref__authors__name">Murray J.I. </li><li class="ref__authors__name">Ball C.A. </li><li class="ref__authors__name">Alexander K.E. </li><li class="ref__authors__name">Matese J.C. </li><li class="ref__authors__name">Perou C.M. </li><li class="ref__authors__name">Hurt M.M. </li><li class="ref__authors__name">Brown P.O. </li><li class="ref__authors__name">Botstein D. </li></ul><div class="ref__title">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.</div><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">1977-2000</span><div class="ref__controls"><a href="#back-bib194" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-0035985177&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1117)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=12058064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_204_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1091%2Fmbc.02-02-0030.&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2002&pages=1977-2000&journal=Mol.+Biol.+Cell&author=Whitfield+M.L.&author=Sherlock+G.&author=Saldanha+A.J.&author=Murray+J.I.&author=Ball+C.A.&author=Alexander+K.E.&author=Matese+J.C.&author=Perou+C.M.&author=Hurt+M.M.&author=Brown+P.O.&author=Botstein+D.&title=Identification+of+genes+periodically+expressed+in+the+human+cell+cycle+and+their+expression+in+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Hayes D.N. </li></ul><div class="ref__title">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span><div class="ref__controls"><a href="#back-bib195" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (710)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_205_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinformatics&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Yin X. </li><li class="ref__authors__name">Hoadley K.A. </li><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Hayward M.C. </li><li class="ref__authors__name">Cabanski C.R. </li><li class="ref__authors__name">Muldrew K. </li><li class="ref__authors__name">Miller C.R. </li><li class="ref__authors__name">Randell S.H. </li><li class="ref__authors__name">Socinski M.A. </li><li>et al.</li></ul><div class="ref__title">Lung squamous cell carcinoma mRNA expression subtypes are reproducible, clinically important, and correspond to normal cell types.</div><span class="ref__series"><i>Clin. Cancer Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4864-4875</span><div class="ref__controls"><a href="#back-bib196" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-77957570386&amp;cf=" onclick="newWindow(this.href);return false">Scopus (163)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=20643781&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_206_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1158%2F1078-0432.CCR-10-0199&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4864-4875&journal=Clin.+Cancer+Res.&author=Wilkerson+M.D.&author=Yin+X.&author=Hoadley+K.A.&author=Liu+Y.&author=Hayward+M.C.&author=Cabanski+C.R.&author=Muldrew+K.&author=Miller+C.R.&author=Randell+S.H.&author=Socinski+M.A.&title=Lung+squamous+cell+carcinoma+mRNA+expression+subtypes+are+reproducible%2C+clinically+important%2C+and+correspond+to+normal+cell+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wong J.J.Y. </li><li class="ref__authors__name">Pung Y.F. </li><li class="ref__authors__name">Sze N.S.-K. </li><li class="ref__authors__name">Chin K.-C. </li></ul><div class="ref__title">HERC5 is an IFN-induced HECT-type E3 protein ligase that mediates type I IFN-induced ISGylation of protein targets.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">103</span></span>: <span class="ref__seriesPages">10735-10740</span><div class="ref__controls"><a href="#back-bib197" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-33746089547&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=16815975&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_207_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1073%2Fpnas.0600397103&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=10735-10740&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Wong+J.J.Y.&author=Pung+Y.F.&author=Sze+N.S.-K.&author=Chin+K.-C.&title=HERC5+is+an+IFN-induced+HECT-type+E3+protein+ligase+that+mediates+type+I+IFN-induced+ISGylation+of+protein+targets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wu Y.-M. </li><li class="ref__authors__name">Cieślik M. </li><li class="ref__authors__name">Lonigro R.J. </li><li class="ref__authors__name">Vats P. </li><li class="ref__authors__name">Reimers M.A. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Ning Y. </li><li class="ref__authors__name">Wang L. </li><li class="ref__authors__name">Kunju L.P. </li><li class="ref__authors__name">de Sarkar N. </li><li>et al.</li><li>PCF/SU2C International Prostate Cancer Dream Team</li></ul><div class="ref__title">Inactivation of CDK12 Delineates a Distinct Immunogenic Class of Advanced Prostate Cancer.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">1770-1782.e14</span><div class="ref__controls"><a href="#back-bib198" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85047481697&amp;cf=" onclick="newWindow(this.href);return false">Scopus (196)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29906450&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_208_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.cell.2018.04.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=1770-1782.e14&journal=Cell&author=PCF%2FSU2C+International+Prostate+Cancer+Dream+Team&title=Inactivation+of+CDK12+Delineates+a+Distinct+Immunogenic+Class+of+Advanced+Prostate+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wu Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Zheng X. </li><li class="ref__authors__name">Dai F. </li><li class="ref__authors__name">Lu Y. </li><li class="ref__authors__name">Dai L. </li><li class="ref__authors__name">Niu M. </li><li class="ref__authors__name">Guo H. </li><li class="ref__authors__name">Li W. </li><li class="ref__authors__name">Xue X. </li><li>et al.</li></ul><div class="ref__title">Circular RNA circCORO1C promotes laryngeal squamous cell carcinoma progression by modulating the let-7c-5p/PBX3 axis.</div><span class="ref__series"><i>Mol. Cancer.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">19</span></span>: <span class="ref__seriesPages">99</span><div class="ref__controls"><a href="#back-bib199" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85085909077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=32487167&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_209_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1186%2Fs12943-020-01215-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=99&journal=Mol.+Cancer&author=Wu+Y.&author=Zhang+Y.&author=Zheng+X.&author=Dai+F.&author=Lu+Y.&author=Dai+L.&author=Niu+M.&author=Guo+H.&author=Li+W.&author=Xue+X.&title=Circular+RNA+circCORO1C+promotes+laryngeal+squamous+cell+carcinoma+progression+by+modulating+the+let-7c-5p%2FPBX3+axis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Xiong Y. </li><li class="ref__authors__name">Lei Q.-Y. </li><li class="ref__authors__name">Zhao S. </li><li class="ref__authors__name">Guan K.-L. </li></ul><div class="ref__title">Regulation of glycolysis and gluconeogenesis by acetylation of PKM and PEPCK.</div><span class="ref__series"><i>Cold Spring Harb. Symp. Quant. Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">76</span></span>: <span class="ref__seriesPages">285-289</span><div class="ref__controls"><a href="#back-bib200" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84867414358&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=22096030&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_210_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1101%2Fsqb.2011.76.010942&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2011&pages=285-289&journal=Cold+Spring+Harb.+Symp.+Quant.+Biol.&author=Xiong+Y.&author=Lei+Q.-Y.&author=Zhao+S.&author=Guan+K.-L.&title=Regulation+of+glycolysis+and+gluconeogenesis+by+acetylation+of+PKM+and+PEPCK">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yang W. </li><li class="ref__authors__name">Lu Z. </li></ul><div class="ref__title">Nuclear PKM2 regulates the Warburg effect.</div><span class="ref__series"><i>Cell Cycle.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">3154-3158</span><div class="ref__controls"><a href="#back-bib201" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84886704519&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24013426&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_211_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4161%2Fcc.26182&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=3154-3158&journal=Cell+Cycle&author=Yang+W.&author=Lu+Z.&title=Nuclear+PKM2+regulates+the+Warburg+effect">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Schulz M.H. </li><li class="ref__authors__name">Long Q. </li><li class="ref__authors__name">Apweiler R. </li><li class="ref__authors__name">Ning Z. </li></ul><div class="ref__title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span><div class="ref__controls"><a href="#back-bib202" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1209)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_212_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yomtoubian S. </li><li class="ref__authors__name">Lee S.B. </li><li class="ref__authors__name">Verma A. </li><li class="ref__authors__name">Izzo F. </li><li class="ref__authors__name">Markowitz G. </li><li class="ref__authors__name">Choi H. </li><li class="ref__authors__name">Cerchietti L. </li><li class="ref__authors__name">Vahdat L. </li><li class="ref__authors__name">Brown K.A. </li><li class="ref__authors__name">Andreopoulou E. </li><li>et al.</li></ul><div class="ref__title">Inhibition of EZH2 Catalytic Activity Selectively Targets a Metastatic Subpopulation in Triple-Negative Breast Cancer.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">755-770.e6</span><div class="ref__controls"><a href="#back-bib203" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85078319103&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31968251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_213_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2019.12.056&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=755-770.e6&journal=Cell+Rep.&author=Yomtoubian+S.&author=Lee+S.B.&author=Verma+A.&author=Izzo+F.&author=Markowitz+G.&author=Choi+H.&author=Cerchietti+L.&author=Vahdat+L.&author=Brown+K.A.&author=Andreopoulou+E.&title=Inhibition+of+EZH2+Catalytic+Activity+Selectively+Targets+a+Metastatic+Subpopulation+in+Triple-Negative+Breast+Cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoshihara K. </li><li class="ref__authors__name">Shahmoradgoli M. </li><li class="ref__authors__name">Martínez E. </li><li class="ref__authors__name">Vegesna R. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Torres-Garcia W. </li><li class="ref__authors__name">Treviño V. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Levine D.A. </li><li>et al.</li></ul><div class="ref__title">Inferring tumour purity and stromal and immune cell admixture from expression data.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span><div class="ref__controls"><a href="#back-bib204" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1382)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_214_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yuan G. </li><li class="ref__authors__name">Flores N.M. </li><li class="ref__authors__name">Hausmann S. </li><li class="ref__authors__name">Lofgren S.M. </li><li class="ref__authors__name">Kharchenko V. </li><li class="ref__authors__name">Angulo-Ibanez M. </li><li class="ref__authors__name">Sengupta D. </li><li class="ref__authors__name">Lu X. </li><li class="ref__authors__name">Czaban I. </li><li class="ref__authors__name">Azhibek D. </li><li>et al.</li></ul><div class="ref__title">Elevated NSD3 histone methylation activity drives squamous cell lung cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2021; </span><span class="ref__seriesVolume">590</span></span>: <span class="ref__seriesPages">504-508</span><div class="ref__controls"><a href="#back-bib205" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85100414615&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=33536620&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_215_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41586-020-03170-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=590&publication_year=2021&pages=504-508&journal=Nature&author=Yuan+G.&author=Flores+N.M.&author=Hausmann+S.&author=Lofgren+S.M.&author=Kharchenko+V.&author=Angulo-Ibanez+M.&author=Sengupta+D.&author=Lu+X.&author=Czaban+I.&author=Azhibek+D.&title=Elevated+NSD3+histone+methylation+activity+drives+squamous+cell+lung+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zecha J. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Kanashova T. </li><li class="ref__authors__name">Avanessian S.C. </li><li class="ref__authors__name">Kane M.H. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Kuster B. </li></ul><div class="ref__title">TMT labeling for the masses: A robust and cost-efficient, in-solution labeling approach.</div><span class="ref__series"><i>Mol. Cell Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span><div class="ref__controls"><a href="#back-bib222" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_216_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+A+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang X. </li><li class="ref__authors__name">Lu F. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Yin F. </li><li class="ref__authors__name">Xu Z. </li><li class="ref__authors__name">Qi D. </li><li class="ref__authors__name">Wu X. </li><li class="ref__authors__name">Cao Y. </li><li class="ref__authors__name">Liang W. </li><li class="ref__authors__name">Liu Y. </li><li>et al.</li></ul><div class="ref__title">Pluripotent stem cell protein Sox2 confers sensitivity to LSD1 inhibition in cancer cells.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">445-457</span><div class="ref__controls"><a href="#back-bib206" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84887021821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (79)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=24139802&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_217_2&amp;dbid=4&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1016%2Fj.celrep.2013.09.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=445-457&journal=Cell+Rep.&author=Zhang+X.&author=Lu+F.&author=Wang+J.&author=Yin+F.&author=Xu+Z.&author=Qi+D.&author=Wu+X.&author=Cao+Y.&author=Liang+W.&author=Liu+Y.&title=Pluripotent+stem+cell+protein+Sox2+confers+sensitivity+to+LSD1+inhibition+in+cancer+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Zou X. </li><li class="ref__authors__name">Qian W. </li><li class="ref__authors__name">Weng X. </li><li class="ref__authors__name">Zhang L. </li><li class="ref__authors__name">Zhang L. </li><li class="ref__authors__name">Wang S. </li><li class="ref__authors__name">Cao X. </li><li class="ref__authors__name">Ma L. </li><li class="ref__authors__name">Wei G. </li><li>et al.</li></ul><div class="ref__title">Enhanced PAPSS2/VCAN sulfation axis is essential for Snail-mediated breast cancer cell migration and metastasis.</div><span class="ref__series"><i>Cell Death Differ.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">565-579</span><div class="ref__controls"><a href="#back-bib207" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85049124890&amp;cf=" onclick="newWindow(this.href);return false">Scopus (35)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29955124&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_218_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41418-018-0147-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=565-579&journal=Cell+Death+Differ.&author=Zhang+Y.&author=Zou+X.&author=Qian+W.&author=Weng+X.&author=Zhang+L.&author=Zhang+L.&author=Wang+S.&author=Cao+X.&author=Ma+L.&author=Wei+G.&title=Enhanced+PAPSS2%2FVCAN+sulfation+axis+is+essential+for+Snail-mediated+breast+cancer+cell+migration+and+metastasis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Thery F. </li><li class="ref__authors__name">Wu N.C. </li><li class="ref__authors__name">Luhmann E.K. </li><li class="ref__authors__name">Dussurget O. </li><li class="ref__authors__name">Foecke M. </li><li class="ref__authors__name">Bredow C. </li><li class="ref__authors__name">Jiménez-Fernández D. </li><li class="ref__authors__name">Leandro K. </li><li class="ref__authors__name">Beling A. </li><li>et al.</li></ul><div class="ref__title">The in vivo ISGylome links ISG15 to metabolic pathways and autophagy upon Listeria monocytogenes infection.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">5383</span><div class="ref__controls"><a href="#back-bib208" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85075610966&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=31772204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_219_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fs41467-019-13393-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=5383&journal=Nat.+Commun.&author=Zhang+Y.&author=Thery+F.&author=Wu+N.C.&author=Luhmann+E.K.&author=Dussurget+O.&author=Foecke+M.&author=Bredow+C.&author=Jim%C3%A9nez-Fern%C3%A1ndez+D.&author=Leandro+K.&author=Beling+A.&title=The+in%C2%A0vivo+ISGylome+links+ISG15+to+metabolic+pathways+and+autophagy+upon+Listeria+monocytogenes+infection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhao D. </li><li class="ref__authors__name">Li F.-L. </li><li class="ref__authors__name">Cheng Z.-L. </li><li class="ref__authors__name">Lei Q.-Y. </li></ul><div class="ref__title">Impact of acetylation on tumor metabolism.</div><span class="ref__series"><i>Mol. Cell. Oncol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">e963452</span><div class="ref__controls"><a href="#back-bib209" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85029407726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=27308346&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_220_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.4161%2F23723548.2014.963452&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=e963452&journal=Mol.+Cell.+Oncol.&author=Zhao+D.&author=Li+F.-L.&author=Cheng+Z.-L.&author=Lei+Q.-Y.&title=Impact+of+acetylation+on+tumor+metabolism">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zheng L. </li><li class="ref__authors__name">Conner S.D. </li></ul><div class="ref__title">Glycogen synthase kinase 3β inhibition enhances Notch1 recycling.</div><span class="ref__series"><i>Mol. Biol. Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">389-395</span><div class="ref__controls"><a href="#back-bib210" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-85042268410&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=29237816&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_221_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1091%2Fmbc.E17-07-0474&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=389-395&journal=Mol.+Biol.+Cell&author=Zheng+L.&author=Conner+S.D.&title=Glycogen+synthase+kinase+3%CE%B2+inhibition+enhances+Notch1+recycling">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhou X. </li><li class="ref__authors__name">Edmonson M.N. </li><li class="ref__authors__name">Wilkinson M.R. </li><li class="ref__authors__name">Patel A. </li><li class="ref__authors__name">Wu G. </li><li class="ref__authors__name">Liu Y. </li><li class="ref__authors__name">Li Y. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Rusch M.C. </li><li class="ref__authors__name">Parker M. </li><li>et al.</li></ul><div class="ref__title">Exploring genomic alteration in pediatric cancer using ProteinPaint.</div><span class="ref__series"><i>Nat. Genet.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">4-6</span><div class="ref__controls"><a href="#back-bib211" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2021.07.016&amp;key=2-s2.0-84971602748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (123)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=8&amp;doi=10.1016/j.cell.2021.07.016&amp;key=26711108&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_222_2&amp;dbid=16&amp;doi=10.1016/j.cell.2021.07.016&amp;key=10.1038%2Fng.3466&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=4-6&journal=Nat.+Genet.&author=Zhou+X.&author=Edmonson+M.N.&author=Wilkinson+M.R.&author=Patel+A.&author=Wu+G.&author=Liu+Y.&author=Li+Y.&author=Zhang+Z.&author=Rusch+M.C.&author=Parker+M.&title=Exploring+genomic+alteration+in+pediatric+cancer+using+ProteinPaint">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: August 5, 2021</div><div class="article-info__date">Accepted:
            July 12,
            2021</div><div class="article-info__date">Received in revised form:
            April 26,
            2021</div><div class="article-info__date">Received:
            November 6,
            2020</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.cell.2021.07.016">https://doi.org/10.1016/j.cell.2021.07.016</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2021 The Author(s). Published by Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S0092867421008576">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421008576" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867421008576%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00857-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867421008576%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1723231561915","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"184","issueNumber":"16","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421008576","coverDate":"202108","type":"XOCS-JOURNAL:SCOPE-FULL","title":"A proteogenomic portrait of lung squamous cell carcinoma","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
