<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="381a1b45-1fff-412e-88a1-a0fe0c54ccce"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867418X00117;article:article:pii\:S0092867419302922;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(19)30292-2" />
    



        <meta property="og:site_name" content="Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg" />
    


        <meta property="og:description" content="A systematic proteogenomic analysis of colon cancer reveals vulnerabilities of potential
clinical value inaccessible from genomic assessment alone." />
    











        <meta name="twitter:title" content="Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg" />
    


        <meta name="twitter:description" content="A systematic proteogenomic analysis of colon cancer reveals vulnerabilities of potential
clinical value inaccessible from genomic assessment alone." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(19)30292-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419302922%3Fshowall%3Dtrue">
    


<title>Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities: Cell</title><meta name="citation_journal_title" content="Cell"/> <meta name="citation_journal_abbrev" content="Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities"/> <meta name="citation_date" content="2019/05/02"/> <meta name="citation_online_date" content="2019/04/25"/> <meta name="citation_volume" content="177"/> <meta name="citation_issue" content="4"/> <meta name="citation_firstpage" content="1035"/> <meta name="citation_lastpage" content="1049.e19"/> <meta name="citation_pmid" content="31031003"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S0092867419302922/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S0092867419302922/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S0092867419302922/pdf"/> <meta name="citation_issn" content="0092-8674"/> <meta name="citation_issn" content="1097-4172"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.cell.2019.03.030"/> <meta name="citation_pmid" content="31031003"/> <meta name="citation_author" content="Suhas Vasaikar"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Xiaojing Wang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Vladislav A. Petyuk"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Sara R. Savage"></meta><meta name="citation_author_institution" content="Department of Biomedical Informatics, Vanderbilt University Medical Center, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yongchao Dou"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Yun Zhang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Zhiao Shi"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Osama A. Arshad"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Marina A. Gritsenko"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Lisa J. Zimmerman"></meta><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Jason E. McDermott"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Therese R. Clauss"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Ronald J. Moore"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Rui Zhao"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Matthew E. Monroe"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Yi-Ting Wang"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Matthew C. Chambers"></meta><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Robbert J.C. Slebos"></meta><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Ken S. Lau"></meta><meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Qianxing Mo"></meta><meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_institution" content="The McDonnell Genome Institute, Washington University in St. Louis, Forest Park Avenue, Campus Box 8501, St. Louis, MO 63108, USA"></meta><meta name="citation_author" content="Matthew Ellis"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author_email" content="karin.rodland@pnnl.gov"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author_institution" content="Department of Cell, Developmental, and Cancer Biology, Oregon Health & Science University, Portland, OR 97221, USA"></meta><meta name="citation_author" content="Daniel C. Liebler"></meta><meta name="citation_author_email" content="daniel.liebler@vanderbilt.edu"></meta><meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author_email" content="tao.liu@pnnl.gov"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_email" content="bing.zhang@bcm.edu"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Akhilesh Pandey"></meta><meta name="citation_author" content="Amanda Paulovich"></meta><meta name="citation_author" content="Andrew Hoofnagle"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author" content="David F. Ransohoff"></meta><meta name="citation_author" content="David Fenyo"></meta><meta name="citation_author" content="David L. Tabb"></meta><meta name="citation_author" content="Douglas A. Levine"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author" content="Eric Kuhn"></meta><meta name="citation_author" content="Forest M. White"></meta><meta name="citation_author" content="Gordon A. Whiteley"></meta><meta name="citation_author" content="Heng Zhu"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author" content="Ie-Ming Shih"></meta><meta name="citation_author" content="Jasmin Bavarva"></meta><meta name="citation_author" content="Jeffrey Whiteaker"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author" content="Karl R. Clauser"></meta><meta name="citation_author" content="Kelly Ruggles"></meta><meta name="citation_author" content="Kimberly Elburn"></meta><meta name="citation_author" content="Linda Hannick"></meta><meta name="citation_author" content="Mark Watson"></meta><meta name="citation_author" content="Mauricio Oberti"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author" content="Melinda E. Sanders"></meta><meta name="citation_author" content="Melissa Borucki"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author" content="Michael Snyder"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Negin Vatanian"></meta><meta name="citation_author" content="Paul A. Rudnick"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="Philip Mertins"></meta><meta name="citation_author" content="R. Reid Townsend"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Robert C. Rivers"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author" content="Sherri R. Davies"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Stephen E. Stein"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author" content="Steven J. Skates"></meta><meta name="citation_author" content="Subha Madhavan"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author" content="Xian Chen"></meta><meta name="citation_author" content="Yingming Zhao"></meta><meta name="citation_author" content="Yue Wang"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_keywords" content="colon cancer; proteomics; proteogenomics; tumor antigen; immune evasion; glycolysis; drug targets; biomarkers; RB1; SOX9"></meta><meta name="citation_reference" content="citation_title=The transcription factor Sox9 is degraded by the ubiquitin-proteasome system and stabilized by a mutation in a ubiquitin-target site; citation_author=H. Akiyama; citation_author=T. Kamitani; citation_author=X. Yang; citation_author=R. Kandyil; citation_author=L.C. Bridgewater; citation_author=M. Fellous; citation_author=Y. Mori-Akiyama; citation_author=B. de Crombrugghe; citation_journal_title=Matrix Biol.; citation_volume=23; citation_firstpage=499; citation_lastpage=505; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens; citation_author=L.G. Almeida; citation_author=N.J. Sakabe; citation_author=A.R. deOliveira; citation_author=M.C. Silva; citation_author=A.S. Mundstein; citation_author=T. Cohen; citation_author=Y.T. Chen; citation_author=R. Chua; citation_author=S. Gurung; citation_author=S. Gnjatic; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=37; citation_firstpage=D816; citation_lastpage=D819; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=HTSeq--a Python framework to work with high-throughput sequencing data; citation_author=S. Anders; citation_author=P.T. Pyl; citation_author=W. Huber; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=166; citation_lastpage=169; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy; citation_author=M. Angelova; citation_author=P. Charoentong; citation_author=H. Hackl; citation_author=M.L. Fischer; citation_author=R. Snajder; citation_author=A.M. Krogsdam; citation_author=M.J. Waldner; citation_author=G. Bindea; citation_author=B. Mlecnik; citation_author=J. Galon; citation_author=Z. Trajanoski; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=64; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Global patterns and trends in colorectal cancer incidence and mortality; citation_author=M. Arnold; citation_author=M.S. Sierra; citation_author=M. Laversanne; citation_author=I. Soerjomataram; citation_author=A. Jemal; citation_author=F. Bray; citation_journal_title=Gut; citation_volume=66; citation_firstpage=683; citation_lastpage=691; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A probability-based approach for high-throughput protein phosphorylation analysis and site localization; citation_author=S.A. Beausoleil; citation_author=J. Villén; citation_author=S.A. Gerber; citation_author=J. Rush; citation_author=S.P. Gygi; citation_journal_title=Nat. Biotechnol.; citation_volume=24; citation_firstpage=1285; citation_lastpage=1292; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=LDHA-Associated Lactic Acid Production Blunts Tumor Immunosurveillance by T and NK Cells; citation_author=A. Brand; citation_author=K. Singer; citation_author=G.E. Koehl; citation_author=M. Kolitzus; citation_author=G. Schoenhammer; citation_author=A. Thiel; citation_author=C. Matos; citation_author=C. Bruss; citation_author=S. Klobuch; citation_author=K. Peter; citation_author=et al.; citation_journal_title=Cell Metab.; citation_volume=24; citation_firstpage=657; citation_lastpage=671; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Comprehensive molecular characterization of human colon and rectal cancer; citation_author=Cancer Genome Atlas Network; citation_journal_title=Nature; citation_volume=487; citation_firstpage=330; citation_lastpage=337; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Comprehensive genomic characterization of squamous cell lung cancers; citation_author=Cancer Genome Atlas Research Network; citation_journal_title=Nature; citation_volume=489; citation_firstpage=519; citation_lastpage=525; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=ncPRO-seq: a tool for annotation and profiling of ncRNAs in sRNA-seq data; citation_author=C.J. Chen; citation_author=N. Servant; citation_author=J. Toedling; citation_author=A. Sarazin; citation_author=A. Marchais; citation_author=E. Duvernois-Berthet; citation_author=V. Cognat; citation_author=V. Colot; citation_author=O. Voinnet; citation_author=E. Heard; citation_author=et al.; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=3147; citation_lastpage=3149; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=The M2 splice isoform of pyruvate kinase is important for cancer metabolism and tumour growth; citation_author=H.R. Christofk; citation_author=M.G. Vander Heiden; citation_author=M.H. Harris; citation_author=A. Ramanathan; citation_author=R.E. Gerszten; citation_author=R. Wei; citation_author=M.D. Fleming; citation_author=S.L. Schreiber; citation_author=L.C. Cantley; citation_journal_title=Nature; citation_volume=452; citation_firstpage=230; citation_lastpage=233; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=TANDEM: matching proteins with tandem mass spectra; citation_author=R. Craig; citation_author=R.C. Beavis; citation_journal_title=Bioinformatics; citation_volume=20; citation_firstpage=1466; citation_lastpage=1467; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Carcinoembryonic antigen as a marker for colorectal cancer: is it clinically useful?; citation_author=M.J. Duffy; citation_journal_title=Clin. Chem.; citation_volume=47; citation_firstpage=624; citation_lastpage=630; citation_publication_date=2001; "></meta><meta name="citation_reference" content="citation_title=A user’s guide to the encyclopedia of DNA elements (ENCODE); citation_author=ENCODE Project Consortium; citation_journal_title=PLoS Biol.; citation_volume=9; citation_firstpage=e1001046; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=COSMIC: somatic cancer genetics at high-resolution; citation_author=S.A. Forbes; citation_author=D. Beare; citation_author=H. Boutselakis; citation_author=S. Bamford; citation_author=N. Bindal; citation_author=J. Tate; citation_author=C.G. Cole; citation_author=S. Ward; citation_author=E. Dawson; citation_author=L. Ponting; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=D1; citation_firstpage=D777; citation_lastpage=D783; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Global proteome analysis of the NCI-60 cell line panel; citation_author=A.M. Gholami; citation_author=H. Hahne; citation_author=Z. Wu; citation_author=F.J. Auer; citation_author=C. Meng; citation_author=M. Wilhelm; citation_author=B. Kuster; citation_journal_title=Cell Rep.; citation_volume=4; citation_firstpage=609; citation_lastpage=620; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=The consensus molecular subtypes of colorectal cancer; citation_author=J. Guinney; citation_author=R. Dienstmann; citation_author=X. Wang; citation_author=A. de Reyniès; citation_author=A. Schlicker; citation_author=C. Soneson; citation_author=L. Marisa; citation_author=P. Roepman; citation_author=G. Nyamundanda; citation_author=P. Angelino; citation_author=et al.; citation_journal_title=Nat. Med.; citation_volume=21; citation_firstpage=1350; citation_lastpage=1356; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=GSVA: gene set variation analysis for microarray and RNA-seq data; citation_author=S. Hänzelmann; citation_author=R. Castelo; citation_author=J. Guinney; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=7; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=PhosphoSitePlus, 2014: mutations, PTMs and recalibrations; citation_author=P.V. Hornbeck; citation_author=B. Zhang; citation_author=B. Murray; citation_author=J.M. Kornhauser; citation_author=V. Latham; citation_author=E. Skrzypek; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D512; citation_lastpage=D520; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data; citation_author=V. Jurtz; citation_author=S. Paul; citation_author=M. Andreatta; citation_author=P. Marcatili; citation_author=B. Peters; citation_author=M. Nielsen; citation_journal_title=J. Immunol.; citation_volume=199; citation_firstpage=3360; citation_lastpage=3368; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=ProteoWizard: open source software for rapid proteomics tools development; citation_author=D. Kessner; citation_author=M. Chambers; citation_author=R. Burke; citation_author=D. Agus; citation_author=P. Mallick; citation_journal_title=Bioinformatics; citation_volume=24; citation_firstpage=2534; citation_lastpage=2536; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases; citation_author=S. Kim; citation_author=N. Gupta; citation_author=P.A. Pevzner; citation_journal_title=J. Proteome Res.; citation_volume=7; citation_firstpage=3354; citation_lastpage=3363; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Dual mechanisms for the inhibition of E2F binding to RB by cyclin-dependent kinase-mediated RB phosphorylation; citation_author=E.S. Knudsen; citation_author=J.Y. Wang; citation_journal_title=Mol. Cell. Biol.; citation_volume=17; citation_firstpage=5771; citation_lastpage=5783; citation_publication_date=1997; "></meta><meta name="citation_reference" content="citation_title=Vaccination with multiple peptides derived from novel cancer-testis antigens can induce specific T-cell responses and clinical responses in advanced esophageal cancer; citation_author=K. Kono; citation_author=Y. Mizukami; citation_author=Y. Daigo; citation_author=A. Takano; citation_author=K. Masuda; citation_author=K. Yoshida; citation_author=T. Tsunoda; citation_author=Y. Kawaguchi; citation_author=Y. Nakamura; citation_author=H. Fujii; citation_journal_title=Cancer Sci.; citation_volume=100; citation_firstpage=1502; citation_lastpage=1509; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=The miR-200 family inhibits epithelial-mesenchymal transition and cancer cell migration by direct targeting of E-cadherin transcriptional repressors ZEB1 and ZEB2; citation_author=M. Korpal; citation_author=E.S. Lee; citation_author=G. Hu; citation_author=Y. Kang; citation_journal_title=J. Biol. Chem.; citation_volume=283; citation_firstpage=14910; citation_lastpage=14914; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=CopywriteR: DNA copy number detection from off-target sequence data; citation_author=T. Kuilman; citation_author=A. Velds; citation_author=K. Kemper; citation_author=M. Ranzani; citation_author=L. Bombardelli; citation_author=M. Hoogstraat; citation_author=E. Nevedomskaya; citation_author=G. Xu; citation_author=J. de Ruiter; citation_author=M.P. Lolkema; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=49; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=ClinVar: public archive of relationships among sequence variation and human phenotype; citation_author=M.J. Landrum; citation_author=J.M. Lee; citation_author=G.R. Riley; citation_author=W. Jang; citation_author=W.S. Rubinstein; citation_author=D.M. Church; citation_author=D.R. Maglott; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=D980; citation_lastpage=D985; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B. Langmead; citation_author=C. Trapnell; citation_author=M. Pop; citation_author=S.L. Salzberg; citation_journal_title=Genome Biol.; citation_volume=10; citation_firstpage=R25; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=PD-1 Blockade in Tumors with Mismatch-Repair Deficiency; citation_author=D.T. Le; citation_author=J.N. Uram; citation_author=H. Wang; citation_author=B.R. Bartlett; citation_author=H. Kemberling; citation_author=A.D. Eyring; citation_author=A.D. Skora; citation_author=B.S. Luber; citation_author=N.S. Azad; citation_author=D. Laheru; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=372; citation_firstpage=2509; citation_lastpage=2520; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome; citation_author=B. Li; citation_author=C.N. Dewey; citation_journal_title=BMC Bioinformatics; citation_volume=12; citation_firstpage=323; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=589; citation_lastpage=595; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst.; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Integrative Omics Analysis Reveals Post-Transcriptionally Enhanced Protective Host Response in Colorectal Cancers with Microsatellite Instability; citation_author=Q. Liu; citation_author=B. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=15; citation_firstpage=766; citation_lastpage=776; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=dbNSFP: a lightweight database of human nonsynonymous SNPs and their functional predictions; citation_author=X. Liu; citation_author=X. Jian; citation_author=E. Boerwinkle; citation_journal_title=Hum. Mutat.; citation_volume=32; citation_firstpage=894; citation_lastpage=899; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=The vigorous immune microenvironment of microsatellite instable colon cancer is balanced by multiple counter-inhibitory checkpoints; citation_author=N.J. Llosa; citation_author=M. Cruise; citation_author=A. Tam; citation_author=E.C. Wicks; citation_author=E.M. Hechenbleikner; citation_author=J.M. Taube; citation_author=R.L. Blosser; citation_author=H. Fan; citation_author=H. Wang; citation_author=B.S. Luber; citation_author=et al.; citation_journal_title=Cancer Discov.; citation_volume=5; citation_firstpage=43; citation_lastpage=51; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=SIGNOR: A Database of Causal Relationships Between Biological Entities-A Short Guide to Searching and Browsing; citation_author=P. Lo Surdo; citation_author=A. Calderone; citation_author=G. Cesareni; citation_author=L. Perfetto; citation_journal_title=Curr. Protoc. Bioinformatics; citation_volume=58; citation_firstpage=8.23.1; citation_lastpage=8.23.16; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Skyline: an open source document editor for creating and analyzing targeted proteomics experiments; citation_author=B. MacLean; citation_author=D.M. Tomazela; citation_author=N. Shulman; citation_author=M. Chambers; citation_author=G.L. Finney; citation_author=B. Frewen; citation_author=R. Kern; citation_author=D.L. Tabb; citation_author=D.C. Liebler; citation_author=M.J. MacCoss; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=966; citation_lastpage=968; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Analysis of somatic microsatellite indels identifies driver events in human tumors; citation_author=Y.E. Maruvka; citation_author=K.W. Mouw; citation_author=R. Karlic; citation_author=P. Parasuraman; citation_author=A. Kamburov; citation_author=P. Polak; citation_author=N.J. Haradhvala; citation_author=J.M. Hess; citation_author=E. Rheinbay; citation_author=Y. Brody; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=35; citation_firstpage=951; citation_lastpage=959; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Oncogenicity of the developmental transcription factor Sox9; citation_author=A. Matheu; citation_author=M. Collado; citation_author=C. Wise; citation_author=L. Manterola; citation_author=L. Cekaite; citation_author=A.J. Tye; citation_author=M. Canamero; citation_author=L. Bujanda; citation_author=A. Schedl; citation_author=K.S. Cheah; citation_author=et al.; citation_journal_title=Cancer Res.; citation_volume=72; citation_firstpage=1301; citation_lastpage=1315; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features; citation_author=M.E. Monroe; citation_author=J.L. Shaw; citation_author=D.S. Daly; citation_author=J.N. Adkins; citation_author=R.D. Smith; citation_journal_title=Comput. Biol. Chem.; citation_volume=32; citation_firstpage=215; citation_lastpage=217; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Derailed endocytosis: an emerging feature of cancer; citation_author=Y. Mosesson; citation_author=G.B. Mills; citation_author=Y. Yarden; citation_journal_title=Nat. Rev. Cancer; citation_volume=8; citation_firstpage=835; citation_lastpage=850; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Circular binary segmentation for the analysis of array-based DNA copy number data; citation_author=A.B. Olshen; citation_author=E.S. Venkatraman; citation_author=R. Lucito; citation_author=M. Wigler; citation_journal_title=Biostatistics; citation_volume=5; citation_firstpage=557; citation_lastpage=572; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=STAT1 and STAT3 in tumorigenesis: two sides of the same coin?; citation_title=JAK-STAT Pathway in Disease; citation_author=S. Pensa; citation_author=G. Regis; citation_author=D. Boselli; citation_author=F. Novelli; citation_author=V. Ploi; citation_firstpage=100; citation_lastpage=121; citation_publication_date=2009; citation_publisher=Landes Bioscience; "></meta><meta name="citation_reference" content="citation_title=DtaRefinery, a software tool for elimination of systematic errors from parent ion mass measurements in tandem mass spectra data sets; citation_author=V.A. Petyuk; citation_author=A.M. Mayampurath; citation_author=M.E. Monroe; citation_author=A.D. Polpitiya; citation_author=S.O. Purvine; citation_author=G.A. Anderson; citation_author=D.G. Camp; citation_author=R.D. Smith; citation_journal_title=Mol. Cell. Proteomics; citation_volume=9; citation_firstpage=486; citation_lastpage=496; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Oncotator: cancer variant annotation tool; citation_author=A.H. Ramos; citation_author=L. Lichtenstein; citation_author=M. Gupta; citation_author=M.S. Lawrence; citation_author=T.J. Pugh; citation_author=G. Saksena; citation_author=M. Meyerson; citation_author=G. Getz; citation_journal_title=Hum. Mutat.; citation_volume=36; citation_firstpage=E2423; citation_lastpage=E2429; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Deciphering the retinoblastoma protein phosphorylation code; citation_author=S.M. Rubin; citation_journal_title=Trends Biochem. Sci.; citation_volume=38; citation_firstpage=12; citation_lastpage=19; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets; citation_author=M.M. Savitski; citation_author=M. Wilhelm; citation_author=H. Hahne; citation_author=B. Kuster; citation_author=M. Bantscheff; citation_journal_title=Mol. Cell. Proteomics; citation_volume=14; citation_firstpage=2394; citation_lastpage=2404; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Primary, Adaptive, and Acquired Resistance to Cancer Immunotherapy; citation_author=P. Sharma; citation_author=S. Hu-Lieskovan; citation_author=J.A. Wargo; citation_author=A. Ribas; citation_journal_title=Cell; citation_volume=168; citation_firstpage=707; citation_lastpage=723; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The RB and p53 pathways in cancer; citation_author=C.J. Sherr; citation_author=F. McCormick; citation_journal_title=Cancer Cell; citation_volume=2; citation_firstpage=103; citation_lastpage=112; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; citation_author=S.T. Sherry; citation_author=M.H. Ward; citation_author=M. Kholodov; citation_author=J. Baker; citation_author=L. Phan; citation_author=E.M. Smigielski; citation_author=K. Sirotkin; citation_journal_title=Nucleic Acids Res.; citation_volume=29; citation_firstpage=308; citation_lastpage=311; citation_publication_date=2001; "></meta><meta name="citation_reference" content="citation_title=Antibody-free, targeted mass-spectrometric approach for quantification of proteins at low picogram per milliliter levels in human plasma/serum; citation_author=T. Shi; citation_author=T.L. Fillmore; citation_author=X. Sun; citation_author=R. Zhao; citation_author=A.A. Schepmoes; citation_author=M. Hossain; citation_author=F. Xie; citation_author=S. Wu; citation_author=J.S. Kim; citation_author=N. Jones; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=15395; citation_lastpage=15400; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Identification of human leukocyte antigen-A24-restricted epitope peptides derived from gene products upregulated in lung and esophageal cancers as novel targets for immunotherapy; citation_author=T. Suda; citation_author=T. Tsunoda; citation_author=Y. Daigo; citation_author=Y. Nakamura; citation_author=H. Tahara; citation_journal_title=Cancer Sci.; citation_volume=98; citation_firstpage=1803; citation_lastpage=1808; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=OptiType: precision HLA typing from next-generation sequencing data; citation_author=A. Szolek; citation_author=B. Schubert; citation_author=C. Mohr; citation_author=M. Sturm; citation_author=M. Feldhahn; citation_author=O. Kohlbacher; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=3310; citation_lastpage=3316; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis; citation_author=D.L. Tabb; citation_author=C.G. Fernando; citation_author=M.C. Chambers; citation_journal_title=J. Proteome Res.; citation_volume=6; citation_firstpage=654; citation_lastpage=661; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Immune Evasion in Tumor’s Own Sweet Way; citation_author=H. Tang; citation_author=Y.X. Fu; citation_journal_title=Cell Metab.; citation_volume=27; citation_firstpage=945; citation_lastpage=946; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A pathology atlas of the human cancer transcriptome; citation_author=M. Uhlen; citation_author=C. Zhang; citation_author=S. Lee; citation_author=E. Sjöstedt; citation_author=L. Fagerberg; citation_author=G. Bidkhori; citation_author=R. Benfeitas; citation_author=M. Arif; citation_author=Z. Liu; citation_author=F. Edfors; citation_author=et al.; citation_journal_title=Science; citation_volume=357; citation_firstpage=eaan2507; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline; citation_author=G.A. Van der Auwera; citation_author=M.O. Carneiro; citation_author=C. Hartl; citation_author=R. Poplin; citation_author=G. Del Angel; citation_author=A. Levy-Moonshine; citation_author=T. Jordan; citation_author=K. Shakir; citation_author=D. Roazen; citation_author=J. Thibault; citation_author=et al.; citation_journal_title=Curr. Protoc. Bioinformatics; citation_volume=43; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=LinkedOmics: analyzing multi-omics data within and across 32 cancer types; citation_author=S.V. Vasaikar; citation_author=P. Straub; citation_author=J. Wang; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_number=D1; citation_firstpage=D956; citation_lastpage=D963; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Cancer genome landscapes; citation_author=B. Vogelstein; citation_author=N. Papadopoulos; citation_author=V.E. Velculescu; citation_author=S. Zhou; citation_author=L.A. Diaz; citation_author=K.W. Kinzler; citation_journal_title=Science; citation_volume=339; citation_firstpage=1546; citation_lastpage=1558; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=3235; citation_lastpage=3237; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data; citation_author=K. Wang; citation_author=M. Li; citation_author=H. Hakonarson; citation_journal_title=Nucleic Acids Res.; citation_volume=38; citation_firstpage=e164; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Protein identification using customized protein sequence databases derived from RNA-Seq data; citation_author=X. Wang; citation_author=R.J. Slebos; citation_author=D. Wang; citation_author=P.J. Halvey; citation_author=D.L. Tabb; citation_author=D.C. Liebler; citation_author=B. Zhang; citation_journal_title=J. Proteome Res.; citation_volume=11; citation_firstpage=1009; citation_lastpage=1017; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Proteome Profiling Outperforms Transcriptome Profiling for Coexpression Based Gene Function Prediction; citation_author=J. Wang; citation_author=Z. Ma; citation_author=S.A. Carr; citation_author=P. Mertins; citation_author=H. Zhang; citation_author=Z. Zhang; citation_author=D.W. Chan; citation_author=M.J. Ellis; citation_author=R.R. Townsend; citation_author=R.D. Smith; citation_author=et al.; citation_journal_title=Mol. Cell. Proteomics; citation_volume=16; citation_firstpage=121; citation_lastpage=134; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Colorectal Cancer Cell Line Proteomes Are Representative of Primary Tumors and Predict Drug Sensitivity; citation_author=J. Wang; citation_author=D. Mouradov; citation_author=X. Wang; citation_author=R.N. Jorissen; citation_author=M.C. Chambers; citation_author=L.J. Zimmerman; citation_author=S. Vasaikar; citation_author=C.G. Love; citation_author=S. Li; citation_author=K. Lowes; citation_author=et al.; citation_journal_title=Gastroenterology; citation_volume=153; citation_firstpage=1082; citation_lastpage=1095; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit; citation_author=J. Wang; citation_author=S. Vasaikar; citation_author=Z. Shi; citation_author=M. Greer; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_number=W1; citation_firstpage=W130; citation_lastpage=W137; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The OMSSAPercolator: an automated tool to validate OMSSA results; citation_author=B. Wen; citation_author=G. Li; citation_author=J.C. Wright; citation_author=C. Du; citation_author=Q. Feng; citation_author=X. Xu; citation_author=J.S. Choudhary; citation_author=J. Wang; citation_journal_title=Proteomics; citation_volume=14; citation_firstpage=1011; citation_lastpage=1014; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=IPeak: An open source tool to combine results from multiple MS/MS search engines; citation_author=B. Wen; citation_author=C. Du; citation_author=G. Li; citation_author=F. Ghali; citation_author=A.R. Jones; citation_author=L. Käll; citation_author=S. Xu; citation_author=R. Zhou; citation_author=Z. Ren; citation_author=Q. Feng; citation_author=et al.; citation_journal_title=Proteomics; citation_volume=15; citation_firstpage=2916; citation_lastpage=2920; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=metaX: a flexible and comprehensive software for processing metabolomics data; citation_author=B. Wen; citation_author=Z. Mei; citation_author=C. Zeng; citation_author=S. Liu; citation_journal_title=BMC Bioinformatics; citation_volume=18; citation_firstpage=183; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations; citation_author=B. Wen; citation_author=X. Wang; citation_author=B. Zhang; citation_journal_title=Genome Res.; citation_volume=29; citation_firstpage=485; citation_lastpage=493; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Inferring tumour purity and stromal and immune cell admixture from expression data; citation_author=K. Yoshihara; citation_author=M. Shahmoradgoli; citation_author=E. Martínez; citation_author=R. Vegesna; citation_author=H. Kim; citation_author=W. Torres-Garcia; citation_author=V. Treviño; citation_author=H. Shen; citation_author=P.W. Laird; citation_author=D.A. Levine; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_firstpage=2612; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of human colon and rectal cancer; citation_author=B. Zhang; citation_author=J. Wang; citation_author=X. Wang; citation_author=J. Zhu; citation_author=Q. Liu; citation_author=Z. Shi; citation_author=M.C. Chambers; citation_author=L.J. Zimmerman; citation_author=K.F. Shaddox; citation_author=S. Kim; citation_author=NCI CPTAC; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=382; citation_lastpage=387; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer; citation_author=H. Zhang; citation_author=T. Liu; citation_author=Z. Zhang; citation_author=S.H. Payne; citation_author=B. Zhang; citation_author=J.E. McDermott; citation_author=J.Y. Zhou; citation_author=V.A. Petyuk; citation_author=L. Chen; citation_author=D. Ray; citation_author=CPTAC Investigators; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_firstpage=755; citation_lastpage=765; citation_publication_date=2016; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2930292-2%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419302922%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(19)30292-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419302922%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(19)30292-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419302922%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2930292-2%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419302922%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2930292-2%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419302922%253Fshowall%253Dtrue&amp;pii=S0092-8674%2819%2930292-2" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=cell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2930292-2%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419302922%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2930292-2%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS0092867419302922%253Fshowall%253Dtrue&amp;pii=S0092-8674%2819%2930292-2" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2019.03.030">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(23)X0017-8"><img src="/cms/asset/atypon:cms:attachment:img:d18e6:rev:1722883367683-3936:pii:S0092867423X00178/cover.tif.jpg" alt="Cover Image - Cell, Volume 187, Issue 16"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S0092-8674(19)30292-2"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Resource</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cell/issue?pii=S0092-8674(18)X0011-7" class="article-header__vol faded">
                                                    Volume 177, ISSUE 4</a><span></span><span>, </span><span class="article-header__pages faded">P1035-1049.e19, </span><span class="article-header__date faded">May 02, 2019</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=177&amp;issue=4&amp;pii=S0092-8674%2818%29X0011-7" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2819%2930292-2" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF [9 MB]</span><span class="alternative">PDF [9 MB]</span></a></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cell/ppt/S0092-8674(19)30292-2.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.cell.2019.03.030" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2019.03.030&amp;pii=S0092-8674%2819%2930292-2" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2019.03.030" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-248" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-248">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2819%2930292-2"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S0092867419302922&amp;issn=00928674&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867419302922&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Suhas Vasaikar">Suhas Vasaikar</a> <a href="#" data-db-target-for="crosref0020" aria-expanded="false" aria-controls="crosref0020" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>14</sup></span></a><div data-db-target-of="crosref0020" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">14 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Suhas Vasaikar</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">14 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Vasaikar%2C+Suhas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chen Huang">Chen Huang</a> <a href="#" data-db-target-for="crosref0035" aria-expanded="false" aria-controls="crosref0035" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>14</sup></span></a><div data-db-target-of="crosref0035" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">14 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">14 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaojing Wang">Xiaojing Wang</a> <a href="#" data-db-target-for="crosref0050" aria-expanded="false" aria-controls="crosref0050" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>12</sup></span></a><div data-db-target-of="crosref0050" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">12 Present address: Department of Epidemiology and Biostatistics, Greehey Children’s Cancer Research Institute, The University of Texas Health Science Center at San Antonio, San Antonio, TX 78229, USA<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>, </sup><a href="#" data-db-target-for="crosref0055" aria-expanded="false" aria-controls="crosref0055" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>14</sup></span></a><div data-db-target-of="crosref0055" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">14 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaojing Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">12 Present address: Department of Epidemiology and Biostatistics, Greehey Children’s Cancer Research Institute, The University of Texas Health Science Center at San Antonio, San Antonio, TX 78229, USA<br/>14 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Xiaojing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Vladislav A. Petyuk">Vladislav A. Petyuk</a> <a href="#" data-db-target-for="crosref0065" aria-expanded="false" aria-controls="crosref0065" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>14</sup></span></a><div data-db-target-of="crosref0065" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">14 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Vladislav A. Petyuk</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">14 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petyuk%2C+Vladislav A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sara R. Savage">Sara R. Savage</a> <a href="#" data-db-target-for="crosref0075" aria-expanded="false" aria-controls="crosref0075" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>14</sup></span></a><div data-db-target-of="crosref0075" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">14 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sara R. Savage</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">14 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biomedical Informatics, Vanderbilt University Medical Center, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Savage%2C+Sara R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bo Wen">Bo Wen</a><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yongchao Dou">Yongchao Dou</a><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yongchao Dou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dou%2C+Yongchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yun Zhang">Yun Zhang</a><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yun Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Yun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Zhiao Shi">Zhiao Shi</a><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Zhiao Shi</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Zhiao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Osama A. Arshad">Osama A. Arshad</a><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Osama A. Arshad</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Arshad%2C+Osama A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marina A. Gritsenko">Marina A. Gritsenko</a><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marina A. Gritsenko</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gritsenko%2C+Marina A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lisa J. Zimmerman">Lisa J. Zimmerman</a><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lisa J. Zimmerman</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zimmerman%2C+Lisa J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jason E. McDermott">Jason E. McDermott</a><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jason E. McDermott</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Therese R. Clauss">Therese R. Clauss</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Therese R. Clauss</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Clauss%2C+Therese R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ronald J. Moore">Ronald J. Moore</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ronald J. Moore</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Moore%2C+Ronald J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rui Zhao">Rui Zhao</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rui Zhao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Rui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew E. Monroe">Matthew E. Monroe</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew E. Monroe</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Monroe%2C+Matthew E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yi-Ting Wang">Yi-Ting Wang</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yi-Ting Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Yi-Ting" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew C. Chambers">Matthew C. Chambers</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew C. Chambers</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chambers%2C+Matthew C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Robbert J.C. Slebos">Robbert J.C. Slebos</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Robbert J.C. Slebos</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Slebos%2C+Robbert J.C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ken S. Lau">Ken S. Lau</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ken S. Lau</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lau%2C+Ken S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Qianxing Mo">Qianxing Mo</a> <a href="#" data-db-target-for="crosref0180" aria-expanded="false" aria-controls="crosref0180" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>13</sup></span></a><div data-db-target-of="crosref0180" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">13 Present address: Department of Biostatistics &amp; Bioinformatics, H. Lee Moffitt Cancer Center &amp; Research Institute, Tampa, FL 33612, USA<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Qianxing Mo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">13 Present address: Department of Biostatistics &amp; Bioinformatics, H. Lee Moffitt Cancer Center &amp; Research Institute, Tampa, FL 33612, USA<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mo%2C+Qianxing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">The McDonnell Genome Institute, Washington University in St. Louis, Forest Park Avenue, Campus Box 8501, St. Louis, MO 63108, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew Ellis">Matthew Ellis</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew Ellis</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Richard D. Smith">Richard D. Smith</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karin D. Rodland">Karin D. Rodland<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0230" aria-expanded="false" aria-controls="crosref0230" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>15</sup></span></a><div data-db-target-of="crosref0230" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">15 Senior author<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karin D. Rodland</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#bcd7ddced5d292ced3d8d0ddd2d8fcccd2d2d092dbd3ca" title="Send email to Karin D. Rodland"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">15 Senior author<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA<p></p>Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97221, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Karin D. Rodland" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#f893998a9196d68a979c9499969cb888969694d69f978e"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel C. Liebler">Daniel C. Liebler<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0245" aria-expanded="false" aria-controls="crosref0245" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>15</sup></span></a><div data-db-target-of="crosref0245" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">15 Senior author<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel C. Liebler</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#5430353a3d31387a383d31363831261422353a303126363d38207a313021" title="Send email to Daniel C. Liebler"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">15 Senior author<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biochemistry, Vanderbilt University, Nashville, TN 37232, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liebler%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Daniel C. Liebler" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#1276737c7b777e3c7e7b77707e77605264737c767760707b7e663c777667"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tao Liu">Tao Liu<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0260" aria-expanded="false" aria-controls="crosref0260" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>15</sup></span></a><div data-db-target-of="crosref0260" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">15 Senior author<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tao Liu</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#3f4b5e501153564a7f4f51515311585049" title="Send email to Tao Liu"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">15 Senior author<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99352, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Tao Liu" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#bcc8ddd392d0d5c9fcccd2d2d092dbd3ca"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref0285" aria-expanded="false" aria-controls="crosref0285" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>15</sup></span></a><div data-db-target-of="crosref0285" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">15 Senior author<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><sup>, </sup><a href="#" data-db-target-for="crosref0290" aria-expanded="false" aria-controls="crosref0290" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>16</sup></span></a><div data-db-target-of="crosref0290" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">16 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#91f3f8fff6bfebf9f0fff6d1f3f2fcbff4f5e4" title="Send email to Bing Zhang"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">15 Senior author<br/>16 Lead Contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Dan L Duncan Comprehensive Cancer Center, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Bing Zhang" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#57353e3930792d3f3639301735343a79323322"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Akhilesh Pandey" class="dropBlock__slider__ctrl">Akhilesh Pandey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Akhilesh back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Akhilesh Pandey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pandey%2C+Akhilesh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amanda Paulovich" class="dropBlock__slider__ctrl">Amanda Paulovich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amanda back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amanda Paulovich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Andrew Hoofnagle" class="dropBlock__slider__ctrl">Andrew Hoofnagle</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Andrew back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Andrew Hoofnagle</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoofnagle%2C+Andrew" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="D.R. Mani" class="dropBlock__slider__ctrl">D.R. Mani</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="D.R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">D.R. Mani</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel W. Chan" class="dropBlock__slider__ctrl">Daniel W. Chan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel W. Chan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David F. Ransohoff" class="dropBlock__slider__ctrl">David F. Ransohoff</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David F. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David F. Ransohoff</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ransohoff%2C+David F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Fenyo" class="dropBlock__slider__ctrl">David Fenyo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Fenyo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Fenyo%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David L. Tabb" class="dropBlock__slider__ctrl">David L. Tabb</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David L. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David L. Tabb</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tabb%2C+David L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Douglas A. Levine" class="dropBlock__slider__ctrl">Douglas A. Levine</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Douglas A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Douglas A. Levine</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Levine%2C+Douglas A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Emily S. Boja" class="dropBlock__slider__ctrl">Emily S. Boja</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Emily S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric Kuhn" class="dropBlock__slider__ctrl">Eric Kuhn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric Kuhn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kuhn%2C+Eric" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Forest M. White" class="dropBlock__slider__ctrl">Forest M. White</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Forest M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Forest M. White</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=White%2C+Forest M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gordon A. Whiteley" class="dropBlock__slider__ctrl">Gordon A. Whiteley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gordon A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gordon A. Whiteley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteley%2C+Gordon A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Heng Zhu" class="dropBlock__slider__ctrl">Heng Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Heng back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Heng Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Heng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hui Zhang" class="dropBlock__slider__ctrl">Hui Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hui Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ie-Ming Shih" class="dropBlock__slider__ctrl">Ie-Ming Shih</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ie-Ming back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ie-Ming Shih</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shih%2C+Ie-Ming" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jasmin Bavarva" class="dropBlock__slider__ctrl">Jasmin Bavarva</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jasmin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jasmin Bavarva</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bavarva%2C+Jasmin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey Whiteaker" class="dropBlock__slider__ctrl">Jeffrey Whiteaker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey Whiteaker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteaker%2C+Jeffrey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karen A. Ketchum" class="dropBlock__slider__ctrl">Karen A. Ketchum</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karen A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karl R. Clauser" class="dropBlock__slider__ctrl">Karl R. Clauser</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karl R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karl R. Clauser</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Clauser%2C+Karl R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kelly Ruggles" class="dropBlock__slider__ctrl">Kelly Ruggles</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kelly back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kelly Ruggles</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ruggles%2C+Kelly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kimberly Elburn" class="dropBlock__slider__ctrl">Kimberly Elburn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kimberly back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kimberly Elburn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Elburn%2C+Kimberly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Linda Hannick" class="dropBlock__slider__ctrl">Linda Hannick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Linda back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Linda Hannick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hannick%2C+Linda" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mark Watson" class="dropBlock__slider__ctrl">Mark Watson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mark back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mark Watson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Watson%2C+Mark" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mauricio Oberti" class="dropBlock__slider__ctrl">Mauricio Oberti</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mauricio back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mauricio Oberti</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Oberti%2C+Mauricio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mehdi Mesri" class="dropBlock__slider__ctrl">Mehdi Mesri</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mehdi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melinda E. Sanders" class="dropBlock__slider__ctrl">Melinda E. Sanders</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melinda E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melinda E. Sanders</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sanders%2C+Melinda E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melissa Borucki" class="dropBlock__slider__ctrl">Melissa Borucki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melissa Borucki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borucki%2C+Melissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael A. Gillette" class="dropBlock__slider__ctrl">Michael A. Gillette</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael A. Gillette</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Snyder" class="dropBlock__slider__ctrl">Michael Snyder</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Snyder</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Snyder%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Negin Vatanian" class="dropBlock__slider__ctrl">Negin Vatanian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Negin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Negin Vatanian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vatanian%2C+Negin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Paul A. Rudnick" class="dropBlock__slider__ctrl">Paul A. Rudnick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Paul A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Paul A. Rudnick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rudnick%2C+Paul A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Philip Mertins" class="dropBlock__slider__ctrl">Philip Mertins</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Philip back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Philip Mertins</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mertins%2C+Philip" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="R. Reid Townsend" class="dropBlock__slider__ctrl">R. Reid Townsend</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="R. Reid back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">R. Reid Townsend</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Townsend%2C+R. Reid" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert C. Rivers" class="dropBlock__slider__ctrl">Robert C. Rivers</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert C. Rivers</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rivers%2C+Robert C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Samuel H. Payne" class="dropBlock__slider__ctrl">Samuel H. Payne</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Samuel H. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sherri R. Davies" class="dropBlock__slider__ctrl">Sherri R. Davies</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sherri R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sherri R. Davies</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Davies%2C+Sherri R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephen E. Stein" class="dropBlock__slider__ctrl">Stephen E. Stein</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephen E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephen E. Stein</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stein%2C+Stephen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Steven A. Carr" class="dropBlock__slider__ctrl">Steven A. Carr</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Steven A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Steven A. Carr</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Steven J. Skates" class="dropBlock__slider__ctrl">Steven J. Skates</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Steven J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Steven J. Skates</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Skates%2C+Steven J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Subha Madhavan" class="dropBlock__slider__ctrl">Subha Madhavan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Subha back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Subha Madhavan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Madhavan%2C+Subha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tara Hiltke" class="dropBlock__slider__ctrl">Tara Hiltke</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Xian Chen" class="dropBlock__slider__ctrl">Xian Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Xian back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Xian Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xian" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yingming Zhao" class="dropBlock__slider__ctrl">Yingming Zhao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yingming back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yingming Zhao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Yingming" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yue Wang" class="dropBlock__slider__ctrl">Yue Wang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yue back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yue Wang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Yue" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhen Zhang" class="dropBlock__slider__ctrl">Zhen Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">12 Present address: Department of Epidemiology and Biostatistics, Greehey Children’s Cancer Research Institute, The University of Texas Health Science Center at San Antonio, San Antonio, TX 78229, USA<br/>13 Present address: Department of Biostatistics &amp; Bioinformatics, H. Lee Moffitt Cancer Center &amp; Research Institute, Tampa, FL 33612, USA<br/>14 These authors contributed equally<br/>15 Senior author<br/>16 Lead Contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">April 25, 2019</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.cell.2019.03.030">https://doi.org/10.1016/j.cell.2019.03.030</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.cell.2019.03.030"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2019.03.030" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cell/fulltext/S0092-8674(19)30400-3" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Trans Effects on Gene Expression Can Drive Omnigenic Inheritance</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="/cell/fulltext/S0092-8674(19)30273-9" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">Volumetric Ca2+ Imaging in the Mouse Brain Using Hybrid Multiplexed Sculpted Light Microscopy</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        

    

        


    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        


    

        


</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Systematic identification of colon cancer-associated proteins and phosphosites</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Proteomics-supported neoantigens and cancer/testis antigens in 78% of the tumors</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Rb phosphorylation is an oncogenic driver and a putative target in colon cancer</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Glycolysis inhibition may render MSI tumors more sensitive to checkpoint blockade</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical Abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical Abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/67d0ef8e-8f47-4219-8d72-3c80ec8a94ce/fx1_lrg.jpg" data-large-image="/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/cf26f780-0444-4700-bf6d-be0bf98863c2/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/67d0ef8e-8f47-4219-8d72-3c80ec8a94ce/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;colon cancer&#34;&amp;ISSN=0092-8674">colon cancer</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=0092-8674">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=0092-8674">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;tumor antigen&#34;&amp;ISSN=0092-8674">tumor antigen</a></li><li><a href="/action/doSearch?AllField=&#34;immune evasion&#34;&amp;ISSN=0092-8674">immune evasion</a></li><li><a href="/action/doSearch?AllField=&#34;glycolysis&#34;&amp;ISSN=0092-8674">glycolysis</a></li><li><a href="/action/doSearch?AllField=&#34;drug targets&#34;&amp;ISSN=0092-8674">drug targets</a></li><li><a href="/action/doSearch?AllField=&#34;biomarkers&#34;&amp;ISSN=0092-8674">biomarkers</a></li><li><a href="/action/doSearch?AllField=&#34;RB1&#34;&amp;ISSN=0092-8674">RB1</a></li><li><a href="/action/doSearch?AllField=&#34;SOX9&#34;&amp;ISSN=0092-8674">SOX9</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">Colorectal cancer (CRC) is the third most common cancer worldwide and the fourth leading cause of cancer-related deaths (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d4132558e928" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Arnold et al., 2017</a><div data-db-target-of="bib5-d4132558e928" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Arnold M. </li><li class="loa__item">Sierra M.S. </li><li class="loa__item">Laversanne M. </li><li class="loa__item">Soerjomataram I. </li><li class="loa__item">Jemal A. </li><li class="loa__item">Bray F. </li></ul><div class="reference-citations__detials">Global patterns and trends in colorectal cancer incidence and mortality.<div class="popup-ref"><span class="ref__series"><i>Gut.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">66</span></span>: <span class="ref__seriesPages">683-691</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1136%2Fgutjnl-2015-310912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26818619&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84962150091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3331)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=683-691&journal=Gut&author=Arnold+M.&author=Sierra+M.S.&author=Laversanne+M.&author=Soerjomataram+I.&author=Jemal+A.&author=Bray+F.&title=Global+patterns+and+trends+in+colorectal+cancer+incidence+and+mortality">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Recent studies of the genomic, transcriptomic, and proteomic landscapes of human CRC have identified many genomic alterations and have revealed extensive molecular heterogeneity of the disease (<div class="dropBlock reference-citations"><a href="#" id="back-bib8" data-db-target-for="bib8-d4132558e931" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Cancer Genome Atlas Network, 2012</a><div data-db-target-of="bib8-d4132558e931" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Network</span><br />Comprehensive molecular characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">487</span></span>: <span class="ref__seriesPages">330-337</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature11252&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22810696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84863922124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6415)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=330-337&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib19" data-db-target-for="bib19-d4132558e931" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e931" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib77" data-db-target-for="bib77-d4132558e931" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e931" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, the rapidly accumulating omics data have yet to bring novel biomarkers and drug targets to the clinic.</div><div class="section-paragraph">Global proteomic differences between tumor and normal tissues, which are critical for cancer biomarker discovery, have not been systematically characterized in large tumor cohorts. Signaling proteins and pathways are often attractive therapeutic targets for cancer treatment, yet global phosphoproteomic analyses on human CRC are lacking. Recent advances in cancer immunotherapy underscore the critical need for biomarkers to predict response to immune checkpoint inhibition and to select neoantigens for personalized vaccine development (<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d4132558e936" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Sharma et al., 2017</a><div data-db-target-of="bib54-d4132558e936" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sharma P. </li><li class="loa__item">Hu-Lieskovan S. </li><li class="loa__item">Wargo J.A. </li><li class="loa__item">Ribas A. </li></ul><div class="reference-citations__detials">Primary, Adaptive, and Acquired Resistance to Cancer Immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">707-723</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28187290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85012158583&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3269)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=707-723&journal=Cell&author=Sharma+P.&author=Hu-Lieskovan+S.&author=Wargo+J.A.&author=Ribas+A.&title=Primary%2C+Adaptive%2C+and+Acquired+Resistance+to+Cancer+Immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Proteogenomics can provide fresh approaches to these needs. Here, we describe a proteogenomic study from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) on a prospectively collected colon cancer cohort to systematically identify new therapeutic opportunities.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic Profiling</span></h3><div class="section-paragraph">We prospectively collected tumor specimens, matched normal adjacent tissues (NATs), and blood samples from 110 colon cancer patients. We performed whole-exome sequencing (WXS), copy-number array, RNA sequencing (RNA-seq), microRNA sequencing (miRNA-seq), and label-free shotgun proteomic analyses on the tumor samples, similar to our previous study (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e947" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e947" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To characterize the proteomes in cancer and normal states, we further performed isobaric tandem mass tag (TMT) labeling-based global and phosphoproteomic analysis on both tumors and NATs (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A). <a href="/cms/10.1016/j.cell.2019.03.030/attachment/87317e5c-393a-47d6-9c62-cb4fee1d653b/mmc1" class="scroll-into-link">Table S1</a> summarizes the clinical and pathological characteristics of the tumors.<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/51a6709b-fa20-49b7-bee7-cefe84668af4/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/e8617521-d0e3-4fac-aa70-3d2701ee388b/gr1_lrg.jpg" data-large-image="/cms/attachment/51a6709b-fa20-49b7-bee7-cefe84668af4/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Schematic Overview of the Study</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Samples and omics platforms for data generation.</div><div class="section-paragraph">(B) Therapeutic hypothesis generation through proteogenomic integration. The colors represent data generated from different omics platforms as indicated by the same colors in (A).</div><div class="section-paragraph">See also <a data-locator="figs1" aria-expanded="false" aria-controls="crosref1325" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs1">Figure S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref1330" href="/cms/10.1016/j.cell.2019.03.030/attachment/87317e5c-393a-47d6-9c62-cb4fee1d653b/mmc1" target="_blank">Table S1</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/51a6709b-fa20-49b7-bee7-cefe84668af4/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e8617521-d0e3-4fac-aa70-3d2701ee388b/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Although this study includes only colon tumors, rather than both colon and rectal tumors in The Cancer Genome Atlas (TCGA) cohort (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e959" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e959" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), the average mRNA profiles were highly correlated between the two cohorts (Pearson’s <em>r</em> = 0.92) as were the average label-free protein profiles (Pearson’s <em>r</em> = 0.96), and these correlations were higher than those between different cancer types or between colon tumors and cell lines (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>A–S1G). Principal component analysis clearly separated the tumors and NATs based on the TMT global or phosphoproteomics data, and no batch effect was observed between the TMT plexes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>H and S1I). Correlation of proteomics data between the label-free and TMT platforms was higher than for either with RNA-seq data (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>J–S1L). Label-free proteomics data from colon tumors outperformed RNA-seq data for gene function prediction, and TMT data further outperformed both (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs1" href="#figs1" data-locator="figs1" id="back-figs1" class="figure-link scroll-into-link">Figures S1</a>M–S1O). These results affirm the consistency of the two proteomic platforms and the added value of proteomics for assessing gene functions. Based on the comprehensive molecular profiling datasets, we performed integrative proteogenomic data analyses, focusing on using global and phosphoproteomics data to improve the interpretation of genomics data and to reveal new therapeutic opportunities (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B).<div class="floatDisplay"><figure id="figs1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/51d30281-9340-4eb2-8590-df93645881d8/figs1.jpg" alt="Figure thumbnail figs1" data-locator="figs1" data-high-res="/cms/attachment/68d3a60b-8278-4421-905f-04b9a0b0cba1/figs1_lrg.jpg" data-large-image="/cms/attachment/51d30281-9340-4eb2-8590-df93645881d8/figs1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S1" class="label figure__label">Figure S1</span><span class="figure__title__text">Data Quality Analysis, Related to <a data-locator="fig1" aria-expanded="false" aria-controls="crosref1395" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig1">Figure 1</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A–D) Average mRNA profile correlation between the prospective cohort (n = 106) and TCGA colorectal cohort (n = 90, A), TCGA breast cohort (n = 102, B), TCGA ovarian cohort (n = 374, C), and colorectal cell lines (n = 44, D). (E-G) Average label-free protein profile correlation between the prospective cohort (n = 100) and TCGA colorectal cohort (n = 95, E), colorectal cell lines (n = 44, F), and NCI-60 cell lines (n = 60, G). (H-I) Unsupervised principal component analysis (PCA) of the TMT global (H) or phosphoproteomics (I) data. Samples analyzed in different TMT plex are shown with different shapes. Tumor and normal samples are shown in different colors. Numbers in the parentheses are variance explained by PC1 and PC2. PCA was performed by metaX. (J-L) Distribution of gene-wise correlation between label-free proteomics and RNA-Seq data (J), between TMT proteomics and RNA-Seq data (K), and between TMT and label-free data (L). For each comparison, only genes quantifiable by both platforms and with the highest variances (top 10%) in either platforms were included in the analysis. (M-O) Scatterplots comparing the gene function prediction performance between co-expression networks derived from mRNA and label-free proteomics and RNA-Seq data (M), from TMT proteomics and RNA-Seq data (N), and from TMT and label-free data (O). Network-based gene function prediction was performed using the random walk-based network propagation algorithm for each KEGG pathway. Prediction performance was evaluated using 5-fold cross validation and quantified on the basis of the area under the receiver operating characteristic curve (AUROC). Results for each KEGG pathway term is represented as a dot.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/51d30281-9340-4eb2-8590-df93645881d8/figs1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/68d3a60b-8278-4421-905f-04b9a0b0cba1/figs1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic Mutations and Their Proteomic Consequences</span></h3><div class="section-paragraph">WXS analysis of 106 tumor specimens and matched blood samples identified 64,010 somatic single nucleotide variants (SNVs) and 7,691 somatic insertions/deletions (INDELs) (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figures S2</a>A and S2B). A focused analysis of microsatellites further identified 6,186 somatic microsatellite INDELs (MS INDELs) (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>C; <a href="/cms/10.1016/j.cell.2019.03.030/attachment/66edcd43-6f7c-4a3f-932c-f67c762caa96/mmc2" class="scroll-into-link">Table S2</a>). In total, we identified 56,592 unique somatic protein altering events (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/422700f5-078e-4f85-b043-ff1f75a164fd/mmc3" class="scroll-into-link">Table S3</a>).<div class="floatDisplay"><figure id="figs2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/483a2b0c-64db-4df1-be92-74bac53324d4/figs2.jpg" alt="Figure thumbnail figs2" data-locator="figs2" data-high-res="/cms/attachment/62f1f796-47e3-4ba6-a2f2-6f5f5a2c1e03/figs2_lrg.jpg" data-large-image="/cms/attachment/483a2b0c-64db-4df1-be92-74bac53324d4/figs2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S2" class="label figure__label">Figure S2</span><span class="figure__title__text">Somatic Mutations and Microsatellite Instability, Related to <a data-locator="fig2" aria-expanded="false" aria-controls="crosref1400" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig2">Figure 2</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Mutation rates for individual tumor samples. Mutation rate was calculated on the basis of MuTect2 somatic single nucleotide variant (SNV) calls. (B) Numbers of MuTect2 identified somatic INDELs for individual tumor samples. (C) Numbers of MSMuTect-Fisher identified somatic microsatellite INDELs (MS-INDELS) for individual tumor samples. (D) Tumor samples classified based on microsatellite instability (MSI) status (MSI-H: red versus MSS: white) or mutation rate (Hypermutated: red versus non-hypermutated: white). (E) Mutation spectrum for individual tumor samples. (F) Mutations in BRAF v600, mismatch repair genes, POLE, and KRAS among all samples.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/483a2b0c-64db-4df1-be92-74bac53324d4/figs2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/62f1f796-47e3-4ba6-a2f2-6f5f5a2c1e03/figs2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The number of MS INDELs showed a clear bimodal distribution, which allowed us to separate the samples into a microsatellite instability-high (MSI-H) group (n = 24) and a microsatellite stable (MSS) group (n = 82, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>D). For the 85 samples with PCR-based MSI testing results, WXS-based assignment agreed completely with PCR assignment (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/66edcd43-6f7c-4a3f-932c-f67c762caa96/mmc2" class="scroll-into-link">Table S2</a>). MSI-H tumors showed a distinct mutational spectrum with an increased proportion of A &gt; G/T &gt; C transitions and decreased G &gt; C/C &gt; G transversions compared to MSS tumors (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>E). The MSI-H group was enriched with mutations in the mismatch repair pathway and in the POLE and BRAF genes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>F).</div><div class="section-paragraph">To identify significantly mutated genes, we grouped the MSI-H and the one hypermutated MSS sample with a POLE mutation into a hypermutated group; the remaining samples formed a non-hypermutated group (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs2" href="#figs2" data-locator="figs2" id="back-figs2" class="figure-link scroll-into-link">Figure S2</a>D). In the non-hypermutated group, we identified eight significantly mutated genes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>A), which all were reported in the TCGA study (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d4132558e1024" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Cancer Genome Atlas Network, 2012</a><div data-db-target-of="bib8-d4132558e1024" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Network</span><br />Comprehensive molecular characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">487</span></span>: <span class="ref__seriesPages">330-337</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature11252&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22810696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84863922124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6415)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=330-337&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In the hypermutated group, we identified nine significantly mutated genes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B), six of which were not reported in the TCGA study. Four genes newly identified in this study, namely CASP5, RNF43, LTN1, and BMPR2, were mutated in more than 50% of the hypermutated samples.<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/3c8330e4-86c5-4719-89cf-c17f92de7540/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/8e95e863-872a-49ec-a9f3-8f3770792361/gr2_lrg.jpg" data-large-image="/cms/attachment/3c8330e4-86c5-4719-89cf-c17f92de7540/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text">Somatic Mutations and Their Proteomic Consequences</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A and B) Significantly mutated genes in non-hypermutated (A) and hypermutated (B) samples. Mutation frequency is shown at the top for each gene. Genes not reported in the TCGA study are shown in bold font.</div><div class="section-paragraph">(C–F) Somatic mutations versus protein/phosphosite abundance change for APC (C), TGFBR2 (D), TP53 (E), and SOX9 (F). For each gene, the top panel lollipop plot visualizes all protein altering somatic mutations detected in this cohort. The size of a lollipop represents the number of samples with corresponding mutation, and the color represents a specific type of mutation as indicated in the figure legend. The location of the post-translational modification (PTM) of interest is indicated by a triangle. The bottom panel co-visualizes the mutation and protein or phosphosite abundance data for individual samples. For mutation data, a colored box denotes the existence of a specific type of mutation as indicated in the figure legend. Gray boxes indicate data are not available. If a given sample has more than one type of mutation, only one type is shown in the following order of priority: stop-gain, frameshift-INDEL, non-frameshift INDEL, and non-synonymous SNV. In the waterfall plot, each bar represents the protein or phosphosite abundance change between tumor and matched normal adjacent tissue for a patient. Red and green bars represent over- and under-expression in tumor, respectively. White space in the waterfall plots indicates missing values.</div><div class="section-paragraph">See also <a data-locator="figs2" aria-expanded="false" aria-controls="crosref1335" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs2">Figure S2</a> and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref1340" href="/cms/10.1016/j.cell.2019.03.030/attachment/66edcd43-6f7c-4a3f-932c-f67c762caa96/mmc2" target="_blank">Tables S2</a> and <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref1345" href="/cms/10.1016/j.cell.2019.03.030/attachment/422700f5-078e-4f85-b043-ff1f75a164fd/mmc3" target="_blank">S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/3c8330e4-86c5-4719-89cf-c17f92de7540/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/8e95e863-872a-49ec-a9f3-8f3770792361/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">The TMT data on matched tumors and NATs allowed us to investigate the proteomic consequences of somatic mutations. Some protein changes could be predicted from the mutations, such as the stop-gain and frameshift mutations in APC, which result in nonsense-mediated mRNA decay or truncated proteins. As expected, tumors with these mutations had reduced abundance of phosphosite APC-T2451, which is located downstream of the mutations (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>C). Similarly, tumor samples bearing frameshift mutations in TGFBR2 showed reduced abundance of phosphosite TGFBR2-S553 (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D).</div><div class="section-paragraph">Some protein changes were difficult to predict based on the mutations. For example, phosphorylation of TP53-S315 was increased in tumors over matched NATs (p = 0.001, paired t test). Three tumors with more than 10-fold increase of TP53-S315 phosphorylation all had an R273 mutation (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>E). We reviewed TP53-S315 phosphorylation data from the CPTAC breast (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d4132558e1043" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Mertins et al., 2016</a><div data-db-target-of="bib45-d4132558e1043" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and ovarian cancer (<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d4132558e1046" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Zhang et al., 2016</a><div data-db-target-of="bib78-d4132558e1046" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang H. </li><li class="loa__item">Liu T. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Zhang B. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Zhou J.Y. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Chen L. </li><li class="loa__item">Ray D. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>CPTAC Investigators</span><br />Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) studies and found TP53-S315 phosphorylation in tumors with R273 mutations was 3.4- to 83-fold above the medians of those cohorts, strengthening the association between these hotspot mutations and increased TP53-S315 phosphorylation.</div><div class="section-paragraph">The high mobility group (HMG) transcription factor SOX9 was recurrently mutated in this cohort, including six stop-gain mutations, eight frameshift mutations, and five nonsynonymous SNVs. According to the “20/20 rule” that classifies genes with more than 20% truncating mutations as tumor suppressor genes (<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d4132558e1051" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Vogelstein et al., 2013</a><div data-db-target-of="bib65-d4132558e1051" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vogelstein B. </li><li class="loa__item">Papadopoulos N. </li><li class="loa__item">Velculescu V.E. </li><li class="loa__item">Zhou S. </li><li class="loa__item">Diaz Jr., L.A. </li><li class="loa__item">Kinzler K.W. </li></ul><div class="reference-citations__detials">Cancer genome landscapes.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">339</span></span>: <span class="ref__seriesPages">1546-1558</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1126%2Fscience.1235122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23539594&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84875490185&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5727)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1546-1558&journal=Science&author=Vogelstein+B.&author=Papadopoulos+N.&author=Velculescu+V.E.&author=Zhou+S.&author=Diaz+L.A.&author=Kinzler+K.W.&title=Cancer+genome+landscapes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), SOX9 should be classified as a tumor suppressor. However, SOX9 protein was significantly overexpressed in tumor samples compared to matched NATs (p = 1.02 × 10<sup>−10</sup>, paired t test, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F), which argues against a tumor suppressor assignment. Even in tumors with the truncating mutations, despite the small number of samples, we still observed statistically significant overexpression of SOX9 (p = 0.04, paired t test, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F). Interestingly, most of the truncating mutations occurred downstream of the HMG-box domain and upstream of the evolutionally conserved ubiquitin-target site K398 (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>F), which is responsible for 26S proteasome dependent degradation of SOX9 (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d4132558e1067" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">Akiyama et al., 2005</a><div data-db-target-of="bib1-d4132558e1067" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Akiyama H. </li><li class="loa__item">Kamitani T. </li><li class="loa__item">Yang X. </li><li class="loa__item">Kandyil R. </li><li class="loa__item">Bridgewater L.C. </li><li class="loa__item">Fellous M. </li><li class="loa__item">Mori-Akiyama Y. </li><li class="loa__item">de Crombrugghe B. </li></ul><div class="reference-citations__detials">The transcription factor Sox9 is degraded by the ubiquitin-proteasome system and stabilized by a mutation in a ubiquitin-target site.<div class="popup-ref"><span class="ref__series"><i>Matrix Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">499-505</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.matbio.2004.10.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=15694126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-13444260932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=499-505&journal=Matrix+Biol.&author=Akiyama+H.&author=Kamitani+T.&author=Yang+X.&author=Kandyil+R.&author=Bridgewater+L.C.&author=Fellous+M.&author=Mori-Akiyama+Y.&author=de+Crombrugghe+B.&title=The+transcription+factor+Sox9+is+degraded+by+the+ubiquitin-proteasome+system+and+stabilized+by+a+mutation+in+a+ubiquitin-target+site">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thus, removal of the K398 ubiquitination site through the truncating mutations may stabilize SOX9 protein and increase protein abundance. Consistent with these data, functional assays support an oncogenic rather than tumor suppressor role of SOX9 in primary CRC cells (<div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d4132558e1070" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Matheu et al., 2012</a><div data-db-target-of="bib43-d4132558e1070" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matheu A. </li><li class="loa__item">Collado M. </li><li class="loa__item">Wise C. </li><li class="loa__item">Manterola L. </li><li class="loa__item">Cekaite L. </li><li class="loa__item">Tye A.J. </li><li class="loa__item">Canamero M. </li><li class="loa__item">Bujanda L. </li><li class="loa__item">Schedl A. </li><li class="loa__item">Cheah K.S. </li><li>et al.</li></ul><div class="reference-citations__detials">Oncogenicity of the developmental transcription factor Sox9.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">72</span></span>: <span class="ref__seriesPages">1301-1315</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1158%2F0008-5472.CAN-11-3660&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22246670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84857700780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=1301-1315&journal=Cancer+Res.&author=Matheu+A.&author=Collado+M.&author=Wise+C.&author=Manterola+L.&author=Cekaite+L.&author=Tye+A.J.&author=Canamero+M.&author=Bujanda+L.&author=Schedl+A.&author=Cheah+K.S.&title=Oncogenicity+of+the+developmental+transcription+factor+Sox9">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Taken together, our somatic mutation analyses identified new significantly mutated genes among MSI-H tumors, and the proteomics data revealed unexpected functional complexity that could not have been predicted from mutation data alone.</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteomics Data Prioritize Somatic Copy-Number Alterations</span></h3><div class="section-paragraph">We performed somatic copy-number alteration (SCNA) analyses with WXS data and Illumina SNP array data, but the SNP array data showed low dynamic range. Hence, we only report SCNAs identified from WXS analysis. We identified very similar arm level SCNAs to the TCGA cohort, including amplifications of 1q, 7p and q, 8p and q, 13q, and 20p and q, and deletions of 1p, 14q, 15q, 17p and q, 18p and q, and 22q (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>A). Focal level SCNAs (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B) did not completely agree with those reported in the TCGA study, which may reflect cohort and/or platform differences. Nonetheless, most focal SCNAs previously reported were found in our cohort. Cytobands in chromosome 20 (20p12.1, 20q13.12, 20q13.13) and chromosome 18 (18q21.2) contained the most frequently amplified and deleted focal regions, respectively (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B).<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/1bd75c32-cca6-46fd-9d9b-50cea43ff2f4/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/f058332e-86f8-4223-9a8f-523597d50669/gr3_lrg.jpg" data-large-image="/cms/attachment/1bd75c32-cca6-46fd-9d9b-50cea43ff2f4/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Somatic Copy-Number Alteration Analysis</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Arm-level somatic copy-number alteration (SCNA) events. Red denotes amplification and blue denotes deletion.</div><div class="section-paragraph">(B) Focal-level SCNA events. Focal peaks with significant copy-number gains (red) and losses (blue) (GISTIC2 Q-values &lt;0.25) are shown. The top ten amplified and deleted cytobands are labeled, with the proportions of amplified or deleted samples shown in the parentheses. Representative genes encoded from these focal peaks are highlighted in approximate positions across the genome.</div><div class="section-paragraph">(C) Effects of copy-number alternations on mRNA and protein abundance. The upper heatmap shows the abundance of significant copy-number correlation with mRNA (left) and protein (right). Significant positive and negative correlations (adj. p &lt; 0.01, Spearman’s correlation coefficient) are indicated by red and blue, respectively. Genes are ordered by chromosome locations on both x- and y-axes. The bottom shows the frequency of significant correlations. Gray bars represent copy-number correlation to mRNA (left) and protein (right), and black bars represent copy-number correlation to both mRNA and protein.</div><div class="section-paragraph">(D) Strategy for prioritizing genes in focal alteration peaks.</div><div class="section-paragraph">(E) Most enriched KEGG pathways and Gene Ontology (GO) biological processes (BP) for genomic drivers inferred in this study.</div><div class="section-paragraph">(F) Six deleted genes involved in endocytosis. Violin plots compare protein expression in tumor and normal adjacent tissue for each gene.</div><div class="section-paragraph">See also <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref1350" href="/cms/10.1016/j.cell.2019.03.030/attachment/95453698-dea5-457c-9f18-d69fd79b7e69/mmc4" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/1bd75c32-cca6-46fd-9d9b-50cea43ff2f4/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/f058332e-86f8-4223-9a8f-523597d50669/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Next, we examined the correlations of SCNA with mRNA and protein abundance. While <em>cis</em>-effects of SCNAs on mRNA abundance were similar to our previous study (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e1094" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e1094" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), <em>cis</em>-effects on protein abundance were stronger (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C), reflecting the greater quantitative precision of the TMT platform compared to the previous label-free analysis. We also confirmed previously reported <em>trans</em>-acting SCNA hotspots on chromosomes 20q, 18, 16, 13, and 7 and identified a new hotspot on chromosome 14 (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C).</div><div class="section-paragraph">Previously, we showed that correlated copy number, mRNA, and protein levels can prioritize copy-number drivers in focal alteration regions. Here, we found that only 59% of such prioritized amplification or deletion drivers showed expected protein-level effects in tumor versus NAT comparisons (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>D). Thus, data from NATs can substantially refine candidate driver lists (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/95453698-dea5-457c-9f18-d69fd79b7e69/mmc4" class="scroll-into-link">Table S4</a>). The final prioritized list included the previously reported 20q amplification drivers HNF4A and TOMM34. We also identified the well-known tumor suppressor SMAD4 in the 18p focal deletion region.</div><div class="section-paragraph">To better understand the genomic drivers inferred in this study, including the 17 significantly mutated genes and the 90 prioritized SCNA genes, we performed enrichment analysis. Not surprisingly, significantly enriched terms included colorectal cancer, cell proliferation, cell death, and Hippo signaling pathway (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>E). Enrichment for endosome organization and endocytosis, including six genes located in different focal deletion regions across the genome (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>F), indicated that multiple deletion events converge to repress the endocytosis pathway, which may allow tumors to gain self-sufficiency in growth signals (<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d4132558e1124" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Mosesson et al., 2008</a><div data-db-target-of="bib47-d4132558e1124" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mosesson Y. </li><li class="loa__item">Mills G.B. </li><li class="loa__item">Yarden Y. </li></ul><div class="reference-citations__detials">Derailed endocytosis: an emerging feature of cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">835-850</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnrc2521&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18948996&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-54549102284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (590)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=835-850&journal=Nat.+Rev.+Cancer&author=Mosesson+Y.&author=Mills+G.B.&author=Yarden+Y.&title=Derailed+endocytosis%3A+an+emerging+feature+of+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Rb Phosphorylation as a Driver and Therapeutic Target in Colon Cancer</span></h3><div class="section-paragraph">One of the recurrently amplified genes in this cohort was the retinoblastoma (RB1) gene (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A). Consistently, the RB1 protein (Rb) was overexpressed in tumors compared to NATs (p = 2.10 × 10<sup>−15</sup>, paired t test, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>B). RB1 was the first tumor suppressor gene identified, and its amplification and overexpression in colon cancer contradict its frequent mutation and deletion in other cancers (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs3" href="#figs3" data-locator="figs3" id="back-figs3" class="figure-link scroll-into-link">Figure S3</a>).<div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/fa411723-5748-400c-98c0-899365c70194/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/e18b87cd-7fed-40df-a33a-9fe5292bc158/gr4_lrg.jpg" data-large-image="/cms/attachment/fa411723-5748-400c-98c0-899365c70194/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Rb Phosphorylation as a Driver and Therapeutic Target in Colon Cancer</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A–C) RB1 Copy-number alteration (CNA) (A), protein log2 fold change (FC) from normal (B), and phosphorylation log2 fold change from normal (C). Samples are ordered by increasing average phosphorylation abundance.</div><div class="section-paragraph">(D and E) Correlations of Rb protein abundance change (D) and average Rb phosphorylation change (E) with estimated E2F1 activity change.</div><div class="section-paragraph">(F–H) Correlation of average Rb phosphorylation change with estimated CDK2 activity change (F), H3.1 phosphorylation change (G), and protein level changes of apoptotic proteins (H).</div><div class="section-paragraph">(I) A model depicting the multi-level regulation of RB1 in colon cancer, highlighting Rb phosphorylation as a driver and therapeutic target in colon cancer.</div><div class="section-paragraph">The p values in (D)–(H) were calculated by Pearson's correlation test. See also <a data-locator="figs3" aria-expanded="false" aria-controls="crosref1355" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs3">Figure S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/fa411723-5748-400c-98c0-899365c70194/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e18b87cd-7fed-40df-a33a-9fe5292bc158/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div><div class="floatDisplay"><figure id="figs3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/442dc650-1613-4632-bfd8-100657f4a3cc/figs3.jpg" alt="Figure thumbnail figs3" data-locator="figs3" data-high-res="/cms/attachment/8349e363-6e8b-4aad-a1e1-12c7e9003f3f/figs3_lrg.jpg" data-large-image="/cms/attachment/442dc650-1613-4632-bfd8-100657f4a3cc/figs3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S3" class="label figure__label">Figure S3</span><span class="figure__title__text">RB1 Mutation Frequency in Human Cancer, Related to <a data-locator="fig4" aria-expanded="false" aria-controls="crosref1405" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig4">Figure 4</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">The tumor suppressor gene RB1 is frequently mutated in retinoblastoma, bladder cancer, small cell lung cancer, and many other human cancers. However, it is rarely mutated and is even amplified in colorectal cancer. Figure was generated using cBioPortal (<a class="inter-ref-link" href="http://www.cbioportal.org/" target="_blank">http://www.cbioportal.org/</a>).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/442dc650-1613-4632-bfd8-100657f4a3cc/figs3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/8349e363-6e8b-4aad-a1e1-12c7e9003f3f/figs3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">As a tumor-suppressor, Rb prevents cell proliferation by inhibiting E2F transcription factors, but this inhibition is abolished by Rb phosphorylation (<div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d4132558e1148" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Rubin, 2013</a><div data-db-target-of="bib52-d4132558e1148" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rubin S.M. </li></ul><div class="reference-citations__detials">Deciphering the retinoblastoma protein phosphorylation code.<div class="popup-ref"><span class="ref__series"><i>Trends Biochem. Sci.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">12-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23218751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871513029&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=12-19&journal=Trends+Biochem.+Sci.&author=Rubin+S.M.&title=Deciphering+the+retinoblastoma+protein+phosphorylation+code">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphoproteomics quantified six Rb phosphorylation sites in at least 50% of all samples, including four sites (i.e., T373, S807, S811, and T826) that regulate E2F binding (<div class="dropBlock reference-citations"><a href="#" id="back-bib26" data-db-target-for="bib26-d4132558e1151" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Knudsen and Wang, 1997</a><div data-db-target-of="bib26-d4132558e1151" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Knudsen E.S. </li><li class="loa__item">Wang J.Y. </li></ul><div class="reference-citations__detials">Dual mechanisms for the inhibition of E2F binding to RB by cyclin-dependent kinase-mediated RB phosphorylation.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">1997; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">5771-5783</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1128%2FMCB.17.10.5771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=9315635&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0030849455&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=5771-5783&journal=Mol.+Cell.+Biol.&author=Knudsen+E.S.&author=Wang+J.Y.&title=Dual+mechanisms+for+the+inhibition+of+E2F+binding+to+RB+by+cyclin-dependent+kinase-mediated+RB+phosphorylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib52" data-db-target-for="bib52-d4132558e1151" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Rubin, 2013</a><div data-db-target-of="bib52-d4132558e1151" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rubin S.M. </li></ul><div class="reference-citations__detials">Deciphering the retinoblastoma protein phosphorylation code.<div class="popup-ref"><span class="ref__series"><i>Trends Biochem. Sci.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">12-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23218751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871513029&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=12-19&journal=Trends+Biochem.+Sci.&author=Rubin+S.M.&title=Deciphering+the+retinoblastoma+protein+phosphorylation+code">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The average abundance of the four sites was higher in colon tumors than in NATs (p &lt; 2.2 × 10<sup>−16</sup>, paired t test) (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). Moreover, the average tumor versus NAT change for phospho-Rb was 1.84-fold, which was significantly higher (p = 0.01, Wilcoxon signed-rank test) than the 1.58-fold average change for total Rb measured from global proteomics. Thus, tumor samples had not only higher total Rb, but also a higher proportion of phospho-Rb within the total Rb pool.</div><div class="section-paragraph">Predicted E2F1 activity changes between tumors and NATs (<a href="#sec5" class="scroll-into-link">STAR Methods</a>) were positively correlated with both total Rb change (Pearson’s <em>r</em> = 0.40, p = 5.1 × 10<sup>−5</sup>, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>D) and phospho-Rb change (Pearson’s <em>r</em> = 0.30, p = 3.5 × 10<sup>−3</sup>, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>E). Because un-phosphorylated Rb inhibits E2F1 activity, the significant positive correlation between total Rb and E2F1 activity suggests that the increase of total Rb in tumors is attributable to phospho-Rb, rather than un-phosphorylated Rb. The positive correlation between E2F1 activity and phospho-Rb may reflect a positive feedback loop (<div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d4132558e1182" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Sherr and McCormick, 2002</a><div data-db-target-of="bib55-d4132558e1182" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sherr C.J. </li><li class="loa__item">McCormick F. </li></ul><div class="reference-citations__detials">The RB and p53 pathways in cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">103-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=12204530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0001510491&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1378)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=103-112&journal=Cancer+Cell&author=Sherr+C.J.&author=McCormick+F.&title=The+RB+and+p53+pathways+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), in which E2F1 transcriptionally upregulates cyclin E, which activates CDK2, thereby phosphorylating Rb and relieving E2F1 inhibition of cell proliferation. Indeed, phospho-Rb change showed the highest correlation with the predicted activity change of CDK2 (<em>r</em> = 0.47, p = 1.8 × 10<sup>−6</sup>, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>F) compared to all other kinases (<a href="#sec5" class="scroll-into-link">STAR Methods</a>). Further, phospho-Rb change was significantly correlated with the phosphorylation change of histone H3.1, a marker for cell proliferation (Pearson’s <em>r</em> = 0.49, p = 2.6 × 10<sup>−4</sup>, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>G). Therefore, our data showed that increased Rb phosphorylation, instead of RB1 mutation or deletion, drives colon cancer proliferation. However, it remains puzzling why RB1 is recurrently amplified in colon tumors, because post-translational downregulation of the inhibitory activity of this overexpressed protein seems mechanistically inefficient. Further analysis showed that Rb phosphorylation change in colon tumors was negatively correlated with the apoptosis hallmark gene set (<a href="#sec5" class="scroll-into-link">STAR Methods</a>, Pearson’s <em>r</em> = −0.28, p = 5.3 × 10<sup>−3</sup>, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>H), which primarily comprises pro-apoptotic genes (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d4132558e1217" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Liberzon et al., 2015</a><div data-db-target-of="bib36-d4132558e1217" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5972)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Thus, phospho-Rb appears to have an antiapoptotic role.</div><div class="section-paragraph">Together, proteogenomic data integration suggests that Rb phosphorylation regulates both proliferation and apoptosis to drive colon cancer development (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>I). This insight reveals a previously unexploited opportunity to target Rb phosphorylation in colon cancer through CDK2 inhibition, which is not possible for cancers driven by RB1 mutation or deletion.</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Colon Cancer-Associated Proteomic Events and Potential Clinical Utilities</span></h3><div class="section-paragraph">TMT global proteomic analysis of the 96 tumor and NAT pairs identified a total of 8,067 proteins. Among the 6,422 proteins that were quantified in at least 50% of the samples (i.e., quantifiable proteins), 2,217 (35%) were significantly increased and 2,527 (39%) were significantly decreased in tumors compared to paired NATs (adjusted [adj.] p &lt; 0.01, Wilcoxon signed-rank test, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A). Of these, 31 increased and 417 decreased proteins had a more than 2-fold abundance change. The 417 proteins elevated in NATs were significantly enriched in muscle-related functions (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B), as expected from normal colon histology, which includes defined muscular structures that are typically absent in tumors. We focused our analysis on the 31 proteins elevated by more than 2-fold in tumors, which we defined as colon cancer-associated proteins (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5" class="scroll-into-link">Table S5</a>).<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/07b7d73b-264b-4446-8d5d-04d9d739683e/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/47c6e981-2046-467f-bd05-01b2c130e317/gr5_lrg.jpg" data-large-image="/cms/attachment/07b7d73b-264b-4446-8d5d-04d9d739683e/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Colon Cancer-Associated Proteomic Events</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Volcano plot indicating proteins overexpressed in tumors or normal adjacent tissues (NATs, light red and blue colors indicate adj. p &lt; 0.01 (sig) whereas red and blue further require more than 2-fold change); other genes are colored in gray.</div><div class="section-paragraph">(B) Gene Ontology biological processes enriched for the 417 proteins downregulated in tumors. Venn diagram depicts the overlap between muscle system process related genes and the 417 proteins.</div><div class="section-paragraph">(C) Log2-fold change between tumor and matched NATs is shown for the 31 cancer-associated proteins (mean in red).</div><div class="section-paragraph">(D) Tumor-cell specific immunohistochemistry (IHC) staining scores defined by the Human Protein Atlas (HPA).</div><div class="section-paragraph">(E) Overlap with plasma proteins, secreted proteins, transmembrane proteins, and enzymes annotated by HPA, as well as known clinical utilities.</div><div class="section-paragraph">(F) Volcano plot indicating phosphosites overexpressed in tumors or NATs. Colors are the same as in (A).</div><div class="section-paragraph">(G) Correlation between tumor-normal protein and phosphorylation site abundance differences (Pearson’s <em>r</em> = 0.81, p &lt; 2.2 × 10<sup>−16</sup>). The purple dashed line indicates the diagonal line. Red points indicate the phosphorylation sites with &gt;2-fold increase. The black arrows highlight 5 of these phosphorylation sites with lower protein abundance in tumors than in NATs.</div><div class="section-paragraph">(H) Overlap of proteins containing cancer-associated phosphorylation sites (phosphoproteome), cancer-associated proteins (proteome), and cancer genes in the Cancer Gene Census (CGC).</div><div class="section-paragraph">(I) Cancer-associated kinases identified by increased phosphorylation of a known kinase activating site in tumor compared to NAT (phosphorylation) or by phosphosite set enrichment analysis based on known kinase-target site relationships (inferred). Gray boxes indicate data are not available. Black boxes indicate the existence of an FDA-approved drug or a drug undergoing clinical trials targeting that kinase.</div><div class="section-paragraph">(J) The number of proteomics-supported neoantigens identified for each sample, with MSI-H and MSS annotation shown at the top.</div><div class="section-paragraph">(K) Three cancer/testis (CT) antigens overexpressed by at least 2-fold in tumors compared to NATs in more than 5% of all samples, with the percentage indicated in brackets. Sample order is the same as in (J). Gray boxes indicate data are not available.</div><div class="section-paragraph">See also <a data-locator="figs4" aria-expanded="false" aria-controls="crosref1360" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs4">Figures S4</a> and <a data-locator="figs5" aria-expanded="false" aria-controls="crosref1365" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs5">S5</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref1370" href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5" target="_blank">Tables S5</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref1375" href="/cms/10.1016/j.cell.2019.03.030/attachment/30cfa83a-177e-4ed2-90a2-2d401364e784/mmc6" target="_blank">S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/07b7d73b-264b-4446-8d5d-04d9d739683e/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/47c6e981-2046-467f-bd05-01b2c130e317/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph"><a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C depicts the distributions of pairwise tumor-NAT differences for the 31 cancer-associated proteins. Most showed highly homogeneous differential expression across the 96 tumor-NAT pairs. Eight proteins were increased in tumor in more than 95% of the pairs, including DDX21 (100%), S100A11 (100%), RSL1D1 (99%), S100P (97%), RPL36A (97%), PLOD2 (96%), SERPINH1 (95%), and GPRC5A (95%). Among the 30 proteins with immunohistochemistry (IHC) staining data in The Human Protein Atlas (HPA), around half showed medium to strong tumor-specific staining in CRC samples (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figures 5</a>D and <a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs4" href="#figs4" data-locator="figs4" id="back-figs4" class="figure-link scroll-into-link">S4</a>).<div class="floatDisplay"><figure id="figs4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/be0c8f28-e55c-4f2c-9a65-fd417fbe2c7a/figs4.jpg" alt="Figure thumbnail figs4" data-locator="figs4" data-high-res="/cms/attachment/5befc21b-4d29-47b2-8c20-c883ef1e4e23/figs4_lrg.jpg" data-large-image="/cms/attachment/be0c8f28-e55c-4f2c-9a65-fd417fbe2c7a/figs4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S4" class="label figure__label">Figure S4</span><span class="figure__title__text">Human Protein Atlas Immunohistochemistry Staining Data for the 31 Cancer-Associated Proteins, Related to <a data-locator="fig5" aria-expanded="false" aria-controls="crosref1410" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Proportions of colorectal tumors with high, medium, or low staining, or not detected (ND) as reported by HPA. (B) Representative IHC images for individual proteins.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/be0c8f28-e55c-4f2c-9a65-fd417fbe2c7a/figs4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/5befc21b-4d29-47b2-8c20-c883ef1e4e23/figs4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We associated the 31 proteins with the human secretome, membrane proteome, and enzymes because these sub-proteomes are enriched with clinically approved biomarkers and drug targets. Nineteen have been found in plasma, 18 are secreted, nine are trans-membrane, and eight are enzymes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E). Fifteen of these proteins have known clinical utilities as diagnostic markers, outcome markers, or therapeutic targets (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5" class="scroll-into-link">Table S5</a>), including CEACAM5, which is the most widely used CRC marker in clinical practice (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d4132558e1259" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Duffy, 2001</a><div data-db-target-of="bib15-d4132558e1259" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Duffy M.J. </li></ul><div class="reference-citations__detials">Carcinoembryonic antigen as a marker for colorectal cancer: is it clinically useful?.<div class="popup-ref"><span class="ref__series"><i>Clin. Chem.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">624-630</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fclinchem%2F47.4.624&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=11274010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0035059972&amp;cf=" onclick="newWindow(this.href);return false">Scopus (610)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2001&pages=624-630&journal=Clin.+Chem.&author=Duffy+M.J.&title=Carcinoembryonic+antigen+as+a+marker+for+colorectal+cancer%3A+is+it+clinically+useful%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Other proteins may merit further investigation.</div><div class="section-paragraph">We also assessed the differences in phosphorylation site abundance between 96 tumor and NAT pairs. Among the 7,295 phosphorylation sites quantified in at least 50% of the paired samples, 2,119 (29%) were significantly increased and 3,053 (42%) were significantly decreased in tumors compared to paired NATs (adj. p &lt; 0.01, Wilcoxon signed-rank test). Of these, 63 were increased and 793 were decreased with a &gt;2-fold abundance change (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>F). We defined the 63 phosphosites mapping to 50 proteins as cancer-associated phosphosites (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5" class="scroll-into-link">Table S5</a>), of which all but one had greater changes in phosphosite abundance than in corresponding protein abundance (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>G, red dots). Despite the overall concordance between phosphosite abundance changes and corresponding protein abundance changes (Pearson’s <em>r</em> = 0.81, p &lt; 2.2 × 10<sup>−16</sup>), some cancer-associated phosphosites mapped to proteins that were decreased in tumor samples (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>G, highlighted by black arrows). Only four of the 50 proteins with a cancer-associated phosphosite also met our criteria as cancer-associated proteins. Moreover, only seven proteins in our combined protein set were documented in the Cancer Gene Census, a comprehensive catalog of genes containing mutations implicated in cancer (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>H). Proteomics and phosphoproteomics data thus are complementary and both reveal additional colon cancer genes that were missed in genomic studies.</div><div class="section-paragraph">Kinases are among the most attractive therapeutic targets for cancer treatment. We predicted cancer-associated kinase activity based on (1) significantly increased phosphorylation of a known kinase activating site in tumor compared to NAT, and on (2) enrichment analysis of the known target sites for each kinase. We identified four kinases by each method, with one kinase, CDK7, identified by both (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>I). Among these kinases, CDK4 is targeted by Food and Drug Administration (FDA)-approved drugs, and the other CDKs (CDK1, CDK2, and CDK7), MELK, and PFKFB3 are targeted by drugs in clinical trials (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5" class="scroll-into-link">Table S5</a>). The last kinase, PI4KB, may merit further investigation as a novel candidate therapeutic target.</div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor Antigens</span></h3><div class="section-paragraph">To facilitate the development of personalized vaccines for cancer immunotherapy, we further analyzed our data to identify candidate tumor antigens, including neoantigens derived from somatic mutations and non-mutated cancer/testis (CT) antigens. To identity candidate neoantigens, we searched the label-free and TMT global proteomics and TMT phosphoproteomics data against customized protein databases incorporating all coding variations identified from matched exome sequencing and RNA-seq data and found 173 proteomics-supported somatic mutations. All peptides of 9–11 amino acids in length that contained one of these somatic mutations were evaluated for human leukocyte antigen (HLA)-I binding affinity. The 88 mutant peptides with high predicted binding affinity to HLA molecules were considered as putative neoantigens (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/30cfa83a-177e-4ed2-90a2-2d401364e784/mmc6" class="scroll-into-link">Table S6</a>). In total, one or more putative neoantigens were identified for 38% of the tumors (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>J).</div><div class="section-paragraph">The TMT global proteomics data identified a total of 16 CT-antigens, and three were increased by at least 2-fold in tumors compared to paired NATs in more than 5% of all tumor-NAT pairs, including IGF2BP3 (51%), SPAG1 (14%), and ATAD2 (8%) (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>K). According to data in HPA, IGF2BP3 protein expression in normal human tissues is restricted to reproductive organs and fetal brain (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>A). While normal colon tissue stains negatively for IGF2BP3, colon cancer stains positively (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs5" href="#figs5" data-locator="figs5" id="back-figs5" class="figure-link scroll-into-link">Figure S5</a>B). Notably, peptides derived from IGF2BP3 significantly induce a tumor-specific cytotoxic T lymphocyte response <em>in vitro</em> (<div class="dropBlock reference-citations"><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d4132558e1318" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Suda et al., 2007</a><div data-db-target-of="bib58-d4132558e1318" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Suda T. </li><li class="loa__item">Tsunoda T. </li><li class="loa__item">Daigo Y. </li><li class="loa__item">Nakamura Y. </li><li class="loa__item">Tahara H. </li></ul><div class="reference-citations__detials">Identification of human leukocyte antigen-A24-restricted epitope peptides derived from gene products upregulated in lung and esophageal cancers as novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">1803-1808</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2007.00603.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=17784873&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-34848851598&amp;cf=" onclick="newWindow(this.href);return false">Scopus (101)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2007&pages=1803-1808&journal=Cancer+Sci.&author=Suda+T.&author=Tsunoda+T.&author=Daigo+Y.&author=Nakamura+Y.&author=Tahara+H.&title=Identification+of+human+leukocyte+antigen-A24-restricted+epitope+peptides+derived+from+gene+products+upregulated+in+lung+and+esophageal+cancers+as+novel+targets%C2%A0for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and <em>in vivo</em> in human esophageal tumors in a phase I clinical trial (<div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d4132558e1324" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Kono et al., 2009</a><div data-db-target-of="bib27-d4132558e1324" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kono K. </li><li class="loa__item">Mizukami Y. </li><li class="loa__item">Daigo Y. </li><li class="loa__item">Takano A. </li><li class="loa__item">Masuda K. </li><li class="loa__item">Yoshida K. </li><li class="loa__item">Tsunoda T. </li><li class="loa__item">Kawaguchi Y. </li><li class="loa__item">Nakamura Y. </li><li class="loa__item">Fujii H. </li></ul><div class="reference-citations__detials">Vaccination with multiple peptides derived from novel cancer-testis antigens can induce specific T-cell responses and clinical responses in advanced esophageal cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">100</span></span>: <span class="ref__seriesPages">1502-1509</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2009.01200.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19459850&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-67650648498&amp;cf=" onclick="newWindow(this.href);return false">Scopus (120)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2009&pages=1502-1509&journal=Cancer+Sci.&author=Kono+K.&author=Mizukami+Y.&author=Daigo+Y.&author=Takano+A.&author=Masuda+K.&author=Yoshida+K.&author=Tsunoda+T.&author=Kawaguchi+Y.&author=Nakamura+Y.&author=Fujii+H.&title=Vaccination+with+multiple+peptides+derived+from+novel+cancer-testis+antigens+can+induce+specific+T-cell+responses+and+clinical+responses+in+advanced+esophageal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).<div class="floatDisplay"><figure id="figs5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/a760d22a-e428-4923-b1f3-fa381f8cbea1/figs5.jpg" alt="Figure thumbnail figs5" data-locator="figs5" data-high-res="/cms/attachment/ee1d4af8-7b7a-4be5-a835-08313cb46c31/figs5_lrg.jpg" data-large-image="/cms/attachment/a760d22a-e428-4923-b1f3-fa381f8cbea1/figs5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S5" class="label figure__label">Figure S5</span><span class="figure__title__text">IGF2BP3 Protein Expression Data from Human Protein Atlas, Related to <a data-locator="fig5" aria-expanded="false" aria-controls="crosref1415" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig5">Figure 5</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) IGF2BP3 (Insulin Like Growth Factor 2 MRNA Binding Protein 3) protein expression is restricted to fetal brain and reproductive organs such as testis, ovary, and placenta. (B) Representative IHC images for IGF2BP3 in normal testis tissue (high), normal colon tissue (not detected), and colon cancer (high).</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/a760d22a-e428-4923-b1f3-fa381f8cbea1/figs5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/ee1d4af8-7b7a-4be5-a835-08313cb46c31/figs5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">In contrast to neoantigens that were enriched in MSI-H tumors and were patient-specific, CT antigens were independent of MSI status and were shared among many patients. Together, we found proteomics-supported neoantigens or CT antigens for 78% of the tumors in this cohort, demonstrating the potential of proteogenomics in identifying tumor antigens for cancer vaccine development.</div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">A Unified View of Colon Cancer Molecular Subtypes</span></h3><div class="section-paragraph">Applying the consensus molecular subtypes (CMS) classifier (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4132558e1334" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e1334" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to 106 tumors with RNA-seq data, we assigned 85 (80%) of these tumors to the four transcriptomic subtypes (CMS1–CMS4, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>A). Applying the CRC proteomic subtype classifier (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e1341" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e1341" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to 100 tumors with label-free proteomics data, we assigned 88 (88%) of these tumors to the five proteomic subtypes (ProS-A–ProS-E, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>B), and this protein expression pattern was largely preserved in the TMT proteomics data (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>C). These results provided independent validation of the previously published molecular classification systems.<div class="floatDisplay"><figure id="figs6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/8c89fdee-d443-4ab9-91bd-b2351ebacfc7/figs6.jpg" alt="Figure thumbnail figs6" data-locator="figs6" data-high-res="/cms/attachment/e40b1719-7311-49ca-b7d5-87d2a8ac31de/figs6_lrg.jpg" data-large-image="/cms/attachment/8c89fdee-d443-4ab9-91bd-b2351ebacfc7/figs6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S6" class="label figure__label">Figure S6</span><span class="figure__title__text">Colon Cancer-Subtype Analysis, Related to <a data-locator="fig6" aria-expanded="false" aria-controls="crosref1420" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig6">Figure 6</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Transcriptomic subtyping based on the consensus molecular subtypes (CMSs) and the RNA-seq data. (B) Proteomic subtyping based on previously published proteomic subtypes and the label-free proteomics data. (C) Visualization of TMT data using the same sample and gene orders as in B. (D-F) CMS3 has a vague molecular boundary. KRAS mutation is highlighted as a key characteristic of CMS3 in the original CMS paper (<div class="base dropBlock"><div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4132558e249" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e249" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div>), however, it was not enriched in the CMS3 subtype in our cohort (D). Upregulation of the metabolism-related pathways, another reported characteristic of CMS3, was recapitulated to a certain extent in our cohort at the transcriptomic level (E); however, this pattern diminished when using proteomics data to estimate pathway activities (F). (G) Comparison of chromosome instabilities across the three UMS subtypes. (H) Comparison of RB1 copy number between the CIN subtype to other two UMS subtypes.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/8c89fdee-d443-4ab9-91bd-b2351ebacfc7/figs6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e40b1719-7311-49ca-b7d5-87d2a8ac31de/figs6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">To test for underlying consistency among the mRNA, protein and MSI-based classifications, we constructed a network in which nodes represent the subtypes from individual classification systems, and weighted edges represent statistically significant association between two subtypes and the level of significance (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A). All these associations were confirmed in the TCGA CRC cohort. The connected subtypes in the association network fell naturally into three groups, as indicated by the dashed circles in <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A. The grouping aligned well with previous transcriptomic and proteomic studies that independently associated CMS1 and ProS-B with MSI-H and hypermutation, CMS2 and ProS-E with chromosome instability (CIN), and CMS4 and ProS-C with epithelial-mesenchymal transition (EMT) (<div class="dropBlock reference-citations"><a href="#" id="back-bib19" data-db-target-for="bib19-d4132558e1359" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e1359" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib77" data-db-target-for="bib77-d4132558e1359" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e1359" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Accordingly, we named the three unified multi-omics subtypes (UMS) as “MSI,” “CIN,” and “Mesenchymal,” respectively. Using a network centrality-based weighted voting algorithm, we assigned 87 out of the 110 tumor samples to the three UMS subtypes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B). The UMS classification eliminated the CMS3 subtype and assigned CMS3 tumors to other UMS subtypes. This was explained by the vague molecular boundary of the CMS3 subtype (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figures S6</a>D–S6F). In addition, three MSI-H tumors with relatively fewer MS INDELs were assigned to the Mesenchymal subtype.<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/d199510f-b80a-4368-b419-370908065f45/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/ae77149c-343f-46e4-9b74-0ee6c271ab39/gr6_lrg.jpg" data-large-image="/cms/attachment/d199510f-b80a-4368-b419-370908065f45/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">A Unified, Multi-omics View of Colon Cancer Subtypes</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) The network representing the association between subtypes defined by genomic (black), transcriptomic (white), and proteomic (gray) classification systems. Edge width denotes the significance of the connections computed by the Fisher’s exact test. The dashed circles indicate the three unified multi-omics subtypes (UMSs).</div><div class="section-paragraph">(B) UMS assignment for samples in the cohort. The genomic, transcriptomic, and proteomic subtypes are also shown for comparison.</div><div class="section-paragraph">(C) Copy-number alteration data grouped by the three UMSs.</div><div class="section-paragraph">(D) Stroma and immune infiltration profiles grouped by the three UMSs. The cytotoxic immune cell cluster is highlighted by blue in the dendrogram.</div><div class="section-paragraph">See also <a data-locator="figs6" aria-expanded="false" aria-controls="crosref1380" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs6">Figure S6</a> and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref1385" href="/cms/10.1016/j.cell.2019.03.030/attachment/abe17f04-7d22-4c30-8f9b-40d50d641f5c/mmc7" target="_blank">Table S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/d199510f-b80a-4368-b419-370908065f45/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/ae77149c-343f-46e4-9b74-0ee6c271ab39/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Next, we performed subtype-based supervised analysis on omics data not used in defining the UMS classification. Tumors in the CIN subtype showed higher chromosome instability compared to those in the other two subtypes (p &lt; 0.01, t test, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>C and <a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">S6</a>G), providing orthogonal confirmation of the UMS classification. Similarly, many of the miRNA and phosphosite markers identified in the supervised analysis (<a href="/cms/10.1016/j.cell.2019.03.030/attachment/abe17f04-7d22-4c30-8f9b-40d50d641f5c/mmc7" class="scroll-into-link">Table S7</a>) have known relationships with the subtype-specific features. Examples include decreased expression of miR-552, miR-592, and miR-181d in the MSI subtype (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d4132558e1379" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">Liu and Zhang, 2016</a><div data-db-target-of="bib37-d4132558e1379" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu Q. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Integrative Omics Analysis Reveals Post-Transcriptionally Enhanced Protective Host Response in Colorectal Cancers with Microsatellite Instability.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">766-776</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Facs.jproteome.5b00847&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26680540&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84960401765&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=766-776&journal=J.%C2%A0Proteome+Res.&author=Liu+Q.&author=Zhang+B.&title=Integrative+Omics+Analysis+Reveals+Post-Transcriptionally+Enhanced+Protective+Host+Response+in+Colorectal+Cancers+with+Microsatellite+Instability">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), decreased expression of the miR-200 family in the Mesenchymal subtype (<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d4132558e1382" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Korpal et al., 2008</a><div data-db-target-of="bib28-d4132558e1382" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Korpal M. </li><li class="loa__item">Lee E.S. </li><li class="loa__item">Hu G. </li><li class="loa__item">Kang Y. </li></ul><div class="reference-citations__detials">The miR-200 family inhibits epithelial-mesenchymal transition and cancer cell migration by direct targeting of E-cadherin transcriptional repressors ZEB1 and ZEB2.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">283</span></span>: <span class="ref__seriesPages">14910-14914</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fjbc.C800074200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18411277&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-47249091921&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1402)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=14910-14914&journal=J.%C2%A0Biol.+Chem.&author=Korpal+M.&author=Lee+E.S.&author=Hu+G.&author=Kang+Y.&title=The+miR-200+family+inhibits+epithelial-mesenchymal+transition+and+cancer+cell+migration+by+direct+targeting+of+E-cadherin+transcriptional+repressors+ZEB1+and+ZEB2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and increased phosphorylation of STAT1 and STAT3 in the MSI and Mesenchymal subtypes, respectively (<div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d4132558e1386" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Pensa et al., 2009</a><div data-db-target-of="bib49-d4132558e1386" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pensa S. </li><li class="loa__item">Regis G. </li><li class="loa__item">Boselli D. </li><li class="loa__item">Novelli F. </li><li class="loa__item">Ploi V. </li></ul><div class="reference-citations__detials">STAT1 and STAT3 in tumorigenesis: two sides of the same coin?.<div class="popup-ref"><span>JAK-STAT Pathway in Disease. </span> <span class="refPublisherName">Landes Bioscience</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2009</span>: <span class="ref__seriesPages">100-121</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=100-121&author=Pensa+S.&author=Regis+G.&author=Boselli+D.&author=Novelli+F.&author=Ploi+V.&isbn=JAK-STAT+Pathway+in+Disease&title=STAT1+and+STAT3+in+tumorigenesis%3A+two+sides+of+the+same+coin%3F">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Notably, we found higher copy-number gain of RB1 in the CIN subtype compared to the other subtypes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs6" href="#figs6" data-locator="figs6" id="back-figs6" class="figure-link scroll-into-link">Figure S6</a>H). Moreover, Rb-S811 and S807 were significantly increased in the CIN subtype compared to the other subtypes (1.61- and 1.51-fold, respectively), and the increase was much stronger than that of total Rb (1.13-fold, <a href="/cms/10.1016/j.cell.2019.03.030/attachment/abe17f04-7d22-4c30-8f9b-40d50d641f5c/mmc7" class="scroll-into-link">Table S7</a>). These results complement our interpretation of the role of Rb phosphorylation in colon cancer and further suggest that CDK2 inhibition may be the most effective in the CIN subtype.</div><div class="section-paragraph">To understand the UMS classification in the context of tumor microenvironment, we performed <em>in silico</em> deconvolution to quantify stromal infiltration level, immune infiltration level, and tumor-infiltrating lymphocyte (TIL) subpopulations based on RNA-seq data. Total stromal infiltration was significantly higher in the Mesenchymal subtype compared to the other two subtypes (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D). Although both the MSI and the Mesenchymal subtypes had higher immune infiltration than the CIN subtype, the MSI subtype was specifically enriched with cytotoxic immune cells, such as NK cells and activated CD8 T cells, whereas the Mesenchymal subtype was enriched with suppressor immune cells, such as myeloid-derived suppressor cells (MDSCs), macrophages, and Treg cells (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>F). The UMS classification thus provided a unified view of three major subtypes of colon cancers with distinct genomic, transcriptomic, proteomic, and microenvironment profiles.</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Increased Glycolysis in the MSI Subtype Is Associated with Immune Suppression</span></h3><div class="section-paragraph">Despite the enrichment of cytotoxic immune cells in the microenvironment, MSI tumors develop and progress, which has been explained by the selective upregulation of immune checkpoint proteins in the tumor microenvironment (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d4132558e1410" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Llosa et al., 2015</a><div data-db-target-of="bib39-d4132558e1410" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Llosa N.J. </li><li class="loa__item">Cruise M. </li><li class="loa__item">Tam A. </li><li class="loa__item">Wicks E.C. </li><li class="loa__item">Hechenbleikner E.M. </li><li class="loa__item">Taube J.M. </li><li class="loa__item">Blosser R.L. </li><li class="loa__item">Fan H. </li><li class="loa__item">Wang H. </li><li class="loa__item">Luber B.S. </li><li>et al.</li></ul><div class="reference-citations__detials">The vigorous immune microenvironment of microsatellite instable colon cancer is balanced by multiple counter-inhibitory checkpoints.<div class="popup-ref"><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">43-51</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1158%2F2159-8290.CD-14-0863&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25358689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84922369377&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1132)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=43-51&journal=Cancer+Discov.&author=Llosa+N.J.&author=Cruise+M.&author=Tam+A.&author=Wicks+E.C.&author=Hechenbleikner+E.M.&author=Taube+J.M.&author=Blosser+R.L.&author=Fan+H.&author=Wang+H.&author=Luber+B.S.&title=The+vigorous+immune+microenvironment+of+microsatellite+instable+colon+cancer+is+balanced+by+multiple+counter-inhibitory+checkpoints">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, only a subset of MSI-H CRCs respond to immune checkpoint inhibitors (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d4132558e1413" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Le et al., 2015</a><div data-db-target-of="bib32-d4132558e1413" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Le D.T. </li><li class="loa__item">Uram J.N. </li><li class="loa__item">Wang H. </li><li class="loa__item">Bartlett B.R. </li><li class="loa__item">Kemberling H. </li><li class="loa__item">Eyring A.D. </li><li class="loa__item">Skora A.D. </li><li class="loa__item">Luber B.S. </li><li class="loa__item">Azad N.S. </li><li class="loa__item">Laheru D. </li><li>et al.</li></ul><div class="reference-citations__detials">PD-1 Blockade in Tumors with Mismatch-Repair Deficiency.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">372</span></span>: <span class="ref__seriesPages">2509-2520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1056%2FNEJMoa1500596&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26028255&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84932628341&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7182)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=2509-2520&journal=N.%C2%A0Engl.+J.+Med.&author=Le+D.T.&author=Uram+J.N.&author=Wang+H.&author=Bartlett+B.R.&author=Kemberling+H.&author=Eyring+A.D.&author=Skora+A.D.&author=Luber+B.S.&author=Azad+N.S.&author=Laheru+D.&title=PD-1+Blockade+in+Tumors+with+Mismatch-Repair+Deficiency">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), suggesting a role for other immune evasion mechanisms.</div><div class="section-paragraph">Comparison of protein and mRNA profiles between the MSI subtype and the other UMS subtypes revealed a broad increase of the glycolytic enzymes in the MSI subtype (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>A). We also found an almost universal decrease in tricarboxylic acid (TCA) cycle enzymes at the protein level, but not at the mRNA level. Thus, our data revealed protein-level adaptations driving a strong Warburg effect in the MSI subtype.<div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/14e33df0-ca85-4f72-a261-2be6c55c6b89/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/364c10e9-edfd-45ce-91c8-414b645ae96e/gr7_lrg.jpg" data-large-image="/cms/attachment/14e33df0-ca85-4f72-a261-2be6c55c6b89/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Increased Glycolysis in the MSI Subtype and Its Association with CD8 T Cell Infiltration</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) MSI subtype-specific alteration of key enzymes involved in the glycolysis and TCA cycle. The MSI subtype-specific RNA and protein changes are shown side-by-side. p values were calculated based on the Wilcoxon rank sum test.</div><div class="section-paragraph">(B) The heatmap showing the protein expression levels of glycolytic enzymes within the MSI subtype. Samples are ordered by increased infiltration of activated CD8 T cells.</div><div class="section-paragraph">(C) The negative correlation between glycolytic activity (inferred by the protein expression of enzymes involved in the pathway) and the activated CD8 T cell level for the MSI subtype.</div><div class="section-paragraph">(D–G) Strong positive correlations were observed between SRM and TMT measurements for CD8A(D), SLC2A3 (E), and PKM2 (F), but not for PKM1 (G).</div><div class="section-paragraph">(H) SRM data showed higher CD8A abundance in MSI/CD8-H tumors (n = 5) than MSI/CD8-L tumors (n = 5).</div><div class="section-paragraph">(I–K) SRM data showed higher protein abundance of SLC2A3 (I) and PKM2 (J) in MSI tumors (n = 10) compared to CIN (n = 5) and Mesenchymal (n = 5) tumors, and in MSI/CD8-L tumors (n = 5) compared to MSI/CD8-H tumors (n = 5). This pattern was not observed for PKM1 (K).</div><div class="section-paragraph">(L) Schematic diagram summarizing the interplay between glycolysis and CD8 T cell activation in MSI tumors, highlighting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade.</div><div class="section-paragraph">The p value in (C) was calculated by Spearman’s correlation test; the p values in (D)–(G) were calculated by Pearson's correlation test; the p values in (H)–(K) were calculated by t test. See also <a data-locator="figs7" aria-expanded="false" aria-controls="crosref1390" class="figure-link caption__link scroll-into-link" href="&#xA;                            #figs7">Figure S7</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/14e33df0-ca85-4f72-a261-2be6c55c6b89/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/364c10e9-edfd-45ce-91c8-414b645ae96e/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Lactate, a key product of Warburg effect, is a potent inhibitor of CD8 T cells (<div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d4132558e1424" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Brand et al., 2016</a><div data-db-target-of="bib7-d4132558e1424" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brand A. </li><li class="loa__item">Singer K. </li><li class="loa__item">Koehl G.E. </li><li class="loa__item">Kolitzus M. </li><li class="loa__item">Schoenhammer G. </li><li class="loa__item">Thiel A. </li><li class="loa__item">Matos C. </li><li class="loa__item">Bruss C. </li><li class="loa__item">Klobuch S. </li><li class="loa__item">Peter K. </li><li>et al.</li></ul><div class="reference-citations__detials">LDHA-Associated Lactic Acid Production Blunts Tumor Immunosurveillance by T and NK Cells.<div class="popup-ref"><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">657-671</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27641098&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84994592466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1089)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=657-671&journal=Cell+Metab.&author=Brand+A.&author=Singer+K.&author=Koehl+G.E.&author=Kolitzus+M.&author=Schoenhammer+G.&author=Thiel+A.&author=Matos+C.&author=Bruss+C.&author=Klobuch+S.&author=Peter+K.&title=LDHA-Associated+Lactic+Acid+Production+Blunts+Tumor+Immunosurveillance+by+T+and+NK+Cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Indeed, almost all glycolytic enzymes were negatively correlated with CD8 infiltration in MSI tumors (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B). Using the median protein abundance of all glycolytic enzymes as a measure of glycolytic activity, we found a statistically significant negative correlation between glycolytic activity and CD8 infiltration (Spearman’s ρ = −0.61, p = 0.02, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C). Interestingly, this relationship was not observed in other colon cancer subtypes or when all colon tumors were analyzed together (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2figs7" href="#figs7" data-locator="figs7" id="back-figs7" class="figure-link scroll-into-link">Figure S7</a>). The interplay between metabolic reprogramming and immune function may apply specifically to immune evasion and checkpoint inhibition resistance in the MSI subtype.<div class="floatDisplay"><figure id="figs7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/d6ff9dd1-9d70-47f8-b8d8-dff39813b2df/figs7.jpg" alt="Figure thumbnail figs7" data-locator="figs7" data-high-res="/cms/attachment/ee560915-c634-412c-9f8d-b324b95eb659/figs7_lrg.jpg" data-large-image="/cms/attachment/d6ff9dd1-9d70-47f8-b8d8-dff39813b2df/figs7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. S7" class="label figure__label">Figure S7</span><span class="figure__title__text">The Correlation between Glycolytic Activity and the Activated CD8 T Cell Level for Different Groups of Tumor Samples, Related to <a data-locator="fig7" aria-expanded="false" aria-controls="crosref1430" class="figure-link caption__link scroll-into-link" href="&#xA;                            #fig7">Figure 7</a></span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) The whole cohort, (B) the MSI subtype, (C) the CIN subtype, and (D) the Mesenchymal subtype.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/d6ff9dd1-9d70-47f8-b8d8-dff39813b2df/figs7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/ee560915-c634-412c-9f8d-b324b95eb659/figs7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S0092-8674(19)30292-2&amp;id=figs7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">To validate these findings, we performed targeted analysis of selected proteins in representative tumor samples using selected reaction monitoring (SRM). The SRM measurements were highly correlated with TMT measurements for the CD8 T cell marker CD8A (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>D) and the glucose transporter SLC2A3 (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>E). Pyruvate kinase PKM, the rate-limiting glycolytic enzyme that catalyzes the last step of glycolysis, has two isoforms. The targeted SRM analysis also was able to measure isoform-specific peptides that were not detected in the TMT analysis, thereby clarifying that PKM2 was the major isoform measured in the TMT analysis (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>F and 7G). It is known that PKM2, rather than PKM1, drives aerobic glycolysis and lactate production in human cancer (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d4132558e1448" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Christofk et al., 2008</a><div data-db-target-of="bib11-d4132558e1448" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Christofk H.R. </li><li class="loa__item">Vander Heiden M.G. </li><li class="loa__item">Harris M.H. </li><li class="loa__item">Ramanathan A. </li><li class="loa__item">Gerszten R.E. </li><li class="loa__item">Wei R. </li><li class="loa__item">Fleming M.D. </li><li class="loa__item">Schreiber S.L. </li><li class="loa__item">Cantley L.C. </li></ul><div class="reference-citations__detials">The M2 splice isoform of pyruvate kinase is important for cancer metabolism and tumour growth.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">452</span></span>: <span class="ref__seriesPages">230-233</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature06734&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18337823&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-40749163248&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2283)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=230-233&journal=Nature&author=Christofk+H.R.&author=Vander+Heiden+M.G.&author=Harris+M.H.&author=Ramanathan+A.&author=Gerszten+R.E.&author=Wei+R.&author=Fleming+M.D.&author=Schreiber+S.L.&author=Cantley+L.C.&title=The+M2+splice+isoform+of+pyruvate+kinase+is+important+for+cancer+metabolism+and+tumour+growth">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). MSI tumors with relatively higher amounts of estimated activated CD8 cells (MSI/CD8-H) had 2.23-fold higher CD8A abundance than those with lower amounts (MSI/CD8-L, <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>H), although the difference was not statistically significant, likely due to the expression of CD8A in both activated and inactivated CD8 cells. In the subtype-based comparisons (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figures 7</a>I–7K), the abundance of SLC2A3 and PKM2 was 1.85- and 1.4-fold higher (p = 0.007 and 0.003, respectively, t test) in the MSI subtype compared to the other two subtypes. Within the MSI subtype, SLC2A3 and PKM2 were 1.19- and 1.7-fold higher in the MSI/CD8-L tumors compared to the MSI/CD8-H tumors. These data were consistent with the global proteomics data. Taken together, our data support the model depicted in <a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>L and suggest that combined therapy of checkpoint and glycolysis inhibition may provide a potent strategy to treat MSI tumors resistant to checkpoint blockade.</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0080" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">We performed an unprecedented molecular characterization of human colon cancer and paired NATs with comprehensive integration of data from multiple proteogenomic platforms. Our study confirmed the value of proteogenomic integration in uncovering novel cancer biology and further demonstrated the utility of proteogenomics in therapeutic hypothesis generation.</div><div class="section-paragraph">We combined the customized proteomics database approach with HLA binding prediction and identified personalized neoantigens for 38% of the patients. mRNA expression has been used to prioritize somatic mutations for personalized neoantigen vaccine development, but proteomics-based filtering prioritizes more effectively because neoepitopes themselves are peptides.</div><div class="section-paragraph">Colon cancer-associated proteins and phosphosites identified from our tumor versus NAT comparisons had very little overlap with known cancer genes in the Cancer Gene Census (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>H), providing a novel information layer to our understanding of colon cancer. Notably, several CT antigens were recurrently overexpressed in tumors. In addition to serving as putative tumor-specific biomarkers, the inherent immunogenicity of CT antigens, as demonstrated by IGF2BP3 in esophageal cancer (<div class="dropBlock reference-citations"><a href="#" id="back-bib27" data-db-target-for="bib27-d4132558e1473" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Kono et al., 2009</a><div data-db-target-of="bib27-d4132558e1473" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kono K. </li><li class="loa__item">Mizukami Y. </li><li class="loa__item">Daigo Y. </li><li class="loa__item">Takano A. </li><li class="loa__item">Masuda K. </li><li class="loa__item">Yoshida K. </li><li class="loa__item">Tsunoda T. </li><li class="loa__item">Kawaguchi Y. </li><li class="loa__item">Nakamura Y. </li><li class="loa__item">Fujii H. </li></ul><div class="reference-citations__detials">Vaccination with multiple peptides derived from novel cancer-testis antigens can induce specific T-cell responses and clinical responses in advanced esophageal cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">100</span></span>: <span class="ref__seriesPages">1502-1509</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2009.01200.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19459850&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-67650648498&amp;cf=" onclick="newWindow(this.href);return false">Scopus (120)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2009&pages=1502-1509&journal=Cancer+Sci.&author=Kono+K.&author=Mizukami+Y.&author=Daigo+Y.&author=Takano+A.&author=Masuda+K.&author=Yoshida+K.&author=Tsunoda+T.&author=Kawaguchi+Y.&author=Nakamura+Y.&author=Fujii+H.&title=Vaccination+with+multiple+peptides+derived+from+novel+cancer-testis+antigens+can+induce+specific+T-cell+responses+and+clinical+responses+in+advanced+esophageal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib58" data-db-target-for="bib58-d4132558e1473" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Suda et al., 2007</a><div data-db-target-of="bib58-d4132558e1473" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Suda T. </li><li class="loa__item">Tsunoda T. </li><li class="loa__item">Daigo Y. </li><li class="loa__item">Nakamura Y. </li><li class="loa__item">Tahara H. </li></ul><div class="reference-citations__detials">Identification of human leukocyte antigen-A24-restricted epitope peptides derived from gene products upregulated in lung and esophageal cancers as novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">1803-1808</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2007.00603.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=17784873&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-34848851598&amp;cf=" onclick="newWindow(this.href);return false">Scopus (101)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2007&pages=1803-1808&journal=Cancer+Sci.&author=Suda+T.&author=Tsunoda+T.&author=Daigo+Y.&author=Nakamura+Y.&author=Tahara+H.&title=Identification+of+human+leukocyte+antigen-A24-restricted+epitope+peptides+derived+from+gene+products+upregulated+in+lung+and+esophageal+cancers+as+novel+targets%C2%A0for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), makes them potentially ideal targets for immunotherapy, especially for MSS colon tumors that are poor candidates for checkpoint inhibition or neoantigen vaccine treatment because of their low neoantigen load.</div><div class="section-paragraph">Our multi-omics-based subtype analysis provided a unified view of colon cancer molecular heterogeneity based on three UMS subtypes (i.e., MSI, CIN, and Mesenchymal). Proteomics data associated decreased CD8 infiltration with increased glycolysis in MSI tumors, which supports the emerging view that increased tumor glycolysis suppresses anti-tumor immunity by impairing T cell function and trafficking to the tumor microenvironment (<div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d4132558e1478" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Tang and Fu, 2018</a><div data-db-target-of="bib61-d4132558e1478" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tang H. </li><li class="loa__item">Fu Y.X. </li></ul><div class="reference-citations__detials">Immune Evasion in Tumor’s Own Sweet Way.<div class="popup-ref"><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">945-946</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=29628418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85044618686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=945-946&journal=Cell+Metab.&author=Tang+H.&author=Fu+Y.X.&title=Immune+Evasion+in+Tumor%E2%80%99s+Own+Sweet+Way">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Therefore, glycolysis inhibition may be considered to overcome the resistance of MSI tumors to immune checkpoint blockade.</div><div class="section-paragraph">In addition to reinforcing or complementing genomics data, proteogenomic integration also may correct inaccurate genomics data-based inferences and lead to unexpected discoveries and therapeutic opportunities. One example is the proteomic identification of SOX9 as an oncogene, whereas it was predicted to be a tumor suppressor based on somatic mutation data. Another example is the phosphoproteomics data-enabled discovery of Rb phosphorylation as an oncogenic driver of colon cancer, suggesting a unique opportunity to target Rb phosphorylation in colon cancer through CDK2 inhibition.</div><div class="section-paragraph">In summary, our integrative proteogenomic characterization revealed new therapeutic opportunities for targeting signaling proteins, metabolic enzymes, and tumor antigens in colon cancer treatment. Although validation of these therapeutic hypotheses is beyond the scope of our current study, these new hypotheses may eventually enable substantial advances in molecularly guided precision therapy of colon cancer. Further interrogation of this deeply characterized colon cancer cohort by other investigators will likely yield additional insights. The primary and processed datasets are available in publicly accessible data repositories and portals (<a data-link="modal" data-target="#image-S0092-8674(19)30292-2gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>A; <a href="#sec5" class="scroll-into-link">STAR Methods</a>), and we anticipate broad usage of these datasets for new biological discoveries and therapeutic hypothesis generation.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0085" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the National Cancer Institute Clinical Proteomic Tumor Analysis Consortium are Matthew J.C. Ellis, Zhiao Shi, Bing Zhang, Jasmin Bavarva, Melissa Borucki, Kimberly Elburn, Linda Hannick, Mathangi Thiagarajan, Negin Vatanian, Samuel H. Payne, Steven A. Carr, Karl R. Clauser, Michael A. Gillette, Eric Kuhn, D.R. Mani, Shuang Cai, Karen A. Ketchum, Ratna R. Thangudu, Gordon A. Whiteley, Amanda Paulovich, Jeffrey Whiteaker, Nathan J. Edwards, Subha Madhavan, Peter B. McGarvey, Daniel W. Chan, Ie-Ming Shih, Hui Zhang, Zhen Zhang, Heng Zhu, Steven J. Skates, Forest M. White, Philip Mertins, Akhilesh Pandey, Robert J.C. Slebos, Emily S. Boja, Tara Hiltke, Christopher R. Kinsinger, Mehdi Mesri, Robert C. Rivers, Henry Rodriguez, Stephen E. Stein, David Fenyo, Kelly Ruggles, Douglas A. Levine, Mauricio Oberti, Tao Liu, Jason E. McDermott, Karin D. Rodland, Richard D. Smith, Lisa J. Zimmerman, Paul A. Rudnick, Michael Snyder, David L. Tabb, Yingming Zhao, Xian Chen, David F. Ransohoff, Andrew Hoofnagle, Daniel C. Liebler, Melinda E. Sanders, Yue Wang, Sherri R. Davies, Li Ding, R. Reid Townsend, and Mark Watson.</div></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec5.1" data-locator="sec5.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key Resources Table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Biological Samples</strong></td></tr><tr><td class="rowsep">Primary tumor samples</td><td class="rowsep">See <a href="#sec5.3" class="scroll-into-link">Experimental Model and Subject Details</a></td><td class="rowsep">N/A</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical Commercial Assays</strong></td></tr><tr><td>Illumina Barcoded Paired-End Library Preparation Kit</td><td>Illumina</td><td>Catalog: PE-410-1001</td></tr><tr><td>Illumina’s Infinium LCG Quad Assay</td><td>Illumina</td><td>Catalog: 15025908</td></tr><tr><td>TruSeq Stranded RNA Sample Preparation Kit</td><td>Illumina</td><td>Catalog: RS-122-2103</td></tr><tr><td>TruSeq Small Total RNA Sample Prep Kit</td><td>Illumina</td><td>Catalog: RS-200-0048</td></tr><tr><td>MSI Analysis System (v1.2)</td><td>Promega</td><td>Catalog: MD1641</td></tr><tr><td>BCA Protein Assay Kit</td><td>Thermo Scientific Pierce</td><td>Catalog: A53225</td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog: 93482</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>Phosphatase Inhibitor Cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase Inhibitor Cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Urea</td><td>Sigma</td><td>Catalog: U0631</td></tr><tr><td>Ammonium bicarbonate</td><td>Sigma</td><td>Catalog: 9830</td></tr><tr><td>Sodium chloride</td><td>Sigma</td><td>Catalog: S7653</td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Calcium chloride</td><td>Sigma</td><td>Catalog: C1016</td></tr><tr><td>Dithiothreitol</td><td>Thermo Scientific</td><td>Catalog: 20291</td></tr><tr><td>Iodoacetamide</td><td>Thermo Scientific</td><td>Catalog: A3221</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Catalog: V517</td></tr><tr><td>Formic acid</td><td>Sigma</td><td>Catalog: 33015</td></tr><tr><td>Reversed-phase tC18 SepPak</td><td>Waters</td><td>Catalog: WAT054925</td></tr><tr><td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</td><td>Sigma</td><td>Catalog: H3375</td></tr><tr><td>Tandem mass tags - 10plex</td><td>Thermo Scientific</td><td>Catalog: 90110</td></tr><tr><td class="rowsep">Ammonium formate</td><td class="rowsep">Sigma</td><td class="rowsep">Catalog: 9735</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited Data</strong></td></tr><tr><td>dbSNP150</td><td><div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d4132558e1717" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Sherry et al., 2001</a><div data-db-target-of="bib56-d4132558e1717" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sherry S.T. </li><li class="loa__item">Ward M.H. </li><li class="loa__item">Kholodov M. </li><li class="loa__item">Baker J. </li><li class="loa__item">Phan L. </li><li class="loa__item">Smigielski E.M. </li><li class="loa__item">Sirotkin K. </li></ul><div class="reference-citations__detials">dbSNP: the NCBI database of genetic variation.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">308-311</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2F29.1.308&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=11125122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0035173378&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5316)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=308-311&journal=Nucleic%C2%A0Acids+Res.&author=Sherry+S.T.&author=Ward+M.H.&author=Kholodov+M.&author=Baker+J.&author=Phan+L.&author=Smigielski+E.M.&author=Sirotkin+K.&title=dbSNP%3A+the+NCBI+database+of+genetic+variation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://annovar.openbioinformatics.org/en/latest/" target="_blank">http://annovar.openbioinformatics.org/en/latest/</a></td></tr><tr><td>COSMIC83</td><td><div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d4132558e1726" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Forbes et al., 2017</a><div data-db-target-of="bib17-d4132558e1726" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Forbes S.A. </li><li class="loa__item">Beare D. </li><li class="loa__item">Boutselakis H. </li><li class="loa__item">Bamford S. </li><li class="loa__item">Bindal N. </li><li class="loa__item">Tate J. </li><li class="loa__item">Cole C.G. </li><li class="loa__item">Ward S. </li><li class="loa__item">Dawson E. </li><li class="loa__item">Ponting L. </li><li>et al.</li></ul><div class="reference-citations__detials">COSMIC: somatic cancer genetics at high-resolution.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D777-D783</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkw1121&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27899578&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85016154270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1390)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D777-D783&journal=Nucleic+Acids+Res.&author=Forbes+S.A.&author=Beare+D.&author=Boutselakis+H.&author=Bamford+S.&author=Bindal+N.&author=Tate+J.&author=Cole+C.G.&author=Ward+S.&author=Dawson+E.&author=Ponting+L.&title=COSMIC%3A+somatic+cancer+genetics+at+high-resolution">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cancer.sanger.ac.uk/cosmic" target="_blank">https://cancer.sanger.ac.uk/cosmic</a></td></tr><tr><td>Clinvar (version 20170905)</td><td><div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d4132558e1735" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Landrum et al., 2014</a><div data-db-target-of="bib30-d4132558e1735" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Landrum M.J. </li><li class="loa__item">Lee J.M. </li><li class="loa__item">Riley G.R. </li><li class="loa__item">Jang W. </li><li class="loa__item">Rubinstein W.S. </li><li class="loa__item">Church D.M. </li><li class="loa__item">Maglott D.R. </li></ul><div class="reference-citations__detials">ClinVar: public archive of relationships among sequence variation and human phenotype.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">D980-D985</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkt1113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24234437&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84891809093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1887)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D980-D985&journal=Nucleic+Acids+Res.&author=Landrum+M.J.&author=Lee+J.M.&author=Riley+G.R.&author=Jang+W.&author=Rubinstein+W.S.&author=Church+D.M.&author=Maglott+D.R.&title=ClinVar%3A+public+archive+of+relationships+among%C2%A0sequence+variation+and+human+phenotype">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://annovar.openbioinformatics.org/en/latest/" target="_blank">http://annovar.openbioinformatics.org/en/latest/</a></td></tr><tr><td>dbNSFP33a</td><td><div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d4132558e1744" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Liu et al., 2011</a><div data-db-target-of="bib38-d4132558e1744" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu X. </li><li class="loa__item">Jian X. </li><li class="loa__item">Boerwinkle E. </li></ul><div class="reference-citations__detials">dbNSFP: a lightweight database of human nonsynonymous SNPs and their functional predictions.<div class="popup-ref"><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">894-899</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fhumu.21517&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21520341&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79960763462&amp;cf=" onclick="newWindow(this.href);return false">Scopus (585)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=894-899&journal=Hum.+Mutat.&author=Liu+X.&author=Jian+X.&author=Boerwinkle+E.&title=dbNSFP%3A+a+lightweight+database+of+human+nonsynonymous+SNPs+and+their+functional+predictions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://annovar.openbioinformatics.org/en/latest/" target="_blank">http://annovar.openbioinformatics.org/en/latest/</a></td></tr><tr><td>PhosphoSitePlus</td><td><div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d4132558e1754" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Hornbeck et al., 2015</a><div data-db-target-of="bib21-d4132558e1754" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2026)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.phosphosite.org/" target="_blank">https://www.phosphosite.org/</a></td></tr><tr><td>Signor</td><td><div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d4132558e1763" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Lo Surdo et al., 2017</a><div data-db-target-of="bib40-d4132558e1763" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lo Surdo P. </li><li class="loa__item">Calderone A. </li><li class="loa__item">Cesareni G. </li><li class="loa__item">Perfetto L. </li></ul><div class="reference-citations__detials">SIGNOR: A Database of Causal Relationships Between Biological Entities-A Short Guide to Searching and Browsing.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">58</span></span>: <span class="ref__seriesPages">8.23.1-8.23.16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fcpbi.28&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85029438175&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2017&pages=8.23.1-8.23.16&journal=Curr.+Protoc.+Bioinformatics&author=Lo+Surdo+P.&author=Calderone+A.&author=Cesareni+G.&author=Perfetto+L.&title=SIGNOR%3A+A+Database+of+Causal+Relationships+Between+Biological+Entities-A+Short+Guide+to+Searching+and+Browsing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://signor.uniroma2.it/" target="_blank">https://signor.uniroma2.it/</a></td></tr><tr><td>The Human Protein Atlas</td><td><div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d4132558e1772" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Uhlen et al., 2017</a><div data-db-target-of="bib62-d4132558e1772" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Uhlen M. </li><li class="loa__item">Zhang C. </li><li class="loa__item">Lee S. </li><li class="loa__item">Sjöstedt E. </li><li class="loa__item">Fagerberg L. </li><li class="loa__item">Bidkhori G. </li><li class="loa__item">Benfeitas R. </li><li class="loa__item">Arif M. </li><li class="loa__item">Liu Z. </li><li class="loa__item">Edfors F. </li><li>et al.</li></ul><div class="reference-citations__detials">A pathology atlas of the human cancer transcriptome.<div class="popup-ref"><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">357</span></span>: <span class="ref__seriesPages">eaan2507</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1126%2Fscience.aan2507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28818916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85028362951&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2170)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2017&pages=eaan2507&journal=Science&author=Uhlen+M.&author=Zhang+C.&author=Lee+S.&author=Sj%C3%B6stedt+E.&author=Fagerberg+L.&author=Bidkhori+G.&author=Benfeitas+R.&author=Arif+M.&author=Liu+Z.&author=Edfors+F.&title=A+pathology+atlas+of+the+human+cancer+transcriptome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.proteinatlas.org/" target="_blank">https://www.proteinatlas.org/</a></td></tr><tr><td class="rowsep">CTdatabase</td><td class="rowsep"><div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d4132558e1781" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Almeida et al., 2009</a><div data-db-target-of="bib2-d4132558e1781" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (308)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td class="rowsep"><a class="inter-ref-link" href="http://www.cta.lncc.br/" target="_blank">http://www.cta.lncc.br/</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Software and Algorithms</strong></td></tr><tr><td>BWA (version 0.7.15-r1140)</td><td><div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d4132558e1793" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Li and Durbin, 2010</a><div data-db-target-of="bib34-d4132558e1793" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Fast and accurate long-read alignment with Burrows-Wheeler transform.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">589-595</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtp698&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20080505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77949587649&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8157)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=589-595&journal=Bioinformatics&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+long-read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td>Bowtie (version 1.1.1)</td><td><div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d4132558e1802" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Langmead et al., 2009</a><div data-db-target-of="bib31-d4132558e1802" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Langmead B. </li><li class="loa__item">Trapnell C. </li><li class="loa__item">Pop M. </li><li class="loa__item">Salzberg S.L. </li></ul><div class="reference-citations__detials">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">R25</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fgb-2009-10-3-r25&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19261174&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-62349130698&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16269)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&journal=Genome+Biol.&author=Langmead+B.&author=Trapnell+C.&author=Pop+M.&author=Salzberg+S.L.&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bowtie-bio.sourceforge.net/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/index.shtml</a></td></tr><tr><td>GATK (version 3.8.0)</td><td><div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d4132558e1811" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Van der Auwera et al., 2013</a><div data-db-target-of="bib63-d4132558e1811" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Van der Auwera G.A. </li><li class="loa__item">Carneiro M.O. </li><li class="loa__item">Hartl C. </li><li class="loa__item">Poplin R. </li><li class="loa__item">Del Angel G. </li><li class="loa__item">Levy-Moonshine A. </li><li class="loa__item">Jordan T. </li><li class="loa__item">Shakir K. </li><li class="loa__item">Roazen D. </li><li class="loa__item">Thibault J. </li><li>et al.</li></ul><div class="reference-citations__detials">From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">43</span> (<span class="refComment">11.10.1-33</span>)</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2F0471250953.bi1110s43&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25431634&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84896009017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4200)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2013&journal=Curr.+Protoc.+Bioinformatics&author=Van+der+Auwera+G.A.&author=Carneiro+M.O.&author=Hartl+C.&author=Poplin+R.&author=Del+Angel+G.&author=Levy-Moonshine+A.&author=Jordan+T.&author=Shakir+K.&author=Roazen+D.&author=Thibault+J.&title=From+FastQ+data+to+high+confidence+variant+calls%3A+the+Genome+Analysis+Toolkit+best+practices+pipeline">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://software.broadinstitute.org/gatk/" target="_blank">https://software.broadinstitute.org/gatk/</a></td></tr><tr><td>Picard (2.9.0)</td><td>Broad Institute</td><td><a class="inter-ref-link" href="https://broadinstitute.github.io/picard/" target="_blank">https://broadinstitute.github.io/picard/</a></td></tr><tr><td>Samtools (1.3.1)</td><td><div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d4132558e1828" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Li et al., 2009</a><div data-db-target-of="bib35-d4132558e1828" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Handsaker B. </li><li class="loa__item">Wysoker A. </li><li class="loa__item">Fennell T. </li><li class="loa__item">Ruan J. </li><li class="loa__item">Homer N. </li><li class="loa__item">Marth G. </li><li class="loa__item">Abecasis G. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials"><span>1000 Genome Project Data Processing Subgroup</span><br />The Sequence Alignment/Map format and SAMtools.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37642)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinformatics&author=1000+Genome+Project+Data+Processing+Subgroup&title=The+Sequence+Alignment%2FMap+format+and+SAMtools">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.htslib.org/" target="_blank">http://www.htslib.org/</a></td></tr><tr><td>STAR (version 2.5.3a)</td><td><div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d4132558e1837" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Dobin et al., 2013</a><div data-db-target-of="bib14-d4132558e1837" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dobin A. </li><li class="loa__item">Davis C.A. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Drenkow J. </li><li class="loa__item">Zaleski C. </li><li class="loa__item">Jha S. </li><li class="loa__item">Batut P. </li><li class="loa__item">Chaisson M. </li><li class="loa__item">Gingeras T.R. </li></ul><div class="reference-citations__detials">STAR: ultrafast universal RNA-seq aligner.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">15-21</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbts635&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23104886&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871809302&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25709)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=15-21&journal=Bioinformatics&author=Dobin+A.&author=Davis+C.A.&author=Schlesinger+F.&author=Drenkow+J.&author=Zaleski+C.&author=Jha+S.&author=Batut+P.&author=Chaisson+M.&author=Gingeras+T.R.&title=STAR%3A+ultrafast+universal+RNA-seq+aligner">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td></tr><tr><td>RSEM (version 1.2.31)</td><td><div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d4132558e1847" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Li and Dewey, 2011</a><div data-db-target-of="bib33-d4132558e1847" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li B. </li><li class="loa__item">Dewey C.N. </li></ul><div class="reference-citations__detials">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13447)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://deweylab.github.io/RSEM/" target="_blank">https://deweylab.github.io/RSEM/</a></td></tr><tr><td>HTseq (version 0.7.2)</td><td><div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d4132558e1856" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Anders et al., 2015</a><div data-db-target-of="bib3-d4132558e1856" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anders S. </li><li class="loa__item">Pyl P.T. </li><li class="loa__item">Huber W. </li></ul><div class="reference-citations__detials">HTSeq--a Python framework to work with high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">166-169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtu638&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25260700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84928987900&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13104)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=166-169&journal=Bioinformatics&author=Anders+S.&author=Pyl+P.T.&author=Huber+W.&title=HTSeq%2D%2Da+Python+framework+to+work+with+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://htseq.readthedocs.io/en/release_0.10.0/history.html#version-0-7-2" target="_blank">https://htseq.readthedocs.io/en/release_0.10.0/history.html#version-0-7-2</a></td></tr><tr><td>ncPRO-seq (version 1.6.1)</td><td><div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d4132558e1865" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Chen et al., 2012</a><div data-db-target-of="bib10-d4132558e1865" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen C.J. </li><li class="loa__item">Servant N. </li><li class="loa__item">Toedling J. </li><li class="loa__item">Sarazin A. </li><li class="loa__item">Marchais A. </li><li class="loa__item">Duvernois-Berthet E. </li><li class="loa__item">Cognat V. </li><li class="loa__item">Colot V. </li><li class="loa__item">Voinnet O. </li><li class="loa__item">Heard E. </li><li>et al.</li></ul><div class="reference-citations__detials">ncPRO-seq: a tool for annotation and profiling of ncRNAs in sRNA-seq data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3147-3149</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbts587&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23044543&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84870420933&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3147-3149&journal=Bioinformatics&author=Chen+C.J.&author=Servant+N.&author=Toedling+J.&author=Sarazin+A.&author=Marchais+A.&author=Duvernois-Berthet+E.&author=Cognat+V.&author=Colot+V.&author=Voinnet+O.&author=Heard+E.&title=ncPRO-seq%3A+a+tool+for+annotation+and+profiling+of+ncRNAs+in+sRNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/ncproseq/" target="_blank">https://sourceforge.net/projects/ncproseq/</a></td></tr><tr><td>customProDB (version 1.14.1)</td><td><div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d4132558e1874" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Wang and Zhang, 2013</a><div data-db-target-of="bib66-d4132558e1874" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">3235-3237</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtt543&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24058055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84894110748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=3235-3237&journal=Bioinformatics&author=Wang+X.&author=Zhang+B.&title=customProDB%3A+an+R+package+to+generate+customized+protein+databases+from+RNA-Seq+data+for+proteomics+search">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://bioconductor.riken.jp/packages/3.4/bioc/html/customProDB.html" target="_blank">http://bioconductor.riken.jp/packages/3.4/bioc/html/customProDB.html</a></td></tr><tr><td>MutsigCV (version 1.3)</td><td><div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d4132558e1883" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Cancer Genome Atlas Research Network, 2012</a><div data-db-target-of="bib9-d4132558e1883" class="dropBlock__holder centered"><div class="dropBlock__body"><div class="reference-citations__detials"><span>Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization of squamous cell lung cancers.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3135)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/MutSigCV/1" target="_blank">http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/MutSigCV/1</a></td></tr><tr><td>Oncotator</td><td><div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d4132558e1892" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Ramos et al., 2015</a><div data-db-target-of="bib51-d4132558e1892" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ramos A.H. </li><li class="loa__item">Lichtenstein L. </li><li class="loa__item">Gupta M. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Pugh T.J. </li><li class="loa__item">Saksena G. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Oncotator: cancer variant annotation tool.<div class="popup-ref"><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">E2423-E2429</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fhumu.22771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25703262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84925965197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (375)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=E2423-E2429&journal=Hum.+Mutat.&author=Ramos+A.H.&author=Lichtenstein+L.&author=Gupta+M.&author=Lawrence+M.S.&author=Pugh+T.J.&author=Saksena+G.&author=Meyerson+M.&author=Getz+G.&title=Oncotator%3A+cancer+variant+annotation+tool">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://portals.broadinstitute.org/oncotator/" target="_blank">http://portals.broadinstitute.org/oncotator/</a></td></tr><tr><td>CMSclassifier</td><td><div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4132558e1901" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e1901" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://rdrr.io/github/Sage-Bionetworks/CMSclassifier" target="_blank">http://rdrr.io/github/Sage-Bionetworks/CMSclassifier</a></td></tr><tr><td>DTARefinery</td><td><div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d4132558e1910" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Petyuk et al., 2010</a><div data-db-target-of="bib50-d4132558e1910" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Mayampurath A.M. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Polpitiya A.D. </li><li class="loa__item">Purvine S.O. </li><li class="loa__item">Anderson G.A. </li><li class="loa__item">Camp 2nd, D.G. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">DtaRefinery, a software tool for elimination of systematic errors from parent ion mass measurements in tandem mass spectra data sets.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">486-496</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M900217-MCP200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20019053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77649174058&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=486-496&journal=Mol.+Cell.+Proteomics&author=Petyuk+V.A.&author=Mayampurath+A.M.&author=Monroe+M.E.&author=Polpitiya+A.D.&author=Purvine+S.O.&author=Anderson+G.A.&author=Camp+D.G.&author=Smith+R.D.&title=DtaRefinery%2C+a+software+tool+for+elimination+of+systematic+errors+from+parent+ion+mass+measurements+in+tandem+mass+spectra+data+sets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>Integrative Omics Group, Pacific Northwest National Laboratory</td></tr><tr><td>MASIC</td><td><div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d4132558e1918" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Monroe et al., 2008</a><div data-db-target-of="bib46-d4132558e1918" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic%C2%A0profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td>Integrative Omics Group, Pacific Northwest National Laboratory</td></tr><tr><td>IPeak</td><td><div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d4132558e1926" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Wen et al., 2015</a><div data-db-target-of="bib73-d4132558e1926" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Du C. </li><li class="loa__item">Li G. </li><li class="loa__item">Ghali F. </li><li class="loa__item">Jones A.R. </li><li class="loa__item">Käll L. </li><li class="loa__item">Xu S. </li><li class="loa__item">Zhou R. </li><li class="loa__item">Ren Z. </li><li class="loa__item">Feng Q. </li><li>et al.</li></ul><div class="reference-citations__detials">IPeak: An open source tool to combine results from multiple MS/MS search engines.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">2916-2920</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fpmic.201400208&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25951428&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84940580015&amp;cf=" onclick="newWindow(this.href);return false">Scopus (31)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=2916-2920&journal=Proteomics&author=Wen+B.&author=Du+C.&author=Li+G.&author=Ghali+F.&author=Jones+A.R.&author=K%C3%A4ll+L.&author=Xu+S.&author=Zhou+R.&author=Ren+Z.&author=Feng+Q.&title=IPeak%3A+An+open+source+tool+to+combine+results+from+multiple+MS%2FMS+search+engines">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/PGB-LIV/mzidlib" target="_blank">https://github.com/PGB-LIV/mzidlib</a></td></tr><tr><td>MS-GF+</td><td><div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d4132558e1935" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Kim and Pevzner, 2014</a><div data-db-target-of="bib24-d4132558e1935" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://omics.pnl.gov/software/ms-gf" target="_blank">https://omics.pnl.gov/software/ms-gf</a></td></tr><tr><td>X!Tandem</td><td><div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d4132558e1945" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Craig and Beavis, 2004</a><div data-db-target-of="bib13-d4132558e1945" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Craig R. </li><li class="loa__item">Beavis R.C. </li></ul><div class="reference-citations__detials">TANDEM: matching proteins with tandem mass spectra.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1466-1467</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbth092&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=14976030&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-3142702204&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2063)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2004&pages=1466-1467&journal=Bioinformatics&author=Craig+R.&author=Beavis+R.C.&title=TANDEM%3A+matching+proteins+with+tandem+mass+spectra">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.thegpm.org/tandem/" target="_blank">https://www.thegpm.org/tandem/</a></td></tr><tr><td>MyriMatch</td><td><div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d4132558e1954" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Tabb et al., 2007</a><div data-db-target-of="bib60-d4132558e1954" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tabb D.L. </li><li class="loa__item">Fernando C.G. </li><li class="loa__item">Chambers M.C. </li></ul><div class="reference-citations__detials">MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">654-661</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr0604054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=17269722&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-33847401893&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=654-661&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Fernando+C.G.&author=Chambers+M.C.&title=MyriMatch%3A+highly+accurate+tandem+mass+spectral+peptide+identification+by+multivariate+hypergeometric+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://proteowizard.sourceforge.net" target="_blank">http://proteowizard.sourceforge.net</a></td></tr><tr><td>PepQuery</td><td><div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d4132558e1963" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Wen et al., 2019</a><div data-db-target-of="bib75-d4132558e1963" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://pepquery.org/" target="_blank">http://pepquery.org/</a></td></tr><tr><td>Customprodbj</td><td>In-house</td><td><a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a></td></tr><tr><td>ProteoWizard</td><td><div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d4132558e1980" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Kessner et al., 2008</a><div data-db-target-of="bib23-d4132558e1980" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kessner D. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Burke R. </li><li class="loa__item">Agus D. </li><li class="loa__item">Mallick P. </li></ul><div class="reference-citations__detials">ProteoWizard: open source software for rapid proteomics tools development.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1395)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://proteowizard.sourceforge.net" target="_blank">http://proteowizard.sourceforge.net</a></td></tr><tr><td>ANNOVAR</td><td><div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d4132558e1989" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Wang et al., 2010</a><div data-db-target-of="bib67-d4132558e1989" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang K. </li><li class="loa__item">Li M. </li><li class="loa__item">Hakonarson H. </li></ul><div class="reference-citations__detials">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">e164</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkq603&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20601685&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77956534324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9421)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e164&journal=Nucleic+Acids+Res.&author=Wang+K.&author=Li+M.&author=Hakonarson+H.&title=ANNOVAR%3A+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://annovar.openbioinformatics.org/en/latest/" target="_blank">http://annovar.openbioinformatics.org/en/latest/</a></td></tr><tr><td>OptiType (version 1.3.1)</td><td><div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d4132558e1998" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Szolek et al., 2014</a><div data-db-target-of="bib59-d4132558e1998" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (449)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/FRED-2/OptiType" target="_blank">https://github.com/FRED-2/OptiType</a></td></tr><tr><td>netMHCpan (version 4.0)</td><td><div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d4132558e2007" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Jurtz et al., 2017</a><div data-db-target-of="bib22-d4132558e2007" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (869)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="http://www.cbs.dtu.dk/services/NetMHCpan/" target="_blank">http://www.cbs.dtu.dk/services/NetMHCpan/</a></td></tr><tr><td>metaX</td><td><div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d4132558e2016" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Wen et al., 2017</a><div data-db-target-of="bib74-d4132558e2016" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Mei Z. </li><li class="loa__item">Zeng C. </li><li class="loa__item">Liu S. </li></ul><div class="reference-citations__detials">metaX: a flexible and comprehensive software for processing metabolomics data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">183</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs12859-017-1579-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28327092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85015943888&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=183&journal=BMC+Bioinformatics&author=Wen+B.&author=Mei+Z.&author=Zeng+C.&author=Liu+S.&title=metaX%3A+a+flexible+and+comprehensive+software+for+processing+metabolomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://github.com/wenbostar/metaX" target="_blank">https://github.com/wenbostar/metaX</a></td></tr><tr><td>GISTIC2 (version 2.0.23)</td><td><div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d4132558e2025" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Mermel et al., 2011</a><div data-db-target-of="bib44-d4132558e2025" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz" target="_blank">ftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz</a></td></tr><tr><td>MSMuTect (version 1.0)</td><td><div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d4132558e2034" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Maruvka et al., 2017</a><div data-db-target-of="bib42-d4132558e2034" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maruvka Y.E. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Karlic R. </li><li class="loa__item">Parasuraman P. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Polak P. </li><li class="loa__item">Haradhvala N.J. </li><li class="loa__item">Hess J.M. </li><li class="loa__item">Rheinbay E. </li><li class="loa__item">Brody Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Analysis of somatic microsatellite indels identifies driver events in human tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">951-959</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt.3966&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28892075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031106017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=951-959&journal=Nat.+Biotechnol.&author=Maruvka+Y.E.&author=Mouw+K.W.&author=Karlic+R.&author=Parasuraman+P.&author=Kamburov+A.&author=Polak+P.&author=Haradhvala+N.J.&author=Hess+J.M.&author=Rheinbay+E.&author=Brody+Y.&title=Analysis+of+somatic+microsatellite+indels+identifies+driver+events+in+human+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://www.nature.com/articles/nbt.3966#supplementary-information" target="_blank">https://www.nature.com/articles/nbt.3966#supplementary-information</a></td></tr><tr><td>WebGestaltR</td><td><div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d4132558e2044" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Wang et al., 2017c</a><div data-db-target-of="bib71-d4132558e2044" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Greer M. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (787)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt%C2%A02017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/WebGestaltR/index.html" target="_blank">https://cran.r-project.org/web/packages/WebGestaltR/index.html</a></td></tr><tr><td>GSVA</td><td><div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d4132558e2053" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Hänzelmann et al., 2013</a><div data-db-target-of="bib20-d4132558e2053" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6889)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/release/bioc/html/GSVA.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/GSVA.html</a></td></tr><tr><td>Agilent Bioanalyzer (version 2.0)</td><td>Agilent Technologies</td><td><a class="inter-ref-link" href="https://www.agilent.com/en/product/bioanalyzer-automated-electrophoresis/bioanalyzer-instrument/2100-bioanalyzer-instrument-228250" target="_blank">https://www.agilent.com/en/product/bioanalyzer-automated-electrophoresis/bioanalyzer-instrument/2100-bioanalyzer-instrument-228250</a></td></tr><tr><td>Skyline</td><td><div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d4132558e2070" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">MacLean et al., 2010</a><div data-db-target-of="bib41-d4132558e2070" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">MacLean B. </li><li class="loa__item">Tomazela D.M. </li><li class="loa__item">Shulman N. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Finney G.L. </li><li class="loa__item">Frewen B. </li><li class="loa__item">Kern R. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Liebler D.C. </li><li class="loa__item">MacCoss M.J. </li></ul><div class="reference-citations__detials">Skyline: an open source document editor for creating and analyzing targeted proteomics experiments.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">966-968</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtq054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20147306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77951965920&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=966-968&journal=Bioinformatics&author=MacLean+B.&author=Tomazela+D.M.&author=Shulman+N.&author=Chambers+M.&author=Finney+G.L.&author=Frewen+B.&author=Kern+R.&author=Tabb+D.L.&author=Liebler+D.C.&author=MacCoss+M.J.&title=Skyline%3A+an+open+source+document+editor+for+creating+and+analyzing+targeted+proteomics+experiments">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></td><td><a class="inter-ref-link" href="https://skyline.ms/project/home/software/Skyline/begin.view" target="_blank">https://skyline.ms/project/home/software/Skyline/begin.view</a></td></tr><tr><td>Genome Studio (version 2.0)</td><td>Illumina</td><td><a class="inter-ref-link" href="https://www.illumina.com/techniques/microarrays/array-data-analysis-experimental-design/genomestudio.html" target="_blank">https://www.illumina.com/techniques/microarrays/array-data-analysis-experimental-design/genomestudio.html</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2819%2930292-2">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec5.2" data-locator="sec5.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Contact for Reagent and Resource Sharing</span></h3><div class="section-paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Bing Zhang (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#3c5e55525b1246545d525b7c5e5f5112595849" target="_blank"><span class="__cf_email__" data-cfemail="e4868d8a83ca9e8c858a83a4868789ca818091">[email&#160;protected]</span></a>).</div></section><section id="sec5.3" data-locator="sec5.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental Model and Subject Details</span></h3><section id="sec5.3.1" data-locator="sec5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimens and Clinical Data</span></h4><div class="section-paragraph">Tumor, adjacent normal, and blood samples were collected by several tissue source sites in strict accordance to the CPTAC-2 colon procurement protocol (<a class="inter-ref-link" href="https://brd.nci.nih.gov/brd/sop/download-pdf/321" target="_blank">https://brd.nci.nih.gov/brd/sop/download-pdf/321</a>) with an informed consent from the patients. The Washington University in St. Louis Institutional Review Board (IRB) reviewed the individual informed consent documents at each tissue source site and determined that the materials sent to the CPTAC biospecimen core resource met the requirements for Exemption 4. The cohort had an inclusion criterion of newly diagnosed, untreated patients undergoing primary surgery for colon adenocarcinoma. Because untreated rectal tumors are difficult to obtain, we only included colon cancers, which represent approximately 70% of all CRCs. Patients with prior history of other malignancies within 12 months, any systemic chemotherapy, endocrine or biological therapy as well as prior radiation therapy to the abdomen or pelvis for any cancer type were excluded from the study. Required clinical information regarding patient history and status of surgery along with relevant diagnostic information were collected using case reports forms. One year follow up information with updated history after completion of the initial treatment regimen were also collected through follow up forms. Deidentified pathology reports and representative diagnostic slide images were utilized to review and qualify cases for this study. The peripheral venous blood from the same patient were collected prior to administration of anesthesia. Segments from qualified tumor specimens were greater than 300mg in mass with at least 60% tumor cell nuclei and less than 20% necrosis. To ensure tissue suitability for phosphoprotein analysis, the tissue and the adjacent normal specimens were collected in less than 30 minutes total ischemic time and embedded in optimal cutting temperature (OCT) compound for processing at a common CPTAC-2 specimen core resource center. Pathologically qualified cases underwent further molecular qualification for extraction and co-isolation of nucleic acids. Tissue segments that were pathology and molecular qualified were shipped to the proteomic characterization centers. DNA and RNA from the same tumor segment and DNA from germline blood were further aliquoted and quantified per protocol. DNA quality was confirmed using gel electrophoresis and Nano drop methods. RNA quality was confirmed using Nano drop and Agilent bioanalyzer. Sufficient yield, a good gel score and passing value of 7 or greater RIN qualified the DNA and RNA, respectively, for sequencing. The analytes were then shipped to the sequencing center. The corresponding clinical data were formatted and distributed through the CPTAC data coordinating center (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S037" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S037</a>). <a href="/cms/10.1016/j.cell.2019.03.030/attachment/87317e5c-393a-47d6-9c62-cb4fee1d653b/mmc1" class="scroll-into-link">Table S1</a> summarizes the clinical and pathological characteristics of the tumors and the specific numbers of samples analyzed by each omics platform. Among the 110 patients in the cohort, there were 65 females (60%) and 45 males (40%), with an average age of 65 (range 40 to 93 years). We did not perform analyses on the two sexes separately because the sample size is too small after sex stratification. Moreover, we were interested in results common to both sexes, and the sex distribution is reasonably balanced.</div></section></section><section id="sec5.4" data-locator="sec5.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Methods Details</span></h3><section id="sec5.4.1" data-locator="sec5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PCR-based MSI Analysis</span></h4><div class="section-paragraph">The MSI Analysis System (version 1.2, Promega), a fluorescent PCR-based assay, was used to detect microsatellite instability (MSI) in the colon tumors. The analysis compares allelic profiles of microsatellite markers generated by amplification of DNA from matching tumor and normal samples, and alleles that are present in the tumor sample but not in corresponding normal samples indicate MSI. The system uses seven markers including five mononucleotide repeat markers (BAT-25, BAT-26, NR-21, NR-24, and MONO-27) and two pentanucleotide repeat markers (Penta C and Penta D). The output data were analyzed with GeneMapper® software (Applied Biosystems) to determine MSI status of the colon tumor samples.</div></section><section id="sec5.4.2" data-locator="sec5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genotyping Array Analysis</span></h4><div class="section-paragraph">Genomic DNA samples were prepared according to Illumina’s Infinium LCG Quad Assay manual protocol. Processed samples were loaded on the HumanOmni5-Quad BeadChips and run on the HiScan platform. SNP and CNP genotyping were performed with the Genome Studio Genotyping Module (Version 2.0, Illumina).</div></section><section id="sec5.4.3" data-locator="sec5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole Exome Sequencing</span></h4><div class="section-paragraph">Genomic DNA samples were used to prepare indexed libraries using the Nextera Rapid Capture Exome kit from Illumina. Library preparation was performed using a semi-automated 96-well plate method, with washing and clean-up/concentration steps performed on the Beckman Coulter Biomek NXP platform and with ZR-96 DNA Clean &amp; Concentrator-5 plates, respectively. Libraries were quantified using the Agilent 2100 Bioanalyzer. Pooled libraries were run on HiSeq4000 (2x150 paired end runs) to achieve a minimum of 150x on target coverage per each sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files, adaptor and low-quality sequences were trimmed. Whole exome sequencing (WXS) data were used for somatic mutation detection, microsatellite instability prediction, and somatic copy number alteration (SCNA) analysis as described below.</div></section><section id="sec5.4.4" data-locator="sec5.4.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic Mutation Detection</span></h4><div class="section-paragraph">We followed the Genome Analysis Toolkit (GATK, version 3.8.0) best practice guideline for somatic short variant discovery (<a class="inter-ref-link" href="https://software.broadinstitute.org/gatk/best-practices/workflow?id=11146" target="_blank">https://software.broadinstitute.org/gatk/best-practices/workflow?id=11146</a>). Briefly, we aligned paired-end WXS reads to the human reference genome (hg19) with BWA-mem (version 0.7.15-r1140). The bam files were further processed by adding read groups, marking duplicates, and re-ordering with Picard tools (version 2.9.0). Consequently, base quality score recalibration and INDEL realignment were performed using GATK modules IndelRealigner and BaseRecalibrator. The cross-individual contamination was then estimated by the GATK module ContEst. Single nucleotide variants (SNVs) and INDELs (insertions/deletions) were called from tumor and matched-normal pairs using MuTect2 from GATK. We filtered out variants from short tandem repeat regions, which were downloaded from the UCSC table browser. The sequence variants were then annotated using customProDB and Oncotator. Significance of candidate mutations was evaluated in non-hypermutated and hypermutated tumors separately using MutSigCV (version 1) in GenePattern, and genes with a false discovery rate (q value) below 0.05 were considered significantly mutated above the background mutation rate.</div></section><section id="sec5.4.5" data-locator="sec5.4.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline Short Variant Discovery from WXS</span></h4><div class="section-paragraph">We followed the GATK best practice guideline for germline short variant discovery from WXS data (<a class="inter-ref-link" href="https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145" target="_blank">https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145</a>). We started from the processed bam files generated in the previous section. HaplotypeCaller was used to generate an intermediate file, GVCF, for each sample. Next, we consolidated all GVCFs from 106 samples into to one GVCF file using the GATK module CombineGV<em>CF.</em> The merged GVCF file was passed to GenotypeGVCFs, a joint genotyping tool for SNP and INDEL calling. We further filtered the variants by variant quality score recalibration, in which machine learning was used to identify annotation profiles of variants likely to be real.</div></section><section id="sec5.4.6" data-locator="sec5.4.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Microsatellite Instability Prediction</span></h4><div class="section-paragraph">MSMuTect (version 1.0) (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d4132558e2149" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Maruvka et al., 2017</a><div data-db-target-of="bib42-d4132558e2149" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maruvka Y.E. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Karlic R. </li><li class="loa__item">Parasuraman P. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Polak P. </li><li class="loa__item">Haradhvala N.J. </li><li class="loa__item">Hess J.M. </li><li class="loa__item">Rheinbay E. </li><li class="loa__item">Brody Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Analysis of somatic microsatellite indels identifies driver events in human tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">951-959</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt.3966&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28892075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031106017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=951-959&journal=Nat.+Biotechnol.&author=Maruvka+Y.E.&author=Mouw+K.W.&author=Karlic+R.&author=Parasuraman+P.&author=Kamburov+A.&author=Polak+P.&author=Haradhvala+N.J.&author=Hess+J.M.&author=Rheinbay+E.&author=Brody+Y.&title=Analysis+of+somatic+microsatellite+indels+identifies+driver+events+in+human+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was applied to the processed bam files for somatic microsatellite INDEL (MS INDEL) calling. MSMuTect uses the Kolmogorov–Smirnov (KS) test to identify microsatellite sites with different alleles between tumor and normal samples. Using tumors with PCR test results, we found that the Fisher’s exact test provided higher sensitivity and specificity compared to the KS test. Therefore, we applied Fisher’s exact test to the histograms generated by MSMuTect and then used the Fisher’s exact test <em>p-value</em> as the filtering criterion, with a <em>p-value</em> cutoff of 0.001. We named this modified method MSMuTect-fisher. Significant MS INDELs with multiple alleles in normal and only one allele in tumor were removed from further analysis. Using 40 MS INDELs as a cutoff, we separated the samples into an MSI-high (MSI-H) group and a microsatellite stable (MSS) group. MSMutSigCV (<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d4132558e2156" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Maruvka et al., 2017</a><div data-db-target-of="bib42-d4132558e2156" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Maruvka Y.E. </li><li class="loa__item">Mouw K.W. </li><li class="loa__item">Karlic R. </li><li class="loa__item">Parasuraman P. </li><li class="loa__item">Kamburov A. </li><li class="loa__item">Polak P. </li><li class="loa__item">Haradhvala N.J. </li><li class="loa__item">Hess J.M. </li><li class="loa__item">Rheinbay E. </li><li class="loa__item">Brody Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Analysis of somatic microsatellite indels identifies driver events in human tumors.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">951-959</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt.3966&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28892075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031106017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=951-959&journal=Nat.+Biotechnol.&author=Maruvka+Y.E.&author=Mouw+K.W.&author=Karlic+R.&author=Parasuraman+P.&author=Kamburov+A.&author=Polak+P.&author=Haradhvala+N.J.&author=Hess+J.M.&author=Rheinbay+E.&author=Brody+Y.&title=Analysis+of+somatic+microsatellite+indels+identifies+driver+events+in+human+tumors">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to detect significantly mutated genes based on identified MS INDELs (p &lt; 0.05).</div></section><section id="sec5.4.7" data-locator="sec5.4.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Annotation of Protein Altering Somatic Mutations</span></h4><div class="section-paragraph">All somatic mutations identified by MuTect2 and MSMuTect, including SNVs, INDELs, and MS INDELs, were annotated using ANNOVAR (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d4132558e2164" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Wang et al., 2010</a><div data-db-target-of="bib67-d4132558e2164" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang K. </li><li class="loa__item">Li M. </li><li class="loa__item">Hakonarson H. </li></ul><div class="reference-citations__detials">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">e164</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkq603&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20601685&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77956534324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9421)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e164&journal=Nucleic+Acids+Res.&author=Wang+K.&author=Li+M.&author=Hakonarson+H.&title=ANNOVAR%3A+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The variants obtained by ANNOVAR were filtered for protein altering events including non-synonymous SNVs, frameshift INDELs, non-frameshift INDELs, and stop gains. <a href="/cms/10.1016/j.cell.2019.03.030/attachment/422700f5-078e-4f85-b043-ff1f75a164fd/mmc3" class="scroll-into-link">Table S3</a> includes all isoforms altered by the somatic mutation events. For the analysis of proteomic consequence, the longest isoform was selected for each somatic mutation event.</div></section><section id="sec5.4.8" data-locator="sec5.4.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic Copy Number Alteration Analysis</span></h4><div class="section-paragraph">Somatic copy number alteration (SCNA) analysis used WXS-derived BAM files that were processed in the somatic mutation detection pipeline. These BAM files were further processed by the R Package CopywriteR (version 1.18.0) (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d4132558e2175" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Kuilman et al., 2015</a><div data-db-target-of="bib29-d4132558e2175" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuilman T. </li><li class="loa__item">Velds A. </li><li class="loa__item">Kemper K. </li><li class="loa__item">Ranzani M. </li><li class="loa__item">Bombardelli L. </li><li class="loa__item">Hoogstraat M. </li><li class="loa__item">Nevedomskaya E. </li><li class="loa__item">Xu G. </li><li class="loa__item">de Ruiter J. </li><li class="loa__item">Lolkema M.P. </li><li>et al.</li></ul><div class="reference-citations__detials">CopywriteR: DNA copy number detection from off-target sequence data.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs13059-015-0617-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25887352&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84937691121&amp;cf=" onclick="newWindow(this.href);return false">Scopus (158)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=49&journal=Genome+Biol.&author=Kuilman+T.&author=Velds+A.&author=Kemper+K.&author=Ranzani+M.&author=Bombardelli+L.&author=Hoogstraat+M.&author=Nevedomskaya+E.&author=Xu+G.&author=de+Ruiter+J.&author=Lolkema+M.P.&title=CopywriteR%3A+DNA+copy+number+detection+from+off-target+sequence+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which uses off-target WXS reads to infer copy number values. 105 tumor and matched-normal pairs had sufficient (&gt; 5 million) off-target reads for SCNA detection as recommended by the software (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d4132558e2178" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Kuilman et al., 2015</a><div data-db-target-of="bib29-d4132558e2178" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kuilman T. </li><li class="loa__item">Velds A. </li><li class="loa__item">Kemper K. </li><li class="loa__item">Ranzani M. </li><li class="loa__item">Bombardelli L. </li><li class="loa__item">Hoogstraat M. </li><li class="loa__item">Nevedomskaya E. </li><li class="loa__item">Xu G. </li><li class="loa__item">de Ruiter J. </li><li class="loa__item">Lolkema M.P. </li><li>et al.</li></ul><div class="reference-citations__detials">CopywriteR: DNA copy number detection from off-target sequence data.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">49</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs13059-015-0617-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25887352&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84937691121&amp;cf=" onclick="newWindow(this.href);return false">Scopus (158)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=49&journal=Genome+Biol.&author=Kuilman+T.&author=Velds+A.&author=Kemper+K.&author=Ranzani+M.&author=Bombardelli+L.&author=Hoogstraat+M.&author=Nevedomskaya+E.&author=Xu+G.&author=de+Ruiter+J.&author=Lolkema+M.P.&title=CopywriteR%3A+DNA+copy+number+detection+from+off-target+sequence+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The circular binary segmentation (CBS) algorithm (<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d4132558e2181" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Olshen et al., 2004</a><div data-db-target-of="bib48-d4132558e2181" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Olshen A.B. </li><li class="loa__item">Venkatraman E.S. </li><li class="loa__item">Lucito R. </li><li class="loa__item">Wigler M. </li></ul><div class="reference-citations__detials">Circular binary segmentation for the analysis of array-based DNA copy number data.<div class="popup-ref"><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">557-572</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbiostatistics%2Fkxh008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=15475419&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-3543105225&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1696)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=557-572&journal=Biostatistics&author=Olshen+A.B.&author=Venkatraman+E.S.&author=Lucito+R.&author=Wigler+M.&title=Circular+binary%C2%A0segmentation+for+the+analysis+of+array-based+DNA+copy+number+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) which is also implemented in the CopywriteR package was used for the copy number segmentation, with the default parameters. From the segmentation result, we used a weighted-sum approach to summarize the chromosome instability for each sample. Specifically, the absolute log2 ratios of all segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was derived by summing up the instability score of all 22 autosomes.</div><div class="section-paragraph">Next, we used GISTIC2 (version 2.0.23) (<div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d4132558e2186" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Mermel et al., 2011</a><div data-db-target-of="bib44-d4132558e2186" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to retrieve gene-level copy number values and call significant copy number alterations in the cohort. We set up a threshold of 0.4 (<em>-ta</em> and <em>-td</em> parameters of GISTIC2) in picking the amplified or deleted regions based on the distribution of germline copy number variants. Moreover, genome regions containing significant numbers of germline copy number variants were excluded from the GISTIC2 population level statistics (the <em>-cnv</em> parameter of GISTIC2). GISTIC2 generated arm level and focal level SCNAs for the cohort with G-Score and FDR Q value indicating the significance and strength of the identified SCNAs.</div><div class="section-paragraph">In order to prioritize SCNA drivers, we selected all the genes located in the focal region with GISTIC2 Q value less than 0.25. Also, we included genes located in the “wide peaks” identified by GISTIC2. For these genes, spearman correlation was calculated between copy number values and their RNA or protein levels across the cohort. We used FDR less than 0.05 and absolute correlation coefficient larger than 0.3 as the cutoff to select candidate genes. As an additional filtering criterion, we further required candidate amplification and deletion drivers to show significant upregulation and downregulation, respectively, in tumor versus matched adjacent normal comparisons (FDR &lt; 0.05, paired t test).</div></section><section id="sec5.4.9" data-locator="sec5.4.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA Sequencing</span></h4><div class="section-paragraph">Indexed cDNA sequencing libraries were prepared from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags. Library preparation was performed similarly to the WXS. Quality control was performed at every step, and the libraries were quantified using the Agilent 2100 Bioanalyzer. Indexed libraries were prepared as equimolar pools and run on HiSeq4000 (2x150 paired end runs) to generate a minimum of 30 million paired-end reads per sample library. The raw Illumina sequence data were demultiplexed and converted to. fastq files, and adaptor and low-quality sequences were trimmed.</div></section><section id="sec5.4.10" data-locator="sec5.4.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA Quantification</span></h4><div class="section-paragraph">mRNA sequencing reads were mapped to the human genome hg19 by STAR (version 2.5.3a) using the 1-pass model. Hg19 sequence and RefSeq annotation were downloaded from the UCSC table browser (03/29/2017). RSEM (version 1.2.31) was used to quantify genes and transcripts expression levels. Gene read counts were calculated using HTseq (version 0.7.2). The RSEM outputs the mRNA RSEM and FPKM (Fragments Per Kilobase of transcript per Million mapped reads) results in table format. The mRNA RSEM data were filtered for genes with median FPKM &gt; 1 for use in downstream analyses.</div></section><section id="sec5.4.11" data-locator="sec5.4.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Short Variant Discovery from RNA-Seq</span></h4><div class="section-paragraph">We followed the GATK Best Practice Variant Detection protocol on RNASeq (<a class="inter-ref-link" href="https://gatkforums.broadinstitute.org/dsde/discussion/3892/the-gatk-best-practices-for-variant-calling-on-rnaseq-in-full-detail" target="_blank">https://gatkforums.broadinstitute.org/dsde/discussion/3892/the-gatk-best-practices-for-variant-calling-on-rnaseq-in-full-detail</a>). We used the STAR 2-pass method to align RNA-Seq reads to the human reference genome (hg19). Specifically, splice junctions detected in the 1-pass alignment run were used to guide the 2-pass alignment. After alignment, the SAM file was processed through the usual Picard processing steps including adding read group, sorting, marking duplicates, and indexing. Next, we applied the GATK pipeline including the modules ‘SplitNCigarReads’, ‘HaplotypeCaller’ and ‘VariantFiltration’. The minimum phred-scaled confidence threshold for calling variants was set to 20. The ‘VariantFiltration’ module excluded SNVs with: a quality by depth score (QD) &lt; 2.0, a Fisher strand score (FS) &gt; 30.0, or clusters of at least 3 SNPs that were within a window of 35 bases between them. These filters ensured: (1) high confidence variant calls based on unfiltered depth of non-reference samples (QD); (2) low strand bias for detection of variants (FS)—as strand bias is indicative of false positive calls; and (3) filtering of many false variant calls introduced by RNA-Seq read mapping errors.</div></section><section id="sec5.4.12" data-locator="sec5.4.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA Sequencing</span></h4><div class="section-paragraph">Indexed small RNA sequencing libraries were prepared from the RNA samples using the TruSeq Small Total RNA Sample Prep Kit, and bar-coded with individual tags. Library preparation was performed similarly to the WXS sequencing. Quality control was performed at every step, and the libraries were quantified using the Agilent 2100 Bioanalyzer. Indexed libraries were prepared as equimolar pools and loaded on the NextSeq500 (1x75 single read) run to generate at least 5,000,000 single reads per sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files.</div></section><section id="sec5.4.13" data-locator="sec5.4.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-Seq Data Analysis</span></h4><div class="section-paragraph">For fastq files from miRNA sequencing, adapters were trimmed using Cutadpt (version 1.13) with a maximum allowed error rate of 0.1. Trimmed reads shorter than 17 or longer than 26 nucleotides in length were excluded from further analysis. Reads were then mapped to the human genome hg19 using Bowtie (version 1.1.1). Mapped reads were then annotated using ncPRO-seq (version 1.6.1) based on the ncPRO-seq hg19 annotation. Mature miRNA annotation was extended 2 bp in both upstream and downstream regions to accommodate inaccurate processing of precursor miRNAs. ncPRO-seq outputs miRNA count and RPM (reads per million mappable reads) results in table format.</div></section><section id="sec5.4.14" data-locator="sec5.4.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Label-free Proteomics Analysis</span></h4><div class="section-paragraph">Label-free shotgun proteomic analyses of the colon tumor samples were done according to the methods described previously (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2231" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2231" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with two changes. First, 6 concatenated basic reverse phase LC fractions were prepared from tryptic digests using the same instrumentation described previously. Second, LC-MS/MS analyses were done with a ThermoFisher QExactive MS instrument. LC-MS/MS shotgun proteomics of the concatenated fraction samples were carried out on a Q Exactive mass spectrometer (ThermoFisher Scientific) equipped with an Easy nLC-1000 (ThermoFisher Scientific) and a Nanoflex source (ThermoFisher Scientific). A 2 μL injection volume of peptides were separated on a PicoFrit (New Objective, Woburn, MA) column (75 μm ID x 110 mm, 10 μm ID tip) packed with ReproSil-Pur C18-AQ resin (3 μm particle size and 120 Å pore size). Peptides were eluted at a flow rate of 300 nL/min, and the mobile phase solvents consisted of water containing 0.1% formic acid (solvent A) and acetonitrile containing 0.1% formic acid (solvent B).</div><div class="section-paragraph">A 100-minute gradient was performed, consisting of the following: 0−5 min, increase to 5% B; 5-90 min, 5−35% B; 90-93 min, 90% B; 93-100 min, 90% B and held at 90% B for 7 min before returning to the initial conditions of 2% B. Mass spectra were acquired over the scan range of <em>m/z</em> 300-1800 at a resolution of 70,000 (AGC target 3 x10<sup>6</sup> and 64 ms max injection time). Data-dependent scans of the top 20 most abundant ions were selected for fragmentation with HCD using an isolation width of 2 <em>m/z</em>, 27% normalized collision energy and a resolution of 17,500 (AGC target 2 x10<sup>5</sup> and 100 ms max injection time). Dynamic exclusion was set to 60 s.</div></section><section id="sec5.4.15" data-locator="sec5.4.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Label-free Proteomics Data Analysis</span></h4><div class="section-paragraph">The raw MS data were converted to MGF and mzML files using ProteoWizard (version 3.0.10462). The MS/MS data were searched by three search engines (MyriMatch version 2.2.10165, X!Tandem version Alanine 2017.02.01, and MS-GF+ version 2017.01.13) through IPeak (<div class="dropBlock reference-citations"><a href="#" id="back-bib72" data-db-target-for="bib72-d4132558e2251" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">Wen et al., 2014</a><div data-db-target-of="bib72-d4132558e2251" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Li G. </li><li class="loa__item">Wright J.C. </li><li class="loa__item">Du C. </li><li class="loa__item">Feng Q. </li><li class="loa__item">Xu X. </li><li class="loa__item">Choudhary J.S. </li><li class="loa__item">Wang J. </li></ul><div class="reference-citations__detials">The OMSSAPercolator: an automated tool to validate OMSSA results.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">1011-1014</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fpmic.201300393&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24504981&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84898621562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1011-1014&journal=Proteomics&author=Wen+B.&author=Li+G.&author=Wright+J.C.&author=Du+C.&author=Feng+Q.&author=Xu+X.&author=Choudhary+J.S.&author=Wang+J.&title=The+OMSSAPercolator%3A+an+automated+tool+to+validate+OMSSA+results">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib73" data-db-target-for="bib73-d4132558e2251" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Wen et al., 2015</a><div data-db-target-of="bib73-d4132558e2251" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Du C. </li><li class="loa__item">Li G. </li><li class="loa__item">Ghali F. </li><li class="loa__item">Jones A.R. </li><li class="loa__item">Käll L. </li><li class="loa__item">Xu S. </li><li class="loa__item">Zhou R. </li><li class="loa__item">Ren Z. </li><li class="loa__item">Feng Q. </li><li>et al.</li></ul><div class="reference-citations__detials">IPeak: An open source tool to combine results from multiple MS/MS search engines.<div class="popup-ref"><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">2916-2920</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fpmic.201400208&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25951428&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84940580015&amp;cf=" onclick="newWindow(this.href);return false">Scopus (31)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=2916-2920&journal=Proteomics&author=Wen+B.&author=Du+C.&author=Li+G.&author=Ghali+F.&author=Jones+A.R.&author=K%C3%A4ll+L.&author=Xu+S.&author=Zhou+R.&author=Ren+Z.&author=Feng+Q.&title=IPeak%3A+An+open+source+tool+to+combine+results+from+multiple+MS%2FMS+search+engines">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) against the RefSeq protein database (03/29/2017, 45929 sequences + 245 contaminant sequences) with decoy sequences. The following parameters were set for database searching: Carbamidomethyl (C) was specified as a fixed modification. Oxidation (M) and Deamidated (NQ) were specified as variable modifications. The precursor mass tolerance for protein identification on MS was 10 ppm, and the product ion tolerance for MS/MS was 0.05 Da. Full cleavage by trypsin was used, with up to two missed cleavages permitted. The results from the three search engines were then integrated by IPeak, which is a tool that combines multiple search engine results. To optimize the number of proteins identified we applied a very stringent filter at 0.1% PSM FDR. Then the protein inference was performed and 1% FDR was controlled using the “picked” protein FDR approach (<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d4132558e2254" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Savitski et al., 2015</a><div data-db-target-of="bib53-d4132558e2254" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Savitski M.M. </li><li class="loa__item">Wilhelm M. </li><li class="loa__item">Hahne H. </li><li class="loa__item">Kuster B. </li><li class="loa__item">Bantscheff M. </li></ul><div class="reference-citations__detials">A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (290)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell.+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+Scalable+Approach+for+Protein+False+Discovery+Rate+Estimation+in+Large+Proteomic%C2%A0Data+Sets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To rescue high quality PSMs that were excluded by the stringent PSM FDR threshold, we relaxed the PSM FDR threshold to 1% for the confidently identified proteins.</div><div class="section-paragraph">Spectral count data were filtered by removing proteins with zero counts in all samples and quantile-normalized using the R package preprocessCore (version 1.42.0, <a class="inter-ref-link" href="https://github.com/bmbolstad/preprocessCore" target="_blank">https://github.com/bmbolstad/preprocessCore</a>). We further filtered low abundant proteins with average raw count &lt; 1.4 as we did previously (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2262" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2262" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The normalized and filtered counts were then log2 transformed for downstream analysis.</div></section><section id="sec5.4.16" data-locator="sec5.4.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein Extraction and Tryptic Digestion for TMT Analysis</span></h4><div class="section-paragraph">For TMT analysis, the tumor and normal colon tissue samples were obtained as OCT-embedded tissue curls through the CPTAC Biospecimen Core Resource. Approximately 100 mg of each of the samples were first subjected to OCT removal procedure by sequential rinsing in 70% ethanol, nanopure water, and 100% ethanol. The tissue samples were then homogenized separately in 600 μL of lysis buffer (8 M urea, 100 mM NH<sub>4</sub>HCO<sub>3</sub>, pH 7.8, 75 mM NaCl, 1 mM EDTA, 10 mM NaF, Sigma phosphatase inhibitor cocktail 2, Sigma phosphatase inhibitor cocktail 3, and 20 μM PUGNAc). Lysates were precleared by centrifugation at 16,500 g for 5 min at 4°C and protein concentrations were determined by BCA assay (Pierce). Proteins were reduced with 5 mM dithiothreitol for 1 h at 37°C, and subsequently alkylated with 10 mM iodoacetamide for 45 min at 25°C in the dark. Samples were diluted 1:2 with 100 mM NH<sub>4</sub>HCO<sub>3</sub>, 1 mM CaCl<sub>2</sub> and digested with sequencing grade modified trypsin (Promega) at 1:50 enzyme-to-substrate ratio. After 3 h of digestion at 37°C, samples were diluted 1:4 with the same buffers and another aliquot of the same amount of trypsin was added to the samples and further incubated at 25°C overnight (∼16 h). The digested samples were then acidified with 100% formic acid to 1% formic acid in the final sample solution. Tryptic peptides were desalted on reversed phase C18 SPE columns (Waters tC18 SepPak, 200mg) and dried using Speed-Vac.</div></section><section id="sec5.4.17" data-locator="sec5.4.17" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT-10 Labeling of Peptides</span></h4><div class="section-paragraph">Desalted peptides from each sample were labeled with 10-plex Tandem Mass Tag (TMT) reagents according to the manufacturer’s instructions (ThermoScientific). Peptides (300 μg) from each of the samples were dissolved in 300 μL of 50 mM HEPES, pH 8.5, and mixed with 3 units of TMT reagent that was dissolved freshly in 123 μL of anhydrous acetonitrile. Channel 131 was used for labeling the internal reference sample (pooled from all tumor and normal samples with equal contribution) throughout the sample analysis. After 1 h incubation at RT, 24 μL of 5% hydroxylamine was added and incubated for 15 min at RT to quench the reaction. Peptides labeled by different TMT reagents were then mixed, dried using Speed-Vac, reconstituted with 3% acetonitrile, 0.1% formic acid and were desalted on C18 SPE columns (Waters tC18 SepPak, 200mg).</div></section><section id="sec5.4.18" data-locator="sec5.4.18" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide Fractionation by Basic Reversed-phase Liquid Chromatography</span></h4><div class="section-paragraph">Approximately 2.5 mg of 10-plex TMT labeled sample was separated on a Waters reversed phase XBridge C18 column (250 mm × 4.6 mm column containing 5-μm particles, and a 4.6 mm × 20 mm guard column) using an Agilent 1200 HPLC System. After the sample loading, the C18 column was washed for 35 min with solvent A (5 mM ammonium formate, pH 10.0), before applying a 100-min LC gradient with solvent B (5 mM ammonium formate, pH 10, 90% acetonitrile). The LC gradient started with a linear increase of solvent A to 10% B in 6 min, then linearly increased to 30% B in 86 min, 10 min to 43% B, 5 min to 55% B and another 8 min back to 100% B. The flow rate was 0.5 mL/min. A total of 96 fractions were collected into a 96 well plate throughout the LC gradient. These fractions were concatenated into 12 fractions by combining 8 fractions that are 12 fractions apart (i.e., combining fractions #1, #13, #25, #37, #49, #61, #73, and #85; #2, #14, #26, #38, #50, #62, #74, and #86; and so on). For proteome analysis, 5% of each concatenated fraction was dried down and re-suspended in 2% acetonitrile, 0.1% formic acid to a peptide concentration of 0.1 μg/μL for LC-MS/MS analysis. The rest of the concatenated fractions (95%) were further concatenated into 6 fractions by combining two concatenated fractions (i.e., combining concatenated fractions #1 and #7; #2 and #8; and so on), dried down, and subjected to immobilized metal affinity chromatography (IMAC) for phosphopeptide enrichment.</div></section><section id="sec5.4.19" data-locator="sec5.4.19" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide Enrichment Using IMAC</span></h4><div class="section-paragraph">Fe<sup>3+</sup>-NTA-agarose beads were freshly prepared using the Ni-NTA Superflow agarose beads (QIAGEN) for phosphopeptide enrichment. For each of the 6 fractions, peptides were reconstituted to 0.5 μg/μL in IMAC binding/wash buffer (80% acetonitrile, 0.1% formic acid) and incubated with 20 μL of the 50% Fe<sup>3+</sup>-conditioned NiNTA bead suspension for 30 min at RT. After incubation, the beads were washed 2 times each with 100 μL of wash buffer on the stage tip packed with 2 discs of Empore C18 material. Phosphopeptides were eluted from the beads on C18 using 60 μL of Elution Buffer (500 mM K<sub>2</sub>HPO<sub>4</sub>, pH 7.0). 50% acetonitrile, 0.1% formic acid was used for elution of phosphopeptides from the C18 stage tips. Samples were dried using Speed-Vac, and later reconstituted with 10 μL of 3% acetonitrile, 0.1% formic acid for LC-MS/MS analysis.</div></section><section id="sec5.4.20" data-locator="sec5.4.20" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for TMT Global Proteome Analysis</span></h4><div class="section-paragraph">The global proteome fractions were separated using a nanoAquity UPLC system (Waters Corporation) by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase (Dr. Maisch GmbH) and slurry packed into a 30-cm length of 360 μm o.d. x 75 μm i.d. fused silica containing a 3-mm sol-gel frit. The trapping column was manufactured in-house using Jupiter 300 C18 5-μm stationary phase (Phenomenex) and slurry packed into a 4-cm length of 360 μm o.d. x 150 μm i.d. fused silica with the final column being sol-gel fritted on both ends. The analytical column was heated to 50°C using an AgileSLEEVE column heater (Analytical Sales and Services, Inc.). The analytical column was equilibrated to 95% Mobile Phase A (MP A, 0.1% formic acid in water) and 5% Mobile Phase B (MP B, 0.1% formic acid in acetonitrile) and maintained at a constant column flow of 200 nL/min. The sample injected (5-μL) was trapped using 100% MP A for 10 min at flow rate of 3 μL/min before being placed in-line with the analytical column and subjected to the gradient profile (min: %MP B): 0:5, 1:8, 44:15, 85:30, 94:55, 102:70, 105:95, 108:95, 115:5, 150:5.</div><div class="section-paragraph">MS analysis was performed using a Q-Exactive Plus mass spectrometer (Thermo Scientific, San Jose, CA). Electrospray voltage (2.2 kV) was applied at a carbon composite union (Valco Instruments Co. Inc.) between the analytical column and electrospray emitter (chemically etched 360-μm o.d. x 20-μm i.d.). The ion transfer tube was set at 250°C. Following a 15-min delay from the end of sample trapping, Orbitrap precursor spectra (AGC 1x10<sup>6</sup>) were collected from 300-1800 m/z for 120 minutes at a resolution of 70K along with the top 12 data dependent Orbitrap HCD MS/MS spectra at a resolution of 35K (AGC 1x10<sup>5</sup>) and max ion time of 100 msec. Masses selected for MS/MS were isolated at a width of 0.7 m/z and fragmented using a normalized collision energy of 32%. Peptide match was set to ‘Preferred’, exclude isotopes was set to ‘on’, and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 20 s to discriminate against previously analyzed ions.</div></section><section id="sec5.4.21" data-locator="sec5.4.21" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS for TMT Phosphoproteome Analysis</span></h4><div class="section-paragraph">The phosphoproteome fractions were separated using a nanoAquity UPLC system (Waters Corporation) by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase (Dr. Maisch GmbH) and slurry packed into a 35-cm length of 360 μm o.d. x 50 μm i.d. fused silica picofrit capillary tubing (New Objective, Inc.). The trapping column was manufactured in-house using Jupiter 300 C18 5-μm stationary phase (Phenomenex) and slurry packed into a 4-cm length of 360 μm o.d. x 150 μm i.d. fused silica with the final column being sol-gel fritted on both ends. The analytical column was heated to 50°C using an AgileSLEEVE column heater. The analytical column was equilibrated to 98% MP A and 2% MP B and maintained at a constant column flow of 120 nL/min. The sample injected (5 μL) was trapped using 100% MP A for 5 min at a flow rate of 3 μL/min before being placed in-line with the analytical column and subjected to the gradient profile (min:%MP B): 0:2, 8:4, 50:15, 85:35, 94:60, 95:95, 105:95, 115:2, 170:2.</div><div class="section-paragraph">MS analysis was performed using an Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). Electrospray voltage (1.8 kV) was applied at a Valco carbon composite union coupling a 360 μm o.d. x 20 μm i.d. fused silica extension from the LC gradient pump to the analytical column and the ion transfer tube was set at 250°C. Following a 40-min delay from the end of sample trapping, Orbitrap precursor spectra (AGC 4x10<sup>5</sup>) were collected from 350-1800 m/z for 120 min at a resolution of 60K along with data dependent Orbitrap HCD MS/MS spectra (centroided) at a resolution of 50K (AGC 1x10<sup>5</sup>) and max ion time of 105 msec for a total duty cycle of 2 s. Masses selected for MS/MS were isolated (quadrupole) at a width of 0.7 m/z and fragmented using a collision energy of 30%. Peptide mode was selected for monoisotopic precursor scan and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 45 s to discriminate against previously analyzed ions between ± 10 ppm.</div></section><section id="sec5.4.22" data-locator="sec5.4.22" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of TMT Global Proteomics Data</span></h4><div class="section-paragraph">LC-MS/MS analysis of the TMT10-labeled, bRPLC fractionated samples generated a total of 264 global proteomics data files. The Thermo RAW files were processed with DTARefinery (<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d4132558e2344" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Petyuk et al., 2010</a><div data-db-target-of="bib50-d4132558e2344" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Mayampurath A.M. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Polpitiya A.D. </li><li class="loa__item">Purvine S.O. </li><li class="loa__item">Anderson G.A. </li><li class="loa__item">Camp 2nd, D.G. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">DtaRefinery, a software tool for elimination of systematic errors from parent ion mass measurements in tandem mass spectra data sets.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">486-496</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M900217-MCP200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20019053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77649174058&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=486-496&journal=Mol.+Cell.+Proteomics&author=Petyuk+V.A.&author=Mayampurath+A.M.&author=Monroe+M.E.&author=Polpitiya+A.D.&author=Purvine+S.O.&author=Anderson+G.A.&author=Camp+D.G.&author=Smith+R.D.&title=DtaRefinery%2C+a+software+tool+for+elimination+of+systematic+errors+from+parent+ion+mass+measurements+in+tandem+mass+spectra+data+sets">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (v1.2) to characterize and correct for any instrument calibration errors, and then with MS-GF+ (<div class="dropBlock reference-citations"><a href="#" id="back-bib25" data-db-target-for="bib25-d4132558e2347" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Kim et al., 2008</a><div data-db-target-of="bib25-d4132558e2347" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Gupta N. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib24" data-db-target-for="bib24-d4132558e2347" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Kim and Pevzner, 2014</a><div data-db-target-of="bib24-d4132558e2347" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (v9881) to match against the RefSeq human protein database (03/29/2017, 45929 sequences), combined with 261 contaminants (e.g., trypsin, keratin). The partially tryptic search used a ± 10 ppm parent ion tolerance, allowed for isotopic error in precursor ion selection, and searched a decoy database composed of the forward and reversed protein sequences. MS-GF+ considered static carbamidomethylation (+57.0215 Da) on Cys residues and TMT modification (+229.1629 Da) on the peptide N terminus and Lys residues, and dynamic oxidation (+15.9949 Da) on Met residues for searching the global proteome data.</div><div class="section-paragraph">Peptide identification stringency was set at a maximum false discovery rate (FDR) of 1% at peptide level using PepQValue &lt; 0.005 and parent ion mass deviation &lt; 7 ppm criteria. A minimum of 6 unique peptides per 1000 amino acids of protein length was required for achieving 1% at the protein level within the full dataset. Inference of parsimonious protein set resulted in the identification of a total of 8,067 common protein groups among the 197 samples.</div><div class="section-paragraph">The intensities of all ten TMT reporter ions were extracted using MASIC software (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d4132558e2354" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Monroe et al., 2008</a><div data-db-target-of="bib46-d4132558e2354" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic%C2%A0profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Next, PSMs passing the confidence thresholds described above were linked to the extracted reporter ion intensities by scan number. The reporter ion intensities from different scans and different bRPLC fractions corresponding to the same gene were grouped. Relative protein abundance was calculated as the ratio of sample abundance to reference abundance using the summed reporter ion intensities from peptides that could be uniquely mapped to a gene. The pooled reference sample was labeled with TMT 131 reagent, allowing comparison of relative protein abundances across different TMT-10 plexes. The relative abundances were log2 transformed and zero-centered for each gene to obtain final, relative abundance values.</div><div class="section-paragraph">Small differences in laboratory conditions and sample handling can result in systematic, sample-specific bias in the quantification of protein levels. In order to mitigate these effects, we computed the median, log2 relative protein abundance for each sample and re-centered to achieve a common median of 0.</div></section><section id="sec5.4.23" data-locator="sec5.4.23" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of Phosphopeptides</span></h4><div class="section-paragraph">Phosphopeptide identification for the 132 phosphoproteomics data files were performed as described above (e.g., peptide level FDR &lt; 1%), with an additional dynamic phosphorylation (+79.9663 Da) on Ser, Thr or Tyr residues. The phosphoproteome data were further processed by the Ascore algorithm (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d4132558e2364" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Beausoleil et al., 2006</a><div data-db-target-of="bib6-d4132558e2364" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beausoleil S.A. </li><li class="loa__item">Villén J. </li><li class="loa__item">Gerber S.A. </li><li class="loa__item">Rush J. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1297)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for phosphorylation site localization, and the top-scoring sequences were reported. For phosphoproteomics data, the TMT-10 quantitative data were not summarized by protein, but left at phosphopeptide level. All the peptides (phosphopeptides and global peptides) were labeled with TMT-10 reagent simultaneously. Separation into phospho- and non-phosphopeptides using IMAC was performed after the labeling. Thus, all the biases upstream of labeling are assumed to be identical between global and phosphoproteomics datasets. Therefore, to account for sample-specific biases in the phosphoproteome analysis, we applied the correction factors derived from mean-centering the global proteomics data.</div></section><section id="sec5.4.24" data-locator="sec5.4.24" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Gene-wise Correlation Between Different Platforms</span></h4><div class="section-paragraph">We calculated gene-wise correlations for each pair of the three platforms, including RNA-Seq, label-free proteomics, and TMT proteomics. For each pair of the platforms, the analysis included the top 10% most variably expressed genes in each platform and quantifiable in both platforms. Spearman’s correlation between the two platforms was calculated for each gene across all the samples.</div></section><section id="sec5.4.25" data-locator="sec5.4.25" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">mRNA and Protein Correlation Across Datasets</span></h4><div class="section-paragraph">We calculated average gene-wise RNA expression and performed the Pearson’s correlation coefficient analysis between the prospective colon tumor samples (N = 106) and TCGA CRC samples (N = 90) (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2377" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2377" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), TCGA breast tumor samples (N = 1102, downloaded from TCGAbiolinks FPKM-UQ (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d4132558e2380" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Colaprico et al., 2016</a><div data-db-target-of="bib12-d4132558e2380" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2047)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic%C2%A0Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), TCGA ovarian tumor samples (N = 374, downloaded from TCGAbiolinks FPKM-UQ), and colorectal cell lines (N = 44) (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d4132558e2383" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Wang et al., 2017b</a><div data-db-target-of="bib70-d4132558e2383" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Mouradov D. </li><li class="loa__item">Wang X. </li><li class="loa__item">Jorissen R.N. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Love C.G. </li><li class="loa__item">Li S. </li><li class="loa__item">Lowes K. </li><li>et al.</li></ul><div class="reference-citations__detials">Colorectal Cancer Cell Line Proteomes Are Representative of Primary Tumors and Predict Drug Sensitivity.<div class="popup-ref"><span class="ref__series"><i>Gastroenterology.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">153</span></span>: <span class="ref__seriesPages">1082-1095</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=abstract&amp;site=ygast-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=fulltext&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=pdf&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28625833&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031699623&amp;cf=" onclick="newWindow(this.href);return false">Scopus (50)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2017&pages=1082-1095&journal=Gastroenterology&author=Wang+J.&author=Mouradov+D.&author=Wang+X.&author=Jorissen+R.N.&author=Chambers+M.C.&author=Zimmerman+L.J.&author=Vasaikar+S.&author=Love+C.G.&author=Li+S.&author=Lowes+K.&title=Colorectal+Cancer+Cell+Line+Proteomes+Are+Representative+of+Primary+Tumors+and+Predict+Drug+Sensitivity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), respectively. Similarly, we calculated average gene-wise protein expression (label-free) and performed the Pearson’s correlation coefficient analysis between the prospective colon tumor samples (N = 100) and TCGA colorectal samples (N = 95) (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2386" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2386" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), colorectal cell lines (N = 44) (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d4132558e2389" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Wang et al., 2017b</a><div data-db-target-of="bib70-d4132558e2389" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Mouradov D. </li><li class="loa__item">Wang X. </li><li class="loa__item">Jorissen R.N. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Love C.G. </li><li class="loa__item">Li S. </li><li class="loa__item">Lowes K. </li><li>et al.</li></ul><div class="reference-citations__detials">Colorectal Cancer Cell Line Proteomes Are Representative of Primary Tumors and Predict Drug Sensitivity.<div class="popup-ref"><span class="ref__series"><i>Gastroenterology.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">153</span></span>: <span class="ref__seriesPages">1082-1095</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=abstract&amp;site=ygast-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=fulltext&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=pdf&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28625833&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031699623&amp;cf=" onclick="newWindow(this.href);return false">Scopus (50)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2017&pages=1082-1095&journal=Gastroenterology&author=Wang+J.&author=Mouradov+D.&author=Wang+X.&author=Jorissen+R.N.&author=Chambers+M.C.&author=Zimmerman+L.J.&author=Vasaikar+S.&author=Love+C.G.&author=Li+S.&author=Lowes+K.&title=Colorectal+Cancer+Cell+Line+Proteomes+Are+Representative+of+Primary+Tumors+and+Predict+Drug+Sensitivity">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and NCI-60 colorectal cell lines (N = 60, quantile-normalized) (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4132558e2393" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Gholami et al., 2013</a><div data-db-target-of="bib18-d4132558e2393" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gholami A.M. </li><li class="loa__item">Hahne H. </li><li class="loa__item">Wu Z. </li><li class="loa__item">Auer F.J. </li><li class="loa__item">Meng C. </li><li class="loa__item">Wilhelm M. </li><li class="loa__item">Kuster B. </li></ul><div class="reference-citations__detials">Global proteome analysis of the NCI-60 cell line panel.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">609-620</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23933261&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84881613658&amp;cf=" onclick="newWindow(this.href);return false">Scopus (237)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=609-620&journal=Cell+Rep.&author=Gholami+A.M.&author=Hahne+H.&author=Wu+Z.&author=Auer+F.J.&author=Meng+C.&author=Wilhelm+M.&author=Kuster+B.&title=Global+proteome+analysis+of+the+NCI-60+cell+line+panel">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), respectively.</div></section><section id="sec5.4.26" data-locator="sec5.4.26" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Co-expression-based Gene Function Prediction</span></h4><div class="section-paragraph">To compare the ability of different gene expression profiling datasets to predict gene function, we constructed <em>k</em>-nearest neighbor co-expression networks as previously described (<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d4132558e2403" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Wang et al., 2017a</a><div data-db-target-of="bib69-d4132558e2403" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Ma Z. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Townsend R.R. </li><li class="loa__item">Smith R.D. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteome Profiling Outperforms Transcriptome Profiling for Coexpression Based Gene Function Prediction.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">121-134</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27836980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85008490068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (96)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=121-134&journal=Mol.+Cell.+Proteomics&author=Wang+J.&author=Ma+Z.&author=Carr+S.A.&author=Mertins+P.&author=Zhang+H.&author=Zhang+Z.&author=Chan+D.W.&author=Ellis+M.J.&author=Townsend+R.R.&author=Smith+R.D.&title=Proteome+Profiling+Outperforms%C2%A0Transcriptome+Profiling+for+Coexpression+Based+Gene+Function+Prediction">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using RNA-Seq, label-free proteomics, and TMT proteomics data, respectively. Network-based gene function prediction was performed using the random walk-based network propagation algorithm (<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d4132558e2406" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Wang et al., 2017a</a><div data-db-target-of="bib69-d4132558e2406" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Ma Z. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Zhang H. </li><li class="loa__item">Zhang Z. </li><li class="loa__item">Chan D.W. </li><li class="loa__item">Ellis M.J. </li><li class="loa__item">Townsend R.R. </li><li class="loa__item">Smith R.D. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteome Profiling Outperforms Transcriptome Profiling for Coexpression Based Gene Function Prediction.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">121-134</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27836980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85008490068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (96)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=121-134&journal=Mol.+Cell.+Proteomics&author=Wang+J.&author=Ma+Z.&author=Carr+S.A.&author=Mertins+P.&author=Zhang+H.&author=Zhang+Z.&author=Chan+D.W.&author=Ellis+M.J.&author=Townsend+R.R.&author=Smith+R.D.&title=Proteome+Profiling+Outperforms%C2%A0Transcriptome+Profiling+for+Coexpression+Based+Gene+Function+Prediction">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Prediction performance was evaluated using 5-fold cross validation for each KEGG pathway and quantified based on the area under the receiver operating characteristic curve (AUROC).</div></section><section id="sec5.4.27" data-locator="sec5.4.27" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor versus Normal Differential Proteomic Analysis</span></h4><div class="section-paragraph">TMT-based global proteomics data were used to perform differential proteome analysis between tumor and matched normal samples. Gene-level data were further filtered for non-missing values in at least 50% of samples. A paired Wilcoxon signed-rank test was performed on overlapping samples to determine differential abundance of proteins between tumor and normal. Proteins with fold change &gt; 2 and Benjamini-Hochberg adjusted <em>p-value</em> &lt; 0.01 were considered to be cancer-associated proteins. GO enrichment analysis was performed using WebGestalt (<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d4132558e2416" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Wang et al., 2017c</a><div data-db-target-of="bib71-d4132558e2416" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang J. </li><li class="loa__item">Vasaikar S. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Greer M. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (787)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt%C2%A02017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For each cancer-associated protein, we checked immunohistochemistry images in colorectal tumors from the Human Protein Atlas (HPA, <a class="inter-ref-link" href="https://www.proteinatlas.org/" target="_blank">https://www.proteinatlas.org/</a>), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The weighted average score (IHC staining) was calculated for each protein by assigning weight to high, medium, low, and not detected respectively.</div></section><section id="sec5.4.28" data-locator="sec5.4.28" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor versus Normal Differential Phosphoproteomic Analysis</span></h4><div class="section-paragraph">Identified phosphopeptides were mapped to UniProt sequences (version July 2017), and named according to the canonical UniProt sequence. If the peptide matched multiple canonical UniProt sequences, the best ID was chosen based on presence of the protein in the proteomics data. If no canonical IDs had proteomics data, or if more than one protein was present in the quantified proteomics data, an ID was chosen at random. For peptides not matching a canonical protein sequence, a matching protein isoform ID was chosen. Peptides were filtered to those with an Ascore ≥ 19 in at least one scan and a Q value &lt; 0.01. Phosphorylation site levels were determined by the median level for all peptides matching that site. Quantified sites and proteins were defined as those containing non-missing values in at least 50% of the matched samples. Log fold change was calculated as the log2 peptide ratios for normal samples subtracted from the log2 peptide ratios for tumor samples. Log fold change was correlated with log2 fold change of protein abundance using Pearson correlation. Differential abundance was performed using the paired Wilcoxon signed-rank test. Phosphorylation sites with fold change &gt; 2 and Benjamini-Hochberg adjusted <em>p-value</em> &lt; 0.01 were considered to be cancer-associated phosphosites.</div></section><section id="sec5.4.29" data-locator="sec5.4.29" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kinase Activity Prediction</span></h4><div class="section-paragraph">We predicted cancer-associated kinase activity based on two methods. First, sites annotated as activating kinase activity in Signor (<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d4132558e2434" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Lo Surdo et al., 2017</a><div data-db-target-of="bib40-d4132558e2434" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lo Surdo P. </li><li class="loa__item">Calderone A. </li><li class="loa__item">Cesareni G. </li><li class="loa__item">Perfetto L. </li></ul><div class="reference-citations__detials">SIGNOR: A Database of Causal Relationships Between Biological Entities-A Short Guide to Searching and Browsing.<div class="popup-ref"><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">58</span></span>: <span class="ref__seriesPages">8.23.1-8.23.16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fcpbi.28&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85029438175&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2017&pages=8.23.1-8.23.16&journal=Curr.+Protoc.+Bioinformatics&author=Lo+Surdo+P.&author=Calderone+A.&author=Cesareni+G.&author=Perfetto+L.&title=SIGNOR%3A+A+Database+of+Causal+Relationships+Between+Biological+Entities-A+Short+Guide+to+Searching+and+Browsing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) were used to predict kinase activity. Increased kinase activity was defined as significantly increased phosphorylation on these sites (Benjamini-Hochberg adjusted <em>p-value</em> &lt; 0.05, Wilcoxon signed-rank test) in tumor compared to matched adjacent normal. Sites with a protein abundance change greater than the phosphorylation abundance change were excluded. Second, we performed phosphosite set enrichment analysis based on known kinase-target site relationships. Unique phosphorylation sites were identified as a 13-mer sequence (±6 amino acids surrounding the phosphorylation site). Phosphorylation sites of kinases were determined by a union of kinase-substrate interactions in PhosphoSitePlus (<div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d4132558e2439" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Hornbeck et al., 2015</a><div data-db-target-of="bib21-d4132558e2439" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hornbeck P.V. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Murray B. </li><li class="loa__item">Kornhauser J.M. </li><li class="loa__item">Latham V. </li><li class="loa__item">Skrzypek E. </li></ul><div class="reference-citations__detials">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2026)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and Signor. The median log2 fold change of sites with at least 50% non-missing values was used to rank the phosphorylation sites and was submitted to WebGestaltR for GSEA analysis. A minimum set size of 3 substrates and 1000 permutations were required.</div></section><section id="sec5.4.30" data-locator="sec5.4.30" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Rb Phosphorylation Quantification and Correlation Analysis</span></h4><div class="section-paragraph">Our phosphoproteomics data quantified six Rb phosphorylation sites with non-missing values in at least 50% of all samples, including four sites (i.e., T373, S807, S811, and T826) that have been reported to directly or indirectly regulate E2F binding (<div class="dropBlock reference-citations"><a href="#" id="back-bib26" data-db-target-for="bib26-d4132558e2447" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Knudsen and Wang, 1997</a><div data-db-target-of="bib26-d4132558e2447" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Knudsen E.S. </li><li class="loa__item">Wang J.Y. </li></ul><div class="reference-citations__detials">Dual mechanisms for the inhibition of E2F binding to RB by cyclin-dependent kinase-mediated RB phosphorylation.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">1997; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">5771-5783</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1128%2FMCB.17.10.5771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=9315635&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0030849455&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=5771-5783&journal=Mol.+Cell.+Biol.&author=Knudsen+E.S.&author=Wang+J.Y.&title=Dual+mechanisms+for+the+inhibition+of+E2F+binding+to+RB+by+cyclin-dependent+kinase-mediated+RB+phosphorylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>, <div class="dropBlock reference-citations"><a href="#" id="back-bib52" data-db-target-for="bib52-d4132558e2447" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">Rubin, 2013</a><div data-db-target-of="bib52-d4132558e2447" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rubin S.M. </li></ul><div class="reference-citations__detials">Deciphering the retinoblastoma protein phosphorylation code.<div class="popup-ref"><span class="ref__series"><i>Trends Biochem. Sci.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">12-19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23218751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871513029&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=12-19&journal=Trends+Biochem.+Sci.&author=Rubin+S.M.&title=Deciphering+the+retinoblastoma+protein+phosphorylation+code">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We used the average of the four sites to quantify phospho-Rb abundance. Correlation between the average Rb phosphorylation levels and other features was performed using Pearson’s correlation. Inferred activity was performed using single sample GSEA implemented in the R package GSVA with a minimum gene set size of 10 (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d4132558e2450" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Hänzelmann et al., 2013</a><div data-db-target-of="bib20-d4132558e2450" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6889)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphorylation sites of kinases were determined by a union of kinase-substrate interactions in PhosphoSitePlus and Signor. The transcriptional targets of E2F transcription factors were collected from <div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d4132558e2453" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">ENCODE Project Consortium, 2011</a><div data-db-target-of="bib16-d4132558e2453" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>ENCODE Project Consortium</span><br />A user’s guide to the encyclopedia of DNA elements (ENCODE).<div class="popup-ref"><span class="ref__series"><i>PLoS Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">e1001046</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1371%2Fjournal.pbio.1001046&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21526222&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79955550445&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1168)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=e1001046&journal=PLoS+Biol.&author=ENCODE+Project+Consortium&title=A+user%E2%80%99s+guide+to+the+encyclopedia+of+DNA+elements+%28ENCODE%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>. Hallmark pathways were collected from MSigDB (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d4132558e2456" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Liberzon et al., 2015</a><div data-db-target-of="bib36-d4132558e2456" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5972)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The tumor-normal protein levels were used to infer activity for E2F1 and the apoptosis hallmark pathway. Although transcriptomic data for E2F1 targets would more directly reflect transcription factor activity, RNA-Seq data were not available for the normal samples. The tumor-normal phosphorylation levels were used to infer CDK2 activity.</div></section><section id="sec5.4.31" data-locator="sec5.4.31" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cancer-testis Antigen Analysis</span></h4><div class="section-paragraph">Cancer-testis (CT) antigens were downloaded from the CTdatabase (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d4132558e2464" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Almeida et al., 2009</a><div data-db-target-of="bib2-d4132558e2464" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Almeida L.G. </li><li class="loa__item">Sakabe N.J. </li><li class="loa__item">deOliveira A.R. </li><li class="loa__item">Silva M.C. </li><li class="loa__item">Mundstein A.S. </li><li class="loa__item">Cohen T. </li><li class="loa__item">Chen Y.T. </li><li class="loa__item">Chua R. </li><li class="loa__item">Gurung S. </li><li class="loa__item">Gnjatic S. </li><li>et al.</li></ul><div class="reference-citations__detials">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (308)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which consists of 269 CT antigens with carefully curated and annotated literature-derived information. The CT antigens that overlapped with the proteomics dataset were used for further analysis.</div></section><section id="sec5.4.32" data-locator="sec5.4.32" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Variant Peptide Identification</span></h4><div class="section-paragraph">To identify variant peptides, we used a customized protein sequence database approach (<div class="dropBlock reference-citations"><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d4132558e2472" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Wang et al., 2012</a><div data-db-target-of="bib68-d4132558e2472" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang X. </li><li class="loa__item">Slebos R.J. </li><li class="loa__item">Wang D. </li><li class="loa__item">Halvey P.J. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Liebler D.C. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">Protein identification using customized protein sequence databases derived from RNA-Seq data.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (141)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases%C2%A0derived+from+RNA-Seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We derived customized protein sequence databases from both matched RNA-Seq and WXS data and then performed database searching using the customized databases for individual samples or individual TMT experiments. For label-free data, we built a customized database for each individual sample based on germline and somatic variants from WXS data and variants from RNA-Seq data. For TMT data, we built a customized database for each TMT experiment based on germline and somatic variants from WXS data and variants from RNA-Seq data derived from corresponding tumor samples. We used Customprodbj (<a class="inter-ref-link" href="https://github.com/bzhanglab/customprodbj" target="_blank">https://github.com/bzhanglab/customprodbj</a>) for customized database construction. We used ANNOVAR (<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d4132558e2478" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Wang et al., 2010</a><div data-db-target-of="bib67-d4132558e2478" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang K. </li><li class="loa__item">Li M. </li><li class="loa__item">Hakonarson H. </li></ul><div class="reference-citations__detials">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">e164</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkq603&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20601685&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77956534324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9421)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e164&journal=Nucleic+Acids+Res.&author=Wang+K.&author=Li+M.&author=Hakonarson+H.&title=ANNOVAR%3A+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for variant annotation and dbSNP150, COSMIC83, dbnsfp33a and Clinvar (version 20170905) were used. For variant peptide identification, MS/MS data were searched by three search engines (MyriMatch version 2.2.10165, X!Tandem version Alanine 2017.02.01, and MS-GF+ version 2017.01.13) through IPeak against the customized protein databases with decoy sequences. The results from the three search engines were then integrated by IPeak and 1% FDR at PSM level was used. We further validated the identified variant peptides using PepQuery (<a class="inter-ref-link" href="http://www.pepquery.org" target="_blank">http://www.pepquery.org</a>) (<div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d4132558e2484" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Wen et al., 2019</a><div data-db-target-of="bib75-d4132558e2484" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wen B. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and only confidently identified variant peptides with PepQuery p value ≤ 0.01 were retained.</div></section><section id="sec5.4.33" data-locator="sec5.4.33" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">MHC-binding Peptide Prediction</span></h4><div class="section-paragraph">We used Optitype (<div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d4132558e2493" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Szolek et al., 2014</a><div data-db-target-of="bib59-d4132558e2493" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Szolek A. </li><li class="loa__item">Schubert B. </li><li class="loa__item">Mohr C. </li><li class="loa__item">Sturm M. </li><li class="loa__item">Feldhahn M. </li><li class="loa__item">Kohlbacher O. </li></ul><div class="reference-citations__detials">OptiType: precision HLA typing from next-generation sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (449)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to perform human leukocyte antigen (HLA) genotyping for each sample based on WXS and RNA-Seq data, respectively, and got exactly the same results. Then we used netMHCpan (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d4132558e2496" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Jurtz et al., 2017</a><div data-db-target-of="bib22-d4132558e2496" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Jurtz V. </li><li class="loa__item">Paul S. </li><li class="loa__item">Andreatta M. </li><li class="loa__item">Marcatili P. </li><li class="loa__item">Peters B. </li><li class="loa__item">Nielsen M. </li></ul><div class="reference-citations__detials">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.<div class="popup-ref"><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (869)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to predict HLA-peptide binding affinity for somatic mutation-derived variant peptides with a length between 8-11 amino acids. The HLA-peptides with IC<sub>50</sub> binding affinity &lt; 150 nM were considered to be neoantigens.</div></section><section id="sec5.4.34" data-locator="sec5.4.34" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Subtype Prediction</span></h4><div class="section-paragraph">For proteomic subtype prediction, we performed row and column z-score transformation for the CPTAC-CRC and prospective colon tumor samples separately. To assign prospective tumors to our previously identified proteomic subtypes (ProS A-E) (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2507" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2507" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), the R package pamr (<a class="inter-ref-link" href="https://cran.r-project.org/web/packages/pamr/index.html" target="_blank">https://cran.r-project.org/web/packages/pamr/index.html</a>) was used to apply predefined signature genes (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4132558e2513" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Zhang et al., 2014</a><div data-db-target-of="bib77-d4132558e2513" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang B. </li><li class="loa__item">Wang J. </li><li class="loa__item">Wang X. </li><li class="loa__item">Zhu J. </li><li class="loa__item">Liu Q. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Zimmerman L.J. </li><li class="loa__item">Shaddox K.F. </li><li class="loa__item">Kim S. </li><li>et al.</li></ul><div class="reference-citations__detials"><span>NCI CPTAC</span><br />Proteogenomic characterization of human colon and rectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to the protein expression matrix. To identify the optimal value of the shrinkage parameter for our PAM prediction model, we selected the value that minimized leave-one-out cross-validated misclassification error for the 79 CPTAC-CRC tumor samples (error rate &lt; 2%). We assigned 88 prospective tumors to the five proteomic subtypes with probability higher than 0.9.</div><div class="section-paragraph">For transcriptomic subtype prediction, we employed the random forest predictor implemented in the R package CMSclassifier (<a class="inter-ref-link" href="https://github.com/Sage-Bionetworks/CMSclassifier" target="_blank">https://github.com/Sage-Bionetworks/CMSclassifier</a>) (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4132558e2521" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e2521" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to assign the consensus molecular subtypes (CMSs) to each sample based on the RSEM gene expression profiles from RNA-Seq. Using a default posterior probability of 0.5 as a threshold for sample classification, we assigned 85 prospective tumors to the four CMS subtypes.</div></section><section id="sec5.4.35" data-locator="sec5.4.35" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unified View of Subtypes</span></h4><div class="section-paragraph">The associations between the three subtype classifications (MSI, CMS and ProS) were assessed by the Fisher’s exact test. Only subtype pairs with an adjusted <em>p-value</em> less than 0.05 were considered to be associated. An association network was constructed based on the results, in which nodes represent the subtypes from individual classification systems, and weighted edges represent statistically significant association between two subtypes and the level of significance. In this network, we found three disjoint subnetworks or modules, which were defined as three unified multi-omics subtypes (UMSs). We further assigned each sample in the cohort to the three UMS subtypes using a network centrality-based weighted voting algorithm described below.</div><div class="section-paragraph">Let <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif" overflow="scroll"><mi>V</mi></math> denote the vertices, <em>E</em> denote the edges and M denote the modules in the network. <em>E</em><sub>i,j</sub> represents the weight of the edge connecting node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif" overflow="scroll"><mi>i</mi></math> and <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif" overflow="scroll"><mi>j</mi></math>. We define<div class="labeled-equation clearfix"><div id="ufd1" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si4.gif" overflow="scroll"><mrow><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mo>−</mo><mi>l</mi><mi>o</mi><msub><mi>g</mi><mrow><mn>10</mn></mrow></msub><mi>A</mi><mi>d</mi><mi>j</mi><mo>.</mo><msub><mi>P</mi><mrow><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></msub></mrow></math></div></div><br></br>Where <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si5.gif" overflow="scroll"><mrow><mi>A</mi><mi>d</mi><mi>j</mi><mo>.</mo><msub><mi>P</mi><mrow><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow></mrow></msub></mrow></math> represents the adjusted <em>p-value</em> from the Fisher’s exact test between node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif" overflow="scroll"><mi>i</mi></math> and node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif" overflow="scroll"><mi>j</mi></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si6.gif" overflow="scroll"><mrow><mo>∀</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>∈</mo><mi>V</mi></mrow></math>.<div class="labeled-equation clearfix"><div id="ufd2" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si7.gif" overflow="scroll"><mrow><msub><mi>W</mi><mrow><mi>i</mi><mo>→</mo><mi>m</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mo>∑</mo><mrow><mi>j</mi><mo>∈</mo><mi>m</mi><mo>;</mo><mo>&lt;</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>&gt;</mo><mo>∈</mo><mi>E</mi><mo>;</mo><mi>i</mi><mo>&lt;</mo><mi>j</mi></mrow></msub><mrow><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow><mrow><msub><mo>∑</mo><mrow><mi>j</mi><mo>,</mo><mi>k</mi><mo>∈</mo><mi>m</mi><mo>,</mo><mo>&lt;</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>&gt;</mo><mo>∈</mo><mi>E</mi><mo>,</mo><mi>j</mi><mo>&lt;</mo><mi>k</mi></mrow></msub><mrow><msub><mi>E</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub></mrow></mrow></mfrac></mrow></math></div></div><br></br>Where <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si8.gif" overflow="scroll"><mrow><msub><mi>W</mi><mrow><mi>i</mi><mo>→</mo><mi>m</mi></mrow></msub></mrow></math> represents the weighted contribution of node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif" overflow="scroll"><mi>i</mi></math> (old subtypes) to the module <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si9.gif" overflow="scroll"><mi>m</mi></math> (new subtypes), <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si10.gif" overflow="scroll"><mrow><mo>∀</mo><mi>i</mi><mo>∈</mo><mi>V</mi><mo>,</mo><mo>∀</mo><mi>m</mi><mo>∈</mo><mi>M</mi></mrow></math>. <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si11.gif" overflow="scroll"><mrow><mrow><mo>〈</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>〉</mo></mrow></mrow></math> denotes the connection between node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif" overflow="scroll"><mi>i</mi></math> and node <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif" overflow="scroll"><mi>j</mi></math>, <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si6.gif" overflow="scroll"><mrow><mo>∀</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>∈</mo><mi>V</mi></mrow></math>.</div><div class="section-paragraph">Finally, for each sample <em>x</em>,<div class="labeled-equation clearfix"><div id="ufd3" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si12.gif" overflow="scroll"><mrow><msub><mi>S</mi><mrow><mi>x</mi><mo>,</mo><mi>m</mi></mrow></msub><mo>=</mo><munder><mo>∑</mo><mrow><mi>A</mi><mi>l</mi><mi>l</mi><mspace width="0.25em"></mspace><mi>i</mi><mspace width="0.25em"></mspace><mi>a</mi><mi>s</mi><mi>s</mi><mi>i</mi><mi>g</mi><mi>n</mi><mi>e</mi><mi>d</mi><mspace width="0.25em"></mspace><mi>t</mi><mi>o</mi><mspace width="0.25em"></mspace><mi>x</mi></mrow></munder><mrow><msub><mi>W</mi><mrow><mi>i</mi><mo>→</mo><mi>m</mi></mrow></msub></mrow></mrow></math></div></div><br></br><div class="labeled-equation clearfix"><div id="ufd4" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si13.gif" overflow="scroll"><mrow><mi>S</mi><mi>u</mi><mi>b</mi><mi>t</mi><mi>y</mi><mi>p</mi><mi>e</mi><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd><mrow><msup><mi>m</mi><mo>'</mo></msup><mo>,</mo><mspace width="0.25em"></mspace><mi>w</mi><mi>h</mi><mi>e</mi><mi>r</mi><mi>e</mi><mspace width="0.25em"></mspace><msub><mi>S</mi><mrow><mi>x</mi><mo>,</mo><msup><mi>m</mi><mo>'</mo></msup></mrow></msub><mo>=</mo><mi>max</mi><mrow><mo>(</mo><mrow><msub><mi>S</mi><mrow><mi>x</mi><mo>,</mo><mi>m</mi></mrow></msub></mrow><mo>)</mo></mrow><mi>i</mi><mi>f</mi><mfrac><mrow><mi>max</mi><mrow><mo>(</mo><mrow><msub><mi>S</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow></mrow><mrow><mo>∑</mo><mrow><msub><mi>S</mi><mi>m</mi></msub></mrow></mrow></mfrac><mo>&gt;</mo><mn>0.7</mn></mrow></mtd></mtr><mtr><mtd><mrow><mo>"</mo><mi>u</mi><mi>n</mi><mi>d</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>d</mi><mi>e</mi><mi>d</mi><mo>"</mo><mo>,</mo><mspace width="0.25em"></mspace><mi>i</mi><mi>f</mi><mfrac><mrow><mi>max</mi><mrow><mo>(</mo><mrow><msub><mi>S</mi><mi>m</mi></msub></mrow><mo>)</mo></mrow></mrow><mrow><mo>∑</mo><mrow><msub><mi>S</mi><mi>m</mi></msub></mrow></mrow></mfrac><mo>&lt;</mo><mn>0.7</mn></mrow></mtd></mtr></mtable></mrow></mrow></mrow></math></div></div><br></br>Where <!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si14.gif" overflow="scroll"><mrow><msub><mi>S</mi><mrow><mi>x</mi><mo>,</mo><mi>m</mi></mrow></msub></mrow></math> represents the score of the UMS subtype <em>m</em> for a given sample <em>x</em>, and <em>m’</em> represents the UMS assigned to the sample <em>x</em>. To have a more definitive classification, we required a new subtype assignment to have a score greater than 70% of the sum of scores of all three new subtypes.</div></section><section id="sec5.4.36" data-locator="sec5.4.36" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Subtype-specific Marker Identification and Pathway Analysis</span></h4><div class="section-paragraph">We performed both site-level phosphorylation and microRNA abundance comparisons across the different UMS subtypes using the Wilcoxon rank sum test to identify subtype-specific markers. For phosphosite markers, we also required their subtype-specific changes to be greater (or in a different direction) than changes at the protein level to highlight phosphorylation-level regulation. To study metabolism differences between CMS3 and the other CMS subtypes, we downloaded the pre-defined metabolic gene sets (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4132558e3121" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Guinney et al., 2015</a><div data-db-target-of="bib19-d4132558e3121" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Guinney J. </li><li class="loa__item">Dienstmann R. </li><li class="loa__item">Wang X. </li><li class="loa__item">de Reyniès A. </li><li class="loa__item">Schlicker A. </li><li class="loa__item">Soneson C. </li><li class="loa__item">Marisa L. </li><li class="loa__item">Roepman P. </li><li class="loa__item">Nyamundanda G. </li><li class="loa__item">Angelino P. </li><li>et al.</li></ul><div class="reference-citations__detials">The consensus molecular subtypes of colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from the Synapse platform (<a class="inter-ref-link" href="https://www.synapse.org/#!Synapse:syn2623706/wiki/" target="_blank">https://www.synapse.org/#!Synapse:syn2623706/wiki/</a>) and used ssGSEA to infer RNA-based and protein-based pathway activities.</div></section><section id="sec5.4.37" data-locator="sec5.4.37" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of Immune and Stromal Cell Infiltration</span></h4><div class="section-paragraph">Overall immune infiltration, stromal infiltration and tumor purity reported in <a href="/cms/10.1016/j.cell.2019.03.030/attachment/87317e5c-393a-47d6-9c62-cb4fee1d653b/mmc1" class="scroll-into-link">Table S1</a> were quantified by the ESTIMATE algorithm (<div class="dropBlock reference-citations"><a href="#bib76" id="back-bib76" data-db-target-for="bib76-d4132558e3135" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Yoshihara et al., 2013</a><div data-db-target-of="bib76-d4132558e3135" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoshihara K. </li><li class="loa__item">Shahmoradgoli M. </li><li class="loa__item">Martínez E. </li><li class="loa__item">Vegesna R. </li><li class="loa__item">Kim H. </li><li class="loa__item">Torres-Garcia W. </li><li class="loa__item">Treviño V. </li><li class="loa__item">Shen H. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Levine D.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Inferring tumour purity and stromal and immune cell admixture from expression data.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5182)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In addition, individual immune cell abundance for the current cohort was inferred by the GSVA method (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d4132558e3138" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Hänzelmann et al., 2013</a><div data-db-target-of="bib20-d4132558e3138" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hänzelmann S. </li><li class="loa__item">Castelo R. </li><li class="loa__item">Guinney J. </li></ul><div class="reference-citations__detials">GSVA: gene set variation analysis for microarray and RNA-seq data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6889)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using a previously published immune cell signature gene panel (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d4132558e3141" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Angelova et al., 2015</a><div data-db-target-of="bib4-d4132558e3141" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Angelova M. </li><li class="loa__item">Charoentong P. </li><li class="loa__item">Hackl H. </li><li class="loa__item">Fischer M.L. </li><li class="loa__item">Snajder R. </li><li class="loa__item">Krogsdam A.M. </li><li class="loa__item">Waldner M.J. </li><li class="loa__item">Bindea G. </li><li class="loa__item">Mlecnik B. </li><li class="loa__item">Galon J. </li><li class="loa__item">Trajanoski Z. </li></ul><div class="reference-citations__detials">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (390)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which involves 33 different immune cell types.</div></section><section id="sec5.4.38" data-locator="sec5.4.38" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Metabolic Pathway Analysis</span></h4><div class="section-paragraph">We performed both protein and mRNA abundance comparisons between the UMS MSI subtype and the other two UMS subtypes using the Wilcoxon rank sum test. Mapping gene-level results to KEGG pathways (<a class="inter-ref-link" href="http://www.genome.jp/kegg" target="_blank">http://www.genome.jp/kegg</a>) identified concordant changes of enzymes involved in the glycolysis and oxidative phosphorylation pathways. To summarize sample-specific glycolytic activities, we calculated the median of the gene-wise z-score-transformed protein abundance of all measured glycolytic genes.</div></section><section id="sec5.4.39" data-locator="sec5.4.39" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">SRM and PRISM-SRM Analysis</span></h4><div class="section-paragraph">We performed targeted analysis of selected proteins, namely CD8A, SLC2A3, PKM1, and PKM2, in representative tumor samples using selected reaction monitoring (SRM). For all proteins, analyses were performed on five MSI samples with relatively higher estimated activated CD8 T cell levels (MSI/CD8-H) and five MSI samples with relatively lower estimated activated CD8 T cell levels (MSI/CD8-L). For the analysis of SLC2A3, PKM1 and PKM2, we also included five samples from the CIN subtype and five from the Mesenchymal subtype.</div><div class="section-paragraph">Proteotypic peptides selected for each target protein were synthesized as crude peptides with the C-terminal lysine or arginine labeled with <sup>13</sup>C/<sup>15</sup>N (New England Peptide, Gardner, MA) The heavy peptides were dissolved individually in 15% acetonitrile (ACN) and 0.1% formic acid (FA) at a nominal concentration of 1.5 mM and mixed at a final concentration of 1 pmol/μL for each peptide. The best transitions and optimal collision energy (CE) for each peptide were determined using a direct infusion experiment, and 3 transitions per peptide were selected after LC-SRM analysis of a colon tissue digest spiked with the heavy peptides to confirm there is no issue with peptide retention time, transition interference, and endogenous peptide detectability.</div><div class="section-paragraph">Reliable detection of CD8A required the use of the highly sensitive high-pressure, high-resolution separations coupled with intelligent selection and multiplexing (PRISM)-SRM method (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d4132558e3167" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Shi et al., 2012</a><div data-db-target-of="bib57-d4132558e3167" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Shi T. </li><li class="loa__item">Fillmore T.L. </li><li class="loa__item">Sun X. </li><li class="loa__item">Zhao R. </li><li class="loa__item">Schepmoes A.A. </li><li class="loa__item">Hossain M. </li><li class="loa__item">Xie F. </li><li class="loa__item">Wu S. </li><li class="loa__item">Kim J.S. </li><li class="loa__item">Jones N. </li><li>et al.</li></ul><div class="reference-citations__detials">Antibody-free, targeted mass-spectrometric approach for quantification of proteins at low picogram per milliliter levels in human plasma/serum.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">15395-15400</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1073%2Fpnas.1204366109&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22949669&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84866538934&amp;cf=" onclick="newWindow(this.href);return false">Scopus (184)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=15395-15400&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Shi+T.&author=Fillmore+T.L.&author=Sun+X.&author=Zhao+R.&author=Schepmoes+A.A.&author=Hossain+M.&author=Xie+F.&author=Wu+S.&author=Kim+J.S.&author=Jones+N.&title=Antibody-free%2C+targeted+mass-spectrometric+approach+for+quantification+of+proteins+at+low+picogram+per+milliliter+levels+in+human+plasma%2Fserum">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A nanoACQUITY UPLC® system (Waters) equipped with a reversed-phase capillary LC column (3-μm Jupiter C18 bonded particles in-house packed in a 200 μm i.d. x 50 cm capillary) was used for the PRISM fractionation. Separations were performed at a flow rates of 2.2 μL/min using 10 mM ammonium formate (pH 10) in water as mobile phase A and 10 mM ammonium formate (pH 10) in 90% ACN as mobile phase B. Forty five microliters of sample with a peptide concentration of 1 μg/μL and 2 fmol/μL of heavy peptide standards was loaded onto the C18 capillary column and separated using a 190-min gradient (min:%B): 35:1, 37:10, 52:15, 87:25, 112:35, 125:45, 150:90, 156:1. The eluent was automatically deposited every minute onto a 96-well plate. Prior to peptide fraction collection, 20 μL of water was added to each well of the 96-well plate to avoid potential loss of peptides and dilute the peptide fractions (∼1:10 dilution) so they can be directly analyzed using LC-SRM (below).</div><div class="section-paragraph">Peptides from all the other target proteins in the colon tissue digests dissolved in 2% ACN/0.1% FA, as well as the CD8A PRISM fractions, were analyzed by LC-SRM using a TSQ Vantage triple quadruple mass spectrometer (Thermo Fisher Scientific) equipped with a nanoACQUITY UPLC system and an ACQUITY UPLC BEH 1.7-μm C18 100 μm i.d. x 10 cm column (Waters). The mobile phases were (A) 0.1% FA in water and (B) 0.1% FA in ACN. Two microliter of 0.25 μg/μL colon digest sample with 5 fmol/μL heavy peptides were loaded onto the column and separated at a flow rate of 400 nL/min using a 72-min gradient as follows (min:%B): 11:0.5, 13.5:10, 17:15, 38:25, 49:38, 50:95, 59:10, 60:95, 64:0.5. For PRISM fractions, 4 μL of each fraction were loaded onto the column and separated with a 35-min gradient as follows (min:%B): 12:0.5, 13:10, 14:15, 20:25, 23:35, 25:95, 26:10, 27:95, 28:0.5. The LC column is operated at a temperature of 44°C. The parameters of the triple quadruple instrument were set to 0.7 fwhm Q1 resolution with 1 s cycle time. Data were acquired in time-scheduled SRM mode (retention time window: 10 min). For PRISM fractions, data were acquired with unscheduled SRM mode.</div><div class="section-paragraph">All SRM data were analyzed using the Skyline software (<div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d4132558e3174" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">MacLean et al., 2010</a><div data-db-target-of="bib41-d4132558e3174" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">MacLean B. </li><li class="loa__item">Tomazela D.M. </li><li class="loa__item">Shulman N. </li><li class="loa__item">Chambers M. </li><li class="loa__item">Finney G.L. </li><li class="loa__item">Frewen B. </li><li class="loa__item">Kern R. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Liebler D.C. </li><li class="loa__item">MacCoss M.J. </li></ul><div class="reference-citations__detials">Skyline: an open source document editor for creating and analyzing targeted proteomics experiments.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">966-968</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtq054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20147306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77951965920&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=966-968&journal=Bioinformatics&author=MacLean+B.&author=Tomazela+D.M.&author=Shulman+N.&author=Chambers+M.&author=Finney+G.L.&author=Frewen+B.&author=Kern+R.&author=Tabb+D.L.&author=Liebler+D.C.&author=MacCoss+M.J.&title=Skyline%3A+an+open+source+document+editor+for+creating+and+analyzing+targeted+proteomics+experiments">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The total peak area ratios of endogenous light peptides and their heavy isotope-labeled internal standards (i.e., L/H peak area ratios) were exported for quantitation. Peak detection and integration were carried out according to two criteria: (1) same retention time and (2) similar L/H peaks area ratios for all transitions. All data were also manually inspected to ensure correct retention time, peak detection and accurate integration. Peptides used in this study were: SLC2A3 (QVTVLELFR), PKM1 (CLAAALIVLTESGR), PKM2 (IYVDDGLISLQVK), CD8A (TWNLGETVELK).</div></section></section><section id="sec5.5" data-locator="sec5.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and Software Availability</span></h3><div class="section-paragraph">Raw genomics data from this study are available at the Sequence Read Archive (SRA), BioProject ID: PRJNA514017 (<a class="inter-ref-link" href="ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a" target="_blank">ftp://ftp-trace.ncbi.nlm.nih.gov/sra/review/SRP178677_20190114_143443_27e795eb0f314edf0479737480ab0f2a</a>). Raw and low-level processed proteomics data from this study are available at the CPTAC Data Portal (<a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S045" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S045</a>). All final data matrices are available at the LinkedOmics (<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d4132558e3188" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Vasaikar et al., 2018</a><div data-db-target-of="bib64-d4132558e3188" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1378)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a class="inter-ref-link" href="http://linkedomics.org/cptac-colon/" target="_blank">http://linkedomics.org/cptac-colon/</a>), which also provides computational tools for further exploration of this dataset.</div></section></section><section><h2 class="top" id="secsectitle0320" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by the National Cancer Institute (NCI) Clinical Proteomic Tumor Analysis Consortium (CPTAC) (U24CA159988, U24CA210954, U24CA160019, and U24CA210955) and the Cancer Prevention &amp; Research Institutes of Texas (CPRIT RR160027) and by funding from the McNair Medical Institute at The Robert and Janice McNair Foundation. The Pacific Northwest National Laboratory (PNNL) proteomics work described herein was performed in the Environmental Molecular Sciences Laboratory, a US Department of Energy (DOE) National Scientific User Facility located at PNNL in Richland, Washington. PNNL is a multi-program national laboratory operated by the Battelle Memorial Institute for the DOE under contract DE-AC05-76RL01830.</div><h3>Author Contributions</h3><div class="section-paragraph">Conceptualization, H.R., R.D.S., K.D.R., D.C.L., T.L., and B.Z.; Methodology, S.V., C.H., X.W., V.A.P., S.R.S., B.W., Y.D., Y.Z., M.A.G., L.J.Z., D.C.L., T.L., and B.Z.; Software, S.V., C.H., X.W., V.A.P., S.S., B.W., Y.D., Y.Z., Z.S., O.A.A., J.E.M., M.E.M., M.C.C., Q.M., L.D., T.L., and B.Z.; Validation, S.V., C.H., X.W., V.A.P., S.R.S., B.W., Y.D., Y.Z., Z.S., O.A.A., L.J.Z., J.E.M., K.S.L., Q.M., M.E., K.D.R., D.C.L., T.L., and B.Z.; Formal Analysis, S.V., C.H., X.W., V.A.P., S.R.S., B.W., Y.D., Y.Z., Z.S., O.A.A., J.E.M., M.C.C., Q.M., L.D., T.L., and B.Z.; Investigation, M.A.G., L.J.Z., T.R.C., R.J.M., R.Z., M.E.M., Y.-T.W., R.J.C.S., D.C.L., and T.L.; Resources, R.D.S., K.D.R., D.C.L., T.L., and B.Z.; Data Curation, S.V., C.H., X.W., V.A.P., S.R.S., B.W., T.L., and B.Z.; Writing – Original Drafts, S.V., C.H., X.W., S.R.S., B.W., Y.D., Y.Z., M.T., T.L., and B.Z.; Write – Review &amp; Editing, all authors; Visualization, S.V., C.H., X.W., S.R.S., B.W., Y.D., and Y.Z.; Supervision, R.D.S., K.D.R., D.C.L., T.L., and B.Z.; Project Administration, M.T., C.R.K., H.R., R.D.S., K.D.R., D.C.L., T.L., and B.Z.; Funding Acquisition, R.D.S., K.D.R., D.C.L., T.L., and B.Z.</div><h3>Declaration of Interests</h3><div class="section-paragraph">The authors declare no competing interests.</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental Information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental Information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/87317e5c-393a-47d6-9c62-cb4fee1d653b/mmc1.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.04
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. Samples Analyzed by Each Omics Platform and the Clinical, Pathological, and Selected Molecular Characteristics of the Tumors, Related to Figure 1</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/66edcd43-6f7c-4a3f-932c-f67c762caa96/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.92
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. Microsatellite Instability Analysis Results, Related to Figure 2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/422700f5-078e-4f85-b043-ff1f75a164fd/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (10.99
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. Somatic Protein Altering Events and Their Occurrence in Individual Tumors, Related to Figure 2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/95453698-dea5-457c-9f18-d69fd79b7e69/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.03
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. Prioritized Copy-Number Drivers, Related to Figure 3</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/0fedc57b-d46f-4fe4-b933-7f096d4a4d2b/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.02
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Colon Cancer-Associated Proteomic Events and Clinical Utilities, Related to Figure 5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/30cfa83a-177e-4ed2-90a2-2d401364e784/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.61
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Variant Peptide and Neoantigen Analysis Results, Related to Figure 5</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.cell.2019.03.030/attachment/abe17f04-7d22-4c30-8f9b-40d50d641f5c/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.24
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S7. MicroRNA and Phosphosite Markers of the UMS Subtypes, Related to Figure 6</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0345" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Akiyama H. </li><li class="ref__authors__name">Kamitani T. </li><li class="ref__authors__name">Yang X. </li><li class="ref__authors__name">Kandyil R. </li><li class="ref__authors__name">Bridgewater L.C. </li><li class="ref__authors__name">Fellous M. </li><li class="ref__authors__name">Mori-Akiyama Y. </li><li class="ref__authors__name">de Crombrugghe B. </li></ul><div class="ref__title">The transcription factor Sox9 is degraded by the ubiquitin-proteasome system and stabilized by a mutation in a ubiquitin-target site.</div><span class="ref__series"><i>Matrix Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">499-505</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-13444260932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=15694126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.matbio.2004.10.002&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=499-505&journal=Matrix+Biol.&author=Akiyama+H.&author=Kamitani+T.&author=Yang+X.&author=Kandyil+R.&author=Bridgewater+L.C.&author=Fellous+M.&author=Mori-Akiyama+Y.&author=de+Crombrugghe+B.&title=The+transcription+factor+Sox9+is+degraded+by+the+ubiquitin-proteasome+system+and+stabilized+by+a+mutation+in+a+ubiquitin-target+site">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Almeida L.G. </li><li class="ref__authors__name">Sakabe N.J. </li><li class="ref__authors__name">deOliveira A.R. </li><li class="ref__authors__name">Silva M.C. </li><li class="ref__authors__name">Mundstein A.S. </li><li class="ref__authors__name">Cohen T. </li><li class="ref__authors__name">Chen Y.T. </li><li class="ref__authors__name">Chua R. </li><li class="ref__authors__name">Gurung S. </li><li class="ref__authors__name">Gnjatic S. </li><li>et al.</li></ul><div class="ref__title">CTdatabase: a knowledge-base of high-throughput and curated data on cancer-testis antigens.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">D816-D819</span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-58149186474&amp;cf=" onclick="newWindow(this.href);return false">Scopus (308)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18838390&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkn673&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D816-D819&journal=Nucleic+Acids+Res.&author=Almeida+L.G.&author=Sakabe+N.J.&author=deOliveira+A.R.&author=Silva+M.C.&author=Mundstein+A.S.&author=Cohen+T.&author=Chen+Y.T.&author=Chua+R.&author=Gurung+S.&author=Gnjatic+S.&title=CTdatabase%3A+a+knowledge-base+of+high-throughput+and+curated+data+on+cancer-testis+antigens">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anders S. </li><li class="ref__authors__name">Pyl P.T. </li><li class="ref__authors__name">Huber W. </li></ul><div class="ref__title">HTSeq--a Python framework to work with high-throughput sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">166-169</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84928987900&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13104)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25260700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtu638&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=166-169&journal=Bioinformatics&author=Anders+S.&author=Pyl+P.T.&author=Huber+W.&title=HTSeq%2D%2Da+Python+framework+to+work+with+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Angelova M. </li><li class="ref__authors__name">Charoentong P. </li><li class="ref__authors__name">Hackl H. </li><li class="ref__authors__name">Fischer M.L. </li><li class="ref__authors__name">Snajder R. </li><li class="ref__authors__name">Krogsdam A.M. </li><li class="ref__authors__name">Waldner M.J. </li><li class="ref__authors__name">Bindea G. </li><li class="ref__authors__name">Mlecnik B. </li><li class="ref__authors__name">Galon J. </li><li class="ref__authors__name">Trajanoski Z. </li></ul><div class="ref__title">Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">64</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84931426001&amp;cf=" onclick="newWindow(this.href);return false">Scopus (390)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25853550&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs13059-015-0620-6&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=64&journal=Genome+Biol.&author=Angelova+M.&author=Charoentong+P.&author=Hackl+H.&author=Fischer+M.L.&author=Snajder+R.&author=Krogsdam+A.M.&author=Waldner+M.J.&author=Bindea+G.&author=Mlecnik+B.&author=Galon+J.&author=Trajanoski+Z.&title=Characterization+of+the+immunophenotypes+and+antigenomes+of+colorectal+cancers+reveals+distinct+tumor+escape+mechanisms+and+novel+targets+for+immunotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Arnold M. </li><li class="ref__authors__name">Sierra M.S. </li><li class="ref__authors__name">Laversanne M. </li><li class="ref__authors__name">Soerjomataram I. </li><li class="ref__authors__name">Jemal A. </li><li class="ref__authors__name">Bray F. </li></ul><div class="ref__title">Global patterns and trends in colorectal cancer incidence and mortality.</div><span class="ref__series"><i>Gut.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">66</span></span>: <span class="ref__seriesPages">683-691</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84962150091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3331)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26818619&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1136%2Fgutjnl-2015-310912&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=683-691&journal=Gut&author=Arnold+M.&author=Sierra+M.S.&author=Laversanne+M.&author=Soerjomataram+I.&author=Jemal+A.&author=Bray+F.&title=Global+patterns+and+trends+in+colorectal+cancer+incidence+and+mortality">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Beausoleil S.A. </li><li class="ref__authors__name">Villén J. </li><li class="ref__authors__name">Gerber S.A. </li><li class="ref__authors__name">Rush J. </li><li class="ref__authors__name">Gygi S.P. </li></ul><div class="ref__title">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1297)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brand A. </li><li class="ref__authors__name">Singer K. </li><li class="ref__authors__name">Koehl G.E. </li><li class="ref__authors__name">Kolitzus M. </li><li class="ref__authors__name">Schoenhammer G. </li><li class="ref__authors__name">Thiel A. </li><li class="ref__authors__name">Matos C. </li><li class="ref__authors__name">Bruss C. </li><li class="ref__authors__name">Klobuch S. </li><li class="ref__authors__name">Peter K. </li><li>et al.</li></ul><div class="ref__title">LDHA-Associated Lactic Acid Production Blunts Tumor Immunosurveillance by T and NK Cells.</div><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">657-671</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84994592466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1089)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27641098&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2016.08.011&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=657-671&journal=Cell+Metab.&author=Brand+A.&author=Singer+K.&author=Koehl+G.E.&author=Kolitzus+M.&author=Schoenhammer+G.&author=Thiel+A.&author=Matos+C.&author=Bruss+C.&author=Klobuch+S.&author=Peter+K.&title=LDHA-Associated+Lactic+Acid+Production+Blunts+Tumor+Immunosurveillance+by+T+and+NK+Cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>Cancer Genome Atlas Network</li></ul><div class="ref__title">Comprehensive molecular characterization of human colon and rectal cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">487</span></span>: <span class="ref__seriesPages">330-337</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84863922124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6415)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22810696&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature11252&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=330-337&journal=Nature&author=Cancer+Genome+Atlas+Network&title=Comprehensive+molecular+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive genomic characterization of squamous cell lung cancers.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">489</span></span>: <span class="ref__seriesPages">519-525</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84866894408&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3135)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22960745&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature11404&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=489&publication_year=2012&pages=519-525&journal=Nature&author=Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen C.J. </li><li class="ref__authors__name">Servant N. </li><li class="ref__authors__name">Toedling J. </li><li class="ref__authors__name">Sarazin A. </li><li class="ref__authors__name">Marchais A. </li><li class="ref__authors__name">Duvernois-Berthet E. </li><li class="ref__authors__name">Cognat V. </li><li class="ref__authors__name">Colot V. </li><li class="ref__authors__name">Voinnet O. </li><li class="ref__authors__name">Heard E. </li><li>et al.</li></ul><div class="ref__title">ncPRO-seq: a tool for annotation and profiling of ncRNAs in sRNA-seq data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3147-3149</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84870420933&amp;cf=" onclick="newWindow(this.href);return false">Scopus (78)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23044543&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbts587&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3147-3149&journal=Bioinformatics&author=Chen+C.J.&author=Servant+N.&author=Toedling+J.&author=Sarazin+A.&author=Marchais+A.&author=Duvernois-Berthet+E.&author=Cognat+V.&author=Colot+V.&author=Voinnet+O.&author=Heard+E.&title=ncPRO-seq%3A+a+tool+for+annotation+and+profiling+of+ncRNAs+in+sRNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Christofk H.R. </li><li class="ref__authors__name">Vander Heiden M.G. </li><li class="ref__authors__name">Harris M.H. </li><li class="ref__authors__name">Ramanathan A. </li><li class="ref__authors__name">Gerszten R.E. </li><li class="ref__authors__name">Wei R. </li><li class="ref__authors__name">Fleming M.D. </li><li class="ref__authors__name">Schreiber S.L. </li><li class="ref__authors__name">Cantley L.C. </li></ul><div class="ref__title">The M2 splice isoform of pyruvate kinase is important for cancer metabolism and tumour growth.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">452</span></span>: <span class="ref__seriesPages">230-233</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-40749163248&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2283)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18337823&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature06734&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=452&publication_year=2008&pages=230-233&journal=Nature&author=Christofk+H.R.&author=Vander+Heiden+M.G.&author=Harris+M.H.&author=Ramanathan+A.&author=Gerszten+R.E.&author=Wei+R.&author=Fleming+M.D.&author=Schreiber+S.L.&author=Cantley+L.C.&title=The+M2+splice+isoform+of+pyruvate+kinase+is+important+for+cancer+metabolism+and+tumour+growth">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Garofano L. </li><li class="ref__authors__name">Cava C. </li><li class="ref__authors__name">Garolini D. </li><li class="ref__authors__name">Sabedot T.S. </li><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Pagnotta S.M. </li><li class="ref__authors__name">Castiglioni I. </li><li>et al.</li></ul><div class="ref__title">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2047)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic%C2%A0Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Craig R. </li><li class="ref__authors__name">Beavis R.C. </li></ul><div class="ref__title">TANDEM: matching proteins with tandem mass spectra.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1466-1467</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-3142702204&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2063)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=14976030&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbth092&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2004&pages=1466-1467&journal=Bioinformatics&author=Craig+R.&author=Beavis+R.C.&title=TANDEM%3A+matching+proteins+with+tandem+mass+spectra">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dobin A. </li><li class="ref__authors__name">Davis C.A. </li><li class="ref__authors__name">Schlesinger F. </li><li class="ref__authors__name">Drenkow J. </li><li class="ref__authors__name">Zaleski C. </li><li class="ref__authors__name">Jha S. </li><li class="ref__authors__name">Batut P. </li><li class="ref__authors__name">Chaisson M. </li><li class="ref__authors__name">Gingeras T.R. </li></ul><div class="ref__title">STAR: ultrafast universal RNA-seq aligner.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">15-21</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871809302&amp;cf=" onclick="newWindow(this.href);return false">Scopus (25709)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23104886&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbts635&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=15-21&journal=Bioinformatics&author=Dobin+A.&author=Davis+C.A.&author=Schlesinger+F.&author=Drenkow+J.&author=Zaleski+C.&author=Jha+S.&author=Batut+P.&author=Chaisson+M.&author=Gingeras+T.R.&title=STAR%3A+ultrafast+universal+RNA-seq+aligner">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Duffy M.J. </li></ul><div class="ref__title">Carcinoembryonic antigen as a marker for colorectal cancer: is it clinically useful?.</div><span class="ref__series"><i>Clin. Chem.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">624-630</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0035059972&amp;cf=" onclick="newWindow(this.href);return false">Scopus (610)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=11274010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fclinchem%2F47.4.624&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2001&pages=624-630&journal=Clin.+Chem.&author=Duffy+M.J.&title=Carcinoembryonic+antigen+as+a+marker+for+colorectal+cancer%3A+is+it+clinically+useful%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>ENCODE Project Consortium</li></ul><div class="ref__title">A user’s guide to the encyclopedia of DNA elements (ENCODE).</div><span class="ref__series"><i>PLoS Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">e1001046</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79955550445&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1168)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21526222&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1371%2Fjournal.pbio.1001046&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=e1001046&journal=PLoS+Biol.&author=ENCODE+Project+Consortium&title=A+user%E2%80%99s+guide+to+the+encyclopedia+of+DNA+elements+%28ENCODE%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Forbes S.A. </li><li class="ref__authors__name">Beare D. </li><li class="ref__authors__name">Boutselakis H. </li><li class="ref__authors__name">Bamford S. </li><li class="ref__authors__name">Bindal N. </li><li class="ref__authors__name">Tate J. </li><li class="ref__authors__name">Cole C.G. </li><li class="ref__authors__name">Ward S. </li><li class="ref__authors__name">Dawson E. </li><li class="ref__authors__name">Ponting L. </li><li>et al.</li></ul><div class="ref__title">COSMIC: somatic cancer genetics at high-resolution.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">D777-D783</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85016154270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1390)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27899578&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkw1121&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D777-D783&journal=Nucleic+Acids+Res.&author=Forbes+S.A.&author=Beare+D.&author=Boutselakis+H.&author=Bamford+S.&author=Bindal+N.&author=Tate+J.&author=Cole+C.G.&author=Ward+S.&author=Dawson+E.&author=Ponting+L.&title=COSMIC%3A+somatic+cancer+genetics+at+high-resolution">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gholami A.M. </li><li class="ref__authors__name">Hahne H. </li><li class="ref__authors__name">Wu Z. </li><li class="ref__authors__name">Auer F.J. </li><li class="ref__authors__name">Meng C. </li><li class="ref__authors__name">Wilhelm M. </li><li class="ref__authors__name">Kuster B. </li></ul><div class="ref__title">Global proteome analysis of the NCI-60 cell line panel.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">609-620</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84881613658&amp;cf=" onclick="newWindow(this.href);return false">Scopus (237)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23933261&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.celrep.2013.07.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=609-620&journal=Cell+Rep.&author=Gholami+A.M.&author=Hahne+H.&author=Wu+Z.&author=Auer+F.J.&author=Meng+C.&author=Wilhelm+M.&author=Kuster+B.&title=Global+proteome+analysis+of+the+NCI-60+cell+line+panel">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Guinney J. </li><li class="ref__authors__name">Dienstmann R. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">de Reyniès A. </li><li class="ref__authors__name">Schlicker A. </li><li class="ref__authors__name">Soneson C. </li><li class="ref__authors__name">Marisa L. </li><li class="ref__authors__name">Roepman P. </li><li class="ref__authors__name">Nyamundanda G. </li><li class="ref__authors__name">Angelino P. </li><li>et al.</li></ul><div class="ref__title">The consensus molecular subtypes of colorectal cancer.</div><span class="ref__series"><i>Nat. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">1350-1356</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84946571753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3274)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26457759&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnm.3967&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1350-1356&journal=Nat.+Med.&author=Guinney+J.&author=Dienstmann+R.&author=Wang+X.&author=de+Reyni%C3%A8s+A.&author=Schlicker+A.&author=Soneson+C.&author=Marisa+L.&author=Roepman+P.&author=Nyamundanda+G.&author=Angelino+P.&title=The+consensus+molecular+subtypes+of+colorectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hänzelmann S. </li><li class="ref__authors__name">Castelo R. </li><li class="ref__authors__name">Guinney J. </li></ul><div class="ref__title">GSVA: gene set variation analysis for microarray and RNA-seq data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">7</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84872202078&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6889)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23323831&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-14-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=7&journal=BMC+Bioinformatics&author=H%C3%A4nzelmann+S.&author=Castelo+R.&author=Guinney+J.&title=GSVA%3A+gene+set+variation+analysis+for+microarray+and+RNA-seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hornbeck P.V. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Murray B. </li><li class="ref__authors__name">Kornhauser J.M. </li><li class="ref__authors__name">Latham V. </li><li class="ref__authors__name">Skrzypek E. </li></ul><div class="ref__title">PhosphoSitePlus, 2014: mutations, PTMs and recalibrations.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D512-D520</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84941120469&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2026)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25514926&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgku1267&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D512-D520&journal=Nucleic+Acids+Res.&author=Hornbeck+P.V.&author=Zhang+B.&author=Murray+B.&author=Kornhauser+J.M.&author=Latham+V.&author=Skrzypek+E.&title=PhosphoSitePlus%2C+2014%3A+mutations%2C+PTMs+and+recalibrations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Jurtz V. </li><li class="ref__authors__name">Paul S. </li><li class="ref__authors__name">Andreatta M. </li><li class="ref__authors__name">Marcatili P. </li><li class="ref__authors__name">Peters B. </li><li class="ref__authors__name">Nielsen M. </li></ul><div class="ref__title">NetMHCpan-4.0: Improved Peptide-MHC Class I Interaction Predictions Integrating Eluted Ligand and Peptide Binding Affinity Data.</div><span class="ref__series"><i>J. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">199</span></span>: <span class="ref__seriesPages">3360-3368</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031724367&amp;cf=" onclick="newWindow(this.href);return false">Scopus (869)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28978689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.4049%2Fjimmunol.1700893&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=199&publication_year=2017&pages=3360-3368&journal=J.%C2%A0Immunol.&author=Jurtz+V.&author=Paul+S.&author=Andreatta+M.&author=Marcatili+P.&author=Peters+B.&author=Nielsen+M.&title=NetMHCpan-4.0%3A+Improved+Peptide-MHC+Class+I+Interaction+Predictions+Integrating+Eluted+Ligand+and+Peptide+Binding+Affinity+Data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kessner D. </li><li class="ref__authors__name">Chambers M. </li><li class="ref__authors__name">Burke R. </li><li class="ref__authors__name">Agus D. </li><li class="ref__authors__name">Mallick P. </li></ul><div class="ref__title">ProteoWizard: open source software for rapid proteomics tools development.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">2534-2536</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-54949129419&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1395)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18606607&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtn323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2534-2536&journal=Bioinformatics&author=Kessner+D.&author=Chambers+M.&author=Burke+R.&author=Agus+D.&author=Mallick+P.&title=ProteoWizard%3A+open+source+software+for+rapid+proteomics+tools+development">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (753)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Gupta N. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (344)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Knudsen E.S. </li><li class="ref__authors__name">Wang J.Y. </li></ul><div class="ref__title">Dual mechanisms for the inhibition of E2F binding to RB by cyclin-dependent kinase-mediated RB phosphorylation.</div><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">1997; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">5771-5783</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0030849455&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=9315635&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1128%2FMCB.17.10.5771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=5771-5783&journal=Mol.+Cell.+Biol.&author=Knudsen+E.S.&author=Wang+J.Y.&title=Dual+mechanisms+for+the+inhibition+of+E2F+binding+to+RB+by+cyclin-dependent+kinase-mediated+RB+phosphorylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kono K. </li><li class="ref__authors__name">Mizukami Y. </li><li class="ref__authors__name">Daigo Y. </li><li class="ref__authors__name">Takano A. </li><li class="ref__authors__name">Masuda K. </li><li class="ref__authors__name">Yoshida K. </li><li class="ref__authors__name">Tsunoda T. </li><li class="ref__authors__name">Kawaguchi Y. </li><li class="ref__authors__name">Nakamura Y. </li><li class="ref__authors__name">Fujii H. </li></ul><div class="ref__title">Vaccination with multiple peptides derived from novel cancer-testis antigens can induce specific T-cell responses and clinical responses in advanced esophageal cancer.</div><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">100</span></span>: <span class="ref__seriesPages">1502-1509</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-67650648498&amp;cf=" onclick="newWindow(this.href);return false">Scopus (120)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19459850&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2009.01200.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2009&pages=1502-1509&journal=Cancer+Sci.&author=Kono+K.&author=Mizukami+Y.&author=Daigo+Y.&author=Takano+A.&author=Masuda+K.&author=Yoshida+K.&author=Tsunoda+T.&author=Kawaguchi+Y.&author=Nakamura+Y.&author=Fujii+H.&title=Vaccination+with+multiple+peptides+derived+from+novel+cancer-testis+antigens+can+induce+specific+T-cell+responses+and+clinical+responses+in+advanced+esophageal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Korpal M. </li><li class="ref__authors__name">Lee E.S. </li><li class="ref__authors__name">Hu G. </li><li class="ref__authors__name">Kang Y. </li></ul><div class="ref__title">The miR-200 family inhibits epithelial-mesenchymal transition and cancer cell migration by direct targeting of E-cadherin transcriptional repressors ZEB1 and ZEB2.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">283</span></span>: <span class="ref__seriesPages">14910-14914</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-47249091921&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1402)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18411277&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fjbc.C800074200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=14910-14914&journal=J.%C2%A0Biol.+Chem.&author=Korpal+M.&author=Lee+E.S.&author=Hu+G.&author=Kang+Y.&title=The+miR-200+family+inhibits+epithelial-mesenchymal+transition+and+cancer+cell+migration+by+direct+targeting+of+E-cadherin+transcriptional+repressors+ZEB1+and+ZEB2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kuilman T. </li><li class="ref__authors__name">Velds A. </li><li class="ref__authors__name">Kemper K. </li><li class="ref__authors__name">Ranzani M. </li><li class="ref__authors__name">Bombardelli L. </li><li class="ref__authors__name">Hoogstraat M. </li><li class="ref__authors__name">Nevedomskaya E. </li><li class="ref__authors__name">Xu G. </li><li class="ref__authors__name">de Ruiter J. </li><li class="ref__authors__name">Lolkema M.P. </li><li>et al.</li></ul><div class="ref__title">CopywriteR: DNA copy number detection from off-target sequence data.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">49</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84937691121&amp;cf=" onclick="newWindow(this.href);return false">Scopus (158)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25887352&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs13059-015-0617-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=49&journal=Genome+Biol.&author=Kuilman+T.&author=Velds+A.&author=Kemper+K.&author=Ranzani+M.&author=Bombardelli+L.&author=Hoogstraat+M.&author=Nevedomskaya+E.&author=Xu+G.&author=de+Ruiter+J.&author=Lolkema+M.P.&title=CopywriteR%3A+DNA+copy+number+detection+from+off-target+sequence+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Landrum M.J. </li><li class="ref__authors__name">Lee J.M. </li><li class="ref__authors__name">Riley G.R. </li><li class="ref__authors__name">Jang W. </li><li class="ref__authors__name">Rubinstein W.S. </li><li class="ref__authors__name">Church D.M. </li><li class="ref__authors__name">Maglott D.R. </li></ul><div class="ref__title">ClinVar: public archive of relationships among sequence variation and human phenotype.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">D980-D985</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84891809093&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1887)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24234437&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkt1113&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D980-D985&journal=Nucleic+Acids+Res.&author=Landrum+M.J.&author=Lee+J.M.&author=Riley+G.R.&author=Jang+W.&author=Rubinstein+W.S.&author=Church+D.M.&author=Maglott+D.R.&title=ClinVar%3A+public+archive+of+relationships+among%C2%A0sequence+variation+and+human+phenotype">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Langmead B. </li><li class="ref__authors__name">Trapnell C. </li><li class="ref__authors__name">Pop M. </li><li class="ref__authors__name">Salzberg S.L. </li></ul><div class="ref__title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">R25</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-62349130698&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16269)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19261174&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fgb-2009-10-3-r25&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2009&pages=R25&journal=Genome+Biol.&author=Langmead+B.&author=Trapnell+C.&author=Pop+M.&author=Salzberg+S.L.&title=Ultrafast+and+memory-efficient+alignment+of+short+DNA+sequences+to+the+human+genome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Le D.T. </li><li class="ref__authors__name">Uram J.N. </li><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Bartlett B.R. </li><li class="ref__authors__name">Kemberling H. </li><li class="ref__authors__name">Eyring A.D. </li><li class="ref__authors__name">Skora A.D. </li><li class="ref__authors__name">Luber B.S. </li><li class="ref__authors__name">Azad N.S. </li><li class="ref__authors__name">Laheru D. </li><li>et al.</li></ul><div class="ref__title">PD-1 Blockade in Tumors with Mismatch-Repair Deficiency.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">372</span></span>: <span class="ref__seriesPages">2509-2520</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84932628341&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7182)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26028255&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1056%2FNEJMoa1500596&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=372&publication_year=2015&pages=2509-2520&journal=N.%C2%A0Engl.+J.+Med.&author=Le+D.T.&author=Uram+J.N.&author=Wang+H.&author=Bartlett+B.R.&author=Kemberling+H.&author=Eyring+A.D.&author=Skora+A.D.&author=Luber+B.S.&author=Azad+N.S.&author=Laheru+D.&title=PD-1+Blockade+in+Tumors+with+Mismatch-Repair+Deficiency">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li B. </li><li class="ref__authors__name">Dewey C.N. </li></ul><div class="ref__title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13447)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Durbin R. </li></ul><div class="ref__title">Fast and accurate long-read alignment with Burrows-Wheeler transform.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">589-595</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77949587649&amp;cf=" onclick="newWindow(this.href);return false">Scopus (8157)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20080505&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtp698&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=589-595&journal=Bioinformatics&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+long-read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Handsaker B. </li><li class="ref__authors__name">Wysoker A. </li><li class="ref__authors__name">Fennell T. </li><li class="ref__authors__name">Ruan J. </li><li class="ref__authors__name">Homer N. </li><li class="ref__authors__name">Marth G. </li><li class="ref__authors__name">Abecasis G. </li><li class="ref__authors__name">Durbin R. </li><li>1000 Genome Project Data Processing Subgroup</li></ul><div class="ref__title">The Sequence Alignment/Map format and SAMtools.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (37642)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinformatics&author=1000+Genome+Project+Data+Processing+Subgroup&title=The+Sequence+Alignment%2FMap+format+and+SAMtools">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Mesirov J.P. </li><li class="ref__authors__name">Tamayo P. </li></ul><div class="ref__title">The Molecular Signatures Database (MSigDB) hallmark gene set collection.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5972)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst.&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Integrative Omics Analysis Reveals Post-Transcriptionally Enhanced Protective Host Response in Colorectal Cancers with Microsatellite Instability.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">766-776</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84960401765&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=26680540&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Facs.jproteome.5b00847&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=766-776&journal=J.%C2%A0Proteome+Res.&author=Liu+Q.&author=Zhang+B.&title=Integrative+Omics+Analysis+Reveals+Post-Transcriptionally+Enhanced+Protective+Host+Response+in+Colorectal+Cancers+with+Microsatellite+Instability">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu X. </li><li class="ref__authors__name">Jian X. </li><li class="ref__authors__name">Boerwinkle E. </li></ul><div class="ref__title">dbNSFP: a lightweight database of human nonsynonymous SNPs and their functional predictions.</div><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">894-899</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79960763462&amp;cf=" onclick="newWindow(this.href);return false">Scopus (585)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21520341&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fhumu.21517&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=894-899&journal=Hum.+Mutat.&author=Liu+X.&author=Jian+X.&author=Boerwinkle+E.&title=dbNSFP%3A+a+lightweight+database+of+human+nonsynonymous+SNPs+and+their+functional+predictions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Llosa N.J. </li><li class="ref__authors__name">Cruise M. </li><li class="ref__authors__name">Tam A. </li><li class="ref__authors__name">Wicks E.C. </li><li class="ref__authors__name">Hechenbleikner E.M. </li><li class="ref__authors__name">Taube J.M. </li><li class="ref__authors__name">Blosser R.L. </li><li class="ref__authors__name">Fan H. </li><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Luber B.S. </li><li>et al.</li></ul><div class="ref__title">The vigorous immune microenvironment of microsatellite instable colon cancer is balanced by multiple counter-inhibitory checkpoints.</div><span class="ref__series"><i>Cancer Discov.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">43-51</span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84922369377&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1132)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25358689&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1158%2F2159-8290.CD-14-0863&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=43-51&journal=Cancer+Discov.&author=Llosa+N.J.&author=Cruise+M.&author=Tam+A.&author=Wicks+E.C.&author=Hechenbleikner+E.M.&author=Taube+J.M.&author=Blosser+R.L.&author=Fan+H.&author=Wang+H.&author=Luber+B.S.&title=The+vigorous+immune+microenvironment+of+microsatellite+instable+colon+cancer+is+balanced+by+multiple+counter-inhibitory+checkpoints">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lo Surdo P. </li><li class="ref__authors__name">Calderone A. </li><li class="ref__authors__name">Cesareni G. </li><li class="ref__authors__name">Perfetto L. </li></ul><div class="ref__title">SIGNOR: A Database of Causal Relationships Between Biological Entities-A Short Guide to Searching and Browsing.</div><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">58</span></span>: <span class="ref__seriesPages">8.23.1-8.23.16</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85029438175&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fcpbi.28&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2017&pages=8.23.1-8.23.16&journal=Curr.+Protoc.+Bioinformatics&author=Lo+Surdo+P.&author=Calderone+A.&author=Cesareni+G.&author=Perfetto+L.&title=SIGNOR%3A+A+Database+of+Causal+Relationships+Between+Biological+Entities-A+Short+Guide+to+Searching+and+Browsing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">MacLean B. </li><li class="ref__authors__name">Tomazela D.M. </li><li class="ref__authors__name">Shulman N. </li><li class="ref__authors__name">Chambers M. </li><li class="ref__authors__name">Finney G.L. </li><li class="ref__authors__name">Frewen B. </li><li class="ref__authors__name">Kern R. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Liebler D.C. </li><li class="ref__authors__name">MacCoss M.J. </li></ul><div class="ref__title">Skyline: an open source document editor for creating and analyzing targeted proteomics experiments.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">966-968</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77951965920&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3379)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20147306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtq054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=966-968&journal=Bioinformatics&author=MacLean+B.&author=Tomazela+D.M.&author=Shulman+N.&author=Chambers+M.&author=Finney+G.L.&author=Frewen+B.&author=Kern+R.&author=Tabb+D.L.&author=Liebler+D.C.&author=MacCoss+M.J.&title=Skyline%3A+an+open+source+document+editor+for+creating+and+analyzing+targeted+proteomics+experiments">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Maruvka Y.E. </li><li class="ref__authors__name">Mouw K.W. </li><li class="ref__authors__name">Karlic R. </li><li class="ref__authors__name">Parasuraman P. </li><li class="ref__authors__name">Kamburov A. </li><li class="ref__authors__name">Polak P. </li><li class="ref__authors__name">Haradhvala N.J. </li><li class="ref__authors__name">Hess J.M. </li><li class="ref__authors__name">Rheinbay E. </li><li class="ref__authors__name">Brody Y. </li><li>et al.</li></ul><div class="ref__title">Analysis of somatic microsatellite indels identifies driver events in human tumors.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">35</span></span>: <span class="ref__seriesPages">951-959</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031106017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (89)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28892075&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnbt.3966&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=951-959&journal=Nat.+Biotechnol.&author=Maruvka+Y.E.&author=Mouw+K.W.&author=Karlic+R.&author=Parasuraman+P.&author=Kamburov+A.&author=Polak+P.&author=Haradhvala+N.J.&author=Hess+J.M.&author=Rheinbay+E.&author=Brody+Y.&title=Analysis+of+somatic+microsatellite+indels+identifies+driver+events+in+human+tumors">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Matheu A. </li><li class="ref__authors__name">Collado M. </li><li class="ref__authors__name">Wise C. </li><li class="ref__authors__name">Manterola L. </li><li class="ref__authors__name">Cekaite L. </li><li class="ref__authors__name">Tye A.J. </li><li class="ref__authors__name">Canamero M. </li><li class="ref__authors__name">Bujanda L. </li><li class="ref__authors__name">Schedl A. </li><li class="ref__authors__name">Cheah K.S. </li><li>et al.</li></ul><div class="ref__title">Oncogenicity of the developmental transcription factor Sox9.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">72</span></span>: <span class="ref__seriesPages">1301-1315</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84857700780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (175)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22246670&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1158%2F0008-5472.CAN-11-3660&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=1301-1315&journal=Cancer+Res.&author=Matheu+A.&author=Collado+M.&author=Wise+C.&author=Manterola+L.&author=Cekaite+L.&author=Tye+A.J.&author=Canamero+M.&author=Bujanda+L.&author=Schedl+A.&author=Cheah+K.S.&title=Oncogenicity+of+the+developmental+transcription+factor+Sox9">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2189)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1138)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=NCI+CPTAC&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Shaw J.L. </li><li class="ref__authors__name">Daly D.S. </li><li class="ref__authors__name">Adkins J.N. </li><li class="ref__authors__name">Smith R.D. </li></ul><div class="ref__title">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.</div><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (122)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic%C2%A0profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mosesson Y. </li><li class="ref__authors__name">Mills G.B. </li><li class="ref__authors__name">Yarden Y. </li></ul><div class="ref__title">Derailed endocytosis: an emerging feature of cancer.</div><span class="ref__series"><i>Nat. Rev. Cancer.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">835-850</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-54549102284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (590)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=18948996&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnrc2521&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=835-850&journal=Nat.+Rev.+Cancer&author=Mosesson+Y.&author=Mills+G.B.&author=Yarden+Y.&title=Derailed+endocytosis%3A+an+emerging+feature+of+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Olshen A.B. </li><li class="ref__authors__name">Venkatraman E.S. </li><li class="ref__authors__name">Lucito R. </li><li class="ref__authors__name">Wigler M. </li></ul><div class="ref__title">Circular binary segmentation for the analysis of array-based DNA copy number data.</div><span class="ref__series"><i>Biostatistics.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">557-572</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-3543105225&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1696)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=15475419&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbiostatistics%2Fkxh008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=557-572&journal=Biostatistics&author=Olshen+A.B.&author=Venkatraman+E.S.&author=Lucito+R.&author=Wigler+M.&title=Circular+binary%C2%A0segmentation+for+the+analysis+of+array-based+DNA+copy+number+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pensa S. </li><li class="ref__authors__name">Regis G. </li><li class="ref__authors__name">Boselli D. </li><li class="ref__authors__name">Novelli F. </li><li class="ref__authors__name">Ploi V. </li></ul><div class="ref__title">STAT1 and STAT3 in tumorigenesis: two sides of the same coin?.</div><span>JAK-STAT Pathway in Disease. </span> <span class="refPublisherName">Landes Bioscience</span>,
        <span class="refPublisherLoc"></span><span class="refDate">2009</span>: <span class="ref__seriesPages">100-121</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&pages=100-121&author=Pensa+S.&author=Regis+G.&author=Boselli+D.&author=Novelli+F.&author=Ploi+V.&isbn=JAK-STAT+Pathway+in+Disease&title=STAT1+and+STAT3+in+tumorigenesis%3A+two+sides+of+the+same+coin%3F">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Mayampurath A.M. </li><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Polpitiya A.D. </li><li class="ref__authors__name">Purvine S.O. </li><li class="ref__authors__name">Anderson G.A. </li><li class="ref__authors__name">Camp 2nd, D.G. </li><li class="ref__authors__name">Smith R.D. </li></ul><div class="ref__title">DtaRefinery, a software tool for elimination of systematic errors from parent ion mass measurements in tandem mass spectra data sets.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">486-496</span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77649174058&amp;cf=" onclick="newWindow(this.href);return false">Scopus (48)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20019053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M900217-MCP200&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=486-496&journal=Mol.+Cell.+Proteomics&author=Petyuk+V.A.&author=Mayampurath+A.M.&author=Monroe+M.E.&author=Polpitiya+A.D.&author=Purvine+S.O.&author=Anderson+G.A.&author=Camp+D.G.&author=Smith+R.D.&title=DtaRefinery%2C+a+software+tool+for+elimination+of+systematic+errors+from+parent+ion+mass+measurements+in+tandem+mass+spectra+data+sets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ramos A.H. </li><li class="ref__authors__name">Lichtenstein L. </li><li class="ref__authors__name">Gupta M. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Pugh T.J. </li><li class="ref__authors__name">Saksena G. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Oncotator: cancer variant annotation tool.</div><span class="ref__series"><i>Hum. Mutat.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">E2423-E2429</span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84925965197&amp;cf=" onclick="newWindow(this.href);return false">Scopus (375)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25703262&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fhumu.22771&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=E2423-E2429&journal=Hum.+Mutat.&author=Ramos+A.H.&author=Lichtenstein+L.&author=Gupta+M.&author=Lawrence+M.S.&author=Pugh+T.J.&author=Saksena+G.&author=Meyerson+M.&author=Getz+G.&title=Oncotator%3A+cancer+variant+annotation+tool">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rubin S.M. </li></ul><div class="ref__title">Deciphering the retinoblastoma protein phosphorylation code.</div><span class="ref__series"><i>Trends Biochem. Sci.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">12-19</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84871513029&amp;cf=" onclick="newWindow(this.href);return false">Scopus (189)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23218751&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.tibs.2012.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=12-19&journal=Trends+Biochem.+Sci.&author=Rubin+S.M.&title=Deciphering+the+retinoblastoma+protein+phosphorylation+code">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Savitski M.M. </li><li class="ref__authors__name">Wilhelm M. </li><li class="ref__authors__name">Hahne H. </li><li class="ref__authors__name">Kuster B. </li><li class="ref__authors__name">Bantscheff M. </li></ul><div class="ref__title">A Scalable Approach for Protein False Discovery Rate Estimation in Large Proteomic Data Sets.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">2394-2404</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84940547273&amp;cf=" onclick="newWindow(this.href);return false">Scopus (290)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25987413&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M114.046995&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2394-2404&journal=Mol.+Cell.+Proteomics&author=Savitski+M.M.&author=Wilhelm+M.&author=Hahne+H.&author=Kuster+B.&author=Bantscheff+M.&title=A+Scalable+Approach+for+Protein+False+Discovery+Rate+Estimation+in+Large+Proteomic%C2%A0Data+Sets">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sharma P. </li><li class="ref__authors__name">Hu-Lieskovan S. </li><li class="ref__authors__name">Wargo J.A. </li><li class="ref__authors__name">Ribas A. </li></ul><div class="ref__title">Primary, Adaptive, and Acquired Resistance to Cancer Immunotherapy.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">168</span></span>: <span class="ref__seriesPages">707-723</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85012158583&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3269)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28187290&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2017.01.017&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=707-723&journal=Cell&author=Sharma+P.&author=Hu-Lieskovan+S.&author=Wargo+J.A.&author=Ribas+A.&title=Primary%2C+Adaptive%2C+and+Acquired+Resistance+to+Cancer+Immunotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sherr C.J. </li><li class="ref__authors__name">McCormick F. </li></ul><div class="ref__title">The RB and p53 pathways in cancer.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">2</span></span>: <span class="ref__seriesPages">103-112</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0001510491&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1378)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=12204530&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2FS1535-6108%2802%2900102-2&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=103-112&journal=Cancer+Cell&author=Sherr+C.J.&author=McCormick+F.&title=The+RB+and+p53+pathways+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sherry S.T. </li><li class="ref__authors__name">Ward M.H. </li><li class="ref__authors__name">Kholodov M. </li><li class="ref__authors__name">Baker J. </li><li class="ref__authors__name">Phan L. </li><li class="ref__authors__name">Smigielski E.M. </li><li class="ref__authors__name">Sirotkin K. </li></ul><div class="ref__title">dbSNP: the NCBI database of genetic variation.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2001; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">308-311</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-0035173378&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5316)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=11125122&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2F29.1.308&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=308-311&journal=Nucleic%C2%A0Acids+Res.&author=Sherry+S.T.&author=Ward+M.H.&author=Kholodov+M.&author=Baker+J.&author=Phan+L.&author=Smigielski+E.M.&author=Sirotkin+K.&title=dbSNP%3A+the+NCBI+database+of+genetic+variation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Shi T. </li><li class="ref__authors__name">Fillmore T.L. </li><li class="ref__authors__name">Sun X. </li><li class="ref__authors__name">Zhao R. </li><li class="ref__authors__name">Schepmoes A.A. </li><li class="ref__authors__name">Hossain M. </li><li class="ref__authors__name">Xie F. </li><li class="ref__authors__name">Wu S. </li><li class="ref__authors__name">Kim J.S. </li><li class="ref__authors__name">Jones N. </li><li>et al.</li></ul><div class="ref__title">Antibody-free, targeted mass-spectrometric approach for quantification of proteins at low picogram per milliliter levels in human plasma/serum.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. USA.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">109</span></span>: <span class="ref__seriesPages">15395-15400</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84866538934&amp;cf=" onclick="newWindow(this.href);return false">Scopus (184)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22949669&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1073%2Fpnas.1204366109&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=15395-15400&journal=Proc.+Natl.+Acad.+Sci.+USA&author=Shi+T.&author=Fillmore+T.L.&author=Sun+X.&author=Zhao+R.&author=Schepmoes+A.A.&author=Hossain+M.&author=Xie+F.&author=Wu+S.&author=Kim+J.S.&author=Jones+N.&title=Antibody-free%2C+targeted+mass-spectrometric+approach+for+quantification+of+proteins+at+low+picogram+per+milliliter+levels+in+human+plasma%2Fserum">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Suda T. </li><li class="ref__authors__name">Tsunoda T. </li><li class="ref__authors__name">Daigo Y. </li><li class="ref__authors__name">Nakamura Y. </li><li class="ref__authors__name">Tahara H. </li></ul><div class="ref__title">Identification of human leukocyte antigen-A24-restricted epitope peptides derived from gene products upregulated in lung and esophageal cancers as novel targets for immunotherapy.</div><span class="ref__series"><i>Cancer Sci.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">98</span></span>: <span class="ref__seriesPages">1803-1808</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-34848851598&amp;cf=" onclick="newWindow(this.href);return false">Scopus (101)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=17784873&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1111%2Fj.1349-7006.2007.00603.x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2007&pages=1803-1808&journal=Cancer+Sci.&author=Suda+T.&author=Tsunoda+T.&author=Daigo+Y.&author=Nakamura+Y.&author=Tahara+H.&title=Identification+of+human+leukocyte+antigen-A24-restricted+epitope+peptides+derived+from+gene+products+upregulated+in+lung+and+esophageal+cancers+as+novel+targets%C2%A0for+immunotherapy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Szolek A. </li><li class="ref__authors__name">Schubert B. </li><li class="ref__authors__name">Mohr C. </li><li class="ref__authors__name">Sturm M. </li><li class="ref__authors__name">Feldhahn M. </li><li class="ref__authors__name">Kohlbacher O. </li></ul><div class="ref__title">OptiType: precision HLA typing from next-generation sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">3310-3316</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84925550130&amp;cf=" onclick="newWindow(this.href);return false">Scopus (449)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25143287&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtu548&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=3310-3316&journal=Bioinformatics&author=Szolek+A.&author=Schubert+B.&author=Mohr+C.&author=Sturm+M.&author=Feldhahn+M.&author=Kohlbacher+O.&title=OptiType%3A+precision+HLA+typing+from+next-generation+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Fernando C.G. </li><li class="ref__authors__name">Chambers M.C. </li></ul><div class="ref__title">MyriMatch: highly accurate tandem mass spectral peptide identification by multivariate hypergeometric analysis.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">654-661</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-33847401893&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=17269722&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr0604054&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=654-661&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Fernando+C.G.&author=Chambers+M.C.&title=MyriMatch%3A+highly+accurate+tandem+mass+spectral+peptide+identification+by+multivariate+hypergeometric+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tang H. </li><li class="ref__authors__name">Fu Y.X. </li></ul><div class="ref__title">Immune Evasion in Tumor’s Own Sweet Way.</div><span class="ref__series"><i>Cell Metab.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">945-946</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85044618686&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=29628418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cmet.2018.03.013&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=945-946&journal=Cell+Metab.&author=Tang+H.&author=Fu+Y.X.&title=Immune+Evasion+in+Tumor%E2%80%99s+Own+Sweet+Way">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Uhlen M. </li><li class="ref__authors__name">Zhang C. </li><li class="ref__authors__name">Lee S. </li><li class="ref__authors__name">Sjöstedt E. </li><li class="ref__authors__name">Fagerberg L. </li><li class="ref__authors__name">Bidkhori G. </li><li class="ref__authors__name">Benfeitas R. </li><li class="ref__authors__name">Arif M. </li><li class="ref__authors__name">Liu Z. </li><li class="ref__authors__name">Edfors F. </li><li>et al.</li></ul><div class="ref__title">A pathology atlas of the human cancer transcriptome.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">357</span></span>: <span class="ref__seriesPages">eaan2507</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85028362951&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2170)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28818916&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1126%2Fscience.aan2507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2017&pages=eaan2507&journal=Science&author=Uhlen+M.&author=Zhang+C.&author=Lee+S.&author=Sj%C3%B6stedt+E.&author=Fagerberg+L.&author=Bidkhori+G.&author=Benfeitas+R.&author=Arif+M.&author=Liu+Z.&author=Edfors+F.&title=A+pathology+atlas+of+the+human+cancer+transcriptome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Van der Auwera G.A. </li><li class="ref__authors__name">Carneiro M.O. </li><li class="ref__authors__name">Hartl C. </li><li class="ref__authors__name">Poplin R. </li><li class="ref__authors__name">Del Angel G. </li><li class="ref__authors__name">Levy-Moonshine A. </li><li class="ref__authors__name">Jordan T. </li><li class="ref__authors__name">Shakir K. </li><li class="ref__authors__name">Roazen D. </li><li class="ref__authors__name">Thibault J. </li><li>et al.</li></ul><div class="ref__title">From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline.</div><span class="ref__series"><i>Curr. Protoc. Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">43</span> (<span class="refComment">11.10.1-33</span>)</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84896009017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4200)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25431634&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2F0471250953.bi1110s43&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2013&journal=Curr.+Protoc.+Bioinformatics&author=Van+der+Auwera+G.A.&author=Carneiro+M.O.&author=Hartl+C.&author=Poplin+R.&author=Del+Angel+G.&author=Levy-Moonshine+A.&author=Jordan+T.&author=Shakir+K.&author=Roazen+D.&author=Thibault+J.&title=From+FastQ+data+to+high+confidence+variant+calls%3A+the+Genome+Analysis+Toolkit+best+practices+pipeline">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Straub P. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1378)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vogelstein B. </li><li class="ref__authors__name">Papadopoulos N. </li><li class="ref__authors__name">Velculescu V.E. </li><li class="ref__authors__name">Zhou S. </li><li class="ref__authors__name">Diaz Jr., L.A. </li><li class="ref__authors__name">Kinzler K.W. </li></ul><div class="ref__title">Cancer genome landscapes.</div><span class="ref__series"><i>Science.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">339</span></span>: <span class="ref__seriesPages">1546-1558</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84875490185&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5727)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=23539594&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1126%2Fscience.1235122&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1546-1558&journal=Science&author=Vogelstein+B.&author=Papadopoulos+N.&author=Velculescu+V.E.&author=Zhou+S.&author=Diaz+L.A.&author=Kinzler+K.W.&title=Cancer+genome+landscapes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">customProDB: an R package to generate customized protein databases from RNA-Seq data for proteomics search.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">3235-3237</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84894110748&amp;cf=" onclick="newWindow(this.href);return false">Scopus (119)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24058055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fbioinformatics%2Fbtt543&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=3235-3237&journal=Bioinformatics&author=Wang+X.&author=Zhang+B.&title=customProDB%3A+an+R+package+to+generate+customized+protein+databases+from+RNA-Seq+data+for+proteomics+search">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang K. </li><li class="ref__authors__name">Li M. </li><li class="ref__authors__name">Hakonarson H. </li></ul><div class="ref__title">ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">e164</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-77956534324&amp;cf=" onclick="newWindow(this.href);return false">Scopus (9421)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=20601685&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkq603&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=e164&journal=Nucleic+Acids+Res.&author=Wang+K.&author=Li+M.&author=Hakonarson+H.&title=ANNOVAR%3A+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Slebos R.J. </li><li class="ref__authors__name">Wang D. </li><li class="ref__authors__name">Halvey P.J. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Liebler D.C. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">Protein identification using customized protein sequence databases derived from RNA-Seq data.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">1009-1017</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84863011504&amp;cf=" onclick="newWindow(this.href);return false">Scopus (141)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=22103967&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1021%2Fpr200766z&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=1009-1017&journal=J.%C2%A0Proteome+Res.&author=Wang+X.&author=Slebos+R.J.&author=Wang+D.&author=Halvey+P.J.&author=Tabb+D.L.&author=Liebler+D.C.&author=Zhang+B.&title=Protein+identification+using+customized+protein+sequence+databases%C2%A0derived+from+RNA-Seq+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Ma Z. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Chan D.W. </li><li class="ref__authors__name">Ellis M.J. </li><li class="ref__authors__name">Townsend R.R. </li><li class="ref__authors__name">Smith R.D. </li><li>et al.</li></ul><div class="ref__title">Proteome Profiling Outperforms Transcriptome Profiling for Coexpression Based Gene Function Prediction.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">121-134</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85008490068&amp;cf=" onclick="newWindow(this.href);return false">Scopus (96)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27836980&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1074%2Fmcp.M116.060301&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=121-134&journal=Mol.+Cell.+Proteomics&author=Wang+J.&author=Ma+Z.&author=Carr+S.A.&author=Mertins+P.&author=Zhang+H.&author=Zhang+Z.&author=Chan+D.W.&author=Ellis+M.J.&author=Townsend+R.R.&author=Smith+R.D.&title=Proteome+Profiling+Outperforms%C2%A0Transcriptome+Profiling+for+Coexpression+Based+Gene+Function+Prediction">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Mouradov D. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Jorissen R.N. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Zimmerman L.J. </li><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Love C.G. </li><li class="ref__authors__name">Li S. </li><li class="ref__authors__name">Lowes K. </li><li>et al.</li></ul><div class="ref__title">Colorectal Cancer Cell Line Proteomes Are Representative of Primary Tumors and Predict Drug Sensitivity.</div><span class="ref__series"><i>Gastroenterology.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">153</span></span>: <span class="ref__seriesPages">1082-1095</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85031699623&amp;cf=" onclick="newWindow(this.href);return false">Scopus (50)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28625833&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=abstract&amp;site=ygast-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=fulltext&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1053%2Fj.gastro.2017.06.008&amp;cf=pdf&amp;site=ygast-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2017&pages=1082-1095&journal=Gastroenterology&author=Wang+J.&author=Mouradov+D.&author=Wang+X.&author=Jorissen+R.N.&author=Chambers+M.C.&author=Zimmerman+L.J.&author=Vasaikar+S.&author=Love+C.G.&author=Li+S.&author=Lowes+K.&title=Colorectal+Cancer+Cell+Line+Proteomes+Are+Representative+of+Primary+Tumors+and+Predict+Drug+Sensitivity">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Greer M. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">W130-W137</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85023191744&amp;cf=" onclick="newWindow(this.href);return false">Scopus (787)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28472511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1093%2Fnar%2Fgkx356&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W130-W137&journal=Nucleic+Acids+Res.&author=Wang+J.&author=Vasaikar+S.&author=Shi+Z.&author=Greer+M.&author=Zhang+B.&title=WebGestalt%C2%A02017%3A+a+more+comprehensive%2C+powerful%2C+flexible+and+interactive+gene+set+enrichment+analysis+toolkit">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Li G. </li><li class="ref__authors__name">Wright J.C. </li><li class="ref__authors__name">Du C. </li><li class="ref__authors__name">Feng Q. </li><li class="ref__authors__name">Xu X. </li><li class="ref__authors__name">Choudhary J.S. </li><li class="ref__authors__name">Wang J. </li></ul><div class="ref__title">The OMSSAPercolator: an automated tool to validate OMSSA results.</div><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">14</span></span>: <span class="ref__seriesPages">1011-1014</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84898621562&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24504981&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fpmic.201300393&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1011-1014&journal=Proteomics&author=Wen+B.&author=Li+G.&author=Wright+J.C.&author=Du+C.&author=Feng+Q.&author=Xu+X.&author=Choudhary+J.S.&author=Wang+J.&title=The+OMSSAPercolator%3A+an+automated+tool+to+validate+OMSSA+results">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Du C. </li><li class="ref__authors__name">Li G. </li><li class="ref__authors__name">Ghali F. </li><li class="ref__authors__name">Jones A.R. </li><li class="ref__authors__name">Käll L. </li><li class="ref__authors__name">Xu S. </li><li class="ref__authors__name">Zhou R. </li><li class="ref__authors__name">Ren Z. </li><li class="ref__authors__name">Feng Q. </li><li>et al.</li></ul><div class="ref__title">IPeak: An open source tool to combine results from multiple MS/MS search engines.</div><span class="ref__series"><i>Proteomics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">2916-2920</span><div class="ref__controls"><a href="#back-bib73" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84940580015&amp;cf=" onclick="newWindow(this.href);return false">Scopus (31)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25951428&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1002%2Fpmic.201400208&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=2916-2920&journal=Proteomics&author=Wen+B.&author=Du+C.&author=Li+G.&author=Ghali+F.&author=Jones+A.R.&author=K%C3%A4ll+L.&author=Xu+S.&author=Zhou+R.&author=Ren+Z.&author=Feng+Q.&title=IPeak%3A+An+open+source+tool+to+combine+results+from+multiple+MS%2FMS+search+engines">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Mei Z. </li><li class="ref__authors__name">Zeng C. </li><li class="ref__authors__name">Liu S. </li></ul><div class="ref__title">metaX: a flexible and comprehensive software for processing metabolomics data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">183</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85015943888&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=28327092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1186%2Fs12859-017-1579-y&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=183&journal=BMC+Bioinformatics&author=Wen+B.&author=Mei+Z.&author=Zeng+C.&author=Liu+S.&title=metaX%3A+a+flexible+and+comprehensive+software+for+processing+metabolomics+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">PepQuery enables fast, accurate, and convenient proteomic validation of novel genomic alterations.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">29</span></span>: <span class="ref__seriesPages">485-493</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-85062586361&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=30610011&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1101%2Fgr.235028.118&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=485-493&journal=Genome+Res.&author=Wen+B.&author=Wang+X.&author=Zhang+B.&title=PepQuery+enables+fast%2C+accurate%2C+and+convenient+proteomic+validation+of+novel+genomic+alterations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoshihara K. </li><li class="ref__authors__name">Shahmoradgoli M. </li><li class="ref__authors__name">Martínez E. </li><li class="ref__authors__name">Vegesna R. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Torres-Garcia W. </li><li class="ref__authors__name">Treviño V. </li><li class="ref__authors__name">Shen H. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Levine D.A. </li><li>et al.</li></ul><div class="ref__title">Inferring tumour purity and stromal and immune cell admixture from expression data.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">2612</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84885673911&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5182)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=24113773&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fncomms3612&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2612&journal=Nat.+Commun.&author=Yoshihara+K.&author=Shahmoradgoli+M.&author=Mart%C3%ADnez+E.&author=Vegesna+R.&author=Kim+H.&author=Torres-Garcia+W.&author=Trevi%C3%B1o+V.&author=Shen+H.&author=Laird+P.W.&author=Levine+D.A.&title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Zhu J. </li><li class="ref__authors__name">Liu Q. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Zimmerman L.J. </li><li class="ref__authors__name">Shaddox K.F. </li><li class="ref__authors__name">Kim S. </li><li>et al.</li><li>NCI CPTAC</li></ul><div class="ref__title">Proteogenomic characterization of human colon and rectal cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">513</span></span>: <span class="ref__seriesPages">382-387</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84907269780&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1052)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=25043054&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1038%2Fnature13438&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=382-387&journal=Nature&author=NCI+CPTAC&title=Proteogenomic+characterization+of+human+colon+and+rectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang H. </li><li class="ref__authors__name">Liu T. </li><li class="ref__authors__name">Zhang Z. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">McDermott J.E. </li><li class="ref__authors__name">Zhou J.Y. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Chen L. </li><li class="ref__authors__name">Ray D. </li><li>et al.</li><li>CPTAC Investigators</li></ul><div class="ref__title">Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">166</span></span>: <span class="ref__seriesPages">755-765</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.cell.2019.03.030&amp;key=2-s2.0-84977611330&amp;cf=" onclick="newWindow(this.href);return false">Scopus (674)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.cell.2019.03.030&amp;key=27372738&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.cell.2019.03.030&amp;key=10.1016%2Fj.cell.2016.05.069&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=755-765&journal=Cell&author=CPTAC+Investigators&title=Integrated+Proteogenomic+Characterization+of+Human+High-Grade+Serous+Ovarian+Cancer">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: April 25, 2019</div><div class="article-info__date">Accepted:
            March 12,
            2019</div><div class="article-info__date">Received in revised form:
            November 22,
            2018</div><div class="article-info__date">Received:
            July 28,
            2018</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.cell.2019.03.030">https://doi.org/10.1016/j.cell.2019.03.030</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2019 Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S0092867419302922">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867419302922" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)30292-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419302922%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)30292-2?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867419302922%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1723231508175","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"177","issueNumber":"4","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867419302922","coverDate":"201905","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
