<!DOCTYPE html>
<html lang="en"     class="pb-page"  data-request-id="08919bb0-efc7-4d24-a10d-8b3464653218"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;requestedJournal:journal:ccell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S1535610821000507;pageGroup:string:Article View;page:string:Show Full Text;wgroup:string:Default Website Group;subPage:string:Full Text;journal:journal:ccell;issue:issue:pii\:S1535610820X00055"/>




<!-- if there is any customization for Responsive Project widget  -->




























        
                <meta name="robots" content="noarchive"/>
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1"/>











        <meta property="og:title" content="Proteogenomic and metabolomic characterization of human glioblastoma" />
    


        <meta property="og:type" content="Article" />
    


        <meta property="og:url" content="https://www.cell.com/cancer-cell/abstract/S1535-6108(21)00050-7" />
    



        <meta property="og:site_name" content="Cancer Cell" />
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg" />
    


        <meta property="og:description" content="Wang et&nbsp;al. perform integrated proteogenomic analysis of adult glioblastoma (GBM),
including metabolomics, lipidomics, and single nuclei RNA-Seq, revealing insights
into the immune landscape of GBM, cell-specific nature of EMT signatures, histone
acetylation in classical GBM, and the existence of signaling hubs which could provide
therapeutic vulnerabilities." />
    











        <meta name="twitter:title" content="Proteogenomic and metabolomic characterization of human glioblastoma" />
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg" />
    


        <meta name="twitter:description" content="Wang et&nbsp;al. perform integrated proteogenomic analysis of adult glioblastoma (GBM),
including metabolomics, lipidomics, and single nuclei RNA-Seq, revealing insights
into the immune landscape of GBM, cell-specific nature of EMT signatures, histone
acetylation in classical GBM, and the existence of signaling hubs which could provide
therapeutic vulnerabilities." />
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00050-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610821000507%3Fshowall%3Dtrue">
    


<title>Proteogenomic and metabolomic characterization of human glioblastoma: Cancer Cell</title><meta name="citation_journal_title" content="Cancer Cell"/> <meta name="citation_journal_abbrev" content="Cancer Cell"/> <meta name="citation_publisher" content="Elsevier"/> <meta name="citation_title" content="Proteogenomic and metabolomic characterization of human glioblastoma"/> <meta name="citation_date" content="2021/04/12"/> <meta name="citation_online_date" content="2021/02/11"/> <meta name="citation_volume" content="39"/> <meta name="citation_issue" content="4"/> <meta name="citation_firstpage" content="509"/> <meta name="citation_lastpage" content="528.e20"/> <meta name="citation_pmid" content="33577785"/> <meta name="citation_abstract_html_url" content="http://www.cell.com/article/S1535610821000507/abstract"/> <meta name="citation_fulltext_html_url" content="http://www.cell.com/article/S1535610821000507/fulltext"/> <meta name="citation_pdf_url" content="http://www.cell.com/article/S1535610821000507/pdf"/> <meta name="citation_issn" content="1535-6108"/> <meta name="citation_issn" content="1878-3686"/> <meta name="citation_language" content="English"/> <meta name="citation_doi" content="10.1016/j.ccell.2021.01.006"/> <meta name="citation_pmid" content="33577785"/> <meta name="citation_author" content="Liang-Bo Wang"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Alla Karpova"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Marina A. Gritsenko"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Jennifer E. Kyle"></meta><meta name="citation_author_orcid" content="0000-0002-9282-9005"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Song Cao"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Yize Li"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Dmitry Rykunov"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Antonio Colaprico"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami, FL 33136, USA"></meta><meta name="citation_author" content="Joseph H. Rothstein"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Runyu Hong"></meta><meta name="citation_author_orcid" content="0000-0002-0941-351X"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Vasileios Stathias"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="BD2K-LINCS Data Coordination and Integration Center, Miami, FL 33136, USA"></meta><meta name="citation_author" content="MacIntosh Cornwell"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Francesca Petralia"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Yige Wu"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Boris Reva"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Karsten Krug"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Pietro Pugliese"></meta><meta name="citation_author_institution" content="Department of Science and Technology, University of Sannio, 82100, Benevento, Italy"></meta><meta name="citation_author" content="Emily Kawaler"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Lindsey K. Olsen"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Wen-Wei Liang"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Xiaoyu Song"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Yongchao Dou"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="Michael C. Wendl"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Department of Genetics, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Department of Mathematics, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Wagma Caravan"></meta><meta name="citation_author_orcid" content="0000-0003-4082-6723"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Wenke Liu"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Daniel Cui Zhou"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Jiayi Ji"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Chia-Feng Tsai"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Vladislav A. Petyuk"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Jamie Moon"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Weiping Ma"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Rosalie K. Chu"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Karl K. Weitz"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Ronald J. Moore"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Matthew E. Monroe"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Rui Zhao"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Xiaolu Yang"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland"></meta><meta name="citation_author" content="Seungyeul Yoo"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Azra Krek"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Alexis Demopoulos"></meta><meta name="citation_author_institution" content="Department of Neurology, Northwell Health System, Lake Success, NY 11042 USA"></meta><meta name="citation_author" content="Houxiang Zhu"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Matthew A. Wyczalkowski"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Joshua F. McMichael"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Brittany L. Henderson"></meta><meta name="citation_author_orcid" content="0000-0002-3449-0058"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Caleb M. Lindgren"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Hannah Boekweg"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Shuangjia Lu"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Jessika Baral"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Lijun Yao"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Kelly G. Stratton"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Lisa M. Bramer"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Erika Zink"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Sneha P. Couvillion"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Kent J. Bloodsworth"></meta><meta name="citation_author_orcid" content="0000-0002-7596-1565"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Shankha Satpathy"></meta><meta name="citation_author_orcid" content="0000-0002-6040-4854"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Weiva Sieh"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author_institution" content="Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Simina M. Boca"></meta><meta name="citation_author_institution" content="Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC 20007, USA"></meta><meta name="citation_author" content="Stephan Schürer"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="BD2K-LINCS Data Coordination and Integration Center, Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Institute for Data Science & Computing, University of Miami, FL 33136, USA"></meta><meta name="citation_author" content="Feng Chen"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Department of Cell Biology and Physiology, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Maciej Wiznerowicz"></meta><meta name="citation_author_institution" content="International Institute for Molecular Oncology, 60-203 Poznań, Poland"></meta><meta name="citation_author_institution" content="Poznań University of Medical Sciences, 61-701 Poznań, Poland"></meta><meta name="citation_author" content="Karen A. Ketchum"></meta><meta name="citation_author_institution" content="ESAC, Inc., Rockville, MD 20850, USA"></meta><meta name="citation_author" content="Emily S. Boja"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Christopher R. Kinsinger"></meta><meta name="citation_author_orcid" content="0000-0001-5015-4428"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Ana I. Robles"></meta><meta name="citation_author_orcid" content="0000-0001-5019-4374"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Tara Hiltke"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mathangi Thiagarajan"></meta><meta name="citation_author_institution" content="Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA"></meta><meta name="citation_author" content="Alexey I. Nesvizhskii"></meta><meta name="citation_author_institution" content="Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author_institution" content="Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA"></meta><meta name="citation_author" content="Bing Zhang"></meta><meta name="citation_author_institution" content="Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA"></meta><meta name="citation_author" content="D.R. Mani"></meta><meta name="citation_author_institution" content="Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA"></meta><meta name="citation_author" content="Michele Ceccarelli"></meta><meta name="citation_author_institution" content='Department of Electrical Engineering and Information Technology, University of Naples &#34;Federico II&#34;, 80128, Naples, Italy'></meta><meta name="citation_author_institution" content="BIOGEM, 83031 Ariano Irpino, Italy"></meta><meta name="citation_author" content="Xi S. Chen"></meta><meta name="citation_author_institution" content="Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA"></meta><meta name="citation_author_institution" content="Division of Biostatistics, Department of Public Health Science, University of Miami, FL 33136, USA"></meta><meta name="citation_author" content="Sandra L. Cottingham"></meta><meta name="citation_author_institution" content="Department of Pathology, Spectrum Health and Helen DeVos Children's Hospital, Grand Rapids, MI 49503, USA"></meta><meta name="citation_author" content="Qing Kay Li"></meta><meta name="citation_author_institution" content="Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA"></meta><meta name="citation_author" content="Albert H. Kim"></meta><meta name="citation_author_institution" content="Department of Neurological Surgery, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="David Fenyö"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Kelly V. Ruggles"></meta><meta name="citation_author_orcid" content="0000-0002-0152-0863"></meta><meta name="citation_author_institution" content="Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author_institution" content="Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA"></meta><meta name="citation_author" content="Henry Rodriguez"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Mehdi Mesri"></meta><meta name="citation_author_institution" content="Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA"></meta><meta name="citation_author" content="Samuel H. Payne"></meta><meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, UT 84602, USA"></meta><meta name="citation_author" content="Adam C. Resnick"></meta><meta name="citation_author_institution" content="Center for Data Driven Discovery in Biomedicine, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA"></meta><meta name="citation_author_institution" content="Division of Neurosurgery, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA"></meta><meta name="citation_author" content="Pei Wang"></meta><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"></meta><meta name="citation_author" content="Richard D. Smith"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Antonio Iavarone"></meta><meta name="citation_author_institution" content="Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10032, USA"></meta><meta name="citation_author_institution" content="Department of Neurology, Columbia University Medical Center, New York, NY 10032, USA"></meta><meta name="citation_author_institution" content="Department of Pathology and Cell Biology, Columbia University Medical Center, New York, NY 10032, USA"></meta><meta name="citation_author_institution" content="Herbert Irving Comprehensive Cancer Center, Columbia University Medical Center, New York, NY 10032, USA"></meta><meta name="citation_author" content="Milan G. Chheda"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Department of Neurology, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Jill S. Barnholtz-Sloan"></meta><meta name="citation_author_institution" content="Case Comprehensive Cancer Center and Department of Population and Quantitative Health Sciences, Case Western Reserve University School of Medicine, Cleveland, OH 44106, USA"></meta><meta name="citation_author_institution" content="Research and Education, University Hospitals Health System, Cleveland, OH 44106, USA"></meta><meta name="citation_author" content="Karin D. Rodland"></meta><meta name="citation_author_email" content="karin.rodland@pnnl.gov"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author_institution" content="Department of Cell, Developmental, and Cancer Biology, Oregon Health & Science University, Portland, OR 97221, USA"></meta><meta name="citation_author" content="Tao Liu"></meta><meta name="citation_author_email" content="tao.liu@pnnl.gov"></meta><meta name="citation_author_institution" content="Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA"></meta><meta name="citation_author" content="Li Ding"></meta><meta name="citation_author_email" content="lding@wustl.edu"></meta><meta name="citation_author_orcid" content="0000-0003-1517-2975"></meta><meta name="citation_author_institution" content="Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Department of Genetics, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author_institution" content="Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA"></meta><meta name="citation_author" content="Anupriya Agarwal"></meta><meta name="citation_author" content="Mitual Amin"></meta><meta name="citation_author" content="Eunkyung An"></meta><meta name="citation_author" content="Matthew L. Anderson"></meta><meta name="citation_author" content="David W. Andrews"></meta><meta name="citation_author" content="Thomas Bauer"></meta><meta name="citation_author" content="Chet Birger"></meta><meta name="citation_author" content="Michael J. Birrer"></meta><meta name="citation_author" content="Lili Blumenberg"></meta><meta name="citation_author" content="William E. Bocik"></meta><meta name="citation_author" content="Uma Borate"></meta><meta name="citation_author" content="Melissa Borucki"></meta><meta name="citation_author" content="Meghan C. Burke"></meta><meta name="citation_author" content="Shuang Cai"></meta><meta name="citation_author" content="Anna P. Calinawan"></meta><meta name="citation_author" content="Steven A. Carr"></meta><meta name="citation_author" content="Sandra Cerda"></meta><meta name="citation_author" content="Daniel W. Chan"></meta><meta name="citation_author" content="Alyssa Charamut"></meta><meta name="citation_author" content="Lin S. Chen"></meta><meta name="citation_author" content="David Chesla"></meta><meta name="citation_author" content="Arul M. Chinnaiyan"></meta><meta name="citation_author" content="Shrabanti Chowdhury"></meta><meta name="citation_author" content="Marcin P. Cieślik"></meta><meta name="citation_author" content="David J. Clark"></meta><meta name="citation_author" content="Houston Culpepper"></meta><meta name="citation_author" content="Tomasz Czernicki"></meta><meta name="citation_author" content="Fulvio D'Angelo"></meta><meta name="citation_author" content="Jacob Day"></meta><meta name="citation_author" content="Stephanie De Young"></meta><meta name="citation_author" content="Emek Demir"></meta><meta name="citation_author" content="Saravana Mohan Dhanasekaran"></meta><meta name="citation_author" content="Rajiv Dhir"></meta><meta name="citation_author" content="Marcin J. Domagalski"></meta><meta name="citation_author" content="Brian Druker"></meta><meta name="citation_author" content="Elizabeth Duffy"></meta><meta name="citation_author" content="Maureen Dyer"></meta><meta name="citation_author" content="Nathan J. Edwards"></meta><meta name="citation_author" content="Robert Edwards"></meta><meta name="citation_author" content="Kimberly Elburn"></meta><meta name="citation_author" content="Matthew J. Ellis"></meta><meta name="citation_author" content="Jennifer Eschbacher"></meta><meta name="citation_author" content="Alicia Francis"></meta><meta name="citation_author" content="Stacey Gabriel"></meta><meta name="citation_author" content="Nikolay Gabrovski"></meta><meta name="citation_author" content="Luciano Garofano"></meta><meta name="citation_author" content="Gad Getz"></meta><meta name="citation_author" content="Michael A. Gillette"></meta><meta name="citation_author" content="Andrew K. Godwin"></meta><meta name="citation_author" content="Denis Golbin"></meta><meta name="citation_author" content="Ziad Hanhan"></meta><meta name="citation_author" content="Linda I. Hannick"></meta><meta name="citation_author" content="Pushpa Hariharan"></meta><meta name="citation_author" content="Barbara Hindenach"></meta><meta name="citation_author" content="Katherine A. Hoadley"></meta><meta name="citation_author" content="Galen Hostetter"></meta><meta name="citation_author" content="Chen Huang"></meta><meta name="citation_author" content="Eric Jaehnig"></meta><meta name="citation_author" content="Scott D. Jewell"></meta><meta name="citation_author" content="Nan Ji"></meta><meta name="citation_author" content="Corbin D. Jones"></meta><meta name="citation_author" content="Alcida Karz"></meta><meta name="citation_author" content="Wojciech Kaspera"></meta><meta name="citation_author" content="Lyndon Kim"></meta><meta name="citation_author" content="Ramani B. Kothadia"></meta><meta name="citation_author" content="Chandan Kumar-Sinha"></meta><meta name="citation_author" content="Jonathan Lei"></meta><meta name="citation_author" content="Felipe D. Leprevost"></meta><meta name="citation_author" content="Kai Li"></meta><meta name="citation_author" content="Yuxing Liao"></meta><meta name="citation_author" content="Jena Lilly"></meta><meta name="citation_author" content="Hongwei Liu"></meta><meta name="citation_author" content="Jan Lubínski"></meta><meta name="citation_author" content="Rashna Madan"></meta><meta name="citation_author" content="William Maggio"></meta><meta name="citation_author" content="Ewa Malc"></meta><meta name="citation_author" content="Anna Malovannaya"></meta><meta name="citation_author" content="Sailaja Mareedu"></meta><meta name="citation_author" content="Sanford P. Markey"></meta><meta name="citation_author" content="Annette Marrero-Oliveras"></meta><meta name="citation_author" content="Nina Martinez"></meta><meta name="citation_author" content="Nicollette Maunganidze"></meta><meta name="citation_author" content="Jason E. McDermott"></meta><meta name="citation_author" content="Peter B. McGarvey"></meta><meta name="citation_author" content="John McGee"></meta><meta name="citation_author" content="Piotr Mieczkowski"></meta><meta name="citation_author" content="Simona Migliozzi"></meta><meta name="citation_author" content="Francesmary Modugno"></meta><meta name="citation_author" content="Rebecca Montgomery"></meta><meta name="citation_author" content="Chelsea J. Newton"></meta><meta name="citation_author" content="Gilbert S. Omenn"></meta><meta name="citation_author" content="Umut Ozbek"></meta><meta name="citation_author" content="Oxana V. Paklina"></meta><meta name="citation_author" content="Amanda G. Paulovich"></meta><meta name="citation_author" content="Amy M. Perou"></meta><meta name="citation_author" content="Alexander R. Pico"></meta><meta name="citation_author" content="Paul D. Piehowski"></meta><meta name="citation_author" content="Dimitris G. Placantonakis"></meta><meta name="citation_author" content="Larisa Polonskaya"></meta><meta name="citation_author" content="Olga Potapova"></meta><meta name="citation_author" content="Barbara Pruetz"></meta><meta name="citation_author" content="Liqun Qi"></meta><meta name="citation_author" content="Shakti Ramkissoon"></meta><meta name="citation_author" content="Adam Resnick"></meta><meta name="citation_author" content="Shannon Richey"></meta><meta name="citation_author" content="Gregory Riggins"></meta><meta name="citation_author" content="Karna Robinson"></meta><meta name="citation_author" content="Nancy Roche"></meta><meta name="citation_author" content="Daniel C. Rohrer"></meta><meta name="citation_author" content="Brian R. Rood"></meta><meta name="citation_author" content="Larissa Rossell"></meta><meta name="citation_author" content="Sara R. Savage"></meta><meta name="citation_author" content="Eric E. Schadt"></meta><meta name="citation_author" content="Yan Shi"></meta><meta name="citation_author" content="Zhiao Shi"></meta><meta name="citation_author" content="Yvonne Shutack"></meta><meta name="citation_author" content="Shilpi Singh"></meta><meta name="citation_author" content="Tara Skelly"></meta><meta name="citation_author" content="Lori J. Sokoll"></meta><meta name="citation_author" content="Jakub Stawicki"></meta><meta name="citation_author" content="Stephen E. Stein"></meta><meta name="citation_author" content="James Suh"></meta><meta name="citation_author" content="Wojciech Szopa"></meta><meta name="citation_author" content="Dave Tabor"></meta><meta name="citation_author" content="Donghui Tan"></meta><meta name="citation_author" content="Darlene Tansil"></meta><meta name="citation_author" content="Ratna R. Thangudu"></meta><meta name="citation_author" content="Cristina Tognon"></meta><meta name="citation_author" content="Elie Traer"></meta><meta name="citation_author" content="Shirley Tsang"></meta><meta name="citation_author" content="Jeffrey Tyner"></meta><meta name="citation_author" content="Ki Sung Um"></meta><meta name="citation_author" content="Dana R. Valley"></meta><meta name="citation_author" content="Suhas Vasaikar"></meta><meta name="citation_author" content="Negin Vatanian"></meta><meta name="citation_author" content="Uma Velvulou"></meta><meta name="citation_author" content="Michael Vernon"></meta><meta name="citation_author" content="Weiqing Wan"></meta><meta name="citation_author" content="Junmei Wang"></meta><meta name="citation_author" content="Alex Webster"></meta><meta name="citation_author" content="Bo Wen"></meta><meta name="citation_author" content="Jeffrey R. Whiteaker"></meta><meta name="citation_author" content="George D. Wilson"></meta><meta name="citation_author" content="Yuriy Zakhartsev"></meta><meta name="citation_author" content="Robert Zelt"></meta><meta name="citation_author" content="Hui Zhang"></meta><meta name="citation_author" content="Liwei Zhang"></meta><meta name="citation_author" content="Zhen Zhang"></meta><meta name="citation_author" content="Grace Zhao"></meta><meta name="citation_author" content="Jun Zhu"></meta><meta name="citation_keywords" content="glioblastoma; proteomics; proteogenomics; signaling; CPTAC; metabolome; lipidome; acetylome; single nuclei RNA-seq"></meta><meta name="citation_reference" content="citation_title=A map of human genome variation from population-scale sequencing; citation_author=G.R. Abecasis; citation_author=D. Altshuler; citation_author=A. Auton; citation_author=L.D. Brooks; citation_author=R.M. Durbin; citation_author=R.A. Gibbs; citation_author=M.E. Hurles; citation_author=G.A. McVean; citation_author=1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=467; citation_firstpage=1061; citation_lastpage=1073; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=HTSeq—a Python framework to work with high-throughput sequencing data; citation_author=S. Anders; citation_author=P.T. Pyl; citation_author=W. Huber; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=166; citation_lastpage=169; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=xCell: digitally portraying the tissue cellular heterogeneity landscape; citation_author=D. Aran; citation_author=Z. Hu; citation_author=A.J. Butte; citation_journal_title=Genome Biol.; citation_volume=18; citation_firstpage=220; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Arg1 expression defines immunosuppressive subsets of tumor-associated macrophages; citation_author=S.P. Arlauckas; citation_author=S.B. Garren; citation_author=C.S. Garris; citation_author=R.H. Kohler; citation_author=J. Oh; citation_author=M.J. Pittet; citation_author=R. Weissleder; citation_journal_title=Theranostics; citation_volume=8; citation_firstpage=5842; citation_lastpage=5854; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Recurrent chimeric fusion RNAs in non-cancer tissues and cells; citation_author=M. Babiceanu; citation_author=F. Qin; citation_author=Z. Xie; citation_author=Y. Jia; citation_author=K. Lopez; citation_author=N. Janus; citation_author=L. Facemire; citation_author=S. Kumar; citation_author=Y. Pang; citation_author=Y. Qi; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=2859; citation_lastpage=2872; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Causal interactions from proteomic profiles: molecular data meets pathway knowledge; citation_author=Ö. Babur; citation_author=A. Luna; citation_author=A. Korkut; citation_author=F. Durupinar; citation_author=M.C. Siper; citation_author=U. Dogrusoz; citation_author=J.E. Aslan; citation_author=C. Sander; citation_author=E. Demir; citation_journal_title=BioRxiv; citation_firstpage=258855; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=MGMT methylation analysis of glioblastoma on the Infinium methylation BeadChip identifies two distinct CpG regions associated with gene silencing and outcome, yielding a prediction model for comparisons across datasets, tumor grades, and CIMP-status; citation_author=P. Bady; citation_author=D. Sciuscio; citation_author=A.-C. Diserens; citation_author=J. Bloch; citation_author=M.J. van den Bent; citation_author=C. Marosi; citation_author=P.-Y. Dietrich; citation_author=M. Weller; citation_author=L. Mariani; citation_author=F.L. Heppner; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=124; citation_firstpage=547; citation_lastpage=560; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Comprehensive characterization of cancer driver genes and mutations; citation_author=M.H. Bailey; citation_author=C. Tokheim; citation_author=E. Porta-Pardo; citation_author=S. Sengupta; citation_author=D. Bertrand; citation_author=A. Weerasinghe; citation_author=A. Colaprico; citation_author=M.C. Wendl; citation_author=J. Kim; citation_author=B. Reardon; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=371; citation_lastpage=385.e18; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1; citation_author=D.A. Barbie; citation_author=P. Tamayo; citation_author=J.S. Boehm; citation_author=S.Y. Kim; citation_author=S.E. Moody; citation_author=I.F. Dunn; citation_author=A.C. Schinzel; citation_author=P. Sandy; citation_author=E. Meylan; citation_author=C. Scholl; citation_author=et al.; citation_journal_title=Nature; citation_volume=462; citation_firstpage=108; citation_lastpage=112; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=A probability-based approach for high-throughput protein phosphorylation analysis and site localization; citation_author=S.A. Beausoleil; citation_author=J. Villén; citation_author=S.A. Gerber; citation_author=J. Rush; citation_author=S.P. Gygi; citation_journal_title=Nat. Biotechnol.; citation_volume=24; citation_firstpage=1285; citation_lastpage=1292; citation_publication_date=2006; "></meta><meta name="citation_reference" content="citation_title=The landscape of the mesenchymal signature in brain tumours; citation_author=J. Behnan; citation_author=G. Finocchiaro; citation_author=G. Hanna; citation_journal_title=Brain J. Neurol.; citation_volume=142; citation_firstpage=847; citation_lastpage=866; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=SUMO1 modification stabilizes CDK6 protein and drives the cell cycle and glioblastoma progression; citation_author=A.C. Bellail; citation_author=J.J. Olson; citation_author=C. Hao; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=4234; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript; citation_author=M. Benelli; citation_author=C. Pescucci; citation_author=G. Marseglia; citation_author=M. Severgnini; citation_author=F. Torricelli; citation_author=A. Magi; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=3232; citation_lastpage=3239; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Potential role of dietary lipids in the prophylaxis of some clinical conditions; citation_author=U. Bhagat; citation_author=U.N. Das; citation_journal_title=Arch. Med. Sci.; citation_volume=11; citation_firstpage=807; citation_lastpage=818; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Fast unfolding of communities in large networks; citation_author=V.D. Blondel; citation_author=J.-L. Guillaume; citation_author=R. Lambiotte; citation_author=E. Lefebvre; citation_journal_title=J. Stat. Mech. Theor. Exp.; citation_volume=2008; citation_firstpage=P10008; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=BlackSheep: a bioconductor and bioconda package for differential extreme value analysis (Bioinformatics); citation_author=L. Blumenberg; citation_author=E. Kawaler; citation_author=M. Cornwell; citation_author=S. Smith; citation_author=K. Ruggles; citation_author=D. Fenyö; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=cIMPACT-NOW update 3: recommended diagnostic criteria for “Diffuse astrocytic glioma, IDH-wildtype, with molecular features of glioblastoma, WHO grade IV; citation_author=D.J. Brat; citation_author=K. Aldape; citation_author=H. Colman; citation_author=E.C. Holland; citation_author=D.N. Louis; citation_author=R.B. Jenkins; citation_author=B.K. Kleinschmidt-DeMasters; citation_author=A. Perry; citation_author=G. Reifenberger; citation_author=R. Stupp; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=136; citation_firstpage=805; citation_lastpage=810; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The somatic genomic landscape of glioblastoma; citation_author=C.W. Brennan; citation_author=R.G.W. Verhaak; citation_author=A. McKenna; citation_author=B. Campos; citation_author=H. Noushmehr; citation_author=S.R. Salama; citation_author=S. Zheng; citation_author=D. Chakravarty; citation_author=J.Z. Sanborn; citation_author=S.H. Berman; citation_author=et al.; citation_journal_title=Cell; citation_volume=155; citation_firstpage=462; citation_lastpage=477; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Metagenes and molecular pattern discovery using matrix factorization; citation_author=J.-P. Brunet; citation_author=P. Tamayo; citation_author=T.R. Golub; citation_author=J.P. Mesirov; citation_journal_title=Proc. Natl. Acad. Sci. U S A; citation_volume=101; citation_firstpage=4164; citation_lastpage=4169; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Integrating single-cell transcriptomic data across different conditions, technologies, and species; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=P. Smibert; citation_author=E. Papalexi; citation_author=Rahul Satija; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=411; citation_lastpage=420; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Orally administered colony stimulating factor 1 receptor inhibitor PLX3397 in recurrent glioblastoma: an Ivy Foundation Early Phase Clinical Trials Consortium phase II study; citation_author=N. Butowski; citation_author=H. Colman; citation_author=J.F. De Groot; citation_author=A.M. Omuro; citation_author=L. Nayak; citation_author=P.Y. Wen; citation_author=T.F. Cloughesy; citation_author=A. Marimuthu; citation_author=S. Haidar; citation_author=A. Perry; citation_author=et al.; citation_journal_title=Neuro-Oncol.; citation_volume=18; citation_firstpage=557; citation_lastpage=564; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=DNA methylation-based classification of central nervous system tumours; citation_author=D. Capper; citation_author=D.T.W. Jones; citation_author=M. Sill; citation_author=V. Hovestadt; citation_author=D. Schrimpf; citation_author=D. Sturm; citation_author=C. Koelsche; citation_author=F. Sahm; citation_author=L. Chavez; citation_author=D.E. Reuss; citation_author=et al.; citation_journal_title=Nature; citation_volume=555; citation_firstpage=469; citation_lastpage=474; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Immune microenvironment in glioblastoma subtypes; citation_author=Z. Chen; citation_author=D. Hambardzumyan; citation_journal_title=Front. Immunol.; citation_volume=9; citation_firstpage=1004; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications; citation_author=X. Chen; citation_author=O. Schulz-Trieglaff; citation_author=R. Shaw; citation_author=B. Barnes; citation_author=F. Schlesinger; citation_author=M. Källberg; citation_author=A.J. Cox; citation_author=S. Kruglyak; citation_author=C.T. Saunders; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=1220; citation_lastpage=1222; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Large-scale profiling of microRNAs for the cancer genome atlas; citation_author=A. Chu; citation_author=G. Robertson; citation_author=D. Brooks; citation_author=A.J. Mungall; citation_author=I. Birol; citation_author=R. Coope; citation_author=Y. Ma; citation_author=S. Jones; citation_author=M.A. Marra; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e3; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples; citation_author=K. Cibulskis; citation_author=M.S. Lawrence; citation_author=S.L. Carter; citation_author=A. Sivachenko; citation_author=D. Jaffe; citation_author=C. Sougnez; citation_author=S. Gabriel; citation_author=M. Meyerson; citation_author=E.S. Lander; citation_author=G. Getz; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=213; citation_lastpage=219; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Integrated proteogenomic characterization of clear cell renal cell carcinoma; citation_author=D.J. Clark; citation_author=S.M. Dhanasekaran; citation_author=F. Petralia; citation_author=J. Pan; citation_author=X. Song; citation_author=Y. Hu; citation_author=F. da Veiga Leprevost; citation_author=B. Reva; citation_author=T.-S.M. Lih; citation_author=H.-Y. Chang; citation_author=et al.; citation_journal_title=Cell; citation_volume=179; citation_firstpage=964; citation_lastpage=983.e31; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data; citation_author=A. Colaprico; citation_author=T.C. Silva; citation_author=C. Olsen; citation_author=L. Garofano; citation_author=C. Cava; citation_author=D. Garolini; citation_author=T.S. Sabedot; citation_author=T.M. Malta; citation_author=S.M. Pagnotta; citation_author=I. Castiglioni; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e71; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Interpreting pathways to discover cancer driver genes with Moonlight; citation_author=A. Colaprico; citation_author=C. Olsen; citation_author=M.H. Bailey; citation_author=G.J. Odom; citation_author=T. Terkelsen; citation_author=T.C. Silva; citation_author=A.V. Olsen; citation_author=L. Cantini; citation_author=A. Zinovyev; citation_author=E. Barillot; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=69; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=DGIdb 3.0: a redesign and expansion of the drug-gene interaction database; citation_author=K.C. Cotto; citation_author=A.H. Wagner; citation_author=Y.-Y. Feng; citation_author=S. Kiwala; citation_author=A.C. Coffman; citation_author=G. Spies; citation_author=A. Wollam; citation_author=N.C. Spies; citation_author=O.L. Griffith; citation_author=M. Griffith; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_firstpage=D1068; citation_lastpage=D1073; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives; citation_author=K. Daily; citation_author=S.J. Ho Sui; citation_author=L.M. Schriml; citation_author=P.J. Dexheimer; citation_author=N. Salomonis; citation_author=R. Schroll; citation_author=S. Bush; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=S. Lotia; citation_author=et al.; citation_journal_title=Sci. Data; citation_volume=4; citation_firstpage=170030; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Survival in glioblastoma: a review on the impact of treatment modalities; citation_author=P.D. Delgado-López; citation_author=E.M. Corrales-García; citation_journal_title=Clin. Transl. Oncol.; citation_volume=18; citation_firstpage=1062; citation_lastpage=1071; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Estimating telomere length from whole genome sequence data; citation_author=Z. Ding; citation_author=M. Mangino; citation_author=A. Aviv; citation_author=T. Spector; citation_author=R. Durbin; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=e75; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=ACSL4 dictates ferroptosis sensitivity by shaping cellular lipid composition; citation_author=S. Doll; citation_author=B. Proneth; citation_author=Y.Y. Tyurina; citation_author=E. Panzilius; citation_author=S. Kobayashi; citation_author=I. Ingold; citation_author=M. Irmler; citation_author=J. Beckers; citation_author=M. Aichler; citation_author=A. Walch; citation_author=et al.; citation_journal_title=Nat. Chem. Biol.; citation_volume=13; citation_firstpage=91; citation_lastpage=98; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic characterization of endometrial carcinoma; citation_author=Y. Dou; citation_author=E.A. Kawaler; citation_author=D.C. Zhou; citation_author=M.A. Gritsenko; citation_author=C. Huang; citation_author=L. Blumenberg; citation_author=A. Karpova; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=S. Satpathy; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_firstpage=729; citation_lastpage=748.e26; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=The human DEPhOsphorylation database DEPOD: a 2015 update; citation_author=G. Duan; citation_author=X. Li; citation_author=M. Köhn; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_firstpage=D531; citation_lastpage=D535; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Role for Akt3/protein kinase Bgamma in attainment of normal brain size; citation_author=R.M. Easton; citation_author=H. Cho; citation_author=K. Roovers; citation_author=D.W. Shineman; citation_author=M. Mizrahi; citation_author=M.S. Forman; citation_author=V.M.-Y. Lee; citation_author=M. Szabolcs; citation_author=R. de Jong; citation_author=T. Oltersdorf; citation_author=et al.; citation_journal_title=Mol. Cell. Biol.; citation_volume=25; citation_firstpage=1869; citation_lastpage=1878; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Histone acetylation: a switch between repressive and permissive chromatin. Second in review series on chromatin dynamics; citation_author=A. Eberharter; citation_author=P.B. Becker; citation_journal_title=EMBO Rep.; citation_volume=3; citation_firstpage=224; citation_lastpage=229; citation_publication_date=2002; "></meta><meta name="citation_reference" content="citation_title=Role of histone deacetylase 2 and its posttranslational modifications in cardiac hypertrophy; citation_author=G.H. Eom; citation_author=H. Kook; citation_journal_title=BMB Rep.; citation_volume=48; citation_firstpage=131; citation_lastpage=138; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=The reactome pathway Knowledgebase; citation_author=A. Fabregat; citation_author=S. Jupe; citation_author=L. Matthews; citation_author=K. Sidiropoulos; citation_author=M. Gillespie; citation_author=P. Garapati; citation_author=R. Haw; citation_author=B. Jassal; citation_author=F. Korninger; citation_author=B. May; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_firstpage=D649; citation_lastpage=D655; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries; citation_author=S. Fisher; citation_author=A. Barry; citation_author=J. Abreu; citation_author=B. Minie; citation_author=J. Nolan; citation_author=T.M. Delorey; citation_author=G. Young; citation_author=T.J. Fennell; citation_author=A. Allen; citation_author=L. Ambrogio; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R1; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=GENCODE reference annotation for the human and mouse genomes; citation_author=A. Frankish; citation_author=M. Diekhans; citation_author=A.-M. Ferreira; citation_author=R. Johnson; citation_author=I. Jungreis; citation_author=J. Loveland; citation_author=J.M. Mudge; citation_author=C. Sisu; citation_author=J. Wright; citation_author=J. Armstrong; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_firstpage=D766; citation_lastpage=D773; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=CIRI: an efficient and unbiased algorithm for de novo circular RNA identification; citation_author=Y. Gao; citation_author=J. Wang; citation_author=F. Zhao; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=4; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Driver fusions and their implications in the development and treatment of human cancers; citation_author=Q. Gao; citation_author=W.-W. Liang; citation_author=S.M. Foltz; citation_author=G. Mutharasu; citation_author=R.G. Jayasinghe; citation_author=S. Cao; citation_author=W.-W. Liao; citation_author=S.M. Reynolds; citation_author=M.A. Wyczalkowski; citation_author=L. Yao; citation_author=et al.; citation_journal_title=Cell Rep.; citation_volume=23; citation_firstpage=227; citation_lastpage=238.e3; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Lipid peroxidation in cell death; citation_author=M.M. Gaschler; citation_author=B.R. Stockwell; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=482; citation_firstpage=419; citation_lastpage=425; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=A flexible R package for nonnegative matrix factorization; citation_author=R. Gaujoux; citation_author=C. Seoighe; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=367; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Correcting systematic bias and instrument measurement drift with mzRefinery; citation_author=B.C. Gibbons; citation_author=M.C. Chambers; citation_author=M.E. Monroe; citation_author=D.L. Tabb; citation_author=S.H. Payne; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=31; citation_firstpage=3838; citation_lastpage=3840; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Complex heatmaps reveal patterns and correlations in multidimensional genomic data; citation_author=Z. Gu; citation_author=R. Eils; citation_author=M. Schlesner; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=2847; citation_lastpage=2849; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Proteome and acetylome analysis identifies novel pathways and targets regulated by Perifosine in neuroblastoma; citation_author=X. Gu; citation_author=Z. Hua; citation_author=Y. Dong; citation_author=Y. Zhan; citation_author=X. Zhang; citation_author=W. Tian; citation_author=Z. Liu; citation_author=C.J. Thiele; citation_author=Z. Li; citation_journal_title=Sci. Rep.; citation_volume=7; citation_firstpage=42062; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods; citation_author=B.J. Haas; citation_author=A. Dobin; citation_author=B. Li; citation_author=N. Stransky; citation_author=N. Pochet; citation_author=A. Regev; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=213; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression; citation_author=C. Hafemeister; citation_author=R. Satija; citation_journal_title=BioRxiv; citation_firstpage=576827; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Method for lipidomic analysis: p53 expression modulation of sulfatide, ganglioside, and phospholipid composition of U87 MG glioblastoma cells; citation_author=H. He; citation_author=C.A. Conrad; citation_author=C.L. Nilsson; citation_author=Y. Ji; citation_author=T.M. Schaub; citation_author=A.G. Marshall; citation_author=M.R. Emmett; citation_journal_title=Anal. Chem.; citation_volume=79; citation_firstpage=8423; citation_lastpage=8430; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=MetaboliteDetector: comprehensive analysis tool for targeted and nontargeted GC/MS based metabolome analysis; citation_author=K. Hiller; citation_author=J. Hangebrauk; citation_author=C. Jäger; citation_author=J. Spura; citation_author=K. Schreiber; citation_author=D. Schomburg; citation_journal_title=Anal. Chem.; citation_volume=81; citation_firstpage=3429; citation_lastpage=3439; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=KinomeXplorer: an integrated platform for kinome biology studies; citation_author=H. Horn; citation_author=E.M. Schoof; citation_author=J. Kim; citation_author=X. Robin; citation_author=M.L. Miller; citation_author=F. Diella; citation_author=A. Palma; citation_author=G. Cesareni; citation_author=L.J. Jensen; citation_author=R. Linding; citation_journal_title=Nat. Methods; citation_volume=11; citation_firstpage=603; citation_lastpage=604; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Orchestrating high-throughput genomic analysis with Bioconductor; citation_author=W. Huber; citation_author=V.J. Carey; citation_author=R. Gentleman; citation_author=S. Anders; citation_author=M. Carlson; citation_author=B.S. Carvalho; citation_author=H.C. Bravo; citation_author=S. Davis; citation_author=L. Gatto; citation_author=T. Girke; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=115; citation_lastpage=121; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Oxidized arachidonic and adrenic PEs navigate cells to ferroptosis; citation_author=V.E. Kagan; citation_author=G. Mao; citation_author=F. Qu; citation_author=J.P.F. Angeli; citation_author=S. Doll; citation_author=C.S. Croix; citation_author=H.H. Dar; citation_author=B. Liu; citation_author=V.A. Tyurin; citation_author=V.B. Ritov; citation_author=et al.; citation_journal_title=Nat. Chem. Biol.; citation_volume=13; citation_firstpage=81; citation_lastpage=90; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The mutational constraint spectrum quantified from variation in 141,456 humans; citation_author=K.J. Karczewski; citation_author=L.C. Francioli; citation_author=G. Tiao; citation_author=B.B. Cummings; citation_author=J. Alföldi; citation_author=Q. Wang; citation_author=R.L. Collins; citation_author=K.M. Laricchia; citation_author=A. Ganna; citation_author=D.P. Birnbaum; citation_author=et al.; citation_journal_title=Nature; citation_volume=581; citation_firstpage=434; citation_lastpage=443; citation_publication_date=2020; "></meta><meta name="citation_reference" content="citation_title=The central nervous system tumor methylation classifier changes neuro-oncology practice for challenging brain tumor diagnoses and directly impacts patient care; citation_author=S. Karimi; citation_author=J.A. Zuccato; citation_author=Y. Mamatjan; citation_author=S. Mansouri; citation_author=S. Suppiah; citation_author=F. Nassiri; citation_author=P. Diamandis; citation_author=D.G. Munoz; citation_author=K.D. Aldape; citation_author=G. Zadeh; citation_journal_title=Clin. Epigenetics; citation_volume=11; citation_firstpage=185; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The library of integrated network-based cellular signatures NIH program: system-level cataloging of human cells response to perturbations; citation_author=A.B. Keenan; citation_author=S.L. Jenkins; citation_author=K.M. Jagodnik; citation_author=S. Koplev; citation_author=E. He; citation_author=D. Torre; citation_author=Z. Wang; citation_author=A.B. Dohlman; citation_author=M.C. Silverstein; citation_author=A. Lachmann; citation_author=et al.; citation_journal_title=Cell Syst.; citation_volume=6; citation_firstpage=13; citation_lastpage=24; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=TERT promoter mutations occur frequently in gliomas and a subset of tumors derived from cells with low rates of self-renewal; citation_author=P.J. Killela; citation_author=Z.J. Reitman; citation_author=Y. Jiao; citation_author=C. Bettegowda; citation_author=N. Agrawal; citation_author=L.A. Diaz; citation_author=A.H. Friedman; citation_author=H. Friedman; citation_author=G.L. Gallia; citation_author=B.C. Giovanella; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. U S A; citation_volume=110; citation_firstpage=6021; citation_lastpage=6026; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis; citation_author=H. Kim; citation_author=H. Park; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=23; citation_firstpage=1495; citation_lastpage=1502; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=MS-GF+ makes progress towards a universal database search tool for proteomics; citation_author=S. Kim; citation_author=P.A. Pevzner; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5277; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases; citation_author=S. Kim; citation_author=N. Gupta; citation_author=P.A. Pevzner; citation_journal_title=J. Proteome Res.; citation_volume=7; citation_firstpage=3354; citation_lastpage=3363; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Strelka2: fast and accurate calling of germline and somatic variants; citation_author=S. Kim; citation_author=K. Scheffler; citation_author=A.L. Halpern; citation_author=M.A. Bekritsky; citation_author=E. Noh; citation_author=M. Källberg; citation_author=X. Chen; citation_author=Y. Kim; citation_author=D. Beyter; citation_author=P. Krusche; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=591; citation_lastpage=594; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole and time-of-flight gas chromatography/mass spectrometry; citation_author=T. Kind; citation_author=G. Wohlgemuth; citation_author=D.Y. Lee; citation_author=Y. Lu; citation_author=M. Palazoglu; citation_author=S. Shahbaz; citation_author=O. Fiehn; citation_journal_title=Anal. Chem.; citation_volume=81; citation_firstpage=10038; citation_lastpage=10048; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=D.C. Koboldt; citation_author=Q. Zhang; citation_author=D.E. Larson; citation_author=D. Shen; citation_author=M.D. McLellan; citation_author=L. Lin; citation_author=C.A. Miller; citation_author=E.R. Mardis; citation_author=L. Ding; citation_author=R.K. Wilson; citation_journal_title=Genome Res.; citation_volume=22; citation_firstpage=568; citation_lastpage=576; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Snakemake—a scalable bioinformatics workflow engine; citation_author=J. Köster; citation_author=S. Rahmann; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=2520; citation_lastpage=2522; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Plasminogen activator inhibitor-1 promotes the recruitment and polarization of macrophages in cancer; citation_author=M.H. Kubala; citation_author=V. Punj; citation_author=V.R. Placencio-Hickok; citation_author=H. Fang; citation_author=G.E. Fernandez; citation_author=R. Sposto; citation_author=Y.A. DeClerck; citation_journal_title=Cell Rep; citation_volume=25; citation_firstpage=2177; citation_lastpage=2191.e7; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=A recurrent activating PLCG1 mutation in cardiac angiosarcomas increases apoptosis resistance and invasiveness of endothelial cells; citation_author=K. Kunze; citation_author=T. Spieker; citation_author=U. Gamerdinger; citation_author=K. Nau; citation_author=J. Berger; citation_author=T. Dreyer; citation_author=J.R. Sindermann; citation_author=A. Hoffmeier; citation_author=S. Gattenlöhner; citation_author=A. Bräuninger; citation_journal_title=Cancer Res.; citation_volume=74; citation_firstpage=6173; citation_lastpage=6183; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=LIQUID: an-open source software for identifying lipids in LC-MS/MS-based lipidomics data; citation_author=J.E. Kyle; citation_author=K.L. Crowell; citation_author=C.P. Casey; citation_author=G.M. Fujimoto; citation_author=S. Kim; citation_author=S.E. Dautel; citation_author=R.D. Smith; citation_author=S.H. Payne; citation_author=T.O. Metz; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=33; citation_firstpage=1744; citation_lastpage=1746; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B. Langmead; citation_author=S.L. Salzberg; citation_journal_title=Nat. Methods; citation_volume=9; citation_firstpage=357; citation_lastpage=359; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription; citation_author=G. LeRoy; citation_author=B. Rickards; citation_author=S.J. Flint; citation_journal_title=Mol. Cell; citation_volume=30; citation_firstpage=51; citation_lastpage=60; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome; citation_author=B. Li; citation_author=C.N. Dewey; citation_journal_title=BMC Bioinformatics; citation_volume=12; citation_firstpage=323; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts; citation_author=S. Li; citation_author=D. Shen; citation_author=J. Shao; citation_author=R. Crowder; citation_author=W. Liu; citation_author=A. Prat; citation_author=X. He; citation_author=S. Liu; citation_author=J. Hoog; citation_author=C. Lu; citation_author=et al.; citation_journal_title=Cell Rep; citation_volume=4; citation_firstpage=1116; citation_lastpage=1130; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=Quantifying circular RNA expression from RNA-seq data using model-based framework; citation_author=M. Li; citation_author=X. Xie; citation_author=J. Zhou; citation_author=M. Sheng; citation_author=X. Yin; citation_author=E.-A. Ko; citation_author=T. Zhou; citation_author=W. Gu; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=33; citation_firstpage=2131; citation_lastpage=2139; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=SIGNOR 2.0, the SIGnaling network open resource 2.0: 2019 update; citation_author=L. Licata; citation_author=P. Lo Surdo; citation_author=M. Iannuccelli; citation_author=A. Palma; citation_author=E. Micarelli; citation_author=L. Perfetto; citation_author=D. Peluso; citation_author=A. Calderone; citation_author=L. Castagnoli; citation_author=G. Cesareni; citation_journal_title=Nucleic Acids Res.; citation_volume=48; citation_firstpage=D504; citation_lastpage=D510; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Current state of immunotherapy for glioblastoma; citation_author=M. Lim; citation_author=Y. Xia; citation_author=C. Bettegowda; citation_author=M. Weller; citation_journal_title=Nat. Rev. Clin. Oncol.; citation_volume=15; citation_firstpage=422; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=An EGFR-ERK-SOX9 signaling cascade links urothelial development and regeneration to cancer; citation_author=S. Ling; citation_author=X. Chang; citation_author=L. Schultz; citation_author=T.K. Lee; citation_author=A. Chaux; citation_author=L. Marchionni; citation_author=G.J. Netto; citation_author=D. Sidransky; citation_author=D.M. Berman; citation_journal_title=Cancer Res.; citation_volume=71; citation_firstpage=3812; citation_lastpage=3821; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=A library of phosphoproteomic and chromatin signatures for characterizing cellular responses to drug perturbations; citation_author=L. Litichevskiy; citation_author=R. Peckner; citation_author=J.G. Abelin; citation_author=J.K. Asiedu; citation_author=A.L. Creech; citation_author=J.F. Davis; citation_author=D. Davison; citation_author=C.M. Dunning; citation_author=J.D. Egertson; citation_author=S. Egri; citation_author=et al.; citation_journal_title=Cell Syst; citation_volume=6; citation_firstpage=424; citation_lastpage=443.e7; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=The long non-coding RNA SNHG1 promotes glioma progression by competitively binding to miR-194 to regulate PHLDA1 expression; citation_author=L. Liu; citation_author=Y. Shi; citation_author=J. Shi; citation_author=H. Wang; citation_author=Y. Sheng; citation_author=Q. Jiang; citation_author=H. Chen; citation_author=X. Li; citation_author=J. Dong; citation_journal_title=Cell Death Dis.; citation_volume=10; citation_firstpage=463; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The 2016 World health organization classification of tumors of the central nervous system: a summary; citation_author=D.N. Louis; citation_author=A. Perry; citation_author=G. Reifenberger; citation_author=A. von Deimling; citation_author=D. Figarella-Branger; citation_author=W.K. Cavenee; citation_author=H. Ohgaki; citation_author=O.D. Wiestler; citation_author=P. Kleihues; citation_author=D.W. Ellison; citation_journal_title=Acta Neuropathol.; citation_volume=131; citation_firstpage=803; citation_lastpage=820; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Announcing cIMPACT-NOW: the consortium to inform molecular and practical approaches to CNS tumor taxonomy; citation_author=D.N. Louis; citation_author=K. Aldape; citation_author=D.J. Brat; citation_author=D. Capper; citation_author=D.W. Ellison; citation_author=C. Hawkins; citation_author=W. Paulus; citation_author=A. Perry; citation_author=G. Reifenberger; citation_author=D. Figarella-Branger; citation_author=et al.; citation_journal_title=Acta Neuropathol.; citation_volume=133; citation_firstpage=1; citation_lastpage=3; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=550; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Machine learning identifies stemness features associated with oncogenic dedifferentiation; citation_author=T.M. Malta; citation_author=A. Sokolov; citation_author=A.J. Gentles; citation_author=T. Burzykowski; citation_author=L. Poisson; citation_author=J.N. Weinstein; citation_author=B. Kamińska; citation_author=J. Huelsken; citation_author=L. Omberg; citation_author=O. Gevaert; citation_author=et al.; citation_journal_title=Cell; citation_volume=173; citation_firstpage=338; citation_lastpage=354.e15; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=IRF8 is a transcriptional determinant for microglial motility; citation_author=T. Masuda; citation_author=N. Nishimoto; citation_author=D. Tomiyama; citation_author=T. Matsuda; citation_author=H. Tozaki-Saitoh; citation_author=T. Tamura; citation_author=S. Kohsaka; citation_author=M. Tsuda; citation_author=K. Inoue; citation_journal_title=Purinergic Signal.; citation_volume=10; citation_firstpage=515; citation_lastpage=521; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Improved quality control processing of peptide-centric LC-MS proteomics data; citation_author=M.M. Matzke; citation_author=K.M. Waters; citation_author=T.O. Metz; citation_author=J.M. Jacobs; citation_author=A.C. Sims; citation_author=R.S. Baric; citation_author=J.G. Pounds; citation_author=B.-J.M. Webb-Robertson; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=27; citation_firstpage=2866; citation_lastpage=2872; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Current state of immunotherapy for treatment of glioblastoma; citation_author=T. McGranahan; citation_author=K.E. Therkelsen; citation_author=S. Ahmad; citation_author=S. Nagpal; citation_journal_title=Curr. Treat. Options Oncol.; citation_volume=20; citation_firstpage=24; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers; citation_author=C.H. Mermel; citation_author=S.E. Schumacher; citation_author=B. Hill; citation_author=M.L. Meyerson; citation_author=R. Beroukhim; citation_author=G. Getz; citation_journal_title=Genome Biol.; citation_volume=12; citation_firstpage=R41; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=Proteogenomics connects somatic mutations to signalling in breast cancer; citation_author=P. Mertins; citation_author=D.R. Mani; citation_author=K.V. Ruggles; citation_author=M.A. Gillette; citation_author=K.R. Clauser; citation_author=P. Wang; citation_author=X. Wang; citation_author=J.W. Qiao; citation_author=S. Cao; citation_author=F. Petralia; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=55; citation_lastpage=62; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features; citation_author=M.E. Monroe; citation_author=J.L. Shaw; citation_author=D.S. Daly; citation_author=J.N. Adkins; citation_author=R.D. Smith; citation_journal_title=Comput. Biol. Chem.; citation_volume=32; citation_firstpage=215; citation_lastpage=217; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=A novel role for Gab1 and SHP2 in epidermal growth factor-induced Ras activation; citation_author=A. Montagner; citation_author=A. Yart; citation_author=M. Dance; citation_author=B. Perret; citation_author=J.-P. Salles; citation_author=P. Raynal; citation_journal_title=J. Biol. Chem.; citation_volume=280; citation_firstpage=5350; citation_lastpage=5360; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=MPLEx: a robust and universal protocol for single-sample integrative proteomic, metabolomic, and lipidomic analyses; citation_author=E.S. Nakayasu; citation_author=C.D. Nicora; citation_author=A.C. Sims; citation_author=K.E. Burnum-Johnson; citation_author=Y.-M. Kim; citation_author=J.E. Kyle; citation_author=M.M. Matzke; citation_author=A.K. Shukla; citation_author=R.K. Chu; citation_author=A.A. Schepmoes; citation_author=et al.; citation_journal_title=MSystems; citation_volume=1; citation_firstpage=e00043; citation_lastpage=16; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Functions and mechanisms of non-histone protein acetylation; citation_author=T. Narita; citation_author=B.T. Weinert; citation_author=C. Choudhary; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=20; citation_firstpage=156; citation_lastpage=174; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=DNA methylation profiling to predict recurrence risk in meningioma: development and validation of a nomogram to optimize clinical management; citation_author=F. Nassiri; citation_author=Y. Mamatjan; citation_author=S. Suppiah; citation_author=J.H. Badhiwala; citation_author=S. Mansouri; citation_author=S. Karimi; citation_author=O. Saarela; citation_author=L. Poisson; citation_author=I. Gepfner-Tuma; citation_author=J. Schittenhelm; citation_author=et al.; citation_journal_title=Neuro-Oncol.; citation_volume=21; citation_firstpage=901; citation_lastpage=910; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Determining cell type abundance and expression from bulk tissues with digital cytometry; citation_author=A.M. Newman; citation_author=C.B. Steen; citation_author=C.L. Liu; citation_author=A.J. Gentles; citation_author=A.A. Chaudhuri; citation_author=F. Scherer; citation_author=M.S. Khodadoust; citation_author=M.S. Esfahani; citation_author=B.A. Luca; citation_author=D. Steiner; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_firstpage=773; citation_lastpage=782; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=CBTRUS statistical report: primary brain and other central nervous system tumors diagnosed in the United States in 2012-2016; citation_author=Q.T. Ostrom; citation_author=G. Cioffi; citation_author=H. Gittleman; citation_author=N. Patil; citation_author=K. Waite; citation_author=C. Kruchko; citation_author=J.S. Barnholtz-Sloan; citation_journal_title=Neuro-Oncol; citation_volume=21; citation_firstpage=v1; citation_lastpage=v100; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation; citation_author=N.A. O’Leary; citation_author=M.W. Wright; citation_author=J.R. Brister; citation_author=S. Ciufo; citation_author=D. Haddad; citation_author=R. McVeigh; citation_author=B. Rajput; citation_author=B. Robbertse; citation_author=B. Smith-White; citation_author=D. Ako-Adjei; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=D733; citation_lastpage=D745; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Modulation of beta-catenin phosphorylation/degradation by cyclin-dependent kinase 2; citation_author=C.S. Park; citation_author=S.I. Kim; citation_author=M.S. Lee; citation_author=C.-Y. Youn; citation_author=D.J. Kim; citation_author=E.-H. Jho; citation_author=W.K. Song; citation_journal_title=J. Biol. Chem.; citation_volume=279; citation_firstpage=19592; citation_lastpage=19599; citation_publication_date=2004; "></meta><meta name="citation_reference" content="citation_title=Scikit-learn: machine learning in Python; citation_author=F. Pedregosa; citation_author=G. Varoquaux; citation_author=A. Gramfort; citation_author=V. Michel; citation_author=B. Thirion; citation_author=O. Grisel; citation_author=M. Blondel; citation_author=P. Prettenhofer; citation_author=R. Weiss; citation_author=V. Dubourg; citation_author=et al.; citation_journal_title=J. Mach. Learn. Res.; citation_volume=12; citation_firstpage=2825; citation_lastpage=2830; citation_publication_date=2011; "></meta><meta name="citation_reference" content="citation_title=SIGNOR: a database of causal relationships between biological entities; citation_author=L. Perfetto; citation_author=L. Briganti; citation_author=A. Calderone; citation_author=A. Cerquone Perpetuini; citation_author=M. Iannuccelli; citation_author=F. Langone; citation_author=L. Licata; citation_author=M. Marinkovic; citation_author=A. Mattioni; citation_author=T. Pavlidou; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=D548; citation_lastpage=D554; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Short-course radiation plus temozolomide in elderly patients with glioblastoma; citation_author=J.R. Perry; citation_author=N. Laperriere; citation_author=C.J. O’Callaghan; citation_author=A.A. Brandes; citation_author=J. Menten; citation_author=C. Phillips; citation_author=M. Fay; citation_author=R. Nishikawa; citation_author=J.G. Cairncross; citation_author=W. Roa; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=376; citation_firstpage=1027; citation_lastpage=1037; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Connecting Omics Signatures of Diseases, Drugs, and Mechanisms of Actions with iLINCS; citation_author=M. Pilarczyk; citation_author=M.F. Najafabadi; citation_author=M. Kouril; citation_author=J. Vasiliauskas; citation_author=W. Niu; citation_author=B. Shamsaei; citation_author=N. Mahi; citation_author=L. Zhang; citation_author=N. Clark; citation_author=Y. Ren; citation_author=et al.; citation_journal_title=BioRxiv; citation_firstpage=826271; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=The two faces of tumor-associated macrophages and their clinical significance in colorectal cancer; citation_author=M.L. Pinto; citation_author=E. Rios; citation_author=C. Durães; citation_author=R. Ribeiro; citation_author=J.C. Machado; citation_author=A. Mantovani; citation_author=M.A. Barbosa; citation_author=F. Carneiro; citation_author=M.J. Oliveira; citation_journal_title=Front. Immunol.; citation_volume=10; citation_firstpage=1875; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data; citation_author=T. Pluskal; citation_author=S. Castillo; citation_author=A. Villar-Briones; citation_author=M. Oresic; citation_journal_title=BMC Bioinformatics; citation_volume=11; citation_firstpage=395; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=DAnTE: a statistical tool for quantitative analysis of -omics data; citation_author=A.D. Polpitiya; citation_author=W.-J. Qian; citation_author=N. Jaitly; citation_author=V.A. Petyuk; citation_author=J.N. Adkins; citation_author=D.G. Camp; citation_author=G.A. Anderson; citation_author=R.D. Smith; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=24; citation_firstpage=1556; citation_lastpage=1558; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=Intramolecular interaction between phosphorylated tyrosine-783 and the C-terminal Src homology 2 domain activates phospholipase C-gamma1; citation_author=B. Poulin; citation_author=F. Sekiya; citation_author=S.G. Rhee; citation_journal_title=Proc. Natl. Acad. Sci. U S A; citation_volume=102; citation_firstpage=4276; citation_lastpage=4281; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis; citation_author=T. Rausch; citation_author=T. Zichner; citation_author=A. Schlattl; citation_author=A.M. Stutz; citation_author=V. Benes; citation_author=J.O. Korbel; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=i333; citation_lastpage=i339; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=Social network architecture of human immune cells unveiled by quantitative proteomics; citation_author=J.C. Rieckmann; citation_author=R. Geiger; citation_author=D. Hornburg; citation_author=T. Wolf; citation_author=K. Kveler; citation_author=D. Jarrossay; citation_author=F. Sallusto; citation_author=S.S. Shen-Orr; citation_author=A. Lanzavecchia; citation_author=M. Mann; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=18; citation_firstpage=583; citation_lastpage=593; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=CORUM: the comprehensive resource of mammalian protein complexes--2009; citation_author=A. Ruepp; citation_author=B. Waegele; citation_author=M. Lechner; citation_author=B. Brauner; citation_author=I. Dunger-Kaltenbach; citation_author=G. Fobo; citation_author=G. Frishman; citation_author=C. Montrone; citation_author=H.-W. Mewes; citation_journal_title=Nucleic Acids Res.; citation_volume=38; citation_firstpage=D497; citation_lastpage=D501; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Integrated genomic analysis of diverse induced pluripotent stem cells from the progenitor cell biology consortium; citation_author=N. Salomonis; citation_author=P.J. Dexheimer; citation_author=L. Omberg; citation_author=R. Schroll; citation_author=S. Bush; citation_author=J. Huo; citation_author=L. Schriml; citation_author=S. Ho Sui; citation_author=M. Keddache; citation_author=C. Mayhew; citation_author=et al.; citation_journal_title=Stem Cell Rep.; citation_volume=7; citation_firstpage=110; citation_lastpage=125; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=TCGA Workflow: analyze cancer genomics and epigenomics data using Bioconductor packages; citation_author=T.C. Silva; citation_author=A. Colaprico; citation_author=C. Olsen; citation_author=F. D’Angelo; citation_author=G. Bontempi; citation_author=M. Ceccarelli; citation_author=H. Noushmehr; citation_journal_title=F1000Res.; citation_volume=5; citation_firstpage=1542; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=One-class detection of cell states in tumor subtypes; citation_author=A. Sokolov; citation_author=E.O. Paull; citation_author=J.M. Stuart; citation_journal_title=Pac. Symp. Biocomput.; citation_volume=21; citation_firstpage=405; citation_lastpage=416; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis; citation_author=X. Song; citation_author=J. Ji; citation_author=K.J. Gleason; citation_author=F. Yang; citation_author=J.A. Martignetti; citation_author=L.S. Chen; citation_author=P. Wang; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=S52; citation_lastpage=S65; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=LINCS Data Portal 2.0: next generation access point for perturbation-response signatures; citation_author=V. Stathias; citation_author=J. Turner; citation_author=A. Koleti; citation_author=D. Vidovic; citation_author=D. Cooper; citation_author=M. Fazel-Najafabadi; citation_author=M. Pilarczyk; citation_author=R. Terryn; citation_author=C. Chung; citation_author=A. Umeano; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=48; citation_firstpage=D431; citation_lastpage=D439; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma; citation_author=R. Stupp; citation_author=W.P. Mason; citation_author=M.J. van den Bent; citation_author=M. Weller; citation_author=B. Fisher; citation_author=M.J.B. Taphoorn; citation_author=K. Belanger; citation_author=A.A. Brandes; citation_author=C. Marosi; citation_author=U. Bogdahn; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=352; citation_firstpage=987; citation_lastpage=996; citation_publication_date=2005; "></meta><meta name="citation_reference" content="citation_title=Effect of tumor-treating fields plus maintenance temozolomide vs maintenance temozolomide alone on survival in patients with glioblastoma: a randomized clinical trial; citation_author=R. Stupp; citation_author=S. Taillibert; citation_author=A. Kanner; citation_author=W. Read; citation_author=D. Steinberg; citation_author=B. Lhermitte; citation_author=S. Toms; citation_author=A. Idbaih; citation_author=M.S. Ahluwalia; citation_author=K. Fink; citation_author=et al.; citation_journal_title=JAMA; citation_volume=318; citation_firstpage=2306; citation_lastpage=2316; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Hotspot mutations in H3F3A and IDH1 define distinct epigenetic and biological subgroups of glioblastoma; citation_author=D. Sturm; citation_author=H. Witt; citation_author=V. Hovestadt; citation_author=D.-A. Khuong-Quang; citation_author=D.T.W. Jones; citation_author=C. Konermann; citation_author=E. Pfaff; citation_author=M. Tönjes; citation_author=M. Sill; citation_author=S. Bender; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=22; citation_firstpage=425; citation_lastpage=437; citation_publication_date=2012; "></meta><meta name="citation_reference" content="citation_title=A next generation connectivity map: L1000 platform and the first 1,000,000 profiles; citation_author=A. Subramanian; citation_author=R. Narayan; citation_author=S.M. Corsello; citation_author=D.D. Peck; citation_author=T.E. Natoli; citation_author=X. Lu; citation_author=J. Gould; citation_author=J.F. Davis; citation_author=A.A. Tubelli; citation_author=J.K. Asiedu; citation_author=et al.; citation_journal_title=Cell; citation_volume=171; citation_firstpage=1437; citation_lastpage=1452.e17; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Database of evidence for precision oncology portal; citation_author=S.Q. Sun; citation_author=R.J. Mashl; citation_author=S. Sengupta; citation_author=A.D. Scott; citation_author=W. Wang; citation_author=P. Batra; citation_author=L.-B. Wang; citation_author=M.A. Wyczalkowski; citation_author=L. Ding; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=34; citation_firstpage=4315; citation_lastpage=4317; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Reproducibility of differential proteomic technologies in CPTAC fractionated xenografts; citation_author=D.L. Tabb; citation_author=X. Wang; citation_author=S.A. Carr; citation_author=K.R. Clauser; citation_author=P. Mertins; citation_author=M.C. Chambers; citation_author=J.D. Holman; citation_author=J. Wang; citation_author=B. Zhang; citation_author=L.J. Zimmerman; citation_author=et al.; citation_journal_title=J. Proteome Res.; citation_volume=15; citation_firstpage=691; citation_lastpage=706; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=A global reference for human genetic variation; citation_author=The 1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=526; citation_firstpage=68; citation_lastpage=74; citation_publication_date=2015; "></meta><meta name="citation_reference" content="citation_title=Bioconda: sustainable and comprehensive software distribution for the life sciences; citation_author=The Bioconda Team; citation_author=B. Grüning; citation_author=R. Dale; citation_author=A. Sjödin; citation_author=B.A. Chapman; citation_author=J. Rowe; citation_author=C.H. Tomkins-Tinch; citation_author=R. Valieris; citation_author=J. Köster; citation_journal_title=Nat. Methods; citation_volume=15; citation_firstpage=475; citation_lastpage=476; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Comprehensive genomic characterization defines human glioblastoma genes and core pathways; citation_author=The Cancer Genome Atlas Research Network; citation_journal_title=Nature; citation_volume=455; citation_firstpage=1061; citation_lastpage=1068; citation_publication_date=2008; "></meta><meta name="citation_reference" content="citation_title=OmniPath: guidelines and gateway for literature-curated signaling pathway resources; citation_author=D. Türei; citation_author=T. Korcsmáros; citation_author=J. Saez-Rodriguez; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=966; citation_lastpage=967; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=LinkedOmics: analyzing multi-omics data within and across 32 cancer types; citation_author=S.V. Vasaikar; citation_author=P. Straub; citation_author=J. Wang; citation_author=B. Zhang; citation_journal_title=Nucleic Acids Res.; citation_volume=46; citation_firstpage=D956; citation_lastpage=D963; citation_publication_date=2018; "></meta><meta name="citation_reference" content="citation_title=Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities; citation_author=S. Vasaikar; citation_author=C. Huang; citation_author=X. Wang; citation_author=V.A. Petyuk; citation_author=S.R. Savage; citation_author=B. Wen; citation_author=Y. Dou; citation_author=Y. Zhang; citation_author=Z. Shi; citation_author=O.A. Arshad; citation_author=et al.; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1035; citation_lastpage=1049.e19; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1; citation_author=R.G.W. Verhaak; citation_author=K.A. Hoadley; citation_author=E. Purdom; citation_author=V. Wang; citation_author=Y. Qi; citation_author=M.D. Wilkerson; citation_author=C.R. Miller; citation_author=L. Ding; citation_author=T. Golub; citation_author=J.P. Mesirov; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=17; citation_firstpage=98; citation_lastpage=110; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=The tyrosine phosphatase Shp2 interacts with NPM-ALK and regulates anaplastic lymphoma cell growth and migration; citation_author=C. Voena; citation_author=C. Conte; citation_author=C. Ambrogio; citation_author=E. Boeri Erba; citation_author=F. Boccalatte; citation_author=S. Mohammed; citation_author=O.N. Jensen; citation_author=G. Palestro; citation_author=G. Inghirami; citation_author=R. Chiarle; citation_journal_title=Cancer Res.; citation_volume=67; citation_firstpage=4278; citation_lastpage=4286; citation_publication_date=2007; "></meta><meta name="citation_reference" content="citation_title=Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment; citation_author=Q. Wang; citation_author=B. Hu; citation_author=X. Hu; citation_author=H. Kim; citation_author=M. Squatrito; citation_author=L. Scarpace; citation_author=A.C. deCarvalho; citation_author=S. Lyu; citation_author=P. Li; citation_author=Y. Li; citation_author=et al.; citation_journal_title=Cancer Cell; citation_volume=32; citation_firstpage=42; citation_lastpage=56.e6; citation_publication_date=2017; "></meta><meta name="citation_reference" content="citation_title=Sequential projection pursuit principal component analysis--dealing with missing data associated with new -omics technologies; citation_author=B.-J.M. Webb-Robertson; citation_author=M.M. Matzke; citation_author=T.O. Metz; citation_author=J.E. McDermott; citation_author=H. Walker; citation_author=K.D. Rodland; citation_author=J.G. Pounds; citation_author=K.M. Waters; citation_journal_title=BioTechniques; citation_volume=54; citation_firstpage=165; citation_lastpage=168; citation_publication_date=2013; "></meta><meta name="citation_reference" content="citation_title=A statistical analysis of the effects of urease pre-treatment on the measurement of the urinary metabolome by gas chromatography-mass spectrometry; citation_author=B.-J. Webb-Robertson; citation_author=Y.-M. Kim; citation_author=E.M. Zink; citation_author=K.A. Hallaian; citation_author=Q. Zhang; citation_author=R. Madupu; citation_author=K.M. Waters; citation_author=T.O. Metz; citation_journal_title=Metabolomics; citation_volume=10; citation_firstpage=897; citation_lastpage=908; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=Welcome to the tidyverse; citation_author=H. Wickham; citation_author=M. Averick; citation_author=J. Bryan; citation_author=W. Chang; citation_author=L. McGowan; citation_author=R. François; citation_author=G. Grolemund; citation_author=A. Hayes; citation_author=L. Henry; citation_author=J. Hester; citation_author=et al.; citation_journal_title=J. Open Source Softw.; citation_volume=4; citation_firstpage=1686; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking; citation_author=M.D. Wilkerson; citation_author=D.N. Hayes; citation_journal_title=Bioinforma. Oxf. Engl.; citation_volume=26; citation_firstpage=1572; citation_lastpage=1573; citation_publication_date=2010; "></meta><meta name="citation_reference" content="citation_title=Copy number analysis of whole-genome data using BIC-seq2 and its application to detection of cancer susceptibility variants; citation_author=R. Xi; citation_author=S. Lee; citation_author=Y. Xia; citation_author=T.-M. Kim; citation_author=P.J. Park; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=6274; citation_lastpage=6286; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=IDH1 and IDH2 mutations in gliomas; citation_author=H. Yan; citation_author=D.W. Parsons; citation_author=G. Jin; citation_author=R. McLendon; citation_author=B.A. Rasheed; citation_author=W. Yuan; citation_author=I. Kos; citation_author=I. Batinic-Haberle; citation_author=S. Jones; citation_author=G.J. Riggins; citation_author=et al.; citation_journal_title=N. Engl. J. Med.; citation_volume=360; citation_firstpage=765; citation_lastpage=773; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads; citation_author=K. Ye; citation_author=M.H. Schulz; citation_author=Q. Long; citation_author=R. Apweiler; citation_author=Z. Ning; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2865; citation_lastpage=2871; citation_publication_date=2009; "></meta><meta name="citation_reference" content="citation_title=MODMatcher: multi-omics data matcher for integrative genomic analysis; citation_author=S. Yoo; citation_author=T. Huang; citation_author=J.D. Campbell; citation_author=E. Lee; citation_author=Z. Tu; citation_author=M.W. Geraci; citation_author=C.A. Powell; citation_author=E.E. Schadt; citation_author=A. Spira; citation_author=J. Zhu; citation_journal_title=PLoS Comput. Biol.; citation_volume=10; citation_firstpage=e1003790; citation_publication_date=2014; "></meta><meta name="citation_reference" content="citation_title=TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach; citation_author=J. Zecha; citation_author=S. Satpathy; citation_author=T. Kanashova; citation_author=S.C. Avanessian; citation_author=M.H. Kane; citation_author=K.R. Clauser; citation_author=P. Mertins; citation_author=S.A. Carr; citation_author=B. Kuster; citation_journal_title=Mol. Cell. Proteomics; citation_volume=18; citation_firstpage=1468; citation_lastpage=1478; citation_publication_date=2019; "></meta><meta name="citation_reference" content="citation_title=INTEGRATE: gene fusion discovery using whole genome and transcriptome data; citation_author=J. Zhang; citation_author=N.M. White; citation_author=H.K. Schmidt; citation_author=R.S. Fulton; citation_author=C. Tomlinson; citation_author=W.C. Warren; citation_author=R.K. Wilson; citation_author=C.A. Maher; citation_journal_title=Genome Res.; citation_volume=26; citation_firstpage=108; citation_lastpage=118; citation_publication_date=2016; "></meta><meta name="citation_reference" content="citation_title=Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes; citation_author=W. Zhou; citation_author=P.W. Laird; citation_author=H. Shen; citation_journal_title=Nucleic Acids Res.; citation_volume=45; citation_firstpage=e22; citation_publication_date=2017; "></meta><meta name="citation_abstract" content="&lt;h2&gt;Summary&lt;/h2&gt;&lt;p&gt;Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for &lt;i&gt;EGFR&lt;/i&gt;-, &lt;i&gt;TP53&lt;/i&gt;-, and &lt;i&gt;RB1&lt;/i&gt;-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in &lt;i&gt;IDH&lt;/i&gt;-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.&lt;/p&gt;"></meta><meta charset="UTF-8" /><!--

    Pagebuild admin UI
-->



<script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8"
            data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "true";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/nb6b~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/nb6b~selected-legacy.css"/>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans"/>
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-cba8bc975dae30d19357.css"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-cba8bc975dae30d19357.css" media="print"/><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-ccell.css"/><script type="text/javascript">
    document.addEventListener('DOMContentLoaded', function(event) {
        var SETTINGS = {
            ENDPOINT: 'https://crossmark.crossref.org/dialog',
            SCRIPT_VERSION: 'v2.0',
            // LOGO_URL: 'http://crossmark-cdn.crossref.org/widget/v2.0/logos/crossmark-logo-square.svg',
            STYLESHEET_URL: 'https://crossmark-cdn.crossref.org/widget/v2.0/style.css',
        };

        var touchStarted = false;
        var touchArea;
        var tapEvent = function(element, callback) {
            element.addEventListener('click', function(event) {
                if (event.ctrlKey || event.shiftKey || event.metaKey || event.which !== 1) {
                    return;
                }
                return callback(event);
            }, false);

            element.addEventListener('touchstart', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                touchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                touchStarted = element;

                event.stopPropagation();
            }, false);

            window.addEventListener('touchstart', function(event) {
                touchStarted = false;
            });

            element.addEventListener('touchmove', function(event) {
                if (event.touches.length > 1) return touchStarted = false;
                var newTouchArea = { x: event.touches[0].screenX, y: event.touches[0].screenY };
                if (Math.pow(touchArea.x - newTouchArea.x, 2) + Math.pow(touchArea.y - newTouchArea.y, 2) > 500) {
                    touchStarted = false;
                }
            }, false);

            element.addEventListener('touchend', function(event) {
                if (touchStarted) {
                    var element = touchStarted;
                    touchStarted = false;

                    var x = event.changedTouches[0].pageX - window.pageXOffset;
                    var y = event.changedTouches[0].pageY - window.pageYOffset;
                    var target = document.elementFromPoint(x, y);

                    return callback(event);
                } else {
                    event.preventDefault();
                }
            }, false);
        };

        var buildQueryString = function(data) {
            var query = [];
            for (var key in data) {
                query.push(encodeURIComponent(key) + '=' + encodeURIComponent(data[key]));
            }
            return '?' + query.join('&');
        };

        var isIos = /iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream;

        // Scan for case-insensitive.
        var getDoiMeta = function() {
            var metaTags = document.querySelectorAll("meta");
            for (var i in metaTags) {
                if (metaTags.hasOwnProperty(i)) {
                    var tag = metaTags[i];
                    if ((tag.name || "").toLowerCase() == "dc.identifier" && tag.scheme == "doi") {
                        return tag;
                    }
                }
            }
        };

        var doiMeta = getDoiMeta();
        var doi = doiMeta ? doiMeta.getAttribute('content').replace(/^(info:doi\/|doi:)/, '') : null;
        if(doi == null) doi = $("#crossMark").data('doi');

        var queryData = {
            doi: doi,
            domain: window.location.hostname,
            uri_scheme: window.location.protocol,
            cm_version: SETTINGS.SCRIPT_VERSION
        };

        var css = document.createElement('link');
        css.setAttribute('href', SETTINGS.STYLESHEET_URL);
        css.setAttribute('type', 'text/css');
        css.setAttribute('rel', 'stylesheet');
        document.querySelector('head').appendChild(css);

        var widget = document.createElement('div');
        widget.setAttribute('id', 'crossmark-widget');

        // Initial display none to avoid a flash on load.
        widget.style.display = 'none';

        widget.innerHTML =
            '<div class="crossmark-reset crossmark-overlay"></div>' +
            '<div class="crossmark-reset crossmark-popup">' +
            '<div class="crossmark-reset crossmark-popup__offset">' +
            '<div class="crossmark-reset crossmark-popup__inner">' +
            '<div class="crossmark-reset crossmark-popup__header">' +
            '<a target="_blank" href="http://www.crossref.org/crossmark">' +
            '<img class="crossmark-reset crossmark-popup__logo">' +
            '</a>' +
            '<button class="crossmark-reset crossmark-popup__btn-close"></button>' +
            '</div>' +
            '<div class="crossmark-reset crossmark-popup__content-wrapper">' +
            '<iframe class="crossmark-reset crossmark-popup__content"></iframe>' +
            '</div>' +
            '</div>' +
            '</div>' +
            '</div>';

        var overlay = widget.querySelector('.crossmark-overlay');
        var popup = widget.querySelector('.crossmark-popup');
        var popupOffset = widget.querySelector('.crossmark-popup__offset');
        var popupInner = widget.querySelector('.crossmark-popup__inner');
        var logo = widget.querySelector('.crossmark-popup__logo');
        var content = widget.querySelector('.crossmark-popup__content');
        var closeButton = widget.querySelector('.crossmark-popup__btn-close');

        if (isIos) popupOffset.classList.add('is-ios');
        // logo.setAttribute('src', SETTINGS.LOGO_URL);

        document.body.appendChild(widget);

        [overlay, popup, closeButton].map(function(element) {
            tapEvent(element, function(event) {
                widget.style.display = 'none';

                event.preventDefault();
                event.stopPropagation();
            });
        });

        tapEvent(popupInner, function(event) {
            event.stopPropagation();
        });

        var initialised = false;

        // delete queryData.domain;

        var links = [].slice.call(document.querySelectorAll('[data-target=crossmark]'), 0);
        links.map(function(link) {
            link.style.cursor = 'pointer';
            link.setAttribute('href', SETTINGS.ENDPOINT + buildQueryString(queryData));

            tapEvent(link, function(event) {
                if (!initialised) {
                    content.setAttribute('src', SETTINGS.ENDPOINT + buildQueryString(queryData));
                    initialised = true;
                }
                widget.style.display = 'block';
                if (isIos) popupInner.style.top = window.scrollY + 'px';

                event.preventDefault();
                event.stopPropagation();
            });
        });
    });
</script><script type="text/javascript">
    var crossmark = {
        sTooltipID: 'crossmark-tooltip-130',
        sTooltipSelector: '#crossmark-tooltip-130',
        sTooltipCopy: 'Click to get updates and verify authenticity.',
        sOpenSelector: '#open-crossmark',
        activateTooltip: function() {
            var that = this;
            $('body').append('<div id="' + this.sTooltipID + '" class="crossmark-tooltip" style="display: none;"><div class="cmtttop"></div><div class="cmttmid"><p>' + this.sTooltipCopy + '</p></div><div class="cmttbot"></div></div>');
            var $icon = $('#crossmark-icon');
            $icon.attr({'title': '', 'alt': ''}).show();
            $icon.mouseover(function(){
                var o = $icon.offset();
                var x = o.left + $icon.width()/2 - $('#crossmark-icon').width()/2;
                var y = o.top - $(that.sTooltipSelector).height() + 10;
                $(that.sTooltipSelector).css({ 'left' : x, 'top' : y }).show();
            });
            $icon.mouseout(function(){
                $(that.sTooltipSelector).hide();
            });
        },
        addStylesheet: function() {
            $('#open-crossmark').parent().eq(0).prepend("<link media='screen' rel='preload' as='style' href='/templates/jsp/crossref.css'/>");
        }
    };
    $(document).ready(function() {
        crossmark.addStylesheet();
        crossmark.activateTooltip();
    });
</script>
        <link rel="stylesheet" type="text/css" href="/templates/jsp/crossref.css"/>
    







        
        <script class="optanon-category-4" type="text/plain" async src="https://www.youtube.com/iframe_api"></script>
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script class="optanon-category-4" async src="https://static.addtoany.com/menu/page.js"></script>
</head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>


<script type="text/javascript">

        if(true) {
            document.getElementById("skipNavigationLink").onclick =function skipElement () {
                document.getElementById('pageBody').setAttribute('tabindex','0');
                document.getElementById('pageBody').focus();
            }

        }

</script>




    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2821%2900050-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610821000507%253Fshowall%253Dtrue" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00050-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610821000507%3Fshowall%3Dtrue"/><input type="hidden" name="referer" value="/cancer-cell/fulltext/S1535-6108(21)00050-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610821000507%3Fshowall%3Dtrue"/><input type="hidden" name="ecommRedirect"/><input type="hidden" name="code" value="cell-site"/><input type="hidden" name="UX3LoginWidget" value="true"/><input type="hidden" name="isPopup" value="true"/><div class="input-group"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"/></div><div class="input-group"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"/><a href="#" class="login__field__action js-show-password">Show</a><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"/><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"/></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2821%2900050-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610821000507%253Fshowall%253Dtrue" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"/><input type="hidden" name="requestResetPassword" value="true"/><input type="hidden" name="codeSite" value="cell-site"/><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"/></div><div class="submit"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"/></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="ccell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br />
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br />
				<br />
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch">
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cancer-cell/home" title="Cancer Cell"><img alt="Cancer Cell" src="/pb-assets/ux3/logos/cell/CancerCell_white-1532516635240.svg"/></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="f0df5ebf-b1ca-4821-b16c-caadb221e181" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"/></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__loginHtml"><a href="https://www.editorialmanager.com/cancer-cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li class="profile-links__logIn"><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2821%2900050-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610821000507%253Fshowall%253Dtrue&amp;pii=S1535-6108%2821%2900050-7" title="Log in to Cell.com" class="show-login">Log in</a></li><li class="profile-links__register"><a href="/action/registration?journalCode=ccell&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2821%2900050-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610821000507%253Fshowall%253Dtrue&amp;directReg=true" title="Register a username and password for free">Register</a></li><li class="hidden-lg"><div class="dropBlock profile-links__user-list__wrapper"><a href="#" data-db-target-for="loginMenu" aria-expanded="false" aria-controls="login-links" title="LogIn" class="profile-links__logIn__mobile">Log in</a><div data-db-target-of="loginMenu" id="login-links" class="dropBlock__holder profile-links__user-list__holder"><ul class="rlist article-tools__sub-list"><li><a href="https://www.editorialmanager.com/cancer-cell" data-aa-conversiondriver="jbs:global-header:submitpage">Submit</span></a></li><li><a href="/action/showLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcancer-cell%2Ffulltext%2FS1535-6108%2821%2900050-7%3F_returnURL%3Dhttps%253A%252F%252Flinkinghub.elsevier.com%252Fretrieve%252Fpii%252FS1535610821000507%253Fshowall%253Dtrue&amp;pii=S1535-6108%2821%2900050-7" title="Log in to Cell.com" class="show-login">Log in</a></li></ul></div></div></li><li class="profile-links__subscribe"><a href="/cancer-cell/subscribe" title="Subscribe to a Cell journal">Subscribe</a></li><li class="profile-links__claim"><a href="/cancer-cell/activate" title="Claim your online subscription">Claim</a></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.ccell.2021.01.006">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin widget-none  widget-compact-all" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d">
                
                
                    
                

                <div class="wrapped " id='CP TOP LEVEL NAV MENU'>
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-1" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About this journal</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            
                <div class="hidden-xs hidden-sm">
            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S1535-6108(23)X0008-7"><img src="/cms/asset/atypon:cms:attachment:img:d3e6:rev:1720427783745-1677:pii:S1535610823X00087/cover.tif.jpg" alt="Cover Image - Cancer Cell, Volume 42, Issue 7"/></a></div><div class="cover-image__meta"></div></div>

            
                </div>
            

        
</div>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life & medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host & Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods & Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology & Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology & Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences & engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cancer-cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.14.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/author-resources" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Author resources</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News & events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/lablinks" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>LabLinks</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help & support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://careers.cell.com/" data-index="1" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Scientific job board
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=ccell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"/><div class="quick-search__toggle"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete"/><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=ccell&amp;ISSN=1535-6108&amp;journalCode=ccell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="ccell"/><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="ccell" checked="checked" aria-label="Search within current journal"/><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"/><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=ccell&amp;ISSN=1535-6108&amp;journalCode=ccell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="dc43d918-6255-4b51-a3b2-9725dca931ea" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    










        <main data-widget-def="UX3articleContent" data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" class="content">
        
    

        


    

            

            

            <article data-widget-id="7f1f4a57-23b9-4395-ab00-2ba3b187e1d7" data-widget-name="UX3articleContent" data-pii="S1535-6108(21)00050-7"><div id="articleHeader" class="article__header"><div class="article-header"><div class="article-header__overlay"></div><div class="what-image__wrapper"><img data-src="/cms/attachment/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg" aria-hidden="true" class="what-image b-lazy" loading="lazy"></img></div><div class="article-header__wrapper"><div class="article-header__top"><div class="row"><div class="col-md-6"><div class="article-header__meta"><span class="article-header__journal">Article</span><span class="article-header__sep">|</span><a href="https://www.cell.com/cancer-cell/issue?pii=S1535-6108(20)X0005-5" class="article-header__vol faded">
                                                    Volume 39, ISSUE 4</a><span></span><span>, </span><span class="article-header__pages faded">P509-528.e20, </span><span class="article-header__date faded">April 12, 2021</span><div class="article-header__download-full-issue"><button title="Download Full Issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=ccell&amp;volume=39&amp;issue=4&amp;pii=S1535-6108%2820%29X0005-5" class="btn--cancel table-of-content__links--item download-full-issue"><span class="svg-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 98 80"><path d="M77 30 71 23 54 40 54 0 44 0 44 40 26 23 19 30 48 59 77 30M88 48 88 68 10 68 10 48 0 48 0 78 98 78 98 48 88 48"></path></svg></span><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started.</p><a href="#" class="table-of-content__alert--close"> Ok</a></div></div></div></div><div class="col-md-6"><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleTools-dropzone"><div data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><div class="article-tools__wrapper"><span class="article-tools__header"><!--.transplant(data-target="article h1" data-remove="false" data-target-class="hidden-xs hidden-sm" data-toggle="transplant" data-transplant="content")--></span><div class="article-tools__holder pull-right"><ul class="rlist rlist--inline article-tools__list"><li class="article-tools__item article-tools__pdf"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__pdf"><a href="#" title="open PDF menu" data-db-target-for="article-tools-pdf" aria-expanded="false" class="article-tools__ctrl article-tools__item__pdf"><i aria-hidden="true" class="icon-gizmo-pdf-file"></i><span class="basic">PDF</span><span class="alternative">PDF</span></a><div data-db-target-of="article-tools-pdf" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="/action/showPdf?pii=S1535-6108%2821%2900050-7" title="Standard PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayStandardPdf ">Standard PDF [6 MB]</a></li><li class="article-tools__sub-item"><a href="/cancer-cell/pdfExtended/S1535-6108(21)00050-7" title="Extended PDF (opens in a new tab)" target="_blank" class="article-tools__item__displayExtendedPdf ">Extended PDF [13 MB]</a></li></ul></div></div></li><li class="article-tools__item article-tools__figures"><div aria-hidden="true" class="dropBlock article-tools__dropblock article-tools__figures"><a href="#" title="open Figures menu" data-db-target-for="article-tools-figures" aria-expanded="false" class="article-tools__ctrl article-tools__item__figures"><i aria-hidden="true" class="icon-gizmo-image_figure"></i><span>Figures</span></a><div data-db-target-of="article-tools-figures" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Figure Viewer (opens in a new tab)" target="_blank" class="article-tools__item__displayFigureViewer js--openFigureViewer ">Figure Viewer</a></li><li class="article-tools__sub-item"><a href="/cancer-cell/ppt/S1535-6108(21)00050-7.ppt" title="Download Figures (PPT) (opens in a new tab)" target="_blank" class="article-tools__item__displayDownloadFigures ">Download Figures (PPT)</a></li></ul></div></div></li><li class="article-tools__item article-tools__save"><div aria-hidden="false" class="dropBlock article-tools__dropblock article-tools__save"><a href="#" title="open Save menu" data-db-target-for="article-tools-save" aria-expanded="false" class="article-tools__ctrl article-tools__item__save"><i aria-hidden="true" class="icon-gizmo-folder-in"></i><span>Save</span></a><div data-db-target-of="article-tools-save" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist article-tools__sub-list"><li class="article-tools__sub-item"><a href="#" title="Add to Online Library" class="article-tools__item__displayAddToOnlineLibrary mendeley ">Add to Online Library</a><span class="powered-by"><span>Powered By</span><i class="icon-mendeley"></i><span>Mendeley</span></span></li><li class="article-tools__sub-item"><a href="https://www.cell.com/personalize/addFavoritePublication?doi=10.1016%2Fj.ccell.2021.01.006" title="Add to My Reading List (opens in a new tab)" target="_blank" class="article-tools__item__displayAddToMyReadingList ">Add to My Reading List</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.ccell.2021.01.006&amp;pii=S1535-6108%2821%2900050-7" title="Export Citation (opens in a new tab)" target="_blank" class="article-tools__item__displayExportCitation ">Export Citation</a></li><li class="article-tools__sub-item"><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.ccell.2021.01.006" title="Create Citation Alert (opens in a new tab)" target="_blank" class="article-tools__item__displayCreateCitationAlert ">Create Citation Alert</a></li></ul></div></div></li><li class="article-tools__item article-tools__coolBar">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><div data-widget-id="6f130e25-c0e4-42df-bb7d-35ca8d725c64" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><a href="#" data-db-target-for="6f130e25-c0e4-42df-bb7d-35ca8d725c64" aria-expanded="false" title="open Share menu" aria-labelledby="share-label-913" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-gizmo-share"></i><span id="share-label-913">Share</span></a><div data-db-target-of="6f130e25-c0e4-42df-bb7d-35ca8d725c64" id="6f130e25-c0e4-42df-bb7d-35ca8d725c64_Pop" class="share__block dropBlock__holder article-tools__dropblock__holder">
    

        


    

            

            

            <div data-widget-id="d7798b2a-1dc2-4a1b-bb92-9415c2383b15" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
    var res = window.location.href;
    res = res.replace(/\(/g, "%28");
    res = res.replace(/\)/g, "%29");    
    var addthis_share = {
       url: res
    }
    var addthis_config = {
       data_track_clickback: false 
    } 
</script> </div>

            

        
<span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cancer-cell/fulltext/S1535-6108%2821%2900050-7"><li class="a2a_listitem_custom"><a role="link" title="Twitter" class="a2a_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter3"></i><span>Twitter</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook3"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin3"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i><span>Email</span></a></li></ul></div></div>

            

        
</li><li><div class="dropBlock article-tools__dropblock article-tools__more"><a href="#" title="Show more tools menu" data-db-target-for="article-tools-more" aria-expanded="false" class="article-tools__ctrl"><i aria-hidden="true" class="icon-dots-three-horizontal"></i><span>more</span></a><div data-db-target-of="article-tools-more" class="dropBlock__holder article-tools__dropblock__holder"><ul class="rlist"><li><a href="http://www.medreprints.com/?pii=S1535610821000507&amp;issn=15356108&amp;source=jbs" title="Reprints" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-gizmo-reprints"></i><span>Reprints</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1535610821000507&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Request" class="article-tools__item__request"><i aria-hidden="true" class="icon-copyright"></i><span>Request</span></a></li></ul></div></div></li><li><a href="#" class="back-to-top"><i aria-hidden="true" class="icon-arrow_u_p"></i><span>Top</span></a></li></ul></div></div></div></div></div></div></div><div class="article-header__middle"><h1 class="article-header__title">Proteogenomic and metabolomic characterization of human glioblastoma</h1><span class="article-header__sub-title"></span><ul class="rlist loa inline-bullet-list ellipsis-dot" data-number-of-author="6"><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au1-authorcard" aria-expanded="false" aria-controls="au1" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Liang-Bo Wang">Liang-Bo Wang</a> <a href="#" data-db-target-for="crosref0020" aria-expanded="false" aria-controls="crosref0020" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0020" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au1-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Liang-Bo Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Liang-Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au2-authorcard" aria-expanded="false" aria-controls="au2" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alla Karpova">Alla Karpova</a> <a href="#" data-db-target-for="crosref0035" aria-expanded="false" aria-controls="crosref0035" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0035" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au2-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alla Karpova</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Karpova%2C+Alla" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au3-authorcard" aria-expanded="false" aria-controls="au3" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Marina A. Gritsenko">Marina A. Gritsenko</a> <a href="#" data-db-target-for="crosref0045" aria-expanded="false" aria-controls="crosref0045" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0045" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au3-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Marina A. Gritsenko</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Gritsenko%2C+Marina A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au4-authorcard" aria-expanded="false" aria-controls="au4" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jennifer E. Kyle">Jennifer E. Kyle</a> <a href="#" data-db-target-for="crosref0055" aria-expanded="false" aria-controls="crosref0055" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0055" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au4-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jennifer E. Kyle</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kyle%2C+Jennifer E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au5-authorcard" aria-expanded="false" aria-controls="au5" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Song Cao">Song Cao</a> <a href="#" data-db-target-for="crosref0070" aria-expanded="false" aria-controls="crosref0070" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0070" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au5-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Song Cao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cao%2C+Song" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au6-authorcard" aria-expanded="false" aria-controls="au6" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yize Li">Yize Li</a> <a href="#" data-db-target-for="crosref0085" aria-expanded="false" aria-controls="crosref0085" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>48</sup></span></a><div data-db-target-of="crosref0085" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au6-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yize Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">48 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Yize" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au7-authorcard" aria-expanded="false" aria-controls="au7" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Dmitry Rykunov">Dmitry Rykunov</a> <a href="#" data-db-target-for="crosref2515" aria-expanded="false" aria-controls="crosref2515" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2515" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au7-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Dmitry Rykunov</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rykunov%2C+Dmitry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au8-authorcard" aria-expanded="false" aria-controls="au8" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Antonio Colaprico">Antonio Colaprico</a> <a href="#" data-db-target-for="crosref2520" aria-expanded="false" aria-controls="crosref2520" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2520" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au8-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Antonio Colaprico</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA<p></p>Division of Biostatistics, Department of Public Health Science, University of Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Colaprico%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au9-authorcard" aria-expanded="false" aria-controls="au9" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Joseph H. Rothstein">Joseph H. Rothstein</a> <a href="#" data-db-target-for="crosref2525" aria-expanded="false" aria-controls="crosref2525" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2525" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au9-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Joseph H. Rothstein</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rothstein%2C+Joseph H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au10-authorcard" aria-expanded="false" aria-controls="au10" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Runyu Hong">Runyu Hong</a> <a href="#" data-db-target-for="crosref2530" aria-expanded="false" aria-controls="crosref2530" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2530" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au10-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Runyu Hong</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hong%2C+Runyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au11-authorcard" aria-expanded="false" aria-controls="au11" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Vasileios Stathias">Vasileios Stathias</a> <a href="#" data-db-target-for="crosref2535" aria-expanded="false" aria-controls="crosref2535" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2535" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au11-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Vasileios Stathias</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA<p></p>Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA<p></p>BD2K-LINCS Data Coordination and Integration Center, Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Stathias%2C+Vasileios" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au12-authorcard" aria-expanded="false" aria-controls="au12" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author MacIntosh Cornwell">MacIntosh Cornwell</a> <a href="#" data-db-target-for="crosref2540" aria-expanded="false" aria-controls="crosref2540" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2540" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au12-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">MacIntosh Cornwell</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cornwell%2C+MacIntosh" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au13-authorcard" aria-expanded="false" aria-controls="au13" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Francesca Petralia">Francesca Petralia</a> <a href="#" data-db-target-for="crosref2545" aria-expanded="false" aria-controls="crosref2545" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>49</sup></span></a><div data-db-target-of="crosref2545" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">49 These authors contributed equally<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au13-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Francesca Petralia</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">49 These authors contributed equally<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petralia%2C+Francesca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au14-authorcard" aria-expanded="false" aria-controls="au14" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yige Wu">Yige Wu</a><div data-db-target-of="au14-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yige Wu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wu%2C+Yige" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au15-authorcard" aria-expanded="false" aria-controls="au15" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Boris Reva">Boris Reva</a><div data-db-target-of="au15-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Boris Reva</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Reva%2C+Boris" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au16-authorcard" aria-expanded="false" aria-controls="au16" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karsten Krug">Karsten Krug</a><div data-db-target-of="au16-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karsten Krug</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krug%2C+Karsten" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au17-authorcard" aria-expanded="false" aria-controls="au17" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pietro Pugliese">Pietro Pugliese</a><div data-db-target-of="au17-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pietro Pugliese</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Science and Technology, University of Sannio, 82100, Benevento, Italy</div></div><a href="/action/doSearch?ContribAuthorRaw=Pugliese%2C+Pietro" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au18-authorcard" aria-expanded="false" aria-controls="au18" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily Kawaler">Emily Kawaler</a><div data-db-target-of="au18-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily Kawaler</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kawaler%2C+Emily" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au19-authorcard" aria-expanded="false" aria-controls="au19" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lindsey K. Olsen">Lindsey K. Olsen</a><div data-db-target-of="au19-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lindsey K. Olsen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Olsen%2C+Lindsey K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au20-authorcard" aria-expanded="false" aria-controls="au20" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wen-Wei Liang">Wen-Wei Liang</a><div data-db-target-of="au20-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wen-Wei Liang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liang%2C+Wen-Wei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au21-authorcard" aria-expanded="false" aria-controls="au21" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaoyu Song">Xiaoyu Song</a><div data-db-target-of="au21-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaoyu Song</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Song%2C+Xiaoyu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au22-authorcard" aria-expanded="false" aria-controls="au22" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Yongchao Dou">Yongchao Dou</a><div data-db-target-of="au22-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Yongchao Dou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Dou%2C+Yongchao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au23-authorcard" aria-expanded="false" aria-controls="au23" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michael C. Wendl">Michael C. Wendl</a><div data-db-target-of="au23-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michael C. Wendl</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Department of Genetics, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Department of Mathematics, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wendl%2C+Michael C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au24-authorcard" aria-expanded="false" aria-controls="au24" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wagma Caravan">Wagma Caravan</a><div data-db-target-of="au24-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wagma Caravan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Caravan%2C+Wagma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au25-authorcard" aria-expanded="false" aria-controls="au25" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Wenke Liu">Wenke Liu</a><div data-db-target-of="au25-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Wenke Liu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Wenke" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au26-authorcard" aria-expanded="false" aria-controls="au26" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Daniel Cui Zhou">Daniel Cui Zhou</a><div data-db-target-of="au26-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Daniel Cui Zhou</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cui Zhou%2C+Daniel" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au27-authorcard" aria-expanded="false" aria-controls="au27" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jiayi Ji">Jiayi Ji</a><div data-db-target-of="au27-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jiayi Ji</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ji%2C+Jiayi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au28-authorcard" aria-expanded="false" aria-controls="au28" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Chia-Feng Tsai">Chia-Feng Tsai</a><div data-db-target-of="au28-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Chia-Feng Tsai</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Tsai%2C+Chia-Feng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au29-authorcard" aria-expanded="false" aria-controls="au29" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Vladislav A. Petyuk">Vladislav A. Petyuk</a><div data-db-target-of="au29-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Vladislav A. Petyuk</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Petyuk%2C+Vladislav A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au30-authorcard" aria-expanded="false" aria-controls="au30" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jamie Moon">Jamie Moon</a><div data-db-target-of="au30-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jamie Moon</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Moon%2C+Jamie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au31-authorcard" aria-expanded="false" aria-controls="au31" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Weiping Ma">Weiping Ma</a><div data-db-target-of="au31-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Weiping Ma</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ma%2C+Weiping" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au32-authorcard" aria-expanded="false" aria-controls="au32" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rosalie K. Chu">Rosalie K. Chu</a><div data-db-target-of="au32-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rosalie K. Chu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chu%2C+Rosalie K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au33-authorcard" aria-expanded="false" aria-controls="au33" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karl K. Weitz">Karl K. Weitz</a><div data-db-target-of="au33-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karl K. Weitz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Weitz%2C+Karl K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au34-authorcard" aria-expanded="false" aria-controls="au34" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ronald J. Moore">Ronald J. Moore</a><div data-db-target-of="au34-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ronald J. Moore</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Moore%2C+Ronald J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au35-authorcard" aria-expanded="false" aria-controls="au35" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew E. Monroe">Matthew E. Monroe</a><div data-db-target-of="au35-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew E. Monroe</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Monroe%2C+Matthew E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au36-authorcard" aria-expanded="false" aria-controls="au36" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Rui Zhao">Rui Zhao</a><div data-db-target-of="au36-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Rui Zhao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Rui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au37-authorcard" aria-expanded="false" aria-controls="au37" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xiaolu Yang">Xiaolu Yang</a><div data-db-target-of="au37-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xiaolu Yang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Yang%2C+Xiaolu" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au38-authorcard" aria-expanded="false" aria-controls="au38" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Seungyeul Yoo">Seungyeul Yoo</a><div data-db-target-of="au38-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Seungyeul Yoo</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Yoo%2C+Seungyeul" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au39-authorcard" aria-expanded="false" aria-controls="au39" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Azra Krek">Azra Krek</a><div data-db-target-of="au39-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Azra Krek</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Krek%2C+Azra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au40-authorcard" aria-expanded="false" aria-controls="au40" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexis Demopoulos">Alexis Demopoulos</a><div data-db-target-of="au40-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexis Demopoulos</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Neurology, Northwell Health System, Lake Success, NY 11042 USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Demopoulos%2C+Alexis" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au41-authorcard" aria-expanded="false" aria-controls="au41" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Houxiang Zhu">Houxiang Zhu</a><div data-db-target-of="au41-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Houxiang Zhu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Houxiang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au42-authorcard" aria-expanded="false" aria-controls="au42" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Matthew A. Wyczalkowski">Matthew A. Wyczalkowski</a><div data-db-target-of="au42-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Matthew A. Wyczalkowski</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wyczalkowski%2C+Matthew A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au43-authorcard" aria-expanded="false" aria-controls="au43" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Joshua F. McMichael">Joshua F. McMichael</a><div data-db-target-of="au43-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Joshua F. McMichael</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=McMichael%2C+Joshua F" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au44-authorcard" aria-expanded="false" aria-controls="au44" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Brittany L. Henderson">Brittany L. Henderson</a><div data-db-target-of="au44-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Brittany L. Henderson</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Henderson%2C+Brittany L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au45-authorcard" aria-expanded="false" aria-controls="au45" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Caleb M. Lindgren">Caleb M. Lindgren</a><div data-db-target-of="au45-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Caleb M. Lindgren</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lindgren%2C+Caleb M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au46-authorcard" aria-expanded="false" aria-controls="au46" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Hannah Boekweg">Hannah Boekweg</a><div data-db-target-of="au46-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Hannah Boekweg</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boekweg%2C+Hannah" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au47-authorcard" aria-expanded="false" aria-controls="au47" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shuangjia Lu">Shuangjia Lu</a><div data-db-target-of="au47-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shuangjia Lu</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Lu%2C+Shuangjia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au48-authorcard" aria-expanded="false" aria-controls="au48" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jessika Baral">Jessika Baral</a><div data-db-target-of="au48-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jessika Baral</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Baral%2C+Jessika" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au49-authorcard" aria-expanded="false" aria-controls="au49" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lijun Yao">Lijun Yao</a><div data-db-target-of="au49-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lijun Yao</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Yao%2C+Lijun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au50-authorcard" aria-expanded="false" aria-controls="au50" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kelly G. Stratton">Kelly G. Stratton</a><div data-db-target-of="au50-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kelly G. Stratton</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Stratton%2C+Kelly G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au51-authorcard" aria-expanded="false" aria-controls="au51" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Lisa M. Bramer">Lisa M. Bramer</a><div data-db-target-of="au51-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Lisa M. Bramer</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Bramer%2C+Lisa M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au52-authorcard" aria-expanded="false" aria-controls="au52" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Erika Zink">Erika Zink</a><div data-db-target-of="au52-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Erika Zink</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zink%2C+Erika" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au53-authorcard" aria-expanded="false" aria-controls="au53" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sneha P. Couvillion">Sneha P. Couvillion</a><div data-db-target-of="au53-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sneha P. Couvillion</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Couvillion%2C+Sneha P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au54-authorcard" aria-expanded="false" aria-controls="au54" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kent J. Bloodsworth">Kent J. Bloodsworth</a><div data-db-target-of="au54-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kent J. Bloodsworth</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Bloodsworth%2C+Kent J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au55-authorcard" aria-expanded="false" aria-controls="au55" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Shankha Satpathy">Shankha Satpathy</a><div data-db-target-of="au55-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Shankha Satpathy</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Satpathy%2C+Shankha" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au56-authorcard" aria-expanded="false" aria-controls="au56" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Weiva Sieh">Weiva Sieh</a><div data-db-target-of="au56-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Weiva Sieh</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA<p></p>Department of Population Health Science and Policy, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Sieh%2C+Weiva" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au57-authorcard" aria-expanded="false" aria-controls="au57" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Simina M. Boca">Simina M. Boca</a><div data-db-target-of="au57-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Simina M. Boca</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC 20007, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boca%2C+Simina M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au58-authorcard" aria-expanded="false" aria-controls="au58" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Stephan Schürer">Stephan Schürer</a><div data-db-target-of="au58-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Stephan Schürer</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA<p></p>Department of Molecular and Cellular Pharmacology, Miller School of Medicine, University of Miami, Miami, FL 33136, USA<p></p>BD2K-LINCS Data Coordination and Integration Center, Miami, FL 33136, USA<p></p>Institute for Data Science &amp; Computing, University of Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Schürer%2C+Stephan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au59-authorcard" aria-expanded="false" aria-controls="au59" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Feng Chen">Feng Chen</a><div data-db-target-of="au59-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Feng Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Department of Cell Biology and Physiology, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Feng" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au60-authorcard" aria-expanded="false" aria-controls="au60" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Maciej Wiznerowicz">Maciej Wiznerowicz</a><div data-db-target-of="au60-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Maciej Wiznerowicz</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">International Institute for Molecular Oncology, 60-203 Poznań, Poland<p></p>Poznań University of Medical Sciences, 61-701 Poznań, Poland</div></div><a href="/action/doSearch?ContribAuthorRaw=Wiznerowicz%2C+Maciej" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au61-authorcard" aria-expanded="false" aria-controls="au61" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karen A. Ketchum">Karen A. Ketchum</a><div data-db-target-of="au61-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karen A. Ketchum</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">ESAC, Inc., Rockville, MD 20850, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ketchum%2C+Karen A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au62-authorcard" aria-expanded="false" aria-controls="au62" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Emily S. Boja">Emily S. Boja</a><div data-db-target-of="au62-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Emily S. Boja</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Boja%2C+Emily S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au63-authorcard" aria-expanded="false" aria-controls="au63" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Christopher R. Kinsinger">Christopher R. Kinsinger</a><div data-db-target-of="au63-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Christopher R. Kinsinger</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kinsinger%2C+Christopher R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au64-authorcard" aria-expanded="false" aria-controls="au64" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Ana I. Robles">Ana I. Robles</a><div data-db-target-of="au64-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Ana I. Robles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Robles%2C+Ana I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au65-authorcard" aria-expanded="false" aria-controls="au65" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tara Hiltke">Tara Hiltke</a><div data-db-target-of="au65-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tara Hiltke</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Hiltke%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au66-authorcard" aria-expanded="false" aria-controls="au66" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mathangi Thiagarajan">Mathangi Thiagarajan</a><div data-db-target-of="au66-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mathangi Thiagarajan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Frederick National Laboratory for Cancer Research, Frederick, MD 21702, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Thiagarajan%2C+Mathangi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au67-authorcard" aria-expanded="false" aria-controls="au67" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Alexey I. Nesvizhskii">Alexey I. Nesvizhskii</a><div data-db-target-of="au67-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Alexey I. Nesvizhskii</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, University of Michigan, Ann Arbor, MI 48109, USA<p></p>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Nesvizhskii%2C+Alexey I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au68-authorcard" aria-expanded="false" aria-controls="au68" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Bing Zhang">Bing Zhang</a><div data-db-target-of="au68-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Bing Zhang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Lester and Sue Smith Breast Center, Baylor College of Medicine, Houston, TX 77030, USA<p></p>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX 77030, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Bing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au69-authorcard" aria-expanded="false" aria-controls="au69" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author D.R. Mani">D.R. Mani</a><div data-db-target-of="au69-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">D.R. Mani</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA 02142, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mani%2C+D.R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au70-authorcard" aria-expanded="false" aria-controls="au70" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Michele Ceccarelli">Michele Ceccarelli</a><div data-db-target-of="au70-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Michele Ceccarelli</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Electrical Engineering and Information Technology, University of Naples "Federico II", 80128, Naples, Italy<p></p>BIOGEM, 83031 Ariano Irpino, Italy</div></div><a href="/action/doSearch?ContribAuthorRaw=Ceccarelli%2C+Michele" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au71-authorcard" aria-expanded="false" aria-controls="au71" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Xi S. Chen">Xi S. Chen</a><div data-db-target-of="au71-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Xi S. Chen</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Sylvester Comprehensive Cancer Center, University of Miami, FL 33136, USA<p></p>Division of Biostatistics, Department of Public Health Science, University of Miami, FL 33136, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Xi S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au72-authorcard" aria-expanded="false" aria-controls="au72" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Sandra L. Cottingham">Sandra L. Cottingham</a><div data-db-target-of="au72-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Sandra L. Cottingham</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Spectrum Health and Helen DeVos Children's Hospital, Grand Rapids, MI 49503, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Cottingham%2C+Sandra L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au73-authorcard" aria-expanded="false" aria-controls="au73" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Qing Kay Li">Qing Kay Li</a><div data-db-target-of="au73-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Qing Kay Li</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Pathology, Johns Hopkins University, Baltimore, MD 21231, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Qing Kay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au74-authorcard" aria-expanded="false" aria-controls="au74" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Albert H. Kim">Albert H. Kim</a><div data-db-target-of="au74-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Albert H. Kim</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Neurological Surgery, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Kim%2C+Albert H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au75-authorcard" aria-expanded="false" aria-controls="au75" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author David Fenyö">David Fenyö</a><div data-db-target-of="au75-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">David Fenyö</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Biochemistry and Molecular Pharmacology, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Fenyö%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au76-authorcard" aria-expanded="false" aria-controls="au76" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Kelly V. Ruggles">Kelly V. Ruggles</a><div data-db-target-of="au76-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Kelly V. Ruggles</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Systems Genetics, NYU Grossman School of Medicine, New York, NY 10016, USA<p></p>Department of Medicine, NYU Grossman School of Medicine, New York, NY 10016, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ruggles%2C+Kelly V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au77-authorcard" aria-expanded="false" aria-controls="au77" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Henry Rodriguez">Henry Rodriguez</a><div data-db-target-of="au77-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Henry Rodriguez</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodriguez%2C+Henry" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au78-authorcard" aria-expanded="false" aria-controls="au78" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Mehdi Mesri">Mehdi Mesri</a><div data-db-target-of="au78-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Mehdi Mesri</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Office of Cancer Clinical Proteomics Research, National Cancer Institute, Bethesda, MD 20892, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Mesri%2C+Mehdi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au79-authorcard" aria-expanded="false" aria-controls="au79" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Samuel H. Payne">Samuel H. Payne</a><div data-db-target-of="au79-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Samuel H. Payne</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Biology, Brigham Young University, Provo, UT 84602, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Payne%2C+Samuel H" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au80-authorcard" aria-expanded="false" aria-controls="au80" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Adam C. Resnick">Adam C. Resnick</a><div data-db-target-of="au80-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Adam C. Resnick</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Center for Data Driven Discovery in Biomedicine, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA<p></p>Division of Neurosurgery, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Resnick%2C+Adam C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au81-authorcard" aria-expanded="false" aria-controls="au81" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Pei Wang">Pei Wang</a><div data-db-target-of="au81-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Pei Wang</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Genetics and Genomic Sciences, Icahn Institute of Genomics and Multiscale Biology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Pei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au82-authorcard" aria-expanded="false" aria-controls="au82" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Richard D. Smith">Richard D. Smith</a><div data-db-target-of="au82-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Richard D. Smith</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Smith%2C+Richard D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au83-authorcard" aria-expanded="false" aria-controls="au83" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Antonio Iavarone">Antonio Iavarone</a><div data-db-target-of="au83-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Antonio Iavarone</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Institute for Cancer Genetics, Columbia University Medical Center, New York, NY 10032, USA<p></p>Department of Neurology, Columbia University Medical Center, New York, NY 10032, USA<p></p>Department of Pathology and Cell Biology, Columbia University Medical Center, New York, NY 10032, USA<p></p>Herbert Irving Comprehensive Cancer Center, Columbia University Medical Center, New York, NY 10032, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Iavarone%2C+Antonio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au84-authorcard" aria-expanded="false" aria-controls="au84" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Milan G. Chheda">Milan G. Chheda</a><div data-db-target-of="au84-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Milan G. Chheda</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Department of Neurology, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Chheda%2C+Milan G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au85-authorcard" aria-expanded="false" aria-controls="au85" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Jill S. Barnholtz-Sloan">Jill S. Barnholtz-Sloan</a><div data-db-target-of="au85-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Jill S. Barnholtz-Sloan</div><div class="article-header__info__body"><div class="article-header__info__social"><ul class="rlist inline-bullet-list"></ul></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Case Comprehensive Cancer Center and Department of Population and Quantitative Health Sciences, Case Western Reserve University School of Medicine, Cleveland, OH 44106, USA<p></p>Research and Education, University Hospitals Health System, Cleveland, OH 44106, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Barnholtz-Sloan%2C+Jill S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au86-authorcard" aria-expanded="false" aria-controls="au86" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Karin D. Rodland">Karin D. Rodland<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au86-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Karin D. Rodland</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#bad1dbc8d3d494c8d5ded6dbd4defacad4d4d694ddd5cc" title="Send email to Karin D. Rodland"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA<p></p>Department of Cell, Developmental, and Cancer Biology, Oregon Health &amp; Science University, Portland, OR 97221, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Rodland%2C+Karin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Karin D. Rodland" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#2b404a5942450559444f474a454f6b5b454547054c445d"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au87-authorcard" aria-expanded="false" aria-controls="au87" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Tao Liu">Tao Liu<i aria-hidden="true" class="icon-gizmo-person faded"></i></a><div data-db-target-of="au87-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Tao Liu</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#6d190c02430104182d1d030301430a021b" title="Send email to Tao Liu"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Biological Sciences Division, Pacific Northwest National Laboratory, Richland, WA 99354, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Tao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Tao Liu" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#1561747a3b797c6055657b7b793b727a63"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item author"><div class="dropBlock article-header__info"><a href="#" data-db-target-for="au88-authorcard" aria-expanded="false" aria-controls="au88" class="loa__item__name article-header__info__ctrl loa__item__email" title="Correspondence information about the author Li Ding">Li Ding<i aria-hidden="true" class="icon-gizmo-person faded"></i></a> <a href="#" data-db-target-for="crosref2550" aria-expanded="false" aria-controls="crosref2550" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span><sup>50</sup></span></a><div data-db-target-of="crosref2550" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">50 Lead contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div data-db-target-of="au88-authorcard" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Li Ding</div><span class="article-header__info__group__label">Correspondence</span><div class="article-header__info__sub-label">Corresponding author</div><div class="article-header__info__body"><div class="article-header__info__social"><a class="article-header__info__email default" href="/cdn-cgi/l/email-protection#bcd0d8d5d2dbfccbc9cfc8d092d9d8c9" title="Send email to Li Ding"><i aria-hidden="true" class="icon-Email"></i><span>
                            Contact
                        </span></a></div><div class="article-header__info__group"><span class="article-header__info__group__label">Footnotes</span><div class="article-header__info__group__body">50 Lead contact<br/></div></div><div class="article-header__info__group"><span class="article-header__info__group__label">Affiliations</span><div class="article-header__info__group__body">Department of Medicine, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Department of Genetics, Washington University in St. Louis, St. Louis, MO 63130, USA<p></p>Siteman Cancer Center, Washington University in St. Louis, St. Louis, MO 63130, USA</div></div><a href="/action/doSearch?ContribAuthorRaw=Ding%2C+Li" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div><div class="dropBlock article-header__info"><a title="Send email to Li Ding" class="loa__item__name  loa__item__email" href="/cdn-cgi/l/email-protection#83efe7eaede4c3f4f6f0f7efade6e7f6"><i aria-hidden="true" class="icon-Email faded"></i></a></div></li><li class="loa__item collaboration-author author"><span><div class="dropBlock article-header__info"><a href="#" data-db-target-for="collab0010" aria-expanded="false" aria-controls="collab0010" class="loa__item__name article-header__info__ctrl loa__item__email">Clinical Proteomic Tumor Analysis Consortium</a><div data-db-target-of="collab0010" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Clinical Proteomic Tumor Analysis Consortium</div><div class="article-header__info__body"><span class="article-header__info__group__label">Authors List</span><ul class="rlist dropBlock__slider"><li class="loa__item"><a href="#" title="Anupriya Agarwal" class="dropBlock__slider__ctrl">Anupriya Agarwal</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anupriya back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anupriya Agarwal</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Agarwal%2C+Anupriya" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Mitual Amin" class="dropBlock__slider__ctrl">Mitual Amin</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Mitual back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Mitual Amin</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Amin%2C+Mitual" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eunkyung An" class="dropBlock__slider__ctrl">Eunkyung An</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eunkyung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eunkyung An</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=An%2C+Eunkyung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew L. Anderson" class="dropBlock__slider__ctrl">Matthew L. Anderson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew L. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew L. Anderson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Anderson%2C+Matthew L" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David W. Andrews" class="dropBlock__slider__ctrl">David W. Andrews</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David W. Andrews</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Andrews%2C+David W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Thomas Bauer" class="dropBlock__slider__ctrl">Thomas Bauer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Thomas back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Thomas Bauer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bauer%2C+Thomas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chet Birger" class="dropBlock__slider__ctrl">Chet Birger</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chet back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chet Birger</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birger%2C+Chet" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael J. Birrer" class="dropBlock__slider__ctrl">Michael J. Birrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael J. Birrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Birrer%2C+Michael J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lili Blumenberg" class="dropBlock__slider__ctrl">Lili Blumenberg</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lili back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lili Blumenberg</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Blumenberg%2C+Lili" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="William E. Bocik" class="dropBlock__slider__ctrl">William E. Bocik</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="William E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">William E. Bocik</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Bocik%2C+William E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Borate" class="dropBlock__slider__ctrl">Uma Borate</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Borate</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borate%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Melissa Borucki" class="dropBlock__slider__ctrl">Melissa Borucki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Melissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Melissa Borucki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Borucki%2C+Melissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Meghan C. Burke" class="dropBlock__slider__ctrl">Meghan C. Burke</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Meghan C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Meghan C. Burke</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Burke%2C+Meghan C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shuang Cai" class="dropBlock__slider__ctrl">Shuang Cai</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shuang back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shuang Cai</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cai%2C+Shuang" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anna P. Calinawan" class="dropBlock__slider__ctrl">Anna P. Calinawan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anna P. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anna P. Calinawan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Calinawan%2C+Anna P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Steven A. Carr" class="dropBlock__slider__ctrl">Steven A. Carr</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Steven A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Steven A. Carr</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Carr%2C+Steven A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sandra Cerda" class="dropBlock__slider__ctrl">Sandra Cerda</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sandra back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sandra Cerda</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cerda%2C+Sandra" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel W. Chan" class="dropBlock__slider__ctrl">Daniel W. Chan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel W. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel W. Chan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chan%2C+Daniel W" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alyssa Charamut" class="dropBlock__slider__ctrl">Alyssa Charamut</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alyssa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alyssa Charamut</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Charamut%2C+Alyssa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lin S. Chen" class="dropBlock__slider__ctrl">Lin S. Chen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lin S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lin S. Chen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chen%2C+Lin S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David Chesla" class="dropBlock__slider__ctrl">David Chesla</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David Chesla</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chesla%2C+David" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Arul M. Chinnaiyan" class="dropBlock__slider__ctrl">Arul M. Chinnaiyan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Arul M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Arul M. Chinnaiyan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chinnaiyan%2C+Arul M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shrabanti Chowdhury" class="dropBlock__slider__ctrl">Shrabanti Chowdhury</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shrabanti back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shrabanti Chowdhury</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Chowdhury%2C+Shrabanti" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marcin P. Cieślik" class="dropBlock__slider__ctrl">Marcin P. Cieślik</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marcin P. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marcin P. Cieślik</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Cieślik%2C+Marcin P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="David J. Clark" class="dropBlock__slider__ctrl">David J. Clark</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="David J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">David J. Clark</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Clark%2C+David J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Houston Culpepper" class="dropBlock__slider__ctrl">Houston Culpepper</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Houston back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Houston Culpepper</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Culpepper%2C+Houston" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tomasz Czernicki" class="dropBlock__slider__ctrl">Tomasz Czernicki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tomasz back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tomasz Czernicki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Czernicki%2C+Tomasz" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Fulvio D'Angelo" class="dropBlock__slider__ctrl">Fulvio D'Angelo</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Fulvio back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Fulvio D'Angelo</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=D'Angelo%2C+Fulvio" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jacob Day" class="dropBlock__slider__ctrl">Jacob Day</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jacob back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jacob Day</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Day%2C+Jacob" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephanie De Young" class="dropBlock__slider__ctrl">Stephanie De Young</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephanie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephanie De Young</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=De Young%2C+Stephanie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Emek Demir" class="dropBlock__slider__ctrl">Emek Demir</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Emek back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Emek Demir</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Demir%2C+Emek" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Saravana Mohan Dhanasekaran" class="dropBlock__slider__ctrl">Saravana Mohan Dhanasekaran</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Saravana Mohan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Saravana Mohan Dhanasekaran</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dhanasekaran%2C+Saravana Mohan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rajiv Dhir" class="dropBlock__slider__ctrl">Rajiv Dhir</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rajiv back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rajiv Dhir</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dhir%2C+Rajiv" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Marcin J. Domagalski" class="dropBlock__slider__ctrl">Marcin J. Domagalski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Marcin J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Marcin J. Domagalski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Domagalski%2C+Marcin J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Brian Druker" class="dropBlock__slider__ctrl">Brian Druker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Brian back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Brian Druker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Druker%2C+Brian" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elizabeth Duffy" class="dropBlock__slider__ctrl">Elizabeth Duffy</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elizabeth back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elizabeth Duffy</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Duffy%2C+Elizabeth" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Maureen Dyer" class="dropBlock__slider__ctrl">Maureen Dyer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Maureen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Maureen Dyer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Dyer%2C+Maureen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nathan J. Edwards" class="dropBlock__slider__ctrl">Nathan J. Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nathan J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nathan J. Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Nathan J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert Edwards" class="dropBlock__slider__ctrl">Robert Edwards</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert Edwards</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Edwards%2C+Robert" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kimberly Elburn" class="dropBlock__slider__ctrl">Kimberly Elburn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kimberly back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kimberly Elburn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Elburn%2C+Kimberly" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Matthew J. Ellis" class="dropBlock__slider__ctrl">Matthew J. Ellis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Matthew J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Matthew J. Ellis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ellis%2C+Matthew J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jennifer Eschbacher" class="dropBlock__slider__ctrl">Jennifer Eschbacher</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jennifer back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jennifer Eschbacher</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Eschbacher%2C+Jennifer" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alicia Francis" class="dropBlock__slider__ctrl">Alicia Francis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alicia back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alicia Francis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Francis%2C+Alicia" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stacey Gabriel" class="dropBlock__slider__ctrl">Stacey Gabriel</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stacey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stacey Gabriel</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabriel%2C+Stacey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nikolay Gabrovski" class="dropBlock__slider__ctrl">Nikolay Gabrovski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nikolay back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nikolay Gabrovski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gabrovski%2C+Nikolay" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Luciano Garofano" class="dropBlock__slider__ctrl">Luciano Garofano</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Luciano back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Luciano Garofano</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Garofano%2C+Luciano" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gad Getz" class="dropBlock__slider__ctrl">Gad Getz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gad back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gad Getz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Getz%2C+Gad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael A. Gillette" class="dropBlock__slider__ctrl">Michael A. Gillette</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael A. Gillette</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Gillette%2C+Michael A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Andrew K. Godwin" class="dropBlock__slider__ctrl">Andrew K. Godwin</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Andrew K. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Andrew K. Godwin</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Godwin%2C+Andrew K" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Denis Golbin" class="dropBlock__slider__ctrl">Denis Golbin</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Denis back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Denis Golbin</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Golbin%2C+Denis" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ziad Hanhan" class="dropBlock__slider__ctrl">Ziad Hanhan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ziad back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ziad Hanhan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hanhan%2C+Ziad" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Linda I. Hannick" class="dropBlock__slider__ctrl">Linda I. Hannick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Linda I. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Linda I. Hannick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hannick%2C+Linda I" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Pushpa Hariharan" class="dropBlock__slider__ctrl">Pushpa Hariharan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Pushpa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Pushpa Hariharan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hariharan%2C+Pushpa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Hindenach" class="dropBlock__slider__ctrl">Barbara Hindenach</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Hindenach</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hindenach%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Katherine A. Hoadley" class="dropBlock__slider__ctrl">Katherine A. Hoadley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Katherine A. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Katherine A. Hoadley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hoadley%2C+Katherine A" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Galen Hostetter" class="dropBlock__slider__ctrl">Galen Hostetter</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Galen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Galen Hostetter</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Hostetter%2C+Galen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chen Huang" class="dropBlock__slider__ctrl">Chen Huang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chen Huang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Huang%2C+Chen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric Jaehnig" class="dropBlock__slider__ctrl">Eric Jaehnig</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric Jaehnig</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jaehnig%2C+Eric" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Scott D. Jewell" class="dropBlock__slider__ctrl">Scott D. Jewell</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Scott D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Scott D. Jewell</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jewell%2C+Scott D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nan Ji" class="dropBlock__slider__ctrl">Nan Ji</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nan Ji</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ji%2C+Nan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Corbin D. Jones" class="dropBlock__slider__ctrl">Corbin D. Jones</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Corbin D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Corbin D. Jones</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Jones%2C+Corbin D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alcida Karz" class="dropBlock__slider__ctrl">Alcida Karz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alcida back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alcida Karz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Karz%2C+Alcida" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Wojciech Kaspera" class="dropBlock__slider__ctrl">Wojciech Kaspera</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Wojciech back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Wojciech Kaspera</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kaspera%2C+Wojciech" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lyndon Kim" class="dropBlock__slider__ctrl">Lyndon Kim</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lyndon back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lyndon Kim</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kim%2C+Lyndon" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ramani B. Kothadia" class="dropBlock__slider__ctrl">Ramani B. Kothadia</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ramani B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ramani B. Kothadia</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kothadia%2C+Ramani B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chandan Kumar-Sinha" class="dropBlock__slider__ctrl">Chandan Kumar-Sinha</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chandan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chandan Kumar-Sinha</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Kumar-Sinha%2C+Chandan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jonathan Lei" class="dropBlock__slider__ctrl">Jonathan Lei</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jonathan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jonathan Lei</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Lei%2C+Jonathan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Felipe D. Leprevost" class="dropBlock__slider__ctrl">Felipe D. Leprevost</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Felipe D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Felipe D. Leprevost</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Leprevost%2C+Felipe D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Kai Li" class="dropBlock__slider__ctrl">Kai Li</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Kai back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Kai Li</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Li%2C+Kai" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yuxing Liao" class="dropBlock__slider__ctrl">Yuxing Liao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yuxing back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yuxing Liao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liao%2C+Yuxing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jena Lilly" class="dropBlock__slider__ctrl">Jena Lilly</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jena back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jena Lilly</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Lilly%2C+Jena" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hongwei Liu" class="dropBlock__slider__ctrl">Hongwei Liu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hongwei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hongwei Liu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Liu%2C+Hongwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jan Lubínski" class="dropBlock__slider__ctrl">Jan Lubínski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jan Lubínski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Lubínski%2C+Jan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rashna Madan" class="dropBlock__slider__ctrl">Rashna Madan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rashna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rashna Madan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Madan%2C+Rashna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="William Maggio" class="dropBlock__slider__ctrl">William Maggio</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="William back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">William Maggio</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maggio%2C+William" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ewa Malc" class="dropBlock__slider__ctrl">Ewa Malc</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ewa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ewa Malc</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malc%2C+Ewa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Anna Malovannaya" class="dropBlock__slider__ctrl">Anna Malovannaya</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Anna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Anna Malovannaya</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Malovannaya%2C+Anna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sailaja Mareedu" class="dropBlock__slider__ctrl">Sailaja Mareedu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sailaja back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sailaja Mareedu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mareedu%2C+Sailaja" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sanford P. Markey" class="dropBlock__slider__ctrl">Sanford P. Markey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sanford P. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sanford P. Markey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Markey%2C+Sanford P" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Annette Marrero-Oliveras" class="dropBlock__slider__ctrl">Annette Marrero-Oliveras</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Annette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Annette Marrero-Oliveras</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Marrero-Oliveras%2C+Annette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nina Martinez" class="dropBlock__slider__ctrl">Nina Martinez</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nina back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nina Martinez</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Martinez%2C+Nina" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nicollette Maunganidze" class="dropBlock__slider__ctrl">Nicollette Maunganidze</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nicollette back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nicollette Maunganidze</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Maunganidze%2C+Nicollette" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jason E. McDermott" class="dropBlock__slider__ctrl">Jason E. McDermott</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jason E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jason E. McDermott</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McDermott%2C+Jason E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Peter B. McGarvey" class="dropBlock__slider__ctrl">Peter B. McGarvey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Peter B. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Peter B. McGarvey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGarvey%2C+Peter B" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="John McGee" class="dropBlock__slider__ctrl">John McGee</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="John back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">John McGee</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=McGee%2C+John" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Piotr Mieczkowski" class="dropBlock__slider__ctrl">Piotr Mieczkowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Piotr back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Piotr Mieczkowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Mieczkowski%2C+Piotr" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Simona Migliozzi" class="dropBlock__slider__ctrl">Simona Migliozzi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Simona back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Simona Migliozzi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Migliozzi%2C+Simona" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Francesmary Modugno" class="dropBlock__slider__ctrl">Francesmary Modugno</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Francesmary back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Francesmary Modugno</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Modugno%2C+Francesmary" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Rebecca Montgomery" class="dropBlock__slider__ctrl">Rebecca Montgomery</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Rebecca back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Rebecca Montgomery</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Montgomery%2C+Rebecca" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Chelsea J. Newton" class="dropBlock__slider__ctrl">Chelsea J. Newton</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Chelsea J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Chelsea J. Newton</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Newton%2C+Chelsea J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gilbert S. Omenn" class="dropBlock__slider__ctrl">Gilbert S. Omenn</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gilbert S. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gilbert S. Omenn</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Omenn%2C+Gilbert S" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Umut Ozbek" class="dropBlock__slider__ctrl">Umut Ozbek</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Umut back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Umut Ozbek</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ozbek%2C+Umut" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Oxana V. Paklina" class="dropBlock__slider__ctrl">Oxana V. Paklina</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Oxana V. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Oxana V. Paklina</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paklina%2C+Oxana V" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amanda G. Paulovich" class="dropBlock__slider__ctrl">Amanda G. Paulovich</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amanda G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amanda G. Paulovich</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Paulovich%2C+Amanda G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Amy M. Perou" class="dropBlock__slider__ctrl">Amy M. Perou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Amy M. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Amy M. Perou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Perou%2C+Amy M" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alexander R. Pico" class="dropBlock__slider__ctrl">Alexander R. Pico</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alexander R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alexander R. Pico</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pico%2C+Alexander R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Paul D. Piehowski" class="dropBlock__slider__ctrl">Paul D. Piehowski</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Paul D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Paul D. Piehowski</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Piehowski%2C+Paul D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dimitris G. Placantonakis" class="dropBlock__slider__ctrl">Dimitris G. Placantonakis</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dimitris G. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dimitris G. Placantonakis</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Placantonakis%2C+Dimitris G" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Larisa Polonskaya" class="dropBlock__slider__ctrl">Larisa Polonskaya</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Larisa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Larisa Polonskaya</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Polonskaya%2C+Larisa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Olga Potapova" class="dropBlock__slider__ctrl">Olga Potapova</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Olga back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Olga Potapova</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Potapova%2C+Olga" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Barbara Pruetz" class="dropBlock__slider__ctrl">Barbara Pruetz</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Barbara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Barbara Pruetz</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Pruetz%2C+Barbara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liqun Qi" class="dropBlock__slider__ctrl">Liqun Qi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liqun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liqun Qi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Qi%2C+Liqun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shakti Ramkissoon" class="dropBlock__slider__ctrl">Shakti Ramkissoon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shakti back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shakti Ramkissoon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Ramkissoon%2C+Shakti" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Adam Resnick" class="dropBlock__slider__ctrl">Adam Resnick</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Adam back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Adam Resnick</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Resnick%2C+Adam" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shannon Richey" class="dropBlock__slider__ctrl">Shannon Richey</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shannon back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shannon Richey</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Richey%2C+Shannon" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Gregory Riggins" class="dropBlock__slider__ctrl">Gregory Riggins</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Gregory back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Gregory Riggins</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Riggins%2C+Gregory" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Karna Robinson" class="dropBlock__slider__ctrl">Karna Robinson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Karna back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Karna Robinson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Robinson%2C+Karna" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Nancy Roche" class="dropBlock__slider__ctrl">Nancy Roche</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Nancy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Nancy Roche</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Roche%2C+Nancy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Daniel C. Rohrer" class="dropBlock__slider__ctrl">Daniel C. Rohrer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Daniel C. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Daniel C. Rohrer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rohrer%2C+Daniel C" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Brian R. Rood" class="dropBlock__slider__ctrl">Brian R. Rood</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Brian R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Brian R. Rood</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rood%2C+Brian R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Larissa Rossell" class="dropBlock__slider__ctrl">Larissa Rossell</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Larissa back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Larissa Rossell</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Rossell%2C+Larissa" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Sara R. Savage" class="dropBlock__slider__ctrl">Sara R. Savage</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Sara R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Sara R. Savage</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Savage%2C+Sara R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Eric E. Schadt" class="dropBlock__slider__ctrl">Eric E. Schadt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Eric E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Eric E. Schadt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Schadt%2C+Eric E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yan Shi" class="dropBlock__slider__ctrl">Yan Shi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yan back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yan Shi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Yan" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhiao Shi" class="dropBlock__slider__ctrl">Zhiao Shi</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhiao back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhiao Shi</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shi%2C+Zhiao" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yvonne Shutack" class="dropBlock__slider__ctrl">Yvonne Shutack</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yvonne back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yvonne Shutack</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Shutack%2C+Yvonne" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shilpi Singh" class="dropBlock__slider__ctrl">Shilpi Singh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shilpi back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shilpi Singh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Singh%2C+Shilpi" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Tara Skelly" class="dropBlock__slider__ctrl">Tara Skelly</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Tara back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Tara Skelly</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Skelly%2C+Tara" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Lori J. Sokoll" class="dropBlock__slider__ctrl">Lori J. Sokoll</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Lori J. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Lori J. Sokoll</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Sokoll%2C+Lori J" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jakub Stawicki" class="dropBlock__slider__ctrl">Jakub Stawicki</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jakub back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jakub Stawicki</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stawicki%2C+Jakub" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Stephen E. Stein" class="dropBlock__slider__ctrl">Stephen E. Stein</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Stephen E. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Stephen E. Stein</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Stein%2C+Stephen E" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="James Suh" class="dropBlock__slider__ctrl">James Suh</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="James back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">James Suh</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Suh%2C+James" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Wojciech Szopa" class="dropBlock__slider__ctrl">Wojciech Szopa</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Wojciech back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Wojciech Szopa</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Szopa%2C+Wojciech" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dave Tabor" class="dropBlock__slider__ctrl">Dave Tabor</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dave back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dave Tabor</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tabor%2C+Dave" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Donghui Tan" class="dropBlock__slider__ctrl">Donghui Tan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Donghui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Donghui Tan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tan%2C+Donghui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Darlene Tansil" class="dropBlock__slider__ctrl">Darlene Tansil</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Darlene back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Darlene Tansil</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tansil%2C+Darlene" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ratna R. Thangudu" class="dropBlock__slider__ctrl">Ratna R. Thangudu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ratna R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ratna R. Thangudu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Thangudu%2C+Ratna R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Cristina Tognon" class="dropBlock__slider__ctrl">Cristina Tognon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Cristina back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Cristina Tognon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tognon%2C+Cristina" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Elie Traer" class="dropBlock__slider__ctrl">Elie Traer</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Elie back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Elie Traer</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Traer%2C+Elie" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Shirley Tsang" class="dropBlock__slider__ctrl">Shirley Tsang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Shirley back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Shirley Tsang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tsang%2C+Shirley" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey Tyner" class="dropBlock__slider__ctrl">Jeffrey Tyner</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey Tyner</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Tyner%2C+Jeffrey" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Ki Sung Um" class="dropBlock__slider__ctrl">Ki Sung Um</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Ki Sung back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Ki Sung Um</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Um%2C+Ki Sung" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Dana R. Valley" class="dropBlock__slider__ctrl">Dana R. Valley</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Dana R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Dana R. Valley</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Valley%2C+Dana R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Suhas Vasaikar" class="dropBlock__slider__ctrl">Suhas Vasaikar</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Suhas back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Suhas Vasaikar</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vasaikar%2C+Suhas" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Negin Vatanian" class="dropBlock__slider__ctrl">Negin Vatanian</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Negin back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Negin Vatanian</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vatanian%2C+Negin" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Uma Velvulou" class="dropBlock__slider__ctrl">Uma Velvulou</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Uma back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Uma Velvulou</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Velvulou%2C+Uma" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Michael Vernon" class="dropBlock__slider__ctrl">Michael Vernon</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Michael back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Michael Vernon</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Vernon%2C+Michael" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Weiqing Wan" class="dropBlock__slider__ctrl">Weiqing Wan</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Weiqing back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Weiqing Wan</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wan%2C+Weiqing" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Junmei Wang" class="dropBlock__slider__ctrl">Junmei Wang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Junmei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Junmei Wang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wang%2C+Junmei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Alex Webster" class="dropBlock__slider__ctrl">Alex Webster</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Alex back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Alex Webster</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Webster%2C+Alex" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Bo Wen" class="dropBlock__slider__ctrl">Bo Wen</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Bo back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Bo Wen</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wen%2C+Bo" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jeffrey R. Whiteaker" class="dropBlock__slider__ctrl">Jeffrey R. Whiteaker</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jeffrey R. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jeffrey R. Whiteaker</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Whiteaker%2C+Jeffrey R" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="George D. Wilson" class="dropBlock__slider__ctrl">George D. Wilson</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="George D. back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">George D. Wilson</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Wilson%2C+George D" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Yuriy Zakhartsev" class="dropBlock__slider__ctrl">Yuriy Zakhartsev</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Yuriy back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Yuriy Zakhartsev</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zakhartsev%2C+Yuriy" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Robert Zelt" class="dropBlock__slider__ctrl">Robert Zelt</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Robert back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Robert Zelt</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zelt%2C+Robert" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Hui Zhang" class="dropBlock__slider__ctrl">Hui Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Hui back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Hui Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Hui" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Liwei Zhang" class="dropBlock__slider__ctrl">Liwei Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Liwei back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Liwei Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Liwei" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Zhen Zhang" class="dropBlock__slider__ctrl">Zhen Zhang</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Zhen back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Zhen Zhang</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhang%2C+Zhen" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Grace Zhao" class="dropBlock__slider__ctrl">Grace Zhao</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Grace back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Grace Zhao</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhao%2C+Grace" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li><li class="loa__item"><a href="#" title="Jun Zhu" class="dropBlock__slider__ctrl">Jun Zhu</a><div class="dropBlock__slider__wrapper"><a href="#" class="dropBlock__slider__back" title="Jun back to list"><i aria-hidden="true" class="icon-arrow_l"></i><span>Back to list</span></a><div class="article-header__info__label">Jun Zhu</div><div class="article-header__info__body"><a href="/action/doSearch?ContribAuthorRaw=Zhu%2C+Jun" class="author__search" target="_blank"><i class="icon-arrow_r"></i><span>Search for articles by this author</span></a></div></div></li></ul></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></span></li> <li class="loa__item"><div class="dropBlock article-header__info article-header__footnotes"><a href="#" data-db-target-for="foot-notes" aria-expanded="false" aria-controls="" class="loa__item__name faded article-header__info__ctrl article-header__footnotes"><span class="show">Show footnotes</span><span class="hide">Hide footnotes</span></a><div data-db-target-of="foot-notes" class="dropBlock__holder article-header__info__holder lefted"><div class="article-header__info__scrollable"><div class="article-header__info__label">Author Footnotes</div><div class="article-header__info__body"><div title="Footnotes" class="article-header__info__item article-header__info__date">48 These authors contributed equally<br/>49 These authors contributed equally<br/>50 Lead contact<br/></div></div><a href="#" title="Close info" class="dropBlock__ctrl--close"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div></div></li></ul><div class="inline-it"><span class="bulleted"><span><span class="article-header__access">Open Access</span></span></span><span class="article-header__publish-date bulleted"><span class="article-header__publish-date__label">Published:</span><span class="article-header__publish-date__value">February 11, 2021</span></span><span class="article-header__doi bulleted"><span class="article-header__doi__label">DOI:</span><a class="article-header__doi__value" href="https://doi.org/10.1016/j.ccell.2021.01.006">https://doi.org/10.1016/j.ccell.2021.01.006</a></span><span class="article-header__mark bulleted"><a id="open-crossmark" data-target="crossmark"><img id="crossmark-icon" width="150" src="//crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" loading="lazy"></img></a><meta name="dc.identifier" scheme="doi" content="doi:10.1016/j.ccell.2021.01.006"></meta></span></div></div><div class="article-header__bottom clearfix row"><div class="col-md-6 static-md"></div><div class="col-md-6 static-md"><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"></script><a href="https://plu.mx/plum/a/?doi=10.1016/j.ccell.2021.01.006" class="plumx-plum-print-popup plum-jbs-theme article-header__metrics" data-popup="left" data-hide-when-empty="true" data-hide-usage="true" data-pass-hidden-categories="true" data-site="plum" data-badge="true" data-size="small">Proteogenomic and metabolomic characterization of human glioblastoma</a></div></div></div><div class="article-header__dropZone pb-dropzone" data-pb-dropzone="articleNavigation-Dropzone"><div data-widget-id="54733c81-009b-4b22-b588-f003dedf997f" data-widget-name="UX3ArticleNavigation" class="content-navigation clearfix"><div class="content-navigation__btn__wrapper--prev"><a href="/cancer-cell/fulltext/S1535-6108(21)00041-6" title="Previous Article" class="content-navigation__btn--pre default"><span class="content-navigation__body pre"><span class="content-navigation__body__top">Previous Article</span><span class="content-navigation__body__title">Mutant p53 suppresses innate immune signaling to promote tumorigenesis</span></span><i aria-hidden="true" class="icon-arrow_l"></i></a></div><div class="content-navigation__extra"></div><div class="content-navigation__btn__wrapper--next"><a href="/cancer-cell/fulltext/S1535-6108(21)00108-2" title="Next Article" class="content-navigation__btn--next default"><span class="content-navigation__body next"><span class="content-navigation__body__top">Next Article</span><span class="content-navigation__body__title">MDMX acts as a pervasive preleukemic-to-acute myeloid leukemia transition mechanism</span></span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div>

            

        

    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div></div></div></div><div class="article__body"><button title="open table of sections" aria-label="open table of sections" type="button" class="left-side-nav__ctrl"><i aria-hidden="true" class="icon-sub-menu"></i></button><div class="left-side-nav col-md-2 col-lg-1"><ul role="menu" class="rlist left-side-nav__upper"></ul><ul role="menu" class="rlist left-side-nav__bottom"></ul></div><div class="adplaceholders hidden-xs hidden-sm not-sticky">
    

        


    

        



    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id='Marketplace-Recs'>
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async></script>
<div id="pbgrd-skyr-c"></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

</div><div class="container"><div class="row"><div class="col-md-2 col-lg-1 hidden-sm"></div><div class="col-md-7 col-lg-9 article__sections"><section><h2 class="top" id="secsectitle0020" data-left-hand-nav="Highlights" tabindex="0"><span class="top__sub"></span><span class="top__text">Highlights</span></h2><div class="section-paragraph"><ul class="ce-list--remove-bullets" id="ulist0010"><li class="ce-list--remove-bullets__list-item" id="u0010"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Phosphorylated PTPN11 and PLCG1 represent a signaling hub in RTK-altered tumors</div></li><li class="ce-list--remove-bullets__list-item" id="u0015"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Four immune GBM subtypes exist, characterized by distinct immune cell populations</div></li><li class="ce-list--remove-bullets__list-item" id="u0020"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Mesenchymal subtype EMT signature is specific to tumor cells but not to stroma</div></li><li class="ce-list--remove-bullets__list-item" id="u0025"><span class="label">•</span><div class="ce-list--remove-bullets__list-item__text">Histone H2B acetylation is enriched in classical GBMs with low macrophage content</div></li></ul></div></section><section><h2 class="top" id="secsectitle0010" data-left-hand-nav="Summary" tabindex="0"><span class="top__sub"></span><span class="top__text">Summary</span></h2><div class="section-paragraph"><div class="section-paragraph">Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for <em>EGFR</em>-, <em>TP53</em>-, and <em>RB1</em>-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in <em>IDH</em>-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.</div></div></section><section><h2 class="top" id="secsectitle0015" data-left-hand-nav="Graphical abstract" tabindex="0"><span class="top__sub"></span><span class="top__text">Graphical abstract</span></h2><div class="section-paragraph"><figure id="fx1" class="graphical-abstract" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg" alt="Figure thumbnail fx1" data-locator="undfig1" data-high-res="/cms/attachment/d1b52466-e8f3-4230-b2bc-63b112742bdd/fx1_lrg.jpg" data-large-image="/cms/attachment/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Graph. Abst." class="label figure__label">Graphical Abstract</span><span class="figure__title__text"></span></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/74e2f79c-8f57-42bc-9031-ae006b66c89d/fx1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/d1b52466-e8f3-4230-b2bc-63b112742bdd/fx1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=fx1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></section><section><h2 class="top" id="secsectitle0025" data-left-hand-nav="Keywords" tabindex="0"><span class="top__sub"></span><span class="top__text">Keywords</span></h2><ul class="rlist keywords-list inline-bullet-list"><li><a href="/action/doSearch?AllField=&#34;glioblastoma&#34;&amp;ISSN=1535-6108">glioblastoma</a></li><li><a href="/action/doSearch?AllField=&#34;proteomics&#34;&amp;ISSN=1535-6108">proteomics</a></li><li><a href="/action/doSearch?AllField=&#34;proteogenomics&#34;&amp;ISSN=1535-6108">proteogenomics</a></li><li><a href="/action/doSearch?AllField=&#34;signaling&#34;&amp;ISSN=1535-6108">signaling</a></li><li><a href="/action/doSearch?AllField=&#34;CPTAC&#34;&amp;ISSN=1535-6108">CPTAC</a></li><li><a href="/action/doSearch?AllField=&#34;metabolome&#34;&amp;ISSN=1535-6108">metabolome</a></li><li><a href="/action/doSearch?AllField=&#34;lipidome&#34;&amp;ISSN=1535-6108">lipidome</a></li><li><a href="/action/doSearch?AllField=&#34;acetylome&#34;&amp;ISSN=1535-6108">acetylome</a></li><li><a href="/action/doSearch?AllField=&#34;single nuclei RNA-seq&#34;&amp;ISSN=1535-6108">single nuclei RNA-seq</a></li></ul></section><div data-pb-dropzone="relatedItems-dropzone" class="relatedItems-dropzone pb-dropzone"><!-- Empty dropzone --></div><section id="sec1" data-locator="sec1" tabindex="0"><h2 class="top" id="secsectitle0030" data-left-hand-nav="Introduction" tabindex="0"><span class="top__sub"></span><span class="top__text">Introduction</span></h2><div class="section-paragraph">Glioblastoma (GBM) is the most common primary malignant brain tumor, with roughly 12,000 new cases annually in the United States and median survival under 2 years (<div class="dropBlock reference-citations"><a href="#bib32" id="back-bib32" data-db-target-for="bib32-d4222834e2133" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib32">Delgado-López and Corrales-García, 2016</a><div data-db-target-of="bib32-d4222834e2133" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Delgado-López P.D. </li><li class="loa__item">Corrales-García E.M. </li></ul><div class="reference-citations__detials">Survival in glioblastoma: a review on the impact of treatment modalities.<div class="popup-ref"><span class="ref__series"><i>Clin. Transl. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1062-1071</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs12094-016-1497-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26960561&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84960422466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=1062-1071&journal=Clin.+Transl.+Oncol.&author=Delgado-L%C3%B3pez+P.D.&author=Corrales-Garc%C3%ADa+E.M.&title=Survival+in+glioblastoma%3A+a+review+on+the+impact+of+treatment+modalities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib99" id="back-bib99" data-db-target-for="bib99-d4222834e2136" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib99">Ostrom et al., 2019</a><div data-db-target-of="bib99-d4222834e2136" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ostrom Q.T. </li><li class="loa__item">Cioffi G. </li><li class="loa__item">Gittleman H. </li><li class="loa__item">Patil N. </li><li class="loa__item">Waite K. </li><li class="loa__item">Kruchko C. </li><li class="loa__item">Barnholtz-Sloan J.S. </li></ul><div class="reference-citations__detials">CBTRUS statistical report: primary brain and other central nervous system tumors diagnosed in the United States in 2012-2016.<div class="popup-ref"><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">v1-v100</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnoz150&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31675094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85074400952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=v1-v100&journal=Neuro-Oncol&author=Ostrom+Q.T.&author=Cioffi+G.&author=Gittleman+H.&author=Patil+N.&author=Waite+K.&author=Kruchko+C.&author=Barnholtz-Sloan+J.S.&title=CBTRUS+statistical+report%3A+primary+brain+and+other+central+nervous+system+tumors+diagnosed+in+the+United+States+in+2012-2016">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The Cancer Genome Atlas (TCGA) (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4222834e2139" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brennan et al., 2013</a><div data-db-target-of="bib18-d4222834e2139" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brennan C.W. </li><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Campos B. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Salama S.R. </li><li class="loa__item">Zheng S. </li><li class="loa__item">Chakravarty D. </li><li class="loa__item">Sanborn J.Z. </li><li class="loa__item">Berman S.H. </li><li>et al.</li></ul><div class="reference-citations__detials">The somatic genomic landscape of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib127" id="back-bib127" data-db-target-for="bib127-d4222834e2142" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib127">The Cancer Genome Atlas Research Network, 2008</a><div data-db-target-of="bib127-d4222834e2142" class="dropBlock__holder"><div class="dropBlock__body"><div class="reference-citations__detials"><span>The Cancer Genome Atlas Research Network</span><br />Comprehensive genomic characterization defines human glioblastoma genes and core pathways.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">455</span></span>: <span class="ref__seriesPages">1061-1068</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature07385&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18772890&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-54549108740&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4993)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=1061-1068&journal=Nature&author=The+Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+defines+human+glioblastoma+genes+and+core+pathways">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and other studies (<div class="dropBlock reference-citations"><a href="#bib139" id="back-bib139" data-db-target-for="bib139-d4222834e2145" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib139">Yan et al., 2009</a><div data-db-target-of="bib139-d4222834e2145" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yan H. </li><li class="loa__item">Parsons D.W. </li><li class="loa__item">Jin G. </li><li class="loa__item">McLendon R. </li><li class="loa__item">Rasheed B.A. </li><li class="loa__item">Yuan W. </li><li class="loa__item">Kos I. </li><li class="loa__item">Batinic-Haberle I. </li><li class="loa__item">Jones S. </li><li class="loa__item">Riggins G.J. </li><li>et al.</li></ul><div class="reference-citations__detials">IDH1 and IDH2 mutations in gliomas.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">360</span></span>: <span class="ref__seriesPages">765-773</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa0808710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19228619&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-60849115270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3455)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2009&pages=765-773&journal=N.%C2%A0Engl.+J.+Med.&author=Yan+H.&author=Parsons+D.W.&author=Jin+G.&author=McLendon+R.&author=Rasheed+B.A.&author=Yuan+W.&author=Kos+I.&author=Batinic-Haberle+I.&author=Jones+S.&author=Riggins+G.J.&title=IDH1+and+IDH2+mutations+in+gliomas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) have reshaped the World Health Organization classification of nervous system tumors (<div class="dropBlock reference-citations"><a href="#bib84" id="back-bib84" data-db-target-for="bib84-d4222834e2149" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib84">Louis et al., 2016</a><div data-db-target-of="bib84-d4222834e2149" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Louis D.N. </li><li class="loa__item">Perry A. </li><li class="loa__item">Reifenberger G. </li><li class="loa__item">von Deimling A. </li><li class="loa__item">Figarella-Branger D. </li><li class="loa__item">Cavenee W.K. </li><li class="loa__item">Ohgaki H. </li><li class="loa__item">Wiestler O.D. </li><li class="loa__item">Kleihues P. </li><li class="loa__item">Ellison D.W. </li></ul><div class="reference-citations__detials">The 2016 World health organization classification of tumors of the central nervous system: a summary.<div class="popup-ref"><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">131</span></span>: <span class="ref__seriesPages">803-820</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-016-1545-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27157931&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2016&pages=803-820&journal=Acta+Neuropathol.&author=Louis+D.N.&author=Perry+A.&author=Reifenberger+G.&author=von+Deimling+A.&author=Figarella-Branger+D.&author=Cavenee+W.K.&author=Ohgaki+H.&author=Wiestler+O.D.&author=Kleihues+P.&author=Ellison+D.W.&title=The+2016+World+health+organization+classification+of+tumors+of+the+central+nervous+system%3A+a+summary">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to include molecular features (<div class="dropBlock reference-citations"><a href="#bib17" id="back-bib17" data-db-target-for="bib17-d4222834e2152" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib17">Brat et al., 2018</a><div data-db-target-of="bib17-d4222834e2152" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brat D.J. </li><li class="loa__item">Aldape K. </li><li class="loa__item">Colman H. </li><li class="loa__item">Holland E.C. </li><li class="loa__item">Louis D.N. </li><li class="loa__item">Jenkins R.B. </li><li class="loa__item">Kleinschmidt-DeMasters B.K. </li><li class="loa__item">Perry A. </li><li class="loa__item">Reifenberger G. </li><li class="loa__item">Stupp R. </li><li>et al.</li></ul><div class="reference-citations__detials">cIMPACT-NOW update 3: recommended diagnostic criteria for “Diffuse astrocytic glioma, IDH-wildtype, with molecular features of glioblastoma, WHO grade IV.<div class="popup-ref"><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">136</span></span>: <span class="ref__seriesPages">805-810</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-018-1913-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30259105&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85053870232&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2018&pages=805-810&journal=Acta+Neuropathol.&author=Brat+D.J.&author=Aldape+K.&author=Colman+H.&author=Holland+E.C.&author=Louis+D.N.&author=Jenkins+R.B.&author=Kleinschmidt-DeMasters+B.K.&author=Perry+A.&author=Reifenberger+G.&author=Stupp+R.&title=cIMPACT-NOW+update+3%3A+recommended+diagnostic+criteria+for+%E2%80%9CDiffuse+astrocytic+glioma%2C+IDH-wildtype%2C+with+molecular+features+of+glioblastoma%2C+WHO+grade+IV">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib85" id="back-bib85" data-db-target-for="bib85-d4222834e2155" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib85">Louis et al., 2017</a><div data-db-target-of="bib85-d4222834e2155" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Louis D.N. </li><li class="loa__item">Aldape K. </li><li class="loa__item">Brat D.J. </li><li class="loa__item">Capper D. </li><li class="loa__item">Ellison D.W. </li><li class="loa__item">Hawkins C. </li><li class="loa__item">Paulus W. </li><li class="loa__item">Perry A. </li><li class="loa__item">Reifenberger G. </li><li class="loa__item">Figarella-Branger D. </li><li>et al.</li></ul><div class="reference-citations__detials">Announcing cIMPACT-NOW: the consortium to inform molecular and practical approaches to CNS tumor taxonomy.<div class="popup-ref"><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">133</span></span>: <span class="ref__seriesPages">1-3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-016-1646-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27909809&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85001090602&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=1-3&journal=Acta+Neuropathol.&author=Louis+D.N.&author=Aldape+K.&author=Brat+D.J.&author=Capper+D.&author=Ellison+D.W.&author=Hawkins+C.&author=Paulus+W.&author=Perry+A.&author=Reifenberger+G.&author=Figarella-Branger+D.&title=Announcing+cIMPACT-NOW%3A+the+consortium+to+inform+molecular+and+practical+approaches+to+CNS+tumor+taxonomy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). GBM is categorized as either <em>IDH</em>-wild type (<em>IDH</em>-WT; ∼90%) or <em>IDH</em>-mutant (∼10%). <em>IDH</em>-WT GBMs fall into three distinct subclasses (proneural, classical, and mesenchymal) based on genomic alterations and gene expression signatures (<div class="dropBlock reference-citations"><a href="#bib131" id="back-bib131" data-db-target-for="bib131-d4222834e2167" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib131">Verhaak et al., 2010</a><div data-db-target-of="bib131-d4222834e2167" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">Hoadley K.A. </li><li class="loa__item">Purdom E. </li><li class="loa__item">Wang V. </li><li class="loa__item">Qi Y. </li><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Miller C.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Golub T. </li><li class="loa__item">Mesirov J.P. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">98-110</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20129251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-73649123907&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4057)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=98-110&journal=Cancer+Cell&author=Verhaak+R.G.W.&author=Hoadley+K.A.&author=Purdom+E.&author=Wang+V.&author=Qi+Y.&author=Wilkerson+M.D.&author=Miller+C.R.&author=Ding+L.&author=Golub+T.&author=Mesirov+J.P.&title=Integrated+genomic+analysis+identifies+clinically+relevant+subtypes+of+glioblastoma+characterized+by+abnormalities+in+PDGFRA%2C+IDH1%2C+EGFR%2C+and+NF1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d4222834e2170" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Wang et al., 2017</a><div data-db-target-of="bib133-d4222834e2170" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang Q. </li><li class="loa__item">Hu B. </li><li class="loa__item">Hu X. </li><li class="loa__item">Kim H. </li><li class="loa__item">Squatrito M. </li><li class="loa__item">Scarpace L. </li><li class="loa__item">deCarvalho A.C. </li><li class="loa__item">Lyu S. </li><li class="loa__item">Li P. </li><li class="loa__item">Li Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">42-56.e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28697342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85027444208&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=42-56.e6&journal=Cancer+Cell&author=Wang+Q.&author=Hu+B.&author=Hu+X.&author=Kim+H.&author=Squatrito+M.&author=Scarpace+L.&author=deCarvalho+A.C.&author=Lyu+S.&author=Li+P.&author=Li+Y.&title=Tumor+evolution+of+glioma-intrinsic+gene+expression+subtypes+associates+with+immunological+changes+in+the+microenvironment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Methylome-based classification is being used to differentially diagnose brain tumors (<div class="dropBlock reference-citations"><a href="#bib58" id="back-bib58" data-db-target-for="bib58-d4222834e2173" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib58">Karimi et al., 2019</a><div data-db-target-of="bib58-d4222834e2173" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karimi S. </li><li class="loa__item">Zuccato J.A. </li><li class="loa__item">Mamatjan Y. </li><li class="loa__item">Mansouri S. </li><li class="loa__item">Suppiah S. </li><li class="loa__item">Nassiri F. </li><li class="loa__item">Diamandis P. </li><li class="loa__item">Munoz D.G. </li><li class="loa__item">Aldape K.D. </li><li class="loa__item">Zadeh G. </li></ul><div class="reference-citations__detials">The central nervous system tumor methylation classifier changes neuro-oncology practice for challenging brain tumor diagnoses and directly impacts patient care.<div class="popup-ref"><span class="ref__series"><i>Clin. Epigenetics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">185</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13148-019-0766-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31806041&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85076173211&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=185&journal=Clin.+Epigenetics&author=Karimi+S.&author=Zuccato+J.A.&author=Mamatjan+Y.&author=Mansouri+S.&author=Suppiah+S.&author=Nassiri+F.&author=Diamandis+P.&author=Munoz+D.G.&author=Aldape+K.D.&author=Zadeh+G.&title=The+central+nervous+system+tumor+methylation+classifier+changes+neuro-oncology+practice+for+challenging+brain+tumor+diagnoses+and+directly+impacts+patient+care">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib97" id="back-bib97" data-db-target-for="bib97-d4222834e2176" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib97">Nassiri et al., 2019</a><div data-db-target-of="bib97-d4222834e2176" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nassiri F. </li><li class="loa__item">Mamatjan Y. </li><li class="loa__item">Suppiah S. </li><li class="loa__item">Badhiwala J.H. </li><li class="loa__item">Mansouri S. </li><li class="loa__item">Karimi S. </li><li class="loa__item">Saarela O. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Gepfner-Tuma I. </li><li class="loa__item">Schittenhelm J. </li><li>et al.</li></ul><div class="reference-citations__detials">DNA methylation profiling to predict recurrence risk in meningioma: development and validation of a nomogram to optimize clinical management.<div class="popup-ref"><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">901-910</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnoz061&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31158293&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071162289&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=901-910&journal=Neuro-Oncol.&author=Nassiri+F.&author=Mamatjan+Y.&author=Suppiah+S.&author=Badhiwala+J.H.&author=Mansouri+S.&author=Karimi+S.&author=Saarela+O.&author=Poisson+L.&author=Gepfner-Tuma+I.&author=Schittenhelm+J.&title=DNA+methylation+profiling+to+predict+recurrence+risk+in+meningioma%3A+development+and+validation+of+a+nomogram+to+optimize+clinical+management">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and may become clinically useful for GBM.</div><div class="section-paragraph">Surgical resection, chemotherapy, and radiotherapy remain the standard of care (<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d4222834e2181" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Stupp et al., 2005</a><div data-db-target-of="bib119-d4222834e2181" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stupp R. </li><li class="loa__item">Mason W.P. </li><li class="loa__item">van den Bent M.J. </li><li class="loa__item">Weller M. </li><li class="loa__item">Fisher B. </li><li class="loa__item">Taphoorn M.J.B. </li><li class="loa__item">Belanger K. </li><li class="loa__item">Brandes A.A. </li><li class="loa__item">Marosi C. </li><li class="loa__item">Bogdahn U. </li><li>et al.</li></ul><div class="reference-citations__detials">Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">352</span></span>: <span class="ref__seriesPages">987-996</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa043330&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15758009&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044366163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12069)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=987-996&journal=N.%C2%A0Engl.+J.+Med.&author=Stupp+R.&author=Mason+W.P.&author=van+den+Bent+M.J.&author=Weller+M.&author=Fisher+B.&author=Taphoorn+M.J.B.&author=Belanger+K.&author=Brandes+A.A.&author=Marosi+C.&author=Bogdahn+U.&title=Radiotherapy+plus+concomitant+and+adjuvant+temozolomide+for+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib104" id="back-bib104" data-db-target-for="bib104-d4222834e2184" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib104">Perry et al., 2017</a><div data-db-target-of="bib104-d4222834e2184" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Perry J.R. </li><li class="loa__item">Laperriere N. </li><li class="loa__item">O’Callaghan C.J. </li><li class="loa__item">Brandes A.A. </li><li class="loa__item">Menten J. </li><li class="loa__item">Phillips C. </li><li class="loa__item">Fay M. </li><li class="loa__item">Nishikawa R. </li><li class="loa__item">Cairncross J.G. </li><li class="loa__item">Roa W. </li><li>et al.</li></ul><div class="reference-citations__detials">Short-course radiation plus temozolomide in elderly patients with glioblastoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">376</span></span>: <span class="ref__seriesPages">1027-1037</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa1611977&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28296618&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85015918057&amp;cf=" onclick="newWindow(this.href);return false">Scopus (357)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2017&pages=1027-1037&journal=N.%C2%A0Engl.+J.+Med.&author=Perry+J.R.&author=Laperriere+N.&author=O%E2%80%99Callaghan+C.J.&author=Brandes+A.A.&author=Menten+J.&author=Phillips+C.&author=Fay+M.&author=Nishikawa+R.&author=Cairncross+J.G.&author=Roa+W.&title=Short-course+radiation+plus+temozolomide+in+elderly+patients+with+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with the recent addition of tumor treating fields (<div class="dropBlock reference-citations"><a href="#bib120" id="back-bib120" data-db-target-for="bib120-d4222834e2187" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib120">Stupp et al., 2017</a><div data-db-target-of="bib120-d4222834e2187" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stupp R. </li><li class="loa__item">Taillibert S. </li><li class="loa__item">Kanner A. </li><li class="loa__item">Read W. </li><li class="loa__item">Steinberg D. </li><li class="loa__item">Lhermitte B. </li><li class="loa__item">Toms S. </li><li class="loa__item">Idbaih A. </li><li class="loa__item">Ahluwalia M.S. </li><li class="loa__item">Fink K. </li><li>et al.</li></ul><div class="reference-citations__detials">Effect of tumor-treating fields plus maintenance temozolomide vs maintenance temozolomide alone on survival in patients with glioblastoma: a randomized clinical trial.<div class="popup-ref"><span class="ref__series"><i>JAMA.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">318</span></span>: <span class="ref__seriesPages">2306-2316</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1001%2Fjama.2017.18718&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29260225&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040033962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (464)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2017&pages=2306-2316&journal=JAMA&author=Stupp+R.&author=Taillibert+S.&author=Kanner+A.&author=Read+W.&author=Steinberg+D.&author=Lhermitte+B.&author=Toms+S.&author=Idbaih+A.&author=Ahluwalia+M.S.&author=Fink+K.&title=Effect+of+tumor-treating+fields+plus+maintenance+temozolomide+vs+maintenance+temozolomide+alone+on+survival+in+patients+with+glioblastoma%3A+a+randomized+clinical+trial">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Promising immunotherapies have been proposed, including immune checkpoint inhibitors, vaccines, chimeric antigen receptor T cell (CAR-T) therapy, and viral therapy, though none have cleared Phase III trials (<div class="dropBlock reference-citations"><a href="#bib80" id="back-bib80" data-db-target-for="bib80-d4222834e2190" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib80">Lim et al., 2018</a><div data-db-target-of="bib80-d4222834e2190" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Lim M. </li><li class="loa__item">Xia Y. </li><li class="loa__item">Bettegowda C. </li><li class="loa__item">Weller M. </li></ul><div class="reference-citations__detials">Current state of immunotherapy for glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">422</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41571-018-0003-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29643471&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85045203677&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=422&journal=Nat.+Rev.+Clin.+Oncol.&author=Lim+M.&author=Xia+Y.&author=Bettegowda+C.&author=Weller+M.&title=Current+state+of+immunotherapy+for+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib89" id="back-bib89" data-db-target-for="bib89-d4222834e2193" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib89">McGranahan et al., 2019</a><div data-db-target-of="bib89-d4222834e2193" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McGranahan T. </li><li class="loa__item">Therkelsen K.E. </li><li class="loa__item">Ahmad S. </li><li class="loa__item">Nagpal S. </li></ul><div class="reference-citations__detials">Current state of immunotherapy for treatment of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Curr. Treat. Options Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">24</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11864-019-0619-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30790064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85061990008&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=24&journal=Curr.+Treat.+Options+Oncol.&author=McGranahan+T.&author=Therkelsen+K.E.&author=Ahmad+S.&author=Nagpal+S.&title=Current+state+of+immunotherapy+for+treatment+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Despite different subtypes, no specific treatment works more effectively in a pre-specified subset of patients based on transcriptomics, though those with MGMT promoter methylation respond better to temozolomide (<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d4222834e2197" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Stupp et al., 2005</a><div data-db-target-of="bib119-d4222834e2197" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stupp R. </li><li class="loa__item">Mason W.P. </li><li class="loa__item">van den Bent M.J. </li><li class="loa__item">Weller M. </li><li class="loa__item">Fisher B. </li><li class="loa__item">Taphoorn M.J.B. </li><li class="loa__item">Belanger K. </li><li class="loa__item">Brandes A.A. </li><li class="loa__item">Marosi C. </li><li class="loa__item">Bogdahn U. </li><li>et al.</li></ul><div class="reference-citations__detials">Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">352</span></span>: <span class="ref__seriesPages">987-996</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa043330&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15758009&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044366163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12069)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=987-996&journal=N.%C2%A0Engl.+J.+Med.&author=Stupp+R.&author=Mason+W.P.&author=van+den+Bent+M.J.&author=Weller+M.&author=Fisher+B.&author=Taphoorn+M.J.B.&author=Belanger+K.&author=Brandes+A.A.&author=Marosi+C.&author=Bogdahn+U.&title=Radiotherapy+plus+concomitant+and+adjuvant+temozolomide+for+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Here, we integrated proteogenomic and metabolomic data from 10 platforms including whole genome sequencing (WGS), whole exome sequencing (WES), RNA sequencing (RNA-seq), microRNA-seq (miRNA-seq), single nuclei RNA-seq (snRNA-seq), DNA methylation arrays, proteome, phosphoproteome, acetylome, lipidome, and metabolome to investigate 99 treatment-naive GBMs prospectively collected by the Clinical Proteomic Tumor Analysis Consortium (CPTAC). We report new immune-based subtypes, activation of DNA repair pathways via upregulated phosphosite levels of DNA repair genes in <em>TP53</em>-mutated tumors, an apparent phospho-signaling bottleneck in receptor tyrosine kinase (RTK)-altered tumors, and enrichment of histone H2B acetylation and low macrophage content in classical-like GBM tumors. We used single-cell data to investigate contributions of various cell types to bulk tumor signatures and analyzed the mesenchymal subtype to discern epithelial-mesenchymal transition (EMT) signatures in tumor and infiltrating immune cells. The data presented here furnish a resource for future GBM studies.</div></section><section id="sec2" data-locator="sec2" tabindex="0"><h2 class="top" id="secsectitle0035" data-left-hand-nav="Results" tabindex="0"><span class="top__sub"></span><span class="top__text">Results</span></h2><section id="sec2.1" data-locator="sec2.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Proteogenomic and metabolomic features delineate molecular subtypes of glioblastoma</span></h3><div class="section-paragraph">We characterized the proteogenomic landscape of 99 GBMs and 10 unmatched GTEx normal brain samples. This cohort has diverse origins and clinical characteristics typical of adult GBM (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/88e921e9-6b3e-4bd6-b49b-24db5cd273c8/mmc2" class="scroll-into-link">Table S1</a>). Six cases harbored <em>IDH1</em> R132H mutations and had earlier disease onset than those with <em>IDH1</em>-WT (median 47 vs. 59 years, t test p = 0.055). We detected one additional non-hotspot <em>IDH1</em> mutation (R222C).</div><div class="section-paragraph">All samples were homogenized and aliquoted for each of the ten different omics assays (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>A and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S1</a>A–S1C; <a href="#sec5" class="scroll-into-link">STAR methods</a>). Mass spectrometry (MS) quantified protein, phosphorylation and acetylation, as previously described (<div class="dropBlock reference-citations"><a href="#bib35" id="back-bib35" data-db-target-for="bib35-d4222834e2233" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib35">Dou et al., 2020</a><div data-db-target-of="bib35-d4222834e2233" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Dou Y. </li><li class="loa__item">Kawaler E.A. </li><li class="loa__item">Zhou D.C. </li><li class="loa__item">Gritsenko M.A. </li><li class="loa__item">Huang C. </li><li class="loa__item">Blumenberg L. </li><li class="loa__item">Karpova A. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Satpathy S. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic characterization of endometrial carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">729-748.e26</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=32059776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85079388976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (39)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=729-748.e26&journal=Cell&author=Dou+Y.&author=Kawaler+E.A.&author=Zhou+D.C.&author=Gritsenko+M.A.&author=Huang+C.&author=Blumenberg+L.&author=Karpova+A.&author=Petyuk+V.A.&author=Savage+S.R.&author=Satpathy+S.&title=Proteogenomic+characterization+of+endometrial+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib92" id="back-bib92" data-db-target-for="bib92-d4222834e2236" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib92">Mertins et al., 2016</a><div data-db-target-of="bib92-d4222834e2236" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mertins P. </li><li class="loa__item">Mani D.R. </li><li class="loa__item">Ruggles K.V. </li><li class="loa__item">Gillette M.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Wang P. </li><li class="loa__item">Wang X. </li><li class="loa__item">Qiao J.W. </li><li class="loa__item">Cao S. </li><li class="loa__item">Petralia F. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomics connects somatic mutations to signalling in breast cancer.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (664)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=Mertins+P.&author=Mani+D.R.&author=Ruggles+K.V.&author=Gillette+M.A.&author=Clauser+K.R.&author=Wang+P.&author=Wang+X.&author=Qiao+J.W.&author=Cao+S.&author=Petralia+F.&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figures S1</a>A-S1C). Metabolome and lipidome levels were respectively measured by label-free gas and liquid chromatography coupled to MS.<div class="floatDisplay"><figure id="gr1" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/eef53957-fddb-42e1-aec5-937e000d28fb/gr1.jpg" alt="Figure thumbnail gr1" data-locator="fig1" data-high-res="/cms/attachment/805a53c5-17c7-4182-89c4-4b3d3090afa9/gr1_lrg.jpg" data-large-image="/cms/attachment/eef53957-fddb-42e1-aec5-937e000d28fb/gr1.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 1" class="label figure__label">Figure 1</span><span class="figure__title__text">Proteogenomic summary of the cohort</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Summary of 10 data types generated in this study.</div><div class="section-paragraph">(B) Overview of significantly altered genes found in at least 5% of samples, showing tumor mutation burden (log<sub>2</sub> WES mutation count) and structural variants, fusions, and CNVs. Subtypes are based on results in panel (C).</div><div class="section-paragraph">(C) Multi-omics clustering of tumor samples by NMF using CNV, expression, and protein and phosphoprotein abundances. Heatmaps show differential expression between subtypes, including DNA methylation, acetylome, metabolome, and lipidome, and characteristic features for each subtype. Pathway enrichment analysis highlights differences between subtypes. Neuron activity related pathways, immune response pathways, and cell cycle pathways were respectively enriched in the nmf1 (proneural-like), nmf2 (mesenchymal-like), and nmf3 (classical-like) subtypes.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2445" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figures S1</a> and <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2450" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">S2</a>, and <a data-locator="mmc2" aria-expanded="false" aria-controls="crosref2455" href="/cms/10.1016/j.ccell.2021.01.006/attachment/88e921e9-6b3e-4bd6-b49b-24db5cd273c8/mmc2" target="_blank">Tables S1</a>, <a data-locator="mmc3" aria-expanded="false" aria-controls="crosref2460" href="/cms/10.1016/j.ccell.2021.01.006/attachment/55251911-b73b-4f07-8f3d-466953b295e9/mmc3" target="_blank">S2</a>, and <a data-locator="mmc4" aria-expanded="false" aria-controls="crosref2465" href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" target="_blank">S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/eef53957-fddb-42e1-aec5-937e000d28fb/gr1.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/805a53c5-17c7-4182-89c4-4b3d3090afa9/gr1_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr1.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Genomic properties of our cohort were comparable to those of TCGA GBM cohort (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4222834e2245" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brennan et al., 2013</a><div data-db-target-of="bib18-d4222834e2245" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brennan C.W. </li><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Campos B. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Salama S.R. </li><li class="loa__item">Zheng S. </li><li class="loa__item">Chakravarty D. </li><li class="loa__item">Sanborn J.Z. </li><li class="loa__item">Berman S.H. </li><li>et al.</li></ul><div class="reference-citations__detials">The somatic genomic landscape of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). We identified many structural variants (SV) in oncogenes, including <em>EGFR</em> and <em>PDGFRA</em>, and tumor suppressors <em>PTEN</em> and <em>NF1</em>. <em>EGFR</em> mutations often co-occurred with <em>EGFR</em> SV and amplification events (p &lt; 0.01). WES and WGS identified <em>TERT</em> promoter (<em>TERT</em>p) mutations with variant allele frequency (VAF) &gt;5% (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>B). Copy number analysis identified common focal and arm-level copy number variations (CNVs) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>A).</div><div class="section-paragraph">We added protein and phosphosite abundance to prior clustering studies of gene expression (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>C and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S2</a>B-S2F; <a href="#sec5" class="scroll-into-link">STAR methods</a>). While our results are concordant with TCGA expression-based classification (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d4222834e2286" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Wang et al., 2017</a><div data-db-target-of="bib133-d4222834e2286" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang Q. </li><li class="loa__item">Hu B. </li><li class="loa__item">Hu X. </li><li class="loa__item">Kim H. </li><li class="loa__item">Squatrito M. </li><li class="loa__item">Scarpace L. </li><li class="loa__item">deCarvalho A.C. </li><li class="loa__item">Lyu S. </li><li class="loa__item">Li P. </li><li class="loa__item">Li Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">42-56.e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28697342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85027444208&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=42-56.e6&journal=Cancer+Cell&author=Wang+Q.&author=Hu+B.&author=Hu+X.&author=Kim+H.&author=Squatrito+M.&author=Scarpace+L.&author=deCarvalho+A.C.&author=Lyu+S.&author=Li+P.&author=Li+Y.&title=Tumor+evolution+of+glioma-intrinsic+gene+expression+subtypes+associates+with+immunological+changes+in+the+microenvironment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), 27 tumors (29%) were classified as a different subtype (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figures 1</a>C and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S2</a>B). Based on similarities with gene expression subtypes, we designated the three clusters observed in <em>IDH</em>-WT tumors as nmf1 (proneural-like; n = 29), nmf2 (mesenchymal-like; n = 37), and nmf3 (classical-like; n = 26). Pathway enrichment analysis of RNA, protein, and phosphosite abundances indicated that nmf1 was enriched for synaptic vesicle cycle and neurotransmission transport; nmf2 was enriched for innate immune response, including neutrophil degranulation, phagocytosis, and extracellular matrix organization; and nmf3 was enriched for mRNA splicing and RNA metabolism. Based on the known functional effects of protein acetylation (<div class="dropBlock reference-citations"><a href="#bib96" id="back-bib96" data-db-target-for="bib96-d4222834e2298" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib96">Narita et al., 2019</a><div data-db-target-of="bib96-d4222834e2298" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Narita T. </li><li class="loa__item">Weinert B.T. </li><li class="loa__item">Choudhary C. </li></ul><div class="reference-citations__detials">Functions and mechanisms of non-histone protein acetylation.<div class="popup-ref"><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">156-174</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41580-018-0081-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30467427&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85057071302&amp;cf=" onclick="newWindow(this.href);return false">Scopus (151)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=156-174&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Narita+T.&author=Weinert+B.T.&author=Choudhary+C.&title=Functions+and+mechanisms+of+non-histone+protein+acetylation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), nmf1/proneural-like cluster had a higher abundance of acetylated proteins involved in the TCA cycle and metabolism of amino acids, whereas the nmf2/mesenchymal-like cluster was enriched for innate immune system activation, peroxisomal protein import and glycolysis. The nmf3/classical-like subtype was enriched for acetylation of chromatin modifiers and DNA repair proteins.</div><div class="section-paragraph">Clinical data associated with the three subtypes indicated that tumors with relatively low multi-omics membership scores for two or more subtypes, i.e. those of “mixed subtype” (n = 12) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>B) were associated with worse prognosis (log rank test p = 1.7e-4; <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>C) compared with those of non-mixed subtype (excluding <em>IDH1</em>-mutant tumors). We identified three proteins associated with poor survival across all tumors: low expression of HIST3H2BB (log rank test p = 0.0034), high expression of MT-CYB (p = 0.03), and high expression of PRODH (p = 0.096). Genome-wide DNA methylation profiling identified six DNA methylation subtypes, including two distinct glioma CpG island methylator phenotype (G-CIMP) subtypes (dm2 and dm6). The dm6 subtype is <em>IDH-</em>mutant-specific with upregulation of chromatin organization pathways, while dm2 consists of <em>IDH</em>-WT tumors with upregulation of transcription and mRNA splicing pathways (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>D). Two subtypes showed elevated expression of different <em>de novo</em> DNA methylases (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>E). We also examined <em>cis</em> associations of DNA methylation with RNA or protein abundances using iProFun (<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d4222834e2326" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Song et al., 2019</a><div data-db-target-of="bib117-d4222834e2326" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" class="scroll-into-link">Table S3</a>). For example, 38 of 90 tumors (42%) exhibited hypermethylation of the <em>MGMT</em> promoter region and significantly decreased MGMT RNA and protein levels (Welch's t test p = 4.9e-11 and 2.6e-6, respectively) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>G; <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/55251911-b73b-4f07-8f3d-466953b295e9/mmc3" class="scroll-into-link">Table S2</a>).</div></section><section id="sec2.2" data-locator="sec2.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Driver genetic alterations influence oncogenic protein abundance and phosphorylation</span></h3><div class="section-paragraph">We associated genetic alterations (mutations, CNVs, fusions, and SVs) with RNA, protein expression and phosphorylation levels, observing 95 <em>cis-trans</em> phosphorylation events (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B ). We found strong <em>cis</em> effects for <em>EGFR</em> and <em>PDGFRA</em>, with significant increases in RNA and protein expression and increased phosphorylation at S1166 and S1067/S1070, respectively. At the <em>trans</em> level, tumors with <em>EGFR</em> alterations presented elevated CTNNB1 (β-catenin) protein despite decreased mRNA, and increased phosphorylation of both PTPN11 (Shp2) at Y62 and PLCG1 at Y783 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B). These observations illustrate the importance of protein measurements to study pathway activation.<div class="floatDisplay"><figure id="gr2" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/dc051b26-306d-4878-97e9-54ae677d2d09/gr2.jpg" alt="Figure thumbnail gr2" data-locator="fig2" data-high-res="/cms/attachment/b35acd49-1d5d-4847-88ae-dd2a77e92165/gr2_lrg.jpg" data-large-image="/cms/attachment/dc051b26-306d-4878-97e9-54ae677d2d09/gr2.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 2" class="label figure__label">Figure 2</span><span class="figure__title__text"><em>Cis</em> and <em>trans</em> effects of SMGs and effects of TP53 regulations on DNA repair genes and RB1 on cell cycle genes</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) <em>Cis</em> and <em>trans</em> effects of significantly mutated genes (y axis) on RNA and protein level (x axis) showing that effects are often similar.</div><div class="section-paragraph">(B) <em>Cis</em> and <em>trans</em> effects of significantly mutated genes (y axis) on protein phosphorylation status (x axis).</div><div class="section-paragraph">(C) Comparison of RNA and protein expression in <em>TP53</em>-mutated versus WT samples. Bottom: alternative <em>TP53</em> splice site for the X126 mutation.</div><div class="section-paragraph">(D) Differentially expressed proteins and phosphoproteins in DNA repair genes for <em>TP53</em>-mutant (n = 32) versus <em>TP53</em> WT (n = 67) samples. Right: a schematic of differences in expression and phosphorylation in the context of known pathway regulation.</div><div class="section-paragraph">(E) <em>RB1</em> alterations associated with protein expression of CDK2, CDK6, MCM2, MCM4, MCM6, and RB1. Right: A schematic of proposed interplay among RB1, MCM2, MCM4, and MCM6 in <em>RB1</em>-altered (n = 10) and WT (n = 89) samples.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2470" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S3</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/dc051b26-306d-4878-97e9-54ae677d2d09/gr2.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/b35acd49-1d5d-4847-88ae-dd2a77e92165/gr2_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr2.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">Tumor suppressors <em>RB1</em>, <em>NF1</em>, <em>PTEN</em>, and <em>ATRX</em> demonstrated good concordance between genetic alterations and decreased RNA, protein, and phosphorylation levels of their respective gene products. Although the general effect of <em>TP53</em> mutations on increased protein stability is known, we identified specific phosphosites that correlate with increased stability. Phosphorylation of TP53 at S315 and TP53BP1 at S1099, S1106, and S1109 correlated with increased TP53 protein expression (Pearson <em>r</em> = 0.89 and 0.53, respectively) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A, 2B, and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S3</a>A). TP53 alterations were largely missense mutations (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>C) within its DNA binding domain (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S3</a>B), including hotspot sites and several rare variants. TP53 protein expression was consistently elevated relative to <em>TP53</em> WT GBMs, an effect not seen at the RNA level. RNA-seq read alignment demonstrated that a recurrent splice-site mutation, X126_splice, results in an alternative splice site, producing an in-frame 7aa truncation in exon 5 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>C and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S3</a>C).</div><div class="section-paragraph">We assessed kinases known to phosphorylate TP53 and its downstream targets. In <em>TP53</em> mutants (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A, 2B, and 2D), we detected elevated protein and/or phosphorylation in ATR, MAPK3, CDK2, and CDK9, while <em>MDM2</em> was decreased at both RNA and protein levels. Tumors with <em>TP53</em> mutations showed upregulated phosphosites, but not increased protein levels, of DNA repair genes (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>D), suggesting specific phosphosite regulation.</div><div class="section-paragraph">We observed negative feedback between <em>RB1</em> and downstream targets, <em>CDK2, CDK6, MCM2, MCM4,</em> and <em>MCM6</em>, while <em>NF1</em> had similar effects on IRF8 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B). <em>RB1</em>-altered samples (12% of the cohort) showed significantly downregulated <em>RB1</em> and upregulated MCM2, MCM4, and MCM6 protein expression (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>E). In addition, in samples with <em>NF1</em> alterations, we observed upregulation of protein and RNA of <em>IRF8</em>, a transcription factor that controls microglial motility (<div class="dropBlock reference-citations"><a href="#bib145" id="back-bib145" data-db-target-for="bib145-d4222834e2442" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib145">Masuda et al., 2014</a><div data-db-target-of="bib145-d4222834e2442" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Masuda T. </li><li class="loa__item">Nishimoto N. </li><li class="loa__item">Tomiyama D. </li><li class="loa__item">Matsuda T. </li><li class="loa__item">Tozaki-Saitoh H. </li><li class="loa__item">Tamura T. </li><li class="loa__item">Kohsaka S. </li><li class="loa__item">Tsuda M. </li><li class="loa__item">Inoue K. </li></ul><div class="reference-citations__detials">IRF8 is a transcriptional determinant for microglial motility.<div class="popup-ref"><span class="ref__series"><i>Purinergic Signal.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">515-521</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11302-014-9413-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24798612&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84906866003&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=515-521&journal=Purinergic+Signal.&author=Masuda+T.&author=Nishimoto+N.&author=Tomiyama+D.&author=Matsuda+T.&author=Tozaki-Saitoh+H.&author=Tamura+T.&author=Kohsaka+S.&author=Tsuda+M.&author=Inoue+K.&title=IRF8+is+a+transcriptional+determinant+for+microglial+motility">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and 2B)<em>.</em></div><div class="section-paragraph">GBMs exploit various telomere lengthening mechanisms. WGS data identified <em>TERT</em>p hotspot mutations in 74% of primary GBMs (NM_198253.2:c.-124C&gt;T [C228T] and c.-146C&gt;T [C250T]) (<div class="dropBlock reference-citations"><a href="#bib60" id="back-bib60" data-db-target-for="bib60-d4222834e2454" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib60">Killela et al., 2013</a><div data-db-target-of="bib60-d4222834e2454" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Killela P.J. </li><li class="loa__item">Reitman Z.J. </li><li class="loa__item">Jiao Y. </li><li class="loa__item">Bettegowda C. </li><li class="loa__item">Agrawal N. </li><li class="loa__item">Diaz L.A. </li><li class="loa__item">Friedman A.H. </li><li class="loa__item">Friedman H. </li><li class="loa__item">Gallia G.L. </li><li class="loa__item">Giovanella B.C. </li><li>et al.</li></ul><div class="reference-citations__detials">TERT promoter mutations occur frequently in gliomas and a subset of tumors derived from cells with low rates of self-renewal.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">110</span></span>: <span class="ref__seriesPages">6021-6026</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.1303607110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23530248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84876067164&amp;cf=" onclick="newWindow(this.href);return false">Scopus (819)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=6021-6026&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Killela+P.J.&author=Reitman+Z.J.&author=Jiao+Y.&author=Bettegowda+C.&author=Agrawal+N.&author=Diaz+L.A.&author=Friedman+A.H.&author=Friedman+H.&author=Gallia+G.L.&author=Giovanella+B.C.&title=TERT+promoter+mutations+occur+frequently+in+gliomas+and+a+subset+of+tumors+derived+from+cells+with+low+rates+of+self-renewal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), resulting in increased <em>TERT</em> RNA expression (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figures S3</a>D and S3E). We also found that <em>ATRX</em> mutations were mutually exclusive with <em>TERT</em>p hotspot mutations and co-occurred with <em>TP53</em> and <em>IDH1</em> R132H mutations (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S3</a>F). Nine <em>ATRX</em> mutants had significantly diminished <em>ATRX</em> RNA and protein levels (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S3</a>G). Immunohistochemistry (IHC) staining confirmed the loss of ATRX protein in tumor cells in <em>ATRX-</em>WT <em>IDH1</em> R132H mutant tumors (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S3</a>H). Despite ATRX loss, expression of its complex partners was not affected at RNA or protein levels, raising questions about the function of the complex in the absence of ATRX protein (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figures S3</a>I and S3J).</div></section><section id="sec2.3" data-locator="sec2.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RTK signaling cascades are activated in GBM</span></h3><div class="section-paragraph">Genomic loci associated with RTKs, such as EGFR, PDGFRA, and MET, are frequently amplified in GBM (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4222834e2497" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brennan et al., 2013</a><div data-db-target-of="bib18-d4222834e2497" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brennan C.W. </li><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Campos B. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Salama S.R. </li><li class="loa__item">Zheng S. </li><li class="loa__item">Chakravarty D. </li><li class="loa__item">Sanborn J.Z. </li><li class="loa__item">Berman S.H. </li><li>et al.</li></ul><div class="reference-citations__detials">The somatic genomic landscape of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We identified 45 tumors with <em>EGFR</em> SVs, all having copy number amplifications, suggesting high concordance between SV and CNV (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>A  and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S4</a>A). All tumors with mutated <em>EGFR</em> and SV have correspondingly high RNA, protein, and Y1172 phosphorylation levels, indicating EGFR pathway activation.<div class="floatDisplay"><figure id="gr3" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/10647e67-3109-425e-91e0-05f26400f802/gr3.jpg" alt="Figure thumbnail gr3" data-locator="fig3" data-high-res="/cms/attachment/3c649fec-6fb8-486f-b9dd-623a08e5b407/gr3_lrg.jpg" data-large-image="/cms/attachment/10647e67-3109-425e-91e0-05f26400f802/gr3.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 3" class="label figure__label">Figure 3</span><span class="figure__title__text">Alterations in RTKs and associations with expression, phosphosite status, and downstream targets</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Structural variations (SV), fusions, mutations (MUT), and copy number variations (CNV) in <em>EGFR, PDGFRA, FGFR3,</em> and <em>MET</em> and their <em>cis</em> effects.</div><div class="section-paragraph">(B) Proteins and phosphosites differentially expressed or phosphorylated between <em>EGFR</em>-altered and <em>EGFR</em> WT samples.</div><div class="section-paragraph">(C) Proteomic association of altered <em>EGFR</em> (n = 46) on protein expression of key genes, compared to samples with EGFR WT (n = 53) (left). PTPN11 level is not affected by <em>EGFR</em> alterations, while phosphorylation of the Y62 site is increased in <em>EGFR</em>-altered samples (right).</div><div class="section-paragraph">(D) Heatmap showing significant (FDR &lt;0.1) <em>cis</em>- and <em>trans</em>-regulated sites of EGFR and PDGFRA kinases. Both EGFR and PDGFR regulate phosphorylation of PTPN11. The schematic (right) shows dual regulation of PTPN11 by EGFR and PDGFRA and the downstream substrates that PTPN11 may dephosphorylate.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2475" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/10647e67-3109-425e-91e0-05f26400f802/gr3.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/3c649fec-6fb8-486f-b9dd-623a08e5b407/gr3_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr3.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">In <em>EGFR</em>-altered samples, high EGFR autophosphorylation was observed, along with increased abundance and phosphorylation of pleckstrin homology-like domain family A member proteins (PHLDA1 and PHLDA3), transcription factor SOX9, cell adhesion protein CTNND2 (δ-catenin), and cell cycle proteins CDK6 and CDKN2C (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figures 3</a>B and 3C). <em>PHLDA1</em> and <em>PHLDA3</em> were not differentially expressed in RNA, consistent with activation of downstream EGFR pathway components (<div class="dropBlock reference-citations"><a href="#bib81" id="back-bib81" data-db-target-for="bib81-d4222834e2524" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib81">Ling et al., 2011</a><div data-db-target-of="bib81-d4222834e2524" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ling S. </li><li class="loa__item">Chang X. </li><li class="loa__item">Schultz L. </li><li class="loa__item">Lee T.K. </li><li class="loa__item">Chaux A. </li><li class="loa__item">Marchionni L. </li><li class="loa__item">Netto G.J. </li><li class="loa__item">Sidransky D. </li><li class="loa__item">Berman D.M. </li></ul><div class="reference-citations__detials">An EGFR-ERK-SOX9 signaling cascade links urothelial development and regeneration to cancer.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">71</span></span>: <span class="ref__seriesPages">3812-3821</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-10-3072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21512138&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79957883840&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3812-3821&journal=Cancer+Res.&author=Ling+S.&author=Chang+X.&author=Schultz+L.&author=Lee+T.K.&author=Chaux+A.&author=Marchionni+L.&author=Netto+G.J.&author=Sidransky+D.&author=Berman+D.M.&title=An+EGFR-ERK-SOX9+signaling+cascade+links+urothelial+development+and+regeneration+to+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib83" id="back-bib83" data-db-target-for="bib83-d4222834e2528" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib83">Liu et al., 2019</a><div data-db-target-of="bib83-d4222834e2528" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liu L. </li><li class="loa__item">Shi Y. </li><li class="loa__item">Shi J. </li><li class="loa__item">Wang H. </li><li class="loa__item">Sheng Y. </li><li class="loa__item">Jiang Q. </li><li class="loa__item">Chen H. </li><li class="loa__item">Li X. </li><li class="loa__item">Dong J. </li></ul><div class="reference-citations__detials">The long non-coding RNA SNHG1 promotes glioma progression by competitively binding to miR-194 to regulate PHLDA1 expression.<div class="popup-ref"><span class="ref__series"><i>Cell Death Dis.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">463</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41419-019-1698-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31189920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85067299482&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=463&journal=Cell+Death+Dis.&author=Liu+L.&author=Shi+Y.&author=Shi+J.&author=Wang+H.&author=Sheng+Y.&author=Jiang+Q.&author=Chen+H.&author=Li+X.&author=Dong+J.&title=The+long+non-coding+RNA+SNHG1+promotes+glioma+progression+by+competitively+binding+to+miR-194+to+regulate+PHLDA1+expression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). SOX9 expression was validated in five of six tumors (three <em>EGFR-</em>altered and three <em>EGFR-</em>WT) using IHC (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S4</a>D).</div><div class="section-paragraph">We also observed increased phosphorylation levels of PTPN11-Y62, PLCG1-Y783, RB1-S795Y805, MAP3K1-S1408, and specific EGFR sites in <em>EGFR</em>-altered samples (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>B, bottom). Notably, the total protein level of PTPN11 was comparable between the two groups, suggesting its activity is regulated primarily by phosphorylation (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>C). A similar pattern is observed with PLCG1 (PLCγ1), where Y783 phosphorylation was significantly higher in <em>EGFR</em>-altered versus <em>EGFR</em>-WT samples (Wilcox false discovery rate [FDR] &lt; 0.01; <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S4</a>C), despite no significant difference in PLCγ1 protein expression (FDR = 0.11). Since phosphorylation of PLCG1 on Y783 is activating (<div class="dropBlock reference-citations"><a href="#bib109" id="back-bib109" data-db-target-for="bib109-d4222834e2556" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib109">Poulin et al., 2005</a><div data-db-target-of="bib109-d4222834e2556" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Poulin B. </li><li class="loa__item">Sekiya F. </li><li class="loa__item">Rhee S.G. </li></ul><div class="reference-citations__detials">Intramolecular interaction between phosphorylated tyrosine-783 and the C-terminal Src homology 2 domain activates phospholipase C-gamma1.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">4276-4281</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.0409590102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15764700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-15444377640&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4276-4281&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Poulin+B.&author=Sekiya+F.&author=Rhee+S.G.&title=Intramolecular+interaction+between+phosphorylated+tyrosine-783+and+the+C-terminal+Src+homology+2+domain+activates+phospholipase+C-gamma1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), this could provide a mechanism for EGFR activation of PLCG1’s known effects on proliferation, migration, and invasiveness (<div class="dropBlock reference-citations"><a href="#bib69" id="back-bib69" data-db-target-for="bib69-d4222834e2559" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib69">Kunze et al., 2014</a><div data-db-target-of="bib69-d4222834e2559" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kunze K. </li><li class="loa__item">Spieker T. </li><li class="loa__item">Gamerdinger U. </li><li class="loa__item">Nau K. </li><li class="loa__item">Berger J. </li><li class="loa__item">Dreyer T. </li><li class="loa__item">Sindermann J.R. </li><li class="loa__item">Hoffmeier A. </li><li class="loa__item">Gattenlöhner S. </li><li class="loa__item">Bräuninger A. </li></ul><div class="reference-citations__detials">A recurrent activating PLCG1 mutation in cardiac angiosarcomas increases apoptosis resistance and invasiveness of endothelial cells.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">6173-6183</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-14-1162&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25252913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84910020229&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=6173-6183&journal=Cancer+Res.&author=Kunze+K.&author=Spieker+T.&author=Gamerdinger+U.&author=Nau+K.&author=Berger+J.&author=Dreyer+T.&author=Sindermann+J.R.&author=Hoffmeier+A.&author=Gattenl%C3%B6hner+S.&author=Br%C3%A4uninger+A.&title=A+recurrent+activating+PLCG1+mutation+in+cardiac+angiosarcomas+increases+apoptosis+resistance+and+invasiveness+of+endothelial+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">We performed a kinase-substrate study for EGFR and PDGFRA and identified high levels of GAB1 phosphorylation at Y689 and Y657, consistent with high EGFR expression. In addition, PTPN11 phosphosites at Y546 and Y584 were associated with high PDGFRA expression (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>D) and have been observed in lung cancers with <em>ALK</em> fusions (<div class="dropBlock reference-citations"><a href="#bib132" id="back-bib132" data-db-target-for="bib132-d4222834e2569" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib132">Voena et al., 2007</a><div data-db-target-of="bib132-d4222834e2569" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Voena C. </li><li class="loa__item">Conte C. </li><li class="loa__item">Ambrogio C. </li><li class="loa__item">Boeri Erba E. </li><li class="loa__item">Boccalatte F. </li><li class="loa__item">Mohammed S. </li><li class="loa__item">Jensen O.N. </li><li class="loa__item">Palestro G. </li><li class="loa__item">Inghirami G. </li><li class="loa__item">Chiarle R. </li></ul><div class="reference-citations__detials">The tyrosine phosphatase Shp2 interacts with NPM-ALK and regulates anaplastic lymphoma cell growth and migration.<div class="popup-ref"><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">67</span></span>: <span class="ref__seriesPages">4278-4286</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-06-4350&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17483340&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-34249277428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=4278-4286&journal=Cancer+Res.&author=Voena+C.&author=Conte+C.&author=Ambrogio+C.&author=Boeri+Erba+E.&author=Boccalatte+F.&author=Mohammed+S.&author=Jensen+O.N.&author=Palestro+G.&author=Inghirami+G.&author=Chiarle+R.&title=The+tyrosine+phosphatase+Shp2+interacts+with+NPM-ALK+and+regulates+anaplastic+lymphoma+cell+growth+and+migration">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Activation of PTPN11 through either EGFR- or PDGFRA-related phosphorylation in GBM suggests it may represent a shared RTK signaling hub. PTPN11, GAB1, and GRB2 form a complex and are co-regulated by RTKs to activate the RAS pathway (<div class="dropBlock reference-citations"><a href="#bib94" id="back-bib94" data-db-target-for="bib94-d4222834e2572" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib94">Montagner et al., 2005</a><div data-db-target-of="bib94-d4222834e2572" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Montagner A. </li><li class="loa__item">Yart A. </li><li class="loa__item">Dance M. </li><li class="loa__item">Perret B. </li><li class="loa__item">Salles J.-P. </li><li class="loa__item">Raynal P. </li></ul><div class="reference-citations__detials">A novel role for Gab1 and SHP2 in epidermal growth factor-induced Ras activation.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">280</span></span>: <span class="ref__seriesPages">5350-5360</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15574420&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-14044272770&amp;cf=" onclick="newWindow(this.href);return false">Scopus (133)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=5350-5360&journal=J.%C2%A0Biol.+Chem.&author=Montagner+A.&author=Yart+A.&author=Dance+M.&author=Perret+B.&author=Salles+J.-P.&author=Raynal+P.&title=A+novel+role+for+Gab1+and+SHP2+in+epidermal+growth+factor-induced+Ras+activation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). <a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figures 2</a>A and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S4</a>C show that EGFR activation status is associated with upregulated GAB1 and downregulated GRB2 protein expression. We validated the elevated GAB1 expression in <em>EGFR</em>-altered tumors using IHC (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S4</a>D).</div></section><section id="sec2.4" data-locator="sec2.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Distinct immune marker expression and epigenetic events characterize GBM immune subtypes</span></h3><div class="section-paragraph">We generated cell-type immune enrichment scores using single-sample GSEA by xCell (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d4222834e2592" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Aran et al., 2017</a><div data-db-target-of="bib3-d4222834e2592" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), finding four distinct immune-based GBM subtypes (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A; <a href="#sec5" class="scroll-into-link">STAR methods</a>). Immune subtype 1 (im1) showed overall higher scores (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S5</a>A), including elevated levels of microglia, macrophages, and lymphocytes. Immune subtypes 2 and 3 (im2 and im3) displayed reciprocal ratios of macrophages and lymphocytes, with im2 higher in macrophages and lower in lymphocytes and im3 having higher neuron score (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S5</a>A). Immune subtype 4 (im4) is distinct from the others, with substantially lower enrichment for all immune cell types (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S5</a>A). The mesenchymal subtype was enriched in im1 and im2 (Fisher test p = 1.65e-15), while <em>IDH</em> mutants were overrepresented in im3 (p = 1.02e-5). The DNA methylation dm3 subtype was strongly associated with im1 (Fisher test p = 1.26e-5), consistent with the association of dm3 with immune gene expression (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>D). The four immune subtypes were confirmed in the TCGA GBM cohort using transcriptome data (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figures S5</a>A and S5B) and protein deconvolution (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>A and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S5</a>C).<div class="floatDisplay"><figure id="gr4" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/0616ce9f-1e54-448c-95ef-a5b1de6f70ee/gr4.jpg" alt="Figure thumbnail gr4" data-locator="fig4" data-high-res="/cms/attachment/4978ed41-50eb-4de4-b417-6ce246f6c346/gr4_lrg.jpg" data-large-image="/cms/attachment/0616ce9f-1e54-448c-95ef-a5b1de6f70ee/gr4.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 4" class="label figure__label">Figure 4</span><span class="figure__title__text">Cell-type enrichment, immune marker expression, and enrichment pathways among the four immune subtypes</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) The four immune subtypes identified by consensus clustering showing cell-type features, immune checkpoints, and potential immunotherapy targets. Differential expression is between tumors of one immune subtype versus the rest based on global protein and phosphoprotein abundance (DEPs/DEPPs: FDR &lt;0.05 and log<sub>2</sub>FC &gt; 0.8) and the corresponding enriched pathways (FDR &lt;0.05 and log<sub>2</sub>FC ≥ 3 markers included in the pathway).</div><div class="section-paragraph">(B) snRNA-seq UMAP plot colored by cell types observed in 18 discovery cohort GBM samples. OPC, oligodendrocyte progenitor cells; TAM, tumor-associated microglia/macrophage; vSMC, vascular smooth muscle cell.</div><div class="section-paragraph">(C) Differentially expressed genes in TAMs between im1 subtype samples versus the remaining cohort. Figure shows genes with absolute value of average log<sub>2</sub>FC &gt; 0.25 and Wilcoxon test FDR-adjusted p values.</div><div class="section-paragraph">(D) Features captured by the deep learning model. Each dot represents a tile of H&amp;E slides in the test set, colored according to prediction score (red: predicted immune-high; blue: predicted immune-low). The 20,000 sampled tiles from 99 patients were clustered by t-SNE to their activation maps (a 1,250-long vector for each tile) from the final layer of the model.</div><div class="section-paragraph">(E) H&amp;E tile images from im4 tumors, with arrows indicating giant cells. The highlighted region contains multiple noncontinuous tiles clustered closely in t-SNE space.</div><div class="section-paragraph">(F) H&amp;E tile images from non-im4 tumors, with arrows indicating the inflammatory cells. The highlighted region contains multiple noncontinuous tiles clustered closely in t-SNE space.</div><div class="section-paragraph">(G) Heatmaps showing snRNA-seq (left) and bulk protein (right) expression of genes upregulated in the nmf2 subtype in tumor cells. Expression values were scaled sample-wise across all cell types (or across tumor cells as labeled) and then averaged across multi-omics subtypes. Protein expression is shown for samples with snRNA-seq available.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2480" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S5</a> and <a data-locator="mmc5" aria-expanded="false" aria-controls="crosref2485" href="/cms/10.1016/j.ccell.2021.01.006/attachment/c8ae7793-ce25-422c-a366-37ffd9051677/mmc5" target="_blank">Table S4</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/0616ce9f-1e54-448c-95ef-a5b1de6f70ee/gr4.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/4978ed41-50eb-4de4-b417-6ce246f6c346/gr4_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr4.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We performed snRNA-seq on 18 GBM samples using the same cryopulverized material from previous analyses (7 im1, 5 im2, 1 im3, 5 im4 samples; <a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A). TAMs comprised the major non-neoplastic cell population in the GBM TME (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>B). Im4 samples showed consistently low percentages of T cell and TAM infiltration (1.3% and 6% on average, respectively) compared with those of the other immune subtypes (7% and 19% on average, respectively). Im1 tumors showed higher scores for microglia and macrophages at the bulk level, but this was not observed in im1-3 in snRNA-seq (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S5</a>D), which may be explained by the bulk data being driven by the percentage of TAMs and the expression of genes in the TAMs themselves. Supporting this hypothesis, we observed genes upregulated at the RNA and protein levels in TAMs in im1 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>C). We validated key immune markers (CD3, CD68, CD163, PD-1, and PD-L1) in five tumors from three immune subtypes using IHC (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figures S4</a>E and S4F).</div><div class="section-paragraph">We identified differentially expressed proteins (DEPs) and phosphoproteins (DEPPs) in known immune targets (<div class="dropBlock reference-citations"><a href="#bib23" id="back-bib23" data-db-target-for="bib23-d4222834e2652" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib23">Chen and Hambardzumyan, 2018</a><div data-db-target-of="bib23-d4222834e2652" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen Z. </li><li class="loa__item">Hambardzumyan D. </li></ul><div class="reference-citations__detials">Immune microenvironment in glioblastoma subtypes.<div class="popup-ref"><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1004</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.3389%2Ffimmu.2018.01004&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29867979&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85046650498&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1004&journal=Front.+Immunol.&author=Chen+Z.&author=Hambardzumyan+D.&title=Immune+microenvironment+in+glioblastoma+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Gene and protein expression levels of immune targets, including negative regulatory immune checkpoints (e.g., PD-1 and TIM-3), chemokines (e.g., CCL2), macrophage-specific cytokines (e.g., CSF-1), and their receptors (CSF-1R), were significantly higher in im1 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A) (<div class="dropBlock reference-citations"><a href="#bib21" id="back-bib21" data-db-target-for="bib21-d4222834e2658" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib21">Butowski et al., 2016</a><div data-db-target-of="bib21-d4222834e2658" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Butowski N. </li><li class="loa__item">Colman H. </li><li class="loa__item">De Groot J.F. </li><li class="loa__item">Omuro A.M. </li><li class="loa__item">Nayak L. </li><li class="loa__item">Wen P.Y. </li><li class="loa__item">Cloughesy T.F. </li><li class="loa__item">Marimuthu A. </li><li class="loa__item">Haidar S. </li><li class="loa__item">Perry A. </li><li>et al.</li></ul><div class="reference-citations__detials">Orally administered colony stimulating factor 1 receptor inhibitor PLX3397 in recurrent glioblastoma: an Ivy Foundation Early Phase Clinical Trials Consortium phase II study.<div class="popup-ref"><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">557-564</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnov245&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26449250&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966552295&amp;cf=" onclick="newWindow(this.href);return false">Scopus (194)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=557-564&journal=Neuro-Oncol.&author=Butowski+N.&author=Colman+H.&author=De+Groot+J.F.&author=Omuro+A.M.&author=Nayak+L.&author=Wen+P.Y.&author=Cloughesy+T.F.&author=Marimuthu+A.&author=Haidar+S.&author=Perry+A.&title=Orally+administered+colony+stimulating+factor+1+receptor+inhibitor+PLX3397+in+recurrent+glioblastoma%3A+an+Ivy+Foundation+Early+Phase+Clinical+Trials+Consortium+phase+II+study">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We also identified overrepresented pathways among DEPs and DEPPs (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/c8ae7793-ce25-422c-a366-37ffd9051677/mmc5" class="scroll-into-link">Table S4</a>). Pathways in im1, including immune system and microglia pathogen phagocytosis, were mostly immune-related, whereas collagen formation and angiogenesis-related proteins were upregulated in im2, neuronal system pathways in im3, and cell cycle and gliogenesis pathways in im4 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>A, <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Table S4</a>).</div><div class="section-paragraph">We analyzed morphologic differences between immune subtypes by applying a deep learning model using sampled tumor tiles from H&amp;E-stained sections. Dimensional reduction (t-SNE) of im4 compared with other clusters (im1-3) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>D-4F) revealed a substantial number of large cells, some of which are giant cells, in im4 tiles, with few in im1-3 tiles. Biological relevance of these cells is unclear. Inflammatory cell fractions were noted in ∼20% of im1-3 tiles compared to 5% in im4 tiles.</div></section><section id="sec2.5" data-locator="sec2.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mesenchymal tumor and microenvironment characteristics</span></h3><div class="section-paragraph">Application of CausalPath (<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d4222834e2681" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Babur et al., 2018</a><div data-db-target-of="bib6-d4222834e2681" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Babur Ö. </li><li class="loa__item">Luna A. </li><li class="loa__item">Korkut A. </li><li class="loa__item">Durupinar F. </li><li class="loa__item">Siper M.C. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Aslan J.E. </li><li class="loa__item">Sander C. </li><li class="loa__item">Demir E. </li></ul><div class="reference-citations__detials">Causal interactions from proteomic profiles: molecular data meets pathway knowledge.<div class="popup-ref"><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">258855</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=258855&journal=BioRxiv&author=Babur+%C3%96.&author=Luna+A.&author=Korkut+A.&author=Durupinar+F.&author=Siper+M.C.&author=Dogrusoz+U.&author=Aslan+J.E.&author=Sander+C.&author=Demir+E.&title=Causal+interactions+from+proteomic+profiles%3A+molecular+data+meets+pathway+knowledge">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to the protein and phosphoprotein expression data (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S7</a>A, <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6" class="scroll-into-link">Table S5</a>) showed upregulation of the hypoxia pathway in mesenchymal tumors, evidenced by significant activation of multiple HIF-1 downstream targets (network permutation p = 0.0012). Increased angiogenesis was also evident in mesenchymal tumors, as demonstrated by upregulation of FLT1, MMP14, ENG, and SERPINE1. We observed complex regulation of macrophage activation and polarization through the upregulation of STAT3, ICAM1, SPI1, and CEBPB. In addition, the M1 polarization marker ARG1 showed increased expression (<div class="dropBlock reference-citations"><a href="#bib4" id="back-bib4" data-db-target-for="bib4-d4222834e2690" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib4">Arlauckas et al., 2018</a><div data-db-target-of="bib4-d4222834e2690" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Arlauckas S.P. </li><li class="loa__item">Garren S.B. </li><li class="loa__item">Garris C.S. </li><li class="loa__item">Kohler R.H. </li><li class="loa__item">Oh J. </li><li class="loa__item">Pittet M.J. </li><li class="loa__item">Weissleder R. </li></ul><div class="reference-citations__detials">Arg1 expression defines immunosuppressive subsets of tumor-associated macrophages.<div class="popup-ref"><span class="ref__series"><i>Theranostics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">5842-5854</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.7150%2Fthno.26888&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30613266&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85057156211&amp;cf=" onclick="newWindow(this.href);return false">Scopus (49)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=5842-5854&journal=Theranostics&author=Arlauckas+S.P.&author=Garren+S.B.&author=Garris+C.S.&author=Kohler+R.H.&author=Oh+J.&author=Pittet+M.J.&author=Weissleder+R.&title=Arg1+expression+defines+immunosuppressive+subsets+of+tumor-associated+macrophages">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), along with SERPINE1 and HCK proteins, which promote M2 polarization (<div class="dropBlock reference-citations"><a href="#bib68" id="back-bib68" data-db-target-for="bib68-d4222834e2693" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib68">Kubala et al., 2018</a><div data-db-target-of="bib68-d4222834e2693" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kubala M.H. </li><li class="loa__item">Punj V. </li><li class="loa__item">Placencio-Hickok V.R. </li><li class="loa__item">Fang H. </li><li class="loa__item">Fernandez G.E. </li><li class="loa__item">Sposto R. </li><li class="loa__item">DeClerck Y.A. </li></ul><div class="reference-citations__detials">Plasminogen activator inhibitor-1 promotes the recruitment and polarization of macrophages in cancer.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2177-2191.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30463014&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85056741174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2177-2191.e7&journal=Cell+Rep&author=Kubala+M.H.&author=Punj+V.&author=Placencio-Hickok+V.R.&author=Fang+H.&author=Fernandez+G.E.&author=Sposto+R.&author=DeClerck+Y.A.&title=Plasminogen+activator+inhibitor-1+promotes+the+recruitment+and+polarization+of+macrophages+in+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The elevated inflammatory response in mesenchymal tumors may result in downstream activation of either hypoxia or macrophage polarization through multiple mediators, including LANE, IL18, and CD40 (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S7</a>A)<em>.</em></div><div class="section-paragraph">While the origin of the mesenchymal signature in GBM has been controversial, snRNA-seq data enabled identifying mesenchymal features in tumor and immune cells. We found EMT-related genes upregulated in the tumor cells of nmf2 samples (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figures 4</a>G and <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">S5</a>E). Among those, mesenchymal markers CHI3L1 and MET had the highest expression in tumor cells, along with other EMT-related genes, such as <em>TNC</em>, <em>ITGA3</em>, and <em>PDPN</em>. Some EMT-related genes were also expressed by nontumor cell types, including TAMs, T-cells, pericytes, endothelial cells, and oligodendrocytes. However, there were few subtype-specific differences in expression of these markers by nontumor cells. Accordingly, all EMT-related genes that were upregulated in tumor cells were also significantly increased in bulk RNA and protein data in nmf2 GBMs (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr4" href="#gr4" data-locator="fig4" id="back-gr4" class="figure-link scroll-into-link">Figure 4</a>G). In addition to immune infiltration, nmf2 GBMs evidently have intrinsic mesenchymal tumor cell-specific properties as measured by bulk proteomics.</div></section><section id="sec2.6" data-locator="sec2.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differential acetylation of histone proteins is associated with specific subtypes and pathways</span></h3><div class="section-paragraph">Histone acetylation regulates gene expression but is frequently aberrant in cancer (<div class="dropBlock reference-citations"><a href="#bib38" id="back-bib38" data-db-target-for="bib38-d4222834e2724" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib38">Eberharter and Becker, 2002</a><div data-db-target-of="bib38-d4222834e2724" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Eberharter A. </li><li class="loa__item">Becker P.B. </li></ul><div class="reference-citations__detials">Histone acetylation: a switch between repressive and permissive chromatin. Second in review series on chromatin dynamics.<div class="popup-ref"><span class="ref__series"><i>EMBO Rep.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">224-229</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fembo-reports%2Fkvf053&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=11882541&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-0036211013&amp;cf=" onclick="newWindow(this.href);return false">Scopus (592)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=224-229&journal=EMBO+Rep.&author=Eberharter+A.&author=Becker+P.B.&title=Histone+acetylation%3A+a+switch+between+repressive+and+permissive+chromatin.+Second+in+review+series+on+chromatin+dynamics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We detected more than 30 acetylation sites on histones H1, H2A, H2B, H3.3, and H4. Unsupervised clustering of these sites across all samples identified subsets of tumors with differentially acetylated histones H1, H2B, H3.3, and H4 (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A). Histone acetylation was generally upregulated in tumors compared to normal samples, with a subset of tumors having elevated H1, H3, and H4 acetylation, while a different cluster exhibited significantly increased acetylation of H2B N-terminal sites (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>A).<div class="floatDisplay"><figure id="gr5" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/54bce86e-ae89-424e-9f4d-f4eff62deae5/gr5.jpg" alt="Figure thumbnail gr5" data-locator="fig5" data-high-res="/cms/attachment/bf86a051-efc5-4703-a70c-9402625acb0e/gr5_lrg.jpg" data-large-image="/cms/attachment/54bce86e-ae89-424e-9f4d-f4eff62deae5/gr5.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 5" class="label figure__label">Figure 5</span><span class="figure__title__text">Histone acetylation associations with immune subtypes and pathways</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Unsupervised clustering of histone protein and site-level acetylation reveals distinct clusters of tumors enriched for acetylation of histones H2B, H3, and H4.</div><div class="section-paragraph">(B) Significant associations between histone acetylation sites and histone acetyltransferase, deacetylases, and bromodomain-containing proteins.</div><div class="section-paragraph">(C) Pathways associated with levels of acetylation of histones H2B, H3, or H4 by multi-omics subtype.</div><div class="section-paragraph">(D) Significant Spearman correlation between xCell scores and acetylation of histone sites (FDR &lt;0.05).</div><div class="section-paragraph">(E) SUMO1 and UBE2I protein expression across samples with high and low H2B acetylation.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2490" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/54bce86e-ae89-424e-9f4d-f4eff62deae5/gr5.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/bf86a051-efc5-4703-a70c-9402625acb0e/gr5_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr5.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We performed Lasso linear regression between histone acetylation sites and the protein and acetylation abundances of histone acetyltransferases (HATs), bromodomain-containing proteins (BRDs), and deacetylases (HDACs). It revealed potential connections between HATs and BRDs and H2B acetylation sites, for example CREBBP and EP300, whose protein and acetylation levels showed substantial respective associations with H2B-K12, K13, K16, K17, and K21 and H2B-K21, and K24 sites (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>B). These observations suggest that H2B hyperacetylation in some tumors may depend on CREBBP/EP300 activity. H2B acetylation sites also correlated with protein and acetylation abundance of BRD1, BRD3, and BRD4 proteins, which bind acetylated histones and mediate transcription (<div class="dropBlock reference-citations"><a href="#bib72" id="back-bib72" data-db-target-for="bib72-d4222834e2739" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib72">LeRoy et al., 2008</a><div data-db-target-of="bib72-d4222834e2739" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">LeRoy G. </li><li class="loa__item">Rickards B. </li><li class="loa__item">Flint S.J. </li></ul><div class="reference-citations__detials">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">51-60</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18406326&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-41549115095&amp;cf=" onclick="newWindow(this.href);return false">Scopus (221)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&journal=Mol.+Cell&author=LeRoy+G.&author=Rickards+B.&author=Flint+S.J.&title=The+double+bromodomain+proteins+Brd2+and+Brd3+couple+histone+acetylation+to+transcription">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We observed significant negative correlation between H2B acetylation and the TME enrichment score (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>D), while other histones showed positive associations. Our analysis identified that some pathways related to immune infiltration, such as ferroptosis, mast cells, and reactive oxygen species pathways had negative correlations with H2B acetylation, while spliceosome, nuclear receptors and SUMOylation pathways were positively correlated (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C). Two key proteins in the SUMOylation pathway, SUMO1 and UBE2I, were upregulated in samples with high H2B acetylation (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>E). Interestingly, UBE2I correlated moderately with cell cycle regulator CDK6 (Pearson <em>r</em> = 0.413), a UBE2I target that is stabilized upon SUMOylation (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S6</a>A) (<div class="dropBlock reference-citations"><a href="#bib12" id="back-bib12" data-db-target-for="bib12-d4222834e2757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib12">Bellail et al., 2014</a><div data-db-target-of="bib12-d4222834e2757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bellail A.C. </li><li class="loa__item">Olson J.J. </li><li class="loa__item">Hao C. </li></ul><div class="reference-citations__detials">SUMO1 modification stabilizes CDK6 protein and drives the cell cycle and glioblastoma progression.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">4234</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fncomms5234&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24953629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84903125838&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=4234&journal=Nat.+Commun.&author=Bellail+A.C.&author=Olson+J.J.&author=Hao+C.&title=SUMO1+modification+stabilizes+CDK6+protein+and+drives+the+cell+cycle+and+glioblastoma+progression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). These observations suggest that H2B acetylation assists in distinguishing immune cells and other cell types.</div></section><section id="sec2.7" data-locator="sec2.7" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lipid composition and metabolomic features associated with GBM subtypes</span></h3><div class="section-paragraph">We quantified 582 lipids in 75 tumors and 7 GTEx normal samples (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>A; <a href="#sec5" class="scroll-into-link">STAR methods</a>), identifying more than 500 lipids that were differentially abundant across the four multi-omics subtypes (Wilcoxon; FDR &lt; 0.05, <a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr1" href="#gr1" data-locator="fig1" id="back-gr1" class="figure-link scroll-into-link">Figure 1</a>C). The mesenchymal subtype demonstrated elevated abundance of triacylglycerols (TGs), as well as depleted levels of phosphatidylcholines (PCs) and other types of phospholipids (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figures 6</a>A and 6B). The proneural-like subtype was enriched for very long chain fatty acid lipids (VLCFAs) and glycerophospholipids with long-chain (LC) polyunsaturated fatty acids (PUFAs) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>B). As for metabolites, the proneural-like cluster exhibited significantly increased levels of creatinine and homocysteine and reduced levels of L-cysteine and palatinitol (Wilcoxon; FDR ≤ 0.05).<div class="floatDisplay"><figure id="gr6" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/e3cb0170-cec6-4600-9946-2b580fe89488/gr6.jpg" alt="Figure thumbnail gr6" data-locator="fig6" data-high-res="/cms/attachment/45ffa233-6dfe-47a6-b7bf-c68fc691fc2c/gr6_lrg.jpg" data-large-image="/cms/attachment/e3cb0170-cec6-4600-9946-2b580fe89488/gr6.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 6" class="label figure__label">Figure 6</span><span class="figure__title__text">Lipidome and metabolome data map to major metabolic and signaling pathways</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Average abundance of all lipids detected across the four tumor subtypes and GTEx normal samples. Lipids are sorted by total number of double-chain bonds and total number of carbons in side chains.</div><div class="section-paragraph">(B) Lipid Mini-On enrichment analysis of lipid properties upregulated in subtype versus second subtype.</div><div class="section-paragraph">(C) Contribution of enzymes that activate PUFAs (<em>ACSL4</em> and <em>ACSL6</em>) to the phospholipid pool and the connection of PUFA-containing PE to ferroptosis.</div><div class="section-paragraph">(D) Protein expression of ACSL6, ACSL4, and ALOX5 across tumor multi-omics subtypes and GTEx normal tissues.</div><div class="section-paragraph">(E) Schematic diagram of lipid conversion reactions essential for cell signaling.</div><div class="section-paragraph">(F) Correlation among DG, phosphatidic acid (PA), and phospholipases C (PLCs; cleaves PIP<sub>2</sub> into DG and IP<sub>3</sub>), Akt kinases (interact with PIP<sub>3</sub>), protein kinases C (PKCs; interact with DG), and DG kinases (DGKs; phosphorylate DG to produce PA).</div><div class="section-paragraph">(G) <em>IDH1</em> mutants display elevated abundance of glucose, glycolytic intermediate metabolites, and 2-HG, along with reduced abundance of glutamate and serine.</div><div class="section-paragraph">(H) GLUD1 protein expression is upregulated in <em>IDH1</em> mutants in both discovery and validation cohorts.</div><div class="section-paragraph">CE, Cholesteryl ester; CL, Cardiolipin; Cer, Ceramide; FA, Fatty acid; GP, Glycerophospholipid; Glc, Glucose; Glu, Glutamate; HexCer, Hexosylceramide; LCFA, Long chain fatty acid; LacCer, Lactosylceramides; MUFA, Monounsaturated fatty acid; OEA, Oleoylethanolamide; PCO, Phosphatidylcholine with an alkyl ether substituent; PCP, Phosphatidylcholine with a plasmalogen substituent; PEO, Phosphatidylethanolamine with an alkyl ether substituent. PEP (in panel A): Phosphatidylethanolamine with a plasmalogen substituent. PEP (G): Phosphoenolpyruvic acid. PI, Phosphatidylinositol; PIO, Phosphatidylinositol with an alkyl ether substituent; PIP, Phosphatidylinositol with a plasmalogen substituent; PLC, Phospholipase C; Pyr, Pyruvic acid; SM, Sphingomyelin; SP, Sphingolipid; 3PG, 3-Phosphoglyceric acid.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2495" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/e3cb0170-cec6-4600-9946-2b580fe89488/gr6.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/45ffa233-6dfe-47a6-b7bf-c68fc691fc2c/gr6_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr6.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">We explored the connection between the differential abundance of 22:4- and 22:6-containing lipids and their metabolically related neuroprotective proteins (<div class="dropBlock reference-citations"><a href="#bib14" id="back-bib14" data-db-target-for="bib14-d4222834e2783" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib14">Bhagat and Das, 2015</a><div data-db-target-of="bib14-d4222834e2783" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bhagat U. </li><li class="loa__item">Das U.N. </li></ul><div class="reference-citations__detials">Potential role of dietary lipids in the prophylaxis of some clinical conditions.<div class="popup-ref"><span class="ref__series"><i>Arch. Med. Sci.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">807-818</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.5114%2Faoms.2015.53302&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26322094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84939169189&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=807-818&journal=Arch.+Med.+Sci.&author=Bhagat+U.&author=Das+U.N.&title=Potential+role+of+dietary+lipids+in+the+prophylaxis+of+some+clinical+conditions">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A co-regulation example is shown with 22:6 (likely docosahexaenoic acid, DHA) and ACSL6 (an acyl-coA synthetase) (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>C). Tumor samples had drastically diminished protein expression of <em>ACSL6</em> (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D) and increased content of DHA-containing phosphatidylglycerols (PGs) (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S6</a>C) and TGs (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S6</a>D), while other phosphatidylethanolamine (PE), PC, and phosphatidylserine (PS) DHA-containing lipids were downregulated (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S6</a>E). In addition, the proneural-like subtype demonstrated elevated expression of ACSL6 and phospholipids carrying DHA compared to the mesenchymal-like subtype. With respect to DHA metabolism, normal tissues were most similar to the proneural-like subtype and least similar to the mesenchymal-like subtype.</div><div class="section-paragraph">H2B acetylation-related pathway analysis identified upregulation of the ferroptosis pathway in mesenchymal-like GBMs (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr5" href="#gr5" data-locator="fig5" id="back-gr5" class="figure-link scroll-into-link">Figure 5</a>C). For example, proteins ACSL4 and ALOX5 (arachidonate 5-lipoxygenase) were significantly upregulated only in the mesenchymal-like subtype (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>D). ACSL4 incorporates arachidonic acid (AA - 20:4) and adrenic acid (AdA - 22:4) into PEs (<div class="dropBlock reference-citations"><a href="#bib34" id="back-bib34" data-db-target-for="bib34-d4222834e2812" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib34">Doll et al., 2017</a><div data-db-target-of="bib34-d4222834e2812" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Doll S. </li><li class="loa__item">Proneth B. </li><li class="loa__item">Tyurina Y.Y. </li><li class="loa__item">Panzilius E. </li><li class="loa__item">Kobayashi S. </li><li class="loa__item">Ingold I. </li><li class="loa__item">Irmler M. </li><li class="loa__item">Beckers J. </li><li class="loa__item">Aichler M. </li><li class="loa__item">Walch A. </li><li>et al.</li></ul><div class="reference-citations__detials">ACSL4 dictates ferroptosis sensitivity by shaping cellular lipid composition.<div class="popup-ref"><span class="ref__series"><i>Nat. Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">91-98</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnchembio.2239&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27842070&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84995468814&amp;cf=" onclick="newWindow(this.href);return false">Scopus (472)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=91-98&journal=Nat.+Chem.+Biol.&author=Doll+S.&author=Proneth+B.&author=Tyurina+Y.Y.&author=Panzilius+E.&author=Kobayashi+S.&author=Ingold+I.&author=Irmler+M.&author=Beckers+J.&author=Aichler+M.&author=Walch+A.&title=ACSL4+dictates+ferroptosis+sensitivity+by+shaping+cellular+lipid+composition">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib56" id="back-bib56" data-db-target-for="bib56-d4222834e2815" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib56">Kagan et al., 2017</a><div data-db-target-of="bib56-d4222834e2815" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kagan V.E. </li><li class="loa__item">Mao G. </li><li class="loa__item">Qu F. </li><li class="loa__item">Angeli J.P.F. </li><li class="loa__item">Doll S. </li><li class="loa__item">Croix C.S. </li><li class="loa__item">Dar H.H. </li><li class="loa__item">Liu B. </li><li class="loa__item">Tyurin V.A. </li><li class="loa__item">Ritov V.B. </li><li>et al.</li></ul><div class="reference-citations__detials">Oxidized arachidonic and adrenic PEs navigate cells to ferroptosis.<div class="popup-ref"><span class="ref__series"><i>Nat. Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">81-90</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnchembio.2238&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27842066&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84995452435&amp;cf=" onclick="newWindow(this.href);return false">Scopus (429)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=81-90&journal=Nat.+Chem.+Biol.&author=Kagan+V.E.&author=Mao+G.&author=Qu+F.&author=Angeli+J.P.F.&author=Doll+S.&author=Croix+C.S.&author=Dar+H.H.&author=Liu+B.&author=Tyurin+V.A.&author=Ritov+V.B.&title=Oxidized+arachidonic+and+adrenic+PEs+navigate+cells+to+ferroptosis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and ALOX5 catalyzes oxidation of PUFAs (<div class="dropBlock reference-citations"><a href="#bib45" id="back-bib45" data-db-target-for="bib45-d4222834e2818" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib45">Gaschler and Stockwell, 2017</a><div data-db-target-of="bib45-d4222834e2818" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaschler M.M. </li><li class="loa__item">Stockwell B.R. </li></ul><div class="reference-citations__detials">Lipid peroxidation in cell death.<div class="popup-ref"><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">482</span></span>: <span class="ref__seriesPages">419-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.bbrc.2016.10.086&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28212725&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85012272811&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2017&pages=419-425&journal=Biochem.+Biophys.+Res.+Commun.&author=Gaschler+M.M.&author=Stockwell+B.R.&title=Lipid+peroxidation+in+cell+death">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Their upregulation could indicate a higher content of oxidized PEs in this subtype. Downregulation of intact PE with PUFAs was also observed in this subtype (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S6</a>G). We also examined diacylglycerol (DG) levels in the context of enzymes related to DG production (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>E). <a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>F shows significant correlation between DGs and AKT1, PLCD3, and PLCG1 protein expression. PLCG1 phosphorylation was also affected by <em>EGFR</em> alterations (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr2" href="#gr2" data-locator="fig2" id="back-gr2" class="figure-link scroll-into-link">Figure 2</a>B).</div><div class="section-paragraph">We compared metabolite abundances in <em>IDH</em>-mutant versus <em>IDH</em>-WT tumors. While 2-HG was the most highly abundant metabolite in <em>IDH</em>-mutant tumors (median log<sub>2</sub>FC = 3.62, FDR &lt; 0.05), we found other differentially present metabolites with p &lt; 0.05, although they did not pass the FDR cutoff (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>G). Several metabolites involved in glycolysis showed increased abundance in <em>IDH</em> mutants, while serine and glutamate levels were reduced. Glutamate may contribute to alpha-ketoglutarate levels and to 2-HG levels via GLUD1- and IDH1-catalyzed reactions. Supporting this hypothesis is the negative correlation between GLUD1 protein expression and glutamate abundance (Pearson’s <em>r</em> = −0.29, p &lt; 0.01) and significant upregulation of GLUD1 in <em>IDH</em> mutants (Wilcoxon p-value &lt; 0.05). Our validation cohort confirmed this elevated expression of GLUD1 in <em>IDH</em>-mutant tumors (Wilcoxon p-value = 0.07, <a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr6" href="#gr6" data-locator="fig6" id="back-gr6" class="figure-link scroll-into-link">Figure 6</a>H).</div></section><section id="sec2.8" data-locator="sec2.8" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key oncogenic pathways and therapeutic opportunities in GBM</span></h3><div class="section-paragraph">We integrated genetic alterations and the RNA, protein, and phosphosite levels per expression subtype to examine three important oncogenic signaling pathways in GBM: RTK/RAS, PI3K/AKT, and p53/cell cycle (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>A). We found that expression outlier percentage was much higher than genetic alteration rate in RTK pathways (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr3" href="#gr3" data-locator="fig3" id="back-gr3" class="figure-link scroll-into-link">Figure 3</a>). Moreover, analyzed tumors harbored at least one genetic alteration or outlier expression in at least one of the three pathways.<div class="floatDisplay"><figure id="gr7" class="" tabindex="0"><img class="b-lazy" loading="lazy" data-src="/cms/attachment/20dd5db6-16d8-417d-91f7-c04d2d2d63d1/gr7.jpg" alt="Figure thumbnail gr7" data-locator="fig7" data-high-res="/cms/attachment/e516b23e-3b3d-4d0f-9100-05dcb58ff2ac/gr7_lrg.jpg" data-large-image="/cms/attachment/20dd5db6-16d8-417d-91f7-c04d2d2d63d1/gr7.jpg"></img><figcaption><div class="figure__caption__body"><div class="figure__title"><span data-fv-label="Fig. 7" class="label figure__label">Figure 7</span><span class="figure__title__text">Summary of pathway alterations and potential therapeutic targets</span></div><div class="accordion"><a href="#" class="accordion__control" aria-expanded="false"><i class="icon-section_arrow_d"></i><span>Show full caption</span></a><div class="accordion__content"><div class="figure__caption__text"><div class="section-paragraph">(A) Three oncogenic pathways frequently altered in GBM. Each gene is annotated with mutational and CNV frequency, RNA, protein, and phosphoprotein abundance, by multi-omics subtype. Horizontal bar below gene box indicates frequency of alteration across all tumors. Also indicated are the proportion of tumors with genetic alterations (first percentage) and protein and phosphoprotein outlier expression (second percentage) for each pathway.</div><div class="section-paragraph">(B) Dysregulated phospho-signaling in RTK, PI3K, WNT, and NOTCH pathways across all tumors. Thickness of a kinase-substrate connecting line indicates degree to which variation in kinase phosphorylation explains observed variation in the substrate phosphosite abundance. Line color indicates percentage of samples with outlier phosphorylation. Kinases governing multiple substrates with substantial phosphorylation outliers may be potential therapeutic targets.</div><div class="section-paragraph">(C) Drug connectivity analysis using alteration-specific transcriptional (CLUE and iLINCS) and phosphoproteomic (P100) signatures (altered tumors versus WT tumors). Twenty compounds that most strongly reverse or enhance the signature are highlighted along with their known mechanisms of action.</div><div class="section-paragraph">See also <a data-locator="mmc1" aria-expanded="false" aria-controls="crosref2500" href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" target="_blank">Figure S7</a> and <a data-locator="mmc6" aria-expanded="false" aria-controls="crosref2505" href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6" target="_blank">Tables S5</a> and <a data-locator="mmc7" aria-expanded="false" aria-controls="crosref2510" href="/cms/10.1016/j.ccell.2021.01.006/attachment/5ab0cf58-b77f-4522-a878-83ed50e51c8d/mmc7" target="_blank">S6</a>.</div></div></div></div><ul class="rlist--inline separator download-links"><li><a target="_blank" href="/cms/attachment/20dd5db6-16d8-417d-91f7-c04d2d2d63d1/gr7.jpg" title="View Large Image in a new tab">View Large
                                            Image
                                        </a></li><li class="download-links__figureViewer"><a href="#" class="figure__open__ctrl">Figure Viewer</a></li><li><a href="/cms/attachment/e516b23e-3b3d-4d0f-9100-05dcb58ff2ac/gr7_lrg.jpg" class="download-links__download-Hi-res" target="_blank" download="">Download Hi-res
                                            image
                                        </a></li><li class="hidden-xs"><a href="/action/downloadFigures?pii=S1535-6108(21)00050-7&amp;id=gr7.jpg" class="download-links__download-This">Download (PPT)</a></li></ul></div></figcaption></figure></div></div><div class="section-paragraph">In the RTK/RAS pathway, classical tumors predominantly showed amplified <em>EGFR</em>, while proneural and <em>IDH</em>-mutant tumors showed amplified <em>PDGFRA</em>, both resulting in higher RNA, protein, and phosphosite abundances of EGFR and PDGFRA, respectively, illustrating convergence at the functional level. For mesenchymal tumors, we observed upregulated MET and downregulated NF1 protein abundance.</div><div class="section-paragraph">In the PI3K pathway, proneural, mesenchymal, and classical tumors showed lower expression of <em>PTEN</em> due to mutations and deletions, which potentially activate AKT1 and AKT2 through phosphatidylinositol (3,4,5)-trisphosphate (PIP<sub>3</sub>). In contrast, AKT3 expression was higher in <em>IDH</em>-mutant and proneural tumors, explained by active expression of <em>AKT3</em> in adult brains (<div class="dropBlock reference-citations"><a href="#bib37" id="back-bib37" data-db-target-for="bib37-d4222834e2893" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib37">Easton et al., 2005</a><div data-db-target-of="bib37-d4222834e2893" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Easton R.M. </li><li class="loa__item">Cho H. </li><li class="loa__item">Roovers K. </li><li class="loa__item">Shineman D.W. </li><li class="loa__item">Mizrahi M. </li><li class="loa__item">Forman M.S. </li><li class="loa__item">Lee V.M.-Y. </li><li class="loa__item">Szabolcs M. </li><li class="loa__item">de Jong R. </li><li class="loa__item">Oltersdorf T. </li><li>et al.</li></ul><div class="reference-citations__detials">Role for Akt3/protein kinase Bgamma in attainment of normal brain size.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1869-1878</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1128%2FMCB.25.5.1869-1878.2005&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15713641&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044382806&amp;cf=" onclick="newWindow(this.href);return false">Scopus (441)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=1869-1878&journal=Mol.+Cell.+Biol.&author=Easton+R.M.&author=Cho+H.&author=Roovers+K.&author=Shineman+D.W.&author=Mizrahi+M.&author=Forman+M.S.&author=Lee+V.M.-Y.&author=Szabolcs+M.&author=de+Jong+R.&author=Oltersdorf+T.&title=Role+for+Akt3%2Fprotein+kinase+Bgamma+in+attainment+of+normal+brain+size">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In the p53/cell cycle pathway, we observed subtype-specific amplification and increased expression of <em>MDM2</em> in mesenchymal and <em>MDM4</em> in proneural and classical tumors. We also observed differences between IDH WT and mutant tumors in <em>CDKN2A/B</em>.</div><div class="section-paragraph">In conjunction with druggability information from DGIdb (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d4222834e2905" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Cotto et al., 2018</a><div data-db-target-of="bib30-d4222834e2905" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cotto K.C. </li><li class="loa__item">Wagner A.H. </li><li class="loa__item">Feng Y.-Y. </li><li class="loa__item">Kiwala S. </li><li class="loa__item">Coffman A.C. </li><li class="loa__item">Spies G. </li><li class="loa__item">Wollam A. </li><li class="loa__item">Spies N.C. </li><li class="loa__item">Griffith O.L. </li><li class="loa__item">Griffith M. </li></ul><div class="reference-citations__detials">DGIdb 3.0: a redesign and expansion of the drug-gene interaction database.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D1068-D1073</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1143&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29156001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85070670869&amp;cf=" onclick="newWindow(this.href);return false">Scopus (214)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1068-D1073&journal=Nucleic+Acids+Res.&author=Cotto+K.C.&author=Wagner+A.H.&author=Feng+Y.-Y.&author=Kiwala+S.&author=Coffman+A.C.&author=Spies+G.&author=Wollam+A.&author=Spies+N.C.&author=Griffith+O.L.&author=Griffith+M.&title=DGIdb+3.0%3A+a+redesign+and+expansion+of+the+drug-gene+interaction+database">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and DEPO (<div class="dropBlock reference-citations"><a href="#bib123" id="back-bib123" data-db-target-for="bib123-d4222834e2908" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib123">Sun et al., 2018</a><div data-db-target-of="bib123-d4222834e2908" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sun S.Q. </li><li class="loa__item">Mashl R.J. </li><li class="loa__item">Sengupta S. </li><li class="loa__item">Scott A.D. </li><li class="loa__item">Wang W. </li><li class="loa__item">Batra P. </li><li class="loa__item">Wang L.-B. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Ding L. </li></ul><div class="reference-citations__detials">Database of evidence for precision oncology portal.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">4315-4317</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbty531&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30535306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85050591786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=4315-4317&journal=Bioinforma.+Oxf.+Engl.&author=Sun+S.Q.&author=Mashl+R.J.&author=Sengupta+S.&author=Scott+A.D.&author=Wang+W.&author=Batra+P.&author=Wang+L.-B.&author=Wyczalkowski+M.A.&author=Ding+L.&title=Database+of+evidence+for+precision+oncology+portal">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we conducted kinase-substrate and outlier analyses to identify druggable pairs (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>B; <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6" class="scroll-into-link">Table S5</a>). We found that GSK3B phosphorylation is positively associated with phosphorylation of its downstream proteins involved in mammalian target of rapamycin (mTOR) signaling (e.g., RPTOR and TSC1) and Wnt signaling (e.g., CTNNB1 and APC). Another player in mTOR signaling, AKT1S1, had many significant connections with AKT1, AKT2, and AKT3 kinases. EGFR was also found to phosphorylate CTNNB1 S33 at the N-terminal, known to mediate CTNNB1 proteasomal degradation (<div class="dropBlock reference-citations"><a href="#bib101" id="back-bib101" data-db-target-for="bib101-d4222834e2917" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib101">Park et al., 2004</a><div data-db-target-of="bib101-d4222834e2917" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Park C.S. </li><li class="loa__item">Kim S.I. </li><li class="loa__item">Lee M.S. </li><li class="loa__item">Youn C.-Y. </li><li class="loa__item">Kim D.J. </li><li class="loa__item">Jho E.-H. </li><li class="loa__item">Song W.K. </li></ul><div class="reference-citations__detials">Modulation of beta-catenin phosphorylation/degradation by cyclin-dependent kinase 2.<div class="popup-ref"><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">279</span></span>: <span class="ref__seriesPages">19592-19599</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=14985333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-2442616062&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=19592-19599&journal=J.%C2%A0Biol.+Chem.&author=Park+C.S.&author=Kim+S.I.&author=Lee+M.S.&author=Youn+C.-Y.&author=Kim+D.J.&author=Jho+E.-H.&author=Song+W.K.&title=Modulation+of+beta-catenin+phosphorylation%2Fdegradation+by+cyclin-dependent+kinase+2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphosite outlier analysis corroborated these findings with interactions for GSK3B, AKT1, MAPK1, MAPK3, and EGFR (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S7</a>B; <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6" class="scroll-into-link">Table S5</a>). The MAP kinase cascade was associated with diverse proteins, including ABL1 kinase, which in turn had a strong association with HDAC2 deacetylase with ∼5% of outliers. We also found increased HDAC2 S422 phosphorylation, coinciding with its functional activity (<div class="dropBlock reference-citations"><a href="#bib39" id="back-bib39" data-db-target-for="bib39-d4222834e2927" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib39">Eom and Kook, 2015</a><div data-db-target-of="bib39-d4222834e2927" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Eom G.H. </li><li class="loa__item">Kook H. </li></ul><div class="reference-citations__detials">Role of histone deacetylase 2 and its posttranslational modifications in cardiac hypertrophy.<div class="popup-ref"><span class="ref__series"><i>BMB Rep.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">131-138</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.5483%2FBMBRep.2015.48.3.242&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25388210&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84928262932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=131-138&journal=BMB+Rep.&author=Eom+G.H.&author=Kook+H.&title=Role+of+histone+deacetylase+2+and+its+posttranslational+modifications+in+cardiac+hypertrophy">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">Using the Library of Integrated Network-Based Cellular Signatures (LINCS) (<div class="dropBlock reference-citations"><a href="#bib59" id="back-bib59" data-db-target-for="bib59-d4222834e2932" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib59">Keenan et al., 2018</a><div data-db-target-of="bib59-d4222834e2932" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Keenan A.B. </li><li class="loa__item">Jenkins S.L. </li><li class="loa__item">Jagodnik K.M. </li><li class="loa__item">Koplev S. </li><li class="loa__item">He E. </li><li class="loa__item">Torre D. </li><li class="loa__item">Wang Z. </li><li class="loa__item">Dohlman A.B. </li><li class="loa__item">Silverstein M.C. </li><li class="loa__item">Lachmann A. </li><li>et al.</li></ul><div class="reference-citations__detials">The library of integrated network-based cellular signatures NIH program: system-level cataloging of human cells response to perturbations.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">13-24</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29199020&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036553710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (115)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=13-24&journal=Cell+Syst.&author=Keenan+A.B.&author=Jenkins+S.L.&author=Jagodnik+K.M.&author=Koplev+S.&author=He+E.&author=Torre+D.&author=Wang+Z.&author=Dohlman+A.B.&author=Silverstein+M.C.&author=Lachmann+A.&title=The+library+of+integrated+network-based+cellular+signatures+NIH+program%3A+system-level+cataloging+of+human+cells+response+to+perturbations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d4222834e2935" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Stathias et al., 2019</a><div data-db-target-of="bib118-d4222834e2935" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stathias V. </li><li class="loa__item">Turner J. </li><li class="loa__item">Koleti A. </li><li class="loa__item">Vidovic D. </li><li class="loa__item">Cooper D. </li><li class="loa__item">Fazel-Najafabadi M. </li><li class="loa__item">Pilarczyk M. </li><li class="loa__item">Terryn R. </li><li class="loa__item">Chung C. </li><li class="loa__item">Umeano A. </li><li>et al.</li></ul><div class="reference-citations__detials">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D431-D439</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkz1023&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077665017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D431-D439&journal=Nucleic+Acids+Res.&author=Stathias+V.&author=Turner+J.&author=Koleti+A.&author=Vidovic+D.&author=Cooper+D.&author=Fazel-Najafabadi+M.&author=Pilarczyk+M.&author=Terryn+R.&author=Chung+C.&author=Umeano+A.&title=LINCS+Data+Portal+2.0%3A+next+generation+access+point+for+perturbation-response+signatures">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we calculated the similarity between alteration-specific RNA or phosphoprotein signatures from our study with corresponding transcriptional (L1000 assay) (<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d4222834e2938" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d4222834e2938" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+next+generation+connectivity+map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and phosphoproteomic LINCS signatures (P100 assay) (<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d4222834e2941" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">Litichevskiy et al., 2018</a><div data-db-target-of="bib82-d4222834e2941" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Litichevskiy L. </li><li class="loa__item">Peckner R. </li><li class="loa__item">Abelin J.G. </li><li class="loa__item">Asiedu J.K. </li><li class="loa__item">Creech A.L. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Davison D. </li><li class="loa__item">Dunning C.M. </li><li class="loa__item">Egertson J.D. </li><li class="loa__item">Egri S. </li><li>et al.</li></ul><div class="reference-citations__detials">A library of phosphoproteomic and chromatin signatures for characterizing cellular responses to drug perturbations.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">424-443.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29655704&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85045118030&amp;cf=" onclick="newWindow(this.href);return false">Scopus (26)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=424-443.e7&journal=Cell+Syst&author=Litichevskiy+L.&author=Peckner+R.&author=Abelin+J.G.&author=Asiedu+J.K.&author=Creech+A.L.&author=Davis+J.F.&author=Davison+D.&author=Dunning+C.M.&author=Egertson+J.D.&author=Egri+S.&title=A+library+of+phosphoproteomic+and+chromatin+signatures+for+characterizing+cellular+responses+to+drug+perturbations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to identify compounds predicted to reverse tumor signatures of the cohort. Phosphoproteomic data yielded more robust results than the transcriptional-response data. For example, while RNA signatures suggested that HDAC inhibitors might reverse the EGFR signature, phosphoproteomic data indicated kinase inhibitors, beyond the expected EGFR inhibitors, to be more highly connected with potential inhibitory effects (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C). In contrast, <em>NF1</em>-altered samples exhibited concordance between the transcriptional and phosphoproteomic perturbation-response analyses: both suggested MAPK inhibitors as top signature-reversing candidates (<a data-link="modal" data-target="#image-S1535-6108(21)00050-7gr7" href="#gr7" data-locator="fig7" id="back-gr7" class="figure-link scroll-into-link">Figure 7</a>C). RB1 alteration and immune subtypes agreed variously when comparing perturbations predicted to reverse the gene-altered cell states using transcriptomics versus phosphoproteomics readouts (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S7</a>C, <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/5ab0cf58-b77f-4522-a878-83ed50e51c8d/mmc7" class="scroll-into-link">Table S6</a>).</div></section></section><section id="sec3" data-locator="sec3" tabindex="0"><h2 class="top" id="secsectitle0080" data-left-hand-nav="Discussion" tabindex="0"><span class="top__sub"></span><span class="top__text">Discussion</span></h2><div class="section-paragraph">GBM was one of the earliest subjects of deep genomic and transcriptomic analysis (<div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4222834e2964" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brennan et al., 2013</a><div data-db-target-of="bib18-d4222834e2964" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brennan C.W. </li><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Campos B. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Salama S.R. </li><li class="loa__item">Zheng S. </li><li class="loa__item">Chakravarty D. </li><li class="loa__item">Sanborn J.Z. </li><li class="loa__item">Berman S.H. </li><li>et al.</li></ul><div class="reference-citations__detials">The somatic genomic landscape of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and targeted MS studies (<div class="dropBlock reference-citations"><a href="#bib49" id="back-bib49" data-db-target-for="bib49-d4222834e2967" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib49">Gu et al., 2017</a><div data-db-target-of="bib49-d4222834e2967" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gu X. </li><li class="loa__item">Hua Z. </li><li class="loa__item">Dong Y. </li><li class="loa__item">Zhan Y. </li><li class="loa__item">Zhang X. </li><li class="loa__item">Tian W. </li><li class="loa__item">Liu Z. </li><li class="loa__item">Thiele C.J. </li><li class="loa__item">Li Z. </li></ul><div class="reference-citations__detials">Proteome and acetylome analysis identifies novel pathways and targets regulated by Perifosine in neuroblastoma.<div class="popup-ref"><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">42062</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fsrep42062&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28165023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85011872870&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42062&journal=Sci.+Rep.&author=Gu+X.&author=Hua+Z.&author=Dong+Y.&author=Zhan+Y.&author=Zhang+X.&author=Tian+W.&author=Liu+Z.&author=Thiele+C.J.&author=Li+Z.&title=Proteome+and+acetylome+analysis+identifies+novel+pathways+and+targets+regulated+by+Perifosine+in+neuroblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib52" id="back-bib52" data-db-target-for="bib52-d4222834e2970" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib52">He et al., 2007</a><div data-db-target-of="bib52-d4222834e2970" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">He H. </li><li class="loa__item">Conrad C.A. </li><li class="loa__item">Nilsson C.L. </li><li class="loa__item">Ji Y. </li><li class="loa__item">Schaub T.M. </li><li class="loa__item">Marshall A.G. </li><li class="loa__item">Emmett M.R. </li></ul><div class="reference-citations__detials">Method for lipidomic analysis: p53 expression modulation of sulfatide, ganglioside, and phospholipid composition of U87 MG glioblastoma cells.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">79</span></span>: <span class="ref__seriesPages">8423-8430</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac071413m&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17929901&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-36448939111&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=8423-8430&journal=Anal.+Chem.&author=He+H.&author=Conrad+C.A.&author=Nilsson+C.L.&author=Ji+Y.&author=Schaub+T.M.&author=Marshall+A.G.&author=Emmett+M.R.&title=Method+for+lipidomic+analysis%3A+p53+expression+modulation+of+sulfatide%2C+ganglioside%2C+and+phospholipid+composition+of+U87+MG+glioblastoma+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). However, most patients are still treated with a standard of care developed almost two decades ago (<div class="dropBlock reference-citations"><a href="#bib119" id="back-bib119" data-db-target-for="bib119-d4222834e2973" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib119">Stupp et al., 2005</a><div data-db-target-of="bib119-d4222834e2973" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stupp R. </li><li class="loa__item">Mason W.P. </li><li class="loa__item">van den Bent M.J. </li><li class="loa__item">Weller M. </li><li class="loa__item">Fisher B. </li><li class="loa__item">Taphoorn M.J.B. </li><li class="loa__item">Belanger K. </li><li class="loa__item">Brandes A.A. </li><li class="loa__item">Marosi C. </li><li class="loa__item">Bogdahn U. </li><li>et al.</li></ul><div class="reference-citations__detials">Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma.<div class="popup-ref"><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">352</span></span>: <span class="ref__seriesPages">987-996</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa043330&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15758009&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044366163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12069)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=987-996&journal=N.%C2%A0Engl.+J.+Med.&author=Stupp+R.&author=Mason+W.P.&author=van+den+Bent+M.J.&author=Weller+M.&author=Fisher+B.&author=Taphoorn+M.J.B.&author=Belanger+K.&author=Brandes+A.A.&author=Marosi+C.&author=Bogdahn+U.&title=Radiotherapy+plus+concomitant+and+adjuvant+temozolomide+for+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), underscoring the need for deeper insights. Here, we extended classical sequencing approaches with comprehensive integration of MS-based proteome, phosphoproteome, acetylome, metabolome, and lipidome analyses and single-cell transcriptomics. Multi-omics analysis identified a subset of patients with mixed subtypes compared with traditional sequencing-based subtypes, who exhibit shortened overall survival. Phosphoproteomic data indicate that PLCG1 and PTPN11 act as a common signaling hub for multiple RTKs.</div><div class="section-paragraph">RNA and protein expression data from bulk tumors indicate that GBM subtypes differ in infiltrating macrophages and the distribution of specific immune cell types. In particular, we discovered an unexpected immune subtype im3 exhibiting a relative depletion of the macrophage-microglia immunosuppressive infiltration typical of GBM, but which contains significant enrichment of T lymphocytes and natural killer (NK) cells. It is also enriched for <em>IDH</em>-mutated tumors. We validated these findings using IHC and an independent patient cohort. Interestingly, im1 TAMs demonstrated upregulation of M2 polarization markers, such as CD163 and MRC1, suggesting a role in tumor promotion (<div class="dropBlock reference-citations"><a href="#bib106" id="back-bib106" data-db-target-for="bib106-d4222834e2980" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib106">Pinto et al., 2019</a><div data-db-target-of="bib106-d4222834e2980" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pinto M.L. </li><li class="loa__item">Rios E. </li><li class="loa__item">Durães C. </li><li class="loa__item">Ribeiro R. </li><li class="loa__item">Machado J.C. </li><li class="loa__item">Mantovani A. </li><li class="loa__item">Barbosa M.A. </li><li class="loa__item">Carneiro F. </li><li class="loa__item">Oliveira M.J. </li></ul><div class="reference-citations__detials">The two faces of tumor-associated macrophages and their clinical significance in colorectal cancer.<div class="popup-ref"><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1875</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.3389%2Ffimmu.2019.01875&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31481956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071761383&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1875&journal=Front.+Immunol.&author=Pinto+M.L.&author=Rios+E.&author=Dur%C3%A3es+C.&author=Ribeiro+R.&author=Machado+J.C.&author=Mantovani+A.&author=Barbosa+M.A.&author=Carneiro+F.&author=Oliveira+M.J.&title=The+two+faces+of+tumor-associated+macrophages+and+their+clinical+significance+in+colorectal+cancer">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">The mesenchymal subtype has high bulk-level RNA expression of EMT signatures (<div class="dropBlock reference-citations"><a href="#bib11" id="back-bib11" data-db-target-for="bib11-d4222834e2985" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib11">Behnan et al., 2019</a><div data-db-target-of="bib11-d4222834e2985" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Behnan J. </li><li class="loa__item">Finocchiaro G. </li><li class="loa__item">Hanna G. </li></ul><div class="reference-citations__detials">The landscape of the mesenchymal signature in brain tumours.<div class="popup-ref"><span class="ref__series"><i>Brain J. Neurol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">142</span></span>: <span class="ref__seriesPages">847-866</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbrain%2Fawz044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30946477&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85064239772&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2019&pages=847-866&journal=Brain+J.+Neurol.&author=Behnan+J.&author=Finocchiaro+G.&author=Hanna+G.&title=The+landscape+of+the+mesenchymal+signature+in+brain+tumours">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), but it was not known if this was due to the tumor cells themselves or the high number of infiltrating immune cells. Our data indicate that tumor cells in the mesenchymal subtype display an enhanced EMT signature, along with an increase in the relative proportion of immune cells in the tumor stroma. Cell-type-specific gene expression indicates that both the tumor cells and the stromal components contribute to the overall mesenchymal signatures. This study reveals an association between H2B acetylation and patterns of protein expression associated with immune cell functions in GBM. Furthermore, comparison of the lipid and metabolic signatures in GBM subtypes and normal brain tissues reveals shared characteristics that may be associated with neuronal phenotypes and IDH status. Similarly, from a metabolomic view, mesenchymal-like GBMs differ substantially from other subtypes and have specific metabolic vulnerabilities not present in other GBMs.</div><div class="section-paragraph">The multidimensional analysis of patient specimens described in this investigation adds context to prior genomic and transcription-based investigations of GBM and suggests avenues for further mechanistic studies. Rapid advancement of single-cell genomics and proteomics technologies will facilitate deeper analyses of GBM heterogeneity and TME interactions. We hope these advances will improve patient stratification for clinical trials and lead, ultimately, to personalized treatments.</div></section><section id="sec4" data-locator="sec4" tabindex="0"><h2 class="top" id="secsectitle0085" data-left-hand-nav="Consortia" tabindex="0"><span class="top__sub"></span><span class="top__text">Consortia</span></h2><div class="section-paragraph">The members of the National Cancer Institute Clinical Proteomic Tumor Analysis Consortium are Anupriya Agarwal, Mitual Amin, Eunkyung An, Matthew L. Anderson, David W. Andrews, Jessika Baral, Jill S. Barnholtz-Sloan, Thomas Bauer, Chet Birger, Michael J. Birrer, Kent J. Bloodsworth, Lili Blumenberg, Simina M. Boca, William E. Bocik, Hannah Boekweg, Emily S. Boja, Uma Borate, Melissa Borucki, Lisa M. Bramer, Meghan C. Burke, Shuang Cai, Anna Pamela. Calinawan, Song Cao, Wagma Caravan, Steven A. Carr, Michele Ceccarelli, Sandra Cerda, Daniel W. Chan, Changcun Chang, Alyssa Charamut, Feng Chen, Lin S. Chen, Xi S. Chen, David Chesla, Milan G. Chheda, Arul M. Chinnaiyan, Shrabanti Chowdhury, Rosalie K. Chu, Marcin P. Cieślik, David J. Clark, Karl Clauser, Antonio Colaprico, MacIntosh Cornwell, Sandra L. Cottingham, Sneha P. Couvillion, Daniel Cui Zhou, Houston Culpepper, Andrzej Czekański, Tomasz Czernicki, Fulvio D'Angelo, Jacob Day, Stephanie De Young, Emek Demir, Alexis Demopoulos, Saravana Mohan Dhanasekaran, Rajiv Dhir, Li Ding, Marcin J. Domagalski, Yongchao Dou, Brian Druker, Elizabeth Duffy, Maureen Dyer, Nathan J. Edwards, Robert Edwards, Kimberly Elburn, Matthew J. Ellis, Jennifer Eschbacher, David Fenyö, Jayson B. Field, Steven Foltz, Alicia Francis, Stacey Gabriel, Nikolay Gabrovski, Luciano Garofano, Yifat Geffen, Gad Getz, Michael A. Gillette, Andrew K. Godwin, Denis Golbin, Charles A. Goldthwaite, Jr., Marina A. Gritsenko, Zeynep Gumus, Ziad Hanhan, Linda I. Hannick, Pushpa Hariharan, Dawn Hayward, David Heiman, Brittany L. Henderson, Tara Hiltke, Barbara Hindenach, Katherine A. Hoadley, Runyu Hong, Galen Hostetter, Chen Huang, Antonio Iavarone, Eric Jaehnig, Paweł Jarmużek, Scott D. Jewell, Jiayi Ji, Nan Ji, Corbin D. Jones, Marcin Jędryka, Bożena Kamińska, Renee Karabon, Alla Karpova, Alcida Karz, Wojciech Kaspera, Emily Kawaler, Selim Kaycee, Karen A. Ketchum, Albert H. Kim, Lyndon Kim, Christopher R. Kinsinger, Ramani B. Kothadia, Azra Krek, Karsten Krug, Chandan Kumar-Sinha, Jennifer E. Kyle, Jonathan Lei, Felipe D. Leprevost, Kai Li, Qin Li, Qing Kay Li, Tian Li, Yize Li, Zibo Li, Wen-Wei Liang, Yuxing Liao, Jena Lilly, Caleb M. Lindgren, Hongwei Liu, Tao Liu, Wenke Liu, Shuangjia Lu, Jan Lubínski, Weiping Ma, Melanie MacMullan, Rashna Madan, William Maggio, Piotr Makarewicz, Ewa Malc, Anna Malovannaya, D. R. Mani, Andrzej Marchel, Sailaja Mareedu, Sanford P. Markey, Annette Marrero-Oliveras, John Martignetti, Nina Martinez, Yosef Maruvka, Rafał Matkowski, Nicollette Maunganidze, Jason E. McDermott, Peter B. McGarvey, John McGee, Joshua F. McMichael, Mehdi Mesri, Piotr Mieczkowski, Simona Migliozzi, Francesmary Modugno, Matthew E. Monroe, Rebecca Montgomery, Jamie Moon, Ronald J. Moore, Sergiusz Nawrocki, Alexey I. Nesvizhskii, Chelsea J. Newton, Lindsey K. Olsen, Gilbert S. Omenn, Umut Ozbek, Oxana V. Paklina, Akash J. Patel, Amanda G. Paulovich, Samuel H. Payne, Amy M. Perou, Francesca Petralia, Vladislav A. Petyuk, Alexander R. Pico, Paul D. Piehowski, Dimitris G. Placantonakis, Larisa Polonskaya, Olga Potapova, Barbara Pruetz, Pietro Pugliese, Liqun Qi, Shakti Ramkissoon, Adam C. Resnick, Boris Reva, Shannon Richey, Gregory Riggins, Anna Roberts-Pilgrim, Karna Robinson, Ana I. Robles, Nancy Roche, Karin D. Rodland, Henry Rodriguez, Daniel C. Rohrer, Brian R. Rood, Larissa Rossell, Joseph Rothstein, Kelly V. Ruggles, Dmitry Rykunov, Shankha Satpathy, Sara R. Savage, Eric E. Schadt, Stephan Schürer, Sohini Sengupta, Yan Shi, Zhiao Shi, Yvonne Shutack, Weiva Sieh, Shilpi Singh, Tara Skelly, Richard D. Smith, Lori J. Sokoll, Xiaoyu Song, Vasileios Stathias, Jakub Stawicki, Stephen E. Stein, Kelly G. Stratton, James Suh, Wojciech Szopa, Dave Tabor, Donghui Tan, Lauren Tang, Darlene Tansil, Ratna R. Thangudu, Mathangi Thiagarajan, Cristina Tognon, Elie Traer, Chia-Feng Tsai, Shirley Tsang, Jeffrey Tyner, Ki Sung Um, Dana R. Valley, Suhas Vasaikar, Negin Vatanian, Uma Velvulou, Michael Vernon, Weiqing Wan, Bing Wang, Junmei Wang, Liang-Bo Wang, Pei Wang, Alex Webster, Sun Wei, Karl K. Weitz, Bo Wen, Michael C. Wendl, Jeffrey R. Whiteaker, George D. Wilson, Maciej Wiznerowicz, Yige Wu, Matthew A. Wyczalkowski, Xiaolu Yang, Lijun Yao, Seungyeul Yoo, Yuriy Zakhartsev, Robert Zelt, Bing Zhang, Hui Zhang, Liwei Zhang, Zhen Zhang, Grace Zhao, Rui Zhao, Yuanli Zhao, Houxiang Zhu, Jun Zhu, Erika Zink.</div></section><section id="sec5" data-locator="sec5" tabindex="0"><h2 class="top" id="secsectitle0090" data-left-hand-nav="STAR★Methods" tabindex="0"><span class="top__sub"></span><span class="top__text">STAR★Methods</span></h2><section id="sec5.1" data-locator="sec5.1" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Key resources table</span></h3><div class="section-paragraph"><div id="undtbl1" class="inline-table" tabindex="0"><div class="inline-table__head no-label"><span class="inline-table__label no-caption">Tabled
                            1</span><span class="inline-table__title"></span></div><table width="100%" cellpadding="0" cellspacing="0" border="0" class="scrollable bordered" tabindex="0"><thead><tr><th class="rowsep">REAGENT or RESOURCE</th><th class="rowsep">SOURCE</th><th class="rowsep">IDENTIFIER</th></tr></thead><tbody><tr><td class="rowsep" colspan="3"><strong>Antibodies</strong></td></tr><tr><td>Mouse monoclonal anti-IDH1-R132H (clone HO9)</td><td>Dianova</td><td>Cat# DIA-H09; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2335716" target="_blank">AB_2335716</a></td></tr><tr><td>Rabbit polyclonal anti-ATRX</td><td>Sigma</td><td>Cat# HPA001906; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_1078249" target="_blank">AB_1078249</a></td></tr><tr><td>Rabbit monoclonal anti-SOX9 (clone D8G8H)</td><td>Cell Signaling</td><td>Cat# 82630S; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2665492" target="_blank">AB_2665492</a></td></tr><tr><td>Mouse monoclonal anti-GAB1 (clone H-7)</td><td>Santa Cruz</td><td>Cat# sc-13319; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2107855" target="_blank">AB_2107855</a></td></tr><tr><td>Rabbit polyclonal anti-CD3 (clone A0452)</td><td>Dako</td><td>Cat# A00452; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2335677" target="_blank">AB_2335677</a></td></tr><tr><td>Mouse monoclonal anti-CD68 (clone KP1)</td><td>Ventana</td><td>Cat# 790-2931; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2335972" target="_blank">AB_2335972</a></td></tr><tr><td>Mouse monoclonal anti-CD163 (clone 10D6)</td><td>Novacastra</td><td>Cat# NCL-L-CD163; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2756375" target="_blank">AB_2756375</a></td></tr><tr><td>Mouse monoclonal anti-PD-1 (clone NAT105)</td><td>Cell Marque</td><td>Cat# 315M-96; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_1160829" target="_blank">AB_1160829</a></td></tr><tr><td>Mouse monoclonal anti-PD-L1 (clone 22C3)</td><td>Dako</td><td>Cat# M3653; RRID: <a class="inter-ref-link" href="http://antibodyregistry.org/AB_2833074" target="_blank">AB_2833074</a></td></tr><tr><td class="rowsep"></td><td class="rowsep"></td><td class="rowsep"></td></tr><tr><td class="rowsep" colspan="3"><strong>Bacterial and virus strains</strong></td></tr><tr><td class="rowsep"></td><td class="rowsep"></td><td class="rowsep"></td></tr><tr><td class="rowsep" colspan="3"><strong>Biological samples</strong></td></tr><tr><td>Primary tumor and normal tissue samples</td><td>This paper</td><td>See Methods: Experimental Model and Subject Details</td></tr><tr><td class="rowsep">Patient-derived xenograft tissue samples</td><td class="rowsep">Washington University in St. Louis</td><td class="rowsep">See Methods: Method Details</td></tr><tr><td class="rowsep" colspan="3"><strong>Chemicals, peptides, and recombinant proteins</strong></td></tr><tr><td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)</td><td>Sigma</td><td>Catalog: H3375</td></tr><tr><td>Acetic Acid, glacial</td><td>Sigma</td><td>Catalog: AX0074-6</td></tr><tr><td>Acetonitrile, HPLC grade</td><td>J.T. Baker</td><td>Catalog: 9829-03</td></tr><tr><td>Acetonitrile anhydrous</td><td>Sigma</td><td>Catalog: 271004</td></tr><tr><td>Ammonium hydroxide solution</td><td>Sigma</td><td>Catalog: 338818</td></tr><tr><td>Aprotinin</td><td>Sigma</td><td>Catalog: A6103</td></tr><tr><td>Dithiothreitol</td><td>Thermo Scientific</td><td>Catalog: 20291</td></tr><tr><td>Ethylenediaminetetraacetic acid</td><td>Sigma</td><td>Catalog: E7889</td></tr><tr><td>Formic acid</td><td>Sigma</td><td>Catalog: 0507</td></tr><tr><td>Iodoacetamide</td><td>Sigma</td><td>Catalog: A3221</td></tr><tr><td>Iron (III) chloride</td><td>Sigma</td><td>Catalog: 451649</td></tr><tr><td>HPLC Grade Water</td><td>J.T. Baker</td><td>Catalog: 4218-03</td></tr><tr><td>Hydroxylamine Solution 50%</td><td>Sigma</td><td>Catalog: 467804</td></tr><tr><td>Leupeptin</td><td>Roche</td><td>Catalog: 11017101001</td></tr><tr><td>Lysyl Endopeptidase</td><td>Wako Chemicals</td><td>Catalog 129-02541</td></tr><tr><td>Methanol, HPLC grade</td><td>Fluka</td><td>Catalog: 34966</td></tr><tr><td>Ni-NTA Superflow Agarose Beads</td><td>Qiagen</td><td>Catalog: 30410</td></tr><tr><td>Phenylmethylsulfonyl fluoride</td><td>Sigma</td><td>Catalog: 93482</td></tr><tr><td>Phosphatase Inhibitor Cocktail 2</td><td>Sigma</td><td>Catalog: P5726</td></tr><tr><td>Phosphatase Inhibitor Cocktail 3</td><td>Sigma</td><td>Catalog: P0044</td></tr><tr><td>Potassium phosphate dibasic</td><td>Sigma</td><td>Catalog: P3786</td></tr><tr><td>Potassium phosphate monobasic</td><td>Sigma</td><td>Catalog: P9791</td></tr><tr><td>PUGNAc</td><td>Sigma</td><td>Catalog: A7229</td></tr><tr><td>Reversed-phase tC18 SepPak</td><td>Waters</td><td>Catalog: WAT054925</td></tr><tr><td>Sequencing grade modified trypsin</td><td>Promega</td><td>Catalog: V517</td></tr><tr><td>Sodium butyrate</td><td>Sigma</td><td>Catalog: 303410</td></tr><tr><td>Sodium chloride</td><td>Sigma</td><td>Catalog: S7653</td></tr><tr><td>Sodium fluoride</td><td>Sigma</td><td>Catalog: S7920</td></tr><tr><td>Tris (hydroxymethyl)aminomethane hydrochloride pH 8.0</td><td>Sigma</td><td>Catalog: T2694</td></tr><tr><td>Trifluoroacetic acid</td><td>Sigma</td><td>Catalog: 91707</td></tr><tr><td class="rowsep">Urea</td><td class="rowsep">Sigma</td><td class="rowsep">Catalog: U0631</td></tr><tr><td class="rowsep" colspan="3"><strong>Critical commercial assays</strong></td></tr><tr><td>BCA Protein Assay Kit</td><td>ThermoFisher Scientific</td><td>Catalog: A53225</td></tr><tr><td>Infinium MethylationEPIC Kit</td><td>Illumina</td><td>Catalog: WG-317-1003</td></tr><tr><td>TMT-11 reagent kit</td><td>ThermoFisher Scientific</td><td>Catalog: A34808</td></tr><tr><td>TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold</td><td>Illumina</td><td>Catalog: RS-122-2301</td></tr><tr><td class="rowsep">PTMScan® Acetyl-Lysine Motif [Ac-K] Kit</td><td class="rowsep">Cell Signaling</td><td class="rowsep">Catalog: 13416</td></tr><tr><td class="rowsep" colspan="3"><strong>Deposited data</strong></td></tr><tr><td>CPTAC GBM proteomic data</td><td>this study</td><td><a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S048" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S048</a>; <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S057" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S057</a>; <a class="inter-ref-link" href="https://pdc.cancer.gov/" target="_blank">https://pdc.cancer.gov/</a></td></tr><tr><td>CPTAC GBM genomic and snRNA-seq data</td><td>this study</td><td><a class="inter-ref-link" href="https://portal.gdc.cancer.gov/projects/CPTAC-3" target="_blank">https://portal.gdc.cancer.gov/projects/CPTAC-3</a></td></tr><tr><td class="rowsep">Clinical and genomic data of the validation cohort</td><td class="rowsep">this study</td><td class="rowsep"><a class="inter-ref-link" href="https://pedcbioportal.kidsfirstdrc.org/study/summary?id=phgg_cbttc" target="_blank">https://pedcbioportal.kidsfirstdrc.org/study/summary?id=phgg_cbttc</a>; <a class="inter-ref-link" href="https://cavatica.sbgenomics.com/u/cavatica/pbta-cbttc/" target="_blank">https://cavatica.sbgenomics.com/u/cavatica/pbta-cbttc/</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Software and algorithms</strong></td></tr><tr><td>Ascore v1.0.6858</td><td>(<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d4222834e3436" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Beausoleil et al., 2006</a><div data-db-target-of="bib10-d4222834e3436" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beausoleil S.A. </li><li class="loa__item">Villén J. </li><li class="loa__item">Gerber S.A. </li><li class="loa__item">Rush J. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1084)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/PNNL-Comp-Mass-Spec/AScore" target="_blank">https://github.com/PNNL-Comp-Mass-Spec/AScore</a></td></tr><tr><td>MASIC</td><td>(<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d4222834e3447" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Monroe et al., 2008</a><div data-db-target-of="bib93-d4222834e3447" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/PNNL-Comp-Mass-Spec/MASIC" target="_blank">https://github.com/PNNL-Comp-Mass-Spec/MASIC</a></td></tr><tr><td>MS-GF+ v9981</td><td>(<div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d4222834e3458" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Kim and Pevzner, 2014</a><div data-db-target-of="bib62-d4222834e3458" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (469)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/MSGFPlus/msgfplus" target="_blank">https://github.com/MSGFPlus/msgfplus</a></td></tr><tr><td>mzRefinery</td><td>(<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d4222834e3469" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Gibbons et al., 2015</a><div data-db-target-of="bib47-d4222834e3469" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbons B.C. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Payne S.H. </li></ul><div class="reference-citations__detials">Correcting systematic bias and instrument measurement drift with mzRefinery.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinforma.+Oxf.+Engl.&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://omics.pnl.gov/software/mzrefinery" target="_blank">https://omics.pnl.gov/software/mzrefinery</a></td></tr><tr><td>BIC-Seq2</td><td>(<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d4222834e3480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Xi et al., 2016</a><div data-db-target-of="bib138-d4222834e3480" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xi R. </li><li class="loa__item">Lee S. </li><li class="loa__item">Xia Y. </li><li class="loa__item">Kim T.-M. </li><li class="loa__item">Park P.J. </li></ul><div class="reference-citations__detials">Copy number analysis of whole-genome data using BIC-seq2 and its application to detection of cancer susceptibility variants.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">6274-6286</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkw491&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27260798&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=6274-6286&journal=Nucleic+Acids+Res.&author=Xi+R.&author=Lee+S.&author=Xia+Y.&author=Kim+T.-M.&author=Park+P.J.&title=Copy+number+analysis+of+whole-genome+data+using+BIC-seq2+and+its+application+to+detection+of+cancer+susceptibility+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://compbio.med.harvard.edu/BIC-seq/" target="_blank">http://compbio.med.harvard.edu/BIC-seq/</a></td></tr><tr><td>GISTIC2 v2.0.22</td><td>(<div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d4222834e3491" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Mermel et al., 2011</a><div data-db-target-of="bib91-d4222834e3491" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1131)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/gistic2" target="_blank">https://github.com/broadinstitute/gistic2</a></td></tr><tr><td>Strelka v2.9.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d4222834e3502" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Kim et al., 2018</a><div data-db-target-of="bib64-d4222834e3502" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li>et al.</li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (172)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/Illumina/strelka" target="_blank">https://github.com/Illumina/strelka</a></td></tr><tr><td>VarScan v2.3.8</td><td>(<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d4222834e3514" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Koboldt et al., 2012</a><div data-db-target-of="bib66-d4222834e3514" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2216)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://dkoboldt.github.io/varscan/" target="_blank">https://dkoboldt.github.io/varscan/</a></td></tr><tr><td>Pindel v0.2.5</td><td>(<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d4222834e3525" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Ye et al., 2009</a><div data-db-target-of="bib140-d4222834e3525" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1160)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/genome/pindel" target="_blank">https://github.com/genome/pindel</a></td></tr><tr><td>MuTect v1.1.7</td><td>(<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d4222834e3536" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Cibulskis et al., 2013</a><div data-db-target-of="bib26-d4222834e3536" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2160)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadinstitute/mutect" target="_blank">https://github.com/broadinstitute/mutect</a></td></tr><tr><td>somaticwrapper v1.3 and v1.5</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/somaticwrapper" target="_blank">https://github.com/ding-lab/somaticwrapper</a></td></tr><tr><td>Samtools v1.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib75" id="back-bib75" data-db-target-for="bib75-d4222834e3555" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib75">Li et al., 2009</a><div data-db-target-of="bib75-d4222834e3555" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Handsaker B. </li><li class="loa__item">Wysoker A. </li><li class="loa__item">Fennell T. </li><li class="loa__item">Ruan J. </li><li class="loa__item">Homer N. </li><li class="loa__item">Marth G. </li><li class="loa__item">Abecasis G. </li><li class="loa__item">Durbin R. </li><li class="loa__item">1000 Genome Project Data Processing Subgroup </li></ul><div class="reference-citations__detials">The sequence alignment/map format and SAMtools.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21968)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinforma.+Oxf.+Engl.&author=Li+H.&author=Handsaker+B.&author=Wysoker+A.&author=Fennell+T.&author=Ruan+J.&author=Homer+N.&author=Marth+G.&author=Abecasis+G.&author=Durbin+R.&author=1000+Genome+Project+Data+Processing+Subgroup+&title=The+sequence+alignment%2Fmap+format+and+SAMtools">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.htslib.org/" target="_blank">https://www.htslib.org/</a></td></tr><tr><td>GATK v4.0.0.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d4222834e3566" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">McKenna et al., 2010</a><div data-db-target-of="bib90-d4222834e3566" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li>et al.</li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10783)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/broadgsa/gatk" target="_blank">https://github.com/broadgsa/gatk</a></td></tr><tr><td>bam-readcount v0.8</td><td>McDonnell Genome Institute</td><td><a class="inter-ref-link" href="https://github.com/genome/bam-readcount" target="_blank">https://github.com/genome/bam-readcount</a></td></tr><tr><td>germlinewrapper v1.1</td><td>Li Ding Lab</td><td><a class="inter-ref-link" href="https://github.com/ding-lab/germlinewrapper" target="_blank">https://github.com/ding-lab/germlinewrapper</a></td></tr><tr><td>Manta v1.6.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d4222834e3593" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Chen et al., 2016</a><div data-db-target-of="bib24-d4222834e3593" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Schulz-Trieglaff O. </li><li class="loa__item">Shaw R. </li><li class="loa__item">Barnes B. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Cox A.J. </li><li class="loa__item">Kruglyak S. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (375)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/Illumina/manta" target="_blank">https://github.com/Illumina/manta</a></td></tr><tr><td>DELLY v0.8.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d4222834e3604" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Rausch et al., 2012</a><div data-db-target-of="bib110-d4222834e3604" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rausch T. </li><li class="loa__item">Zichner T. </li><li class="loa__item">Schlattl A. </li><li class="loa__item">Stutz A.M. </li><li class="loa__item">Benes V. </li><li class="loa__item">Korbel J.O. </li></ul><div class="reference-citations__detials">DELLY: structural variant discovery by integrated paired-end and split-read analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">i333-i339</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22962449&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84866440781&amp;cf=" onclick="newWindow(this.href);return false">Scopus (759)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i333-i339&journal=Bioinformatics&author=Rausch+T.&author=Zichner+T.&author=Schlattl+A.&author=Stutz+A.M.&author=Benes+V.&author=Korbel+J.O.&title=DELLY%3A+structural+variant+discovery+by+integrated+paired-end+and+split-read+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/dellytools/delly" target="_blank">https://github.com/dellytools/delly</a></td></tr><tr><td>Telseq v0.0.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d4222834e3615" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Ding et al., 2014</a><div data-db-target-of="bib33-d4222834e3615" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ding Z. </li><li class="loa__item">Mangino M. </li><li class="loa__item">Aviv A. </li><li class="loa__item">Spector T. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Estimating telomere length from whole genome sequence data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">e75</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku181&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24609383&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84899643560&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=e75&journal=Nucleic+Acids+Res.&author=Ding+Z.&author=Mangino+M.&author=Aviv+A.&author=Spector+T.&author=Durbin+R.&title=Estimating+telomere+length+from+whole+genome+sequence+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/zd1/telseq" target="_blank">https://github.com/zd1/telseq</a></td></tr><tr><td>HTSeq v0.11.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d4222834e3627" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Anders et al., 2015</a><div data-db-target-of="bib2-d4222834e3627" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anders S. </li><li class="loa__item">Pyl P.T. </li><li class="loa__item">Huber W. </li></ul><div class="reference-citations__detials">HTSeq—a Python framework to work with high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">166-169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtu638&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25260700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84928987900&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7209)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=166-169&journal=Bioinformatics&author=Anders+S.&author=Pyl+P.T.&author=Huber+W.&title=HTSeq%E2%80%94a+Python+framework+to+work+with+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/simon-anders/htseq" target="_blank">https://github.com/simon-anders/htseq</a></td></tr><tr><td>EricScript v0.5.5</td><td>(<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d4222834e3638" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benelli et al., 2012</a><div data-db-target-of="bib13-d4222834e3638" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benelli M. </li><li class="loa__item">Pescucci C. </li><li class="loa__item">Marseglia G. </li><li class="loa__item">Severgnini M. </li><li class="loa__item">Torricelli F. </li><li class="loa__item">Magi A. </li></ul><div class="reference-citations__detials">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sites.google.com/site/bioericscript/" target="_blank">https://sites.google.com/site/bioericscript/</a></td></tr><tr><td>INTEGRATE v0.2.6</td><td>(<div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d4222834e3649" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Zhang et al., 2016</a><div data-db-target-of="bib143-d4222834e3649" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang J. </li><li class="loa__item">White N.M. </li><li class="loa__item">Schmidt H.K. </li><li class="loa__item">Fulton R.S. </li><li class="loa__item">Tomlinson C. </li><li class="loa__item">Warren W.C. </li><li class="loa__item">Wilson R.K. </li><li class="loa__item">Maher C.A. </li></ul><div class="reference-citations__detials">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/integrate-fusion/" target="_blank">https://sourceforge.net/projects/integrate-fusion/</a></td></tr><tr><td>STAR-Fusion v1.5.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d4222834e3660" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Haas et al., 2019</a><div data-db-target-of="bib50-d4222834e3660" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haas B.J. </li><li class="loa__item">Dobin A. </li><li class="loa__item">Li B. </li><li class="loa__item">Stransky N. </li><li class="loa__item">Pochet N. </li><li class="loa__item">Regev A. </li></ul><div class="reference-citations__detials">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (51)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/STAR-Fusion/STAR-Fusion" target="_blank">https://github.com/STAR-Fusion/STAR-Fusion</a></td></tr><tr><td>BWA v0.7.17-r1188</td><td>(<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d4222834e3671" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Li and Durbin, 2009</a><div data-db-target-of="bib74-d4222834e3671" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Fast and accurate short read alignment with Burrows-Wheeler transform.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1754-1760</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp324&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19451168&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-67649884743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19451)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1754-1760&journal=Bioinforma.+Oxf.+Engl.&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td>CIRI v2.0.6</td><td>(<div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d4222834e3682" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Gao et al., 2015</a><div data-db-target-of="bib43-d4222834e3682" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhao F. </li></ul><div class="reference-citations__detials">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-014-0571-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25583365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84937578024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (409)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=4&journal=Genome+Biol.&author=Gao+Y.&author=Wang+J.&author=Zhao+F.&title=CIRI%3A+an+efficient+and+unbiased+algorithm+for+de+novo+circular+RNA+identification">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://sourceforge.net/projects/ciri/" target="_blank">https://sourceforge.net/projects/ciri/</a></td></tr><tr><td>RSEM v1.3.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d4222834e3693" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Li and Dewey, 2011</a><div data-db-target-of="bib73-d4222834e3693" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li B. </li><li class="loa__item">Dewey C.N. </li></ul><div class="reference-citations__detials">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6978)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://deweylab.github.io/RSEM/" target="_blank">https://deweylab.github.io/RSEM/</a></td></tr><tr><td>Bowtie2 v2.3.3</td><td>(<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d4222834e3704" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Langmead and Salzberg, 2012</a><div data-db-target-of="bib71-d4222834e3704" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Langmead B. </li><li class="loa__item">Salzberg S.L. </li></ul><div class="reference-citations__detials">Fast gapped-read alignment with Bowtie 2.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">357-359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.1923&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22388286&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84859210032&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17228)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=357-359&journal=Nat.+Methods&author=Langmead+B.&author=Salzberg+S.L.&title=Fast+gapped-read+alignment+with+Bowtie+2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td>R-rollup</td><td>(<div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d4222834e3715" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Polpitiya et al., 2008</a><div data-db-target-of="bib108-d4222834e3715" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Polpitiya A.D. </li><li class="loa__item">Qian W.-J. </li><li class="loa__item">Jaitly N. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Camp D.G. </li><li class="loa__item">Anderson G.A. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">DAnTE: a statistical tool for quantitative analysis of -omics data.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1556-1558</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtn217&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18453552&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-46249116675&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=1556-1558&journal=Bioinforma.+Oxf.+Engl.&author=Polpitiya+A.D.&author=Qian+W.-J.&author=Jaitly+N.&author=Petyuk+V.A.&author=Adkins+J.N.&author=Camp+D.G.&author=Anderson+G.A.&author=Smith+R.D.&title=DAnTE%3A+a+statistical+tool+for+quantitative+analysis+of+-omics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://omics.pnl.gov/software/danter" target="_blank">https://omics.pnl.gov/software/danter</a></td></tr><tr><td>MODMatcher</td><td>(<div class="dropBlock reference-citations"><a href="#bib141" id="back-bib141" data-db-target-for="bib141-d4222834e3726" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Yoo et al., 2014</a><div data-db-target-of="bib141-d4222834e3726" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoo S. </li><li class="loa__item">Huang T. </li><li class="loa__item">Campbell J.D. </li><li class="loa__item">Lee E. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Geraci M.W. </li><li class="loa__item">Powell C.A. </li><li class="loa__item">Schadt E.E. </li><li class="loa__item">Spira A. </li><li class="loa__item">Zhu J. </li></ul><div class="reference-citations__detials">MODMatcher: multi-omics data matcher for integrative genomic analysis.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/integrativenetworkbiology/Modmatcher" target="_blank">https://github.com/integrativenetworkbiology/Modmatcher</a></td></tr><tr><td>ConsensusClusterPlus v1.48.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d4222834e3737" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib137-d4222834e3737" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinforma.+Oxf.+Engl.&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/packages/ConsensusClusterPlus/" target="_blank">https://bioconductor.org/packages/ConsensusClusterPlus/</a></td></tr><tr><td>louvain-igraph v0.6.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d4222834e3749" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Blondel et al., 2008</a><div data-db-target-of="bib15-d4222834e3749" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blondel V.D. </li><li class="loa__item">Guillaume J.-L. </li><li class="loa__item">Lambiotte R. </li><li class="loa__item">Lefebvre E. </li></ul><div class="reference-citations__detials">Fast unfolding of communities in large networks.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Mech. Theor. Exp.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">2008</span></span>: <span class="ref__seriesPages">P10008</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1088%2F1742-5468%2F2008%2F10%2FP10008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-56349094785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7916)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2008&publication_year=2008&pages=P10008&journal=J.%C2%A0Stat.+Mech.+Theor.+Exp.&author=Blondel+V.D.&author=Guillaume+J.-L.&author=Lambiotte+R.&author=Lefebvre+E.&title=Fast+unfolding+of+communities+in+large+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://doi.org/10.5281/zenodo.1054103" target="_blank">https://doi.org/10.5281/zenodo.1054103</a></td></tr><tr><td>TCGAbiolinks v2.11.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d4222834e3760" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Colaprico et al., 2016</a><div data-db-target-of="bib28-d4222834e3760" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/TCGAbiolinks/" target="_blank">http://bioconductor.org/packages/TCGAbiolinks/</a></td></tr><tr><td>iProFun</td><td>(<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d4222834e3771" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Song et al., 2019</a><div data-db-target-of="bib117-d4222834e3771" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/songxiaoyu/iProFun" target="_blank">https://github.com/songxiaoyu/iProFun</a></td></tr><tr><td>BlackSheep</td><td>(<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d4222834e3782" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Blumenberg et al., 2019</a><div data-db-target-of="bib16-d4222834e3782" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blumenberg L. </li><li class="loa__item">Kawaler E. </li><li class="loa__item">Cornwell M. </li><li class="loa__item">Smith S. </li><li class="loa__item">Ruggles K. </li><li class="loa__item">Fenyö D. </li></ul><div class="reference-citations__detials">BlackSheep: a bioconductor and bioconda package for differential extreme value analysis (Bioinformatics).<div class="popup-ref"><span class="refDate">2019</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+a+bioconductor+and+bioconda+package+for+differential+extreme+value+analysis+%28Bioinformatics%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/ruggleslab/blackSheep" target="_blank">https://github.com/ruggleslab/blackSheep</a></td></tr><tr><td>xCell v1.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d4222834e3793" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Aran et al., 2017</a><div data-db-target-of="bib3-d4222834e3793" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://xcell.ucsf.edu/" target="_blank">http://xcell.ucsf.edu/</a></td></tr><tr><td>CIBERSORTx</td><td>(<div class="dropBlock reference-citations"><a href="#bib98" id="back-bib98" data-db-target-for="bib98-d4222834e3804" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Newman et al., 2019</a><div data-db-target-of="bib98-d4222834e3804" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Steen C.B. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Chaudhuri A.A. </li><li class="loa__item">Scherer F. </li><li class="loa__item">Khodadoust M.S. </li><li class="loa__item">Esfahani M.S. </li><li class="loa__item">Luca B.A. </li><li class="loa__item">Steiner D. </li><li>et al.</li></ul><div class="reference-citations__detials">Determining cell type abundance and expression from bulk tissues with digital cytometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">773-782</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41587-019-0114-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31061481&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85065314034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (187)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=773-782&journal=Nat.+Biotechnol.&author=Newman+A.M.&author=Steen+C.B.&author=Liu+C.L.&author=Gentles+A.J.&author=Chaudhuri+A.A.&author=Scherer+F.&author=Khodadoust+M.S.&author=Esfahani+M.S.&author=Luca+B.A.&author=Steiner+D.&title=Determining+cell+type+abundance+and+expression+from+bulk+tissues+with+digital+cytometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cibersortx.stanford.edu/" target="_blank">https://cibersortx.stanford.edu/</a></td></tr><tr><td>MoonlightR v1.12.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d4222834e3815" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Colaprico et al., 2020</a><div data-db-target-of="bib29-d4222834e3815" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://bioconductor.org/packages/MoonlightR" target="_blank">http://bioconductor.org/packages/MoonlightR</a></td></tr><tr><td>CausalPath v.7c5b934</td><td>(<div class="dropBlock reference-citations"><a href="#bib6" id="back-bib6" data-db-target-for="bib6-d4222834e3826" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib6">Babur et al., 2018</a><div data-db-target-of="bib6-d4222834e3826" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Babur Ö. </li><li class="loa__item">Luna A. </li><li class="loa__item">Korkut A. </li><li class="loa__item">Durupinar F. </li><li class="loa__item">Siper M.C. </li><li class="loa__item">Dogrusoz U. </li><li class="loa__item">Aslan J.E. </li><li class="loa__item">Sander C. </li><li class="loa__item">Demir E. </li></ul><div class="reference-citations__detials">Causal interactions from proteomic profiles: molecular data meets pathway knowledge.<div class="popup-ref"><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">258855</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=258855&journal=BioRxiv&author=Babur+%C3%96.&author=Luna+A.&author=Korkut+A.&author=Durupinar+F.&author=Siper+M.C.&author=Dogrusoz+U.&author=Aslan+J.E.&author=Sander+C.&author=Demir+E.&title=Causal+interactions+from+proteomic+profiles%3A+molecular+data+meets+pathway+knowledge">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://github.com/PathwayAndDataAnalysis/causalpath" target="_blank">https://github.com/PathwayAndDataAnalysis/causalpath</a></td></tr><tr><td>Seurat v3.1.2</td><td>(<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d4222834e3837" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Butler et al., 2018</a><div data-db-target-of="bib20-d4222834e3837" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Butler A. </li><li class="loa__item">Hoffman P. </li><li class="loa__item">Smibert P. </li><li class="loa__item">Papalexi E. </li><li class="loa__item">Satija Rahul </li></ul><div class="reference-citations__detials">Integrating single-cell transcriptomic data across different conditions, technologies, and species.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">411-420</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.4096&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29608179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85046298440&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1751)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=411-420&journal=Nat.+Biotechnol.&author=Butler+A.&author=Hoffman+P.&author=Smibert+P.&author=Papalexi+E.&author=Satija+Rahul&title=Integrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+species">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://cran.r-project.org/web/packages/Seurat" target="_blank">https://cran.r-project.org/web/packages/Seurat</a></td></tr><tr><td>edgeR v3.28.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d4222834e3848" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Robinson et al., 2010</a><div data-db-target-of="bib112-d4222834e3848" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson M.D. </li><li class="loa__item">McCarthy D.J. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">139-140</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp616&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19910308&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75249087100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13601)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=139-140&journal=Bioinformatics&author=Robinson+M.D.&author=McCarthy+D.J.&author=Smyth+G.K.&title=edgeR%3A+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.bioconductor.org/packages/edgeR/" target="_blank">https://www.bioconductor.org/packages/edgeR/</a></td></tr><tr><td>CLUE (data v1.1.1.2 and software v1.1.1.43)</td><td>(<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d4222834e3859" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d4222834e3859" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+next+generation+connectivity+map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://clue.io/" target="_blank">https://clue.io/</a></td></tr><tr><td>iLINCS v2.7.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d4222834e3871" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Pilarczyk et al., 2019</a><div data-db-target-of="bib105-d4222834e3871" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pilarczyk M. </li><li class="loa__item">Najafabadi M.F. </li><li class="loa__item">Kouril M. </li><li class="loa__item">Vasiliauskas J. </li><li class="loa__item">Niu W. </li><li class="loa__item">Shamsaei B. </li><li class="loa__item">Mahi N. </li><li class="loa__item">Zhang L. </li><li class="loa__item">Clark N. </li><li class="loa__item">Ren Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Connecting Omics Signatures of Diseases, Drugs, and Mechanisms of Actions with iLINCS.<div class="popup-ref"><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">826271</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=826271&journal=BioRxiv&author=Pilarczyk+M.&author=Najafabadi+M.F.&author=Kouril+M.&author=Vasiliauskas+J.&author=Niu+W.&author=Shamsaei+B.&author=Mahi+N.&author=Zhang+L.&author=Clark+N.&author=Ren+Y.&title=Connecting+Omics+Signatures+of+Diseases%2C+Drugs%2C+and+Mechanisms+of+Actions+with+iLINCS">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.ilincs.org/" target="_blank">https://www.ilincs.org/</a></td></tr><tr><td>R v3.6</td><td>R Development Core Team</td><td><a class="inter-ref-link" href="https://www.r-project.org/" target="_blank">https://www.R-project.org/</a></td></tr><tr><td>Bioconductor v3.9</td><td>(<div class="dropBlock reference-citations"><a href="#bib55" id="back-bib55" data-db-target-for="bib55-d4222834e3890" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib55">Huber et al., 2015</a><div data-db-target-of="bib55-d4222834e3890" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Huber W. </li><li class="loa__item">Carey V.J. </li><li class="loa__item">Gentleman R. </li><li class="loa__item">Anders S. </li><li class="loa__item">Carlson M. </li><li class="loa__item">Carvalho B.S. </li><li class="loa__item">Bravo H.C. </li><li class="loa__item">Davis S. </li><li class="loa__item">Gatto L. </li><li class="loa__item">Girke T. </li><li>et al.</li></ul><div class="reference-citations__detials">Orchestrating high-throughput genomic analysis with Bioconductor.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">115-121</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.3252&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25633503&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84961289551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1269)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=115-121&journal=Nat.+Methods&author=Huber+W.&author=Carey+V.J.&author=Gentleman+R.&author=Anders+S.&author=Carlson+M.&author=Carvalho+B.S.&author=Bravo+H.C.&author=Davis+S.&author=Gatto+L.&author=Girke+T.&title=Orchestrating+high-throughput+genomic+analysis+with+Bioconductor">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconductor.org/" target="_blank">https://bioconductor.org/</a></td></tr><tr><td>Tidyverse</td><td>(<div class="dropBlock reference-citations"><a href="#bib136" id="back-bib136" data-db-target-for="bib136-d4222834e3901" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib136">Wickham et al., 2019</a><div data-db-target-of="bib136-d4222834e3901" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wickham H. </li><li class="loa__item">Averick M. </li><li class="loa__item">Bryan J. </li><li class="loa__item">Chang W. </li><li class="loa__item">McGowan L. </li><li class="loa__item">François R. </li><li class="loa__item">Grolemund G. </li><li class="loa__item">Hayes A. </li><li class="loa__item">Henry L. </li><li class="loa__item">Hester J. </li><li>et al.</li></ul><div class="reference-citations__detials">Welcome to the tidyverse.<div class="popup-ref"><span class="ref__series"><i>J. Open Source Softw.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1686</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.21105%2Fjoss.01686&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=1686&journal=J.%C2%A0Open+Source+Softw.&author=Wickham+H.&author=Averick+M.&author=Bryan+J.&author=Chang+W.&author=McGowan+L.&author=Fran%C3%A7ois+R.&author=Grolemund+G.&author=Hayes+A.&author=Henry+L.&author=Hester+J.&title=Welcome+to+the+tidyverse">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.tidyverse.org/" target="_blank">https://www.tidyverse.org/</a></td></tr><tr><td>Python v3.7</td><td>Python Software Foundation</td><td><a class="inter-ref-link" href="https://www.python.org/" target="_blank">https://www.python.org/</a></td></tr><tr><td>Bioconda</td><td>(<div class="dropBlock reference-citations"><a href="#bib126" id="back-bib126" data-db-target-for="bib126-d4222834e3920" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib126">The Bioconda Team et al., 2018</a><div data-db-target-of="bib126-d4222834e3920" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">The Bioconda Team </li><li class="loa__item">Grüning B. </li><li class="loa__item">Dale R. </li><li class="loa__item">Sjödin A. </li><li class="loa__item">Chapman B.A. </li><li class="loa__item">Rowe J. </li><li class="loa__item">Tomkins-Tinch C.H. </li><li class="loa__item">Valieris R. </li><li class="loa__item">Köster J. </li></ul><div class="reference-citations__detials">Bioconda: sustainable and comprehensive software distribution for the life sciences.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">475-476</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41592-018-0046-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29967506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85049501214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (151)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=475-476&journal=Nat.+Methods&author=The+Bioconda+Team+&author=Gr%C3%BCning+B.&author=Dale+R.&author=Sj%C3%B6din+A.&author=Chapman+B.A.&author=Rowe+J.&author=Tomkins-Tinch+C.H.&author=Valieris+R.&author=K%C3%B6ster+J.&title=Bioconda%3A+sustainable+and+comprehensive+software+distribution+for+the+life+sciences">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://bioconda.github.io/" target="_blank">https://bioconda.github.io/</a></td></tr><tr><td>Snakemake v5.6</td><td>(<div class="dropBlock reference-citations"><a href="#bib67" id="back-bib67" data-db-target-for="bib67-d4222834e3931" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib67">Köster and Rahmann, 2012</a><div data-db-target-of="bib67-d4222834e3931" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Köster J. </li><li class="loa__item">Rahmann S. </li></ul><div class="reference-citations__detials">Snakemake—a scalable bioinformatics workflow engine.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">2520-2522</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts480&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22908215&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84867306721&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2520-2522&journal=Bioinformatics&author=K%C3%B6ster+J.&author=Rahmann+S.&title=Snakemake%E2%80%94a+scalable+bioinformatics+workflow+engine">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://snakemake.readthedocs.io/" target="_blank">https://snakemake.readthedocs.io/</a></td></tr><tr><td>ComplexHeatmap</td><td>(<div class="dropBlock reference-citations"><a href="#bib48" id="back-bib48" data-db-target-for="bib48-d4222834e3942" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib48">Gu et al., 2016</a><div data-db-target-of="bib48-d4222834e3942" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gu Z. </li><li class="loa__item">Eils R. </li><li class="loa__item">Schlesner M. </li></ul><div class="reference-citations__detials">Complex heatmaps reveal patterns and correlations in multidimensional genomic data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2847-2849</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtw313&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27207943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84992187637&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1009)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2847-2849&journal=Bioinformatics&author=Gu+Z.&author=Eils+R.&author=Schlesner+M.&title=Complex+heatmaps+reveal+patterns+and+correlations+in+multidimensional+genomic+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.bioconductor.org/packages/ComplexHeatmap/" target="_blank">https://www.bioconductor.org/packages/ComplexHeatmap/</a></td></tr><tr><td class="rowsep">scikit-learn v0.23.2</td><td class="rowsep">(<div class="dropBlock reference-citations"><a href="#bib102" id="back-bib102" data-db-target-for="bib102-d4222834e3953" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib102">Pedregosa et al., 2011</a><div data-db-target-of="bib102-d4222834e3953" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pedregosa F. </li><li class="loa__item">Varoquaux G. </li><li class="loa__item">Gramfort A. </li><li class="loa__item">Michel V. </li><li class="loa__item">Thirion B. </li><li class="loa__item">Grisel O. </li><li class="loa__item">Blondel M. </li><li class="loa__item">Prettenhofer P. </li><li class="loa__item">Weiss R. </li><li class="loa__item">Dubourg V. </li><li>et al.</li></ul><div class="reference-citations__detials">Scikit-learn: machine learning in Python.<div class="popup-ref"><span class="ref__series"><i>J. Mach. Learn. Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">2825-2830</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&journal=J.%C2%A0Mach.+Learn.+Res.&author=Pedregosa+F.&author=Varoquaux+G.&author=Gramfort+A.&author=Michel+V.&author=Thirion+B.&author=Grisel+O.&author=Blondel+M.&author=Prettenhofer+P.&author=Weiss+R.&author=Dubourg+V.&title=Scikit-learn%3A+machine+learning+in+Python">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td class="rowsep"><a class="inter-ref-link" href="https://scikit-learn.org/" target="_blank">https://scikit-learn.org/</a></td></tr><tr><td class="rowsep" colspan="3"><strong>Other</strong></td></tr><tr><td>RefSeq (downloaded from UCSC Genome Browser on 2018-06-29)</td><td>(<div class="dropBlock reference-citations"><a href="#bib100" id="back-bib100" data-db-target-for="bib100-d4222834e3967" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib100">O’Leary et al., 2016</a><div data-db-target-of="bib100-d4222834e3967" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">O’Leary N.A. </li><li class="loa__item">Wright M.W. </li><li class="loa__item">Brister J.R. </li><li class="loa__item">Ciufo S. </li><li class="loa__item">Haddad D. </li><li class="loa__item">McVeigh R. </li><li class="loa__item">Rajput B. </li><li class="loa__item">Robbertse B. </li><li class="loa__item">Smith-White B. </li><li class="loa__item">Ako-Adjei D. </li><li>et al.</li></ul><div class="reference-citations__detials">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D733-D745</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1189&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26553804&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D733-D745&journal=Nucleic+Acids+Res.&author=O%E2%80%99Leary+N.A.&author=Wright+M.W.&author=Brister+J.R.&author=Ciufo+S.&author=Haddad+D.&author=McVeigh+R.&author=Rajput+B.&author=Robbertse+B.&author=Smith-White+B.&author=Ako-Adjei+D.&title=Reference+sequence+%28RefSeq%29+database+at+NCBI%3A+current+status%2C+taxonomic+expansion%2C+and+functional+annotation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank">https://www.ncbi.nlm.nih.gov/refseq/</a>; <a class="inter-ref-link" href="https://genome.ucsc.edu/cgi-bin/hgTables" target="_blank">https://genome.ucsc.edu/cgi-bin/hgTables</a>; RRID:SCR_003496</td></tr><tr><td>GENCODE v22 (download from GDC Reference Files)</td><td>(<div class="dropBlock reference-citations"><a href="#bib42" id="back-bib42" data-db-target-for="bib42-d4222834e3983" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib42">Frankish et al., 2019</a><div data-db-target-of="bib42-d4222834e3983" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Frankish A. </li><li class="loa__item">Diekhans M. </li><li class="loa__item">Ferreira A.-M. </li><li class="loa__item">Johnson R. </li><li class="loa__item">Jungreis I. </li><li class="loa__item">Loveland J. </li><li class="loa__item">Mudge J.M. </li><li class="loa__item">Sisu C. </li><li class="loa__item">Wright J. </li><li class="loa__item">Armstrong J. </li><li>et al.</li></ul><div class="reference-citations__detials">GENCODE reference annotation for the human and mouse genomes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D766-D773</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgky955&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30357393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85059554160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D766-D773&journal=Nucleic+Acids+Res.&author=Frankish+A.&author=Diekhans+M.&author=Ferreira+A.-M.&author=Johnson+R.&author=Jungreis+I.&author=Loveland+J.&author=Mudge+J.M.&author=Sisu+C.&author=Wright+J.&author=Armstrong+J.&title=GENCODE+reference+annotation+for+the+human+and+mouse+genomes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.gencodegenes.org/" target="_blank">https://www.gencodegenes.org/</a>; <a class="inter-ref-link" href="https://gdc.cancer.gov/about-data/data-harmonization-and-generation/gdc-reference-files" target="_blank">https://gdc.cancer.gov/about-data/data-harmonization-and-generation/gdc-reference-files</a></td></tr><tr><td>gnomAD v2.1</td><td>(<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d4222834e3997" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Karczewski et al., 2020</a><div data-db-target-of="bib57-d4222834e3997" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karczewski K.J. </li><li class="loa__item">Francioli L.C. </li><li class="loa__item">Tiao G. </li><li class="loa__item">Cummings B.B. </li><li class="loa__item">Alföldi J. </li><li class="loa__item">Wang Q. </li><li class="loa__item">Collins R.L. </li><li class="loa__item">Laricchia K.M. </li><li class="loa__item">Ganna A. </li><li class="loa__item">Birnbaum D.P. </li><li>et al.</li></ul><div class="reference-citations__detials">The mutational constraint spectrum quantified from variation in 141,456 humans.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">581</span></span>: <span class="ref__seriesPages">434-443</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41586-020-2308-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=32461654&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85085542423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (613)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2020&pages=434-443&journal=Nature&author=Karczewski+K.J.&author=Francioli+L.C.&author=Tiao+G.&author=Cummings+B.B.&author=Alf%C3%B6ldi+J.&author=Wang+Q.&author=Collins+R.L.&author=Laricchia+K.M.&author=Ganna+A.&author=Birnbaum+D.P.&title=The+mutational+constraint+spectrum+quantified+from+variation+in+141%2C456+humans">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://gnomad.broadinstitute.org/" target="_blank">https://gnomad.broadinstitute.org/</a></td></tr><tr><td>The 1000 genomes project (final phase release on 2013-05-02)</td><td>(<div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d4222834e4008" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">The 1000 Genomes Project Consortium, 2015</a><div data-db-target-of="bib125-d4222834e4008" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">The 1000 Genomes Project Consortium </li></ul><div class="reference-citations__detials">A global reference for human genetic variation.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">526</span></span>: <span class="ref__seriesPages">68-74</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature15393&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26432245&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84943171338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5449)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=68-74&journal=Nature&author=The+1000+Genomes+Project+Consortium+&title=A+global+reference+for+human+genetic+variation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://www.internationalgenome.org/" target="_blank">https://www.internationalgenome.org/</a></td></tr><tr><td>OmniPath (downloaded on 2018-03-29)</td><td>(<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d4222834e4019" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Türei et al., 2016</a><div data-db-target-of="bib128-d4222834e4019" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Türei D. </li><li class="loa__item">Korcsmáros T. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">OmniPath: guidelines and gateway for literature-curated signaling pathway resources.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">966-967</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.4077&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27898060&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84999106855&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=966-967&journal=Nat.+Methods&author=T%C3%BCrei+D.&author=Korcsm%C3%A1ros+T.&author=Saez-Rodriguez+J.&title=OmniPath%3A+guidelines+and+gateway+for+literature-curated+signaling+pathway+resources">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://omnipathdb.org/" target="_blank">http://omnipathdb.org/</a></td></tr><tr><td>DEPOD (downloaded on 2018-03-29)</td><td>(<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d4222834e4030" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Duan et al., 2015</a><div data-db-target-of="bib36-d4222834e4030" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Duan G. </li><li class="loa__item">Li X. </li><li class="loa__item">Köhn M. </li></ul><div class="reference-citations__detials">The human DEPhOsphorylation database DEPOD: a 2015 update.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D531-D535</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku1009&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25332398&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84941037753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D531-D535&journal=Nucleic+Acids+Res.&author=Duan+G.&author=Li+X.&author=K%C3%B6hn+M.&title=The+human+DEPhOsphorylation+database+DEPOD%3A+a+2015+update">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://depod.bioss.uni-freiburg.de/" target="_blank">http://depod.bioss.uni-freiburg.de/</a></td></tr><tr><td>CORUM (downloaded on 2018-06-29)</td><td>(<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d4222834e4041" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Ruepp et al., 2010</a><div data-db-target-of="bib113-d4222834e4041" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruepp A. </li><li class="loa__item">Waegele B. </li><li class="loa__item">Lechner M. </li><li class="loa__item">Brauner B. </li><li class="loa__item">Dunger-Kaltenbach I. </li><li class="loa__item">Fobo G. </li><li class="loa__item">Frishman G. </li><li class="loa__item">Montrone C. </li><li class="loa__item">Mewes H.-W. </li></ul><div class="reference-citations__detials">CORUM: the comprehensive resource of mammalian protein complexes--2009.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">D497-D501</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkp914&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19884131&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75549091794&amp;cf=" onclick="newWindow(this.href);return false">Scopus (500)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D497-D501&journal=Nucleic+Acids+Res.&author=Ruepp+A.&author=Waegele+B.&author=Lechner+M.&author=Brauner+B.&author=Dunger-Kaltenbach+I.&author=Fobo+G.&author=Frishman+G.&author=Montrone+C.&author=Mewes+H.-W.&title=CORUM%3A+the+comprehensive+resource+of+mammalian+protein+complexes%2D%2D2009">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://mips.helmholtz-muenchen.de/corum/" target="_blank">https://mips.helmholtz-muenchen.de/corum/</a></td></tr><tr><td>SIGNOR v2.0 (downloaded on 2018-10-29)</td><td>(<div class="dropBlock reference-citations"><a href="#bib79" id="back-bib79" data-db-target-for="bib79-d4222834e4052" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib79">Licata et al., 2019</a><div data-db-target-of="bib79-d4222834e4052" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Licata L. </li><li class="loa__item">Lo Surdo P. </li><li class="loa__item">Iannuccelli M. </li><li class="loa__item">Palma A. </li><li class="loa__item">Micarelli E. </li><li class="loa__item">Perfetto L. </li><li class="loa__item">Peluso D. </li><li class="loa__item">Calderone A. </li><li class="loa__item">Castagnoli L. </li><li class="loa__item">Cesareni G. </li></ul><div class="reference-citations__detials">SIGNOR 2.0, the SIGnaling network open resource 2.0: 2019 update.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D504-D510</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D504-D510&journal=Nucleic+Acids+Res.&author=Licata+L.&author=Lo+Surdo+P.&author=Iannuccelli+M.&author=Palma+A.&author=Micarelli+E.&author=Perfetto+L.&author=Peluso+D.&author=Calderone+A.&author=Castagnoli+L.&author=Cesareni+G.&title=SIGNOR+2.0%2C+the+SIGnaling+network+open+resource+2.0%3A+2019+update">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://signor.uniroma2.it/" target="_blank">https://signor.uniroma2.it/</a></td></tr><tr><td>Reactome (downloaded on 2018-11-01)</td><td>(<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d4222834e4063" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Fabregat et al., 2018</a><div data-db-target-of="bib40-d4222834e4063" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fabregat A. </li><li class="loa__item">Jupe S. </li><li class="loa__item">Matthews L. </li><li class="loa__item">Sidiropoulos K. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Garapati P. </li><li class="loa__item">Haw R. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Korninger F. </li><li class="loa__item">May B. </li><li>et al.</li></ul><div class="reference-citations__detials">The reactome pathway Knowledgebase.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (926)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+reactome+pathway+Knowledgebase">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://reactome.org/" target="_blank">https://reactome.org/</a></td></tr><tr><td>NetworKIN 3.0</td><td>(<div class="dropBlock reference-citations"><a href="#bib54" id="back-bib54" data-db-target-for="bib54-d4222834e4074" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib54">Horn et al., 2014</a><div data-db-target-of="bib54-d4222834e4074" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Horn H. </li><li class="loa__item">Schoof E.M. </li><li class="loa__item">Kim J. </li><li class="loa__item">Robin X. </li><li class="loa__item">Miller M.L. </li><li class="loa__item">Diella F. </li><li class="loa__item">Palma A. </li><li class="loa__item">Cesareni G. </li><li class="loa__item">Jensen L.J. </li><li class="loa__item">Linding R. </li></ul><div class="reference-citations__detials">KinomeXplorer: an integrated platform for kinome biology studies.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">603-604</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.2968&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24874572&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84901788851&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=603-604&journal=Nat.+Methods&author=Horn+H.&author=Schoof+E.M.&author=Kim+J.&author=Robin+X.&author=Miller+M.L.&author=Diella+F.&author=Palma+A.&author=Cesareni+G.&author=Jensen+L.J.&author=Linding+R.&title=KinomeXplorer%3A+an+integrated+platform+for+kinome+biology+studies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="https://networkin.info/" target="_blank">https://networkin.info/</a></td></tr><tr><td>LINCS data portal (P100 Level 4)</td><td>(<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d4222834e4085" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Stathias et al., 2019</a><div data-db-target-of="bib118-d4222834e4085" class="dropBlock__holder centered"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stathias V. </li><li class="loa__item">Turner J. </li><li class="loa__item">Koleti A. </li><li class="loa__item">Vidovic D. </li><li class="loa__item">Cooper D. </li><li class="loa__item">Fazel-Najafabadi M. </li><li class="loa__item">Pilarczyk M. </li><li class="loa__item">Terryn R. </li><li class="loa__item">Chung C. </li><li class="loa__item">Umeano A. </li><li>et al.</li></ul><div class="reference-citations__detials">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D431-D439</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkz1023&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077665017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D431-D439&journal=Nucleic+Acids+Res.&author=Stathias+V.&author=Turner+J.&author=Koleti+A.&author=Vidovic+D.&author=Cooper+D.&author=Fazel-Najafabadi+M.&author=Pilarczyk+M.&author=Terryn+R.&author=Chung+C.&author=Umeano+A.&title=LINCS+Data+Portal+2.0%3A+next+generation+access+point+for+perturbation-response+signatures">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>)</td><td><a class="inter-ref-link" href="http://lincsportal.ccs.miami.edu/dcic-portal/" target="_blank">http://lincsportal.ccs.miami.edu/dcic-portal/</a></td></tr></tbody></table><div class="inline-table__tail"></div><ul class="rlist--inline separator download-links"><li><a title="Open Full Table" class="table__open__ctrl" target="_blank" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S1535-6108%2821%2900050-7">
                            Open table in a new tab
                        </a></li></ul></div></div></section><section id="sec5.2" data-locator="sec5.2" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Resource availability</span></h3><section id="sec5.2.1" data-locator="sec5.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lead contact</span></h4><div class="section-paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Li Ding (<a class="inter-ref-link" href="/cdn-cgi/l/email-protection#c0aca4a9aea780b7b5b3b4aceea5a4b5" target="_blank"><span class="__cf_email__" data-cfemail="6a060e03040d2a1d1f191e06440f0e1f">[email&#160;protected]</span></a>).</div></section><section id="sec5.2.2" data-locator="sec5.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Materials availability</span></h4><div class="section-paragraph">This study did not generate new unique reagents.</div></section><section id="sec5.2.3" data-locator="sec5.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Data and code availability</span></h4><div class="section-paragraph">Clinical data and raw proteomic data reported in this paper can be accessed via the CPTAC Data Portal at: <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/cptac/s/S048" target="_blank">https://cptac-data-portal.georgetown.edu/cptac/s/S048</a>. Genomic, transcriptomic, and snRNA-seq data files can be accessed via Genomic Data Commons (GDC) at: <a class="inter-ref-link" href="https://portal.gdc.cancer.gov/projects/CPTAC-3" target="_blank">https://portal.gdc.cancer.gov/projects/CPTAC-3</a>. Clinical and processed genomic data of our validation cohort from Children’s Brain Tumor Tissue Consortium (CBTTC) can be accessed via PedcBioPortal at: <a class="inter-ref-link" href="https://pedcbioportal.kidsfirstdrc.org/study/summary?id=phgg_cbttc" target="_blank">https://pedcbioportal.kidsfirstdrc.org/study/summary?id=phgg_cbttc</a> and via CAVATICA at: <a class="inter-ref-link" href="https://cavatica.sbgenomics.com/u/cavatica/pbta-cbttc/" target="_blank">https://cavatica.sbgenomics.com/u/cavatica/pbta-cbttc/</a>. Proteomic data files can be accessed via Proteomic Data Commons (PDC) at: <a class="inter-ref-link" href="https://pdc.cancer.gov/" target="_blank">https://pdc.cancer.gov/</a>. Processed data used in this publication can be found the CPTAC Data Portal at: <a class="inter-ref-link" href="https://cptac-data-portal.georgetown.edu/study-summary/S057" target="_blank">https://cptac-data-portal.georgetown.edu/study-summary/S057</a>, at <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/55251911-b73b-4f07-8f3d-466953b295e9/mmc3" class="scroll-into-link">Table S2</a>, the cptac Python package, and LinkedOmics (<div class="dropBlock reference-citations"><a href="#bib129" id="back-bib129" data-db-target-for="bib129-d4222834e4134" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib129">Vasaikar et al., 2018</a><div data-db-target-of="bib129-d4222834e4134" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S.V. </li><li class="loa__item">Straub P. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li></ul><div class="reference-citations__detials">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section></section><section id="sec5.3" data-locator="sec5.3" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Additional resources</span></h3><div class="section-paragraph">The CPTAC program website, which includes details about program initiatives, investigators, and datasets, can be accessed at: <a class="inter-ref-link" href="https://proteomics.cancer.gov/programs/cptac" target="_blank">https://proteomics.cancer.gov/programs/cptac</a>.</div></section><section id="sec5.4" data-locator="sec5.4" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Experimental model and subject details</span></h3><section id="sec5.4.1" data-locator="sec5.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Specimens and clinical data</span></h4><div class="section-paragraph">Tumor and germline blood samples from 99 qualified cases were collected from 10 tissue source sites in strict accordance with the CPTAC-3 protocol with informed consent from the patients. No adjacent tissue was collected as part of this study, however, 10 normal samples from the frontal cortex were used in the analysis from the GTEx project (<a class="inter-ref-link" href="https://gtexportal.org/" target="_blank">https://gtexportal.org/</a>). This study contained both males (n = 55) and females (n = 44) from 6 different countries. Histopathologically defined adult glioblastoma tumors were only considered for analysis, with an age range of 24-88. Clinical data were obtained from the tissue source sites and reviewed for correctness and completeness of data.</div></section><section id="sec5.4.2" data-locator="sec5.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample processing</span></h4><div class="section-paragraph">The CPTAC Biospecimen Core Resource (BCR) at the Pathology and Biorepository Core of the Van Andel Research Institute in Grand Rapids, Michigan manufactured and distributed biospecimen kits to the Tissue Source Sites (TSS) located in the US, Europe, and Asia. Each kit contains a set of pre-manufactured labels for unique tracking of every specimen respective to TSS location, disease, and sample type, used to track the specimens through the BCR to the CPTAC proteomic and genomic characterization centers.</div><div class="section-paragraph">Tissue specimens averaging 200 mg were snap-frozen by the TSS within a 30 min cold ischemic time (CIT) (CIT average = 13 min) and an adjacent segment was formalin-fixed paraffin-embedded (FFPE) and H&amp;E stained by the TSS for quality assessment to meet the CPTAC GBM requirements. Routinely, several tissue segments for each case were collected. Tissues were flash frozen in liquid nitrogen (LN<sub>2</sub>) then transferred to a liquid nitrogen freezer for storage until approval for shipment to the BCR.</div><div class="section-paragraph">Specimens were shipped using a cryoport that maintained an average temperature of under −140°C to the BCR with a time and temperature tracker to monitor the shipment. Receipt of specimens at the BCR included a physical inspection and review of the time and temperature tracker data for specimen integrity, followed by barcode entry into a biospecimen tracking database. Specimens were again placed in LN<sub>2</sub> storage until further processing. Acceptable GBM tumor tissue segments were determined by TSS pathologists based on the percent viable tumor nuclei (&gt;60%), total cellularity (&gt;50%), and necrosis (&lt;50%). Segments received at the BCR were verified by BCR and Leidos Biomedical Research (LBR) pathologists and the percent of total area of tumor in the segment was also documented. Additionally, disease-specific working group pathology experts reviewed the morphology to clarify or standardize specific disease classifications and correlation to the proteomic and genomic data.</div><div class="section-paragraph">Specimens selected for the discovery set were determined on the maximal percent in the pathology criteria and best weight. Specimens were pulled from the biorepository using an LN<sub>2</sub> cryocart to maintain specimen integrity and then cryopulverized. The cryopulverized specimen was divided into aliquots for DNA (30 mg) and RNA (30 mg) isolation and proteomics (50 mg) for molecular characterization. Nucleic acids were isolated and stored at −80°C until further processing and distribution; cryopulverized protein material was returned to the LN<sub>2</sub> freezer until distribution. Shipment of the cryopulverized segments used cryoports for distribution to the proteomic characterization centers and shipment of the nucleic acids used dry ice shippers for distribution to the genomic characterization centers; a shipment manifest accompanied all distributions for the receipt and integrity inspection of the specimens at the destination. The DNA sequencing was performed at the Broad Institute, Cambridge, MA and RNA sequencing was performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses was sent to the Proteomic Characterization Center (PCC) at Pacific Northwest National Laboratory (PNNL), Richland, Washington.</div></section><section id="sec5.4.3" data-locator="sec5.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Validation cohort sample collection and processing</span></h4><div class="section-paragraph">High grade glioma adolescent and young adults (AYA) cohort was used in validation studies were collected through Children’s Brain Tumor Tissue Consortium (CBTTC) sites including Children’s Hospital of Philadelphia (CHOP), Seattle Children's Hospital, Meyer Children's Hospital, UCSF Benioff Children's Hospital, University of Pittsburgh, Lurie Children's Hospital, Children's National Medical Center, and through the HUP-CHOP Neurosurgery Tumor Tissue Bank Collaborative at the Hospital of University of Pennsylvania. All samples were fresh frozen collected at the time of surgery, shipped and stored in BioRC (Biorepository Resource Center) at Children’s Hospital of Philadelphia. 30 mg tissue pieces were cut/chipped off using disposable scalpels on dry ice and delivered to Fred Hutchinson Cancer Research Center for sample preparation and proteomic analysis.</div></section></section><section id="sec5.5" data-locator="sec5.5" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Method details</span></h3><section id="sec5.5.1" data-locator="sec5.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample processing for genomic DNA and total RNA extraction</span></h4><div class="section-paragraph">Our study sampled a single site of the primary tumor from surgical resections, due to the internal requirement to process a minimum of 125 mg of tumor issue and 50 mg of adjacent normal tissue. DNA and RNA were extracted from tumor and blood normal specimens in a co-isolation protocol using Qiagen’s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3-5 mL) to serve as matched normal reference material. The Qubit™ dsDNA BR Assay Kit was used with the Qubit® 2.0 Fluorometer to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through various genomic assays was sent for genomic characterization. RNA quality was quantified using both the NanoDrop 8000 and quality assessed using Agilent Bioanalyzer. A sample that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity match for germline, normal adjacent tissue, and tumor tissue was assayed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and stratification.</div></section><section id="sec5.5.2" data-locator="sec5.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Whole exome sequencing</span></h4><section id="sec5.5.2.1" data-locator="sec5.5.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Library construction</span></h4><div class="section-paragraph">Library construction was performed as described in (<div class="dropBlock reference-citations"><a href="#bib41" id="back-bib41" data-db-target-for="bib41-d4222834e4200" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib41">Fisher et al., 2011</a><div data-db-target-of="bib41-d4222834e4200" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fisher S. </li><li class="loa__item">Barry A. </li><li class="loa__item">Abreu J. </li><li class="loa__item">Minie B. </li><li class="loa__item">Nolan J. </li><li class="loa__item">Delorey T.M. </li><li class="loa__item">Young G. </li><li class="loa__item">Fennell T.J. </li><li class="loa__item">Allen A. </li><li class="loa__item">Ambrogio L. </li><li>et al.</li></ul><div class="reference-citations__detials">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with the following modifications: initial genomic DNA input into shearing was reduced from 3 μg to 20-250 ng in 50 μL of solution. For adapter ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies, with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30 μL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted.</div></section><section id="sec5.5.2.2" data-locator="sec5.5.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">In-solution hybrid selection</span></h4><div class="section-paragraph">After library construction, libraries were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina's Nextera Exome Kit and following the manufacturer’s suggested protocol, with the following exceptions. First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina’s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system.</div></section><section id="sec5.5.2.3" data-locator="sec5.5.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of libraries for cluster amplification and sequencing</span></h4><div class="section-paragraph">After post-capture enrichment, library pools were quantified using qPCR (automated assay on the Agilent Bravo) using a kit purchased from KAPA Biosystems with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2 nM.</div></section><section id="sec5.5.2.4" data-locator="sec5.5.2.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster amplification and sequencing</span></h4><div class="section-paragraph">Cluster amplification of DNA libraries was performed according to the manufacturer’s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flowcells were sequenced utilizing sequencing-by-synthesis chemistry. The flowcells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76 cycle runs with two 8 cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq 4000 paired-end runs to achieve a minimum of 150x on target coverage per each sample library. The raw Illumina sequence data were demultiplexed and converted to fastq files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling.</div></section></section><section id="sec5.5.3" data-locator="sec5.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">PCR-free whole genome sequencing</span></h4><section id="sec5.5.3.1" data-locator="sec5.5.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preparation of libraries for cluster amplification and sequencing</span></h4><div class="section-paragraph">An aliquot of genomic DNA (350 ng in 50 μL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size selection was performed using a SPRI cleanup. Library preparation was performed using a commercially available kit provided by KAPA Biosystems (KAPA Hyper Prep without amplification module) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (purchased from IDT). Following sample preparation, libraries were quantified using quantitative PCR (kit purchased from KAPA Biosystems), with probes specific to the ends of the adapters. This assay was automated using Agilent’s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7 nM and pooled into 24-plexes.</div></section><section id="sec5.5.3.2" data-locator="sec5.5.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cluster amplification and sequencing (HiSeq X)</span></h4><div class="section-paragraph">Sample pools were combined with HiSeq X Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer’s protocol (Illumina) with the Illumina cBot. Flowcells were sequenced to a minimum of 15x on HiSeq X utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAMs containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated LIMS messaging.</div></section></section><section id="sec5.5.4" data-locator="sec5.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Illumina infinium methylationEPIC beadchip array</span></h4><div class="section-paragraph">The MethylationEPIC array uses an 8-sample version of the Illumina Beadchip capturing &gt; 850,000 DNA methylation sites per sample. 250 ng of DNA was used for the bisulfite conversation using Infinium MethylationEPIC BeadChip Kit. The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype calling pipeline. Data generated consisted of raw idats and a sample sheet.</div></section><section id="sec5.5.5" data-locator="sec5.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA sequencing</span></h4><section id="sec5.5.5.1" data-locator="sec5.5.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quality assurance and quality control of RNA analytes</span></h4><div class="section-paragraph">All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs &gt; 8.0 were considered high quality.</div></section><section id="sec5.5.5.2" data-locator="sec5.5.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA-seq library construction</span></h4><div class="section-paragraph">Total RNA-seq library construction was performed from the RNA samples using the TruSeq Stranded RNA Sample Preparation Kit and bar-coded with individual tags following the manufacturer’s instructions (Illumina, Inc. San Diego, CA). Libraries were prepared on an Agilent Bravo Automated Liquid Handling System. Quality control was performed at every step and the libraries were quantified using the TapeStation system.</div></section><section id="sec5.5.5.3" data-locator="sec5.5.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Total RNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were prepared and run on HiSeq 4000 paired end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. Typically, these were pools of four samples. The raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were quantified. Samples were then assessed for quality by mapping reads to the hg38 human genome reference, estimating the total number of reads that mapped, amount of RNA mapping to coding regions, amount of rRNA in sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository.</div></section><section id="sec5.5.5.4" data-locator="sec5.5.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA-seq library construction</span></h4><div class="section-paragraph">miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and bar-coded with individual tags following the manufacturer’s instructions. Libraries were prepared on Sciclone Liquid Handling Workstation Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex Kit specifications using a Pippin Prep system (Sage Science, Beverly, MA).</div></section><section id="sec5.5.5.5" data-locator="sec5.5.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA sequencing</span></h4><div class="section-paragraph">Indexed libraries were loaded on the Hiseq 4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (<div class="dropBlock reference-citations"><a href="#bib25" id="back-bib25" data-db-target-for="bib25-d4222834e4264" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib25">Chu et al., 2016</a><div data-db-target-of="bib25-d4222834e4264" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chu A. </li><li class="loa__item">Robertson G. </li><li class="loa__item">Brooks D. </li><li class="loa__item">Mungall A.J. </li><li class="loa__item">Birol I. </li><li class="loa__item">Coope R. </li><li class="loa__item">Ma Y. </li><li class="loa__item">Jones S. </li><li class="loa__item">Marra M.A. </li></ul><div class="reference-citations__detials">Large-scale profiling of microRNAs for the cancer genome atlas.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res.&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+the+cancer+genome+atlas">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Samples were assessed for the number of miRNAs called, species diversity, and total abundance. Samples passing quality control were uploaded to the GDC repository.</div></section><section id="sec5.5.5.6" data-locator="sec5.5.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Single-nuclei RNA library preparation and sequencing</span></h4><div class="section-paragraph">About 20-30 mg of cryopulverized powder from GBM specimens was resuspended in Lysis buffer (10 mM Tris-HCl (pH 7.4); 10 mM NaCl; 3 mM MgCl2; and 0.1% NP-40). This suspension was pipetted gently for 6-8 times, incubated on ice for 30 seconds, and pipetted again for 4-6 times. The lysate containing free nuclei was filtered through a 40 μm cell strainer. We washed the filter with 1 mL Wash and Resuspension buffer (1X PBS + 2% BSA + 0.2 U/μL RNase inhibitor) and combined the flow through with the original filtrate. After a 6-minute centrifugation at 500 x g and 4°C, the nuclei pellet was resuspended in 500 μL of Wash and Resuspension buffer. After staining by DRAQ5, the nuclei were further purified by Fluorescence Activated Cell Sorting (FACS). FACS-purified nuclei were centrifuged again and resuspended in a small volume (about 30 μL). After counting and microscopic inspection of nuclei quality, the nuclei preparation was diluted to about 1,000 nuclei/μL. About 20,000 nuclei were used for single-nuclei RNA sequencing (snRNA-seq) by the 10X Chromium platform. We loaded the single nuclei onto a Chromium Chip B Single Cell Kit, 48 rxns (10x Genomics, PN-1000073) and processed them through the Chromium Controller to generate GEMs (Gel Beads in Emulsion). We then prepared the sequencing libraries with the Chromium Single Cell 3' GEM, Library &amp; Gel Bead Kit v3, 16 rxns (10x Genomics, PN-1000075) following the manufacturer’s protocol. Sequencing was performed on an Illumina NovaSeq 6000 S4 flow cell. The libraries were pooled and sequenced using the XP workflow according to the manufacturer's protocol with a 28x8x98bp sequencing recipe. The resulting sequencing files were available as FASTQs per sample after demultiplexing.</div></section></section><section id="sec5.5.6" data-locator="sec5.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">MS sample processing and data collection</span></h4><section id="sec5.5.6.1" data-locator="sec5.5.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein extraction and Lys-C/Trypsin tandem digestion</span></h4><div class="section-paragraph">Approximately 50 mg of each of the cryopulverized tumor and normal tissues were homogenized separately in 200 μL of lysis buffer (8 M urea, 75 mM NaCl, 50 mM Tris, pH 8.0, 1 mM EDTA, 2 μg/mL aprotinin, 10 μg/mL leupeptin, 1 mM PMSF, 10 mM NaF, 1:100 v/v Sigma phosphatase inhibitor cocktail 2, 1:100 v/v Sigma phosphatase inhibitor cocktail 3, 20 μM PUGNAc, and 5 mM sodium butyrate). Lysates were precleared by centrifugation at 20,000 x g for 10 min at 4°C and protein concentrations were determined by BCA assay (ThermoFisher Scientific) and adjusted to 8 μg/μL with lysis buffer. Proteins were reduced with 5 mM dithiothreitol for 1 h at 37°C and subsequently alkylated with 10 mM iodoacetamide for 45 min at 25°C in the dark. Samples were diluted 1:3 with 50 mM Tris, pH 8.0 and digested with Lys-C (Wako) at 1:50 enzyme-to-substrate ratio. After 2 h of digestion at 25°C, an aliquot of the same amount of sequencing-grade modified trypsin (Promega, V5117) was added to the samples and further incubated at 25°C for 14 h. The digested samples were then acidified with 100% formic acid to 1% of the final concentration of formic acid and centrifuged for 15 min at 1,500 x g at 4°C before transferring samples into new tubes leaving the resulting pellet behind. After 3 fold dilution with 0.1% formic acid, tryptic peptides were desalted on C18 SPE (Waters tC18 SepPak, WAT054925) and dried using Speed-Vac.</div></section><section id="sec5.5.6.2" data-locator="sec5.5.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TMT-11 labeling of peptides</span></h4><div class="section-paragraph">Desalted peptides from each sample were labeled with 11-plex TMT reagents (ThermoFisher Scientific). Peptides (400 μg) from each of the samples were dissolved in 80 μL of 50 mM HEPES, pH 8.5 solution, and mixed with 400 μg of TMT reagent that was dissolved freshly in 20 μL of anhydrous acetonitrile according to the optimized TMT labeling protocol described previously (<div class="dropBlock reference-citations"><a href="#bib142" id="back-bib142" data-db-target-for="bib142-d4222834e4285" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib142">Zecha et al., 2019</a><div data-db-target-of="bib142-d4222834e4285" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zecha J. </li><li class="loa__item">Satpathy S. </li><li class="loa__item">Kanashova T. </li><li class="loa__item">Avanessian S.C. </li><li class="loa__item">Kane M.H. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Kuster B. </li></ul><div class="reference-citations__detials">TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell.+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+a+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Channel 126 was used for labeling the internal reference sample (pooled from all tumor and normal samples) throughout the sample analysis. After 1 h incubation at RT, 60 μL 50 mM HEPES pH8.5, 20% ACN solution was added to dilute the samples, and 12 μL of 5% hydroxylamine was added and incubated for 15 min at RT to quench the labeling reaction. Peptides labeled by different TMT reagents were then mixed, dried using Speed-Vac, reconstituted with 3% acetonitrile, 0.1% formic acid and desalted on tC18 SepPak SPE columns.</div></section><section id="sec5.5.6.3" data-locator="sec5.5.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Peptide fractionation by basic reversed-phase liquid chromatography (bRPLC)</span></h4><div class="section-paragraph">Approximately 3.5 mg of 11-plex TMT labeled sample was separated on a reversed phase Agilent Zorbax 300 Extend-C18 column (250 mm × 4.6 mm column containing 3.5-μm particles) using the Agilent 1200 HPLC System. Solvent A was 4.5 mM ammonium formate, pH 10, 2% acetonitrile and solvent B was 4.5 mM ammonium formate, pH 10, 90% acetonitrile. The flow rate was 1 mL/min and the injection volume was 900 μL. The LC gradient started with a linear increase of solvent B to 16% in 6 min, then linearly increased to 40% B in 60 min, 4 min to 44% B, 5 min to 60% B and another 14 of 60% solvent B. A total of 96 fractions were collected into a 96 well plate throughout the LC gradient. These fractions were concatenated into 24 fractions by combining 4 fractions that are 24 fractions apart (i.e., combining fractions #1, #25, #49, and #73; #2, #26, #50, and #74; and so on). For proteome analysis, 5% of each concatenated fraction was dried down and re-suspended in 2% acetonitrile, 0.1% formic acid to a peptide concentration of 0.1 mg/mL for LC-MS/MS analysis. The rest of the fractions (95%) were further concatenated into 12 fractions (i.e., by combining fractions #1 and #13; #3 and #15; and so on), dried down, and subjected to immobilized metal affinity chromatography (IMAC) for phosphopeptide enrichment.</div></section><section id="sec5.5.6.4" data-locator="sec5.5.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphopeptide enrichment using IMAC</span></h4><div class="section-paragraph">Fe<sup>3+</sup>-NTA-agarose beads were freshly prepared using the Ni-NTA Superflow agarose beads (QIAGEN, #30410) for phosphopeptide enrichment. For each of the 12 fractions, peptides were reconstituted in 500 μL IMAC binding/wash buffer (80% acetonitrile, 0.1% trifluoroacetic acid) and incubated with 20 μL of the 50% bead suspension for 30 min at RT. After incubation, the beads were sequentially washed with 50 μL of wash buffer (1X), 50 μL of 50% acetonitrile, 0.1% trifluoroacetic acid (1X), 50 μL of wash buffer (1X), and 50 μL of 1% formic acid (1X) on the stage tip packed with 2 discs of Empore C18 material (Empore Octadecyl C18, 47 mm; Supleco, 66883-U). Phosphopeptides were eluted from the beads on C18 using 70 μL of elution buffer (500 mM K<sub>2</sub>HPO<sub>4</sub>, pH 7.0). Sixty microliter of 50% acetonitrile, 0.1% formic acid was used for elution of phosphopeptides from the C18 stage tips after two washes with 100 μL of 1% formic acid. Samples were dried using Speed-Vac and later reconstituted with 12 μL of 3% acetonitrile, 0.1% formic acid for LC-MS/MS analysis.</div></section><section id="sec5.5.6.5" data-locator="sec5.5.6.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immunoaffinity purification of acetylated peptides</span></h4><div class="section-paragraph">Tryptic peptides from the flow-through of IMAC were combined into four samples follow concatenation scheme by combining 3 fractions that were 4 fractions apart (i.e., combining fractions #1, #5 and #9 as a new fraction) and dried down using Speed-Vac. The dried peptides were reconstituted in 1.4 mL of the immunoaffinity purification (IAP) buffer (50 mM MOPS/NaOH pH 7.2, 10 mM Na<sub>2</sub>HPO<sub>4</sub> and 50 mM NaCl). After dissolving the peptide, the pH of the peptide solution was checked using pH indicator paper. The antibody beads from PTMScan® Acetyl-Lysine Motif [Ac-K] Kit (Cell Signaling, #13416) were freshly prepared. Briefly, the antibody beads were centrifuged at 2,000 x g for 30 sec and all buffers from the beads were removed; the antibody beads were then washed with 1 mL of IAP buffer for four times and finally resuspend in 40 μL of IAP buffer. For each fraction, half of the antibody in each tube was transferred to the peptide solution and incubated on a rotator overnight at 4°C. After removing the supernatant, the reacted beads were washed with 1 mL of PBS buffer five times. For the elution of acetylated peptides, the antibody beads were incubated 2 times each with 50 μL of 0.15% TFA at room temperature for 10 min. The eluted peptides were transferred to the stage tip packed with two discs of Empore C18 material. The C18 stage tips were washed by 1% formic acid and 50% acetonitrile, and 0.1% formic acid was used for elution of peptides from the C18 stage tips. The eluted peptides were dried using Speed-Vac, and reconstituted with 13 μL of 2% acetonitrile, 0.1% formic acid contained 0.01% DDM (n-Dodecyl β-D-maltoside) right before the LC-MS/MS analysis.</div><div class="section-paragraph">The acetylated peptides prepared by IP from the IMAC flow-through may very well miss those peptides that are both phosphorylated and acetylated. Splitting the samples for independent IP and IMAC may improve the chance of recovering such peptides, assuming having both PTMs on the same peptide does not impact the affinity of either the IP or IMAC process. However, acetylated peptides are estimated to be 10 times lower in abundance than the phosphopeptides, hence much larger input may be needed to recover the dual-modified peptides. Given the extremely low stoichiometry of these dual-modified peptides and the sample size limitations, it was not pursued in this work.</div></section><section id="sec5.5.6.6" data-locator="sec5.5.6.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS/MS analysis</span></h4><div class="section-paragraph">Fractionated samples prepared for global proteome, phosphoproteome, and acetylome analysis were separated using a nanoACQUITY UPLC system (Waters) by reversed-phase HPLC. The analytical column was manufactured in-house using ReproSil-Pur 120 C18-AQ 1.9 μm stationary phase (Dr. Maisch GmbH) and slurry packed into a 25-cm length of 360 μm o.d. x 75 μm i.d. fused silica picofrit capillary tubing (New Objective). The analytical column was heated to 50°C using an AgileSLEEVE column heater (Analytical Sales and Services). The analytical column was equilibrated to 98% Mobile Phase A (MP A, 0.1% formic acid/3% acetonitrile) and 2% Mobile Phase B (MP B, 0.1% formic acid/90% acetonitrile) and maintained at a constant column flow of 200 nL/min. The sample was injected into a 5-μL loop placed in-line with the analytical column which initiated the gradient profile (min:%MP B): 0:2, 1:6, 85:30, 94:60, 95:90, 100:90, 101:50, 110:50 (for global proteome and phosphoproteome analysis); 0:2, 1:6, 235:30, 244:60, 245:90, 250:90, 251:50, 260:50 (for acetylome analysis). The column was allowed to equilibrate at start conditions for 30 minutes between analytical runs.</div><div class="section-paragraph">MS analysis was performed using an Orbitrap Fusion Lumos mass spectrometer (ThermoFisher Scientific). The global proteome and phosphoproteome samples were analyzed under identical conditions. Electrospray voltage (1.8 kV) was applied at a carbon composite union (Valco Instruments) coupling a 360 μm o.d. x 20 μm i.d. fused silica extension from the LC gradient pump to the analytical column and the ion transfer tube was set at 250°C. Following a 25-min delay from the time of sample injection, Orbitrap precursor spectra (AGC 4 × 10<sup>5</sup>) were collected from 350-1800 m/z for 110 min at a resolution of 60K along with data dependent Orbitrap HCD MS/MS spectra (centroid) at a resolution of 50K (AGC 1 × 10<sup>5</sup>) and max ion time of 105 ms for a total duty cycle of 2 seconds. Masses selected for MS/MS were isolated (quadrupole) at a width of 0.7 m/z and fragmented using a collision energy of 30%. Peptide mode was selected for monoisotopic precursor scan and charge state screening was enabled to reject unassigned 1+, 7+, 8+, and &gt; 8+ ions with a dynamic exclusion time of 45 seconds to discriminate against previously analyzed ions between ±10 ppm. The acetylome samples were analyzed under similar conditions except that the max ion time was 125 ms.</div></section><section id="sec5.5.6.7" data-locator="sec5.5.6.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Construction and utilization of the comparative reference samples</span></h4><div class="section-paragraph">As a quality control measure, two different types of “Comparative Reference” (“CompRef”) patient-derived xenograft (PDX) samples were generated as previously described (<div class="dropBlock reference-citations"><a href="#bib76" id="back-bib76" data-db-target-for="bib76-d4222834e4338" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib76">Li et al., 2013</a><div data-db-target-of="bib76-d4222834e4338" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li S. </li><li class="loa__item">Shen D. </li><li class="loa__item">Shao J. </li><li class="loa__item">Crowder R. </li><li class="loa__item">Liu W. </li><li class="loa__item">Prat A. </li><li class="loa__item">He X. </li><li class="loa__item">Liu S. </li><li class="loa__item">Hoog J. </li><li class="loa__item">Lu C. </li><li>et al.</li></ul><div class="reference-citations__detials">Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1116-1130</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24055055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84884559238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (376)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1116-1130&journal=Cell+Rep&author=Li+S.&author=Shen+D.&author=Shao+J.&author=Crowder+R.&author=Liu+W.&author=Prat+A.&author=He+X.&author=Liu+S.&author=Hoog+J.&author=Lu+C.&title=Endocrine-therapy-resistant+ESR1+variants+revealed+by+genomic+characterization+of+breast-cancer-derived+xenografts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib124" id="back-bib124" data-db-target-for="bib124-d4222834e4341" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib124">Tabb et al., 2016</a><div data-db-target-of="bib124-d4222834e4341" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Tabb D.L. </li><li class="loa__item">Wang X. </li><li class="loa__item">Carr S.A. </li><li class="loa__item">Clauser K.R. </li><li class="loa__item">Mertins P. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Holman J.D. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhang B. </li><li class="loa__item">Zimmerman L.J. </li><li>et al.</li></ul><div class="reference-citations__detials">Reproducibility of differential proteomic technologies in CPTAC fractionated xenografts.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">691-706</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Facs.jproteome.5b00859&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26653538&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84960343160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=691-706&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Wang+X.&author=Carr+S.A.&author=Clauser+K.R.&author=Mertins+P.&author=Chambers+M.C.&author=Holman+J.D.&author=Wang+J.&author=Zhang+B.&author=Zimmerman+L.J.&title=Reproducibility+of+differential+proteomic+technologies+in+CPTAC+fractionated+xenografts">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and used to monitor the longitudinal performance of the proteomics workflow throughout the course of this study. Briefly, the PDX tumors from established basal and luminal breast cancer intrinsic subtypes were raised subcutaneously in 8-week old NOD.Cg-Prkdc<sup>scid</sup> Il2rg<sup>tm1Wjl</sup>/SzJ mice (Jackson Laboratories, Bar Harbor, ME) using procedures reviewed and approved by the Institutional Animal Care and Use Committee at Washington University in St. Louis. Xenografts were grown in multiple mice, pooled, and cryopulverized to provide a sufficient amount of uniform material for the duration of the study. Full proteome, phosphoproteome and acetylome process replicates of each of the two types of CompRef samples were prepared and analyzed as standalone 11-plex TMT experiments alongside every 4 TMT-11 experiments of the study samples, using the same analysis protocol as the patient samples. These interstitially analyzed CompRef samples were evaluated for depth of proteome, phosphoproteome, and acetylome coverage and for consistency in quantitative comparison between the basal and luminal models.</div></section><section id="sec5.5.6.8" data-locator="sec5.5.6.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Global proteome and phosphoproteome analysis of high grade brain tumor samples</span></h4><div class="section-paragraph">An independent cohort of 39 high grade (age 14-39 years old) brain tumors obtained from the Children’s Brain Tumor Tissue Consortium (CBTTC) were analyzed using the same procedures for TMT-based quantitative global proteome and phosphoproteome analysis of the adult GBM and normal brain tissue samples, with modifications in front-end protein extraction and digestion described as follows: Approximately 50 mg of each of brain tumor tissues were cryopulverized and lysed separately in 800 μL of lysis buffer (6 M urea, 25 mM Tris, pH 8.0, 1 mM EDTA, 1 mM EGTA, 1:100 v/v Sigma protease inhibitor, 1:100 v/v Sigma phosphatase inhibitor cocktail 2, and 1:100 v/v Sigma phosphatase inhibitor cocktail 3). Lysates were precleared by centrifugation at 20,000 x g for 10 min at 4°C. After adjusting the protein concentration to approximately 1.5 μg/μL, proteins were reduced with 5 mM dithiothreitol for 1 h at 37°C, and subsequently alkylated with 10 mM iodoacetamide for 45 min at 25°C in the dark. Samples were diluted to 2 M urea concentration with 25 mM Tris, pH 8.0 and digested with Lys-C at 1:50 enzyme-to-substrate ratio. After 2 h of digestion at 25°C, aliquot of sequencing grade modified trypsin at 1:25 enzyme-to-substrate ratio was added to the samples and further incubated at 25°C for 14 h.</div></section></section><section id="sec5.5.7" data-locator="sec5.5.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Polar metabolites and lipid mass spectrometry</span></h4><section id="sec5.5.7.1" data-locator="sec5.5.7.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Metabolite and lipid extraction</span></h4><div class="section-paragraph">Lipids and metabolite extracts were generated from the same pulverized tissue with a minimum of 30 mg using a modified Folch extraction (<div class="dropBlock reference-citations"><a href="#bib95" id="back-bib95" data-db-target-for="bib95-d4222834e4363" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib95">Nakayasu et al., 2016</a><div data-db-target-of="bib95-d4222834e4363" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Nakayasu E.S. </li><li class="loa__item">Nicora C.D. </li><li class="loa__item">Sims A.C. </li><li class="loa__item">Burnum-Johnson K.E. </li><li class="loa__item">Kim Y.-M. </li><li class="loa__item">Kyle J.E. </li><li class="loa__item">Matzke M.M. </li><li class="loa__item">Shukla A.K. </li><li class="loa__item">Chu R.K. </li><li class="loa__item">Schepmoes A.A. </li><li>et al.</li></ul><div class="reference-citations__detials">MPLEx: a robust and universal protocol for single-sample integrative proteomic, metabolomic, and lipidomic analyses.<div class="popup-ref"><span class="ref__series"><i>MSystems.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">e00043-16</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1128%2FmSystems.00043-16&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27822525&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85041917042&amp;cf=" onclick="newWindow(this.href);return false">Scopus (62)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=e00043-16&journal=MSystems&author=Nakayasu+E.S.&author=Nicora+C.D.&author=Sims+A.C.&author=Burnum-Johnson+K.E.&author=Kim+Y.-M.&author=Kyle+J.E.&author=Matzke+M.M.&author=Shukla+A.K.&author=Chu+R.K.&author=Schepmoes+A.A.&title=MPLEx%3A+a+robust+and+universal+protocol+for+single-sample+integrative+proteomic%2C+metabolomic%2C+and+lipidomic+analyses">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Additional solvent was added such that the final volume was proportionate to the mass of the sample ensuring the solvent ratio is 3:8:4 H<sub>2</sub>O:CHCl<sub>3</sub>:MeOH. Sample were vortexed for 30 sec, chilled in an ice block for 5 min, and vortexed again for 30 sec. The samples were then centrifuged at 10,000 x g for 10 min at 4°C. The polar metabolite extract was transferred into a glass vial, dried in a speedvac, and stored at -20°C until chemical derivatization for gas chromatography mass spectrometry (GC-MS) analysis. The total lipid extract (TLE) was transferred into a glass vial, dried in a speedvac, and then reconstituted in 500 μL 1:1 chloroform/methanol for storage at -20°C until analysis.</div></section><section id="sec5.5.7.2" data-locator="sec5.5.7.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Chemical derivatization of polar metabolites</span></h4><div class="section-paragraph">Polar metabolites along with 50% of the TLE were chemically derivatized prior to metabolomics analysis. Chemical derivatization of metabolites was previously detailed (<div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d4222834e4377" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Webb-Robertson et al., 2014</a><div data-db-target-of="bib135-d4222834e4377" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Webb-Robertson B.-J. </li><li class="loa__item">Kim Y.-M. </li><li class="loa__item">Zink E.M. </li><li class="loa__item">Hallaian K.A. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Madupu R. </li><li class="loa__item">Waters K.M. </li><li class="loa__item">Metz T.O. </li></ul><div class="reference-citations__detials">A statistical analysis of the effects of urease pre-treatment on the measurement of the urinary metabolome by gas chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Metabolomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">897-908</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11306-014-0642-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25254001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84894268438&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=897-908&journal=Metabolomics&author=Webb-Robertson+B.-J.&author=Kim+Y.-M.&author=Zink+E.M.&author=Hallaian+K.A.&author=Zhang+Q.&author=Madupu+R.&author=Waters+K.M.&author=Metz+T.O.&title=A+statistical+analysis+of+the+effects+of+urease+pre-treatment+on+the+measurement+of+the+urinary+metabolome+by+gas+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To protect carbonyl groups and reduce the number of tautomeric isomers, 20 μL of methoxyamine in pyridine (30 mg/mL) was added to each sample, followed by vortexing for 30 seconds and incubation at 37°C with generous shaking for 90 minutes. To derivatize hydroxyl and amine groups to trimethylsilylated (TMS) forms, 80 μL of N-methyl-N-(trimethylsilyl)trifluoroacetamide (MSTFA) with 1% trimethylchlorosilane (TMCS) was added to each vial, followed by vortexing for 10 seconds and incubation at 37°C with shaking for 30 minutes. The samples were allowed to cool to room temperature and were analysed on the GC-MS the same day.</div></section><section id="sec5.5.7.3" data-locator="sec5.5.7.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">GC-MS analysis</span></h4><div class="section-paragraph">An Agilent GC 7890A coupled with a single quadrupole MSD 5975C was used to analyze chemically derivatized metabolites. GC-MS analysis was previously detailed (<div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d4222834e4385" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Webb-Robertson et al., 2014</a><div data-db-target-of="bib135-d4222834e4385" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Webb-Robertson B.-J. </li><li class="loa__item">Kim Y.-M. </li><li class="loa__item">Zink E.M. </li><li class="loa__item">Hallaian K.A. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Madupu R. </li><li class="loa__item">Waters K.M. </li><li class="loa__item">Metz T.O. </li></ul><div class="reference-citations__detials">A statistical analysis of the effects of urease pre-treatment on the measurement of the urinary metabolome by gas chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Metabolomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">897-908</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11306-014-0642-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25254001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84894268438&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=897-908&journal=Metabolomics&author=Webb-Robertson+B.-J.&author=Kim+Y.-M.&author=Zink+E.M.&author=Hallaian+K.A.&author=Zhang+Q.&author=Madupu+R.&author=Waters+K.M.&author=Metz+T.O.&title=A+statistical+analysis+of+the+effects+of+urease+pre-treatment+on+the+measurement+of+the+urinary+metabolome+by+gas+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, 1 μL of each sample was injected onto a HP-5MS column (30 m × 0.25 mm × 0.25 μm; Agilent Technologies, Inc). The injection port temperature was held at 250°C throughout the analysis. The GC oven was held at 60°C for 1 minute after injection then increased to 325°C by 10°C/min, followed by a 5-minute hold at 325°C. Total analysis time was 34 minutes per injection. The helium gas flow rates were determined by the Agilent Retention Time Locking function based on analysis of deuterated myristic acid. Data were collected over the mass range 50 - 550 m/z. A mixture of fatty acid methyl esters (C8-C28) was analyzed once per day at the beginning of each batch together with the samples for retention index alignment purposes during subsequent data analysis.</div></section><section id="sec5.5.7.4" data-locator="sec5.5.7.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">LC-MS analysis</span></h4><div class="section-paragraph">Stored plasma TLEs were dried <em>in vacuo</em> (45 min) and reconstituted in 5 μL chloroform plus 95 μL of methanol. The TLEs were analyzed as outlined in the previous study (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d4222834e4395" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Kyle et al., 2017</a><div data-db-target-of="bib70-d4222834e4395" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kyle J.E. </li><li class="loa__item">Crowell K.L. </li><li class="loa__item">Casey C.P. </li><li class="loa__item">Fujimoto G.M. </li><li class="loa__item">Kim S. </li><li class="loa__item">Dautel S.E. </li><li class="loa__item">Smith R.D. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Metz T.O. </li></ul><div class="reference-citations__detials">LIQUID: an-open source software for identifying lipids in LC-MS/MS-based lipidomics data.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1744-1746</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtx046&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28158427&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85021049695&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=1744-1746&journal=Bioinforma.+Oxf.+Engl.&author=Kyle+J.E.&author=Crowell+K.L.&author=Casey+C.P.&author=Fujimoto+G.M.&author=Kim+S.&author=Dautel+S.E.&author=Smith+R.D.&author=Payne+S.H.&author=Metz+T.O.&title=LIQUID%3A+an-open+source+software+for+identifying+lipids+in+LC-MS%2FMS-based+lipidomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). A Waters Acquity UPLC H class system interfaced with a Velos-ETD Orbitrap mass spectrometer was used for liquid chromatography tandem mass spectrometry (LC-MS/MS) analyses. 10 μL of reconstituted sample was injected onto a Waters CSH column (3.0 mm x 150 mm x 1.7 μm particle size) and separated over a 34-minute gradient (mobile phase A: ACN/H<sub>2</sub>O (40:60) containing 10 mM ammonium acetate; mobile phase B: ACN/IPA (10:90) containing 10 mM ammonium acetate) at a flow rate of 250 μL/min. Eluting lipids were introduced to the MS via electrospray ionization in both positive and negative modes, and lipids were fragmented using higher-energy collision dissociation (HCD) and collision-induced dissociation (CID).</div></section><section id="sec5.5.7.5" data-locator="sec5.5.7.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Metabolite identification and data processing</span></h4><div class="section-paragraph">Metabolite identifications and data processing were conducted as previously detailed (<div class="dropBlock reference-citations"><a href="#bib135" id="back-bib135" data-db-target-for="bib135-d4222834e4406" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib135">Webb-Robertson et al., 2014</a><div data-db-target-of="bib135-d4222834e4406" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Webb-Robertson B.-J. </li><li class="loa__item">Kim Y.-M. </li><li class="loa__item">Zink E.M. </li><li class="loa__item">Hallaian K.A. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Madupu R. </li><li class="loa__item">Waters K.M. </li><li class="loa__item">Metz T.O. </li></ul><div class="reference-citations__detials">A statistical analysis of the effects of urease pre-treatment on the measurement of the urinary metabolome by gas chromatography-mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Metabolomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">897-908</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11306-014-0642-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25254001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84894268438&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=897-908&journal=Metabolomics&author=Webb-Robertson+B.-J.&author=Kim+Y.-M.&author=Zink+E.M.&author=Hallaian+K.A.&author=Zhang+Q.&author=Madupu+R.&author=Waters+K.M.&author=Metz+T.O.&title=A+statistical+analysis+of+the+effects+of+urease+pre-treatment+on+the+measurement+of+the+urinary+metabolome+by+gas+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). GC-MS raw data files were processed using Metabolite Detector software v2.0.6 beta (<div class="dropBlock reference-citations"><a href="#bib53" id="back-bib53" data-db-target-for="bib53-d4222834e4409" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib53">Hiller et al., 2009</a><div data-db-target-of="bib53-d4222834e4409" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hiller K. </li><li class="loa__item">Hangebrauk J. </li><li class="loa__item">Jäger C. </li><li class="loa__item">Spura J. </li><li class="loa__item">Schreiber K. </li><li class="loa__item">Schomburg D. </li></ul><div class="reference-citations__detials">MetaboliteDetector: comprehensive analysis tool for targeted and nontargeted GC/MS based metabolome analysis.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">3429-3439</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac802689c&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19358599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-66149088821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=3429-3439&journal=Anal.+Chem.&author=Hiller+K.&author=Hangebrauk+J.&author=J%C3%A4ger+C.&author=Spura+J.&author=Schreiber+K.&author=Schomburg+D.&title=MetaboliteDetector%3A+comprehensive+analysis+tool+for+targeted+and+nontargeted+GC%2FMS+based+metabolome+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Retention indices (RI) of detected metabolites were calculated based on the analysis of the FAMEs mixture, followed by their chromatographic alignment across all analyses after deconvolution. Metabolites were identified by matching experimental spectra to an augmented version of the Agilent Fiehn Metabolomics Retention Time Locked (RTL) Library (<div class="dropBlock reference-citations"><a href="#bib65" id="back-bib65" data-db-target-for="bib65-d4222834e4412" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib65">Kind et al., 2009</a><div data-db-target-of="bib65-d4222834e4412" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kind T. </li><li class="loa__item">Wohlgemuth G. </li><li class="loa__item">Lee D.Y. </li><li class="loa__item">Lu Y. </li><li class="loa__item">Palazoglu M. </li><li class="loa__item">Shahbaz S. </li><li class="loa__item">Fiehn O. </li></ul><div class="reference-citations__detials">FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole and time-of-flight gas chromatography/mass spectrometry.<div class="popup-ref"><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">10038-10048</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac9019522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19928838&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-72449186517&amp;cf=" onclick="newWindow(this.href);return false">Scopus (759)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=10038-10048&journal=Anal.+Chem.&author=Kind+T.&author=Wohlgemuth+G.&author=Lee+D.Y.&author=Lu+Y.&author=Palazoglu+M.&author=Shahbaz+S.&author=Fiehn+O.&title=FiehnLib%3A+mass+spectral+and+retention+index+libraries+for+metabolomics+based+on+quadrupole+and+time-of-flight+gas+chromatography%2Fmass+spectrometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), containing spectra and validated retention indices. All metabolite identifications were manually validated. The NIST 08 GC-MS library was also used to cross validate the spectral matching scores obtained using the Agilent library and to provide identifications for metabolites that were initially unidentified. The three most abundant fragment ions in the spectra of each identified metabolite were automatically determined by Metabolite Detector, and their summed abundances were integrated across the GC elution profile. A matrix of identified metabolites, unidentified metabolite features, and their corresponding abundances for each sample in the batch were exported for statistics.</div></section><section id="sec5.5.7.6" data-locator="sec5.5.7.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Lipid idetification and data processing</span></h4><div class="section-paragraph">LC-MS/MS lipidomics data were analyzed using LIQUID (Lipid Informed Quantitation and Identification) (<div class="dropBlock reference-citations"><a href="#bib70" id="back-bib70" data-db-target-for="bib70-d4222834e4420" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib70">Kyle et al., 2017</a><div data-db-target-of="bib70-d4222834e4420" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kyle J.E. </li><li class="loa__item">Crowell K.L. </li><li class="loa__item">Casey C.P. </li><li class="loa__item">Fujimoto G.M. </li><li class="loa__item">Kim S. </li><li class="loa__item">Dautel S.E. </li><li class="loa__item">Smith R.D. </li><li class="loa__item">Payne S.H. </li><li class="loa__item">Metz T.O. </li></ul><div class="reference-citations__detials">LIQUID: an-open source software for identifying lipids in LC-MS/MS-based lipidomics data.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1744-1746</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtx046&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28158427&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85021049695&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=1744-1746&journal=Bioinforma.+Oxf.+Engl.&author=Kyle+J.E.&author=Crowell+K.L.&author=Casey+C.P.&author=Fujimoto+G.M.&author=Kim+S.&author=Dautel+S.E.&author=Smith+R.D.&author=Payne+S.H.&author=Metz+T.O.&title=LIQUID%3A+an-open+source+software+for+identifying+lipids+in+LC-MS%2FMS-based+lipidomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Confident identifications were selected by manually evaluating the MS/MS spectra for diagnostic and corresponding acyl chain fragments of the identified lipid. In addition, the precursor isotopic profile, extracted ion chromatogram, and mass measurement error along with the elution time were evaluated. To facilitate quantification of lipids, a reference database for lipids identified from the MS/MS data was created and features from each analysis were then aligned to the reference database based on their identification, m/z and retention time using MZmine 2 (<div class="dropBlock reference-citations"><a href="#bib107" id="back-bib107" data-db-target-for="bib107-d4222834e4423" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib107">Pluskal et al., 2010</a><div data-db-target-of="bib107-d4222834e4423" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pluskal T. </li><li class="loa__item">Castillo S. </li><li class="loa__item">Villar-Briones A. </li><li class="loa__item">Oresic M. </li></ul><div class="reference-citations__detials">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">395</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-11-395&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20650010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954772536&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1479)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=395&journal=BMC+Bioinformatics&author=Pluskal+T.&author=Castillo+S.&author=Villar-Briones+A.&author=Oresic+M.&title=MZmine+2%3A+modular+framework+for+processing%2C+visualizing%2C+and+analyzing+mass+spectrometry-based+molecular+profile+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Aligned features were manually verified and peak apex intensity values were exported for subsequent statistical analysis.</div></section></section><section id="sec5.5.8" data-locator="sec5.5.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immunohistochemistry (IHC) validation of genetic alteration downstream impact and immune cell compositions</span></h4><div class="section-paragraph">IHC stains for IDH1, ATRX, SOX9, GAB1, CD3, CD68, CD163, PD-1, and PD-L1 were performed at the Johns Hopkins Hospital clinical IHC laboratory using the autostainers (Ventana XT and Dako). Briefly, tissue blocks were cut into 5-micron thickness sections prior to incubation with primary antibodies. Heat antigen retrieval was performed to enhance signal detection. Primary antibodies were diluted according to standard protocols and/or manufacturer suggestions. A mouse-HRP and/or rabbit-AP polymer detection systems were used to develop immunostaining. Slides were counterstained with hematoxylin and dehydrated for permanent mounting. Appropriate positive and negative controls were also included during the assay.</div></section></section><section id="sec5.6" data-locator="sec5.6" tabindex="0"><h3 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification and statistical analysis</span></h3><section id="sec5.6.1" data-locator="sec5.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Tumor exclusion criteria</span></h4><div class="section-paragraph">One sample (C3L-03747) was excluded from the downstream analysis since it failed the expert pathology review (high necrosis) and had low correlation of RNA and protein or phosphoprotein.</div></section><section id="sec5.6.2" data-locator="sec5.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Genomic data analysis</span></h4><section id="sec5.6.2.1" data-locator="sec5.6.2.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Harmonized genome alignment</span></h4><div class="section-paragraph">WGS, WES, RNA-Seq sequence data were harmonized by NCI Genomic Data Commons (GDC) <a class="inter-ref-link" href="https://gdc.cancer.gov/about-data/gdc-data-harmonization" target="_blank">https://gdc.cancer.gov/about-data/gdc-data-harmonization</a>, which included alignment to GDC’s hg38 human reference genome (GRCh38.d1.vd1) and additional quality checks. All the downstream genomic processing was based on the GDC aligned BAMs to ensure reproducibility. However, RNA-Seq of 9 GTEx and 4 CPTAC samples didn’t have the GDC harmonized BAMs available at the time of the analysis. We followed GDC’s pipeline (same tool and parameters) to align those RNA-Seq samples. To ensure our alignment pipeline is identical to GDC, we randomly selected 10 samples with GDC BAMs available to apply our pipeline and obtain their gene level read count. All selected samples had identical gene counts using GDC or our BAMs.</div></section><section id="sec5.6.2.2" data-locator="sec5.6.2.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Copy number variant calling</span></h4><div class="section-paragraph">Copy Number Variant (CNV) were detected using BIC-Seq2 (NBICseq-norm v0.2.4 and NBICseq-seg v0.7.2) (<div class="dropBlock reference-citations"><a href="#bib138" id="back-bib138" data-db-target-for="bib138-d4222834e4455" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib138">Xi et al., 2016</a><div data-db-target-of="bib138-d4222834e4455" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Xi R. </li><li class="loa__item">Lee S. </li><li class="loa__item">Xia Y. </li><li class="loa__item">Kim T.-M. </li><li class="loa__item">Park P.J. </li></ul><div class="reference-citations__detials">Copy number analysis of whole-genome data using BIC-seq2 and its application to detection of cancer susceptibility variants.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">6274-6286</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkw491&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27260798&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=6274-6286&journal=Nucleic+Acids+Res.&author=Xi+R.&author=Lee+S.&author=Xia+Y.&author=Kim+T.-M.&author=Park+P.J.&title=Copy+number+analysis+of+whole-genome+data+using+BIC-seq2+and+its+application+to+detection+of+cancer+susceptibility+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from WGS tumor and normal paired BAMs using Li Ding Lab’s BIC-Seq2 pipeline v2.0 <a class="inter-ref-link" href="https://github.com/ding-lab/BICSEQ2" target="_blank">https://github.com/ding-lab/BICSEQ2</a>. We used a bin size of 100bp and a lambda of 3 (smoothing parameter for CNV segmentation). To further summarize the arm-level copy number change, we used a weighted sum approach (<div class="dropBlock reference-citations"><a href="#bib130" id="back-bib130" data-db-target-for="bib130-d4222834e4461" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib130">Vasaikar et al., 2019</a><div data-db-target-of="bib130-d4222834e4461" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Vasaikar S. </li><li class="loa__item">Huang C. </li><li class="loa__item">Wang X. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Savage S.R. </li><li class="loa__item">Wen B. </li><li class="loa__item">Dou Y. </li><li class="loa__item">Zhang Y. </li><li class="loa__item">Shi Z. </li><li class="loa__item">Arshad O.A. </li><li>et al.</li></ul><div class="reference-citations__detials">Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (107)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.A.&author=Savage+S.R.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.A.&title=Proteogenomic+analysis+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), in which the segment-level log<sub>2</sub> copy ratios for all the segments located in the given arm were added up with the length of each segment being weighted. We then used GISTIC2 v2.0.22 (<div class="dropBlock reference-citations"><a href="#bib91" id="back-bib91" data-db-target-for="bib91-d4222834e4467" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib91">Mermel et al., 2011</a><div data-db-target-of="bib91-d4222834e4467" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Mermel C.H. </li><li class="loa__item">Schumacher S.E. </li><li class="loa__item">Hill B. </li><li class="loa__item">Meyerson M.L. </li><li class="loa__item">Beroukhim R. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1131)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to integrate results from individual patients and identify genomic regions recurrently amplified or deleted in our samples. The threshold of arm-level CNV was 0.3 for gain and -0.3 for loss.</div><div class="section-paragraph">We defined a tumor with chr7 amplification or chr10 deletion when the GISTIC results of at least one chromosome arm exceeded the threshold (± 0.3). For samples that both chromosome arms were within the GISTIC threshold or its GISTIC result was not available, we considered the tumor with the chromosome amplification or deletion if the averaged CNV values exceeded the threshold (± 0.2).</div></section><section id="sec5.6.2.3" data-locator="sec5.6.2.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Somatic variant calling</span></h4><div class="section-paragraph">Somatic variants were called from WES tumor and normal paired BAMs using somaticwrapper v1.5, a pipeline designed for detection of somatic variants from tumor and normal exome data. The pipeline merges and filters variant calls from four callers: Strelka v2.9.2 (<div class="dropBlock reference-citations"><a href="#bib64" id="back-bib64" data-db-target-for="bib64-d4222834e4477" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib64">Kim et al., 2018</a><div data-db-target-of="bib64-d4222834e4477" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Scheffler K. </li><li class="loa__item">Halpern A.L. </li><li class="loa__item">Bekritsky M.A. </li><li class="loa__item">Noh E. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Chen X. </li><li class="loa__item">Kim Y. </li><li class="loa__item">Beyter D. </li><li class="loa__item">Krusche P. </li><li>et al.</li></ul><div class="reference-citations__detials">Strelka2: fast and accurate calling of germline and somatic variants.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (172)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), VarScan v2.3.8 (<div class="dropBlock reference-citations"><a href="#bib66" id="back-bib66" data-db-target-for="bib66-d4222834e4480" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib66">Koboldt et al., 2012</a><div data-db-target-of="bib66-d4222834e4480" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Koboldt D.C. </li><li class="loa__item">Zhang Q. </li><li class="loa__item">Larson D.E. </li><li class="loa__item">Shen D. </li><li class="loa__item">McLellan M.D. </li><li class="loa__item">Lin L. </li><li class="loa__item">Miller C.A. </li><li class="loa__item">Mardis E.R. </li><li class="loa__item">Ding L. </li><li class="loa__item">Wilson R.K. </li></ul><div class="reference-citations__detials">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2216)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), Pindel v0.2.5 (<div class="dropBlock reference-citations"><a href="#bib140" id="back-bib140" data-db-target-for="bib140-d4222834e4483" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib140">Ye et al., 2009</a><div data-db-target-of="bib140-d4222834e4483" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ye K. </li><li class="loa__item">Schulz M.H. </li><li class="loa__item">Long Q. </li><li class="loa__item">Apweiler R. </li><li class="loa__item">Ning Z. </li></ul><div class="reference-citations__detials">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1160)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and MuTect v1.1.7 (<div class="dropBlock reference-citations"><a href="#bib26" id="back-bib26" data-db-target-for="bib26-d4222834e4486" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib26">Cibulskis et al., 2013</a><div data-db-target-of="bib26-d4222834e4486" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cibulskis K. </li><li class="loa__item">Lawrence M.S. </li><li class="loa__item">Carter S.L. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Jaffe D. </li><li class="loa__item">Sougnez C. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Meyerson M. </li><li class="loa__item">Lander E.S. </li><li class="loa__item">Getz G. </li></ul><div class="reference-citations__detials">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2160)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). SNV calls were obtained from Strelka, Varscan, and Mutect. Indel calls were obtained from Stralka2, Varscan, and Pindel. The following filters were applied to get variant calls of high confidence:<ul class="ce-list--remove-bullets" id="olist0010"><li class="ce-list--remove-bullets__list-item" id="o0010"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Normal VAF ≤ 0.02 and tumor VAF ≥ 0.05</div></li><li class="ce-list--remove-bullets__list-item" id="o0015"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Read depth in tumor ≥ 14 and normal ≥ 8</div></li><li class="ce-list--remove-bullets__list-item" id="o0020"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Indel length &lt; 100 bp</div></li><li class="ce-list--remove-bullets__list-item" id="o0025"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">All variants must be called by 2 or more callers</div></li><li class="ce-list--remove-bullets__list-item" id="o0030"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">All variants must be exonic</div></li><li class="ce-list--remove-bullets__list-item" id="o0035"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Exclude variants in dbSNP but not in COSMIC</div></li></ul></div><div class="section-paragraph">We additionally called somatic whole-genome variants using WGS tumor and normal paired BAMs using somaticwrapper v1.3 identical to the exome version except that we kept non-exonic variants.</div></section><section id="sec5.6.2.4" data-locator="sec5.6.2.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Germline variant calling and annotation</span></h4><div class="section-paragraph">Germline variant calling was performed using Li Ding Lab’s pipeline germlinewrapper v1.1, which implements multiple tools for the detection of germline INDELs and SNVs. Germline SNVs were identified using VarScan v2.3.8 (with parameters: --min-var-freq 0.10 --p-value 0.10, --min-coverage 3 --strand-filter 1) operating on a mpileup stream produced by samtools v1.2 (with parameters: -q 1 -Q 13) and GATK v4.0.0.0 (<div class="dropBlock reference-citations"><a href="#bib90" id="back-bib90" data-db-target-for="bib90-d4222834e4521" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib90">McKenna et al., 2010</a><div data-db-target-of="bib90-d4222834e4521" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">McKenna A. </li><li class="loa__item">Hanna M. </li><li class="loa__item">Banks E. </li><li class="loa__item">Sivachenko A. </li><li class="loa__item">Cibulskis K. </li><li class="loa__item">Kernytsky A. </li><li class="loa__item">Garimella K. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Gabriel S. </li><li class="loa__item">Daly M. </li><li>et al.</li></ul><div class="reference-citations__detials">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10783)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using its haplotype caller in single-sample mode with duplicate and unmapped reads removed and retaining calls with a minimum quality threshold of 10. All resulting variants were limited to the coding region of the full-length transcripts obtained from Ensembl release 95 plus additional two base pairs flanking each exon to cover splice donor/acceptor sites. We required variants to have allelic depth ≥ 5 reads for the alternative allele in both tumor and normal samples. We used bam-readcount v0.8 for reference and alternative alleles quantification (with parameters: -q 10 -b 15) in both normal and tumor samples. Additionally, we filtered all variants with ≥ 0.05% frequency in gnomAD v2.1 (<div class="dropBlock reference-citations"><a href="#bib57" id="back-bib57" data-db-target-for="bib57-d4222834e4524" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib57">Karczewski et al., 2020</a><div data-db-target-of="bib57-d4222834e4524" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Karczewski K.J. </li><li class="loa__item">Francioli L.C. </li><li class="loa__item">Tiao G. </li><li class="loa__item">Cummings B.B. </li><li class="loa__item">Alföldi J. </li><li class="loa__item">Wang Q. </li><li class="loa__item">Collins R.L. </li><li class="loa__item">Laricchia K.M. </li><li class="loa__item">Ganna A. </li><li class="loa__item">Birnbaum D.P. </li><li>et al.</li></ul><div class="reference-citations__detials">The mutational constraint spectrum quantified from variation in 141,456 humans.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">581</span></span>: <span class="ref__seriesPages">434-443</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41586-020-2308-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=32461654&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85085542423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (613)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2020&pages=434-443&journal=Nature&author=Karczewski+K.J.&author=Francioli+L.C.&author=Tiao+G.&author=Cummings+B.B.&author=Alf%C3%B6ldi+J.&author=Wang+Q.&author=Collins+R.L.&author=Laricchia+K.M.&author=Ganna+A.&author=Birnbaum+D.P.&title=The+mutational+constraint+spectrum+quantified+from+variation+in+141%2C456+humans">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and The 1000 Genomes Project (<div class="dropBlock reference-citations"><a href="#bib125" id="back-bib125" data-db-target-for="bib125-d4222834e4527" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib125">The 1000 Genomes Project Consortium, 2015</a><div data-db-target-of="bib125-d4222834e4527" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">The 1000 Genomes Project Consortium </li></ul><div class="reference-citations__detials">A global reference for human genetic variation.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">526</span></span>: <span class="ref__seriesPages">68-74</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature15393&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26432245&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84943171338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5449)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=68-74&journal=Nature&author=The+1000+Genomes+Project+Consortium+&title=A+global+reference+for+human+genetic+variation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.6.2.5" data-locator="sec5.6.2.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">TERT promoter mutation calling</span></h4><div class="section-paragraph">We used bam-readcount to count reads in WGS tumor and blood normal BAMs at the known hotspot positions at hg38 chr5:1295113 and chr5:1295135. We called a mutation if it was not observed in matching blood normal BAM and VAF &gt; 5%. For all tumor samples lacking a <em>TERT</em>p hotspot mutation, we performed the readcount across the entire TERT promoter region from chr5:1294200 to chr5:1295601 (hg38). In these cases, we applied a more stringent VAF cutoff of 10%.</div></section><section id="sec5.6.2.6" data-locator="sec5.6.2.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Structural variant calling</span></h4><div class="section-paragraph">Structural variants (SVs) were called by Manta v1.6.0 (<div class="dropBlock reference-citations"><a href="#bib24" id="back-bib24" data-db-target-for="bib24-d4222834e4542" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib24">Chen et al., 2016</a><div data-db-target-of="bib24-d4222834e4542" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Chen X. </li><li class="loa__item">Schulz-Trieglaff O. </li><li class="loa__item">Shaw R. </li><li class="loa__item">Barnes B. </li><li class="loa__item">Schlesinger F. </li><li class="loa__item">Källberg M. </li><li class="loa__item">Cox A.J. </li><li class="loa__item">Kruglyak S. </li><li class="loa__item">Saunders C.T. </li></ul><div class="reference-citations__detials">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (375)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and DELLY v0.8.1 (<div class="dropBlock reference-citations"><a href="#bib110" id="back-bib110" data-db-target-for="bib110-d4222834e4545" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib110">Rausch et al., 2012</a><div data-db-target-of="bib110-d4222834e4545" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rausch T. </li><li class="loa__item">Zichner T. </li><li class="loa__item">Schlattl A. </li><li class="loa__item">Stutz A.M. </li><li class="loa__item">Benes V. </li><li class="loa__item">Korbel J.O. </li></ul><div class="reference-citations__detials">DELLY: structural variant discovery by integrated paired-end and split-read analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">i333-i339</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22962449&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84866440781&amp;cf=" onclick="newWindow(this.href);return false">Scopus (759)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i333-i339&journal=Bioinformatics&author=Rausch+T.&author=Zichner+T.&author=Schlattl+A.&author=Stutz+A.M.&author=Benes+V.&author=Korbel+J.O.&title=DELLY%3A+structural+variant+discovery+by+integrated+paired-end+and+split-read+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from WGS tumor and normal paired BAMs. We ran Manta on canonical chromosomes with the default record- and sample-level filters. For DELLY, we followed somatic SV workflow Only SV calls with PASS filter status were kept for downstream analysis. Lastly, we manually reviewed all the SV calls in the genes of interest (e.g. EGFR and PDGFRA).</div></section><section id="sec5.6.2.7" data-locator="sec5.6.2.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">DNA methylation microarray processing</span></h4><div class="section-paragraph">Raw methylation idat files were downloaded from CPTAC DCC and GDC. Beta values of CpG loci were reported after functional normalization, quality check, common SNP filtering, and probe annotation using Li Ding Lab’s methylation pipeline v1.1 <a class="inter-ref-link" href="https://github.com/ding-lab/cptac_methylation" target="_blank">https://github.com/ding-lab/cptac_methylation</a>. Resulting beta values of methylation were used for downstream analysis.</div></section><section id="sec5.6.2.8" data-locator="sec5.6.2.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Classification of MGMT promoter DNA methylation status</span></h4><div class="section-paragraph">We applied the MGMT-STP27 model (<div class="dropBlock reference-citations"><a href="#bib7" id="back-bib7" data-db-target-for="bib7-d4222834e4561" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib7">Bady et al., 2012</a><div data-db-target-of="bib7-d4222834e4561" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bady P. </li><li class="loa__item">Sciuscio D. </li><li class="loa__item">Diserens A.-C. </li><li class="loa__item">Bloch J. </li><li class="loa__item">van den Bent M.J. </li><li class="loa__item">Marosi C. </li><li class="loa__item">Dietrich P.-Y. </li><li class="loa__item">Weller M. </li><li class="loa__item">Mariani L. </li><li class="loa__item">Heppner F.L. </li><li>et al.</li></ul><div class="reference-citations__detials">MGMT methylation analysis of glioblastoma on the Infinium methylation BeadChip identifies two distinct CpG regions associated with gene silencing and outcome, yielding a prediction model for comparisons across datasets, tumor grades, and CIMP-status.<div class="popup-ref"><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">124</span></span>: <span class="ref__seriesPages">547-560</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-012-1016-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22810491&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84866542816&amp;cf=" onclick="newWindow(this.href);return false">Scopus (179)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2012&pages=547-560&journal=Acta+Neuropathol.&author=Bady+P.&author=Sciuscio+D.&author=Diserens+A.-C.&author=Bloch+J.&author=van+den+Bent+M.J.&author=Marosi+C.&author=Dietrich+P.-Y.&author=Weller+M.&author=Mariani+L.&author=Heppner+F.L.&title=MGMT+methylation+analysis+of+glioblastoma+on+the+Infinium+methylation+BeadChip+identifies+two+distinct+CpG+regions+associated+with+gene+silencing+and+outcome%2C+yielding+a+prediction+model+for+comparisons+across+datasets%2C+tumor+grades%2C+and+CIMP-status">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to determine the MGMT promoter DNA methylation status, which is a logistic regression prediction model based on the M values of the two probes in the MGMT promoter region, cg12434587 and cg12981137. M-values were converted from the beta values of the processed microarray data by <em>M</em> = log<sub>2</sub>(<em>beta</em> / (1 - <em>beta</em>)). 9 tumors with low data quality of DNA methylation array were excluded from the prediction. 38 out of the remaining 90 tumors (42%) were predicted to be MGMT promoter DNA hypermethylated.</div></section><section id="sec5.6.2.9" data-locator="sec5.6.2.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Telomere length quantification and telomere genotyping</span></h4><div class="section-paragraph">We used Telseq v0.0.1 (<div class="dropBlock reference-citations"><a href="#bib33" id="back-bib33" data-db-target-for="bib33-d4222834e4578" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib33">Ding et al., 2014</a><div data-db-target-of="bib33-d4222834e4578" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ding Z. </li><li class="loa__item">Mangino M. </li><li class="loa__item">Aviv A. </li><li class="loa__item">Spector T. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Estimating telomere length from whole genome sequence data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">e75</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku181&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24609383&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84899643560&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=e75&journal=Nucleic+Acids+Res.&author=Ding+Z.&author=Mangino+M.&author=Aviv+A.&author=Spector+T.&author=Durbin+R.&title=Estimating+telomere+length+from+whole+genome+sequence+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to estimate the telomere length using WXS and WGS tumor and blood normal paired BAMs. We defined telomere length ratio as the ratio between the estimated tumor telomere length and the estimated blood normal telomere length. While WXS and WGS-based telomere length ratios were well correlated, we used WGS based lengths for the downstream analysis. We defined long telomere phenotype as tumors with WGS telomere length ratio &gt; 1.2, and short telomere phenotype as WGS telomere length ratio &lt; 0.8.</div><div class="section-paragraph">We identified telomere genotypes as the following:<ul class="ce-list--remove-bullets" id="olist0015"><li class="ce-list--remove-bullets__list-item" id="o0040"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text"><em>TERT</em>p hotspot if tumor has <em>TERT</em>p hotspot mutation</div></li><li class="ce-list--remove-bullets__list-item" id="o0045"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text"><em>ATRX</em>mut for all remaining tumors with only <em>ATRX</em> mutation</div></li><li class="ce-list--remove-bullets__list-item" id="o0050"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text"><em>ATRX</em>mut <em>IDH1</em>mut for all remaining tumors with both <em>ATRX</em> and <em>IDH1</em> mutated</div></li><li class="ce-list--remove-bullets__list-item" id="o0055"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text"><em>IDH1</em>mut for all remaining tumors with only <em>IDH1</em> mutation</div></li><li class="ce-list--remove-bullets__list-item" id="o0060"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text"><em>TERT</em>p not hotspot for all remaining tumors without hotspot mutation in <em>TERT</em> promoter and expressing <em>TERT</em></div></li><li class="ce-list--remove-bullets__list-item" id="o0065"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">WT for the remaining tumors that do not fall into any category</div></li></ul></div></section></section><section id="sec5.6.3" data-locator="sec5.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA quantification and analysis</span></h4><section id="sec5.6.3.1" data-locator="sec5.6.3.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA quantification</span></h4><div class="section-paragraph">We obtained the gene-level readcount, Fragments Per Kilobase of transcript per Million mapped reads (FPKM) and FPKM Upper Quartile (FPKM-UQ) values by following the GDC’s RNA-Seq pipeline (Expression mRNA Pipeline) <a class="inter-ref-link" href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/" target="_blank">https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/</a>, with the exception of running the quantification tools in the stranded mode. We used HTSeq v0.11.2 (<div class="dropBlock reference-citations"><a href="#bib2" id="back-bib2" data-db-target-for="bib2-d4222834e4639" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib2">Anders et al., 2015</a><div data-db-target-of="bib2-d4222834e4639" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Anders S. </li><li class="loa__item">Pyl P.T. </li><li class="loa__item">Huber W. </li></ul><div class="reference-citations__detials">HTSeq—a Python framework to work with high-throughput sequencing data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">166-169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtu638&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25260700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84928987900&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7209)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=166-169&journal=Bioinformatics&author=Anders+S.&author=Pyl+P.T.&author=Huber+W.&title=HTSeq%E2%80%94a+Python+framework+to+work+with+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to calculate the gene-level stranded readcount (parameters: -r pos -f bam -a 10 -s reverse -t exon -i gene_id -m intersection-nonempty --nonunique=none) using GENCODE v22 (Ensembl v79) annotation downloaded from GDC (gencode.gene.info.v22.tsv). The readcount was then converted to FPKM and FPKM-UQ using the same formula described in GDC’s Expression mRNA Pipeline documentation.</div></section><section id="sec5.6.3.2" data-locator="sec5.6.3.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">RNA fusion detection</span></h4><div class="section-paragraph">We used three callers, STAR-Fusion v1.5.0 (<div class="dropBlock reference-citations"><a href="#bib50" id="back-bib50" data-db-target-for="bib50-d4222834e4647" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib50">Haas et al., 2019</a><div data-db-target-of="bib50-d4222834e4647" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Haas B.J. </li><li class="loa__item">Dobin A. </li><li class="loa__item">Li B. </li><li class="loa__item">Stransky N. </li><li class="loa__item">Pochet N. </li><li class="loa__item">Regev A. </li></ul><div class="reference-citations__detials">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (51)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), INTEGRATE v0.2.6 (<div class="dropBlock reference-citations"><a href="#bib143" id="back-bib143" data-db-target-for="bib143-d4222834e4650" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib143">Zhang et al., 2016</a><div data-db-target-of="bib143-d4222834e4650" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhang J. </li><li class="loa__item">White N.M. </li><li class="loa__item">Schmidt H.K. </li><li class="loa__item">Fulton R.S. </li><li class="loa__item">Tomlinson C. </li><li class="loa__item">Warren W.C. </li><li class="loa__item">Wilson R.K. </li><li class="loa__item">Maher C.A. </li></ul><div class="reference-citations__detials">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.<div class="popup-ref"><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and EricScript v0.5.5 (<div class="dropBlock reference-citations"><a href="#bib13" id="back-bib13" data-db-target-for="bib13-d4222834e4653" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib13">Benelli et al., 2012</a><div data-db-target-of="bib13-d4222834e4653" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Benelli M. </li><li class="loa__item">Pescucci C. </li><li class="loa__item">Marseglia G. </li><li class="loa__item">Severgnini M. </li><li class="loa__item">Torricelli F. </li><li class="loa__item">Magi A. </li></ul><div class="reference-citations__detials">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), to call consensus fusion/chimeric events in our samples. Calls by each tool using tumor and normal RNA-Seq data were then merged into a single file and extensive filtering is done. As STAR-Fusion has higher sensitivity, calls made by this tool with higher supporting evidence (defined by fusion fragments per million total reads, or FFPM &gt; 0.1) were required, or a given fusion must be reported by at least 2 callers. We then removed fusions present in our panel of blacklisted or normal fusions, which included uncharacterized genes, immunoglobulin genes, mitochondrial genes, and others, as well as fusions from the same gene or paralog genes and fusions reported in TCGA normal samples (<div class="dropBlock reference-citations"><a href="#bib44" id="back-bib44" data-db-target-for="bib44-d4222834e4656" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib44">Gao et al., 2018</a><div data-db-target-of="bib44-d4222834e4656" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao Q. </li><li class="loa__item">Liang W.-W. </li><li class="loa__item">Foltz S.M. </li><li class="loa__item">Mutharasu G. </li><li class="loa__item">Jayasinghe R.G. </li><li class="loa__item">Cao S. </li><li class="loa__item">Liao W.-W. </li><li class="loa__item">Reynolds S.M. </li><li class="loa__item">Wyczalkowski M.A. </li><li class="loa__item">Yao L. </li><li>et al.</li></ul><div class="reference-citations__detials">Driver fusions and their implications in the development and treatment of human cancers.<div class="popup-ref"><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">227-238.e3</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29617662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044744284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (149)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=227-238.e3&journal=Cell+Rep.&author=Gao+Q.&author=Liang+W.-W.&author=Foltz+S.M.&author=Mutharasu+G.&author=Jayasinghe+R.G.&author=Cao+S.&author=Liao+W.-W.&author=Reynolds+S.M.&author=Wyczalkowski+M.A.&author=Yao+L.&title=Driver+fusions+and+their+implications+in+the+development+and+treatment+of+human+cancers">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), GTEx tissues (reported in STAR-Fusion output), and non-cancer cell studies (<div class="dropBlock reference-citations"><a href="#bib5" id="back-bib5" data-db-target-for="bib5-d4222834e4659" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib5">Babiceanu et al., 2016</a><div data-db-target-of="bib5-d4222834e4659" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Babiceanu M. </li><li class="loa__item">Qin F. </li><li class="loa__item">Xie Z. </li><li class="loa__item">Jia Y. </li><li class="loa__item">Lopez K. </li><li class="loa__item">Janus N. </li><li class="loa__item">Facemire L. </li><li class="loa__item">Kumar S. </li><li class="loa__item">Pang Y. </li><li class="loa__item">Qi Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Recurrent chimeric fusion RNAs in non-cancer tissues and cells.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">2859-2872</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkw032&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26837576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84963985214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=2859-2872&journal=Nucleic+Acids+Res.&author=Babiceanu+M.&author=Qin+F.&author=Xie+Z.&author=Jia+Y.&author=Lopez+K.&author=Janus+N.&author=Facemire+L.&author=Kumar+S.&author=Pang+Y.&author=Qi+Y.&title=Recurrent+chimeric+fusion+RNAs+in+non-cancer+tissues+and+cells">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Finally, we removed normal fusions from the tumor fusions to curate the final set.</div></section><section id="sec5.6.3.3" data-locator="sec5.6.3.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">miRNA quantification</span></h4><div class="section-paragraph">miRNA-Seq FASTQ files were downloaded from GDC. We reported the mature miRNA and precursor miRNA expression in TPM (Transcripts Per Million) after adapter trimming, quality check, alignment, annotation, reads counting using Li Ding Lab’s miRNA pipeline <a class="inter-ref-link" href="https://github.com/ding-lab/CPTAC_miRNA" target="_blank">https://github.com/ding-lab/CPTAC_miRNA</a>. The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression from its precursor miRNAs.</div></section><section id="sec5.6.3.4" data-locator="sec5.6.3.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Circular RNA prediction and quantification</span></h4><div class="section-paragraph">The hg38 reference genome and GDC’s annotations were used for the circRNA analysis. First, CIRI v2.0.6 (<div class="dropBlock reference-citations"><a href="#bib43" id="back-bib43" data-db-target-for="bib43-d4222834e4675" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib43">Gao et al., 2015</a><div data-db-target-of="bib43-d4222834e4675" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gao Y. </li><li class="loa__item">Wang J. </li><li class="loa__item">Zhao F. </li></ul><div class="reference-citations__detials">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-014-0571-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25583365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84937578024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (409)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=4&journal=Genome+Biol.&author=Gao+Y.&author=Wang+J.&author=Zhao+F.&title=CIRI%3A+an+efficient+and+unbiased+algorithm+for+de+novo+circular+RNA+identification">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used to call circular RNA with default parameters and BWA v0.7.17-r1188 (<div class="dropBlock reference-citations"><a href="#bib74" id="back-bib74" data-db-target-for="bib74-d4222834e4678" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib74">Li and Durbin, 2009</a><div data-db-target-of="bib74-d4222834e4678" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li H. </li><li class="loa__item">Durbin R. </li></ul><div class="reference-citations__detials">Fast and accurate short read alignment with Burrows-Wheeler transform.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1754-1760</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp324&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19451168&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-67649884743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19451)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1754-1760&journal=Bioinforma.+Oxf.+Engl.&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used as a mapping tool. The cutoff of supporting reads for circRNA was set to 10. Then a pseudo-linear transcript strategy was used to quantify circular RNA expression (<div class="dropBlock reference-citations"><a href="#bib77" id="back-bib77" data-db-target-for="bib77-d4222834e4681" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib77">Li et al., 2017</a><div data-db-target-of="bib77-d4222834e4681" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li M. </li><li class="loa__item">Xie X. </li><li class="loa__item">Zhou J. </li><li class="loa__item">Sheng M. </li><li class="loa__item">Yin X. </li><li class="loa__item">Ko E.-A. </li><li class="loa__item">Zhou T. </li><li class="loa__item">Gu W. </li></ul><div class="reference-citations__detials">Quantifying circular RNA expression from RNA-seq data using model-based framework.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinforma.+Oxf.+Engl.&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3’ end was copied to the 5’ end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM v1.3.1 (<div class="dropBlock reference-citations"><a href="#bib73" id="back-bib73" data-db-target-for="bib73-d4222834e4684" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib73">Li and Dewey, 2011</a><div data-db-target-of="bib73-d4222834e4684" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Li B. </li><li class="loa__item">Dewey C.N. </li></ul><div class="reference-citations__detials">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6978)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) with Bowtie2 v2.3.3 (<div class="dropBlock reference-citations"><a href="#bib71" id="back-bib71" data-db-target-for="bib71-d4222834e4687" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib71">Langmead and Salzberg, 2012</a><div data-db-target-of="bib71-d4222834e4687" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Langmead B. </li><li class="loa__item">Salzberg S.L. </li></ul><div class="reference-citations__detials">Fast gapped-read alignment with Bowtie 2.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">357-359</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.1923&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22388286&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84859210032&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17228)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=357-359&journal=Nat.+Methods&author=Langmead+B.&author=Salzberg+S.L.&title=Fast+gapped-read+alignment+with+Bowtie+2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) as the mapping tool was used to quantify circular RNA expression based on the mixed transcripts. After quantification, the upper quantile method was applied for normalization and the normalized matrix was log<sub>2</sub>-transformed.</div></section></section><section id="sec5.6.4" data-locator="sec5.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">snRNA-seq quantification and analysis</span></h4><section id="sec5.6.4.1" data-locator="sec5.6.4.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">snRNA-seq data preprocessing</span></h4><div class="section-paragraph">For each sample, we obtained the unfiltered feature-barcode matrix per sample by passing the demultiplexed FASTQs to Cell Ranger v3.1.0 ‘count’ command using default parameters and a customized pre-mRNA GRCh38 genome reference was built to capture both exonic and intronic reads. The customized genome reference modified the transcript annotation from the 10x Genomics pre-built human genome reference 3.0.0 (GRCh38 and Ensembl 93).</div><div class="section-paragraph">Seurat v3.1.2 (<div class="dropBlock reference-citations"><a href="#bib20" id="back-bib20" data-db-target-for="bib20-d4222834e4704" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib20">Butler et al., 2018</a><div data-db-target-of="bib20-d4222834e4704" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Butler A. </li><li class="loa__item">Hoffman P. </li><li class="loa__item">Smibert P. </li><li class="loa__item">Papalexi E. </li><li class="loa__item">Satija Rahul </li></ul><div class="reference-citations__detials">Integrating single-cell transcriptomic data across different conditions, technologies, and species.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">411-420</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.4096&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29608179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85046298440&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1751)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=411-420&journal=Nat.+Biotechnol.&author=Butler+A.&author=Hoffman+P.&author=Smibert+P.&author=Papalexi+E.&author=Satija+Rahul&title=Integrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+species">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib51" id="back-bib51" data-db-target-for="bib51-d4222834e4707" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib51">Hafemeister and Satija, 2019</a><div data-db-target-of="bib51-d4222834e4707" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Hafemeister C. </li><li class="loa__item">Satija R. </li></ul><div class="reference-citations__detials">Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression.<div class="popup-ref"><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">576827</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=576827&journal=BioRxiv&author=Hafemeister+C.&author=Satija+R.&title=Normalization+and+variance+stabilization+of+single-cell+RNA-seq+data+using+regularized+negative+binomial+regression">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) was used for all subsequent analysis. We constructed a Seurat object using the unfiltered feature-barcode matrix for each sample. A series of quality filters were applied to the data to remove those cell barcodes which fell into any one of these categories recommended by Seurat: too few total transcript counts (&lt; 300); possible debris with too few genes expressed (&lt; 200) and too few UMIs (&lt; 1,000); possible more than one cell with too many genes expressed (&gt; 10,000) and too many UMIs (&gt; 10,000); possible dead cell or a sign of cellular stress and apoptosis with too high proportion of mitochondrial gene expression over the total transcript counts (&gt; 10%).</div><div class="section-paragraph">Each sample was scaled and normalized using Seurat’s ‘SCTransform’ function to correct for batch effects (with parameters: vars.to.regress = c("nCount_RNA", "percent.mito"), variable.features.n = 3000). We then merged all samples and repeated the same scaling and normalization method. All cells in the merged Seurat object were then clustered using the original Louvain algorithm (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d4222834e4712" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Blondel et al., 2008</a><div data-db-target-of="bib15-d4222834e4712" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blondel V.D. </li><li class="loa__item">Guillaume J.-L. </li><li class="loa__item">Lambiotte R. </li><li class="loa__item">Lefebvre E. </li></ul><div class="reference-citations__detials">Fast unfolding of communities in large networks.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Mech. Theor. Exp.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">2008</span></span>: <span class="ref__seriesPages">P10008</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1088%2F1742-5468%2F2008%2F10%2FP10008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-56349094785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7916)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2008&publication_year=2008&pages=P10008&journal=J.%C2%A0Stat.+Mech.+Theor.+Exp.&author=Blondel+V.D.&author=Guillaume+J.-L.&author=Lambiotte+R.&author=Lefebvre+E.&title=Fast+unfolding+of+communities+in+large+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and the top 30 PCA dimensions via Seurat's ‘FindNeighbors’ and ‘FindClusters’ (with parameters: resolution = 0.5) functions. The resulting merged and normalized matrix was used for the subsequent analysis.</div></section><section id="sec5.6.4.2" data-locator="sec5.6.4.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">snRNA-seq cell type annotation</span></h4><div class="section-paragraph">Cell types were assigned to each cluster by manually reviewing the expression of marker genes. The marker genes used were TMEM119, P2RY12, SLC2A5, TGFBR1, GPR34, SALL1, GAS6, MERTK, C1QA, C3, PROS1, CD68, ADGRE1, AIF1, CX3CR1, TREM2, ITGAM, SPI1, CSF1R, LAPTM5, RGS1, PTPRC (Microglia); LGALS2, FCER1G, FCN1, CSTA, S100A8, S100A9, S100A12, LYZ, CD68, CD14 (Monocytes); AIF1, CD68, LST1, IFITM2 (Macrophages); Microglia, macrophages were named together as tumor-associated microglia/macrophages. CD8A, CD8B, CD3E, CD3D, PRF1, GZMA, GZMB, GZMK, GZMH, CD4, IL7R, LTB, LDHB, CD69, FAS, KLRG1, CD28, DPP4 (CD4/CD8 T-cells); CD19, CD79A, CD79B, MS4A1, SDC1, IGHG1, IGHG3, IGH4 (B-cells/Plasma); MBP, PLP1, CLDN11, MOG, KLK6, CNDP1, GJB1, MAG, NKX6-2, OPALIN, FOLH1, CARNS1, MOBP, ERMN, TMEM125, CNTN2, ENPP2, SH3GL3, MAL, TF, ST18, TPPP (Oligodendrocytes); PPP1R1B, CPNE6, NTSR2, GJB6, SLC39A12, GABRA2, WIF1, GABRG1, HHATL, C16orf89, ACSBG1, FBXO2, MMP28, SNCG, RANBP3L, IQCA1, SLC14A1 (Astrocytes); FSTL5, GAD2, GRIN1, SYNPR, GABRG2, DLX5, SULT4A1, RBFOX3, CALY, SLC6A17, SLC32A1, CCK, GABRA1, CDH9, DLX6-AS1, KCNC2, MIR7-3HG, FRMPD4, CAMKV, PCP4L1 (Neurons); EMCN, FLT1, PECAM1, KDR, PLVAP, PLVAP, TEK, VWF, ACTA2, ANGPT2, COL1A1, COL3A1, COL5A1, COL12A1, EMILIN1, LUM (Stroma).</div></section><section id="sec5.6.4.3" data-locator="sec5.6.4.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">snRNA-seq analysis</span></h4><div class="section-paragraph">Differentially expressed genes within TAM cells, T-cells and Tumor cells were identified by FindMarkers function comparing cells belonging to one subtype (immune subtype or multi-omics subtype) to the rest. Wilcoxon statistical test was used. log<sub>2</sub>FC &gt; 0.25 and FDR &lt; 0.05 was used to filter DEGs.</div></section></section><section id="sec5.6.5" data-locator="sec5.6.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">MS data interpretation</span></h4><section id="sec5.6.5.1" data-locator="sec5.6.5.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of TMT global proteomics data</span></h4><div class="section-paragraph">LC-MS/MS analysis of the TMT11-labeled, bRPLC fractionated samples generated a total of 264 global proteomics data files. The Thermo RAW files were processed with mzRefinery to characterize and correct for any instrument calibration errors, and then with MS-GF+ v9881(<div class="dropBlock reference-citations"><a href="#bib47" id="back-bib47" data-db-target-for="bib47-d4222834e4736" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib47">Gibbons et al., 2015</a><div data-db-target-of="bib47-d4222834e4736" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gibbons B.C. </li><li class="loa__item">Chambers M.C. </li><li class="loa__item">Monroe M.E. </li><li class="loa__item">Tabb D.L. </li><li class="loa__item">Payne S.H. </li></ul><div class="reference-citations__detials">Correcting systematic bias and instrument measurement drift with mzRefinery.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinforma.+Oxf.+Engl.&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib62" id="back-bib62" data-db-target-for="bib62-d4222834e4739" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib62">Kim and Pevzner, 2014</a><div data-db-target-of="bib62-d4222834e4739" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">MS-GF+ makes progress towards a universal database search tool for proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (469)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib63" id="back-bib63" data-db-target-for="bib63-d4222834e4742" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib63">Kim et al., 2008</a><div data-db-target-of="bib63-d4222834e4742" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim S. </li><li class="loa__item">Gupta N. </li><li class="loa__item">Pevzner P.A. </li></ul><div class="reference-citations__detials">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.<div class="popup-ref"><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to match against the RefSeq human protein sequence database downloaded on June 29, 2018 (hg38; 41,734 proteins), combined with 264 contaminants (e.g., trypsin, keratin). The partially tryptic search used a ± 10 ppm parent ion tolerance, allowed for isotopic error in precursor ion selection, and searched a decoy database composed of the forward and reversed protein sequences. MS-GF+ considered static carbamidomethylation (+57.0215 Da) on Cys residues and TMT modification (+229.1629 Da) on the peptide N-terminus and Lys residues, and dynamic oxidation (+15.9949 Da) on Met residues for searching the global proteome data.</div><div class="section-paragraph">Peptide identification stringency was set at a maximum 1% FDR at peptide level using PepQValue &lt; 0.005 and parent ion mass deviation &lt; 7 ppm criteria. A minimum of 6 unique peptides per 1000 amino acids of protein length was then required for achieving 1% at the protein level within the full data set. Inference of parsimonious protein set at gene level resulted in the identification of protein groups covering 11,141 genes.</div><div class="section-paragraph">The intensities of all 11 TMT reporter ions were extracted using MASIC software (<div class="dropBlock reference-citations"><a href="#bib93" id="back-bib93" data-db-target-for="bib93-d4222834e4749" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib93">Monroe et al., 2008</a><div data-db-target-of="bib93-d4222834e4749" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Monroe M.E. </li><li class="loa__item">Shaw J.L. </li><li class="loa__item">Daly D.S. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.<div class="popup-ref"><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Next, PSMs passing the confidence thresholds described above were linked to the extracted reporter ion intensities by scan number. The reporter ion intensities from different scans and different bRPLC fractions corresponding to the same gene were grouped. Relative protein abundance was calculated as the ratio of sample abundance to reference abundance using the summed reporter ion intensities from peptides that could be uniquely mapped to a gene. The pooled reference sample was labeled with TMT 126 reagent, allowing comparison of relative protein abundances across different TMT-11 plexes. The relative abundances were log<sub>2</sub> transformed and zero-centered for each gene to obtain final relative abundance values.</div><div class="section-paragraph">Small differences in laboratory conditions and sample handling can result in systematic, sample-specific bias in the quantification of protein levels. In order to mitigate these effects, we computed the median, log<sub>2</sub> relative protein abundance for each sample and re-centered to achieve a common median of 0.</div></section><section id="sec5.6.5.2" data-locator="sec5.6.5.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of phosphopeptides</span></h4><div class="section-paragraph">Phosphopeptide identification for the 132 phosphoproteomics data files were performed as in the global proteome data analysis described above (e.g., peptide level FDR &lt; 1%), with an additional dynamic phosphorylation (+79.9663 Da) on Ser, Thr, or Tyr residues. The phosphoproteome data were further processed by the Ascore algorithm (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d4222834e4765" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Beausoleil et al., 2006</a><div data-db-target-of="bib10-d4222834e4765" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beausoleil S.A. </li><li class="loa__item">Villén J. </li><li class="loa__item">Gerber S.A. </li><li class="loa__item">Rush J. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1084)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) for phosphorylation site localization, and the top-scoring sequences were reported. For phosphoproteomic datasets, the TMT-11 quantitative data were not summarized by protein but left at the phosphopeptide level. All peptides (phosphopeptides and global peptides) were labeled with TMT-11 reagent simultaneously. Separation into phospho- and non-phosphopeptides using IMAC was performed after the labeling. Thus, all the biases upstream of labeling are assumed to be identical between global and phosphoproteomic datasets. Therefore, to account for sample-specific biases in the phosphoproteome analysis, we applied the correction factors derived from median-centering the global proteomic dataset.</div></section><section id="sec5.6.5.3" data-locator="sec5.6.5.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Quantification of acetylated peptides</span></h4><div class="section-paragraph">Acetylated peptide identification for the 44 acetylome data files were performed as in the global proteome data analysis described above, with additional dynamic acetylation (+42.0105 Da) and carbamylation (+43.0058 Da) on Lys residues. The acetylation site localization, protein inference, and quantification of the acetylome data were performed in identical fashion as in the phosphoproteome data.</div></section><section id="sec5.6.5.4" data-locator="sec5.6.5.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Preprocessing of proteomics tables</span></h4><div class="section-paragraph">Due to the quantification of small values close to 0 on spectrum level, some extreme positive or negative values were generated after log<sub>2</sub> transform of relative protein/phosphopeptide/acetyl peptide abundance, which may have negative impact on the downstream analysis of the data sets. To identify TMT outliers with extreme values, we perform inter-TMT <em>t</em>-test for each individual protein/phosphopeptide/acetyl peptide. For a specific protein/phosphopeptide/acetyl peptide, relative abundance level of each TMT value was compared against all the other TMT values using Spearman two-sample test. Outlier was defined if the p-value passed a certain threshold. In the global proteome data, 153 TMT values were identified as outliers with inter-TMT <em>t</em>-test p-value lower than 10e-6, as a result 1,530 data points (0.14% of all observations) were removed from the data sets. In the phosphoproteome data, 379 TMT values were identified as outliers with inter-TMT <em>t</em>-test p-value lower than 10e-10, resulting in 3,790 data points (0.09% of all observations) removed from the data sets. In the acetylome data, 12 TMT values were identified as outliers with inter-TMT <em>t</em>-test p-value lower than 10e-14, and 120 data points (0.015% of all observations) were removed from the data sets.</div><div class="section-paragraph">Batch effects were checked using the log<sub>2</sub> relative protein/phosphopeptide abundance or protein/acetyl peptide abundance, and removed using Combat algorithm (<div class="dropBlock reference-citations"><a href="#bib10" id="back-bib10" data-db-target-for="bib10-d4222834e4794" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib10">Beausoleil et al., 2006</a><div data-db-target-of="bib10-d4222834e4794" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Beausoleil S.A. </li><li class="loa__item">Villén J. </li><li class="loa__item">Gerber S.A. </li><li class="loa__item">Rush J. </li><li class="loa__item">Gygi S.P. </li></ul><div class="reference-citations__detials">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1084)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) after TMT outlier filtering. Imputation was performed after batch effect correction to produce a different version of the data tables for some of the data analysis tools that are sensitive to missing values. The proteins/phosphopeptide/acetyl peptide with missing rate less than 50% were selected and imputed with the DreamAI algorithm <a class="inter-ref-link" href="https://github.com/WangLab-MSSM/DreamAI" target="_blank">https://github.com/WangLab-MSSM/DreamAI</a> tailored for proteomics data.</div></section><section id="sec5.6.5.5" data-locator="sec5.6.5.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample outlier identification of metabolome and lipidome</span></h4><div class="section-paragraph">A robust Mahalanobis distance based on biomolecule abundance vectors (rMd-PAV) was calculated to identify potential sample outliers in the data (<div class="dropBlock reference-citations"><a href="#bib88" id="back-bib88" data-db-target-for="bib88-d4222834e4805" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib88">Matzke et al., 2011</a><div data-db-target-of="bib88-d4222834e4805" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Matzke M.M. </li><li class="loa__item">Waters K.M. </li><li class="loa__item">Metz T.O. </li><li class="loa__item">Jacobs J.M. </li><li class="loa__item">Sims A.C. </li><li class="loa__item">Baric R.S. </li><li class="loa__item">Pounds J.G. </li><li class="loa__item">Webb-Robertson B.-J.M. </li></ul><div class="reference-citations__detials">Improved quality control processing of peptide-centric LC-MS proteomics data.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2866-2872</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtr479&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21852304&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-80053982558&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=2866-2872&journal=Bioinforma.+Oxf.+Engl.&author=Matzke+M.M.&author=Waters+K.M.&author=Metz+T.O.&author=Jacobs+J.M.&author=Sims+A.C.&author=Baric+R.S.&author=Pounds+J.G.&author=Webb-Robertson+B.-J.M.&title=Improved+quality+control+processing+of+peptide-centric+LC-MS+proteomics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). For proteomics data, this distance was calculated based on four metrics: average correlation with samples in the same group, skewness of biomolecule abundance distribution, the proportion of missing data, and median absolute deviation of abundances. These metrics, minus the proportion missing, were used for the metabolomics and lipidomics datasets. To confirm any sample outliers identified by rMd-PAV, a correlation heatmap was generated and sequential projection pursuit principal component analysis (PCA) was run (<div class="dropBlock reference-citations"><a href="#bib134" id="back-bib134" data-db-target-for="bib134-d4222834e4808" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib134">Webb-Robertson et al., 2013</a><div data-db-target-of="bib134-d4222834e4808" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Webb-Robertson B.-J.M. </li><li class="loa__item">Matzke M.M. </li><li class="loa__item">Metz T.O. </li><li class="loa__item">McDermott J.E. </li><li class="loa__item">Walker H. </li><li class="loa__item">Rodland K.D. </li><li class="loa__item">Pounds J.G. </li><li class="loa__item">Waters K.M. </li></ul><div class="reference-citations__detials">Sequential projection pursuit principal component analysis--dealing with missing data associated with new -omics technologies.<div class="popup-ref"><span class="ref__series"><i>BioTechniques.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">54</span></span>: <span class="ref__seriesPages">165-168</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.2144%2F000113978&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23477384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84876495055&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2013&pages=165-168&journal=BioTechniques&author=Webb-Robertson+B.-J.M.&author=Matzke+M.M.&author=Metz+T.O.&author=McDermott+J.E.&author=Walker+H.&author=Rodland+K.D.&author=Pounds+J.G.&author=Waters+K.M.&title=Sequential+projection+pursuit+principal+component+analysis%2D%2Ddealing+with+missing+data+associated+with+new+-omics+technologies">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). No sample outliers were identified in the proteomics dataset. One outlier, C3N-01366, was removed from the metabolomics dataset; C3N-01370 was removed from the positive lipid dataset and C3L-03968 from the negative lipid dataset.</div></section><section id="sec5.6.5.6" data-locator="sec5.6.5.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Normalization and protein quantification of metabolome and lipidome</span></h4><div class="section-paragraph">Global median centering, where each sample is normalized to the median of its observed values, was used to normalize all datasets. Protein quantification was accomplished via R-rollup (<div class="dropBlock reference-citations"><a href="#bib108" id="back-bib108" data-db-target-for="bib108-d4222834e4816" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib108">Polpitiya et al., 2008</a><div data-db-target-of="bib108-d4222834e4816" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Polpitiya A.D. </li><li class="loa__item">Qian W.-J. </li><li class="loa__item">Jaitly N. </li><li class="loa__item">Petyuk V.A. </li><li class="loa__item">Adkins J.N. </li><li class="loa__item">Camp D.G. </li><li class="loa__item">Anderson G.A. </li><li class="loa__item">Smith R.D. </li></ul><div class="reference-citations__detials">DAnTE: a statistical tool for quantitative analysis of -omics data.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1556-1558</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtn217&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18453552&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-46249116675&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=1556-1558&journal=Bioinforma.+Oxf.+Engl.&author=Polpitiya+A.D.&author=Qian+W.-J.&author=Jaitly+N.&author=Petyuk+V.A.&author=Adkins+J.N.&author=Camp+D.G.&author=Anderson+G.A.&author=Smith+R.D.&title=DAnTE%3A+a+statistical+tool+for+quantitative+analysis+of+-omics+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), in which peptides were scaled by a reference peptide and the protein abundance was set as the median of the scaled peptides.</div></section></section><section id="sec5.6.6" data-locator="sec5.6.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Other proteogenomic analysis</span></h4><section id="sec5.6.6.1" data-locator="sec5.6.6.1" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Sample labeling check across data types</span></h4><div class="section-paragraph">While multiple omics data enhance our understanding of complex molecular mechanisms underlying GBM, it is sometimes inevitable to have sample errors including sample swapping, shifting or data contamination. Working on error-containing data is dangerous since it could lead to a wrong scientific location. Therefore, it is required to confirm whether different types of molecular data are pertained from the same individuals prior to data integration or public sharing. For the GBM dataset, we checked sample labeling across different types of data as described previously (<div class="dropBlock reference-citations"><a href="#bib27" id="back-bib27" data-db-target-for="bib27-d4222834e4827" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib27">Clark et al., 2019</a><div data-db-target-of="bib27-d4222834e4827" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Clark D.J. </li><li class="loa__item">Dhanasekaran S.M. </li><li class="loa__item">Petralia F. </li><li class="loa__item">Pan J. </li><li class="loa__item">Song X. </li><li class="loa__item">Hu Y. </li><li class="loa__item">da Veiga Leprevost F. </li><li class="loa__item">Reva B. </li><li class="loa__item">Lih T.-S.M. </li><li class="loa__item">Chang H.-Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated proteogenomic characterization of clear cell renal cell carcinoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e31</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e31&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Using MODMatcher (<div class="dropBlock reference-citations"><a href="#bib141" id="back-bib141" data-db-target-for="bib141-d4222834e4830" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib141">Yoo et al., 2014</a><div data-db-target-of="bib141-d4222834e4830" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Yoo S. </li><li class="loa__item">Huang T. </li><li class="loa__item">Campbell J.D. </li><li class="loa__item">Lee E. </li><li class="loa__item">Tu Z. </li><li class="loa__item">Geraci M.W. </li><li class="loa__item">Powell C.A. </li><li class="loa__item">Schadt E.E. </li><li class="loa__item">Spira A. </li><li class="loa__item">Zhu J. </li></ul><div class="reference-citations__detials">MODMatcher: multi-omics data matcher for integrative genomic analysis.<div class="popup-ref"><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), we confirmed that all samples were well aligned among RNA-Seq, proteomics and CNV (WGS) data.</div></section><section id="sec5.6.6.2" data-locator="sec5.6.6.2" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Ancestry prediction using SNPs from 1000 genomes project</span></h4><div class="section-paragraph">We used a reference panel of genotypes and a clustering based on principal components to identify likely ancestry. We selected 107,765 coding SNPs with a minor allele frequency &gt; 0.02 from the final phase release of The 1000 Genomes Project (<div class="dropBlock reference-citations"><a href="#bib1" id="back-bib1" data-db-target-for="bib1-d4222834e4838" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib1">1000 Genomes Project Consortium et al., 2010</a><div data-db-target-of="bib1-d4222834e4838" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Abecasis G.R. </li><li class="loa__item">Altshuler D. </li><li class="loa__item">Auton A. </li><li class="loa__item">Brooks L.D. </li><li class="loa__item">Durbin R.M. </li><li class="loa__item">Gibbs R.A. </li><li class="loa__item">Hurles M.E. </li><li class="loa__item">McVean G.A. </li></ul><div class="reference-citations__detials"><span>1000 Genomes Project Consortium</span><br />A map of human genome variation from population-scale sequencing.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">467</span></span>: <span class="ref__seriesPages">1061-1073</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature09534&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20981092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-78049390036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5331)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2010&pages=1061-1073&journal=Nature&author=1000+Genomes+Project+Consortium&title=A+map+of+human+genome+variation+from+population-scale+sequencing">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). From this set of loci, we measured the depth and allele counts of each sample in our cohort using bam-readcount v0.8.0. Genotypes were then called for each sample based on the following criteria: 0/0 if reference count ≥ 8 and alternate count &lt; 4; 0/1 if reference count ≥ 4 and alternate count ≥ 4; 1/1 if reference count &lt; 4 and alternate count ≥ 8; and ./. (missing) otherwise. After excluding markers with missingness &gt; 5%, 70,968 markers were kept for analysis. We performed PCA on the 1000 Genomes samples to identify the top 20 principal components. We then projected our cohort onto the 20-dimensional space representing the 1000 Genomes data. We then trained a random forest classifier with the 1000 Genomes dataset using these 20 principal components. The 1000 Genomes dataset was split 80/20 for training and validation respectively. On the validation dataset our classifier achieved 99.6% accuracy. We then used the fitted classifier to predict the likely ancestry of our cohort.</div></section><section id="sec5.6.6.3" data-locator="sec5.6.6.3" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omics subtyping using non-negative matrix factorization (NMF)</span></h4><div class="section-paragraph">We selected the following proteogenomic features to the sample availability: CNV, bulk RNA, protein, and phosphoprotein expression. Due to limited sample amounts, not all tumors were analyzed for DNA methylome, metabolome, and miRNA. We used non-negative matrix factorization (NMF) implemented in the NMF R-package (<div class="dropBlock reference-citations"><a href="#bib46" id="back-bib46" data-db-target-for="bib46-d4222834e4846" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib46">Gaujoux and Seoighe, 2010</a><div data-db-target-of="bib46-d4222834e4846" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Gaujoux R. </li><li class="loa__item">Seoighe C. </li></ul><div class="reference-citations__detials">A flexible R package for nonnegative matrix factorization.<div class="popup-ref"><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (444)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to perform unsupervised clustering of tumor samples and to identify proteogenomic features that show characteristic expression patterns for each cluster. Briefly, given a factorization rank <em>k</em> (where <em>k</em> is the number of clusters), NMF decomposes a p × <em>n</em> data matrix <em>V</em> into two matrices <em>W</em> and <em>H</em> such that multiplication of <em>W</em> and <em>H</em> approximates <em>V</em>. Matrix <em>H</em> is a <em>k</em> × <em>n</em> matrix whose entries represent weights for each sample (1 to <em>N</em>) to contribute to each cluster (1 to <em>k</em>), whereas matrix <em>W</em> is a p × <em>k</em> matrix representing weights for each feature (1 to p) to contribute to each cluster (1 to <em>k</em>). Matrix <em>H</em> was used to assign samples to clusters by choosing the <em>k</em> with maximum score in each column of <em>H</em>. For each sample, we calculated a cluster membership score as the maximal fractional score of the corresponding column in matrix <em>H</em>. We defined a “cluster core” as the set of samples with cluster membership score &gt; 0.5. Matrix <em>W</em> containing the weights of each feature to a certain cluster was used to derive a list of representative features separating the clusters using the method proposed in (<div class="dropBlock reference-citations"><a href="#bib61" id="back-bib61" data-db-target-for="bib61-d4222834e4897" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib61">Kim and Park, 2007</a><div data-db-target-of="bib61-d4222834e4897" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Kim H. </li><li class="loa__item">Park H. </li></ul><div class="reference-citations__detials">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinforma.+Oxf.+Engl.&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">To enable integrative multi-omics clustering we enforced all data types (and converted if necessary) to represent log<sub>2</sub>-ratios to either a common reference measured in each TMT plex (proteome, phosphoproteome), an <em>in silico</em> common reference calculated as the median abundance across all samples (RNA gene expression) or to gene copy numbers relative to matching normal blood sample (CNV). All data tables were then concatenated and all rows containing missing values were removed. To remove uninformative features from the dataset prior to NMF clustering, we removed features with the lowest standard deviation (bottom 5<sup>th</sup> percentile) across all samples. Each row in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores.</div><div class="section-paragraph">Since NMF requires a non-negative input matrix we converted the z-scores in the data matrix into a non-negative matrix as follows:<ul class="ce-list--remove-bullets" id="olist0020"><li class="ce-list--remove-bullets__list-item" id="o0070"><span class="label">1)</span><div class="ce-list--remove-bullets__list-item__text">Create one data matrix with all negative numbers zeroed</div></li><li class="ce-list--remove-bullets__list-item" id="o0075"><span class="label">2)</span><div class="ce-list--remove-bullets__list-item__text">Create another data matrix with all positive numbers zeroed and the signs of all negative numbers removed</div></li><li class="ce-list--remove-bullets__list-item" id="o0080"><span class="label">3)</span><div class="ce-list--remove-bullets__list-item__text">Concatenate both matrices resulting in a data matrix twice as large as the original, but with positive values only and zeros and hence appropriate for NMF</div></li></ul></div><div class="section-paragraph">The resulting matrix was then passed to NMF analysis in R using the factorization method described in (<div class="dropBlock reference-citations"><a href="#bib19" id="back-bib19" data-db-target-for="bib19-d4222834e4927" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib19">Brunet et al., 2004</a><div data-db-target-of="bib19-d4222834e4927" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brunet J.-P. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Golub T.R. </li><li class="loa__item">Mesirov J.P. </li></ul><div class="reference-citations__detials">Metagenes and molecular pattern discovery using matrix factorization.<div class="popup-ref"><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1043)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). To determine the optimal factorization rank <em>k</em> (number of clusters) for the multi-omic data matrix, we tested a range of clusters between <em>k</em> = 2 and 8. For each <em>k</em>, we factorized matrix <em>V</em> using 50 iterations with random initializations of <em>W</em> and <em>H</em>. To determine the optimal factorization rank, we calculated cophenetic correlation coefficients to measure how well the intrinsic structure of the data is recapitulated after clustering. Finally, we picked the <em>k</em> with maximal cophenetic correlation for cluster numbers between <em>k</em> = 3 and 8.</div><div class="section-paragraph">To achieve robust factorization of the multi-omics data matrix <em>V</em>, we took the optimal factorization rank <em>k</em>, repeated the NMF analysis for 200 iterations with random initializations of <em>W</em> and <em>H</em>, and partitioned the samples into clusters as described above. Due to the non-negative transformation of the z-scored data matrix, feature weight matrix <em>W</em> contained two separate weights for positive and negative z-scores of each feature, respectively. To revert the non-negative transformation and to derive a single signed weight for each feature, we first normalized each row in matrix <em>W</em> by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score the initial data matrix. Thus, the resulting transformed matrix <em>W</em><sub>signed</sub> contained signed cluster weights for each feature in the input matrix.</div><div class="section-paragraph">For each cluster, we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights <em>W</em><sub>signed</sub> onto hallmark pathway gene sets (<div class="dropBlock reference-citations"><a href="#bib78" id="back-bib78" data-db-target-for="bib78-d4222834e4972" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib78">Liberzon et al., 2015</a><div data-db-target-of="bib78-d4222834e4972" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Liberzon A. </li><li class="loa__item">Birger C. </li><li class="loa__item">Thorvaldsdóttir H. </li><li class="loa__item">Ghandi M. </li><li class="loa__item">Mesirov J.P. </li><li class="loa__item">Tamayo P. </li></ul><div class="reference-citations__detials">The Molecular Signatures Database (MSigDB) hallmark gene set collection.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1644)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) using ssGSEA (<div class="dropBlock reference-citations"><a href="#bib9" id="back-bib9" data-db-target-for="bib9-d4222834e4975" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib9">Barbie et al., 2009</a><div data-db-target-of="bib9-d4222834e4975" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Barbie D.A. </li><li class="loa__item">Tamayo P. </li><li class="loa__item">Boehm J.S. </li><li class="loa__item">Kim S.Y. </li><li class="loa__item">Moody S.E. </li><li class="loa__item">Dunn I.F. </li><li class="loa__item">Schinzel A.C. </li><li class="loa__item">Sandy P. </li><li class="loa__item">Meylan E. </li><li class="loa__item">Scholl C. </li><li>et al.</li></ul><div class="reference-citations__detials">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1164)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) available on <a class="inter-ref-link" href="https://github.com/broadinstitute/ssGSEA2.0" target="_blank">https://github.com/broadinstitute/ssGSEA2.0</a> (parameters: gene.set.database="h.all.v6.2.symbols.gmt" sample.norm.type="rank" weight=1 statistic="area.under.RES" output.score.type="NES" nperm=1000 global.fdr=TRUE min.overlap=5 correl.type="z.score"). To derive a single weight for each gene measured across all omics data types, we retained the weight with maximal absolute amplitude. We then associated the resulting clusters to sample-level variables by testing for overrepresentation in the cluster core sample sets using Fisher’s exact test. The following clinical variables were used: expression subtype, sex, vital status, and smoking history.</div><div class="section-paragraph">The entire NMF workflow has been implemented as a module on Broad's Cloud platform Terra (<a class="inter-ref-link" href="https://app.terra.bio/" target="_blank">https://app.terra.bio/</a>). The docker containers encapsulating the source code and the required R packages for NMF clustering and ssGSEA were available on Dockerhub (broadcptac/pgdac_mo_nmf:9, broadcptac/pgdac_ssgsea:5).</div></section><section id="sec5.6.6.4" data-locator="sec5.6.6.4" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Expression based TCGA subtyping</span></h4><div class="section-paragraph">Gene expression based subtypes were based on the 150 genes created by Wang et al., the most recent TCGA subtyping effort (<div class="dropBlock reference-citations"><a href="#bib133" id="back-bib133" data-db-target-for="bib133-d4222834e4991" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib133">Wang et al., 2017</a><div data-db-target-of="bib133-d4222834e4991" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wang Q. </li><li class="loa__item">Hu B. </li><li class="loa__item">Hu X. </li><li class="loa__item">Kim H. </li><li class="loa__item">Squatrito M. </li><li class="loa__item">Scarpace L. </li><li class="loa__item">deCarvalho A.C. </li><li class="loa__item">Lyu S. </li><li class="loa__item">Li P. </li><li class="loa__item">Li Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">42-56.e6</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28697342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85027444208&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=42-56.e6&journal=Cancer+Cell&author=Wang+Q.&author=Hu+B.&author=Hu+X.&author=Kim+H.&author=Squatrito+M.&author=Scarpace+L.&author=deCarvalho+A.C.&author=Lyu+S.&author=Li+P.&author=Li+Y.&title=Tumor+evolution+of+glioma-intrinsic+gene+expression+subtypes+associates+with+immunological+changes+in+the+microenvironment">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which contained 50 highly expressed genes in classical, proneural, and mesenchymal IDH WT tumors. Tumors with recurrent mutations in IDH1/2 (IDH1 R132H specifically in our cohort) were assigned to be IDH mutant tumors. We then performed consensus clustering on all tumors based on the selected gene expression in log<sub>2</sub>(FPKM-UQ + 1) using ConsensusClusterPlus R package (parameters: maxK = 10 reps = 2000 pItem = 0.8 pFeature = 1 clusterAlg = "hc" distance = "pearson" seed = 201909). We chose the total number of clusters k = 5 based on the delta area plot of consensus CDF. The clusters were annotated with the TCGA subtypes based on their gene expression profiles. Three clusters (r1, r4, and r5) were merged due to their similar expression signature, which was identical to the clustering result while choosing k = 3.</div></section><section id="sec5.6.6.5" data-locator="sec5.6.6.5" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unsupervised clustering of DNA methylation</span></h4><div class="section-paragraph">Methylation subtypes were segregated based on the top 8,000 most variable probes using k-means consensus clustering as previously described (<div class="dropBlock reference-citations"><a href="#bib121" id="back-bib121" data-db-target-for="bib121-d4222834e5002" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib121">Sturm et al., 2012</a><div data-db-target-of="bib121-d4222834e5002" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sturm D. </li><li class="loa__item">Witt H. </li><li class="loa__item">Hovestadt V. </li><li class="loa__item">Khuong-Quang D.-A. </li><li class="loa__item">Jones D.T.W. </li><li class="loa__item">Konermann C. </li><li class="loa__item">Pfaff E. </li><li class="loa__item">Tönjes M. </li><li class="loa__item">Sill M. </li><li class="loa__item">Bender S. </li><li>et al.</li></ul><div class="reference-citations__detials">Hotspot mutations in H3F3A and IDH1 define distinct epigenetic and biological subgroups of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">425-437</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23079654&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84867606428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1026)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=425-437&journal=Cancer+Cell&author=Sturm+D.&author=Witt+H.&author=Hovestadt+V.&author=Khuong-Quang+D.-A.&author=Jones+D.T.W.&author=Konermann+C.&author=Pfaff+E.&author=T%C3%B6njes+M.&author=Sill+M.&author=Bender+S.&title=Hotspot+mutations+in+H3F3A+and+IDH1+define+distinct+epigenetic+and+biological+subgroups+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We first removed underperforming probes (<div class="dropBlock reference-citations"><a href="#bib144" id="back-bib144" data-db-target-for="bib144-d4222834e5005" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib144">Zhou et al., 2017</a><div data-db-target-of="bib144-d4222834e5005" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Zhou W. </li><li class="loa__item">Laird P.W. </li><li class="loa__item">Shen H. </li></ul><div class="reference-citations__detials">Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">e22</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27924034&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=e22&journal=Nucleic+Acids+Res.&author=Zhou+W.&author=Laird+P.W.&author=Shen+H.&title=Comprehensive+characterization%2C+annotation+and+innovative+use+of+Infinium+DNA+methylation+BeadChip+probes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and then the samples with more than 30% missing values. Remaining missing values were imputed using the mean of the corresponding probe value. We then performed clustering 1000 times using the ConsensusClusterPlus R package (parameters: maxK = 10 reps = 1000 pItem = 0.8 pFeature = 1 clusterAlg = "km" distance = "euclidean"). We choose k = 6 based on the delta area plot of consensus CDF.</div></section><section id="sec5.6.6.6" data-locator="sec5.6.6.6" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">MolecularNeuroPathology (MNP) DNA Methylation Classification of Central Nervous System (CNS) Tumors</span></h4><div class="section-paragraph">We applied the existing DNA methylation classification of CNS tumors developed by the MolecularNeuroPathology group (<div class="dropBlock reference-citations"><a href="#bib22" id="back-bib22" data-db-target-for="bib22-d4222834e5013" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib22">Capper et al., 2018</a><div data-db-target-of="bib22-d4222834e5013" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Capper D. </li><li class="loa__item">Jones D.T.W. </li><li class="loa__item">Sill M. </li><li class="loa__item">Hovestadt V. </li><li class="loa__item">Schrimpf D. </li><li class="loa__item">Sturm D. </li><li class="loa__item">Koelsche C. </li><li class="loa__item">Sahm F. </li><li class="loa__item">Chavez L. </li><li class="loa__item">Reuss D.E. </li><li>et al.</li></ul><div class="reference-citations__detials">DNA methylation-based classification of central nervous system tumours.<div class="popup-ref"><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">555</span></span>: <span class="ref__seriesPages">469-474</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature26000&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29539639&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044262739&amp;cf=" onclick="newWindow(this.href);return false">Scopus (573)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=555&publication_year=2018&pages=469-474&journal=Nature&author=Capper+D.&author=Jones+D.T.W.&author=Sill+M.&author=Hovestadt+V.&author=Schrimpf+D.&author=Sturm+D.&author=Koelsche+C.&author=Sahm+F.&author=Chavez+L.&author=Reuss+D.E.&title=DNA+methylation-based+classification+of+central+nervous+system+tumours">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to our cohort. The processed microarray beta values and the classification of the MNP cohort (v11b2) were downloaded from the GEO dataset <a class="inter-ref-link" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90496" target="_blank">GSE90496</a> (the MNP reference set) and the supplemental tables of the original publication, consisting of total 600 GBM tumors and control samples. We included probes from CPTAC samples with &lt; 20% of missing values across all samples. Remaining missing values were imputed using the mean of the corresponding probe value. Top 10,000 variable probes out of the total 375,969 shared probes across MNP and CPTAC GBM samples were selected to construct the shared DNA methylome space. We used PCA (parameters: random_state = 202012) to extract the top 30 principal components and assigned the MNP classification to CPTAC samples using nearest neighbor implemented by scikit-learn (KNeighborsClassifier with parameters: k = 9, algorithm='brute'). While an official MNP DNA methylation classifier exists online (<a class="inter-ref-link" href="https://www.molecularneuropathology.org/mnp" target="_blank">https://www.molecularneuropathology.org/mnp</a>), we were not able to access it since our registration was not approved by the site at the time of writing.</div></section><section id="sec5.6.6.7" data-locator="sec5.6.6.7" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unsupervised clustering of miRNA expression</span></h4><div class="section-paragraph">Unsupervised miRNA expression subtype identification was performed on mature miRNAs expression (log<sub>2</sub> TPM) from 98 tumors with miRNA-seq available using Louvain clustering (<div class="dropBlock reference-citations"><a href="#bib15" id="back-bib15" data-db-target-for="bib15-d4222834e5030" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib15">Blondel et al., 2008</a><div data-db-target-of="bib15-d4222834e5030" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blondel V.D. </li><li class="loa__item">Guillaume J.-L. </li><li class="loa__item">Lambiotte R. </li><li class="loa__item">Lefebvre E. </li></ul><div class="reference-citations__detials">Fast unfolding of communities in large networks.<div class="popup-ref"><span class="ref__series"><i>J. Stat. Mech. Theor. Exp.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">2008</span></span>: <span class="ref__seriesPages">P10008</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1088%2F1742-5468%2F2008%2F10%2FP10008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-56349094785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7916)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2008&publication_year=2008&pages=P10008&journal=J.%C2%A0Stat.+Mech.+Theor.+Exp.&author=Blondel+V.D.&author=Guillaume+J.-L.&author=Lambiotte+R.&author=Lefebvre+E.&title=Fast+unfolding+of+communities+in+large+networks">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) implemented in louvain-igraph v0.6.1. Top 50 differentially expressed miRNAs from each miRNA-based subtype were selected (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" class="scroll-into-link">Table S3</a>).</div></section><section id="sec5.6.6.8" data-locator="sec5.6.6.8" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Unsupervised clustering of individual data type</span></h4><div class="section-paragraph">We also applied clustering across all tumors using individual data types including RNA, protein, phosphoprotein, acetylprotein, lipids, and metabolites. The DreamAI-imputed expression values were used for protein, phosphoprotein and acetylprotein. For other data types, features with too many missing values were discarded (RNA: 40%; metabolites: 80% of all tumors). We then selected the top 95% most variable remaining features to perform consensus clustering using ConsensusClusterPlus R package (parameters: maxK = 10 reps = 1000 pItem = 0.8 pFeature = 1 clusterAlg = "km" distance = "euclidean" seed = 201909). Optimal number of k was selected based on the delta area plot of consensus CDF. The resulting clusters per data type were shown in <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1" class="scroll-into-link">Figure S2</a>F.</div></section><section id="sec5.6.6.9" data-locator="sec5.6.6.9" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Determination of stemness score</span></h4><div class="section-paragraph">Stemness scores were calculated as previously described (<div class="dropBlock reference-citations"><a href="#bib87" id="back-bib87" data-db-target-for="bib87-d4222834e5049" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib87">Malta et al., 2018</a><div data-db-target-of="bib87-d4222834e5049" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Malta T.M. </li><li class="loa__item">Sokolov A. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Burzykowski T. </li><li class="loa__item">Poisson L. </li><li class="loa__item">Weinstein J.N. </li><li class="loa__item">Kamińska B. </li><li class="loa__item">Huelsken J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Gevaert O. </li><li>et al.</li></ul><div class="reference-citations__detials">Machine learning identifies stemness features associated with oncogenic dedifferentiation.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e15</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (258)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e15&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kami%C5%84ska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+learning+identifies+stemness+features+associated+with+oncogenic+dedifferentiation">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Firstly, we used MoonlightR (<div class="dropBlock reference-citations"><a href="#bib29" id="back-bib29" data-db-target-for="bib29-d4222834e5052" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib29">Colaprico et al., 2020</a><div data-db-target-of="bib29-d4222834e5052" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Bailey M.H. </li><li class="loa__item">Odom G.J. </li><li class="loa__item">Terkelsen T. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen A.V. </li><li class="loa__item">Cantini L. </li><li class="loa__item">Zinovyev A. </li><li class="loa__item">Barillot E. </li><li>et al.</li></ul><div class="reference-citations__detials">Interpreting pathways to discover cancer driver genes with Moonlight.<div class="popup-ref"><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to query, download, and preprocess the pluripotent stem cell samples (ESC and iPSC) from the Progenitor Cell Biology Consortium (PCBC) dataset (<div class="dropBlock reference-citations"><a href="#bib31" id="back-bib31" data-db-target-for="bib31-d4222834e5055" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib31">Daily et al., 2017</a><div data-db-target-of="bib31-d4222834e5055" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Daily K. </li><li class="loa__item">Ho Sui S.J. </li><li class="loa__item">Schriml L.M. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Salomonis N. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li class="loa__item">Lotia S. </li><li>et al.</li></ul><div class="reference-citations__detials">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.<div class="popup-ref"><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib114" id="back-bib114" data-db-target-for="bib114-d4222834e5058" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib114">Salomonis et al., 2016</a><div data-db-target-of="bib114-d4222834e5058" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Salomonis N. </li><li class="loa__item">Dexheimer P.J. </li><li class="loa__item">Omberg L. </li><li class="loa__item">Schroll R. </li><li class="loa__item">Bush S. </li><li class="loa__item">Huo J. </li><li class="loa__item">Schriml L. </li><li class="loa__item">Ho Sui S. </li><li class="loa__item">Keddache M. </li><li class="loa__item">Mayhew C. </li><li>et al.</li></ul><div class="reference-citations__detials">Integrated genomic analysis of diverse induced pluripotent stem cells from the progenitor cell biology consortium.<div class="popup-ref"><span class="ref__series"><i>Stem Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Rep.&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+genomic+analysis+of+diverse+induced+pluripotent+stem+cells+from+the+progenitor+cell+biology+consortium">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Secondly, to calculate the stemness scores based on mRNA expression, we built a predictive model using one-class logistic regression (OCLR) (<div class="dropBlock reference-citations"><a href="#bib116" id="back-bib116" data-db-target-for="bib116-d4222834e5061" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib116">Sokolov et al., 2016</a><div data-db-target-of="bib116-d4222834e5061" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Sokolov A. </li><li class="loa__item">Paull E.O. </li><li class="loa__item">Stuart J.M. </li></ul><div class="reference-citations__detials">One-class detection of cell states in tumor subtypes.<div class="popup-ref"><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-class+detection+of+cell+states+in+tumor+subtypes">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) on Progenitor Cell Biology Consortium (PCBC) dataset. For mRNA expression-based signatures, to ensure compatibility with our cohort, we first mapped the Ensembl IDs to Human Genome Organization (HUGO) gene names and dropped any genes that had no such mapping. The resulting training matrix contained 12,945 mRNA expression values measured across all available PCBC samples. To calculate mRNA-based sternness index (mRNASi), we used FPKM-UQ mRNA expression values for all CPTAC GBM tumors and GTEx samples. We used TCGAanalyze_Stemness function from the R package TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d4222834e5065" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Colaprico et al., 2016</a><div data-db-target-of="bib28-d4222834e5065" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and following our previously described workflow (<div class="dropBlock reference-citations"><a href="#bib115" id="back-bib115" data-db-target-for="bib115-d4222834e5068" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib115">Silva et al., 2016</a><div data-db-target-of="bib115-d4222834e5068" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Silva T.C. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Olsen C. </li><li class="loa__item">D’Angelo F. </li><li class="loa__item">Bontempi G. </li><li class="loa__item">Ceccarelli M. </li><li class="loa__item">Noushmehr H. </li></ul><div class="reference-citations__detials">TCGA Workflow: analyze cancer genomics and epigenomics data using Bioconductor packages.<div class="popup-ref"><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1542</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.12688%2Ff1000research.8923.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28232861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85013753091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1542&journal=F1000Res.&author=Silva+T.C.&author=Colaprico+A.&author=Olsen+C.&author=D%E2%80%99Angelo+F.&author=Bontempi+G.&author=Ceccarelli+M.&author=Noushmehr+H.&title=TCGA+Workflow%3A+analyze+cancer+genomics+and+epigenomics+data+using+Bioconductor+packages">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), with "stemSig" argument set to PCBC stemSig.</div></section><section id="sec5.6.6.10" data-locator="sec5.6.6.10" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Multi-omics cis association analysis using iProFun</span></h4><div class="section-paragraph">We integrated somatic mutation, CNV, DNA methylation, RNA, protein, phosphorylation (phospho) and acetylation (acetyl) levels via iProFun (<div class="dropBlock reference-citations"><a href="#bib117" id="back-bib117" data-db-target-for="bib117-d4222834e5076" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib117">Song et al., 2019</a><div data-db-target-of="bib117-d4222834e5076" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Song X. </li><li class="loa__item">Ji J. </li><li class="loa__item">Gleason K.J. </li><li class="loa__item">Yang F. </li><li class="loa__item">Martignetti J.A. </li><li class="loa__item">Chen L.S. </li><li class="loa__item">Wang P. </li></ul><div class="reference-citations__detials">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.<div class="popup-ref"><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to investigate the functional impacts of DNA alterations in GBM. All data types were preprocessed to eliminate potential issues for analysis such as batch effects, missing data and major unmeasured confounding effects before the iProFun analysis. As phosphoprotein and acetylprotein were measured in a small subset of the genes in comparison with RNA and protein, we considered three sets of iProFun analysis using different combination functional outcomes (mRNA/protein, mRNA/protein/phospho, and mRNA/protein/acetyl) to include as many as possible genes and omics for investigation. For each set of outcomes (e.g. RNA and protein), we considered their levels perturbed jointly by three DNA alterations (somatic mutation, CNV, and DNA methylation). The effects of DNA methylation on molecular traits are usually smaller than mutation and CNV, and thus adjusting their effects in analysis is critical to obtain unconfounded associations for methylation. In addition, we adjusted age, sex, and tumor purity in the analysis. Tumor purity was determined using xCell (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d4222834e5079" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Aran et al., 2017</a><div data-db-target-of="bib3-d4222834e5079" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) from RNA-Seq data.</div><div class="section-paragraph">The iProFun procedure was applied to a total of 7,464 genes with measured RNA/protein, 4,433 genes with measured RNA/protein/phospho, and 1,315 genes with measured RNA/protein/acetyl data, respectively, for their <em>cis</em> regulatory patterns in tumors. For example, when we considered DNA methylation for its effects on RNA/protein/phospho, we started with the traditional linear regression for each of the three outcomes separately:<div class="labeled-equation clearfix"><div id="ufd1" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si1.gif"><mrow><mi>R</mi><mi>N</mi><mi>A</mi><mspace width="0.25em"></mspace><mo>∼</mo><mspace width="0.25em"></mspace><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo linebreak="goodbreak">+</mo><mi>c</mi><mi>o</mi><mi>v</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>s</mi></mrow></math></div></div><br></br><div class="labeled-equation clearfix"><div id="ufd2" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si2.gif"><mrow><mi>p</mi><mi>r</mi><mi>o</mi><mi>t</mi><mi>e</mi><mi>i</mi><mi>n</mi><mspace width="0.25em"></mspace><mo>∼</mo><mspace width="0.25em"></mspace><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo linebreak="goodbreak">+</mo><mi>c</mi><mi>o</mi><mi>v</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>s</mi></mrow></math></div></div><br></br><div class="labeled-equation clearfix"><div id="ufd3" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si3.gif"><mrow><mi>p</mi><mi>h</mi><mi>o</mi><mi>s</mi><mi>p</mi><mi>h</mi><mi>o</mi><mspace width="0.25em"></mspace><mo>∼</mo><mspace width="0.25em"></mspace><mi>m</mi><mi>e</mi><mi>t</mi><mi>h</mi><mi>y</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo linebreak="goodbreak">+</mo><mi>c</mi><mi>o</mi><mi>v</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>s</mi></mrow></math></div></div><br></br></div><div class="section-paragraph">The covariates here include CNV, somatic mutations (genes with mutation rate ≥ 10%), age, sex, and tumor purity. Then iProFun took the association summary statistics from these three regressions as input to call posterior probabilities of belonging to each of the eight possible configurations (e.g., “None”, “RNA only”, “protein only”, “phospho only” “RNA &amp; protein”, “RNA &amp; phospho”, “protein &amp; phospho” and “all three”) and to determine significance associations.</div><div class="section-paragraph">A gene was identified to present significant and biologically meaningful association if the association passes three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities &gt; 75%, and (3) empirical false discovery rate (eFDR) &lt; 10%. Specifically, the biological filtering criterion requires that CNV presents positive associations with all the types of molecular quantitative traits (QTs), DNA methylation presents negative associations with all the types of molecular QTs, and mutation requires the association across all outcome platforms preserve consistent directions (either positive or negative). Secondly, a significance was called only if the posterior probabilities &gt; 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. For example, the posterior probability of a methylation being associated with mRNA expression levels was obtained by summing up the posterior probabilities in the following four association patterns – “RNA only”, “RNA &amp; protein”, “RNA &amp; phospho” and “all three”, all of which were consistent with methylation being associated with mRNA expression. Lastly, we calculated the empirical FDR (eFDR) via 100 permutations per molecular QTs by shuffling the label of the molecular QTs and required eFDR &lt; 10% by selecting a minimal cutoff value of α that 75% &lt; α &lt; 100%. The eFDR is calculated by:<div class="labeled-equation clearfix"><div id="ufd4" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si4.gif"><mrow><mi>e</mi><mi>F</mi><mi>D</mi><mi>R</mi><mo linebreak="badbreak">=</mo><mfrac><mrow><mi>A</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.25em"></mspace><mo>#</mo><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi><mi>s</mi><mspace width="0.25em"></mspace><mi>w</mi><mi>i</mi><mi>t</mi><mi>h</mi><mspace width="0.25em"></mspace><mi>p</mi><mi>o</mi><mi>s</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>o</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>p</mi><mi>r</mi><mi>o</mi><mi>b</mi><mi>a</mi><mi>b</mi><mi>i</mi><mi>l</mi><mi>i</mi><mi>t</mi><mi>y</mi><mspace width="0.25em"></mspace><mo linebreak="badbreak">&gt;</mo><mspace width="0.25em"></mspace><mi>α</mi><mspace width="0.25em"></mspace><mi>i</mi><mi>n</mi><mspace width="0.25em"></mspace><mi>p</mi><mi>e</mi><mi>r</mi><mi>m</mi><mi>u</mi><mi>t</mi><mi>a</mi><mi>t</mi><mi>e</mi><mi>d</mi><mspace width="0.25em"></mspace><mi>d</mi><mi>a</mi><mi>t</mi><mi>a</mi></mrow><mrow><mi>A</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.25em"></mspace><mo>#</mo><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi><mi>s</mi><mspace width="0.25em"></mspace><mi>w</mi><mi>i</mi><mi>t</mi><mi>h</mi><mspace width="0.25em"></mspace><mi>p</mi><mi>o</mi><mi>s</mi><mi>t</mi><mi>e</mi><mi>r</mi><mi>i</mi><mi>o</mi><mi>r</mi><mspace width="0.25em"></mspace><mi>p</mi><mi>r</mi><mi>o</mi><mi>b</mi><mi>a</mi><mi>b</mi><mi>i</mi><mi>l</mi><mi>i</mi><mi>t</mi><mi>y</mi><mspace width="0.25em"></mspace><mo linebreak="badbreak">&gt;</mo><mspace width="0.25em"></mspace><mi>α</mi><mspace width="0.25em"></mspace><mi>i</mi><mi>n</mi><mspace width="0.25em"></mspace><mi>o</mi><mi>r</mi><mi>i</mi><mi>g</mi><mi>i</mi><mi>n</mi><mi>a</mi><mi>l</mi><mspace width="0.25em"></mspace><mi>d</mi><mi>a</mi><mi>t</mi><mi>a</mi></mrow></mfrac></mrow></math></div></div><br></br></div><div class="section-paragraph"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" class="scroll-into-link">Table S3</a> presents the results of whether the DNA methylation/CNV/mutation of a gene has perturbed any of its cis QTs (mRNA, protein, phosphoprotein and acetylprotein).</div></section><section id="sec5.6.6.11" data-locator="sec5.6.6.11" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Mutation impact on the RNA, proteome, phosphoproteome, lipidome and metabolome</span></h4><div class="section-paragraph">We aggregated a set of interacting proteins (e.g. kinase/phosphatase-substrate or complex partners) from OmniPath (downloaded on 2018-03-29) (<div class="dropBlock reference-citations"><a href="#bib128" id="back-bib128" data-db-target-for="bib128-d4222834e5557" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib128">Türei et al., 2016</a><div data-db-target-of="bib128-d4222834e5557" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Türei D. </li><li class="loa__item">Korcsmáros T. </li><li class="loa__item">Saez-Rodriguez J. </li></ul><div class="reference-citations__detials">OmniPath: guidelines and gateway for literature-curated signaling pathway resources.<div class="popup-ref"><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">966-967</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.4077&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27898060&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84999106855&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=966-967&journal=Nat.+Methods&author=T%C3%BCrei+D.&author=Korcsm%C3%A1ros+T.&author=Saez-Rodriguez+J.&title=OmniPath%3A+guidelines+and+gateway+for+literature-curated+signaling+pathway+resources">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), DEPOD (downloaded on 2018-03-29) (<div class="dropBlock reference-citations"><a href="#bib36" id="back-bib36" data-db-target-for="bib36-d4222834e5560" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib36">Duan et al., 2015</a><div data-db-target-of="bib36-d4222834e5560" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Duan G. </li><li class="loa__item">Li X. </li><li class="loa__item">Köhn M. </li></ul><div class="reference-citations__detials">The human DEPhOsphorylation database DEPOD: a 2015 update.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D531-D535</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku1009&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25332398&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84941037753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D531-D535&journal=Nucleic+Acids+Res.&author=Duan+G.&author=Li+X.&author=K%C3%B6hn+M.&title=The+human+DEPhOsphorylation+database+DEPOD%3A+a+2015+update">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), CORUM (downloaded on 2018-06-29) (<div class="dropBlock reference-citations"><a href="#bib113" id="back-bib113" data-db-target-for="bib113-d4222834e5563" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib113">Ruepp et al., 2010</a><div data-db-target-of="bib113-d4222834e5563" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Ruepp A. </li><li class="loa__item">Waegele B. </li><li class="loa__item">Lechner M. </li><li class="loa__item">Brauner B. </li><li class="loa__item">Dunger-Kaltenbach I. </li><li class="loa__item">Fobo G. </li><li class="loa__item">Frishman G. </li><li class="loa__item">Montrone C. </li><li class="loa__item">Mewes H.-W. </li></ul><div class="reference-citations__detials">CORUM: the comprehensive resource of mammalian protein complexes--2009.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">D497-D501</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkp914&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19884131&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75549091794&amp;cf=" onclick="newWindow(this.href);return false">Scopus (500)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D497-D501&journal=Nucleic+Acids+Res.&author=Ruepp+A.&author=Waegele+B.&author=Lechner+M.&author=Brauner+B.&author=Dunger-Kaltenbach+I.&author=Fobo+G.&author=Frishman+G.&author=Montrone+C.&author=Mewes+H.-W.&title=CORUM%3A+the+comprehensive+resource+of+mammalian+protein+complexes%2D%2D2009">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), Signor2 (downloaded on 2018-10-29) (<div class="dropBlock reference-citations"><a href="#bib103" id="back-bib103" data-db-target-for="bib103-d4222834e5566" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib103">Perfetto et al., 2016</a><div data-db-target-of="bib103-d4222834e5566" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Perfetto L. </li><li class="loa__item">Briganti L. </li><li class="loa__item">Calderone A. </li><li class="loa__item">Cerquone Perpetuini A. </li><li class="loa__item">Iannuccelli M. </li><li class="loa__item">Langone F. </li><li class="loa__item">Licata L. </li><li class="loa__item">Marinkovic M. </li><li class="loa__item">Mattioni A. </li><li class="loa__item">Pavlidou T. </li><li>et al.</li></ul><div class="reference-citations__detials">SIGNOR: a database of causal relationships between biological entities.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D548-D554</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1048&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26467481&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84976871607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D548-D554&journal=Nucleic+Acids+Res.&author=Perfetto+L.&author=Briganti+L.&author=Calderone+A.&author=Cerquone+Perpetuini+A.&author=Iannuccelli+M.&author=Langone+F.&author=Licata+L.&author=Marinkovic+M.&author=Mattioni+A.&author=Pavlidou+T.&title=SIGNOR%3A+a+database+of+causal+relationships+between+biological+entities">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and Reactome (downloaded on 2018-11-01) (<div class="dropBlock reference-citations"><a href="#bib40" id="back-bib40" data-db-target-for="bib40-d4222834e5569" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib40">Fabregat et al., 2018</a><div data-db-target-of="bib40-d4222834e5569" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Fabregat A. </li><li class="loa__item">Jupe S. </li><li class="loa__item">Matthews L. </li><li class="loa__item">Sidiropoulos K. </li><li class="loa__item">Gillespie M. </li><li class="loa__item">Garapati P. </li><li class="loa__item">Haw R. </li><li class="loa__item">Jassal B. </li><li class="loa__item">Korninger F. </li><li class="loa__item">May B. </li><li>et al.</li></ul><div class="reference-citations__detials">The reactome pathway Knowledgebase.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (926)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+reactome+pathway+Knowledgebase">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). We focused our analyses on 18 GBM SMGs previously reported in the literature: <em>PIK3R1, PIK3CA, PTEN, RB1, TP53, EGFR, IDH1, BRAF, NF1, PDGFRA, ATRX,</em> and <em>TERT</em>p (<div class="dropBlock reference-citations"><a href="#bib8" id="back-bib8" data-db-target-for="bib8-d4222834e5577" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib8">Bailey et al., 2018</a><div data-db-target-of="bib8-d4222834e5577" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Bailey M.H. </li><li class="loa__item">Tokheim C. </li><li class="loa__item">Porta-Pardo E. </li><li class="loa__item">Sengupta S. </li><li class="loa__item">Bertrand D. </li><li class="loa__item">Weerasinghe A. </li><li class="loa__item">Colaprico A. </li><li class="loa__item">Wendl M.C. </li><li class="loa__item">Kim J. </li><li class="loa__item">Reardon B. </li><li>et al.</li></ul><div class="reference-citations__detials">Comprehensive characterization of cancer driver genes and mutations.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">371-385.e18</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29625053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044623848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (506)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=371-385.e18&journal=Cell&author=Bailey+M.H.&author=Tokheim+C.&author=Porta-Pardo+E.&author=Sengupta+S.&author=Bertrand+D.&author=Weerasinghe+A.&author=Colaprico+A.&author=Wendl+M.C.&author=Kim+J.&author=Reardon+B.&title=Comprehensive+characterization+of+cancer+driver+genes+and+mutations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>; <div class="dropBlock reference-citations"><a href="#bib18" id="back-bib18" data-db-target-for="bib18-d4222834e5580" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib18">Brennan et al., 2013</a><div data-db-target-of="bib18-d4222834e5580" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Brennan C.W. </li><li class="loa__item">Verhaak R.G.W. </li><li class="loa__item">McKenna A. </li><li class="loa__item">Campos B. </li><li class="loa__item">Noushmehr H. </li><li class="loa__item">Salama S.R. </li><li class="loa__item">Zheng S. </li><li class="loa__item">Chakravarty D. </li><li class="loa__item">Sanborn J.Z. </li><li class="loa__item">Berman S.H. </li><li>et al.</li></ul><div class="reference-citations__detials">The somatic genomic landscape of glioblastoma.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div><div class="section-paragraph">For each interacting protein pair, we split samples with and without mutations in partner A and compare expression levels (RNA, protein and phosphosites) both in <em>cis</em> (partner A) and in <em>trans</em> (partner B), calculating a median difference in expression and testing for significance with the Wilcoxon rank sum test, with the Benjamini-Hochberg multiple test correction. For mutational impact analysis on lipidome or metabolome, all possible pairs between SMGs and metabolites/lipids were tested.</div></section><section id="sec5.6.6.12" data-locator="sec5.6.6.12" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Protein and RNA marker identification for multi-omics mixed subtype</span></h4><div class="section-paragraph">By comparing the tumors of multi-omics mixed subtypes (nmf_cluster_membership score ≤ 0.55) to other (non-mixed) tumors, we identified 276 differentially expressed genes and 690 differentially expressed proteins. For each differentially expressed gene/protein, we binned all the tumors into three groups based on their expression level: high, medium, and low. We then compared the survival outcome (log-rank test) in the high expression group to the low expression group using the functions TCGAanalyze_divideGroups(), TCGAanalyze_SurvivalKM(), and TCGAanalyze_survival() from TCGAbiolinks (<div class="dropBlock reference-citations"><a href="#bib28" id="back-bib28" data-db-target-for="bib28-d4222834e5594" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib28">Colaprico et al., 2016</a><div data-db-target-of="bib28-d4222834e5594" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Colaprico A. </li><li class="loa__item">Silva T.C. </li><li class="loa__item">Olsen C. </li><li class="loa__item">Garofano L. </li><li class="loa__item">Cava C. </li><li class="loa__item">Garolini D. </li><li class="loa__item">Sabedot T.S. </li><li class="loa__item">Malta T.M. </li><li class="loa__item">Pagnotta S.M. </li><li class="loa__item">Castiglioni I. </li><li>et al.</li></ul><div class="reference-citations__detials">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), and the function surv_median() from the R package survminer. We identified 19 genes and 40 proteins with significantly differential survival outcome (<a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" class="scroll-into-link">Table S3</a>).</div></section><section id="sec5.6.6.13" data-locator="sec5.6.6.13" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Kinase-substrate pairs regression analysis</span></h4><div class="section-paragraph">For each kinase-substrate protein pair supported by previous experimental evidence (OmniPath, NetworKIN, DEPOD, and SIGNOR), we tested the associations between all sufficiently detected phosphosites on the substrate and the kinase. For a kinase-substrate pair to be tested, we required both kinase protein/phosphoprotein expression and phosphosite phosphorylation to be observed in at least 20 samples in the respective datasets and the overlapped dataset. We then applied the linear regression model using lm function in R to test for the relation between kinase and substrate phosphosite. For the <em>i</em>-th trial for kinase phosphosite abundance in the <em>cis</em> associations, kinase phosphosite abundance <em>A</em><sub>i</sub> depends on kinase protein expression <em>S</em><sub>i</sub> and error <em>E</em><sub>i</sub>,<div class="labeled-equation clearfix"><div id="ufd5" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si5.gif"><mrow><msub><mi>A</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><msub><mi>M</mi><mn>1</mn></msub><msub><mi>S</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>B</mi><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>i</mi></msub></mrow></math></div></div><br></br></div><div class="section-paragraph">For the <em>i</em>-th trial for kinase phosphosite abundance in the <em>trans</em> associations, substrate phosphosite abundance <em>A</em><sub>i</sub> depends on kinase phosphosite expression <em>K</em><sub>i</sub> substrate protein expression <em>S</em><sub>i</sub> and error <em>E</em><sub>i</sub>,<div class="labeled-equation clearfix"><div id="ufd6" class="formula" tabindex="0"><!--${MathJax: TeX-AMS-MML_HTMLorMML}--><math altimg="si6.gif"><mrow><msub><mi>A</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><msub><mi>M</mi><mn>1</mn></msub><msub><mi>S</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>M</mi><mn>2</mn></msub><msub><mi>K</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>B</mi><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>i</mi></msub></mrow></math></div></div><br></br>where the regression slope <em>M</em> coefficients are determined by least-square calculation. The resulting p-values were adjusted for multiple testing using the Benjamini-Hochberg procedure.</div><div class="section-paragraph">For the broader investigation of signaling cascades, we included total 214 kinases and 43 phosphatases if they satisfied either of the genetic alteration criteria or at least three criteria below:<ul class="ce-list--remove-bullets" id="olist0025"><li class="ce-list--remove-bullets__list-item" id="o0085"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">5% and more tumors with copy number alterations</div></li><li class="ce-list--remove-bullets__list-item" id="o0090"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">2 and more tumors with somatic mutations</div></li><li class="ce-list--remove-bullets__list-item" id="o0095"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Top 20% variable gene expression</div></li><li class="ce-list--remove-bullets__list-item" id="o0100"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Top 35% variable protein expression</div></li><li class="ce-list--remove-bullets__list-item" id="o0105"><span class="label">●</span><div class="ce-list--remove-bullets__list-item__text">Significantly different RNA or protein expression between tumor and normal (FDR ≤ 0.01)</div></li></ul></div></section><section id="sec5.6.6.14" data-locator="sec5.6.6.14" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Differential proteomic, phosphosite, metabolome and lipidome analysis</span></h4><div class="section-paragraph">TMT-based global proteomic, phosphoproteomic, and acetylation, as well as metabolome and lipidome data were used to perform pairwise differential analysis between groups of samples. A Wilcoxon rank-sum test was performed to determine differential abundance of proteins, PTMs and metabolites. At least four samples in both groups were required to have non-missing values and the p-value was adjusted using the Benjamini-Hochberg procedure. For phosphorylation markers in each genomic subtype, the adjusted p-value for the protein change was required to be ≥ 0.05.</div></section><section id="sec5.6.6.15" data-locator="sec5.6.6.15" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Phosphoproteome outlier analysis</span></h4><div class="section-paragraph">Outlier Analysis was done using BlackSheep’s DEVA analysis (<div class="dropBlock reference-citations"><a href="#bib16" id="back-bib16" data-db-target-for="bib16-d4222834e5757" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib16">Blumenberg et al., 2019</a><div data-db-target-of="bib16-d4222834e5757" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Blumenberg L. </li><li class="loa__item">Kawaler E. </li><li class="loa__item">Cornwell M. </li><li class="loa__item">Smith S. </li><li class="loa__item">Ruggles K. </li><li class="loa__item">Fenyö D. </li></ul><div class="reference-citations__detials">BlackSheep: a bioconductor and bioconda package for differential extreme value analysis (Bioinformatics).<div class="popup-ref"><span class="refDate">2019</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+a+bioconductor+and+bioconda+package+for+differential+extreme+value+analysis+%28Bioinformatics%29">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Phosphopeptide analysis was done on data that was aggregated per protein, summing together outlier values across all phosphosites. Protein analysis was performed using TMT-based global proteomic data, RNA analysis was done using FPKM-UQ normalized transcript data. The DEVA method calculates interquartile range (IQR) and median values for the given dataset, and then defines outliers as values greater than the median plus 1.5x IQR. Features were prefiltered to include an outlier value in at least 30% of samples in the group of interest and for features that had a higher proportion of features in the group of interest compared to the rest of the population. Statistics were calculated using a Fisher’s exact test and p-values were corrected using the Benjamini-Hochberg procedure. Druggability of a gene/protein was performed using DGIdbR (<div class="dropBlock reference-citations"><a href="#bib30" id="back-bib30" data-db-target-for="bib30-d4222834e5760" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib30">Cotto et al., 2018</a><div data-db-target-of="bib30-d4222834e5760" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Cotto K.C. </li><li class="loa__item">Wagner A.H. </li><li class="loa__item">Feng Y.-Y. </li><li class="loa__item">Kiwala S. </li><li class="loa__item">Coffman A.C. </li><li class="loa__item">Spies G. </li><li class="loa__item">Wollam A. </li><li class="loa__item">Spies N.C. </li><li class="loa__item">Griffith O.L. </li><li class="loa__item">Griffith M. </li></ul><div class="reference-citations__detials">DGIdb 3.0: a redesign and expansion of the drug-gene interaction database.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D1068-D1073</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1143&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29156001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85070670869&amp;cf=" onclick="newWindow(this.href);return false">Scopus (214)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1068-D1073&journal=Nucleic+Acids+Res.&author=Cotto+K.C.&author=Wagner+A.H.&author=Feng+Y.-Y.&author=Kiwala+S.&author=Coffman+A.C.&author=Spies+G.&author=Wollam+A.&author=Spies+N.C.&author=Griffith+O.L.&author=Griffith+M.&title=DGIdb+3.0%3A+a+redesign+and+expansion+of+the+drug-gene+interaction+database">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>).</div></section><section id="sec5.6.6.16" data-locator="sec5.6.6.16" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Copy number impact on transcriptome and proteome</span></h4><div class="section-paragraph">To evaluate copy number impact on RNA and protein expression, we applied gene-wise correlation analysis on CNV versus RNA expression and on CNV versus protein expression. Correlation was performed by Pearson’s correlation method. Both correlation coefficient and p-value were computed and adjusted by the Benjamini-Hochberg procedure.</div></section><section id="sec5.6.6.17" data-locator="sec5.6.6.17" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cell type enrichment deconvolution using gene expression</span></h4><div class="section-paragraph">The abundance of each cell type was inferred by the xCell web tool (<div class="dropBlock reference-citations"><a href="#bib3" id="back-bib3" data-db-target-for="bib3-d4222834e5773" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib3">Aran et al., 2017</a><div data-db-target-of="bib3-d4222834e5773" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Aran D. </li><li class="loa__item">Hu Z. </li><li class="loa__item">Butte A.J. </li></ul><div class="reference-citations__detials">xCell: digitally portraying the tissue cellular heterogeneity landscape.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (379)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>), which performed the cell type enrichment analysis from gene expression data for 64 immune and stromal cell types (default xCell signature). xCell is a gene signatures-based method learned from thousands of pure cell types from various sources. We input the FPKM-UQ expression matrix of this study in xCell using the expression levels ranking.</div></section><section id="sec5.6.6.18" data-locator="sec5.6.6.18" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Immune clustering using cell type enrichment scores</span></h4><div class="section-paragraph">Immune subtypes of the GBM tumors were generated on the consensus clustering of the cell type enrichment scores by xCell (<div class="dropBlock reference-citations"><a href="#bib137" id="back-bib137" data-db-target-for="bib137-d4222834e5781" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib137">Wilkerson and Hayes, 2010</a><div data-db-target-of="bib137-d4222834e5781" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Wilkerson M.D. </li><li class="loa__item">Hayes D.N. </li></ul><div class="reference-citations__detials">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.<div class="popup-ref"><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinforma.+Oxf.+Engl.&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Among the 64 cell types tested in xCell, we selected 40 cell types with at least 2 samples with xCell enrichment p &lt; 0.01, which filtered out the cell types not typical in the brain. xCell generated an immune score per sample that integrates the enrichment scores B cells, CD4+ T-cells, CD8+ T-cells, DC, eosinophils, macrophages, monocytes, mast cells, neutrophils, and NK cells. In addition, we included microglia using the scores by ssGSEA based on its marker genes: <em>P2RY12, TMEM119, SLC2A5, TGFBR1, GPR34, SALL1, GAS6, MERTK, C1QA, PROS1, CD68, ADGRE1, AIF1, CX3CR1, TREM2,</em> and <em>ITGAM</em>. The microglia ssGSEA score was computed using the R package GSVA (gsva function with method='ssgsea'). We performed consensus immune clustering based on the z-score normalized xCell and microglia scores. The consensus clustering was determined by the R package ConsensusClusterPlus (parameters: clusterAlg='kmdist' distance='spearman').</div></section><section id="sec5.6.6.19" data-locator="sec5.6.6.19" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Cell type enrichment deconvolution using protein abundance</span></h4><div class="section-paragraph">We applied CIBERSORTx (<div class="dropBlock reference-citations"><a href="#bib98" id="back-bib98" data-db-target-for="bib98-d4222834e5793" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib98">Newman et al., 2019</a><div data-db-target-of="bib98-d4222834e5793" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Newman A.M. </li><li class="loa__item">Steen C.B. </li><li class="loa__item">Liu C.L. </li><li class="loa__item">Gentles A.J. </li><li class="loa__item">Chaudhuri A.A. </li><li class="loa__item">Scherer F. </li><li class="loa__item">Khodadoust M.S. </li><li class="loa__item">Esfahani M.S. </li><li class="loa__item">Luca B.A. </li><li class="loa__item">Steiner D. </li><li>et al.</li></ul><div class="reference-citations__detials">Determining cell type abundance and expression from bulk tissues with digital cytometry.<div class="popup-ref"><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">773-782</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41587-019-0114-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31061481&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85065314034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (187)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=773-782&journal=Nat.+Biotechnol.&author=Newman+A.M.&author=Steen+C.B.&author=Liu+C.L.&author=Gentles+A.J.&author=Chaudhuri+A.A.&author=Scherer+F.&author=Khodadoust+M.S.&author=Esfahani+M.S.&author=Luca+B.A.&author=Steiner+D.&title=Determining+cell+type+abundance+and+expression+from+bulk+tissues+with+digital+cytometry">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) to compute immune cell fractions from bulk protein abundance. To characterize the immune context using proteomics, we generated a signature matrix based on the dataset from Rieckmann et al. (<div class="dropBlock reference-citations"><a href="#bib111" id="back-bib111" data-db-target-for="bib111-d4222834e5796" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib111">Rieckmann et al., 2017</a><div data-db-target-of="bib111-d4222834e5796" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Rieckmann J.C. </li><li class="loa__item">Geiger R. </li><li class="loa__item">Hornburg D. </li><li class="loa__item">Wolf T. </li><li class="loa__item">Kveler K. </li><li class="loa__item">Jarrossay D. </li><li class="loa__item">Sallusto F. </li><li class="loa__item">Shen-Orr S.S. </li><li class="loa__item">Lanzavecchia A. </li><li class="loa__item">Mann M. </li><li>et al.</li></ul><div class="reference-citations__detials">Social network architecture of human immune cells unveiled by quantitative proteomics.<div class="popup-ref"><span class="ref__series"><i>Nat. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">583-593</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fni.3693&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28263321&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85014542002&amp;cf=" onclick="newWindow(this.href);return false">Scopus (130)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=583-593&journal=Nat.+Immunol.&author=Rieckmann+J.C.&author=Geiger+R.&author=Hornburg+D.&author=Wolf+T.&author=Kveler+K.&author=Jarrossay+D.&author=Sallusto+F.&author=Shen-Orr+S.S.&author=Lanzavecchia+A.&author=Mann+M.&title=Social+network+architecture+of+human+immune+cells+unveiled+by+quantitative+proteomics">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Briefly, 28 distinct human hematopoietic cell types from peripheral blood of healthy donors were sorted by flow cytometry. Erythrocytes and platelets were excluded from subsequent analyses. Cellular proteomes were analyzed in single runs by high-resolution MS using a quadrupole Orbitrap instrument. Each cell type proteome was measured from four donors. The proteomic dataset included 10,134 proteins and 104 steady state samples. The samples were first scaled to have mean zero and standard deviation equal to one. We grouped the 26 subtypes into nine cell types: B cells, basophils, dendritic cells, eosinophils, monocytes, natural killer cells (NKs), neutrophils, CD4+ T-cells and CD8+ T-cells. We took imputed values from <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4" class="scroll-into-link">Table S3</a> of the Rieckmann et al. paper to generate a signature matrix of these nine cell types. CIBERSORT was applied to the GBM imputed protein abundance matrix using a batch correction and relative values. CD8+ T-cells cells and NKs were merged by summing their relative values. Z-scores of relative values were used for boxplots and heatmap.</div></section><section id="sec5.6.6.20" data-locator="sec5.6.6.20" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Deep learning histopathology image analysis</span></h4><div class="section-paragraph">We trained deep learning models for 3 different prediction tasks based on histopathology images, including the G-CIMP phenotype (positive and negative), immune response (im4 subtype as immune low and the rest of the tumors as immune high), and telomere length (short, normal, and long). Digital histopathology slides and associated quantified features (cellularity, necrosis, tumor nuclei, age, tumor weight) of samples used in proteomics analysis were downloaded from The Cancer Imaging Archive (TCIA) database. Labels were at per-case (patient) level. The images and their corresponding labels were then divided into 3 datasets at per-case level with 70% of cases in training set, 15% of cases in validation set, and 15% of cases in testing set. Due to the large size of the scanned histopathology slides, they were tiled into 299-by-299-pixel pieces with overlapping area of 49 pixels from each edge at 20X, 10X, and 5X resolution. In this process, tiles with over 30% of background pixels were removed. Qualified tiles, quantified features, and labels of each set were then loaded into a designated TFrecords file. After the data preparation, convolutional neural network (CNN) architectures, including InceptionV1 to V4, InceptionResNetV1 and V2, and self-designed simple CNNs, were trained from scratch. Statistical metrics, such as area under ROC, area under PRC, and top-1 accuracy, were used to evaluate the performance. The best model for each task was picked at the minimum validation loss point. Trained models were tested on the testing set and the statistical metrics of the testing set were used to compare the performance of different models on the same tasks.</div><div class="section-paragraph">A visualization method designed to unveil the features learned by the models was applied to discover histological features associated with G-CIMP phenotype, telomere length, and immune response in the cohort. Firstly, the activation score vectors of each tile from the fully connected layer immediately before the output layer in the testing set were extracted as representation of the input samples. A randomly sampled subset of these activation score vectors was dimensionally reduced into 2-dimensional space by t-SNE with each point representing an image tile. Overlay of prediction scores on these points revealed clusters corresponding to the labels. Finally, experienced pathologists examined the tiles in each of these clusters and summarized the general histological features in these clusters, which served as the representation of the histological features of these subgroups.</div></section><section id="sec5.6.6.21" data-locator="sec5.6.6.21" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Gene set enrichment analysis</span></h4><div class="section-paragraph">Differential Expressed Genes (DEGs) were identified using DESeq2 (<div class="dropBlock reference-citations"><a href="#bib86" id="back-bib86" data-db-target-for="bib86-d4222834e5814" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib86">Love et al., 2014</a><div data-db-target-of="bib86-d4222834e5814" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Love M.I. </li><li class="loa__item">Huber W. </li><li class="loa__item">Anders S. </li></ul><div class="reference-citations__detials">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.<div class="popup-ref"><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">550</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-014-0550-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25516281&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84924629414&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16300)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=550&journal=Genome+Biol.&author=Love+M.I.&author=Huber+W.&author=Anders+S.&title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) by applying the minimal pre-filtering to keep only genes that have at least 10 reads in total. We selected the genes which had FDR ≤ 0.01 and absolute fold change larger than 2. To designate the representative pathways of immune subtypes, we selected the DEGs between the two immune subtypes and then underwent a pathway enrichment analysis of Hallmark, KEGG, and Reactome. The overrepresented pathways were selected (FDR &lt; 0.1, only pathways with at least 10 genes observed in each data type are considered).</div><div class="section-paragraph">To identify significantly enriched Hallmark, KEGG, PID, and REACTOME gene sets of each immune cluster, we applied the ssGSEA on all proteins to calculate the normalized enrichment score (NES) for each gene set in each sample. Then we performed the pairwise t-test of NES among the 2 immune clusters and adjusted the p-values by FDR. We ranked gene sets by FDR and selected the top 50 gene sets (all FDR &lt; 0.01) of each immune cluster.</div></section><section id="sec5.6.6.22" data-locator="sec5.6.6.22" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Histone protein and acetylation calculation</span></h4><div class="section-paragraph">Core histones H2A, H2B, H3 and H4, and linker histone H1 are encoded by multiple genes with minor changes in their sequence. Accordingly, we detected a number of peptides and acetylated peptides corresponding to either of the core histones and H1 histone. To facilitate the interpretation of histone acetylation events, we averaged acetylation values for peptides mapped on different gene encoding practically the same histone protein.</div></section><section id="sec5.6.6.23" data-locator="sec5.6.6.23" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Histone acetylation association with HATs and HDACs</span></h4><div class="section-paragraph">To test the association between HATs/HDACs protein and acetylation levels of histone sites, we fitted Lasso regression model with HATs/HDACs and histone protein expression as independent variables and a histone acetylation site as a dependent variable. Lasso regression has been chosen because it takes expression of all enzymes into account simultaneously and is insensitive to highly correlated dependent variables. We performed 300 bootstraps with 80% training data and 20% testing data, and reported averaged coefficients returned by the model across 300 iterations.</div></section><section id="sec5.6.6.24" data-locator="sec5.6.6.24" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Pathway enrichment analysis along histones H2B and H3/H4 acetylation axes</span></h4><div class="section-paragraph">We investigated pathways from Hallmark, KEGG, WIkipathways, and REACTOME, positively or negatively aligned with averaged H2B and H3/H4 acetylation level. H2B acetylation was calculated by averaging acetylation of all H2B peptides detected. Since H3 and H4 histones are strongly correlated with each other, we averaged acetylation of histones H3 and H4 peptides together to obtain H3/H4 acetylation value.</div><div class="section-paragraph">We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway; then a difference in a pathway activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: (1) probability of pathway proteins to occupy by random chance the observed positions in a list of tumor proteins ranked by expression level from the top to the bottom and, similarly, (2) probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top. Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above “top” and “bottom” probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the “top” and “bottom” probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher’s exact test, given protein’s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor. The thermodynamic interpretation of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway’s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway’s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random. Given any biological axis, e.g. histone acetylation levels in each of tumors, one can determine pathways which are significantly correlated or anti correlated with the axis.</div></section><section id="sec5.6.6.25" data-locator="sec5.6.6.25" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">Causative pathway interaction discovery using CausalPath</span></h4><div class="section-paragraph">To discover the causative pathway interactions in our proteomic and phosphoproteomic data, we took the normalized expression of protein with &lt; 10% missing values and phosphoprotein with &lt; 25% missing values across all tumor and normal samples as the input to CausalPath (commit 7c5b934). We ran CausalPath in the mode that tests the mean values between test and control groups (value-transformation = significant-change-of-mean), where the test group being the tumors of one subtype and control group being the rest of the tumors. The pathway interaction discovery data source was Pathway Commons v9 (built-in-network-resource-selection = PC). Additionally, we enabled the causal reasoning if all the downstream targets of a gene were active or inactive (calculate-network-significance = true, use-network-significance-for-causal-reasoning = true, permutations-for-significance = 10000). The causative interactions with FDR &lt; 0.05 were extracted and visualized (fdr-threshold-for-data-significance = 0.05 phosphoprotein, fdr-threshold-for-data-significance = 0.05 protein, fdr-threshold-for-network-significance = 0.05). Full result tables were available in <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6" class="scroll-into-link">Table S5</a>.</div></section><section id="sec5.6.6.26" data-locator="sec5.6.6.26" tabindex="0"><h4 class="sectionTitle"><span class="top__sub"></span><span class="top__text">L1000 and P100 drug connectivity analysis</span></h4><div class="section-paragraph">For mRNA abundance, RNA-seq read counts were used to perform differential expression analysis between gene-altered and WT samples using edgeR (<div class="dropBlock reference-citations"><a href="#bib112" id="back-bib112" data-db-target-for="bib112-d4222834e5850" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib112">Robinson et al., 2010</a><div data-db-target-of="bib112-d4222834e5850" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Robinson M.D. </li><li class="loa__item">McCarthy D.J. </li><li class="loa__item">Smyth G.K. </li></ul><div class="reference-citations__detials">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.<div class="popup-ref"><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">139-140</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp616&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19910308&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75249087100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13601)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=139-140&journal=Bioinformatics&author=Robinson+M.D.&author=McCarthy+D.J.&author=Smyth+G.K.&title=edgeR%3A+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). The significantly differentially expressed genes (FC ≥ 1 and FDR &lt; 0.05) were then used as input in the subsequent analysis. For protein and phosphoprotein abundance, a Wilcoxon rank sum test was performed to determine differential abundance of protein and phosphoprotein measurements between gene altered and WT samples. Protein with FDR &lt; 0.05 were considered as differentially expressed.</div><div class="section-paragraph">The differentially expressed genes between gene-altered and WT samples were filtered for the 978 genes measured in the L1000 assay and then were processed using the CLUE (<div class="dropBlock reference-citations"><a href="#bib122" id="back-bib122" data-db-target-for="bib122-d4222834e5855" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib122">Subramanian et al., 2017</a><div data-db-target-of="bib122-d4222834e5855" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Subramanian A. </li><li class="loa__item">Narayan R. </li><li class="loa__item">Corsello S.M. </li><li class="loa__item">Peck D.D. </li><li class="loa__item">Natoli T.E. </li><li class="loa__item">Lu X. </li><li class="loa__item">Gould J. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Tubelli A.A. </li><li class="loa__item">Asiedu J.K. </li><li>et al.</li></ul><div class="reference-citations__detials">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles.<div class="popup-ref"><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+next+generation+connectivity+map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) (summary connectivity score) and iLINCS (<div class="dropBlock reference-citations"><a href="#bib105" id="back-bib105" data-db-target-for="bib105-d4222834e5858" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib105">Pilarczyk et al., 2019</a><div data-db-target-of="bib105-d4222834e5858" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Pilarczyk M. </li><li class="loa__item">Najafabadi M.F. </li><li class="loa__item">Kouril M. </li><li class="loa__item">Vasiliauskas J. </li><li class="loa__item">Niu W. </li><li class="loa__item">Shamsaei B. </li><li class="loa__item">Mahi N. </li><li class="loa__item">Zhang L. </li><li class="loa__item">Clark N. </li><li class="loa__item">Ren Y. </li><li>et al.</li></ul><div class="reference-citations__detials">Connecting Omics Signatures of Diseases, Drugs, and Mechanisms of Actions with iLINCS.<div class="popup-ref"><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">826271</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=826271&journal=BioRxiv&author=Pilarczyk+M.&author=Najafabadi+M.F.&author=Kouril+M.&author=Vasiliauskas+J.&author=Niu+W.&author=Shamsaei+B.&author=Mahi+N.&author=Zhang+L.&author=Clark+N.&author=Ren+Y.&title=Connecting+Omics+Signatures+of+Diseases%2C+Drugs%2C+and+Mechanisms+of+Actions+with+iLINCS">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) connectivity algorithms. The resulting drug connectivities were aggregated to the compound level using the summary connectivity score in CLUE and the Connected Perturbations Z-score in iLINCS. Target annotations for the ranked compounds were extracted from CLUE and iLINCS and combined in a single list.</div><div class="section-paragraph">Level 4 P100 data were downloaded from the LINCS Data Portal (<div class="dropBlock reference-citations"><a href="#bib118" id="back-bib118" data-db-target-for="bib118-d4222834e5863" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib118">Stathias et al., 2019</a><div data-db-target-of="bib118-d4222834e5863" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Stathias V. </li><li class="loa__item">Turner J. </li><li class="loa__item">Koleti A. </li><li class="loa__item">Vidovic D. </li><li class="loa__item">Cooper D. </li><li class="loa__item">Fazel-Najafabadi M. </li><li class="loa__item">Pilarczyk M. </li><li class="loa__item">Terryn R. </li><li class="loa__item">Chung C. </li><li class="loa__item">Umeano A. </li><li>et al.</li></ul><div class="reference-citations__detials">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures.<div class="popup-ref"><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D431-D439</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkz1023&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077665017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D431-D439&journal=Nucleic+Acids+Res.&author=Stathias+V.&author=Turner+J.&author=Koleti+A.&author=Vidovic+D.&author=Cooper+D.&author=Fazel-Najafabadi+M.&author=Pilarczyk+M.&author=Terryn+R.&author=Chung+C.&author=Umeano+A.&title=LINCS+Data+Portal+2.0%3A+next+generation+access+point+for+perturbation-response+signatures">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>) and were used to calculate drug connectivities on the phosphoprotein level as previously described (<div class="dropBlock reference-citations"><a href="#bib82" id="back-bib82" data-db-target-for="bib82-d4222834e5866" class="reference-citations__ctrl" aria-expanded="false" aria-controls="bib82">Litichevskiy et al., 2018</a><div data-db-target-of="bib82-d4222834e5866" class="dropBlock__holder"><div class="dropBlock__body"><ul title="list of authors" class="rlist reference-citations__loa inline-bullet-list"><li class="loa__item">Litichevskiy L. </li><li class="loa__item">Peckner R. </li><li class="loa__item">Abelin J.G. </li><li class="loa__item">Asiedu J.K. </li><li class="loa__item">Creech A.L. </li><li class="loa__item">Davis J.F. </li><li class="loa__item">Davison D. </li><li class="loa__item">Dunning C.M. </li><li class="loa__item">Egertson J.D. </li><li class="loa__item">Egri S. </li><li>et al.</li></ul><div class="reference-citations__detials">A library of phosphoproteomic and chromatin signatures for characterizing cellular responses to drug perturbations.<div class="popup-ref"><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">424-443.e7</span></div></div><div class="reference-citations__links"><ul class="rlist inline-bullet-list"><li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29655704&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85045118030&amp;cf=" onclick="newWindow(this.href);return false">Scopus (26)</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=424-443.e7&journal=Cell+Syst&author=Litichevskiy+L.&author=Peckner+R.&author=Abelin+J.G.&author=Asiedu+J.K.&author=Creech+A.L.&author=Davis+J.F.&author=Davison+D.&author=Dunning+C.M.&author=Egertson+J.D.&author=Egri+S.&title=A+library+of+phosphoproteomic+and+chromatin+signatures+for+characterizing+cellular+responses+to+drug+perturbations">Google Scholar</a></li></ul></div></div><a href="#" class="dropBlock__ctrl--close" title="Close info"><i aria-hidden="true" class="icon-close_thin"></i></a></div></div>). Firstly, the P100 probes were mapped to the CPTAC phosphopeptides using their respective modified peptide sequences. The spearman correlation was then calculated between the phosphoproteomic mutation signature and each drug-treated P100 experiment. The correlation was then converted to a connectivity score by comparing the distribution of the observed drug-signature correlations to a background distribution of correlations using the two-sample Kolmogorov-Smirnov (KS) test. The directionality of the connectivity score (positive/negative) was determined by the comparison of the medians of the two distributions (The score is negative when the median of the observed distribution is lower than the one of the background distribution). Significant connectivity scores (p-value &lt; 0.05) were then aggregated to the compound level by their mean across all tested cell lines.</div><div class="section-paragraph">Full result tables were available in <a href="/cms/10.1016/j.ccell.2021.01.006/attachment/5ab0cf58-b77f-4522-a878-83ed50e51c8d/mmc7" class="scroll-into-link">Table S6</a>.</div></section></section></section></section><section><h2 class="top" id="secsectitle0595" data-left-hand-nav="Acknowledgments" tabindex="0"><span class="top__sub"></span><span class="top__text">Acknowledgments</span></h2><div class="section-paragraph">This work was supported by grants U24CA210972, U24CA210955, U24CA210954, U24CA210985, U24CA210993, U24CA210967, U24CA210986, U01CA214125, and U24CA210979 from the National Cancer Institute's Clinical Proteomic Tumor Analysis Consortium, by grant R01HG009711 from National Human Genome Research Institute to L.D., and R01NS107833 from National Institutes of Health to M.G.C. The MS-based proteomics work was performed at the Environmental Molecular Sciences Laboratory (grid.436923.9), a U.S. Department of Energy National Scientific User Facility located at the Pacific Northwest National Laboratory operated under contract DE-AC05-76RL01830. We thank Dr. Charles A. Goldthwaite for manuscript editing.</div><h3>Author contributions</h3><div class="section-paragraph">Study Conception &amp; Design: J.S.B., K.D.R., T.L., L.D.</div><div class="section-paragraph">Performed Experiment or Data Collection: M.A.G., W.C., C.T., R.K.C., K.K.W., R.J.M., M.E.M., R.Z., X.Y., E.Z., S.P.C., K.J.B., F.C.</div><div class="section-paragraph">Computation &amp; Statistical Analysis: L.W., AKarpova, J.E.K., S.C., Y.L., D.R., A.C., J.H.R., R.H., V.S., MCornwell, F.P., Y.W., B.R., K.K., P.P., E.K., L.K.O., WLiang, X.S., Y.D., M.C.W., WLiu, D.C.Z., J.J., V.A.P., J.M., W.M., S.Y., AKrek, A.D., H.Z., B.L.H., C.M.L., H.B., S.L., J.B., L.Y., K.G.S., L.M.B., SSatpathy, W.S., M.W., A.I.N., B.Z., D.R.M., MCeccarelli, X.S.C., D.F., K.V.R., S.H.P., P.W., L.D.</div><div class="section-paragraph">Data Interpretation &amp; Biological Analysis: L.W., AKarpova, J.E.K., S.C., Y.L., D.R., A.C., J.H.R., R.H., V.S., MCornwell, F.P., Y.W., B.R., K.K., P.P., E.K., L.K.O., WLiang, X.S., Y.D., M.C.W., WLiu, D.C.Z., J.J., S.Y., AKrek, A.D., B.L.H., C.M.L., H.B., S.L., J.B., L.Y., W.S., M.W., K.A.K., S.L.C., Q.K.L., J.S.B., L.D.</div><div class="section-paragraph">Writing – Original Drafts: L.W., AKarpova, M.A.G., J.E.K., S.C., Y.L., D.R., V.S., M.A.W., K.D.R., T.L., L.D.</div><div class="section-paragraph">Writing – Review &amp; Editing: L.W., AKarpova, M.A.G., J.E.K., S.C., Y.L., A.C., V.S., M.C.W., J.F.M., S.M.B., E.S.B., A.H.K., M.M., S.H.P., A.I., M.G.C., J.S.B., K.D.R., T.L., L.D.</div><div class="section-paragraph">Supervision: SSchürer, F.C., A.I.N., B.Z., D.R.M., MCeccarelli, X.S.C., D.F., K.V.R., H.R., S.H.P., A.C.R., P.W., R.D.S., J.S.B., K.D.R., T.L., L.D.</div><div class="section-paragraph">Administration: E.S.B., C.R.K., A.I.R., T.H., M.T., A.H.K., H.R., M.M., K.D.R., T.L., L.D.</div><h3>Declaration of interests</h3><div class="section-paragraph">S.Y. is employed by Sema4. A.H.K. consults for Monteris Medical. P.W. is a statistical consultant for Sema4. M.G.C. receives research support from Orbus Therapeutics and NeoimmuneTech Inc, and royalties from UpToDate.</div></section><section id="app2" data-locator="app2" tabindex="0"><h2 class="top" id="supplementaryMaterial" data-left-hand-nav="Supplemental information" tabindex="0"><span class="top__sub"></span><span class="top__text">Supplemental information</span></h2><div class="section-paragraph"><ul class="rlist"><li class="supplemental-information__item" id="mmc1"><div class="supplemental-information__icon icon-gizmo-pdf-file2"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/87362f58-35c5-477d-a566-db582618cc6c/mmc1.pdf" class="supplemental-information__download " target="_blank">
                                            Download .pdf (4.77
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    pdf files
                                </a></div></div><div class="supplemental-information__title">Document S1. Figures S1–S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc2"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/88e921e9-6b3e-4bd6-b49b-24db5cd273c8/mmc2.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.06
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S1. GBM patient and GTEx normal samples clinical characteristics, additional annotations, and survival analysis of different molecular features and subtypes, related to Figures 1, S1, and S2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc3"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/55251911-b73b-4f07-8f3d-466953b295e9/mmc3.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (123.25
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S2. All processed genomic and proteomic data, related to Figures 1, 2, 3, 4, 5, 6, 7, and S1–S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc4"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/71970c26-ff68-46ff-bbaf-d5e23f90b417/mmc4.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (.34
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S3. Signature features of multi-omics subtypes and unsupervised clustering of miRNA expression, related to Figures 1, S1, and S2</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc5"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/c8ae7793-ce25-422c-a366-37ffd9051677/mmc5.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (3.38
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S4. Immune subtype details, related to Figures 4 and S4</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc6"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/2a64e9d4-0561-4c45-8a52-443550df188f/mmc6.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (46.24
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S5. Kinase-substrate signaling cascades of all selected kinases and phosphatases, outlier status of all phosphosites, and complete causal path graph, related to Figures 7 and S7</div></li></ul><ul class="rlist"><li class="supplemental-information__item" id="mmc7"><div class="supplemental-information__icon icon-gizmo-interactive-case-insights"></div><div class="supplemental-information__links clearfix"><div class="col-sm-6 gutterless"><a href="/cms/10.1016/j.ccell.2021.01.006/attachment/5ab0cf58-b77f-4522-a878-83ed50e51c8d/mmc7.xlsx" class="supplemental-information__download " target="_blank">
                                            Download .xlsx (1.06
                                            MB)
                                        </a></div><div class="col-sm-6 gutterless"><a href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" class="supplemental-information__help " target="_blank">
                                    Help with
                                    xlsx files
                                </a></div></div><div class="supplemental-information__title">Table S6. Drug connectivity scores of iLINCS, CLUE, and P100 of all signatures, related to Figures 7 and S7</div></li></ul></div></section><div data-pb-dropzone="extra-dropzone" class="extra-dropzone pb-dropzone"><!-- Empty dropzone --></div><section class="refs"><h2 class="top" id="secsectitle0620" data-left-hand-nav="References" tabindex="0"><span class="top__sub"></span><span class="top__text">References</span></h2><ol class="ref__wrapper"><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Abecasis G.R. </li><li class="ref__authors__name">Altshuler D. </li><li class="ref__authors__name">Auton A. </li><li class="ref__authors__name">Brooks L.D. </li><li class="ref__authors__name">Durbin R.M. </li><li class="ref__authors__name">Gibbs R.A. </li><li class="ref__authors__name">Hurles M.E. </li><li class="ref__authors__name">McVean G.A. </li><li>1000 Genomes Project Consortium</li></ul><div class="ref__title">A map of human genome variation from population-scale sequencing.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">467</span></span>: <span class="ref__seriesPages">1061-1073</span><div class="ref__controls"><a href="#back-bib1" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-78049390036&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5331)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20981092&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature09534&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2010&pages=1061-1073&journal=Nature&author=1000+Genomes+Project+Consortium&title=A+map+of+human+genome+variation+from+population-scale+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Anders S. </li><li class="ref__authors__name">Pyl P.T. </li><li class="ref__authors__name">Huber W. </li></ul><div class="ref__title">HTSeq—a Python framework to work with high-throughput sequencing data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">166-169</span><div class="ref__controls"><a href="#back-bib2" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84928987900&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7209)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25260700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtu638&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=166-169&journal=Bioinformatics&author=Anders+S.&author=Pyl+P.T.&author=Huber+W.&title=HTSeq%E2%80%94a+Python+framework+to+work+with+high-throughput+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Aran D. </li><li class="ref__authors__name">Hu Z. </li><li class="ref__authors__name">Butte A.J. </li></ul><div class="ref__title">xCell: digitally portraying the tissue cellular heterogeneity landscape.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">220</span><div class="ref__controls"><a href="#back-bib3" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85034614124&amp;cf=" onclick="newWindow(this.href);return false">Scopus (379)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29141660&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-017-1349-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=220&journal=Genome+Biol.&author=Aran+D.&author=Hu+Z.&author=Butte+A.J.&title=xCell%3A+digitally+portraying+the+tissue+cellular+heterogeneity+landscape">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Arlauckas S.P. </li><li class="ref__authors__name">Garren S.B. </li><li class="ref__authors__name">Garris C.S. </li><li class="ref__authors__name">Kohler R.H. </li><li class="ref__authors__name">Oh J. </li><li class="ref__authors__name">Pittet M.J. </li><li class="ref__authors__name">Weissleder R. </li></ul><div class="ref__title">Arg1 expression defines immunosuppressive subsets of tumor-associated macrophages.</div><span class="ref__series"><i>Theranostics.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">8</span></span>: <span class="ref__seriesPages">5842-5854</span><div class="ref__controls"><a href="#back-bib4" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85057156211&amp;cf=" onclick="newWindow(this.href);return false">Scopus (49)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30613266&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.7150%2Fthno.26888&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=5842-5854&journal=Theranostics&author=Arlauckas+S.P.&author=Garren+S.B.&author=Garris+C.S.&author=Kohler+R.H.&author=Oh+J.&author=Pittet+M.J.&author=Weissleder+R.&title=Arg1+expression+defines+immunosuppressive+subsets+of+tumor-associated+macrophages">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Babiceanu M. </li><li class="ref__authors__name">Qin F. </li><li class="ref__authors__name">Xie Z. </li><li class="ref__authors__name">Jia Y. </li><li class="ref__authors__name">Lopez K. </li><li class="ref__authors__name">Janus N. </li><li class="ref__authors__name">Facemire L. </li><li class="ref__authors__name">Kumar S. </li><li class="ref__authors__name">Pang Y. </li><li class="ref__authors__name">Qi Y. </li><li>et al.</li></ul><div class="ref__title">Recurrent chimeric fusion RNAs in non-cancer tissues and cells.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">2859-2872</span><div class="ref__controls"><a href="#back-bib5" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84963985214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (83)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26837576&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkw032&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=2859-2872&journal=Nucleic+Acids+Res.&author=Babiceanu+M.&author=Qin+F.&author=Xie+Z.&author=Jia+Y.&author=Lopez+K.&author=Janus+N.&author=Facemire+L.&author=Kumar+S.&author=Pang+Y.&author=Qi+Y.&title=Recurrent+chimeric+fusion+RNAs+in+non-cancer+tissues+and+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Babur Ö. </li><li class="ref__authors__name">Luna A. </li><li class="ref__authors__name">Korkut A. </li><li class="ref__authors__name">Durupinar F. </li><li class="ref__authors__name">Siper M.C. </li><li class="ref__authors__name">Dogrusoz U. </li><li class="ref__authors__name">Aslan J.E. </li><li class="ref__authors__name">Sander C. </li><li class="ref__authors__name">Demir E. </li></ul><div class="ref__title">Causal interactions from proteomic profiles: molecular data meets pathway knowledge.</div><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">258855</span><div class="ref__controls"><a href="#back-bib6" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=258855&journal=BioRxiv&author=Babur+%C3%96.&author=Luna+A.&author=Korkut+A.&author=Durupinar+F.&author=Siper+M.C.&author=Dogrusoz+U.&author=Aslan+J.E.&author=Sander+C.&author=Demir+E.&title=Causal+interactions+from+proteomic+profiles%3A+molecular+data+meets+pathway+knowledge">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bady P. </li><li class="ref__authors__name">Sciuscio D. </li><li class="ref__authors__name">Diserens A.-C. </li><li class="ref__authors__name">Bloch J. </li><li class="ref__authors__name">van den Bent M.J. </li><li class="ref__authors__name">Marosi C. </li><li class="ref__authors__name">Dietrich P.-Y. </li><li class="ref__authors__name">Weller M. </li><li class="ref__authors__name">Mariani L. </li><li class="ref__authors__name">Heppner F.L. </li><li>et al.</li></ul><div class="ref__title">MGMT methylation analysis of glioblastoma on the Infinium methylation BeadChip identifies two distinct CpG regions associated with gene silencing and outcome, yielding a prediction model for comparisons across datasets, tumor grades, and CIMP-status.</div><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">124</span></span>: <span class="ref__seriesPages">547-560</span><div class="ref__controls"><a href="#back-bib7" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84866542816&amp;cf=" onclick="newWindow(this.href);return false">Scopus (179)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22810491&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-012-1016-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2012&pages=547-560&journal=Acta+Neuropathol.&author=Bady+P.&author=Sciuscio+D.&author=Diserens+A.-C.&author=Bloch+J.&author=van+den+Bent+M.J.&author=Marosi+C.&author=Dietrich+P.-Y.&author=Weller+M.&author=Mariani+L.&author=Heppner+F.L.&title=MGMT+methylation+analysis+of+glioblastoma+on+the+Infinium+methylation+BeadChip+identifies+two+distinct+CpG+regions+associated+with+gene+silencing+and+outcome%2C+yielding+a+prediction+model+for+comparisons+across+datasets%2C+tumor+grades%2C+and+CIMP-status">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bailey M.H. </li><li class="ref__authors__name">Tokheim C. </li><li class="ref__authors__name">Porta-Pardo E. </li><li class="ref__authors__name">Sengupta S. </li><li class="ref__authors__name">Bertrand D. </li><li class="ref__authors__name">Weerasinghe A. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Wendl M.C. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Reardon B. </li><li>et al.</li></ul><div class="ref__title">Comprehensive characterization of cancer driver genes and mutations.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">371-385.e18</span><div class="ref__controls"><a href="#back-bib8" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044623848&amp;cf=" onclick="newWindow(this.href);return false">Scopus (506)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29625053&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.02.060&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=371-385.e18&journal=Cell&author=Bailey+M.H.&author=Tokheim+C.&author=Porta-Pardo+E.&author=Sengupta+S.&author=Bertrand+D.&author=Weerasinghe+A.&author=Colaprico+A.&author=Wendl+M.C.&author=Kim+J.&author=Reardon+B.&title=Comprehensive+characterization+of+cancer+driver+genes+and+mutations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Barbie D.A. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Boehm J.S. </li><li class="ref__authors__name">Kim S.Y. </li><li class="ref__authors__name">Moody S.E. </li><li class="ref__authors__name">Dunn I.F. </li><li class="ref__authors__name">Schinzel A.C. </li><li class="ref__authors__name">Sandy P. </li><li class="ref__authors__name">Meylan E. </li><li class="ref__authors__name">Scholl C. </li><li>et al.</li></ul><div class="ref__title">Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">462</span></span>: <span class="ref__seriesPages">108-112</span><div class="ref__controls"><a href="#back-bib9" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-70449091786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1164)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19847166&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature08460&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=108-112&journal=Nature&author=Barbie+D.A.&author=Tamayo+P.&author=Boehm+J.S.&author=Kim+S.Y.&author=Moody+S.E.&author=Dunn+I.F.&author=Schinzel+A.C.&author=Sandy+P.&author=Meylan+E.&author=Scholl+C.&title=Systematic+RNA+interference+reveals+that+oncogenic+KRAS-driven+cancers+require+TBK1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Beausoleil S.A. </li><li class="ref__authors__name">Villén J. </li><li class="ref__authors__name">Gerber S.A. </li><li class="ref__authors__name">Rush J. </li><li class="ref__authors__name">Gygi S.P. </li></ul><div class="ref__title">A probability-based approach for high-throughput protein phosphorylation analysis and site localization.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2006; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1285-1292</span><div class="ref__controls"><a href="#back-bib10" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-33749853607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1084)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=16964243&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt1240&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=1285-1292&journal=Nat.+Biotechnol.&author=Beausoleil+S.A.&author=Vill%C3%A9n+J.&author=Gerber+S.A.&author=Rush+J.&author=Gygi+S.P.&title=A+probability-based+approach+for+high-throughput+protein+phosphorylation+analysis+and+site+localization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Behnan J. </li><li class="ref__authors__name">Finocchiaro G. </li><li class="ref__authors__name">Hanna G. </li></ul><div class="ref__title">The landscape of the mesenchymal signature in brain tumours.</div><span class="ref__series"><i>Brain J. Neurol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">142</span></span>: <span class="ref__seriesPages">847-866</span><div class="ref__controls"><a href="#back-bib11" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85064239772&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30946477&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbrain%2Fawz044&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2019&pages=847-866&journal=Brain+J.+Neurol.&author=Behnan+J.&author=Finocchiaro+G.&author=Hanna+G.&title=The+landscape+of+the+mesenchymal+signature+in+brain+tumours">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bellail A.C. </li><li class="ref__authors__name">Olson J.J. </li><li class="ref__authors__name">Hao C. </li></ul><div class="ref__title">SUMO1 modification stabilizes CDK6 protein and drives the cell cycle and glioblastoma progression.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">4234</span><div class="ref__controls"><a href="#back-bib12" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84903125838&amp;cf=" onclick="newWindow(this.href);return false">Scopus (58)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24953629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fncomms5234&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=4234&journal=Nat.+Commun.&author=Bellail+A.C.&author=Olson+J.J.&author=Hao+C.&title=SUMO1+modification+stabilizes+CDK6+protein+and+drives+the+cell+cycle+and+glioblastoma+progression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Benelli M. </li><li class="ref__authors__name">Pescucci C. </li><li class="ref__authors__name">Marseglia G. </li><li class="ref__authors__name">Severgnini M. </li><li class="ref__authors__name">Torricelli F. </li><li class="ref__authors__name">Magi A. </li></ul><div class="ref__title">Discovering chimeric transcripts in paired-end RNA-seq data by using EricScript.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">3232-3239</span><div class="ref__controls"><a href="#back-bib13" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84870815979&amp;cf=" onclick="newWindow(this.href);return false">Scopus (77)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23093608&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts617&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=3232-3239&journal=Bioinformatics&author=Benelli+M.&author=Pescucci+C.&author=Marseglia+G.&author=Severgnini+M.&author=Torricelli+F.&author=Magi+A.&title=Discovering+chimeric+transcripts+in+paired-end+RNA-seq+data+by+using+EricScript">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Bhagat U. </li><li class="ref__authors__name">Das U.N. </li></ul><div class="ref__title">Potential role of dietary lipids in the prophylaxis of some clinical conditions.</div><span class="ref__series"><i>Arch. Med. Sci.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">807-818</span><div class="ref__controls"><a href="#back-bib14" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84939169189&amp;cf=" onclick="newWindow(this.href);return false">Scopus (14)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26322094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.5114%2Faoms.2015.53302&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=807-818&journal=Arch.+Med.+Sci.&author=Bhagat+U.&author=Das+U.N.&title=Potential+role+of+dietary+lipids+in+the+prophylaxis+of+some+clinical+conditions">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Blondel V.D. </li><li class="ref__authors__name">Guillaume J.-L. </li><li class="ref__authors__name">Lambiotte R. </li><li class="ref__authors__name">Lefebvre E. </li></ul><div class="ref__title">Fast unfolding of communities in large networks.</div><span class="ref__series"><i>J. Stat. Mech. Theor. Exp.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">2008</span></span>: <span class="ref__seriesPages">P10008</span><div class="ref__controls"><a href="#back-bib15" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-56349094785&amp;cf=" onclick="newWindow(this.href);return false">Scopus (7916)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1088%2F1742-5468%2F2008%2F10%2FP10008&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2008&publication_year=2008&pages=P10008&journal=J.%C2%A0Stat.+Mech.+Theor.+Exp.&author=Blondel+V.D.&author=Guillaume+J.-L.&author=Lambiotte+R.&author=Lefebvre+E.&title=Fast+unfolding+of+communities+in+large+networks">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Blumenberg L. </li><li class="ref__authors__name">Kawaler E. </li><li class="ref__authors__name">Cornwell M. </li><li class="ref__authors__name">Smith S. </li><li class="ref__authors__name">Ruggles K. </li><li class="ref__authors__name">Fenyö D. </li></ul><div class="ref__title">BlackSheep: a bioconductor and bioconda package for differential extreme value analysis (Bioinformatics).</div><span class="refDate">2019</span><div class="ref__controls"><a href="#back-bib16" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Blumenberg+L.&author=Kawaler+E.&author=Cornwell+M.&author=Smith+S.&author=Ruggles+K.&author=Feny%C3%B6+D.&title=BlackSheep%3A+a+bioconductor+and+bioconda+package+for+differential+extreme+value+analysis+%28Bioinformatics%29">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brat D.J. </li><li class="ref__authors__name">Aldape K. </li><li class="ref__authors__name">Colman H. </li><li class="ref__authors__name">Holland E.C. </li><li class="ref__authors__name">Louis D.N. </li><li class="ref__authors__name">Jenkins R.B. </li><li class="ref__authors__name">Kleinschmidt-DeMasters B.K. </li><li class="ref__authors__name">Perry A. </li><li class="ref__authors__name">Reifenberger G. </li><li class="ref__authors__name">Stupp R. </li><li>et al.</li></ul><div class="ref__title">cIMPACT-NOW update 3: recommended diagnostic criteria for “Diffuse astrocytic glioma, IDH-wildtype, with molecular features of glioblastoma, WHO grade IV.</div><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">136</span></span>: <span class="ref__seriesPages">805-810</span><div class="ref__controls"><a href="#back-bib17" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85053870232&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30259105&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-018-1913-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2018&pages=805-810&journal=Acta+Neuropathol.&author=Brat+D.J.&author=Aldape+K.&author=Colman+H.&author=Holland+E.C.&author=Louis+D.N.&author=Jenkins+R.B.&author=Kleinschmidt-DeMasters+B.K.&author=Perry+A.&author=Reifenberger+G.&author=Stupp+R.&title=cIMPACT-NOW+update+3%3A+recommended+diagnostic+criteria+for+%E2%80%9CDiffuse+astrocytic+glioma%2C+IDH-wildtype%2C+with+molecular+features+of+glioblastoma%2C+WHO+grade+IV">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brennan C.W. </li><li class="ref__authors__name">Verhaak R.G.W. </li><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Campos B. </li><li class="ref__authors__name">Noushmehr H. </li><li class="ref__authors__name">Salama S.R. </li><li class="ref__authors__name">Zheng S. </li><li class="ref__authors__name">Chakravarty D. </li><li class="ref__authors__name">Sanborn J.Z. </li><li class="ref__authors__name">Berman S.H. </li><li>et al.</li></ul><div class="ref__title">The somatic genomic landscape of glioblastoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">155</span></span>: <span class="ref__seriesPages">462-477</span><div class="ref__controls"><a href="#back-bib18" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84885074034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2249)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24120142&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2013.09.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=462-477&journal=Cell&author=Brennan+C.W.&author=Verhaak+R.G.W.&author=McKenna+A.&author=Campos+B.&author=Noushmehr+H.&author=Salama+S.R.&author=Zheng+S.&author=Chakravarty+D.&author=Sanborn+J.Z.&author=Berman+S.H.&title=The+somatic+genomic+landscape+of+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Brunet J.-P. </li><li class="ref__authors__name">Tamayo P. </li><li class="ref__authors__name">Golub T.R. </li><li class="ref__authors__name">Mesirov J.P. </li></ul><div class="ref__title">Metagenes and molecular pattern discovery using matrix factorization.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">101</span></span>: <span class="ref__seriesPages">4164-4169</span><div class="ref__controls"><a href="#back-bib19" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-1642529511&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1043)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15016911&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.0308531101&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4164-4169&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Brunet+J.-P.&author=Tamayo+P.&author=Golub+T.R.&author=Mesirov+J.P.&title=Metagenes+and+molecular+pattern+discovery+using+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Butler A. </li><li class="ref__authors__name">Hoffman P. </li><li class="ref__authors__name">Smibert P. </li><li class="ref__authors__name">Papalexi E. </li><li class="ref__authors__name">Satija Rahul </li></ul><div class="ref__title">Integrating single-cell transcriptomic data across different conditions, technologies, and species.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">36</span></span>: <span class="ref__seriesPages">411-420</span><div class="ref__controls"><a href="#back-bib20" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85046298440&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1751)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29608179&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.4096&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=411-420&journal=Nat.+Biotechnol.&author=Butler+A.&author=Hoffman+P.&author=Smibert+P.&author=Papalexi+E.&author=Satija+Rahul&title=Integrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+species">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Butowski N. </li><li class="ref__authors__name">Colman H. </li><li class="ref__authors__name">De Groot J.F. </li><li class="ref__authors__name">Omuro A.M. </li><li class="ref__authors__name">Nayak L. </li><li class="ref__authors__name">Wen P.Y. </li><li class="ref__authors__name">Cloughesy T.F. </li><li class="ref__authors__name">Marimuthu A. </li><li class="ref__authors__name">Haidar S. </li><li class="ref__authors__name">Perry A. </li><li>et al.</li></ul><div class="ref__title">Orally administered colony stimulating factor 1 receptor inhibitor PLX3397 in recurrent glioblastoma: an Ivy Foundation Early Phase Clinical Trials Consortium phase II study.</div><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">557-564</span><div class="ref__controls"><a href="#back-bib21" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966552295&amp;cf=" onclick="newWindow(this.href);return false">Scopus (194)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26449250&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnov245&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=557-564&journal=Neuro-Oncol.&author=Butowski+N.&author=Colman+H.&author=De+Groot+J.F.&author=Omuro+A.M.&author=Nayak+L.&author=Wen+P.Y.&author=Cloughesy+T.F.&author=Marimuthu+A.&author=Haidar+S.&author=Perry+A.&title=Orally+administered+colony+stimulating+factor+1+receptor+inhibitor+PLX3397+in+recurrent+glioblastoma%3A+an+Ivy+Foundation+Early+Phase+Clinical+Trials+Consortium+phase+II+study">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Capper D. </li><li class="ref__authors__name">Jones D.T.W. </li><li class="ref__authors__name">Sill M. </li><li class="ref__authors__name">Hovestadt V. </li><li class="ref__authors__name">Schrimpf D. </li><li class="ref__authors__name">Sturm D. </li><li class="ref__authors__name">Koelsche C. </li><li class="ref__authors__name">Sahm F. </li><li class="ref__authors__name">Chavez L. </li><li class="ref__authors__name">Reuss D.E. </li><li>et al.</li></ul><div class="ref__title">DNA methylation-based classification of central nervous system tumours.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">555</span></span>: <span class="ref__seriesPages">469-474</span><div class="ref__controls"><a href="#back-bib22" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044262739&amp;cf=" onclick="newWindow(this.href);return false">Scopus (573)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29539639&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature26000&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=555&publication_year=2018&pages=469-474&journal=Nature&author=Capper+D.&author=Jones+D.T.W.&author=Sill+M.&author=Hovestadt+V.&author=Schrimpf+D.&author=Sturm+D.&author=Koelsche+C.&author=Sahm+F.&author=Chavez+L.&author=Reuss+D.E.&title=DNA+methylation-based+classification+of+central+nervous+system+tumours">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen Z. </li><li class="ref__authors__name">Hambardzumyan D. </li></ul><div class="ref__title">Immune microenvironment in glioblastoma subtypes.</div><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">1004</span><div class="ref__controls"><a href="#back-bib23" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85046650498&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29867979&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.3389%2Ffimmu.2018.01004&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1004&journal=Front.+Immunol.&author=Chen+Z.&author=Hambardzumyan+D.&title=Immune+microenvironment+in+glioblastoma+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Schulz-Trieglaff O. </li><li class="ref__authors__name">Shaw R. </li><li class="ref__authors__name">Barnes B. </li><li class="ref__authors__name">Schlesinger F. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Cox A.J. </li><li class="ref__authors__name">Kruglyak S. </li><li class="ref__authors__name">Saunders C.T. </li></ul><div class="ref__title">Manta: rapid detection of structural variants and indels for germline and cancer sequencing applications.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">1220-1222</span><div class="ref__controls"><a href="#back-bib24" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966658505&amp;cf=" onclick="newWindow(this.href);return false">Scopus (375)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26647377&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtv710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1220-1222&journal=Bioinformatics&author=Chen+X.&author=Schulz-Trieglaff+O.&author=Shaw+R.&author=Barnes+B.&author=Schlesinger+F.&author=K%C3%A4llberg+M.&author=Cox+A.J.&author=Kruglyak+S.&author=Saunders+C.T.&title=Manta%3A+rapid+detection+of+structural+variants+and+indels+for+germline+and+cancer+sequencing+applications">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Chu A. </li><li class="ref__authors__name">Robertson G. </li><li class="ref__authors__name">Brooks D. </li><li class="ref__authors__name">Mungall A.J. </li><li class="ref__authors__name">Birol I. </li><li class="ref__authors__name">Coope R. </li><li class="ref__authors__name">Ma Y. </li><li class="ref__authors__name">Jones S. </li><li class="ref__authors__name">Marra M.A. </li></ul><div class="ref__title">Large-scale profiling of microRNAs for the cancer genome atlas.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e3</span><div class="ref__controls"><a href="#back-bib25" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84959526423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (56)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26271990&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv808&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e3&journal=Nucleic+Acids+Res.&author=Chu+A.&author=Robertson+G.&author=Brooks+D.&author=Mungall+A.J.&author=Birol+I.&author=Coope+R.&author=Ma+Y.&author=Jones+S.&author=Marra+M.A.&title=Large-scale+profiling+of+microRNAs+for+the+cancer+genome+atlas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Lawrence M.S. </li><li class="ref__authors__name">Carter S.L. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Jaffe D. </li><li class="ref__authors__name">Sougnez C. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Meyerson M. </li><li class="ref__authors__name">Lander E.S. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">Sensitive detection of somatic point mutations in impure and heterogeneous cancer samples.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">213-219</span><div class="ref__controls"><a href="#back-bib26" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84874025843&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2160)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23396013&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnbt.2514&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=213-219&journal=Nat.+Biotechnol.&author=Cibulskis+K.&author=Lawrence+M.S.&author=Carter+S.L.&author=Sivachenko+A.&author=Jaffe+D.&author=Sougnez+C.&author=Gabriel+S.&author=Meyerson+M.&author=Lander+E.S.&author=Getz+G.&title=Sensitive+detection+of+somatic+point+mutations+in+impure+and+heterogeneous+cancer+samples">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Clark D.J. </li><li class="ref__authors__name">Dhanasekaran S.M. </li><li class="ref__authors__name">Petralia F. </li><li class="ref__authors__name">Pan J. </li><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Hu Y. </li><li class="ref__authors__name">da Veiga Leprevost F. </li><li class="ref__authors__name">Reva B. </li><li class="ref__authors__name">Lih T.-S.M. </li><li class="ref__authors__name">Chang H.-Y. </li><li>et al.</li></ul><div class="ref__title">Integrated proteogenomic characterization of clear cell renal cell carcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">179</span></span>: <span class="ref__seriesPages">964-983.e31</span><div class="ref__controls"><a href="#back-bib27" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85074028553&amp;cf=" onclick="newWindow(this.href);return false">Scopus (59)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31675502&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.10.007&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=964-983.e31&journal=Cell&author=Clark+D.J.&author=Dhanasekaran+S.M.&author=Petralia+F.&author=Pan+J.&author=Song+X.&author=Hu+Y.&author=da+Veiga+Leprevost+F.&author=Reva+B.&author=Lih+T.-S.M.&author=Chang+H.-Y.&title=Integrated+proteogenomic+characterization+of+clear+cell+renal+cell+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Garofano L. </li><li class="ref__authors__name">Cava C. </li><li class="ref__authors__name">Garolini D. </li><li class="ref__authors__name">Sabedot T.S. </li><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Pagnotta S.M. </li><li class="ref__authors__name">Castiglioni I. </li><li>et al.</li></ul><div class="ref__title">TCGAbiolinks: an R/Bioconductor package for integrative analysis of TCGA data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">e71</span><div class="ref__controls"><a href="#back-bib28" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84966269471&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26704973&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1507&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=e71&journal=Nucleic+Acids+Res.&author=Colaprico+A.&author=Silva+T.C.&author=Olsen+C.&author=Garofano+L.&author=Cava+C.&author=Garolini+D.&author=Sabedot+T.S.&author=Malta+T.M.&author=Pagnotta+S.M.&author=Castiglioni+I.&title=TCGAbiolinks%3A+an+R%2FBioconductor+package+for+integrative+analysis+of+TCGA+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">Bailey M.H. </li><li class="ref__authors__name">Odom G.J. </li><li class="ref__authors__name">Terkelsen T. </li><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Olsen A.V. </li><li class="ref__authors__name">Cantini L. </li><li class="ref__authors__name">Zinovyev A. </li><li class="ref__authors__name">Barillot E. </li><li>et al.</li></ul><div class="ref__title">Interpreting pathways to discover cancer driver genes with Moonlight.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">69</span><div class="ref__controls"><a href="#back-bib29" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077479923&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31900418&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41467-019-13803-0&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=69&journal=Nat.+Commun.&author=Colaprico+A.&author=Olsen+C.&author=Bailey+M.H.&author=Odom+G.J.&author=Terkelsen+T.&author=Silva+T.C.&author=Olsen+A.V.&author=Cantini+L.&author=Zinovyev+A.&author=Barillot+E.&title=Interpreting+pathways+to+discover+cancer+driver+genes+with+Moonlight">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Cotto K.C. </li><li class="ref__authors__name">Wagner A.H. </li><li class="ref__authors__name">Feng Y.-Y. </li><li class="ref__authors__name">Kiwala S. </li><li class="ref__authors__name">Coffman A.C. </li><li class="ref__authors__name">Spies G. </li><li class="ref__authors__name">Wollam A. </li><li class="ref__authors__name">Spies N.C. </li><li class="ref__authors__name">Griffith O.L. </li><li class="ref__authors__name">Griffith M. </li></ul><div class="ref__title">DGIdb 3.0: a redesign and expansion of the drug-gene interaction database.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D1068-D1073</span><div class="ref__controls"><a href="#back-bib30" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85070670869&amp;cf=" onclick="newWindow(this.href);return false">Scopus (214)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29156001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1143&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1068-D1073&journal=Nucleic+Acids+Res.&author=Cotto+K.C.&author=Wagner+A.H.&author=Feng+Y.-Y.&author=Kiwala+S.&author=Coffman+A.C.&author=Spies+G.&author=Wollam+A.&author=Spies+N.C.&author=Griffith+O.L.&author=Griffith+M.&title=DGIdb+3.0%3A+a+redesign+and+expansion+of+the+drug-gene+interaction+database">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Daily K. </li><li class="ref__authors__name">Ho Sui S.J. </li><li class="ref__authors__name">Schriml L.M. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li class="ref__authors__name">Lotia S. </li><li>et al.</li></ul><div class="ref__title">Molecular, phenotypic, and sample-associated data to describe pluripotent stem cell lines and derivatives.</div><span class="ref__series"><i>Sci. Data.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">170030</span><div class="ref__controls"><a href="#back-bib31" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85016445988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28350385&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fsdata.2017.30&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=170030&journal=Sci.+Data&author=Daily+K.&author=Ho+Sui+S.J.&author=Schriml+L.M.&author=Dexheimer+P.J.&author=Salomonis+N.&author=Schroll+R.&author=Bush+S.&author=Keddache+M.&author=Mayhew+C.&author=Lotia+S.&title=Molecular%2C+phenotypic%2C+and+sample-associated+data+to+describe+pluripotent+stem+cell+lines+and+derivatives">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Delgado-López P.D. </li><li class="ref__authors__name">Corrales-García E.M. </li></ul><div class="ref__title">Survival in glioblastoma: a review on the impact of treatment modalities.</div><span class="ref__series"><i>Clin. Transl. Oncol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1062-1071</span><div class="ref__controls"><a href="#back-bib32" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84960422466&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26960561&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs12094-016-1497-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=1062-1071&journal=Clin.+Transl.+Oncol.&author=Delgado-L%C3%B3pez+P.D.&author=Corrales-Garc%C3%ADa+E.M.&title=Survival+in+glioblastoma%3A+a+review+on+the+impact+of+treatment+modalities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ding Z. </li><li class="ref__authors__name">Mangino M. </li><li class="ref__authors__name">Aviv A. </li><li class="ref__authors__name">Spector T. </li><li class="ref__authors__name">Durbin R. </li></ul><div class="ref__title">Estimating telomere length from whole genome sequence data.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">42</span></span>: <span class="ref__seriesPages">e75</span><div class="ref__controls"><a href="#back-bib33" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84899643560&amp;cf=" onclick="newWindow(this.href);return false">Scopus (60)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24609383&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku181&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=e75&journal=Nucleic+Acids+Res.&author=Ding+Z.&author=Mangino+M.&author=Aviv+A.&author=Spector+T.&author=Durbin+R.&title=Estimating+telomere+length+from+whole+genome+sequence+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Doll S. </li><li class="ref__authors__name">Proneth B. </li><li class="ref__authors__name">Tyurina Y.Y. </li><li class="ref__authors__name">Panzilius E. </li><li class="ref__authors__name">Kobayashi S. </li><li class="ref__authors__name">Ingold I. </li><li class="ref__authors__name">Irmler M. </li><li class="ref__authors__name">Beckers J. </li><li class="ref__authors__name">Aichler M. </li><li class="ref__authors__name">Walch A. </li><li>et al.</li></ul><div class="ref__title">ACSL4 dictates ferroptosis sensitivity by shaping cellular lipid composition.</div><span class="ref__series"><i>Nat. Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">91-98</span><div class="ref__controls"><a href="#back-bib34" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84995468814&amp;cf=" onclick="newWindow(this.href);return false">Scopus (472)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27842070&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnchembio.2239&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=91-98&journal=Nat.+Chem.+Biol.&author=Doll+S.&author=Proneth+B.&author=Tyurina+Y.Y.&author=Panzilius+E.&author=Kobayashi+S.&author=Ingold+I.&author=Irmler+M.&author=Beckers+J.&author=Aichler+M.&author=Walch+A.&title=ACSL4+dictates+ferroptosis+sensitivity+by+shaping+cellular+lipid+composition">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Kawaler E.A. </li><li class="ref__authors__name">Zhou D.C. </li><li class="ref__authors__name">Gritsenko M.A. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Blumenberg L. </li><li class="ref__authors__name">Karpova A. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Satpathy S. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic characterization of endometrial carcinoma.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">180</span></span>: <span class="ref__seriesPages">729-748.e26</span><div class="ref__controls"><a href="#back-bib35" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85079388976&amp;cf=" onclick="newWindow(this.href);return false">Scopus (39)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=32059776&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2020.01.026&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=180&publication_year=2020&pages=729-748.e26&journal=Cell&author=Dou+Y.&author=Kawaler+E.A.&author=Zhou+D.C.&author=Gritsenko+M.A.&author=Huang+C.&author=Blumenberg+L.&author=Karpova+A.&author=Petyuk+V.A.&author=Savage+S.R.&author=Satpathy+S.&title=Proteogenomic+characterization+of+endometrial+carcinoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Duan G. </li><li class="ref__authors__name">Li X. </li><li class="ref__authors__name">Köhn M. </li></ul><div class="ref__title">The human DEPhOsphorylation database DEPOD: a 2015 update.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">43</span></span>: <span class="ref__seriesPages">D531-D535</span><div class="ref__controls"><a href="#back-bib36" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84941037753&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25332398&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgku1009&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D531-D535&journal=Nucleic+Acids+Res.&author=Duan+G.&author=Li+X.&author=K%C3%B6hn+M.&title=The+human+DEPhOsphorylation+database+DEPOD%3A+a+2015+update">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Easton R.M. </li><li class="ref__authors__name">Cho H. </li><li class="ref__authors__name">Roovers K. </li><li class="ref__authors__name">Shineman D.W. </li><li class="ref__authors__name">Mizrahi M. </li><li class="ref__authors__name">Forman M.S. </li><li class="ref__authors__name">Lee V.M.-Y. </li><li class="ref__authors__name">Szabolcs M. </li><li class="ref__authors__name">de Jong R. </li><li class="ref__authors__name">Oltersdorf T. </li><li>et al.</li></ul><div class="ref__title">Role for Akt3/protein kinase Bgamma in attainment of normal brain size.</div><span class="ref__series"><i>Mol. Cell. Biol.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1869-1878</span><div class="ref__controls"><a href="#back-bib37" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044382806&amp;cf=" onclick="newWindow(this.href);return false">Scopus (441)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15713641&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1128%2FMCB.25.5.1869-1878.2005&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2005&pages=1869-1878&journal=Mol.+Cell.+Biol.&author=Easton+R.M.&author=Cho+H.&author=Roovers+K.&author=Shineman+D.W.&author=Mizrahi+M.&author=Forman+M.S.&author=Lee+V.M.-Y.&author=Szabolcs+M.&author=de+Jong+R.&author=Oltersdorf+T.&title=Role+for+Akt3%2Fprotein+kinase+Bgamma+in+attainment+of+normal+brain+size">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Eberharter A. </li><li class="ref__authors__name">Becker P.B. </li></ul><div class="ref__title">Histone acetylation: a switch between repressive and permissive chromatin. Second in review series on chromatin dynamics.</div><span class="ref__series"><i>EMBO Rep.</i> <span class="ref__seriesDate">2002; </span><span class="ref__seriesVolume">3</span></span>: <span class="ref__seriesPages">224-229</span><div class="ref__controls"><a href="#back-bib38" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-0036211013&amp;cf=" onclick="newWindow(this.href);return false">Scopus (592)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=11882541&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fembo-reports%2Fkvf053&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=224-229&journal=EMBO+Rep.&author=Eberharter+A.&author=Becker+P.B.&title=Histone+acetylation%3A+a+switch+between+repressive+and+permissive+chromatin.+Second+in+review+series+on+chromatin+dynamics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Eom G.H. </li><li class="ref__authors__name">Kook H. </li></ul><div class="ref__title">Role of histone deacetylase 2 and its posttranslational modifications in cardiac hypertrophy.</div><span class="ref__series"><i>BMB Rep.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">131-138</span><div class="ref__controls"><a href="#back-bib39" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84928262932&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25388210&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.5483%2FBMBRep.2015.48.3.242&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=131-138&journal=BMB+Rep.&author=Eom+G.H.&author=Kook+H.&title=Role+of+histone+deacetylase+2+and+its+posttranslational+modifications+in+cardiac+hypertrophy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fabregat A. </li><li class="ref__authors__name">Jupe S. </li><li class="ref__authors__name">Matthews L. </li><li class="ref__authors__name">Sidiropoulos K. </li><li class="ref__authors__name">Gillespie M. </li><li class="ref__authors__name">Garapati P. </li><li class="ref__authors__name">Haw R. </li><li class="ref__authors__name">Jassal B. </li><li class="ref__authors__name">Korninger F. </li><li class="ref__authors__name">May B. </li><li>et al.</li></ul><div class="ref__title">The reactome pathway Knowledgebase.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D649-D655</span><div class="ref__controls"><a href="#back-bib40" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040929878&amp;cf=" onclick="newWindow(this.href);return false">Scopus (926)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29145629&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1132&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&journal=Nucleic+Acids+Res.&author=Fabregat+A.&author=Jupe+S.&author=Matthews+L.&author=Sidiropoulos+K.&author=Gillespie+M.&author=Garapati+P.&author=Haw+R.&author=Jassal+B.&author=Korninger+F.&author=May+B.&title=The+reactome+pathway+Knowledgebase">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Fisher S. </li><li class="ref__authors__name">Barry A. </li><li class="ref__authors__name">Abreu J. </li><li class="ref__authors__name">Minie B. </li><li class="ref__authors__name">Nolan J. </li><li class="ref__authors__name">Delorey T.M. </li><li class="ref__authors__name">Young G. </li><li class="ref__authors__name">Fennell T.J. </li><li class="ref__authors__name">Allen A. </li><li class="ref__authors__name">Ambrogio L. </li><li>et al.</li></ul><div class="ref__title">A scalable, fully automated process for construction of sequence-ready human exome targeted capture libraries.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R1</span><div class="ref__controls"><a href="#back-bib41" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-78650762588&amp;cf=" onclick="newWindow(this.href);return false">Scopus (249)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21205303&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fgb-2011-12-1-r1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R1&journal=Genome+Biol.&author=Fisher+S.&author=Barry+A.&author=Abreu+J.&author=Minie+B.&author=Nolan+J.&author=Delorey+T.M.&author=Young+G.&author=Fennell+T.J.&author=Allen+A.&author=Ambrogio+L.&title=A+scalable%2C+fully+automated+process+for+construction+of+sequence-ready+human+exome+targeted+capture+libraries">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Frankish A. </li><li class="ref__authors__name">Diekhans M. </li><li class="ref__authors__name">Ferreira A.-M. </li><li class="ref__authors__name">Johnson R. </li><li class="ref__authors__name">Jungreis I. </li><li class="ref__authors__name">Loveland J. </li><li class="ref__authors__name">Mudge J.M. </li><li class="ref__authors__name">Sisu C. </li><li class="ref__authors__name">Wright J. </li><li class="ref__authors__name">Armstrong J. </li><li>et al.</li></ul><div class="ref__title">GENCODE reference annotation for the human and mouse genomes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">47</span></span>: <span class="ref__seriesPages">D766-D773</span><div class="ref__controls"><a href="#back-bib42" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85059554160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (460)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30357393&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgky955&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D766-D773&journal=Nucleic+Acids+Res.&author=Frankish+A.&author=Diekhans+M.&author=Ferreira+A.-M.&author=Johnson+R.&author=Jungreis+I.&author=Loveland+J.&author=Mudge+J.M.&author=Sisu+C.&author=Wright+J.&author=Armstrong+J.&title=GENCODE+reference+annotation+for+the+human+and+mouse+genomes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao Y. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhao F. </li></ul><div class="ref__title">CIRI: an efficient and unbiased algorithm for de novo circular RNA identification.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">16</span></span>: <span class="ref__seriesPages">4</span><div class="ref__controls"><a href="#back-bib43" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84937578024&amp;cf=" onclick="newWindow(this.href);return false">Scopus (409)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25583365&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-014-0571-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=4&journal=Genome+Biol.&author=Gao+Y.&author=Wang+J.&author=Zhao+F.&title=CIRI%3A+an+efficient+and+unbiased+algorithm+for+de+novo+circular+RNA+identification">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gao Q. </li><li class="ref__authors__name">Liang W.-W. </li><li class="ref__authors__name">Foltz S.M. </li><li class="ref__authors__name">Mutharasu G. </li><li class="ref__authors__name">Jayasinghe R.G. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Liao W.-W. </li><li class="ref__authors__name">Reynolds S.M. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Yao L. </li><li>et al.</li></ul><div class="ref__title">Driver fusions and their implications in the development and treatment of human cancers.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">227-238.e3</span><div class="ref__controls"><a href="#back-bib44" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044744284&amp;cf=" onclick="newWindow(this.href);return false">Scopus (149)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29617662&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.03.050&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=227-238.e3&journal=Cell+Rep.&author=Gao+Q.&author=Liang+W.-W.&author=Foltz+S.M.&author=Mutharasu+G.&author=Jayasinghe+R.G.&author=Cao+S.&author=Liao+W.-W.&author=Reynolds+S.M.&author=Wyczalkowski+M.A.&author=Yao+L.&title=Driver+fusions+and+their+implications+in+the+development+and+treatment+of+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gaschler M.M. </li><li class="ref__authors__name">Stockwell B.R. </li></ul><div class="ref__title">Lipid peroxidation in cell death.</div><span class="ref__series"><i>Biochem. Biophys. Res. Commun.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">482</span></span>: <span class="ref__seriesPages">419-425</span><div class="ref__controls"><a href="#back-bib45" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85012272811&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28212725&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.bbrc.2016.10.086&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=482&publication_year=2017&pages=419-425&journal=Biochem.+Biophys.+Res.+Commun.&author=Gaschler+M.M.&author=Stockwell+B.R.&title=Lipid+peroxidation+in+cell+death">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gaujoux R. </li><li class="ref__authors__name">Seoighe C. </li></ul><div class="ref__title">A flexible R package for nonnegative matrix factorization.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">367</span><div class="ref__controls"><a href="#back-bib46" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954069195&amp;cf=" onclick="newWindow(this.href);return false">Scopus (444)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20598126&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-11-367&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=367&journal=BMC+Bioinformatics&author=Gaujoux+R.&author=Seoighe+C.&title=A+flexible+R+package+for+nonnegative+matrix+factorization">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gibbons B.C. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Payne S.H. </li></ul><div class="ref__title">Correcting systematic bias and instrument measurement drift with mzRefinery.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">31</span></span>: <span class="ref__seriesPages">3838-3840</span><div class="ref__controls"><a href="#back-bib47" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26243018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=3838-3840&journal=Bioinforma.+Oxf.+Engl.&author=Gibbons+B.C.&author=Chambers+M.C.&author=Monroe+M.E.&author=Tabb+D.L.&author=Payne+S.H.&title=Correcting+systematic+bias+and+instrument+measurement+drift+with+mzRefinery">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gu Z. </li><li class="ref__authors__name">Eils R. </li><li class="ref__authors__name">Schlesner M. </li></ul><div class="ref__title">Complex heatmaps reveal patterns and correlations in multidimensional genomic data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">2847-2849</span><div class="ref__controls"><a href="#back-bib48" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84992187637&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1009)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27207943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtw313&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=2847-2849&journal=Bioinformatics&author=Gu+Z.&author=Eils+R.&author=Schlesner+M.&title=Complex+heatmaps+reveal+patterns+and+correlations+in+multidimensional+genomic+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Gu X. </li><li class="ref__authors__name">Hua Z. </li><li class="ref__authors__name">Dong Y. </li><li class="ref__authors__name">Zhan Y. </li><li class="ref__authors__name">Zhang X. </li><li class="ref__authors__name">Tian W. </li><li class="ref__authors__name">Liu Z. </li><li class="ref__authors__name">Thiele C.J. </li><li class="ref__authors__name">Li Z. </li></ul><div class="ref__title">Proteome and acetylome analysis identifies novel pathways and targets regulated by Perifosine in neuroblastoma.</div><span class="ref__series"><i>Sci. Rep.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">42062</span><div class="ref__controls"><a href="#back-bib49" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85011872870&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28165023&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fsrep42062&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42062&journal=Sci.+Rep.&author=Gu+X.&author=Hua+Z.&author=Dong+Y.&author=Zhan+Y.&author=Zhang+X.&author=Tian+W.&author=Liu+Z.&author=Thiele+C.J.&author=Li+Z.&title=Proteome+and+acetylome+analysis+identifies+novel+pathways+and+targets+regulated+by+Perifosine+in+neuroblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Haas B.J. </li><li class="ref__authors__name">Dobin A. </li><li class="ref__authors__name">Li B. </li><li class="ref__authors__name">Stransky N. </li><li class="ref__authors__name">Pochet N. </li><li class="ref__authors__name">Regev A. </li></ul><div class="ref__title">Accuracy assessment of fusion transcript detection via read-mapping and de novo fusion transcript assembly-based methods.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">213</span><div class="ref__controls"><a href="#back-bib50" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85073741338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (51)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31639029&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-019-1842-9&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=213&journal=Genome+Biol.&author=Haas+B.J.&author=Dobin+A.&author=Li+B.&author=Stransky+N.&author=Pochet+N.&author=Regev+A.&title=Accuracy+assessment+of+fusion+transcript+detection+via+read-mapping+and+de+novo+fusion+transcript+assembly-based+methods">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hafemeister C. </li><li class="ref__authors__name">Satija R. </li></ul><div class="ref__title">Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression.</div><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">576827</span><div class="ref__controls"><a href="#back-bib51" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=576827&journal=BioRxiv&author=Hafemeister+C.&author=Satija+R.&title=Normalization+and+variance+stabilization+of+single-cell+RNA-seq+data+using+regularized+negative+binomial+regression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">He H. </li><li class="ref__authors__name">Conrad C.A. </li><li class="ref__authors__name">Nilsson C.L. </li><li class="ref__authors__name">Ji Y. </li><li class="ref__authors__name">Schaub T.M. </li><li class="ref__authors__name">Marshall A.G. </li><li class="ref__authors__name">Emmett M.R. </li></ul><div class="ref__title">Method for lipidomic analysis: p53 expression modulation of sulfatide, ganglioside, and phospholipid composition of U87 MG glioblastoma cells.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">79</span></span>: <span class="ref__seriesPages">8423-8430</span><div class="ref__controls"><a href="#back-bib52" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-36448939111&amp;cf=" onclick="newWindow(this.href);return false">Scopus (54)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17929901&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac071413m&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2007&pages=8423-8430&journal=Anal.+Chem.&author=He+H.&author=Conrad+C.A.&author=Nilsson+C.L.&author=Ji+Y.&author=Schaub+T.M.&author=Marshall+A.G.&author=Emmett+M.R.&title=Method+for+lipidomic+analysis%3A+p53+expression+modulation+of+sulfatide%2C+ganglioside%2C+and+phospholipid+composition+of+U87+MG+glioblastoma+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Hiller K. </li><li class="ref__authors__name">Hangebrauk J. </li><li class="ref__authors__name">Jäger C. </li><li class="ref__authors__name">Spura J. </li><li class="ref__authors__name">Schreiber K. </li><li class="ref__authors__name">Schomburg D. </li></ul><div class="ref__title">MetaboliteDetector: comprehensive analysis tool for targeted and nontargeted GC/MS based metabolome analysis.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">3429-3439</span><div class="ref__controls"><a href="#back-bib53" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-66149088821&amp;cf=" onclick="newWindow(this.href);return false">Scopus (243)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19358599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac802689c&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=3429-3439&journal=Anal.+Chem.&author=Hiller+K.&author=Hangebrauk+J.&author=J%C3%A4ger+C.&author=Spura+J.&author=Schreiber+K.&author=Schomburg+D.&title=MetaboliteDetector%3A+comprehensive+analysis+tool+for+targeted+and+nontargeted+GC%2FMS+based+metabolome+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Horn H. </li><li class="ref__authors__name">Schoof E.M. </li><li class="ref__authors__name">Kim J. </li><li class="ref__authors__name">Robin X. </li><li class="ref__authors__name">Miller M.L. </li><li class="ref__authors__name">Diella F. </li><li class="ref__authors__name">Palma A. </li><li class="ref__authors__name">Cesareni G. </li><li class="ref__authors__name">Jensen L.J. </li><li class="ref__authors__name">Linding R. </li></ul><div class="ref__title">KinomeXplorer: an integrated platform for kinome biology studies.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">603-604</span><div class="ref__controls"><a href="#back-bib54" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84901788851&amp;cf=" onclick="newWindow(this.href);return false">Scopus (152)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24874572&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.2968&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=603-604&journal=Nat.+Methods&author=Horn+H.&author=Schoof+E.M.&author=Kim+J.&author=Robin+X.&author=Miller+M.L.&author=Diella+F.&author=Palma+A.&author=Cesareni+G.&author=Jensen+L.J.&author=Linding+R.&title=KinomeXplorer%3A+an+integrated+platform+for+kinome+biology+studies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Huber W. </li><li class="ref__authors__name">Carey V.J. </li><li class="ref__authors__name">Gentleman R. </li><li class="ref__authors__name">Anders S. </li><li class="ref__authors__name">Carlson M. </li><li class="ref__authors__name">Carvalho B.S. </li><li class="ref__authors__name">Bravo H.C. </li><li class="ref__authors__name">Davis S. </li><li class="ref__authors__name">Gatto L. </li><li class="ref__authors__name">Girke T. </li><li>et al.</li></ul><div class="ref__title">Orchestrating high-throughput genomic analysis with Bioconductor.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">115-121</span><div class="ref__controls"><a href="#back-bib55" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84961289551&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1269)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25633503&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.3252&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=115-121&journal=Nat.+Methods&author=Huber+W.&author=Carey+V.J.&author=Gentleman+R.&author=Anders+S.&author=Carlson+M.&author=Carvalho+B.S.&author=Bravo+H.C.&author=Davis+S.&author=Gatto+L.&author=Girke+T.&title=Orchestrating+high-throughput+genomic+analysis+with+Bioconductor">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kagan V.E. </li><li class="ref__authors__name">Mao G. </li><li class="ref__authors__name">Qu F. </li><li class="ref__authors__name">Angeli J.P.F. </li><li class="ref__authors__name">Doll S. </li><li class="ref__authors__name">Croix C.S. </li><li class="ref__authors__name">Dar H.H. </li><li class="ref__authors__name">Liu B. </li><li class="ref__authors__name">Tyurin V.A. </li><li class="ref__authors__name">Ritov V.B. </li><li>et al.</li></ul><div class="ref__title">Oxidized arachidonic and adrenic PEs navigate cells to ferroptosis.</div><span class="ref__series"><i>Nat. Chem. Biol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">81-90</span><div class="ref__controls"><a href="#back-bib56" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84995452435&amp;cf=" onclick="newWindow(this.href);return false">Scopus (429)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27842066&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnchembio.2238&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=81-90&journal=Nat.+Chem.+Biol.&author=Kagan+V.E.&author=Mao+G.&author=Qu+F.&author=Angeli+J.P.F.&author=Doll+S.&author=Croix+C.S.&author=Dar+H.H.&author=Liu+B.&author=Tyurin+V.A.&author=Ritov+V.B.&title=Oxidized+arachidonic+and+adrenic+PEs+navigate+cells+to+ferroptosis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Karczewski K.J. </li><li class="ref__authors__name">Francioli L.C. </li><li class="ref__authors__name">Tiao G. </li><li class="ref__authors__name">Cummings B.B. </li><li class="ref__authors__name">Alföldi J. </li><li class="ref__authors__name">Wang Q. </li><li class="ref__authors__name">Collins R.L. </li><li class="ref__authors__name">Laricchia K.M. </li><li class="ref__authors__name">Ganna A. </li><li class="ref__authors__name">Birnbaum D.P. </li><li>et al.</li></ul><div class="ref__title">The mutational constraint spectrum quantified from variation in 141,456 humans.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2020; </span><span class="ref__seriesVolume">581</span></span>: <span class="ref__seriesPages">434-443</span><div class="ref__controls"><a href="#back-bib57" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85085542423&amp;cf=" onclick="newWindow(this.href);return false">Scopus (613)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=32461654&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41586-020-2308-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=581&publication_year=2020&pages=434-443&journal=Nature&author=Karczewski+K.J.&author=Francioli+L.C.&author=Tiao+G.&author=Cummings+B.B.&author=Alf%C3%B6ldi+J.&author=Wang+Q.&author=Collins+R.L.&author=Laricchia+K.M.&author=Ganna+A.&author=Birnbaum+D.P.&title=The+mutational+constraint+spectrum+quantified+from+variation+in+141%2C456+humans">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Karimi S. </li><li class="ref__authors__name">Zuccato J.A. </li><li class="ref__authors__name">Mamatjan Y. </li><li class="ref__authors__name">Mansouri S. </li><li class="ref__authors__name">Suppiah S. </li><li class="ref__authors__name">Nassiri F. </li><li class="ref__authors__name">Diamandis P. </li><li class="ref__authors__name">Munoz D.G. </li><li class="ref__authors__name">Aldape K.D. </li><li class="ref__authors__name">Zadeh G. </li></ul><div class="ref__title">The central nervous system tumor methylation classifier changes neuro-oncology practice for challenging brain tumor diagnoses and directly impacts patient care.</div><span class="ref__series"><i>Clin. Epigenetics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">185</span><div class="ref__controls"><a href="#back-bib58" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85076173211&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31806041&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13148-019-0766-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=185&journal=Clin.+Epigenetics&author=Karimi+S.&author=Zuccato+J.A.&author=Mamatjan+Y.&author=Mansouri+S.&author=Suppiah+S.&author=Nassiri+F.&author=Diamandis+P.&author=Munoz+D.G.&author=Aldape+K.D.&author=Zadeh+G.&title=The+central+nervous+system+tumor+methylation+classifier+changes+neuro-oncology+practice+for+challenging+brain+tumor+diagnoses+and+directly+impacts+patient+care">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Keenan A.B. </li><li class="ref__authors__name">Jenkins S.L. </li><li class="ref__authors__name">Jagodnik K.M. </li><li class="ref__authors__name">Koplev S. </li><li class="ref__authors__name">He E. </li><li class="ref__authors__name">Torre D. </li><li class="ref__authors__name">Wang Z. </li><li class="ref__authors__name">Dohlman A.B. </li><li class="ref__authors__name">Silverstein M.C. </li><li class="ref__authors__name">Lachmann A. </li><li>et al.</li></ul><div class="ref__title">The library of integrated network-based cellular signatures NIH program: system-level cataloging of human cells response to perturbations.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">13-24</span><div class="ref__controls"><a href="#back-bib59" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036553710&amp;cf=" onclick="newWindow(this.href);return false">Scopus (115)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29199020&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2017.11.001&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=13-24&journal=Cell+Syst.&author=Keenan+A.B.&author=Jenkins+S.L.&author=Jagodnik+K.M.&author=Koplev+S.&author=He+E.&author=Torre+D.&author=Wang+Z.&author=Dohlman+A.B.&author=Silverstein+M.C.&author=Lachmann+A.&title=The+library+of+integrated+network-based+cellular+signatures+NIH+program%3A+system-level+cataloging+of+human+cells+response+to+perturbations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Killela P.J. </li><li class="ref__authors__name">Reitman Z.J. </li><li class="ref__authors__name">Jiao Y. </li><li class="ref__authors__name">Bettegowda C. </li><li class="ref__authors__name">Agrawal N. </li><li class="ref__authors__name">Diaz L.A. </li><li class="ref__authors__name">Friedman A.H. </li><li class="ref__authors__name">Friedman H. </li><li class="ref__authors__name">Gallia G.L. </li><li class="ref__authors__name">Giovanella B.C. </li><li>et al.</li></ul><div class="ref__title">TERT promoter mutations occur frequently in gliomas and a subset of tumors derived from cells with low rates of self-renewal.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">110</span></span>: <span class="ref__seriesPages">6021-6026</span><div class="ref__controls"><a href="#back-bib60" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84876067164&amp;cf=" onclick="newWindow(this.href);return false">Scopus (819)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23530248&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.1303607110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=6021-6026&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Killela+P.J.&author=Reitman+Z.J.&author=Jiao+Y.&author=Bettegowda+C.&author=Agrawal+N.&author=Diaz+L.A.&author=Friedman+A.H.&author=Friedman+H.&author=Gallia+G.L.&author=Giovanella+B.C.&title=TERT+promoter+mutations+occur+frequently+in+gliomas+and+a+subset+of+tumors+derived+from+cells+with+low+rates+of+self-renewal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Park H. </li></ul><div class="ref__title">Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">23</span></span>: <span class="ref__seriesPages">1495-1502</span><div class="ref__controls"><a href="#back-bib61" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-34547844077&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17483501&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtm134&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=1495-1502&journal=Bioinforma.+Oxf.+Engl.&author=Kim+H.&author=Park+H.&title=Sparse+non-negative+matrix+factorizations+via+alternating+non-negativity-constrained+least+squares+for+microarray+data+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">MS-GF+ makes progress towards a universal database search tool for proteomics.</div><span class="ref__series"><i>Nat. Commun.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">5277</span><div class="ref__controls"><a href="#back-bib62" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84923247212&amp;cf=" onclick="newWindow(this.href);return false">Scopus (469)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25358478&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fncomms6277&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=5277&journal=Nat.+Commun.&author=Kim+S.&author=Pevzner+P.A.&title=MS-GF%2B+makes+progress+towards+a+universal+database+search+tool+for+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Gupta N. </li><li class="ref__authors__name">Pevzner P.A. </li></ul><div class="ref__title">Spectral probabilities and generating functions of tandem mass spectra: a strike against decoy databases.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">3354-3363</span><div class="ref__controls"><a href="#back-bib63" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-50149117169&amp;cf=" onclick="newWindow(this.href);return false">Scopus (292)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18597511&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fpr8001244&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=3354-3363&journal=J.%C2%A0Proteome+Res.&author=Kim+S.&author=Gupta+N.&author=Pevzner+P.A.&title=Spectral+probabilities+and+generating+functions+of+tandem+mass+spectra%3A+a+strike+against+decoy+databases">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Scheffler K. </li><li class="ref__authors__name">Halpern A.L. </li><li class="ref__authors__name">Bekritsky M.A. </li><li class="ref__authors__name">Noh E. </li><li class="ref__authors__name">Källberg M. </li><li class="ref__authors__name">Chen X. </li><li class="ref__authors__name">Kim Y. </li><li class="ref__authors__name">Beyter D. </li><li class="ref__authors__name">Krusche P. </li><li>et al.</li></ul><div class="ref__title">Strelka2: fast and accurate calling of germline and somatic variants.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">591-594</span><div class="ref__controls"><a href="#back-bib64" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85049953163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (172)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30013048&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41592-018-0051-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=591-594&journal=Nat.+Methods&author=Kim+S.&author=Scheffler+K.&author=Halpern+A.L.&author=Bekritsky+M.A.&author=Noh+E.&author=K%C3%A4llberg+M.&author=Chen+X.&author=Kim+Y.&author=Beyter+D.&author=Krusche+P.&title=Strelka2%3A+fast+and+accurate+calling+of+germline+and+somatic+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kind T. </li><li class="ref__authors__name">Wohlgemuth G. </li><li class="ref__authors__name">Lee D.Y. </li><li class="ref__authors__name">Lu Y. </li><li class="ref__authors__name">Palazoglu M. </li><li class="ref__authors__name">Shahbaz S. </li><li class="ref__authors__name">Fiehn O. </li></ul><div class="ref__title">FiehnLib: mass spectral and retention index libraries for metabolomics based on quadrupole and time-of-flight gas chromatography/mass spectrometry.</div><span class="ref__series"><i>Anal. Chem.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">81</span></span>: <span class="ref__seriesPages">10038-10048</span><div class="ref__controls"><a href="#back-bib65" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-72449186517&amp;cf=" onclick="newWindow(this.href);return false">Scopus (759)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19928838&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Fac9019522&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=10038-10048&journal=Anal.+Chem.&author=Kind+T.&author=Wohlgemuth+G.&author=Lee+D.Y.&author=Lu+Y.&author=Palazoglu+M.&author=Shahbaz+S.&author=Fiehn+O.&title=FiehnLib%3A+mass+spectral+and+retention+index+libraries+for+metabolomics+based+on+quadrupole+and+time-of-flight+gas+chromatography%2Fmass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Koboldt D.C. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Larson D.E. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">McLellan M.D. </li><li class="ref__authors__name">Lin L. </li><li class="ref__authors__name">Miller C.A. </li><li class="ref__authors__name">Mardis E.R. </li><li class="ref__authors__name">Ding L. </li><li class="ref__authors__name">Wilson R.K. </li></ul><div class="ref__title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">568-576</span><div class="ref__controls"><a href="#back-bib66" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84863229597&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2216)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22300766&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.129684.111&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=568-576&journal=Genome+Res.&author=Koboldt+D.C.&author=Zhang+Q.&author=Larson+D.E.&author=Shen+D.&author=McLellan+M.D.&author=Lin+L.&author=Miller+C.A.&author=Mardis+E.R.&author=Ding+L.&author=Wilson+R.K.&title=VarScan+2%3A+somatic+mutation+and+copy+number+alteration+discovery+in+cancer+by+exome+sequencing">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Köster J. </li><li class="ref__authors__name">Rahmann S. </li></ul><div class="ref__title">Snakemake—a scalable bioinformatics workflow engine.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">2520-2522</span><div class="ref__controls"><a href="#back-bib67" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84867306721&amp;cf=" onclick="newWindow(this.href);return false">Scopus (597)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22908215&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts480&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=2520-2522&journal=Bioinformatics&author=K%C3%B6ster+J.&author=Rahmann+S.&title=Snakemake%E2%80%94a+scalable+bioinformatics+workflow+engine">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kubala M.H. </li><li class="ref__authors__name">Punj V. </li><li class="ref__authors__name">Placencio-Hickok V.R. </li><li class="ref__authors__name">Fang H. </li><li class="ref__authors__name">Fernandez G.E. </li><li class="ref__authors__name">Sposto R. </li><li class="ref__authors__name">DeClerck Y.A. </li></ul><div class="ref__title">Plasminogen activator inhibitor-1 promotes the recruitment and polarization of macrophages in cancer.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2177-2191.e7</span><div class="ref__controls"><a href="#back-bib68" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85056741174&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30463014&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2018.10.082&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2177-2191.e7&journal=Cell+Rep&author=Kubala+M.H.&author=Punj+V.&author=Placencio-Hickok+V.R.&author=Fang+H.&author=Fernandez+G.E.&author=Sposto+R.&author=DeClerck+Y.A.&title=Plasminogen+activator+inhibitor-1+promotes+the+recruitment+and+polarization+of+macrophages+in+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kunze K. </li><li class="ref__authors__name">Spieker T. </li><li class="ref__authors__name">Gamerdinger U. </li><li class="ref__authors__name">Nau K. </li><li class="ref__authors__name">Berger J. </li><li class="ref__authors__name">Dreyer T. </li><li class="ref__authors__name">Sindermann J.R. </li><li class="ref__authors__name">Hoffmeier A. </li><li class="ref__authors__name">Gattenlöhner S. </li><li class="ref__authors__name">Bräuninger A. </li></ul><div class="ref__title">A recurrent activating PLCG1 mutation in cardiac angiosarcomas increases apoptosis resistance and invasiveness of endothelial cells.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">74</span></span>: <span class="ref__seriesPages">6173-6183</span><div class="ref__controls"><a href="#back-bib69" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84910020229&amp;cf=" onclick="newWindow(this.href);return false">Scopus (33)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25252913&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-14-1162&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=6173-6183&journal=Cancer+Res.&author=Kunze+K.&author=Spieker+T.&author=Gamerdinger+U.&author=Nau+K.&author=Berger+J.&author=Dreyer+T.&author=Sindermann+J.R.&author=Hoffmeier+A.&author=Gattenl%C3%B6hner+S.&author=Br%C3%A4uninger+A.&title=A+recurrent+activating+PLCG1+mutation+in+cardiac+angiosarcomas+increases+apoptosis+resistance+and+invasiveness+of+endothelial+cells">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Kyle J.E. </li><li class="ref__authors__name">Crowell K.L. </li><li class="ref__authors__name">Casey C.P. </li><li class="ref__authors__name">Fujimoto G.M. </li><li class="ref__authors__name">Kim S. </li><li class="ref__authors__name">Dautel S.E. </li><li class="ref__authors__name">Smith R.D. </li><li class="ref__authors__name">Payne S.H. </li><li class="ref__authors__name">Metz T.O. </li></ul><div class="ref__title">LIQUID: an-open source software for identifying lipids in LC-MS/MS-based lipidomics data.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">1744-1746</span><div class="ref__controls"><a href="#back-bib70" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85021049695&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28158427&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtx046&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=1744-1746&journal=Bioinforma.+Oxf.+Engl.&author=Kyle+J.E.&author=Crowell+K.L.&author=Casey+C.P.&author=Fujimoto+G.M.&author=Kim+S.&author=Dautel+S.E.&author=Smith+R.D.&author=Payne+S.H.&author=Metz+T.O.&title=LIQUID%3A+an-open+source+software+for+identifying+lipids+in+LC-MS%2FMS-based+lipidomics+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Langmead B. </li><li class="ref__authors__name">Salzberg S.L. </li></ul><div class="ref__title">Fast gapped-read alignment with Bowtie 2.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">9</span></span>: <span class="ref__seriesPages">357-359</span><div class="ref__controls"><a href="#back-bib71" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84859210032&amp;cf=" onclick="newWindow(this.href);return false">Scopus (17228)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22388286&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.1923&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=357-359&journal=Nat.+Methods&author=Langmead+B.&author=Salzberg+S.L.&title=Fast+gapped-read+alignment+with+Bowtie+2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">LeRoy G. </li><li class="ref__authors__name">Rickards B. </li><li class="ref__authors__name">Flint S.J. </li></ul><div class="ref__title">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription.</div><span class="ref__series"><i>Mol. Cell.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">30</span></span>: <span class="ref__seriesPages">51-60</span><div class="ref__controls"><a href="#back-bib72" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-41549115095&amp;cf=" onclick="newWindow(this.href);return false">Scopus (221)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18406326&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.molcel.2008.01.018&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&journal=Mol.+Cell&author=LeRoy+G.&author=Rickards+B.&author=Flint+S.J.&title=The+double+bromodomain+proteins+Brd2+and+Brd3+couple+histone+acetylation+to+transcription">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li B. </li><li class="ref__authors__name">Dewey C.N. </li></ul><div class="ref__title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">323</span><div class="ref__controls"><a href="#back-bib73" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79961123152&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6978)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21816040&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-12-323&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=323&journal=BMC+Bioinformatics&author=Li+B.&author=Dewey+C.N.&title=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Durbin R. </li></ul><div class="ref__title">Fast and accurate short read alignment with Burrows-Wheeler transform.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">1754-1760</span><div class="ref__controls"><a href="#back-bib74" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-67649884743&amp;cf=" onclick="newWindow(this.href);return false">Scopus (19451)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19451168&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp324&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=1754-1760&journal=Bioinforma.+Oxf.+Engl.&author=Li+H.&author=Durbin+R.&title=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li H. </li><li class="ref__authors__name">Handsaker B. </li><li class="ref__authors__name">Wysoker A. </li><li class="ref__authors__name">Fennell T. </li><li class="ref__authors__name">Ruan J. </li><li class="ref__authors__name">Homer N. </li><li class="ref__authors__name">Marth G. </li><li class="ref__authors__name">Abecasis G. </li><li class="ref__authors__name">Durbin R. </li><li class="ref__authors__name">1000 Genome Project Data Processing Subgroup </li></ul><div class="ref__title">The sequence alignment/map format and SAMtools.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2078-2079</span><div class="ref__controls"><a href="#back-bib75" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-68549104404&amp;cf=" onclick="newWindow(this.href);return false">Scopus (21968)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19505943&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp352&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2078-2079&journal=Bioinforma.+Oxf.+Engl.&author=Li+H.&author=Handsaker+B.&author=Wysoker+A.&author=Fennell+T.&author=Ruan+J.&author=Homer+N.&author=Marth+G.&author=Abecasis+G.&author=Durbin+R.&author=1000+Genome+Project+Data+Processing+Subgroup+&title=The+sequence+alignment%2Fmap+format+and+SAMtools">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li S. </li><li class="ref__authors__name">Shen D. </li><li class="ref__authors__name">Shao J. </li><li class="ref__authors__name">Crowder R. </li><li class="ref__authors__name">Liu W. </li><li class="ref__authors__name">Prat A. </li><li class="ref__authors__name">He X. </li><li class="ref__authors__name">Liu S. </li><li class="ref__authors__name">Hoog J. </li><li class="ref__authors__name">Lu C. </li><li>et al.</li></ul><div class="ref__title">Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts.</div><span class="ref__series"><i>Cell Rep.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1116-1130</span><div class="ref__controls"><a href="#back-bib76" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84884559238&amp;cf=" onclick="newWindow(this.href);return false">Scopus (376)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24055055&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.celrep.2013.08.022&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1116-1130&journal=Cell+Rep&author=Li+S.&author=Shen+D.&author=Shao+J.&author=Crowder+R.&author=Liu+W.&author=Prat+A.&author=He+X.&author=Liu+S.&author=Hoog+J.&author=Lu+C.&title=Endocrine-therapy-resistant+ESR1+variants+revealed+by+genomic+characterization+of+breast-cancer-derived+xenografts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Li M. </li><li class="ref__authors__name">Xie X. </li><li class="ref__authors__name">Zhou J. </li><li class="ref__authors__name">Sheng M. </li><li class="ref__authors__name">Yin X. </li><li class="ref__authors__name">Ko E.-A. </li><li class="ref__authors__name">Zhou T. </li><li class="ref__authors__name">Gu W. </li></ul><div class="ref__title">Quantifying circular RNA expression from RNA-seq data using model-based framework.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">33</span></span>: <span class="ref__seriesPages">2131-2139</span><div class="ref__controls"><a href="#back-bib77" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85020877053&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28334396&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtx129&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2131-2139&journal=Bioinforma.+Oxf.+Engl.&author=Li+M.&author=Xie+X.&author=Zhou+J.&author=Sheng+M.&author=Yin+X.&author=Ko+E.-A.&author=Zhou+T.&author=Gu+W.&title=Quantifying+circular+RNA+expression+from+RNA-seq+data+using+model-based+framework">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liberzon A. </li><li class="ref__authors__name">Birger C. </li><li class="ref__authors__name">Thorvaldsdóttir H. </li><li class="ref__authors__name">Ghandi M. </li><li class="ref__authors__name">Mesirov J.P. </li><li class="ref__authors__name">Tamayo P. </li></ul><div class="ref__title">The Molecular Signatures Database (MSigDB) hallmark gene set collection.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">417-425</span><div class="ref__controls"><a href="#back-bib78" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84955328286&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1644)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26771021&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=417-425&journal=Cell+Syst&author=Liberzon+A.&author=Birger+C.&author=Thorvaldsd%C3%B3ttir+H.&author=Ghandi+M.&author=Mesirov+J.P.&author=Tamayo+P.&title=The+Molecular+Signatures+Database+%28MSigDB%29+hallmark+gene+set+collection">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Licata L. </li><li class="ref__authors__name">Lo Surdo P. </li><li class="ref__authors__name">Iannuccelli M. </li><li class="ref__authors__name">Palma A. </li><li class="ref__authors__name">Micarelli E. </li><li class="ref__authors__name">Perfetto L. </li><li class="ref__authors__name">Peluso D. </li><li class="ref__authors__name">Calderone A. </li><li class="ref__authors__name">Castagnoli L. </li><li class="ref__authors__name">Cesareni G. </li></ul><div class="ref__title">SIGNOR 2.0, the SIGnaling network open resource 2.0: 2019 update.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D504-D510</span><div class="ref__controls"><a href="#back-bib79" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D504-D510&journal=Nucleic+Acids+Res.&author=Licata+L.&author=Lo+Surdo+P.&author=Iannuccelli+M.&author=Palma+A.&author=Micarelli+E.&author=Perfetto+L.&author=Peluso+D.&author=Calderone+A.&author=Castagnoli+L.&author=Cesareni+G.&title=SIGNOR+2.0%2C+the+SIGnaling+network+open+resource+2.0%3A+2019+update">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Lim M. </li><li class="ref__authors__name">Xia Y. </li><li class="ref__authors__name">Bettegowda C. </li><li class="ref__authors__name">Weller M. </li></ul><div class="ref__title">Current state of immunotherapy for glioblastoma.</div><span class="ref__series"><i>Nat. Rev. Clin. Oncol.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">422</span><div class="ref__controls"><a href="#back-bib80" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85045203677&amp;cf=" onclick="newWindow(this.href);return false">Scopus (234)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29643471&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41571-018-0003-5&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=422&journal=Nat.+Rev.+Clin.+Oncol.&author=Lim+M.&author=Xia+Y.&author=Bettegowda+C.&author=Weller+M.&title=Current+state+of+immunotherapy+for+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ling S. </li><li class="ref__authors__name">Chang X. </li><li class="ref__authors__name">Schultz L. </li><li class="ref__authors__name">Lee T.K. </li><li class="ref__authors__name">Chaux A. </li><li class="ref__authors__name">Marchionni L. </li><li class="ref__authors__name">Netto G.J. </li><li class="ref__authors__name">Sidransky D. </li><li class="ref__authors__name">Berman D.M. </li></ul><div class="ref__title">An EGFR-ERK-SOX9 signaling cascade links urothelial development and regeneration to cancer.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">71</span></span>: <span class="ref__seriesPages">3812-3821</span><div class="ref__controls"><a href="#back-bib81" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79957883840&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21512138&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-10-3072&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=3812-3821&journal=Cancer+Res.&author=Ling+S.&author=Chang+X.&author=Schultz+L.&author=Lee+T.K.&author=Chaux+A.&author=Marchionni+L.&author=Netto+G.J.&author=Sidransky+D.&author=Berman+D.M.&title=An+EGFR-ERK-SOX9+signaling+cascade+links+urothelial+development+and+regeneration+to+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Litichevskiy L. </li><li class="ref__authors__name">Peckner R. </li><li class="ref__authors__name">Abelin J.G. </li><li class="ref__authors__name">Asiedu J.K. </li><li class="ref__authors__name">Creech A.L. </li><li class="ref__authors__name">Davis J.F. </li><li class="ref__authors__name">Davison D. </li><li class="ref__authors__name">Dunning C.M. </li><li class="ref__authors__name">Egertson J.D. </li><li class="ref__authors__name">Egri S. </li><li>et al.</li></ul><div class="ref__title">A library of phosphoproteomic and chromatin signatures for characterizing cellular responses to drug perturbations.</div><span class="ref__series"><i>Cell Syst.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">6</span></span>: <span class="ref__seriesPages">424-443.e7</span><div class="ref__controls"><a href="#back-bib82" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85045118030&amp;cf=" onclick="newWindow(this.href);return false">Scopus (26)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29655704&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cels.2018.03.012&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=424-443.e7&journal=Cell+Syst&author=Litichevskiy+L.&author=Peckner+R.&author=Abelin+J.G.&author=Asiedu+J.K.&author=Creech+A.L.&author=Davis+J.F.&author=Davison+D.&author=Dunning+C.M.&author=Egertson+J.D.&author=Egri+S.&title=A+library+of+phosphoproteomic+and+chromatin+signatures+for+characterizing+cellular+responses+to+drug+perturbations">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Liu L. </li><li class="ref__authors__name">Shi Y. </li><li class="ref__authors__name">Shi J. </li><li class="ref__authors__name">Wang H. </li><li class="ref__authors__name">Sheng Y. </li><li class="ref__authors__name">Jiang Q. </li><li class="ref__authors__name">Chen H. </li><li class="ref__authors__name">Li X. </li><li class="ref__authors__name">Dong J. </li></ul><div class="ref__title">The long non-coding RNA SNHG1 promotes glioma progression by competitively binding to miR-194 to regulate PHLDA1 expression.</div><span class="ref__series"><i>Cell Death Dis.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">463</span><div class="ref__controls"><a href="#back-bib83" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85067299482&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31189920&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41419-019-1698-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=463&journal=Cell+Death+Dis.&author=Liu+L.&author=Shi+Y.&author=Shi+J.&author=Wang+H.&author=Sheng+Y.&author=Jiang+Q.&author=Chen+H.&author=Li+X.&author=Dong+J.&title=The+long+non-coding+RNA+SNHG1+promotes+glioma+progression+by+competitively+binding+to+miR-194+to+regulate+PHLDA1+expression">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Louis D.N. </li><li class="ref__authors__name">Perry A. </li><li class="ref__authors__name">Reifenberger G. </li><li class="ref__authors__name">von Deimling A. </li><li class="ref__authors__name">Figarella-Branger D. </li><li class="ref__authors__name">Cavenee W.K. </li><li class="ref__authors__name">Ohgaki H. </li><li class="ref__authors__name">Wiestler O.D. </li><li class="ref__authors__name">Kleihues P. </li><li class="ref__authors__name">Ellison D.W. </li></ul><div class="ref__title">The 2016 World health organization classification of tumors of the central nervous system: a summary.</div><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">131</span></span>: <span class="ref__seriesPages">803-820</span><div class="ref__controls"><a href="#back-bib84" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27157931&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-016-1545-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2016&pages=803-820&journal=Acta+Neuropathol.&author=Louis+D.N.&author=Perry+A.&author=Reifenberger+G.&author=von+Deimling+A.&author=Figarella-Branger+D.&author=Cavenee+W.K.&author=Ohgaki+H.&author=Wiestler+O.D.&author=Kleihues+P.&author=Ellison+D.W.&title=The+2016+World+health+organization+classification+of+tumors+of+the+central+nervous+system%3A+a+summary">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Louis D.N. </li><li class="ref__authors__name">Aldape K. </li><li class="ref__authors__name">Brat D.J. </li><li class="ref__authors__name">Capper D. </li><li class="ref__authors__name">Ellison D.W. </li><li class="ref__authors__name">Hawkins C. </li><li class="ref__authors__name">Paulus W. </li><li class="ref__authors__name">Perry A. </li><li class="ref__authors__name">Reifenberger G. </li><li class="ref__authors__name">Figarella-Branger D. </li><li>et al.</li></ul><div class="ref__title">Announcing cIMPACT-NOW: the consortium to inform molecular and practical approaches to CNS tumor taxonomy.</div><span class="ref__series"><i>Acta Neuropathol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">133</span></span>: <span class="ref__seriesPages">1-3</span><div class="ref__controls"><a href="#back-bib85" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85001090602&amp;cf=" onclick="newWindow(this.href);return false">Scopus (55)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27909809&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs00401-016-1646-x&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2017&pages=1-3&journal=Acta+Neuropathol.&author=Louis+D.N.&author=Aldape+K.&author=Brat+D.J.&author=Capper+D.&author=Ellison+D.W.&author=Hawkins+C.&author=Paulus+W.&author=Perry+A.&author=Reifenberger+G.&author=Figarella-Branger+D.&title=Announcing+cIMPACT-NOW%3A+the+consortium+to+inform+molecular+and+practical+approaches+to+CNS+tumor+taxonomy">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Love M.I. </li><li class="ref__authors__name">Huber W. </li><li class="ref__authors__name">Anders S. </li></ul><div class="ref__title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">550</span><div class="ref__controls"><a href="#back-bib86" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84924629414&amp;cf=" onclick="newWindow(this.href);return false">Scopus (16300)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25516281&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fs13059-014-0550-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=550&journal=Genome+Biol.&author=Love+M.I.&author=Huber+W.&author=Anders+S.&title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Malta T.M. </li><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Burzykowski T. </li><li class="ref__authors__name">Poisson L. </li><li class="ref__authors__name">Weinstein J.N. </li><li class="ref__authors__name">Kamińska B. </li><li class="ref__authors__name">Huelsken J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Gevaert O. </li><li>et al.</li></ul><div class="ref__title">Machine learning identifies stemness features associated with oncogenic dedifferentiation.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">173</span></span>: <span class="ref__seriesPages">338-354.e15</span><div class="ref__controls"><a href="#back-bib87" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85044967234&amp;cf=" onclick="newWindow(this.href);return false">Scopus (258)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29625051&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2018.03.034&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2018&pages=338-354.e15&journal=Cell&author=Malta+T.M.&author=Sokolov+A.&author=Gentles+A.J.&author=Burzykowski+T.&author=Poisson+L.&author=Weinstein+J.N.&author=Kami%C5%84ska+B.&author=Huelsken+J.&author=Omberg+L.&author=Gevaert+O.&title=Machine+learning+identifies+stemness+features+associated+with+oncogenic+dedifferentiation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Masuda T. </li><li class="ref__authors__name">Nishimoto N. </li><li class="ref__authors__name">Tomiyama D. </li><li class="ref__authors__name">Matsuda T. </li><li class="ref__authors__name">Tozaki-Saitoh H. </li><li class="ref__authors__name">Tamura T. </li><li class="ref__authors__name">Kohsaka S. </li><li class="ref__authors__name">Tsuda M. </li><li class="ref__authors__name">Inoue K. </li></ul><div class="ref__title">IRF8 is a transcriptional determinant for microglial motility.</div><span class="ref__series"><i>Purinergic Signal.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">515-521</span><div class="ref__controls"><a href="#back-bib145" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84906866003&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=24798612&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11302-014-9413-8&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=515-521&journal=Purinergic+Signal.&author=Masuda+T.&author=Nishimoto+N.&author=Tomiyama+D.&author=Matsuda+T.&author=Tozaki-Saitoh+H.&author=Tamura+T.&author=Kohsaka+S.&author=Tsuda+M.&author=Inoue+K.&title=IRF8+is+a+transcriptional+determinant+for+microglial+motility">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Matzke M.M. </li><li class="ref__authors__name">Waters K.M. </li><li class="ref__authors__name">Metz T.O. </li><li class="ref__authors__name">Jacobs J.M. </li><li class="ref__authors__name">Sims A.C. </li><li class="ref__authors__name">Baric R.S. </li><li class="ref__authors__name">Pounds J.G. </li><li class="ref__authors__name">Webb-Robertson B.-J.M. </li></ul><div class="ref__title">Improved quality control processing of peptide-centric LC-MS proteomics data.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">27</span></span>: <span class="ref__seriesPages">2866-2872</span><div class="ref__controls"><a href="#back-bib88" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-80053982558&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21852304&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtr479&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2011&pages=2866-2872&journal=Bioinforma.+Oxf.+Engl.&author=Matzke+M.M.&author=Waters+K.M.&author=Metz+T.O.&author=Jacobs+J.M.&author=Sims+A.C.&author=Baric+R.S.&author=Pounds+J.G.&author=Webb-Robertson+B.-J.M.&title=Improved+quality+control+processing+of+peptide-centric+LC-MS+proteomics+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McGranahan T. </li><li class="ref__authors__name">Therkelsen K.E. </li><li class="ref__authors__name">Ahmad S. </li><li class="ref__authors__name">Nagpal S. </li></ul><div class="ref__title">Current state of immunotherapy for treatment of glioblastoma.</div><span class="ref__series"><i>Curr. Treat. Options Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">24</span><div class="ref__controls"><a href="#back-bib89" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85061990008&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30790064&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11864-019-0619-4&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=24&journal=Curr.+Treat.+Options+Oncol.&author=McGranahan+T.&author=Therkelsen+K.E.&author=Ahmad+S.&author=Nagpal+S.&title=Current+state+of+immunotherapy+for+treatment+of+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">McKenna A. </li><li class="ref__authors__name">Hanna M. </li><li class="ref__authors__name">Banks E. </li><li class="ref__authors__name">Sivachenko A. </li><li class="ref__authors__name">Cibulskis K. </li><li class="ref__authors__name">Kernytsky A. </li><li class="ref__authors__name">Garimella K. </li><li class="ref__authors__name">Altshuler D. </li><li class="ref__authors__name">Gabriel S. </li><li class="ref__authors__name">Daly M. </li><li>et al.</li></ul><div class="ref__title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">1297-1303</span><div class="ref__controls"><a href="#back-bib90" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77956295988&amp;cf=" onclick="newWindow(this.href);return false">Scopus (10783)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20644199&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.107524.110&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=1297-1303&journal=Genome+Res.&author=McKenna+A.&author=Hanna+M.&author=Banks+E.&author=Sivachenko+A.&author=Cibulskis+K.&author=Kernytsky+A.&author=Garimella+K.&author=Altshuler+D.&author=Gabriel+S.&author=Daly+M.&title=The+Genome+Analysis+Toolkit%3A+a+MapReduce+framework+for+analyzing+next-generation+DNA+sequencing+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mermel C.H. </li><li class="ref__authors__name">Schumacher S.E. </li><li class="ref__authors__name">Hill B. </li><li class="ref__authors__name">Meyerson M.L. </li><li class="ref__authors__name">Beroukhim R. </li><li class="ref__authors__name">Getz G. </li></ul><div class="ref__title">GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers.</div><span class="ref__series"><i>Genome Biol.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">R41</span><div class="ref__controls"><a href="#back-bib91" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-79955166265&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1131)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=21527027&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2Fgb-2011-12-4-r41&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=R41&journal=Genome+Biol.&author=Mermel+C.H.&author=Schumacher+S.E.&author=Hill+B.&author=Meyerson+M.L.&author=Beroukhim+R.&author=Getz+G.&title=GISTIC2.0+facilitates+sensitive+and+confident+localization+of+the+targets+of+focal+somatic+copy-number+alteration+in+human+cancers">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Mani D.R. </li><li class="ref__authors__name">Ruggles K.V. </li><li class="ref__authors__name">Gillette M.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Wang P. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Qiao J.W. </li><li class="ref__authors__name">Cao S. </li><li class="ref__authors__name">Petralia F. </li><li>et al.</li></ul><div class="ref__title">Proteogenomics connects somatic mutations to signalling in breast cancer.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">534</span></span>: <span class="ref__seriesPages">55-62</span><div class="ref__controls"><a href="#back-bib92" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84973617999&amp;cf=" onclick="newWindow(this.href);return false">Scopus (664)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27251275&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature18003&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=55-62&journal=Nature&author=Mertins+P.&author=Mani+D.R.&author=Ruggles+K.V.&author=Gillette+M.A.&author=Clauser+K.R.&author=Wang+P.&author=Wang+X.&author=Qiao+J.W.&author=Cao+S.&author=Petralia+F.&title=Proteogenomics+connects+somatic+mutations+to+signalling+in+breast+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Monroe M.E. </li><li class="ref__authors__name">Shaw J.L. </li><li class="ref__authors__name">Daly D.S. </li><li class="ref__authors__name">Adkins J.N. </li><li class="ref__authors__name">Smith R.D. </li></ul><div class="ref__title">MASIC: a software program for fast quantitation and flexible visualization of chromatographic profiles from detected LC-MS(/MS) features.</div><span class="ref__series"><i>Comput. Biol. Chem.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">215-217</span><div class="ref__controls"><a href="#back-bib93" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-43249087428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (81)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18440872&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.compbiolchem.2008.02.006&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=215-217&journal=Comput.+Biol.+Chem.&author=Monroe+M.E.&author=Shaw+J.L.&author=Daly+D.S.&author=Adkins+J.N.&author=Smith+R.D.&title=MASIC%3A+a+software+program+for+fast+quantitation+and+flexible+visualization+of+chromatographic+profiles+from+detected+LC-MS%28%2FMS%29+features">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Montagner A. </li><li class="ref__authors__name">Yart A. </li><li class="ref__authors__name">Dance M. </li><li class="ref__authors__name">Perret B. </li><li class="ref__authors__name">Salles J.-P. </li><li class="ref__authors__name">Raynal P. </li></ul><div class="ref__title">A novel role for Gab1 and SHP2 in epidermal growth factor-induced Ras activation.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">280</span></span>: <span class="ref__seriesPages">5350-5360</span><div class="ref__controls"><a href="#back-bib94" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-14044272770&amp;cf=" onclick="newWindow(this.href);return false">Scopus (133)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15574420&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M410012200&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=5350-5360&journal=J.%C2%A0Biol.+Chem.&author=Montagner+A.&author=Yart+A.&author=Dance+M.&author=Perret+B.&author=Salles+J.-P.&author=Raynal+P.&title=A+novel+role+for+Gab1+and+SHP2+in+epidermal+growth+factor-induced+Ras+activation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nakayasu E.S. </li><li class="ref__authors__name">Nicora C.D. </li><li class="ref__authors__name">Sims A.C. </li><li class="ref__authors__name">Burnum-Johnson K.E. </li><li class="ref__authors__name">Kim Y.-M. </li><li class="ref__authors__name">Kyle J.E. </li><li class="ref__authors__name">Matzke M.M. </li><li class="ref__authors__name">Shukla A.K. </li><li class="ref__authors__name">Chu R.K. </li><li class="ref__authors__name">Schepmoes A.A. </li><li>et al.</li></ul><div class="ref__title">MPLEx: a robust and universal protocol for single-sample integrative proteomic, metabolomic, and lipidomic analyses.</div><span class="ref__series"><i>MSystems.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">1</span></span>: <span class="ref__seriesPages">e00043-16</span><div class="ref__controls"><a href="#back-bib95" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85041917042&amp;cf=" onclick="newWindow(this.href);return false">Scopus (62)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27822525&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1128%2FmSystems.00043-16&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=e00043-16&journal=MSystems&author=Nakayasu+E.S.&author=Nicora+C.D.&author=Sims+A.C.&author=Burnum-Johnson+K.E.&author=Kim+Y.-M.&author=Kyle+J.E.&author=Matzke+M.M.&author=Shukla+A.K.&author=Chu+R.K.&author=Schepmoes+A.A.&title=MPLEx%3A+a+robust+and+universal+protocol+for+single-sample+integrative+proteomic%2C+metabolomic%2C+and+lipidomic+analyses">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Narita T. </li><li class="ref__authors__name">Weinert B.T. </li><li class="ref__authors__name">Choudhary C. </li></ul><div class="ref__title">Functions and mechanisms of non-histone protein acetylation.</div><span class="ref__series"><i>Nat. Rev. Mol. Cell Biol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">20</span></span>: <span class="ref__seriesPages">156-174</span><div class="ref__controls"><a href="#back-bib96" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85057071302&amp;cf=" onclick="newWindow(this.href);return false">Scopus (151)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30467427&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41580-018-0081-3&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=156-174&journal=Nat.+Rev.+Mol.+Cell+Biol.&author=Narita+T.&author=Weinert+B.T.&author=Choudhary+C.&title=Functions+and+mechanisms+of+non-histone+protein+acetylation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Nassiri F. </li><li class="ref__authors__name">Mamatjan Y. </li><li class="ref__authors__name">Suppiah S. </li><li class="ref__authors__name">Badhiwala J.H. </li><li class="ref__authors__name">Mansouri S. </li><li class="ref__authors__name">Karimi S. </li><li class="ref__authors__name">Saarela O. </li><li class="ref__authors__name">Poisson L. </li><li class="ref__authors__name">Gepfner-Tuma I. </li><li class="ref__authors__name">Schittenhelm J. </li><li>et al.</li></ul><div class="ref__title">DNA methylation profiling to predict recurrence risk in meningioma: development and validation of a nomogram to optimize clinical management.</div><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">901-910</span><div class="ref__controls"><a href="#back-bib97" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071162289&amp;cf=" onclick="newWindow(this.href);return false">Scopus (30)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31158293&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnoz061&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=901-910&journal=Neuro-Oncol.&author=Nassiri+F.&author=Mamatjan+Y.&author=Suppiah+S.&author=Badhiwala+J.H.&author=Mansouri+S.&author=Karimi+S.&author=Saarela+O.&author=Poisson+L.&author=Gepfner-Tuma+I.&author=Schittenhelm+J.&title=DNA+methylation+profiling+to+predict+recurrence+risk+in+meningioma%3A+development+and+validation+of+a+nomogram+to+optimize+clinical+management">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Newman A.M. </li><li class="ref__authors__name">Steen C.B. </li><li class="ref__authors__name">Liu C.L. </li><li class="ref__authors__name">Gentles A.J. </li><li class="ref__authors__name">Chaudhuri A.A. </li><li class="ref__authors__name">Scherer F. </li><li class="ref__authors__name">Khodadoust M.S. </li><li class="ref__authors__name">Esfahani M.S. </li><li class="ref__authors__name">Luca B.A. </li><li class="ref__authors__name">Steiner D. </li><li>et al.</li></ul><div class="ref__title">Determining cell type abundance and expression from bulk tissues with digital cytometry.</div><span class="ref__series"><i>Nat. Biotechnol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">37</span></span>: <span class="ref__seriesPages">773-782</span><div class="ref__controls"><a href="#back-bib98" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85065314034&amp;cf=" onclick="newWindow(this.href);return false">Scopus (187)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31061481&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41587-019-0114-2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=773-782&journal=Nat.+Biotechnol.&author=Newman+A.M.&author=Steen+C.B.&author=Liu+C.L.&author=Gentles+A.J.&author=Chaudhuri+A.A.&author=Scherer+F.&author=Khodadoust+M.S.&author=Esfahani+M.S.&author=Luca+B.A.&author=Steiner+D.&title=Determining+cell+type+abundance+and+expression+from+bulk+tissues+with+digital+cytometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ostrom Q.T. </li><li class="ref__authors__name">Cioffi G. </li><li class="ref__authors__name">Gittleman H. </li><li class="ref__authors__name">Patil N. </li><li class="ref__authors__name">Waite K. </li><li class="ref__authors__name">Kruchko C. </li><li class="ref__authors__name">Barnholtz-Sloan J.S. </li></ul><div class="ref__title">CBTRUS statistical report: primary brain and other central nervous system tumors diagnosed in the United States in 2012-2016.</div><span class="ref__series"><i>Neuro-Oncol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">v1-v100</span><div class="ref__controls"><a href="#back-bib99" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85074400952&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31675094&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fneuonc%2Fnoz150&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=v1-v100&journal=Neuro-Oncol&author=Ostrom+Q.T.&author=Cioffi+G.&author=Gittleman+H.&author=Patil+N.&author=Waite+K.&author=Kruchko+C.&author=Barnholtz-Sloan+J.S.&title=CBTRUS+statistical+report%3A+primary+brain+and+other+central+nervous+system+tumors+diagnosed+in+the+United+States+in+2012-2016">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">O’Leary N.A. </li><li class="ref__authors__name">Wright M.W. </li><li class="ref__authors__name">Brister J.R. </li><li class="ref__authors__name">Ciufo S. </li><li class="ref__authors__name">Haddad D. </li><li class="ref__authors__name">McVeigh R. </li><li class="ref__authors__name">Rajput B. </li><li class="ref__authors__name">Robbertse B. </li><li class="ref__authors__name">Smith-White B. </li><li class="ref__authors__name">Ako-Adjei D. </li><li>et al.</li></ul><div class="ref__title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D733-D745</span><div class="ref__controls"><a href="#back-bib100" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26553804&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1189&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D733-D745&journal=Nucleic+Acids+Res.&author=O%E2%80%99Leary+N.A.&author=Wright+M.W.&author=Brister+J.R.&author=Ciufo+S.&author=Haddad+D.&author=McVeigh+R.&author=Rajput+B.&author=Robbertse+B.&author=Smith-White+B.&author=Ako-Adjei+D.&title=Reference+sequence+%28RefSeq%29+database+at+NCBI%3A+current+status%2C+taxonomic+expansion%2C+and+functional+annotation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Park C.S. </li><li class="ref__authors__name">Kim S.I. </li><li class="ref__authors__name">Lee M.S. </li><li class="ref__authors__name">Youn C.-Y. </li><li class="ref__authors__name">Kim D.J. </li><li class="ref__authors__name">Jho E.-H. </li><li class="ref__authors__name">Song W.K. </li></ul><div class="ref__title">Modulation of beta-catenin phosphorylation/degradation by cyclin-dependent kinase 2.</div><span class="ref__series"><i>J. Biol. Chem.</i> <span class="ref__seriesDate">2004; </span><span class="ref__seriesVolume">279</span></span>: <span class="ref__seriesPages">19592-19599</span><div class="ref__controls"><a href="#back-bib101" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-2442616062&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=14985333&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=abstract&amp;site=jbc-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=fulltext&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fjbc.M314208200&amp;cf=pdf&amp;site=jbc-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=19592-19599&journal=J.%C2%A0Biol.+Chem.&author=Park+C.S.&author=Kim+S.I.&author=Lee+M.S.&author=Youn+C.-Y.&author=Kim+D.J.&author=Jho+E.-H.&author=Song+W.K.&title=Modulation+of+beta-catenin+phosphorylation%2Fdegradation+by+cyclin-dependent+kinase+2">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pedregosa F. </li><li class="ref__authors__name">Varoquaux G. </li><li class="ref__authors__name">Gramfort A. </li><li class="ref__authors__name">Michel V. </li><li class="ref__authors__name">Thirion B. </li><li class="ref__authors__name">Grisel O. </li><li class="ref__authors__name">Blondel M. </li><li class="ref__authors__name">Prettenhofer P. </li><li class="ref__authors__name">Weiss R. </li><li class="ref__authors__name">Dubourg V. </li><li>et al.</li></ul><div class="ref__title">Scikit-learn: machine learning in Python.</div><span class="ref__series"><i>J. Mach. Learn. Res.</i> <span class="ref__seriesDate">2011; </span><span class="ref__seriesVolume">12</span></span>: <span class="ref__seriesPages">2825-2830</span><div class="ref__controls"><a href="#back-bib102" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=2825-2830&journal=J.%C2%A0Mach.+Learn.+Res.&author=Pedregosa+F.&author=Varoquaux+G.&author=Gramfort+A.&author=Michel+V.&author=Thirion+B.&author=Grisel+O.&author=Blondel+M.&author=Prettenhofer+P.&author=Weiss+R.&author=Dubourg+V.&title=Scikit-learn%3A+machine+learning+in+Python">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Perfetto L. </li><li class="ref__authors__name">Briganti L. </li><li class="ref__authors__name">Calderone A. </li><li class="ref__authors__name">Cerquone Perpetuini A. </li><li class="ref__authors__name">Iannuccelli M. </li><li class="ref__authors__name">Langone F. </li><li class="ref__authors__name">Licata L. </li><li class="ref__authors__name">Marinkovic M. </li><li class="ref__authors__name">Mattioni A. </li><li class="ref__authors__name">Pavlidou T. </li><li>et al.</li></ul><div class="ref__title">SIGNOR: a database of causal relationships between biological entities.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">D548-D554</span><div class="ref__controls"><a href="#back-bib103" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84976871607&amp;cf=" onclick="newWindow(this.href);return false">Scopus (112)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26467481&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkv1048&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D548-D554&journal=Nucleic+Acids+Res.&author=Perfetto+L.&author=Briganti+L.&author=Calderone+A.&author=Cerquone+Perpetuini+A.&author=Iannuccelli+M.&author=Langone+F.&author=Licata+L.&author=Marinkovic+M.&author=Mattioni+A.&author=Pavlidou+T.&title=SIGNOR%3A+a+database+of+causal+relationships+between+biological+entities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Perry J.R. </li><li class="ref__authors__name">Laperriere N. </li><li class="ref__authors__name">O’Callaghan C.J. </li><li class="ref__authors__name">Brandes A.A. </li><li class="ref__authors__name">Menten J. </li><li class="ref__authors__name">Phillips C. </li><li class="ref__authors__name">Fay M. </li><li class="ref__authors__name">Nishikawa R. </li><li class="ref__authors__name">Cairncross J.G. </li><li class="ref__authors__name">Roa W. </li><li>et al.</li></ul><div class="ref__title">Short-course radiation plus temozolomide in elderly patients with glioblastoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">376</span></span>: <span class="ref__seriesPages">1027-1037</span><div class="ref__controls"><a href="#back-bib104" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85015918057&amp;cf=" onclick="newWindow(this.href);return false">Scopus (357)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28296618&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa1611977&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2017&pages=1027-1037&journal=N.%C2%A0Engl.+J.+Med.&author=Perry+J.R.&author=Laperriere+N.&author=O%E2%80%99Callaghan+C.J.&author=Brandes+A.A.&author=Menten+J.&author=Phillips+C.&author=Fay+M.&author=Nishikawa+R.&author=Cairncross+J.G.&author=Roa+W.&title=Short-course+radiation+plus+temozolomide+in+elderly+patients+with+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pilarczyk M. </li><li class="ref__authors__name">Najafabadi M.F. </li><li class="ref__authors__name">Kouril M. </li><li class="ref__authors__name">Vasiliauskas J. </li><li class="ref__authors__name">Niu W. </li><li class="ref__authors__name">Shamsaei B. </li><li class="ref__authors__name">Mahi N. </li><li class="ref__authors__name">Zhang L. </li><li class="ref__authors__name">Clark N. </li><li class="ref__authors__name">Ren Y. </li><li>et al.</li></ul><div class="ref__title">Connecting Omics Signatures of Diseases, Drugs, and Mechanisms of Actions with iLINCS.</div><span class="ref__series"><i>BioRxiv.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume"></span></span>: <span class="ref__seriesPages">826271</span><div class="ref__controls"><a href="#back-bib105" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=826271&journal=BioRxiv&author=Pilarczyk+M.&author=Najafabadi+M.F.&author=Kouril+M.&author=Vasiliauskas+J.&author=Niu+W.&author=Shamsaei+B.&author=Mahi+N.&author=Zhang+L.&author=Clark+N.&author=Ren+Y.&title=Connecting+Omics+Signatures+of+Diseases%2C+Drugs%2C+and+Mechanisms+of+Actions+with+iLINCS">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pinto M.L. </li><li class="ref__authors__name">Rios E. </li><li class="ref__authors__name">Durães C. </li><li class="ref__authors__name">Ribeiro R. </li><li class="ref__authors__name">Machado J.C. </li><li class="ref__authors__name">Mantovani A. </li><li class="ref__authors__name">Barbosa M.A. </li><li class="ref__authors__name">Carneiro F. </li><li class="ref__authors__name">Oliveira M.J. </li></ul><div class="ref__title">The two faces of tumor-associated macrophages and their clinical significance in colorectal cancer.</div><span class="ref__series"><i>Front. Immunol.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">1875</span><div class="ref__controls"><a href="#back-bib106" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071761383&amp;cf=" onclick="newWindow(this.href);return false">Scopus (38)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31481956&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.3389%2Ffimmu.2019.01875&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1875&journal=Front.+Immunol.&author=Pinto+M.L.&author=Rios+E.&author=Dur%C3%A3es+C.&author=Ribeiro+R.&author=Machado+J.C.&author=Mantovani+A.&author=Barbosa+M.A.&author=Carneiro+F.&author=Oliveira+M.J.&title=The+two+faces+of+tumor-associated+macrophages+and+their+clinical+significance+in+colorectal+cancer">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Pluskal T. </li><li class="ref__authors__name">Castillo S. </li><li class="ref__authors__name">Villar-Briones A. </li><li class="ref__authors__name">Oresic M. </li></ul><div class="ref__title">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data.</div><span class="ref__series"><i>BMC Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">11</span></span>: <span class="ref__seriesPages">395</span><div class="ref__controls"><a href="#back-bib107" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954772536&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1479)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20650010&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1186%2F1471-2105-11-395&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=395&journal=BMC+Bioinformatics&author=Pluskal+T.&author=Castillo+S.&author=Villar-Briones+A.&author=Oresic+M.&title=MZmine+2%3A+modular+framework+for+processing%2C+visualizing%2C+and+analyzing+mass+spectrometry-based+molecular+profile+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Polpitiya A.D. </li><li class="ref__authors__name">Qian W.-J. </li><li class="ref__authors__name">Jaitly N. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Adkins J.N. </li><li class="ref__authors__name">Camp D.G. </li><li class="ref__authors__name">Anderson G.A. </li><li class="ref__authors__name">Smith R.D. </li></ul><div class="ref__title">DAnTE: a statistical tool for quantitative analysis of -omics data.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">24</span></span>: <span class="ref__seriesPages">1556-1558</span><div class="ref__controls"><a href="#back-bib108" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-46249116675&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18453552&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtn217&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=1556-1558&journal=Bioinforma.+Oxf.+Engl.&author=Polpitiya+A.D.&author=Qian+W.-J.&author=Jaitly+N.&author=Petyuk+V.A.&author=Adkins+J.N.&author=Camp+D.G.&author=Anderson+G.A.&author=Smith+R.D.&title=DAnTE%3A+a+statistical+tool+for+quantitative+analysis+of+-omics+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Poulin B. </li><li class="ref__authors__name">Sekiya F. </li><li class="ref__authors__name">Rhee S.G. </li></ul><div class="ref__title">Intramolecular interaction between phosphorylated tyrosine-783 and the C-terminal Src homology 2 domain activates phospholipase C-gamma1.</div><span class="ref__series"><i>Proc. Natl. Acad. Sci. U S A.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">102</span></span>: <span class="ref__seriesPages">4276-4281</span><div class="ref__controls"><a href="#back-bib109" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-15444377640&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15764700&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1073%2Fpnas.0409590102&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=4276-4281&journal=Proc.+Natl.+Acad.+Sci.+U+S+A&author=Poulin+B.&author=Sekiya+F.&author=Rhee+S.G.&title=Intramolecular+interaction+between+phosphorylated+tyrosine-783+and+the+C-terminal+Src+homology+2+domain+activates+phospholipase+C-gamma1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rausch T. </li><li class="ref__authors__name">Zichner T. </li><li class="ref__authors__name">Schlattl A. </li><li class="ref__authors__name">Stutz A.M. </li><li class="ref__authors__name">Benes V. </li><li class="ref__authors__name">Korbel J.O. </li></ul><div class="ref__title">DELLY: structural variant discovery by integrated paired-end and split-read analysis.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">28</span></span>: <span class="ref__seriesPages">i333-i339</span><div class="ref__controls"><a href="#back-bib110" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84866440781&amp;cf=" onclick="newWindow(this.href);return false">Scopus (759)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=22962449&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbts378&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=i333-i339&journal=Bioinformatics&author=Rausch+T.&author=Zichner+T.&author=Schlattl+A.&author=Stutz+A.M.&author=Benes+V.&author=Korbel+J.O.&title=DELLY%3A+structural+variant+discovery+by+integrated+paired-end+and+split-read+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Rieckmann J.C. </li><li class="ref__authors__name">Geiger R. </li><li class="ref__authors__name">Hornburg D. </li><li class="ref__authors__name">Wolf T. </li><li class="ref__authors__name">Kveler K. </li><li class="ref__authors__name">Jarrossay D. </li><li class="ref__authors__name">Sallusto F. </li><li class="ref__authors__name">Shen-Orr S.S. </li><li class="ref__authors__name">Lanzavecchia A. </li><li class="ref__authors__name">Mann M. </li><li>et al.</li></ul><div class="ref__title">Social network architecture of human immune cells unveiled by quantitative proteomics.</div><span class="ref__series"><i>Nat. Immunol.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">583-593</span><div class="ref__controls"><a href="#back-bib111" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85014542002&amp;cf=" onclick="newWindow(this.href);return false">Scopus (130)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28263321&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fni.3693&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=583-593&journal=Nat.+Immunol.&author=Rieckmann+J.C.&author=Geiger+R.&author=Hornburg+D.&author=Wolf+T.&author=Kveler+K.&author=Jarrossay+D.&author=Sallusto+F.&author=Shen-Orr+S.S.&author=Lanzavecchia+A.&author=Mann+M.&title=Social+network+architecture+of+human+immune+cells+unveiled+by+quantitative+proteomics">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Robinson M.D. </li><li class="ref__authors__name">McCarthy D.J. </li><li class="ref__authors__name">Smyth G.K. </li></ul><div class="ref__title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">139-140</span><div class="ref__controls"><a href="#back-bib112" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75249087100&amp;cf=" onclick="newWindow(this.href);return false">Scopus (13601)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19910308&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp616&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=139-140&journal=Bioinformatics&author=Robinson+M.D.&author=McCarthy+D.J.&author=Smyth+G.K.&title=edgeR%3A+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ruepp A. </li><li class="ref__authors__name">Waegele B. </li><li class="ref__authors__name">Lechner M. </li><li class="ref__authors__name">Brauner B. </li><li class="ref__authors__name">Dunger-Kaltenbach I. </li><li class="ref__authors__name">Fobo G. </li><li class="ref__authors__name">Frishman G. </li><li class="ref__authors__name">Montrone C. </li><li class="ref__authors__name">Mewes H.-W. </li></ul><div class="ref__title">CORUM: the comprehensive resource of mammalian protein complexes--2009.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">38</span></span>: <span class="ref__seriesPages">D497-D501</span><div class="ref__controls"><a href="#back-bib113" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-75549091794&amp;cf=" onclick="newWindow(this.href);return false">Scopus (500)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19884131&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkp914&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D497-D501&journal=Nucleic+Acids+Res.&author=Ruepp+A.&author=Waegele+B.&author=Lechner+M.&author=Brauner+B.&author=Dunger-Kaltenbach+I.&author=Fobo+G.&author=Frishman+G.&author=Montrone+C.&author=Mewes+H.-W.&title=CORUM%3A+the+comprehensive+resource+of+mammalian+protein+complexes%2D%2D2009">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Salomonis N. </li><li class="ref__authors__name">Dexheimer P.J. </li><li class="ref__authors__name">Omberg L. </li><li class="ref__authors__name">Schroll R. </li><li class="ref__authors__name">Bush S. </li><li class="ref__authors__name">Huo J. </li><li class="ref__authors__name">Schriml L. </li><li class="ref__authors__name">Ho Sui S. </li><li class="ref__authors__name">Keddache M. </li><li class="ref__authors__name">Mayhew C. </li><li>et al.</li></ul><div class="ref__title">Integrated genomic analysis of diverse induced pluripotent stem cells from the progenitor cell biology consortium.</div><span class="ref__series"><i>Stem Cell Rep.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">7</span></span>: <span class="ref__seriesPages">110-125</span><div class="ref__controls"><a href="#back-bib114" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27293150&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.stemcr.2016.05.006&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=110-125&journal=Stem+Cell+Rep.&author=Salomonis+N.&author=Dexheimer+P.J.&author=Omberg+L.&author=Schroll+R.&author=Bush+S.&author=Huo+J.&author=Schriml+L.&author=Ho+Sui+S.&author=Keddache+M.&author=Mayhew+C.&title=Integrated+genomic+analysis+of+diverse+induced+pluripotent+stem+cells+from+the+progenitor+cell+biology+consortium">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Silva T.C. </li><li class="ref__authors__name">Colaprico A. </li><li class="ref__authors__name">Olsen C. </li><li class="ref__authors__name">D’Angelo F. </li><li class="ref__authors__name">Bontempi G. </li><li class="ref__authors__name">Ceccarelli M. </li><li class="ref__authors__name">Noushmehr H. </li></ul><div class="ref__title">TCGA Workflow: analyze cancer genomics and epigenomics data using Bioconductor packages.</div><span class="ref__series"><i>F1000Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">5</span></span>: <span class="ref__seriesPages">1542</span><div class="ref__controls"><a href="#back-bib115" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85013753091&amp;cf=" onclick="newWindow(this.href);return false">Scopus (2)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28232861&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.12688%2Ff1000research.8923.2&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1542&journal=F1000Res.&author=Silva+T.C.&author=Colaprico+A.&author=Olsen+C.&author=D%E2%80%99Angelo+F.&author=Bontempi+G.&author=Ceccarelli+M.&author=Noushmehr+H.&title=TCGA+Workflow%3A+analyze+cancer+genomics+and+epigenomics+data+using+Bioconductor+packages">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sokolov A. </li><li class="ref__authors__name">Paull E.O. </li><li class="ref__authors__name">Stuart J.M. </li></ul><div class="ref__title">One-class detection of cell states in tumor subtypes.</div><span class="ref__series"><i>Pac. Symp. Biocomput.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">21</span></span>: <span class="ref__seriesPages">405-416</span><div class="ref__controls"><a href="#back-bib116" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26776204&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=405-416&journal=Pac.+Symp.+Biocomput.&author=Sokolov+A.&author=Paull+E.O.&author=Stuart+J.M.&title=One-class+detection+of+cell+states+in+tumor+subtypes">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Song X. </li><li class="ref__authors__name">Ji J. </li><li class="ref__authors__name">Gleason K.J. </li><li class="ref__authors__name">Yang F. </li><li class="ref__authors__name">Martignetti J.A. </li><li class="ref__authors__name">Chen L.S. </li><li class="ref__authors__name">Wang P. </li></ul><div class="ref__title">Insights into impact of DNA copy number alteration and methylation on the proteogenomic landscape of human ovarian cancer via a multi-omics integrative analysis.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">S52-S65</span><div class="ref__controls"><a href="#back-bib117" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85071348150&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31227599&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.RA118.001220&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=S52-S65&journal=Mol.+Cell.+Proteomics&author=Song+X.&author=Ji+J.&author=Gleason+K.J.&author=Yang+F.&author=Martignetti+J.A.&author=Chen+L.S.&author=Wang+P.&title=Insights+into+impact+of+DNA+copy+number+alteration+and+methylation+on+the+proteogenomic+landscape+of+human+ovarian+cancer+via+a+multi-omics+integrative+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Stathias V. </li><li class="ref__authors__name">Turner J. </li><li class="ref__authors__name">Koleti A. </li><li class="ref__authors__name">Vidovic D. </li><li class="ref__authors__name">Cooper D. </li><li class="ref__authors__name">Fazel-Najafabadi M. </li><li class="ref__authors__name">Pilarczyk M. </li><li class="ref__authors__name">Terryn R. </li><li class="ref__authors__name">Chung C. </li><li class="ref__authors__name">Umeano A. </li><li>et al.</li></ul><div class="ref__title">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">48</span></span>: <span class="ref__seriesPages">D431-D439</span><div class="ref__controls"><a href="#back-bib118" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85077665017&amp;cf=" onclick="newWindow(this.href);return false">Scopus (11)</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkz1023&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2019&pages=D431-D439&journal=Nucleic+Acids+Res.&author=Stathias+V.&author=Turner+J.&author=Koleti+A.&author=Vidovic+D.&author=Cooper+D.&author=Fazel-Najafabadi+M.&author=Pilarczyk+M.&author=Terryn+R.&author=Chung+C.&author=Umeano+A.&title=LINCS+Data+Portal+2.0%3A+next+generation+access+point+for+perturbation-response+signatures">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Stupp R. </li><li class="ref__authors__name">Mason W.P. </li><li class="ref__authors__name">van den Bent M.J. </li><li class="ref__authors__name">Weller M. </li><li class="ref__authors__name">Fisher B. </li><li class="ref__authors__name">Taphoorn M.J.B. </li><li class="ref__authors__name">Belanger K. </li><li class="ref__authors__name">Brandes A.A. </li><li class="ref__authors__name">Marosi C. </li><li class="ref__authors__name">Bogdahn U. </li><li>et al.</li></ul><div class="ref__title">Radiotherapy plus concomitant and adjuvant temozolomide for glioblastoma.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2005; </span><span class="ref__seriesVolume">352</span></span>: <span class="ref__seriesPages">987-996</span><div class="ref__controls"><a href="#back-bib119" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-20044366163&amp;cf=" onclick="newWindow(this.href);return false">Scopus (12069)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=15758009&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa043330&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=987-996&journal=N.%C2%A0Engl.+J.+Med.&author=Stupp+R.&author=Mason+W.P.&author=van+den+Bent+M.J.&author=Weller+M.&author=Fisher+B.&author=Taphoorn+M.J.B.&author=Belanger+K.&author=Brandes+A.A.&author=Marosi+C.&author=Bogdahn+U.&title=Radiotherapy+plus+concomitant+and+adjuvant+temozolomide+for+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Stupp R. </li><li class="ref__authors__name">Taillibert S. </li><li class="ref__authors__name">Kanner A. </li><li class="ref__authors__name">Read W. </li><li class="ref__authors__name">Steinberg D. </li><li class="ref__authors__name">Lhermitte B. </li><li class="ref__authors__name">Toms S. </li><li class="ref__authors__name">Idbaih A. </li><li class="ref__authors__name">Ahluwalia M.S. </li><li class="ref__authors__name">Fink K. </li><li>et al.</li></ul><div class="ref__title">Effect of tumor-treating fields plus maintenance temozolomide vs maintenance temozolomide alone on survival in patients with glioblastoma: a randomized clinical trial.</div><span class="ref__series"><i>JAMA.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">318</span></span>: <span class="ref__seriesPages">2306-2316</span><div class="ref__controls"><a href="#back-bib120" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040033962&amp;cf=" onclick="newWindow(this.href);return false">Scopus (464)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29260225&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_121_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1001%2Fjama.2017.18718&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2017&pages=2306-2316&journal=JAMA&author=Stupp+R.&author=Taillibert+S.&author=Kanner+A.&author=Read+W.&author=Steinberg+D.&author=Lhermitte+B.&author=Toms+S.&author=Idbaih+A.&author=Ahluwalia+M.S.&author=Fink+K.&title=Effect+of+tumor-treating+fields+plus+maintenance+temozolomide+vs+maintenance+temozolomide+alone+on+survival+in+patients+with+glioblastoma%3A+a+randomized+clinical+trial">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sturm D. </li><li class="ref__authors__name">Witt H. </li><li class="ref__authors__name">Hovestadt V. </li><li class="ref__authors__name">Khuong-Quang D.-A. </li><li class="ref__authors__name">Jones D.T.W. </li><li class="ref__authors__name">Konermann C. </li><li class="ref__authors__name">Pfaff E. </li><li class="ref__authors__name">Tönjes M. </li><li class="ref__authors__name">Sill M. </li><li class="ref__authors__name">Bender S. </li><li>et al.</li></ul><div class="ref__title">Hotspot mutations in H3F3A and IDH1 define distinct epigenetic and biological subgroups of glioblastoma.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2012; </span><span class="ref__seriesVolume">22</span></span>: <span class="ref__seriesPages">425-437</span><div class="ref__controls"><a href="#back-bib121" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84867606428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1026)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23079654&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2012.08.024&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=425-437&journal=Cancer+Cell&author=Sturm+D.&author=Witt+H.&author=Hovestadt+V.&author=Khuong-Quang+D.-A.&author=Jones+D.T.W.&author=Konermann+C.&author=Pfaff+E.&author=T%C3%B6njes+M.&author=Sill+M.&author=Bender+S.&title=Hotspot+mutations+in+H3F3A+and+IDH1+define+distinct+epigenetic+and+biological+subgroups+of+glioblastoma">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Subramanian A. </li><li class="ref__authors__name">Narayan R. </li><li class="ref__authors__name">Corsello S.M. </li><li class="ref__authors__name">Peck D.D. </li><li class="ref__authors__name">Natoli T.E. </li><li class="ref__authors__name">Lu X. </li><li class="ref__authors__name">Gould J. </li><li class="ref__authors__name">Davis J.F. </li><li class="ref__authors__name">Tubelli A.A. </li><li class="ref__authors__name">Asiedu J.K. </li><li>et al.</li></ul><div class="ref__title">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">171</span></span>: <span class="ref__seriesPages">1437-1452.e17</span><div class="ref__controls"><a href="#back-bib122" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85036572148&amp;cf=" onclick="newWindow(this.href);return false">Scopus (565)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29195078&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2017.10.049&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452.e17&journal=Cell&author=Subramanian+A.&author=Narayan+R.&author=Corsello+S.M.&author=Peck+D.D.&author=Natoli+T.E.&author=Lu+X.&author=Gould+J.&author=Davis+J.F.&author=Tubelli+A.A.&author=Asiedu+J.K.&title=A+next+generation+connectivity+map%3A+L1000+platform+and+the+first+1%2C000%2C000+profiles">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Sun S.Q. </li><li class="ref__authors__name">Mashl R.J. </li><li class="ref__authors__name">Sengupta S. </li><li class="ref__authors__name">Scott A.D. </li><li class="ref__authors__name">Wang W. </li><li class="ref__authors__name">Batra P. </li><li class="ref__authors__name">Wang L.-B. </li><li class="ref__authors__name">Wyczalkowski M.A. </li><li class="ref__authors__name">Ding L. </li></ul><div class="ref__title">Database of evidence for precision oncology portal.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">34</span></span>: <span class="ref__seriesPages">4315-4317</span><div class="ref__controls"><a href="#back-bib123" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85050591786&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30535306&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbty531&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=4315-4317&journal=Bioinforma.+Oxf.+Engl.&author=Sun+S.Q.&author=Mashl+R.J.&author=Sengupta+S.&author=Scott+A.D.&author=Wang+W.&author=Batra+P.&author=Wang+L.-B.&author=Wyczalkowski+M.A.&author=Ding+L.&title=Database+of+evidence+for+precision+oncology+portal">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Tabb D.L. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Chambers M.C. </li><li class="ref__authors__name">Holman J.D. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li><li class="ref__authors__name">Zimmerman L.J. </li><li>et al.</li></ul><div class="ref__title">Reproducibility of differential proteomic technologies in CPTAC fractionated xenografts.</div><span class="ref__series"><i>J. Proteome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">691-706</span><div class="ref__controls"><a href="#back-bib124" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84960343160&amp;cf=" onclick="newWindow(this.href);return false">Scopus (29)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26653538&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1021%2Facs.jproteome.5b00859&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=691-706&journal=J.%C2%A0Proteome+Res.&author=Tabb+D.L.&author=Wang+X.&author=Carr+S.A.&author=Clauser+K.R.&author=Mertins+P.&author=Chambers+M.C.&author=Holman+J.D.&author=Wang+J.&author=Zhang+B.&author=Zimmerman+L.J.&title=Reproducibility+of+differential+proteomic+technologies+in+CPTAC+fractionated+xenografts">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">The 1000 Genomes Project Consortium </li></ul><div class="ref__title">A global reference for human genetic variation.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2015; </span><span class="ref__seriesVolume">526</span></span>: <span class="ref__seriesPages">68-74</span><div class="ref__controls"><a href="#back-bib125" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84943171338&amp;cf=" onclick="newWindow(this.href);return false">Scopus (5449)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26432245&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature15393&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=68-74&journal=Nature&author=The+1000+Genomes+Project+Consortium+&title=A+global+reference+for+human+genetic+variation">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">The Bioconda Team </li><li class="ref__authors__name">Grüning B. </li><li class="ref__authors__name">Dale R. </li><li class="ref__authors__name">Sjödin A. </li><li class="ref__authors__name">Chapman B.A. </li><li class="ref__authors__name">Rowe J. </li><li class="ref__authors__name">Tomkins-Tinch C.H. </li><li class="ref__authors__name">Valieris R. </li><li class="ref__authors__name">Köster J. </li></ul><div class="ref__title">Bioconda: sustainable and comprehensive software distribution for the life sciences.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">15</span></span>: <span class="ref__seriesPages">475-476</span><div class="ref__controls"><a href="#back-bib126" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85049501214&amp;cf=" onclick="newWindow(this.href);return false">Scopus (151)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29967506&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fs41592-018-0046-7&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=475-476&journal=Nat.+Methods&author=The+Bioconda+Team+&author=Gr%C3%BCning+B.&author=Dale+R.&author=Sj%C3%B6din+A.&author=Chapman+B.A.&author=Rowe+J.&author=Tomkins-Tinch+C.H.&author=Valieris+R.&author=K%C3%B6ster+J.&title=Bioconda%3A+sustainable+and+comprehensive+software+distribution+for+the+life+sciences">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li>The Cancer Genome Atlas Research Network</li></ul><div class="ref__title">Comprehensive genomic characterization defines human glioblastoma genes and core pathways.</div><span class="ref__series"><i>Nature.</i> <span class="ref__seriesDate">2008; </span><span class="ref__seriesVolume">455</span></span>: <span class="ref__seriesPages">1061-1068</span><div class="ref__controls"><a href="#back-bib127" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-54549108740&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4993)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=18772890&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_128_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnature07385&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2008&pages=1061-1068&journal=Nature&author=The+Cancer+Genome+Atlas+Research+Network&title=Comprehensive+genomic+characterization+defines+human+glioblastoma+genes+and+core+pathways">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Türei D. </li><li class="ref__authors__name">Korcsmáros T. </li><li class="ref__authors__name">Saez-Rodriguez J. </li></ul><div class="ref__title">OmniPath: guidelines and gateway for literature-curated signaling pathway resources.</div><span class="ref__series"><i>Nat. Methods.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">13</span></span>: <span class="ref__seriesPages">966-967</span><div class="ref__controls"><a href="#back-bib128" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84999106855&amp;cf=" onclick="newWindow(this.href);return false">Scopus (106)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27898060&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_129_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1038%2Fnmeth.4077&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=966-967&journal=Nat.+Methods&author=T%C3%BCrei+D.&author=Korcsm%C3%A1ros+T.&author=Saez-Rodriguez+J.&title=OmniPath%3A+guidelines+and+gateway+for+literature-curated+signaling+pathway+resources">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S.V. </li><li class="ref__authors__name">Straub P. </li><li class="ref__authors__name">Wang J. </li><li class="ref__authors__name">Zhang B. </li></ul><div class="ref__title">LinkedOmics: analyzing multi-omics data within and across 32 cancer types.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2018; </span><span class="ref__seriesVolume">46</span></span>: <span class="ref__seriesPages">D956-D963</span><div class="ref__controls"><a href="#back-bib129" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85040936922&amp;cf=" onclick="newWindow(this.href);return false">Scopus (326)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=29136207&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkx1090&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D956-D963&journal=Nucleic+Acids+Res.&author=Vasaikar+S.V.&author=Straub+P.&author=Wang+J.&author=Zhang+B.&title=LinkedOmics%3A+analyzing+multi-omics+data+within+and+across+32+cancer+types">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Vasaikar S. </li><li class="ref__authors__name">Huang C. </li><li class="ref__authors__name">Wang X. </li><li class="ref__authors__name">Petyuk V.A. </li><li class="ref__authors__name">Savage S.R. </li><li class="ref__authors__name">Wen B. </li><li class="ref__authors__name">Dou Y. </li><li class="ref__authors__name">Zhang Y. </li><li class="ref__authors__name">Shi Z. </li><li class="ref__authors__name">Arshad O.A. </li><li>et al.</li></ul><div class="ref__title">Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities.</div><span class="ref__series"><i>Cell.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">177</span></span>: <span class="ref__seriesPages">1035-1049.e19</span><div class="ref__controls"><a href="#back-bib130" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85064886726&amp;cf=" onclick="newWindow(this.href);return false">Scopus (107)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=31031003&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_131_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.cell.2019.03.030&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=2019&pages=1035-1049.e19&journal=Cell&author=Vasaikar+S.&author=Huang+C.&author=Wang+X.&author=Petyuk+V.A.&author=Savage+S.R.&author=Wen+B.&author=Dou+Y.&author=Zhang+Y.&author=Shi+Z.&author=Arshad+O.A.&title=Proteogenomic+analysis+of+human+colon+cancer+reveals+new+therapeutic+opportunities">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Verhaak R.G.W. </li><li class="ref__authors__name">Hoadley K.A. </li><li class="ref__authors__name">Purdom E. </li><li class="ref__authors__name">Wang V. </li><li class="ref__authors__name">Qi Y. </li><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Miller C.R. </li><li class="ref__authors__name">Ding L. </li><li class="ref__authors__name">Golub T. </li><li class="ref__authors__name">Mesirov J.P. </li><li>et al.</li></ul><div class="ref__title">Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">17</span></span>: <span class="ref__seriesPages">98-110</span><div class="ref__controls"><a href="#back-bib131" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-73649123907&amp;cf=" onclick="newWindow(this.href);return false">Scopus (4057)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20129251&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccr.2009.12.020&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=98-110&journal=Cancer+Cell&author=Verhaak+R.G.W.&author=Hoadley+K.A.&author=Purdom+E.&author=Wang+V.&author=Qi+Y.&author=Wilkerson+M.D.&author=Miller+C.R.&author=Ding+L.&author=Golub+T.&author=Mesirov+J.P.&title=Integrated+genomic+analysis+identifies+clinically+relevant+subtypes+of+glioblastoma+characterized+by+abnormalities+in+PDGFRA%2C+IDH1%2C+EGFR%2C+and+NF1">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Voena C. </li><li class="ref__authors__name">Conte C. </li><li class="ref__authors__name">Ambrogio C. </li><li class="ref__authors__name">Boeri Erba E. </li><li class="ref__authors__name">Boccalatte F. </li><li class="ref__authors__name">Mohammed S. </li><li class="ref__authors__name">Jensen O.N. </li><li class="ref__authors__name">Palestro G. </li><li class="ref__authors__name">Inghirami G. </li><li class="ref__authors__name">Chiarle R. </li></ul><div class="ref__title">The tyrosine phosphatase Shp2 interacts with NPM-ALK and regulates anaplastic lymphoma cell growth and migration.</div><span class="ref__series"><i>Cancer Res.</i> <span class="ref__seriesDate">2007; </span><span class="ref__seriesVolume">67</span></span>: <span class="ref__seriesPages">4278-4286</span><div class="ref__controls"><a href="#back-bib132" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-34249277428&amp;cf=" onclick="newWindow(this.href);return false">Scopus (66)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=17483340&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1158%2F0008-5472.CAN-06-4350&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=4278-4286&journal=Cancer+Res.&author=Voena+C.&author=Conte+C.&author=Ambrogio+C.&author=Boeri+Erba+E.&author=Boccalatte+F.&author=Mohammed+S.&author=Jensen+O.N.&author=Palestro+G.&author=Inghirami+G.&author=Chiarle+R.&title=The+tyrosine+phosphatase+Shp2+interacts+with+NPM-ALK+and+regulates+anaplastic+lymphoma+cell+growth+and+migration">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wang Q. </li><li class="ref__authors__name">Hu B. </li><li class="ref__authors__name">Hu X. </li><li class="ref__authors__name">Kim H. </li><li class="ref__authors__name">Squatrito M. </li><li class="ref__authors__name">Scarpace L. </li><li class="ref__authors__name">deCarvalho A.C. </li><li class="ref__authors__name">Lyu S. </li><li class="ref__authors__name">Li P. </li><li class="ref__authors__name">Li Y. </li><li>et al.</li></ul><div class="ref__title">Tumor evolution of glioma-intrinsic gene expression subtypes associates with immunological changes in the microenvironment.</div><span class="ref__series"><i>Cancer Cell.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">32</span></span>: <span class="ref__seriesPages">42-56.e6</span><div class="ref__controls"><a href="#back-bib133" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85027444208&amp;cf=" onclick="newWindow(this.href);return false">Scopus (384)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=28697342&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=abstract&amp;site=cell-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=fulltext&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1016%2Fj.ccell.2017.06.003&amp;cf=pdf&amp;site=cell-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2017&pages=42-56.e6&journal=Cancer+Cell&author=Wang+Q.&author=Hu+B.&author=Hu+X.&author=Kim+H.&author=Squatrito+M.&author=Scarpace+L.&author=deCarvalho+A.C.&author=Lyu+S.&author=Li+P.&author=Li+Y.&title=Tumor+evolution+of+glioma-intrinsic+gene+expression+subtypes+associates+with+immunological+changes+in+the+microenvironment">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Webb-Robertson B.-J.M. </li><li class="ref__authors__name">Matzke M.M. </li><li class="ref__authors__name">Metz T.O. </li><li class="ref__authors__name">McDermott J.E. </li><li class="ref__authors__name">Walker H. </li><li class="ref__authors__name">Rodland K.D. </li><li class="ref__authors__name">Pounds J.G. </li><li class="ref__authors__name">Waters K.M. </li></ul><div class="ref__title">Sequential projection pursuit principal component analysis--dealing with missing data associated with new -omics technologies.</div><span class="ref__series"><i>BioTechniques.</i> <span class="ref__seriesDate">2013; </span><span class="ref__seriesVolume">54</span></span>: <span class="ref__seriesPages">165-168</span><div class="ref__controls"><a href="#back-bib134" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84876495055&amp;cf=" onclick="newWindow(this.href);return false">Scopus (6)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=23477384&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.2144%2F000113978&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2013&pages=165-168&journal=BioTechniques&author=Webb-Robertson+B.-J.M.&author=Matzke+M.M.&author=Metz+T.O.&author=McDermott+J.E.&author=Walker+H.&author=Rodland+K.D.&author=Pounds+J.G.&author=Waters+K.M.&title=Sequential+projection+pursuit+principal+component+analysis%2D%2Ddealing+with+missing+data+associated+with+new+-omics+technologies">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Webb-Robertson B.-J. </li><li class="ref__authors__name">Kim Y.-M. </li><li class="ref__authors__name">Zink E.M. </li><li class="ref__authors__name">Hallaian K.A. </li><li class="ref__authors__name">Zhang Q. </li><li class="ref__authors__name">Madupu R. </li><li class="ref__authors__name">Waters K.M. </li><li class="ref__authors__name">Metz T.O. </li></ul><div class="ref__title">A statistical analysis of the effects of urease pre-treatment on the measurement of the urinary metabolome by gas chromatography-mass spectrometry.</div><span class="ref__series"><i>Metabolomics.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">897-908</span><div class="ref__controls"><a href="#back-bib135" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84894268438&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25254001&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1007%2Fs11306-014-0642-1&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=897-908&journal=Metabolomics&author=Webb-Robertson+B.-J.&author=Kim+Y.-M.&author=Zink+E.M.&author=Hallaian+K.A.&author=Zhang+Q.&author=Madupu+R.&author=Waters+K.M.&author=Metz+T.O.&title=A+statistical+analysis+of+the+effects+of+urease+pre-treatment+on+the+measurement+of+the+urinary+metabolome+by+gas+chromatography-mass+spectrometry">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wickham H. </li><li class="ref__authors__name">Averick M. </li><li class="ref__authors__name">Bryan J. </li><li class="ref__authors__name">Chang W. </li><li class="ref__authors__name">McGowan L. </li><li class="ref__authors__name">François R. </li><li class="ref__authors__name">Grolemund G. </li><li class="ref__authors__name">Hayes A. </li><li class="ref__authors__name">Henry L. </li><li class="ref__authors__name">Hester J. </li><li>et al.</li></ul><div class="ref__title">Welcome to the tidyverse.</div><span class="ref__series"><i>J. Open Source Softw.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">4</span></span>: <span class="ref__seriesPages">1686</span><div class="ref__controls"><a href="#back-bib136" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.21105%2Fjoss.01686&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=1686&journal=J.%C2%A0Open+Source+Softw.&author=Wickham+H.&author=Averick+M.&author=Bryan+J.&author=Chang+W.&author=McGowan+L.&author=Fran%C3%A7ois+R.&author=Grolemund+G.&author=Hayes+A.&author=Henry+L.&author=Hester+J.&title=Welcome+to+the+tidyverse">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Wilkerson M.D. </li><li class="ref__authors__name">Hayes D.N. </li></ul><div class="ref__title">ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking.</div><span class="ref__series"><i>Bioinforma. Oxf. Engl.</i> <span class="ref__seriesDate">2010; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">1572-1573</span><div class="ref__controls"><a href="#back-bib137" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-77954182362&amp;cf=" onclick="newWindow(this.href);return false">Scopus (0)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=20427518&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtq170&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1572-1573&journal=Bioinforma.+Oxf.+Engl.&author=Wilkerson+M.D.&author=Hayes+D.N.&title=ConsensusClusterPlus%3A+a+class+discovery+tool+with+confidence+assessments+and+item+tracking">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Xi R. </li><li class="ref__authors__name">Lee S. </li><li class="ref__authors__name">Xia Y. </li><li class="ref__authors__name">Kim T.-M. </li><li class="ref__authors__name">Park P.J. </li></ul><div class="ref__title">Copy number analysis of whole-genome data using BIC-seq2 and its application to detection of cancer susceptibility variants.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">44</span></span>: <span class="ref__seriesPages">6274-6286</span><div class="ref__controls"><a href="#back-bib138" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27260798&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fnar%2Fgkw491&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=6274-6286&journal=Nucleic+Acids+Res.&author=Xi+R.&author=Lee+S.&author=Xia+Y.&author=Kim+T.-M.&author=Park+P.J.&title=Copy+number+analysis+of+whole-genome+data+using+BIC-seq2+and+its+application+to+detection+of+cancer+susceptibility+variants">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yan H. </li><li class="ref__authors__name">Parsons D.W. </li><li class="ref__authors__name">Jin G. </li><li class="ref__authors__name">McLendon R. </li><li class="ref__authors__name">Rasheed B.A. </li><li class="ref__authors__name">Yuan W. </li><li class="ref__authors__name">Kos I. </li><li class="ref__authors__name">Batinic-Haberle I. </li><li class="ref__authors__name">Jones S. </li><li class="ref__authors__name">Riggins G.J. </li><li>et al.</li></ul><div class="ref__title">IDH1 and IDH2 mutations in gliomas.</div><span class="ref__series"><i>N. Engl. J. Med.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">360</span></span>: <span class="ref__seriesPages">765-773</span><div class="ref__controls"><a href="#back-bib139" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-60849115270&amp;cf=" onclick="newWindow(this.href);return false">Scopus (3455)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19228619&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1056%2FNEJMoa0808710&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2009&pages=765-773&journal=N.%C2%A0Engl.+J.+Med.&author=Yan+H.&author=Parsons+D.W.&author=Jin+G.&author=McLendon+R.&author=Rasheed+B.A.&author=Yuan+W.&author=Kos+I.&author=Batinic-Haberle+I.&author=Jones+S.&author=Riggins+G.J.&title=IDH1+and+IDH2+mutations+in+gliomas">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Ye K. </li><li class="ref__authors__name">Schulz M.H. </li><li class="ref__authors__name">Long Q. </li><li class="ref__authors__name">Apweiler R. </li><li class="ref__authors__name">Ning Z. </li></ul><div class="ref__title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</div><span class="ref__series"><i>Bioinformatics.</i> <span class="ref__seriesDate">2009; </span><span class="ref__seriesVolume">25</span></span>: <span class="ref__seriesPages">2865-2871</span><div class="ref__controls"><a href="#back-bib140" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-70350694443&amp;cf=" onclick="newWindow(this.href);return false">Scopus (1160)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=19561018&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1093%2Fbioinformatics%2Fbtp394&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=2865-2871&journal=Bioinformatics&author=Ye+K.&author=Schulz+M.H.&author=Long+Q.&author=Apweiler+R.&author=Ning+Z.&title=Pindel%3A+a+pattern+growth+approach+to+detect+break+points+of+large+deletions+and+medium+sized+insertions+from+paired-end+short+reads">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Yoo S. </li><li class="ref__authors__name">Huang T. </li><li class="ref__authors__name">Campbell J.D. </li><li class="ref__authors__name">Lee E. </li><li class="ref__authors__name">Tu Z. </li><li class="ref__authors__name">Geraci M.W. </li><li class="ref__authors__name">Powell C.A. </li><li class="ref__authors__name">Schadt E.E. </li><li class="ref__authors__name">Spira A. </li><li class="ref__authors__name">Zhu J. </li></ul><div class="ref__title">MODMatcher: multi-omics data matcher for integrative genomic analysis.</div><span class="ref__series"><i>PLoS Comput. Biol.</i> <span class="ref__seriesDate">2014; </span><span class="ref__seriesVolume">10</span></span>: <span class="ref__seriesPages">e1003790</span><div class="ref__controls"><a href="#back-bib141" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84924380084&amp;cf=" onclick="newWindow(this.href);return false">Scopus (18)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=25122495&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1371%2Fjournal.pcbi.1003790&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=e1003790&journal=PLoS+Comput.+Biol.&author=Yoo+S.&author=Huang+T.&author=Campbell+J.D.&author=Lee+E.&author=Tu+Z.&author=Geraci+M.W.&author=Powell+C.A.&author=Schadt+E.E.&author=Spira+A.&author=Zhu+J.&title=MODMatcher%3A+multi-omics+data+matcher+for+integrative+genomic+analysis">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zecha J. </li><li class="ref__authors__name">Satpathy S. </li><li class="ref__authors__name">Kanashova T. </li><li class="ref__authors__name">Avanessian S.C. </li><li class="ref__authors__name">Kane M.H. </li><li class="ref__authors__name">Clauser K.R. </li><li class="ref__authors__name">Mertins P. </li><li class="ref__authors__name">Carr S.A. </li><li class="ref__authors__name">Kuster B. </li></ul><div class="ref__title">TMT labeling for the masses: a robust and cost-efficient, in-solution labeling approach.</div><span class="ref__series"><i>Mol. Cell. Proteomics.</i> <span class="ref__seriesDate">2019; </span><span class="ref__seriesVolume">18</span></span>: <span class="ref__seriesPages">1468-1478</span><div class="ref__controls"><a href="#back-bib142" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-85068772355&amp;cf=" onclick="newWindow(this.href);return false">Scopus (44)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=30967486&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Abstract" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=abstract&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Abstract</a></li>
<li><a id="FullText" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=fulltext&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text</a></li>
<li><a id="FullTextPDF" href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=4&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1074%2Fmcp.TIR119.001385&amp;cf=pdf&amp;site=mcpro-site" onclick="newWindow(this.href);return false">Full Text PDF</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=1468-1478&journal=Mol.+Cell.+Proteomics&author=Zecha+J.&author=Satpathy+S.&author=Kanashova+T.&author=Avanessian+S.C.&author=Kane+M.H.&author=Clauser+K.R.&author=Mertins+P.&author=Carr+S.A.&author=Kuster+B.&title=TMT+labeling+for+the+masses%3A+a+robust+and+cost-efficient%2C+in-solution+labeling+approach">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhang J. </li><li class="ref__authors__name">White N.M. </li><li class="ref__authors__name">Schmidt H.K. </li><li class="ref__authors__name">Fulton R.S. </li><li class="ref__authors__name">Tomlinson C. </li><li class="ref__authors__name">Warren W.C. </li><li class="ref__authors__name">Wilson R.K. </li><li class="ref__authors__name">Maher C.A. </li></ul><div class="ref__title">INTEGRATE: gene fusion discovery using whole genome and transcriptome data.</div><span class="ref__series"><i>Genome Res.</i> <span class="ref__seriesDate">2016; </span><span class="ref__seriesVolume">26</span></span>: <span class="ref__seriesPages">108-118</span><div class="ref__controls"><a href="#back-bib143" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="Scopus" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=2-s2.0-84956655943&amp;cf=" onclick="newWindow(this.href);return false">Scopus (53)</a></li>
<li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=26556708&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a id="Crossref" href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=16&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=10.1101%2Fgr.186114.114&amp;cf=" onclick="newWindow(this.href);return false">Crossref</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=108-118&journal=Genome+Res.&author=Zhang+J.&author=White+N.M.&author=Schmidt+H.K.&author=Fulton+R.S.&author=Tomlinson+C.&author=Warren+W.C.&author=Wilson+R.K.&author=Maher+C.A.&title=INTEGRATE%3A+gene+fusion+discovery+using+whole+genome+and+transcriptome+data">Google Scholar</a></li></ul></div></li><li class="ref"><ul title="list of authors" class="rlist inline-bullet-list ref__authors ref__inline-block"><li class="ref__authors__name">Zhou W. </li><li class="ref__authors__name">Laird P.W. </li><li class="ref__authors__name">Shen H. </li></ul><div class="ref__title">Comprehensive characterization, annotation and innovative use of Infinium DNA methylation BeadChip probes.</div><span class="ref__series"><i>Nucleic Acids Res.</i> <span class="ref__seriesDate">2017; </span><span class="ref__seriesVolume">45</span></span>: <span class="ref__seriesPages">e22</span><div class="ref__controls"><a href="#back-bib144" title="View in Article" class="ref__controls__viewInArticle scroll-into-link">View
                        in Article
                        <i class="icon-section_arrow_u"></i></a><ul class="rlist inline-bullet-list"><li><a id="PubMed" href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=8&amp;doi=10.1016/j.ccell.2021.01.006&amp;key=27924034&amp;cf=" onclick="newWindow(this.href);return false">PubMed</a></li>
<li><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=e22&journal=Nucleic+Acids+Res.&author=Zhou+W.&author=Laird+P.W.&author=Shen+H.&title=Comprehensive+characterization%2C+annotation+and+innovative+use+of+Infinium+DNA+methylation+BeadChip+probes">Google Scholar</a></li></ul></div></li></ol></section><section class="article-info"><h2 class="top" id="articleInformation">Article info</h2><section><h3 class="publicationHistory__label">Publication history</h3><div class="article-info__dates"><div class="article-info__date">Published: February 11, 2021</div><div class="article-info__date">Accepted:
            January 11,
            2021</div><div class="article-info__date">Received in revised form:
            June 2,
            2020</div><div class="article-info__date">Received:
            February 3,
            2020</div></div></section><section><h3>Identification</h3><p>DOI: <a href="https://doi.org/10.1016/j.ccell.2021.01.006">https://doi.org/10.1016/j.ccell.2021.01.006</a></p></section><section><h3>Copyright</h3><div class="copyright">© 2021 Elsevier Inc.</div></section><section class="userLicenseRpl">                                              <h3>User license</h3>                                               <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" class="userLicLnk" target="blank">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</a> |                                                      <div class="dropBlock reference-citations">                                                     <a data-db-target-for="modal-user-license" href="#" class="reference-citations__ctrl" >How you can reuse</a>                                                     <img src="/templates/jsp/_style2/_marlin/images/information.png" alt="Information Icon" class="infoIcon"/>                                                     <div data-db-target-of="modal-user-license" class="dropBlock__holder reference-citations__holder">                                                      <div class="reference-citations__border"></div>                                             		 <div class="modal-header"><span class="userLicTitle">Creative Commons Attribution – NonCommercial – NoDerivs (CC BY-NC-ND 4.0)</span></div>                                                       <div class="modal-body"><p><h2>Permitted</h2>
<h3>For non-commercial purposes:</h3>
<ul>
<li>Read, print &amp; download</li>
<li>Redistribute or republish the final article</li>

<li>Text &amp; data mine</li>
<li>Translate the article (private use only, not for distribution)</li>
<li>Reuse portions or extracts from the article in other works</li></ul>
<h2>Not Permitted</h2><ul>

<li>Sell or re-use for commercial purposes</li>



<li>Distribute translations or adaptations of the article</li></ul>
<br><a href="https://www.elsevier.com/about/open-science/open-access" target= "blank">Elsevier's open access license policy</a></p></div>                                                       </div>                                                      </div>                                              </section><section><h3>ScienceDirect</h3><a href="https://www.sciencedirect.com/user/institution/login?targetUrl=/science/article/pii/S1535610821000507">Access this article on ScienceDirect</a></section></section><div id="figure-viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous figure" class="frameButton previousFrameButton frames__prev"><i class="icon-arrow_l"></i></a><a href="#" title="Next figure" class="frameButton nextFrameButton frames__next"><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in article</span></a></div><div class="thumbstrip"><a class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="#" title="View Large Image in a new tab" class="viewLargeImage"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">View Large Image</span></a></li><li><a target="_blank" href="#" title="View high resolution image in a new tab" class="downloadThisButton" download=""><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image </span></a></li><li class="hidden-xs hidden-sm"><a href="#" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div class="figures-placeholder"></div><div class="tables-placeholder"></div>
    

        


    

        


    

        


    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

        


    

        


    

            

            

            <section><h2 id="relatedArticles" class="top">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="f03b945a-ad7e-4ca2-9d54-9b913bce3420" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S1535610821000507" class="ux3DeferredOnScrollWidget"></div></section>

            

        
<div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title"></div></nav><div class="frames"><ul class="frames__list"></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li><a target="_blank" href="/specs/products/marlin/components/figure-viewer/images/facets-2015-0010-f10-standard.jpg" title="View high resolution image in a new tab" class="downloadThisButton"><i class="icon icon-gizmo-image_figure figure-viewer__download__viewer-size-icon"></i><span class="figure-viewer__download__text">Download Hi-res image</span></a></li><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?articleNumber=e00489" target="_blank" title="Doanload .PPT" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><div data-toggle="transplant" data-direction="from" data-transplant="self" data-target=".adplaceholder--slot2, .adplaceholder--slot3 " data-remove="false" class="adplaceholder__transplant hidden-md hidden-lg"></div></div><div class="col-md-3 col-lg-2"></div></div></div></div></article>

            

        

        </main>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id='ad for nucleus-single-molecule-technologies'>
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">
URL_var = 'https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00050-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610821000507%3Fshowall%3Dtrue';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00050-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1535610821000507%3Fshowall%3Dtrue';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async defer src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped " >
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement"/>
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"/>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"/>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"/>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"/>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"/>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"/>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"/>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"/>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"/>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"/>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"/>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Life &amp; medical sciences journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		<li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<div class="group__title">Physical sciences &amp; engineering journals</div>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<div class="group__title">Multidisciplinary journals</div>
	<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<div class="group__title">Authors</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Reviewers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">News &amp; events</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
				<li>
					<a alt="Lablinks" href="/lablinks">LabLinks</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<div class="group__title">Multimedia</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<div class="group__title">About</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Contact</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<div class="group__title">Careers</div>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
				<li>
					<a alt="Scientific job board" href="https://careers.cell.com/" target="_blank">Scientific job board</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<div class="group__title">Access</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Collections</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<div class="group__title">Information</div>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr/>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr/>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie settings</button></b> for this site.<br />

All content on this site: Copyright © 2024 Elsevier Inc., its licensors, and contributors. <br/>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br/>For all open access content, the Creative Commons licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>

#brandingRow {
   background-color: #fff;
   padding: 7px 32px;
   color: #007DBC;
   font-size: 14px;
   clear: both;
   width: 100%
}

#brandingRow a {
   text-decoration: none;
   color: #007DBC;
}

#brandingRow a:hover {
   text-decoration: underline;
}

#brandingRowRight {
   display: inline;
   float: right;
   width: auto;
   padding: 0;
   margin: 0;
}

div#brandingRowInnerWrapper {
    max-width: 1440px;
    margin-left: auto;
    margin-right: auto;
    padding: 0 16px;
}

a#cpIconLnk {
   display: flex;
   margin-right: 12px;
}

@media screen and (max-width: 600px) {
    #brandingRow{
        font-size: 12px;
    }
}

@media screen and (max-width: 750px) {
    XXXsvg#cp_logo{
        margin-right: 10px;
    }
}

@media screen and (max-width: 800px) {
    a#cpIconLnk {
       display: none;
    }
}

.header__wrapper.clearfix .col-xs-6:first-of-type {
    width: 55%;
	padding-left: 32px;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
    width: 45%;
}
@media screen and (min-width: 800px) and (max-width: 1000px) {
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width: 75%;
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width: 25%;
    }
}

@media (max-width: 1139px) {
	.header__wrapper.clearfix .col-xs-6:first-of-type {
		padding-left: 16px;
	}
	
	#brandingRow {
		padding: 7px 16px 7px 16px;
	}
	
}

/* code for global header */

.header__wrapper {
   /* max-width: 1440px; */
   margin-left: auto;
   margin-right: auto;
}

@media (min-width: 769px) {
    .ux3 .quick-search .dropBlock [data-db-target-of], .quick-search .dropBlock [data-db-target-of] {
       width: 100vw !important;
       max-width: 1440px;
    }
} 

/* adjust for CP infinity icon */

svg#cp_logo {
    float: none;
    height: 32px;
    width: auto;
    margin-left: 0px;
    margin-top: 0px;
}

header .journal-logos {
    margin-right: 30px;
}

header .journal-logos:last-of-type {
    margin-right: 0px;
}

@media (min-width: 768px) {
    header .journal-logos {
       margin-left: 0 !important;
    }

/* adjust for logos with descender characters */
header .journal-logos.descender {
    margin-top: 8px;
}
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "ynexus" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");

	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
	$("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press'><svg id='cp_logo' width='324' height='204' viewBox='0 0 324 204' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M242.8 59.3C235.9 60.3 230.4 65.3 228.7 72.1L223.2 95C222.5 98.1 219.8 100.4 216.7 100.7L197.9 102.9C187.5 104 179.4 113 180.2 123.4C181 134.2 190.6 142 201.2 140.9C208.1 139.9 213.6 134.9 215.3 128.1L220.8 105.2C221.5 102.1 224.2 99.8 227.3 99.5L246.1 97.3C256.5 96.2 264.6 87.2 263.8 76.8C263 66 253.4 58.1 242.8 59.3Z' fill='#007DBC'/><path d='M221.7 0C165.5 0 120 45.5 120 101.7C120 157.9 165.5 203.4 221.7 203.4C277.9 203.4 323.4 157.9 323.4 101.7C323.4 45.5 277.8 0 221.7 0ZM251 135L221.3 138.4C216.1 139 211.8 142.8 210.6 147.9L205.6 168.8C204.1 175.1 198 179.2 191.6 178.2C167 174.5 146.5 155.6 142.8 130.1C138.2 97.9 161.1 68.6 193.2 65.1L222.9 61.7C228.1 61.1 232.4 57.3 233.6 52.2L238.6 31.3C240.1 25 246.2 20.9 252.6 21.9C277.2 25.6 297.7 44.5 301.4 70C306 102.2 283 131.5 251 135Z' fill='#007DBC'/><path d='M54.9 33.8L46.9 79L47.5 79.6C53 74.1 58.9 70.2 65.1 67.8C71.4 65.4 78.4 64.2 86.4 64.2C96.2 64.2 105 66 112.5 69.6C120.1 73.2 126.5 78 131.7 84.1C136.9 90.2 140.9 97.3 143.6 105.5C146.3 113.7 147.7 122.5 147.7 131.8C147.7 142 145.7 151.5 141.7 160.3C137.7 169.1 132.4 176.7 125.6 183.2C118.9 189.6 111.1 194.6 102.2 198.1C93.3 201.6 83.8 203.3 73.8 203.1C64.2 203.1 54.9 201.8 46.1 199.3C37.3 196.7 29.5 192.8 22.7 187.5C15.9 182.2 10.4 175.6 6.4 167.8C2.3 159.9 0.2 150.8 0 140.4H40.3C41.2 149.5 44.7 156.7 50.5 162.1C56.4 167.5 63.8 170.2 72.9 170.2C78.2 170.2 83 169.1 87.2 166.9C91.5 164.7 95.1 161.9 98 158.4C100.9 154.9 103.2 150.8 104.8 146.2C106.4 141.6 107.2 136.8 107.2 131.8C107.2 126.7 106.4 121.9 104.9 117.3C103.4 112.7 101.1 108.8 98.1 105.4C95.1 102 91.5 99.3 87.3 97.4C83.1 95.5 78.3 94.6 72.8 94.6C65.6 94.6 59.7 95.9 55.2 98.4C50.6 100.9 46.4 105 42.4 110.5H6.4L26 0.5H137V33.7H54.9V33.8Z' fill='#007DBC'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");

}

</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-9fa7ff1ab4c041fb83f6.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"1535-6108","name":"Cancer Cell"},"page":{"loadTimestamp":"1723231879102","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:375335|ae:228598","accountName":"ae:Journal Branded Solutions - Guests|ae:ScienceDirect Guests","departmentId":"ae:388406|ae:291352","ipAddress":"73.149.180.196","loginStatus":"anonymous","userId":"ae:19319093|ae:12975512"},"content":[{"accessType":"article:free:openaccess","volumeNumber":"39","issueNumber":"4","viewState":"fulltext","format":"MIME-HTML","id":"pii:S1535610821000507","coverDate":"202104","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Proteogenomic and metabolomic characterization of human glioblastoma","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
</body>
</html>
